<h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy)</h1>
<p><a href="https://doi.org/10.1038/s41598-020-79738-5">doi:10.1038/s41598-020-79738-5</a></p>

<p><a href=None>None</a></p>
<p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p><h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden</h1>
<p><a href="https://doi.org/10.7717/peerj.8424">doi:10.7717/peerj.8424</a></p>

<p><a href=None>None</a></p>
<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p><h1>Biodiversity between sand grains: Meiofauna composition across southern and western Sweden assessed by metabarcoding</h1>
<p><a href="https://doi.org/10.3897/BDJ.8.e51813">doi:10.3897/BDJ.8.e51813</a></p>

<p><a href=None>None</a></p>
<p>The meiofauna is an important part of the marine ecosystem, but its composition and distribution patterns are relatively unexplored. Here we assessed the biodiversity and community structure of meiofauna from five locations on the Swedish western and southern coasts using a high-throughput DNA sequencing (metabarcoding) approach. The mitochondrial cytochrome oxidase 1 (COI) mini-barcode and nuclear 18S small ribosomal subunit (18S) V1-V2 region were amplified and sequenced using Illumina MiSeq technology. Our analyses revealed a higher number of species than previously found in other areas: thirteen samples comprising 6.5 dm3 sediment revealed 708 COI and 1,639 18S metazoan OTUs. Across all sites, the majority of the metazoan biodiversity was assigned to <i>Arthropoda</i>, <i>Nematoda</i> and <i>Platyhelminthes</i>. Alpha and beta diversity measurements showed that community composition differed significantly amongst sites. OTUs initially assigned to <i>Acoela</i>, <i>Gastrotricha</i> and the two <i>Platyhelminthes</i> sub-groups <i>Macrostomorpha</i> and <i>Rhabdocoela</i> were further investigated and assigned to species using a phylogeny-based taxonomy approach. Our results demonstrate that there is great potential for discovery of new meiofauna species even in some of the most extensively studied locations.</p><h1>Debugging metabarcoding for insect biodiversity studies</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Insects are one of the most abundant and diverse animal groups, and they include many valuable ecological indicator species, but taxonomic discovery projects and biodiversity surveys targeting this group are often challenging. While mass trapping devices allow the collection of insects in great numbers, the task of identifying the species present is a painstaking and resource-demanding process. Metabarcoding, that is, high throughput sequencing of PCR-amplified species-specific genetic markers in environmental samples, promises to solve this problem. However, metabarcoding is still in its infancy. In this thesis, I optimized metabarcoding methods for inventorying and accelerating species discovery of terrestrial insects. In <b>paper I,</b> we designed new PCR primers for mitochondrial markers and evaluated them against existing ones using <i>in silico</i> methods. We showed that the best marker for metabarcoding of insects is 16S because of its broad taxonomic coverage and low amplification bias. However, there is significantly more reference data for COI, and its taxonomic coverage is reasonable when using sufficiently degenerate primers (mixes of primer sequences). In <b>paper II,</b> we applied 16S and COI metabarcoding to different types of samples of the same insect communities: Malaise trap samples (preservative ethanol or homogenized samples) and soil samples. The results show that the two-marker strategy increases biodiversity detection over single-marker analyses. They also show that 16S is better than COI for metabarcoding of eDNA samples because the less degenerate 16S primers do not amplify as many off-target organisms. Finally, the results show that analyses of tissue homogenate and preservative ethanol yield strikingly different results. Large and heavily sclerotized insects do not leak DNA into preservative ethanol like small and weakly sclerotized ones do, but their DNA tends to swamp the DNA of the latter in homogenized samples.  In <b>paper III</b> we evaluated the performance of various non-destructive mild lysis treatments and DNA purification methods. We subjected mock community samples to incubation in either a milder or a more aggressive digestion buffer for a short or a long period of incubation. The DNA was then extracted using either a manual or an automated purification protocol. We found that the milder digestion buffer and the shorter incubation time preserved the morphology of the insect best while at the same time giving the most accurate DNA metabarcoding results; the purification protocol had little or no effect on metabarcoding results. Finally, in <b>paper IV,</b> we explored the received wisdom that high concentrations of ethanol, although optimal for preservation of the DNA, make insects fragile and difficult to work with from a morphological point of view. We preserved insects in different ethanol concentrations and subjected them to damaging processes, such as shaking or transporting. We verified that high concentrations of ethanol induce brittleness, although the effect is less pronounced in robust insects. Our results also indicate that shipping by mail is safe for samples preserved at intermediate concentrations (70 or 80 %). In summary, this thesis represents a significant step forward in the development of methods for preserving and analyzing samples of terrestrial insects for biodiversity surveys, monitoring programs, and taxonomic research projects.</p><h1>Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea</h1>
<p><a href="https://doi.org/10.3389/fmicb.2016.00679">doi:10.3389/fmicb.2016.00679</a></p>

<p><a href=None>None</a></p>
<p>Microbial plankton form the productive base of both marine and freshwater ecosystems and are key drivers of global biogeochemical cycles of carbon and nutrients. Plankton diversity is immense with representations from all major phyla within the three domains of life. So far, plankton monitoring has mainly been based on microscopic identification, which has limited sensitivity and reproducibility, not least because of the numerical majority of plankton being unidentifiable under the light microscope. High-throughput sequencing of taxonomic marker genes offers a means to identify taxa inaccessible by traditional methods; thus, recent studies have unveiled an extensive previously unknown diversity of plankton. Here, we conducted ultra-deep Illumina sequencing (average 105 sequences/sample) of rRNA gene amplicons of surface water eukaryotic and bacterial plankton communities sampled in summer along a 2000 km transect following the salinity gradient of the Baltic Sea. Community composition was strongly correlated with salinity for both bacterial and eukaryotic plankton assemblages, highlighting the importance of salinity for structuring the biodiversity within this ecosystem. In contrast, no clear trends in alpha-diversity for bacterial or eukaryotic communities could be detected along the transect. The distribution of major planktonic taxa followed expected patterns as observed in monitoring programs, but groups novel to the Baltic Sea were also identified, such as relatives to the coccolithophore Erniliana huxleyi detected in the northern Baltic Sea. This study provides the first ultra-deep sequencing-based survey on eukaryotic and bacterial plankton biogeography in the Baltic Sea.</p><h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding</h1>
<p><a href="https://doi.org/10.3897/mycokeys.39.28109">doi:10.3897/mycokeys.39.28109</a></p>

<p><a href=None>None</a></p>
<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p><h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient</h1>
<p><a href="https://doi.org/10.1111/mec.15561">doi:10.1111/mec.15561</a></p>

<p><a href=None>None</a></p>
<p>Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates &lt; 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%–6.9%), when compared to low OC sediments (1.2%–2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode <i>Sabatieria</i> and foraminifera <i>Elphidium</i> in high OC, and <i>Cryptocaryon</i>‐like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p><h1>Lake sedimentary dna research on past terrestrial and aquatic biodiversity: Overview and recommendations</h1>
<p><a href="https://doi.org/10.3390/quat4010006">doi:10.3390/quat4010006</a></p>

<p><a href=None>None</a></p>
<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p><h1>Microbial DNA Sequencing in Environmental Studies</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>The field of microbial ecology has just entered a new era of rapid technological development and generation of big data. The high-throughput sequencing techniques presently available provide an opportunity to extensively inventorize the blueprints of life. Now, millions of microbes of natural microbial communities can be studied simultaneously without prior cultivation. New species and new functions (genes) can be discovered just by mining sequencing data. However, there is still a tremendous number of microorganisms not yet examined, nor are the ecosystem functions these carry out. The modern genomic technologies can contribute to solve environmental problems and help us understand ecosystems, but to most efficiently do so, methods need to be continuously optimised.<b> </b>During my Ph. D. studies, I developed a method to survey eukaryotic microbial diversity with a higher accuracy, and applied various sequencing-based approaches in an attempt to answer questions of importance in environmental research and ecology. In <b>PAPER-I</b>, we developed a set of 18S rRNA gene PCR primers with high taxonomic coverage, meeting the requirements of currently popular sequencing technologies and matching the richness of 18S rRNA reference sequences accumulated so far. In <b>PAPER-II</b>, we conducted the first sequencing-based spatial survey on the combined eukaryotic and bacterial planktonic community in the Baltic Sea to uncover the relationship of microbial diversity and environmental conditions. Here, the 18S primers designed in <b>PAPER-I</b> and a pair of broad-coverage 16S primers were employed to target the rRNA genes of protists and bacterioplankton for amplicon sequencing. In <b>PAPER-III</b>, we integrated metagenomic, metabarcoding, and metatranscriptomic data in an effort to scrutinise the protein synthesis potential (i.e., activity) of microbes in the sediment at a depth of 460 m in the Baltic Sea and, thus, disclosing microbial diversity and their possible ecological functions within such an extreme environment. Lastly, in <b>PAPER-IV</b>, we compared the performance of <i>E. coli</i> culturing, high-throughput sequencing, and portable real-time sequencing in tracking wastewater contamination in an urban stormwater system. From the aspects of cost, mobility and accuracy, we evaluated the usage of sequencing-based approaches in civil engineering, and for the first time, validated the real-time sequencing device in use within water quality monitoring. In summary, these studies demonstrate how DNA sequencing of microbial communities can be applied in environmental monitoring and ecological research.</p><h1>Microbial metagenomics in the Baltic Sea : Recent advancements and prospects for environmental monitoring</h1>
<p><a href="https://doi.org/10.1007/s13280-015-0663-7">doi:10.1007/s13280-015-0663-7</a></p>

<p><a href=None>None</a></p>
<p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed.</p><h1>Perspectives from Ten Years of Protist Studies by High-Throughput Metabarcoding</h1>
<p><a href="https://doi.org/10.1111/jeu.12813">doi:10.1111/jeu.12813</a></p>

<p><a href=None>None</a></p>
<p>During the last decade, high-throughput metabarcoding became routine for analyzing protistan diversity and distributions in nature. Amid a multitude of exciting findings, scientists have also identified and addressed technical and biological limitations, although problems still exist for inference of meaningful taxonomic and ecological knowledge based on shortDNAsequences. Given the extensive use of this approach, it is critical to settle our understanding on its strengths and weaknesses and to synthesize up-to-date methodological and conceptual trends. This article summarizes key scientific and technical findings, and identifies current and future directions in protist research that uses metabarcoding.</p><h1>Systematic comparison of the relative accuracy of vegetation surveys and soil DNA metabarcoding : Assessing plant biodiversity at different spatial scales</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Analysis of soil-derived DNA has been shown to minimize problems seen during traditional vegetation surveys, e.g. by matching the eDNA to a reference database for taxonomic identification rather than relying solely on taxonomic expertise. However, it has been debated to what extent and how accurately eDNA acts as a proxy for biodiversity. The reliability on eDNA and the awareness on influencing factors is also important for palaeoenvironmental reconstructions where above-ground vegetation cannot be used. This study aimed to investigate how well modern soil-derived eDNA reflects the contemporary vascular vegetation communities in a subarctic environment, and if the efficiency of the taxonomic identification differed between spatial scales. Near-surface soil samples at altitudinal gradients along numerous transects were collected in combination with vegetation surveys. The eDNA was amplified through metabarcoding using the P6 loop region of the chloroplast trnL intron, followed by a high-throughput sequencing. No difference in the number of identified taxa between eDNA and vegetation survey was seen at landscape scale. In contrast, the number of identified taxa was consistently higher in the vegetation survey at smaller spatial scales. The efficiency of identified taxa per scale remained stable for the vegetation survey, whereas for eDNA, a decreasing trend was seen. This study highlights the variations on taxa identification between both methods and which factors might cause it. Combining the methods allows for a more precise modern biodiversity estimation, as well as to minimize wrongful conclusions. This allows for a more accurate palaeoenvironmental reconstructions, which in turn can improve future species conservation decisions.</p><h1>Best practices in metabarcoding of fungi: From experimental design to results</h1>
<p><a href="https://doi.org/10.1111/Mec.16460">doi:10.1111/Mec.16460</a></p>

<p><a href=None>None</a></p>
<p>The development of high-throughput sequencing (HTS) technologies has greatly improved our capacity to identify fungi and unveil their ecological roles across a variety of ecosystems. Here we provide an overview of current best practices in metabarcoding analysis of fungal communities, from experimental design through molecular and computational analyses. By reanalysing published data sets, we demonstrate that operational taxonomic units (OTUs) outperform amplified sequence variants (ASVs) in recovering fungal diversity, a finding that is particularly evident for long markers. Additionally, analysis of the full-length ITS region allows more accurate taxonomic placement of fungi and other eukaryotes compared to the ITS2 subregion. Finally, we show that specific methods for compositional data analyses provide more reliable estimates of shifts in community structure. We conclude that metabarcoding analyses of fungi are especially promising for integrating fungi into the full microbiome and broader ecosystem functioning context, recovery of novel fungal lineages and ancient organisms as well as barcoding of old specimens including type material.</p><h1>LANDMark: an ensemble approach to the supervised selection of biomarkers in high-throughput sequencing data</h1>
<p><a href="https://doi.org/10.1186/S12859-022-04631-Z">doi:10.1186/S12859-022-04631-Z</a></p>

<p><a href=None>None</a></p>
<p>Identification of biomarkers, which are measurable characteristics of biological datasets, can be challenging. Although amplicon sequence variants (ASVs) can be considered potential biomarkers, identifying important ASVs in high-throughput sequencing datasets is challenging. Noise, algorithmic failures to account for specific distributional properties, and feature interactions can complicate the discovery of ASV biomarkers. In addition, these issues can impact the replicability of various models and elevate false-discovery rates. Contemporary machine learning approaches can be leveraged to address these issues. Ensembles of decision trees are particularly effective at classifying the types of data commonly generated in high-throughput sequencing (HTS) studies due to their robustness when the number of features in the training data is orders of magnitude larger than the number of samples. In addition, when combined with appropriate model introspection algorithms, machine learning algorithms can also be used to discover and select potential biomarkers. However, the construction of these models could introduce various biases which potentially obfuscate feature discovery.</p><h1>Disentangling bias for non-destructive insect metabarcoding</h1>
<p><a href="https://doi.org/10.7717/Peerj.12981">doi:10.7717/Peerj.12981</a></p>

<p><a href=None>None</a></p>
<p>A fast and reliable method for obtaining a species-level identification is a fundamental requirement for a wide range of activities, from plant protection and invasive species management to biodiversity assessments and ecological studies. For insects, novel molecular techniques such as DNA metabarcoding have emerged as a rapid alternative to traditional morphological identification, reducing the dependence on limited taxonomic experts. Until recently, molecular techniques have required a destructive DNA extraction, precluding the possibility of preserving voucher specimens for future studies, or species descriptions. Here we paired insect metabarcoding with two recent non-destructive DNA extraction protocols, to obtain a rapid and high-throughput taxonomic identification of diverse insect taxa while retaining a physical voucher specimen. The aim of this work was to explore how non-destructive extraction protocols impact the semi-quantitative nature of metabarcoding, which alongside species presence/absence also provides a quantitative, but biased, representation of their relative abundances. By using a series of mock communities representing each stage of a typical metabarcoding workflow we were able to determine how different morphological (<i>i.e</i>., insect biomass and exoskeleton hardness) and molecular traits (<i>i.e</i>., primer mismatch and amplicon GC%), interact with different protocol steps to introduce quantitative bias into non-destructive metabarcoding results. We discuss the relevance of taxonomic bias to metabarcoding identification of insects and potential approaches to account for it.</p><h1>Next generation restoration metrics: Using soil eDNA bacterial community data to measure trajectories towards rehabilitation targets</h1>
<p><a href="https://doi.org/S0301-4797(22)00321-8">doi:S0301-4797(22)00321-8</a></p>

<p><a href=None>None</a></p>
<p>In post-mining rehabilitation, successful mine closure planning requires specific, measurable, achievable, relevant and time-bound (SMART) completion criteria, such as returning ecological communities to match a target level of similarity to reference sites. Soil microbiota are fundamentally linked to the restoration of degraded ecosystems, helping to underpin ecological functions and plant communities. High-throughput sequencing of soil eDNA to characterise these communities offers promise to help monitor and predict ecological progress towards reference states. Here we demonstrate a novel methodology for monitoring and evaluating ecological restoration using three long-term (>25 year) case study post-mining rehabilitation soil eDNA-based bacterial community datasets. Specifically, we developed rehabilitation trajectory assessments based on similarity to reference data from restoration chronosequence datasets. Recognising that numerous alternative options for microbiota data processing have potential to influence these assessments, we comprehensively examined the influence of standard versus compositional data analyses, different ecological distance measures, sequence grouping approaches, eliminating rare taxa, and the potential for excessive spatial autocorrelation to impact on results. Our approach reduces the complexity of information that often overwhelms ecologically-relevant patterns in microbiota studies, and enables prediction of recovery time, with explicit inclusion of uncertainty in assessments. We offer a step change in the development of quantitative microbiota-based SMART metrics for measuring rehabilitation success. Our approach may also have wider applications where restorative processes facilitate the shift of microbiota towards reference states.</p><h1>Ichthyoplankton DNA metabarcoding: Challenges and perspectives</h1>
<p><a href="https://doi.org/10.1111/Mec.16387">doi:10.1111/Mec.16387</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding has been widely used to access and monitor species. However, several challenges remain open for its mainstream application in ecological studies, particularly when dealing with a quantitative approach. In a From the Cover article in this issue of Molecular Ecology, Mariac et al. (2021) report species-level ichthyoplankton dynamics for 97 fish species from two Amazon river basins using a clever quantitative metabarcoding approach employing a probe capture method. They clearly show that most species spawned during the floods, although ~20% also spawned mainly during the receding period and some other year-round, but interestingly, species from the same genus reproduced in distinct periods (i.e., inverse phenology). Opportunistically, Mariac et al. (2021) reported that during an intense hydrological anomaly, several species had a sharp reduction in spawning activity, demonstrating a quick response to environmental cues. This is an interesting result since the speed at which fish species can react to environmental changes, during the spawning period, is largely unknown. Thus, this study brings remarkable insights into basic life history information that is imperative for proposing strategies that could lead to a realistic framework for sustainable fisheries management practices and conservation, fundamental for an understudied and threatened realm, such as the Amazon River basin.</p><h1>Airborne environmental DNA for terrestrial vertebrate community monitoring</h1>
<p><a href="https://doi.org/S0960-9822(21)01690-0">doi:S0960-9822(21)01690-0</a></p>

<p><a href=None>None</a></p>
<p>Biodiversity monitoring at the community scale is a critical element of assessing and studying species distributions, ecology, diversity, and movements, and it is key to understanding and tracking environmental and anthropogenic effects on natural ecosystems.<sup>1-4</sup> Vertebrates in terrestrial ecosystems are experiencing extinctions and declines in both population numbers and sizes due to increasing threats from human activities and environmental change.<sup>5-8</sup> Terrestrial vertebrate monitoring using existing methods is generally costly and laborious, and although environmental DNA (eDNA) is becoming the tool of choice to assess biodiversity, few sample types effectively capture terrestrial vertebrate diversity. We hypothesized that eDNA captured from air could allow straightforward collection and characterization of terrestrial vertebrate communities. We filtered air at three localities in the Copenhagen Zoo: a stable, outside between the outdoor enclosures, and in the Rainforest House. Through metabarcoding of airborne eDNA, we detected 49 vertebrate species spanning 26 orders and 37 families: 30 mammal, 13 bird, 4 fish, 1 amphibian, and 1 reptile species. These spanned animals kept at the zoo, species occurring in the zoo surroundings, and species used as feed in the zoo. The detected species comprise a range of taxonomic orders and families, sizes, behaviors, and abundances. We found shorter distance to the air sampling device and higher animal biomass to increase the probability of detection. We hereby show that airborne eDNA can offer a fundamentally new way of studying and monitoring terrestrial communities.</p><h1>Zooplankton diversity monitoring strategy for the urban coastal region using metabarcoding analysis</h1>
<p><a href="https://doi.org/10.1038/S41598-021-03656-3">doi:10.1038/S41598-021-03656-3</a></p>

<p><a href=None>None</a></p>
<p>Marine ecosystems in urban coastal areas are exposed to many risks due to human activity. Thus, long-term and continuous monitoring of zooplankton diversity is necessary. High-throughput DNA metabarcoding has gained recognition as an efficient and highly sensitive approach to accurately describing the species diversity of marine zooplankton assemblages. In this study, we collected 30 zooplankton samples at about 2-week intervals for 1 year. Zooplankton diversity showing a typical four season pattern. Of the "total" and "common" zooplankton, we assigned 267 and 64 taxa. The cluster structure and seasonal diversity pattern were rough when only the "common" zooplankton was used. Our study examined how to maximize the benefits of metabarcoding for monitoring zooplankton diversity in urban coastal areas. The results suggest that to take full advantage of metabarcoding when monitoring a zooplankton community, it is necessary to carefully investigate potential ecosystem threats (non-indigenous species) through sufficient curation rather than disregarding low-abundance operational taxonomic units.</p><h1>A high-throughput amplicon sequencing approach for population-wide species diversity and composition survey</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13576">doi:10.1111/1755-0998.13576</a></p>

<p><a href=None>None</a></p>
<p>Management of agricultural pests requires an understanding of pest species diversity, their interactions with beneficial insects and spatial-temporal patterns of pest abundance. Invasive and agriculturally important insect pests can build up very high populations, especially in cropping landscapes. Traditionally, sampling effort for species identification involves small sample sizes and is labour intensive. Here, we describe a multiprimer high throughput sequencing (HTS) metabarcoding method and associated analytical workflow for a rapid, intensive, high-volume survey of pest species compositions. We demonstrate our method using the taxonomically challenging Bemisia pest cryptic species complex as examples. The whiteflies Bemisia including the"tabaci" species are agriculturally important capable of vectoring diverse plant viruses that cause diseases and crop losses. Our multiprimer metabarcoding HTS amplicon approach simultaneously process high volumes of whitefly individuals, with efficiency to detect rare (i.e., 1%) test-species, while our improved whitefly primers for metabarcoding also detected beneficial hymenopteran parasitoid species from whitefly nymphs. Field-testing our redesigned Bemisia metabarcoding primer sets across the Tanzania, Uganda and Malawi cassava cultivation landscapes, we identified the sub-Saharan Africa 1 Bemisia putative species as the dominant pest species, with other cryptic Bemisia species being detected at various abundances. We also provide evidence that Bemisia species compositions can be affected by host crops and sampling techniques that target either nymphs or adults. Our multiprimer HTS metabarcoding method incorporated two overlapping amplicons of 472 bp and 518 bp that spanned the entire 657 bp 3' barcoding region for Bemisia, and is particularly suitable to molecular diagnostic surveys of this highly cryptic insect pest species complex that also typically exhibited high population densities in heavy crop infestation episodes. Our approach can be adopted to understand species biodiversity across landscapes, with broad implications for improving transboundary biosecurity preparedness, thus contributing to molecular ecological knowledge and the development of control strategies for high-density, cryptic, pest-species complexes.</p><h1>High-Throughput Sequencing of Environmental DNA as a Tool for Monitoring Eukaryotic Communities and Potential Pathogens in a Coastal Upwelling Ecosystem</h1>
<p><a href="https://doi.org/10.3389/Fvets.2021.765606">doi:10.3389/Fvets.2021.765606</a></p>

<p><a href=None>None</a></p>
<p>The marine environment includes diverse microeukaryotic organisms that play important functional roles in the ecosystem. With molecular approaches, eukaryotic taxonomy has been improved, complementing classical analysis. In this study, DNA metabarcoding was performed to describe putative pathogenic eukaryotic microorganisms in sediment and marine water fractions collected in Galicia (NW Spain) from 2016 to 2018. The composition of eukaryotic communities was distinct between sediment and water fractions. Protists were the most diverse group, with the clade TSAR (Stramenopiles, Alveolata, Rhizaria, and Telonemida) as the primary representative organisms in the environment. Harmful algae and invasive species were frequently detected. Potential pathogens, invasive pathogenic organisms as well as the causative agents of harmful phytoplanktonic blooms were identified in this marine ecosystem. Most of the identified pathogens have a crucial impact on the aquacultural sector or affect to relevant species in the marine ecosystem, such as diatoms. Moreover, pathogens with medical and veterinary importance worldwide were also found, as well as pathogens that affect diatoms. The evaluation of the health of a marine ecosystem that directly affects the aquacultural sector with a zoonotic concern was performed with the metabarcoding assay.</p><h1>RNA metabarcoding helps reveal zooplankton community response to environmental stressors</h1>
<p><a href="https://doi.org/S0269-7491(21)02028-5">doi:S0269-7491(21)02028-5</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding can provide a high-throughput and rapid method for characterizing responses of communities to environmental stressors. However, within bulk samples, DNA metabarcoding hardly distinguishes live from the dead organisms. Here, both DNA and RNA metabarcoding were applied and compared in experimental freshwater mesocosms conducted for assessment of ecotoxicological responses of zooplankton communities to remediation treatment until 38 days post oil-spill. Furthermore, a novel indicator of normalized vitality (NV), sequence counts of RNA metabarcoding normalized by that of DNA metabarcoding, was developed for assessment of ecological responses. DNA and RNA metabarcoding detected similar taxa richness and rank of relative abundances. Both DNA and RNA metabarcoding demonstrated slight shifts in measured α-diversities in response to treatments. NV presented relatively greater magnitudes of differential responses of community compositions to treatments compared to DNA or RNA metabarcoding. NV declined from the start of the experiment (3 days pre-spill) to the end (38 days post-spill). NV also differed between Rotifer and Arthropoda, possibly due to differential life histories and sizes of organisms. NV could be a useful indicator for characterizing ecological responses to anthropogenic influence; however, the biology of target organisms and subsequent RNA production need to be considered.</p><h1>Diversity and ecology of protists revealed by metabarcoding</h1>
<p><a href="https://doi.org/S0960-9822(21)01056-3">doi:S0960-9822(21)01056-3</a></p>

<p><a href=None>None</a></p>
<p>Protists are the dominant eukaryotes in the biosphere where they play key functional roles. While protists have been studied for over a century, it is the high-throughput sequencing of molecular markers from environmental samples - the approach of metabarcoding - that has revealed just how diverse, and abundant, these small organisms are. Metabarcoding is now routine to survey environmental diversity, so data have rapidly accumulated from a multitude of environments and at different sampling scales. This mass of data has provided unprecedented opportunities to study the taxonomic and functional diversity of protists, and how this diversity is organised in space and time. Here, we use metabarcoding as a common thread to discuss the state of knowledge in protist diversity research, from technical considerations of the approach to important insights gained on diversity patterns and the processes that might have structured this diversity. In addition to these insights, we conclude that metabarcoding is on the verge of an exciting added dimension thanks to the maturation of high-throughput long-read sequencing, so that a robust eco-evolutionary framework of protist diversity is within reach.</p><h1>A Standardized Procedure for Monitoring Harmful Algal Blooms in Chile by Metabarcoding Analysis</h1>
<p><a href="https://doi.org/10.3791/62967">doi:10.3791/62967</a></p>

<p><a href=None>None</a></p>
<p>Harmful algae blooms (HABs) monitoring has been implemented worldwide, and Chile, a country famous for its fisheries and aquaculture, has intensively used microscopic and toxin analyses for decades for this purpose. Molecular biological methods, such as high-throughput DNA sequencing and bacterial assemblage-based approaches, are just beginning to be introduced in Chilean HAB monitoring, and the procedures have not yet been standardized. Here, 16S rRNA and 18S rRNA metabarcoding analyses for monitoring Chilean HABs are introduced stepwise. According to a recent hypothesis, algal-bacterial mutualistic association plays a critical synergetic or antagonistic relationship accounting for bloom initiation, maintenance, and regression. Thus, monitoring HAB from algal-bacterial perspectives may provide a broader understanding of HAB mechanisms and the basis for early warning. Metabarcoding analysis is one of the best suited molecular-based tools for this purpose because it can detect massive algal-bacterial taxonomic information in a sample. The visual procedures of sampling to metabarcoding analysis herein provide specific instructions, aiming to reduce errors and collection of reliable data.</p><h1>Coming of age for COI metabarcoding of whole organism community DNA: Towards bioinformatic harmonisation</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13502">doi:10.1111/1755-0998.13502</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of DNA extracted from community samples of whole organisms (whole organism community DNA, wocDNA) is increasingly being applied to terrestrial, marine and freshwater metazoan communities to provide rapid, accurate and high resolution data for novel molecular ecology research. The growth of this field has been accompanied by considerable development that builds on microbial metabarcoding methods to develop appropriate and efficient sampling and laboratory protocols for whole organism metazoan communities. However, considerably less attention has focused on ensuring bioinformatic methods are adapted and applied comprehensively in wocDNA metabarcoding. In this study we examined over 600 papers and identified 111 studies that performed COI metabarcoding of wocDNA. We then systematically reviewed the bioinformatic methods employed by these papers to identify the state-of-the-art. Our results show that the increasing use of wocDNA COI metabarcoding for metazoan diversity is characterised by a clear absence of bioinformatic harmonisation, and the temporal trends show little change in this situation. The reviewed literature showed (i) high heterogeneity across pipelines, tasks and tools used, (ii) limited or no adaptation of bioinformatic procedures to the nature of the COI fragment, and (iii) a worrying underreporting of tasks, software and parameters. Based upon these findings we propose a set of recommendations that we think the metabarcoding community should consider to ensure that bioinformatic methods are appropriate, comprehensive and comparable. We believe that adhering to these recommendations will improve the long-term integrative potential of wocDNA COI metabarcoding for biodiversity science.</p><h1>Seeing through sedimented waters: environmental DNA reduces the phantom diversity of sharks and rays in turbid marine habitats</h1>
<p><a href="https://doi.org/10.1186/S12862-021-01895-6">doi:10.1186/S12862-021-01895-6</a></p>

<p><a href=None>None</a></p>
<p>Sharks and rays are some of the most threatened marine taxa due to the high levels of bycatch and significant demand for meat and fin-related products in many Asian communities. At least 25% of shark and ray species are considered to be threatened with extinction. In particular, the density of reef sharks in the Pacific has declined to 3-10% of pre-human levels. Elasmobranchs are thought to be sparse in highly urbanised and turbid environments. Low visibility coupled with the highly elusive behaviour of sharks and rays pose a challenge to diversity estimation and biomonitoring efforts as sightings are limited to chance encounters or from carcasses ensnared in nets. Here we utilised an eDNA metabarcoding approach to enhance the precision of elasmobranch diversity estimates in urbanised marine environments.</p><h1>Characterizing parasitic nematode faunas in faeces and soil using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1186/S13071-021-04935-8">doi:10.1186/S13071-021-04935-8</a></p>

<p><a href=None>None</a></p>
<p>Gastrointestinal parasitic nematodes can impact fecundity, development, behaviour, and survival in wild vertebrate populations. Conventional monitoring of gastrointestinal parasitic nematodes in wild populations involves morphological identification of eggs, larvae, and adults from faeces or intestinal samples. Adult worms are typically required for species-level identification, meaning intestinal material from dead animals is needed to characterize the nematode community with high taxonomic resolution. DNA metabarcoding of environmental samples is increasingly used for time- and cost-effective, high-throughput biodiversity monitoring of small-bodied organisms, including parasite communities. Here, we evaluate the potential of DNA metabarcoding of faeces and soil samples for non-invasive monitoring of gastrointestinal parasitic nematode communities in a wild ruminant population.</p><h1>Towards reproducible metabarcoding data: Lessons from an international cross-laboratory experiment</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13485">doi:10.1111/1755-0998.13485</a></p>

<p><a href=None>None</a></p>
<p>Advances in high-throughput sequencing (HTS) are revolutionizing monitoring in marine environments by enabling rapid, accurate and holistic detection of species within complex biological samples. Research institutions worldwide increasingly employ HTS methods for biodiversity assessments. However, variance in laboratory procedures, analytical workflows and bioinformatic pipelines impede the transferability and comparability of results across research groups. An international experiment was conducted to assess the consistency of metabarcoding results derived from identical samples and primer sets using varying laboratory procedures. Homogenized biofouling samples collected from four coastal locations (Australia, Canada, New Zealand and the USA) were distributed to 12 independent laboratories. Participants were asked to follow one of two HTS library preparation workflows. While DNA extraction, primers and bioinformatic analyses were purposefully standardized to allow comparison, many other technical variables were allowed to vary among laboratories (amplification protocols, type of instrument used, etc.). Despite substantial variation observed in raw results, the primary signal in the data was consistent, with the samples grouping strongly by geographical origin for all data sets. Simple post hoc data clean-up by removing low-quality samples gave the best improvement in sample classification for nuclear 18S rRNA gene data, with an overall 92.81% correct group attribution. For mitochondrial COI gene data, the best classification result (95.58%) was achieved after correction for contamination errors. The identified critical methodological factors that introduced the greatest variability (preservation buffer, sample defrosting, template concentration, DNA polymerase, PCR enhancer) should be of great assistance in standardizing future biodiversity studies using metabarcoding.</p><h1>Environmental DNA Metabarcoding: A Novel Method for Biodiversity Monitoring of Marine Fish Communities</h1>
<p><a href="https://doi.org/10.1146/Annurev-Marine-041421-082251">doi:10.1146/Annurev-Marine-041421-082251</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is genetic material that has been shed from macroorganisms. It has received increased attention as an indirect marker for biodiversity monitoring. This article reviews the current status of eDNA metabarcoding (simultaneous detection of multiple species) as a noninvasive and cost-effective approach for monitoring marine fish communities and discusses the prospects for this growing field. eDNA metabarcoding coamplifies short fragments of fish eDNA across a wide variety of taxa and, coupled with high-throughput sequencing technologies, allows massively parallel sequencing to be performed simultaneously for dozens to hundreds of samples. It can predict species richness in a given area, detect habitat segregation and biogeographic patterns from small to large spatial scales, and monitor the spatiotemporal dynamics of fish communities. In addition, it can detect an anthropogenic impact on fish communities through evaluation of their functional diversity. Recognizing the strengths and limitations of eDNA metabarcoding will help ensure that continuous biodiversity monitoring at multiple sites will be useful for ecosystem conservation and sustainable use of fishery resources, possibly contributing to achieving the targets of the United Nations' Sustainable Development Goal 14 for 2030.</p><h1>Just keep it simple? Benchmarking the accuracy of taxonomy assignment software in metabarcoding studies</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13473">doi:10.1111/1755-0998.13473</a></p>

<p><a href=None>None</a></p>
<p>How do you put a name on an unknown piece of DNA? From microbes to mammals, high-throughput metabarcoding studies provide a more objective view of natural communities, overcoming many of the inherent limitations of traditional field surveys and microscopy-based observations (Deiner et al., 2017). Taxonomy assignment is one of the most critical aspects of any metabarcoding study, yet this important bioinformatics task is routinely overlooked. Biodiversity surveys and conservation efforts often depend on formal species inventories: the presence (or absence) of species, and the number of individuals reported across space and time. However, computational workflows applied in eukaryotic metabarcoding studies were originally developed for use with bacterial/archaeal data sets, where microbial researchers rely on one conserved locus (nuclear 16S rRNA) and have access to vast databases with good coverage across most prokaryotic lineages - a situation not mirrored in most multicellular taxa. In this issue of Molecular Ecology Resources, Hleap et al. (2021) carry out an extensive benchmarking exercise focused on taxonomy assignment strategies for eukaryotic metabarcoding studies utilizing the mitochondrial Cytochrome C oxidase I marker gene (COI). They assess the performance and accuracy of software tools representing diverse methodological approaches: from "simple" strategies based on sequence similarity and composition, to model-based phylogenetic and probabilistic classification tools. Contrary to popular assumptions, less complex approaches (BLAST and the QIIME2 feature classifier) consistently outperformed more sophisticated mathematical algorithms and were highly accurate for assigning taxonomy at higher levels (e.g. family). Lower-level assignments at the genus and species level still pose significant challenge for most existing algorithms, and sparse eukaryotic reference databases further limit software performance. This study illuminates current best practices for metabarcoding taxonomy assignments, and underscores the need for community-driven efforts to expand taxonomic and geographic representation in reference DNA barcode databases.</p><h1>High-throughput sequencing on preservative ethanol is effective at jointly examining infraspecific and taxonomic diversity, although bioinformatics pipelines do not perform equally</h1>
<p><a href="https://doi.org/10.1002/Ece3.7453">doi:10.1002/Ece3.7453</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of amplicons (HTSA) has been proposed as an effective approach to evaluate taxonomic and genetic diversity at the same time. However, there are still uncertainties as to how the results produced by different bioinformatics treatments impact the conclusions drawn on biodiversity and population genetics indices.We evaluated the ability of six bioinformatics pipelines to recover taxonomic and genetic diversity from HTSA data obtained from controlled assemblages. To that end, 20 assemblages were produced using 354 colonies of <i>Botrylloides</i> spp., sampled in the wild in ten marinas around Brittany (France). We used DNA extracted from preservative ethanol (ebDNA) after various time of storage (3, 6, and 12 months), and from a bulk of preserved specimens (bulkDNA). DNA was amplified with primers designed for targeting this ascidian genus. Results obtained from HTSA data were compared with Sanger sequencing on individual zooids (i.e., individual barcoding).Species identification and relative abundance determined with HTSA data from either ebDNA or bulkDNA were similar to those obtained with traditional individual barcoding. However, after 12 months of storage, the correlation between HTSA and individual-based data was lower than after shorter durations. The six bioinformatics pipelines were able to depict accurately the genetic diversity using standard population genetics indices (H<sub>S</sub> and F<sub>ST</sub>), despite producing false positives and missing rare haplotypes. However, they did not perform equally and dada2 was the only pipeline able to retrieve all expected haplotypes.This study showed that ebDNA is a nondestructive alternative for both species identification and haplotype recovery, providing storage does not last more than 6 months before DNA extraction. Choosing the bioinformatics pipeline is a matter of compromise, aiming to retrieve all true haplotypes while avoiding false positives. We here recommend to process HTSA data using dada2, including a chimera-removal step. Even if the possibility to use multiplexed primer sets deserves further investigation to expand the taxonomic coverage in future similar studies, we showed that primers targeting a particular genus allowed to reliably analyze this genus within a complex community.</p><h1>Potential applications of CRISPR/Cas for next-generation biomonitoring of harmful algae blooms: A review</h1>
<p><a href="https://doi.org/S1568-9883(21)00056-1">doi:S1568-9883(21)00056-1</a></p>

<p><a href=None>None</a></p>
<p>Research on harmful algal and cyanobacterial blooms (HABs and CHABs) has risen dramatically due to their increasing global distribution, frequency, and intensity. These blooms jeopardize public health, ecosystem function, sustainability and can have negative economic impacts. Numerous monitoring programs have been established using light microscopy, liquid chromatography coupled to mass spectrometry (LC-MS), ELISA, and spectrophotometry to monitor HABs/CHABs outbreaks. Recently, DNA/RNA-based molecular methods have been integrated into these programs to replace or complement traditional methods through analyzing environmental DNA and RNA (eDNA/eRNA) with techniques such as quantitative polymerase chain reaction (qPCR), fluorescent in situ hybridization (FISH), sandwich hybridization assay (SHA), isothermal amplification methods, and microarrays. These have enabled the detection of rare or cryptic species, enhanced sample throughput, and reduced costs and the need for visual taxonomic expertise. However, these methods have limitations, such as the need for high capital investment in equipment or detection uncertainties, including determining whether organisms are viable. In this review, we discuss the potential of newly developed molecular diagnosis technology based on Clustered Regularly Interspaced Short Palindromic Repeats/Cas proteins (CRISPR/Cas), which utilizes the prokaryotic adaptative immune systems of bacteria and archaea. Cas12 and Cas13-based platforms can detect both DNA and RNA with attomolar sensitivity within an hour. CRISPR/Cas diagnostic is a rapid, inexpensive, specific, and ultrasensitive technology that, with some further development, will provide many new platforms that can be used for HABs/CHABs biomonitoring and research.</p><h1>eDNAFlow, an automated, reproducible and scalable workflow for analysis of environmental DNA sequences exploiting Nextflow and Singularity</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13356">doi:10.1111/1755-0998.13356</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of environmental DNA (eDNA) when coupled with high throughput sequencing is revolutionising the way biodiversity can be monitored across a wide range of applications. However, the large number of tools deployed in downstream bioinformatic analyses often places a challenge in configuration and maintenance of a workflow, and consequently limits the research reproducibility. Furthermore, scalability needs to be considered to handle the growing amount of data due to increase in sequence output and the scale of project. Here, we describe eDNAFlow, a fully automated workflow that employs a number of state-of-the-art applications to process eDNA data from raw sequences (single-end or paired-end) to generation of curated and noncurated zero-radius operational taxonomic units (ZOTUs) and their abundance tables. This pipeline is based on Nextflow and Singularity which enable a scalable, portable and reproducible workflow using software containers on a local computer, clouds and high-performance computing (HPC) clusters. Finally, we present an in-house Python script to assign taxonomy to ZOTUs based on user specified thresholds for assigning lowest common ancestor (LCA). We demonstrate the utility and efficiency of the pipeline using an example of a published coral diversity biomonitoring study. Our results were congruent with the aforementioned study. The scalability of the pipeline is also demonstrated through analysis of a large data set containing 154 samples. To our knowledge, this is the first automated bioinformatic pipeline for eDNA analysis using two powerful tools: Nextflow and Singularity. This pipeline addresses two major challenges in the analysis of eDNA data; scalability and reproducibility.</p><h1>Connecting high-throughput biodiversity inventories: Opportunities for a site-based genomic framework for global integration and synthesis</h1>
<p><a href="https://doi.org/10.1111/Mec.15797">doi:10.1111/Mec.15797</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing (HTS) is increasingly being used for the characterization and monitoring of biodiversity. If applied in a structured way, across broad geographical scales, it offers the potential for a much deeper understanding of global biodiversity through the integration of massive quantities of molecular inventory data generated independently at local, regional and global scales. The universality, reliability and efficiency of HTS data can potentially facilitate the seamless linking of data among species assemblages from different sites, at different hierarchical levels of diversity, for any taxonomic group and regardless of prior taxonomic knowledge. However, collective international efforts are required to optimally exploit the potential of site-based HTS data for global integration and synthesis, efforts that at present are limited to the microbial domain. To contribute to the development of an analogous strategy for the nonmicrobial terrestrial domain, an international symposium entitled "Next Generation Biodiversity Monitoring" was held in November 2019 in Nicosia (Cyprus). The symposium brought together evolutionary geneticists, ecologists and biodiversity scientists involved in diverse regional and global initiatives using HTS as a core tool for biodiversity assessment. In this review, we summarize the consensus that emerged from the 3-day symposium. We converged on the opinion that an effective terrestrial Genomic Observatories network for global biodiversity integration and synthesis should be spatially led and strategically united under the umbrella of the metabarcoding approach. Subsequently, we outline an HTS-based strategy to collectively build an integrative framework for site-based biodiversity data generation.</p><h1>Monitoring of spatiotemporal occupancy patterns of fish and amphibian species in a lentic aquatic system using environmental DNA</h1>
<p><a href="https://doi.org/10.1111/Mec.15742">doi:10.1111/Mec.15742</a></p>

<p><a href=None>None</a></p>
<p>To effectively monitor, manage and protect aquatic species and understand their interactions, knowledge of their spatiotemporal distribution is needed. In this study, we used a fine-scale spatiotemporal water sampling design, followed by environmental DNA (eDNA) 12S metabarcoding, to investigate occupancy patterns of a natural community of fish and amphibian species in a lentic system. In the same system, we experimentally estimated the spatial and temporal dispersion of eDNA by placing a community of different fish and amphibian species in cages at one side of the pond, creating a controlled point of eDNA emission. Analyses of this cage community revealed a sharp spatial decline in detection rates and relative eDNA quantities at a distance of 5-10 m from the source, depending on the species and its abundance. In addition, none of the caged species could be detected 1 week after removal from the system. This indicates high eDNA decay rates and limited spatial eDNA dispersal, facilitating high local resolution for monitoring spatial occupancy patterns of aquatic species. Remarkably, for seven of the nine cage species, the presence of a single individual could be detected by pooling water of subsamples taken across the whole water body, illustrating the high sensitivity of the eDNA sampling and detection method applied. Finally, our work demonstrated that a fine-scale sampling design in combination with eDNA metabarcoding can cover total biodiversity very precisely and allows the construction of consistent spatiotemporal patterns of relative abundance and local distribution of free-living fish and amphibian species in a lentic ecosystem.</p><h1>Natrix: a Snakemake-based workflow for processing, clustering, and taxonomically assigning amplicon sequencing reads</h1>
<p><a href="https://doi.org/10.1186/S12859-020-03852-4">doi:10.1186/S12859-020-03852-4</a></p>

<p><a href=None>None</a></p>
<p>Sequencing of marker genes amplified from environmental samples, known as amplicon sequencing, allows us to resolve some of the hidden diversity and elucidate evolutionary relationships and ecological processes among complex microbial communities. The analysis of large numbers of samples at high sequencing depths generated by high throughput sequencing technologies requires efficient, flexible, and reproducible bioinformatics pipelines. Only a few existing workflows can be run in a user-friendly, scalable, and reproducible manner on different computing devices using an efficient workflow management system.</p><h1>Massively parallel sequencing is unlocking the potential of environmental trace evidence</h1>
<p><a href="https://doi.org/S1872-4973(20)30165-4">doi:S1872-4973(20)30165-4</a></p>

<p><a href=None>None</a></p>
<p>Massively parallel sequencing (MPS) has revolutionised the field of genomics enabling substantial advances in human DNA profiling. Further, the advent of MPS now allows biological signatures to be obtained from complex DNA mixtures and trace amounts of low biomass samples. Environmental samples serve as ideal forms of contact trace evidence as detection at a scene can establish a link between a suspect, location and victim. Many studies have applied MPS technology to characterise the biodiversity within high biomass environmental samples (such as soil and water) to address questions related to ecology, conservation, climate change and human health. However, translation of these tools to forensic science remains in its infancy, due in part to the merging of traditional forensic ecology practices with unfamiliar DNA technologies and complex datasets. In addition, people and objects also carry low biomass environmental signals which have recently been shown to reflect a specific individual or location. The sensitivity, and reducing cost, of MPS is now unlocking the power of both high and low biomass environmental DNA (eDNA) samples as useful sources of genetic information in forensic science. This paper discusses the potential of eDNA to forensic science by reviewing the most explored applications that are leading the integration of this technology into the field. We introduce novel areas of forensic ecology that could also benefit from these tools with a focus on linking a suspect to a scene or establishing provenance of an unknown sample and discuss the current limitations and validation recommendations to achieve translation of eDNA into casework.</p><h1>BIOCOM-PIPE: a new user-friendly metabarcoding pipeline for the characterization of microbial diversity from 16S, 18S and 23S rRNA gene amplicons</h1>
<p><a href="https://doi.org/10.1186/S12859-020-03829-3">doi:10.1186/S12859-020-03829-3</a></p>

<p><a href=None>None</a></p>
<p>The ability to compare samples or studies easily using metabarcoding so as to better interpret microbial ecology results is an upcoming challenge. A growing number of metabarcoding pipelines are available, each with its own benefits and limitations. However, very few have been developed to offer the opportunity to characterize various microbial communities (e.g., archaea, bacteria, fungi, photosynthetic microeukaryotes) with the same tool.</p><h1>Space-time dynamics in monitoring neotropical fish communities using eDNA metabarcoding</h1>
<p><a href="https://doi.org/S0048-9697(20)35625-4">doi:S0048-9697(20)35625-4</a></p>

<p><a href=None>None</a></p>
<p>The biodiverse Neotropical ecoregion remains insufficiently assessed, poorly managed, and threatened by unregulated human activities. Novel, rapid and cost-effective DNA-based approaches are valuable to improve understanding of the biological communities and for biomonitoring in remote areas. Here, we evaluate the potential of environmental DNA (eDNA) metabarcoding for assessing the structure and distribution of fish communities by analysing water and sediment from 11 locations along the Jequitinhonha River catchment (Brazil). Each site was sampled twice, before and after a major rain event in a five-week period and fish diversity was estimated using high-throughput sequencing of 12S rRNA amplicons. In total, 252 Molecular Operational Taxonomic Units (MOTUs) and 34 fish species were recovered, including endemic, introduced, and previously unrecorded species for this basin. Spatio-temporal variation of eDNA from fish assemblages was observed and species richness was nearly twice as high before the major rain event compared to afterwards. Yet, peaks of diversity were primarily associated with only four of the locations. No correlation between β-diversity and longitudinal distance or presence of dams was detected, but low species richness observed at sites located near dams might that these anthropogenic barriers may have an impact on local fish diversity. Unexpectedly high α-diversity levels recorded at the river mouth suggest that these sections should be further evaluated as putative "eDNA reservoirs" for rapid monitoring. By uncovering spatio-temporal changes, unrecorded biodiversity components, and putative anthropogenic impacts on fish assemblages, we further strengthen the potential of eDNA metabarcoding as a biomonitoring tool, especially in regions often neglected or difficult to access.</p><h1>Advancing biodiversity assessments with environmental DNA: Long-read technologies help reveal the drivers of Amazonian fungal diversity</h1>
<p><a href="https://doi.org/10.1002/Ece3.6477">doi:10.1002/Ece3.6477</a></p>

<p><a href=None>None</a></p>
<p>Fungi are a key component of tropical biodiversity. However, due to their inconspicuous and largely subterranean nature, they are usually neglected in biodiversity inventories. The goal of this study was to identify the key determinants of fungal richness, community composition, and turnover in tropical rainforests. We tested specifically for the effect of soil properties, habitat, and locality in Amazonia. For these analyses, we used high-throughput sequencing data of short and long reads of fungal DNA present in soil and organic litter samples, combining existing and novel genomic data. Habitat type (phytophysiognomy) emerges as the strongest factor explaining fungal community composition. Naturally open areas-campinas-are the richest habitat overall. Soil properties have different effects depending on the soil layer (litter or mineral soil) and the choice of genetic marker. We suggest that campinas could be a neglected hotspot of fungal diversity. An underlying cause for their rich diversity may be the overall low soil fertility, which increases the reliance on biotic interactions essential for nutrient absorption in these environments, notably ectomycorrhizal fungi-plant associations. Our results highlight the advantages of using both short and long DNA reads produced through high-throughput sequencing to characterize fungal diversity. While short reads can suffice for diversity and community comparison, long reads add taxonomic precision and have the potential to reveal population diversity.</p><h1>Executing multi-taxa eDNA ecological assessment via traditional metrics and interactive networks</h1>
<p><a href="https://doi.org/S0048-9697(20)32318-4">doi:S0048-9697(20)32318-4</a></p>

<p><a href=None>None</a></p>
<p>Current approaches to ecological assessment are limited by the traditional morpho-taxonomic methods presently employed and the inability to meet increasing demands for rapid assessments. Advancements in high throughput sequencing now enable rapid high-resolution ecological assessment using environmental DNA (eDNA). Here we test the ability of using eDNA-based ecological assessment methods against traditional assessment of two key indicator groups (diatoms and macroinvertebrates) and show how eDNA across multiple gene regions (COI, rbcL, 12S and 18S) can be used to infer interactive networks that link to ecological assessment criteria. We compared results between taxonomic and eDNA based assessments and found significant positive associations between macroinvertebrate (p < 0.001 R<sup>2</sup> = 0.645) and diatom (p = 0.015, R<sup>2</sup> = 0.222) assessment metrics. We further assessed the ability of eDNA based assessment to identify environmentally sensitive genera and found an order of magnitude greater potential for 18S, versus COI or rbcL, to determine environmental filtering of ecologically assessed communities. Lastly, we compared the ability of traditional metrics against co-occurrence network properties of our combined 18S, COI and rbcL indicator genera to infer habitat quality measures currently used by managers. We found that transitivity (network connectivity), linkage density and cohesion were significantly associated with habitat modification scores (HMS), whereas network properties were inconsistent with linking to the habitat quality score (HQS) metric. The incorporation of multi-marker eDNA network assessment opens up a means for finer scale ecological assessment, currently limited using traditional methods. While utilization of eDNA-based assessment is recommended, direct comparisons with traditional approaches are difficult as the methods are intrinsically different and should be treated as such with regards to future research. Overall, our findings show that eDNA can be used for effective ecological assessment while offering a wider range of scope and application compared to traditional assessment methods.</p><h1>DNA Metabarcoding as a Tool for Disentangling Food Webs in Agroecosystems</h1>
<p><a href="https://doi.org/E294">doi:E294</a></p>

<p><a href=None>None</a></p>
<p>Better knowledge of food webs and related ecological processes is fundamental to understanding the functional role of biodiversity in ecosystems. This is particularly true for pest regulation by natural enemies in agroecosystems. However, it is generally difficult to decipher the impact of predators, as they often leave no direct evidence of their activity. Metabarcoding via high-throughput sequencing (HTS) offers new opportunities for unraveling trophic linkages between generalist predators and their prey, and ultimately identifying key ecological drivers of natural pest regulation. Here, this approach proved effective in deciphering the diet composition of key predatory arthropods (nine species.; 27 prey taxa), insectivorous birds (one species, 13 prey taxa) and bats (one species; 103 prey taxa) sampled in a millet-based agroecosystem in Senegal. Such information makes it possible to identify the diet breadth and preferences of predators (e.g., mainly moths for bats), to design a qualitative trophic network, and to identify patterns of intraguild predation across arthropod predators, insectivorous vertebrates and parasitoids. Appropriateness and limitations of the proposed molecular-based approach for assessing the diet of crop pest predators and trophic linkages are discussed.</p><h1>Environmental DNA analysis shows high potential as a tool for estimating intraspecific genetic diversity in a wild fish population</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13165">doi:10.1111/1755-0998.13165</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis has recently been used as a new tool for estimating intraspecific diversity. However, whether known haplotypes contained in a sample can be detected correctly using eDNA-based methods has been examined only by an aquarium experiment. Here, we tested whether the haplotypes of Ayu fish (Plecoglossus altivelis altivelis) detected in a capture survey could also be detected from an eDNA sample derived from the field that contained various haplotypes with low concentrations and foreign substances. A water sample and Ayu specimens collected from a river on the same day were analysed by eDNA analysis and Sanger sequencing, respectively. The 10 L water sample was divided into 20 filters for each of which 15 PCR replications were performed. After high-throughput sequencing, denoising was performed using two of the most widely used denoising packages, unoise3 and dada2. Of the 42 haplotypes obtained from the Sanger sequencing of 96 specimens, 38 (unoise3) and 41 (dada2) haplotypes were detected by eDNA analysis. When dada2 was used, except for one haplotype, haplotypes owned by at least two specimens were detected from all the filter replications. Accordingly, although it is important to note that eDNA-based method has some limitations and some risk of false positive and false negative, this study showed that the eDNA analysis for evaluating intraspecific genetic diversity provides comparable results for large-scale capture-based conventional methods. Our results suggest that eDNA-based methods could become a more efficient survey method for investigating intraspecific genetic diversity in the field.</p><h1>A taxonomy-free approach based on machine learning to assess the quality of rivers with diatoms</h1>
<p><a href="https://doi.org/S0048-9697(20)31413-3">doi:S0048-9697(20)31413-3</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are a compulsory biological quality element in the ecological assessment of rivers according to the Water Framework Directive. The application of current official indices requires the identification of individuals to species or lower rank under a microscope based on the valve morphology. This is a highly time-consuming task, often susceptible of disagreements among analysts. In alternative, the use of DNA metabarcoding combined with High-Throughput Sequencing (HTS) has been proposed. The sequences obtained from environmental DNA are clustered into Operational Taxonomic Units (OTUs), which can be assigned to a taxon using reference databases, and from there calculate biotic indices. However, there is still a high percentage of unassigned OTUs to species due to the incompleteness of reference libraries. Alternatively, we tested a new taxonomy-free approach based on diatom community samples to assess rivers. A combination of three machine learning techniques is used to build models that predict diatom OTUs expected in test sites, under reference conditions, from environmental data. The Observed/Expected OTUs ratio indicates the deviation from reference condition and is converted into a quality class. This approach was never used with diatoms neither with OTUs data. To evaluate its efficiency, we built a model based on OTUs lists (HYDGEN) and another based on taxa lists from morphological identification (HYDMORPH), and also calculated a biotic index (IPS). The models were trained and tested with data from 81 sites (44 reference sites) from central Portugal. Both models were considered accurate (linear regression for Observed and Expected richness: R<sup>2</sup> ≈ 0.7, interception ≈ 0.8) and sensitive to global anthropogenic disturbance (Rs<sup>2</sup> > 0.30 p < 0.006 for global disturbance). Yet, the HYDGEN model based on molecular data was sensitive to more types of pressures (such as, changes in land use and habitat quality), which gives promising insights to its use for bioassessment of rivers.</p><h1>A Review and Perspective of eDNA Application to Eutrophication and HAB Control in Freshwater and Marine Ecosystems</h1>
<p><a href="https://doi.org/E417">doi:E417</a></p>

<p><a href=None>None</a></p>
<p>Changing ecological communities in response to anthropogenic activities and climate change has become a worldwide problem. The eutrophication of waterbodies in freshwater and seawater caused by the effects of human activities and nutrient inputs could result in harmful algae blooms (HABs), decreases water quality, reductions in biodiversity and threats to human health. Rapid and accurate monitoring and assessment of aquatic ecosystems are imperative. Environmental DNA (eDNA) analysis using high-throughput sequencing has been demonstrated to be an effective and sensitive assay for detecting and monitoring single or multiple species in different samples. In this study, we review the potential applications of eDNA approaches in controlling and mitigating eutrophication and HABs in freshwater and marine ecosystems. We use recent studies to highlight how eDNA methods have been shown to be a useful tool for providing comprehensive data in studies of eutrophic freshwater and marine environments. We also provide perspectives on using eDNA techniques to reveal molecular mechanisms in biological processes and mitigate eutrophication and HABs in aquatic ecosystems. Finally, we discuss the feasible applications of eDNA for monitoring biodiversity, surveying species communities and providing instructions for the conservation and management of the environment by integration with traditional methods and other advanced techniques.</p><h1>Standards for Methods Utilizing Environmental DNA for Detection of Fish Species</h1>
<p><a href="https://doi.org/E296">doi:E296</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) techniques are gaining attention as cost-effective, non-invasive strategies for acquiring information on fish and other aquatic organisms from water samples. Currently, eDNA approaches are used to detect specific fish species and determine fish community diversity. Various protocols used with eDNA methods for aquatic organism detection have been reported in different eDNA studies, but there are no general recommendations for fish detection. Herein, we reviewed 168 papers to supplement and highlight the key criteria for each step of eDNA technology in fish detection and provide general suggestions for eliminating detection errors. Although there is no unified recommendation for the application of diverse eDNA in detecting fish species, in most cases, 1 or 2 L surface water collection and eDNA capture on 0.7-μm glass fiber filters followed by extraction with a DNeasy Blood and Tissue Kit or PowerWater DNA Isolation Kit are useful for obtaining high-quality eDNA. Subsequently, species-specific quantitative polymerase chain reaction (qPCR) assays based on mitochondrial cytochrome b gene markers or eDNA metabarcoding based on both <i>12S</i> and <i>16S rRNA</i> markers via high-throughput sequencing can effectively detect target DNA or estimate species richness. Furthermore, detection errors can be minimized by mitigating contamination, negative control, PCR replication, and using multiple genetic markers. Our aim is to provide a useful strategy for fish eDNA technology that can be applied by researchers, advisors, and managers.</p><h1>High-throughput sequencing for community analysis: the promise of DNA barcoding to uncover diversity, relatedness, abundances and interactions in spider communities</h1>
<p><a href="https://doi.org/10.1007/S00427-020-00652-X">doi:10.1007/S00427-020-00652-X</a></p>

<p><a href=None>None</a></p>
<p>Large-scale studies on community ecology are highly desirable but often difficult to accomplish due to the considerable investment of time, labor and, money required to characterize richness, abundance, relatedness, and interactions. Nonetheless, such large-scale perspectives are necessary for understanding the composition, dynamics, and resilience of biological communities. Small invertebrates play a central role in ecosystems, occupying critical positions in the food web and performing a broad variety of ecological functions. However, it has been particularly difficult to adequately characterize communities of these animals because of their exceptionally high diversity and abundance. Spiders in particular fulfill key roles as both predator and prey in terrestrial food webs and are hence an important focus of ecological studies. In recent years, large-scale community analyses have benefitted tremendously from advances in DNA barcoding technology. High-throughput sequencing (HTS), particularly DNA metabarcoding, enables community-wide analyses of diversity and interactions at unprecedented scales and at a fraction of the cost that was previously possible. Here, we review the current state of the application of these technologies to the analysis of spider communities. We discuss amplicon-based DNA barcoding and metabarcoding for the analysis of community diversity and molecular gut content analysis for assessing predator-prey relationships. We also highlight applications of the third generation sequencing technology for long read and portable DNA barcoding. We then address the development of theoretical frameworks for community-level studies, and finally highlight critical gaps and future directions for DNA analysis of spider communities.</p><h1>Genetic Biomonitoring and Biodiversity Assessment Using Portable Sequencing Technologies: Current Uses and Future Directions</h1>
<p><a href="https://doi.org/E858">doi:E858</a></p>

<p><a href=None>None</a></p>
<p>We live in an era of unprecedented biodiversity loss, affecting the taxonomic composition of ecosystems worldwide. The immense task of quantifying human imprints on global ecosystems has been greatly simplified by developments in high-throughput DNA sequencing technology (HTS). Approaches like DNA metabarcoding enable the study of biological communities at unparalleled detail. However, current protocols for HTS-based biodiversity exploration have several drawbacks. They are usually based on short sequences, with limited taxonomic and phylogenetic information content. Access to expensive HTS technology is often restricted in developing countries. Ecosystems of particular conservation priority are often remote and hard to access, requiring extensive time from field collection to laboratory processing of specimens. The advent of inexpensive mobile laboratory and DNA sequencing technologies show great promise to facilitate monitoring projects in biodiversity hot-spots around the world. Recent attention has been given to portable DNA sequencing studies related to infectious organisms, such as bacteria and viruses, yet relatively few studies have focused on applying these tools to Eukaryotes, such as plants and animals. Here, we outline the current state of genetic biodiversity monitoring of higher Eukaryotes using Oxford Nanopore Technology's MinION portable sequencing platform, as well as summarize areas of recent development.</p><h1>GenBank is a reliable resource for 21st century biodiversity research</h1>
<p><a href="https://doi.org/10.1073/Pnas.1911714116">doi:10.1073/Pnas.1911714116</a></p>

<p><a href=None>None</a></p>
<p>Traditional methods of characterizing biodiversity are increasingly being supplemented and replaced by approaches based on DNA sequencing alone. These approaches commonly involve extraction and high-throughput sequencing of bulk samples from biologically complex communities or samples of environmental DNA (eDNA). In such cases, vouchers for individual organisms are rarely obtained, often unidentifiable, or unavailable. Thus, identifying these sequences typically relies on comparisons with sequences from genetic databases, particularly GenBank. While concerns have been raised about biases and inaccuracies in laboratory and analytical methods, comparatively little attention has been paid to the taxonomic reliability of GenBank itself. Here we analyze the metazoan mitochondrial sequences of GenBank using a combination of distance-based clustering and phylogenetic analysis. Because of their comparatively rapid evolutionary rates and consequent high taxonomic resolution, mitochondrial sequences represent an invaluable resource for the detection of the many small and often undescribed organisms that represent the bulk of animal diversity. We show that metazoan identifications in GenBank are surprisingly accurate, even at low taxonomic levels (likely <1% error rate at the genus level). This stands in contrast to previously voiced concerns based on limited analyses of particular groups and the fact that individual researchers currently submit annotated sequences to GenBank without significant external taxonomic validation. Our encouraging results suggest that the rapid uptake of DNA-based approaches is supported by a bioinformatic infrastructure capable of assessing both the losses to biodiversity caused by global change and the effectiveness of conservation efforts aimed at slowing or reversing these losses.</p><h1>DNA metabarcoding adds valuable information for management of biodiversity in roadside stormwater ponds</h1>
<p><a href="https://doi.org/10.1002/Ece3.5503">doi:10.1002/Ece3.5503</a></p>

<p><a href=None>None</a></p>
<p>Stormwater ponds are used to compensate for the adverse effects that road runoff might have on the natural environment. Depending on their design and placement, stormwater ponds can act as both refugia and traps for local biodiversity. To evaluate the impact of stormwater ponds on biodiversity, it is critical to use effective and precise methods for identification of life associated with the water body. DNA metabarcoding has recently become a promising tool for identification and assessment of freshwater biodiversity.Using both morphology and DNA metabarcoding, we analyze species richness and biological composition of samples from 12 stormwater ponds and investigate the impact of pond size and pollution levels in the sediments and water column on the macroinvertebrate community structure.DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. The (dis)similarity of macroinvertebrate community composition in different ponds showed that the ponds appear better separated in the results obtained by DNA metabarcoding, but that the explained variation is higher for the results obtained by morphologically identification, since it provides abundance data.The reliance on morphological methods has limited our perception of the aquatic biodiversity in response to anthropogenic stressors, thereby providing inaccurate information for appropriate design and management of stormwater ponds; these drawbacks can be overcome by DNA metabarcoding. <i>Synthesis and applications</i>. The results indicate that DNA metabarcoding is a useful tool in identifying species, especially Diptera, which are difficult to determine. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing a more accurate information regarding the way the ponds function and how they are affected by management.</p><h1>Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance</h1>
<p><a href="https://doi.org/Giz092">doi:Giz092</a></p>

<p><a href=None>None</a></p>
<p>Trap-based surveillance strategies are widely used for monitoring of invasive insect species, aiming to detect newly arrived exotic taxa as well as track the population levels of established or endemic pests. Where these surveillance traps have low specificity and capture non-target endemic species in excess of the target pests, the need for extensive specimen sorting and identification creates a major diagnostic bottleneck. While the recent development of standardized molecular diagnostics has partly alleviated this requirement, the single specimen per reaction nature of these methods does not readily scale to the sheer number of insects trapped in surveillance programmes. Consequently, target lists are often restricted to a few high-priority pests, allowing unanticipated species to avoid detection and potentially establish populations. DNA metabarcoding has recently emerged as a method for conducting simultaneous, multi-species identification of complex mixed communities and may lend itself ideally to rapid diagnostics of bulk insect trap samples. Moreover, the high-throughput nature of recent sequencing platforms could enable the multiplexing of hundreds of diverse trap samples on a single flow cell, thereby providing the means to dramatically scale up insect surveillance in terms of both the quantity of traps that can be processed concurrently and number of pest species that can be targeted. In this review of the metabarcoding literature, we explore how DNA metabarcoding could be tailored to the detection of invasive insects in a surveillance context and highlight the unique technical and regulatory challenges that must be considered when implementing high-throughput sequencing technologies into sensitive diagnostic applications.</p><h1>High-throughput environmental DNA analysis informs a biological assessment of an urban stream</h1>
<p><a href="https://doi.org/10.1016/J.Ecolind.2019.04.088">doi:10.1016/J.Ecolind.2019.04.088</a></p>

<p><a href=None>None</a></p>
<p>There is growing interest in the use of DNA barcoding and metabarcoding approaches to aid biological assessments and monitoring of waterbodies. While biodiversity measured by morphology and by DNA often has been found correlated, few studies have compared DNA data to established measures of impairment such as multimetric pollution tolerance indices used by many bioassessment programs. We incorporated environmental DNA (eDNA) metabarcoding of seston into a rigorous watershed-scale biological assessment of an urban stream to examine the extent to which eDNA richness and diversity patterns were correlated with multimetric indices and ecological impairment status designations. We also evaluated different filtering approaches and taxonomic classifications to identify best practices for environmental assessments. Seston eDNA revealed a wide diversity of eukaryotic taxa but was dominated by diatoms (36%). Differentiation among sites in alpha and beta diversity was greater when operational taxonomic units (OTUs) were classified taxonomically, but coarse resolution taxonomy (kingdom) was more informative than finer resolution taxonomy (family, genus). Correlations of DNA richness and diversity with multimetric indices for fish and macroinvertebrates were generally weak, possibly because Metazoa were not highly represented in our DNA dataset. Nonetheless, sites could be differentiated based on ecological impairment status, with more impaired sites having lower eDNA diversity as measured by the Shannon index, but higher taxonomic richness. Significant environmental drivers of community structure, as inferred from constrained ordination analyses, differed among kingdoms within the eDNA dataset, as well as from fish and macrobenthos, suggesting that eDNA provides novel environmental information. These results suggest that even a simple seston eDNA filtering protocol can provide biodiversity information of value to stream bioassessment programs. The approach bears further investigation as a potentially useful rapid assessment protocol to supplement more intensive field sampling efforts.</p><h1>Evaluating next-generation sequencing (NGS) methods for routine monitoring of wild bees: Metabarcoding, mitogenomics or NGS barcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13013">doi:10.1111/1755-0998.13013</a></p>

<p><a href=None>None</a></p>
<p>Implementing cost-effective monitoring programs for wild bees remains challenging due to the high costs of sampling and specimen identification. To reduce costs, next-generation sequencing (NGS)-based methods have lately been suggested as alternatives to morphology-based identifications. To provide a comprehensive presentation of the advantages and weaknesses of different NGS-based identification methods, we assessed three of the most promising ones, namely metabarcoding, mitogenomics and NGS barcoding. Using a regular monitoring data set (723 specimens identified using morphology), we found that NGS barcoding performed best for both species presence/absence and abundance data, producing only few false positives (3.4%) and no false negatives. In contrast, the proportion of false positives and false negatives was higher using metabarcoding and mitogenomics. Although strong correlations were found between biomass and read numbers, abundance estimates significantly skewed the communities' composition in these two techniques. NGS barcoding recovered the same ecological patterns as morphology. Ecological conclusions based on metabarcoding and mitogenomics were similar to those based on morphology when using presence/absence data, but different when using abundance data. In terms of workload and cost, we show that metabarcoding and NGS barcoding can compete with morphology, but not mitogenomics which was consistently more expensive. Based on these results, we advocate that NGS barcoding is currently the seemliest NGS method for monitoring of wild bees. Furthermore, this method has the advantage of potentially linking DNA sequences with preserved voucher specimens, which enable morphological re-examination and will thus produce verifiable records which can be fed into faunistic databases.</p><h1>SLIM: a flexible web application for the reproducible processing of environmental DNA metabarcoding data</h1>
<p><a href="https://doi.org/10.1186/S12859-019-2663-2">doi:10.1186/S12859-019-2663-2</a></p>

<p><a href=None>None</a></p>
<p>High-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) has become a routine tool for biodiversity survey and ecological studies. By including sample-specific tags in the primers prior PCR amplification, it is possible to multiplex hundreds of samples in a single sequencing run. The analysis of millions of sequences spread into hundreds to thousands of samples prompts for efficient, automated yet flexible analysis pipelines. Various algorithms and software have been developed to perform one or multiple processing steps, such as paired-end reads assembly, chimera filtering, Operational Taxonomic Unit (OTU) clustering and taxonomic assignment. Some of these software are now well established and widely used by scientists as part of their workflow. Wrappers that are capable to process metabarcoding data from raw sequencing data to annotated OTU-to-sample matrix were also developed to facilitate the analysis for non-specialist users. Yet, most of them require basic bioinformatic or command-line knowledge, which can limit the accessibility to such integrative toolkits. Furthermore, for flexibility reasons, these tools have adopted a step-by-step approach, which can prevent an easy automation of the workflow, and hence hamper the analysis reproducibility.</p><h1>Supervised machine learning outperforms taxonomy-based environmental DNA metabarcoding applied to biomonitoring</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12926">doi:10.1111/1755-0998.12926</a></p>

<p><a href=None>None</a></p>
<p>Biodiversity monitoring is the standard for environmental impact assessment of anthropogenic activities. Several recent studies showed that high-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) could overcome many limitations of the traditional morphotaxonomy-based bioassessment. Recently, we demonstrated that supervised machine learning (SML) can be used to predict accurate biotic indices values from eDNA metabarcoding data, regardless of the taxonomic affiliation of the sequences. However, it is unknown to which extent the accuracy of such models depends on taxonomic resolution of molecular markers or how SML compares with metabarcoding approaches targeting well-established bioindicator species. In this study, we address these issues by training predictive models upon five different ribosomal bacterial and eukaryotic markers and measuring their performance to assess the environmental impact of marine aquaculture on independent data sets. Our results show that all tested markers are yielding accurate predictive models and that they all outperform the assessment relying solely on taxonomically assigned sequences. Remarkably, we did not find any significant difference in the performance of the models built using universal eukaryotic or prokaryotic markers. Using any molecular marker with a taxonomic range broad enough to comprise different potential bioindicator taxa, SML approach can overcome the limits of taxonomy-based eDNA bioassessment.</p><h1>Omics Advances in Ecotoxicology</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.7B06494">doi:10.1021/Acs.Est.7B06494</a></p>

<p><a href=None>None</a></p>
<p>Toxic substances in the environment generate adverse effects at all levels of biological organization from the molecular level to community and ecosystem. Given this complexity, it is not surprising that ecotoxicologists have struggled to address the full consequences of toxic substance release at ecosystem level, due to the limits of observational and experimental tools to reveal the changes in deep structure at different levels of organization. -Omics technologies, consisting of genomics and ecogenomics, have the power to reveal, in unprecedented detail, the cellular processes of an individual or biodiversity of a community in response to environmental change with high sample/observation throughput. This represents a historic opportunity to transform the way we study toxic substances in ecosystems, through direct linkage of ecological effects with the systems biology of organisms. Three recent examples of -omics advance in the assessment of toxic substances are explored here: (1) the use of functional genomics in the discovery of novel molecular mechanisms of toxicity of chemicals in the environment; (2) the development of laboratory pipelines of dose-dependent, reduced transcriptomics to support high-throughput chemical testing at the biological pathway level; and (3) the use of eDNA metabarcoding approaches for assessing chemical effects on biological communities in mesocosm experiments and through direct observation in field monitoring. -Omics advances in ecotoxicological studies not only generate new knowledge regarding mechanisms of toxicity and environmental effect, improving the relevance and immediacy of laboratory toxicological assessment, but can provide a wholly new paradigm for ecotoxicology by linking ecological models to mechanism-based, systems biology approaches.</p><h1>Scaling up: A guide to high-throughput genomic approaches for biodiversity analysis</h1>
<p><a href="https://doi.org/10.1111/Mec.14478">doi:10.1111/Mec.14478</a></p>

<p><a href=None>None</a></p>
<p>The purpose of this review is to present the most common and emerging DNA-based methods used to generate data for biodiversity and biomonitoring studies. As environmental assessment and monitoring programmes may require biodiversity information at multiple levels, we pay particular attention to the DNA metabarcoding method and discuss a number of bioinformatic tools and considerations for producing DNA-based indicators using operational taxonomic units (OTUs), taxa at a variety of ranks and community composition. By developing the capacity to harness the advantages provided by the newest technologies, investigators can "scale up" by increasing the number of samples and replicates processed, the frequency of sampling over time and space, and even the depth of sampling such as by sequencing more reads per sample or more markers per sample. The ability to scale up is made possible by the reduced hands-on time and cost per sample provided by the newest kits, platforms and software tools. Results gleaned from broad-scale monitoring will provide opportunities to address key scientific questions linked to biodiversity and its dynamics across time and space as well as being more relevant for policymakers, enabling science-based decision-making, and provide a greater socio-economic impact. As genomic approaches are continually evolving, we provide this guide to methods used in biodiversity genomics.</p><h1>Genetic monitoring of open ocean biodiversity: An evaluation of DNA metabarcoding for processing continuous plankton recorder samples</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12740">doi:10.1111/1755-0998.12740</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is an efficient method for measuring biodiversity, but the process of initiating long-term DNA-based monitoring programmes, or integrating with conventional programs, is only starting. In marine ecosystems, plankton surveys using the continuous plankton recorder (CPR) have characterized biodiversity along transects covering millions of kilometres with time-series spanning decades. We investigated the potential for use of metabarcoding in CPR surveys. Samples (n = 53) were collected in two Southern Ocean transects and metazoans identified using standard microscopic methods and by high-throughput sequencing of a cytochrome c oxidase subunit I marker. DNA increased the number of metazoan species identified and provided high-resolution taxonomy of groups problematic in conventional surveys (e.g., larval echinoderms and hydrozoans). Metabarcoding also generally produced more detections than microscopy, but this sensitivity may make cross-contamination during sampling a problem. In some samples, the prevalence of DNA from large plankton such as krill masked the presence of smaller species. We investigated adding a fixed amount of exogenous DNA to samples as an internal control to allow determination of relative plankton biomass. Overall, the metabarcoding data represent a substantial shift in perspective, making direct integration into current long-term time-series challenging. We discuss a number of hurdles that exist for progressing DNA metabarcoding from the current snapshot studies to the requirements of a long-term monitoring programme. Given the power and continually increasing efficiency of metabarcoding, it is almost certain this approach will play an important role in future plankton monitoring.</p><h1>FuzzyID2: A software package for large data set species identification via barcoding and metabarcoding using hidden Markov models and fuzzy set methods</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12738">doi:10.1111/1755-0998.12738</a></p>

<p><a href=None>None</a></p>
<p>Species identification through DNA barcoding or metabarcoding has become a key approach for biodiversity evaluation and ecological studies. However, the rapid accumulation of barcoding data has created some difficulties: for instance, global enquiries to a large reference library can take a very long time. We here devise a two-step searching strategy to speed identification procedures of such queries. This firstly uses a Hidden Markov Model (HMM) algorithm to narrow the searching scope to genus level and then determines the corresponding species using minimum genetic distance. Moreover, using a fuzzy membership function, our approach also estimates the credibility of assignment results for each query. To perform this task, we developed a new software pipeline, FuzzyID2, using Python and C++. Performance of the new method was assessed using eight empirical data sets ranging from 70 to 234,535 barcodes. Five data sets (four animal, one plant) deployed the conventional barcode approach, one used metabarcodes, and two were eDNA-based. The results showed mean accuracies of generic and species identification of 98.60% (with a minimum of 95.00% and a maximum of 100.00%) and 94.17% (with a range of 84.40%-100.00%), respectively. Tests with simulated NGS sequences based on realistic eDNA and metabarcode data demonstrated that FuzzyID2 achieved a significantly higher identification success rate than the commonly used Blast method, and the TIPP method tends to find many fewer species than either FuzztID2 or Blast. Furthermore, data sets with tens of thousands of barcodes need only a few seconds for each query assignment using FuzzyID2. Our approach provides an efficient and accurate species identification protocol for biodiversity-related projects with large DNA sequence data sets.</p><h1>International Barcode of Life: Focus on big biodiversity in South Africa</h1>
<p><a href="https://doi.org/10.1139/Gen-2017-0210">doi:10.1139/Gen-2017-0210</a></p>

<p><a href=None>None</a></p>
<p>Participants in the 7th International Barcode of Life Conference (Kruger National Park, South Africa, 20-24 November 2017) share the latest findings in DNA barcoding research and its increasingly diversified applications. Here, we review prevailing trends synthesized from among 429 invited and contributed abstracts, which are collated in this open-access special issue of Genome. Hosted for the first time on the African continent, the 7th Conference places special emphasis on the evolutionary origins, biogeography, and conservation of African flora and fauna. Within Africa and elsewhere, DNA barcoding and related techniques are being increasingly used for wildlife forensics and for the validation of commercial products, such as medicinal plants and seafood species. A striking trend of the conference is the dramatic rise of studies on environmental DNA (eDNA) and on diverse uses of high-throughput sequencing techniques. Emerging techniques in these areas are opening new avenues for environmental biomonitoring, managing species-at-risk and invasive species, and revealing species interaction networks in unprecedented detail. Contributors call for the development of validated community standards for high-throughput sequence data generation and analysis, to enable the full potential of these methods to be realized for understanding and managing biodiversity on a global scale.</p><h1>Environmental DNA metabarcoding: Transforming how we survey animal and plant communities</h1>
<p><a href="https://doi.org/10.1111/Mec.14350">doi:10.1111/Mec.14350</a></p>

<p><a href=None>None</a></p>
<p>The genomic revolution has fundamentally changed how we survey biodiversity on earth. High-throughput sequencing ("HTS") platforms now enable the rapid sequencing of DNA from diverse kinds of environmental samples (termed "environmental DNA" or "eDNA"). Coupling HTS with our ability to associate sequences from eDNA with a taxonomic name is called "eDNA metabarcoding" and offers a powerful molecular tool capable of noninvasively surveying species richness from many ecosystems. Here, we review the use of eDNA metabarcoding for surveying animal and plant richness, and the challenges in using eDNA approaches to estimate relative abundance. We highlight eDNA applications in freshwater, marine and terrestrial environments, and in this broad context, we distill what is known about the ability of different eDNA sample types to approximate richness in space and across time. We provide guiding questions for study design and discuss the eDNA metabarcoding workflow with a focus on primers and library preparation methods. We additionally discuss important criteria for consideration of bioinformatic filtering of data sets, with recommendations for increasing transparency. Finally, looking to the future, we discuss emerging applications of eDNA metabarcoding in ecology, conservation, invasion biology, biomonitoring, and how eDNA metabarcoding can empower citizen science and biodiversity education.</p><h1>From Microscopy to Genomic Approach in Soil Biodiversity Assessment</h1>
<p><a href="https://doi.org/10.21775/Cimb.027.195">doi:10.21775/Cimb.027.195</a></p>

<p><a href=None>None</a></p>
<p>Soil biota represents a major component of the earth's biodiversity and for over 200 years, the microscopy approach was the only way to explore it. In the last decade, the DNA-based technique has been adopted in soil ecology. Due to the rapid development of cutting-edge technology, the field is transitioning from barcoding individuals to metabarcoding communities. With the advent of next-generation sequencing and a rapid decline in sequencing cost, it has become feasible to assess soil biodiversity at species level. This review article summarizes current approaches in soil biodiversity research along with their advantages and disadvantages.</p><h1>DNA in a bottle-Rapid metabarcoding survey for early alerts of invasive species in ports</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0183347">doi:10.1371/Journal.Pone.0183347</a></p>

<p><a href=None>None</a></p>
<p>Biota monitoring in ports is increasingly needed for biosecurity reasons and safeguarding marine biodiversity from biological invasion. Present and future international biosecurity directives can be accomplished only if the biota acquired by maritime traffic in ports is controlled. Methodologies for biota inventory are diverse and now rely principally on extensive and labor-intensive sampling along with taxonomic identification by experts. In this study, we employed an extremely simplified environmental DNA (eDNA) sampling methodology from only three 1-L bottles of water per port, followed by metabarcoding (high-throughput sequencing and DNA-based species identification) using 18S rDNA and Cytochrome oxidase I as genetic barcodes. Eight Bay of Biscay ports with available inventory of fouling invertebrates were employed as a case study. Despite minimal sampling efforts, three invasive invertebrates were detected: the barnacle Austrominius modestus, the tubeworm Ficopomatus enigmaticus and the polychaete Polydora triglanda. The same species have been previously found from visual and DNA barcoding (genetic identification of individuals) surveys in the same ports. The current costs of visual surveys, conventional DNA barcoding and this simplified metabarcoding protocol were compared. The results encourage the use of metabarcoding for early biosecurity alerts.</p><h1>Predicting the Ecological Quality Status of Marine Environments from eDNA Metabarcoding Data Using Supervised Machine Learning</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.7B01518">doi:10.1021/Acs.Est.7B01518</a></p>

<p><a href=None>None</a></p>
<p>Monitoring biodiversity is essential to assess the impacts of increasing anthropogenic activities in marine environments. Traditionally, marine biomonitoring involves the sorting and morphological identification of benthic macro-invertebrates, which is time-consuming and taxonomic-expertise demanding. High-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) represents a promising alternative for benthic monitoring. However, an important fraction of eDNA sequences remains unassigned or belong to taxa of unknown ecology, which prevent their use for assessing the ecological quality status. Here, we show that supervised machine learning (SML) can be used to build robust predictive models for benthic monitoring, regardless of the taxonomic assignment of eDNA sequences. We tested three SML approaches to assess the environmental impact of marine aquaculture using benthic foraminifera eDNA, a group of unicellular eukaryotes known to be good bioindicators, as features to infer macro-invertebrates based biotic indices. We found similar ecological status as obtained from macro-invertebrates inventories. We argue that SML approaches could overcome and even bypass the cost and time-demanding morpho-taxonomic approaches in future biomonitoring.</p><h1>Annual time-series analysis of aqueous eDNA reveals ecologically relevant dynamics of lake ecosystem biodiversity</h1>
<p><a href="https://doi.org/10.1038/Ncomms14087">doi:10.1038/Ncomms14087</a></p>

<p><a href=None>None</a></p>
<p>The use of environmental DNA (eDNA) in biodiversity assessments offers a step-change in sensitivity, throughput and simultaneous measures of ecosystem diversity and function. There remains, however, a need to examine eDNA persistence in the wild through simultaneous temporal measures of eDNA and biota. Here, we use metabarcoding of two markers of different lengths, derived from an annual time series of aqueous lake eDNA to examine temporal shifts in ecosystem biodiversity and in an ecologically important group of macroinvertebrates (Diptera: Chironomidae). The analyses allow different levels of detection and validation of taxon richness and community composition (β-diversity) through time, with shorter eDNA fragments dominating the eDNA community. Comparisons between eDNA, community DNA, taxonomy and UK species abundance data further show significant relationships between diversity estimates derived across the disparate methodologies. Our results reveal the temporal dynamics of eDNA and validate the utility of eDNA metabarcoding for tracking seasonal diversity at the ecosystem scale.</p><h1>eDNA metabarcoding: a promising method for anuran surveys in highly diverse tropical forests</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12643">doi:10.1111/1755-0998.12643</a></p>

<p><a href=None>None</a></p>
<p>Understanding the geographical distribution and community composition of species is crucial to monitor species persistence and define effective conservation strategies. Environmental DNA (eDNA) has emerged as a powerful noninvasive tool for species detection. However, most eDNA survey methods have been developed and applied in temperate zones. We tested the feasibility of using eDNA to survey anurans in tropical streams in the Brazilian Atlantic forest and compared the results with short-term visual and audio surveys. We detected all nine species known to inhabit our focal streams with one single visit for eDNA sampling. We found a higher proportion of sequence reads and larger number of positive PCR replicates for more common species and for those with life cycles closely associated with the streams, factors that may contribute to increased release of DNA in the water. However, less common species were also detected in eDNA samples, demonstrating the detection power of this method. Filtering larger volumes of water resulted in a higher probability of detection. Our data also show it is important to sample multiple sites along streams, particularly for detection of target species with lower population densities. For the three focal species in our study, the eDNA metabarcoding method had a greater capacity of detection per sampling event than our rapid field surveys, and thus, has the potential to circumvent some of the challenges associated with traditional approaches. Our results underscore the utility of eDNA metabarcoding as an efficient method to survey anuran species in tropical streams of the highly biodiverse Brazilian Atlantic forest.</p><h1>One step forwards for the routine use of high-throughput DNA sequencing in environmental monitoring. An efficient and standardizable method to maximize the detection of environmental bacteria</h1>
<p><a href="https://doi.org/10.1002/Mbo3.421">doi:10.1002/Mbo3.421</a></p>

<p><a href=None>None</a></p>
<p>We propose an innovative, repeatable, and reliable experimental workflow to concentrate and detect environmental bacteria in drinking water using molecular techniques. We first concentrated bacteria in water samples using tangential flow filtration and then we evaluated two methods of environmental DNA extraction. We performed tests on both artificially contaminated water samples and real drinking water samples. The efficiency of the experimental workflow was measured through qPCR. The successful applicability of the high-throughput DNA sequencing (HTS) approach was demonstrated on drinking water samples. Our results demonstrate the feasibility of our approach in high-throughput-based studies, and we suggest incorporating it in monitoring strategies to have a better representation of the microbial community. In the recent years, HTS techniques have become key tools in the study of microbial communities. To make the leap from academic laboratories to the routine monitoring (e.g., water treatment plants laboratories), we here propose an experimental workflow suitable for the introduction of HTS as a standard method for detecting environmental bacteria.</p><h1>Barcoding the food chain: from Sanger to high-throughput sequencing</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Society faces the complex challenge of supporting biodiversity and ecosystem functioning, while ensuring food security by providing safe traceable food through an ever-more-complex global food chain. The increase in human mobility brings the added threat of pests, parasites, and invaders that further complicate our agro-industrial efforts. DNA barcoding technologies allow researchers to identify both individual species, and, when combined with universal primers and high-throughput sequencing techniques, the diversity within mixed samples (metabarcoding). These tools are already being employed to detect market substitutions, trace pests through the forensic evaluation of trace "environmental DNA", and to track parasitic infections in livestock. The potential of DNA barcoding to contribute to increased security of the food chain is clear, but challenges remain in regulation and the need for validation of experimental analysis. Here, we present an overview of the current uses and challenges of applied DNA barcoding in agriculture, from agro-ecosystems within farmland to the kitchen table.</p><h1>Censusing marine eukaryotic diversity in the twenty-first century</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0331">doi:10.1098/Rstb.2015.0331</a></p>

<p><a href=None>None</a></p>
<p>The ocean constitutes one of the vastest and richest biomes on our planet. Most recent estimations, all based on indirect approaches, suggest that there are millions of marine eukaryotic species. Moreover, a large majority of these are small (less than 1 mm), cryptic and still unknown to science. However, this knowledge gap, caused by the lack of diagnostic morphological features in small organisms and the limited sampling of the global ocean, is currently being filled, thanks to new DNA-based approaches. The molecular technique of PCR amplification of homologous gene regions combined with high-throughput sequencing, routinely used to census unculturable prokaryotes, is now also being used to characterize whole communities of marine eukaryotes. Here, we review how this methodological advancement has helped to better quantify the magnitude and patterns of marine eukaryotic diversity, with an emphasis on taxonomic groups previously largely overlooked. We then discuss obstacles remaining to achieve a global understanding of marine eukaryotic diversity. In particular, we argue that 18S variable regions do not provide sufficient taxonomic resolution to census marine life, and suggest combining broad eukaryotic surveys targeting the 18S rRNA region with more taxon-focused analyses of hypervariable regions to improve our understanding of the diversity of species, the functional units of marine ecosystems.This article is part of the themed issue 'From DNA barcodes to biomes'.</p><h1>A new way to contemplate Darwin's tangled bank: how DNA barcodes are reconnecting biodiversity science and biomonitoring</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0330">doi:10.1098/Rstb.2015.0330</a></p>

<p><a href=None>None</a></p>
<p>Encompassing the breadth of biodiversity in biomonitoring programmes has been frustrated by an inability to simultaneously identify large numbers of species accurately and in a timely fashion. Biomonitoring infers the state of an ecosystem from samples collected and identified using the best available taxonomic knowledge. The advent of DNA barcoding has now given way to the extraction of bulk DNA from mixed samples of organisms in environmental samples through the development of high-throughput sequencing (HTS). This DNA metabarcoding approach allows an unprecedented view of the true breadth and depth of biodiversity, but its adoption poses two important challenges. First, bioinformatics techniques must simultaneously perform complex analyses of large datasets and translate the results of these analyses to a range of users. Second, the insights gained from HTS need to be amalgamated with concepts such as Linnaean taxonomy and indicator species, which are less comprehensive but more intuitive. It is clear that we are moving beyond proof-of-concept studies to address the challenge of implementation of this new approach for environmental monitoring and regulation. Interpreting Darwin's 'tangled bank' through a DNA lens is now a reality, but the question remains: how can this information be generated and used reliably, and how does it relate to accepted norms in ecosystem study?This article is part of the themed issue 'From DNA barcodes to biomes'.</p><h1>Pollen DNA barcoding: current applications and future prospects</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0200">doi:10.1139/Gen-2015-0200</a></p>

<p><a href=None>None</a></p>
<p>Identification of the species origin of pollen has many applications, including assessment of plant-pollinator networks, reconstruction of ancient plant communities, product authentication, allergen monitoring, and forensics. Such applications, however, have previously been limited by microscopy-based identification of pollen, which is slow, has low taxonomic resolution, and has few expert practitioners. One alternative is pollen DNA barcoding, which could overcome these issues. Recent studies demonstrate that both chloroplast and nuclear barcoding markers can be amplified from pollen. These recent validations of pollen metabarcoding indicate that now is the time for researchers in various fields to consider applying these methods to their research programs. In this paper, we review the nascent field of pollen DNA barcoding and discuss potential new applications of this technology, highlighting existing limitations and future research developments that will improve its utility in a wide range of applications. </p><h1>Advances in DNA metabarcoding for food and wildlife forensic species identification</h1>
<p><a href="https://doi.org/10.1007/S00216-016-9595-8">doi:10.1007/S00216-016-9595-8</a></p>

<p><a href=None>None</a></p>
<p>Species identification using DNA barcodes has been widely adopted by forensic scientists as an effective molecular tool for tracking adulterations in food and for analysing samples from alleged wildlife crime incidents. DNA barcoding is an approach that involves sequencing of short DNA sequences from standardized regions and comparison to a reference database as a molecular diagnostic tool in species identification. In recent years, remarkable progress has been made towards developing DNA metabarcoding strategies, which involves next-generation sequencing of DNA barcodes for the simultaneous detection of multiple species in complex samples. Metabarcoding strategies can be used in processed materials containing highly degraded DNA e.g. for the identification of endangered and hazardous species in traditional medicine. This review aims to provide insight into advances of plant and animal DNA barcoding and highlights current practices and recent developments for DNA metabarcoding of food and wildlife forensic samples from a practical point of view. Special emphasis is placed on new developments for identifying species listed in the Convention on International Trade of Endangered Species (CITES) appendices for which reliable methods for species identification may signal and/or prevent illegal trade. Current technological developments and challenges of DNA metabarcoding for forensic scientists will be assessed in the light of stakeholders' needs. </p><h1>Protist metabarcoding and environmental biomonitoring: Time for change</h1>
<p><a href="https://doi.org/S0932-4739(16)30003-7">doi:S0932-4739(16)30003-7</a></p>

<p><a href=None>None</a></p>
<p>High-throughput amplicon sequencing of environmental DNA and/or RNA proved to be a powerful tool to describe protist diversity. This new approach called also the metabarcoding has totally transformed our view of protist diversity, revealing a large number of novel lineages and expanding the range of protist phylogenetic diversity at almost every taxonomic level. However, until now the objectives of the vast majority of metabarcoding studies were purely academic. Practical applications of protist metabarcoding are surprisingly scarce, despite the fact that several groups of protists are commonly used as bioindicators of environmental impacts in freshwater or marine ecosystems. Here, we are reviewing studies that examine the ecological applications of metabarcoding for two groups of well-known protist bioindicators: diatoms and foraminifera. The results of these studies show that despite some biological and technical biases, molecular data quite faithfully reflect the morphology-based biotic indices and provide a similar assessment of ecosystem status. In view of these results, protist metabarcoding appears as a rapid and accurate tool for the evaluation of the quality of aquatic ecosystems. Hence, we plead for integration of protist metabarcoding in future biomonitoring projects as a complement of traditional methods and a source of new biosensors for environmental impact assessment. </p><h1>DNA metabarcoding of insects and allies: an evaluation of primers and pipelines</h1>
<p><a href="https://doi.org/10.1017/S0007485315000681">doi:10.1017/S0007485315000681</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding, the coupling of DNA-based species identification and high-throughput sequencing, offers enormous promise for arthropod biodiversity studies but factors such as cost, speed and ease-of-use of bioinformatic pipelines, crucial for making the leapt from demonstration studies to a real-world application, have not yet been adequately addressed. Here, four published and one newly designed primer sets were tested across a diverse set of 80 arthropod species, representing 11 orders, to establish optimal protocols for Illumina-based metabarcoding of tropical Malaise trap samples. Two primer sets which showed the highest amplification success with individual specimen polymerase chain reaction (PCR, 98%) were used for bulk PCR and Illumina MiSeq sequencing. The sequencing outputs were subjected to both manual and simple metagenomics quality control and filtering pipelines. We obtained acceptable detection rates after bulk PCR and high-throughput sequencing (80-90% of input species) but analyses were complicated by putative heteroplasmic sequences and contamination. The manual pipeline produced similar or better outputs to the simple metagenomics pipeline (1.4 compared with 0.5 expected:unexpected Operational Taxonomic Units). Our study suggests that metabarcoding is slowly becoming as cheap, fast and easy as conventional DNA barcoding, and that Malaise trap metabarcoding may soon fulfill its potential, providing a thermometer for biodiversity. </p><h1>Standardizing metadata and taxonomic identification in metabarcoding studies</h1>
<p><a href="https://doi.org/10.1186/S13742-015-0074-5">doi:10.1186/S13742-015-0074-5</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing-based metabarcoding studies produce vast amounts of ecological data, but a lack of consensus on standardization of metadata and how to refer to the species recovered severely hampers reanalysis and comparisons among studies. Here we propose an automated workflow covering data submission, compression, storage and public access to allow easy data retrieval and inter-study communication. Such standardized and readily accessible datasets facilitate data management, taxonomic comparisons and compilation of global metastudies. </p><h1>obitools: a unix-inspired software package for DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12428">doi:10.1111/1755-0998.12428</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding offers new perspectives in biodiversity research. This recently developed approach to ecosystem study relies heavily on the use of next-generation sequencing (NGS) and thus calls upon the ability to deal with huge sequence data sets. The obitools package satisfies this requirement thanks to a set of programs specifically designed for analysing NGS data in a DNA metabarcoding context. Their capacity to filter and edit sequences while taking into account taxonomic annotation helps to set up tailor-made analysis pipelines for a broad range of DNA metabarcoding applications, including biodiversity surveys or diet analyses. The obitools package is distributed as an open source software available on the following website: http://metabarcoding.org/obitools. A Galaxy wrapper is available on the GenOuest core facility toolshed: http://toolshed.genouest.org. </p><h1>PhytoREF: a reference database of the plastidial 16S rRNA gene of photosynthetic eukaryotes with curated taxonomy</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12401">doi:10.1111/1755-0998.12401</a></p>

<p><a href=None>None</a></p>
<p>Photosynthetic eukaryotes have a critical role as the main producers in most ecosystems of the biosphere. The ongoing environmental metabarcoding revolution opens the perspective for holistic ecosystems biological studies of these organisms, in particular the unicellular microalgae that often lack distinctive morphological characters and have complex life cycles. To interpret environmental sequences, metabarcoding necessarily relies on taxonomically curated databases containing reference sequences of the targeted gene (or barcode) from identified organisms. To date, no such reference framework exists for photosynthetic eukaryotes. In this study, we built the PhytoREF database that contains 6490 plastidial 16S rDNA reference sequences that originate from a large diversity of eukaryotes representing all known major photosynthetic lineages. We compiled 3333 amplicon sequences available from public databases and 879 sequences extracted from plastidial genomes, and generated 411 novel sequences from cultured marine microalgal strains belonging to different eukaryotic lineages. A total of 1867 environmental Sanger 16S rDNA sequences were also included in the database. Stringent quality filtering and a phylogeny-based taxonomic classification were applied for each 16S rDNA sequence. The database mainly focuses on marine microalgae, but sequences from land plants (representing half of the PhytoREF sequences) and freshwater taxa were also included to broaden the applicability of PhytoREF to different aquatic and terrestrial habitats. PhytoREF, accessible via a web interface (http://phytoref.fr), is a new resource in molecular ecology to foster the discovery, assessment and monitoring of the diversity of photosynthetic eukaryotes using high-throughput sequencing. </p><h1>Environmental genes and genomes: understanding the differences and challenges in the approaches and software for their analyses</h1>
<p><a href="https://doi.org/10.1093/Bib/Bbv001">doi:10.1093/Bib/Bbv001</a></p>

<p><a href=None>None</a></p>
<p>DNA-based taxonomic and functional profiling is widely used for the characterization of organismal communities across a rapidly increasing array of research areas that include the role of microbiomes in health and disease, biomonitoring, and estimation of both microbial and metazoan species richness. Two principal approaches are currently used to assign taxonomy to DNA sequences: DNA metabarcoding and metagenomics. When initially developed, each of these approaches mandated their own particular methods for data analysis; however, with the development of high-throughput sequencing (HTS) techniques they have begun to share many aspects in data set generation and processing. In this review we aim to define the current characteristics, goals and boundaries of each field, and describe the different software used for their analysis. We argue that an appreciation of the potential and limitations of each method can help underscore the improvements required by each field so as to better exploit the richness of current HTS-based data sets. </p><h1>Next-generation environmental diversity surveys of foraminifera: preparing the future</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Foraminifera are commonly defined as marine testate protists, and their diversity is mainly assessed on the basis of the morphology of their agglutinated or mineralized tests. Diversity surveys based on environmental DNA (eDNA) have dramatically changed this view by revealing an unexpected diversity of naked and organic-walled lineages as well as detecting foraminiferal lineages in soil and freshwater environments. Moreover, single-cell analyses have allowed discrimination among genetically distinctive types within almost every described morphospecies. In view of these studies, the foraminiferal diversity appeared to be largely underestimated, but its accurate estimation was impeded by the low speed and coverage of a cloning-based eDNA approach. With the advent of high-throughput sequencing (HTS) technologies, these limitations disappeared in favor of exhaustive descriptions of foraminiferal diversity in numerous samples. Yet, the biases and errors identified in early HTS studies raised some questions about the accuracy of HTS data and their biological interpretation. Among the most controversial issues affecting the reliability of HTS diversity estimates are (1) the impact of technical and biological biases, (2) the sensitivity and specificity of taxonomic sequence assignment, (3) the ability to distinguish rare species, and (4) the quantitative interpretation of HTS data. Here, we document the lessons learned from previous HTS surveys and present the current advances and applications focusing on foraminiferal eDNA. We discuss the problems associated with HTS approaches and predict the future trends and avenues that hold promises for surveying foraminiferal diversity accurately and efficiently. </p><h1>Next generation restoration genetics: applications and opportunities</h1>
<p><a href="https://doi.org/10.1016/J.Tplants.2014.03.011">doi:10.1016/J.Tplants.2014.03.011</a></p>

<p><a href=None>None</a></p>
<p>Restoration ecology is a young scientific discipline underpinning improvements in the rapid global expansion of ecological restoration. The application of molecular tools over the past 20 years has made an important contribution to understanding genetic factors influencing ecological restoration success. Here we illustrate how recent advances in next generation sequencing (NGS) methods are revolutionising the practical contribution of genetics to restoration. Novel applications include a dramatically enhanced capacity to measure adaptive variation for optimal seed sourcing, high-throughput assessment and monitoring of natural and restored biological communities aboveground and belowground, and gene expression analysis as a measure of genetic resilience of restored populations. Challenges remain in data generation, handling and analysis, and how best to apply NGS for practical outcomes in restoration. </p><h1>Improved inference of taxonomic richness from environmental DNA</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0071974">doi:10.1371/Journal.Pone.0071974</a></p>

<p><a href=None>None</a></p>
<p>Accurate estimation of biological diversity in environmental DNA samples using high-throughput amplicon pyrosequencing must account for errors generated by PCR and sequencing. We describe a novel approach to distinguish the underlying sequence diversity in environmental DNA samples from errors that uses information on the abundance distribution of similar sequences across independent samples, as well as the frequency and diversity of sequences within individual samples. We have further refined this approach into a bioinformatics pipeline, Amplicon Pyrosequence Denoising Program (APDP) that is able to process raw sequence datasets into a set of validated sequences in formats compatible with commonly used downstream analyses packages. We demonstrate, by sequencing complex environmental samples and mock communities, that APDP is effective for removing errors from deeply sequenced datasets comprising biological and technical replicates, and can efficiently denoise single-sample datasets. APDP provides more conservative diversity estimates for complex datasets than other approaches; however, for some applications this may provide a more accurate and appropriate level of resolution, and result in greater confidence that returned sequences reflect the diversity of the underlying sample. </p><h1>Towards next-generation biodiversity assessment using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05470.X">doi:10.1111/J.1365-294X.2012.05470.X</a></p>

<p><a href=None>None</a></p>
<p>Virtually all empirical ecological studies require species identification during data collection. DNA metabarcoding refers to the automated identification of multiple species from a single bulk sample containing entire organisms or from a single environmental sample containing degraded DNA (soil, water, faeces, etc.). It can be implemented for both modern and ancient environmental samples. The availability of next-generation sequencing platforms and the ecologists' need for high-throughput taxon identification have facilitated the emergence of DNA metabarcoding. The potential power of DNA metabarcoding as it is implemented today is limited mainly by its dependency on PCR and by the considerable investment needed to build comprehensive taxonomic reference libraries. Further developments associated with the impressive progress in DNA sequencing will eliminate the currently required DNA amplification step, and comprehensive taxonomic reference libraries composed of whole organellar genomes and repetitive ribosomal nuclear DNA can be built based on the well-curated DNA extract collections maintained by standardized barcoding initiatives. The near-term future of DNA metabarcoding has an enormous potential to boost data acquisition in biodiversity research.</p><h1>Next-generation sequencing technologies for environmental DNA research</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05538.X">doi:10.1111/J.1365-294X.2012.05538.X</a></p>

<p><a href=None>None</a></p>
<p>Since 2005, advances in next-generation sequencing technologies have revolutionized biological science. The analysis of environmental DNA through the use of specific gene markers such as species-specific DNA barcodes has been a key application of next-generation sequencing technologies in ecological and environmental research. Access to parallel, massive amounts of sequencing data, as well as subsequent improvements in read length and throughput of different sequencing platforms, is leading to a better representation of sample diversity at a reasonable cost. New technologies are being developed rapidly and have the potential to dramatically accelerate ecological and environmental research. The fast pace of development and improvements in next-generation sequencing technologies can reflect on broader and more robust applications in environmental DNA research. Here, we review the advantages and limitations of current next-generation sequencing technologies in regard to their application for environmental DNA analysis.</p><h1>Monitoring endangered freshwater biodiversity using environmental DNA</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2011.05418.X">doi:10.1111/J.1365-294X.2011.05418.X</a></p>

<p><a href=None>None</a></p>
<p>Freshwater ecosystems are among the most endangered habitats on Earth, with thousands of animal species known to be threatened or already extinct. Reliable monitoring of threatened organisms is crucial for data-driven conservation actions but remains a challenge owing to nonstandardized methods that depend on practical and taxonomic expertise, which is rapidly declining. Here, we show that a diversity of rare and threatened freshwater animals--representing amphibians, fish, mammals, insects and crustaceans--can be detected and quantified based on DNA obtained directly from small water samples of lakes, ponds and streams. We successfully validate our findings in a controlled mesocosm experiment and show that DNA becomes undetectable within 2 weeks after removal of animals, indicating that DNA traces are near contemporary with presence of the species. We further demonstrate that entire faunas of amphibians and fish can be detected by high-throughput sequencing of DNA extracted from pond water. Our findings underpin the ubiquitous nature of DNA traces in the environment and establish environmental DNA as a tool for monitoring rare and threatened species across a wide range of taxonomic groups.</p><h1>Global soil microbiomes: A new frontline of biome-ecology research</h1>
<p><a href="https://doi.org/10.1111/geb.13487">doi:10.1111/geb.13487</a></p>

<p><a href=None>None</a></p>
<p>Aim: Organisms on our planet form spatially congruent and functionally distinct communities, which at large geographical scales are called “biomes”. Understanding their pattern and function is vital for sustainable use and protection of biodiversity. Current global terrestrial biome classifications are based primarily on climate characteristics and functional aspects of plant community assembly. These and other existing biome schemes do not take account of soil organisms, including highly diverse and functionally important microbial groups. We aimed to define large-scale structure in the diversity of soil microbes (soil microbiomes), pinpoint the environmental drivers shaping it and identify resemblance and mismatch with existing terrestrial biome schemes. Location: Global. Time period: Current. Major taxa studied: Soil eukaryotes and prokaryotes. Methods: We collected soil samples from natural environments world-wide, incorporating most known terrestrial biomes. We used high-throughput sequencing to characterize soil biotic communities and k-means clustering to define soil microbiomes describing the diversity of microbial eukaryotic and prokaryotic groups. We used climatic data and soil variables measured in the field to identify the environmental variables shaping soil microbiome structure. Results: We recorded strong correlations among fungal, bacterial, archaeal, plant and animal communities, defined a system of global soil microbiomes (producing seven biome types for microbial eukaryotes and six biome types for prokaryotes) and showed that these are typically structured by pH alongside temperature. None of the soil microbiomes are directly paralleled by any current terrestrial biome scheme, with mismatch most substantial for prokaryotes and for microbial eukaryotes in cold climates; nor do they consistently distinguish grassland and forest ecosystems. Main conclusions: Existing terrestrial biome classifications represent a limited surrogate for the large-scale diversity patterns of microbial soil organisms. We show that empirically defined soil microbiomes are attainable using metabarcoding and statistical clustering approaches and suggest that they can have wide application in theoretical and applied biodiversity research.</p><h1>High-frequency and long-term observations of eDNA from imperiled salmonids in a coastal stream: Temporal dynamics, relationships with environmental factors, and comparisons with conventional observations</h1>
<p><a href="https://doi.org/10.1002/edn3.293">doi:10.1002/edn3.293</a></p>

<p><a href=None>None</a></p>
<p>A greater understanding of eDNA behavior in the environment is needed before it can be employed for ecosystem monitoring applications. The objectives of this study were to use autonomous sampling to conduct long-term, high-frequency monitoring of the eDNA of native salmonid species in a Californian coastal stream, describe temporal variation of eDNA on multiple scales and identify environmental factors that drive this variation, and evaluate the ability of the eDNA datasets to detect rare species and represent organismal abundance. Using high-throughput autonomous environmental sample processors (ESPs) and qPCR, we enumerated eDNA concentrations from 674 water samples collected at subdaily intervals over 360 days at a single site. We detected eDNA from two imperiled salmonids (coho salmon Oncorhynchus kisutch and steelhead/rainbow trout O. mykiss) in most samples; O. kisutch eDNA was generally in lower concentration and more variable than O. mykiss eDNA. High-frequency (i.e., subdaily and daily) variability in salmonid eDNA concentrations showed occasional patchiness (i.e., large differences between consecutive samples), while seasonal differences were observed consistent with the ecology of the species at this site. Salmonid eDNA concentrations were significantly associated with creek discharge, photoperiod, and whether the creek mouth was open or closed by a seasonal sandbar. The release of hatchery-origin O. kisutch parr into the stream was associated with a significant increase in eDNA concentration for the remainder of the study. We compared eDNA signals with fish abundance data collected from traps located at the site. Fish were detected more often by eDNA than from trapping. Significant positive associations between fish abundance and eDNA concentrations were observed for O. mykiss; however, no such associations were observed for O. kisutch. This study adds to our knowledge on the occurrence and behavior of fish eDNA in lotic systems and informs future biomonitoring efforts using automated sampling technology.</p><h1>Coupling ecological network analysis with high-throughput sequencing-based surveys: Lessons from the next-generation biomonitoring project</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2021.10.007">doi:10.1016/bs.aecr.2021.10.007</a></p>

<p><a href=None>None</a></p>
<p>Biomonitoring ecosystems is necessary in order to evaluate risks and to efficiently manage ecosystems and their associated services. Agrosystems are the target of multiple stressors that can affect many species through effects cascading along food webs. However, classic biomonitoring, focused on species diversity or indicator species, might be a poor predictor of the risk of such whole-ecosystem perturbations. Thanks to high-throughput sequencing methods, however, it might be possible to obtain sufficient information about entire ecological communities to infer the functioning of their associated interaction networks, and thus monitor more closely the risk of the collapse of entire food webs due to external stressors. In the course of the ‘next-generation biomonitoring’ project, we collectively sought to experiment with this idea of inferring ecological networks on the basis of metabarcoding information gathered on different systems. We here give an overview of issues and preliminary results associated with this endeavour and highlight the main difficulties that such next-generation biomonitoring is still facing. Going from sampling protocols up to methods for comparing inferred networks, through biomolecular, bioinformatic, and network inference, we review all steps of the process, with a view towards generality and transferability towards other systems.</p><h1>Potential of high-throughput eDNA sequencing of soil fungi and bacteria for monitoring ecological restoration in ultramafic substrates: The case study of the New Caledonian biodiversity hotspot</h1>
<p><a href="https://doi.org/10.1016/j.ecoleng.2021.106416">doi:10.1016/j.ecoleng.2021.106416</a></p>

<p><a href=None>None</a></p>
<p>Due to their central role in ecosystems functioning and their ability to rapidly respond to environmental changes, soil microorganisms could potentially be used for monitoring ecosystems recovery in the context of degraded land restoration. However, these belowground organisms have been, to date, largely neglected. Here, we investigated fungal and bacterial community diversity, composition, and structure from ultramafic soils in New Caledonia, an archipelago in the southwest Pacific recognized as a priority for conservation and restoration. The emerging approach of high-throughput amplicon sequencing of environmental DNA (eDNA) – metabarcoding of eDNA – was used to compare soil microbial communities from four different native vegetation types, representing different stages of a chronosequence and defined as reference ecosystems, to five distinct post-mining sites revegetated several years ago. Our results clearly revealed changes in soil microbial phyla and functional groups along the reference chronosequence and variable responses at the different revegetated sites, with two of the five sites showing a good trajectory of recovery. We thus propose three ratios as metrics for monitoring the restoration trajectory of soil microorganisms: the Ascomycota:Basidiomycota and Saprotrophic:Ectomycorrhizal ratios for fungi, and the Cyanobacteria:Chloroflexi ratios for bacteria. Our study, combined with recent works undertaken in other geographical areas, underpins the great promise that could represent soil microbial eDNA metabarcoding for monitoring restoration progress and success. With the emergence of these new cost-effective and scalable sequencing technologies, soil microbes could, in the near future, be included in guidelines for restoration operations in complement to more conventional approaches.</p><h1>The future of dna barcoding: Reflections from early career researchers</h1>
<p><a href="https://doi.org/10.3390/d13070313">doi:10.3390/d13070313</a></p>

<p><a href=None>None</a></p>
<p>Over the last two decades, the use of DNA barcodes has transformed our ability to identify and assess life on our planet. Both strengths and weaknesses of the method have been exemplified through thousands of peer-reviewed scientific articles. Given the novel sequencing approaches, currently capable of generating millions of reads at low cost, we reflect on the questions: What will the future bring for DNA barcoding? Will identification of species using short, standardized fragments of DNA stand the test of time? We present reflected opinions of early career biodiversity researchers in the form of a SWOT analysis and discuss answers to these questions.</p><h1>metabaR: An r package for the evaluation and improvement of DNA metabarcoding data quality</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13552">doi:10.1111/2041-210X.13552</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is becoming the tool of choice for biodiversity assessment across taxa and environments. Yet, the artefacts present in metabarcoding datasets often preclude a proper interpretation of ecological patterns. Bioinformatic pipelines to remove experimental noise exist. However, these often only partially target produced artefacts, or are marker specific. In addition, assessments of data curation quality and chosen filtering thresholds are seldom available in existing pipelines, partly due to the lack of appropriate visualisation tools. Here, we present metabaR, an r package that provides a comprehensive suite of tools to effectively curate DNA metabarcoding data after basic bioinformatic analyses. In particular, metabaR uses experimental negative or positive controls to identify different types of artefactual sequences, that is, contaminants and tag-jumps. It also flags potentially dysfunctional PCRs based on PCR replicate similarities when those are available. Finally, metabaR provides tools to visualise DNA metabarcoding data characteristics in their experimental context as well as their distribution, and facilitates assessment of the appropriateness of data curation filtering thresholds. metabaR is applicable to any DNA metabarcoding experimental design but is most powerful when the design includes experimental controls and replicates. More generally, the simplicity and flexibility of the package makes it applicable any DNA marker, and data generated with any sequencing platform, and pre-analysed with any bioinformatic pipeline. Its outputs are easily usable for downstream analyses with any ecological r package. metabaR complements existing bioinformatics pipelines by providing scientists with a variety of functions to effectively clean DNA metabarcoding data and avoid serious misinterpretations. It thus offers a promising platform for automatised data quality assessments of DNA metabarcoding data for environmental research and biomonitoring.</p><h1>eDNAir: Proof of concept that animal DNA can be collected from air sampling</h1>
<p><a href="https://doi.org/10.7717/peerj.11030">doi:10.7717/peerj.11030</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is one of the fastest developing tools for species biomonitoring and ecological research. However, despite substantial interest from research, commercial and regulatory sectors, it has remained primarily a tool for aquatic systems with a small amount of work in substances such as soil, snow and rain. Here we demonstrate that eDNA can be collected from air and used to identify mammals. Our proof of concept successfully demonstrated that eDNA sampled from air contained mixed templates which reflect the species known to be present within a confined space and that this material can be accessed using existing sampling methods. We anticipate this demonstration will initiate a much larger research programme in terrestrial airDNA sampling and that this may rapidly advance biomonitoring approaches. Lastly, we outline these and potential related applications we expect to benefit from this development.</p><h1>High-throughput sequencing view on the magnitude of global fungal diversity</h1>
<p><a href="https://doi.org/10.1007/s13225-021-00472-y">doi:10.1007/s13225-021-00472-y</a></p>

<p><a href=None>None</a></p>
<p>High-throughput DNA sequencing has dramatically transformed several areas of biodiversity research including mycology. Despite limitations, high-throughput sequencing is nowadays a predominant method to characterize the alpha and beta diversity of fungal communities. Across the papers utilizing high-throughput sequencing approaches to study natural habitats in terrestrial ecosystems worldwide, > 200 studies published until 2019 have generated over 250 million sequences of the primary mycological metabarcoding marker, the nuclear ribosomal internal transcribed spacer 2 (ITS2). Here we show that at a 97% sequence similarity threshold, the total richness of non-singleton fungal taxa across the studies published so far is 1.08 million, mostly Ascomycota (56.8% of the taxa) and Basidiomycota (36.7% of the taxa). The Chao-1 estimate of the total extant fungal diversity based on this dataset is 6.28 million taxa, representing a conservative estimate of global fungal species richness. Soil and litter represent the habitats with the highest alpha diversity of fungi followed by air, plant shoots, plant roots and deadwood with Chao-1 predictions, for samples containing 5000 sequences, of 1219, 569, 392, 228, 215 and 140 molecular species, respectively. Based on the high-throughput sequencing data, the highest proportion of unknown fungal species is associated with samples of lichen and plant tissues. When considering the use of high-throughput sequencing for the estimation of global fungal diversity, the limitations of the method have to be taken into account, some of which are sequencing platform-specific while others are inherent to the metabarcoding approaches of species representation. In this respect, high-throughput sequencing data can complement fungal diversity predictions based on methods of traditional mycology and increase our understanding of fungal biodiversity.</p><h1>Revealing hidden plant diversity in arid environments</h1>
<p><a href="https://doi.org/10.1111/ecog.05100">doi:10.1111/ecog.05100</a></p>

<p><a href=None>None</a></p>
<p>Estimating total plant diversity in extreme or hyperarid environments can be challenging, as adaptations to pronounced climate variability include evading prolonged stress periods through seeds or specialized underground organs. Short-term surveys of these ecosystems are thus likely poor estimators of actual diversity. Here we develop a multimethod strategy to obtain a more complete understanding of plant diversity from a community in the Atacama Desert. We explicitly test environmental DNA-based techniques (eDNA) to see if they can reveal the observed and ‘hidden' (dormant or locally rare) species. To estimate total plant diversity, we performed long-term traditional surveys during eight consecutive years, including El Niño and La Niña events, we then analyzed eDNA from soil samples using high-throughput sequencing. We further used soil pollen analysis and soil seed bank germination assays to identify ‘hidden' species. Each approach offers different subsets of current biodiversity at different taxonomic, spatial and temporal resolution, with a total of 92 taxa identified along the transect. Traditional field surveys identified 77 plant species over eight consecutive years. Observed community composition greatly varies interannually, with only 22 species seen every year. eDNA analysis revealed 37 taxa, eight of which were ‘hidden' in our field surveys. Soil samples contain a viable seed bank of 21 taxa. Soil pollen (27 taxa) and eDNA analysis show affinities with vegetation at the landscape scale but a weak relationship to local plot diversity. Multimethod approaches (including eDNA) in deserts are valuable tools that add to a comprehensive assessment of biodiversity in such extreme environments, where using a single method or observations over a few years is insufficient. Our results can also explain the resilience of Atacama plant communities as ‘hidden' taxa may have been active in the recent past or could even emerge in the future as accelerated global environmental change continues unabated.</p><h1>Metabarcoding From Microbes to Mammals: Comprehensive Bioassessment on a Global Scale</h1>
<p><a href="https://doi.org/10.3389/fevo.2020.581835">doi:10.3389/fevo.2020.581835</a></p>

<p><a href=None>None</a></p>
<p>Global biodiversity loss is unprecedented, and threats to existing biodiversity are growing. Given pervasive global change, a major challenge facing resource managers is a lack of scalable tools to rapidly and consistently measure Earth's biodiversity. Environmental genomic tools provide some hope in the face of this crisis, and DNA metabarcoding, in particular, is a powerful approach for biodiversity assessment at large spatial scales. However, metabarcoding studies are variable in their taxonomic, temporal, or spatial scope, investigating individual species, specific taxonomic groups, or targeted communities at local or regional scales. With the advent of modern, ultra-high throughput sequencing platforms, conducting deep sequencing metabarcoding surveys with multiple DNA markers will enhance the breadth of biodiversity coverage, enabling comprehensive, rapid bioassessment of all the organisms in a sample. Here, we report on a systematic literature review of 1,563 articles published about DNA metabarcoding and summarize how this approach is rapidly revolutionizing global bioassessment efforts. Specifically, we quantify the stakeholders using DNA metabarcoding, the dominant applications of this technology, and the taxonomic groups assessed in these studies. We show that while DNA metabarcoding has reached global coverage, few studies deliver on its promise of near-comprehensive biodiversity assessment. We then outline how DNA metabarcoding can help us move toward real-time, global bioassessment, illustrating how different stakeholders could benefit from DNA metabarcoding. Next, we address barriers to widespread adoption of DNA metabarcoding, highlighting the need for standardized sampling protocols, experts and computational resources to handle the deluge of genomic data, and standardized, open-source bioinformatic pipelines. Finally, we explore how technological and scientific advances will realize the promise of total biodiversity assessment in a sample—from microbes to mammals—and unlock the rich information genomics exposes, opening new possibilities for merging whole-system DNA metabarcoding with (1) abundance and biomass quantification, (2) advanced modeling, such as species occupancy models, to improve species detection, (3) population genetics, (4) phylogenetics, and (5) food web and functional gene analysis. While many challenges need to be addressed to facilitate widespread adoption of environmental genomic approaches, concurrent scientific and technological advances will usher in methods to supplement existing bioassessment tools reliant on morphological and abiotic data. This expanded toolbox will help ensure that the best tool is used for the job and enable exciting integrative techniques that capitalize on multiple tools. Collectively, these new approaches will aid in addressing the global biodiversity crisis we now face.</p><h1>Applications of environmental DNA (eDNA) in ecology and conservation: opportunities, challenges and prospects</h1>
<p><a href="https://doi.org/10.1007/s10531-020-01980-0">doi:10.1007/s10531-020-01980-0</a></p>

<p><a href=None>None</a></p>
<p>Conserving biodiversity in the face of ever-increasing human pressure is hampered by our lack of basic information on species occurrence, distribution, abundance, habitat requirements, and threats. Obtaining this information requires efficient and sensitive methods capable of detecting and quantifying true occurrence and diversity, including rare, cryptic and elusive species. Environmental DNA (eDNA) is an emerging technique that can increase our ability to detect and quantify biodiversity, by overcoming some of the challenges of labor-intensive traditional surveys. The application of eDNA in ecology and conservation has grown enormously in recent years, but without a concurrent growth in appreciation of its strengths and limitations. In many situations, eDNA may either not work, or it may work but not provide the information needed. Problems with (1) imperfect detection, (2) abundance quantification, (3) taxonomic assignment, (4) eDNA spatial and temporal dynamics, (5) data analysis and interpretation, and (6) assessing ecological status have all been significant. The technique has often been used without a careful evaluation of the technical challenges and complexities involved, and a determination made that eDNA is the appropriate method for the species or environment of interest. It is therefore important to evaluate the scope and relevance of eDNA-based studies, and to identify critical considerations that need to be taken into account before using the approach. We review and synthesize eDNA studies published to date to highlight the opportunities and limitations of utilizing eDNA in ecology and conservation. We identify potential ways of reducing limitations in eDNA analysis, and demonstrate how eDNA and traditional surveys can complement each other.</p><h1>A practical guide to DNA metabarcoding for entomological ecologists</h1>
<p><a href="https://doi.org/10.1111/een.12831">doi:10.1111/een.12831</a></p>

<p><a href=None>None</a></p>
<p>1. DNA metabarcoding is a cost-effective species identification approach with great potential to assist entomological ecologists. This review presents a practical guide to help entomological ecologists design their own DNA metabarcoding studies and ensure that sound ecological conclusions can be obtained. 2. The review considers approaches to field sampling, laboratory work, and bioinformatic analyses, with the aim of providing the background knowledge needed to make decisions at each step of a DNA metabarcoding workflow. 3. Although most conventional sampling methods can be adapted to DNA metabarcoding, this review highlights techniques that will ensure suitable DNA preservation during field sampling and laboratory storage. The review also calls for a greater understanding of the occurrence, transportation, and deposition of environmental DNA when applying DNA metabarcoding approaches for different ecosystems. 4. Accurate species detection with DNA metabarcoding needs to consider biases introduced during DNA extraction and PCR amplification, cross-contamination resulting from inappropriate amplicon library preparation, and downstream bioinformatic analyses. Quantifying species abundance with DNA metabarcoding is in its infancy, yet recent studies demonstrate promise for estimating relative species abundance from DNA sequencing reads. 5. Given that bioinformatics is one of the biggest hurdles for researchers new to DNA metabarcoding, several useful graphical user interface programs are recommended for sequence data processing, and the application of emerging sequencing technologies is discussed.</p><h1>Relevance of plant and fungal DNA metabarcoding in aerobiology</h1>
<p><a href="https://doi.org/10.1007/s10453-019-09574-2">doi:10.1007/s10453-019-09574-2</a></p>

<p><a href=None>None</a></p>
<p>Aerobiology studies biological particles present in the air of both indoor and outdoor environments. In the past few years, the application of molecular techniques has considerably improved the knowledge of organismal diversity in air samples and highlighted its ecological and clinical importance. DNA metabarcoding in particular has been recently applied worldwide to describe the biological composition of airborne bacteria, fungi and plants. Here, we provide a comprehensive overview of those aerobiology studies which, in the past few years, have applied DNA metabarcoding techniques. We focus specifically on researches including plants and fungi and review the multiplicity of experimental plans, sampling procedures, laboratory processing methods, the choice of the DNA barcode(s), the selection of sequencing platforms and bioinformatics pipelines for the comparability of data obtained from multiple origins.</p><h1>Diatom metabarcoding applied to large scale monitoring networks: Optimization of bioinformatics strategies using Mothur software</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2019.105775">doi:10.1016/j.ecolind.2019.105775</a></p>

<p><a href=None>None</a></p>
<p>Benthic diatoms are routinely used as ecological indicators in rivers. A standardized methodology is based on biofilm sampling, species identification, and counting under microscope. DNA-metabarcoding is an alternative methodology that can identify species and assess their proportion based on high-throughput DNA sequencing. Sequence data is analyzed with bioinformatics tools, and several strategies can be chosen. The strategy choice can affect communities composition and structure, and therefore the resulting ecological assessment. We wanted to optimize the bioinformatics strategy to obtain the closest results to microscopy. This was done in the framework of the Mothur pipeline. Here, 447 samples from French rivers were analyzed in the monitoring context of the European Water Framework Directive. Samples were analyzed both with DNA metabarcoding and microscopy. A usual bioinformatics strategy in Mothur includes clustering DNA-sequences into Operational Taxonomic Units (OTUs). Different algorithms exist for this. From a subsample of 142 samples, we showed that some strategies (Furthest neighbor) gave closer results to microscopy than others (Opticlust) in terms of community structure and diatom index values. However, we showed that OTU clustering was not necessary for ecological monitoring: Direct taxonomic assignment of individual sequence units (ISU) gave similar results to those obtained in microscopy. Interestingly, direct assignment enabled the detection of more species 2 to 3 times faster in terms of computation time compared to the OTU strategy. However, it remained important to remove low quality and chimeric sequences; if not, biomonitoring results differed greatly from microscopy. We showed that it was preferable to have a loose taxonomical identification threshold instead of a stringent one. This allowed detecting more species, which could participate in the index calculation and increased its performance. Indeed, in diatoms, phylogenetically neighbor species often have similar ecologies, and this explains why it is preferable, in a biomonitoring framework, to identify more species with less stringency instead of identifying few species with stringency. Finally, the best strategy (direct assignment of filtered ISU with a loose taxonomical threshold of 60%) was applied to the 447 samples covering a large diversity of ecological qualities. These data were then used to produce quality index values, using a quantification correction factor taking into account species biovolumes. Compared to microscopy, the DNA-based method assigned the same quality class for 66% of the samples, and 72% of the samples had an index value (ranging from 0 to 20) with less than one point difference from microscopy.</p><h1>Advances and prospects of environmental DNA in neotropical rainforests</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2020.01.001">doi:10.1016/bs.aecr.2020.01.001</a></p>

<p><a href=None>None</a></p>
<p>The rainforests of the Neotropics shelter a vast diversity of plant, animal and microscopic species that provide critical ecosystem goods and services for both local and worldwide populations. These environments face a major crisis due to increased deforestation, pollution, and climate change, emphasizing the need for more effective conservation efforts. The adequate monitoring of these ecosystems has proven a complex and time consuming endeavour, which depends on ever dwindling taxonomic expertise. To date, many species remain undiscovered, let alone described, with otherwise limited information regarding known species population distributions and densities. Overcoming these knowledge shortfalls and practical limitations is becoming increasingly possible through techniques based on environmental DNA (eDNA), i.e., DNA that can be obtained from environmental samples (e.g. tissues, soil, sediment, water, etc.). When coupled with high-throughput sequencing, these techniques now enable realistic, cost-effective, and standardisable biodiversity assessments. This opens up enormous opportunities for advancing our understanding of complex and species-rich tropical communities, but also in facilitating large-scale biomonitoring programs in the neotropics. In this review, we provide a brief introduction to eDNA methods, and an overview of their current and potential uses in both terrestrial and aquatic ecosystems of neotropical rainforests. We also discuss the limits and challenges of these methods for our understanding and monitoring of biodiversity, as well as future research and applied perspectives of these techniques in neotropical rainforests, and beyond.</p><h1>Studying Ecosystems With DNA Metabarcoding: Lessons From Biomonitoring of Aquatic Macroinvertebrates</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00434">doi:10.3389/fevo.2019.00434</a></p>

<p><a href=None>None</a></p>
<p>An ongoing challenge for ecological studies has been the collection of data with high precision and accuracy at a suitable scale to detect and manage critical global change processes. A major hurdle has been the time-consuming and challenging process of sorting and identification of organisms, but the rapid development of DNA metabarcoding as a biodiversity observation tool provides a potential solution. As high-throughput sequencing becomes more rapid and cost-effective, a “big data” revolution is anticipated, based on higher and more accurate taxonomic resolution, more efficient detection, and greater sample processing capacity. These advances have the potential to amplify the power of ecological studies to detect change and diagnose its cause, through a methodology termed “Biomonitoring 2.0.” Despite its promise, the unfamiliar terminology and pace of development in high-throughput sequencing technologies has contributed to a growing concern that an unproven technology is supplanting tried and tested approaches, lowering trust among potential users, and reducing uptake by ecologists and environmental management practitioners. While it is reasonable to exercise caution, we argue that any criticism of new methods must also acknowledge the shortcomings and lower capacity of current observation methods. Broader understanding of the statistical properties of metabarcoding data will help ecologists to design, test and review evidence for new hypotheses. We highlight the uncertainties and challenges underlying DNA metabarcoding and traditional methods for compositional analysis, specifically comparing the interpretation of otherwise identical bulk-community samples of freshwater benthic invertebrates. We explore how taxonomic resolution, sample similarity, taxon misidentification, and taxon abundance affect the statistical properties of these samples, but recognize these issues are relevant to applications across all ecosystem types. In conclusion, metabarcoding has the capacity to improve the quality and utility of ecological data, and consequently the quality of new research and efficacy of management responses.</p><h1>Diatom DNA Metabarcoding for Biomonitoring: Strategies to Avoid Major Taxonomical and Bioinformatical Biases Limiting Molecular Indices Capacities</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00409">doi:10.3389/fevo.2019.00409</a></p>

<p><a href=None>None</a></p>
<p>Recent years provided intense progression in the implementation of molecular techniques in a wide variety of research fields in ecology. Biomonitoring and bioassessment can greatly benefit from DNA metabarcoding and High-Throughput Sequencing (HTS) methods that potentially provide reliable, high quantity and quality standardized data in a cost- and time-efficient way. However, DNA metabarcoding has its drawbacks, introducing biases at all the steps of the process, particularly during bioinformatics treatments used to prepare HTS data for ecological analyses. The high diversity of bioinformatics methods (e.g., OTU clustering, chimera detection, taxonomic assignment) and parameters (e.g., percentage similarity threshold used to define OTUs) make inter-studies comparison difficult, limiting the development of standardized and easy-accessible bioassessment procedures for routine freshwater monitoring. In order to study and overcome these drawbacks, we constructed four de novo indices to assess river ecological status based on the same biological samples of diatoms analyzed with morphological and molecular methods. The biological inventories produced are (i) morphospecies identified by microscopy, (ii) OTUs provided via metabarcoding and hierarchical clustering of sequences using a 95% similarity threshold, (iii) individual sequence units (ISUs) via metabarcoding and only minimal bioinformatical quality filtering, and (iv) exact sequence variants (ESVs) using DADA2 denoising algorithm. The indices based on molecular data operated directly with ecological values estimated for OTUs/ ISUs/ ESVs. Our study used an approach of bypassing taxonomic assignment, so bias related to unclassified sequences missing from reference libraries could be handled and no information on ecology of sequences is lost. Additionally, we showed that the indices based on ISUs and ESVs were equivalent, outperforming the OTU-based one in terms of predictive power and accuracy by revealing the hidden ecological information of sequences that are otherwise clustered in the same OTU (intra-species/intra-population variability). Furthermore, ISUs, ESVs, and morphospecies indices provided similar estimation of site ecological status, validating that ISUs with limited bioinformatics treatments may be used for DNA freshwater monitoring. Our study is a proof of concept where taxonomy- and clustering-free approach is presented, that we believe is a step forward a standardized and comparable DNA bioassessment, complementary to morphological methods.</p><h1>Gaps in DNA-Based Biomonitoring Across the Globe</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00337">doi:10.3389/fevo.2019.00337</a></p>

<p><a href=None>None</a></p>
<p>DNA-based methodology has proven to be a vital tool for ecosystem assessment and monitoring. Increasingly, high-throughput approaches such as DNA metabarcoding are being used to address more complex questions, including ecological network analyses through machine learning. Despite the technological advances which allow for such questions to be posed, there remains inherent limitations in studies utilizing DNA metabarcoding, referring to environmental sample type targeted, geographical coverage and lack of standardized field and laboratory procedures. Additionally, DNA reference databases are lacking information from taxa, resulting in unidentified sequences, and underrepresentation of some taxa. These issues need to be addressed to enable a more representative approach to ecosystem monitoring to allow for detection and monitoring of global ecosystem change.</p><h1>A review on the applications and recent advances in environmental DNA (eDNA) metagenomics</h1>
<p><a href="https://doi.org/10.1007/s11157-019-09501-4">doi:10.1007/s11157-019-09501-4</a></p>

<p><a href=None>None</a></p>
<p>Progressions accomplished in the field of molecular biology amid the most recent decade has changed the way in which we contemplate biodiversity on earth. Together with the accessibility of new molecular techniques like high-throughput sequencing (HTS), a prompt metamorphosis was attained in understanding the behaviour and biodiversity patterns of biological systems at a level never before possible. The DNA gathered from different environmental samples (named as environmental DNA or eDNA) when coupled with HTS offers a powerful tool by empowering the census of individual species on a global scale in real time. The applications of eDNA are transpiring in different domains, for example, trophic and community ecology (functional diversity, ecosystem dynamics and prey–predator interactions), biomonitoring, conservation biology (single and multi species detection, abundance estimates), invasion biology (early species detection, passive surveillance) and environmental assessment (detection of anthropogenic contamination, microbial source tracking). However, more empirical data is required to standardize the specific sampling procedures to achieve in the best possible way. Although the application of eDNA is intensifying swiftly at a global scale, there are still some knowledge gaps, especially with methods and applications. These procedures require some refinements and validations to diminish the burden of false positives/negatives. Considering these impediments, we mainly concentrated in pooling together the most recent outcome of research articles (2008–2019) available in eDNA analysis that converse about diverse ecosystems (freshwater, marine and terrestrial habitats). We likewise discussed developments and limitations that are generally concerned with eDNA exercise in the present review.</p><h1>Anacapa Toolkit: An environmental DNA toolkit for processing multilocus metabarcode datasets</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13214">doi:10.1111/2041-210X.13214</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is a promising method to monitor species and community diversity that is rapid, affordable and non-invasive. The longstanding needs of the eDNA community are modular informatics tools, comprehensive and customizable reference databases, flexibility across high-throughput sequencing platforms, fast multilocus metabarcode processing and accurate taxonomic assignment. Improvements in bioinformatics tools make addressing each of these demands within a single toolkit a reality. The new modular metabarcode sequence toolkit Anacapa (https://github.com/limey-bean/Anacapa/) addresses the above needs, allowing users to build comprehensive reference databases and assign taxonomy to raw multilocus metabarcode sequence data. A novel aspect of Anacapa is its database building module, “Creating Reference libraries Using eXisting tools” (CRUX), which generates comprehensive reference databases for specific user-defined metabarcoding loci. The Quality Control and ASV Parsing module sorts and processes multiple metabarcoding loci and processes merged, unmerged and unpaired reads maximizing recovered diversity. DADA2 then detects amplicon sequence variants (ASVs) and the Anacapa Classifier module aligns these ASVs to CRUX-generated reference databases using Bowtie2. Lastly, taxonomy is assigned to ASVs with confidence scores using a Bayesian Lowest Common Ancestor (BLCA) method. The Anacapa Toolkit also includes an r package, ranacapa, for automated results exploration through standard biodiversity statistical analysis. Benchmarking tests verify that the Anacapa Toolkit effectively and efficiently generates comprehensive reference databases that capture taxonomic diversity, and can assign taxonomy to both MiSeq and HiSeq-length sequence data. We demonstrate the value of the Anacapa Toolkit in assigning taxonomy to seawater eDNA samples collected in southern California. The Anacapa Toolkit improves the functionality of eDNA and streamlines biodiversity assessment and management by generating metabarcode specific databases, processing multilocus data, retaining a larger proportion of sequencing reads and expanding non-traditional eDNA targets. All the components of the Anacapa Toolkit are open and available in a virtual container to ease installation.</p><h1>Scaling up DNA metabarcoding for freshwater macrozoobenthos monitoring</h1>
<p><a href="https://doi.org/10.1111/fwb.13220">doi:10.1111/fwb.13220</a></p>

<p><a href=None>None</a></p>
<p>The viability of DNA metabarcoding for assessment of freshwater macrozoobenthos has been demonstrated over the recent years. The method has matured to a stage where it can be applied to monitoring at a large scale, keeping pace with increased high-throughput sequencing capacity. However, workflows and sample tagging need to be optimised to accommodate for hundreds of samples within a single sequencing run. Here, we conceptualise a streamlined metabarcoding workflow, in which samples are processed in 96-well plates. Each sample is replicated starting with tissue extraction. Negative and positive controls are included to ensure data reliability. With our newly developed fusion primer sets for the BF2 + BR2 primer pair up to three 96-well plates (288 wells) can be uniquely tagged for a single Illumina sequencing run. By including Illumina indices, tagging can be extended to thousands of samples. We hope that our metabarcoding workflow will be used as a practical guide for future large-scale biodiversity assessments involving freshwater invertebrates. However, as this is just one possible metabarcoding approach, we hope this article will stimulate discussion and publication of alternatives and extensions to this method.</p><h1>Environmental DNA (eDNA)-metabarcoding-based early monitoring and warning for invasive species in aquatic ecosystems</h1>
<p><a href="https://doi.org/10.17520/biods.2018233">doi:10.17520/biods.2018233</a></p>

<p><a href=None>None</a></p>
<p>Biological invasion is a major threat to multiple ecosystems across the globe, causing severe damages to ecological integrity, loss of biodiversity, economic and social development and even human health. With the rapid development in aquaculture, shipping and aquarium and ornamental trades in the past several decades, China has become one of the countries most influenced by invasive species. Studies have clearly shown that the development and application of robust early monitoring and warning is one of the most effective ways to prevent and possibly control invasive species in aquatic ecosystems. Compared to terrestrial ecosystems, there remain several technical difficulties for developing early monitoring and warning in aquatic habitats. The technical challenges are mainly due to several features of aquatic biological communities such as high biodiversity and complex structure, a large number of microscopic species, extremely low population density and lack of available taxonomic keys for species identification. With the rapid development of high-throughput sequencing techniques, environmental DNA (eDNA)-metabarcoding has become the top priority method for developing the early monitoring and warning programs in aquatic ecosystems. In this review, we aim to synthesize the research progress on eDNA-metabarcoding and its application to early monitoring and warning of invasive species in aquatic ecosystems. In addition, we briefly discuss the technological advantages of eDNA-metabarcoding for the early monitoring and warning programs. Finally, we propose research perspectives for solutions to technical issues for false positive and false negative errors in the eDNA-metabarcoding process.</p><h1>DNA barcoding and emerging reference construction and data analysis technologies</h1>
<p><a href="https://doi.org/10.17520/biods.201820">doi:10.17520/biods.201820</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding has been growing exponentially in terms of the number of barcode generated as well as its applications, e.g. as conservation tools in: species identification for damaged specimens, diet analysis from gut content and feces, biodiversity assessment from environmental DNA (eDNA), bulk arthropod samples or invertebrate-derived DNA (iDNA). These applications often require coupling with high throughput sequencing (HTS) technologies, and when done so are referred to as metabarcoding. Here, we discuss the methods used to generate reference barcodes using cost-efficient HTS platforms, and introduce several rules-of-thumb and some widely-used tools to conduct data quality control, denoising, and Operational Taxonomic Units (OTUs) clustering. We hope this review will help readers better understand how these emerging technologies can be implemented alongside existing technologies to accelerate biodiversity assessments in an accurate and efficient way.</p><h1>Biodiversity assessments in the 21st century: The potential of insect traps to complement environmental samples for estimating eukaryotic and prokaryotic diversity using high-throughput DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1139/gen-2018-0096">doi:10.1139/gen-2018-0096</a></p>

<p><a href=None>None</a></p>
<p>The rapid loss of biodiversity, coupled with difficulties in species identification, call for innovative approaches to assess biodiversity. Insects make up a substantial proportion of extant diversity and play fundamental roles in any given ecosystem. To complement morphological species identification, new techniques such as metabarcoding make it possible to quantify insect diversity and insect-ecosystem interactions through DNA sequencing. Here we examine the potential of bulk insect samples (i.e., containing many non-sorted specimens) to assess prokaryote and eukaryote biodiversity and to complement the taxonomic coverage of soil samples. We sampled 25 sites on three continents and in various ecosystems, collecting insects with SLAM traps (Brazil) and Malaise traps (South Africa and Sweden). We then compared our diversity estimates with the results obtained with biodiversity data from soil samples from the same localities. We found a largely different taxonomic composition between the soil and insect samples, testifying to the potential of bulk insect samples to complement soil samples. Finally, we found that non-destructive DNA extraction protocols, which preserve insect specimens for morphological studies, constitute a promising choice for cost-effective biodiversity assessments. We propose that the sampling and sequencing of insect samples should become a standard complement for biodiversity studies based on environmental DNA.</p><h1>Needle in a haystack? A comparison of eDNA metabarcoding and targeted qPCR for detection of the great crested newt (Triturus cristatus)</h1>
<p><a href="https://doi.org/10.1002/ece3.4013">doi:10.1002/ece3.4013</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis is a rapid, cost-effective, non-invasive biodiversity monitoring tool which utilises DNA left behind in the environment by organisms for species detection. The method is used as a species-specific survey tool for rare or invasive species across a broad range of ecosystems. Recently, eDNA and “metabarcoding” have been combined to describe whole communities rather than focusing on single target species. However, whether metabarcoding is as sensitive as targeted approaches for rare species detection remains to be evaluated. The great crested newt Triturus cristatus is a flagship pond species of international conservation concern and the first UK species to be routinely monitored using eDNA. We evaluate whether eDNA metabarcoding has comparable sensitivity to targeted real-time quantitative PCR (qPCR) for T. cristatus detection. Extracted eDNA samples (N = 532) were screened for T. cristatus by qPCR and analysed for all vertebrate species using high-throughput sequencing technology. With qPCR and a detection threshold of 1 of 12 positive qPCR replicates, newts were detected in 50% of ponds. Detection decreased to 32% when the threshold was increased to 4 of 12 positive qPCR replicates. With metabarcoding, newts were detected in 34% of ponds without a detection threshold, and in 28% of ponds when a threshold (0.028%) was applied. Therefore, qPCR provided greater detection than metabarcoding but metabarcoding detection with no threshold was equivalent to qPCR with a stringent detection threshold. The proportion of T. cristatus sequences in each sample was positively associated with the number of positive qPCR replicates (qPCR score) suggesting eDNA metabarcoding may be indicative of eDNA concentration. eDNA metabarcoding holds enormous potential for holistic biodiversity assessment and routine freshwater monitoring. We advocate this community approach to freshwater monitoring to guide management and conservation, whereby entire communities can be initially surveyed to best inform use of funding and time for species-specific surveys.</p><h1>The need for standardisation: Exemplified by a description of the diversity, community structure and ecological indices of soil nematodes</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2017.12.002">doi:10.1016/j.ecolind.2017.12.002</a></p>

<p><a href=None>None</a></p>
<p>Molecular approaches are offering a supplement to, or even the possibility of replacing morphological identification of soil fauna, because of advantages for throughput, coverage and objectivity. We determined ecological indices of nematode community data from four sets of duplicate soil cores, based on morphological identification of nematodes after elutriation from 200 g soil and high throughput sequencing (HTS) targeting nematodes both after being elutriated from soils and DNA extracted directly from 10 g soil. HTS (at genus and species level) increased the taxonomic resolution compared to morphology (at family level). DNA extracted from elutriated nematodes identified more nematode taxa than when extracted from soil, due to an enrichment in nematode sequences. Each method also gave a different ecological footprint for the nematode community. Standardisation to previously determined indices based on morphological identification is needed in order to provide more meaningful information about soil quality and for ecological monitoring.</p><h1>High-throughput techniques as support for knowledge-based spatial conservation prioritization in mangrove ecosystems</h1>
<p><a href="https://doi.org/10.1007/978-3-319-73016-5_24">doi:10.1007/978-3-319-73016-5_24</a></p>

<p><a href=None>None</a></p>
<p>The conservation of functioning ecosystems worldwide is warranted by the need for reliable and sustainable provision of ecosystem services locally, regionally and globally. Mangroves provide numerous ecosystem services both to local human communities, e.g., coastal protection or food security, and to mankind worldwide, e.g., climate change-mitigation. Nonetheless they still lack protection in many places of occurrence. Here we base spatial prioritization and planning of mangrove conservation on functional biodiversity and service-relevant ecosystem processes, being studied through cutting-edge genetic and chemical analyses of sediments to unravel the links between biodiversity, biotic interactions, ecosystem processes and ecosystem services. We nonetheless recommend multidisciplinary approaches when planning protected area networks for the sustainable use and provision of ecosystem services and pledge for (i) considering and prioritizing societal, biological and economic values of mangroves, (ii) integrating adjacent ecosystems to maintain connectivity, and (iii) taking into account the spatial and temporal dynamics of mangrove ecosystems and their community composition under global change, i.e. changes in the spatial distribution of species and services over time. Beyond the example of mangroves and the turnover of organic matter in mangrove sediments described herein, our approach to spatial conservation prioritization and planning is applicable to any other ecosystems and their services.</p><h1>Molecular approaches for an operational marine biodiversity observation network</h1>
<p><a href="https://doi.org/10.1016/B978-0-12-805052-1.00032-2">doi:10.1016/B978-0-12-805052-1.00032-2</a></p>

<p><a href=None>None</a></p>
<p>Biological health underpins ecosystem services, including the production of food from the sea. Therefore, many policy frameworks rely on biological observations to guide the sustainable use of marine resources. For example, a marine biodiversity observation network can provide evidence of ecosystem degradation, remediation success, and progress toward goals of sustainable development. The utility of biodiversity observations is limited, however, if measurements are not standardized to allow comparisons across space and time or if they are too difficult or expensive to make routinely or at large scale. Molecular approaches (e.g., metagenomics, metabarcoding, eDNA, metatranscriptomics) offer efficiencies in sample collection and processing and provide analysis across trophic levels compared to traditional methods. Increased affordability and throughput of nucleic acid sequencing allow integration of molecular biological techniques into operational monitoring with the potential benefit of expanding the spatial and temporal scale of biological observations.</p><h1>Linking DNA Metabarcoding and Text Mining to Create Network-Based Biomonitoring Tools: A Case Study on Boreal Wetland Macroinvertebrate Communities</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2018.09.001">doi:10.1016/bs.aecr.2018.09.001</a></p>

<p><a href=None>None</a></p>
<p>Ecological networks are powerful tools for visualizing biodiversity data and assessing ecosystem health and function. Constructing these networks requires considerable empirical efforts, and this remains highly challenging due to sampling limitations and the laborious and notoriously limited, error-prone process of traditional taxonomic identification. Recent advancements in high-throughput gene sequencing and high-performance computing provide new ways to address these challenges. DNA metabarcoding, a method of bulk taxonomic identification from DNA extracted from environmental samples, can generate detailed biodiversity information through a standardizable analytical pipeline for species detection. When this biodiversity information is annotated with prior knowledge on taxon interactions, body size, and trophic position, it is possible to generate trait-based networks, which we call “heuristic food webs”. Although curating trait matrices for constructing heuristic food webs is a laborious, often intractable process using manual literature surveys, it can be greatly accelerated via text mining, allowing knowledge of relevant traits to be gathered across large databases. To explore this possibility, we employed a General Architecture for Text Engineering (GATE) system to create a hybrid text-mining pipeline combining rule-based and machine-learning modules. This pipeline was then used to query online repositories of published papers for missing data on a key trait, body size, that could not be gathered from existing trophic link libraries of freshwater benthic macroinvertebrates. Combining text-mined body size information with feeding information from existing sources allowed us to generate a database of over 20,000 pairwise trophic interactions. Next, we developed a pipeline that uses taxa lists generated from DNA metabarcoding and annotates this matrix with trophic information from existing databases and text-mined body size data. In this way, we generated heuristic food webs for wetland sites within a large delta complex formed by the confluence of the Peace and Athabasca rivers in northern Alberta: the Peace–Athabasca delta. Finally, we used these putative food webs and their network properties to resolve spatial and temporal differences between the benthic subwebs of wetlands in the Peace and Athabasca sectors of the delta complex. Specifically, we asked two questions. (1) How do food web properties (e.g. number of links, linkage density, trophic height) differ between the wetlands of the Peace and Athabasca deltas? (2) How do food web properties change temporally in wetlands of the two deltas? We discuss using DNA-generated, trait-based food webs as a powerful tool for rapid bioassessment, assess the limitations of our current approach, and outline a path forward to make this powerful tool more widely available for land managers and conservation biologists.</p><h1>Why We Need Sustainable Networks Bridging Countries, Disciplines, Cultures and Generations for Aquatic Biomonitoring 2.0: A Perspective Derived From the DNAqua-Net COST Action</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2018.01.001">doi:10.1016/bs.aecr.2018.01.001</a></p>

<p><a href=None>None</a></p>
<p>Aquatic biomonitoring has become an essential task in Europe and many other regions as a consequence of strong anthropogenic pressures affecting the health of lakes, rivers, oceans and groundwater. A typical assessment of the environmental quality status, such as it is required by European but also North American and other legislation, relies on matching the composition of assemblages of organisms identified using morphological criteria present in aquatic ecosystems to those expected in the absence of anthropogenic pressures. Through decade-long and difficult intercalibration exercises among networks of regulators and scientists in European countries, a pragmatic biomonitoring approach was developed and adopted, which now produces invaluable information. Nonetheless, this approach is based on several hundred different protocols, making it susceptible to issues with comparability, scale and resolution. Furthermore, data acquisition is often slow due to a lack of taxonomic experts for many taxa and regions and time-consuming morphological identification of organisms. High-throughput genetic screening methods such as (e)DNA metabarcoding have been proposed as a possible solution to these shortcomings. Such “next-generation biomonitoring”, also termed “biomonitoring 2.0”, has many advantages over the traditional approach in terms of speed, comparability and costs. It also creates the potential to include new bioindicators and thereby further improves the assessment of aquatic ecosystem health. However, several major conceptual and technological challenges still hinder its implementation into legal and regulatory frameworks. Academic scientists sometimes tend to overlook legal or socioeconomic constraints, which regulators have to consider on a regular basis. Moreover, quantification of species abundance or biomass remains a significant bottleneck to releasing the full potential of these approaches. Here, we highlight the main challenges for next-generation aquatic biomonitoring and outline principles and good practices to address these with an emphasis on bridging traditional disciplinary boundaries between academics, regulators, stakeholders and industry.</p><h1>Environmental DNA: For biodiversity research and monitoring</h1>
<p><a href="https://doi.org/10.1093/oso/9780198767220.001.0001">doi:10.1093/oso/9780198767220.001.0001</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA), i.e. DNA released in the environment by any living form, represents a formidable opportunity to gather high-throughput and standard information on the distribution or feeding habits of species. It has therefore great potential for applications in ecology and biodiversity management. However, this research field is fast-moving, involves different areas of expertise and currently lacks standard approaches, which calls for an up-to-date and comprehensive synthesis. Environmental DNA for biodiversity research and monitoring covers current methods based on eDNA, with a particular focus on “eDNA metabarcoding”. Intended for scientists and managers, it provides the background information to allow the design of sound experiments. It revisits all steps necessary to produce high-quality metabarcoding data such as sampling, metabarcode design, optimization of PCR and sequencing protocols, as well as analysis of large sequencing datasets. All these different steps are presented by discussing the potential and current challenges of eDNA-based approaches to infer parameters on biodiversity or ecological processes. The last chapters of this book review how DNA metabarcoding has been used so far to unravel novel patterns of diversity in space and time, to detect particular species, and to answer new ecological questions in various ecosystems and for various organisms. Environmental DNA for biodiversity research and monitoring constitutes an essential reading for all graduate students, researchers and practitioners who do not have a strong background in molecular genetics and who are willing to use eDNA approaches in ecology and biomonitoring.</p><h1>Combining NCBI and BOLD databases for OTU assignment in metabarcoding and metagenomic datasets: The BOLD-NCBI-Merger</h1>
<p><a href="https://doi.org/10.3897/mbmg.1.22262">doi:10.3897/mbmg.1.22262</a></p>

<p><a href=None>None</a></p>
<p>Background Metabarcoding and metagenomic approaches are becoming routine techniquesfor use in biodiversity assessment and in ecological studies. The assignment of taxonomic information to millions of sequences obtained via high-throughput sequencing is challenging, as many DNA reference libraries are lacking information on certain taxonomic groups and can contain erroneous sequences. Combining different reference databases is therefore a promising approach for maximising taxonomic coverage and reliability of results. New information The "BOLD-NCBI-Merger" bash script is introduced, which combines sequence data obtained from the National Centre for Biotechnology Information (NCBI) GenBank and the Barcode of Life Database (BOLD) and prepares it for taxonomic assignment with the software MEGAN.</p><h1>Performance of DNA metabarcoding, standard barcoding, and morphological approach in the identification of hostparasitoid interactions</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0187803">doi:10.1371/journal.pone.0187803</a></p>

<p><a href=None>None</a></p>
<p>Understanding interactions between herbivores and parasitoids is essential for successful biodiversity protection and monitoring and for biological pest control. Morphological identifications employ insect rearing and are complicated by insects' high diversity and crypsis. DNA barcoding has been successfully used in studies of host-parasitoid interactions as it can substantially increase the recovered real host-parasitoid diversity distorted by overlooked species complexes, or by species with slight morphological differences. However, this approach does not allow the simultaneous detection and identification of host(s) and parasitoid(s). Recently, high-throughput sequencing has shown high potential for surveying ecological communities and trophic interactions. Using mock samples comprising insect larvae and their parasitoids, we tested the potential of DNA metabarcoding for identifying individuals involved in host-parasitoid interactions to different taxonomic levels, and compared it to standard DNA barcoding and morphological approaches. For DNA metabarcoding, we targeted the standard barcoding marker cytochrome oxidase subunit I using highly degenerate primers, 2∗300 bp sequencing on a MiSeq platform, and RTAX classification using paired-end reads. Additionally, using a large host-parasitoid dataset from a Central European floodplain forest, we assess the completeness and usability of a local reference library by confronting the number of Barcoding Index Numbers obtained by standard barcoding with the number of morphotypes. Overall, metabarcoding recovery was high, identifying 92.8% of the taxa present in mock samples, and identification success within individual taxonomic levels did not significantly differ among metabarcoding, standard barcoding, and morphology. Based on the current local reference library, 39.4% parasitoid and 90.7% host taxa were identified to the species level. DNA barcoding estimated higher parasitoid diversity than morphotyping, especially in groups with high level of crypsis. This study suggests the potential of metabarcoding for effectively recovering host-parasitoid diversity, together with more accurate identifications obtained from building reliable and comprehensive reference libraries, especially for parasitoids.</p><h1>Invasions Toolkit: Current Methods for Tracking the Spread and Impact of Invasive Species</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2016.10.009">doi:10.1016/bs.aecr.2016.10.009</a></p>

<p><a href=None>None</a></p>
<p>Biological invasions exert multiple pervasive effects on ecosystems, potentially disrupting species interactions and global ecological processes. Our ability to successfully predict and manage the ecosystem-level impacts of biological invasions is strongly dependent on our capacity to empirically characterize complex biological interactions and their spatiotemporal dynamics. In this chapter, we argue that the comprehensive integration of multiple complementary tools within the explicit context of ecological networks is essential for providing mechanistic insight into invasion processes and their impact across organizational levels. We provide an overview of traditional (stable isotopes, populations genetics) and emerging (metabarcoding, citizen science) techniques and methods, and their practical implementation in the context of biological invasions. We also present several currently available models and machine-learning approaches that could be used for predicting novel or undocumented interactions, thus allowing a more robust and cost-effective forecast of network and ecosystem stability. Finally, we discuss the importance of methodological advancements on the emergence of scientific and societal challenges for investigating local and global species histories with several skill sets.</p><h1>Assessing strengths and weaknesses of DNA metabarcoding-based macroinvertebrate identification for routine stream monitoring</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12789">doi:10.1111/2041-210X.12789</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding holds great promise for the assessment of macroinvertebrates in stream ecosystems. However, few large-scale studies have compared the performance of DNA metabarcoding with that of routine morphological identification. We performed metabarcoding using four primer sets on macroinvertebrate samples from 18 stream sites across Finland. The samples were collected in 2013 and identified based on morphology as part of a Finnish stream monitoring program. Specimens were morphologically classified, following standardised protocols, to the lowest taxonomic level for which identification was feasible in the routine national monitoring. DNA metabarcoding identified more than twice the number of taxa than the morphology-based protocol, and also yielded a higher taxonomic resolution. For each sample, we detected more taxa by metabarcoding than by the morphological method, and all four primer sets exhibited comparably good performance. Sequence read abundance and the number of specimens per taxon (a proxy for biomass) were significantly correlated in each sample, although the adjusted R2 values were low. With a few exceptions, the ecological status assessment metrics calculated from morphological and DNA metabarcoding datasets were similar. Given the recent reduction in sequencing costs, metabarcoding is currently approximately as expensive as morphology-based identification. Using samples obtained in the field, we demonstrated that DNA metabarcoding can achieve comparable assessment results to current protocols relying on morphological identification. Thus, metabarcoding represents a feasible and reliable method to identify macroinvertebrates in stream bioassessment, and offers powerful advantage over morphological identification in providing identification for taxonomic groups that are unfeasible to identify in routine protocols. To unlock the full potential of DNA metabarcoding for ecosystem assessment, however, it will be necessary to address key problems with current laboratory protocols and reference databases.</p><h1>Metabarcoding techniques for assessing biodiversity of marine animal forests</h1>
<p><a href="https://doi.org/10.1007/978-3-319-21012-4_53">doi:10.1007/978-3-319-21012-4_53</a></p>

<p><a href=None>None</a></p>
<p>The "marine animal forests" are among the most diverse ecosystems in the Biosphere. However, exhaustive biodiversity assessment of these communities has been so far elusive. The real extent of biodiversity and its temporal and spatial variability patterns remain unknown for most animal forests, mainly due to the inability of traditional taxonomy methods to cope with such degree of diversity and structural complexity. The development of metabarcoding techniques has revolutionized biomonitoring. Using this approach, thousands of species present in any environmental sample can be detected by high-throughput DNA sequencing and identified using public databases. Though initially limited to homogeneous substrates such as plankton or sediments, the applications of metabarcoding have been recently extended to communities on heterogeneous complex hard bottom substrates. Here we present novel metabarcoding protocols, based on the use of short fragments of 18S rRNA or cytochrome c oxidase I genes as genetic markers. We aim to develop methods for robust, reproducible eukaryotic biodiversity assessment of structurally complex communities such as marine animal forests, allowing characterization of communities living on hard-bottom substrates or other marine benthic ecosystems. We propose some guidelines focusing on sampling techniques, sample preprocessing, DNA extraction, selection of genetic markers, and bioinformatic pipelines, including steps such as sequence filtering (removal of low quality reads), clustering algorithms for delimiting molecular operational taxonomic units, and automated taxonomic assignment using reference databases. We expect these recommendations will help marine ecologists to become familiar with the paradigm shift that metabarcoding represents in the way marine ecosystems will be monitored and managed in the next future.</p><h1>Freshwater biomonitoring in the Information Age</h1>
<p><a href="https://doi.org/10.1002/fee.1490">doi:10.1002/fee.1490</a></p>

<p><a href=None>None</a></p>
<p>Freshwaters worldwide face serious threats, making their protection increasingly important. Freshwater monitoring has historically produced valuable data and continues to develop. Rapid improvements to biomolecular techniques are revolutionizing the way scientists describe biological communities and are bringing about major changes in biomonitoring. Combined with high-throughput sequencing, DNA metabarcoding is fast and cost-effective, generating massive amounts of data. In a world with numerous ecological threats, “big data” constitute a tremendous opportunity to improve the efficiency of biological monitoring. These fundamental changes in biomonitoring will require freshwater ecologists and environmental managers to reconsider how they handle large amounts of data.</p><h1>DNA sequencing as a tool to monitor marine ecological status</h1>
<p><a href="https://doi.org/10.3389/fmars.2017.00107">doi:10.3389/fmars.2017.00107</a></p>

<p><a href=None>None</a></p>
<p>Many ocean policies mandate integrated, ecosystem-based approaches to marine monitoring, driving a global need for efficient, low-cost bioindicators of marine ecological quality. Most traditional methods to assess biological quality rely on specialized expertise to provide visual identification of a limited set of specific taxonomic groups, a time-consuming process that can provide a narrow view of ecological status. In addition, microbial assemblages drive food webs but are not amenable to visual inspection and thus are largely excluded from detailed inventory. Molecular-based assessments of biodiversity and ecosystem function offer advantages over traditional methods and are increasingly being generated for a suite of taxa using a "microbes to mammals" or "barcodes to biomes" approach. Progress in these efforts coupled with continued improvements in high-throughput sequencing and bioinformatics pave the way for sequence data to be employed in formal integrated ecosystem evaluation, including food web assessments, as called for in the European Union Marine Strategy Framework Directive. DNA sequencing of bioindicators, both traditional (e.g., benthic macroinvertebrates, ichthyoplankton) and emerging (e.g., microbial assemblages, fish via eDNA), promises to improve assessment of marine biological quality by increasing the breadth, depth, and throughput of information and by reducing costs and reliance on specialized taxonomic expertise.</p><h1>Benchmarking DNA metabarcoding for biodiversity-based monitoring and assessment</h1>
<p><a href="https://doi.org/10.3389/fmars.2016.00096">doi:10.3389/fmars.2016.00096</a></p>

<p><a href=None>None</a></p>
<p>Characterization of biodiversity has been extensively used to confidently monitor and assess environmental status. Yet, visual morphology, traditionally and widely used for species identification in coastal and marine ecosystem communities, is tedious and entails limitations. Metabarcoding coupled with high-throughput sequencing (HTS) represents an alternative to rapidly, accurately, and cost-effectively analyze thousands of environmental samples simultaneously, and this method is increasingly used to characterize the metazoan taxonomic composition of a wide variety of environments. However, a comprehensive study benchmarking visual and metabarcoding-based taxonomic inferences that validates this technique for environmental monitoring is still lacking. Here, we compare taxonomic inferences of benthic macroinvertebrate samples of known taxonomic composition obtained using alternative metabarcoding protocols based on a combination of different DNA sources, barcodes of the mitochondrial cytochrome oxidase I gene and amplification conditions. Our results highlight the influence of the metabarcoding protocol in the obtained taxonomic composition and suggest the better performance of an alternative 313 bp length barcode to the traditionally 658 bp length one used for metazoan metabarcoding. Additionally, we show that a biotic index inferred from the list of macroinvertebrate taxa obtained using DNA-based taxonomic assignments is comparable to that inferred using morphological identification. Thus, our analyses prove metabarcoding valid for environmental status assessment and will contribute to accelerating the implementation of this technique to regular monitoring programs.</p><h1>Application of environmental DNA metabarcoding in ecology</h1>
<p><a href="https://doi.org/10.5846/stxb201501150125">doi:10.5846/stxb201501150125</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA metabarcoding is defined as high throughput DNA sequencing used for simultaneous molecular identification of multiple taxa in a complex environmental sample (i.e., soil, water, feces, etc.). In recent years, this method has aroused widespread concern among scientists and has been increasingly used in the field of ecology, such as biodiversity research, monitoring of aquatic, rare, or invasive species. This article describes the significance and the methods of environmental DNA metabarcoding, and highlights its application in the fields of species monitoring, detection of rare and invasive species, biodiversity, and diet analysis. Finally, this article summarizes the challenges in application of environmental DNA metabarcoding in ecological research and discusses the future prospects of its development.</p><h1>How will the 'molecular revolution' contribute to biological recording?</h1>
<p><a href="https://doi.org/10.1111/bij.12516">doi:10.1111/bij.12516</a></p>

<p><a href=None>None</a></p>
<p>Soaring throughput, plummeting costs, and increased sensitivity for assaying degraded or low-concentration DNA are driving a revolution in the way that we monitor biodiversity. Arguably the biggest 'game-changer' is environmental DNA (eDNA), which refers to free-floating DNA released by organisms into their environment. Rare or elusive species can be detected with greater sensitivity and accuracy using eDNA than by most conventional methods, and we have the capability to screen and describe whole communities, as well as perform targeted monitoring of single species. This paper discusses the basic approaches for molecular monitoring of biodiversity, provides case studies to demonstrate the effectiveness of the techniques, and considers any challenges and limitations that could impact molecular biological recording. It is argued that eDNA surveys offer exciting new opportunities to engage the public in biological recording and that molecular approaches will complement conventional surveys, enabling unprecedented insights into species distributions. Finally, with the number of eDNA studies increasing at a rapid pace, it is argued that there is a need to rapidly establish ways of managing molecular records. Integrating molecular records into existing biological records databases would enhance our understanding of species distributions and may be something that the Biological Records Centre should be considering to mark its landmark anniversary.</p><h1>Rare biosphere exploration using high-throughput sequencing: research progress and perspectives</h1>
<p><a href="https://doi.org/10.1007/s10592-014-0678-9">doi:10.1007/s10592-014-0678-9</a></p>

<p><a href=None>None</a></p>
<p>Identification of rare species and mapping their distributions is crucial for understanding natural species distributions and causes and consequences of accelerating species declines. However, detection of rare species in both terrestrial and especially aquatic communities typically dominated by numerous microscopic species (i.e. rare biosphere) represents a formidable technical challenge. Rapid advances in high-throughput sequencing (HTS) technologies have revolutionized biodiversity studies in the rare biosphere, and also stimulated associated debates. Here we summarize research progress, discuss debates and problems, and propose possible solutions and future studies to address these issues. In addition, we provide take-home messages for experimental design and data interpretation when utilizing HTS techniques for rare biosphere exploration in ecology and conservation biology.</p><h1>Next generation sequencing for characterizing biodiversity: promises and challenges</h1>
<p><a href="https://doi.org/10.1007/s10709-015-9816-7">doi:10.1007/s10709-015-9816-7</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding approaches are used to describe biodiversity by analysing specimens or environmental samples in taxonomic, phylogenetic and ecological studies. While sharing data among these disciplines would be highly valuable, this remains difficult because of contradictory requirements. The properties making a DNA barcode efficient for specimen identification or species delimitation are hardly reconcilable with those required for a powerful analysis of degraded DNA from environmental samples. The use of next generation sequencing methods open up the way towards the development of new markers (e.g., multilocus barcodes) that would overcome such limitations. However, several challenges should be taken up for coordinating actions at the interface between taxonomy, ecology, molecular biology and bioinformatics in order to develop methods and protocols compatible with both taxonomic and ecological studies.</p><h1>Metabarcoding approach for the ballast water surveillance - An advantageous solution or an awkward challenge?</h1>
<p><a href="https://doi.org/10.1016/j.marpolbul.2015.01.008">doi:10.1016/j.marpolbul.2015.01.008</a></p>

<p><a href=None>None</a></p>
<p>Transfer of organisms with ships' ballast water is recognized as a major pathway of non-indigenous species introduction and addressed in a few recent legislative initiatives. Among other they imply scientific and technical research and monitoring to be conducted in a efficient and reliable way. The recent development of DNA barcoding and metabarcoding technologies opens new opportunities for biodiversity and biosecurity surveillance. In the current study, the performance of metabarcoding approach was assessed in comparison to the conventional (visual) observations, during the en route experimental ballast water survey. Opportunities and limitations of the molecular method were identified from taxonomical datasets rendered by two molecular markers of different degree of universality - the universal cytochrome oxydase sub-unit I gene and a fragment of RuBisCO gene. The cost-efficacy and possible improvements of these methods are discussed for the further successful development and implementation of the approach in ballast water control and NIS surveillance.</p><h1>DNA (meta)barcoding of biological invasions: a powerful tool to elucidate invasion processes and help managing aliens</h1>
<p><a href="https://doi.org/10.1007/s10530-015-0854-y">doi:10.1007/s10530-015-0854-y</a></p>

<p><a href=None>None</a></p>
<p>Biological invasions are a major threat to the world’s biodiversity with consequences on ecosystem structure and functioning, species evolution, and human well-being (through ecosystem services). Conservation of biological diversity and management of biological resources require multi-level management strategies on non-native species, in order to (1) prevent biological introductions, (2) detect non-native species at an early stage of the introduction, and (3) eradicate or maintain at a low level of population density non-native species that were successfully introduced. A pre-requisite to any control measures on non-native species is the ability to rapidly and accurately identify the putative threatening alien species. DNA barcoding, and its recent extension, DNA metabarcoding are complementary tools that have proved their value in the identification of living beings. Here we review their use in the identification of non-native species at several steps of the introduction processes, and how they can be applied in the control and management of biological introductions. Through examples covering various taxa and ecosystems (terrestrial, freshwater, marine), we highlight the strengths and weaknesses of approaches that we foresee as crucial in the implementation of early warning strategies.</p><h1>Accounting for uncertainty in species delineation during the analysis of environmental DNA sequence data</h1>
<p><a href="https://doi.org/10.1111/j.2041-210X.2011.00122.x">doi:10.1111/j.2041-210X.2011.00122.x</a></p>

<p><a href=None>None</a></p>
<p>Defining species boundaries represents a significant challenge in biodiversity studies, especially as these studies increasingly rely on high-throughput DNA sequencing technologies. A promising approach for delineating species in environmental sequence data combines phylogenetics and coalescence theory to estimate species boundaries from distributions of lineage birth rates within multispecies coalescent trees. Existing methods for interpreting these models utilize hypothetico-deductive reasoning to identify thresholds associated with a mixed speciation-coalescent model that fits the data better than a null model. Here, I describe an alternative approach that ranks and assigns weights to models based on their fit to the data using information criteria and then uses model averaging to estimate parameters and species probabilities. This approach is applied to data from two independent studies that address (i) patterns of cospeciation in an aphid-bacterial symbiosis and (ii) diversity of bacterial communities associated with the human gut. In both of these cases, accounting for uncertainty during model selection allowed greater flexibility to detect variable (with respect to time) speciation-coalescent thresholds among lineages. The precision of the predicted species boundaries varied among the studies, and the variance-to-mean ratio for richness estimates ranged from 0.023 to 0.079. Sample-based estimates of gut bacteria richness revealed that accounting for uncertainty during species delineation increased the variance in the estimates of population means (by individual from which the samples were taken or by sex of the individuals) by up to 7.5%. In ecological and evolutionary studies, conclusions are highly dependent on the classification system that is adopted; given the uncertainty in species boundaries observed here, ignoring this source of error (as is common practice) likely results in inflated type I error rates. The approach described here represents an objective, theory-based method for predicting species boundaries and explicitly incorporates uncertainty in the classification system into biodiversity estimation, thus allowing researchers to better address the causes and consequences of biodiversity. © 2011 The Author. Methods in Ecology and Evolution © 2011 British Ecological Society.</p>