<h1>A method for the reintroduction of entire benthic invertebrate communities in formerly degraded streams</h1>
<p><a href="https://doi.org/10.1016/j.limno.2019.125689">doi:10.1016/j.limno.2019.125689</a></p>

<p><a href=None>None</a></p>
<p>Reintroduction is a commonly used tool in wildlife conservation. Although freshwater ecosystems are among the most threatened worldwide, reintroductions of entire benthic invertebrate communities have never been reported before. Here, for the first time, we developed a comprehensive method for the reintroduction of entire communities in formerly degraded streams that describes the selection of donor and recipient streams, the transfer of specimens, mortality rates, and a monitoring scheme. The stream selection procedure of the new method is based on widely available environmental variables and richness metrics that are regularly monitored by water authorities to fulfill the requirements of water policy frameworks (e.g. EU WFD). The sampling and transfer of specimens is also based on common and simply applicable methods (i.e. kick sampling). For each recipient stream, three donor streams were selected with a complementary set of reference species missing at the recipient stream. The sampling at each donor stream was done six times a year to cover different seasons and life stages. We applied our method to one stream (10â¯km2) and one river (1608â¯km2) in central Germany. In total, we transferred 325,197 and 401,233 specimens from three different donor streams, including 45 (28,682 specimens) and 50 (47,716 specimens) missing reference taxa into each recipient stream and river, respectively. Average mortality rates varied between 0.97% for all specimens and 1.72% for reference specimens. We recommend a post-reintroduction monitoring that includes new methods, i.e. environmental DNA and high throughput sequencing. Our proposed reintroduction method is based on commonly used variables, metrics and sampling techniques. Accordingly, this method requires only minor modifications to become applicable in other countries. The simultaneous reintroduction of a large number of reference taxa will increase the likelihood to reestablish near-natural benthic invertebrate communities in formerly degraded streams. This may increase the public and political acceptance of costly river restoration projects that so far, do not often reach a good ecological status even after decades, due to a large-scale depletion of potential source populations. Our method will help in the reintroduction of reference species in restored streams where natural recolonization is not possible.</p><h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy)</h1>
<p><a href="https://doi.org/10.1038/s41598-020-79738-5">doi:10.1038/s41598-020-79738-5</a></p>

<p><a href=None>None</a></p>
<p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p><h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden</h1>
<p><a href="https://doi.org/10.7717/peerj.8424">doi:10.7717/peerj.8424</a></p>

<p><a href=None>None</a></p>
<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p><h1>Biodiversity between sand grains: Meiofauna composition across southern and western Sweden assessed by metabarcoding</h1>
<p><a href="https://doi.org/10.3897/BDJ.8.e51813">doi:10.3897/BDJ.8.e51813</a></p>

<p><a href=None>None</a></p>
<p>The meiofauna is an important part of the marine ecosystem, but its composition and distribution patterns are relatively unexplored. Here we assessed the biodiversity and community structure of meiofauna from five locations on the Swedish western and southern coasts using a high-throughput DNA sequencing (metabarcoding) approach. The mitochondrial cytochrome oxidase 1 (COI) mini-barcode and nuclear 18S small ribosomal subunit (18S) V1-V2 region were amplified and sequenced using Illumina MiSeq technology. Our analyses revealed a higher number of species than previously found in other areas: thirteen samples comprising 6.5 dm3 sediment revealed 708 COI and 1,639 18S metazoan OTUs. Across all sites, the majority of the metazoan biodiversity was assigned to <i>Arthropoda</i>, <i>Nematoda</i> and <i>Platyhelminthes</i>. Alpha and beta diversity measurements showed that community composition differed significantly amongst sites. OTUs initially assigned to <i>Acoela</i>, <i>Gastrotricha</i> and the two <i>Platyhelminthes</i> sub-groups <i>Macrostomorpha</i> and <i>Rhabdocoela</i> were further investigated and assigned to species using a phylogeny-based taxonomy approach. Our results demonstrate that there is great potential for discovery of new meiofauna species even in some of the most extensively studied locations.</p><h1>Debugging metabarcoding for insect biodiversity studies</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Insects are one of the most abundant and diverse animal groups, and they include many valuable ecological indicator species, but taxonomic discovery projects and biodiversity surveys targeting this group are often challenging. While mass trapping devices allow the collection of insects in great numbers, the task of identifying the species present is a painstaking and resource-demanding process. Metabarcoding, that is, high throughput sequencing of PCR-amplified species-specific genetic markers in environmental samples, promises to solve this problem. However, metabarcoding is still in its infancy. In this thesis, I optimized metabarcoding methods for inventorying and accelerating species discovery of terrestrial insects. In <b>paper I,</b> we designed new PCR primers for mitochondrial markers and evaluated them against existing ones using <i>in silico</i> methods. We showed that the best marker for metabarcoding of insects is 16S because of its broad taxonomic coverage and low amplification bias. However, there is significantly more reference data for COI, and its taxonomic coverage is reasonable when using sufficiently degenerate primers (mixes of primer sequences). In <b>paper II,</b> we applied 16S and COI metabarcoding to different types of samples of the same insect communities: Malaise trap samples (preservative ethanol or homogenized samples) and soil samples. The results show that the two-marker strategy increases biodiversity detection over single-marker analyses. They also show that 16S is better than COI for metabarcoding of eDNA samples because the less degenerate 16S primers do not amplify as many off-target organisms. Finally, the results show that analyses of tissue homogenate and preservative ethanol yield strikingly different results. Large and heavily sclerotized insects do not leak DNA into preservative ethanol like small and weakly sclerotized ones do, but their DNA tends to swamp the DNA of the latter in homogenized samples.  In <b>paper III</b> we evaluated the performance of various non-destructive mild lysis treatments and DNA purification methods. We subjected mock community samples to incubation in either a milder or a more aggressive digestion buffer for a short or a long period of incubation. The DNA was then extracted using either a manual or an automated purification protocol. We found that the milder digestion buffer and the shorter incubation time preserved the morphology of the insect best while at the same time giving the most accurate DNA metabarcoding results; the purification protocol had little or no effect on metabarcoding results. Finally, in <b>paper IV,</b> we explored the received wisdom that high concentrations of ethanol, although optimal for preservation of the DNA, make insects fragile and difficult to work with from a morphological point of view. We preserved insects in different ethanol concentrations and subjected them to damaging processes, such as shaking or transporting. We verified that high concentrations of ethanol induce brittleness, although the effect is less pronounced in robust insects. Our results also indicate that shipping by mail is safe for samples preserved at intermediate concentrations (70 or 80 %). In summary, this thesis represents a significant step forward in the development of methods for preserving and analyzing samples of terrestrial insects for biodiversity surveys, monitoring programs, and taxonomic research projects.</p><h1>Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea</h1>
<p><a href="https://doi.org/10.3389/fmicb.2016.00679">doi:10.3389/fmicb.2016.00679</a></p>

<p><a href=None>None</a></p>
<p>Microbial plankton form the productive base of both marine and freshwater ecosystems and are key drivers of global biogeochemical cycles of carbon and nutrients. Plankton diversity is immense with representations from all major phyla within the three domains of life. So far, plankton monitoring has mainly been based on microscopic identification, which has limited sensitivity and reproducibility, not least because of the numerical majority of plankton being unidentifiable under the light microscope. High-throughput sequencing of taxonomic marker genes offers a means to identify taxa inaccessible by traditional methods; thus, recent studies have unveiled an extensive previously unknown diversity of plankton. Here, we conducted ultra-deep Illumina sequencing (average 105 sequences/sample) of rRNA gene amplicons of surface water eukaryotic and bacterial plankton communities sampled in summer along a 2000 km transect following the salinity gradient of the Baltic Sea. Community composition was strongly correlated with salinity for both bacterial and eukaryotic plankton assemblages, highlighting the importance of salinity for structuring the biodiversity within this ecosystem. In contrast, no clear trends in alpha-diversity for bacterial or eukaryotic communities could be detected along the transect. The distribution of major planktonic taxa followed expected patterns as observed in monitoring programs, but groups novel to the Baltic Sea were also identified, such as relatives to the coccolithophore Erniliana huxleyi detected in the northern Baltic Sea. This study provides the first ultra-deep sequencing-based survey on eukaryotic and bacterial plankton biogeography in the Baltic Sea.</p><h1>Efficient traffic monitoring in 5G Core Network</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>5G is an enabler to several new use cases. To support all of them, the network infrastructure must be flexible and it should adapt to the different situations. This feature is powered by SDN, NFV, and Automation, three of the main pillars on which the 5G network is built.Traditional network management approaches may not be suitable for the 5G Core Network User Plane, which holds strict requirements in terms of latency and throughput. Therefore, Artificial Intelligence agents have been proposed to manage the 5G in a more efficient manner, delivering a more optimized allocation of the resources. This approach requires real-time monitoring of the data passing by the Core Network, a feature not standardized by the current protocols. In this thesis, the design of a monitoring protocol for the 5G Core Network User Plane has been studied, focusing on precise measurement of latencies. Then, a In-band Network Telemetry (INT) framework has been implemented on top of a User Plane Function prototype. The prototype is built on top of a novel User Plane implementation, based on chaining of atomic functions called micro-UPFs (µUPFs).While the main focus of this work has been latency measurement, packet counters, byte counters and Inter Packet Gap values can be collected from the framework, proving the main KPIs of a 5G User Plane. The INT framework has been implemented through two new µUPFs, one for updating the INT metadata and one for collecting them. These metadata are attached to the user packets as GTP-U extended header, maintaining compatibility with the standard protocol. Moreover, the implemented framework allows high flexibility through dynamic tuning of the parameters, providing mechanisms to reduce the amount of telemetry data generated and, thus, the system overhead.The framework has been tested on a physical setup of four server machines, abstracting a Core Network User Plane, connected with 10 Gbps NICs. In all the tests performed, the performances of the User Plane are affected by the new functionalities only when INT metadata are inserted very frequently. The results show that is possible to monitor the three main KPIs of a 5G User Plane without heavily limiting the system performances.</p><h1>Fungal communities of West African ectomycorrhizal woodlands</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Ectomycorrhizal symbiosis is a widespread mutualist relationship between fungi and plants, often trees. It is most well-known in temperate and boreal forests, but also occurs in a variety of tropical ecosystems, including Sudanian savanna woodlands and gallery forests in West Africa. In addition to their important role in nutrient cycling, many ectomycorrhizal fungi also produce edible mushrooms as their fruit-bodies. In this thesis, I explore spatial patterns of above-ground and below-ground diversity in West African ectomycorrhizal communities, as well as the use of wild edible fungi by the local human population. In <b>Papers I</b> and <b>II</b>, I used soil DNA metabarcoding to investigate fungal communities in their vegetative state. <b>Paper I</b> used measurements of the scale of spatial autocorrelation within the fungal community in Sudanian woodlands in Benin as a test to compare different high-throughput DNA sequencing strategies, including short (≈350 bp) amplicon barcoding using the Illumina, Ion Torrent, and PacBio sequencing platforms, as well as long (≈1550 bp) amplicon barcoding using PacBio. There were some differences in species richness and community composition recovered by the two amplicon lengths, attributable to known primer biases for the short amplicons, but these did not lead to different ecological results. Additionally, <b>Paper I</b> introduced new software packages for analysis of long-amplicon metabarcoding data and integrating phylogenetic information into sequence-based taxonomic identification. In <b>Paper II</b> we sampled both Sudanian woodland and gallery forest sites in five countries across West Africa using long-amplicon metabarcoding with PacBio. We found significant differences in fungal community composition between Sudanian woodland and gallery forest sites, but not between different tree species within each vegetation type. <b>Papers III</b> and <b>IV</b> focused above-ground to study the natural production and human use of the mushrooms themselves. In <b>Paper III</b> we exhaustively collected mushrooms from nine Sudanian woodland plots in Benin during three consecutive rainy seasons. We measured the total biomass produced of each morphospecies, as well as environmental variables related to microclimate, host tree availability, and soil chemistry. Mushroom production and diversity were negatively correlated with soil nitrogen levels, and positively correlated with soil phosphorus levels. Although there were no clear differences in the fungal communities associated with the four host trees present in our plots, greater host tree diversity was associated with greater fungal diversity and productivity. Finally, <b>Paper IV</b> combined interviews with local people from four ethnic groups in five villages near the study sites from <b>Paper III</b> about their knowledge and preferences for edible mushrooms with DNA barcoding of specimens. Knowledge and preferences for different mushrooms varied between ethnic groups, but people living in a village where their ethnic group is a minority tended to absorb knowledge about mushrooms from the majority group in the village. Women over age 35 were the most knowledgeable about mushrooms, but the degree of gender difference varied between groups. We compiled a list of the most choice edible mushrooms in the area, which can inform the possibility for commercial trade in wild mushrooms to supplement the income of rural people and encourage sustainable forest management.</p><h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding</h1>
<p><a href="https://doi.org/10.3897/mycokeys.39.28109">doi:10.3897/mycokeys.39.28109</a></p>

<p><a href=None>None</a></p>
<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p><h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient</h1>
<p><a href="https://doi.org/10.1111/mec.15561">doi:10.1111/mec.15561</a></p>

<p><a href=None>None</a></p>
<p>Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates &lt; 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%–6.9%), when compared to low OC sediments (1.2%–2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode <i>Sabatieria</i> and foraminifera <i>Elphidium</i> in high OC, and <i>Cryptocaryon</i>‐like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p><h1>High-throughput sequencing of African <i>chikanda</i> cake highlights conservation challenges in orchids</h1>
<p><a href="https://doi.org/10.1007/s10531-017-1343-7">doi:10.1007/s10531-017-1343-7</a></p>

<p><a href=None>None</a></p>
<p><i>Chikanda</i> is a traditional dish made with wild-harvested ground orchid tubers belonging to three orchidioid genera, <i>Disa</i>, <i>Satyrium</i> and <i>Habenaria</i>, all of which are CITES appendix II-listed. Identification of collected orchid tubers is very difficult and documentation of constituent species in prepared <i>chikanda</i> has hitherto been impossible. Here amplicon metabarcoding was used in samples of six prepared <i>chikanda</i> cakes to study genetic sequence diversity and species diversity in this product. Molecular operational taxonomic unit identification using similarity-matching reveals that species of all three genera were present in the <i>chikanda</i> samples studied. <i>Disa</i> was present in all of the samples, <i>Satyrium</i> in five out of six and <i>Habenaria</i> in one of the samples, as well as a number of other plants. The fact that each sample contained orchids and the presence of a wide variety of species from all genera in this traditional dish raise serious concerns about the sustainability of this trade and the future of wild orchid populations in the main harvest areas. This proof-of-concept study shows that Ion-Torrent PGM is a cost-effective scalable platform for metabarcoding using the relatively long nrITS1 and nrITS2 regions. Furthermore, nrITS metabarcoding can be successfully used for the detection of specific ingredients in a highly-processed food product at genus level, and this makes it a useful tool in the detection of possible conservation issues arising from commercialized trade or processed plant products.</p><h1>Lake sedimentary dna research on past terrestrial and aquatic biodiversity: Overview and recommendations</h1>
<p><a href="https://doi.org/10.3390/quat4010006">doi:10.3390/quat4010006</a></p>

<p><a href=None>None</a></p>
<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p><h1>Lake Sedimentary DNA Research on Past Terrestrial and Aquatic Biodiversity : Overview and Recommendations</h1>
<p><a href="https://doi.org/10.3390/quat4010006">doi:10.3390/quat4010006</a></p>

<p><a href=None>None</a></p>
<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p><h1>Microbial DNA Sequencing in Environmental Studies</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>The field of microbial ecology has just entered a new era of rapid technological development and generation of big data. The high-throughput sequencing techniques presently available provide an opportunity to extensively inventorize the blueprints of life. Now, millions of microbes of natural microbial communities can be studied simultaneously without prior cultivation. New species and new functions (genes) can be discovered just by mining sequencing data. However, there is still a tremendous number of microorganisms not yet examined, nor are the ecosystem functions these carry out. The modern genomic technologies can contribute to solve environmental problems and help us understand ecosystems, but to most efficiently do so, methods need to be continuously optimised.<b> </b>During my Ph. D. studies, I developed a method to survey eukaryotic microbial diversity with a higher accuracy, and applied various sequencing-based approaches in an attempt to answer questions of importance in environmental research and ecology. In <b>PAPER-I</b>, we developed a set of 18S rRNA gene PCR primers with high taxonomic coverage, meeting the requirements of currently popular sequencing technologies and matching the richness of 18S rRNA reference sequences accumulated so far. In <b>PAPER-II</b>, we conducted the first sequencing-based spatial survey on the combined eukaryotic and bacterial planktonic community in the Baltic Sea to uncover the relationship of microbial diversity and environmental conditions. Here, the 18S primers designed in <b>PAPER-I</b> and a pair of broad-coverage 16S primers were employed to target the rRNA genes of protists and bacterioplankton for amplicon sequencing. In <b>PAPER-III</b>, we integrated metagenomic, metabarcoding, and metatranscriptomic data in an effort to scrutinise the protein synthesis potential (i.e., activity) of microbes in the sediment at a depth of 460 m in the Baltic Sea and, thus, disclosing microbial diversity and their possible ecological functions within such an extreme environment. Lastly, in <b>PAPER-IV</b>, we compared the performance of <i>E. coli</i> culturing, high-throughput sequencing, and portable real-time sequencing in tracking wastewater contamination in an urban stormwater system. From the aspects of cost, mobility and accuracy, we evaluated the usage of sequencing-based approaches in civil engineering, and for the first time, validated the real-time sequencing device in use within water quality monitoring. In summary, these studies demonstrate how DNA sequencing of microbial communities can be applied in environmental monitoring and ecological research.</p><h1>Microbial metagenomics in the Baltic Sea : Recent advancements and prospects for environmental monitoring</h1>
<p><a href="https://doi.org/10.1007/s13280-015-0663-7">doi:10.1007/s13280-015-0663-7</a></p>

<p><a href=None>None</a></p>
<p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed.</p><h1>Mycobiomes of sympatric Amorphophallus albispathus (Araceae) and Camellia sinensis (Theaceae) – a case study reveals clear tissue preferences and differences in diversity and composition</h1>
<p><a href="https://doi.org/10.1007/s11557-018-1375-8">doi:10.1007/s11557-018-1375-8</a></p>

<p><a href=None>None</a></p>
<p>Multiple biotic and abiotic parameters influence the dynamics of individual fungal species and entire communities. Major drivers for tropical plant endophytes are undoubtedly seasonality, local habitat conditions and biogeography. However, host specialization and tissue preferences also contribute to the structuring of endophytic mycobiomes. To elucidate such specializations and preferences, we sampled two commercially important, unrelated plant species, Amorphophallus albispathus and Camellia sinensis (tea plant) simultaneously at close proximity. The mycobiomes of different tissue types were assessed with high-throughput amplicon sequencing of the internal transcribed spacer DNA region. Both plants hosted different fungal communities and varied in α- and β-diversity, despite their neighboring occurrence. However, the fungal assemblages of Amorphophallus leaflets shared taxa with the mycobiomes of tea leaves, thereby suggesting common driving forces for leaf-inhabiting fungi irrespective of host plant identity. The mycobiome composition and diversity of tea leaves was clearly driven by leaf age. We suggest that the very youngest tea leaves are colonized by stochastic processes, while mycobiomes of old leaves are rather similar as the result of progressive succession. The biodiversity of fungi associated with A. albispathus was characterized by a large number of unclassified OTUs (at genus and species level) and by tissue-specific composition.This study is the first cultivation-independent high-throughput assessment of fungal biodiversity of an Amorphophallus species, and additionally expands the knowledge base on fungi associated with tea plants.</p><h1>Perspectives from Ten Years of Protist Studies by High-Throughput Metabarcoding</h1>
<p><a href="https://doi.org/10.1111/jeu.12813">doi:10.1111/jeu.12813</a></p>

<p><a href=None>None</a></p>
<p>During the last decade, high-throughput metabarcoding became routine for analyzing protistan diversity and distributions in nature. Amid a multitude of exciting findings, scientists have also identified and addressed technical and biological limitations, although problems still exist for inference of meaningful taxonomic and ecological knowledge based on shortDNAsequences. Given the extensive use of this approach, it is critical to settle our understanding on its strengths and weaknesses and to synthesize up-to-date methodological and conceptual trends. This article summarizes key scientific and technical findings, and identifies current and future directions in protist research that uses metabarcoding.</p><h1>Systematic comparison of the relative accuracy of vegetation surveys and soil DNA metabarcoding : Assessing plant biodiversity at different spatial scales</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Analysis of soil-derived DNA has been shown to minimize problems seen during traditional vegetation surveys, e.g. by matching the eDNA to a reference database for taxonomic identification rather than relying solely on taxonomic expertise. However, it has been debated to what extent and how accurately eDNA acts as a proxy for biodiversity. The reliability on eDNA and the awareness on influencing factors is also important for palaeoenvironmental reconstructions where above-ground vegetation cannot be used. This study aimed to investigate how well modern soil-derived eDNA reflects the contemporary vascular vegetation communities in a subarctic environment, and if the efficiency of the taxonomic identification differed between spatial scales. Near-surface soil samples at altitudinal gradients along numerous transects were collected in combination with vegetation surveys. The eDNA was amplified through metabarcoding using the P6 loop region of the chloroplast trnL intron, followed by a high-throughput sequencing. No difference in the number of identified taxa between eDNA and vegetation survey was seen at landscape scale. In contrast, the number of identified taxa was consistently higher in the vegetation survey at smaller spatial scales. The efficiency of identified taxa per scale remained stable for the vegetation survey, whereas for eDNA, a decreasing trend was seen. This study highlights the variations on taxa identification between both methods and which factors might cause it. Combining the methods allows for a more precise modern biodiversity estimation, as well as to minimize wrongful conclusions. This allows for a more accurate palaeoenvironmental reconstructions, which in turn can improve future species conservation decisions.</p><h1>The wolf reference genome sequence (Canis lupus lupus) and its implications for Canis spp. population genomics</h1>
<p><a href="https://doi.org/10.1186/s12864-017-3883-3">doi:10.1186/s12864-017-3883-3</a></p>

<p><a href=None>None</a></p>
<p>Analysis of soil-derived DNA has been shown to minimize problems seen during traditional vegetation surveys, e.g. by matching the eDNA to a reference database for taxonomic identification rather than relying solely on taxonomic expertise. However, it has been debated to what extent and how accurately eDNA acts as a proxy for biodiversity. The reliability on eDNA and the awareness on influencing factors is also important for palaeoenvironmental reconstructions where above-ground vegetation cannot be used. This study aimed to investigate how well modern soil-derived eDNA reflects the contemporary vascular vegetation communities in a subarctic environment, and if the efficiency of the taxonomic identification differed between spatial scales. Near-surface soil samples at altitudinal gradients along numerous transects were collected in combination with vegetation surveys. The eDNA was amplified through metabarcoding using the P6 loop region of the chloroplast trnL intron, followed by a high-throughput sequencing. No difference in the number of identified taxa between eDNA and vegetation survey was seen at landscape scale. In contrast, the number of identified taxa was consistently higher in the vegetation survey at smaller spatial scales. The efficiency of identified taxa per scale remained stable for the vegetation survey, whereas for eDNA, a decreasing trend was seen. This study highlights the variations on taxa identification between both methods and which factors might cause it. Combining the methods allows for a more precise modern biodiversity estimation, as well as to minimize wrongful conclusions. This allows for a more accurate palaeoenvironmental reconstructions, which in turn can improve future species conservation decisions.</p><h1>What do we learn from cultures in the omics age? High-throughput sequencing and cultivation of leaf-inhabiting endophytes from beech (Fagus sylvatica L.) revealed complementary community composition but similar correlations with local habitat conditions</h1>
<p><a href="https://doi.org/10.3897/mycokeys.20.11265">doi:10.3897/mycokeys.20.11265</a></p>

<p><a href=None>None</a></p>
<p>Comparative simultaneous studies of environmental high-throughput sequencing (HTS) and cultivation of plant-associated fungi have rarely been conducted in the past years. For the present contribution, HTS and extinction culturing were applied for the same leaf samples of European beech (Fagus sylvatica) in order to trace both “real” environmental drivers as well as method-dependent signals of the observed mycobiomes. Both approaches resulted in non-overlapping community composition and pronounced differences in taxonomic classification and trophic stages. However, both methods revealed similar correlations of the fungal communities with local environmental conditions. Our results indicate undeniable advantages of HTS over cultivation in terms of revealing a good representation of the major functional guilds, rare taxa and biodiversity signals of leaf-inhabiting fungi. On the other hand our results demonstrate that the immense body of literature about cultivable endophytic fungi can and should be used for the interpretation of community signals and environmental correlations obtained from HTS studies and that cultivation studies should be continued at the highest standards, e.g. when sequencing facilities are not available or if such surveys are expanded into functional aspects with experiments on living isolates.</p><h1>Short- and long-read metabarcoding of the eukaryotic rRNA operon: evaluation of primers and comparison to shotgun metagenomics sequencing</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13623">doi:10.1111/1755-0998.13623</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing for analysis of microbial diversity has evolved vastly over the last decade. Currently the go-to method for studying microbial eukaryotes is short-read metabarcoding of variable regions of the 18S rRNA gene with <500 bp amplicons. However, there is a growing interest in applying long-read sequencing of amplicons covering the rRNA operon for improving taxonomic resolution. For both methods, the choice of primers is crucial. It determines if community members are covered, if they can be identified at a satisfactory taxonomic level, and if the obtained community profile is representative. Here, we designed new primers targeting 18S and 28S rRNA based on 177,934 and 21,072 database sequences, respectively. The primers were evaluated in silico along with published primers on reference sequence databases and marine metagenomics datasets. We further evaluated a subset of the primers for short- and long-read sequencing on environmental samples in vitro and compared the obtained community profile with primer-unbiased metagenomic sequencing. Of the short-read pairs, a new V6-V8 pair and the V4_Balzano pair used with a simplified PCR protocol provided good results in silico and in vitro. Fewer differences were observed between the long-read primer pairs. The long-read amplicons and ITS1 alone provided higher taxonomic resolution than V4. Together, our results represent a reference and guide for selection of robust primers for research on and environmental monitoring of microbial eukaryotes.</p><h1>DNA sequence and taxonomic gap analyses to quantify the coverage of aquatic cyanobacteria and eukaryotic microalgae in reference databases: Results of a survey in the Alpine region</h1>
<p><a href="https://doi.org/S0048-9697(22)02268-9">doi:S0048-9697(22)02268-9</a></p>

<p><a href=None>None</a></p>
<p>The taxonomic identification of organisms based on the amplification of specific genetic markers (metabarcoding) implicitly requires adequate discriminatory information and taxonomic coverage of environmental DNA sequences in taxonomic databases. These requirements were quantitatively examined by comparing the determination of cyanobacteria and microalgae obtained by metabarcoding and light microscopy. We used planktic and biofilm samples collected in 37 lakes and 22 rivers across the Alpine region. We focused on two of the most used and best represented genetic markers in the reference databases, namely the 16S rRNA and 18S rRNA genes. A sequence gap analysis using blastn showed that, in the identity range of 99-100%, approximately 30% (plankton) and 60% (biofilm) of the sequences did not find any close counterpart in the reference databases (NCBI GenBank). Similarly, a taxonomic gap analysis showed that approximately 50% of the cyanobacterial and eukaryotic microalgal species identified by light microscopy were not represented in the reference databases. In both cases, the magnitude of the gaps differed between the major taxonomic groups. Even considering the species determined under the microscope and represented in the reference databases, 22% and 26% were still not included in the results obtained by the blastn at percentage levels of identity ≥95% and ≥97%, respectively. The main causes were the absence of matching sequences due to amplification and/or sequencing failure and potential misidentification in the microscopy step. Our results quantitatively demonstrated that in metabarcoding the main obstacles in the classification of 16S rRNA and 18S rRNA sequences and interpretation of high-throughput sequencing biomonitoring data were due to the existence of important gaps in the taxonomic completeness of the reference databases and the short length of reads. The study focused on the Alpine region, but the extent of the gaps could be much greater in other less investigated geographic areas.</p><h1>Best practices in metabarcoding of fungi: From experimental design to results</h1>
<p><a href="https://doi.org/10.1111/Mec.16460">doi:10.1111/Mec.16460</a></p>

<p><a href=None>None</a></p>
<p>The development of high-throughput sequencing (HTS) technologies has greatly improved our capacity to identify fungi and unveil their ecological roles across a variety of ecosystems. Here we provide an overview of current best practices in metabarcoding analysis of fungal communities, from experimental design through molecular and computational analyses. By reanalysing published data sets, we demonstrate that operational taxonomic units (OTUs) outperform amplified sequence variants (ASVs) in recovering fungal diversity, a finding that is particularly evident for long markers. Additionally, analysis of the full-length ITS region allows more accurate taxonomic placement of fungi and other eukaryotes compared to the ITS2 subregion. Finally, we show that specific methods for compositional data analyses provide more reliable estimates of shifts in community structure. We conclude that metabarcoding analyses of fungi are especially promising for integrating fungi into the full microbiome and broader ecosystem functioning context, recovery of novel fungal lineages and ancient organisms as well as barcoding of old specimens including type material.</p><h1>Detection and population genomics of sea turtle species via noninvasive environmental DNA analysis of nesting beach sand tracks and oceanic water</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13617">doi:10.1111/1755-0998.13617</a></p>

<p><a href=None>None</a></p>
<p>Elusive aquatic wildlife, such as endangered sea turtles, are difficult to monitor and conserve. As novel molecular and genetic technologies develop, it is possible to adapt and optimize them for wildlife conservation. One such technology is environmental (e)DNA - the detection of DNA shed from organisms into their surrounding environments. We developed species-specific green (Chelonia mydas) and loggerhead (Caretta caretta) sea turtle probe-based qPCR assays, which can detect and quantify sea turtle eDNA in controlled (captive tank water and sand samples) and free ranging (oceanic water samples and nesting beach sand) settings. eDNA detection complemented traditional in-water sea turtle monitoring by enabling detection even when turtles were not visually observed. Furthermore, we report that high throughput shotgun sequencing of eDNA sand samples enabled sea turtle population genetic studies and pathogen monitoring, demonstrating that noninvasive eDNA techniques are viable and efficient alternatives to biological sampling (e.g., biopsies and blood draws). Genetic information was obtained from sand many hours after nesting events, without having to observe or interact with the target individual. This greatly reduces the sampling stress experienced by nesting mothers and emerging hatchlings, and avoids sacrificing viable eggs for genetic analysis. The detection of pathogens from sand indicates significant potential for increased wildlife disease monitoring capacity and viral variant surveillance. Together, these results demonstrate the potential of eDNA approaches to ultimately help understand and conserve threatened species such as sea turtles.</p><h1>LANDMark: an ensemble approach to the supervised selection of biomarkers in high-throughput sequencing data</h1>
<p><a href="https://doi.org/10.1186/S12859-022-04631-Z">doi:10.1186/S12859-022-04631-Z</a></p>

<p><a href=None>None</a></p>
<p>Identification of biomarkers, which are measurable characteristics of biological datasets, can be challenging. Although amplicon sequence variants (ASVs) can be considered potential biomarkers, identifying important ASVs in high-throughput sequencing datasets is challenging. Noise, algorithmic failures to account for specific distributional properties, and feature interactions can complicate the discovery of ASV biomarkers. In addition, these issues can impact the replicability of various models and elevate false-discovery rates. Contemporary machine learning approaches can be leveraged to address these issues. Ensembles of decision trees are particularly effective at classifying the types of data commonly generated in high-throughput sequencing (HTS) studies due to their robustness when the number of features in the training data is orders of magnitude larger than the number of samples. In addition, when combined with appropriate model introspection algorithms, machine learning algorithms can also be used to discover and select potential biomarkers. However, the construction of these models could introduce various biases which potentially obfuscate feature discovery.</p><h1>[Application and progress in high-throughput sequencing technology for meat adulteration detection]</h1>
<p><a href="https://doi.org/10.13345/J.Cjb.210113">doi:10.13345/J.Cjb.210113</a></p>

<p><a href=None>None</a></p>
<p>Adulteration in meat products is a widespread issue that could lead to serious threats to public health and religious violations. Technology that offers rapid, sensitive, accurate and reliable detection of meat species is the key to an effectual monitoring and control against meat adulteration. In recent years, high-throughput sequencing-based DNA metabarcoding technology has developed rapidly. With the characteristics of being high-throughput, highly precise and high-speed, this technology can simultaneously identify multiple species in complex samples, thus offering pronounced advantages in the surveillance of adulteration in meat and meat products. Starting with an introduction of the major developments in the high-throughput sequencing technology in the past two decades, this review provides an overview of the technical characteristics and research methods of DNA metabarcoding, summarizes the application of DNA metabarcoding technology in meat adulteration detection over the last few years, discusses the challenges of using DNA metabarcoding technology in the detection of meat adulteration, and provides future prospects on the development of this technology.</p><h1>Disentangling bias for non-destructive insect metabarcoding</h1>
<p><a href="https://doi.org/10.7717/Peerj.12981">doi:10.7717/Peerj.12981</a></p>

<p><a href=None>None</a></p>
<p>A fast and reliable method for obtaining a species-level identification is a fundamental requirement for a wide range of activities, from plant protection and invasive species management to biodiversity assessments and ecological studies. For insects, novel molecular techniques such as DNA metabarcoding have emerged as a rapid alternative to traditional morphological identification, reducing the dependence on limited taxonomic experts. Until recently, molecular techniques have required a destructive DNA extraction, precluding the possibility of preserving voucher specimens for future studies, or species descriptions. Here we paired insect metabarcoding with two recent non-destructive DNA extraction protocols, to obtain a rapid and high-throughput taxonomic identification of diverse insect taxa while retaining a physical voucher specimen. The aim of this work was to explore how non-destructive extraction protocols impact the semi-quantitative nature of metabarcoding, which alongside species presence/absence also provides a quantitative, but biased, representation of their relative abundances. By using a series of mock communities representing each stage of a typical metabarcoding workflow we were able to determine how different morphological (<i>i.e</i>., insect biomass and exoskeleton hardness) and molecular traits (<i>i.e</i>., primer mismatch and amplicon GC%), interact with different protocol steps to introduce quantitative bias into non-destructive metabarcoding results. We discuss the relevance of taxonomic bias to metabarcoding identification of insects and potential approaches to account for it.</p><h1>Next generation restoration metrics: Using soil eDNA bacterial community data to measure trajectories towards rehabilitation targets</h1>
<p><a href="https://doi.org/S0301-4797(22)00321-8">doi:S0301-4797(22)00321-8</a></p>

<p><a href=None>None</a></p>
<p>In post-mining rehabilitation, successful mine closure planning requires specific, measurable, achievable, relevant and time-bound (SMART) completion criteria, such as returning ecological communities to match a target level of similarity to reference sites. Soil microbiota are fundamentally linked to the restoration of degraded ecosystems, helping to underpin ecological functions and plant communities. High-throughput sequencing of soil eDNA to characterise these communities offers promise to help monitor and predict ecological progress towards reference states. Here we demonstrate a novel methodology for monitoring and evaluating ecological restoration using three long-term (>25 year) case study post-mining rehabilitation soil eDNA-based bacterial community datasets. Specifically, we developed rehabilitation trajectory assessments based on similarity to reference data from restoration chronosequence datasets. Recognising that numerous alternative options for microbiota data processing have potential to influence these assessments, we comprehensively examined the influence of standard versus compositional data analyses, different ecological distance measures, sequence grouping approaches, eliminating rare taxa, and the potential for excessive spatial autocorrelation to impact on results. Our approach reduces the complexity of information that often overwhelms ecologically-relevant patterns in microbiota studies, and enables prediction of recovery time, with explicit inclusion of uncertainty in assessments. We offer a step change in the development of quantitative microbiota-based SMART metrics for measuring rehabilitation success. Our approach may also have wider applications where restorative processes facilitate the shift of microbiota towards reference states.</p><h1>Exploring the microbial community inhabiting the phosphogypsum stacks of Huelva (SW SPAIN) by a high throughput 16S/18S rDNA sequencing approach</h1>
<p><a href="https://doi.org/S0166-445X(22)00030-3">doi:S0166-445X(22)00030-3</a></p>

<p><a href=None>None</a></p>
<p>Around 100 Mt of phosphogypsum (PG) have been deposited in large stacks on the salt marshes of the Tinto River estuary in Huelva (SW Spain), covering about 1000 ha. These stacks contain extremely acidic water (pH < 2) with high concentrations of pollutants which can cause emissions into their surroundings, generating important environmental concerns. Despite many chemical, geological or hydrological studies have been conducted to characterize the PG stacks of Huelva, the microbial community inhabiting this extreme environment remains unexplored. Using a 16S/18S-rRNA-high throughput sequencing approach, we have uncovered the main taxonomic groups able to live in the acidic metal-contaminated water, which is in direct contact with the PG, demonstrating for the first time the existence of a huge diversity of microbial species in these extreme conditions. In addition, the physicochemical characteristics of the water sampled have been analyzed. These studies have revealed that the most abundant bacteria found in two different leachate samples of the PG stacks belong to the genera Acidiphilium, Pseudomonas, Leptosprillum, Acidithrix, or Acidithiobacillus, typically found in acid mine drainage (AMD) environments, which in total represent around 50% of the total bacterial community. Biodiversity of eukaryotes in PG water is lower than that of prokaryotes, especially in the water collected from the perimeter channel that surrounds the PG stacks, where the pH reaches a value of 1.5 and the activity concentrations exceed 300 Bq L<sup>-1</sup> for <sup>238</sup>U or 20 Bq L<sup>-1</sup> for <sup>210</sup>Po, values which are from four to five orders of magnitude higher than those usually found in unperturbed surface waters. Even so, an unexpected diversity of algae, fungi, and ciliates have been found in the PG stacks of Huelva, where chlorophyte microalgae and basidiomycetes fungi are the most abundant eukaryotes. Additional bioinformatics tools have been used to perform a functional analysis and predict the most common metabolic pathways in the PG microbiota. The obtained data indicate that the extreme conditions of these PG stacks hide an unexpected microbial diversity, which can play an important role in the dynamics of the contaminating compounds of the PG and provide new strains with unique biotechnological applications.</p><h1>Metabarcoding reveals massive species diversity of Diptera in a subtropical ecosystem</h1>
<p><a href="https://doi.org/10.1002/Ece3.8535">doi:10.1002/Ece3.8535</a></p>

<p><a href=None>None</a></p>
<p>Diptera is often considered to be the richest insect group due to its great species diversity and broad ecological versatility. However, data on dipteran diversity from subtropical ecosystems have hitherto been scarce, due to the lack of studies conducted at an appropriate large scale. We investigated the diversity and composition of Diptera communities on Tianmu Mountain, Zhejiang, China, using DNA metabarcoding technology, and evaluated their dynamic responses to the effects of slope aspect, season, and altitudinal zone. A total of 5,092 operational taxonomic units (OTUs) were discovered and tentatively assigned to 72 dipteran families, including 2 family records new for China and 30 family records new for the locality. Cecidomyiidae, Sciaridae, and Phoridae were the predominant families, representing 53.6% of total OTUs, while 52 families include >95% unidentified and presumed undescribed species. We found that the community structure of Diptera was significantly affected by aspect, seasonality (month) and elevation, with richer diversity harbored in north-facing than south-facing slopes, and seasonality a more profound driver of community structure and diversity than elevation. Overall, massive species diversity of Diptera communities was discovered in this subtropical ecosystem of east China. The huge diversity of potentially undescribed species only revealed by metabarcoding now requires more detailed taxonomic study, as a step toward an evolutionary integration that accumulates information on species' geographic ranges, ecological traits, functional roles, and species interactions, and thus places the local communities in the context of the growing knowledge base of global biodiversity and its response to environmental change.</p><h1>Ichthyoplankton DNA metabarcoding: Challenges and perspectives</h1>
<p><a href="https://doi.org/10.1111/Mec.16387">doi:10.1111/Mec.16387</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding has been widely used to access and monitor species. However, several challenges remain open for its mainstream application in ecological studies, particularly when dealing with a quantitative approach. In a From the Cover article in this issue of Molecular Ecology, Mariac et al. (2021) report species-level ichthyoplankton dynamics for 97 fish species from two Amazon river basins using a clever quantitative metabarcoding approach employing a probe capture method. They clearly show that most species spawned during the floods, although ~20% also spawned mainly during the receding period and some other year-round, but interestingly, species from the same genus reproduced in distinct periods (i.e., inverse phenology). Opportunistically, Mariac et al. (2021) reported that during an intense hydrological anomaly, several species had a sharp reduction in spawning activity, demonstrating a quick response to environmental cues. This is an interesting result since the speed at which fish species can react to environmental changes, during the spawning period, is largely unknown. Thus, this study brings remarkable insights into basic life history information that is imperative for proposing strategies that could lead to a realistic framework for sustainable fisheries management practices and conservation, fundamental for an understudied and threatened realm, such as the Amazon River basin.</p><h1>Metabarcoding of feces and intestinal contents to determine carnivorous diets in red-crowned cranes in eastern Hokkaido, Japan</h1>
<p><a href="https://doi.org/10.1292/Jvms.21-0516">doi:10.1292/Jvms.21-0516</a></p>

<p><a href=None>None</a></p>
<p>The red-crowned crane Grus japonensis in Hokkaido, Japan forms a closed population as a residence that is independent of the mainland population. Based on observations of a limited number of individuals as well as cranes in captivity, red-crowned cranes are omnivores and eat fish, worms, insects and plants in their own territories except in winter, when they are fed with dent corn that is supplied in eastern Hokkaido. DNA metabarcoding based on high throughput sequencing was carried out using universal primer sets for cytochrome oxidase subunit I gene. Feces from 27 chicks collected in June and July in the period from 2016 to 2018 and intestinal contents from 33 adult and subadult cranes that were found dead almost throughout year in 2006-2013 in the field in eastern Hokkaido were used. Although compositions varied considerably in the cranes, both insects and fish were found in adults and subadults to the same extents, while insects were predominant in chicks. Both insects and fish were detected in all seasons for adults and subadults. Horse flies, scarab beetles and weevils accounted for the most of the insects regardless of the life stage. Dace, stickleback, flatfish and sculpin were the major fish species in adults, while chicks ate almost only stickleback. The results provide the first comprehensive data on carnivorous diets in wild red-crowned cranes in eastern Hokkaido as basis for conservation of red-crowned cranes, for which the life style and area continue to change.</p><h1>Airborne environmental DNA for terrestrial vertebrate community monitoring</h1>
<p><a href="https://doi.org/S0960-9822(21)01690-0">doi:S0960-9822(21)01690-0</a></p>

<p><a href=None>None</a></p>
<p>Biodiversity monitoring at the community scale is a critical element of assessing and studying species distributions, ecology, diversity, and movements, and it is key to understanding and tracking environmental and anthropogenic effects on natural ecosystems.<sup>1-4</sup> Vertebrates in terrestrial ecosystems are experiencing extinctions and declines in both population numbers and sizes due to increasing threats from human activities and environmental change.<sup>5-8</sup> Terrestrial vertebrate monitoring using existing methods is generally costly and laborious, and although environmental DNA (eDNA) is becoming the tool of choice to assess biodiversity, few sample types effectively capture terrestrial vertebrate diversity. We hypothesized that eDNA captured from air could allow straightforward collection and characterization of terrestrial vertebrate communities. We filtered air at three localities in the Copenhagen Zoo: a stable, outside between the outdoor enclosures, and in the Rainforest House. Through metabarcoding of airborne eDNA, we detected 49 vertebrate species spanning 26 orders and 37 families: 30 mammal, 13 bird, 4 fish, 1 amphibian, and 1 reptile species. These spanned animals kept at the zoo, species occurring in the zoo surroundings, and species used as feed in the zoo. The detected species comprise a range of taxonomic orders and families, sizes, behaviors, and abundances. We found shorter distance to the air sampling device and higher animal biomass to increase the probability of detection. We hereby show that airborne eDNA can offer a fundamentally new way of studying and monitoring terrestrial communities.</p><h1>Soil Fungal Diversity of the Aguarongo Andean Forest (Ecuador)</h1>
<p><a href="https://doi.org/1289">doi:1289</a></p>

<p><a href=None>None</a></p>
<p>Fungi represent an essential component of ecosystems, functioning as decomposers and biotrophs, and they are one of the most diverse groups of Eukarya. In the tropics, many species are unknown. In this work, high-throughput DNA sequencing was used to discover the biodiversity of soil fungi in the Aguarongo forest reserve, one of the richest biodiversity hotspots in Ecuador. The rDNA metabarcoding analysis revealed the presence of seven phyla: <i>Ascomycota</i>, <i>Basidiomycota</i>, <i>Mortierellomycota</i>, <i>Mucoromycota</i>, <i>Glomeromycota</i>, <i>Chytridiomycota</i>, and <i>Monoblepharomycota</i>. A total of 440 identified species were recorded. They mainly belonged to <i>Ascomycota</i> (263) and <i>Basidiomycota</i> (127). In <i>Mortierellomycota</i>, 12 species were recorded, among which <i>Podila verticillata</i> is extremely frequent and represents the dominant species in the entire mycobiota of Aguarongo. The present research provides the first account of the entire soil mycobiota in the Aguarongo forest, where many fungal species exist that have strong application potential in agriculture, bioremediation, chemical, and the food industry. The Aguarongo forest hides a huge number of unknown fungal species that could be assessed, and its protection is of the utmost importance.</p><h1>Zooplankton diversity monitoring strategy for the urban coastal region using metabarcoding analysis</h1>
<p><a href="https://doi.org/10.1038/S41598-021-03656-3">doi:10.1038/S41598-021-03656-3</a></p>

<p><a href=None>None</a></p>
<p>Marine ecosystems in urban coastal areas are exposed to many risks due to human activity. Thus, long-term and continuous monitoring of zooplankton diversity is necessary. High-throughput DNA metabarcoding has gained recognition as an efficient and highly sensitive approach to accurately describing the species diversity of marine zooplankton assemblages. In this study, we collected 30 zooplankton samples at about 2-week intervals for 1 year. Zooplankton diversity showing a typical four season pattern. Of the "total" and "common" zooplankton, we assigned 267 and 64 taxa. The cluster structure and seasonal diversity pattern were rough when only the "common" zooplankton was used. Our study examined how to maximize the benefits of metabarcoding for monitoring zooplankton diversity in urban coastal areas. The results suggest that to take full advantage of metabarcoding when monitoring a zooplankton community, it is necessary to carefully investigate potential ecosystem threats (non-indigenous species) through sufficient curation rather than disregarding low-abundance operational taxonomic units.</p><h1>A high-throughput amplicon sequencing approach for population-wide species diversity and composition survey</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13576">doi:10.1111/1755-0998.13576</a></p>

<p><a href=None>None</a></p>
<p>Management of agricultural pests requires an understanding of pest species diversity, their interactions with beneficial insects and spatial-temporal patterns of pest abundance. Invasive and agriculturally important insect pests can build up very high populations, especially in cropping landscapes. Traditionally, sampling effort for species identification involves small sample sizes and is labour intensive. Here, we describe a multiprimer high throughput sequencing (HTS) metabarcoding method and associated analytical workflow for a rapid, intensive, high-volume survey of pest species compositions. We demonstrate our method using the taxonomically challenging Bemisia pest cryptic species complex as examples. The whiteflies Bemisia including the"tabaci" species are agriculturally important capable of vectoring diverse plant viruses that cause diseases and crop losses. Our multiprimer metabarcoding HTS amplicon approach simultaneously process high volumes of whitefly individuals, with efficiency to detect rare (i.e., 1%) test-species, while our improved whitefly primers for metabarcoding also detected beneficial hymenopteran parasitoid species from whitefly nymphs. Field-testing our redesigned Bemisia metabarcoding primer sets across the Tanzania, Uganda and Malawi cassava cultivation landscapes, we identified the sub-Saharan Africa 1 Bemisia putative species as the dominant pest species, with other cryptic Bemisia species being detected at various abundances. We also provide evidence that Bemisia species compositions can be affected by host crops and sampling techniques that target either nymphs or adults. Our multiprimer HTS metabarcoding method incorporated two overlapping amplicons of 472 bp and 518 bp that spanned the entire 657 bp 3' barcoding region for Bemisia, and is particularly suitable to molecular diagnostic surveys of this highly cryptic insect pest species complex that also typically exhibited high population densities in heavy crop infestation episodes. Our approach can be adopted to understand species biodiversity across landscapes, with broad implications for improving transboundary biosecurity preparedness, thus contributing to molecular ecological knowledge and the development of control strategies for high-density, cryptic, pest-species complexes.</p><h1>Analysis of Serological Biomarkers of SARS-CoV-2 Infection in Convalescent Samples From Severe, Moderate and Mild COVID-19 Cases</h1>
<p><a href="https://doi.org/10.3389/Fimmu.2021.748291">doi:10.3389/Fimmu.2021.748291</a></p>

<p><a href=None>None</a></p>
<p>Precision monitoring of antibody responses during the COVID-19 pandemic is increasingly important during large scale vaccine rollout and rise in prevalence of <i>Severe Acute Respiratory Syndrome-related Coronavirus-2</i> (SARS-CoV-2) variants of concern (VOC). Equally important is defining Correlates of Protection (CoP) for SARS-CoV-2 infection and COVID-19 disease. Data from epidemiological studies and vaccine trials identified virus neutralising antibodies (Nab) and SARS-CoV-2 antigen-specific (notably RBD and S) binding antibodies as candidate CoP. In this study, we used the World Health Organisation (WHO) international standard to benchmark neutralising antibody responses and a large panel of binding antibody assays to compare convalescent sera obtained from: a) COVID-19 patients; b) SARS-CoV-2 seropositive healthcare workers (HCW) and c) seronegative HCW. The ultimate aim of this study is to identify biomarkers of humoral immunity that could be used to differentiate severe from mild or asymptomatic SARS-CoV-2 infections. Some of these biomarkers could be used to define CoP in further serological studies using samples from vaccination breakthrough and/or re-infection cases. Whenever suitable, the antibody levels of the samples studied were expressed in International Units (IU) for virus neutralisation assays or in Binding Antibody Units (BAU) for ELISA tests. In this work we used commercial and non-commercial antibody binding assays; a lateral flow test for detection of SARS-CoV-2-specific IgG/IgM; a high throughput multiplexed particle flow cytometry assay for SARS-CoV-2 Spike (S), Nucleocapsid (N) and Receptor Binding Domain (RBD) proteins); a multiplex antigen semi-automated immuno-blotting assay measuring IgM, IgA and IgG; a pseudotyped microneutralisation test (pMN) and an electroporation-dependent neutralisation assay (EDNA). Our results indicate that overall, severe COVID-19 patients showed statistically significantly higher levels of SARS-CoV-2-specific neutralising antibodies (average 1029 IU/ml) than those observed in seropositive HCW with mild or asymptomatic infections (379 IU/ml) and that clinical severity scoring, based on WHO guidelines was tightly correlated with neutralisation and RBD/S antibodies. In addition, there was a positive correlation between severity, N-antibody assays and intracellular virus neutralisation.</p><h1>Benefits of merging paired-end reads before pre-processing environmental metagenomics data</h1>
<p><a href="https://doi.org/S1874-7787(21)00080-5">doi:S1874-7787(21)00080-5</a></p>

<p><a href=None>None</a></p>
<p>High throughput sequencing of environmental DNA has applications in biodiversity monitoring, taxa abundance estimation, understanding the dynamics of community ecology, and marine species studies and conservation. Environmental DNA, especially, marine eDNA, has a fast degradation rate. Aside from the good quality reads, the data could have a significant number of reads that fall slightly below the default PHRED quality threshold of 30 on sequencing. For quality control, trimming methods are employed, which generally precede the merging of the read pairs. However, in the case of eDNA, a significant percentage of reads within the acceptable quality score range are also dropped.</p><h1>Metabarcoding analysis of trophic sources and linkages in the plankton community of the Kuroshio and neighboring waters</h1>
<p><a href="https://doi.org/10.1038/S41598-021-02083-8">doi:10.1038/S41598-021-02083-8</a></p>

<p><a href=None>None</a></p>
<p>Trophic sources and pathways supporting early life stages are crucial for survival of forage fishes recruiting around the oligotrophic and unproductive Kuroshio. However, information is limited for the Kuroshio planktonic food web and its trophodynamics because of its high biodiversity. Here, we explore trophic sources and linkages in the Kuroshio plankton community using metabarcoding analysis of gut-content DNA for 22 mesozooplankton groups. The major prey was dinoflagellates and calanoids for omnivorous groups, and calanoids and gelatinous organisms for carnivorous groups. Larvaceans and hydrozoans were the most frequently appeared prey for both omnivores and carnivores, whereas they were minor constituents of the available prey in water samples. Although calanoids overlapped as major prey items for both omnivores and carnivores because they were the most available, contributions from phytoplankton and gelatinous prey differed among taxonomic groups. Further analysis of the metabarcoding data showed that in addition to omnivorous copepods like calanoids, gelatinous groups like larvaceans and hydrozoans were important hubs in the planktonic food web with their multiple trophic linkages to many components. These findings suggest that gelatinous organisms are important as supplementary prey and provide evidence of niche segregation on trophic sources among mesozooplankton groups in the Kuroshio.</p><h1>High-Throughput Sequencing of Environmental DNA as a Tool for Monitoring Eukaryotic Communities and Potential Pathogens in a Coastal Upwelling Ecosystem</h1>
<p><a href="https://doi.org/10.3389/Fvets.2021.765606">doi:10.3389/Fvets.2021.765606</a></p>

<p><a href=None>None</a></p>
<p>The marine environment includes diverse microeukaryotic organisms that play important functional roles in the ecosystem. With molecular approaches, eukaryotic taxonomy has been improved, complementing classical analysis. In this study, DNA metabarcoding was performed to describe putative pathogenic eukaryotic microorganisms in sediment and marine water fractions collected in Galicia (NW Spain) from 2016 to 2018. The composition of eukaryotic communities was distinct between sediment and water fractions. Protists were the most diverse group, with the clade TSAR (Stramenopiles, Alveolata, Rhizaria, and Telonemida) as the primary representative organisms in the environment. Harmful algae and invasive species were frequently detected. Potential pathogens, invasive pathogenic organisms as well as the causative agents of harmful phytoplanktonic blooms were identified in this marine ecosystem. Most of the identified pathogens have a crucial impact on the aquacultural sector or affect to relevant species in the marine ecosystem, such as diatoms. Moreover, pathogens with medical and veterinary importance worldwide were also found, as well as pathogens that affect diatoms. The evaluation of the health of a marine ecosystem that directly affects the aquacultural sector with a zoonotic concern was performed with the metabarcoding assay.</p><h1>High-throughput sequencing of faeces provides evidence for dispersal of parasites and pathogens by migratory waterbirds</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13548">doi:10.1111/1755-0998.13548</a></p>

<p><a href=None>None</a></p>
<p>Examination of faecal material has demonstrated how a broad range of organisms are distributed by bird movements. Such research has largely focused on dispersal of plant seeds by frugivores and of freshwater organisms by waterbirds. However, with few exceptions (e.g. avian influenza, Ebola virus), there is a dearth of evidence for transport of parasites and pathogens. High-throughput sequencing methods now provide a powerful means of addressing this knowledge gap by elucidating faecal contents in unprecedented detail. We collected faeces excreted by a range of migratory waterbirds in south-west Spain and pooled faecal DNA to create libraries reflective of feeding behavior. We created sets of libraries using high-throughput metagenomic and amplicon sequencing. For the latter we employed two sets of primers to broadly target the V4 region of the 18S rRNA gene (one set amplifying the region across all eukaryotes, the other excluding amplification of metazoans). Libraries revealed a wide diversity of eukaryotes, including parasites of the faecal producers themselves, parasites of food items, or those incidentally ingested. We also detected novel microbial eukaryotic taxa and found that parasite assemblage profiles were relatively distinct. Comparing the performance of the methods used supports their joint use for future studies of diversity and abundance. Because viable stages of many parasites are likely to be present in faeces, our results suggest significant levels of bird-mediated dispersal of parasites (both from avian and other hosts). Our methods revealed much hidden biodiversity, and allowed identification of the individuals who produced the faecal samples to species level, facilitating the study of interaction networks.</p><h1>Preservation methods of honey bee-collected pollen are not a source of bias in ITS2 metabarcoding</h1>
<p><a href="https://doi.org/10.1007/S10661-021-09563-4">doi:10.1007/S10661-021-09563-4</a></p>

<p><a href=None>None</a></p>
<p>Pollen metabarcoding is emerging as a powerful tool for ecological research and offers unprecedented scale in citizen science projects for environmental monitoring via honey bees. Biases in metabarcoding can be introduced at any stage of sample processing and preservation is at the forefront of the pipeline. While in metabarcoding studies pollen has been preserved at - 20 °C (FRZ), this is not the best method for citizen scientists. Herein, we compared this method with ethanol (EtOH), silica gel (SG) and room temperature (RT) for preservation of pollen collected from hives in Austria and Denmark. After ~ 4 months of storage, DNAs were extracted with a food kit, and their quality and concentration measured. Most DNA extracts exhibited 260/280 absorbance ratios close to the optimal 1.8, with RT samples from Austria performing slightly worse than FRZ and SG samples (P < 0.027). Statistical differences were also detected for DNA concentration, with EtOH samples producing lower yields than RT and FRZ samples in both countries and SG in Austria (P < 0.042). Yet, qualitative and quantitative assessments of floral composition obtained using high-throughput sequencing with the ITS2 barcode gave non-significant effects of preservation methods on richness, relative abundance and Shannon diversity, in both countries. While freezing and ethanol are commonly employed for archiving tissue for molecular applications, desiccation is cheaper and easier to use regarding both storage and transportation. Since SG is less dependent on ambient humidity and less prone to contamination than RT, we recommend SG for preserving pollen for metabarcoding. SG is straightforward for laymen to use and hence robust for widespread application in citizen science studies.</p><h1>Can moderate heavy metal soil contaminations due to cement production influence the surrounding soil bacterial communities?</h1>
<p><a href="https://doi.org/10.1007/S10646-021-02494-3">doi:10.1007/S10646-021-02494-3</a></p>

<p><a href=None>None</a></p>
<p>Events of soil contamination by heavy metals are mostly related to human activities that release these metals into the environment as emissions or effluents. Among the industrial activities related to heavy metal pollution, cement production plants are considered one of the most common sources. In this work we applied the High-throughput sequencing approach called 16 S rDNA metabarcoding to perform the taxonomic characterization of the prokaryotic communities of the soil surrounding three cement plants as well as two areas outside the influence of the cement plants that represented agricultural production environments free of heavy metal contamination (control areas). We applied the environmental genomics approaches known as "structural community metrics" (α- and β-diversity metrics) and "functional community metrics" (PICRUSt2 approach) to verify whether or not the effects of heavy metal contamination in the study area generated impacts on soil bacterial communities. We found that the impact related to the elevation of heavy metal concentration due to the operation of cement plants in the surrounding soil can be considered smooth according to globally recognized indices such as Igeo. However, we identified that both the taxonomic and functional structures of the communities surrounding cement plants were different from those found in the control areas. We consider that our findings contribute significantly to the general understanding of the effects of heavy metals on the soil ecosystem by showing that light contamination can disturb the dynamics of ecosystem services provided by soil, specifically those associated with microbial metabolism.</p><h1>RNA metabarcoding helps reveal zooplankton community response to environmental stressors</h1>
<p><a href="https://doi.org/S0269-7491(21)02028-5">doi:S0269-7491(21)02028-5</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding can provide a high-throughput and rapid method for characterizing responses of communities to environmental stressors. However, within bulk samples, DNA metabarcoding hardly distinguishes live from the dead organisms. Here, both DNA and RNA metabarcoding were applied and compared in experimental freshwater mesocosms conducted for assessment of ecotoxicological responses of zooplankton communities to remediation treatment until 38 days post oil-spill. Furthermore, a novel indicator of normalized vitality (NV), sequence counts of RNA metabarcoding normalized by that of DNA metabarcoding, was developed for assessment of ecological responses. DNA and RNA metabarcoding detected similar taxa richness and rank of relative abundances. Both DNA and RNA metabarcoding demonstrated slight shifts in measured α-diversities in response to treatments. NV presented relatively greater magnitudes of differential responses of community compositions to treatments compared to DNA or RNA metabarcoding. NV declined from the start of the experiment (3 days pre-spill) to the end (38 days post-spill). NV also differed between Rotifer and Arthropoda, possibly due to differential life histories and sizes of organisms. NV could be a useful indicator for characterizing ecological responses to anthropogenic influence; however, the biology of target organisms and subsequent RNA production need to be considered.</p><h1>Diversity and ecology of protists revealed by metabarcoding</h1>
<p><a href="https://doi.org/S0960-9822(21)01056-3">doi:S0960-9822(21)01056-3</a></p>

<p><a href=None>None</a></p>
<p>Protists are the dominant eukaryotes in the biosphere where they play key functional roles. While protists have been studied for over a century, it is the high-throughput sequencing of molecular markers from environmental samples - the approach of metabarcoding - that has revealed just how diverse, and abundant, these small organisms are. Metabarcoding is now routine to survey environmental diversity, so data have rapidly accumulated from a multitude of environments and at different sampling scales. This mass of data has provided unprecedented opportunities to study the taxonomic and functional diversity of protists, and how this diversity is organised in space and time. Here, we use metabarcoding as a common thread to discuss the state of knowledge in protist diversity research, from technical considerations of the approach to important insights gained on diversity patterns and the processes that might have structured this diversity. In addition to these insights, we conclude that metabarcoding is on the verge of an exciting added dimension thanks to the maturation of high-throughput long-read sequencing, so that a robust eco-evolutionary framework of protist diversity is within reach.</p><h1>Strategies for sample labelling and library preparation in DNA metabarcoding studies</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13512">doi:10.1111/1755-0998.13512</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of DNA extracted from environmental or bulk specimen samples is increasingly used to profile biota in basic and applied biodiversity research because of its targeted nature that allows sequencing of genetic markers from many samples in parallel. To achieve this, PCR amplification is carried out with primers designed to target a taxonomically informative marker within a taxonomic group, and sample-specific nucleotide identifiers are added to the amplicons prior to sequencing. The latter enables assignment of the sequences back to the samples they originated from. Nucleotide identifiers can be added during the metabarcoding PCR and during "library preparation", that is, when amplicons are prepared for sequencing. Different strategies to achieve this labelling exist. All have advantages, challenges and limitations, some of which can lead to misleading results, and in the worst case compromise the fidelity of the metabarcoding data. Given the range of questions addressed using metabarcoding, ensuring that data generation is robust and fit for the chosen purpose is critically important for practitioners seeking to employ metabarcoding for biodiversity assessments. Here, we present an overview of the three main workflows for sample-specific labelling and library preparation in metabarcoding studies on Illumina sequencing platforms; one-step PCR, two-step PCR, and tagged PCR. Further, we distill the key considerations for researchers seeking to select an appropriate metabarcoding strategy for their specific study. Ultimately, by gaining insights into the consequences of different metabarcoding workflows, we hope to further consolidate the power of metabarcoding as a tool to assess biodiversity across a range of applications.</p><h1>A Standardized Procedure for Monitoring Harmful Algal Blooms in Chile by Metabarcoding Analysis</h1>
<p><a href="https://doi.org/10.3791/62967">doi:10.3791/62967</a></p>

<p><a href=None>None</a></p>
<p>Harmful algae blooms (HABs) monitoring has been implemented worldwide, and Chile, a country famous for its fisheries and aquaculture, has intensively used microscopic and toxin analyses for decades for this purpose. Molecular biological methods, such as high-throughput DNA sequencing and bacterial assemblage-based approaches, are just beginning to be introduced in Chilean HAB monitoring, and the procedures have not yet been standardized. Here, 16S rRNA and 18S rRNA metabarcoding analyses for monitoring Chilean HABs are introduced stepwise. According to a recent hypothesis, algal-bacterial mutualistic association plays a critical synergetic or antagonistic relationship accounting for bloom initiation, maintenance, and regression. Thus, monitoring HAB from algal-bacterial perspectives may provide a broader understanding of HAB mechanisms and the basis for early warning. Metabarcoding analysis is one of the best suited molecular-based tools for this purpose because it can detect massive algal-bacterial taxonomic information in a sample. The visual procedures of sampling to metabarcoding analysis herein provide specific instructions, aiming to reduce errors and collection of reliable data.</p><h1>Coming of age for COI metabarcoding of whole organism community DNA: Towards bioinformatic harmonisation</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13502">doi:10.1111/1755-0998.13502</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of DNA extracted from community samples of whole organisms (whole organism community DNA, wocDNA) is increasingly being applied to terrestrial, marine and freshwater metazoan communities to provide rapid, accurate and high resolution data for novel molecular ecology research. The growth of this field has been accompanied by considerable development that builds on microbial metabarcoding methods to develop appropriate and efficient sampling and laboratory protocols for whole organism metazoan communities. However, considerably less attention has focused on ensuring bioinformatic methods are adapted and applied comprehensively in wocDNA metabarcoding. In this study we examined over 600 papers and identified 111 studies that performed COI metabarcoding of wocDNA. We then systematically reviewed the bioinformatic methods employed by these papers to identify the state-of-the-art. Our results show that the increasing use of wocDNA COI metabarcoding for metazoan diversity is characterised by a clear absence of bioinformatic harmonisation, and the temporal trends show little change in this situation. The reviewed literature showed (i) high heterogeneity across pipelines, tasks and tools used, (ii) limited or no adaptation of bioinformatic procedures to the nature of the COI fragment, and (iii) a worrying underreporting of tasks, software and parameters. Based upon these findings we propose a set of recommendations that we think the metabarcoding community should consider to ensure that bioinformatic methods are appropriate, comprehensive and comparable. We believe that adhering to these recommendations will improve the long-term integrative potential of wocDNA COI metabarcoding for biodiversity science.</p><h1>Seeing through sedimented waters: environmental DNA reduces the phantom diversity of sharks and rays in turbid marine habitats</h1>
<p><a href="https://doi.org/10.1186/S12862-021-01895-6">doi:10.1186/S12862-021-01895-6</a></p>

<p><a href=None>None</a></p>
<p>Sharks and rays are some of the most threatened marine taxa due to the high levels of bycatch and significant demand for meat and fin-related products in many Asian communities. At least 25% of shark and ray species are considered to be threatened with extinction. In particular, the density of reef sharks in the Pacific has declined to 3-10% of pre-human levels. Elasmobranchs are thought to be sparse in highly urbanised and turbid environments. Low visibility coupled with the highly elusive behaviour of sharks and rays pose a challenge to diversity estimation and biomonitoring efforts as sightings are limited to chance encounters or from carcasses ensnared in nets. Here we utilised an eDNA metabarcoding approach to enhance the precision of elasmobranch diversity estimates in urbanised marine environments.</p><h1>Environmental DNA sampling protocols for the surveillance of marine non-indigenous species in Irish coastal waters</h1>
<p><a href="https://doi.org/S0025-326X(21)00927-9">doi:S0025-326X(21)00927-9</a></p>

<p><a href=None>None</a></p>
<p>Understanding the spread and distribution of Non-Indigenous Species (NIS) is key when implementing legislation to maintain good ecosystem health. Environmental DNA (eDNA) has shown great potential to detect aquatic organisms in a rapid and cost-effective way, however their applicability to new environments must be validated prior to their implementation. Here, we tested different field sampling methods in combination with eDNA metabarcoding to develop a tool to detect NIS. Large and small volumes of seawater were filtered, in addition to the collection of sediment and horizontal tow net samples at 12 locations across four distinct geographic areas in Ireland. The biggest dissimilarity in the species recovered was found between sediment and town net samples. Tow nets showed to be the most efficient. A total of 357 taxa were identified, including 16 NIS. Fine mesh tow nets were identified as the most cost-efficient for large-scale monitoring and surveillance of NIS.</p><h1>Dung beetles as samplers of mammals in Malaysian Borneo-a test of high throughput metabarcoding of iDNA</h1>
<p><a href="https://doi.org/10.7717/Peerj.11897">doi:10.7717/Peerj.11897</a></p>

<p><a href=None>None</a></p>
<p>Invertebrate-derived DNA (iDNA) sampling in biodiversity surveys is becoming increasingly widespread, with most terrestrial studies relying on DNA derived from the gut contents of blood-feeding invertebrates, such as leeches and mosquitoes. Dung beetles (superfamily Scarabaeoidea) primarily feed on the faecal matter of terrestrial vertebrates and offer several potential benefits over blood-feeding invertebrates as samplers of vertebrate DNA. Importantly, these beetles can be easily captured in large numbers using simple, inexpensive baited traps, are globally distributed, and occur in a wide range of habitats. To build on the few existing studies demonstrating the potential of dung beetles as sources of mammalian DNA, we subjected the large-bodied, Bornean dung beetle (<i>Catharsius renaudpauliani</i>) to a controlled feeding experiment. We analysed DNA from gut contents at different times after feeding using qPCR techniques. Here, we first describe the window of DNA persistence within a dung beetle digestive tract. We found that the ability to successfully amplify cattle DNA decayed over relatively short time periods, with DNA copy number decreasing by two orders of magnitude in just 6 h. In addition, we sampled communities of dung beetles from a lowland tropical rainforest in Sabah, Malaysia, in order to test whether it is possible to identify vertebrate sequences from dung beetle iDNA. We sequenced both the gut contents from large dung beetle species, as well as whole communities of smaller beetles. We successfully identified six mammalian species from our samples, including the bearded pig (<i>Sus barbatus</i>) and the sambar deer (<i>Rusa unicolor</i>)-both vulnerable species on the IUCN red list. Our results represent the first use of dung beetle iDNA to sample Southeast Asian vertebrate fauna, and highlight the potential for dung beetle iDNA to be used in future biodiversity monitoring surveys.</p><h1>Inventory of the benthic eukaryotic diversity in the oldest European lake</h1>
<p><a href="https://doi.org/10.1002/Ece3.7907">doi:10.1002/Ece3.7907</a></p>

<p><a href=None>None</a></p>
<p>We have profound knowledge on biodiversity on Earth including plants and animals. In the recent decade, we have also increased our understanding on microorganisms in different hosts and the environment. However, biodiversity is not equally well studied among different biodiversity groups and Earth's systems with eukaryotes in freshwater sediments being among the least known. In this study, we used high-throughput sequencing of the 18S rRNA gene to investigate the entire diversity of benthic eukaryotes in three distinct habitats (littoral sediment and hard substrate, profundal sediment) of Lake Ohrid, the oldest European lake. Eukaryotic sequences were dominated by annelid and arthropod animals (54% of all eukaryotic reads) and protists (Ochrophyta and Ciliophora; together 40% of all reads). Eukaryotic diversity was 15% higher in the deep profundal than on either near-surface hard substrates or littoral sediments. The three habitats differed in their taxonomic and functional community composition. Specifically, heterotrophic organisms accounted for 92% of the reads in the profundal, whereas phototrophs accounted for 43% on the littoral hard substrate. The profundal community was the most homogeneous, and its network was the most complex, suggesting its highest stability among the sampled habitats.</p><h1>Localized reshaping of the fungal community in response to a forest fungal pathogen reveals resilience of Mediterranean mycobiota</h1>
<p><a href="https://doi.org/S0048-9697(21)04657-X">doi:S0048-9697(21)04657-X</a></p>

<p><a href=None>None</a></p>
<p>Mediterranean forests are facing the impact of pests such as the soilborne Phytophthora cambivora, the causal agent of Ink disease, and this impact is made more severe by global changes. The status and resilience of the soil microbial ecosystem in areas with such a disturbance are little known; however, the assessment of the microbial community is fundamental to preserve the ecosystem functioning under emerging challenges. We profile soil fungal communities in a chestnut stand affected by ink disease in Italy using metabarcoding, and couple high-throughput sequencing with physico-chemical parameters and dendrometric measurements. Since the site also includes an area where the disease symptoms seem to be suppressed, we performed several analyses to search for determinants that may contribute to such difference. We demonstrate that neither pathogen presence nor trees decline associate with the reduction of the residing community diversity and functions, but rather with microbial network reshaping through substitutions and new interactions, despite a conservation of core taxa. We predict interactions between taxa and parameters such as soil pH and C/N ratio, and suggest that disease incidence may also relate with disappearance of pathogen antagonists, including ericoid- and ectomycorrhizal (ECM) fungi. By combining metabarcoding and field studies, we infer the resilient status of the fungal community towards a biotic stressor, and provide a benchmark for the study of other threatened ecosystems.</p><h1>Characterizing parasitic nematode faunas in faeces and soil using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1186/S13071-021-04935-8">doi:10.1186/S13071-021-04935-8</a></p>

<p><a href=None>None</a></p>
<p>Gastrointestinal parasitic nematodes can impact fecundity, development, behaviour, and survival in wild vertebrate populations. Conventional monitoring of gastrointestinal parasitic nematodes in wild populations involves morphological identification of eggs, larvae, and adults from faeces or intestinal samples. Adult worms are typically required for species-level identification, meaning intestinal material from dead animals is needed to characterize the nematode community with high taxonomic resolution. DNA metabarcoding of environmental samples is increasingly used for time- and cost-effective, high-throughput biodiversity monitoring of small-bodied organisms, including parasite communities. Here, we evaluate the potential of DNA metabarcoding of faeces and soil samples for non-invasive monitoring of gastrointestinal parasitic nematode communities in a wild ruminant population.</p><h1>Towards reproducible metabarcoding data: Lessons from an international cross-laboratory experiment</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13485">doi:10.1111/1755-0998.13485</a></p>

<p><a href=None>None</a></p>
<p>Advances in high-throughput sequencing (HTS) are revolutionizing monitoring in marine environments by enabling rapid, accurate and holistic detection of species within complex biological samples. Research institutions worldwide increasingly employ HTS methods for biodiversity assessments. However, variance in laboratory procedures, analytical workflows and bioinformatic pipelines impede the transferability and comparability of results across research groups. An international experiment was conducted to assess the consistency of metabarcoding results derived from identical samples and primer sets using varying laboratory procedures. Homogenized biofouling samples collected from four coastal locations (Australia, Canada, New Zealand and the USA) were distributed to 12 independent laboratories. Participants were asked to follow one of two HTS library preparation workflows. While DNA extraction, primers and bioinformatic analyses were purposefully standardized to allow comparison, many other technical variables were allowed to vary among laboratories (amplification protocols, type of instrument used, etc.). Despite substantial variation observed in raw results, the primary signal in the data was consistent, with the samples grouping strongly by geographical origin for all data sets. Simple post hoc data clean-up by removing low-quality samples gave the best improvement in sample classification for nuclear 18S rRNA gene data, with an overall 92.81% correct group attribution. For mitochondrial COI gene data, the best classification result (95.58%) was achieved after correction for contamination errors. The identified critical methodological factors that introduced the greatest variability (preservation buffer, sample defrosting, template concentration, DNA polymerase, PCR enhancer) should be of great assistance in standardizing future biodiversity studies using metabarcoding.</p><h1>Occurrence and distribution of microbial pollutants in coastal areas of the Adriatic Sea influenced by river discharge</h1>
<p><a href="https://doi.org/S0269-7491(21)01254-9">doi:S0269-7491(21)01254-9</a></p>

<p><a href=None>None</a></p>
<p>The transport of a variety of pollutants from agricultural, industrial and urbanised areas makes rivers major contributors to the contamination of coastal marine environments. Too little is known of their role in carrying pathogens to the coast. We used DNA-based metabarcoding data to describe the microbial community composition in seawater and sediment collected in front of the estuary of the Tronto, the Chienti and the Esino, three Italian rivers with different pollution levels that empty into the north-central Adriatic Sea, and to detect and measure within these communities the relative abundance of microbial pollutants, including traditional faecal indicators and alternative faecal and sewage-associated pollutants. We then applied the FORENSIC algorithm to distinguish human from non-human sources of microbial pollution and FAPROTAX to map prokaryotic clades to established metabolic or other ecologically relevant functions. Finally, we searched the dataset for other common pathogenic taxa. Seawater and sediment contained numerous potentially pathogenic bacteria, mainly faecal and sewage-associated. The samples collected in front of the Tronto estuary showed the highest level of contamination, likely sewage-associated. The pathogenic signature showed a weak but positive correlation with some nutrients and strong correlations with some polycyclic aromatic hydrocarbons. This study confirms that rivers transport pathogenic bacteria to the coastal sea and highlights the value of expanding the use of HTS data, source tracking and functional identification tools to detect microbial pollutants and identify their sources with a view to gaining a better understanding of the pathways of sewage-associated discharges to the sea.</p><h1>Next-Generation Sequencing and Its Impacts on Entomological Research in Ecology and Evolution</h1>
<p><a href="https://doi.org/10.1007/S13744-021-00895-X">doi:10.1007/S13744-021-00895-X</a></p>

<p><a href=None>None</a></p>
<p>The advent of NGS-based methods has been profoundly transforming entomological research. Through continual development and improvement of different methods and sequencing platforms, NGS has promoted mass elucidation of partial or whole genetic materials associated with beneficial insects, pests (of agriculture, forestry and animal, and human health), and species of conservation concern, helping to unravel ecological and evolutionary mechanisms and characterizing survival, trophic interactions, and dispersal. It is shifting the scale of biodiversity and environmental analyses from individuals and biodiversity indicator species to the large-scale study of communities and ecosystems using bulk samples of species or a mixed "soup" of environmental DNA. As the NGS-based methods have become more affordable, complexity demystified, and specificity and sensitivity proven, their use in entomological research has spread widely. This article presents several examples on how NGS-based methods have been used in entomology to provide incentives to apply them when appropriate and to open our minds to the expected advances in entomology that are yet to come.</p><h1>Environmental DNA Metabarcoding: A Novel Method for Biodiversity Monitoring of Marine Fish Communities</h1>
<p><a href="https://doi.org/10.1146/Annurev-Marine-041421-082251">doi:10.1146/Annurev-Marine-041421-082251</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is genetic material that has been shed from macroorganisms. It has received increased attention as an indirect marker for biodiversity monitoring. This article reviews the current status of eDNA metabarcoding (simultaneous detection of multiple species) as a noninvasive and cost-effective approach for monitoring marine fish communities and discusses the prospects for this growing field. eDNA metabarcoding coamplifies short fragments of fish eDNA across a wide variety of taxa and, coupled with high-throughput sequencing technologies, allows massively parallel sequencing to be performed simultaneously for dozens to hundreds of samples. It can predict species richness in a given area, detect habitat segregation and biogeographic patterns from small to large spatial scales, and monitor the spatiotemporal dynamics of fish communities. In addition, it can detect an anthropogenic impact on fish communities through evaluation of their functional diversity. Recognizing the strengths and limitations of eDNA metabarcoding will help ensure that continuous biodiversity monitoring at multiple sites will be useful for ecosystem conservation and sustainable use of fishery resources, possibly contributing to achieving the targets of the United Nations' Sustainable Development Goal 14 for 2030.</p><h1>Illumina iSeq 100 and MiSeq exhibit similar performance in freshwater fish environmental DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1038/S41598-021-95360-5">doi:10.1038/S41598-021-95360-5</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis is a method of detecting DNA from environmental samples and is used as a biomonitoring tool. In recent studies, Illumina MiSeq has been the most extensively used tool for eDNA metabarcoding. The Illumina iSeq 100 (hereafter, iSeq), one of the high-throughput sequencers (HTS), has a relatively simple workflow and is potentially more affordable than other HTS. However, its utility in eDNA metabarcoding has still not been investigated. In the present study, we applied fish eDNA metabarcoding to 40 water samples from river and lake ecosystems to assess the difference in species detectability and composition between iSeq and MiSeq. To check differences in sequence quality and errors, we also assessed differences in read changes between the two HTS. There were similar sequence qualities between iSeq and MiSeq. Significant difference was observed in the number of species between two HTS, but no difference was observed in species composition between the two HTS. Additionally, the species compositions in common with the conventional method were the same between the two HTS. According to the results, using the same amplicon library for sequencing, two HTS would exhibit a similar performance of fish species detection using eDNA metabarcoding.</p><h1>A molecular method for biomonitoring of an exotic plant-pest: Leafmining for environmental DNA</h1>
<p><a href="https://doi.org/10.1111/Mec.16092">doi:10.1111/Mec.16092</a></p>

<p><a href=None>None</a></p>
<p>Understanding how invasive species respond to novel environments is limited by a lack of sensitivity and throughput in conventional biomonitoring methods. Arthropods in particular are often difficult to monitor due to their small size, rapid lifecycles, and/or visual similarities with co-occurring species. This is true for the agromyzid leafminer fly, Liriomyza sativae, a global pest of vegetable and nursery industries that has recently established in Australia. A robust method based on environmental DNA (eDNA) was developed exploiting traces of DNA left inside "empty" leaf mines, which are straightforward to collect and persist longer in the environment than the fly. This extends the window of possible diagnosis to at least 28 days after a leaf mine becomes empty. The test allowed for visually indistinguishable leafmining damage caused by L. sativae to be genetically differentiated from that of other flies. Field application resulted in the identification of new local plant hosts for L. sativae, including widely distributed weeds and common garden crops, which has important implications for the pest's ability to spread. Moreover, the test confirmed the presence of a previously unknown population of L. sativae on an island in the Torres Strait. The developed eDNA method is likely to become an important tool for L. sativae and other leafmining species of biosecurity significance, which, historically, have been difficult to detect, diagnose and monitor. More generally, eDNA is emerging as a highly sensitive and labour-efficient surveillance tool for difficult to survey species to improve outcomes for agricultural industries, global health, and the environment.</p><h1>Description of Microbial Communities of Phosphate Mine Wastes in Morocco, a Semi-Arid Climate, Using High-Throughput Sequencing and Functional Prediction</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2021.666936">doi:10.3389/Fmicb.2021.666936</a></p>

<p><a href=None>None</a></p>
<p>Soil microbiota are vital for successful revegetation, as they play a critical role in nutrient cycles, soil functions, and plant growth and health. A rehabilitation scenario of the abandoned Kettara mine (Morocco) includes covering acidic tailings with alkaline phosphate mine wastes to limit water infiltration and hence acid mine drainage. Revegetation of phosphate wastes is the final step to this rehabilitation plan. However, revegetation is hard on this type of waste in semi-arid areas and only a few plants managed to grow naturally after 5 years on the store-and-release cover. As we know that belowground biodiversity is a key component for aboveground functioning, we sought to know if any structural problem in phosphate waste communities could explain the almost absence of plants. To test this hypothesis, bacterial and archaeal communities present in these wastes were assessed by 16S rRNA metabarcoding. Exploration of taxonomic composition revealed a quite diversified community assigned to 19 Bacterial and two Archaeal phyla, similar to other studies, that do not appear to raise any particular issues of structural problems. The dominant sequences belonged to <i>Proteobacteria</i>, <i>Bacteroidetes</i>, <i>Actinobacteria</i>, and <i>Gemmatimonadetes</i> and to the genera <i>Massilia</i>, <i>Sphingomonas</i>, and <i>Adhaeribacter</i>. LEfSe analysis identified 19 key genera, and metagenomic functional prediction revealed a broader phylogenetic range of taxa than expected, with all identified genera possessing at least one plant growth-promoting trait. Around 47% of the sequences were also related to genera possessing strains that facilitate plant development under biotic and environmental stress conditions, such as drought and heat.</p><h1>Just keep it simple? Benchmarking the accuracy of taxonomy assignment software in metabarcoding studies</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13473">doi:10.1111/1755-0998.13473</a></p>

<p><a href=None>None</a></p>
<p>How do you put a name on an unknown piece of DNA? From microbes to mammals, high-throughput metabarcoding studies provide a more objective view of natural communities, overcoming many of the inherent limitations of traditional field surveys and microscopy-based observations (Deiner et al., 2017). Taxonomy assignment is one of the most critical aspects of any metabarcoding study, yet this important bioinformatics task is routinely overlooked. Biodiversity surveys and conservation efforts often depend on formal species inventories: the presence (or absence) of species, and the number of individuals reported across space and time. However, computational workflows applied in eukaryotic metabarcoding studies were originally developed for use with bacterial/archaeal data sets, where microbial researchers rely on one conserved locus (nuclear 16S rRNA) and have access to vast databases with good coverage across most prokaryotic lineages - a situation not mirrored in most multicellular taxa. In this issue of Molecular Ecology Resources, Hleap et al. (2021) carry out an extensive benchmarking exercise focused on taxonomy assignment strategies for eukaryotic metabarcoding studies utilizing the mitochondrial Cytochrome C oxidase I marker gene (COI). They assess the performance and accuracy of software tools representing diverse methodological approaches: from "simple" strategies based on sequence similarity and composition, to model-based phylogenetic and probabilistic classification tools. Contrary to popular assumptions, less complex approaches (BLAST and the QIIME2 feature classifier) consistently outperformed more sophisticated mathematical algorithms and were highly accurate for assigning taxonomy at higher levels (e.g. family). Lower-level assignments at the genus and species level still pose significant challenge for most existing algorithms, and sparse eukaryotic reference databases further limit software performance. This study illuminates current best practices for metabarcoding taxonomy assignments, and underscores the need for community-driven efforts to expand taxonomic and geographic representation in reference DNA barcode databases.</p><h1>Relative species abundance estimation in artificial mixtures of insects using mito-metagenomics and a correction factor for the mitochondrial DNA copy number</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13464">doi:10.1111/1755-0998.13464</a></p>

<p><a href=None>None</a></p>
<p>Mito-metagenomics (MMG) is becoming an alternative to amplicon metabarcoding for the assessment of biodiversity in complex biological samples using high-throughput sequencing. Whereas MMG overcomes the biases introduced by the PCR step in the generation of amplicons, it is not yet a technique free of shortcomings. First, as the reads are obtained from shotgun sequencing, a very low proportion of reads map into the mitogenomes, so a high sequencing effort is needed. Second, as the number of mitogenomes per cell can vary among species, the relative species abundance (RSA) in a mixture could be wrongly estimated. Here, we challenge the MMG method to estimate the RSA using artificial libraries of 17 insect species whose complete genomes are available on public repositories. With fresh specimens of these species, we created single-species libraries to calibrate the bioinformatic pipeline and mixed-species libraries to estimate the RSA. Our results showed that the MMG approach confidently recovers the species list of the mixtures, even when they contain congeneric species. The method was also able to estimate the abundance of a species across different samples (within-species estimation) but failed to estimate the RSA within a single sample (across-species estimation) unless a correction factor accounting for the variable number of mitogenomes per cell was used. To estimate this correction factor, we used the proportion of reads mapping into mitogenomes in the single-species libraries and the lengths of the whole genomes and mitogenomes.</p><h1>Dumpster diving for diatom plastid 16S rRNA genes</h1>
<p><a href="https://doi.org/10.7717/Peerj.11576">doi:10.7717/Peerj.11576</a></p>

<p><a href=None>None</a></p>
<p>High throughput sequencing is improving the efficiency of monitoring diatoms, which inhabit and support aquatic ecosystems across the globe. In this study, we explored the potential of a standard V4 515F-806RB primer pair in recovering diatom plastid 16S rRNA sequences. We used PhytoREF to classify the 16S reads from our freshwater biofilm field sampling from three stream segments across two streams in south-eastern Australia and retrieved diatom community data from other, publicly deposited, Australian 16S amplicon datasets. When these diatom operational taxonomic units (OTUs) were traced using the default RDPII and NCBI databases, 68% were characterized as uncultured cyanobacteria. We analysed the 16S rRNA sequences from 72 stream biofilm samples, separated the chloroplast OTUs, and classified them using the PhytoREF database. After filtering the reads attributed to Bacillariophyta (relative abundance >1%), 71 diatom OTUs comprising more than 90% of the diatom reads in each stream biofilm sample were identified. Beta-diversity analyses demonstrated significantly different diatom assemblages and discrimination among river segments. To further test the approach, the diatom OTUs from our biofilm sampling were used as reference sequences to identify diatom reads from other Australian 16S rRNA datasets in the NCBI-SRA database. Across the three selected public datasets, 67 of our 71 diatom OTUs were detected in other Australian ecosystems. Our results show that diatom plastid 16S rRNA genes are readily amplified with existing 515F-806RB primer sets. Therefore, the volume of existing 16S rRNA amplicon datasets initially generated for microbial community profiling can also be used to detect, characterize, and map diatom distribution to inform phylogeny and ecological health assessments, and can be extended into a range of ecological and industrial applications. To our knowledge, this study represents the first attempt to classify freshwater samples using this approach and the first application of PhytoREF in Australia.</p><h1>eDNA metabarcoding revealed differential structures of aquatic communities in a dynamic freshwater ecosystem shaped by habitat heterogeneity</h1>
<p><a href="https://doi.org/S0013-9351(21)00896-3">doi:S0013-9351(21)00896-3</a></p>

<p><a href=None>None</a></p>
<p>Freshwater ecosystems have been threatened by complicated disturbances from both natural and anthropogenic variables, especially in dynamic and complex river basins. The environmental DNA (eDNA)-based approach provides a broader spectrum and higher throughput way of biomonitoring for biodiversity assessment compared with traditional morphological survey. Most eDNA metabarcoding studies have been limited to a few specific taxa/groups and habitat scopes. Here we applied the eDNA metabarcoding to characterize the structures and spatial variations of zooplankton and fish communities among different habitat types in a highly dynamic and complex freshwater ecosystem of the Daqing River basin (DRB). The results showed that varied species spectra of zooplankton and fish communities were identified and unique dominant species occurred across habitats. Additionally, markedly spatial distributions of biotic community structures were observed in areas with different habitat characteristics. Natural variables, including geographic distances and gradient ratio, as well as anthropogenic factors of chemical oxygen demand (COD) and organic chemicals demonstrated significant effects but different outcomes on the structures of zooplankton and fish communities. Moreover, the relative abundances of specific aquatic taxa were associated with the gradient of particular environmental variables. This case study verified the distribution patterns and differentiation mechanisms of biotic communities under habitat heterogeneity could be captured by application of eDNA biomonitoring. And habitat-specific and even species-specific environmental stressors would be diagnosed for improving management of complex river basins.</p><h1>Phospholipid fatty acid (PLFA) analysis as a tool to estimate absolute abundances from compositional 16S rRNA bacterial metabarcoding data</h1>
<p><a href="https://doi.org/S0167-7012(21)00139-1">doi:S0167-7012(21)00139-1</a></p>

<p><a href=None>None</a></p>
<p>Microbial biodiversity monitoring through the analysis of DNA extracted from environmental samples is increasingly popular because it is perceived as being rapid, cost-effective, and flexible concerning the sample types studied. DNA can be extracted from diverse media before high-throughput sequencing of the prokaryotic 16S rRNA gene is used to characterize the taxonomic diversity and composition of the sample (known as metabarcoding). While sources of bias in metabarcoding methodologies are widely acknowledged, previous studies have focused mainly on the effects of these biases within a single substrate type, and relatively little is known of how these vary across substrates. We investigated the effect of substrate type (water, microbial mats, lake sediments, stream sediments, soil and a mock microbial community) on the relative performance of DNA metabarcoding in parallel with phospholipid fatty acid (PLFA) analysis. Quantitative estimates of the biomass of different taxonomic groups in samples were made through the analysis of PLFAs, and these were compared to the relative abundances of microbial taxa estimated from metabarcoding. Furthermore, we used the PLFA-based quantitative estimates of the biomass to adjust relative abundances of microbial groups determined by metabarcoding to provide insight into how the biomass of microbial taxa from PLFA analysis can improve understanding of microbial communities from environmental DNA samples. We used two sets of PLFA biomarkers that differed in their number of PLFAs to evaluate how PLFA biomarker selection influences biomass estimates. Metabarcoding and PLFA analysis provided significantly different views of bacterial composition, and these differences varied among substrates. We observed the most notable differences for the Gram-negative bacteria, which were overrepresented by metabarcoding in comparison to PLFA analysis. In contrast, the relative biomass and relative sequence abundances aligned reasonably well for Cyanobacteria across the tested freshwater substrates. Adjusting relative abundances of microbial taxa estimated by metabarcoding with PLFA-based quantification estimates of the microbial biomass led to significant changes in the microbial community compositions in all substrates. We recommend including independent estimates of the biomass of microbial groups to increase comparability among metabarcoding libraries from environmental samples, especially when comparing communities associated with different substrates.</p><h1>On the phenology of protists: recurrent patterns reveal seasonal variation of protistan (Rhizaria: Cercozoa and Endomyxa) communities in tree canopies</h1>
<p><a href="https://doi.org/Fiab081">doi:Fiab081</a></p>

<p><a href=None>None</a></p>
<p>Tree canopies are colonized by billions of highly specialized microorganisms that are well adapted to the highly variable microclimatic conditions, caused by diurnal fluctuations and seasonal changes. In this study, we investigated seasonality patterns of protists in the tree canopies of a temperate floodplain forest via high-throughput sequencing with group-specific primers for the phyla Cercozoa and Endomyxa. We observed consistent seasonality, and identified divergent spring and autumn taxa. Tree crowns were characterized by a dominance of bacterivores and omnivores, while eukaryvores gained a distinctly larger share in litter and soil communities on the ground. In the canopy seasonality was largest among communities detected on the foliar surface: In spring, higher variance within alpha diversity of foliar samples indicated greater heterogeneity during initial colonization. However, communities underwent compositional changes during the aging of leaves in autumn, highly reflecting recurring phenological changes during protistan colonization. Surprisingly, endomyxan root pathogens appeared to be exceptionally abundant across tree canopies during autumn, demonstrating a potential role of the canopy surface as a physical filter for air-dispersed propagules. Overall, about 80% of detected OTUs could not be assigned to known species-representing dozens of microeukaryotic taxa whose canopy inhabitants are waiting to be discovered.</p><h1>Consistent declines in aquatic biodiversity across diverse domains of life in rivers impacted by surface coal mining</h1>
<p><a href="https://doi.org/10.1002/Eap.2389">doi:10.1002/Eap.2389</a></p>

<p><a href=None>None</a></p>
<p>The rivers of Appalachia (United States) are among the most biologically diverse freshwater ecosystems in the temperate zone and are home to numerous endemic aquatic organisms. Throughout the Central Appalachian ecoregion, extensive surface coal mines generate alkaline mine drainage that raises the pH, salinity, and trace element concentrations in downstream waters. Previous regional assessments have found significant declines in stream macroinvertebrate and fish communities after draining these mined areas. Here, we expand these assessments with a more comprehensive evaluation across a broad range of organisms (bacteria, algae, macroinvertebrates, all eukaryotes, and fish) using high-throughput amplicon sequencing of environmental DNA (eDNA). We collected water samples from 93 streams in Central Appalachia (West Virginia, United States) spanning a gradient of mountaintop coal mining intensity and legacy to assess how this land use alters downstream water chemistry and affects aquatic biodiversity. For each group of organisms, we identified the sensitive and tolerant taxa along the gradient and calculated stream specific conductivity thresholds in which large synchronous declines in diversity were observed. Streams below mining operations had steep declines in diversity (-18 to -41%) and substantial shifts in community composition that were consistent across multiple taxonomic groups. Overall, large synchronous declines in bacterial, algal, and macroinvertebrate communities occurred even at low levels of mining impact at stream specific conductivity thresholds of 150-200 µS/cm that are substantially below the current U.S. Environmental Protection Agency aquatic life benchmark of 300 µS/cm for Central Appalachian streams. We show that extensive coal surface mining activities led to the extirpation of 40% of biodiversity from impacted rivers throughout the region and that current water quality criteria are likely not protective for many groups of aquatic organisms.</p><h1>Assessing the in situ bacterial diversity and composition at anthropogenically active sites using the environmental DNA (eDNA)</h1>
<p><a href="https://doi.org/S0025-326X(21)00627-5">doi:S0025-326X(21)00627-5</a></p>

<p><a href=None>None</a></p>
<p>In this study, we identified the in situ bacterial groups and their community structure in coastal waters influenced by anthropogenic inputs. The use of environmental DNA (eDNA) and high throughput sequencing (HTS) were employed to derive accurate and reliable information on bacterial abundance. The V3 and V4 hypervariable regions of the 16S rRNA gene were amplified and the sequences were clustered into operational taxonomic units to analyze the site-specific variations in community composition. The percentage composition within the bacterial orders varied significantly among nearshore anthropogenic hotspots and offshore (5 km) samples. The microbial network constructed taking the bacterial abundance as nodes displayed strong positive and negative correlations within the bacterial families. Overall, the use of eDNA coupled with HTS is an incredible means for monitoring and assessing the abundance of bacterial communities and also serves as a biomonitoring tool to understand the degree of anthropogenic contamination in coastal waters.</p><h1>The largest moss carpet transplant in Antarctica and its bryosphere cryptic biodiversity</h1>
<p><a href="https://doi.org/10.1007/S00792-021-01235-Y">doi:10.1007/S00792-021-01235-Y</a></p>

<p><a href=None>None</a></p>
<p>As part of the reconstruction of the Brazilian Antarctic Station on King George Island, three areas of moss carpet were transplanted to minimize the impact of the new facilities on the local biodiversity. A total of 650 m<sup>2</sup> of moss carpet was transplanted to neighboring but previously uncolonized locations and has subsequently survived for the last 3 years. Antarctic moss carpets typically comprise low moss species diversity and are often monospecific. We investigated the cryptic biodiversity that was transplanted along with the carpets using a metabarcoding approach through high throughput sequencing. We targeted 16S rRNA for Bacteria and Archaea, ITS for Fungi and Viridiplantae and Cox1 for Metazoa. We detected DNA representing 263 taxa from five Kingdoms (Chromista, Fungi, Metazoa, Protista and Viridiplantae), two Domains (Archaea and Bacteria) and 33 Phyla associated with the carpet. This diversity included one Archaea, 189 Bacteria, 24 Chromista, 19 Fungi, eight Metazoa, seven Protista and 16 Viridiplantae. Bacteria was the most abundant, rich and diverse group, with Chromista second in diversity and richness. Metazoa was less diverse but second highest in dominance. This is the first study to attempt transplanting a significant area of moss carpet to minimize anthropogenic environmental damage in Antarctica and to use metabarcoding as a proxy to assess diversity associated with Antarctic moss carpets, further highlighting the importance of such habitats for other organisms and their importance for conservation.</p><h1>Assessing the ecological quality status of the highly polluted Bagnoli area (Tyrrhenian Sea, Italy) using foraminiferal eDNA metabarcoding</h1>
<p><a href="https://doi.org/S0048-9697(21)02942-9">doi:S0048-9697(21)02942-9</a></p>

<p><a href=None>None</a></p>
<p>Morphology-based benthic foraminifera indices are increasingly used worldwide for biomonitoring the ecological quality of marine sediments. The recent development of foraminiferal eDNA metabarcoding offers a reliable, time-, and cost-effective alternative to morphology-based foraminiferal biomonitoring. However, the practical applications of these new tools are still highly limited. In the present study, we evaluate the response of benthic foraminifera and define the ecological quality status (EcoQS) in the Bagnoli area (Tyrrhenian Sea, Italy) based on a traditional morphology-based approach and eDNA metabarcoding. The geochemical data show that several sites in front of the former industrial plant contain higher concentrations of potentially toxic elements than the effect range median and are characterized by the highest total organic carbon (TOC) content, whereas the distantly located sites can be considered relatively low- to unpolluted. Significant differences (i.e., diversity and assemblage composition) in both morphological and molecular datasets were found between the relatively low- to unpolluted and the most polluted areas. Similarly, the selected ecological indices of both morphological and molecular datasets strikingly and congruently resulted in a clear separation following the environmental stress gradient. The molecular indices (i.e., g-exp(H'<sub>bc</sub>), g-Foram AMBI, and g-Foram AMBI-MOTUs) reliably identified poor-to-bad EcoQS in the polluted area in front of the former industrial plant. On the other hand, the Foram-AMBI based on morphology well identified an overall trend but seemed to overestimate the EcoQS if the traditional class boundaries were considered. The congruent and complementary trends between morphological and metabarcoding data observed in the case of the Bagnoli site further support the application of foraminiferal metabarcoding in routine biomonitoring to assess the environmental impacts of heavily polluted marine areas.</p><h1>Winogradsky columns as a strategy to study typically rare microbial eukaryotes</h1>
<p><a href="https://doi.org/S0932-4739(21)00042-0">doi:S0932-4739(21)00042-0</a></p>

<p><a href=None>None</a></p>
<p>Winogradsky columns have been widely used to study soil microbial communities, but the vast majority of those investigations have focused on the ecology and diversity of bacteria. In contrast, microbial eukaryotes (ME) have been regularly overlooked in studies based on experimental soil columns. Despite the recognized ecological relevance of ME in soil communities, investigations focused on ME diversity and the abundance of certain groups of interest are still scarce. In the present study, we used DNA metabarcoding (high-throughput sequencing of the V4 region of the 18S rRNA locus) to survey the ME diversity and abundance in an experimental Winogradsky soil column. Consistent with previous surveys in natural soils, our survey identified members of Cercozoa (Rhizaria; 31.2%), Apicomplexa and Ciliophora (Alveolata; 12.5%) as the predominant ME groups, but at particular depths we also detected the abundant presence of ME lineages that are typically rare in natural environments, such as members of the Vampyrellida (Rhizaria) and Breviatea (Amorphea). Our survey demonstrates that experimental soil columns are an efficient enrichment-culture approach that can enhance investigations about the diversity and ecology of ME in soils.</p><h1>To the canopy and beyond: Air dispersal as a mechanism of ubiquitous protistan pathogen assembly in tree canopies</h1>
<p><a href="https://doi.org/S0932-4739(21)00040-7">doi:S0932-4739(21)00040-7</a></p>

<p><a href=None>None</a></p>
<p>Cercozoa and Oomycota contain a huge biodiversity and important pathogens of forest trees and other vegetation. We analyzed air dispersal of these protistan phyla with an air sampler near-ground (~2 m) and in tree crowns (~25 m) of three tree species (oak, linden and ash) in a temperate floodplain forest in March (before leafing) and May (after leaf unfolding) 2019 with a cultivation-independent high-throughput metabarcoding approach. We found a high diversity of Cercozoa and Oomycota in air samples with 122 and 81 OTUs, respectively. Especially oomycetes showed a significant difference in community composition between both sampling dates. Differences in community composition between air samples in tree canopies and close to the ground were however negligible, and also tree species identity did not affect communities in air samples, indicating that the distribution of protistan propagules through the air was not spatially restricted in the forest ecosystem. OTUs of plant pathogens, whose host species did not occur in the forest, demonstrate dispersal of propagules from outside the forest biome. Overall, our results lead to a better understanding of the stochastic processes of air dispersal of protists and protistan pathogens, a prerequisite to understand the mechanisms of their community assembly in forest ecosystems.</p><h1>DNA metabarcoding reveals the significant influence of anthropogenic effects on microeukaryotic communities in urban waterbodies</h1>
<p><a href="https://doi.org/S0269-7491(21)00918-0">doi:S0269-7491(21)00918-0</a></p>

<p><a href=None>None</a></p>
<p>Biological monitoring and assessment are the first and most fundamental steps towards diagnosing ecological or environmental quality. Increasing anthropogenic impact on urban ecosystems has prompted the development of less expensive and more efficient bioassessment approaches. Generally, a morphospecies based approach is effective for plants and large organisms but challenging for the microbial biosphere. To overcome this challenge, we used high-throughput DNA sequencing for predicting anthropogenic effects on microeukaryotic communities in urban waterbodies along a pollution gradient in Wuhan City, central China in summer 2019. Our results indicated that microeukaryotic community structure was distinct between non-urban polluted reservoir and urban polluted waterbodies. The heterogeneity of environmental condition significantly affected the microeukaryotic diversity, community structure, and species interactions. Integrated co-occurring network analysis revealed that the pollution gradient has a significant adverse impact on network complexity and network dissimilarity. These results revealed that the significant variation in anthropogenically-driven environmental condition shaped microeukaryotic communities in urban freshwater ecosystems. Furthermore, we observed that the relative abundance of indicative OTUs were significantly and negatively correlated with pollution level and these indicative OTUs could be used to predict the water quality status with up to 77% success. Thus, our multiple approaches combining 18S rDNA amplicon sequencing, co-occurring network and indicator species analyses suggest that this study gives a novel approach based on microeukaryotic communities to assess and predict the water quality status of urban aquatic environments.</p><h1>The diversity of soil mesofauna declines after bamboo invasion in subtropical China</h1>
<p><a href="https://doi.org/S0048-9697(21)03053-9">doi:S0048-9697(21)03053-9</a></p>

<p><a href=None>None</a></p>
<p>Plant invasions often act as ecosystem 'simplifiers' to simplify diversity and community structure of soil biota. However, inconsistent relationships between plant invasion and soil fauna have been found and few studies have addressed how soil fauna communities change upon plant invasions across taxa and feeding guilds. Here, we investigated the effects of moso bamboo (Phyllostachys edulis) invasion in subtropical China on soil mesofauna communities using novel high-throughput sequencing (HTS). Specifically, we analyzed the spatio-temporal dynamics of fauna diversity and feeding guilds in the litter and soil layers for three stages of moso bamboo invasion, i.e., uninvaded (secondary broadleaved forest), moderately invaded (mixed bamboo forest) and completely invaded (P. edulis forest). Overall, we found that the completely invaded bamboo forest decreased species richness and diversity of total fauna, herbivores, and microbivores consistently across different soil layers, but less so detritivores and predators. Although we did not find any interaction effects of bamboo invasion and soil layers on soil fauna diversity indices, significant interaction effects were found on the community composition, for total fauna and their feeding guilds. Specifically, the detrimental effects of bamboo invasion on the trophic structure of soil fauna communities were more profound in the litter layer than in the soil layer, suggesting that a litter layer with more diverse taxa does not mean higher resistance to plant invasion in maintaining the soil food web structure. Taken together, our findings suggest that different responses within fauna feeding guilds to plant invasion were pervasive, and a deeper soil layer may better alleviate the negative effects of pant invasion on fauna community structure. These shifts in soil biodiversity may further degrade ecosystem functioning.</p><h1>High-throughput sequencing on preservative ethanol is effective at jointly examining infraspecific and taxonomic diversity, although bioinformatics pipelines do not perform equally</h1>
<p><a href="https://doi.org/10.1002/Ece3.7453">doi:10.1002/Ece3.7453</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of amplicons (HTSA) has been proposed as an effective approach to evaluate taxonomic and genetic diversity at the same time. However, there are still uncertainties as to how the results produced by different bioinformatics treatments impact the conclusions drawn on biodiversity and population genetics indices.We evaluated the ability of six bioinformatics pipelines to recover taxonomic and genetic diversity from HTSA data obtained from controlled assemblages. To that end, 20 assemblages were produced using 354 colonies of <i>Botrylloides</i> spp., sampled in the wild in ten marinas around Brittany (France). We used DNA extracted from preservative ethanol (ebDNA) after various time of storage (3, 6, and 12 months), and from a bulk of preserved specimens (bulkDNA). DNA was amplified with primers designed for targeting this ascidian genus. Results obtained from HTSA data were compared with Sanger sequencing on individual zooids (i.e., individual barcoding).Species identification and relative abundance determined with HTSA data from either ebDNA or bulkDNA were similar to those obtained with traditional individual barcoding. However, after 12 months of storage, the correlation between HTSA and individual-based data was lower than after shorter durations. The six bioinformatics pipelines were able to depict accurately the genetic diversity using standard population genetics indices (H<sub>S</sub> and F<sub>ST</sub>), despite producing false positives and missing rare haplotypes. However, they did not perform equally and dada2 was the only pipeline able to retrieve all expected haplotypes.This study showed that ebDNA is a nondestructive alternative for both species identification and haplotype recovery, providing storage does not last more than 6 months before DNA extraction. Choosing the bioinformatics pipeline is a matter of compromise, aiming to retrieve all true haplotypes while avoiding false positives. We here recommend to process HTSA data using dada2, including a chimera-removal step. Even if the possibility to use multiplexed primer sets deserves further investigation to expand the taxonomic coverage in future similar studies, we showed that primers targeting a particular genus allowed to reliably analyze this genus within a complex community.</p><h1>Sampling forest soils to describe fungal diversity and composition. Which is the optimal sampling size in mediterranean pure and mixed pine oak forests?</h1>
<p><a href="https://doi.org/S1878-6146(21)00012-X">doi:S1878-6146(21)00012-X</a></p>

<p><a href=None>None</a></p>
<p>Soil sampling is a critical step affecting perceived fungal diversity, however sampling optimization for high-throughput-DNA sequencing studies have never been tested in Mediterranean forest ecosystems. We identified the minimum number of pooled samples needed to obtain a reliable description of fungal communities in terms of diversity and composition in three different Mediterranean forests (pine, oak, and mixed-pine-oak). Twenty soil samples were randomly selected in each of the three plots per type. Samples obtained in 100 m<sup>2</sup> plots were pooled to obtain mixtures of 3, 6, 10, 15, 20 samples, and sequenced using Illumina MiSeq of fungal ITS2 amplicons. Pooling three soil samples in Pinus and Quercus stands provided consistent richness estimations, while at least six samples were needed in mixed-stands. β-diversity decreased with increasing sample pools in monospecific-stands, while there was no effect of sample pool size on mixed-stands. Soil sample pooling had no effect over species composition. We estimate that three samples would be already optimal to describe fungal richness and composition in Mediterranean pure stands, while at least six samples would be needed in mixed stands.</p><h1>Using zooplankton metabarcoding to assess the efficacy of different techniques to clean-up an oil-spill in a boreal lake</h1>
<p><a href="https://doi.org/S0166-445X(21)00106-5">doi:S0166-445X(21)00106-5</a></p>

<p><a href=None>None</a></p>
<p>Regulators require adequate information to select best practices with less ecosystem impacts for remediation of freshwater ecosystems after oil spills. Zooplankton are valuable indicators of aquatic ecosystem health as they play pivotal roles in biochemical cycles while stabilizing food webs. Compared with morphological identification, metabarcoding holds promise for cost-effective, high-throughput, and benchmarkable biomonitoring of zooplankton communities. The objective of this study was to apply DNA and RNA metabarcoding of zooplankton for ecotoxicological assessment and compare it with traditional morphological identification in experimental shoreline enclosures in a boreal lake. These identification methods were also applied in context of assessing response of the zooplankton community exposed to simulated spills of diluted bitumen (dilbit), with experimental remediation practices (enhanced monitored natural recovery and shoreline cleaner application). Metabarcoding detected boreal zooplankton taxa up to the genus level, with a total of 24 shared genera, and while metabarcoding-based relative abundance served as an acceptable proxy for biomass inferred by morphological identification (ρ ≥ 0.52). Morphological identification determined zooplankton community composition changes due to treatments at 11 days post-spill (PERMANOVA, p = 0.0143) while metabarcoding methods indicated changes in zooplankton richness and communities at 38 days post-spill (T-test, p < 0.05; PERMANOVA, p ≤ 0.0429). Shoreline cleaner application overall seemed to have the largest impact on zooplankton communities relative to enhanced monitored natural recovery, regardless of zooplankton identification method. Both metabarcoding and morphological identification were able to discern the differences between the two experimental remediation practices. Metabarcoding of zooplankton could provide informative results for ecotoxicological assessment of the remediation practices of dilbit, advancing our knowledge of best practices for remediating oil-impacted aquatic ecosystems while serving to accelerate the assessment of at-risk freshwater ecosystems.</p><h1>Profile hidden Markov model sequence analysis can help remove putative pseudogenes from DNA barcoding and metabarcoding datasets</h1>
<p><a href="https://doi.org/10.1186/S12859-021-04180-X">doi:10.1186/S12859-021-04180-X</a></p>

<p><a href=None>None</a></p>
<p>Pseudogenes are non-functional copies of protein coding genes that typically follow a different molecular evolutionary path as compared to functional genes. The inclusion of pseudogene sequences in DNA barcoding and metabarcoding analysis can lead to misleading results. None of the most widely used bioinformatic pipelines used to process marker gene (metabarcode) high throughput sequencing data specifically accounts for the presence of pseudogenes in protein-coding marker genes. The purpose of this study is to develop a method to screen for nuclear mitochondrial DNA segments (nuMTs) in large COI datasets. We do this by: (1) describing gene and nuMT characteristics from an artificial COI barcode dataset, (2) show the impact of two different pseudogene removal methods on perturbed community datasets with simulated nuMTs, and (3) incorporate a pseudogene filtering step in a bioinformatic pipeline that can be used to process Illumina paired-end COI metabarcode sequences. Open reading frame length and sequence bit scores from hidden Markov model (HMM) profile analysis were used to detect pseudogenes.</p><h1>Potential applications of CRISPR/Cas for next-generation biomonitoring of harmful algae blooms: A review</h1>
<p><a href="https://doi.org/S1568-9883(21)00056-1">doi:S1568-9883(21)00056-1</a></p>

<p><a href=None>None</a></p>
<p>Research on harmful algal and cyanobacterial blooms (HABs and CHABs) has risen dramatically due to their increasing global distribution, frequency, and intensity. These blooms jeopardize public health, ecosystem function, sustainability and can have negative economic impacts. Numerous monitoring programs have been established using light microscopy, liquid chromatography coupled to mass spectrometry (LC-MS), ELISA, and spectrophotometry to monitor HABs/CHABs outbreaks. Recently, DNA/RNA-based molecular methods have been integrated into these programs to replace or complement traditional methods through analyzing environmental DNA and RNA (eDNA/eRNA) with techniques such as quantitative polymerase chain reaction (qPCR), fluorescent in situ hybridization (FISH), sandwich hybridization assay (SHA), isothermal amplification methods, and microarrays. These have enabled the detection of rare or cryptic species, enhanced sample throughput, and reduced costs and the need for visual taxonomic expertise. However, these methods have limitations, such as the need for high capital investment in equipment or detection uncertainties, including determining whether organisms are viable. In this review, we discuss the potential of newly developed molecular diagnosis technology based on Clustered Regularly Interspaced Short Palindromic Repeats/Cas proteins (CRISPR/Cas), which utilizes the prokaryotic adaptative immune systems of bacteria and archaea. Cas12 and Cas13-based platforms can detect both DNA and RNA with attomolar sensitivity within an hour. CRISPR/Cas diagnostic is a rapid, inexpensive, specific, and ultrasensitive technology that, with some further development, will provide many new platforms that can be used for HABs/CHABs biomonitoring and research.</p><h1>DNA metabarcoding and morphological methods show complementary patterns in the metacommunity organization of lentic epiphytic diatoms</h1>
<p><a href="https://doi.org/S0048-9697(21)02481-5">doi:S0048-9697(21)02481-5</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are important organisms in freshwater ecosystems due to their position as primary producers and therefore, analyzing their assemblages provides relevant information on ecosystem functioning. Diatoms have historically been identified based on morphological traits, which is time-consuming and requires well-trained specialists. Nevertheless, DNA barcoding offers an alternative approach to overcome some limitations of the morphological method. Here, we assess if both approaches are comparable methods to study patterns and mechanisms (including environmental filtering and dispersal limitation) of epiphytic diatom metacommunities using a comprehensive dataset from 22 Mediterranean ponds at different taxonomic resolutions. We used a fragment of rbcL barcode gene combined with High-Throughput Sequencing to infer diatom community composition. The overall degree of correspondence between both approaches was assessed by Procrustean rotation analysis and Procrustean randomization tests, whereas the role of local environmental variables and geographical distances was studied using a comprehensive combination of BIOENV, Mantel tests and distance-based redundancy analysis. Our results showed a relatively poor correspondence in the compositional variation of diatom metacommunity between both approaches. We speculate that the incompleteness of the reference database and the bioinformatics processing are the biases most likely affecting the molecular approach, whereas the limited counting effort and the presence of cryptic species are presumably the major biases related with the morphological method. On the other hand, variation in diatom community composition detected with both approaches was strongly related to the environmental template, which may be related with the narrow community-environment relationships in diatoms. Nevertheless, we found no significant relationship between compositional variation and geographical distances. Overall, our work shows the complementary nature of both approaches and highlights the importance of DNA metabarcoding to address empirical research questions of community ecology in freshwaters, especially once the reference databases include most genotypes of occurring taxa and bioinformatics biases are overcome.</p><h1>Using next generation sequencing of alpine plants to improve fecal metabarcoding diet analysis for Dall's sheep</h1>
<p><a href="https://doi.org/10.1186/S13104-021-05590-Z">doi:10.1186/S13104-021-05590-Z</a></p>

<p><a href=None>None</a></p>
<p>Dall's sheep (Ovis dalli dalli) are important herbivores in the mountainous ecosystems of northwestern North America, and recent declines in some populations have sparked concern. Our aim was to improve capabilities for fecal metabarcoding diet analysis of Dall's sheep and other herbivores by contributing new sequence data for arctic and alpine plants. This expanded reference library will provide critical reference sequence data that will facilitate metabarcoding diet analysis of Dall's sheep and thus improve understanding of plant-animal interactions in a region undergoing rapid climate change.</p><h1>The aerobiome uncovered: Multi-marker metabarcoding reveals potential drivers of turn-over in the full microbial community in the air</h1>
<p><a href="https://doi.org/S0160-4120(21)00176-8">doi:S0160-4120(21)00176-8</a></p>

<p><a href=None>None</a></p>
<p>Air is a major conduit for the dispersal of organisms at the local and the global scale. Most research has focused on the dispersal of plants, vertebrates and human disease agents. However, the air represents a key dispersal medium also for bacteria, fungi and protists. Many of those represent potential pathogens of animals and plants and have until now gone largely unrecorded. Here we studied the turnover in composition of the entire aerobiome, the collective diversity of airborne microorganisms. For that we performed daily analyses of all prokaryotes and eukaryotes (including plants) using multi-marker high-throughput sequencing for a total of three weeks. We linked the resulting communities to local weather conditions, to assess determinants of aerobiome composition and distribution. We observed hundreds of microbial taxa, mostly belonging to spore-forming organisms including fungi, but also protists. Additionally, we detected many potential human- and plant-pathogens. Community composition fluctuated on a daily basis and was linked to concurrent weather conditions, particularly air pressure and temperature. Using network analyses, we identified taxonomically diverse groups of organisms with correlated temporal dynamics. In part, this was due to co-variation with environmental conditions, while we could also detect specific host-parasite interactions. This study provides the first full inventory of the aerobiome and identifies putative drivers of its dynamics in terms of taxon composition. This knowledge can help develop early warning systems against pathogens and improve our understanding of microbial dispersal.</p><h1>DNA Metabarcoding Reveals Broad Presence of Plant Pathogenic Oomycetes in Soil From Internationally Traded Plants</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2021.637068">doi:10.3389/Fmicb.2021.637068</a></p>

<p><a href=None>None</a></p>
<p>Plants with roots and soil clumps transported over long distances in plant trading can harbor plant pathogenic oomycetes, facilitating disease outbreaks that threaten ecosystems, biodiversity, and food security. Tools to detect the presence of such oomycetes with a sufficiently high throughput and broad scope are currently not part of international phytosanitary testing regimes. In this work, DNA metabarcoding targeting the internal transcribed spacer (ITS) region was employed to broadly detect and identify oomycetes present in soil from internationally shipped plants. This method was compared to traditional isolation-based detection and identification after an enrichment step. DNA metabarcoding showed widespread presence of potentially plant pathogenic <i>Phytophthora</i> and <i>Pythium</i> species in internationally transported rhizospheric soil with <i>Pythium</i> being the overall most abundant genus observed. Baiting, a commonly employed enrichment method for <i>Phytophthora</i> species, led to an increase of golden-brown algae in the soil samples, but did not increase the relative or absolute abundance of potentially plant pathogenic oomycetes. Metabarcoding of rhizospheric soil yielded DNA sequences corresponding to oomycete isolates obtained after enrichment and identified them correctly but did not always detect the isolated oomycetes in the same samples. This work provides a proof of concept and outlines necessary improvements for the use of environmental DNA (eDNA) and metabarcoding as a standalone phytosanitary assessment tool for broad detection and identification of plant pathogenic oomycetes.</p><h1>Diversity of fungal DNA in lake sediments on Vega Island, north-east Antarctic Peninsula assessed using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1007/S00792-021-01226-Z">doi:10.1007/S00792-021-01226-Z</a></p>

<p><a href=None>None</a></p>
<p>We assessed the diversity of fungal DNA present in sediments of three lakes on Vega Island, north-east Antarctic Peninsula using metabarcoding through high-throughput sequencing (HTS). A total of 640,902 fungal DNA reads were detected, which were assigned to 224 taxa of the phyla Ascomycota, Rozellomycota, Basidiomycota, Chytridiomycota and Mortierellomycota, in rank order of abundance. The most abundant genera were Pseudogymnoascus, Penicillium and Mortierella. However, a majority (423,508, 66%) of the reads, representing by 43 ASVs, could only be assigned at higher taxonomic levels and may represent taxa not currently included in the sequence databases used or be new or previously unreported taxa present in Antarctic lakes. The three lakes were characterized by high sequence diversity, richness, and moderate dominance indices. The ASVs were dominated by psychrotolerant and cosmopolitan cold-adapted Ascomycota, Basidiomycota and Mortierellomycota commonly reported in Antarctic environments. However, other taxa detected included unidentified members of Rozellomycota and Chytridiomycota species not previously reported in Antarctic lakes. The assigned diversity was composed mainly of taxa recognized as decomposers and pathogens of plants and invertebrates.</p><h1>Mosquito Identification From Bulk Samples Using DNA Metabarcoding: a Protocol to Support Mosquito-Borne Disease Surveillance in Canada</h1>
<p><a href="https://doi.org/10.1093/Jme/Tjab046">doi:10.1093/Jme/Tjab046</a></p>

<p><a href=None>None</a></p>
<p>Approximately 80 species of mosquitoes (Diptera: Culicidae) have been documented in Canada. Exotic species such as Aedes albopictus (Skuse) (Diptera: Culicidae) are becoming established. Recently occurring endemic mosquito-borne diseases (MBD) in Canada including West-Nile virus (WNV) and Eastern Equine Encephalitis (EEE) are having significant public health impacts. Here we explore the use of DNA metabarcoding to identify mosquitoes from CDC light-trap collections from two locations in eastern Canada. Two primer pairs (BF2-BR2 and F230) were used to amplify regions of the cytochrome c oxidase subunit I (CO1) gene. High throughput sequencing was conducted using an Illumina MiSeq platform and GenBank-based species identification was applied using a QIIME 1.9 bioinformatics pipeline. From a site in southeastern Ontario, Canada, 26 CDC light trap collections of 72 to >300 individual mosquitoes were used to explore the capacity of DNA metabarcoding to identify and quantify captured mosquitoes. The DNA metabarcoding method identified 33 species overall while 24 species were identified by key. Using replicates from each trap, the dried biomass needed to identify the majority of species was determined to be 76 mg (equivalent to approximately 72 mosquitoes), and at least two replicates from the dried biomass would be needed to reliably detect the majority of species in collections of 144-215 mosquitoes and three replicates would be advised for collections with >215 mosquitoes. This study supports the use of DNA metabarcoding as a mosquito surveillance tool in Canada which can help identify the emergence of new mosquito-borne disease potential threats.</p><h1>Microbial community and abiotic effects on aquatic bacterial communities in north temperate lakes</h1>
<p><a href="https://doi.org/S0048-9697(21)01839-8">doi:S0048-9697(21)01839-8</a></p>

<p><a href=None>None</a></p>
<p>The aquatic bacterial community (BC) plays a vital role in determining the nature and rate of ecosystem function. However, the biotic and abiotic factors influencing BC structure and function are largely unknown. Hence, the current study characterizes the impact of biotic and abiotic factors on aquatic bacterial biodiversity to determine whether the dominant effects are biotic or abiotic by partitioning their relative effects across temperate Canadian lakes. We collected water samples from sixty southern Ontario lakes and characterized their BC and microbial eukaryotic community (MEC) compositions using high throughput metabarcode sequencing of 16S and 18S rRNA gene fragments. The diversity and richness of aquatic BCs differed considerably among our study lakes, and those differences were explained by environmental, spatial, and biotic (MEC) factors (31%, 23%, and 23% of variance explained, respectively). The relatively large contribution from biotic and abiotic factors (54%), relative to spatial effects, shows deterministic processes prevail in shaping BC assembly in freshwater lakes. However, spatial effects also contributed significantly, highlighting the role of stochastic processes (ecological drift and coupled with limited dispersal) in shaping BC structure. Furthermore, our co-occurrence network analysis showed strong positive and negative interactions within and between the BCs and MECs, indicating mutualistic or antagonistic co-occurrence patterns relationships play important roles in driving the variation in BC composition among our sampled lakes. Considered together, our community analyses show that deterministic and stochastic processes combined contribute to determining the aquatic BC composition, and hence likely function as well, across a broad array of temperate freshwater lakes.</p><h1>Culture-Dependent and Amplicon Sequencing Approaches Reveal Diversity and Distribution of Black Fungi in Antarctic Cryptoendolithic Communities</h1>
<p><a href="https://doi.org/213">doi:213</a></p>

<p><a href=None>None</a></p>
<p>In the harshest environmental conditions of the Antarctic desert, normally incompatible with active life, microbes are adapted to exploit the cryptoendolithic habitat (i.e., pore spaces of rocks) and represent the predominant life-forms. In the rocky niche, microbes take advantage of the thermal buffering, physical stability, protection against UV radiation, excessive solar radiation, and water retention-of paramount importance in one of the driest environments on Earth. In this work, high-throughput sequencing and culture-dependent approaches have been combined, for the first time, to untangle the diversity and distribution of black fungi in the Antarctic cryptoendolithic microbial communities, hosting some of the most extreme-tolerant microorganisms. Rock samples were collected in a vast area, along an altitudinal gradient and opposite sun exposure-known to influence microbial diversity-with the aim to compare and integrate results gained with the two approaches. Among black fungi, <i>Friedmanniomyces endolithicus</i> was confirmed as the most abundant taxon. Despite the much stronger power of the high-throughput sequencing, several species were not retrieved with DNA sequencing and were detectable by cultivation only. We conclude that both culture-dependent and -independent analyses are needed for a complete overview of black fungi diversity. The reason why some species remain undetectable with molecular methods are speculated upon. The effect of environmental parameters such as sun exposure on relative abundance was clearer if based on the wider biodiversity detected with the molecular approach.</p><h1>eDNA metabarcoding of small plankton samples to detect fish larvae and their preys from Atlantic and Pacific waters</h1>
<p><a href="https://doi.org/10.1038/S41598-021-86731-Z">doi:10.1038/S41598-021-86731-Z</a></p>

<p><a href=None>None</a></p>
<p>Zooplankton community inventories are the basis of fisheries management for containing fish larvae and their preys; however, the visual identification of early-stage larvae (the "missing biomass") is difficult and laborious. Here, eDNA metabarcoding was employed to detect zooplankton species of interest for fisheries from open and coastal waters. High-Throughput sequencing (HTS) from environmental samples using small water volumes has been proposed to detect species of interest whose DNA is the most abundant. We analyzed 6-L water samples taken from subtropical and tropical waters using Cytochrome oxidase I (COI) gene as metabarcode. In the open ocean, several commercial fish larvae and invertebrate species important in fish diet were found from metabarcodes and confirmed from individual barcoding. Comparing Atlantic, Mediterranean, Red Sea, and Pacific samples we found a lower taxonomic depth of OTU assignments in samples from tropical waters than in those from temperate ones, suggesting large gaps in reference databases for those areas; thus a higher effort of zooplankton barcoding in tropical oceans is highly recommended. This and similar simplified sampling protocols could be applied in early detection of species important for fisheries.</p><h1>Metabarcoding Reveals Changes in Benthic Eukaryote and Prokaryote Community Composition along a Tropical Marine Sediment Nickel Gradient</h1>
<p><a href="https://doi.org/10.1002/Etc.5039">doi:10.1002/Etc.5039</a></p>

<p><a href=None>None</a></p>
<p>The Southeast Asia and Melanesia region has extensive nickel (Ni)-rich lateritic regoliths formed from the tropical weathering of ultramafic rocks. As the global demand for Ni continues to rise, these lateritic regoliths are increasingly being exploited for their economic benefit. Mining of these regoliths contributes to the enrichment of coastal sediments in trace metals, especially Ni. The present study used high-throughput sequencing (metabarcoding) to determine changes in eukaryote (18s v7 recombinant DNA [rDNA] and diatom-specific subregion of the 18s v4 rDNA) and prokaryote (16s v4 rDNA) community compositions along a sediment Ni concentration gradient offshore from a large lateritized ultramafic regolith in New Caledonia (Vavouto Bay). Significant changes in the eukaryote, diatom, and prokaryote community compositions were found along the Ni concentration gradient. These changes correlated most with the dilute-acid extractable concentration of Ni in the sediments, which explained 26, 23, and 19% of the variation for eukaryote, diatom, and prokaryote community compositions, respectively. Univariate analyses showed that there was no consistent change in indices of biodiversity, evenness, or richness. Diatom richness and diversity did, however, decrease as sediment acid extractable-Ni concentrations increased. Threshold indicator taxa analysis was conducted separately for each of the 3 targeted genes to detect changes in taxa whose occurrences decreased or increased along the acid extractable-Ni concentration gradient. Based on these data, 46 mg acid extractable-Ni/kg was determined as a threshold value where sensitive species began to disappear. In the case of the estuarine sediments offshore from lateritized ultramafic regolith in New Caledonia, this is recommended as an interim threshold value until further lines of evidence can contribute to a region-specific Ni sediment quality guideline value. Environ Toxicol Chem 2021;40:1894-1907. © 2021 SETAC.</p><h1>Debar: A sequence-by-sequence denoiser for COI-5P DNA barcode data</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13384">doi:10.1111/1755-0998.13384</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding and metabarcoding are now widely used to advance species discovery and biodiversity assessments. High-throughput sequencing (HTS) has expanded the volume and scope of these analyses, but elevated error rates introduce noise into sequence records that can inflate estimates of biodiversity. Denoising -the separation of biological signal from instrument (technical) noise-of barcode and metabarcode data currently employs abundance-based methods which do not capitalize on the highly conserved structure of the cytochrome c oxidase subunit I (COI) region employed as the animal barcode. This manuscript introduces debar, an R package that utilizes a profile hidden Markov model to denoise indel errors in COI sequences introduced by instrument error. In silico studies demonstrated that debar recognized 95% of artificially introduced indels in COI sequences. When applied to real-world data, debar reduced indel errors in circular consensus sequences obtained with the Sequel platform by 75%, and those generated on the Ion Torrent S5 by 94%. The false correction rate was less than 0.1%, indicating that debar is receptive to the majority of true COI variation in the animal kingdom. In conclusion, the debar package improves DNA barcode and metabarcode workflows by aiding the generation of more accurate sequences aiding the characterization of species diversity.</p><h1>DNA Metabarcoding for the Characterization of Terrestrial Microbiota-Pitfalls and Solutions</h1>
<p><a href="https://doi.org/361">doi:361</a></p>

<p><a href=None>None</a></p>
<p>Soil-borne microbes are major ecological players in terrestrial environments since they cycle organic matter, channel nutrients across trophic levels and influence plant growth and health. Therefore, the identification, taxonomic characterization and determination of the ecological role of members of soil microbial communities have become major topics of interest. The development and continuous improvement of high-throughput sequencing platforms have further stimulated the study of complex microbiota in soils and plants. The most frequently used approach to study microbiota composition, diversity and dynamics is polymerase chain reaction (PCR), amplifying specific taxonomically informative gene markers with the subsequent sequencing of the amplicons. This methodological approach is called DNA metabarcoding. Over the last decade, DNA metabarcoding has rapidly emerged as a powerful and cost-effective method for the description of microbiota in environmental samples. However, this approach involves several processing steps, each of which might introduce significant biases that can considerably compromise the reliability of the metabarcoding output. The aim of this review is to provide state-of-the-art background knowledge needed to make appropriate decisions at each step of a DNA metabarcoding workflow, highlighting crucial steps that, if considered, ensures an accurate and standardized characterization of microbiota in environmental studies.</p><h1>Simultaneous absolute quantification and sequencing of fish environmental DNA in a mesocosm by quantitative sequencing technique</h1>
<p><a href="https://doi.org/10.1038/S41598-021-83318-6">doi:10.1038/S41598-021-83318-6</a></p>

<p><a href=None>None</a></p>
<p>The combination of high-throughput sequencing technology and environmental DNA (eDNA) analysis has the potential to be a powerful tool for comprehensive, non-invasive monitoring of species in the environment. To understand the correlation between the abundance of eDNA and that of species in natural environments, we have to obtain quantitative eDNA data, usually via individual assays for each species. The recently developed quantitative sequencing (qSeq) technique enables simultaneous phylogenetic identification and quantification of individual species by counting random tags added to the 5' end of the target sequence during the first DNA synthesis. Here, we applied qSeq to eDNA analysis to test its effectiveness in biodiversity monitoring. eDNA was extracted from water samples taken over 4 days from aquaria containing five fish species (Hemigrammocypris neglectus, Candidia temminckii, Oryzias latipes, Rhinogobius flumineus, and Misgurnus anguillicaudatus), and quantified by qSeq and microfluidic digital PCR (dPCR) using a TaqMan probe. The eDNA abundance quantified by qSeq was consistent with that quantified by dPCR for each fish species at each sampling time. The correlation coefficients between qSeq and dPCR were 0.643, 0.859, and 0.786 for H. neglectus, O. latipes, and M. anguillicaudatus, respectively, indicating that qSeq accurately quantifies fish eDNA.</p><h1>eDNAFlow, an automated, reproducible and scalable workflow for analysis of environmental DNA sequences exploiting Nextflow and Singularity</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13356">doi:10.1111/1755-0998.13356</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of environmental DNA (eDNA) when coupled with high throughput sequencing is revolutionising the way biodiversity can be monitored across a wide range of applications. However, the large number of tools deployed in downstream bioinformatic analyses often places a challenge in configuration and maintenance of a workflow, and consequently limits the research reproducibility. Furthermore, scalability needs to be considered to handle the growing amount of data due to increase in sequence output and the scale of project. Here, we describe eDNAFlow, a fully automated workflow that employs a number of state-of-the-art applications to process eDNA data from raw sequences (single-end or paired-end) to generation of curated and noncurated zero-radius operational taxonomic units (ZOTUs) and their abundance tables. This pipeline is based on Nextflow and Singularity which enable a scalable, portable and reproducible workflow using software containers on a local computer, clouds and high-performance computing (HPC) clusters. Finally, we present an in-house Python script to assign taxonomy to ZOTUs based on user specified thresholds for assigning lowest common ancestor (LCA). We demonstrate the utility and efficiency of the pipeline using an example of a published coral diversity biomonitoring study. Our results were congruent with the aforementioned study. The scalability of the pipeline is also demonstrated through analysis of a large data set containing 154 samples. To our knowledge, this is the first automated bioinformatic pipeline for eDNA analysis using two powerful tools: Nextflow and Singularity. This pipeline addresses two major challenges in the analysis of eDNA data; scalability and reproducibility.</p><h1>Temporal and Spatial Blood Feeding Patterns of Urban Mosquitoes in the San Juan Metropolitan Area, Puerto Rico</h1>
<p><a href="https://doi.org/129">doi:129</a></p>

<p><a href=None>None</a></p>
<p>Urban ecosystems are a patchwork of habitats that host a broad diversity of animal species. Insects comprise a large portion of urban biodiversity which includes many pest species, including those that transmit pathogens. Mosquitoes (Diptera: Culicidae) inhabit urban environments and rely on sympatric vertebrate species to complete their life cycles, and in this process transmit pathogens to animals and humans. Given that mosquitoes feed upon vertebrates, they can also act as efficient samplers that facilitate detection of vertebrate species that utilize urban ecosystems. In this study, we analyzed DNA extracted from mosquito blood meals collected temporally in multiple neighborhoods of the San Juan Metropolitan Area, Puerto Rico to evaluate the presence of vertebrate fauna. DNA was collected from 604 individual mosquitoes that represented two common urban species, <i>Culex quinquefasciatus</i> (<i>n</i> = 586) and <i>Aedes aegypti</i> (<i>n</i> = 18). <i>Culex quinquefasciatus</i> fed on 17 avian taxa (81.2% of blood meals), seven mammalian taxa (17.9%), and one reptilian taxon (0.85%). Domestic chickens dominated these blood meals both temporally and spatially, and no statistically significant shift from birds to mammals was detected. <i>Aedes aegypti</i> blood meals were from a less diverse group, with two avian taxa (11.1%) and three mammalian taxa (88.9%) identified. The blood meals we identified provided a snapshot of the vertebrate community in the San Juan Metropolitan Area and have potential implications for vector-borne pathogen transmission.</p><h1>The influence of intraspecific sequence variation during DNA metabarcoding: A case study of eleven fungal species</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13329">doi:10.1111/1755-0998.13329</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding has become a powerful approach for analysing complex communities from environmental samples, but there are still methodological challenges limiting its full potential. While conserved DNA markers, like 16S and 18S, often are not able to discriminate among closely related species, other more variable markers - like the fungal ITS region, may include considerable intraspecific variation, which can lead to oversplitting of species during DNA metabarcoding analyses. Here we assessed the effects of intraspecific sequence variation in DNA metabarcoding by analysing local populations of eleven fungal species. We investigated the allelic diversity of ITS2 haplotypes using both Sanger sequencing and high throughput sequencing (HTS) coupled with error correction with the software dada2. All the eleven species, except one, included some level of intraspecific variation in the ITS2 region. Overall, we observed a high correspondence between haplotypes generated by Sanger sequencing and HTS, with the exception of a few additional haplotypes detected using either approach. These extra haplotypes, typically occurring in low frequencies, were probably due to PCR and sequencing errors or intragenomic variation in the rDNA region. The presence of intraspecific (and possibly intragenomic) variation in ITS2 suggest that haplotypes (or ASVs) should not be used as basic units in ITS-based fungal community analyses, but an extra clustering step is needed to approach species-level resolution.</p><h1>A multi-kingdom metabarcoding study on cattle grazing Alpine pastures discloses intra-seasonal shifts in plant selection and faecal microbiota</h1>
<p><a href="https://doi.org/10.1038/S41598-020-79474-W">doi:10.1038/S41598-020-79474-W</a></p>

<p><a href=None>None</a></p>
<p>Diet selection by grazing livestock may affect animal performance as well as the biodiversity of grazed areas. Recent DNA barcoding techniques allow to assess dietary plant composition in faecal samples, which may be additionally integrated by the description of gut microbiota. In this high throughput metabarcoding study, we investigated the diversity of plant, fungal and bacterial taxa in faecal samples of lactating cows of two breeds grazing an Alpine semi-natural grassland during summer. The estimated plant composition of the diet comprised 67 genera and 39 species, which varied remarkably during summer, suggesting a decline of the diet forage value with the advancing of the vegetative season. The fungal community included Neocallimastigomycota gut symbionts, but also Ascomycota and Basidiomycota plant parasite and coprophilous taxa, likely ingested during grazing. The proportion of ingested fungi was remarkably higher than in other studies, and varied during summer, although less than that observed for plants. Some variation related to breed was also detected. The gut bacterial taxa remained stable through the summer but displayed a breed-specific composition. The study provided insights in the reciprocal organisms' interactions affecting, and being affected by, the foraging behaviour: plants showed a high temporal variation, fungi a smaller one, while bacteria had practically none; conversely, the same kingdoms showed the opposite gradient of variation as respect to the animal host breed, as bacteria revealed to be the group mostly characterized by host-specificity.</p><h1>Connecting high-throughput biodiversity inventories: Opportunities for a site-based genomic framework for global integration and synthesis</h1>
<p><a href="https://doi.org/10.1111/Mec.15797">doi:10.1111/Mec.15797</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing (HTS) is increasingly being used for the characterization and monitoring of biodiversity. If applied in a structured way, across broad geographical scales, it offers the potential for a much deeper understanding of global biodiversity through the integration of massive quantities of molecular inventory data generated independently at local, regional and global scales. The universality, reliability and efficiency of HTS data can potentially facilitate the seamless linking of data among species assemblages from different sites, at different hierarchical levels of diversity, for any taxonomic group and regardless of prior taxonomic knowledge. However, collective international efforts are required to optimally exploit the potential of site-based HTS data for global integration and synthesis, efforts that at present are limited to the microbial domain. To contribute to the development of an analogous strategy for the nonmicrobial terrestrial domain, an international symposium entitled "Next Generation Biodiversity Monitoring" was held in November 2019 in Nicosia (Cyprus). The symposium brought together evolutionary geneticists, ecologists and biodiversity scientists involved in diverse regional and global initiatives using HTS as a core tool for biodiversity assessment. In this review, we summarize the consensus that emerged from the 3-day symposium. We converged on the opinion that an effective terrestrial Genomic Observatories network for global biodiversity integration and synthesis should be spatially led and strategically united under the umbrella of the metabarcoding approach. Subsequently, we outline an HTS-based strategy to collectively build an integrative framework for site-based biodiversity data generation.</p><h1>Genotype by sequencing: An alternative new method to amplicon metabarcoding and shotgun metagenomics for the assessment of eukaryote biodiversity</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13320">doi:10.1111/1755-0998.13320</a></p>

<p><a href=None>None</a></p>
<p>The use of high-throughput DNA sequencing (HTS) has revolutionized the assessment of biodiversity in plant and animal communities. There are two main approaches to estimate the identity and the relative species abundance (RSA) in complex mixtures using HTS: amplicon metabarcoding and shotgun metagenomics. While amplicon metabarcoding targets one or a few genomic regions, shotgun metagenomics randomly explores the genome of the species. In this issue of Molecular Ecology Resources, Wagemaker et al. (2021) present a new method, multi-species Genotyping by Sequencing (msGBS), as an alternative middle ground between metabarcoding and metagenomics. They apply the technique to mixtures of plant roots and report the remarkable capacity of msGBS to estimate the RSA. If validated in other laboratories and biological communities, msGBS might become a third method to explore the biodiversity of biological communities, especially of plants, where current techniques are struggling to get sufficient taxonomic resolution.</p><h1>Assessment of fungal diversity present in lakes of Maritime Antarctica using DNA metabarcoding: a temporal microcosm experiment</h1>
<p><a href="https://doi.org/10.1007/S00792-020-01212-X">doi:10.1007/S00792-020-01212-X</a></p>

<p><a href=None>None</a></p>
<p>We evaluated the fungal diversity in two lakes on the South Shetland Islands, using DNA metabarcoding through high-throughput sequencing (HTS). A microcosm experiment was deployed for two consecutive years in lakes on Deception and King George islands to capture potential decomposer freshwater fungi. Analyses of the baits revealed 258,326 DNA reads distributed in 34 fungal taxa of the phyla Ascomycota, Basidiomycota, Mortierellomycota, Chytridiomycota and Rozellomycota. Tetracladium marchalianum, Tetracladium sp., Rozellomycota sp., Fungal sp. 1 and Fungal sp. 2 were the most common taxa detected. However, the majority of the communities comprised intermediate and rare taxa. Both fungal communities displayed moderate indices of diversity, richness and dominance. Only six taxa were detected in both lakes, including the most dominant T. marchalianum and Tetracladium sp. The high numbers of reads of the known aquatic saprotrophic hyphomycetes T. marchalianum and Tetracladium sp. in the baits suggest that these fungi may digest organic material in Antarctic lakes, releasing available carbon and nutrients to the other aquatic organisms present in the complex lake food web. Our data confirm that the use of cotton baits together with HTS approaches can be appropriate to study the diversity of resident freshwater fungi present in Antarctic lakes.</p><h1>From Forest Soil to the Canopy: Increased Habitat Diversity Does Not Increase Species Richness of Cercozoa and Oomycota in Tree Canopies</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2020.592189">doi:10.3389/Fmicb.2020.592189</a></p>

<p><a href=None>None</a></p>
<p>Tree canopies provide habitats for diverse and until now, still poorly characterized communities of microbial eukaryotes. One of the most general patterns in community ecology is the increase in species richness with increasing habitat diversity. Thus, environmental heterogeneity of tree canopies should be an important factor governing community structure and diversity in this subsystem of forest ecosystems. Nevertheless, it is unknown if similar patterns are reflected at the microbial scale within unicellular eukaryotes (protists). In this study, high-throughput sequencing of two prominent protistan taxa, Cercozoa (Rhizaria) and Oomycota (Stramenopiles), was performed. Group specific primers were used to comprehensively analyze their diversity in various microhabitats of a floodplain forest from the forest floor to the canopy region. Beta diversity indicated highly dissimilar protistan communities in the investigated microhabitats. However, the majority of operational taxonomic units (OTUs) was present in all samples, and therefore differences in beta diversity were mainly related to species performance (i.e., relative abundance). Accordingly, habitat diversity strongly favored distinct protistan taxa in terms of abundance, but due to their almost ubiquitous distribution the effect of species richness on community composition was negligible.</p><h1>Progress in the use of DNA barcodes in the identification and classification of medicinal plants</h1>
<p><a href="https://doi.org/S0147-6513(20)31528-1">doi:S0147-6513(20)31528-1</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding is an emerging molecular identification and classification technology that has been applied to medicinal plants since 2008. The application of this technique has greatly ensured the safety and effectiveness of medicinal materials. In this paper, we review the application of DNA barcoding and some related technologies over the past 10 years with respect to improving our knowledge of medicinal plant identification and authentication. From single locus-based DNA barcodes to combined markers to genome-scale levels, DNA barcodes contribute more and more genetic information. At the same time, other technologies, such as high-resolution melting (HRM), have been combined with DNA barcoding. With the development of next-generation sequencing (NGS), metabarcoding technology has also been shown to identify species in mixed samples successfully. As a widely used and effective tool, DNA barcoding will become more useful over time in the field of medicinal plants.</p><h1>Vertebrate-Aedes aegypti and Culex quinquefasciatus (Diptera)-arbovirus transmission networks: Non-human feeding revealed by meta-barcoding and next-generation sequencing</h1>
<p><a href="https://doi.org/10.1371/Journal.Pntd.0008867">doi:10.1371/Journal.Pntd.0008867</a></p>

<p><a href=None>None</a></p>
<p>Aedes aegypti mosquito-borne viruses including Zika (ZIKV), dengue (DENV), yellow fever (YFV), and chikungunya (CHIKV) have emerged and re-emerged globally, resulting in an elevated burden of human disease. Aedes aegypti is found worldwide in tropical, sub-tropical, and temperate areas. The characterization of mosquito blood meals is essential to understand the transmission dynamics of mosquito-vectored pathogens.</p><h1>Metabarcoding on both environmental DNA and RNA highlights differences between fungal communities sampled in different habitats</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0244682">doi:10.1371/Journal.Pone.0244682</a></p>

<p><a href=None>None</a></p>
<p>In recent years, metabarcoding has become a key tool to describe microbial communities from natural and artificial environments. Thanks to its high throughput nature, metabarcoding efficiently explores microbial biodiversity under different conditions. It can be performed on environmental (e)DNA to describe so-called total microbial community, or from environmental (e)RNA to describe active microbial community. As opposed to total microbial communities, active ones exclude dead or dormant organisms. For what concerns Fungi, which are mostly filamentous microorganisms, the relationship between DNA-based (total) and RNA-based (active) communities is unclear. In the present study, we evaluated the consequences of performing metabarcoding on both soil and wood-extracted eDNA and eRNA to delineate molecular operational taxonomic units (MOTUs) and differentiate fungal communities according to the environment they originate from. DNA and RNA-based communities differed not only in their taxonomic composition, but also in the relative abundances of several functional guilds. From a taxonomic perspective, we showed that several higher taxa are globally more represented in either "active" or "total" microbial communities. We also observed that delineation of MOTUs based on their co-occurrence among DNA and RNA sequences highlighted differences between the studied habitats that were overlooked when all MOTUs were considered, including those identified exclusively by eDNA sequences. We conclude that metabarcoding on eRNA provides original functional information on the specific roles of several taxonomic or functional groups that would not have been revealed using eDNA alone.</p><h1>Changes in soil microbial communities in post mine ecological restoration: Implications for monitoring using high throughput DNA sequencing</h1>
<p><a href="https://doi.org/S0048-9697(20)35791-0">doi:S0048-9697(20)35791-0</a></p>

<p><a href=None>None</a></p>
<p>The ecological restoration of ecosystem services and biodiversity is a key intervention used to reverse the impacts of anthropogenic activities such as mining. Assessment of the performance of restoration against completion criteria relies on biodiversity monitoring. However, monitoring usually overlooks soil microbial communities (SMC), despite increased awareness of their pivotal role in many ecological functions. Recent advances in cost, scalability and technology has led to DNA sequencing being considered as a cost-effective biological monitoring tool, particularly for otherwise difficult to survey groups such as microbes. However, such approaches for monitoring complex restoration sites such as post-mined landscapes have not yet been tested. Here we examine bacterial and fungal communities across chronosequences of mine site restoration at three locations in Western Australia to determine if there are consistent changes in SMC diversity, community composition and functional capacity. Although we detected directional changes in community composition indicative of microbial recovery, these were inconsistent between locations and microbial taxa (bacteria or fungi). Assessing functional diversity provided greater understanding of changes in site conditions and microbial recovery than could be determined through assessment of community composition alone. These results demonstrate that high-throughput amplicon sequencing of environmental DNA (eDNA) is an effective approach for monitoring the complex changes in SMC following restoration. Future monitoring of mine site restoration using eDNA should consider archiving samples to provide improved understanding of changes in communities over time. Expansion to include other biological groups (e.g. soil fauna) and substrates would also provide a more holistic understanding of biodiversity recovery.</p><h1>Monitoring and Surveillance of Aerial Mycobiota of Rice Paddy through DNA Metabarcoding and qPCR</h1>
<p><a href="https://doi.org/E372">doi:E372</a></p>

<p><a href=None>None</a></p>
<p>The airborne mycobiota has been understudied in comparison with the mycobiota present in other agricultural environments. Traditional, culture-based methods allow the study of a small fraction of the organisms present in the atmosphere, thus missing important information. In this study, the aerial mycobiota in a rice paddy has been examined during the cropping season (from June to September 2016) using qPCRs for two important rice pathogens (<i>Pyricularia oryzae</i> and <i>Bipolaris oryzae</i>) and by using DNA metabarcoding of the fungal ITS region. The metabarcoding results demonstrated a higher alpha diversity (Shannon-Wiener diversity index H' and total number of observed species) at the beginning of the trial (June), suggesting a higher level of community complexity, compared with the end of the season. The main taxa identified by HTS analysis showed a shift in their relative abundance that drove the cluster separation as a function of time and temperature. The most abundant OTUs corresponded to genera such as <i>Cladosporium</i>, <i>Alternaria, Myrothecium</i>, or <i>Pyricularia</i>. Changes in the mycobiota composition were clearly dependent on the average air temperature with a potential impact on disease development in rice. In parallel, oligotyping analysis was performed to obtain a sub-OTU identification which revealed the presence of several oligotypes of <i>Pyricularia</i> and <i>Bipolaris</i> with relative abundance changing during monitoring.</p><h1>Taxonomic Diversity of Pico-/Nanoeukaryotes Is Related to Dissolved Oxygen and Productivity, but Functional Composition Is Shaped by Limiting Nutrients in Eutrophic Coastal Oceans</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2020.601037">doi:10.3389/Fmicb.2020.601037</a></p>

<p><a href=None>None</a></p>
<p>Pico-/nanoeukaryotes (P/NEs) comprise both primary producers and bacterial predators, playing important biogeochemical and ecological roles in the marine microbial loop. Besides the difference in size, these small-sized fractions can be distinguished from microplankton by certain functional and ecological traits. Nevertheless, little information is available regarding patterns of their taxonomic and functional diversity and community composition along environmental gradients in coastal marine ecosystems. In this study, we applied high-throughput sequencing of 18S rRNA gene to assess the taxonomic species richness and community composition of P/NEs in surface waters of Bohai Sea and North Yellow Sea, northern China spanning a 600-km distance during summer and winter of 2011. The richness of operational taxonomic units (OTUs) formed a U-shaped relationship with concentration of chlorophyll <i>a</i> (Chl-<i>a</i>, a proxy of primary productivity), but a stronger, negative relationship with concentration of dissolved oxygen (DO). These two factors also significantly co-varied with the OTU-based community composition of P/NEs. The effect of geographic distance on community composition of P/NEs was negligible. Among the three functional groups defined by trophic traits, heterotrophs had the highest OTU richness, which exhibited a U-shaped relationship with both DO and Chl-<i>a</i>. The community of P/NEs was dominated by heterotrophs and mixotrophs in terms of read numbers, which showed a trade-off along the gradient of phosphate, but no significant changes along DO and Chl-<i>a</i> gradients, indicating functional redundancy. Similarly, the proportion of phototrophs was significantly and positively correlated with the concentration of silicate. Our results indicate that taxonomic and functional composition of P/NEs are decoupled on a regional scale, and limiting nutrients are important factors in modulating functional composition of these microorganisms in the studied area. These findings contribute toward gaining a better understanding of how diversity of small eukaryotes and their functions are structured in coastal oceans and the effect of environmental changes on the structuring process.</p><h1>Food Resource Sharing of Alder Leaf Beetle Specialists (Coleoptera: Chrysomelidae) as Potential Insect-Plant Interface for Horizontal Transmission of Endosymbionts</h1>
<p><a href="https://doi.org/10.1093/Ee/Nvaa111">doi:10.1093/Ee/Nvaa111</a></p>

<p><a href=None>None</a></p>
<p>Recent studies suggest that endosymbionts of herbivore insects can be horizontally transferred to other herbivores feeding on the same host plants, whereby the plant acts as an intermediate stage in the chain of transmission. If this mechanism operates, it is also expected that insect communities sharing the same host plant will have higher chances to share their endosymbionts. In this study, we use a high-throughput 16S rRNA metabarcoding approach to investigate the presence, diversity, and potential sharing of endosymbionts in several species of leaf beetles (Coleoptera: Chrysomelidae) of a local community specialized on an alder diet in North America. Rickettsia and Wolbachia were predominant in the sample, with strong evidence for each species having their own dominant infection, of either or both types of bacteria. However, all species shared a much lower proportion of a particular Wolbachia type, compatible with the same strain dominant in one of the species of leaf beetles. Crucially, the same 16S rRNA haplotype of Wolbachia was found on alder leaf extracts. The combined evidence and the absence of this strain in a syntopic species of leaf beetle feeding on a different host plant support the hypothesis that at least the initial stages of the mechanism that would allow horizontal transmission of endosymbionts across species feeding on the same plant is possible. The accessibility and characteristics of endosymbiont associations of this system make it suitable for deeper analyses of their diversity and transmission in natural conditions.</p><h1>The pico-sized Mamiellophyceae and a novel Bathycoccus clade from the summer plankton of Russian Arctic Seas and adjacent waters</h1>
<p><a href="https://doi.org/Fiaa251">doi:Fiaa251</a></p>

<p><a href=None>None</a></p>
<p>Global climate changes and anthropogenic activity greatly impact Arctic marine biodiversity including phytoplankton which contribute greatly to atmospheric oxygen production. Thus the study of microalgae has rising topicality. Class Mamiellophyceae is an important component of phototrophic picoplankton. To gain more knowledge about Mamiellophyceae distribution and diversity special studies were performed in such remote areas as the Russian Arctic seas. A metabarcoding of pico-sized Mamiellophyceae was undertaken by high-throughput sequencing of the 18S rRNA gene sequence V4 region from samples collected in July-September 2017 in the Barents, Kara and Laptev seas, and in the adjacent waters of the Norwegian Sea. Our study is the first to show that Mamiellophyceae among the summer picoplankton of Russian Arctic seas are diverse and represented by 16 algae species/phylotypes. We discovered a new candidate species of Bathycoccus assigned to a new Bathycoccus clade A-uncultured Bathycoccus Kara 2017. It was found that several Micromonas species can co-exist, with Micromonas polaris dominating north of 72°N. The presence of Ostreococcus tauri, Ostreococcus lucimarinus and Ostreococcus mediterraneus at high latitudes beyond 65°N was documented for the first time, similar to findings for some other taxa. Our results will be important for obtaining a global view of Mamiellophyceae community dynamics.</p><h1>DNA diet profiles with high-resolution animal tracking data reveal levels of prey selection relative to habitat choice in a crepuscular insectivorous bird</h1>
<p><a href="https://doi.org/10.1002/Ece3.6893">doi:10.1002/Ece3.6893</a></p>

<p><a href=None>None</a></p>
<p>Given the global decline of many invertebrate food resources, it is fundamental to understand the dietary requirements of insectivores. We give new insights into the functional relationship between the spatial habitat use, food availability, and diet of a crepuscular aerial insectivore, the European Nightjar (<i>Caprimulgus europaeus</i>) by relating spatial use data with high-throughput sequencing (HTS) combined with DNA metabarcoding. Our study supports the predictions that nightjars collect a substantial part of their daily nourishment from foraging locations, sometimes at considerable distance from nesting sites. Lepidopterans comprise 65% of nightjars' food source. Nightjars tend to select larger species of <i>Lepidoptera</i> (>19 mm) which suggests that nightjars optimize the efficiency of foraging trips by selecting the most energetically favorable-larger-prey items. We anticipate that our findings may shed additional light on the interactions between invertebrate communities and higher trophic levels, which is required to understand the repercussions of changing food resources on individual- and population-level processes.</p><h1>Efficient Monitoring of Adult and Immature Mosquitoes Through Metabarcoding of Bulk Samples: A Case Study for Non-Model Culicids With Unique Ecologies</h1>
<p><a href="https://doi.org/10.1093/Jme/Tjaa267">doi:10.1093/Jme/Tjaa267</a></p>

<p><a href=None>None</a></p>
<p>The rapid and economical monitoring of mosquitos is imperative to understanding the dynamics of both disease vectors and nuisance species. In light of technological advances in mosquito sampling and DNA sequencing, health agencies can now utilize the full potential of metabarcoding pipelines for rapid and standardizable surveillance. Here, we describe mosquito spatial and temporal variation, with particular focus on Mansonia Blanchard species, in the Madeira (Rondônia State) and the Ribeira (São Paulo) watersheds, Brazil using metabarcoding of the D2 rDNA marker. Sampling and molecular pipelines were used to evaluate the taxonomic contribution of mosquitos in pools of culicids collected en masse from macrophyte-roots (immatures) and from Mosquito Magnet traps and protected human landings (adults). Results for adult captures are comparable to morphological diagnoses and clarify previously unknown temporal and spatial species turnover. Metabarcoding of immature stages also confirmed the extent of the geographical distribution of some species and each taxon's association with macrophyte species. Given the benefits of metabarcoding, such as taxonomic acuity, high throughput processing, and objectivity, we suggest such techniques should be more fully incorporated into culicid monitoring schemes. The metabarcoding protocol described herein paired with standardized field sampling schemes, when used by mosquito monitoring professionals, offers substantial improvements in terms of practicality, speed and cost.</p><h1>Monitoring of spatiotemporal occupancy patterns of fish and amphibian species in a lentic aquatic system using environmental DNA</h1>
<p><a href="https://doi.org/10.1111/Mec.15742">doi:10.1111/Mec.15742</a></p>

<p><a href=None>None</a></p>
<p>To effectively monitor, manage and protect aquatic species and understand their interactions, knowledge of their spatiotemporal distribution is needed. In this study, we used a fine-scale spatiotemporal water sampling design, followed by environmental DNA (eDNA) 12S metabarcoding, to investigate occupancy patterns of a natural community of fish and amphibian species in a lentic system. In the same system, we experimentally estimated the spatial and temporal dispersion of eDNA by placing a community of different fish and amphibian species in cages at one side of the pond, creating a controlled point of eDNA emission. Analyses of this cage community revealed a sharp spatial decline in detection rates and relative eDNA quantities at a distance of 5-10 m from the source, depending on the species and its abundance. In addition, none of the caged species could be detected 1 week after removal from the system. This indicates high eDNA decay rates and limited spatial eDNA dispersal, facilitating high local resolution for monitoring spatial occupancy patterns of aquatic species. Remarkably, for seven of the nine cage species, the presence of a single individual could be detected by pooling water of subsamples taken across the whole water body, illustrating the high sensitivity of the eDNA sampling and detection method applied. Finally, our work demonstrated that a fine-scale sampling design in combination with eDNA metabarcoding can cover total biodiversity very precisely and allows the construction of consistent spatiotemporal patterns of relative abundance and local distribution of free-living fish and amphibian species in a lentic ecosystem.</p><h1>Protist taxonomic and functional diversity in soil, freshwater and marine ecosystems</h1>
<p><a href="https://doi.org/S0160-4120(20)32217-0">doi:S0160-4120(20)32217-0</a></p>

<p><a href=None>None</a></p>
<p>Protists dominate eukaryotic diversity and play key functional roles in all ecosystems, particularly by catalyzing carbon and nutrient cycling. To date, however, a comparative analysis of their taxonomic and functional diversity that compares the major ecosystems on Earth (soil, freshwater and marine systems) is missing. Here, we present a comparison of protist diversity based on standardized high throughput 18S rRNA gene sequencing of soil, freshwater and marine environmental DNA. Soil and freshwater protist communities were more similar to each other than to marine protist communities, with virtually no overlap of Operational Taxonomic Units (OTUs) between terrestrial and marine habitats. Soil protists showed higher γ diversity than aquatic samples. Differences in taxonomic composition of the communities led to changes in a functional diversity among ecosystems, as expressed in relative abundance of consumers, phototrophs and parasites. Phototrophs (eukaryotic algae) dominated freshwater systems (49% of the sequences) and consumers soil and marine ecosystems (59% and 48%, respectively). The individual functional groups were composed of ecosystem- specific taxonomic groups. Parasites were equally common in all ecosystems, yet, terrestrial systems hosted more OTUs assigned to parasites of macro-organisms while aquatic systems contained mostly microbial parasitoids. Together, we show biogeographic patterns of protist diversity across major ecosystems on Earth, preparing the way for more focused studies that will help understanding the multiple roles of protists in the biosphere.</p><h1>Freshwater diatom biomonitoring through benthic kick-net metabarcoding</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0242143">doi:10.1371/Journal.Pone.0242143</a></p>

<p><a href=None>None</a></p>
<p>Biomonitoring is an essential tool for assessing ecological conditions and informing management strategies. The application of DNA metabarcoding and high throughput sequencing has improved data quantity and resolution for biomonitoring of taxa such as macroinvertebrates, yet, there remains the need to optimise these methods for other taxonomic groups. Diatoms have a longstanding history in freshwater biomonitoring as bioindicators of water quality status. However, multi-substrate periphyton collection, a common diatom sampling practice, is time-consuming and thus costly in terms of labour. This study examined whether the benthic kick-net technique used for macroinvertebrate biomonitoring could be applied to bulk-sample diatoms for metabarcoding. To test this approach, we collected samples using both conventional multi-substrate microhabitat periphyton collections and bulk-tissue kick-net methodologies in parallel from replicated sites with different habitat status (good/fair). We found there was no significant difference in community assemblages between conventional periphyton collection and kick-net methodologies or site status, but there was significant difference between diatom communities depending on site (P = 0.042). These results show the diatom taxonomic coverage achieved through DNA metabarcoding of kick-net is suitable for ecological biomonitoring applications. The shift to a more robust sampling approach and capturing diatoms as well as macroinvertebrates in a single sampling event has the potential to significantly improve efficiency of biomonitoring programmes that currently only use the kick-net technique to sample macroinvertebrates.</p><h1>Natrix: a Snakemake-based workflow for processing, clustering, and taxonomically assigning amplicon sequencing reads</h1>
<p><a href="https://doi.org/10.1186/S12859-020-03852-4">doi:10.1186/S12859-020-03852-4</a></p>

<p><a href=None>None</a></p>
<p>Sequencing of marker genes amplified from environmental samples, known as amplicon sequencing, allows us to resolve some of the hidden diversity and elucidate evolutionary relationships and ecological processes among complex microbial communities. The analysis of large numbers of samples at high sequencing depths generated by high throughput sequencing technologies requires efficient, flexible, and reproducible bioinformatics pipelines. Only a few existing workflows can be run in a user-friendly, scalable, and reproducible manner on different computing devices using an efficient workflow management system.</p><h1>Decision-making and best practices for taxonomy-free environmental DNA metabarcoding in biomonitoring using Hill numbers</h1>
<p><a href="https://doi.org/10.1111/Mec.15725">doi:10.1111/Mec.15725</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is raising expectations for biomonitoring of organisms that have hitherto been neglected. To bypass current limitations in taxonomic assignments due to incomplete or erroneous reference databases, taxonomy-free approaches are proposed for biomonitoring at the level of operational taxonomic units (OTUs). This is challenging, because OTUs cannot be annotated and directly compared against classically derived taxonomic data. The application of good stringency treatments to infer the validity of OTUs and clear understanding of the consequences of such treatments is especially relevant for biodiversity assessments. We investigated how common practices of stringency filtering affect eDNA diversity estimates in the statistical framework of Hill numbers. We collected water eDNA samples at 61 sites across a 740-km<sup>2</sup> river catchment, reflecting a spatially realistic scenario in biomonitoring. After bioinformatic processing of the data, we studied how different stringency treatments affect conclusions with respect to biodiversity at the catchment and site levels. The applied stringency treatments were based on the consistent appearance of OTUs across filter replicates, a relative abundance cut-off and rarefaction. We detected large differences in diversity estimates when accounting for presence/absence only, such that detected diversity at the catchment scale differed by an order of magnitude between the treatments. These differences disappeared when using stringency treatments with increasing weighting of the OTU abundances. Our study demonstrated the usefulness of Hill numbers for biodiversity analyses and comparisons of eDNA data sets that strongly differ in diversity. We recommend best practice for data stringency filtering for biomonitoring using eDNA.</p><h1>Assessment of microbial plankton diversity as an ecological indicator in the NW Mediterranean coast</h1>
<p><a href="https://doi.org/S0025-326X(20)30809-2">doi:S0025-326X(20)30809-2</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of microbial assemblages has been proposed as an alternative methodology to the traditional ones used in marine monitoring and environmental assessment. Here, we evaluated pico- and nanoplankton diversity as ecological indicators in NW Mediterranean coastal waters by comparing their diversity in samples subjected to varying degrees of continental pressures. Using metabarcoding of the 16S and 18S rRNA genes, we explored whether alphadiversity indices, abundance of Operational Taxonomic Units and taxonomic groups (and their ratios) provide information on the ecological quality of coastal waters. Our results revealed that only eukaryotic diversity metrics and a limited number of prokaryotic and eukaryotic taxa displayed potential in assessing continental influences in our surveyed area, resulting thus in a restrained potential of microbial plankton diversity as an ecological indicator. Therefore, incorporating microbial plankton diversity in environmental assessment could not always result in a significant improvement of current marine monitoring strategies.</p><h1>Massively parallel sequencing is unlocking the potential of environmental trace evidence</h1>
<p><a href="https://doi.org/S1872-4973(20)30165-4">doi:S1872-4973(20)30165-4</a></p>

<p><a href=None>None</a></p>
<p>Massively parallel sequencing (MPS) has revolutionised the field of genomics enabling substantial advances in human DNA profiling. Further, the advent of MPS now allows biological signatures to be obtained from complex DNA mixtures and trace amounts of low biomass samples. Environmental samples serve as ideal forms of contact trace evidence as detection at a scene can establish a link between a suspect, location and victim. Many studies have applied MPS technology to characterise the biodiversity within high biomass environmental samples (such as soil and water) to address questions related to ecology, conservation, climate change and human health. However, translation of these tools to forensic science remains in its infancy, due in part to the merging of traditional forensic ecology practices with unfamiliar DNA technologies and complex datasets. In addition, people and objects also carry low biomass environmental signals which have recently been shown to reflect a specific individual or location. The sensitivity, and reducing cost, of MPS is now unlocking the power of both high and low biomass environmental DNA (eDNA) samples as useful sources of genetic information in forensic science. This paper discusses the potential of eDNA to forensic science by reviewing the most explored applications that are leading the integration of this technology into the field. We introduce novel areas of forensic ecology that could also benefit from these tools with a focus on linking a suspect to a scene or establishing provenance of an unknown sample and discuss the current limitations and validation recommendations to achieve translation of eDNA into casework.</p><h1>BIOCOM-PIPE: a new user-friendly metabarcoding pipeline for the characterization of microbial diversity from 16S, 18S and 23S rRNA gene amplicons</h1>
<p><a href="https://doi.org/10.1186/S12859-020-03829-3">doi:10.1186/S12859-020-03829-3</a></p>

<p><a href=None>None</a></p>
<p>The ability to compare samples or studies easily using metabarcoding so as to better interpret microbial ecology results is an upcoming challenge. A growing number of metabarcoding pipelines are available, each with its own benefits and limitations. However, very few have been developed to offer the opportunity to characterize various microbial communities (e.g., archaea, bacteria, fungi, photosynthetic microeukaryotes) with the same tool.</p><h1>Assessment of littoral algal diversity from the northern Gulf of Mexico using environmental DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/Jpy.13087">doi:10.1111/Jpy.13087</a></p>

<p><a href=None>None</a></p>
<p>Traditional methods for algal biodiversity monitoring are costly and time inefficient because they rely on high-level taxonomic expertise to address species identity problems involving phenotypic plasticity and morphological convergence. These problems are exacerbated in regions such as the Gulf of Mexico, that has a limited history of phycological exploration, but that are economically important or threatened by numerous anthropogenic stressors. Given the high pace of disturbance to natural systems, there is a critical need for expedient and cost-effective tools for the study of benthic algal communities. Here we document the use of environmental DNA metabarcoding, using the partial LSU rDNA and 23S rDNA plastid molecular markers, to elucidate littoral algal diversity in the Northern Gulf of Mexico. We assigned 73.7% of algal OTUs to genus and 59.6% to species ranks. Our current study detected molecular signals for 35 algal/protist species with no previous reports in the Gulf of Mexico, thus providing an important, molecular-validated, baseline of species richness for this region. We also make several bioinformatic recommendations for the efficient use of high-throughput sequence data to assess biological communities.</p><h1>Environmental drivers affecting the bacterial community of intertidal sediments in the Yellow Sea</h1>
<p><a href="https://doi.org/S0048-9697(20)36255-0">doi:S0048-9697(20)36255-0</a></p>

<p><a href=None>None</a></p>
<p>Intertidal flats, as transition zones where terrestrial and marine ecosystems meet, provide unique environments and play an important role in marine ecosystems. In particular, the environmental characteristics of tidal marshes show are different than those of bare flats, especially in the rhizosphere. However, unlike the rhizosphere in terrestrial ecosystems, the rhizosphere of plants in tidal marsh areas and the associated microbial community have been the focus of very little research. Thus, this study investigated the diversity and variation in bacterial communities in the rhizosphere of a Phragmites australis and Suaeda japonica and along the sediment depths. High-throughput sequencing was performed by amplifying the 16S rRNA gene of environmental DNA extracted from sediment cores, and indicator species were identified with respect to the vegetation type and sediment depth. The most abundant phylum was Proteobacteria, followed by Chloroflexi, Bacteroidetes, Acidobacteria, and Firmicutes. In general, the results indicated that not only vegetation type and sediment depth themselves but also their interaction resulted in significant differences among the bacterial communities. The envfit results revealed that the environmental variables of sediment, such as mud content, organic matter, total organic carbon, and total nitrogen, had significant effects on the bacterial community structure. The indicator species varied depending on the vegetation type and sediment depth, showing significant correlations with certain selected environmental variables, but were fundamentally related to the rhizosphere. Overall, this study revealed the key factors that determine the bacterial community structure in tidal marshes and the indicator species according to vegetation type in the little studied rhizosphere of the intertidal ecosystem.</p><h1>High-resolution multi-marker DNA metabarcoding reveals sexual dietary differentiation in a bird with minor dimorphism</h1>
<p><a href="https://doi.org/10.1002/Ece3.6687">doi:10.1002/Ece3.6687</a></p>

<p><a href=None>None</a></p>
<p>Although sexual dietary differentiation is well known in birds, it is usually linked with significant morphological dimorphism between males and females, with lower differentiation reported in sexually monomorphic or only slightly dimorphic species. However, this may be an artifact of poor taxonomic resolution achieved in most conventional dietary studies, which may be unable to detect subtle intraspecific differentiation in prey consumption. Here, we show the power of multi-marker metabarcoding to address these issues, focusing on a slightly dimorphic generalist passerine, the black wheatear <i>Oenanthe leucura</i>. Using markers from four genomic regions (18S, 16S, COI, and <i>trn</i>L), we analyzed fecal droppings collected from 93 adult black wheatears during the breeding season. We found that sexes were rather similar in bill and body features, though males had a slightly thicker bill and longer wings and tail than females. Diet was dominated in both sexes by a very wide range of arthropod species and a few fleshy fruits, but the overall diet diversity was higher for males than females, and there was a much higher frequency of occurrence of ants in female (58%) than male (29%) diets. We hypothesize that the observed sexual differentiation was likely related to females foraging closer to their offspring on abundant prey, while males consumed a wider variety of prey while foraging more widely. Overall, our results suggest that dietary sexual differentiation in birds may be more widespread than recognized at present and that multi-marker DNA metabarcoding is a particularly powerful tool to unveiling such differences.</p><h1>Fish and crustacean biodiversity in an outer maritime estuary of the Pearl River Delta revealed by environmental DNA</h1>
<p><a href="https://doi.org/S0025-326X(20)30825-0">doi:S0025-326X(20)30825-0</a></p>

<p><a href=None>None</a></p>
<p>Understanding the faunal community structure in the estuary would be crucial in assessing the health of the ecosystem. The poor visibility in the estuarine area due to the outflow from the Pearl River hinders the conventional visual census in assessing the megafaunal biodiversity. In this study, the fish and crustacean biodiversity of Hong Kong's western waters, i.e. the outer maritime estuary of the PRD, were studied through the metabarcoding of environmental DNA (eDNA). eDNA from the seawater and sediment samples was extracted from five sites in the region. After testing the performance of two genetic markers, amplicons of the cytochrome oxidase I, amplified by polymerase chain reaction, were subjected to Illumina high-throughput sequencing (MiSeq) analysis. A total of 22 fish species from 17 families and 34 crustacean species from 27 families were identified by blasting the sequences against the NCBI GenBank database, demonstrating segregation between samples from different sites. This study provides insight on the detail distribution of fish assembly in PRD, when compared with a previous eDNA study in the inner brackish PRD.</p><h1>Bathymetric gradient shapes the community composition rather than the species richness of deep-sea benthic ciliates</h1>
<p><a href="https://doi.org/S0048-9697(20)36152-0">doi:S0048-9697(20)36152-0</a></p>

<p><a href=None>None</a></p>
<p>The bathymetric gradient is one of the most important factors that regulate the distribution of life. However, community variations of benthic ciliates along bathymetric gradients in the deep sea remain rather unexplored. In this study, we hypothesize that in the deep sea, the bathymetric gradient shapes the benthic ciliate community composition rather than the species richness. Here, we evaluated the distribution patterns and drivers of benthic ciliate communities of an abyssal plain, a seamount, and a trench with water depths ranging from 800 m down to 6600 m by high throughput eDNA sequencing and statistical analyses. We observed no significant correlation between ciliate operated taxonomic unit (OTU) richness and water depth. A meta-analysis, which combined our previously published data from the neritic habitats, supports the notion that water depth exceeding 800 m has little effect on the richness of benthic ciliate species. In contrast, the composition of deep-sea ciliate communities was significantly distinct in different habitats along the bathymetric gradients. A SourceTracker analysis revealed extremely low connectivity among ciliate communities along the bathymetric gradients. More than 95% of the community dissimilarity in the deep-sea floor was attributed to species replacement, which might be caused by environmental sorting or historical constraints. Furthermore, the observed community variations could be ascribed more to water depth than to geographic distance. The findings imply that the strong force of environmental sorting along the bathymetric gradients and the low connectivity among the ciliate communities might lead to an isolated evolution. This could shape the community composition rather than the species richness, which is mainly determined by the limited nutrient availability and the extreme environmental conditions in the deep sea.</p><h1>Seabed mining could come at a high price for a unique fauna</h1>
<p><a href="https://doi.org/10.1111/Mec.15678">doi:10.1111/Mec.15678</a></p>

<p><a href=None>None</a></p>
<p>The deep seafloor is teeming with life, most of which remains poorly known to science. It also constitutes an important reserve of natural resources, particularly minerals, that mining companies will start harvesting in the next few years (Nat Rev Earth Environ, 1, 2020, 158). In this context, broad biodiversity assessments of deep-sea ecosystems are urgently needed to establish a baseline prior to mining. However, significant gaps in our taxonomic knowledge and the high cost of sampling in the deep sea limit the effectiveness of conventional morphology-based surveys. In this issue of Molecular Ecology, Laroche et al. (Mol Ecol, 2020) capitalize on high throughput molecular methods to conduct one of the most detailed and rigorous surveys of the composition and biogeography of deep-seafloor metazoan communities to date. The authors show that deep seamounts in the Clarion Clipperton Zone are inhabited by rich metazoan communities that are distinct from those of the surrounding abyssal plains. These results have important conservation implications: if communities on deep seamounts were to persist after large-scale industrial mining operations on the surrounding plains, the seamounts would not serve as appropriate reservoirs to repopulate impacted areas.</p><h1>Comparing fish prey diversity for a critically endangered aquatic mammal in a reserve and the wild using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.1038/S41598-020-73648-2">doi:10.1038/S41598-020-73648-2</a></p>

<p><a href=None>None</a></p>
<p>Using environmental DNA (eDNA) metabarcoding, we compared fish diversity in two distinct water bodies within the Yangtze River Basin with known populations of the critically endangered Yangtze finless porpoise (Neophocaena asiaeorientalis; YFP): the Tian-e-Zhou Reserve and Poyang Lake. We aimed to create a fish surveying tool for use in the Yangtze River Basin, while also gaining a better understanding of the prey distribution and diversity within two of the remaining strongholds of YFP. 16S rRNA universal primers were developed to amplify fish eDNA. After high-throughput sequencing and stringent data filtering, we identified a total of 75 fish species (6 orders, 9 families, 57 genera) across seasons and regions. Nine of the 75 fish species were among the 28 known YFP prey species, three of which were detected in all water samples. Our eDNA metabarcoding identified many species that had been previously captured using traditional netting practices, but also numerous species not previously collected in these water bodies. Fish diversity was higher in Poyang Lake than in Tian-e-Zhou Reserve, as well as higher in the spring than in summer. These methods provide a broadly applicable tool to quantify fish diversity and distributions throughout the Yangtze River Basin, and to inform conservation strategies of YFP.</p><h1>Assessing the responses of <i>Sphagnum</i> micro-eukaryotes to climate changes using high throughput sequencing</h1>
<p><a href="https://doi.org/10.7717/Peerj.9821">doi:10.7717/Peerj.9821</a></p>

<p><a href=None>None</a></p>
<p>Current projections suggest that climate warming will be accompanied by more frequent and severe drought events. Peatlands store ca. one third of the world's soil organic carbon. Warming and drought may cause peatlands to become carbon sources through stimulation of microbial activity increasing ecosystem respiration, with positive feedback effect on global warming. Micro-eukaryotes play a key role in the carbon cycle through food web interactions and therefore, alterations in their community structure and diversity may affect ecosystem functioning and could reflect these changes. We assessed the diversity and community composition of <i>Sphagnum</i>-associated eukaryotic microorganisms inhabiting peatlands and their response to experimental drought and warming using high throughput sequencing of environmental DNA. Under drier conditions, micro-eukaryotic diversity decreased, the relative abundance of autotrophs increased and that of osmotrophs (including Fungi and Peronosporomycetes) decreased. Furthermore, we identified climate change indicators that could be used as early indicators of change in peatland microbial communities and ecosystem functioning. The changes we observed indicate a shift towards a more "terrestrial" community in response to drought, in line with observed changes in the functioning of the ecosystem.</p><h1>Space-time dynamics in monitoring neotropical fish communities using eDNA metabarcoding</h1>
<p><a href="https://doi.org/S0048-9697(20)35625-4">doi:S0048-9697(20)35625-4</a></p>

<p><a href=None>None</a></p>
<p>The biodiverse Neotropical ecoregion remains insufficiently assessed, poorly managed, and threatened by unregulated human activities. Novel, rapid and cost-effective DNA-based approaches are valuable to improve understanding of the biological communities and for biomonitoring in remote areas. Here, we evaluate the potential of environmental DNA (eDNA) metabarcoding for assessing the structure and distribution of fish communities by analysing water and sediment from 11 locations along the Jequitinhonha River catchment (Brazil). Each site was sampled twice, before and after a major rain event in a five-week period and fish diversity was estimated using high-throughput sequencing of 12S rRNA amplicons. In total, 252 Molecular Operational Taxonomic Units (MOTUs) and 34 fish species were recovered, including endemic, introduced, and previously unrecorded species for this basin. Spatio-temporal variation of eDNA from fish assemblages was observed and species richness was nearly twice as high before the major rain event compared to afterwards. Yet, peaks of diversity were primarily associated with only four of the locations. No correlation between β-diversity and longitudinal distance or presence of dams was detected, but low species richness observed at sites located near dams might that these anthropogenic barriers may have an impact on local fish diversity. Unexpectedly high α-diversity levels recorded at the river mouth suggest that these sections should be further evaluated as putative "eDNA reservoirs" for rapid monitoring. By uncovering spatio-temporal changes, unrecorded biodiversity components, and putative anthropogenic impacts on fish assemblages, we further strengthen the potential of eDNA metabarcoding as a biomonitoring tool, especially in regions often neglected or difficult to access.</p><h1>Environmental DNA metabarcoding reveals estuarine benthic community response to nutrient enrichment - Evidence from an in-situ experiment</h1>
<p><a href="https://doi.org/S0269-7491(20)36160-1">doi:S0269-7491(20)36160-1</a></p>

<p><a href=None>None</a></p>
<p>Nutrient loading is a major threat to estuaries and coastal environments worldwide, therefore, it is critical that we have good monitoring tools to detect early signs of degradation in these ecologically important and vulnerable ecosystems. Traditionally, bottom-dwelling macroinvertebrates have been used for ecological health assessment but recent advances in environmental genomics mean we can now characterize less visible forms of biodiversity, offering a more holistic view of the ecosystem and potentially providing early warning signals of disturbance. We carried out a manipulative nutrient enrichment experiment (0, 150 and 600 g N fertilizer m<sup>-2</sup>) in two estuaries in New Zealand to assess the effects of nutrient loading on benthic communities. After seven months of enrichment, environmental DNA (eDNA) metabarcoding was used to examine the response of eukaryotic (18S rRNA), diatom only (rbcL) and bacterial (16S rRNA) communities. Multivariate analyses demonstrated changes in eukaryotic, diatom and bacterial communities in response to nutrient enrichment at both sites, despite differing environmental conditions. These patterns aligned with changes in macrofaunal communities identified using traditional morphological techniques, confirming concordance between disturbance indicators detected by eDNA and current monitoring approaches. Clear shifts in eukaryotic and bacterial indicator taxa were seen in response to nutrient loading while changes in diatom only communities were more subtle. Community changes were discernible between 0 and 150 g N m<sup>-2</sup> treatments, suggesting that estuary health assessment tools could be developed to detect early signs of degradation. Increasing variation in community structure associated with nutrient loading could also be used as an indicator of stress or approaching tipping points. This work represents a first step towards the development of molecular-based estuary monitoring tools, which could provide a more holistic and standardized approach to ecosystem health assessment with faster turn-around times and lower costs.</p><h1>Root-Associated Mycobiome Differentiate between Habitats Supporting Production of Different Truffle Species in Serbian Riparian Forests</h1>
<p><a href="https://doi.org/E1331">doi:E1331</a></p>

<p><a href=None>None</a></p>
<p>Balkan lowlands bordering with the Pannonia region are inhabited by diverse riparian forests that support production of different truffle species, predominantly the most prized white truffle of Piedmont (<i>Tuber magnatum</i> Pico), but also other commercial species (<i>T.</i><i>macrosporum</i> Vitt., <i>T</i>. <i>aestivum</i> Vitt.). Surprisingly, little is known about the native root-associated mycobiome (RAM) of these lowland truffle-producing forests. Therefore, in this study we aim at exploring and comparing the RAMs of three different truffle-producing forests from Kolubara river plane in Serbia. Molecular methods based on next generation sequencing (NGS) were used to evaluate the diversity of root-associated fungal communities and to elucidate the influence of environmental factors on their differentiation. To our knowledge, this is the first study from such habitats with a particular focus on comparative analysis of the RAM in different truffle-producing habitats using a high-throughput sequencing approach. Our results indicated that the alpha diversity of investigated fungal communities was not significantly different between different truffle-producing forests and within a specific forest type, while the seasonal differences in the alpha diversity were only observed in the white truffle-producing forests. Taxonomic profiling at phylum level indicated the dominance of fungal OTUs belonging to phylum Ascomycota and Basidiomycota, with very minor presence of other phyla. Distinct community structures of root-associated mycobiomes were observed for white, mixed, and black truffle-producing forests. The core mycobiome analysis indicated a fair share of fungal genera present exclusively in white and black truffle-producing forest, while the core genera of mixed truffle-producing forests were shared with both white and black truffle-producing forests. The majority of detected fungal OTUs in all three forest types were symbiotrophs, with ectomycorrhizal fungi being a dominant functional guild. Apart from assumed vegetation factor, differentiation of fungal communities was driven by factors connected to the distance from the river and exposure to fluvial activities, soil age, structure, and pH. Overall, Pannonian riparian forests appear to host diverse root-associated fungal communities that are strongly shaped by variation in soil conditions.</p><h1>Modelling technical and biological biases in macroinvertebrate community assessment from bulk preservative using multiple metabarcoding markers</h1>
<p><a href="https://doi.org/10.1111/Mec.15620">doi:10.1111/Mec.15620</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding from the ethanol used to store macroinvertebrate bulk samples is a convenient methodological option in molecular biodiversity assessment and biomonitoring of aquatic ecosystems, as it preserves specimens and reduces problems associated with sample sorting. However, this method may be affected by errors and biases, which need to be thoroughly quantified before it can be mainstreamed into biomonitoring programmes. Here, we used 80 unsorted macroinvertebrate samples collected in Portugal under a Water Framework Directive monitoring programme, to compare community diversity and taxonomic composition metrics estimated through morphotaxonomy versus metabarcoding from storage ethanol using three markers (COI-M19BR2, 16S-Inse01 and 18S-Euka02) and a multimarker approach. A preliminary in silico analysis showed that the three markers were adequate for the target taxa, with detection failures related primarily to the lack of adequate barcodes in public databases. Metabarcoding of ethanol samples retrieved far less taxa per site (alpha diversity) than morphotaxonomy, albeit with smaller differences for COI-M19BR2 and the multimarker approach, while estimates of taxa turnover (beta diversity) among sites were similar across methods. Using generalized linear mixed models, we found that after controlling for differences in read coverage across samples, the probability of detection of a taxon was positively related to its proportional abundance, and negatively so to the presence of heavily sclerotized exoskeleton (e.g., Coleoptera). Overall, using our experimental protocol with different template dilutions, the COI marker showed the best performance, but we recommend the use of a multimarker approach to detect a wider range of taxa in freshwater macroinvertebrate samples. Further methodological development and optimization efforts are needed to reduce biases associated with body armouring and rarity in some macroinvertebrate taxa.</p><h1>Diversity and Ecology of Chlorophyta (Viridiplantae) Assemblages in Protected and Non-protected Sites in Deception Island (Antarctica, South Shetland Islands) Assessed Using an NGS Approach</h1>
<p><a href="https://doi.org/10.1007/S00248-020-01584-9">doi:10.1007/S00248-020-01584-9</a></p>

<p><a href=None>None</a></p>
<p>Assessment of the diversity of algal assemblages in Antarctica has until now largely relied on traditional microbiological culture approaches. Here we used DNA metabarcoding through high-throughput sequencing (HTS) to assess the uncultured algal diversity at two sites on Deception Island, Antarctica. The first was a relatively undisturbed site within an Antarctic Specially Protected Area (ASPA 140), and the second was a site heavily impacted by human visitation, the Whalers Bay historic site. We detected 65 distinct algal taxa, 50 from within ASPA 140 and 61 from Whalers Bay. Of these taxa, 46 were common to both sites, and 19 only occurred at one site. Algal richness was about six times greater than reported in previous studies using culture methods. A high proportion of DNA reads obtained was assigned to the highly invasive species Caulerpa webbiana at Whalers Bay, and the potentially pathogenic genus Desmodesmus was found at both sites. Our data demonstrate that important differences exist between these two protected and human-impacted sites on Deception Island in terms of algal diversity, richness, and abundance. The South Shetland Islands have experienced considerable effects of climate change in recent decades, while warming through geothermal activity on Deception Island itself makes this island one of the most vulnerable to colonization by non-native species. The detection of DNA of non-native taxa highlights concerns about how human impacts, which take place primarily through tourism and national research operations, may influence future biological colonization processes in Antarctica.</p><h1>Metabarcoding quantifies differences in accumulation of ballast water borne biodiversity among three port systems in the United States</h1>
<p><a href="https://doi.org/S0048-9697(20)34985-8">doi:S0048-9697(20)34985-8</a></p>

<p><a href=None>None</a></p>
<p>Characterizing biodiversity conveyed in ships' ballast water (BW), a global driver of biological invasions, is critically important for understanding risks posed by this key vector and establishing baselines to evaluate changes associated with BW management. Here we employ high throughput sequence (HTS) metabarcoding of the 18S small subunit rRNA to test for and quantify differences in the accumulation of BW-borne biodiversity among three distinct recipient port systems in the United States. These systems were located on three different coasts (Pacific, Gulf, and Atlantic) and chosen to reflect distinct trade patterns and source port biogeography. Extensive sampling of BW tanks (n = 116) allowed detailed exploration of molecular diversity accumulation. Our results indicate that saturation of introduced zooplankton diversity may be achieved quickly, with fewer than 25 tanks needed to achieve 95% of the total extrapolated diversity, if source biogeography is relatively limited. However, as predicted, port systems with much broader source geographies require more extensive sampling to estimate diversity, which continues to accumulate after sampling >100 discharges. The ability to identify BW sources using molecular indicators was also found to depend on the breadth of source biogeography and the extent to which sources had been sampled. These findings have implications both for the effort required to fully understand introduced diversity and for projecting risks associated with future changes to maritime traffic that may increase source biogeography for many recipient ports. Our data also suggest that molecular diversity may not decline significantly with BW age, indicating either that some organisms survive longer than recognized in previous studies or that nucleic acids from dead organisms persist in BW tanks. We present evidence for detection of potentially invasive species in arriving BW but discuss important caveats that preclude strong inferences regarding the presence of living representatives of these species in BW tanks.</p><h1>Environmental DNA assessment of airborne plant and fungal seasonal diversity</h1>
<p><a href="https://doi.org/S0048-9697(20)33770-0">doi:S0048-9697(20)33770-0</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding and metagenomics analyses can improve taxonomic resolution in biodiversity studies. Only recently, these techniques have been applied in aerobiology, to target bacteria, fungi and plants in airborne samples. Here, we present a nine-month aerobiological study applying eDNA metabarcoding in which we analyzed simultaneously airborne diversity and variation of fungi and plants across five locations in North and Central Italy. We correlated species composition with the ecological characteristics of the sites and the seasons. The most abundant taxa among all sites and seasons were the fungal genera Cladosporium, Alternaria, and Epicoccum and the plant genera Brassica, Corylus, Cupressus and Linum, the latter being much more variable among sites. PERMANOVA and indicator species analyses showed that the plant diversity from air samples is significantly correlated with seasons, while that of fungi varied according to the interaction between seasons and sites. The results consolidate the performance of a new eDNA metabarcoding pipeline for the simultaneous amplification and analysis of airborne plant and fungal particles. They also highlight the promising complementarity of this approach with more traditional biomonitoring frameworks and routine reports of air quality provided by environmental agencies.</p><h1>Tracking seasonal changes in diversity of pollen allergen exposure: Targeted metabarcoding of a subtropical aerobiome</h1>
<p><a href="https://doi.org/S0048-9697(20)34718-5">doi:S0048-9697(20)34718-5</a></p>

<p><a href=None>None</a></p>
<p>The importance of grass pollen to the global burden of allergic respiratory disease is well established but exposure to subtropical and temperate pollens is difficult to discern. Current monitoring of airborne pollen relies on light microscopy, limiting identification of taxa to family level. This informs seasonal fluctuations in pollen aerobiology but restricts analysis of aerobiological composition. We aimed to test the utility of DNA metabarcoding to identify specific taxa contributing to the aerobiome of environmental air samples, using routine pollen and spore monitoring equipment, as well as assess temporal variation of Poaceae pollen across an entire season. Airborne pollen concentrations were determined by light microscopy over two pollen seasons in the subtropical city of Brisbane (27°32'S, 153°00E), Australia. Thirty daily pollen samples were subjected to high throughput sequencing of the plastid rbcL amplicon. Amplicons corresponded to plants observed in the local biogeographical region with up to 3238 different operational taxonomic units (OTU) detected. The aerobiome sequencing data frequently identified pollen to genus levels with significant quantitative differences in aerobiome diversity between the months and seasons detected. Moreover, multiple peaks of Chloridoideae and Panicoideae pollen were evident over the collection period confirming these grasses as the dominant Poaceae pollen source across the season. Targeted high throughput sequencing of routinely collected airborne pollen samples appears to offer utility to track temporal changes in the aerobiome and shifts in pollen exposure. Precise identification of the composition and temporal distributions of airborne pollen is important for tracking biodiversity and for management of allergic respiratory disease.</p><h1>The limited spatial scale of dispersal in soil arthropods revealed with whole-community haplotype-level metabarcoding</h1>
<p><a href="https://doi.org/10.1111/Mec.15591">doi:10.1111/Mec.15591</a></p>

<p><a href=None>None</a></p>
<p>Soil arthropod communities are highly diverse and critical for ecosystem functioning. However, our knowledge of spatial structure and the underlying processes of community assembly are scarce, hampered by limited empirical data on species diversity and turnover. We implement a high-throughput sequencing approach to generate comparative data for thousands of arthropods at three hierarchical levels: genetic, species and supra-specific lineages. A joint analysis of the spatial arrangement across these levels can reveal the predominant processes driving the variation in biological assemblages at the local scale. This multihierarchical approach was performed using haplotype-level COI metabarcoding of entire communities of mites, springtails and beetles from three Iberian mountain regions. Tens of thousands of specimens were extracted from deep and superficial soil layers and produced comparative phylogeographic data for >1,000 codistributed species and nearly 3,000 haplotypes. Local assemblage composition differed greatly between grasslands and forests and, within each habitat, showed strong spatial structure and high endemicity. Distance decay was high at all levels, even at the scale of a few kilometres or less. The local distance decay patterns were self-similar for the haplotypes and higher hierarchical entities, and this fractal structure was similar in all regions, suggesting that uniform processes of limited dispersal determine local-scale community assembly. Our results from whole-community metabarcoding provide insight into how dispersal limitations constrain mesofauna community structure within local spatial settings over evolutionary timescales. If generalized across wider areas, the high turnover and endemicity in the soil locally may indicate extremely high richness globally, challenging our current estimations of total arthropod diversity on Earth.</p><h1>Advancing biodiversity assessments with environmental DNA: Long-read technologies help reveal the drivers of Amazonian fungal diversity</h1>
<p><a href="https://doi.org/10.1002/Ece3.6477">doi:10.1002/Ece3.6477</a></p>

<p><a href=None>None</a></p>
<p>Fungi are a key component of tropical biodiversity. However, due to their inconspicuous and largely subterranean nature, they are usually neglected in biodiversity inventories. The goal of this study was to identify the key determinants of fungal richness, community composition, and turnover in tropical rainforests. We tested specifically for the effect of soil properties, habitat, and locality in Amazonia. For these analyses, we used high-throughput sequencing data of short and long reads of fungal DNA present in soil and organic litter samples, combining existing and novel genomic data. Habitat type (phytophysiognomy) emerges as the strongest factor explaining fungal community composition. Naturally open areas-campinas-are the richest habitat overall. Soil properties have different effects depending on the soil layer (litter or mineral soil) and the choice of genetic marker. We suggest that campinas could be a neglected hotspot of fungal diversity. An underlying cause for their rich diversity may be the overall low soil fertility, which increases the reliance on biotic interactions essential for nutrient absorption in these environments, notably ectomycorrhizal fungi-plant associations. Our results highlight the advantages of using both short and long DNA reads produced through high-throughput sequencing to characterize fungal diversity. While short reads can suffice for diversity and community comparison, long reads add taxonomic precision and have the potential to reveal population diversity.</p><h1>Fine sediment and flow velocity impact bacterial community and functional profile more than nutrient enrichment</h1>
<p><a href="https://doi.org/10.1002/Eap.2212">doi:10.1002/Eap.2212</a></p>

<p><a href=None>None</a></p>
<p>Freshwater ecosystems face many simultaneous pressures due to human activities. Consequently, there has been a rapid loss of freshwater biodiversity and an increase in biomonitoring programs. Our study assessed the potential of benthic stream bacterial communities as indicators of multiple-stressor impacts associated with urbanization and agricultural intensification. We conducted a fully crossed four-factor experiment in 64 flow-through mesocosms fed by a pristine montane stream (21 d of colonization, 21 d of manipulations) and investigated the effects of nutrient enrichment, flow-velocity reduction and added fine sediment after 2 and 3 weeks of stressor exposure. We used high-throughput sequencing and metabarcoding techniques (16S rRNA genes), as well as curated biological databases (METAGENassit, MetaCyc), to identify changes in bacterial relative abundances and predicted metabolic functional profile. Sediment addition and flow-velocity reduction were the most pervasive stressors. They both increased α-diversity and had strong taxon-specific effects on community composition and predicted functions. Sediment and flow velocity also interacted frequently, with 88% of all bacterial response variables showing two-way interactions and 33% showing three-way interactions including nutrient enrichment. Changes in relative abundances of common taxa were associated with shifts in dominant predicted functions, which can be extrapolated to underlaying stream-wide mechanisms such as carbon use and bacterial energy production pathways. Observed changes were largely stable over time and occurred after just 2 weeks of exposure, demonstrating that bacterial communities can be well-suited for early detection of multiple stressors. Overall, added sediment and reduced flow velocity impacted both bacterial community structure and predicted function more than nutrient enrichment. In future research and stream management, a holistic approach to studying multiple-stressor impacts should include multiple trophic levels with their functional responses, to enhance our mechanistic understanding of complex stressor effects and promote establishment of more efficient biomonitoring programs.</p><h1>Meta-omics highlights the diversity, activity and adaptations of fungi in deep oceanic crust</h1>
<p><a href="https://doi.org/10.1111/1462-2920.15181">doi:10.1111/1462-2920.15181</a></p>

<p><a href=None>None</a></p>
<p>The lithified oceanic crust, lower crust gabbros in particular, has remained largely unexplored by microbiologists. Recently, evidence for heterogeneously distributed viable and transcriptionally active autotrophic and heterotrophic microbial populations within low-biomass communities was found down to 750 m below the seafloor at the Atlantis Bank Gabbro Massif, Indian Ocean. Here, we report on the diversity, activity and adaptations of fungal communities in the deep oceanic crust from ~10 to 780 mbsf by combining metabarcoding analyses with mid/high-throughput culturing approaches. Metabarcoding along with culturing indicate a low diversity of viable fungi, mostly affiliated to ubiquitous (terrestrial and aquatic environments) taxa. Ecophysiological analyses coupled with metatranscriptomics point to viable and transcriptionally active fungal populations engaged in cell division, translation, protein modifications and other vital cellular processes. Transcript data suggest possible adaptations for surviving in the nutrient-poor, lithified deep biosphere that include the recycling of organic matter. These active communities appear strongly influenced by the presence of cracks and veins in the rocks where fluids and resulting rock alteration create micro-niches.</p><h1>Nematodes as bioindicators of polluted sediments using metabarcoding and microscopic taxonomy</h1>
<p><a href="https://doi.org/S0160-4120(20)31877-8">doi:S0160-4120(20)31877-8</a></p>

<p><a href=None>None</a></p>
<p>The use of bioindicator species is a widely applied approach to evaluate ecological conditions, and several indices have been designed for this purpose. To assess the impact of pollution, especially in sediments, a pollution-sensitive index based on nematodes, one of the most abundant and species-rich groups of metazoa, was developed. The NemaSPEAR[%] index in its original form relies on the morphological inspection of nematode species. The application of a morphologically based NemaSPEAR[%] at the genus-level was previously validated. The present study evaluated a NemaSPEAR[%] index based on metabarcoding of nematode communities and tested the potential of fragments from the 28S rDNA, 18S rDNA and cytochrome c oxidase subunit I (COI) genes. In general, molecular-based results tended to show a poorer condition than morphology-based results for the investigated sites. At the genus level, NemaSPEAR[%] values based on morphological data strongly correlated with those based on molecular data for both the 28S rDNA and the 18S rDNA gene fragments (R<sup>2</sup> = 0.86 and R<sup>2</sup> = 0.74, respectively). Within the dominant genera (>3%) identified by morphology, 68% were detected by at least one of the two ribosomal markers. At the species level, however, concordance was less pronounced, as there were several deviations of the molecular from the morphological data. These differences could mostly be attributed to shortcomings in the reference database used in the molecular-based assignments. Our pilot study shows that a molecularly based, genus-level NemaSPEAR[%] can be successfully applied to evaluate polluted sediment. Future studies need to validate this approach further, e.g. with bulk extractions of whole meiofaunal communities in order to circumvent time-consuming nematode isolation. Further database curation with abundant NemaSPEAR[%] species will also increase the applicability of this approach.</p><h1>Tagsteady: A metabarcoding library preparation protocol to avoid false assignment of sequences to samples</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13227">doi:10.1111/1755-0998.13227</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of environmental DNA (eDNA) and DNA extracted from bulk specimen samples is a powerful tool in studies of biodiversity, diet and ecological interactions as its inherent labelling of amplicons allows sequencing of taxonomically informative genetic markers from many samples in parallel. However, the occurrence of so-called 'tag-jumps' can cause incorrect assignment of sequences to samples and artificially inflate diversity. Two steps during library preparation of pools of 5' nucleotide-tagged amplicons have been suggested to cause tag-jumps: (a) T4 DNA polymerase blunt-ending in the end-repair step and (b) postligation PCR amplification of amplicon libraries. The discovery of tag-jumps has led to recommendations to only carry out metabarcoding PCR amplifications with primers carrying twin-tags to ensure that tag-jumps cannot result in false assignments of sequences to samples. As this increases both cost and workload, a metabarcoding library preparation protocol which circumvents the two steps that causes tag-jumps is needed. Here, we demonstrate Tagsteady, a PCR-free metabarcoding Illumina library preparation protocol for pools of nucleotide-tagged amplicons that enables efficient and cost-effective generation of metabarcoding data with virtually no tag-jumps. We use pools of twin-tagged amplicons to investigate the effect of T4 DNA polymerase blunt-ending and postligation PCR on the occurrence of tag-jumps and demonstrate that both blunt-ending and postligation PCR, alone or together, can result in detrimental amounts of tag-jumps (here, up to ca. 49% of total sequences), while leaving both steps out (the Tagsteady protocol) results in amounts of sequences carrying new combinations of used tags (tag-jumps) comparable to background contamination.</p><h1>Diel Dynamics of Freshwater Bacterial Communities at Beaches in Lake Erie and Lake St. Clair, Windsor, Ontario</h1>
<p><a href="https://doi.org/10.1007/S00248-020-01539-0">doi:10.1007/S00248-020-01539-0</a></p>

<p><a href=None>None</a></p>
<p>Bacteria play a key role in freshwater biogeochemical cycling as well as water safety, but short-term trends in freshwater bacterial community composition and dynamics are not yet well characterized. We sampled four public beaches in southern Ontario, Canada; in June, July, and August (2016) over a 24-h (diel) cycle at 2-h intervals. Using high-throughput sequencing of 16S rRNA gene, we found substantial bi-hourly and day/night variation in the bacterial communities with considerable fluctuation in the relative abundance of Actinobacteria and Proteobacteria phyla. Moreover, relative abundance of Enterobacteriaceae (associated with potential health risk) was significantly high at night in some dial cycles. Diversity was significantly high at night across most of the diel sampling events. qPCR assays showed a substantial bi-hourly variation of Escherichia coli levels with a significant high level of E. coli at night hours in comparison with day hours and the lowest levels at noon and during the afternoon hours. Taken together, these findings highlighted a considerable short-term temporal variation of bacterial communities which helps better understanding of freshwater bacterial dynamics and their ecology. E. coli monitoring showed that multiple samples in different hours will provide more accurate picture of freshwater safety and human health risk. Graphical abstract.</p><h1>Diversity hotspot and unique community structure of foraminifera in the world's deepest marine blue hole - Sansha Yongle Blue Hole</h1>
<p><a href="https://doi.org/10.1038/S41598-020-67221-0">doi:10.1038/S41598-020-67221-0</a></p>

<p><a href=None>None</a></p>
<p>Marine blue holes are precious geological heritages with high scientific research values. Their physical and chemical characteristics are unique because of the steep-walled structure and isolated water column which create isolated ecosystems in geographically restricted areas. The Sansha Yongle Blue Hole (SYBH) is the world's deepest marine blue hole. Here, we generated the first DNA metabarcoding dataset from SYBH sediment focusing on foraminifera, a group of protists that have colonized various marine environments. We collected sediment samples from SYBH along a depth gradient to characterize the foraminiferal diversity and compared them with the foraminiferal diversity of the costal Jiaozhou Bay (JZB) and the abyssal Northwest Pacific Ocean (NWP). We amplified the SSU rDNA of foraminifera and sequenced them with high-throughput sequencing. The results showed that the foraminiferal assemblages in SYBH were vertically structured in response to the abiotic gradients and diversity was higher than in JZB and NWP. This study illustrates the capacity of foraminifera to colonize hostile environments and shows that blue holes are natural laboratories to explore physiological innovation associated with anoxia.</p><h1>Environmental DNA from plastic and textile marine litter detects exotic and nuisance species nearby ports</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0228811">doi:10.1371/Journal.Pone.0228811</a></p>

<p><a href=None>None</a></p>
<p>Marine debris is currently a significant source of environmental and economic problems. Floating litter can be employed by marine organisms as a surface to attach to and use as spreading vector. Human activities are promoting the expansion of potentially harmful species into novel ecosystems, endangering autochthonous communities. In this project, more than 1,000 litter items were collected and classified from five beaches eastwards the port of Gijon, in Asturias, Spain. Next generation sequencing was employed to study biofouling communities attached to items of different materials. A dominance of DNA from Florideophyceae, Dinophyceae and Arthropoda was found, and four non-indigenous species (NIS) were identified. Results showed a clear preference of Florideophyceae and Bryozoa to attach on textile surfaces versus plastic ones. Considering that these taxa contain several highly invasive species described to date, these data emphasize the potential of textile marine debris as a vector for dispersal of NIS. Moreover, the closest beaches to the port contained a more similar biota profile than the farther ones, confirming that both plastic and textile marine litter can be vectors for species dispersal from ports.</p><h1>A new method of metabarcoding Microsporidia and their hosts reveals high levels of microsporidian infections in mosquitoes (Culicidae)</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13205">doi:10.1111/1755-0998.13205</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding offers new perspectives, especially with regard to the high-throughput identification and diagnostics of pathogens. Microsporidia are an example of widely distributed, opportunistic and pathogenic microorganisms in which molecular identification is important for both environmental research and clinical diagnostics. We have developed a method for parallel detection of both microsporidian infection and the host species. We designed new primer sets: one specific for the classical Microsporidia (targeting the hypervariable V5 region of small subunit [ssu] rDNA), and a second one targeting a shortened fragment of the COI gene (standard metazoan DNA-barcode); both markers are well suited for next generation sequencing. Analysis of the ssu rDNA data set representing 607 microsporidian species (120 genera) indicated that the V5 region enables identification of >98% species in the data set (596/607). To test the method, we used microsporidians that infect mosquitoes in natural populations. Using mini-COI data, all field-collected mosquitoes were unambiguously assigned to seven species; among them almost 60% of specimens were positive for at least 11 different microsporidian species, including a new microsporidian ssu rDNA sequence (Microsporidium sp. PL01). Phylogenetic analysis showed that this species belongs to one of the two main clades in the Terresporidia. We found a high rate of microsporidian co-infections (9.4%). The numbers of sequence reads for the operational taxonomic units suggest that the occurrence of Nosema spp. in co-infections could benefit them; however, this observation should be retested using a more intensive host sampling. Our results show that DNA barcoding is a rapid and cost-effective method for deciphering sample diversity in greater resolution, including the hidden biodiversity that may be overlooked using classical methodology.</p><h1>Executing multi-taxa eDNA ecological assessment via traditional metrics and interactive networks</h1>
<p><a href="https://doi.org/S0048-9697(20)32318-4">doi:S0048-9697(20)32318-4</a></p>

<p><a href=None>None</a></p>
<p>Current approaches to ecological assessment are limited by the traditional morpho-taxonomic methods presently employed and the inability to meet increasing demands for rapid assessments. Advancements in high throughput sequencing now enable rapid high-resolution ecological assessment using environmental DNA (eDNA). Here we test the ability of using eDNA-based ecological assessment methods against traditional assessment of two key indicator groups (diatoms and macroinvertebrates) and show how eDNA across multiple gene regions (COI, rbcL, 12S and 18S) can be used to infer interactive networks that link to ecological assessment criteria. We compared results between taxonomic and eDNA based assessments and found significant positive associations between macroinvertebrate (p < 0.001 R<sup>2</sup> = 0.645) and diatom (p = 0.015, R<sup>2</sup> = 0.222) assessment metrics. We further assessed the ability of eDNA based assessment to identify environmentally sensitive genera and found an order of magnitude greater potential for 18S, versus COI or rbcL, to determine environmental filtering of ecologically assessed communities. Lastly, we compared the ability of traditional metrics against co-occurrence network properties of our combined 18S, COI and rbcL indicator genera to infer habitat quality measures currently used by managers. We found that transitivity (network connectivity), linkage density and cohesion were significantly associated with habitat modification scores (HMS), whereas network properties were inconsistent with linking to the habitat quality score (HQS) metric. The incorporation of multi-marker eDNA network assessment opens up a means for finer scale ecological assessment, currently limited using traditional methods. While utilization of eDNA-based assessment is recommended, direct comparisons with traditional approaches are difficult as the methods are intrinsically different and should be treated as such with regards to future research. Overall, our findings show that eDNA can be used for effective ecological assessment while offering a wider range of scope and application compared to traditional assessment methods.</p><h1>Rapid range shifts and megafaunal extinctions associated with late Pleistocene climate change</h1>
<p><a href="https://doi.org/10.1038/S41467-020-16502-3">doi:10.1038/S41467-020-16502-3</a></p>

<p><a href=None>None</a></p>
<p>Large-scale changes in global climate at the end of the Pleistocene significantly impacted ecosystems across North America. However, the pace and scale of biotic turnover in response to both the Younger Dryas cold period and subsequent Holocene rapid warming have been challenging to assess because of the scarcity of well dated fossil and pollen records that covers this period. Here we present an ancient DNA record from Hall's Cave, Texas, that documents 100 vertebrate and 45 plant taxa from bulk fossils and sediment. We show that local plant and animal diversity dropped markedly during Younger Dryas cooling, but while plant diversity recovered in the early Holocene, animal diversity did not. Instead, five extant and nine extinct large bodied animals disappeared from the region at the end of the Pleistocene. Our findings suggest that climate change affected the local ecosystem in Texas over the Pleistocene-Holocene boundary, but climate change on its own may not explain the disappearance of the megafauna at the end of the Pleistocene.</p><h1>Comparing diversity levels in environmental samples: DNA sequence capture and metabarcoding approaches using 18S and COI genes</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13201">doi:10.1111/1755-0998.13201</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA studies targeting multiple taxa using metabarcoding provide remarkable insights into levels of species diversity in any habitat. The main drawbacks are the presence of primer bias and difficulty in identifying rare species. We tested a DNA sequence-capture method in parallel with the metabarcoding approach to reveal possible advantages of one method over the other. Both approaches were performed using the same eDNA samples and the same 18S and COI regions, followed by high throughput sequencing. Metabarcoded eDNA libraries were PCR amplified with one primer pair from 18S and COI genes. DNA sequence-capture libraries were enriched with 3,639 baits targeting the same gene regions. We tested amplicon sequence variants (ASVs) and operational taxonomic units (OTUs) in silico approaches for both markers and methods, using for this purpose the metabarcoding data set. ASVs methods uncovered more species for the COI gene, whereas the opposite occurred for the 18S gene, suggesting that clustering reads into OTUs could bias diversity richness especially using 18S with relaxed thresholds. Additionally, metabarcoding and DNA sequence-capture recovered 80%-90% of the control sample species. DNA sequence-capture was 8x more expensive, nonetheless it identified 1.5x more species for COI and 13x more genera for 18S than metabarcoding. Both approaches offer reliable results, sharing ca. 40% species and 72% families and retrieve more taxa when nuclear and mitochondrial markers are combined. eDNA metabarcoding is quite well established and low-cost, whereas DNA-sequence capture for biodiversity assessment is still in its infancy, is more time-consuming but provides more taxonomic assignments.</p><h1>Quantitative evaluation of intraspecific genetic diversity in a natural fish population using environmental DNA analysis</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13200">doi:10.1111/1755-0998.13200</a></p>

<p><a href=None>None</a></p>
<p>Recent advances in environmental DNA (eDNA) analysis using high-throughput sequencing (HTS) enable evaluation of intraspecific genetic diversity in a population. As the intraspecific genetic diversity provides invaluable information for wildlife conservation and management, there is an increasing demand to apply eDNA analysis to population genetics and the phylogeography by quantitative evaluation of intraspecific diversity. However, quantitative evaluations of intraspecific genetic diversity using eDNA is not straightforward because the number of eDNA sequence reads obtained by HTS may not be an index of the quantity of eDNA. In this study, to quantitatively evaluate genetic diversity using eDNA analysis, we applied a quantitative eDNA metabarcoding method using the internal standard DNAs. We targeted Ayu (Plecoglossus altivelis altivelis) and added internal standard DNAs with known copy numbers to each eDNA sample obtained from three rivers during the library preparation process. The sequence reads of each Ayu haplotype were successfully converted to DNA copy numbers based on the relationship between the copy numbers and sequence reads of the internal standard DNAs. In all rivers, the calculated copy number of each haplotype showed a significant positive correlation with the haplotype frequency estimated by a capture-based survey. Furthermore, estimates of genetic indicators such as nucleotide diversity based on the eDNA copy numbers were comparable with those estimated based on a capture-based study. Our results demonstrate that eDNA analysis with internal standard DNAs enables reasonable quantification of intraspecific genetic diversity, and this method could thus be a promising tool in the field of population genetics and phylogeography.</p><h1>Surprising low diversity of the plant pathogen Phytophthora in Amazonian forests</h1>
<p><a href="https://doi.org/10.1111/1462-2920.15099">doi:10.1111/1462-2920.15099</a></p>

<p><a href=None>None</a></p>
<p>The genus Phytophthora represents a group of plant pathogens with broad global distribution. The majority of them cause the collar and root-rot of diverse plant species. Little is known about Phytophthora communities in forest ecosystems, especially in the Neotropical forests where natural enemies could maintain the huge plant diversity via negative density dependence. We characterized the diversity of soil-borne Phytophthora communities in the North French Guiana rainforest and investigated how they are structured by host identity and environmental factors. In this little-explored habitat, 250 soil cores were sampled from 10 plots hosting 10 different plant families across three forest environments (Terra Firme, Seasonally Flooded and White Sand). Phytophthora diversity was studied using a baiting approach and metabarcoding (High-Throughput Sequencing) on environmental DNA extracted from both soil samples and baiting-leaves. These three approaches revealed very similar communities, characterized by an unexpected low diversity of Phytophthora species, with the dominance of two cryptic species close to Phytophthora heveae. As expected, the Phytophthora community composition of the French Guiana rainforest was significantly impacted by the host plant family and environment. However, these plant pathogen communities are very small and are dominated by generalist species, questioning their potential roles as drivers of plant diversity in these Amazonian forests.</p><h1>Ecosystems monitoring powered by environmental genomics: A review of current strategies with an implementation roadmap</h1>
<p><a href="https://doi.org/10.1111/Mec.15472">doi:10.1111/Mec.15472</a></p>

<p><a href=None>None</a></p>
<p>A decade after environmental scientists integrated high-throughput sequencing technologies in their toolbox, the genomics-based monitoring of anthropogenic impacts on the biodiversity and functioning of ecosystems is yet to be implemented by regulatory frameworks. Despite the broadly acknowledged potential of environmental genomics to this end, technical limitations and conceptual issues still stand in the way of its broad application by end-users. In addition, the multiplicity of potential implementation strategies may contribute to a perception that the routine application of this methodology is premature or "in development", hence restraining regulators from binding these tools into legal frameworks. Here, we review recent implementations of environmental genomics-based methods, applied to the biomonitoring of ecosystems. By taking a general overview, without narrowing our perspective to particular habitats or groups of organisms, this paper aims to compare, review and discuss the strengths and limitations of four general implementation strategies of environmental genomics for monitoring: (a) Taxonomy-based analyses focused on identification of known bioindicators or described taxa; (b) De novo bioindicator analyses; (c) Structural community metrics including inferred ecological networks; and (d) Functional community metrics (metagenomics or metatranscriptomics). We emphasise the utility of the three latter strategies to integrate meiofauna and microorganisms that are not traditionally utilised in biomonitoring because of difficult taxonomic identification. Finally, we propose a roadmap for the implementation of environmental genomics into routine monitoring programmes that leverage recent analytical advancements, while pointing out current limitations and future research needs.</p><h1>Assessment of Ciguatera and Other Phycotoxin-Related Risks in Anaho Bay (Nuku Hiva Island, French Polynesia): Molecular, Toxicological, and Chemical Analyses of Passive Samplers</h1>
<p><a href="https://doi.org/E321">doi:E321</a></p>

<p><a href=None>None</a></p>
<p>Ciguatera poisoning is a foodborne illness caused by the consumption of seafood contaminated with ciguatoxins (CTXs) produced by dinoflagellates from the genera <i>Gambierdiscus</i> and <i>Fukuyoa</i>. The suitability of Solid Phase Adsorption Toxin Tracking (SPATT) technology for the monitoring of dissolved CTXs in the marine environment has recently been demonstrated. To refine the use of this passive monitoring tool in ciguateric areas, the effects of deployment time and sampler format on the adsorption of CTXs by HP20 resin were assessed in Anaho Bay (Nuku Hiva Island, French Polynesia), a well-known ciguatera hotspot. Toxicity data assessed by means of the mouse neuroblastoma cell-based assay (CBA-N2a) showed that a 24 h deployment of 2.5 g of resin allowed concentrating quantifiable amounts of CTXs on SPATT samplers. The CTX levels varied with increasing deployment time, resin load, and surface area. In addition to CTXs, okadaic acid (OA) and dinophysistoxin-1 (DTX1) were also detected in SPATT extracts using liquid chromatography coupled to tandem mass spectrometry (LC-MS/MS), consistent with the presence of <i>Gambierdiscus</i> and <i>Prorocentrum</i> species in the environment, as assessed by quantitative polymerase chain reaction (qPCR) and high-throughput sequencing (HTS) metabarcoding analyses conducted on passive window screen (WS) artificial substrate samples. Although these preliminary findings await further confirmation in follow-up studies, they highlight the usefulness of SPATT samplers in the routine surveillance of CP risk on a temporal scale, and the monitoring of other phycotoxin-related risks in ciguatera-prone areas.</p><h1>DNA Metabarcoding as a Tool for Disentangling Food Webs in Agroecosystems</h1>
<p><a href="https://doi.org/E294">doi:E294</a></p>

<p><a href=None>None</a></p>
<p>Better knowledge of food webs and related ecological processes is fundamental to understanding the functional role of biodiversity in ecosystems. This is particularly true for pest regulation by natural enemies in agroecosystems. However, it is generally difficult to decipher the impact of predators, as they often leave no direct evidence of their activity. Metabarcoding via high-throughput sequencing (HTS) offers new opportunities for unraveling trophic linkages between generalist predators and their prey, and ultimately identifying key ecological drivers of natural pest regulation. Here, this approach proved effective in deciphering the diet composition of key predatory arthropods (nine species.; 27 prey taxa), insectivorous birds (one species, 13 prey taxa) and bats (one species; 103 prey taxa) sampled in a millet-based agroecosystem in Senegal. Such information makes it possible to identify the diet breadth and preferences of predators (e.g., mainly moths for bats), to design a qualitative trophic network, and to identify patterns of intraguild predation across arthropod predators, insectivorous vertebrates and parasitoids. Appropriateness and limitations of the proposed molecular-based approach for assessing the diet of crop pest predators and trophic linkages are discussed.</p><h1>Vegetation type determines spore deposition within a forest-agricultural mosaic landscape</h1>
<p><a href="https://doi.org/Fiaa082">doi:Fiaa082</a></p>

<p><a href=None>None</a></p>
<p>Predicting fungal community assembly is partly limited by our understanding of the factors driving the composition of deposited spores. We studied the relative contribution of vegetation, geographical distance, seasonality and weather to fungal spore deposition across three vegetation types. Active and passive spore traps were established in agricultural fields, deciduous forests and coniferous forests across a geographic gradient of ∼600 km. Active traps captured the spore community suspended in air, reflecting the potential deposition, whereas passive traps reflected realized deposition. Fungal species were identified by metabarcoding of the ITS2 region. The composition of spore communities captured by passive traps differed more between vegetation types than across regions separated by >100 km, indicating that vegetation type was the strongest driver of composition of deposited spores. By contrast, vegetation contributed less to potential deposition, which followed a seasonal pattern. Within the same site, the spore communities captured by active traps differed from those captured by passive traps. Realized deposition tended to be dominated by spores of species related to vegetation. Temperature was negatively correlated with the fungal species richness of both potential and realized deposition. Our results indicate that vegetation may be able to maintain similar fungal communities across distances, and likely be the driving factor of fungal spore deposition at landscape level.</p><h1>Evaluation and sensitivity analysis of diatom DNA metabarcoding for WFD bioassessment of Mediterranean rivers</h1>
<p><a href="https://doi.org/S0048-9697(20)31958-6">doi:S0048-9697(20)31958-6</a></p>

<p><a href=None>None</a></p>
<p>Our study of 164 diatom samples from Catalonia (NE Spain) is the first to evaluate the applicability of DNA metabarcoding, based on high throughput sequencing (HTS) using a 312-bp rbcL marker, for biomonitoring Mediterranean rivers. For this, we compared the values of a biotic index (IPS) and the ecological status classes derived from them, between light microscope-based (LM) and HTS methods. Very good correspondence between methods gives encouraging results concerning the applicability of DNA metabarcoding for Catalan rivers for the EU Water Framework Directive (WFD). However, in 10 sites, the ecological status class was downgraded from "Good"/"High" obtained by LM to "Moderate"/"Poor"/"Bad" by HTS; these "critical" sites are especially important, because the WFD requires remedial action by water managers for any river with Moderate or lower status. We investigated the contribution of each species to the IPS using a "leave-one-out" sensitivity analysis, paying special attention to critical sites. Discrepancies in IPS between LM and HTS were mainly due to the misidentification and overlooking in LM of a few species, which were better recovered by HTS. This bias was particularly important in the case of Fistulifera saprophila, whose clear underrepresentation in LM was important for explaining 8 out of the 10 critical sites and probably reflected destruction of weakly-silicified frustules during sample preparation. Differences between species in the rbcL copy number per cell affected the relative abundance obtained by HTS for Achnanthidium minutissimum, Nitzschia inconspicua and Ulnaria ulna, which were also identified by the sensitivity analysis as important for the WFD. Only minor IPS discrepancies were attributed to the incompleteness of the reference library, as most of the abundant and influential species (to the IPS) were well represented there. Finally, we propose that leave-one-out analysis is a good method for identifying priority species for isolation and barcoding.</p><h1>Local factors drive bacterial and microeukaryotic community composition in lake surface sediment collected across an altitudinal gradient</h1>
<p><a href="https://doi.org/Fiaa070">doi:Fiaa070</a></p>

<p><a href=None>None</a></p>
<p>Lake surface sediments are dominated by microorganisms that play significant roles in biogeochemical cycling within lakes. There is limited knowledge on the relative importance of local environmental factors and altitude on bacterial and microeukaryotic community richness and composition in lake sediments. In the present study, surface sediment samples were collected from 40 lakes along an altitude gradient (2-1215 m). Microbial communities were characterized using 16S (bacteria) and 18S (microeukaryotes) rRNA gene metabarcoding. Bacterial and microeukaryotic richness were not correlated with altitude but instead to environmental variables (e.g. area of water in the catchment (bacteria: R = -0.43). For both bacteria and microeukaryotes, dissimilarity in the community structure had a higher correlation to combined environmental variables (without altitude) (bacteria: R = 0.53; microeukaryotes: R = 0.55) than altitude alone (bacteria: R = 0.34; microeukaryotes: R = 0.47). Sediment sulfur and productive grassland were important variables in determining the relative abundance of sulfate reducing bacteria. Nitrospira, was positively related to altitude but negatively to water column total organic carbon and the proportion of productive grassland in the catchment. Little overlap in amplicon sequence variants was shown amongst lakes. This has important considerations for management decisions, suggesting that to protect biodiversity, conservation of numerous lakes and lake types is required.</p><h1>Study on the correlation between soil microbial diversity and ambient environmental factors influencing the safflower distribution in Xinjiang</h1>
<p><a href="https://doi.org/10.1002/Jobm.201900626">doi:10.1002/Jobm.201900626</a></p>

<p><a href=None>None</a></p>
<p>The effects of soil microbial properties and physiographical factors on safflower distributions in the main safflower plantations of Xinjiang province in China were studied. This study may help determine the basis of the environmental factors for evaluating the geoherbalism of this medicinal plant. The soil microbial biodiversity in the bulk soil and rhizosphere of safflower at different growth stages and from different sampling plots were characterized by analyzing the environmental DNAs in the samples. With general primers targeting the 16S ribosomal DNA for bacteria and the internal transcribed spacer 1 gene for fungi, the study was performed using marker gene amplification coupled with Illumina HiSeq high-throughput sequencing technologies. Correlation analysis and a distance-based redundancy analysis were performed to determine the dominant factors affecting the distribution of the microorganism in safflower soils. A total of 16517 bacterial operational taxonomic units (OTUs) were obtained from all the 108 soil samples of nine safflower sampling plots. At the phylum level, 48 phyla have been identified with Actinobacteria (32.9%) and proteobacteria (28.7%) being predominant. For fungi, 8746 OTUs were obtained, which belonged to seven phyla with Ascomycota overwhelmingly superior in relative abundance. A significant positive correlation was found between soil microbe quantity and ASL (above sea level). Safflower was sensitive to changes in elevation, growing more abundantly in the mountainous regions at heights of around 1,200 m above sea level. It is concluded that the dominant factors affecting the distribution of microorganisms in safflower soils were soil moisture, available N, and ASL.</p><h1>Environmental DNA analysis shows high potential as a tool for estimating intraspecific genetic diversity in a wild fish population</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13165">doi:10.1111/1755-0998.13165</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis has recently been used as a new tool for estimating intraspecific diversity. However, whether known haplotypes contained in a sample can be detected correctly using eDNA-based methods has been examined only by an aquarium experiment. Here, we tested whether the haplotypes of Ayu fish (Plecoglossus altivelis altivelis) detected in a capture survey could also be detected from an eDNA sample derived from the field that contained various haplotypes with low concentrations and foreign substances. A water sample and Ayu specimens collected from a river on the same day were analysed by eDNA analysis and Sanger sequencing, respectively. The 10 L water sample was divided into 20 filters for each of which 15 PCR replications were performed. After high-throughput sequencing, denoising was performed using two of the most widely used denoising packages, unoise3 and dada2. Of the 42 haplotypes obtained from the Sanger sequencing of 96 specimens, 38 (unoise3) and 41 (dada2) haplotypes were detected by eDNA analysis. When dada2 was used, except for one haplotype, haplotypes owned by at least two specimens were detected from all the filter replications. Accordingly, although it is important to note that eDNA-based method has some limitations and some risk of false positive and false negative, this study showed that the eDNA analysis for evaluating intraspecific genetic diversity provides comparable results for large-scale capture-based conventional methods. Our results suggest that eDNA-based methods could become a more efficient survey method for investigating intraspecific genetic diversity in the field.</p><h1>A taxonomy-free approach based on machine learning to assess the quality of rivers with diatoms</h1>
<p><a href="https://doi.org/S0048-9697(20)31413-3">doi:S0048-9697(20)31413-3</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are a compulsory biological quality element in the ecological assessment of rivers according to the Water Framework Directive. The application of current official indices requires the identification of individuals to species or lower rank under a microscope based on the valve morphology. This is a highly time-consuming task, often susceptible of disagreements among analysts. In alternative, the use of DNA metabarcoding combined with High-Throughput Sequencing (HTS) has been proposed. The sequences obtained from environmental DNA are clustered into Operational Taxonomic Units (OTUs), which can be assigned to a taxon using reference databases, and from there calculate biotic indices. However, there is still a high percentage of unassigned OTUs to species due to the incompleteness of reference libraries. Alternatively, we tested a new taxonomy-free approach based on diatom community samples to assess rivers. A combination of three machine learning techniques is used to build models that predict diatom OTUs expected in test sites, under reference conditions, from environmental data. The Observed/Expected OTUs ratio indicates the deviation from reference condition and is converted into a quality class. This approach was never used with diatoms neither with OTUs data. To evaluate its efficiency, we built a model based on OTUs lists (HYDGEN) and another based on taxa lists from morphological identification (HYDMORPH), and also calculated a biotic index (IPS). The models were trained and tested with data from 81 sites (44 reference sites) from central Portugal. Both models were considered accurate (linear regression for Observed and Expected richness: R<sup>2</sup> ≈ 0.7, interception ≈ 0.8) and sensitive to global anthropogenic disturbance (Rs<sup>2</sup> > 0.30 p < 0.006 for global disturbance). Yet, the HYDGEN model based on molecular data was sensitive to more types of pressures (such as, changes in land use and habitat quality), which gives promising insights to its use for bioassessment of rivers.</p><h1>A Review and Perspective of eDNA Application to Eutrophication and HAB Control in Freshwater and Marine Ecosystems</h1>
<p><a href="https://doi.org/E417">doi:E417</a></p>

<p><a href=None>None</a></p>
<p>Changing ecological communities in response to anthropogenic activities and climate change has become a worldwide problem. The eutrophication of waterbodies in freshwater and seawater caused by the effects of human activities and nutrient inputs could result in harmful algae blooms (HABs), decreases water quality, reductions in biodiversity and threats to human health. Rapid and accurate monitoring and assessment of aquatic ecosystems are imperative. Environmental DNA (eDNA) analysis using high-throughput sequencing has been demonstrated to be an effective and sensitive assay for detecting and monitoring single or multiple species in different samples. In this study, we review the potential applications of eDNA approaches in controlling and mitigating eutrophication and HABs in freshwater and marine ecosystems. We use recent studies to highlight how eDNA methods have been shown to be a useful tool for providing comprehensive data in studies of eutrophic freshwater and marine environments. We also provide perspectives on using eDNA techniques to reveal molecular mechanisms in biological processes and mitigate eutrophication and HABs in aquatic ecosystems. Finally, we discuss the feasible applications of eDNA for monitoring biodiversity, surveying species communities and providing instructions for the conservation and management of the environment by integration with traditional methods and other advanced techniques.</p><h1>Assessing insect biodiversity with automatic light traps in Brazil: Pearls and pitfalls of metabarcoding samples in preservative ethanol</h1>
<p><a href="https://doi.org/10.1002/Ece3.6042">doi:10.1002/Ece3.6042</a></p>

<p><a href=None>None</a></p>
<p>Automated species identification based on data produced with metabarcoding offers an alternative for assessing biodiversity of bulk insect samples obtained with traps. We used a standard two-step PCR approach to amplify a 313 bp fragment of the barcoding region of the mitochondrial COI gene. The PCR products were sequenced on an Illumina MiSeq platform, and the OTUs production and taxonomic identifications were performed with a customized pipeline and database. The DNA used in the PCR procedures was extracted directly from the preservative ethanol of bulk insect samples obtained with automatic light traps in 12 sampling areas located in different biomes of Brazil, during wet and dry seasons. Agricultural field and forest edge habitats were collected for all sampling areas. A total of 119 insect OTUs and nine additional OTUs assigned to other arthropod taxa were obtained at a ≥97% sequence similarity level. The alpha and beta diversity analyses comparing biomes, habitats, and seasons were mostly inconclusive, except for a significant difference in beta diversity between biomes. In this study, we were able to metabarcode and HTS adult insects from their preservative medium. Notwithstanding, our results underrepresent the true magnitude of insect diversity expected from samples obtained with automatic light traps in Brazil. Although biological and technical factors might have impacted our results, measures to optimize and standardize eDNA HTS should be in place to improve taxonomic coverage of samples of unknown diversity and stored in suboptimal conditions, which is the case of most eDNA samples.</p><h1>Standards for Methods Utilizing Environmental DNA for Detection of Fish Species</h1>
<p><a href="https://doi.org/E296">doi:E296</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) techniques are gaining attention as cost-effective, non-invasive strategies for acquiring information on fish and other aquatic organisms from water samples. Currently, eDNA approaches are used to detect specific fish species and determine fish community diversity. Various protocols used with eDNA methods for aquatic organism detection have been reported in different eDNA studies, but there are no general recommendations for fish detection. Herein, we reviewed 168 papers to supplement and highlight the key criteria for each step of eDNA technology in fish detection and provide general suggestions for eliminating detection errors. Although there is no unified recommendation for the application of diverse eDNA in detecting fish species, in most cases, 1 or 2 L surface water collection and eDNA capture on 0.7-μm glass fiber filters followed by extraction with a DNeasy Blood and Tissue Kit or PowerWater DNA Isolation Kit are useful for obtaining high-quality eDNA. Subsequently, species-specific quantitative polymerase chain reaction (qPCR) assays based on mitochondrial cytochrome b gene markers or eDNA metabarcoding based on both <i>12S</i> and <i>16S rRNA</i> markers via high-throughput sequencing can effectively detect target DNA or estimate species richness. Furthermore, detection errors can be minimized by mitigating contamination, negative control, PCR replication, and using multiple genetic markers. Our aim is to provide a useful strategy for fish eDNA technology that can be applied by researchers, advisors, and managers.</p><h1>Towards conserving natural diversity: A biotic inventory by observations, specimens, DNA barcoding and high-throughput sequencing methods</h1>
<p><a href="https://doi.org/10.3897/Bdj.8.E50124">doi:10.3897/Bdj.8.E50124</a></p>

<p><a href=None>None</a></p>
<p>The Kenai National Wildlife Refuge has been given a broad conservation mandate to conserve natural diversity. A prerequisite for fulfilling this purpose is to be able to identify the species and communities that make up that biodiversity. We tested a set of varied methods for inventory and monitoring of plants, birds and terrestrial invertebrates on a grid of 40 sites in a 938 ha study area in the Slikok Creek watershed, Kenai Peninsula, Alaska. We sampled plants and lichens through observation and specimen-based methods. We surveyed birds using bird call surveys on variable circular plots. We sampled terrestrial arthropods by sweep net sampling, processing samples with High Throughput Sequencing methods. We surveyed for earthworms, using the hot mustard extraction method and identified worm specimens by morphology and DNA barcoding. We examined community membership using clustering methods and Nonmetric Multidimensional Scaling. We documented a total of 4,764 occurrences of 984 species and molecular operational taxonomic units: 87 vascular plants, 51 mosses, 12 liverworts, 111 lichens, 43 vertebrates, 663 arthropods, 9 molluscs and 8 annelid worms. Amongst these records, 102 of the arthropod species appeared to be new records for Alaska. We found three non-native species: <i>Deroceras agreste</i> (Linnaeus, 1758) (Stylommatophora: Agriolimacidae), <i>Dendrobaena octaedra</i> (Savigny, 1826) (Crassiclitellata: Lumbricidae) and <i>Heterarthrus nemoratus</i> (Fallén, 1808) (Hymenoptera: Tenthredinidae). Both <i>D. octaedra</i> and <i>H. nemoratus</i> were found at sites distant from obvious human disturbance. The 40 sites were grouped into five community groups: upland mixed forest, black spruce forest, open deciduous forest, shrub-sedge bog and willow. We demonstrated that, at least for a subset of species that could be detected using these methods, we were able to document current species distributions and assemblages in a way that could be efficiently repeated for the purposes of biomonitoring. While our methods could be improved and additional methods and groups could be added, our combination of techniques yielded a substantial portion of the data necessary for fulfilling Kenai National Wildlife Refuge's broad conservation purposes.</p><h1>Species detection and delineation in the marine planktonic diatoms Chaetoceros and Bacteriastrum through metabarcoding: making biological sense of haplotype diversity</h1>
<p><a href="https://doi.org/10.1111/1462-2920.14984">doi:10.1111/1462-2920.14984</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing (HTS) metabarcoding is commonly applied to assess phytoplankton diversity. Usually, haplotypes are grouped into operational taxonomic units (OTUs) through clustering, whereby the resulting number of OTUs depends on chosen similarity thresholds. We applied, instead, a phylogenetic approach to infer taxa among 18S rDNA V4-metabarcode haplotypes gathered from 48 time-series samples using the marine planktonic diatoms Chaetoceros and Bacteriastrum as test case. The 73 recovered taxa comprised both solitary haplotypes and polytomies, the latter composed each of a highly abundant, dominant haplotype and one to several minor, peripheral haplotypes. The solitary and dominant haplotypes usually matched reference sequences, enabling species assignation of taxa. We hypothesise that the super-abundance of reads in dominant haplotypes results from the homogenization effect of concerted evolution. Reads of populous peripheral haplotypes and dominant haplotypes show comparable distribution patterns over the sample dates, suggesting that they are part of the same population. Many taxa revealed marked seasonality, with closely related ones generally showing distinct periodicity, whereas others occur year-round. Phylogenies inferred from metabarcode haplotypes enable delineation of biologically meaningful taxa, whereas OTUs resulting from clustering algorithms often deviate markedly from such taxa.</p><h1>Land use as a driver for protist community structure in soils under agricultural use across Europe</h1>
<p><a href="https://doi.org/S0048-9697(20)30738-5">doi:S0048-9697(20)30738-5</a></p>

<p><a href=None>None</a></p>
<p>Soil biodiversity is threatened by intensification of land use. The consequences of different land use on belowground biodiversity remain insufficiently explored for soil protists. Alongside being abundant and extremely diverse in soil, protists provide many ecosystem services: key players in the microbial loop, turnover of organic matter and stimulation of plant growth-promoting rhizobacteria. However, we lack knowledge of effects of site, land use intensity and management on diversity of soil protists. Here we assessed protist communities in four European arable sites with contrasting land use intensities at each site: Lusignan, France; Moskanjci, Slovenia; Castro Verde, Portugal and Scheyern, Germany as well as two grassland sites: Hainich, Germany and Lancaster, UK. Each site has consistent agricultural management history of low and high land use intensities quantified in terms of land use index (LUI). We employed high-throughput sequencing of environmental DNA, targeting the V4 region of the 18S rRNA gene. By assigning the protist composition to trophic groups, we inspected for effects of management, and other biotic and abiotic variables. While overall protist richness was unaffected by LUI within sites, specific trophic groups such as plant pathogens and saprotrophs were affected. Effects on protist biome across land uses and sites were also observed. LUI sensitive taxa were taxonomically diverse in each plot, and their trophic groups responded in specific patterns to specific practices. The most abundant trophic group was phagotrophs (73%), followed by photoautotrophs (16%), plant pathogens (4%), animal parasites (2%) and saprotrophs (1%). Community compositions and factors affecting the structure of individual trophic groups differed between land uses and management systems. The agricultural management selected for distinct protist populations as well as specific functional traits, and the protist community and diversity were indeed affected by site, LUI and management, which indicates the ecological significance of protists in the soil food web.</p><h1>High-throughput sequencing for community analysis: the promise of DNA barcoding to uncover diversity, relatedness, abundances and interactions in spider communities</h1>
<p><a href="https://doi.org/10.1007/S00427-020-00652-X">doi:10.1007/S00427-020-00652-X</a></p>

<p><a href=None>None</a></p>
<p>Large-scale studies on community ecology are highly desirable but often difficult to accomplish due to the considerable investment of time, labor and, money required to characterize richness, abundance, relatedness, and interactions. Nonetheless, such large-scale perspectives are necessary for understanding the composition, dynamics, and resilience of biological communities. Small invertebrates play a central role in ecosystems, occupying critical positions in the food web and performing a broad variety of ecological functions. However, it has been particularly difficult to adequately characterize communities of these animals because of their exceptionally high diversity and abundance. Spiders in particular fulfill key roles as both predator and prey in terrestrial food webs and are hence an important focus of ecological studies. In recent years, large-scale community analyses have benefitted tremendously from advances in DNA barcoding technology. High-throughput sequencing (HTS), particularly DNA metabarcoding, enables community-wide analyses of diversity and interactions at unprecedented scales and at a fraction of the cost that was previously possible. Here, we review the current state of the application of these technologies to the analysis of spider communities. We discuss amplicon-based DNA barcoding and metabarcoding for the analysis of community diversity and molecular gut content analysis for assessing predator-prey relationships. We also highlight applications of the third generation sequencing technology for long read and portable DNA barcoding. We then address the development of theoretical frameworks for community-level studies, and finally highlight critical gaps and future directions for DNA analysis of spider communities.</p><h1>High-throughput DNA barcoding of oligochaetes for abundance-based indices to assess the biological quality of sediments in streams and lakes</h1>
<p><a href="https://doi.org/10.1038/S41598-020-58703-2">doi:10.1038/S41598-020-58703-2</a></p>

<p><a href=None>None</a></p>
<p>Aquatic oligochaete communities are valuable indicators of the biological quality of sediments in streams and lakes, but identification of specimens to the species level based on morphological features requires solid expertise in taxonomy and is possible only for a fraction of specimens present in a sample. The identification of aquatic oligochaetes using DNA barcodes would facilitate their use in biomonitoring and allow a wider use of this taxonomic group for ecological diagnoses. Previous approaches based on DNA metabarcoding of samples composed of total sediments or pools of specimens have been proposed for assessing the biological quality of ecosystems, but such methods do not provide precise information on species abundance, which limits the value of resulting ecological diagnoses. Here, we tested how a DNA barcoding approach based on high-throughput sequencing of sorted and genetically tagged specimens performed to assess oligochaete species diversity and abundance and the biological quality of sediments in streams and lakes. We applied both molecular and morphological approaches at 13 sites in Swiss streams and at 7 sites in Lake Geneva. We genetically identified 33 or 66 specimens per site. For both approaches, we used the same index calculations. We found that the ecological diagnoses derived from the genetic approach matched well with those of the morphological approach and that the genetic identification of only 33 specimens per site provided enough ecological information for correctly estimating the biological quality of sediments in streams and lakes.</p><h1>Diversity and microhabitat associations of Labyrinthula spp. in the Indian River Lagoon System</h1>
<p><a href="https://doi.org/10.3354/Dao03431">doi:10.3354/Dao03431</a></p>

<p><a href=None>None</a></p>
<p>Seagrasses create foundational habitats in coastal ecosystems. One contributing factor to their global decline is disease, primarily caused by parasites in the genus Labyrinthula. To explore the relationship between seagrass and Labyrinthula spp. diversity in coastal waters, we examined the diversity and microhabitat association of Labyrinthula spp. in 2 inlets on Florida's Atlantic Coast, the Indian River Lagoon (IRL) and Banana River. We used amplicon-based high throughput sequencing with 2 newly designed primers to amplify Labyrinthula spp. from 5 seagrass species, water, and sediments to determine their spatial distribution and microhabitat associations. The SSU primer set identified 12 Labyrinthula zero-radius operational taxonomic units (ZOTUs), corresponding to at least 8 putative species. The ITS1 primer set identified 2 ZOTUs, corresponding to at least 2 putative species. Based on our phylogenetic analyses, which include sequences from previous studies that assigned seagrass-related pathogenicity to Labyrinthula clades, all but one of the ZOTUs that we recovered with the SSU primers were from non-pathogenic species, while the 2 ZOTUs recovered with the ITS1 primers were from pathogenic species. Some of the ZOTUs were widespread across the sampling sites and microhabitats (e.g. SSU ZOTU_10), and most were present in more than one site. Our results demonstrate that targeted metabarcoding is a useful tool for examining the relationships between seagrass and Labyrinthula diversity in coastal waters.</p><h1>Absolute quantitation of microbes using 16S rRNA gene metabarcoding: A rapid normalization of relative abundances by quantitative PCR targeting a 16S rRNA gene spike-in standard</h1>
<p><a href="https://doi.org/10.1002/Mbo3.977">doi:10.1002/Mbo3.977</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of the 16S rRNA gene is commonly used to characterize microbial communities, by estimating the relative abundance of microbes. Here, we present a method to retrieve the concentrations of the 16S rRNA gene per gram of any environmental sample using a synthetic standard in minuscule amounts (100 ppm to 1% of the 16S rRNA sequences) that is added to the sample before DNA extraction and quantified by two quantitative polymerase chain reaction (qPCR) reactions. This allows normalizing by the initial microbial density, taking into account the DNA recovery yield. We quantified the internal standard and the total load of 16S rRNA genes by qPCR. The qPCR for the latter uses the exact same primers as those used for Illumina sequencing of the V3-V4 hypervariable regions of the 16S rRNA gene to increase accuracy. We are able to calculate the absolute concentration of the species per gram of sample, taking into account the DNA recovery yield. This is crucial for an accurate estimate as the yield varied between 40% and 84%. This method avoids sacrificing a high proportion of the sequencing effort to quantify the internal standard. If sacrificing a part of the sequencing effort to the internal standard is acceptable, we however recommend that the internal standard accounts for 30% of the environmental 16S rRNA genes to avoid the PCR bias associated with rare phylotypes. The method proposed here was tested on a feces sample but can be applied more broadly on any environmental sample. This method offers a real improvement of metabarcoding of microbial communities since it makes the method quantitative with limited efforts.</p><h1>Exploring the taxonomic composition of two fungal communities on the Swedish west coast through metabarcoding</h1>
<p><a href="https://doi.org/10.3897/Bdj.7.E35332">doi:10.3897/Bdj.7.E35332</a></p>

<p><a href=None>None</a></p>
<p>Fungi are heterotrophic, unicellular or filamentous organisms that exhibit a wide range of different lifestyles as, e.g., symbionts, parasites, and saprotrophs. Mycologists have traditionally considered fungi to be a nearly exclusively terrestrial group of organisms, but it is now known that fungi have a significant presence in aquatic environments as well. We know little about most fungi in limnic and marine systems, including aspects of their taxonomy, ecology, and geographic distribution. The present study seeks to improve our knowledge of fungi in the marine environment. The fungal communities of two coastal marine environments of the Kattegat sea, Sweden, were explored with metabarcoding techniques using the nuclear ribosomal internal transcribed spacer 2 (ITS2) metabarcode. Our data add new information to the current picture of fungal community composition in benthic and coastal habitats in Northern Europe.</p><h1>Host tree organ is the primary driver of endophytic fungal community structure in a hemiboreal forest</h1>
<p><a href="https://doi.org/Fiz199">doi:Fiz199</a></p>

<p><a href=None>None</a></p>
<p>Despite numerous studies on plant endophytes, little is known about fungal communities associated with different aboveground tissues of living trees. We used high-throughput sequencing to compare the diversity and community structure of fungi inhabiting leaves, branches and trunks of Alnus incana and Corylus avellana growing at three hemiboreal forest sites. Our analysis revealed that tree organs are the main determinants of the structure of fungal communities, whereas the effects of host species and locality remained secondary and negligible, respectively. The structure of fungal communities in trunks was the most distinct compared to that in leaves and branches. The foliar fungal communities were more similar within than between individual trees, implying that certain fungi may grow through parts of the tree crown. The weak effect of locality compared to host organs and species identity suggests that the structural variation of fungal communities in the aboveground parts of trees depends mainly on deterministic factors rather than dispersal limitation.</p><h1>Long-read metabarcoding of the eukaryotic rDNA operon to phylogenetically and taxonomically resolve environmental diversity</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13117">doi:10.1111/1755-0998.13117</a></p>

<p><a href=None>None</a></p>
<p>High-throughput DNA metabarcoding of amplicon sizes below 500 bp has revolutionized the analysis of environmental microbial diversity. However, these short regions contain limited phylogenetic signal, which makes it impractical to use environmental DNA in full phylogenetic inferences. This lesser phylogenetic resolution of short amplicons may be overcome by new long-read sequencing technologies. To test this idea, we amplified soil DNA and used PacBio Circular Consensus Sequencing (CCS) to obtain an ~4500-bp region spanning most of the eukaryotic small subunit (18S) and large subunit (28S) ribosomal DNA genes. We first treated the CCS reads with a novel curation workflow, generating 650 high-quality operational taxonomic units (OTUs) containing the physically linked 18S and 28S regions. To assign taxonomy to these OTUs, we developed a phylogeny-aware approach based on the 18S region that showed greater accuracy and sensitivity than similarity-based methods. The taxonomically annotated OTUs were then combined with available 18S and 28S reference sequences to infer a well-resolved phylogeny spanning all major groups of eukaryotes, allowing us to accurately derive the evolutionary origin of environmental diversity. A total of 1,019 sequences were included, of which a majority (58%) corresponded to the new long environmental OTUs. The long reads also allowed us to directly investigate the relationships among environmental sequences themselves, which represents a key advantage over the placement of short reads on a reference phylogeny. Together, our results show that long amplicons can be treated in a full phylogenetic framework to provide greater taxonomic resolution and a robust evolutionary perspective to environmental DNA.</p><h1>eDNA metabarcoding in zooplankton improves the ecological status assessment of aquatic ecosystems</h1>
<p><a href="https://doi.org/S0160-4120(19)32222-6">doi:S0160-4120(19)32222-6</a></p>

<p><a href=None>None</a></p>
<p>Aquatic ecosystems are monitored worldwide using a range of biological quality elements that are morphologically identified. The environmental DNA (eDNA)-based approach has unprecedented advantages (e.g., high throughput, high efficiency and low cost) for biodiversity surveys in both freshwater and marine ecosystems compared with traditional sampling and image recognition. The use of eDNA has been mostly limited to biodiversity estimation, how to apply the eDNA approach in assessing the ecological health status is largely unexplored. Here, using zooplankton as an example, we examined the application of eDNA monitoring for ecological status assessment in an aquatic ecosystem. The results showed that eDNA monitoring reflected the spatial and temporal variations in zooplankton structure. Both species composition and bio-interactions varied significantly between sampling seasons (dry, normal and wet). A total of 60 different zooplankton indices were calculated based on eDNA monitoring and most of these indices were highly correlated with the level of water pollution, which was indicated by the water quality index in one or all three seasons. Both qualitative and quantitative eDNA-based biological indices were correlated with water quality. The season-dependent eDNA zooplankton integrity index (IZI) reflected the ecological status, and this method improves the timeliness of bioassessment.</p><h1>Foraging strategies are maintained despite workforce reduction: A multidisciplinary survey on the pollen collected by a social pollinator</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0224037">doi:10.1371/Journal.Pone.0224037</a></p>

<p><a href=None>None</a></p>
<p>The way pollinators gather resources may play a key role for buffering their population declines. Social pollinators like bumblebees could adjust their foraging after significant workforce reductions to keep provisions to the colony optimal, especially in terms of pollen diversity and quantity. To test what effects a workforce reduction causes on the foraging for pollen, commercially-acquired colonies of the bumblebee Bombus terrestris were allowed to forage in the field and they were experimentally manipulated by removing half the number of workers. For each bumblebee, the pollen pellets were taxonomically identified with DNA metabarcoding of the ITS2 region followed by a statistical filtering based on ROC curves to filter out underrepresented OTUs. Video cameras and network analyses were employed to investigate changes in foraging strategies and behaviour. After filtering out the false-positives, HTS metabarcoding yielded a high plant diversity in the pollen pellets; for plant identity and pollen quantity traits no differences emerged between samples from treated and from control colonies, suggesting that plant choice was influenced mainly by external factors such as the plant phenology. The colonies responded to the removal of 50% of their workers by increasing the foraging activity of the remaining workers, while only negligible changes were found in diet breadth and indices describing the structure of the pollen transport network. Therefore, a consistency in the bumblebees' feeding strategies emerges in the short term despite the lowered workforce.</p><h1>Genetic Biomonitoring and Biodiversity Assessment Using Portable Sequencing Technologies: Current Uses and Future Directions</h1>
<p><a href="https://doi.org/E858">doi:E858</a></p>

<p><a href=None>None</a></p>
<p>We live in an era of unprecedented biodiversity loss, affecting the taxonomic composition of ecosystems worldwide. The immense task of quantifying human imprints on global ecosystems has been greatly simplified by developments in high-throughput DNA sequencing technology (HTS). Approaches like DNA metabarcoding enable the study of biological communities at unparalleled detail. However, current protocols for HTS-based biodiversity exploration have several drawbacks. They are usually based on short sequences, with limited taxonomic and phylogenetic information content. Access to expensive HTS technology is often restricted in developing countries. Ecosystems of particular conservation priority are often remote and hard to access, requiring extensive time from field collection to laboratory processing of specimens. The advent of inexpensive mobile laboratory and DNA sequencing technologies show great promise to facilitate monitoring projects in biodiversity hot-spots around the world. Recent attention has been given to portable DNA sequencing studies related to infectious organisms, such as bacteria and viruses, yet relatively few studies have focused on applying these tools to Eukaryotes, such as plants and animals. Here, we outline the current state of genetic biodiversity monitoring of higher Eukaryotes using Oxford Nanopore Technology's MinION portable sequencing platform, as well as summarize areas of recent development.</p><h1>Microbiota Associated with Dromedary Camel Milk from Algerian Sahara</h1>
<p><a href="https://doi.org/10.1007/S00284-019-01788-4">doi:10.1007/S00284-019-01788-4</a></p>

<p><a href=None>None</a></p>
<p>Despite the fact that camel milk represents a valuable food source, the fungal diversity of raw camel milk has been poorly studied so far. Here, we investigated the fungal and bacterial communities found in dromedary camel milk from Ghardaia, a representative region of Algerian Sahara. The application of both culture-dependent and independent molecular techniques, based on dHPLC analysis and metabarcoding of ITS region, provided a complementary biodiversity assessment of camel milk fungi which was composed of 15 different taxa. Yeast species belonged to Filobasidium, Naganishia, Malassezia, Mrakia, Rhodotorula, and Yarrowia genera; and mold species belonged to Fusarium, Cladosporium, and Penicillium genera. All three techniques revealed that the fungal community was dominated by species belonging to the former genus Cryptococcus (Filobasidium and Naganishia) although none of them was able to encompass the entire fungal diversity alone. In addition, massive parallel 16S rRNA tag sequencing was applied to gain an insight into the diversity of bacterial communities which were dominated by Pseudomonas spp. Our results provide an initial insight about fungal and bacterial population found in dromedary camel milk from Algerian Sahara.</p><h1>Latent Dirichlet Allocation reveals spatial and taxonomic structure in a DNA-based census of soil biodiversity from a tropical forest</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13109">doi:10.1111/1755-0998.13109</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of amplicons from environmental DNA samples permits rapid, standardized and comprehensive biodiversity assessments. However, retrieving and interpreting the structure of such data sets requires efficient methods for dimensionality reduction. Latent Dirichlet Allocation (LDA) can be used to decompose environmental DNA samples into overlapping assemblages of co-occurring taxa. It is a flexible model-based method adapted to uneven sample sizes and to large and sparse data sets. Here, we compare LDA performance on abundance and occurrence data, and we quantify the robustness of the LDA decomposition by measuring its stability with respect to the algorithm's initialization. We then apply LDA to a survey of 1,131 soil DNA samples that were collected in a 12-ha plot of primary tropical forest and amplified using standard primers for bacteria, protists, fungi and metazoans. The analysis reveals that bacteria, protists and fungi exhibit a strong spatial structure, which matches the topographical features of the plot, while metazoans do not, confirming that microbial diversity is primarily controlled by environmental variation at the studied scale. We conclude that LDA is a sensitive, robust and computationally efficient method to detect and interpret the structure of large DNA-based biodiversity data sets. We finally discuss the possible future applications of this approach for the study of biodiversity.</p><h1>GenBank is a reliable resource for 21st century biodiversity research</h1>
<p><a href="https://doi.org/10.1073/Pnas.1911714116">doi:10.1073/Pnas.1911714116</a></p>

<p><a href=None>None</a></p>
<p>Traditional methods of characterizing biodiversity are increasingly being supplemented and replaced by approaches based on DNA sequencing alone. These approaches commonly involve extraction and high-throughput sequencing of bulk samples from biologically complex communities or samples of environmental DNA (eDNA). In such cases, vouchers for individual organisms are rarely obtained, often unidentifiable, or unavailable. Thus, identifying these sequences typically relies on comparisons with sequences from genetic databases, particularly GenBank. While concerns have been raised about biases and inaccuracies in laboratory and analytical methods, comparatively little attention has been paid to the taxonomic reliability of GenBank itself. Here we analyze the metazoan mitochondrial sequences of GenBank using a combination of distance-based clustering and phylogenetic analysis. Because of their comparatively rapid evolutionary rates and consequent high taxonomic resolution, mitochondrial sequences represent an invaluable resource for the detection of the many small and often undescribed organisms that represent the bulk of animal diversity. We show that metazoan identifications in GenBank are surprisingly accurate, even at low taxonomic levels (likely <1% error rate at the genus level). This stands in contrast to previously voiced concerns based on limited analyses of particular groups and the fact that individual researchers currently submit annotated sequences to GenBank without significant external taxonomic validation. Our encouraging results suggest that the rapid uptake of DNA-based approaches is supported by a bioinformatic infrastructure capable of assessing both the losses to biodiversity caused by global change and the effectiveness of conservation efforts aimed at slowing or reversing these losses.</p><h1>Stream Algal Biofilm Community Diversity Along An Acid Mine Drainage Recovery Gradient Using Multimarker Metabarcoding</h1>
<p><a href="https://doi.org/10.1111/Jpy.12935">doi:10.1111/Jpy.12935</a></p>

<p><a href=None>None</a></p>
<p>In southeastern Ohio, active remediation of streams affected by Acid Mine Drainage (AMD) has proven to be successful for some streams, while others have not recovered based on macroinvertebrate assessment. In this study, biofilms were collected from three Moderately Impaired, three Recovered, and two Unimpaired streams. The biodiversity was characterized by metabarcoding using two universal barcode markers (16S and 18S) along with two algal specific markers (UPA and rbcL) and high-throughput amplicon sequencing. For each marker, the ordination of Bray-Curtis Index calculated from the total Amplicon Sequence Variants (ASVs) present in each stream showed the Unimpaired and Recovered streams clustered, while Moderately Impaired streams were more distant. Focusing on the algal ASVs, the Shannon index for the rbcL, and UPA markers showed significantly lower alpha diversity in Moderately Impaired streams compared to Unimpaired streams, but the Recovered streams were not significantly different from the other two stream categories. The two universal markers together captured all algal phyla providing an outline of the diversity, but the two algal specific markers produced a greater number of ASVs and taxonomic depth for algal taxa. Further examination of the UPA marker revealed a drastic decrease in relative abundance of diatoms in Moderately Impaired streams compared to Recovered and Unimpaired streams. Likewise, diatom genera identified in the rbcL data and indicative of stream water quality showed marked differences in relative abundance among stream categories. Although all markers were useful, the algal-specific UPA and rbcL contributed more insights into algal community differences among stream categories.</p><h1>Genetic diversity of primate strongylid nematodes: Do sympatric nonhuman primates and humans share their strongylid worms?</h1>
<p><a href="https://doi.org/10.1111/Mec.15257">doi:10.1111/Mec.15257</a></p>

<p><a href=None>None</a></p>
<p>The close phylogenetic relationship between humans and nonhuman primates (NHPs) can result in a high potential for pathogen exchange. In recent decades, NHP and human interactions have become more frequent due to increasing habitat encroachment and ecotourism. Strongylid communities, which include members of several genera, are typically found in NHPs. Using optimized high-throughput sequencing for strain-level identification of primate strongylids, we studied the structure of strongylid communities in NHPs and humans co-habiting a tropical forest ecosystem in the Central African Republic. General taxonomic assignment of 85 ITS-2 haplotypes indicated that the studied primates harbour at least nine genera of strongylid nematodes, with Oesophagostomum and Necator being the most prevalent. We detected both host-specific and shared strongylid haplotypes. Skin-penetrating Necator gorillaehaplotypes were shared between humans and gorillas but Necator americanus were much more restricted to humans. Strongylid communities of local hunter-gatherers employed as trackers were more similar to those of gorillas compared to their relatives, who spent more time in villages. This was due to lower abundance of human-origin N. americanus in both gorillas and trackers. Habituated gorillas or those under habituation did not show larger overlap of strongylids with humans compared to unhabituated. We concluded that the occurrence of the human-specific strongylids in gorillas does not increase with direct contact between gorillas and humans due to the habituation. Overall, our results indicate that the degree of habitat sharing between hosts, together with mode of parasite transmission, are important factors for parasite spillover among primates.</p><h1>DNA metabarcoding adds valuable information for management of biodiversity in roadside stormwater ponds</h1>
<p><a href="https://doi.org/10.1002/Ece3.5503">doi:10.1002/Ece3.5503</a></p>

<p><a href=None>None</a></p>
<p>Stormwater ponds are used to compensate for the adverse effects that road runoff might have on the natural environment. Depending on their design and placement, stormwater ponds can act as both refugia and traps for local biodiversity. To evaluate the impact of stormwater ponds on biodiversity, it is critical to use effective and precise methods for identification of life associated with the water body. DNA metabarcoding has recently become a promising tool for identification and assessment of freshwater biodiversity.Using both morphology and DNA metabarcoding, we analyze species richness and biological composition of samples from 12 stormwater ponds and investigate the impact of pond size and pollution levels in the sediments and water column on the macroinvertebrate community structure.DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. The (dis)similarity of macroinvertebrate community composition in different ponds showed that the ponds appear better separated in the results obtained by DNA metabarcoding, but that the explained variation is higher for the results obtained by morphologically identification, since it provides abundance data.The reliance on morphological methods has limited our perception of the aquatic biodiversity in response to anthropogenic stressors, thereby providing inaccurate information for appropriate design and management of stormwater ponds; these drawbacks can be overcome by DNA metabarcoding. <i>Synthesis and applications</i>. The results indicate that DNA metabarcoding is a useful tool in identifying species, especially Diptera, which are difficult to determine. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing a more accurate information regarding the way the ponds function and how they are affected by management.</p><h1>COI metabarcoding primer choice affects richness and recovery of indicator taxa in freshwater systems</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0220953">doi:10.1371/Journal.Pone.0220953</a></p>

<p><a href=None>None</a></p>
<p>Mixed community or environmental DNA marker gene sequencing has become a commonly used technique for biodiversity analyses in freshwater systems. Many cytochrome c oxidase subunit I (COI) primer sets are now available for such work. The purpose of this study is to test whether COI primer choice affects the recovery of arthropod richness, beta diversity, and recovery of target assemblages in the benthos kick-net samples typically used in freshwater biomonitoring. We examine six commonly used COI primer sets on samples collected from six freshwater sites. Biodiversity analyses show that richness is sensitive to primer choice and the combined use of multiple COI amplicons recovers higher richness. Thus, to recover maximum richness, multiple primer sets should be used with COI metabarcoding. In ordination analyses based on community dissimilarity, samples consistently cluster by site regardless of amplicon choice or PCR replicate. Thus, for broadscale community analyses, overall beta diversity patterns are robust to COI marker choice. Recovery of traditional freshwater bioindicator assemblages such as Ephemeroptera, Trichoptera, Plectoptera, and Chironomidae as well as Arthropoda site indicators were differentially detected by each amplicon tested. This work will help future biodiversity and biomonitoring studies develop not just standardized, but optimized workflows that either maximize taxon-detection or the selection of amplicons for water quality or Arthropoda site indicators.</p><h1>Global distribution and diversity of <i>Chaetoceros</i> (Bacillariophyta, Mediophyceae): integration of classical and novel strategies</h1>
<p><a href="https://doi.org/10.7717/Peerj.7410">doi:10.7717/Peerj.7410</a></p>

<p><a href=None>None</a></p>
<p>Information on taxa distribution is a prerequisite for many research fields, and biological records are a major source of data contributing to biogeographic studies. The Global Biodiversity Information Facility (GBIF) and the Ocean Biogeographic Information System (OBIS) are important infrastructures facilitating free and open access to classical biological data from several sources in both temporal and spatial scales. Over the last ten years, high throughput sequencing (HTS) metabarcoding data have become available, which constitute a great source of detailed occurrence data. Among the global sampling projects that have contributed to such data are Tara Oceans and the Ocean Sampling Day (OSD). Integration of classical and metabarcoding data may aid a more comprehensive assessment of the geographic range of species, especially of microscopic ones such as protists. Rare, small and cryptic species are often ignored in surveys or mis-assigned with the classical approaches. Here we show how integration of data from various sources can contribute to insight in the biogeography and diversity at the genus- and species-level using <i>Chaetoceros</i> as study system<i>,</i> one of the most diverse and abundant genera among marine planktonic diatoms. <i>Chaetoceros</i> records were extracted from GBIF and OBIS and literature data were collected by means of a Google Scholar search. <i>Chaetoceros</i> references barcodes where mapped against the metabarcode datasets of Tara Oceans (210 sites) and OSD (144 sites). We compared the resolution of different data sources in determining the global distribution of the genus and provided examples, at the species level, of detection of cryptic species, endemism and cosmopolitan or restricted distributions. Our results highlighted at genus level a comparable picture from the different sources but a more complete assessment when data were integrated. Both the importance of the integration but also the challenges related to it were illustrated. <i>Chaetoceros</i> data collected in this study are organised and available in the form of tables and maps, providing a powerful tool and a baseline for further research in e.g., ecology, conservation and evolutionary biology.</p><h1>A novel metabarcoding diagnostic tool to explore protozoan haemoparasite diversity in mammals: a proof-of-concept study using canines from the tropics</h1>
<p><a href="https://doi.org/10.1038/S41598-019-49118-9">doi:10.1038/S41598-019-49118-9</a></p>

<p><a href=None>None</a></p>
<p>Haemoparasites are responsible for some of the most prevalent and debilitating canine illnesses across the globe, whilst also posing a significant zoonotic risk to humankind. Nowhere are the effects of such parasites more pronounced than in developing countries in the tropics where the abundance and diversity of ectoparasites that transmit these pathogens reaches its zenith. Here we describe the use of a novel next-generation sequencing (NGS) metabarcoding based approach to screen for a range of blood-borne apicomplexan and kinetoplastid parasites from populations of temple dogs in Bangkok, Thailand. Our methodology elucidated high rates of Hepatozoon canis and Babesia vogeli infection, whilst also being able to characterise co-infections. In addition, our approach was confirmed to be more sensitive than conventional endpoint PCR diagnostic methods. Two kinetoplastid infections were also detected, including one by Trypanosoma evansi, a pathogen that is rarely screened for in dogs and another by Parabodo caudatus, a poorly documented organism that has been previously reported inhabiting the urinary tract of a dog with haematuria. Such results demonstrate the power of NGS methodologies to unearth rare and unusual pathogens, especially in regions of the world where limited information on canine vector-borne haemoparasites exist.</p><h1>Current insights into fungal species diversity and perspective on naming the environmental DNA sequences of fungi</h1>
<p><a href="https://doi.org/10.1080/21501203.2019.1614106">doi:10.1080/21501203.2019.1614106</a></p>

<p><a href=None>None</a></p>
<p>The global bio-diversity of fungi has been extensively investigated and their species number has been estimated. Notably, the development of molecular phylogeny has revealed an unexpected fungal diversity and utilisation of culture-independent approaches including high-throughput amplicon sequencing has dramatically increased number of fungal operational taxonomic units. A number of novel taxa including new divisions, classes, orders and new families have been established in last decade. Many cryptic species were identified by molecular phylogeny. Based on recently generated data from culture-dependent and -independent survey on same samples, the fungal species on the earth were estimated to be 12 (11.7-13.2) million compared to 2.2-3.8 million species recently estimated by a variety of the estimation techniques. Moreover, it has been speculated that the current use of high-throughput sequencing techniques would reveal an even higher diversity than our current estimation. Recently, the formal classification of environmental sequences and permission of DNA sequence data as fungal names' type were proposed but strongly objected by the mycologist community. Surveys on fungi in unusual niches have indicated that many previously regarded "unculturable fungi" could be cultured on certain substrates under specific conditions. Moreover, the high-throughput amplicon sequencing, shotgun metagenomics and a single-cell genomics could be a powerful means to detect novel taxa. Here, we propose to separate the fungal types into physical type based on specimen, genome DNA (gDNA) type based on complete genome sequence of culturable and uncluturable fungal specimen and digital type based on environmental DNA sequence data. The physical and gDNA type should have priority, while the digital type can be temporal supplementary before the physical type and gDNA type being available. The fungal name based on the "digital type" could be assigned as the "clade" name + species name. The "clade" name could be the name of genus, family or order, etc. which the sequence of digital type affiliates to. Facilitating future cultivation efforts should be encouraged. Also, with the advancement in knowledge of fungi inhabiting various environments mostly because of rapid development of new detection technologies, more information should be expected for fungal diversity on our planet.</p><h1>Relative Performance of MinION (Oxford Nanopore Technologies) versus Sequel (Pacific Biosciences) Third-Generation Sequencing Instruments in Identification of Agricultural and Forest Fungal Pathogens</h1>
<p><a href="https://doi.org/E01368-19">doi:E01368-19</a></p>

<p><a href=None>None</a></p>
<p>Culture-based molecular identification methods have revolutionized detection of pathogens, yet these methods are slow and may yield inconclusive results from environmental materials. The second-generation sequencing tools have much-improved precision and sensitivity of detection, but these analyses are costly and may take several days to months. Of the third-generation sequencing techniques, the portable MinION device (Oxford Nanopore Technologies) has received much attention because of its small size and possibility of rapid analysis at reasonable cost. Here, we compare the relative performances of two third-generation sequencing instruments, MinION and Sequel (Pacific Biosciences), in identification and diagnostics of fungal and oomycete pathogens from conifer (Pinaceae) needles and potato (<i>Solanum tuberosum</i>) leaves and tubers. We demonstrate that the Sequel instrument is efficient for metabarcoding of complex samples, whereas MinION is not suited for this purpose due to a high error rate and multiple biases. However, we find that MinION can be utilized for rapid and accurate identification of dominant pathogenic organisms and other associated organisms from plant tissues following both amplicon-based and PCR-free metagenomics approaches. Using the metagenomics approach with shortened DNA extraction and incubation times, we performed the entire MinION workflow, from sample preparation through DNA extraction, sequencing, bioinformatics, and interpretation, in 2.5 h. We advocate the use of MinION for rapid diagnostics of pathogens and potentially other organisms, but care needs to be taken to control or account for multiple potential technical biases.<b>IMPORTANCE</b> Microbial pathogens cause enormous losses to agriculture and forestry, but current combined culturing- and molecular identification-based detection methods are too slow for rapid identification and application of countermeasures. Here, we develop new and rapid protocols for Oxford Nanopore MinION-based third-generation diagnostics of plant pathogens that greatly improve the speed of diagnostics. However, due to high error rate and technical biases in MinION, the Pacific BioSciences Sequel platform is more useful for in-depth amplicon-based biodiversity monitoring (metabarcoding) from complex environmental samples.</p><h1>Altitude and fungal diversity influence the structure of Antarctic cryptoendolithic Bacteria communities</h1>
<p><a href="https://doi.org/10.1111/1758-2229.12788">doi:10.1111/1758-2229.12788</a></p>

<p><a href=None>None</a></p>
<p>Endolithic growth within rocks is a critical adaptation of microbes living in harsh environments where exposure to extreme temperature, radiation, and desiccation limits the predominant life forms, such as in the ice-free regions of Continental Antarctica. The microbial diversity of the endolithic communities in these areas has been sparsely examined. In this work, diversity and composition of bacterial assemblages in the cryptoendolithic lichen-dominated communities of Victoria Land (Continental Antarctica) were explored using a high-throughput metabarcoding approach, targeting the V4 region of 16S rDNA. Rocks were collected in 12 different localities (from 14 different sites), along a gradient ranging from 1000 to 3300 m a.s.l. and at a sea distance ranging from 29 to 96 km. The results indicate Actinobacteria and Proteobacteria are the dominant taxa in all samples and defined a 'core' group of bacterial taxa across all sites. The structure of bacteria communities is correlated with the fungal counterpart and among the environmental parameters considered, altitude was found to influence bacterial biodiversity, while distance from sea had no evident influence.</p><h1>The effects of spatial and temporal replicate sampling on eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.7717/Peerj.7335">doi:10.7717/Peerj.7335</a></p>

<p><a href=None>None</a></p>
<p>The heterogeneous nature of environmental DNA (eDNA) and its effects on species detection and community composition estimates has been highlighted in several studies in the past decades. Mostly in the context of spatial distribution over large areas, in fewer occasions looking at spatial distribution within a single body of water. Temporal variation of eDNA, similarly, has mostly been studied as seasonality, observing changes over large periods of time, and often only for small groups of organisms such as fish and amphibians.</p><h1>Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance</h1>
<p><a href="https://doi.org/Giz092">doi:Giz092</a></p>

<p><a href=None>None</a></p>
<p>Trap-based surveillance strategies are widely used for monitoring of invasive insect species, aiming to detect newly arrived exotic taxa as well as track the population levels of established or endemic pests. Where these surveillance traps have low specificity and capture non-target endemic species in excess of the target pests, the need for extensive specimen sorting and identification creates a major diagnostic bottleneck. While the recent development of standardized molecular diagnostics has partly alleviated this requirement, the single specimen per reaction nature of these methods does not readily scale to the sheer number of insects trapped in surveillance programmes. Consequently, target lists are often restricted to a few high-priority pests, allowing unanticipated species to avoid detection and potentially establish populations. DNA metabarcoding has recently emerged as a method for conducting simultaneous, multi-species identification of complex mixed communities and may lend itself ideally to rapid diagnostics of bulk insect trap samples. Moreover, the high-throughput nature of recent sequencing platforms could enable the multiplexing of hundreds of diverse trap samples on a single flow cell, thereby providing the means to dramatically scale up insect surveillance in terms of both the quantity of traps that can be processed concurrently and number of pest species that can be targeted. In this review of the metabarcoding literature, we explore how DNA metabarcoding could be tailored to the detection of invasive insects in a surveillance context and highlight the unique technical and regulatory challenges that must be considered when implementing high-throughput sequencing technologies into sensitive diagnostic applications.</p><h1>Improving eDNA-based protist diversity assessments using networks of amplicon sequence variants</h1>
<p><a href="https://doi.org/10.1111/1462-2920.14764">doi:10.1111/1462-2920.14764</a></p>

<p><a href=None>None</a></p>
<p>Effective and precise grouping of highly similar sequences remains a major bottleneck in the evaluation of high-throughput sequencing datasets. Amplicon sequence variants (ASVs) offer a promising alternative that may supersede the widely used operational taxonomic units (OTUs) in environmental sequencing studies. We compared the performance of a recently developed pipeline based on the algorithm DADA2 for obtaining ASVs against a pipeline based on the algorithm SWARM for obtaining OTUs. Illumina-sequencing of 29 individual ciliate species resulted in up to 11 ASVs per species, while SWARM produced up to 19 OTUs per species. To improve the congruency between species diversity and molecular diversity, we applied sequence similarity networks (SSNs) for second-level sequence grouping into network sequence clusters (NSCs). At 100% sequence similarity in SWARM-SSNs, NSC numbers decreased from 7.9-fold overestimation without abundance filter, to 4.5-fold overestimation when an abundance filter was applied. For the DADA2-SSN approach, NSC numbers decreased from 3.5-fold to 3-fold overestimation. Rand index cluster analyses predicted best binning results between 97% and 94% sequence similarity for both DADA2-SSNs and SWARM-SSNs. Depending on the ecological questions addressed in an environmental sequencing study with protists we recommend ASVs as replacement for OTUs, best in combination with SSNs.</p><h1>Advancing the integration of multi-marker metabarcoding data in dietary analysis of trophic generalists</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13060">doi:10.1111/1755-0998.13060</a></p>

<p><a href=None>None</a></p>
<p>The application of DNA metabarcoding to dietary analysis of trophic generalists requires using multiple markers in order to overcome problems of primer specificity and bias. However, limited attention has been given to the integration of information from multiple markers, particularly when they partly overlap in the taxa amplified, and vary in taxonomic resolution and biases. Here, we test the use of a mix of universal and specific markers, provide criteria to integrate multi-marker metabarcoding data and a python script to implement such criteria and produce a single list of taxa ingested per sample. We then compare the results of dietary analysis based on morphological methods, single markers, and the proposed combination of multiple markers. The study was based on the analysis of 115 faeces from a small passerine, the Black Wheatears (Oenanthe leucura). Morphological analysis detected far fewer plant taxa (12) than either a universal 18S marker (57) or the plant trnL marker (124). This may partly reflect the detection of secondary ingestion by molecular methods. Morphological identification also detected far fewer taxa (23) than when using 18S (91) or the arthropod markers IN16STK (244) and ZBJ (231), though each method missed or underestimated some prey items. Integration of multi-marker data provided far more detailed dietary information than any single marker and estimated higher frequencies of occurrence of all taxa. Overall, our results show the value of integrating data from multiple, taxonomically overlapping markers in an example dietary data set.</p><h1>Land use is a determinant of plant pathogen alpha- but not beta-diversity</h1>
<p><a href="https://doi.org/10.1111/Mec.15177">doi:10.1111/Mec.15177</a></p>

<p><a href=None>None</a></p>
<p>Little is known about the diversity patterns of plant pathogens and how they change with land use at a broad scale. We employed DNA metabarcoding to describe the diversity and composition of putative plant pathogen communities in three substrates (soil, roots, and leaves) across five major land uses at a national scale. Almost all plant pathogen communities (fungi, oomycetes, and bacteria) showed strong responses to land use and substrate type. Land use category could explain up to 24% of the variance in composition between communities. Alpha-diversity (richness) of plant pathogens was consistently lower in natural forests than in agricultural systems. In planted forests, there was also generally low pathogen alpha-diversity in soil and roots, but alpha-diversity in leaves was high compared with most other land uses. In contrast to alpha-diversity, differences in within-land use beta-diversity of plant pathogens (the predictability of plant pathogen communities within land use) were subtle. Our results show that large-scale patterns and distributions of putative plant pathogens can be determined using metabarcoding, allowing some of the first landscape level insights into these critically important communities.</p><h1>An optimised eDNA protocol for detecting fish in lentic and lotic freshwaters using a small water volume</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0219218">doi:10.1371/Journal.Pone.0219218</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA is increasingly being used for assessing the presence and relative abundance of fish in freshwater, but existing protocols typically rely on filtering large volumes of water which is not always practical. We compared the effects of water volume, filtration type and eDNA extraction procedures in the detection of fish in three freshwater bodies (pond, lake and river) using a short fragment of the 12s rRNA mtDNA gene. Quantification of eDNA capture efficiency after DNA extraction, as well as amplification efficiency, were evaluated by conventional PCR and quantitative PCR. No significant differences on eDNA capture yield were found among freshwater bodies, but increasing water volume had a positive effect on eDNA capture and amplification efficiency. Although highest eDNA capture rates were obtained using 2 L of filtered water, 100 mL syringe filtration in combination with ethanol- sodium acetate precipitation proved to be more practical and increased quantitative PCR amplification efficiency by 6.4%. Our results indicate that such method may be optimal to detect fish species effectively across both lotic and lentic freshwater environments.</p><h1>Global distribution of Trebouxiophyceae diversity explored by high-throughput sequencing and phylogenetic approaches</h1>
<p><a href="https://doi.org/10.1111/1462-2920.14738">doi:10.1111/1462-2920.14738</a></p>

<p><a href=None>None</a></p>
<p>Trebouxiophyceae are a ubiquitous class of Chlorophyta encountered in aquatic and terrestrial environments. Most taxa are photosynthetic, and many acts as photobionts in symbiotic relationships, while others are free-living. Trebouxiophyceae have also been widely investigated for their use for biotechnological applications. In this work, we aimed at obtaining a comprehensive image of their diversity by compiling the information of 435 freshwater, soil and marine environmental DNA samples surveyed with Illumina sequencing technology in order to search for the most relevant environments for bioprospecting. Freshwater and soil were most diverse and shared more than half of all operational taxonomic units (OTUs), however, their communities were significantly distinct. Oceans hosted the highest genetic novelty, and did not share any OTUs with the other environments; also, marine samples host more diversity in warm waters. Symbiotic genera usually found in lichens such as Trebouxia, Myrmecia and Symbiochloris were also abundantly detected in the ocean, suggesting either free-living lifestyles or unknown symbiotic relationships with marine planktonic organisms. Altogether, our study opens the way to new prospection for trebouxiophycean strains, especially in understudied environments like the ocean.</p><h1>The menu varies with metabarcoding practices: A case study with the bat Plecotus auritus</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0219135">doi:10.1371/Journal.Pone.0219135</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of feces has revolutionized the knowledge of animal diets by providing unprecedented resolution of consumed resources. However, it is still unclear how different methodological approaches influence the ecological conclusions that can be drawn from such data. Here, we propose a critical evaluation of several data treatments on the inferred diet of the bat Plecotus auritus using guano regularly collected from various colonies throughout the entire active season. First and unlike previous claims, our data indicates that DNA extracted from large amounts of fecal material issued from guano accumulates yield broader taxonomic diversity of prey than smaller numbers of pellets would do, provided that extraction buffer volumes are adapted to such increased amounts of material. Second, trophic niche analyses based on prey occurrence data uncover strong seasonality in the bat's diet and major differences among neighboring maternity colonies. Third, while the removal of rare prey items is not always warranted as it introduces biases affecting particularly samples with greater prey species richness. Fourth, examination of distinct taxonomic depths in diet analyses highlights different aspects of food consumption providing a better understanding of the consumer's diet. Finally, the biologically meaningful patterns recovered with presence-absence approaches are virtually lost when attempting to quantify prey consumed using relative read abundances. Even in an ideal situation where reference barcodes are available for most potential prey species, inferring realistic patterns of prey consumption remains relatively challenging. Although best practice in metabarcoding analyses will depend on the aims of the study, several previous methodological recommendations seem unwarranted for studying such diverse diets as that of brown long-eared bats.</p><h1>High-throughput environmental DNA analysis informs a biological assessment of an urban stream</h1>
<p><a href="https://doi.org/10.1016/J.Ecolind.2019.04.088">doi:10.1016/J.Ecolind.2019.04.088</a></p>

<p><a href=None>None</a></p>
<p>There is growing interest in the use of DNA barcoding and metabarcoding approaches to aid biological assessments and monitoring of waterbodies. While biodiversity measured by morphology and by DNA often has been found correlated, few studies have compared DNA data to established measures of impairment such as multimetric pollution tolerance indices used by many bioassessment programs. We incorporated environmental DNA (eDNA) metabarcoding of seston into a rigorous watershed-scale biological assessment of an urban stream to examine the extent to which eDNA richness and diversity patterns were correlated with multimetric indices and ecological impairment status designations. We also evaluated different filtering approaches and taxonomic classifications to identify best practices for environmental assessments. Seston eDNA revealed a wide diversity of eukaryotic taxa but was dominated by diatoms (36%). Differentiation among sites in alpha and beta diversity was greater when operational taxonomic units (OTUs) were classified taxonomically, but coarse resolution taxonomy (kingdom) was more informative than finer resolution taxonomy (family, genus). Correlations of DNA richness and diversity with multimetric indices for fish and macroinvertebrates were generally weak, possibly because Metazoa were not highly represented in our DNA dataset. Nonetheless, sites could be differentiated based on ecological impairment status, with more impaired sites having lower eDNA diversity as measured by the Shannon index, but higher taxonomic richness. Significant environmental drivers of community structure, as inferred from constrained ordination analyses, differed among kingdoms within the eDNA dataset, as well as from fish and macrobenthos, suggesting that eDNA provides novel environmental information. These results suggest that even a simple seston eDNA filtering protocol can provide biodiversity information of value to stream bioassessment programs. The approach bears further investigation as a potentially useful rapid assessment protocol to supplement more intensive field sampling efforts.</p><h1>Single Fragment or Bulk Soil DNA Metabarcoding: Which is Better for Characterizing Biological Taxa Found in Surface Soils for Sample Separation?</h1>
<p><a href="https://doi.org/E431">doi:E431</a></p>

<p><a href=None>None</a></p>
<p>In forensic geology casework, sample size typically limits routine characterization of material using bulk approaches. To address this, DNA-based characterization of biological taxa has received attention, as the taxa present can be useful for sample-to-sample comparisons and source attribution. In our initial work, low biodiversity was captured when DNA barcodes were Sanger-sequenced from plant and insect fragments isolated from 10 forensic-type surface soils. Considering some forensic laboratories now have access to massively parallel sequencing platforms, we assessed whether biological taxa present in the same surface soils could be better characterized using DNA metabarcoding. To achieve this, plant and animal barcodes were amplified and sequenced on an Illumina MiniSeq for three different DNA sample types (<i>n</i> = 50): individual fragments used in our initial study, and 250 and 100 mg of bulk soil (from the 10 sites used in the initial study). A total of 572 unique target barcode sequences passed quality filtering and were used in downstream statistical analyses: 54, 321, and 285 for individual fragments, 100 mg, and 250 mg bulk soil samples, respectively. Plant barcodes permitted some spatial separation of sample sites in non-metric multidimensional scaling plots; better separation was obtained for samples prepared from bulk soil. This study confirmed that bulk soil DNA metabarcoding is a better approach for characterizing biological taxa present in surface soils, which could supplement traditional geologic examinations.</p><h1>Towards an in-depth characterization of Symbiodiniaceae in tropical giant clams via metabarcoding of pooled multi-gene amplicons</h1>
<p><a href="https://doi.org/10.7717/Peerj.6898">doi:10.7717/Peerj.6898</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing is revolutionizing our ability to comprehensively characterize free-living and symbiotic Symbiodiniaceae, a diverse dinoflagellate group that plays a critical role in coral reef ecosystems. Most studies however, focus on a single marker for metabarcoding Symbiodiniaceae, potentially missing important ecological traits that a combination of markers may capture. In this proof-of-concept study, we used a small set of symbiotic giant clam (<i>Tridacna maxima</i>) samples obtained from nine French Polynesian locations and tested a dual-index sequence library preparation method that pools and simultaneously sequences multiple Symbiodiniaceae gene amplicons per sample for in-depth biodiversity assessments. The rationale for this approach was to allow the metabarcoding of multiple genes without extra costs associated with additional single amplicon dual indexing and library preparations. Our results showed that the technique effectively recovered very similar proportions of sequence reads and dominant Symbiodiniaceae clades among the three pooled gene amplicons investigated per sample, and captured varying levels of phylogenetic resolution enabling a more comprehensive assessment of the diversity present. The pooled Symbiodiniaceae multi-gene metabarcoding approach described here is readily scalable, offering considerable analytical cost savings while providing sufficient phylogenetic information and sequence coverage.</p><h1>Taxonomic and Functional Characterization of the Microbial Community During Spontaneous <i>in vitro</i> Fermentation of Riesling Must</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2019.00697">doi:10.3389/Fmicb.2019.00697</a></p>

<p><a href=None>None</a></p>
<p>Although there is an extensive tradition of research into the microbes that underlie the winemaking process, much remains to be learnt. We combined the high-throughput sequencing (HTS) tools of metabarcoding and metagenomics, to characterize how microbial communities of Riesling musts sampled at four different vineyards, and their subsequent spontaneously fermented derivatives, vary. We specifically explored community variation relating to three points: (i) how microbial communities vary by vineyard; (ii) how community biodiversity changes during alcoholic fermentation; and (iii) how microbial community varies between musts that successfully complete alcoholic fermentation and those that become 'stuck' in the process. Our metabarcoding data showed a general influence of microbial composition at the vineyard level. Two of the vineyards (4 and 5) had strikingly a change in the differential abundance of <i>Metschnikowia</i>. We therefore additionally performed shotgun metagenomic sequencing on a subset of the samples to provide preliminary insights into the potential relevance of this observation, and used the data to both investigate functional potential and reconstruct draft genomes (bins). At these two vineyards, we also observed an increase in non-Saccharomycetaceae fungal functions, and a decrease in bacterial functions during the early fermentation stage. The binning results yielded 11 coherent bins, with both vineyards sharing the yeast bins <i>Hanseniaspora</i> and <i>Saccharomyces</i>. Read recruitment and functional analysis of this data revealed that during fermentation, a high abundance of <i>Metschnikowia</i> might serve as a biocontrol agent against bacteria, via a putative iron depletion pathway, and this in turn could help <i>Saccharomyces</i> dominate the fermentation. During alcoholic fermentation, we observed a general decrease in biodiversity in both the metabarcoding and metagenomic data. Unexpected <i>Micrococcus</i> behavior was observed in vineyard 4 according to metagenomic analyses based on reference-based read mapping. Analysis of open reading frames using these data showed an increase of functions assigned to class Actinobacteria in the end of fermentation. Therefore, we hypothesize that bacteria might sit-and-wait until <i>Saccharomyces</i> activity slows down. Complementary approaches to annotation instead of relying a single database provide more coherent information true species. Lastly, our metabarcoding data enabled us to identify a relationship between stuck fermentations and <i>Starmerella</i> abundance. Given that robust chemical analysis indicated that although the stuck samples contained residual glucose, all fructose had been consumed, we hypothesize that this was because fructophilic <i>Starmerella</i>, rather than <i>Saccharomyces</i>, dominated these fermentations. Overall, our results showcase the different ways in which metagenomic analyses can improve our understanding of the wine alcoholic fermentation process.</p><h1>Benthic Diatom Communities in an Alpine River Impacted by Waste Water Treatment Effluents as Revealed Using DNA Metabarcoding</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2019.00653">doi:10.3389/Fmicb.2019.00653</a></p>

<p><a href=None>None</a></p>
<p>Freshwater ecosystems are continuously affected by anthropogenic pressure. One of the main sources of contamination comes from wastewater treatment plant (WWTP) effluents that contain wide range of micro- and macropollutants. Chemical composition, toxicity levels and impact of treated effluents (TEs) on the recipient aquatic ecosystems may strongly differ depending on the wastewater origin. Compared to urban TEs, hospital ones may contain more active pharmaceutical substances. Benthic diatoms are relevant ecological indicators because of their high species and ecological diversity and rapid response to human pressure. They are routinely used for water quality monitoring. However, there is a knowledge gap on diatom communities' development and behavior in treated wastewater in relation to prevailing micro- and macropollutants. In this study, we aim to (1) investigate the response of diatom communities to urban and hospital TEs, and (2) evaluate TEs effect on communities in the recipient river. Environmental biofilms were colonized in TEs and the recipient river up- and downstream from the WWTP output to study benthic diatoms using DNA metabarcoding combined with high-throughput sequencing (HTS). In parallel, concentrations of nutrients, pharmaceuticals and seasonal conditions were recorded. Diatom metabarcoding showed that benthic communities differed strongly in their diversity and structure depending on the habitat. TE sites were generally dominated by few genera with polysaprobic preferences belonging to the motile guild, while river sites favored diverse communities from oligotrophic and oligosaprobic groups. Seasonal changes were visible to lower extent. To categorize parameters important for diatom changes we performed redundancy analysis which suggested that communities within TE sites were associated to higher concentrations of beta-blockers and non-steroidal anti-inflammatory drugs in urban effluents vs. antibiotics and orthophosphate in hospital effluents. Furthermore, indicator species analysis showed that 27% of OTUs detected in river downstream communities were indicator for urban or hospital TE sites and were absent in the river upstream. Finally, biological diatom index (BDI) calculated to evaluate the ecological status of the recipient river suggested water quality decrease linked to the release of TEs. Thus, in-depth assessment of diatom community composition using DNA metabarcoding is proposed as a promising technique to highlight the disturbing effect of pollutants in Alpine rivers.</p><h1>Using eDNA to biomonitor the fish community in a tropical oligotrophic lake</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0215505">doi:10.1371/Journal.Pone.0215505</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is an effective approach for detecting vertebrates and plants, especially in aquatic ecosystems, but prior studies have largely examined eDNA in cool temperate settings. By contrast, this study employs eDNA to survey the fish fauna in tropical Lake Bacalar (Mexico) with the additional goal of assessing the possible presence of invasive fishes, such as Amazon sailfin catfish and tilapia. Sediment and water samples were collected from eight stations in Lake Bacalar on three occasions over a 4-month interval. Each sample was stored in the presence or absence of lysis buffer to compare eDNA recovery. Short fragments (184-187 bp) of the cytochrome c oxidase I (COI) gene were amplified using fusion primers and then sequenced on Ion Torrent PGM or S5 before their source species were determined using a custom reference sequence database constructed on BOLD. In total, eDNA sequences were recovered from 75 species of vertebrates including 47 fishes, 15 birds, 7 mammals, 5 reptiles, and 1 amphibian. Although all species are known from this region, six fish species represent new records for the study area, while two require verification. Sequences for five species (2 birds, 2 mammals, 1 reptile) were only detected from sediments, while sequences from 52 species were only recovered from water. Because DNA from the Amazon sailfin catfish was not detected, we used a mock eDNA experiment to confirm our methods would enable its detection. In summary, we developed protocols that recovered eDNA from tropical oligotrophic aquatic ecosystems and confirmed their effectiveness in detecting fishes and diverse species of vertebrates.</p><h1>Distribution patterns of dinoflagellate communities along the Songhua River</h1>
<p><a href="https://doi.org/10.7717/Peerj.6733">doi:10.7717/Peerj.6733</a></p>

<p><a href=None>None</a></p>
<p>Dinoflagellates have the potential to pose severe ecological and economic damages to aquatic ecosystems. It is therefore largely needed to understand the causes and consequences of distribution patterns of dinoflagellate communities in order to manage potential environmental problems. However, a majority of studies have focused on marine ecosystems, while the geographical distribution patterns of dinoflagellate communities and associated determinants in freshwater ecosystems remain unexplored, particularly in running water ecosystems such as rivers and streams.</p><h1>Assessing the influence of sample tagging and library preparation on DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13018">doi:10.1111/1755-0998.13018</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding is increasingly used to assess species diversity by high-throughput sequencing where millions of sequences can be generated in parallel and multiple samples can be analysed in one sequencing run. Generating amplified fragments with a unique sequence identifier ('tag') for each sample is crucial, as it allows assigning sequences to the original samples. The tagging through so-called fusion primers is a fast and cheap alternative to commercially produced ligation-based kits. However, little is known about potential bias and inconsistencies introduced by the long nucleotide tail attached to those primers, which could lead to deficient reports of community composition in metabarcoding studies. We therefore tested the consistency and taxa detection efficiency of fusion primers in (1) a one-step and (2) two-step PCR protocol as well as (3) a commercially manufactured Illumina kit using mock communities of known composition. The Illumina kit delivered the most consistent results and detected the highest number of taxa. However, success of the two-step PCR approach was only marginally lower compared to the kit with the additional advantage of a much more competitive price per library. While most taxa were also detected with the one-step PCR approach, the consistency between replicates including read abundance was substantially lower. Our results highlight that method choice depends on the precision needed for analysis as well as on economic considerations and recommend the Illumina kit to obtain most accurate results and the two-step PCR approach as a much cheaper yet very robust alternative.</p><h1>Evaluating next-generation sequencing (NGS) methods for routine monitoring of wild bees: Metabarcoding, mitogenomics or NGS barcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13013">doi:10.1111/1755-0998.13013</a></p>

<p><a href=None>None</a></p>
<p>Implementing cost-effective monitoring programs for wild bees remains challenging due to the high costs of sampling and specimen identification. To reduce costs, next-generation sequencing (NGS)-based methods have lately been suggested as alternatives to morphology-based identifications. To provide a comprehensive presentation of the advantages and weaknesses of different NGS-based identification methods, we assessed three of the most promising ones, namely metabarcoding, mitogenomics and NGS barcoding. Using a regular monitoring data set (723 specimens identified using morphology), we found that NGS barcoding performed best for both species presence/absence and abundance data, producing only few false positives (3.4%) and no false negatives. In contrast, the proportion of false positives and false negatives was higher using metabarcoding and mitogenomics. Although strong correlations were found between biomass and read numbers, abundance estimates significantly skewed the communities' composition in these two techniques. NGS barcoding recovered the same ecological patterns as morphology. Ecological conclusions based on metabarcoding and mitogenomics were similar to those based on morphology when using presence/absence data, but different when using abundance data. In terms of workload and cost, we show that metabarcoding and NGS barcoding can compete with morphology, but not mitogenomics which was consistently more expensive. Based on these results, we advocate that NGS barcoding is currently the seemliest NGS method for monitoring of wild bees. Furthermore, this method has the advantage of potentially linking DNA sequences with preserved voucher specimens, which enable morphological re-examination and will thus produce verifiable records which can be fed into faunistic databases.</p><h1>Diversity of photosynthetic picoeukaryotes in eutrophic shallow lakes as assessed by combining flow cytometry cell-sorting and high throughput sequencing</h1>
<p><a href="https://doi.org/Fiz038">doi:Fiz038</a></p>

<p><a href=None>None</a></p>
<p>Photosynthetic picoeukaryotes (PPE) are key components of primary production in marine and freshwater ecosystems. In contrast with those of marine environments, freshwater PPE groups have received little attention. In this work, we used flow cytometry cell sorting, microscopy and metabarcoding to investigate the composition of small photosynthetic eukaryote communities from six eutrophic shallow lakes in South America, Argentina. We compared the total molecular diversity obtained from PPE sorted populations as well as from filtered total plankton samples (FTP). Most reads obtained from sorted populations belonged to the classes: Trebouxiophyceae, Chlorophyceae and Bacillariophyceae. We retrieved sequences from non-photosynthetic groups, such as Chytridiomycetes and Ichthyosporea which contain a number of described parasites, indicating that these organisms were probably in association with the autotrophic cells sorted. Dominant groups among sorted PPEs were poorly represented in FTP and their richness was on average lower than in the sorted samples. A significant number of operational taxonomic units (OTUs) were exclusively found in sorting samples, emphasizing that sequences from FTP underestimate the diversity of PPE. Moreover, 22% of the OTUs found among the dominant groups had a low similarity (<95%) with reported sequences in public databases, demonstrating a high potential for novel diversity in these lakes.</p><h1>Targeted Amplification and Sequencing of Ancient Environmental and Sedimentary DNA</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-9176-1_16">doi:10.1007/978-1-4939-9176-1_16</a></p>

<p><a href=None>None</a></p>
<p>All organisms release their DNA into the environment through processes such as excretion and the senescence of tissues and limbs. This DNA, often referred to as environmental DNA (eDNA) or sedimentary ancient DNA (sedaDNA), can be recovered from both present-day and ancient soils, fecal samples, bodies of water and lake cores, and even air. While eDNA is a potentially useful record of past and present biodiversity, several challenges complicate data generation and interpretation of results. Most importantly, eDNA samples tend to be highly taxonomically mixed, and the target organism or group of organisms may be present at very low abundance within this mixture. To overcome this challenge, enrichment approaches are often used to target specific taxa of interest. Here, we describe a protocol to amplify metabarcodes or short, variable loci that identify lineages within broad taxonomic groups (e.g., plants, mammals), using the polymerase chain reaction (PCR) with established generic "barcode" primers. We also provide a catalog of animal and plant barcode primers that, because they target relatively short fragments of DNA, are potentially suitable for use with degraded DNA.</p><h1>[Application of DNA metabarcoding in species identification of Chinese herbal medicines]</h1>
<p><a href="https://doi.org/10.19540/J.Cnki.Cjcmm.2019.0001">doi:10.19540/J.Cnki.Cjcmm.2019.0001</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding,one rapid and robust method using specific standard DNA fragments,has been widely used for rapid species identification of a bulk sample through high-throughput sequencing technologies.While it has been widely used in the studies of metagenomics,animal and plant biodiversity,it has gradually come to be used as a profitable method in species identification of mixed Chinese herbal medicines.In this paper,we mainly summarize the current studies of the application of DNA metabarcoding in species identification of mixed Chinese herbal medicines.Moreover,high-throughput sequencing technologies adopted in those studies,such as Sanger,the next-generation,and third-generation sequencing technologies,are discussed.It is conducted to provide a theoretical guidance for the application of DNA metabarcoding in species identification of mixed Chinese herbal medicines and in more other biodiversity studies.</p><h1>Evaluation of detection probabilities at the water-filtering and initial PCR steps in environmental DNA metabarcoding using a multispecies site occupancy model</h1>
<p><a href="https://doi.org/10.1038/S41598-019-40233-1">doi:10.1038/S41598-019-40233-1</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is a recently developed method to assess biodiversity based on a high-throughput parallel DNA sequencing applied to DNA present in the ecosystem. Although eDNA metabarcoding enables a rapid assessment of biodiversity, it is prone to species detection errors that may occur at sequential steps in field sampling, laboratory experiments, and bioinformatics. In this study, we illustrate how the error rates in the eDNA metabarcoding-based species detection can be accounted for by applying the multispecies occupancy modelling framework. We report a case study with the eDNA sample from an aquarium tank in which the detection probabilities of species in the two major steps of eDNA metabarcoding, filtration and PCR, across a range of PCR annealing temperatures, were examined. We also show that the results can be used to examine the efficiency of species detection under a given experimental design and setting, in terms of the efficiency of species detection, highlighting the usefulness of the multispecies site occupancy modelling framework to study the optimum conditions for molecular experiments.</p><h1>Nanopore sequencing of long ribosomal DNA amplicons enables portable and simple biodiversity assessments with high phylogenetic resolution across broad taxonomic scale</h1>
<p><a href="https://doi.org/Giz006">doi:Giz006</a></p>

<p><a href=None>None</a></p>
<p>In light of the current biodiversity crisis, DNA barcoding is developing into an essential tool to quantify state shifts in global ecosystems. Current barcoding protocols often rely on short amplicon sequences, which yield accurate identification of biological entities in a community but provide limited phylogenetic resolution across broad taxonomic scales. However, the phylogenetic structure of communities is an essential component of biodiversity. Consequently, a barcoding approach is required that unites robust taxonomic assignment power and high phylogenetic utility. A possible solution is offered by sequencing long ribosomal DNA (rDNA) amplicons on the MinION platform (Oxford Nanopore Technologies).</p><h1>Changes in Trophic Groups of Protists With Conversion of Rainforest Into Rubber and Oil Palm Plantations</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2019.00240">doi:10.3389/Fmicb.2019.00240</a></p>

<p><a href=None>None</a></p>
<p>Protists, abundant but enigmatic single-celled eukaryotes, are important soil microbiota providing numerous ecosystem functions. We employed high-throughput sequencing of environmental DNA, targeting the V4 region of the 18S rRNA gene, to characterize changes in their abundance, species richness, and community structure with conversion of lowland rainforest into rubber agroforest (jungle rubber), and rubber and oil palm plantations; typical agricultural systems in Sumatra, Indonesia. We identified 5,204 operational taxonomic units (OTUs) at 97% identity threshold of protists from 32 sites. Protists species richness was similar in rainforest, jungle rubber and oil palm plantations but significantly lower in rubber plantations. After standardization, 4,219 OTUs were assigned to five trophic groups, and inspected for effects of land-use change, and potential biotic and abiotic driving factors. The most abundant trophic group was phagotrophs (52%), followed by animal parasites (29%), photoautotrophs (12%), plant parasites (1%), and symbionts (<1%). However, the relative abundance and OTU richness of phagotrophs and photoautotrophs increased significantly with increasing land-use intensity. This was similar, but less pronounced, for the relative abundance of symbionts. Animal and plant parasites decreased significantly in abundance and species richness with increasing land-use intensity. Community compositions and factors affecting the structure of individual trophic groups differed between land-use systems. Parasites were presumably mainly driven by the abundance and species richness of their hosts, while phagotrophs by changes in soil pH and increase in Gram-positive bacteria, and photoautotrophs by light availability. Overall, the results show that relative species richness, relative abundance, and community composition of individual trophic groups of protists in tropical lowland rainforest significantly differ from that in converted ecosystems. This is likely associated with changes in ecosystem functioning. The study provides novel insight into protist communities and their changes with land-use intensity in tropical lowland ecosystems. We show, that trophic groups of protists are powerful indicators reflecting changes in the functioning of ecosystems with conversion of rainforest into monoculture plantations.</p><h1>Metabarcoding free-living marine nematodes using curated 18S and CO1 reference sequence databases for species-level taxonomic assignments</h1>
<p><a href="https://doi.org/10.1002/Ece3.4814">doi:10.1002/Ece3.4814</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing has the potential to describe biological communities with high efficiency yet comprehensive assessment of diversity with species-level resolution remains one of the most challenging aspects of metabarcoding studies. We investigated the utility of curated ribosomal and mitochondrial nematode reference sequence databases for determining phylum-specific species-level clustering thresholds. We compiled 438 ribosomal and 290 mitochondrial sequences which identified 99% and 94% as the species delineation clustering threshold, respectively. These thresholds were evaluated in HTS data from mock communities containing 39 nematode species as well as environmental samples from Vietnam. We compared the taxonomic description of the mocks generated by two read-merging and two clustering algorithms and the cluster-free Dada2 pipeline. Taxonomic assignment with the RDP classifier was assessed under different training sets. Our results showed that 36/39 mock nematode species were identified across the molecular markers (18S: 32, JB2: 19, JB3: 21) in UClust_ref OTUs at their respective clustering thresholds, outperforming UParse_denovo and the commonly used 97% similarity. Dada2 generated the most realistic number of ASVs (18S: 83, JB2: 75, JB3: 82), collectively identifying 30/39 mock species. The ribosomal marker outperformed the mitochondrial markers in terms of species and genus-level detections for both OTUs and ASVs. The number of taxonomic assignments of OTUs/ASVs was highest when the smallest reference database containing only nematode sequences was used and when sequences were truncated to the respective amplicon length. Overall, OTUs generated more species-level detections, which were, however, associated with higher error rates compared to ASVs. Genus-level assignments using ASVs exhibited higher accuracy and lower error rates compared to species-level assignments, suggesting that this is the most reliable pipeline for rapid assessment of alpha diversity from environmental samples.</p><h1>SLIM: a flexible web application for the reproducible processing of environmental DNA metabarcoding data</h1>
<p><a href="https://doi.org/10.1186/S12859-019-2663-2">doi:10.1186/S12859-019-2663-2</a></p>

<p><a href=None>None</a></p>
<p>High-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) has become a routine tool for biodiversity survey and ecological studies. By including sample-specific tags in the primers prior PCR amplification, it is possible to multiplex hundreds of samples in a single sequencing run. The analysis of millions of sequences spread into hundreds to thousands of samples prompts for efficient, automated yet flexible analysis pipelines. Various algorithms and software have been developed to perform one or multiple processing steps, such as paired-end reads assembly, chimera filtering, Operational Taxonomic Unit (OTU) clustering and taxonomic assignment. Some of these software are now well established and widely used by scientists as part of their workflow. Wrappers that are capable to process metabarcoding data from raw sequencing data to annotated OTU-to-sample matrix were also developed to facilitate the analysis for non-specialist users. Yet, most of them require basic bioinformatic or command-line knowledge, which can limit the accessibility to such integrative toolkits. Furthermore, for flexibility reasons, these tools have adopted a step-by-step approach, which can prevent an easy automation of the workflow, and hence hamper the analysis reproducibility.</p><h1>Unmasking the identity of toxigenic cyanobacteria driving a multi-toxin bloom by high-throughput sequencing of cyanotoxins genes and 16S rRNA metabarcoding</h1>
<p><a href="https://doi.org/S0048-9697(19)30564-9">doi:S0048-9697(19)30564-9</a></p>

<p><a href=None>None</a></p>
<p>Cyanobacterial harmful algal blooms (CyanoHABs) are complex communities that include coexisting toxic and non-toxic strains only distinguishable by genetic methods. This study shows a water-management oriented use of next generation sequencing (NGS) to specifically pinpoint toxigenic cyanobacteria within a bloom simultaneously containing three of the most widespread cyanotoxins (the hepatotoxins microcystins, MCs; and the neurotoxins anatoxin-a, ATX, and saxitoxins, STXs). The 2013 summer bloom in Rosarito reservoir (Spain) comprised 33 cyanobacterial OTUs based on 16S rRNA metabarcoding, 7 of which accounted for as much as 96.6% of the community. Cyanotoxins and their respective biosynthesis genes were concurrently present throughout the entire bloom event including: MCs and mcyE gene; ATX and anaF gene; and STXs and sxtI gene. NGS applied to amplicons of cyanotoxin-biosynthesis genes unveiled 6 toxigenic OTUs, comprising 3 involved in MCs production (Planktothrix agardhii and 2 Microcystis spp.), 2 in ATX production (Cuspidothrix issatschenkoi and Phormidium/Tychonema spp.) and 1 in STXs production (Aphanizomenon gracile). These toxigenic taxa were also present in 16S rRNA OTUs list and their relative abundance was positively correlated with the respective toxin concentrations. Our results point at MC-producing P. agardhii and ATX-producing C. issatschenkoi as the main contributors to the moderate toxin concentrations observed, and suggest that their distribution in Southern Europe is broader than previously thought. Our findings also stress the need for monitoring low-abundance cyanobacteria (<1% relative abundance) in cyanotoxicity studies, and provide novel data on the presence of picocyanobacteria and potentially ATX-producing benthic taxa (e.g., Phormidium) in deep thermally-stratified water bodies. This study showcases a straightforward use of amplicon metagenomics of cyanotoxin biosynthesis genes in a multi-toxin bloom thus illustrating the broad applicability of NGS for water management in risk-oriented monitoring of CyanoHABs.</p><h1>High-Throughput Sequencing to Investigate Phytopathogenic Fungal Propagules Caught in Baited Insect Traps</h1>
<p><a href="https://doi.org/E15">doi:E15</a></p>

<p><a href=None>None</a></p>
<p>Studying the means of dispersal of plant pathogens is crucial to better understand the dynamic interactions involved in plant infections. On one hand, entomologists rely mostly on both traditional molecular methods and morphological characteristics, to identify pests. On the other hand, high-throughput sequencing (HTS) is becoming the go-to avenue for scientists studying phytopathogens. These organisms sometimes infect plants, together with insects. Considering the growing number of exotic insect introductions in Canada, forest pest-management efforts would benefit from the development of a high-throughput strategy to investigate the phytopathogenic fungal and oomycete species interacting with wood-boring insects. We recycled formerly discarded preservative fluids from the Canadian Food Inspection Agency annual survey using insect traps and analysed more than one hundred samples originating from across Canada. Using the Ion Torrent Personal Genome Machine (PGM) HTS technology and fusion primers, we performed metabarcoding to screen unwanted fungi and oomycetes species, including <i>Phytophthora</i> spp. Community profiling was conducted on the four different wood-boring, insect-attracting semiochemicals; although the preservative (contained ethanol) also attracted other insects. Phytopathogenic fungi (e.g., <i>Leptographium</i> spp<i>.</i> and <i>Meria</i> <i>laricis</i> in the pine sawyer semiochemical) and oomycetes (mainly <i>Peronospora</i> spp. and <i>Pythium</i> aff. <i>hypogynum</i> in the General Longhorn semiochemical), solely associated with one of the four types of semiochemicals, were detected. This project demonstrated that the insect traps' semiochemical microbiome represents a new and powerful matrix for screening phytopathogens. Compared to traditional diagnostic techniques, the fluids allowed for a faster and higher throughput assessment of the biodiversity contained within. Additionally, minimal modifications to this approach would allow it to be used in other phytopathology fields.</p><h1>Assessing high-throughput environmental DNA extraction methods for meta-barcode characterization of aquatic microbial communities</h1>
<p><a href="https://doi.org/10.2166/Wh.2018.108">doi:10.2166/Wh.2018.108</a></p>

<p><a href=None>None</a></p>
<p>The characterization of microbial community dynamics using genomic methods is rapidly expanding, impacting many fields including medical, ecological, and environmental research and applications. One of the biggest challenges for such studies is the isolation of environmental DNA (eDNA) from a variety of samples, diverse microbes, and widely variable community compositions. The current study developed environmentally friendly, user safe, economical, and high throughput eDNA extraction methods for mixed aquatic microbial communities and tested them using 16 s rRNA gene meta-barcoding. Five different lysis buffers including (1) cetyltrimethylammonium bromide (CTAB), (2) digestion buffer (DB), (3) guanidinium isothiocyanate (GITC), (4) sucrose lysis (SL), and (5) SL-CTAB, coupled with four different purification methods: (1) phenol-chloroform-isoamyl alcohol (PCI), (2) magnetic Bead-Robotic, (3) magnetic Bead-Manual, and (4) membrane-filtration were tested for their efficacy in extracting eDNA from recreational freshwater samples. Results indicated that the CTAB-PCI and SL-Bead-Robotic methods yielded the highest genomic eDNA concentrations and succeeded in detecting the core microbial community including the rare microbes. However, our study recommends the SL-Bead-Robotic eDNA extraction protocol because this method is safe, environmentally friendly, rapid, high-throughput and inexpensive.</p><h1>Broadening the taxonomic scope of coral reef palaeoecological studies using ancient DNA</h1>
<p><a href="https://doi.org/10.1111/Mec.15038">doi:10.1111/Mec.15038</a></p>

<p><a href=None>None</a></p>
<p>Marine environments face acute pressures from human impacts, often resulting in substantial changes in community structure. On the inshore Great Barrier Reef (GBR), palaeoecological studies show the collapse of the previously dominant coral Acropora from the impacts of degraded water quality associated with European colonization. Even more dramatic impacts can result in the replacement of corals by fleshy macroalgae on modern reefs, but their past distribution is unknown because they leave no fossil record. Here, we apply DNA metabarcoding and high-throughput sequencing of the 18S rDNA gene on palaeoenvironmental DNA (aeDNA) derived from sediment cores at two sites on Pandora Reef (GBR), to enhance palaeoecological studies by incorporating key soft-bodied taxa, including macroalgae. We compared temporal trends in this aeDNA record with those of coral genera derived from macrofossils. Multivariate analysis of 12 eukaryotic groups from the aeDNA community showed wide variability over the past 750 years. The occurrence of brown macroalgae was negatively correlated only with the dominant coral at both sites. The occurrence of coralline and green macroalgae was positively correlated with only the dominant coral at one of the sites, where we also observed a significant association between the whole coral community and the occurrence of each of the three macroalgae groups. Our results demonstrate that reef sediments can provide a valuable archive for understanding the past distribution and occurrence of important soft-bodied reef dwellers. Combining information from fossils and aeDNA provides an enhanced understanding of temporal changes of reefs ecosystems at decadal to millennial timescales.</p><h1>Environmental DNA metabarcoding for fish community analysis in backwater lakes: A comparison of capture methods</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0210357">doi:10.1371/Journal.Pone.0210357</a></p>

<p><a href=None>None</a></p>
<p>The use of environmental DNA (eDNA) methods for community analysis has recently been developed. High-throughput parallel DNA sequencing (HTS), called eDNA metabarcoding, has been increasingly used in eDNA studies to examine multiple species. However, eDNA metabarcoding methodology requires validation based on traditional methods in all natural ecosystems before a reliable method can be established. To date, relatively few studies have performed eDNA metabarcoding of fishes in aquatic environments where fish communities were intensively surveyed using multiple traditional methods. Here, we have compared fish communities' data from eDNA metabarcoding with seven conventional multiple capture methods in 31 backwater lakes in Hokkaido, Japan. We found that capture and field surveys of fishes were often interrupted by macrophytes and muddy sediments in the 31 lakes. We sampled 1 L of the surface water and analyzed eDNA using HTS. We also surveyed the fish communities using seven different capture methods, including various types of nets and electrofishing. At some sites, we could not detect any eDNA, presumably because of the polymerase chain reaction (PCR) inhibition. We also detected the marine fish species as sewage-derived eDNA. Comparisons of eDNA metabarcoding and capture methods showed that the detected fish communities were similar between the two methods, with an overlap of 70%. Thus, our study suggests that to detect fish communities in backwater lakes, the performance of eDNA metabarcoding with the use of 1 L surface water sampling is similar to that of capturing methods. Therefore, eDNA metabarcoding can be used for fish community analysis but environmental factors that can cause PCR inhibition, should be considered in eDNA applications.</p><h1>Measuring global fish species richness with eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12929">doi:10.1111/1755-0998.12929</a></p>

<p><a href=None>None</a></p>
<p>Despite mounting threats to global freshwater and marine biodiversity, including climate change, habitat alteration, overharvesting and pollution, we struggle to know which species are present below the water's surface that are suffering from these stressors. However, the idea that a water sample containing environmental DNA (eDNA) can be screened using high-throughput sequencing and bioinformatics to reveal the identity of aquatic species is a revolutionary advance for studying the patterns of species extirpation, invasive species establishment and the dynamics of species richness. To date, many of the critical tests of fisheries diversity using this metabarcoding approach have been conducted in lower diversity systems (<40 fish species), but in this issue of Molecular Ecology Resources, Cilleros et al. (2018) described their eDNA application in the species-rich French Guiana fishery (>200 fish species) and showed the greater potential and some limitations of using eDNA in species-rich environments.</p><h1>Are drivers of root-associated fungal community structure context specific?</h1>
<p><a href="https://doi.org/10.1038/S41396-019-0350-Y">doi:10.1038/S41396-019-0350-Y</a></p>

<p><a href=None>None</a></p>
<p>The composition and structure of plant-root-associated fungal communities are determined by local abiotic and biotic conditions. However, the relative influence and identity of relationships to abiotic and biotic factors may differ across environmental and ecological contexts, and fungal functional groups. Thus, understanding which aspects of root-associated fungal community ecology generalise across contexts is the first step towards a more predictive framework. We investigated how the relative importance of biotic and abiotic factors scale across environmental and ecological contexts using high-throughput sequencing (ca. 55 M Illumina metabarcoding sequences) of >260 plant-root-associated fungal communities from six UK salt marshes across two geographic regions (South-East and North-West England) in winter and summer. Levels of root-associated fungal diversity were comparable with forests and temperate grasslands, quadrupling previous estimates of salt-marsh fungal diversity. Whilst abiotic variables were generally most important, a range of site- and spatial scale-specific abiotic and biotic drivers of diversity and community composition were observed. Consequently, predictive models of diversity trained on one site, extrapolated poorly to others. Fungal taxa from the same functional groups responded similarly to the specific drivers of diversity and composition. Thus site, spatial scale and functional group are key factors that, if accounted for, may lead to a more predictive understanding of fungal community ecology.</p><h1>From feces to data: A metabarcoding method for analyzing consumed and available prey in a bird-insect food web</h1>
<p><a href="https://doi.org/10.1002/Ece3.4787">doi:10.1002/Ece3.4787</a></p>

<p><a href=None>None</a></p>
<p>Diets play a key role in understanding trophic interactions. Knowing the actual structure of food webs contributes greatly to our understanding of biodiversity and ecosystem functioning. The research of prey preferences of different predators requires knowledge not only of the prey consumed, but also of what is available. In this study, we applied DNA metabarcoding to analyze the diet of 4 bird species (willow tits <i>Poecile montanus</i>, Siberian tits <i>Poecile cinctus</i>, great tits <i>Parus major</i> and blue tits <i>Cyanistes caeruleus</i>) by using the feces of nestlings. The availability of their assumed prey (Lepidoptera) was determined from feces of larvae (frass) collected from the main foraging habitat, birch (<i>Betula</i> spp.) canopy. We identified 53 prey species from the nestling feces, of which 11 (21%) were also detected from the frass samples (eight lepidopterans). Approximately 80% of identified prey species in the nestling feces represented lepidopterans, which is in line with the earlier studies on the parids' diet. A subsequent laboratory experiment showed a threshold for fecal sample size and the barcoding success, suggesting that the smallest frass samples do not contain enough larval DNA to be detected by high-throughput sequencing. To summarize, we apply metabarcoding for the first time in a combined approach to identify available prey (through frass) and consumed prey (via nestling feces), expanding the scope and precision for future dietary studies on insectivorous birds.</p><h1>Biodiversity assessments in the 21st century: the potential of insect traps to complement environmental samples for estimating eukaryotic and prokaryotic diversity using high-throughput DNA metabarcoding <sup>1</sup></h1>
<p><a href="https://doi.org/10.1139/Gen-2018-0096">doi:10.1139/Gen-2018-0096</a></p>

<p><a href=None>None</a></p>
<p>The rapid loss of biodiversity, coupled with difficulties in species identification, call for innovative approaches to assess biodiversity. Insects make up a substantial proportion of extant diversity and play fundamental roles in any given ecosystem. To complement morphological species identification, new techniques such as metabarcoding make it possible to quantify insect diversity and insect-ecosystem interactions through DNA sequencing. Here we examine the potential of bulk insect samples (i.e., containing many non-sorted specimens) to assess prokaryote and eukaryote biodiversity and to complement the taxonomic coverage of soil samples. We sampled 25 sites on three continents and in various ecosystems, collecting insects with SLAM traps (Brazil) and Malaise traps (South Africa and Sweden). We then compared our diversity estimates with the results obtained with biodiversity data from soil samples from the same localities. We found a largely different taxonomic composition between the soil and insect samples, testifying to the potential of bulk insect samples to complement soil samples. Finally, we found that non-destructive DNA extraction protocols, which preserve insect specimens for morphological studies, constitute a promising choice for cost-effective biodiversity assessments. We propose that the sampling and sequencing of insect samples should become a standard complement for biodiversity studies based on environmental DNA.</p><h1>Clay beads as artificial trapping matrices for monitoring bacterial distribution among urban stormwater infiltration systems and their connected aquifers</h1>
<p><a href="https://doi.org/10.1007/S10661-019-7190-0">doi:10.1007/S10661-019-7190-0</a></p>

<p><a href=None>None</a></p>
<p>Stormwater infiltration systems (SIS) have been developed to limit surface runoff and flooding in urban areas. The impacts of such practices on the ecological and biological quality of groundwater ecosystems remain poorly studied due to the lack of efficient methodologies to assess microbiological quality of aquifers. In the present study, a monitoring method based on the incubation of artificial matrices (clay beads) is presented to evaluate microbial biomass, microbial activities, and bacterial community structure. Four microbial variables (biomass, dehydrogenase and hydrolytic activities, bacterial community structures) were measured on clay beads incubated in three urban water types (stormwater surface runoffs, SIS-impacted and non-impacted groundwaters) for six SIS. Analyses based on next-generation sequencing (NGS) of partial rrs (16S rRNA) PCR products (V5-V6) were used to compare bacterial community structures of biofilms on clay beads after 10 days of incubation with those of waters collected from the same sampling points at three occasions. Biofilm biomass and activities on clay beads were indicative of nutrient transfers from surface to SIS-impacted groundwaters. Biofilms allowed impacts of SIS on groundwater bacterial community structures to be determined. Although bacterial communities on clay beads did not perfectly match those of waters, clay beads captured the most abundant bacterial taxa. They also captured bacterial taxa that were not detected in waters collected at three occasions during the incubation, demonstrating the integrative character of this approach. Monitoring biofilms on clay beads also allowed the tracking of bacterial genera containing species representing health concerns.</p><h1>Annotated 18S and 28S rDNA reference sequences of taxa in the planktonic diatom family Chaetocerotaceae</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0208929">doi:10.1371/Journal.Pone.0208929</a></p>

<p><a href=None>None</a></p>
<p>The species-rich diatom family Chaetocerotaceae is common in the coastal marine phytoplankton worldwide where it is responsible for a substantial part of the primary production. Despite its relevance for the global cycling of carbon and silica, many species are still described only morphologically, and numerous specimens do not fit any described taxa. Nowadays, studies to assess plankton biodiversity deploy high throughput sequencing metabarcoding of the 18S rDNA V4 region, but to translate the gathered metabarcodes into biologically meaningful taxa, there is a need for reference barcodes. However, 18S reference barcodes for this important family are still relatively scarce. We provide 18S rDNA and partial 28S rDNA reference sequences of 443 morphologically characterized chaetocerotacean strains. We gathered 164 of the 216 18S sequences and 244 of the 413 28S sequences of strains from the Gulf of Naples, Atlantic France, and Chile. Inferred phylogenies showed 84 terminal taxa in seven principal clades. Two of these clades included terminal taxa whose rDNA sequences contained spliceosomal and Group IC1 introns. Regarding the commonly used metabarcode markers in planktonic diversity studies, all terminal taxa can be discriminated with the 18S V4 hypervariable region; its primers fit their targets in all but two species, and the V4-tree topology is similar to that of the 18S. Hence V4-metabarcodes of unknown Chaetocerotaceae are assignable to the family. Regarding the V9 hypervariable region, most terminal taxa can be discriminated, but several contain introns in their primer targets. Moreover, poor phylogenetic resolution of the V9 region affects placement of metabarcodes of putative but unknown chaetocerotacean taxa, and hence, uncertainty in taxonomic assignment, even of higher taxa.</p><h1>Diatom diversity through HTS-metabarcoding in coastal European seas</h1>
<p><a href="https://doi.org/10.1038/S41598-018-36345-9">doi:10.1038/S41598-018-36345-9</a></p>

<p><a href=None>None</a></p>
<p>Diatoms constitute a diverse lineage of unicellular organisms abundant and ecologically important in aquatic ecosystems. Compared to other protists, their biology and taxonomy are well-studied, offering the opportunity to combine traditional approaches and new technologies. We examined a dataset of diatom 18S rRNA- and rDNA- (V4 region) reads from different plankton size-fractions and sediments from six European coastal marine sites, with the aim of identifying peculiarities and commonalities with respect to the whole protistan community. Almost all metabarcodes (99.6%) were assigned to known genera (121) and species (236), the most abundant of which were those already known from classic studies and coincided with those seen in light microscopy. rDNA and rRNA showed comparable patterns for the dominant taxa, but rRNA revealed a much higher diversity particularly in the sediment communities. Peculiar to diatoms is a tight bentho-pelagic coupling, with many benthic or planktonic species colonizing both water column and sediments and the dominance of planktonic species in both habitats. Overall metabarcoding results reflected the marked specificity of diatoms compared to other protistan groups in terms of morphological and ecological characteristics, at the same time confirming their great potential in the description of protist communities.</p><h1>Belowground fungal community diversity and composition associated with Norway spruce along an altitudinal gradient</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0208493">doi:10.1371/Journal.Pone.0208493</a></p>

<p><a href=None>None</a></p>
<p>Altitudinal gradients provide valuable information about the effects of environmental variables on changes in species richness and composition as well as the distribution of below ground fungal communities. Since most knowledge in this respect has been gathered on aboveground communities, we focused our study towards the characterization of belowground fungal communities associated with two different ages of Norway spruce (Picea abies) trees along an altitudinal gradient. By sequencing the internal transcribed spacer (ITS) region on the Illumina platform, we investigated the fungal communities in a floristically and geologically relatively well explored forest on the slope of Mt. Iseler of the Bavarian Alps. From fine roots and rhizosphere of a total of 90 of Norway spruce trees from 18 plots we detected 1285 taxa, with a range of 167 to 506 (average 377) taxa per plot. Fungal taxa are distributed over 96 different orders belonging to the phyla Ascomycota, Basidiomycota, Chrytridiomycota, Glomeromycota, and Mucoromycota. Overall the Agaricales (438 taxa) and Tremellales (81 taxa) belonging to the Basidiomycota and the Hypocreales (65 spp.) and Helotiales (61 taxa) belonging to the Ascomycota represented the taxon richest orders. The evaluation of our multivariate generalized mixed models indicate that the altitude has a significant influence on the composition of the fungal communities (p < 0.003) and that tree age determines community diversity (p < 0.05). A total of 47 ecological guilds were detected, of which the ectomycorrhizal and saprophytic guilds were the most taxon-rich. Our ITS amplicon Illumina sequencing approach allowed us to characterize a high fungal community diversity that would not be possible to capture with fruiting body surveys alone. We conclude that it is an invaluable tool for diverse monitoring tasks and inventorying biodiversity, especially in the detection of microorganisms developing very ephemeral and/or inconspicuous fruiting bodies or lacking them all together. Results suggest that the altitude mainly influences the community composition, whereas fungal diversity becomes higher in mature/older trees. Finally, we demonstrate that novel techniques from bacterial microbiome analyses are also useful for studying fungal diversity and community structure in a DNA metabarcoding approach, but that incomplete reference sequence databases so far limit effective identification.</p><h1>Evaluating the efficacy of sample collection approaches and DNA metabarcoding for identifying the diversity of plants utilized by nectivorous bats</h1>
<p><a href="https://doi.org/10.1139/Gen-2018-0102">doi:10.1139/Gen-2018-0102</a></p>

<p><a href=None>None</a></p>
<p>In this study, we evaluated the efficacy of sample collection approaches and DNA metabarcoding to identify plants utilized by nectivorous bats. Samples included guano collected from beneath bat roosts and pollen-swabs from bat fur, both of which were subjected to DNA metabarcoding and visual identification of pollen (microscopy) to measure plant diversity. Our objectives were to determine whether DNA metabarcoding could detect likely food plants of nectivorous bats, whether sample types would produce different estimates of plant diversity, and to compare results of DNA metabarcoding to visual identification. Visual identification found that 99% of pollen was from Agave, which is thought to be the bats' main food source. The dominant taxon found by metabarcoding was also Agavoideae, but a broader diversity of plant species was also detected, many of which are likely "by-catch" from the broader environment. Metabarcoding outcomes differed between sample types, likely because pollen-swabs measured the plant species visited by bats and guano samples measured all items consumed in the bat's diet, even those that were not pollen or nectar. Overall, metabarcoding is a powerful, high-throughput tool to understand bat ecology and species interactions, but careful analysis of results is necessary to derive accurate ecological conclusions.</p><h1>Protist Biodiversity and Biogeography in Lakes From Four Brazilian River-Floodplain Systems</h1>
<p><a href="https://doi.org/10.1111/Jeu.12703">doi:10.1111/Jeu.12703</a></p>

<p><a href=None>None</a></p>
<p>The biodiversity and biogeography of protists inhabiting many ecosystems have been intensely studied using different sequencing approaches, but tropical ecosystems are relatively under-studied. Here, we sampled planktonic waters from 32 lakes associated with four different river-floodplains systems in Brazil, and sequenced the DNA using a metabarcoding approach with general eukaryotic primers. The lakes were dominated by the largely free-living Discoba (mostly the Euglenida), Ciliophora, and Ochrophyta. There was low community similarity between lakes even within the same river-floodplain. The protists inhabiting these floodplain systems comprise part of the large and relatively undiscovered diversity in the tropics.</p><h1>Metabarcoding using multiplexed markers increases species detection in complex zooplankton communities</h1>
<p><a href="https://doi.org/10.1111/Eva.12694">doi:10.1111/Eva.12694</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding combines DNA barcoding with high-throughput sequencing, often using one genetic marker to understand complex and taxonomically diverse samples. However, species-level identification depends heavily on the choice of marker and the selected primer pair, often with a trade-off between successful species amplification and taxonomic resolution. We present a versatile metabarcoding protocol for biomonitoring that involves the use of two barcode markers (COI and 18S) and four primer pairs in a single high-throughput sequencing run, via sample multiplexing. We validate the protocol using a series of 24 mock zooplanktonic communities incorporating various levels of genetic variation. With the use of a single marker and single primer pair, the highest species recovery was 77%. With all three COI fragments, we detected 62%-83% of species across the mock communities, while the use of the 18S fragment alone resulted in the detection of 73%-75% of species. The species detection level was significantly improved to 89%-93% when both markers were used. Furthermore, multiplexing did not have a negative impact on the proportion of reads assigned to each species and the total number of species detected was similar to when markers were sequenced alone. Overall, our metabarcoding approach utilizing two barcode markers and multiple primer pairs per barcode improved species detection rates over a single marker/primer pair by 14% to 35%, making it an attractive and relatively cost-effective method for biomonitoring natural zooplankton communities. We strongly recommend combining evolutionary independent markers and, when necessary, multiple primer pairs per marker to increase species detection (i.e., reduce false negatives) in metabarcoding studies.</p><h1>Metabarcoding successfully tracks temporal changes in eukaryotic communities in coastal sediments</h1>
<p><a href="https://doi.org/10.1093/Femsec/Fiy226">doi:10.1093/Femsec/Fiy226</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding is a method that combines high-throughput DNA sequencing and DNA-based identification. Previously, this method has been successfully used to target spatial variation of eukaryote communities in marine sediments, however, the temporal changes in these communities remain understudied. Here, we follow the temporal changes of the eukaryote communities in Baltic Sea surface sediments collected from two coastal localities during three seasons of two consecutive years. Our study reveals that the structure of the sediment eukaryotic ecosystem was primarily driven by annual and seasonal changes in prevailing environmental conditions, whereas spatial variation was a less significant factor in explaining the variance in eukaryotic communities over time. Therefore, our data suggests that shifts in regional climate regime or large-scale changes in the environment are the overdriving factors in shaping the coastal eukaryotic sediment ecosystems rather than small-scale changes in local environmental conditions or heterogeneity in ecosystem structure. More studies targeting temporal changes are needed to further understand the long-term trends in ecosystem stability and response to climate change. Furthermore, this work contributes to the recent efforts in developing metabarcoding applications for environmental biomonitoring, proving a comprehensive option for traditional monitoring approaches.</p><h1>Author Correction: Comparing Sanger sequencing and high-throughput metabarcoding for inferring photobiont diversity in lichens</h1>
<p><a href="https://doi.org/10.1038/S41598-018-35358-8">doi:10.1038/S41598-018-35358-8</a></p>

<p><a href=None>None</a></p>
<p>A correction to this article has been published and is linked from the HTML and PDF versions of this paper. The error has been fixed in the paper.</p><h1>Approaches to integrating genetic data into ecological networks</h1>
<p><a href="https://doi.org/10.1111/Mec.14941">doi:10.1111/Mec.14941</a></p>

<p><a href=None>None</a></p>
<p>As molecular tools for assessing trophic interactions become common, research is increasingly focused on the construction of interaction networks. Here, we demonstrate three key methods for incorporating DNA data into network ecology and discuss analytical considerations using a model consisting of plants, insects, bats and their parasites from the Costa Rica dry forest. The simplest method involves the use of Sanger sequencing to acquire long sequences to validate or refine field identifications, for example of bats and their parasites, where one specimen yields one sequence and one identification. This method can be fully quantified and resolved and these data resemble traditional ecological networks. For more complex taxonomic identifications, we target multiple DNA loci, for example from a seed or fruit pulp sample in faeces. These networks are also well resolved but gene targets vary in resolution and quantification is difficult. Finally, for mixed templates such as faecal contents of insectivorous bats, we use DNA metabarcoding targeting two sequence lengths (157 and 407 bp) of one gene region and a MOTU, BLAST and BIN association approach to resolve nodes. This network type is complex to generate and analyse, and we discuss the implications of this type of resolution on network analysis. Using these data, we construct the first molecular-based network of networks containing 3,304 interactions between 762 nodes of eight trophic functions and involving parasitic, mutualistic and predatory interactions. We provide a comparison of the relative strengths and weaknesses of these data types in network ecology.</p><h1>Protist Diversity and Seasonal Dynamics in Skagerrak Plankton Communities as Revealed by Metabarcoding and Microscopy</h1>
<p><a href="https://doi.org/10.1111/Jeu.12700">doi:10.1111/Jeu.12700</a></p>

<p><a href=None>None</a></p>
<p>Protist community composition and seasonal dynamics are of major importance for the production of higher trophic levels, such as zooplankton and fish. Our aim was to reveal how the protist community in the Skagerrak changes through the seasons by combining high-throughput sequencing and microscopy of plankton collected monthly over two years. The V4 region of the 18S rRNA gene was amplified by eukaryote universal primers from the total RNA/cDNA. We found a strong seasonal variation in protist composition and proportional abundances, and a difference between two depths within the euphotic zone. Highest protist richness was found in late summer-early autumn, and lowest in winter. Temperature was the abiotic factor explaining most of the variation in diversity. Dinoflagellates was the most abundant and diverse group followed by ciliates and diatoms. We found about 70 new taxa recorded for the first time in the Skagerrak. The seasonal pattern in relative read abundance of major phytoplankton groups was well in accordance with microscopical biovolumes. This is the first metabarcoding study of the protist plankton community of all taxonomic groups and through seasons in the Skagerrak, which may serve as a baseline for future surveys to reveal effects of climate and environmental changes.</p><h1>Shotgun metagenomics of honey DNA: Evaluation of a methodological approach to describe a multi-kingdom honey bee derived environmental DNA signature</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0205575">doi:10.1371/Journal.Pone.0205575</a></p>

<p><a href=None>None</a></p>
<p>Honey bees are considered large-scale monitoring tools due to their environmental exploration and foraging activities. Traces of these activities can be recovered in the honey that also may reflect the hive ecological micro-conditions in which it has been produced. This study applied a next generation sequencing platform (Ion Torrent) for shotgun metagenomic analysis of honey environmental DNA (eDNA). The study tested a methodological framework to interpret DNA sequence information useful to describe the complex ecosystems of the honey bee colony superorganism, its pathosphere and the heterogeneity of the agroecological environments and environmental sources that left DNA marks in the honey. Analysis of two honeys reported sequence reads from five main organism groups (kingdoms or phyla): arthropods (that mainly included reads from Apis mellifera, several other members of the Hymenotpera, in addition to members of the Diptera, Coleoptera and Lepidoptera, as well as aphids and mites), plants (that clearly confirmed the botanical origin of the two honeys, i.e. orange tree blossom and eucalyptus tree blossom honeys), fungi and bacteria (including common hive and honey bee gut microorganisms, honey bee pathogens and plant pathogens), and viruses (which accounted for the largest number of reads in both honeys, mainly assigned to Apis mellifera filamentous virus). The shotgun metagenomic approach that was used in this study can be applied in large scale experiments that might have multiple objectives according to the multi-kingdom derived eDNA that is contained in the honey.</p><h1>The endangered northern bettong, Bettongia tropica, performs a unique and potentially irreplaceable dispersal function for ectomycorrhizal truffle fungi</h1>
<p><a href="https://doi.org/10.1111/Mec.14916">doi:10.1111/Mec.14916</a></p>

<p><a href=None>None</a></p>
<p>Organisms that are highly connected in food webs often perform unique and vital functions within ecosystems. Understanding the unique ecological roles played by highly connected organisms and the consequences of their loss requires a comprehensive understanding of the functional redundancy amongst organisms. One important, yet poorly understood, food web is that between truffle-forming ectomycorrhizal fungi and their mammalian consumers and dispersers. Mammalian fungal specialists rely on fungi as a food source, and they consume and disperse a higher diversity and abundance of fungi than do mycophagous mammals with generalist diets. Therefore, we hypothesize that mammalian fungal specialists are functionally distinct because they disperse a set of fungal taxa not fully nested within the set consumed by the combined generalist mammalian community (i.e., functional redundancy of fungal dispersal is limited). Using high-throughput sequencing, we compared the fungal composition of 93 scats from the endangered fungal specialist northern bettong (Bettongia tropica) and 120 scats from nine co-occurring generalist mammal species across three sites and three seasons. Compared with other generalist mammals, B. tropica consumed a more diverse fungal diet with more unique taxa. This aligns with our hypothesis that B. tropica performs a unique dispersal function for ectomycorrhizal truffle fungi. Additionally, modelling of mammalian extinctions predicted rapid loss of food web connections which could result in loss of gene flow for truffle taxa. Our results suggest that this system is sensitive to the extinction of highly connected specialist species like B. tropica and their loss could have consequences for ectomycorrhizal truffle fungal diversity. This suggests that the conservation of fungal specialists is imperative to maintaining ectomycorrhizal fungal diversity and healthy plant-mycorrhizal relationships.</p><h1>High-throughput identification and diagnostics of pathogens and pests: Overview and practical recommendations</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12959">doi:10.1111/1755-0998.12959</a></p>

<p><a href=None>None</a></p>
<p>High-throughput identification technologies provide efficient tools for understanding the ecology and functioning of microorganisms. Yet, these methods have been only rarely used for monitoring and testing ecological hypotheses in plant pathogens and pests in spite of their immense importance in agriculture, forestry and plant community dynamics. The main objectives of this manuscript are the following: (a) to provide a comprehensive overview about the state-of-the-art high-throughput quantification and molecular identification methods used to address population dynamics, community ecology and host associations of microorganisms, with a specific focus on antagonists such as pathogens, viruses and pests; (b) to compile available information and provide recommendations about specific protocols and workable primers for bacteria, fungi, oomycetes and insect pests; and (c) to provide examples of novel methods used in other microbiological disciplines that are of great potential use for testing specific biological hypotheses related to pathology. Finally, we evaluate the overall perspectives of the state-of-the-art and still evolving methods for diagnostics and population- and community-level ecological research of pathogens and pests.</p><h1>Metabarcoding of shrimp stomach content: Harnessing a natural sampler for fish biodiversity monitoring</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12956">doi:10.1111/1755-0998.12956</a></p>

<p><a href=None>None</a></p>
<p>Given their positioning and biological productivity, estuaries have long represented key providers of ecosystem services and consequently remain under remarkable pressure from numerous forms of anthropogenic impact. The monitoring of fish communities in space and time is one of the most widespread and established approaches to assess the ecological status of estuaries and other coastal habitats, but traditional fish surveys are invasive, costly, labour intensive and highly selective. Recently, the application of metabarcoding techniques, on either sediment or aqueous environmental DNA, has rapidly gained popularity. Here, we evaluate the application of a novel, high-throughput DNA-based monitoring tool to assess fish diversity, based on the analysis of the gut contents of a generalist predator/scavenger, the European brown shrimp, Crangon crangon. Sediment and shrimp samples were collected from eight European estuaries, and DNA metabarcoding (using both 12S and COI markers) was carried out to infer fish assemblage composition. We detected 32 teleost species (16 and 20, for 12S and COI, respectively). Twice as many species were recovered using metabarcoding than by traditional net surveys. By comparing and interweaving trophic, environmental DNA and traditional survey-based techniques, we show that the DNA-assisted gut content analysis of a ubiquitous, easily accessible, generalist species may serve as a powerful, rapid and cost-effective tool for large-scale, routine estuarine biodiversity monitoring.</p><h1>Shifts in community composition and co-occurrence patterns of phyllosphere fungi inhabiting <i>Mussaenda shikokiana</i> along an elevation gradient</h1>
<p><a href="https://doi.org/10.7717/Peerj.5767">doi:10.7717/Peerj.5767</a></p>

<p><a href=None>None</a></p>
<p>The altitudinal effects on the distributions of phyllosphere fungal assemblages in conspecific plants remain poorly elucidated. To address this, phyllosphere fungal communities associated with <i>Mussaenda shikokiana</i> were investigated at four sites across a 350 m elevation gradient in a subtropical forest by employing Illumina metabarcoding of the fungal internal transcribed spacer 2 (ITS2) region. Our results demonstrated that phyllosphere fungal assemblages with a single host possessed high taxonomic diversity and multiple trophic guilds. OTU richness was significantly influenced by elevation. The elevation gradient also entailed distinct shifts in the community composition of phyllosphere fungi, which was significantly related to geographical distance and mean annual temperature (MAT). Additionally, comparison of phyllosphere fungal networks showed reduced connectivity with increasing elevation. Our data provide insights on the distribution and interactions of the phyllosphere fungal community associated with a single host along a short elevation gradient.</p><h1>Development and testing of an 18S rRNA phylogenetic microarray for marine sediments</h1>
<p><a href="https://doi.org/S0167-7012(18)30829-7">doi:S0167-7012(18)30829-7</a></p>

<p><a href=None>None</a></p>
<p>There is increasing interest in finding new, more efficient methods for routine monitoring of anthropogenic effects on benthic biodiversity and ecosystems. A range of molecular methods have been developed for assessing biodiversity the last decades. Particularly interesting are microarrays targeting phylogenetic marker genes, such as the small subunit of ribosomal RNA in eukaryotes (18S rRNA). This method can detect a large number of taxonomic groups in several samples simultaneously within a relatively short time and has the potential for incorporation in automated remote sensing pipelines. In this study we developed and tested a microarray for eukaryotes in marine sediments. The probes were designed to target 18S rRNA OTUs obtained through metabarcoding of marine sediments. The resulting microarray was tested using both a spiked sample consisting of 50 plasmid-clones and further, samples of genomic DNA extracted from marine sediments. We developed a filtration pipeline to eliminate noise and reduce the number of false positives, making it possible to detect and quantify most of the OTUs with ≥ 0.1% abundance in the spiked sample. Our data indicated that the microarray was specific at higher taxonomic levels. However, the specificity decreased with increasing sequence similarity suggesting cross-hybridization between closely related OTUs. When using genomic DNA isolated from marine sediment there was a positive correlation between hybridization intensity signals and abundance of sequencing reads, suggesting a quantitative behavior of the microarray. Overall, the data suggest a potential for microarrays as a tool for high throughput sediment monitoring.</p><h1>Metabarcoding of marine environmental DNA based on mitochondrial and nuclear genes</h1>
<p><a href="https://doi.org/10.1038/S41598-018-32917-X">doi:10.1038/S41598-018-32917-X</a></p>

<p><a href=None>None</a></p>
<p>We establish the new approach of environmental DNA (eDNA) analyses for the North Sea. Our study uses a multigene approach, including the mitochondrial cytochrome-c-oxidase subunit I (COI) gene for analyzing species composition and the nuclear hypervariable region V8 of 18S rDNA for analyzing supraspecific biodiversity. A new minibarcode primer (124 bp) was created on the basis of a metazoan COI barcode library with 506 species and tested in silico, in vitro, and in situ. We applied high throughput sequencing to filtrates of 23 near-bottom water samples taken at three seasons from 14 stations. The set of COI primers allowed amplification of mitochondrial minibarcodes for diverse metazoan phyla and the differentiation at the species level for more than 99% of the specimens in the dataset. Our results revealed that the number of sequences is not consistent with proportions in the given DNA mixture. Altogether, environmental sequences could be assigned to 114 species and to 12 metazoan phyla. A spatial distribution of taxa recovered by eDNA was congruent with known distributions. Finally, the successful detection of species and biodiversity depends on a comprehensive sequence reference database. Our study offers a powerful tool for future biodiversity research, including the detection of nonnative species.</p><h1>High-throughput metabarcoding reveals the effect of physicochemical soil properties on soil and litter biodiversity and community turnover across Amazonia</h1>
<p><a href="https://doi.org/10.7717/Peerj.5661">doi:10.7717/Peerj.5661</a></p>

<p><a href=None>None</a></p>
<p>Knowledge on the globally outstanding Amazonian biodiversity and its environmental determinants stems almost exclusively from aboveground organisms, notably plants. In contrast, the environmental factors and habitat preferences that drive diversity patterns for micro-organisms in the ground remain elusive, despite the fact that micro-organisms constitute the overwhelming majority of life forms in any given location, in terms of both diversity and abundance. Here we address how the diversity and community turnover of operational taxonomic units (OTU) of organisms in soil and litter respond to soil physicochemical properties; whether OTU diversities and community composition in soil and litter are correlated with each other; and whether they respond in a similar way to soil properties.</p><h1>DNA metabarcoding unveils multiscale trophic variation in a widespread coastal opportunist</h1>
<p><a href="https://doi.org/10.1111/Mec.14886">doi:10.1111/Mec.14886</a></p>

<p><a href=None>None</a></p>
<p>A thorough understanding of ecological networks relies on comprehensive information on trophic relationships among species. Since unpicking the diet of many organisms is unattainable using traditional morphology-based approaches, the application of high-throughput sequencing methods represents a rapid and powerful way forward. Here, we assessed the application of DNA metabarcoding with nearly universal primers for the mitochondrial marker cytochrome c oxidase I in defining the trophic ecology of adult brown shrimp, Crangon crangon, in six European estuaries. The exact trophic role of this abundant and widespread coastal benthic species is somewhat controversial, while information on geographical variation remains scant. Results revealed a highly opportunistic behaviour. Shrimp stomach contents contained hundreds of taxa (>1,000 molecular operational taxonomic units), of which 291 were identified as distinct species, belonging to 35 phyla. Only twenty ascertained species had a mean relative abundance of more than 0.5%. Predominant species included other abundant coastal and estuarine taxa, including the shore crab Carcinus maenas and the amphipod Corophium volutator. Jacobs' selectivity index estimates based on DNA extracted from both shrimp stomachs and sediment samples were used to assess the shrimp's trophic niche indicating a generalist diet, dominated by crustaceans, polychaetes and fish. Spatial variation in diet composition, at regional and local scales, confirmed the highly flexible nature of this trophic opportunist. Furthermore, the detection of a prevalent, possibly endoparasitic fungus (Purpureocillium lilacinum) in the shrimp's stomach demonstrates the wide range of questions that can be addressed using metabarcoding, towards a more robust reconstruction of ecological networks.</p><h1>Biodiversity estimates and ecological interpretations of meiofaunal communities are biased by the taxonomic approach</h1>
<p><a href="https://doi.org/10.1038/S42003-018-0119-2">doi:10.1038/S42003-018-0119-2</a></p>

<p><a href=None>None</a></p>
<p>Accurate assessments of biodiversity are crucial to advising ecosystem-monitoring programs and understanding ecosystem function. Nevertheless, a standard operating procedure to assess biodiversity accurately and consistently has not been established. This is especially true for meiofauna, a diverse community (>20 phyla) of small benthic invertebrates that have fundamental ecological roles. Recent studies show that metabarcoding is a cost-effective and time-effective method to estimate meiofauna biodiversity, in contrast to morphological-based taxonomy. Here, we compare biodiversity assessments of a diverse meiofaunal community derived by applying multiple taxonomic methods based on comparative morphology, molecular phylogenetic analysis, DNA barcoding of individual specimens, and metabarcoding of environmental DNA. We show that biodiversity estimates are strongly biased across taxonomic methods and phyla. Such biases affect understanding of community structures and ecological interpretations. This study supports the urgency of improving aspects of environmental high-throughput sequencing and the value of taxonomists in correctly understanding biodiversity estimates.</p><h1>Toward an ecoregion scale evaluation of eDNA metabarcoding primers: A case study for the freshwater fish biodiversity of the Murray-Darling Basin (Australia)</h1>
<p><a href="https://doi.org/10.1002/Ece3.4387">doi:10.1002/Ece3.4387</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of environmental DNA (i.e., eDNA metabarcoding) has become an increasingly popular method for monitoring aquatic biodiversity. At present, such analyses require target-specific primers to amplify DNA barcodes from co-occurring species, and this initial amplification can introduce biases. Understanding the performance of different primers is thus recommended prior to undertaking any metabarcoding initiative. While multiple software programs are available to evaluate metabarcoding primers, all programs have their own strengths and weaknesses. Therefore, a robust in silico workflow for the evaluation of metabarcoding primers will benefit from the use of multiple programs. Furthermore, geographic differences in species biodiversity are likely to influence the performance of metabarcoding primers and further complicate the evaluation process. Here, an in silico workflow is presented that can be used to evaluate the performance of metabarcoding primers on an ecoregion scale. This workflow was used to evaluate the performance of published and newly developed eDNA metabarcoding primers for the freshwater fish biodiversity of the Murray-Darling Basin (Australia). To validate the in silico workflow, a subset of the primers, including one newly designed primer pair, were used in metabarcoding analyses of an artificial DNA community and eDNA samples. The results show that the in silico workflow allows for a robust evaluation of metabarcoding primers and can reveal important trade-offs that need to be considered when selecting the most suitable primer. Additionally, a new primer pair was described and validated that allows for more robust taxonomic assignments and is less influenced by primer biases compared to commonly used fish metabarcoding primers.</p><h1>Using noninvasive metagenomics to characterize viral communities from wildlife</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12946">doi:10.1111/1755-0998.12946</a></p>

<p><a href=None>None</a></p>
<p>Microbial communities play an important role in organismal and ecosystem health. While high-throughput metabarcoding has revolutionized the study of bacterial communities, generating comparable viral communities has proven elusive, particularly in wildlife samples where the diversity of viruses and limited quantities of viral nucleic acid present distinctive challenges. Metagenomic sequencing is a promising solution for studying viral communities, but the lack of standardized methods currently precludes comparisons across host taxa or localities. Here, we developed an untargeted shotgun metagenomic sequencing protocol to generate comparable viral communities from noninvasively collected faecal and oropharyngeal swabs. Using samples from common vampire bats (Desmodus rotundus), a key species for virus transmission to humans and domestic animals, we tested how different storage media, nucleic acid extraction procedures and enrichment steps affect viral community detection. Based on finding viral contamination in foetal bovine serum, we recommend storing swabs in RNAlater or another nonbiological medium. We recommend extracting nucleic acid directly from swabs rather than from supernatant or pelleted material, which had undetectable levels of viral RNA. Results from a low-input RNA library preparation protocol suggest that ribosomal RNA depletion and light DNase treatment reduce host and bacterial nucleic acid, and improve virus detection. Finally, applying our approach to twelve pooled samples from seven localities in Peru, we showed that detected viral communities saturated at the attained sequencing depth, allowing unbiased comparisons of viral community composition. Future studies using the methods outlined here will elucidate the determinants of viral communities across host species, environments and time.</p><h1>Using metabarcoding to compare the suitability of two blood-feeding leech species for sampling mammalian diversity in North Borneo</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12943">doi:10.1111/1755-0998.12943</a></p>

<p><a href=None>None</a></p>
<p>The application of high-throughput sequencing (HTS) for metabarcoding of mixed samples offers new opportunities in conservation biology. Recently, the successful detection of prey DNA from the guts of leeches has raised the possibility that these, and other blood-feeding invertebrates, might serve as useful samplers of mammals. Yet little is known about whether sympatric leech species differ in their feeding preferences, and whether this has a bearing on their relative suitability for monitoring local mammalian diversity. To address these questions, we collected spatially matched samples of two congeneric leech species Haemadipsa picta and Haemadipsa sumatrana from lowland rainforest in Borneo. For each species, we pooled ~500 leeches into batches of 10 individuals, performed PCR to target a section of the mammalian 16S rRNA locus and undertook sequencing of amplicon libraries using an Illumina MiSeq. In total, we identified sequences from 14 mammalian genera, spanning nine families and five orders. We found greater numbers of detections, and higher diversity of OTUs, in H. picta compared with H. sumatrana, with rodents only present in the former leech species. However, comparison of samples from across the landscape revealed no significant difference in mammal community composition between the leech species. We therefore suggest that H. picta is the more suitable iDNA sampler in this degraded Bornean forest. We conclude that the choice of invertebrate sampler can influence the detectability of different mammal groups and that this should be accounted for when designing iDNA studies.</p><h1>Metabarcoding by capture using a single COI probe (MCSP) to identify and quantify fish species in ichthyoplankton swarms</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0202976">doi:10.1371/Journal.Pone.0202976</a></p>

<p><a href=None>None</a></p>
<p>The ability to determine the composition and relative frequencies of fish species in large ichthyoplankton swarms could have extremely important ecological applications However, this task is currently hampered by methodological limitations. We proposed a new method for Amazonian species based on hybridization capture of the COI gene DNA from a distant species (Danio rerio), absent from our study area (the Amazon basin). The COI sequence of this species is approximately equidistant from all COI of Amazonian species available. By using this sequence as probe we successfully facilitated the simultaneous identification of fish larvae belonging to the order Siluriformes and to the Characiformes represented in our ichthyoplankton samples. Species relative frequencies, estimated by the number of reads, showed almost perfect correlations with true frequencies estimated by a Sanger approach, allowing the development of a quantitative approach. We also proposed a further improvement to a previous protocol, which enables lowering the sequencing effort by 40 times. This new Metabarcoding by Capture using a Single Probe (MCSP) methodology could have important implications for ecology, fisheries management and conservation in fish biodiversity hotspots worldwide. Our approach could easily be extended to other plant and animal taxa.</p><h1>Over 2.5 million COI sequences in GenBank and growing</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0200177">doi:10.1371/Journal.Pone.0200177</a></p>

<p><a href=None>None</a></p>
<p>The increasing popularity of cytochrome c oxidase subunit 1 (COI) DNA metabarcoding warrants a careful look at the underlying reference databases used to make high-throughput taxonomic assignments. The objectives of this study are to document trends and assess the future usability of COI records for metabarcode identification. The number of COI records deposited to the NCBI nucleotide database has increased by a geometric average of 51% per year, from 8,137 records deposited in 2003 to a cumulative total of ~ 2.5 million by the end of 2017. About half of these records are fully identified to the species rank, 92% are at least 500 bp in length, 74% have a country annotation, and 51% have latitude-longitude annotations. To ensure the future usability of COI records in GenBank we suggest: 1) Improving the geographic representation of COI records, 2) Improving the cross-referencing of COI records in the Barcode of Life Data System and GenBank to facilitate consolidation and incorporation into existing bioinformatic pipelines, 3) Adherence to the minimum information about a marker gene sequence guidelines, and 4) Integrating metabarcodes from eDNA and mixed community studies with existing reference sequences. The growth of COI reference records over the past 15 years has been substantial and is likely to be a resource across many fields for years to come.</p><h1>Protocols for Investigating the Leaf Mycobiome Using High-Throughput DNA Sequencing</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-8724-5_4">doi:10.1007/978-1-4939-8724-5_4</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of taxon-specific loci, or DNA metabarcoding, has become an invaluable tool for investigating the composition of plant-associated fungal communities and for elucidating plant-fungal interactions. While sequencing fungal communities has become routine, there remain numerous potential sources of systematic error that can introduce biases and compromise metabarcoding data. This chapter presents a protocol for DNA metabarcoding of the leaf mycobiome based on current best practices to minimize errors through careful laboratory practices and validation.</p><h1>Barcoding blood meals: New vertebrate-specific primer sets for assigning taxonomic identities to host DNA from mosquito blood meals</h1>
<p><a href="https://doi.org/10.1371/Journal.Pntd.0006767">doi:10.1371/Journal.Pntd.0006767</a></p>

<p><a href=None>None</a></p>
<p>The transmission dynamics of mosquito-vectored pathogens are, in part, mediated by mosquito host-feeding patterns. These patterns are elucidated using blood meal analysis, a collection of serological and molecular techniques that determine the taxonomic identities of the host animals from which blood meals are derived. Modern blood meal analyses rely on polymerase chain reaction (PCR), DNA sequencing, and bioinformatic comparisons of blood meal DNA sequences to reference databases. Ideally, primers used in blood meal analysis PCRs amplify templates from a taxonomically diverse range of vertebrates, produce a short amplicon, and avoid co-amplification of non-target templates. Few primer sets that fit these requirements are available for the cytochrome c oxidase subunit I (COI) gene, the species identification marker with the highest taxonomic coverage in reference databases. Here, we present new primer sets designed to amplify fragments of the DNA barcoding region of the vertebrate COI gene, while avoiding co-amplification of mosquito templates, without multiplexed or nested PCR. Primers were validated using host vertebrate DNA templates from mosquito blood meals of known origin, representing all terrestrial vertebrate classes, and field-collected mosquito blood meals of unknown origin. We found that the primers were generally effective in amplifying vertebrate host, but not mosquito DNA templates. Applied to the sample of unknown mosquito blood meals, > 98% (60/61) of blood meals samples were reliably identified, demonstrating the feasibility of identifying mosquito hosts with the new primers. These primers are beneficial in that they can be used to amplify COI templates from a diverse range of vertebrate hosts using standard PCR, thereby streamlining the process of identifying the hosts of mosquitoes, and could be applied to next generation DNA sequencing and metabarcoding approaches.</p><h1>Finding the pond through the weeds: eDNA reveals underestimated diversity of pondweeds</h1>
<p><a href="https://doi.org/10.1002/Aps3.1155">doi:10.1002/Aps3.1155</a></p>

<p><a href=None>None</a></p>
<p>The detection of environmental DNA (eDNA) using high-throughput sequencing has rapidly emerged as a method to detect organisms from environmental samples. However, eDNA studies of aquatic biomes have focused on surveillance of animal species with less emphasis on plants. Pondweeds are important bioindicators of freshwater ecosystems, although their diversity is underestimated due to difficulties in morphological identification and monitoring.</p><h1>Why the COI barcode should be the community DNA metabarcode for the metazoa</h1>
<p><a href="https://doi.org/10.1111/Mec.14844">doi:10.1111/Mec.14844</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of complex metazoan communities is increasingly being used to measure biodiversity in terrestrial, freshwater and marine ecosystems, revolutionizing our ability to observe patterns and infer processes regarding the origin and conservation of biodiversity. A fundamentally important question is which genetic marker to amplify, and although the mitochondrial cytochrome oxidase subunit I (COI) gene is one of the more widely used markers in metabarcoding for the Metazoa, doubts have recently been raised about its suitability. We argue that (a) the extensive coverage of reference sequence databases for COI; (b) the variation it presents; (c) the comparative advantages for denoising protein-coding genes; and (d) recent advances in DNA sequencing protocols argue in favour of standardizing for the use of COI for metazoan community samples. We also highlight where research efforts should focus to maximize the utility of metabarcoding.</p><h1>Multi-marker metabarcoding approach to study mesozooplankton at basin scale</h1>
<p><a href="https://doi.org/10.1038/S41598-018-30157-7">doi:10.1038/S41598-018-30157-7</a></p>

<p><a href=None>None</a></p>
<p>Zooplankton plays a pivotal role in marine ecosystems and the characterisation of its biodiversity still represents a challenge for marine ecologists. In this study, mesozooplankton composition from 46 samples collected in summer along the western Adriatic Sea, was retrieved by DNA metabarcoding analysis. For the first time, the highly variable fragments of the mtDNA COI and the V9 region of 18S rRNA genes were used in a combined matrix to compile an inventory of mesozooplankton at basin scale. The number of sequences retrieved after quality filtering were 824,148 and 223,273 for COI and 18S (V9), respectively. The taxonomical assignment against reference sequences, using 95% (for COI) and 97% (for 18S) similarity thresholds, recovered 234 taxa. NMDS plots and cluster analysis divided coastal from offshore samples and the most representative species of these clusters were distributed according to the dominant surface current pattern of the Adriatic for the summer period. For selected sampling sites, mesozooplankton species were also identified under a stereo microscope providing insights on the strength and weakness of the two approaches. In addition, DNA metabarcoding was shown to be helpful for the monitoring of non-indigenous marine metazoans and spawning areas of commercial fish species. We defined pros and cons of applying this approach at basin scale and the benefits of combining the datasets from two genetic markers.</p><h1>Evaluating freshwater macroinvertebrates from eDNA metabarcoding: A river Nalón case study</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0201741">doi:10.1371/Journal.Pone.0201741</a></p>

<p><a href=None>None</a></p>
<p>Rivers are a vital resource for human wellbeing. To reduce human impact on water bodies, the European Union has established an essential regulatory framework for protection and sustainable management (WFD; 2000/60/EC). In this strategy, reliable and economic bioindicators are a fundamental component. Benthic macroinvertebrates are the group most commonly used as bioindicators through all European countries. However, their conventional assessment currently entails serious cost-efficiency limitations. In this study, we have tested the reliability of metabarcoding as a tool to record river macroinvertebrates using samples from a mock community (in vitro validation) and eDNA extracted for field validation from water from six sites within a north Iberian river (River Nalón, Asturias, Spain). Two markers (V4 region within the nuclear 18S rDNA and a fragment of the mitochondrial COI gene) were amplified and sequenced using an Illumina platform. The molecular technique has proven to be more sensitive than the visual one. A cost-benefit analysis shows that the metabarcoding approach is more expensive than conventional techniques for determining macroinvertebrate communities but requires fewer sampling and identification efforts. Our results suggest metabarcoding is a useful tool for alternative assessment of freshwater quality.</p><h1>Ballast Water Exchange and Invasion Risk Posed by Intracoastal Vessel Traffic: An Evaluation Using High Throughput Sequencing</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.8B02108">doi:10.1021/Acs.Est.8B02108</a></p>

<p><a href=None>None</a></p>
<p>Ballast water remains a potent vector of non-native aquatic species introductions, despite increased global efforts to reduce risk of ballast water mediated invasions. This is particularly true of intracoastal vessel traffic, whose characteristics may limit the feasibility and efficacy of management through ballast water exchange (BWE). Here we utilize high throughput sequencing (HTS) to assess biological communities associated with ballast water being delivered to Valdez, Alaska from multiple source ports along the Pacific Coast of the United States. Our analyses indicate that BWE has a significant but modest effect on ballast water assemblages. Although overall richness was not reduced with exchange, we detected losses of some common benthic coastal taxa (e.g., decapods, mollusks, bryozoans, cnidaria) and gains of open ocean taxa (e.g., certain copepods, diatoms, and dinoflagellates), including some potentially toxic species. HTS-based metabarcoding identified significantly differentiated biodiversity signatures from individual source ports; this signal persisted, though weakened, in vessels undergoing BWE, indicating incomplete faunal turnover associated with management. Our analysis also enabled identification of taxa that may be of particular concern if established in Alaskan waters. While these results reveal a clear effect of BWE on diversity in intracoastal transit, they also indicate continued introduction risk of non-native and harmful taxa.</p><h1>Updating Plakobranchus cf. ianthobapsus (Gastropoda, Sacoglossa) host use: Diverse algal-animal interactions revealed by NGS with implications for invasive species management</h1>
<p><a href="https://doi.org/S1055-7903(18)30054-X">doi:S1055-7903(18)30054-X</a></p>

<p><a href=None>None</a></p>
<p>Sacoglossa, the "sap sucking" sea slugs, are highly specialized herbivores and the only metazoans that exhibit kleptoplasty, the sequestration and retention of chloroplasts from algae. Plakobranchus is one of the most generalistic herbivores within this order, with as many as 12 reported "algal host" (i.e. kleptoplast source) species. However, kleptoplast diversity studies conducted on Plakobranchus to date most likely underestimated the full diversity of kleptoplast sources within the studied populations due to limitations of the molecular techniques employed. Here, we apply a high throughput sequencing technique to assess kleptoplast diversity of Plakobranchus cf. ianthobapsus' from 10 sites across the Main Hawaiian Islands during winter and summer seasons. In so doing, we effectively used P. cf. ianthobapsus as a novel sampling tool to explore diminutive algal communities, including the current distribution of the invasive alga "Avrainvillea amadelpha." Our results show that P. cf. ianthobapsus sequesters chloroplasts from 23 algal species from across the siphonous green algal order Bryopsidales. We identified "Avrainvillea amadelpha" and Codium edule as new host species for P. cf. ianthobapusus, but their rarity among the data suggests they were most likely less preferential as hosts and were possibly utilized due to low abundance or unavailability of more preferable species, and therefore a response to starvation risk. Additionally, the identification of the highly invasive siphonous green alga "A. amadelpha" as a kleptoplast source provides new fine-scale range and distribution data for this problematic species. Overall kleptoplast diversity does not differ among sites, except in a coral-dominated, (i.e. not algal dominated) environment, suggesting that siphonous algal assemblages are common in algal-dominated ecosystems in the Hawaiian Islands. Diversity dissimilarity among seasons was recovered from the majority of sites sampled, supporting the need for seasonal data collection in algal diversity assessments. This case study using metabarcoding of sacoglossan kleptoplasts provides deeper insights into these plant-animal interactions with a better understanding of host use than previous studies using traditional molecular methods and illustrates how algal diversity studies on the scale of plastids can have implications for understanding algal community structure and invasive species dynamics.</p><h1>Slippage of degenerate primers can cause variation in amplicon length</h1>
<p><a href="https://doi.org/10.1038/S41598-018-29364-Z">doi:10.1038/S41598-018-29364-Z</a></p>

<p><a href=None>None</a></p>
<p>It is well understood that homopolymer regions should be avoided for primer binding to prevent off-target amplification. However, in metabarcoding, it is often difficult to avoid primer degeneracy in order to maximize taxa detection. We here investigate primer binding specificity using different primer sets from several invertebrate metabarcoding studies. Our results indicate that primers frequently bound 1-2 bp upstream in taxa where a homopolymer region was present in the amplification direction. Primer binding 1 bp downstream was observed less frequently. This primer slippage leads to taxon-specific length variation in amplicons and subsequent length variation in recovered sequences. Some widely used primer sets were severely affected by this bias, while others were not. While this variation will only have small impacts on the designation of Operational Taxonomic Units (OTUs) by clustering algorithms that ignore terminal gaps, primer sets employed in metabarcoding projects should be evaluated for their sensitivity to slippage. Moreover, steps should be taken to reduce slippage by improving protocols for primer design. For example, the flanking region adjacent to the 3' end of the primer is not considered by current primer development software although GC clamps in this position could mitigate slippage.</p><h1>Multifaceted DNA metabarcoding: Validation of a noninvasive, next-generation approach to studying bat populations</h1>
<p><a href="https://doi.org/10.1111/Eva.12644">doi:10.1111/Eva.12644</a></p>

<p><a href=None>None</a></p>
<p>As multiple species of bats are currently experiencing dramatic declines in populations due to white-nose syndrome (WNS) and other factors, conservation managers have an urgent need for data on the ecology and overall status of populations of once-common bat species. Standard approaches to obtain data on bat populations often involve capture and handling, requiring extensive expertise and unavoidably resulting in stress to the bats. New methods to rapidly obtain critical data are needed that minimize both the stress on bats and the spread of WNS. Guano provides a noninvasive source of DNA that includes information from the bat, but also dietary items, parasites, and pathogens. DNA metabarcoding is a high-throughput, DNA-based identification technique to assess the biodiversity of environmental or fecal samples. We investigated the use of multifaceted DNA metabarcoding (MDM), a technique combining next-generation DNA sequencing (NGS), DNA barcodes, and bioinformatic analysis, to simultaneously collect data on multiple parameters of interest (bat species composition, individual genotype, sex ratios, diet, parasites, and presence of WNS) from fecal samples using a single NGS run. We tested the accuracy of each MDM assay using samples in which these parameters were previously determined using conventional approaches. We found that assays for bat species identification, insect diet, parasite diversity, and genotype were both sensitive and accurate, the assay to detect WNS was highly sensitive but requires careful sample processing steps to ensure the reliability of results, while assays for nectivorous diet and sex showed lower sensitivity. MDM was able to quantify multiple data classes from fecal samples simultaneously, and results were consistent whether we included assays for a single data class or multiple data classes. Overall, MDM is a useful approach that employs noninvasive sampling and a customizable suite of assays to gain important and largely accurate information on bat ecology and population dynamics.</p><h1>Enigmatic Diphyllatea eukaryotes: culturing and targeted PacBio RS amplicon sequencing reveals a higher order taxonomic diversity and global distribution</h1>
<p><a href="https://doi.org/10.1186/S12862-018-1224-Z">doi:10.1186/S12862-018-1224-Z</a></p>

<p><a href=None>None</a></p>
<p>The class Diphyllatea belongs to a group of enigmatic unicellular eukaryotes that play a key role in reconstructing the morphological innovation and diversification of early eukaryotic evolution. Despite its evolutionary significance, very little is known about the phylogeny and species diversity of Diphyllatea. Only three species have described morphology, being taxonomically divided by flagella number, two or four, and cell size. Currently, one 18S rRNA Diphyllatea sequence is available, with environmental sequencing surveys reporting only a single partial sequence from a Diphyllatea-like organism. Accordingly, geographical distribution of Diphyllatea based on molecular data is limited, despite morphological data suggesting the class has a global distribution. We here present a first attempt to understand species distribution, diversity and higher order structure of Diphyllatea.</p><h1>Microplastic bacterial communities in the Bay of Brest: Influence of polymer type and size</h1>
<p><a href="https://doi.org/S0269-7491(18)31478-7">doi:S0269-7491(18)31478-7</a></p>

<p><a href=None>None</a></p>
<p>Microplastics (<5 mm) exhibit intrinsic features such as density, hydrophobic surface, or high surface/volume ratio, that are known to promote microbial colonization and biofilm formation in marine ecosystems. Yet, a relatively low number of studies have investigated the nature of microplastic associated bacterial communities in coastal ecosystems and the potential factors influencing their composition and structure. Here, we characterized microplastics collected in the Bay of Brest by manual sorting followed by Raman spectroscopy and studied their associated bacterial assemblages using 16S amplicon high-throughput sequencing. Our methodology allowed discriminating polymer type (polyethylene, polypropylene and polystyrene) within small size ranges (0.3-1 vs. 1-2 vs. 2-5 mm) of microplastics collected. Data showed high species richness and diversity on microplastics compared to surrounding seawater samples encompassing both free living and particle attached bacteria. Even though a high proportion of operational taxonomic units (OTU; 94 ± 4%) was shared among all plastic polymers, polystyrene fragments exhibited distinct bacterial assemblages as compared to polyethylene and polypropylene samples. No effect of microplastic size was revealed regardless of polymer type, site and date of collection. The Vibrio genus was commonly detected in the microplastic fraction and specific PCR were performed to determine the presence of potentially pathogenic Vibrio strains (namely V. aestuarianus and the V. splendidus polyphyletic group). V. splendidus related species harboring putative oyster pathogens were detected on most microplastic pools (77%) emphasizing the need of further research to understand the role of microplastics on pathogen population transport and ultimate disease emergence.</p><h1>Supervised machine learning outperforms taxonomy-based environmental DNA metabarcoding applied to biomonitoring</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12926">doi:10.1111/1755-0998.12926</a></p>

<p><a href=None>None</a></p>
<p>Biodiversity monitoring is the standard for environmental impact assessment of anthropogenic activities. Several recent studies showed that high-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) could overcome many limitations of the traditional morphotaxonomy-based bioassessment. Recently, we demonstrated that supervised machine learning (SML) can be used to predict accurate biotic indices values from eDNA metabarcoding data, regardless of the taxonomic affiliation of the sequences. However, it is unknown to which extent the accuracy of such models depends on taxonomic resolution of molecular markers or how SML compares with metabarcoding approaches targeting well-established bioindicator species. In this study, we address these issues by training predictive models upon five different ribosomal bacterial and eukaryotic markers and measuring their performance to assess the environmental impact of marine aquaculture on independent data sets. Our results show that all tested markers are yielding accurate predictive models and that they all outperform the assessment relying solely on taxonomically assigned sequences. Remarkably, we did not find any significant difference in the performance of the models built using universal eukaryotic or prokaryotic markers. Using any molecular marker with a taxonomic range broad enough to comprise different potential bioindicator taxa, SML approach can overcome the limits of taxonomy-based eDNA bioassessment.</p><h1>Needle in a haystack? A comparison of eDNA metabarcoding and targeted qPCR for detection of the great crested newt (<i>Triturus cristatus</i>)</h1>
<p><a href="https://doi.org/10.1002/Ece3.4013">doi:10.1002/Ece3.4013</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis is a rapid, cost-effective, non-invasive biodiversity monitoring tool which utilises DNA left behind in the environment by organisms for species detection. The method is used as a species-specific survey tool for rare or invasive species across a broad range of ecosystems. Recently, eDNA and "metabarcoding" have been combined to describe whole communities rather than focusing on single target species. However, whether metabarcoding is as sensitive as targeted approaches for rare species detection remains to be evaluated. The great crested newt <i>Triturus cristatus</i> is a flagship pond species of international conservation concern and the first UK species to be routinely monitored using eDNA. We evaluate whether eDNA metabarcoding has comparable sensitivity to targeted real-time quantitative PCR (qPCR) for <i>T. cristatus</i> detection. Extracted eDNA samples (<i>N</i> = 532) were screened for <i>T. cristatus</i> by qPCR and analysed for all vertebrate species using high-throughput sequencing technology. With qPCR and a detection threshold of 1 of 12 positive qPCR replicates, newts were detected in 50% of ponds. Detection decreased to 32% when the threshold was increased to 4 of 12 positive qPCR replicates. With metabarcoding, newts were detected in 34% of ponds without a detection threshold, and in 28% of ponds when a threshold (0.028%) was applied. Therefore, qPCR provided greater detection than metabarcoding but metabarcoding detection with no threshold was equivalent to qPCR with a stringent detection threshold. The proportion of <i>T. cristatus</i> sequences in each sample was positively associated with the number of positive qPCR replicates (qPCR score) suggesting eDNA metabarcoding may be indicative of eDNA concentration. eDNA metabarcoding holds enormous potential for holistic biodiversity assessment and routine freshwater monitoring. We advocate this community approach to freshwater monitoring to guide management and conservation, whereby entire communities can be initially surveyed to best inform use of funding and time for species-specific surveys.</p><h1>Testing clustering strategies for metabarcoding-based investigation of community-environment interactions</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12922">doi:10.1111/1755-0998.12922</a></p>

<p><a href=None>None</a></p>
<p>The degradation of freshwater ecosystems has become a common ecological and environmental problem globally. Owing to the complexity of biological communities, there remain tremendous technical challenges for investigating influence of environmental stressors (e.g., chemical pollution) on biological communities. High-throughput sequencing-based metabarcoding provides a powerful tool to reveal complex interactions between environments and biological communities. Among many technical issues, the clustering strategies for operational taxonomic units (OTUs) which are crucial for assessing biodiversity of communities, may affect final conclusions. Here, we used zooplankton communities along an environmental pollution gradient in the Chaobai River in Northern China to test different clustering strategies, including nonclustering and clustering with varied thresholds. Our results showed that though the number of OTUs estimated by nonclustering strategies and clustering strategies with divergence thresholds of 99%-97% largely varied, they were able to identify the same set of significant environmental and spatial variables responsible for geographical distributions of zooplankton communities. In addition, the ecological conclusions obtained by clustering thresholds of 99%-97% were consistent with nonclustering strategies, where for all eight clustering scenarios we detected that species sorting predicted by environmental variables overrode dispersal as the dominant factor in structuring zooplankton communities. However, clustering with the divergence thresholds of <95% affected the environmental and spatial variables identified. We conclude that both newly developed nonclustering methods and traditional clustering methods with divergence thresholds ≥97% were reliable to reveal mechanisms of complex community-environment interactions, although different clustering strategies could lead to largely varied biodiversity estimates such as those for α-diversity.</p><h1>The choice of universal primers and the characteristics of the species mixture determine when DNA metabarcoding can be quantitative</h1>
<p><a href="https://doi.org/10.1111/Mec.14776">doi:10.1111/Mec.14776</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is a technique used to survey biodiversity in many ecological settings, but there are doubts about whether it can provide quantitative results, that is, the proportions of each species in the mixture as opposed to a species list. While there are several experimental studies that report quantitative metabarcoding results, there are a similar number that fail to do so. Here, we provide the rationale to understand under what circumstances the technique can be quantitative. In essence, we simulate a mixture of DNA of S species with a defined initial abundance distribution. In the simulated PCR, each species increases its concentration following a certain amplification efficiency. The final DNA concentration will reflect the initial one when the efficiency is similar for all species; otherwise, the initial and final DNA concentrations would be poorly related. Although there are many known factors that modulate amplification efficiency, we focused on the number of primer-template mismatches, arguably the most important one. We used 15 common primers pairs targeting the mitochondrial COI region and the mitogenomes of ca. 1,200 insect species. The results showed that some primers pairs produced quantitative results under most circumstances, whereas some other primers failed to do so. In conclusion, depending on the primer pair used in the PCR amplification and on the characteristics of the mixture analysed (i.e., high species richness, low evenness), DNA metabarcoding can provide a quantitative estimate of the relative abundances of different species.</p><h1>Optimization and performance testing of a sequence processing pipeline applied to detection of nonindigenous species</h1>
<p><a href="https://doi.org/10.1111/Eva.12604">doi:10.1111/Eva.12604</a></p>

<p><a href=None>None</a></p>
<p>Genetic taxonomic assignment can be more sensitive than morphological taxonomic assignment, particularly for small, cryptic or rare species. Sequence processing is essential to taxonomic assignment, but can also produce errors because optimal parameters are not known a priori. Here, we explored how sequence processing parameters influence taxonomic assignment of 18S sequences from bulk zooplankton samples produced by 454 pyrosequencing. We optimized a sequence processing pipeline for two common research goals, estimation of species richness and early detection of aquatic invasive species (AIS), and then tested most optimal models' performances through simulations. We tested 1,050 parameter sets on 18S sequences from 20 AIS to determine optimal parameters for each research goal. We tested optimized pipelines' performances (detectability and sensitivity) by computationally inoculating sequences of 20 AIS into ten bulk zooplankton samples from ports across Canada. We found that optimal parameter selection generally depends on the research goal. However, regardless of research goal, we found that metazoan 18S sequences produced by 454 pyrosequencing should be trimmed to 375-400 bp and sequence quality filtering should be relaxed (1.5 ≤ maximum expected error ≤ 3.0, Phred score = 10). Clustering and denoising were only viable for estimating species richness, because these processing steps made some species undetectable at low sequence abundances which would not be useful for early detection of AIS. With parameter sets optimized for early detection of AIS, 90% of AIS were detected with fewer than 11 target sequences, regardless of whether clustering or denoising was used. Despite developments in next-generation sequencing, sequence processing remains an important issue owing to difficulties in balancing false-positive and false-negative errors in metabarcoding data.</p><h1>Can non-destructive DNA extraction of bulk invertebrate samples be used for metabarcoding?</h1>
<p><a href="https://doi.org/10.7717/Peerj.4980">doi:10.7717/Peerj.4980</a></p>

<p><a href=None>None</a></p>
<p>High throughput DNA sequencing of bulk invertebrate samples or metabarcoding is becoming increasingly used to provide profiles of biological communities for environmental monitoring. As metabarcoding becomes more widely applied, new reference DNA barcodes linked to individual specimens identified by taxonomists are needed. This can be achieved through using DNA extraction methods that are not only suitable for metabarcoding but also for building reference DNA barcode libraries.</p><h1>Tackling critical parameters in metazoan meta-barcoding experiments: a preliminary study based on <i>coxI</i> DNA barcode</h1>
<p><a href="https://doi.org/10.7717/Peerj.4845">doi:10.7717/Peerj.4845</a></p>

<p><a href=None>None</a></p>
<p>Nowadays DNA meta-barcoding is a powerful instrument capable of quickly discovering the biodiversity of an environmental sample by integrating the DNA barcoding approach with High Throughput Sequencing technologies. It mainly consists of the parallel reading of informative genomic fragment/s able to discriminate living entities. Although this approach has been widely studied, it still needs optimization in some necessary steps requested in its advanced accomplishment. A fundamental element concerns the standardization of bioinformatic analyses pipelines. The aim of the present study was to underline a number of critical parameters of laboratory material preparation and taxonomic assignment pipelines in DNA meta-barcoding experiments using the cytochrome oxidase subunit-I (<i>coxI</i>) barcode region, known as a suitable molecular marker for animal species identification. We compared nine taxonomic assignment pipelines, including a custom in-house method, based on Hidden Markov Models. Moreover, we evaluated the potential influence of universal primers amplification bias in qPCR, as well as the correlation between GC content with taxonomic assignment results. The pipelines were tested on a community of known terrestrial invertebrates collected by pitfall traps from a chestnut forest in Italy. Although the present analysis was not exhaustive and needs additional investigation, our results suggest some potential improvements in laboratory material preparation and the introduction of additional parameters in taxonomic assignment pipelines. These include the correct setup of OTU clustering threshold, the calibration of GC content affecting sequencing quality and taxonomic classification, as well as the evaluation of PCR primers amplification bias on the final biodiversity pattern. Thus, careful attention and further validation/optimization of the above-mentioned variables would be required in a DNA meta-barcoding experimental routine.</p><h1>Metabarcoding analysis on European coastal samples reveals new molecular metazoan diversity</h1>
<p><a href="https://doi.org/10.1038/S41598-018-27509-8">doi:10.1038/S41598-018-27509-8</a></p>

<p><a href=None>None</a></p>
<p>Although animals are among the best studied organisms, we still lack a full description of their diversity, especially for microscopic taxa. This is partly due to the time-consuming and costly nature of surveying animal diversity through morphological and molecular studies of individual taxa. A powerful alternative is the use of high-throughput environmental sequencing, providing molecular data from all organisms sampled. We here address the unknown diversity of animal phyla in marine environments using an extensive dataset designed to assess eukaryotic ribosomal diversity among European coastal locations. A multi-phylum assessment of marine animal diversity that includes water column and sediments, oxic and anoxic environments, and both DNA and RNA templates, revealed a high percentage of novel 18S rRNA sequences in most phyla, suggesting that marine environments have not yet been fully sampled at a molecular level. This novelty is especially high among Platyhelminthes, Acoelomorpha, and Nematoda, which are well studied from a morphological perspective and abundant in benthic environments. We also identified, based on molecular data, a potentially novel group of widespread tunicates. Moreover, we recovered a high number of reads for Ctenophora and Cnidaria in the smaller fractions suggesting their gametes might play a greater ecological role than previously suspected.</p><h1>Advances and challenges in barcoding of microbes, parasites, and their vectors and reservoirs</h1>
<p><a href="https://doi.org/10.1017/S0031182018000884">doi:10.1017/S0031182018000884</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding is now a common tool in parasitology and epidemiology, which require good methods for identification not only of parasites and pathogens but vectors and reservoirs. This special issue presents some advances and challenges in barcoding of microbes, parasites, and their vectors and reservoirs. DNA barcoding found new applications in disease ecology, conservation parasitology, environmental parasitology and in paleoparasitology. New technologies such as next-generation sequencing and matrix-assisted laser desorption-ionization time-of-flight have made it now possible to investigate large samples of specimens. By allowing the investigation of parasites at the interface between environment, biodiversity, animal and human health, barcoding and biobanking have important policy outcomes as well as ethics and legal implications. The special issue 'Advances and challenges in the barcoding of parasites, vectors and reservoirs' illustrates some recent advances and proposes new avenues for research in barcoding in parasitology.</p><h1>Debugging diversity - a pan-continental exploration of the potential of terrestrial blood-feeding leeches as a vertebrate monitoring tool</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12912">doi:10.1111/1755-0998.12912</a></p>

<p><a href=None>None</a></p>
<p>The use of environmental DNA (eDNA) has become an applicable noninvasive tool with which to obtain information about biodiversity. A subdiscipline of eDNA is iDNA (invertebrate-derived DNA), where genetic material ingested by invertebrates is used to characterize the biodiversity of the species that served as hosts. While promising, these techniques are still in their infancy, as they have only been explored on limited numbers of samples from only a single or a few different locations. In this study, we investigate the suitability of iDNA extracted from more than 3,000 haematophagous terrestrial leeches as a tool for detecting a wide range of terrestrial vertebrates across five different geographical regions on three different continents. These regions cover almost the full geographical range of haematophagous terrestrial leeches, thus representing all parts of the world where this method might apply. We identify host taxa through metabarcoding coupled with high-throughput sequencing on Illumina and IonTorrent sequencing platforms to decrease economic costs and workload and thereby make the approach attractive for practitioners in conservation management. We identified hosts in four different taxonomic vertebrate classes: mammals, birds, reptiles and amphibians, belonging to at least 42 different taxonomic families. We find that vertebrate blood ingested by haematophagous terrestrial leeches throughout their distribution is a viable source of DNA with which to examine a wide range of vertebrates. Thus, this study provides encouraging support for the potential of haematophagous terrestrial leeches as a tool for detecting and monitoring terrestrial vertebrate biodiversity.</p><h1>Comparing Sanger sequencing and high-throughput metabarcoding for inferring photobiont diversity in lichens</h1>
<p><a href="https://doi.org/10.1038/S41598-018-26947-8">doi:10.1038/S41598-018-26947-8</a></p>

<p><a href=None>None</a></p>
<p>The implementation of HTS (high-throughput sequencing) approaches is rapidly changing our understanding of the lichen symbiosis, by uncovering high bacterial and fungal diversity, which is often host-specific. Recently, HTS methods revealed the presence of multiple photobionts inside a single thallus in several lichen species. This differs from Sanger technology, which typically yields a single, unambiguous algal sequence per individual. Here we compared HTS and Sanger methods for estimating the diversity of green algal symbionts within lichen thalli using 240 lichen individuals belonging to two species of lichen-forming fungi. According to HTS data, Sanger technology consistently yielded the most abundant photobiont sequence in the sample. However, if the second most abundant photobiont exceeded 30% of the total HTS reads in a sample, Sanger sequencing generally failed. Our results suggest that most lichen individuals in the two analyzed species, Lasallia hispanica and L. pustulata, indeed contain a single, predominant green algal photobiont. We conclude that Sanger sequencing is a valid approach to detect the dominant photobionts in lichen individuals and populations. We discuss which research areas in lichen ecology and evolution will continue to benefit from Sanger sequencing, and which areas will profit from HTS approaches to assessing symbiont diversity.</p><h1>Non-biological synthetic spike-in controls and the AMPtk software pipeline improve mycobiome data</h1>
<p><a href="https://doi.org/10.7717/Peerj.4925">doi:10.7717/Peerj.4925</a></p>

<p><a href=None>None</a></p>
<p>High-throughput amplicon sequencing (HTAS) of conserved DNA regions is a powerful technique to characterize microbial communities. Recently, spike-in mock communities have been used to measure accuracy of sequencing platforms and data analysis pipelines. To assess the ability of sequencing platforms and data processing pipelines using fungal internal transcribed spacer (ITS) amplicons, we created two ITS spike-in control mock communities composed of cloned DNA in plasmids: a biological mock community, consisting of ITS sequences from fungal taxa, and a synthetic mock community (SynMock), consisting of non-biological ITS-like sequences. Using these spike-in controls we show that: (1) a non-biological synthetic control (e.g., SynMock) is the best solution for parameterizing bioinformatics pipelines, (2) pre-clustering steps for variable length amplicons are critically important, (3) a major source of bias is attributed to the initial polymerase chain reaction (PCR) and thus HTAS read abundances are typically not representative of starting values. We developed AMPtk, a versatile software solution equipped to deal with variable length amplicons and quality filter HTAS data based on spike-in controls. While we describe herein a non-biological SynMock community for ITS sequences, the concept and AMPtk software can be widely applied to any HTAS dataset to improve data quality.</p><h1>Counting with DNA in metabarcoding studies: How should we convert sequence reads to dietary data?</h1>
<p><a href="https://doi.org/10.1111/Mec.14734">doi:10.1111/Mec.14734</a></p>

<p><a href=None>None</a></p>
<p>Advances in DNA sequencing technology have revolutionized the field of molecular analysis of trophic interactions, and it is now possible to recover counts of food DNA sequences from a wide range of dietary samples. But what do these counts mean? To obtain an accurate estimate of a consumer's diet should we work strictly with data sets summarizing frequency of occurrence of different food taxa, or is it possible to use relative number of sequences? Both approaches are applied to obtain semi-quantitative diet summaries, but occurrence data are often promoted as a more conservative and reliable option due to taxa-specific biases in recovery of sequences. We explore representative dietary metabarcoding data sets and point out that diet summaries based on occurrence data often overestimate the importance of food consumed in small quantities (potentially including low-level contaminants) and are sensitive to the count threshold used to define an occurrence. Our simulations indicate that using relative read abundance (RRA) information often provides a more accurate view of population-level diet even with moderate recovery biases incorporated; however, RRA summaries are sensitive to recovery biases impacting common diet taxa. Both approaches are more accurate when the mean number of food taxa in samples is small. The ideas presented here highlight the need to consider all sources of bias and to justify the methods used to interpret count data in dietary metabarcoding studies. We encourage researchers to continue addressing methodological challenges and acknowledge unanswered questions to help spur future investigations in this rapidly developing area of research.</p><h1>Development of a qPCR assay to detect and quantify ichthyotoxic flagellates along the Norwegian coast, and the first Norwegian record of Fibrocapsa japonica (Raphidophyceae)</h1>
<p><a href="https://doi.org/S1568-9883(18)30051-9">doi:S1568-9883(18)30051-9</a></p>

<p><a href=None>None</a></p>
<p>Blooms of ichthyotoxic microalgae pose a great challenge to the aquaculture industry world-wide, and there is a need for fast and specific methods for their detection and quantification in monitoring programs. In this study, quantitative real-time PCR (qPCR) assays for the detection and enumeration of three ichthyotoxic flagellates: the dinoflagellate Karenia mikimotoi (Miyake & Kominami ex Oda) Hansen & Moestrup and the two raphidophytes Heterosigma akashiwo (Hada) Hada ex Hara & Chihara and Fibrocapsa japonica Toriumi & Takano were developed. Further, a previously published qPCR assay for the dinoflagellate Karlodinium veneficum (Ballantine) Larsen was used. Monthly samples collected for three years (Aug 2009-Jun 2012) in outer Oslofjorden, Norway were analysed, and the results compared with light microscopy cell counts. The results indicate a higher sensitivity and a lower detection limit (down to 1 cell L<sup>-1</sup>) for both qPCR assays. Qualitative and semi-quantitative results were further compared with those obtained by environmental 454 high throughput sequencing (HTS, metabarcoding) and scanning electron microscopy (SEM) examination from the same samplings. All four species were detected by qPCR and HTS and/or SEM in outer Oslofjorden (Aug 2009-Jun 2012); Karlodinium veneficum was present year-round, whereas Karenia mikimotoi, Heterosigma akashiwo and Fibrocapsa japonica appeared mainly during the autumn in all three years. This is the first observation of Fibrocapsa japonica in Norwegian coastal waters. This species has previously been recorded off the Swedish west coast and German Bight, which may suggest a northward dispersal.</p><h1>The effect of filtration method on the efficiency of environmental DNA capture and quantification via metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12899">doi:10.1111/1755-0998.12899</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is a promising tool for rapid and noninvasive biodiversity monitoring. eDNA density is low in environmental samples, and a capture method, such as filtration, is often required to concentrate eDNA for downstream analyses. In this study, six treatments, with differing filter types and pore sizes for eDNA capture, were compared for their efficiency and accuracy to assess fish community structure with known fish abundance and biomass via eDNA metabarcoding. Our results showed that different filters (with the exception of 20-μm large-pore filters) were broadly consistent in their DNA capture ability. The 0.45-μm filters performed the best in terms of total DNA yield, probability of species detection, repeatability within pond and consistency between ponds. However performance of 0.45-μm filters was only marginally better than for 0.8-μm filters, while filtration time was significantly longer. Given this trade-off, the 0.8-μm filter is the optimal pore size of membrane filter for turbid, eutrophic and high fish density ponds analysed here. The 0.45-μm Sterivex enclosed filters performed reasonably well and are suitable in situations where on-site filtration is required. Finally, prefilters are applied only if absolutely essential for reducing the filtration time or increasing the throughput volume of the capture filters. In summary, we found encouraging similarity in the results obtained from different filtration methods, but the optimal pore size of filter or filter type might strongly depend on the water type under study.</p><h1>Development and implementation of eco-genomic tools for aquatic ecosystem biomonitoring: the SYNAQUA French-Swiss program</h1>
<p><a href="https://doi.org/10.1007/S11356-018-2172-2">doi:10.1007/S11356-018-2172-2</a></p>

<p><a href=None>None</a></p>
<p>The effectiveness of environmental protection measures is based on the early identification and diagnosis of anthropogenic pressures. Similarly, restoration actions require precise monitoring of changes in the ecological quality of ecosystems, in order to highlight their effectiveness. Monitoring the ecological quality relies on bioindicators, which are organisms revealing the pressures exerted on the environment through the composition of their communities. Their implementation, based on the morphological identification of species, is expensive because it requires time and experts in taxonomy. Recent genomic tools should provide access to reliable and high-throughput environmental monitoring by directly inferring the composition of bioindicators' communities from their DNA (metabarcoding). The French-Swiss program SYNAQUA (INTERREG France-Switzerland 2017-2019) proposes to use and validate the tools of environmental genomic for biomonitoring and aims ultimately at their implementation in the regulatory bio-surveillance. SYNAQUA will test the metabarcoding approach focusing on two bioindicators, diatoms, and aquatic oligochaetes, which are used in freshwater biomonitoring in France and Switzerland. To go towards the renewal of current biomonitoring practices, SYNAQUA will (1) bring together different actors: scientists, environmental managers, consulting firms, and biotechnological companies, (2) apply this approach on a large scale to demonstrate its relevance, (3) propose robust and reliable tools, and (4) raise public awareness and train the various actors likely to use these new tools. Biomonitoring approaches based on such environmental genomic tools should address the European need for reliable, higher-throughput monitoring to improve the protection of aquatic environments under multiple pressures, guide their restoration, and follow their evolution.</p><h1>Estimating intraspecific genetic diversity from community DNA metabarcoding data</h1>
<p><a href="https://doi.org/10.7717/Peerj.4644">doi:10.7717/Peerj.4644</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is used to generate species composition data for entire communities. However, sequencing errors in high-throughput sequencing instruments are fairly common, usually requiring reads to be clustered into operational taxonomic units (OTUs), losing information on intraspecific diversity in the process. While Cytochrome c oxidase subunit I (COI) haplotype information is limited in resolving intraspecific diversity it is nevertheless often useful e.g. in a phylogeographic context, helping to formulate hypotheses on taxon distribution and dispersal.</p><h1>Mycobiome of Cysts of the Soybean Cyst Nematode Under Long Term Crop Rotation</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2018.00386">doi:10.3389/Fmicb.2018.00386</a></p>

<p><a href=None>None</a></p>
<p>The soybean cyst nematode (SCN), <i>Heterodera glycines</i> Ichinohe (Phylum Nematoda), is a major pathogen of soybean. It causes substantial yield losses worldwide and is difficult to control because the cyst protects the eggs which can remain viable for nearly a decade. Crop rotation with non-host crops and use of biocontrol organisms such as fungi and bacteria offer promising approaches, but remain hampered by lack of knowledge of the biology of nematode parasitic organisms. We used a high-throughput metabarcoding approach to characterize fungal communities associated with the SCN cyst, a microenvironment in soil that may harbor both nematode parasites and plant pathogens. SCN cysts were collected from a long-term crop rotation experiment in Southeastern Minnesota at three time points over two growing seasons to characterize diversity of fungi inhabiting cysts and to examine how crop rotation and seasonal variation affects fungal communities. A majority of fungi in cysts belonged to Ascomycota and Basidiomycota, but the presence of several early diverging fungal subphyla thought to be primarily plant and litter associated, including Mortierellomycotina and Glomeromycotina (e.g., arbuscular mycorrhizal fungi), suggests a possible role as nematode egg parasites. Species richness varied by both crop rotation and season and was higher in early years of crop rotation and in fall at the end of the growing season. Crop rotation and season also impacted fungal community composition and identified several classes of fungi, including Eurotiomycetes, Sordariomycetes, and Orbiliomycetes (e.g., nematode trapping fungi), with higher relative abundance in early soybean rotations. The relative abundance of several genera was correlated with increasing years of soybean. Fungal communities also varied by season and were most divergent at midseason. The percentage of OTUs assigned to Mortierellomycotina_cls_Incertae_sedis and Sordariomycetes increased at midseason, while Orbiliomycetes decreased at midseason, and Glomeromycetes increased in fall. Ecological guilds of fungi containing an animal-pathogen lifestyle, as well as potential egg-parasitic taxa previously isolated from parasitized SCN eggs, increased at midseason. The animal pathogen guilds included known (e.g., <i>Pochonia chlamydosporia</i>) and new candidate biocontrol organisms. This research advances knowledge of the ecology of nematophagous fungi in agroecosystems and their use as biocontrol agents of the SCN.</p><h1>High-throughput terrestrial biodiversity assessments: mitochondrial metabarcoding, metagenomics or metatranscriptomics?</h1>
<p><a href="https://doi.org/10.1080/24701394.2018.1455189">doi:10.1080/24701394.2018.1455189</a></p>

<p><a href=None>None</a></p>
<p>Consensus on the optimal high-throughput sequencing (HTS) approach to examine biodiversity in mixed terrestrial arthropod samples has not been reached. Metatranscriptomics could increase the proportion of taxonomically informative mitochondrial reads in HTS outputs but has not been investigated for terrestrial arthropod samples. We compared the efficiency of 16S rRNA metabarcoding, metagenomics and metatranscriptomics for detecting species in a mixed terrestrial arthropod sample (pooled DNA/RNA from 38 taxa). 16S rRNA metabarcoding and nuclear rRNA-depleted metatranscriptomics had the highest detection rate with 97% of input species detected. Based on cytochrome c oxidase I, metagenomics had the highest detection rate with 82% of input species detected, but metatranscriptomics produced a larger proportion of reads matching (Sanger) reference sequences. Metatranscriptomics with nuclear rRNA depletion may offer advantages over metabarcoding through reducing the number of spurious operational taxonomic units while retaining high detection rates, and offers natural enrichment of mitochondrial sequences which may enable increased species detection rates compared with metagenomics.</p><h1>DNA metabarcoding uncovers fungal diversity of mixed airborne samples in Italy</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0194489">doi:10.1371/Journal.Pone.0194489</a></p>

<p><a href=None>None</a></p>
<p>Fungal spores and mycelium fragments are particles which become and remain airborne and have been subjects of aerobiological studies. The presence and the abundance of taxa in aerobiological samples can be very variable and impaired by changeable climatic conditions. Because many fungi produce mycotoxins and both their mycelium fragments and spores are potential allergens, monitoring the presence of these taxa is of key importance. So far data on exposure and sensitization to fungal allergens are mainly based on the assessment of few, easily identifiable taxa and focused only on certain environments. The microscopic method used to analyze aerobiological samples and the inconspicuous fungal characters do not allow a in depth taxonomical identification. Here, we present a first assessment of fungal diversity from airborne samples using a DNA metabarcoding analysis. The nuclear ITS2 region was selected as barcode to catch fungal diversity in mixed airborne samples gathered during two weeks in four sites of North-Eastern and Central Italy. We assessed the taxonomic composition and diversity within and among the sampled sites and compared the molecular data with those obtained by traditional microscopy. The molecular analyses provide a tenfold more comprehensive determination of the taxa than the traditional morphological inspections. Our results prove that the metabarcoding analysis is a promising approach to increases quality and sensitivity of the aerobiological monitoring. The laboratory and bioinformatic workflow implemented here is now suitable for routine, high-throughput, regional analyses of airborne fungi.</p><h1>Automated high throughput animal CO1 metabarcode classification</h1>
<p><a href="https://doi.org/10.1038/S41598-018-22505-4">doi:10.1038/S41598-018-22505-4</a></p>

<p><a href=None>None</a></p>
<p>We introduce a method for assigning names to CO1 metabarcode sequences with confidence scores in a rapid, high-throughput manner. We compiled nearly 1 million CO1 barcode sequences appropriate for classifying arthropods and chordates. Compared to our previous Insecta classifier, the current classifier has more than three times the taxonomic coverage, including outgroups, and is based on almost five times as many reference sequences. Unlike other popular rDNA metabarcoding markers, we show that classification performance is similar across the length of the CO1 barcoding region. We show that the RDP classifier can make taxonomic assignments about 19 times faster than the popular top BLAST hit method and reduce the false positive rate from nearly 100% to 34%. This is especially important in large-scale biodiversity and biomonitoring studies where datasets can become very large and the taxonomic assignment problem is not trivial. We also show that reference databases are becoming more representative of current species diversity but that gaps still exist. We suggest that it would benefit the field as a whole if all investigators involved in metabarocoding studies, through collaborations with taxonomic experts, also planned to barcode representatives of their local biota as a part of their projects.</p><h1>Detailed ecological associations of triatomines revealed by metabarcoding and next-generation sequencing: implications for triatomine behavior and Trypanosoma cruzi transmission cycles</h1>
<p><a href="https://doi.org/10.1038/S41598-018-22455-X">doi:10.1038/S41598-018-22455-X</a></p>

<p><a href=None>None</a></p>
<p>Trypanosoma cruzi is the agent of Chagas disease, transmitted by hematophagous triatomine vectors. Establishing transmission cycles is key to understand the epidemiology of the disease, but integrative assessments of ecological interactions shaping parasite transmission are still limited. Current approaches also lack sensitivity to assess the full extent of this ecological diversity. Here we developed a metabarcoding approach based on next-generation sequencing to identify triatomine gut microbiome, vertebrate feeding hosts, and parasite diversity and their potential interactions. We detected a dynamic microbiome in Triatoma dimidiata, including 23 bacterial orders, which differed according to blood sources. Fourteen vertebrate species served as blood sources, corresponding to domestic, synantropic and sylvatic species, although four (human, dog, cow and mice) accounted for over 50% of blood sources. Importantly, bugs fed on multiple hosts, with up to 11 hosts identified per bug, indicating very frequent host-switching. A high clonal diversity of T. cruzi was detected, with up to 20 haplotypes per bug. This analysis provided much greater sensitivity to detect multiple blood meals and multiclonal infections with T. cruzi, which should be taken into account to develop transmission networks, and characterize the risk for human infection, eventually leading to a better control of disease transmission.</p><h1>Omics Advances in Ecotoxicology</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.7B06494">doi:10.1021/Acs.Est.7B06494</a></p>

<p><a href=None>None</a></p>
<p>Toxic substances in the environment generate adverse effects at all levels of biological organization from the molecular level to community and ecosystem. Given this complexity, it is not surprising that ecotoxicologists have struggled to address the full consequences of toxic substance release at ecosystem level, due to the limits of observational and experimental tools to reveal the changes in deep structure at different levels of organization. -Omics technologies, consisting of genomics and ecogenomics, have the power to reveal, in unprecedented detail, the cellular processes of an individual or biodiversity of a community in response to environmental change with high sample/observation throughput. This represents a historic opportunity to transform the way we study toxic substances in ecosystems, through direct linkage of ecological effects with the systems biology of organisms. Three recent examples of -omics advance in the assessment of toxic substances are explored here: (1) the use of functional genomics in the discovery of novel molecular mechanisms of toxicity of chemicals in the environment; (2) the development of laboratory pipelines of dose-dependent, reduced transcriptomics to support high-throughput chemical testing at the biological pathway level; and (3) the use of eDNA metabarcoding approaches for assessing chemical effects on biological communities in mesocosm experiments and through direct observation in field monitoring. -Omics advances in ecotoxicological studies not only generate new knowledge regarding mechanisms of toxicity and environmental effect, improving the relevance and immediacy of laboratory toxicological assessment, but can provide a wholly new paradigm for ecotoxicology by linking ecological models to mechanism-based, systems biology approaches.</p><h1>Assessing Performance of Spore Samplers in Monitoring Aeromycobiota and Fungal Plant Pathogen Diversity in Canada</h1>
<p><a href="https://doi.org/E02601-17">doi:E02601-17</a></p>

<p><a href=None>None</a></p>
<p>Spore samplers are widely used in pathogen surveillance but not so much for monitoring the composition of aeromycobiota. In Canada, a nationwide spore-sampling network (AeroNet) was established as a pilot project to assess fungal community composition in air and rain samples collected using three different spore samplers in the summers of 2010 and 2011. Metabarcodes of the internal transcribed spacer (ITS) were exhaustively characterized for three of the network sites, in British Columbia (BC), Québec (QC), and Prince Edward Island (PEI), to compare performance of the samplers. Sampler type accounted for ca. 20% of the total explainable variance in aeromycobiota compositional heterogeneity, with air samplers recovering more Ascomycota and rain samplers recovering more Basidiomycota. Spore samplers showed different abilities to collect 27 fungal genera that are plant pathogens. For instance, <i>Cladosporium</i> spp., <i>Drechslera</i> spp., and <i>Entyloma</i> spp. were collected mainly by air samplers, while <i>Fusarium</i> spp., <i>Microdochium</i> spp., and <i>Ustilago</i> spp. were recovered more frequently with rain samplers. The diversity and abundance of some fungi were significantly affected by sampling location and time (e.g., <i>Alternaria</i> and <i>Bipolaris</i>) and weather conditions (e.g., <i>Mycocentrospora</i> and <i>Leptosphaeria</i>), and depended on using ITS1 or ITS2 as the barcoding region (e.g., <i>Epicoccum</i> and <i>Botrytis</i>). The observation that Canada's aeromycobiota diversity correlates with cooler, wetter conditions and northward wind requires support from more long-term data sets. Our vision of the AeroNet network, combined with high-throughput sequencing (HTS) and well-designed sampling strategies, may contribute significantly to a national biovigilance network for protecting plants of agricultural and economic importance in Canada.<b>IMPORTANCE</b> The current study compared the performance of spore samplers for collecting broad-spectrum air- and rain-borne fungal pathogens using a metabarcoding approach. The results provided a thorough characterization of the aeromycobiota in the coastal regions of Canada in relation to the influence of climatic factors. This study lays the methodological basis to eventually develop knowledge-based guidance on pest surveillance by assisting in the selection of appropriate spore samplers.</p><h1>A cross-taxa study using environmental DNA/RNA metabarcoding to measure biological impacts of offshore oil and gas drilling and production operations</h1>
<p><a href="https://doi.org/S0025-326X(17)30999-2">doi:S0025-326X(17)30999-2</a></p>

<p><a href=None>None</a></p>
<p>Standardized ecosystem-based monitoring surveys are critical for providing information on marine ecosystem health. Environmental DNA/RNA (eDNA/eRNA) metabarcoding may facilitate such surveys by quickly and effectively characterizing multi-trophic levels. In this study, we assessed the suitability of eDNA/eRNA metabarcoding to evaluate changes in benthic assemblages of bacteria, Foraminifera and other eukaryotes along transects at three offshore oil and gas (O&G) drilling and production sites, and compared these to morphologically characterized macro-faunal assemblages. Bacterial communities were the most responsive to O&G activities, followed by Foraminifera, and macro-fauna (the latter assessed by morphology). The molecular approach enabled detection of hydrocarbon degrading taxa such as the bacteria Alcanivorax and Microbulbifer at petroleum impacted stations. Most identified indicator taxa, notably among macro-fauna, were highly specific to site conditions. Based on our results we suggest that eDNA/eRNA metabarcoding can be used as a stand-alone method for biodiversity assessment or as a complement to morphology-based monitoring approaches.</p><h1>Environmental DNA metabarcoding of benthic bacterial communities indicates the benthic footprint of salmon aquaculture</h1>
<p><a href="https://doi.org/S0025-326X(17)31022-6">doi:S0025-326X(17)31022-6</a></p>

<p><a href=None>None</a></p>
<p>We evaluated benthic bacterial communities as bioindicators in environmental impact assessments of salmon aquaculture, a rapidly growing sector of seafood industry. Sediment samples (n=72) were collected from below salmon cages towards distant reference sites. Bacterial community profiles inferred from DNA metabarcodes were compared to reference data from standard macrofauna biomonitoring surveys of the same samples. Deltaproteobacteria were predominant in immediate vicinity of the salmon cages. Along the transect, significant shifts in bacterial community structures were observed with Gammaproteobacteria dominating the less-impacted sites. Alpha- and beta-diversity measures of bacterial communities correlated significantly with macrofauna diversity metrics and with five ecological status indices. Benthic bacterial communities mirror the reaction of macrofauna bioindicators to environmental disturbances caused by salmon farming. The implementation of bacterial eDNA metabarcoding in future Strategic Framework Directives is an alternative cost-effective high-throughput biomonitoring solution, providing a basis for management strategies in a matter of days rather than months.</p><h1>Surface ocean metabarcoding confirms limited diversity in planktonic foraminifera but reveals unknown hyper-abundant lineages</h1>
<p><a href="https://doi.org/10.1038/S41598-018-20833-Z">doi:10.1038/S41598-018-20833-Z</a></p>

<p><a href=None>None</a></p>
<p>Since the advent of DNA metabarcoding surveys, the planktonic realm is considered a treasure trove of diversity, inhabited by a small number of abundant taxa, and a hugely diverse and taxonomically uncharacterized consortium of rare species. Here we assess if the apparent underestimation of plankton diversity applies universally. We target planktonic foraminifera, a group of protists whose known morphological diversity is limited, taxonomically resolved and linked to ribosomal DNA barcodes. We generated a pyrosequencing dataset of ~100,000 partial 18S rRNA foraminiferal sequences from 32 size fractioned photic-zone plankton samples collected at 8 stations in the Indian and Atlantic Oceans during the Tara Oceans expedition (2009-2012). We identified 69 genetic types belonging to 41 morphotaxa in our metabarcoding dataset. The diversity saturated at local and regional scale as well as in the three size fractions and the two depths sampled indicating that the diversity of foraminifera is modest and finite. The large majority of the newly discovered lineages occur in the small size fraction, neglected by classical taxonomy. These unknown lineages dominate the bulk [>0.8 µm] size fraction, implying that a considerable part of the planktonic foraminifera community biomass has its origin in unknown lineages.</p><h1>Effects of predation stress and food ration on perch gut microbiota</h1>
<p><a href="https://doi.org/10.1186/S40168-018-0400-0">doi:10.1186/S40168-018-0400-0</a></p>

<p><a href=None>None</a></p>
<p>Gut microbiota provide functions of importance to influence hosts' food digestion, metabolism, and protection against pathogens. Factors that affect the composition and functions of gut microbial communities are well studied in humans and other animals; however, we have limited knowledge of how natural food web factors such as stress from predators and food resource rations could affect hosts' gut microbiota and how it interacts with host sex. In this study, we designed a two-factorial experiment exposing perch (Perca fluviatilis) to a predator (pike, Esox lucius), and different food ratios, to examine the compositional and functional changes of perch gut microbiota based on 16S rRNA amplicon sequencing. We also investigated if those changes are host sex dependent.</p><h1>Species delimitation for the molecular taxonomy and ecology of the widely distributed microbial eukaryote genus <i>Euplotes</i> (Alveolata, Ciliophora)</h1>
<p><a href="https://doi.org/20172159">doi:20172159</a></p>

<p><a href=None>None</a></p>
<p>Recent advances in high-throughput sequencing and metabarcoding technologies are revolutionizing our understanding of the diversity and ecology of microbial eukaryotes (protists). The interpretation of protist diversity and the elucidation of their ecosystem function are, however, impeded by problems with species delimitation, especially as it applies to molecular taxonomy. Here, using the ciliate <i>Euplotes</i> as an example, we describe approaches for species delimitation based on integrative taxonomy by using evolutionary and ecological perspectives and selecting the most appropriate metabarcoding gene markers as proxies for species units. Our analyses show that: <i>Euplotes</i> (<i>sensu lato</i>) comprises six distinct clades, mainly as result of ecological speciation; the validity of the genera <i>Euplotes</i> (<i>sensu stricto</i>), <i>Euplotoides</i>, <i>Euplotopsis</i> and <i>Moneuplotes</i> are not supported; the <i>vannus</i>-type group, which includes species without distinct morphological differences, seems to be undergoing incipient speciation and contains cryptic species; the hypervariable V4 region of the small subunit rDNA and D1-D2 region of the large subunit rDNA are the promising candidates for general species delimitation in <i>Euplotes</i>.</p><h1>Sorting specimen-rich invertebrate samples with cost-effective NGS barcodes: Validating a reverse workflow for specimen processing</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12751">doi:10.1111/1755-0998.12751</a></p>

<p><a href=None>None</a></p>
<p>Biologists frequently sort specimen-rich samples to species. This process is daunting when based on morphology, and disadvantageous if performed using molecular methods that destroy vouchers (e.g., metabarcoding). An alternative is barcoding every specimen in a bulk sample and then presorting the specimens using DNA barcodes, thus mitigating downstream morphological work on presorted units. Such a "reverse workflow" is too expensive using Sanger sequencing, but we here demonstrate that is feasible with an next-generation sequencing (NGS) barcoding pipeline that allows for cost-effective high-throughput generation of short specimen-specific barcodes (313 bp of COI; laboratory cost <$0.50 per specimen) through next-generation sequencing of tagged amplicons. We applied our approach to a large sample of tropical ants, obtaining barcodes for 3,290 of 4,032 specimens (82%). NGS barcodes and their corresponding specimens were then sorted into molecular operational taxonomic units (mOTUs) based on objective clustering and Automated Barcode Gap Discovery (ABGD). High diversity of 88-90 mOTUs (4% clustering) was found and morphologically validated based on preserved vouchers. The mOTUs were overwhelmingly in agreement with morphospecies (match ratio 0.95 at 4% clustering). Because of lack of coverage in existing barcode databases, only 18 could be accurately identified to named species, but our study yielded new barcodes for 48 species, including 28 that are potentially new to science. With its low cost and technical simplicity, the NGS barcoding pipeline can be implemented by a large range of laboratories. It accelerates invertebrate species discovery, facilitates downstream taxonomic work, helps with building comprehensive barcode databases and yields precise abundance information.</p><h1>Scaling up: A guide to high-throughput genomic approaches for biodiversity analysis</h1>
<p><a href="https://doi.org/10.1111/Mec.14478">doi:10.1111/Mec.14478</a></p>

<p><a href=None>None</a></p>
<p>The purpose of this review is to present the most common and emerging DNA-based methods used to generate data for biodiversity and biomonitoring studies. As environmental assessment and monitoring programmes may require biodiversity information at multiple levels, we pay particular attention to the DNA metabarcoding method and discuss a number of bioinformatic tools and considerations for producing DNA-based indicators using operational taxonomic units (OTUs), taxa at a variety of ranks and community composition. By developing the capacity to harness the advantages provided by the newest technologies, investigators can "scale up" by increasing the number of samples and replicates processed, the frequency of sampling over time and space, and even the depth of sampling such as by sequencing more reads per sample or more markers per sample. The ability to scale up is made possible by the reduced hands-on time and cost per sample provided by the newest kits, platforms and software tools. Results gleaned from broad-scale monitoring will provide opportunities to address key scientific questions linked to biodiversity and its dynamics across time and space as well as being more relevant for policymakers, enabling science-based decision-making, and provide a greater socio-economic impact. As genomic approaches are continually evolving, we provide this guide to methods used in biodiversity genomics.</p><h1>Quantitative species-level ecology of reef fish larvae via metabarcoding</h1>
<p><a href="https://doi.org/10.1038/S41559-017-0413-2">doi:10.1038/S41559-017-0413-2</a></p>

<p><a href=None>None</a></p>
<p>The larval pool of coral reef fish has a crucial role in the dynamics of adult fish populations. However, large-scale species-level monitoring of species-rich larval pools has been technically impractical. Here, we use high-throughput metabarcoding to study larval ecology in the Gulf of Aqaba, a region that is inhabited by >500 reef fish species. We analysed 9,933 larvae from 383 samples that were stratified over sites, depth and time. Metagenomic DNA extracted from pooled larvae was matched to a mitochondrial cytochrome c oxidase subunit I barcode database compiled for 77% of known fish species within this region. This yielded species-level reconstruction of the larval community, allowing robust estimation of larval spatio-temporal distributions. We found significant correlations between species abundance in the larval pool and in local adult assemblages, suggesting a major role for larval supply in determining local adult densities. We documented larval flux of species whose adults were never documented in the region, suggesting environmental filtering as the reason for the absence of these species. Larvae of several deep-sea fishes were found in shallow waters, supporting their dispersal over shallow bathymetries, potentially allowing Lessepsian migration into the Mediterranean Sea. Our method is applicable to any larval community and could assist coral reef conservation and fishery management efforts.</p><h1>Performance of DNA metabarcoding, standard barcoding, and morphological approach in the identification of host-parasitoid interactions</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0187803">doi:10.1371/Journal.Pone.0187803</a></p>

<p><a href=None>None</a></p>
<p>Understanding interactions between herbivores and parasitoids is essential for successful biodiversity protection and monitoring and for biological pest control. Morphological identifications employ insect rearing and are complicated by insects' high diversity and crypsis. DNA barcoding has been successfully used in studies of host-parasitoid interactions as it can substantially increase the recovered real host-parasitoid diversity distorted by overlooked species complexes, or by species with slight morphological differences. However, this approach does not allow the simultaneous detection and identification of host(s) and parasitoid(s). Recently, high-throughput sequencing has shown high potential for surveying ecological communities and trophic interactions. Using mock samples comprising insect larvae and their parasitoids, we tested the potential of DNA metabarcoding for identifying individuals involved in host-parasitoid interactions to different taxonomic levels, and compared it to standard DNA barcoding and morphological approaches. For DNA metabarcoding, we targeted the standard barcoding marker cytochrome oxidase subunit I using highly degenerate primers, 2*300 bp sequencing on a MiSeq platform, and RTAX classification using paired-end reads. Additionally, using a large host-parasitoid dataset from a Central European floodplain forest, we assess the completeness and usability of a local reference library by confronting the number of Barcoding Index Numbers obtained by standard barcoding with the number of morphotypes. Overall, metabarcoding recovery was high, identifying 92.8% of the taxa present in mock samples, and identification success within individual taxonomic levels did not significantly differ among metabarcoding, standard barcoding, and morphology. Based on the current local reference library, 39.4% parasitoid and 90.7% host taxa were identified to the species level. DNA barcoding estimated higher parasitoid diversity than morphotyping, especially in groups with high level of crypsis. This study suggests the potential of metabarcoding for effectively recovering host-parasitoid diversity, together with more accurate identifications obtained from building reliable and comprehensive reference libraries, especially for parasitoids.</p><h1>Metagenomic sequencing of environmental DNA reveals marine faunal assemblages from the West Antarctic Peninsula</h1>
<p><a href="https://doi.org/S1874-7787(17)30258-1">doi:S1874-7787(17)30258-1</a></p>

<p><a href=None>None</a></p>
<p>The West Antarctic Peninsula (WAP) is the fastest warming region in Antarctica where climate impact on the cold-adapted marine ecosystem is already visible. To monitor faunal changes in remote vast bodies of Antarctic waters, efficient and informative tools are essential. High-throughput sequencing of environmental DNA (eDNA) has emerged as one such tool for monitoring biodiversity and ecosystems, as it increases detection sensitivity of taxa, and sampling is often simpler and less costly than traditional collection methods. We collected water samples from four WAP shallow (≤300m) shelf regions, recovered the eDNA therein, and performed metagenomic shotgun sequencing and analyses to determine the effectiveness of this method to assess marine benthic faunal diversity; this includes the detection of deep-water predatory king crabs whose potential shoreward expansion to warming shelves has sparked much concern. Using a customized bioinformatics pipeline, we identified abundant signatures of common benthic invertebrate fauna, endemic notothenioid fishes, as well as lithodid king crabs. We also uncovered species richness and diversity comparable to biological inventories compiled by the use of traditional survey methods, supporting the efficacy of the eDNA shotgun sequencing approach. As the rate of eDNA degradation affects faunal detection sensitivity, we also quantified mitochondrial ND2 gene copies in eDNA derived from a WAP icefish and found ND2 copies persisted to at least 20days in the cold WAP water, much longer than values reported for temperate environments. We propose that eDNA metagenomic sequencing complements traditional sampling, and combining both will enable more inclusive biodiversity detection and faunal change monitoring in the vast Southern Ocean.</p><h1>Genetic monitoring of open ocean biodiversity: An evaluation of DNA metabarcoding for processing continuous plankton recorder samples</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12740">doi:10.1111/1755-0998.12740</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is an efficient method for measuring biodiversity, but the process of initiating long-term DNA-based monitoring programmes, or integrating with conventional programs, is only starting. In marine ecosystems, plankton surveys using the continuous plankton recorder (CPR) have characterized biodiversity along transects covering millions of kilometres with time-series spanning decades. We investigated the potential for use of metabarcoding in CPR surveys. Samples (n = 53) were collected in two Southern Ocean transects and metazoans identified using standard microscopic methods and by high-throughput sequencing of a cytochrome c oxidase subunit I marker. DNA increased the number of metazoan species identified and provided high-resolution taxonomy of groups problematic in conventional surveys (e.g., larval echinoderms and hydrozoans). Metabarcoding also generally produced more detections than microscopy, but this sensitivity may make cross-contamination during sampling a problem. In some samples, the prevalence of DNA from large plankton such as krill masked the presence of smaller species. We investigated adding a fixed amount of exogenous DNA to samples as an internal control to allow determination of relative plankton biomass. Overall, the metabarcoding data represent a substantial shift in perspective, making direct integration into current long-term time-series challenging. We discuss a number of hurdles that exist for progressing DNA metabarcoding from the current snapshot studies to the requirements of a long-term monitoring programme. Given the power and continually increasing efficiency of metabarcoding, it is almost certain this approach will play an important role in future plankton monitoring.</p><h1>Fungal Community Structure and As-Resistant Fungi in a Decommissioned Gold Mine Site</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2017.02202">doi:10.3389/Fmicb.2017.02202</a></p>

<p><a href=None>None</a></p>
<p>Although large quantities of heavy metal laden wastes are released in an uncontrolled manner by gold mining activities with ensuing contamination of the surrounding areas, there is scant information on the mycobiota of gold-mine sites. Thus, the present study was aimed to describe the fungal community structure in three differently As- and Hg-polluted soils collected from the Pestarena decommissioned site by using Illumina® metabarcoding. Fungal richness was found to increase as the contamination level increased while biodiversity was not related to the concentrations of inorganic toxicants. Within the phylum Zygomigota which, irrespective of the contamination level, was predominant in all the soils under study, the most abundant genera were <i>Mucor</i> and <i>Mortierella</i>. The relative abundances of Basidiomycota, instead, tended to raise as the contamination increased; within this phylum the most abundant genera were <i>Cryptococcus</i> and <i>Pseudotomentella</i>. The abundance of Ascomycota, ranging from about 8 to 21%, was not related to the contamination level. The relative abundances of those genera (i.e., <i>Penicillium, Trichoderma</i>, and <i>Chaetomium</i>), the cultivable isolates of which exhibited significant As-resistance, were lower than the set threshold (0.5%). Mass balances obtained from As-exposure experiments with these isolates showed that the main mechanisms involved in counteracting the toxicant were accumulation and, above all, volatilization, the respective extents of which ranged from 0.6 to 5.9% and from 6.4 to 31.2% in dependence of the isolate.</p><h1>FuzzyID2: A software package for large data set species identification via barcoding and metabarcoding using hidden Markov models and fuzzy set methods</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12738">doi:10.1111/1755-0998.12738</a></p>

<p><a href=None>None</a></p>
<p>Species identification through DNA barcoding or metabarcoding has become a key approach for biodiversity evaluation and ecological studies. However, the rapid accumulation of barcoding data has created some difficulties: for instance, global enquiries to a large reference library can take a very long time. We here devise a two-step searching strategy to speed identification procedures of such queries. This firstly uses a Hidden Markov Model (HMM) algorithm to narrow the searching scope to genus level and then determines the corresponding species using minimum genetic distance. Moreover, using a fuzzy membership function, our approach also estimates the credibility of assignment results for each query. To perform this task, we developed a new software pipeline, FuzzyID2, using Python and C++. Performance of the new method was assessed using eight empirical data sets ranging from 70 to 234,535 barcodes. Five data sets (four animal, one plant) deployed the conventional barcode approach, one used metabarcodes, and two were eDNA-based. The results showed mean accuracies of generic and species identification of 98.60% (with a minimum of 95.00% and a maximum of 100.00%) and 94.17% (with a range of 84.40%-100.00%), respectively. Tests with simulated NGS sequences based on realistic eDNA and metabarcode data demonstrated that FuzzyID2 achieved a significantly higher identification success rate than the commonly used Blast method, and the TIPP method tends to find many fewer species than either FuzztID2 or Blast. Furthermore, data sets with tens of thousands of barcodes need only a few seconds for each query assignment using FuzzyID2. Our approach provides an efficient and accurate species identification protocol for biodiversity-related projects with large DNA sequence data sets.</p><h1>DNA metabarcoding for high-throughput monitoring of estuarine macrobenthic communities</h1>
<p><a href="https://doi.org/10.1038/S41598-017-15823-6">doi:10.1038/S41598-017-15823-6</a></p>

<p><a href=None>None</a></p>
<p>Morphology-based profiling of benthic communities has been extensively applied to aquatic ecosystems' health assessment. However, it remains a low-throughput, and sometimes ambiguous, procedure. Despite DNA metabarcoding has been applied to marine benthos, a comprehensive approach providing species-level identifications for estuarine macrobenthos is still lacking. Here we report a combination of experimental and field studies to assess the aptitude of COI metabarcoding to provide robust species-level identifications for high-throughput monitoring of estuarine macrobenthos. To investigate the ability of metabarcoding to detect all species present in bulk DNA extracts, we contrived three phylogenetically diverse communities, and applied four different primer pairs to generate PCR products within the COI barcode region. Between 78-83% of the species in the contrived communities were recovered through HTS. Subsequently, we compared morphology and metabarcoding-based approaches to determine the species composition from four distinct estuarine sites. Our results indicate that species richness would be considerably underestimated if only morphological methods were used: globally 27 species identified through morphology versus 61 detected by metabarcoding. Although further refinement is required to improve efficiency and output of this approach, here we show the great aptitude of COI metabarcoding to provide high quality and auditable species identifications in estuarine macrobenthos monitoring.</p><h1>International Barcode of Life: Focus on big biodiversity in South Africa</h1>
<p><a href="https://doi.org/10.1139/Gen-2017-0210">doi:10.1139/Gen-2017-0210</a></p>

<p><a href=None>None</a></p>
<p>Participants in the 7th International Barcode of Life Conference (Kruger National Park, South Africa, 20-24 November 2017) share the latest findings in DNA barcoding research and its increasingly diversified applications. Here, we review prevailing trends synthesized from among 429 invited and contributed abstracts, which are collated in this open-access special issue of Genome. Hosted for the first time on the African continent, the 7th Conference places special emphasis on the evolutionary origins, biogeography, and conservation of African flora and fauna. Within Africa and elsewhere, DNA barcoding and related techniques are being increasingly used for wildlife forensics and for the validation of commercial products, such as medicinal plants and seafood species. A striking trend of the conference is the dramatic rise of studies on environmental DNA (eDNA) and on diverse uses of high-throughput sequencing techniques. Emerging techniques in these areas are opening new avenues for environmental biomonitoring, managing species-at-risk and invasive species, and revealing species interaction networks in unprecedented detail. Contributors call for the development of validated community standards for high-throughput sequence data generation and analysis, to enable the full potential of these methods to be realized for understanding and managing biodiversity on a global scale.</p><h1>Metabarcoding of freshwater invertebrates to detect the effects of a pesticide spill</h1>
<p><a href="https://doi.org/10.1111/Mec.14410">doi:10.1111/Mec.14410</a></p>

<p><a href=None>None</a></p>
<p>Biomonitoring underpins the environmental assessment of freshwater ecosystems and guides management and conservation. Current methodology for surveys of (macro)invertebrates uses coarse taxonomic identification where species-level resolution is difficult to obtain. Next-generation sequencing of entire assemblages (metabarcoding) provides a new approach for species detection, but requires further validation. We used metabarcoding of invertebrate assemblages with two fragments of the cox1 "barcode" and partial nuclear ribosomal (SSU) genes, to assess the effects of a pesticide spill in the River Kennet (southern England). Operational taxonomic unit (OTU) recovery was tested under 72 parameters (read denoising, filtering, pair merging and clustering). Similar taxonomic profiles were obtained under a broad range of parameters. The SSU marker recovered Platyhelminthes and Nematoda, missed by cox1, while Rotifera were only amplified with cox1. A reference set was created from all available barcode entries for Arthropoda in the BOLD database and clustered into OTUs. The River Kennet metabarcoding produced matches to 207 of these reference OTUs, five times the number of species recognized with morphological monitoring. The increase was due to the following: greater taxonomic resolution (e.g., splitting a single morphotaxon "Chironomidae" into 55 named OTUs); splitting of Linnaean binomials into multiple molecular OTUs; and the use of a filtration-flotation protocol for extraction of minute specimens (meiofauna). Community analyses revealed strong differences between "impacted" vs. "control" samples, detectable with each gene marker, for each major taxonomic group, and for meio- and macrofaunal samples separately. Thus, highly resolved taxonomic data can be extracted at a fraction of the time and cost of traditional nonmolecular methods, opening new avenues for freshwater invertebrate biodiversity monitoring and molecular ecology.</p><h1>Algorithm for post-clustering curation of DNA amplicon data yields reliable biodiversity estimates</h1>
<p><a href="https://doi.org/10.1038/S41467-017-01312-X">doi:10.1038/S41467-017-01312-X</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is promising for cost-effective biodiversity monitoring, but reliable diversity estimates are difficult to achieve and validate. Here we present and validate a method, called LULU, for removing erroneous molecular operational taxonomic units (OTUs) from community data derived by high-throughput sequencing of amplified marker genes. LULU identifies errors by combining sequence similarity and co-occurrence patterns. To validate the LULU method, we use a unique data set of high quality survey data of vascular plants paired with plant ITS2 metabarcoding data of DNA extracted from soil from 130 sites in Denmark spanning major environmental gradients. OTU tables are produced with several different OTU definition algorithms and subsequently curated with LULU, and validated against field survey data. LULU curation consistently improves α-diversity estimates and other biodiversity metrics, and does not require a sequence reference database; thus, it represents a promising method for reliable biodiversity estimation.</p><h1>Flooding Duration Affects the Structure of Terrestrial and Aquatic Microbial Eukaryotic Communities</h1>
<p><a href="https://doi.org/10.1007/S00248-017-1085-9">doi:10.1007/S00248-017-1085-9</a></p>

<p><a href=None>None</a></p>
<p>The increasing number and duration of inundations is reported to be a consequence of climate change and may severely compromise non-adapted macroorganisms. The effect of flooding events on terrestrial and aquatic microbial communities is, however, less well understood. They may respond to the changed abiotic properties of their native habitat, and the native community may change due to the introduction of alien species. We designed an experiment to investigate the effect of five different flooding durations on the terrestrial and aquatic communities of eukaryotic microorganism, using the AquaFlow mesocosms. With amplicon sequencing of the small subunit (SSU) and internal transcribed spacer (ITS) rRNA gene regions, we analyzed community compositions directly before and after flooding. Subsequently, they were monitored for another 28 days, to determine the sustainability of community changes. Our results revealed a temporary increase in similarity between terrestrial and aquatic communities according to OTU composition (operational taxonomic unit, serves as a proxy for species). Increased similarity was mainly caused by the transmission of OTUs from water to soil. A minority of these were able to persist in soil until the end of the experiment. By contrast, the vast majority of soil OTUs was not transmitted to water. Flooding duration affected the community structure (abundance) more than composition (occurrence). Terrestrial communities responded immediately to flooding and the flooding duration influenced the community changes. Independent from flooding duration, all terrestrial communities recovered largely after flooding, indicating a remarkable resilience to the applied disturbances. Aquatic communities responded immediately to the applied inundations too. At the end of the experiment, they grouped according to the applied flooding duration and the amount of ammonium and chloride that leached from the soil. This indicates a sustained long-term response of the aquatic communities to flooding events.</p><h1>Indigenous species barcode database improves the identification of zooplankton</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0185697">doi:10.1371/Journal.Pone.0185697</a></p>

<p><a href=None>None</a></p>
<p>Incompleteness and inaccuracy of DNA barcode databases is considered an important hindrance to the use of metabarcoding in biodiversity analysis of zooplankton at the species-level. Species barcoding by Sanger sequencing is inefficient for organisms with small body sizes, such as zooplankton. Here mitochondrial cytochrome c oxidase I (COI) fragment barcodes from 910 freshwater zooplankton specimens (87 morphospecies) were recovered by a high-throughput sequencing platform, Ion Torrent PGM. Intraspecific divergence of most zooplanktons was < 5%, except Branchionus leydign (Rotifer, 14.3%), Trichocerca elongate (Rotifer, 11.5%), Lecane bulla (Rotifer, 15.9%), Synchaeta oblonga (Rotifer, 5.95%) and Schmackeria forbesi (Copepod, 6.5%). Metabarcoding data of 28 environmental samples from Lake Tai were annotated by both an indigenous database and NCBI Genbank database. The indigenous database improved the taxonomic assignment of metabarcoding of zooplankton. Most zooplankton (81%) with barcode sequences in the indigenous database were identified by metabarcoding monitoring. Furthermore, the frequency and distribution of zooplankton were also consistent between metabarcoding and morphology identification. Overall, the indigenous database improved the taxonomic assignment of zooplankton.</p><h1>Insights into the harmful algal flora in northwestern Mediterranean coastal lagoons revealed by pyrosequencing metabarcodes of the 28S rRNA gene</h1>
<p><a href="https://doi.org/S1568-9883(16)30370-5">doi:S1568-9883(16)30370-5</a></p>

<p><a href=None>None</a></p>
<p>This study investigated the genetic diversity of phytoplankton communities in six shallow lagoons located on the French coast of the northwestern Mediterranean Sea that represented a trophic gradient ranging from oligotrophic to hypereutrophic. The phytoplankton communities were sampled once a month from spring (May) to the beginning of autumn (September/early October) in 2012 and fractionated by size. Metabarcodes were generated from cDNAs by targeting the D1-D2 region of the 28S rRNA gene and pyrosequenced using Roche 454 technology. Examination of the annotated barcodes revealed harmful algal species not previously documented in these lagoons. Three ichthyotoxic species belonging to Pfiesteriaceae were detected: Luciella masanensis was relatively widespread and abundant in many samples, whereas Pfiesteria piscicida and Stoeckeria changwonensis were found as single barcode sequences. Furthermore, a phylogenetic analysis of barcodes annotated as belonging to Pfiesteriaceae suggested the existence of two previously undescribed clades. The other toxic or potentially harmful dinoflagellates detected through rare barcodes were Dinophysis acuminata, Vulcanodinium rugosum, Alexandrium andersonii and A. ostenfeldii. The two most abundant dinoflagellate taxa were Gymnodinium litoralis and Akashiwo sanguinea with respect to sequence numbers. Four diatom species from the genus Pseudo-nitzschia that potentially produce domoic acid were identified (P. galaxiae, P. delicatissima, P. brasiliana and P. calliantha). These observations are discussed in terms of the literature and monitoring records related to the identified taxa in this Mediterranean area.</p><h1>The Protistan Microbiome of Grassland Soil: Diversity in the Mesoscale</h1>
<p><a href="https://doi.org/S1434-4610(17)30034-2">doi:S1434-4610(17)30034-2</a></p>

<p><a href=None>None</a></p>
<p>Genomic data for less than one quarter of ∼1.8 million named species on earth exist in public databases like GenBank. Little information exists on the estimated one million small sized (1-100μm) heterotrophic nanoflagellates and ciliates and their taxa-area relationship. We analyzed environmental DNA from 150 geo-referenced grassland plots representing topographical and land-use ranges typical for Central Europe. High through-put barcoding allowed the identification of operational taxonomic units (OTUs) at species level, with high pairwise identity to reference sequences (≥99.7%), but also the identification of sequences at the genus (≥97%) and class (≥80%) taxonomic level. Species richness analyses revealed, on average, 100 genus level OTUs (332 unique individual read (UIR) and 56 class level OTUs per gram of soil sample in the mesoscale (1-1000km). Database shortfalls were highlighted by increased uncertain taxonomic lineages at lower resolution (≥80% sequence identity). No single barcode occurred ubiquitously across all sites. Taxa-area relationships indicated that OTUs spread over the entire mesoscale were more similar than in the local scale and increased land-use (fertilization, mowing and grazing) promoted taxa-area separation. Only a small fraction of sequences strictly matched reference library sequences, suggesting a large protistan "dark matter" in soil which warrants further research.</p><h1>Environmental DNA metabarcoding: Transforming how we survey animal and plant communities</h1>
<p><a href="https://doi.org/10.1111/Mec.14350">doi:10.1111/Mec.14350</a></p>

<p><a href=None>None</a></p>
<p>The genomic revolution has fundamentally changed how we survey biodiversity on earth. High-throughput sequencing ("HTS") platforms now enable the rapid sequencing of DNA from diverse kinds of environmental samples (termed "environmental DNA" or "eDNA"). Coupling HTS with our ability to associate sequences from eDNA with a taxonomic name is called "eDNA metabarcoding" and offers a powerful molecular tool capable of noninvasively surveying species richness from many ecosystems. Here, we review the use of eDNA metabarcoding for surveying animal and plant richness, and the challenges in using eDNA approaches to estimate relative abundance. We highlight eDNA applications in freshwater, marine and terrestrial environments, and in this broad context, we distill what is known about the ability of different eDNA sample types to approximate richness in space and across time. We provide guiding questions for study design and discuss the eDNA metabarcoding workflow with a focus on primers and library preparation methods. We additionally discuss important criteria for consideration of bioinformatic filtering of data sets, with recommendations for increasing transparency. Finally, looking to the future, we discuss emerging applications of eDNA metabarcoding in ecology, conservation, invasion biology, biomonitoring, and how eDNA metabarcoding can empower citizen science and biodiversity education.</p><h1>From Microscopy to Genomic Approach in Soil Biodiversity Assessment</h1>
<p><a href="https://doi.org/10.21775/Cimb.027.195">doi:10.21775/Cimb.027.195</a></p>

<p><a href=None>None</a></p>
<p>Soil biota represents a major component of the earth's biodiversity and for over 200 years, the microscopy approach was the only way to explore it. In the last decade, the DNA-based technique has been adopted in soil ecology. Due to the rapid development of cutting-edge technology, the field is transitioning from barcoding individuals to metabarcoding communities. With the advent of next-generation sequencing and a rapid decline in sequencing cost, it has become feasible to assess soil biodiversity at species level. This review article summarizes current approaches in soil biodiversity research along with their advantages and disadvantages.</p><h1>DNA in a bottle-Rapid metabarcoding survey for early alerts of invasive species in ports</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0183347">doi:10.1371/Journal.Pone.0183347</a></p>

<p><a href=None>None</a></p>
<p>Biota monitoring in ports is increasingly needed for biosecurity reasons and safeguarding marine biodiversity from biological invasion. Present and future international biosecurity directives can be accomplished only if the biota acquired by maritime traffic in ports is controlled. Methodologies for biota inventory are diverse and now rely principally on extensive and labor-intensive sampling along with taxonomic identification by experts. In this study, we employed an extremely simplified environmental DNA (eDNA) sampling methodology from only three 1-L bottles of water per port, followed by metabarcoding (high-throughput sequencing and DNA-based species identification) using 18S rDNA and Cytochrome oxidase I as genetic barcodes. Eight Bay of Biscay ports with available inventory of fouling invertebrates were employed as a case study. Despite minimal sampling efforts, three invasive invertebrates were detected: the barnacle Austrominius modestus, the tubeworm Ficopomatus enigmaticus and the polychaete Polydora triglanda. The same species have been previously found from visual and DNA barcoding (genetic identification of individuals) surveys in the same ports. The current costs of visual surveys, conventional DNA barcoding and this simplified metabarcoding protocol were compared. The results encourage the use of metabarcoding for early biosecurity alerts.</p><h1>Large-scale differences in microbial biodiversity discovery between 16S amplicon and shotgun sequencing</h1>
<p><a href="https://doi.org/10.1038/S41598-017-06665-3">doi:10.1038/S41598-017-06665-3</a></p>

<p><a href=None>None</a></p>
<p>Modern metagenomic environmental DNA studies are almost completely reliant on next-generation sequencing, making evaluations of these methods critical. We compare two next-generation sequencing techniques - amplicon and shotgun - on water samples across four of Brazil's major river floodplain systems (Amazon, Araguaia, Paraná, and Pantanal). Less than 50% of phyla identified via amplicon sequencing were recovered from shotgun sequencing, clearly challenging the dogma that mid-depth shotgun recovers more diversity than amplicon-based approaches. Amplicon sequencing also revealed ~27% more families. Overall the amplicon data were more robust across both biodiversity and community ecology analyses at different taxonomic scales. Our work doubles the sampling size in similar environmental studies, and novelly integrates environmental data (e.g., pH, temperature, nutrients) from each site, revealing divergent correlations depending on which data are used. While myriad variants on NGS techniques and bioinformatic pipelines are available, our results point to core differences that have not been highlighted in any studies to date. Given the low number of taxa identified when coupling shotgun data with clade-based taxonomic algorithms, previous studies that quantified biodiversity using such bioinformatic tools should be viewed cautiously or re-analyzed. Nonetheless, shotgun has complementary advantages that should be weighed when designing projects.</p><h1>Revealing higher than expected meiofaunal diversity in Antarctic sediments: a metabarcoding approach</h1>
<p><a href="https://doi.org/10.1038/S41598-017-06687-X">doi:10.1038/S41598-017-06687-X</a></p>

<p><a href=None>None</a></p>
<p>An increasing number of studies are showing that Antarctic mega- and macrofauna are highly diverse, however, little is known about meiofaunal biodiversity in sediment communities, which are a vital part of a healthy and functional ecosystem. This is the first study to analyse community DNA (targeting meiofauna) using metabarcoding to investigate biodiversity levels in sediment communities of the Antarctic Peninsula. The results show that almost all of the meiofaunal biodiversity in the benthic habitat has yet to be characterised, levels of biodiversity were higher than expected and similar to temperate regions, albeit with the existence of potentially new and locally adapted species never described before at the molecular level. The Rothera meiofaunal sample sites showed four dominant eukaryotic groups, the nematodes, arthropods, platyhelminthes, and the annelids; some of which could comprise species complexes. Comparisons with deep-sea data from the same region suggest little exchange of Operational Taxonomic Units (OTUs) between depths with the nematodes prevalent at all depths, but sharing the shallow water benthos with the copepods. This study provides a preliminary analysis of benthic Antarctic Peninsula meiofauna using high throughput sequencing which substantiates how little is known on the biodiversity of one of the most diverse, yet underexplored communities of the Antarctic: the benthos.</p><h1>Using metagenomics to show the efficacy of forest restoration in the New Jersey Pine Barrens</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0199">doi:10.1139/Gen-2015-0199</a></p>

<p><a href=None>None</a></p>
<p>The Franklin Parker Preserve within the New Jersey Pine Barrens contains 5000 acres of wetlands habitat, including old-growth Atlantic white cedar (or AWC; Chamaecyparis thyoides) swamps, cranberry bogs, and former cranberry bogs undergoing restoration into AWC forests. This study showed that the C-use efficiency was greater in the old-growth AWC soils than in soils from 8-year-old mid-stage restored AWC stands, which were greater than found in soil from 4-year-old AWC stands-the latter two stands being restored from long-term cranberry bogs. A metagenomic analysis of eDNA extracted from these soils showed that the C-cycle trends were associated with increases in the relative numbers of DNA sequences from several copiotrophic bacterial groups (Bacteroidetes and Proteobacteria), complex C-decomposing fungal groups (Sordiomycetes, Mortierellales, and Thelephorales), and collembolan and formicid invertebrates. All groups are indicators of successionally more advanced soils, and critical for soil C-cycle activities. These data suggest that the restoration activities studied are enhancing critical guilds of soil biota, and increasing C-use efficiency in the soils of restored habitats, and that the use of metagenomic analysis of soil eDNA can be used in the development of assessment models for soil recovery of wetlands following restoration.</p><h1>Taxa-area relationship of aquatic fungi on deciduous leaves</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0181545">doi:10.1371/Journal.Pone.0181545</a></p>

<p><a href=None>None</a></p>
<p>One of the fundamental patterns in macroecology is the increase in the number of observed taxa with size of sampled area. For microbes, the shape of this relationship remains less clear. The current study assessed the diversity of aquatic fungi, by the traditional approach based on conidial morphology (captures reproducing aquatic hyphomycetes) and next generation sequencing (NGS; captures other fungi as well), on graded sizes of alder leaves (0.6 to 13.6 cm2). Leaves were submerged in two streams in geographically distant locations: the Oliveira Stream in Portugal and the Boss Brook in Canada. Decay rates of alder leaves and fungal sporulation rates did not differ between streams. Fungal biomass was higher in Boss Brook than in Oliveira Stream, and in both streams almost 100% of the reads belonged to active fungal taxa. In general, larger leaf areas tended to harbour more fungi, but these findings were not consistent between techniques. Morphospecies-based diversity increased with leaf area in Boss Brook, but not in Oliveira Stream; metabarcoding data showed an opposite trend. The higher resolution of metabarcoding resulted in steeper taxa-accumulation curves than morphospecies-based assessments (fungal conidia morphology). Fungal communities assessed by metabarcoding were spatially structured by leaf area in both streams. Metabarcoding promises greater resolution to assess biodiversity patterns in aquatic fungi and may be more accurate for assessing taxa-area relationships and local to global diversity ratios.</p><h1>Predicting the Ecological Quality Status of Marine Environments from eDNA Metabarcoding Data Using Supervised Machine Learning</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.7B01518">doi:10.1021/Acs.Est.7B01518</a></p>

<p><a href=None>None</a></p>
<p>Monitoring biodiversity is essential to assess the impacts of increasing anthropogenic activities in marine environments. Traditionally, marine biomonitoring involves the sorting and morphological identification of benthic macro-invertebrates, which is time-consuming and taxonomic-expertise demanding. High-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) represents a promising alternative for benthic monitoring. However, an important fraction of eDNA sequences remains unassigned or belong to taxa of unknown ecology, which prevent their use for assessing the ecological quality status. Here, we show that supervised machine learning (SML) can be used to build robust predictive models for benthic monitoring, regardless of the taxonomic assignment of eDNA sequences. We tested three SML approaches to assess the environmental impact of marine aquaculture using benthic foraminifera eDNA, a group of unicellular eukaryotes known to be good bioindicators, as features to infer macro-invertebrates based biotic indices. We found similar ecological status as obtained from macro-invertebrates inventories. We argue that SML approaches could overcome and even bypass the cost and time-demanding morpho-taxonomic approaches in future biomonitoring.</p><h1>Analysis of phytoplankton assemblage structure in the Mediterranean Sea based on high-throughput sequencing of partial 18S rRNA sequences</h1>
<p><a href="https://doi.org/S1874-7787(17)30144-7">doi:S1874-7787(17)30144-7</a></p>

<p><a href=None>None</a></p>
<p>Studying taxonomic and ecological diversity of phytoplankton assemblages is often difficult because morphological analysis cannot provide a complete description of their composition. Therefore, more robust and feasible approaches have to be chosen to elucidate the interactions between environmental and human pressures and phytoplankton assemblages. The Ocean Sampling Day (OSD) allowed collecting seawater samples from a wide range of oceanic regions including the Mediterranean Sea. In this study, a total of 754,167 V4-18S ribosomal DNA (rDNA) metabarcodes derived from 20 plankton samples collected at 19 sampling sites across the coastal areas of the Mediterranean Sea were analyzed to explore the relationships between phytoplankton assemblages' composition, sub-regional environmental features and human pressures. We reduced the whole set of autotroph plankton (1398 OTUs) to a smaller number of ecologically relevant entities (205 taxa) and used the latter for analysing the structure of phytoplankton assemblages. Chaetoceros was the only genus occurring in all the samples, while the number of taxa was maximum in the W Mediterranean. Based on the assigned OTUs, the structure of E Mediterranean phytoplankton was the most homogeneous. Further, phytoplankton assemblages from the three Mediterranean sub-regions (Western, Adriatic and Eastern) were significantly different (R=0.25, p=0.0136) based on Jaccard similarity. We also observed that phytoplankton diversity and human impact on marine ecosystems were not significantly related to each other based on Mantel's test.</p><h1>Zooplankton community structure along a pollution gradient at fine geographical scales in river ecosystems: The importance of species sorting over dispersal</h1>
<p><a href="https://doi.org/10.1111/Mec.14199">doi:10.1111/Mec.14199</a></p>

<p><a href=None>None</a></p>
<p>The release of anthropogenic pollution into freshwater ecosystems has largely transformed biodiversity and its geographical distribution patterns globally. However, for many communities including ecologically crucial ones such as zooplankton, it is largely unknown how different communities respond to environmental pollution. Collectively, dispersal and species sorting are two competing processes in determining the structure and geographical distribution of zooplankton communities in running water ecosystems such as rivers. At fine geographical scales, dispersal is usually considered as the dominant factor; however, the relative role of species sorting has not been evaluated well, mainly because significant environmental gradients rarely exist along continuously flowing rivers. The Chaobai River in northern China represents a rare system, where a significant environmental gradient exists at fine scales. Here, we employed high-throughput sequencing to characterize complex zooplankton communities collected from the Chaobai River, and tested the relative roles of dispersal and species sorting in determining zooplankton community structure along the pollution gradient. Our results showed distinct patterns of zooplankton communities along the environmental gradient, and chemical pollutant-related factors such as total phosphorus and chlorophyll-a were identified as the major drivers for the observed patterns. Further partial redundancy analyses showed that species sorting overrode the effect of dispersal to shape local zooplankton community structure. Thus, our results reject the dispersal hypothesis and support the concept that species sorting caused by local pollution can largely determine the zooplankton community structure when significant environmental gradients exist at fine geographical scales in highly polluted running water ecosystems.</p><h1>Functional metagenomics of oil-impacted mangrove sediments reveals high abundance of hydrolases of biotechnological interest</h1>
<p><a href="https://doi.org/10.1007/S11274-017-2307-5">doi:10.1007/S11274-017-2307-5</a></p>

<p><a href=None>None</a></p>
<p>Mangroves are located in coastal wetlands and are susceptible to the consequences of oil spills, what may threaten the diversity of microorganisms responsible for the nutrient cycling and the consequent ecosystem functioning. Previous reports show that high concentration of oil favors the incidence of epoxide hydrolases and haloalkane dehalogenases in mangroves. This finding has guided the goals of this study in an attempt to broaden the analysis to other hydrolases and thereby verify whether oil contamination interferes with the prevalence of particular hydrolases and their assigned microorganisms. For this, an in-depth survey of the taxonomic and functional microbial diversity recovered in a fosmid library (Library_Oil Mgv) constructed from oil-impacted Brazilian mangrove sediment was carried out. Fosmid DNA of the whole library was extracted and submitted to Illumina HiSeq sequencing. The resulting Library Oil_Mgv dataset was further compared with those obtained by direct sequencing of environmental DNA from Brazilian mangroves (from distinct regions and affected by distinct sources of contamination), focusing on hydrolases with potential use in biotechnological processes. The most abundant hydrolases found were proteases, esterases and amylases, with similar occurrence profile in all datasets. The main microbial groups harboring such hydrolase-encoding genes were distinct in each mangrove, and in the fosmid library these enzymes were mainly assigned to Chloroflexaceae (for amylases), Planctomycetaceae (for esterases) and Bradyrhizobiaceae (for proteases). Assembly and analysis of Library_Oil Mgv reads revealed three potentially novel enzymes, one epoxide hydrolase, one xylanase and one amylase, to be further investigated via heterologous expression assays.</p><h1>DNA metabarcoding to assess indoor fungal communities: Electrostatic dust collectors and Illumina sequencing</h1>
<p><a href="https://doi.org/S0167-7012(17)30124-0">doi:S0167-7012(17)30124-0</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is increasingly being used to characterize the microbiological composition of both the indoor and outdoor environments of dwellings. Our study aimed to evaluate metabarcoding and bioinformatic analysis resulting from calibrated samples and samples collected by an electrostatic dust collector (EDC) in dwellings with no moisture problems. Thus, the fungal communities of 14 dwellings (eastern France, Franche-Comté region) were analyzed by Illumina MiSeq technology after amplification of the ITS2 region. Using the standard samples of 11 species of yeasts and molds allowed us to validate the Operational taxonomic units (OTU) assignment. These calibrated samples also showed a low amplification bias, a low rate of sequencing errors and the semi-quantitative nature of the technique. Only one species from the calibrated samples (Lichtheimia corymbifera) was less amplified probably due to the presence of two mismatches in its3 primer. EDC analysis identified 3594OTU with 75% of reads corresponding to 30 genera. The main genera are those usually found by culture techniques (Penicillium, Aspergillus and Cladosporium), but findings also indicate others less commonly isolated in culture such as Epicoccum, the fourth detected genus in our study. The type of heating systems was correlated with fungal diversity. We found less diversity in the dwellings with wood heating and larger quantities of Epicoccum nigrum verified by qPCR. DNA metabarcoding analysis applied to EDC seems promising. However, we think that it must be used along with qPCR, to obtain a more global view of microbial ecology and relative quantification of species of interest within communities.</p><h1>Metabarcoding monitoring analysis: the pros and cons of using co-extracted environmental DNA and RNA data to assess offshore oil production impacts on benthic communities</h1>
<p><a href="https://doi.org/10.7717/Peerj.3347">doi:10.7717/Peerj.3347</a></p>

<p><a href=None>None</a></p>
<p>Sequencing environmental DNA (eDNA) is increasingly being used as an alternative to traditional morphological-based identification to characterize biological assemblages and monitor anthropogenic impacts in marine environments. Most studies only assess eDNA which, compared to eRNA, can persist longer in the environment after cell death. Therefore, eRNA may provide a more immediate census of the environment due to its relatively weaker stability, leading some researchers to advocate for the use of eRNA as an additional, or perhaps superior proxy for portraying ecological changes. A variety of pre-treatment techniques for screening eDNA and eRNA derived operational taxonomic units (OTUs) have been employed prior to statistical analyses, including removing singleton taxa (i.e., OTUs found only once) and discarding those not present in both eDNA and eRNA datasets. In this study, we used bacterial (16S ribosomal RNA gene) and eukaryotic (18S ribosomal RNA gene) eDNA- and eRNA-derived data from benthic communities collected at increasing distances along a transect from an oil production platform (Taranaki, New Zealand). Macro-infauna (visual classification of benthic invertebrates) and physico-chemical data were analyzed in parallel. We tested the effect of removing singleton taxa, and removing taxa not present in the eDNA and eRNA libraries from the same environmental sample (trimmed by shared OTUs), by comparing the impact of the oil production platform on alpha- and beta-diversity of the eDNA/eRNA-based biological assemblages, and by correlating these to the morphologically identified macro-faunal communities and the physico-chemical data. When trimmed by singletons, presence/absence information from eRNA data represented the best proxy to detect changes on species diversity for both bacteria and eukaryotes. However, assessment of quantitative beta-diversity from read abundance information of bacteria eRNA did not, contrary to eDNA, reveal any impact from the oil production activity. Overall, the data appeared more robust when trimmed by shared OTUs, showing a greater effect of the platform on alpha- and beta-diversity. Trimming by shared OTUs likely removes taxa derived from legacy DNA and technical artefacts introduced through reverse transcriptase, polymerase-chain-reaction and sequencing. Findings from our scoping study suggest that metabarcoding-based biomonitoring surveys should, if funds, time and expertise allow, be assessed using both eDNA and eRNA products.</p><h1>The importance of standardization for biodiversity comparisons: A case study using autonomous reef monitoring structures (ARMS) and metabarcoding to measure cryptic diversity on Mo'orea coral reefs, French Polynesia</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0175066">doi:10.1371/Journal.Pone.0175066</a></p>

<p><a href=None>None</a></p>
<p>The advancement of metabarcoding techniques, declining costs of high-throughput sequencing and development of systematic sampling devices, such as autonomous reef monitoring structures (ARMS), have provided the means to gather a vast amount of diversity data from cryptic marine communities. However, such increased capability could also lead to analytical challenges if the methods used to examine these communities across local and global scales are not standardized. Here we compare and assess the underlying biases of four ARMS field processing methods, preservation media, and current bioinformatic pipelines in evaluating diversity from cytochrome c oxidase I metabarcoding data. Illustrating the ability of ARMS-based metabarcoding to capture a wide spectrum of biodiversity, 3,372 OTUs and twenty-eight phyla, including 17 of 33 marine metazoan phyla, were detected from 3 ARMS (2.607 m2 area) collected on coral reefs in Mo'orea, French Polynesia. Significant differences were found between processing and preservation methods, demonstrating the need to standardize methods for biodiversity comparisons. We recommend the use of a standardized protocol (NOAA method) combined with DMSO preservation of tissues for sessile macroorganisms because it gave a more accurate representation of the underlying communities, is cost effective and removes chemical restrictions associated with sample transportation. We found that sequences identified at ≥ 97% similarity increased more than 7-fold (5.1% to 38.6%) using a geographically local barcode inventory, highlighting the importance of local species inventories. Phylogenetic approaches that assign higher taxonomic ranks accrued phylum identification errors (9.7%) due to sparse taxonomic coverage of the understudied cryptic coral reef community in public databases. However, a ≥ 85% sequence identity cut-off provided more accurate results (0.7% errors) and enabled phylum level identifications of 86.3% of the sequence reads. With over 1600 ARMS deployed, standardizing methods and improving databases are imperative to provide unprecedented global baseline assessments of understudied cryptic marine species in a rapidly changing world.</p><h1>Effective mosquito and arbovirus surveillance using metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12682">doi:10.1111/1755-0998.12682</a></p>

<p><a href=None>None</a></p>
<p>Effective vector and arbovirus surveillance requires timely and accurate screening techniques that can be easily upscaled. Next-generation sequencing (NGS) is a high-throughput technology that has the potential to modernize vector surveillance. When combined with DNA barcoding, it is termed 'metabarcoding.' The aim of our study was to establish a metabarcoding protocol to characterize pools of mosquitoes and screen them for virus. Pools contained 100 morphologically identified individuals, including one Ross River virus (RRV) infected mosquito, with three species present at different proportions: 1, 5, 94%. Nucleic acid extracted from both crude homogenate and supernatant was used to amplify a 269-bp section of the mitochondrial cytochrome c oxidase subunit I (COI) locus. Additionally, a 67-bp region of the RRV E2 gene was amplified from synthesized cDNA to screen for RRV. Amplicon sequencing was performed using an Illumina MiSeq, and bioinformatic analysis was performed using a DNA barcode database of Victorian mosquitoes. Metabarcoding successfully detected all mosquito species and RRV in every positive sample tested. The limits of species detection were also examined by screening a pool of 1000 individuals, successfully identifying the species and RRV from a single mosquito. The primers used for amplification, number of PCR cycles and total number of individuals present all have effects on the quantification of species in mixed bulk samples. Based on the results, a number of recommendations for future metabarcoding studies are presented. Overall, metabarcoding shows great promise for providing a new alternative approach to screening large insect surveillance trap catches.</p><h1>Sensitivity and accuracy of high-throughput metabarcoding methods for early detection of invasive fish species</h1>
<p><a href="https://doi.org/10.1038/Srep46393">doi:10.1038/Srep46393</a></p>

<p><a href=None>None</a></p>
<p>High-throughput DNA metabarcoding has gained recognition as a potentially powerful tool for biomonitoring, including early detection of aquatic invasive species (AIS). DNA based techniques are advancing, but our understanding of the limits to detection for metabarcoding complex samples is inadequate. For detecting AIS at an early stage of invasion when the species is rare, accuracy at low detection limits is key. To evaluate the utility of metabarcoding in future fish community monitoring programs, we conducted several experiments to determine the sensitivity and accuracy of routine metabarcoding methods. Experimental mixes used larval fish tissue from multiple "common" species spiked with varying proportions of tissue from an additional "rare" species. Pyrosequencing of genetic marker, COI (cytochrome c oxidase subunit I) and subsequent sequence data analysis provided experimental evidence of low-level detection of the target "rare" species at biomass percentages as low as 0.02% of total sample biomass. Limits to detection varied interspecifically and were susceptible to amplification bias. Moreover, results showed some data processing methods can skew sequence-based biodiversity measurements from corresponding relative biomass abundances and increase false absences. We suggest caution in interpreting presence/absence and relative abundance in larval fish assemblages until metabarcoding methods are optimized for accuracy and precision.</p><h1>Does warming by open-top chambers induce change in the root-associated fungal community of the arctic dwarf shrub Cassiope tetragona (Ericaceae)?</h1>
<p><a href="https://doi.org/10.1007/S00572-017-0767-Y">doi:10.1007/S00572-017-0767-Y</a></p>

<p><a href=None>None</a></p>
<p>Climate change may alter mycorrhizal communities, which impact ecosystem characteristics such as carbon sequestration processes. These impacts occur at a greater magnitude in Arctic ecosystems, where the climate is warming faster than in lower latitudes. Cassiope tetragona (L.) D. Don is an Arctic plant species in the Ericaceae family with a circumpolar range. C. tetragona has been reported to form ericoid mycorrhizal (ErM) as well as ectomycorrhizal (ECM) symbioses. In this study, the fungal taxa present within roots of C. tetragona plants collected from Svalbard were investigated using DNA metabarcoding. In light of ongoing climate change in the Arctic, the effects of artificial warming by open-top chambers (OTCs) on the fungal root community of C. tetragona were evaluated. We detected only a weak effect of warming by OTCs on the root-associated fungal communities that was masked by the spatial variation between sampling sites. The root fungal community of C. tetragona was dominated by fungal groups in the Basidiomycota traditionally classified as either saprotrophic or ECM symbionts, including the orders Sebacinales and Agaricales and the genera Clavaria, Cortinarius, and Mycena. Only a minor proportion of the operational taxonomic units (OTUs) could be annotated as ErM-forming fungi. This indicates that C. tetragona may be forming mycorrhizal symbioses with typically ECM-forming fungi, although no characteristic ECM root tips were observed. Previous studies have indicated that some saprophytic fungi may also be involved in biotrophic associations, but whether the saprotrophic fungi in the roots of C. tetragona are involved in biotrophic associations remains unclear. The need for more experimental and microscopy-based studies to reveal the nature of the fungal associations in C. tetragona roots is emphasized.</p><h1>Patterns of temporal and enemy niche use by a community of leaf cone moths (Caloptilia) coexisting on maples (Acer) as revealed by metabarcoding</h1>
<p><a href="https://doi.org/10.1111/Mec.14105">doi:10.1111/Mec.14105</a></p>

<p><a href=None>None</a></p>
<p>The diversity of herbivorous insects is often considered a function of host plant diversity. However, recent research has uncovered many examples of closely related herbivores using the same host plant(s), suggesting that partitioning of host plants is not the only mechanism generating diversity. Herbivores sharing hosts may utilize different parts of the same plant, but such resource partitioning is often not apparent; hence, the factors that allow closely related herbivores to coexist are still largely undetermined. We examined whether partitioning of phenology or natural enemies may explain the coexistence of leaf cone moths (Caloptilia; Gracillariidae) associated with maples (Acer; Sapindaceae). Larval activity of 10 sympatric Caloptilia species found on nine maple species was monitored every 2-3 weeks for a total of 13 sampling events, and an exhaustive search for internal parasitoid wasps was conducted using high-throughput sequencing. Blocking primers were used to facilitate the detection of wasp larvae inside moth tissue. We found considerable phenological overlap among Caloptilia species, with two clear peaks in July and September-October. Coexisting Caloptilia species also had largely overlapping parasitoid communities; a total of 13 chalcid and ichneumon wasp species attacked Caloptilia in a nonspecific fashion at an overall parasitism rate of 46.4%. Although coexistence may be facilitated by factors not accounted for in this study, it appears that niche partitioning is not necessary for closely related herbivores to stably coexist on shared hosts. Co-occurrence without resource partitioning may provide an additional axis along which herbivorous insects attain increased species richness.</p><h1>Taxonomy-free molecular diatom index for high-throughput eDNA biomonitoring</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12668">doi:10.1111/1755-0998.12668</a></p>

<p><a href=None>None</a></p>
<p>Current biodiversity assessment and biomonitoring are largely based on the morphological identification of selected bioindicator taxa. Recently, several attempts have been made to use eDNA metabarcoding as an alternative tool. However, until now, most applied metabarcoding studies have been based on the taxonomic assignment of sequences that provides reference to morphospecies ecology. Usually, only a small portion of metabarcoding data can be used due to a limited reference database and a lack of phylogenetic resolution. Here, we investigate the possibility to overcome these limitations using a taxonomy-free approach that allows the computing of a molecular index directly from eDNA data without any reference to morphotaxonomy. As a case study, we use the benthic diatoms index, commonly used for monitoring the biological quality of rivers and streams. We analysed 87 epilithic samples from Swiss rivers, the ecological status of which was established based on the microscopic identification of diatom species. We compared the diatom index derived from eDNA data obtained with or without taxonomic assignment. Our taxonomy-free approach yields promising results by providing a correct assessment for 77% of examined sites. The main advantage of this method is that almost 95% of OTUs could be used for index calculation, compared to 35% in the case of the taxonomic assignment approach. Its main limitations are under-sampling and the need to calibrate the index based on the microscopic assessment of diatoms communities. However, once calibrated, the taxonomy-free molecular index can be easily standardized and applied in routine biomonitoring, as a complementary tool allowing fast and cost-effective assessment of the biological quality of watercourses.</p><h1>Protistan Biogeography: A Snapshot Across a Major Shipping Corridor Spanning Two Oceans</h1>
<p><a href="https://doi.org/S1434-4610(16)30095-5">doi:S1434-4610(16)30095-5</a></p>

<p><a href=None>None</a></p>
<p>Deciphering patterns of protistan taxa is a crucial step for understanding anthropogenic and environmental impacts on biogeography. We characterized and compared protistan communities from environmental samples collected along a major shipping corridor, the Panama Canal, and the Bocas del Toro archipelago. We used metabarcoding with high throughput sequencing (HTS) with the V4 hypervariable region of the ribosomal gene complex (rDNA). We detected many protistan taxa, including a variety of parasitic and toxic taxa. There were 1,296 OTUs shared across all three regions, with an additional 342-1,526 OTUs occurring across two or more regions, suggesting some mixing within the Caribbean and across the Isthmus. In general, this mixing did not impact community similarity, which was primarily distinct across regions. When OTUs identified as gregarines were analyzed separately, most samples grouped by region and communities were distinct across the Canal. Shipping traffic through the Panama Canal could move some taxa across regions; however, different environmental conditions in the two oceans may limit their establishment. Overall our results suggest that contemporary protistan biogeographic patterns are likely caused by a complex combination of factors, including anthropogenic dispersal and environmental tolerance.</p><h1>Revegetation rewilds the soil bacterial microbiome of an old field</h1>
<p><a href="https://doi.org/10.1111/Mec.14081">doi:10.1111/Mec.14081</a></p>

<p><a href=None>None</a></p>
<p>Ecological restoration is a globally important and well-financed management intervention used to combat biodiversity declines and land degradation. Most restoration aims to increase biodiversity towards a reference state, but there are concerns that intended outcomes are not reached due to unsuccessful interventions and land-use legacy issues. Monitoring biodiversity recovery is essential to measure success; however, most projects remain insufficiently monitored. Current field-based methods are hard to standardize and are limited in their ability to assess important components of ecosystems, such as bacteria. High-throughput amplicon sequencing of environmental DNA (metabarcoding of eDNA) has been proposed as a cost-effective, scalable and uniform ecological monitoring solution, but its application in restoration remains largely untested. Here we show that metabarcoding of soil eDNA is effective at demonstrating the return of the native bacterial community in an old field following native plant revegetation. Bacterial composition shifted significantly after 8 years of revegetation, where younger sites were more similar to cleared sites and older sites were more similar to remnant stands. Revegetation of the native plant community strongly impacted on the belowground bacterial community, despite the revegetated sites having a long and dramatically altered land-use history (i.e. >100 years grazing). We demonstrate that metabarcoding of eDNA provides an effective way of monitoring changes in bacterial communities that would otherwise go unchecked with conventional monitoring of restoration projects. With further development, awareness of microbial diversity in restoration has significant scope for improving the efficacy of restoration interventions more broadly.</p><h1>The best of both worlds: A combined approach for analyzing microalgal diversity via metabarcoding and morphology-based methods</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0172808">doi:10.1371/Journal.Pone.0172808</a></p>

<p><a href=None>None</a></p>
<p>An increasing number of studies use next generation sequencing (NGS) to analyze complex communities, but is the method sensitive enough when it comes to identification and quantification of species? We compared NGS with morphology-based identification methods in an analysis of microalgal (periphyton) communities. We conducted a mesocosm experiment in which we allowed two benthic grazer species to feed upon benthic biofilms, which resulted in altered periphyton communities. Morphology-based identification and 454 (Roche) pyrosequencing of the V4 region in the small ribosomal unit (18S) rDNA gene were used to investigate the community change caused by grazing. Both the NGS-based data and the morphology-based method detected a marked shift in the biofilm composition, though the two methods varied strongly in their abilities to detect and quantify specific taxa, and neither method was able to detect all species in the biofilms. For quantitative analysis, we therefore recommend using both metabarcoding and microscopic identification when assessing the community composition of eukaryotic microorganisms.</p><h1>The importance of molecular markers and primer design when characterizing biodiversity from environmental DNA</h1>
<p><a href="https://doi.org/10.1139/Gen-2016-0100">doi:10.1139/Gen-2016-0100</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) comprises DNA fragments that have been shed into the environment by organisms, and which can be extracted from environmental samples such as water or soil. Characterization of eDNA can allow researchers to infer the presence or absence of species from a particular site without the need to locate and identify individuals, and therefore may provide an extremely valuable tool for quantifying biodiversity. However, as is often the case with relatively new protocols, methodological challenges remain. A number of earlier reviews have discussed these challenges, but none have provided extensive treatment of the critical decisions surrounding molecular markers and primer development for use in eDNA assays. This review discusses a number of options and approaches that can be used when determining which primers and gene regions are most appropriate for either targeted species detection or metabarcoding macro-organisms from eDNA. The latter represents a new field that is growing rapidly, and which has the potential to revolutionize future assessments of community and ecosystem diversity.</p><h1>Priming of a DNA metabarcoding approach for species identification and inventory in marine macrobenthic communities</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0220">doi:10.1139/Gen-2015-0220</a></p>

<p><a href=None>None</a></p>
<p>In marine and estuarine benthic communities, the inventory and estimation of species richness are often hampered by the need for broad taxonomic expertise across several phyla. The use of DNA metabarcoding has emerged as a powerful tool for the fast assessment of species composition in a diversity of ecological communities. Here, we tested the amplification success of five primer sets targeting different COI-5P regions by 454 pyrosequencing to maximize the recovery of two simulated macrobenthic communities containing 21 species (SimCom1 and SimCom 2). Species identification was first performed against a compiled reference library of macrobenthic species. Reads with similarity results to reference sequences between 70% and 97% were then submitted to GenBank and BOLD to attempt the identification of concealed species in the bulk sample. The combination of at least three primer sets was able to recover more species than any primer set alone, achieving 85% of represented species in SimCom1 and 76% in SimCom2. Our approach was successful to detect low-frequency specimens, as well as concealed species, in the bulk sample, indicating the potential for the application of this approach on marine bioassessment and inventory, including the detection of "hidden" biodiversity that would hardly be possible based on morphology only.</p><h1>Annual time-series analysis of aqueous eDNA reveals ecologically relevant dynamics of lake ecosystem biodiversity</h1>
<p><a href="https://doi.org/10.1038/Ncomms14087">doi:10.1038/Ncomms14087</a></p>

<p><a href=None>None</a></p>
<p>The use of environmental DNA (eDNA) in biodiversity assessments offers a step-change in sensitivity, throughput and simultaneous measures of ecosystem diversity and function. There remains, however, a need to examine eDNA persistence in the wild through simultaneous temporal measures of eDNA and biota. Here, we use metabarcoding of two markers of different lengths, derived from an annual time series of aqueous lake eDNA to examine temporal shifts in ecosystem biodiversity and in an ecologically important group of macroinvertebrates (Diptera: Chironomidae). The analyses allow different levels of detection and validation of taxon richness and community composition (β-diversity) through time, with shorter eDNA fragments dominating the eDNA community. Comparisons between eDNA, community DNA, taxonomy and UK species abundance data further show significant relationships between diversity estimates derived across the disparate methodologies. Our results reveal the temporal dynamics of eDNA and validate the utility of eDNA metabarcoding for tracking seasonal diversity at the ecosystem scale.</p><h1>DNA Metabarcoding of Amazonian Ichthyoplankton Swarms</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0170009">doi:10.1371/Journal.Pone.0170009</a></p>

<p><a href=None>None</a></p>
<p>Tropical rainforests harbor extraordinary biodiversity. The Amazon basin is thought to hold 30% of all river fish species in the world. Information about the ecology, reproduction, and recruitment of most species is still lacking, thus hampering fisheries management and successful conservation strategies. One of the key understudied issues in the study of population dynamics is recruitment. Fish larval ecology in tropical biomes is still in its infancy owing to identification difficulties. Molecular techniques are very promising tools for the identification of larvae at the species level. However, one of their limits is obtaining individual sequences with large samples of larvae. To facilitate this task, we developed a new method based on the massive parallel sequencing capability of next generation sequencing (NGS) coupled with hybridization capture. We focused on the mitochondrial marker cytochrome oxidase I (COI). The results obtained using the new method were compared with individual larval sequencing. We validated the ability of the method to identify Amazonian catfish larvae at the species level and to estimate the relative abundance of species in batches of larvae. Finally, we applied the method and provided evidence for strong temporal variation in reproductive activity of catfish species in the Ucayalí River in the Peruvian Amazon. This new time and cost effective method enables the acquisition of large datasets, paving the way for a finer understanding of reproductive dynamics and recruitment patterns of tropical fish species, with major implications for fisheries management and conservation.</p><h1>Age-related environmental gradients influence invertebrate distribution in the Prince Charles Mountains, East Antarctica</h1>
<p><a href="https://doi.org/10.1098/Rsos.160296">doi:10.1098/Rsos.160296</a></p>

<p><a href=None>None</a></p>
<p>The potential impact of environmental change on terrestrial Antarctic ecosystems can be explored by inspecting biodiversity patterns across large-scale gradients. Unfortunately, morphology-based surveys of Antarctic invertebrates are time-consuming and limited by the cryptic nature of many taxa. We used biodiversity information derived from high-throughput sequencing (HTS) to elucidate the relationship between soil properties and invertebrate biodiversity in the Prince Charles Mountains, East Antarctica. Across 136 analysed soil samples collected from Mount Menzies, Mawson Escarpment and Lake Terrasovoje, we found invertebrate distribution in the Prince Charles Mountains significantly influenced by soil salinity and/or sulfur content. Phyla Tardigrada and Arachnida occurred predominantly in low-salinity substrates with abundant nutrients, whereas Bdelloidea (Rotifera) and Chromadorea (Nematoda) were more common in highly saline substrates. A significant correlation between invertebrate occurrence, soil salinity and time since deglaciation indicates that terrain age indirectly influences Antarctic terrestrial biodiversity, with more recently deglaciated areas supporting greater diversity. Our study demonstrates the value of HTS metabarcoding to investigate environmental constraints on inconspicuous soil biodiversity across large spatial scales.</p><h1>Environmental DNA metabarcoding reveals primary chemical contaminants in freshwater sediments from different land-use types</h1>
<p><a href="https://doi.org/S0045-6535(16)31858-6">doi:S0045-6535(16)31858-6</a></p>

<p><a href=None>None</a></p>
<p>Land-use intensification threatens freshwater biodiversity. Freshwater eukaryotic communities are affected by multiple chemical contaminants with a land-use specific manner. However, biodiversities of eukaryotes and their associations with multiple chemical contaminants are largely unknown. This study characterized in situ eukaryotic communities in sediments exposed to mixtures of chemical contaminants and assessed relationships between various environmental variables and eukaryotic communities in sediments from the Nanfei River. Eukaryotic communities in the sediment samples were dominated by Annelida, Arthropoda, Rotifera, Ochrophyta, Chlorophyta and Ciliophora. Alpha-diversities (Shannon entropy) and structures of eukaryotic communities were significantly different between land-use types. According to the results of multiple statistical tests (PCoA, distLM, Mantel and network analysis), dissimilarity of eukaryotic community structures revealed the key effects of pyrethroid insecticides, manganese, zinc, lead, chromium and polycyclic aromatic hydrocarbons (PAHs) on eukaryotic communities in the sediment samples from the Nanfei River. Furthermore, taxa associated with land-use types were identified and several sensitive eukaryotic taxa to some of the primary contaminants were identified as potential indicators to monitor effects of the primary chemical contaminants. Overall, environmental DNA metabarcoding on in situ eukaryotic communities provided a powerful tool for biomonitoring and identifying primary contaminants and their complex effects on benthic eukaryotic communities in freshwater sediments.</p><h1>DNA metabarcoding reveals that 200-μm-size-fractionated filtering is unable to discriminate between planktonic microbial and large eukaryotes</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12652">doi:10.1111/1755-0998.12652</a></p>

<p><a href=None>None</a></p>
<p>Microeukaryotic plankton (0.2-200 μm) are critical components of aquatic ecosystems and key players in global ecological processes. High-throughput sequencing is currently revolutionizing their study on an unprecedented scale. However, it is currently unclear whether we can accurately, effectively and quantitatively depict the microeukaryotic plankton communities using traditional size-fractionated filtering combined with molecular methods. To address this, we analysed the eukaryotic plankton communities both with, and without, prefiltering with a 200 μm pore-size sieve -by using SSU rDNA-based high-throughput sequencing on 16 samples with three replicates in each sample from two subtropical reservoirs sampled from January to October in 2013. We found that ~25% reads were classified as metazoan in both size groups. The species richness, alpha and beta diversity of plankton community and relative abundance of reads in 99.2% eukaryotic OTUs showed no significant changes after prefiltering with a 200 μm pore-size sieve. We further found that both >0.2 μm and 0.2-200 μm eukaryotic plankton communities, especially the abundant plankton subcommunities, exhibited very similar, and synchronous, spatiotemporal patterns and processes associated with almost identical environmental drivers. The lack of an effect on community structure from prefiltering suggests that environmental DNA from larger metazoa is introduced into the smaller size class. Therefore, size-fractionated filtering with 200 μm is insufficient to discriminate between the eukaryotic plankton size groups in metabarcoding approaches. Our results also highlight the importance of sequencing depth, and strict quality filtering of reads, when designing studies to characterize microeukaryotic plankton communities.</p><h1>Recommendations for developing and applying genetic tools to assess and manage biological invasions in marine ecosystems</h1>
<p><a href="https://doi.org/10.1016/J.Marpol.2017.08.014">doi:10.1016/J.Marpol.2017.08.014</a></p>

<p><a href=None>None</a></p>
<p>The European Union's Marine Strategy Framework Directive (MSFD) aims to adopt integrated ecosystem management approaches to achieve or maintain "Good Environmental Status" for marine waters, habitats and resources, including mitigation of the negative effects of non-indigenous species (NIS). The Directive further seeks to promote broadly standardized monitoring efforts and assessment of temporal trends in marine ecosystem condition, incorporating metrics describing the distribution and impacts of NIS. Accomplishing these goals will require application of advanced tools for NIS surveillance and risk assessment, particularly given known challenges associated with surveying and monitoring with traditional methods. In the past decade, a host of methods based on nucleic acids (DNA and RNA) analysis have been developed or advanced that promise to dramatically enhance capacity in assessing and managing NIS. However, ensuring that these rapidly evolving approaches remain accessible and responsive to the needs of resource managers remains a challenge. This paper provides recommendations for future development of these genetic tools for assessment and management of NIS in marine systems, within the context of the explicit requirements of the MSFD. Issues considered include technological innovation, methodological standardization, data sharing and collaboration, and the critical importance of shared foundational resources, particularly integrated taxonomic expertise. Though the recommendations offered here are not exhaustive, they provide a basis for future intentional (and international) collaborative development of a genetic toolkit for NIS research, capable of fulfilling the immediate and long term goals of marine ecosystem and resource conservation.</p><h1>eDNA metabarcoding: a promising method for anuran surveys in highly diverse tropical forests</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12643">doi:10.1111/1755-0998.12643</a></p>

<p><a href=None>None</a></p>
<p>Understanding the geographical distribution and community composition of species is crucial to monitor species persistence and define effective conservation strategies. Environmental DNA (eDNA) has emerged as a powerful noninvasive tool for species detection. However, most eDNA survey methods have been developed and applied in temperate zones. We tested the feasibility of using eDNA to survey anurans in tropical streams in the Brazilian Atlantic forest and compared the results with short-term visual and audio surveys. We detected all nine species known to inhabit our focal streams with one single visit for eDNA sampling. We found a higher proportion of sequence reads and larger number of positive PCR replicates for more common species and for those with life cycles closely associated with the streams, factors that may contribute to increased release of DNA in the water. However, less common species were also detected in eDNA samples, demonstrating the detection power of this method. Filtering larger volumes of water resulted in a higher probability of detection. Our data also show it is important to sample multiple sites along streams, particularly for detection of target species with lower population densities. For the three focal species in our study, the eDNA metabarcoding method had a greater capacity of detection per sampling event than our rapid field surveys, and thus, has the potential to circumvent some of the challenges associated with traditional approaches. Our results underscore the utility of eDNA metabarcoding as an efficient method to survey anuran species in tropical streams of the highly biodiverse Brazilian Atlantic forest.</p><h1>Vertical Structure of Phyllosphere Fungal Communities in a Tropical Forest in Thailand Uncovered by High-Throughput Sequencing</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0166669">doi:10.1371/Journal.Pone.0166669</a></p>

<p><a href=None>None</a></p>
<p>Phyllosphere fungi harbor a tremendous species diversity and play important ecological roles. However, little is known about their distribution patterns within forest ecosystems. We examined how species diversity and community composition of phyllosphere fungi change along a vertical structure in a tropical forest in Thailand. Fungal communities in 144 leaf samples from 19 vertical layers (1.28-34.4 m above ground) of 73 plant individuals (27 species) were investigated by metabarcoding analysis using Ion Torrent sequencing. In total, 1,524 fungal operational taxonomic units (OTUs) were detected among 890,710 reads obtained from the 144 leaf samples. Taxonomically diverse fungi belonging to as many as 24 orders of Ascomycota and 21 orders of Basidiomycota were detected, most of which inhabited limited parts of the lowest layers closest to the forest floor. Species diversity of phyllosphere fungi was the highest in the lowest layers closest to the forest floor, decreased with increasing height, and lowest in the canopy; 742 and 55 fungal OTUs were detected at the lowest and highest layer, respectively. On the layers close to the forest floor, phyllosphere fungal communities were mainly composed of low frequency OTUs and largely differentiated among plant individuals. Conversely, in the canopy, fungal communities consisted of similar OTUs across plant individuals, and as many as 86.1%-92.7% of the OTUs found in the canopy (≥22 m above ground) were also distributed in the lower layers. Overall, our study showed the variability of phyllosphere fungal communities along the vertical gradient of plant vegetation and environmental conditions, suggesting the significance of biotic and abiotic variation for the species diversity of phyllosphere fungi.</p><h1>16S rRNA Amplicon Sequencing for Epidemiological Surveys of Bacteria in Wildlife</h1>
<p><a href="https://doi.org/E00032-16">doi:E00032-16</a></p>

<p><a href=None>None</a></p>
<p>The human impact on natural habitats is increasing the complexity of human-wildlife interactions and leading to the emergence of infectious diseases worldwide. Highly successful synanthropic wildlife species, such as rodents, will undoubtedly play an increasingly important role in transmitting zoonotic diseases. We investigated the potential for recent developments in 16S rRNA amplicon sequencing to facilitate the multiplexing of the large numbers of samples needed to improve our understanding of the risk of zoonotic disease transmission posed by urban rodents in West Africa. In addition to listing pathogenic bacteria in wild populations, as in other high-throughput sequencing (HTS) studies, our approach can estimate essential parameters for studies of zoonotic risk, such as prevalence and patterns of coinfection within individual hosts. However, the estimation of these parameters requires cleaning of the raw data to mitigate the biases generated by HTS methods. We present here an extensive review of these biases and of their consequences, and we propose a comprehensive trimming strategy for managing these biases. We demonstrated the application of this strategy using 711 commensal rodents, including 208 <i>Mus musculus domesticus</i>, 189 <i>Rattus rattus</i>, 93 <i>Mastomys natalensis</i>, and 221 <i>Mastomys erythroleucus</i>, collected from 24 villages in Senegal. Seven major genera of pathogenic bacteria were detected in their spleens: <i>Borrelia</i>, <i>Bartonella</i>, <i>Mycoplasma</i>, <i>Ehrlichia</i>, <i>Rickettsia</i>, <i>Streptobacillus</i>, and <i>Orientia</i>. <i>Mycoplasma</i>, <i>Ehrlichia</i>, <i>Rickettsia</i>, <i>Streptobacillus</i>, and <i>Orientia</i> have never before been detected in West African rodents. Bacterial prevalence ranged from 0% to 90% of individuals per site, depending on the bacterial taxon, rodent species, and site considered, and 26% of rodents displayed coinfection. The 16S rRNA amplicon sequencing strategy presented here has the advantage over other molecular surveillance tools of dealing with a large spectrum of bacterial pathogens without requiring assumptions about their presence in the samples. This approach is therefore particularly suitable to continuous pathogen surveillance in the context of disease-monitoring programs. <b>IMPORTANCE</b> Several recent public health crises have shown that the surveillance of zoonotic agents in wildlife is important to prevent pandemic risks. High-throughput sequencing (HTS) technologies are potentially useful for this surveillance, but rigorous experimental processes are required for the use of these effective tools in such epidemiological contexts. In particular, HTS introduces biases into the raw data set that might lead to incorrect interpretations. We describe here a procedure for cleaning data before estimating reliable biological parameters, such as positivity, prevalence, and coinfection, using 16S rRNA amplicon sequencing on an Illumina MiSeq platform. This procedure, applied to 711 rodents collected in West Africa, detected several zoonotic bacterial species, including some at high prevalence, despite their never before having been reported for West Africa. In the future, this approach could be adapted for the monitoring of other microbes such as protists, fungi, and even viruses.</p><h1>Diversity and temporal patterns of planktonic protist assemblages at a Mediterranean Long Term Ecological Research site</h1>
<p><a href="https://doi.org/Fiw200">doi:Fiw200</a></p>

<p><a href=None>None</a></p>
<p>We tracked temporal changes in protist diversity at the Long Term Ecological Research (LTER) station MareChiara in the Gulf of Naples (Mediterranean Sea) on eight dates in 2011 using a metabarcoding approach. Illumina analysis of the V4 and V9 fragments of the 18S rDNA produced 869 522 and 1 410 071 sequences resulting in 6517 and 6519 OTUs, respectively. Marked compositional variations were recorded across the year, with less than 2% of OTUs shared among all samples and similar patterns for the two marker tags. Alveolata, Stramenopiles and Rhizaria were the most represented groups. A comparison with light microscopy data indicated an over-representation of Dinophyta in the sequence dataset, whereas Bacillariophyta showed comparable taxonomic patterns between sequence and light microscopy data. Shannon diversity values were stable from February to September, increasing thereafter with a peak in December. Community variance was mainly explained by seasonality (as temperature), trophic status (as chlorophyll a), and influence of coastal waters (as salinity). Overall, the background knowledge of the system provided a sound context for the result interpretation, showing that LTER sites provide an ideal setting for high-throughput sequencing (HTS) metabarcoding characterisation of protist assemblages and their relationships with environmental variations.</p><h1>One step forwards for the routine use of high-throughput DNA sequencing in environmental monitoring. An efficient and standardizable method to maximize the detection of environmental bacteria</h1>
<p><a href="https://doi.org/10.1002/Mbo3.421">doi:10.1002/Mbo3.421</a></p>

<p><a href=None>None</a></p>
<p>We propose an innovative, repeatable, and reliable experimental workflow to concentrate and detect environmental bacteria in drinking water using molecular techniques. We first concentrated bacteria in water samples using tangential flow filtration and then we evaluated two methods of environmental DNA extraction. We performed tests on both artificially contaminated water samples and real drinking water samples. The efficiency of the experimental workflow was measured through qPCR. The successful applicability of the high-throughput DNA sequencing (HTS) approach was demonstrated on drinking water samples. Our results demonstrate the feasibility of our approach in high-throughput-based studies, and we suggest incorporating it in monitoring strategies to have a better representation of the microbial community. In the recent years, HTS techniques have become key tools in the study of microbial communities. To make the leap from academic laboratories to the routine monitoring (e.g., water treatment plants laboratories), we here propose an experimental workflow suitable for the introduction of HTS as a standard method for detecting environmental bacteria.</p><h1>Assessing myxozoan presence and diversity using environmental DNA</h1>
<p><a href="https://doi.org/S0020-7519(16)30195-3">doi:S0020-7519(16)30195-3</a></p>

<p><a href=None>None</a></p>
<p>Amplicon sequencing on a High Throughput Sequencing platform (custom barcoding) was used to detect and characterise myxosporean communities in environmental DNA samples from marine and freshwater environments and in faeces of animals that may serve as hosts or whose prey may host myxosporean infections. A diversity of myxozoans in filtered water samples and in faeces of piscivores (otters and great cormorants) was detected, demonstrating the suitability of lineage-specific amplicons for characterising otherwise difficult to sample parasite communities. The importance of using this approach was highlighted by the lack of myxosporean detection using commonly employed, broadly targeted eukaryotic primers. These results suggest that, despite being frequently present in eDNA samples, myxozoans have been generally overlooked in "eukaryote-wide" surveys. Lineage-specific primers, in contrast, detected 107 operational taxonomic units that were assigned to both the "freshwater" and "marine" myxosporean lineages. Only 7% of these OTUs clustered with sequences in GenBank, providing evidence for substantial undescribed myxosporean diversity. Many new operational taxonomic units, including those found in otter faeces, clustered with a clade of myxosporeans previously characterised by sequences from invertebrate hosts and water samples only. Because myxozoan species identification is heavily reliant on molecular signatures, lineage-specific amplicon sequencing offers an effective and non-destructive means of improving our knowledge of myxozoan diversity. In addition, the analysis of myxozoan DNA in faeces of piscivores offers a potentially efficient method of sampling for diversity and revealing life cycles as piscivore activities may integrate myxozoan infections in fish over relatively broad spatial scales.</p><h1>Barcoding the food chain: from Sanger to high-throughput sequencing</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Society faces the complex challenge of supporting biodiversity and ecosystem functioning, while ensuring food security by providing safe traceable food through an ever-more-complex global food chain. The increase in human mobility brings the added threat of pests, parasites, and invaders that further complicate our agro-industrial efforts. DNA barcoding technologies allow researchers to identify both individual species, and, when combined with universal primers and high-throughput sequencing techniques, the diversity within mixed samples (metabarcoding). These tools are already being employed to detect market substitutions, trace pests through the forensic evaluation of trace "environmental DNA", and to track parasitic infections in livestock. The potential of DNA barcoding to contribute to increased security of the food chain is clear, but challenges remain in regulation and the need for validation of experimental analysis. Here, we present an overview of the current uses and challenges of applied DNA barcoding in agriculture, from agro-ecosystems within farmland to the kitchen table.</p><h1>Long-term dynamics in microbial eukaryotes communities: a palaeolimnological view based on sedimentary DNA</h1>
<p><a href="https://doi.org/10.1111/Mec.13893">doi:10.1111/Mec.13893</a></p>

<p><a href=None>None</a></p>
<p>Assessing the extent to which changes in lacustrine biodiversity are affected by anthropogenic or climatic forces requires extensive palaeolimnological data. We used high-throughput sequencing to generate time-series data encompassing over 2200 years of microbial eukaryotes (protists and Fungi) diversity changes from the sedimentary DNA record of two lakes (Lake Bourget in French Alps and Lake Igaliku in Greenland). From 176 samples, we sequenced a large diversity of microbial eukaryotes, with a total 16 386 operational taxonomic units distributed within 50 phylogenetic groups. Thus, microbial groups, such as Chlorophyta, Dinophyceae, Haptophyceae and Ciliophora, that were not previously considered in lacustrine sediment record analyses appeared to be potential biological markers of trophic status changes. Our data suggest that shifts in relative abundance of extant species, including shifts between rare and abundant taxa, drive ecosystem responses to local and global environmental changes. Community structure shift events were concomitant with major climate variations (more particularly in Lake Igaliku). However, this study shows that the impacts of climatic fluctuations may be overpassed by the high-magnitude eutrophication impacts, as observed in the eutrophicated Lake Bourget. Overall, our data show that DNA preserved in sediment constitutes a precious archive of information on past biodiversity changes.</p><h1>Fungal diversity and seasonal succession in ash leaves infected by the invasive ascomycete Hymenoscyphus fraxineus</h1>
<p><a href="https://doi.org/10.1111/Nph.14204">doi:10.1111/Nph.14204</a></p>

<p><a href=None>None</a></p>
<p>High biodiversity is regarded as a barrier against biological invasions. We hypothesized that the invasion success of the pathogenic ascomycete Hymenoscyphus fraxineus threatening common ash in Europe relates to differences in dispersal and colonization success between the invader and the diverse native competitors. Ash leaf mycobiome was monitored by high-throughput sequencing of the fungal internal transcribed spacer region (ITS) and quantitative PCR profiling of H. fraxineus DNA. Initiation of ascospore production by H. fraxineus after overwintering was followed by pathogen accumulation in asymptomatic leaves. The induction of necrotic leaf lesions coincided with escalation of H. fraxineus DNA levels and changes in proportion of biotrophs, followed by an increase of ubiquitous endophytes with pathogenic potential. H. fraxineus uses high propagule pressure to establish in leaves as quiescent thalli that switch to pathogenic mode once these thalli reach a certain threshold - the massive feedback from the saprophytic phase enables this fungus to challenge host defenses and the resident competitors in mid-season when their density in host tissues is still low. Despite the general correspondence between the ITS-1 and ITS-2 datasets, marker biases were observed, which suggests that multiple barcodes provide better overall representation of mycobiomes.</p><h1>Enhancement of the Knowledge on Fungal Communities in Directly Brined Aloreña de Málaga Green Olive Fermentations by Metabarcoding Analysis</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0163135">doi:10.1371/Journal.Pone.0163135</a></p>

<p><a href=None>None</a></p>
<p>Nowadays, our knowledge of the fungal biodiversity in fermented vegetables is limited although these microorganisms could have a great influence on the quality and safety of this kind of food. This work uses a metagenetic approach to obtain basic knowledge of the fungal community ecology during the course of fermentation of natural Aloreña de Málaga table olives, from reception of raw material to edible fruits. For this purpose, samples of brines and fruits were collected from two industries in Guadalhorce Valley (Málaga, Spain) at different moments of fermentation (0, 7, 30 and 120 days). The physicochemical and microbial counts performed during fermentation showed the typical evolution of this type of processes, mainly dominated by yeasts in apparent absence of Enterobacteriaceae and Lactobacillaceae. High-throughput barcoded pyrosequencing analysis of ITS1-5.8S-ITS2 region showed a low biodiversity of the fungal community, with the presence at 97% identity of 29 different fungal genera included in 105 operational taxonomic units (OTUs). The most important genera in the raw material at the moment of reception in the industry were Penicillium, Cladosporium, Malassezia, and Candida, whilst after 4 months of fermentation in brines Zygotorulaspora and Pichia were predominant, whereas in fruits were Candida, Penicillium, Debaryomyces and Saccharomyces. The fungal genera Penicillium, Pichia, and Zygotorulaspora were shared among the three types of substrates during all the course of fermentation, representing the core fungal population for this table olive specialty. A phylogenetic analysis of the ITS sequences allowed a more accurate assignment of diverse OTUs to Pichia manshurica, Candida parapsilosis/C. tropicalis, Candida diddensiae, and Citeromyces nyonensis clades. This study highlights the existence of a complex fungal consortium in olive fermentations including phytopathogenic, saprofitic, spoilage and fermentative genera. Insights into the ecology, identification and quantification of fungi species in olive fermentation will facilitate the design of new strategies to improve the quality and safety of this fermented vegetable. </p><h1>A quantitative protocol for DNA metabarcoding of springtails (Collembola)</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0228">doi:10.1139/Gen-2015-0228</a></p>

<p><a href=None>None</a></p>
<p>We developed a novel protocol with superior quantitative analysis results for DNA metabarcoding of Collembola, a major soil microarthropod order. Degenerate PCR primers were designed for conserved regions in the mitochondrial cytochrome c oxidase subunit I (mtCOI) and 16S ribosomal RNA (mt16S) genes based on published collembolan mitogenomes. The best primer pair was selected based on its ability to amplify each gene, irrespective of the species. DNA was extracted from 10 natural communities sampled in a temperate forest (with typically 25-30 collembolan species per 10 soil samples) and 10 mock communities (with seven cultured collembolan species). The two gene regions were then amplified using the selected primers, ligated with adapters for 454 technology, and sequenced. Examination of the natural community samples showed that 32 and 36 operational taxonomic units (defined at a 90% sequence similarity threshold) were recovered from the mtCOI and mt16S data, respectively, which were comparable to the results of the microscopic identification of 25 morphospecies. Further, sequence abundances for each collembolan species from the mtCOI and mt16S data of the mock communities, after normalization by using a species as the internal control, showed good correlation with the number of individuals in the samples (R = 0.91-0.99), although relative species abundances within a mock community sample estimated from sequences were skewed from community composition in terms of the number of individuals or biomass of the species. Thus, this protocol enables the comparison of collembolan communities in a quantitative manner by metabarcoding. </p><h1>First evaluation of foraminiferal metabarcoding for monitoring environmental impact from an offshore oil drilling site</h1>
<p><a href="https://doi.org/10.1016/J.Marenvres.2016.08.009">doi:10.1016/J.Marenvres.2016.08.009</a></p>

<p><a href=None>None</a></p>
<p>At present, environmental impacts from offshore oil and gas activities are partly determined by measuring changes in macrofauna diversity. Morphological identification of macrofauna is time-consuming, expensive and dependent on taxonomic expertise. In this study, we evaluated the applicability of using foraminiferal-specific metabarcoding for routine monitoring. Sediment samples were collected along distance gradients from two oil platforms off Taranaki (New Zealand) and their physico-chemical properties, foraminiferal environmental DNA/RNA, and macrofaunal composition analyzed. Macrofaunal and foraminiferal assemblages showed similar shifts along impact gradients, but responded differently to environmental perturbations. Macrofauna were affected by hypoxia, whereas sediment grain size appeared to drive shifts in foraminifera. We identified eight foraminiferal molecular operational taxonomic units that have potential to be used as bioindicator taxa. Our results show that metabarcoding represents an effective tool for assessing foraminiferal communities near offshore oil and gas platforms, and that it can be used to complement current monitoring techniques. </p><h1>Deep COI sequencing of standardized benthic samples unveils overlooked diversity of Jordanian coral reefs in the northern Red Sea</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0208">doi:10.1139/Gen-2015-0208</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing (HTS) of DNA barcodes (metabarcoding), particularly when combined with standardized sampling protocols, is one of the most promising approaches for censusing overlooked cryptic invertebrate communities. We present biodiversity estimates based on sequencing of the cytochrome c oxidase subunit 1 (COI) gene for coral reefs of the Gulf of Aqaba, a semi-enclosed system in the northern Red Sea. Samples were obtained from standardized sampling devices (Autonomous Reef Monitoring Structures (ARMS)) deployed for 18 months. DNA barcoding of non-sessile specimens >2 mm revealed 83 OTUs in six phyla, of which only 25% matched a reference sequence in public databases. Metabarcoding of the 2 mm - 500 μm and sessile bulk fractions revealed 1197 OTUs in 15 animal phyla, of which only 4.9% matched reference barcodes. These results highlight the scarcity of COI data for cryptobenthic organisms of the Red Sea. Compared with data obtained using similar methods, our results suggest that Gulf of Aqaba reefs are less diverse than two Pacific coral reefs but much more diverse than an Atlantic oyster reef at a similar latitude. The standardized approaches used here show promise for establishing baseline data on biodiversity, monitoring the impacts of environmental change, and quantifying patterns of diversity at regional and global scales. </p><h1>Mycamoeba gemmipara nov. gen., nov. sp., the First Cultured Member of the Environmental Dermamoebidae Clade LKM74 and its Unusual Life Cycle</h1>
<p><a href="https://doi.org/10.1111/Jeu.12357">doi:10.1111/Jeu.12357</a></p>

<p><a href=None>None</a></p>
<p>Since the first environmental DNA surveys, entire groups of sequences called "environmental clades" did not have any cultured representative. LKM74 is an amoebozoan clade affiliated to Dermamoebidae, whose presence is pervasively reported in soil and freshwater. We obtained an isolate from soil that we assigned to LKM74 by molecular phylogeny, close related to freshwater clones. We described Mycamoeba gemmipara based on observations made with light- and transmission electron microscopy. It is an extremely small amoeba with typical lingulate shape. Unlike other Dermamoebidae, it lacked ornamentation on its cell membrane, and condensed chromatin formed characteristic patterns in the nucleus. M. gemmipara displayed a unique life cycle: trophozoites formed walled coccoid stages which grew through successive buddings and developed into branched structures holding cysts. These structures, measuring hundreds of micrometres, are built as the exclusive product of osmotrophic feeding. To demonstrate that M. gemmipara is a genuine soil inhabitant, we screened its presence in an environmental soil DNA diversity survey performed on an experimental setup where pig cadavers were left to decompose in soils to follow changes in eukaryotic communities. Mycamoeba gemmipara was present in all samples, although related reads were uncommon underneath the cadaver.</p><h1>Real-time PCR detection of Didemnum perlucidum (Monniot, 1983) and Didemnum vexillum (Kott, 2002) in an applied routine marine biosecurity context</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12581">doi:10.1111/1755-0998.12581</a></p>

<p><a href=None>None</a></p>
<p>Prevention and early detection are well recognized as the best strategies for minimizing the risks posed by nonindigenous species (NIS) that have the potential to become marine pests. Central to this is the ability to rapidly and accurately identify the presence of NIS, often from complex environmental samples like biofouling and ballast water. Molecular tools have been increasingly applied to assist with the identification of NIS and can prove particularly useful for taxonomically difficult groups like ascidians. In this study, we have developed real-time PCR assays suited to the specific identification of the ascidians Didemnum perlucidum and Didemnum vexillum. Despite being recognized as important global pests, this is the first time specific molecular detection methods have been developed that can support the early identification and detection of these species from a broad range of environmental sample types. These fast, robust and high-throughput assays represent powerful tools for routine marine biosecurity surveillance, as detection and confirmation of the early presence of species could assist in the timely establishment of emergency responses and control strategies. This study applied the developed assays to confirm the ability to detect Didemnid eDNA in water samples. While previous work has focused on detection of marine larvae from water samples, the development of real-time PCR assays specifically aimed at detecting eDNA of sessile invertebrate species in the marine environment represents a world first and a significant step forwards in applied marine biosecurity surveillance. Demonstrated success in the detection of D. perlucidum eDNA from water samples at sites where it could not be visually identified suggests value in incorporating such assays into biosecurity survey designs targeting Didemnid species.</p><h1>Scaling up discovery of hidden diversity in fungi: impacts of barcoding approaches</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0336">doi:10.1098/Rstb.2015.0336</a></p>

<p><a href=None>None</a></p>
<p>The fungal kingdom is a hyperdiverse group of multicellular eukaryotes with profound impacts on human society and ecosystem function. The challenge of documenting and describing fungal diversity is exacerbated by their typically cryptic nature, their ability to produce seemingly unrelated morphologies from a single individual and their similarity in appearance to distantly related taxa. This multiplicity of hurdles resulted in the early adoption of DNA-based comparisons to study fungal diversity, including linking curated DNA sequence data to expertly identified voucher specimens. DNA-barcoding approaches in fungi were first applied in specimen-based studies for identification and discovery of taxonomic diversity, but are now widely deployed for community characterization based on sequencing of environmental samples. Collectively, fungal barcoding approaches have yielded important advances across biological scales and research applications, from taxonomic, ecological, industrial and health perspectives. A major outstanding issue is the growing problem of 'sequences without names' that are somewhat uncoupled from the traditional framework of fungal classification based on morphology and preserved specimens. This review summarizes some of the most significant impacts of fungal barcoding, its limitations, and progress towards the challenge of effective utilization of the exponentially growing volume of data gathered from high-throughput sequencing technologies.This article is part of the themed issue 'From DNA barcodes to biomes'.</p><h1>Censusing marine eukaryotic diversity in the twenty-first century</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0331">doi:10.1098/Rstb.2015.0331</a></p>

<p><a href=None>None</a></p>
<p>The ocean constitutes one of the vastest and richest biomes on our planet. Most recent estimations, all based on indirect approaches, suggest that there are millions of marine eukaryotic species. Moreover, a large majority of these are small (less than 1 mm), cryptic and still unknown to science. However, this knowledge gap, caused by the lack of diagnostic morphological features in small organisms and the limited sampling of the global ocean, is currently being filled, thanks to new DNA-based approaches. The molecular technique of PCR amplification of homologous gene regions combined with high-throughput sequencing, routinely used to census unculturable prokaryotes, is now also being used to characterize whole communities of marine eukaryotes. Here, we review how this methodological advancement has helped to better quantify the magnitude and patterns of marine eukaryotic diversity, with an emphasis on taxonomic groups previously largely overlooked. We then discuss obstacles remaining to achieve a global understanding of marine eukaryotic diversity. In particular, we argue that 18S variable regions do not provide sufficient taxonomic resolution to census marine life, and suggest combining broad eukaryotic surveys targeting the 18S rRNA region with more taxon-focused analyses of hypervariable regions to improve our understanding of the diversity of species, the functional units of marine ecosystems.This article is part of the themed issue 'From DNA barcodes to biomes'.</p><h1>A new way to contemplate Darwin's tangled bank: how DNA barcodes are reconnecting biodiversity science and biomonitoring</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0330">doi:10.1098/Rstb.2015.0330</a></p>

<p><a href=None>None</a></p>
<p>Encompassing the breadth of biodiversity in biomonitoring programmes has been frustrated by an inability to simultaneously identify large numbers of species accurately and in a timely fashion. Biomonitoring infers the state of an ecosystem from samples collected and identified using the best available taxonomic knowledge. The advent of DNA barcoding has now given way to the extraction of bulk DNA from mixed samples of organisms in environmental samples through the development of high-throughput sequencing (HTS). This DNA metabarcoding approach allows an unprecedented view of the true breadth and depth of biodiversity, but its adoption poses two important challenges. First, bioinformatics techniques must simultaneously perform complex analyses of large datasets and translate the results of these analyses to a range of users. Second, the insights gained from HTS need to be amalgamated with concepts such as Linnaean taxonomy and indicator species, which are less comprehensive but more intuitive. It is clear that we are moving beyond proof-of-concept studies to address the challenge of implementation of this new approach for environmental monitoring and regulation. Interpreting Darwin's 'tangled bank' through a DNA lens is now a reality, but the question remains: how can this information be generated and used reliably, and how does it relate to accepted norms in ecosystem study?This article is part of the themed issue 'From DNA barcodes to biomes'.</p><h1>Plant diversity accurately predicts insect diversity in two tropical landscapes</h1>
<p><a href="https://doi.org/10.1111/Mec.13770">doi:10.1111/Mec.13770</a></p>

<p><a href=None>None</a></p>
<p>Plant diversity surely determines arthropod diversity, but only moderate correlations between arthropod and plant species richness had been observed until Basset et al. (Science, 338, 2012 and 1481) finally undertook an unprecedentedly comprehensive sampling of a tropical forest and demonstrated that plant species richness could indeed accurately predict arthropod species richness. We now require a high-throughput pipeline to operationalize this result so that we can (i) test competing explanations for tropical arthropod megadiversity, (ii) improve estimates of global eukaryotic species diversity, and (iii) use plant and arthropod communities as efficient proxies for each other, thus improving the efficiency of conservation planning and of detecting forest degradation and recovery. We therefore applied metabarcoding to Malaise-trap samples across two tropical landscapes in China. We demonstrate that plant species richness can accurately predict arthropod (mostly insect) species richness and that plant and insect community compositions are highly correlated, even in landscapes that are large, heterogeneous and anthropogenically modified. Finally, we review how metabarcoding makes feasible highly replicated tests of the major competing explanations for tropical megadiversity.</p><h1>Analysis of Illumina MiSeq Metabarcoding Data: Application to Benthic Indices for Environmental Monitoring</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_16">doi:10.1007/978-1-4939-3774-5_16</a></p>

<p><a href=None>None</a></p>
<p>This protocol details the analysis of Illumina MiSeq amplicon libraries derived from marine benthic macroinvertebrate samples and based on two barcodes of the mitochondrial cytochrome oxidase 1 (CO1) gene: a "short region," covered by overlapping forward and reverse reads and a "long region" for which forward and reverse reads do not overlap. Aside from providing guidelines for analyzing both types of amplicons, we show how amplicon reads can be used for the calculation of benthic indices for environmental monitoring.</p><h1>Visualizing Patterns of Marine Eukaryotic Diversity from Metabarcoding Data Using QIIME</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_15">doi:10.1007/978-1-4939-3774-5_15</a></p>

<p><a href=None>None</a></p>
<p>PCR amplification followed by deep sequencing of homologous gene regions is increasingly used to characterize the diversity and taxonomic composition of marine eukaryotic communities. This approach may generate millions of sequences for hundreds of samples simultaneously. Therefore, tools that researchers can use to visualize complex patterns of diversity for these massive datasets are essential. Efforts by microbiologists to understand the Earth and human microbiomes using high-throughput sequencing of the 16S rRNA gene has led to the development of several user-friendly, open-source software packages that can be similarly used to analyze eukaryotic datasets. Quantitative Insights Into Microbial Ecology (QIIME) offers some of the most helpful data visualization tools. Here, we describe functionalities to import OTU tables generated with any molecular marker (e.g., 18S, COI, ITS) and associated metadata into QIIME. We then present a range of analytical tools implemented within QIIME that can be used to obtain insights about patterns of alpha and beta diversity for marine eukaryotes.</p><h1>Preparation of Amplicon Libraries for Metabarcoding of Marine Eukaryotes Using Illumina MiSeq: The Adapter Ligation Method</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_14">doi:10.1007/978-1-4939-3774-5_14</a></p>

<p><a href=None>None</a></p>
<p>Amplicon-based studies of marine microscopic eukaryotes, also referred to as metabarcoding studies, can be performed to analyze patterns of biodiversity or predator-prey interactions targeting the mitochondrial cytochrome oxidase 1 (CO1) or the small ribosomal subunit (SSU) markers. Because high-throughput sequencing (HTS) Illumina platforms provide millions of reads per run, hundreds of samples may be sequenced simultaneously. This protocol details the preparation of multiplexed amplicon libraries for Illumina MiSeq sequencing. We describe a strategy for sample multiplexing using a combination of tailed PCR primers and ligation of indexed adapters.</p><h1>Metabarcoding Marine Sediments: Preparation of Amplicon Libraries</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_12">doi:10.1007/978-1-4939-3774-5_12</a></p>

<p><a href=None>None</a></p>
<p>The accurate assessment of community composition and ultimately species identification is of utmost importance in any ecological and evolutionary study. Advances in sequencing technologies have allowed the unraveling of levels of biodiversity never imagined before when applied to large-scale environmental DNA studies (also termed metabarcoding/metagenetics/metasystematics/environmental barcoding). Here, we describe a detailed protocol to assess eukaryotic biodiversity in marine sediments, identifying key steps that should not be neglected when preparing Next-Generation Sequencing (NGS) amplicon libraries: DNA extraction, multiple PCR amplification of DNA barcode markers with index/ tag-primers, and final Illumina MiSeq sequencing library preparation.</p><h1>High-throughput metabarcoding of eukaryotic diversity for environmental monitoring of offshore oil-drilling activities</h1>
<p><a href="https://doi.org/10.1111/Mec.13761">doi:10.1111/Mec.13761</a></p>

<p><a href=None>None</a></p>
<p>As global exploitation of available resources increases, operations extend towards sensitive and previously protected ecosystems. It is important to monitor such areas in order to detect, understand and remediate environmental responses to stressors. The natural heterogeneity and complexity of communities means that accurate monitoring requires high resolution, both temporally and spatially, as well as more complete assessments of taxa. Increased resolution and taxonomic coverage is economically challenging using current microscopy-based monitoring practices. Alternatively, DNA sequencing-based methods have been suggested for cost-efficient monitoring, offering additional insights into ecosystem function and disturbance. Here, we applied DNA metabarcoding of eukaryotic communities in marine sediments, in areas of offshore drilling on the Norwegian continental shelf. Forty-five samples, collected from seven drilling sites in the Troll/Oseberg region, were assessed, using the small subunit ribosomal RNA gene as a taxonomic marker. In agreement with results based on classical morphology-based monitoring, we were able to identify changes in sediment communities surrounding oil platforms. In addition to overall changes in community structure, we identified several potential indicator taxa, responding to pollutants associated with drilling fluids. These included the metazoan orders Macrodasyida, Macrostomida and Ceriantharia, as well as several ciliates and other protist taxa, typically not targeted by environmental monitoring programmes. Analysis of a co-occurrence network to study the distribution of taxa across samples provided a framework for better understanding the impact of anthropogenic activities on the benthic food web, generating novel, testable hypotheses of trophic interactions structuring benthic communities.</p><h1>A Bigger Toolbox: Biotechnology in Biodiversity Conservation</h1>
<p><a href="https://doi.org/S0167-7799(16)30091-9">doi:S0167-7799(16)30091-9</a></p>

<p><a href=None>None</a></p>
<p>Conservation biology needs a bigger toolbox to meet unprecedented challenges. Genomics, fueled by declining sequencing costs, offers novel tools with increased precision for genetic questions previously answered with a few molecular markers, as well as completely new possibilities. Metabarcoding promises quicker, cheaper, and more accurate assessments of biodiversity in groups that are difficult to assess by traditional methods, while sequencing low-quality DNA extends the range of useable materials to include museum specimens, archeological remains, and environmental samples. Genomic and transcriptomic data can be used to assess the potential of populations to adapt to new challenges. In the near future, gene-editing tools may help endangered species cope with change, while gene drives control unwanted species and help wanted ones. De-extinction has become a serious prospect.</p><h1>New Primers for Discovering Fungal Diversity Using Nuclear Large Ribosomal DNA</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0159043">doi:10.1371/Journal.Pone.0159043</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding has become an important tool in the discovery of biodiversity, including fungi, which are the second most speciose group of eukaryotes, with diverse and important ecological roles in terrestrial ecosystems. We have designed and tested new PCR primers that target the D1 variable region of nuclear large subunit (LSU) ribosomal DNA; one set that targets the phylum Ascomycota and another that recovers all other fungal phyla. The primers yield amplicons compatible with the Illumina MiSeq platform, which is cost-effective and has a lower error rate than other high throughput sequencing platforms. The new primer set LSU200A-F/LSU476A-R (Ascomycota) yielded 95-98% of reads of target taxa from environmental samples, and primers LSU200-F/LSU481-R (all other fungi) yielded 72-80% of target reads. Both primer sets have fairly low rates of data loss, and together they cover a wide variety of fungal taxa. We compared our results with these primers by amplifying and sequencing a subset of samples using the previously described ITS3_KYO2/ITS4_KYO3 primers, which amplify the internal transcribed spacer 2 (ITS2) of Ascomycota and Basidiomycota. With approximately equivalent read depth, our LSU primers recovered a greater number and phylogenetic diversity of sequences than the ITS2 primers. For instance, ITS3_KYO2/ITS4_KYO3 primers failed to pick up any members of Eurotiales, Mytilinidiales, Pezizales, Saccharomycetales, or Venturiales within Ascomycota, or members of Exobasidiomycetes, Microbotryomycetes, Pucciniomycetes, or Tremellomycetes within Basidiomycota, which were retrieved in good numbers from the same samples by our LSU primers. Among the OTUs recovered using the LSU primers were 127 genera and 28 species that were not obtained using the ITS2 primers, although the ITS2 primers recovered 10 unique genera and 16 species that were not obtained using either of the LSU primers These features identify the new primer sets developed in this study as useful complements to other universal primers for the study of fungal diversity and community composition. </p><h1>The effects of parameter choice on defining molecular operational taxonomic units and resulting ecological analyses of metabarcoding data</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>The combination of DNA barcoding and high-throughput (next-generation) sequencing (metabarcoding) provides many promises but also serious challenges. Generating a reliable comparable estimate of biodiversity remains a central challenge to the application of the technology. Many approaches have been used to turn millions of sequences into distinct taxonomic units. However, the extent to which these methods impact the outcome of simple ecological analyses is not well understood. Here we performed a simple analysis of dietary overlap by skinks and shrews on Ile Aux Aigrettes, Mauritius. We used a combination of filtering thresholds and clustering algorithms on a COI metabarcoding dataset and demonstrate that all bioinformatics parameters will have interacting effects on molecular operational taxonomic unit (MOTU) recovery rates. These effects generated estimates covering two orders of magnitude. However, the effect on a simple ecological analysis was not large and, despite the wide variation in estimates of niche overlap, the same ecological conclusion was drawn in most cases. We advise that a conservative clustering programme coupled with larger sequence divergences to define a cluster, the removal of singletons, rigorous length filtering, and stringent match criteria for Molecular Identifier tags are preferable to avoid MOTU inflation and that the same parameters be used in all comparative analyses.</p><h1>Pollen DNA barcoding: current applications and future prospects</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0200">doi:10.1139/Gen-2015-0200</a></p>

<p><a href=None>None</a></p>
<p>Identification of the species origin of pollen has many applications, including assessment of plant-pollinator networks, reconstruction of ancient plant communities, product authentication, allergen monitoring, and forensics. Such applications, however, have previously been limited by microscopy-based identification of pollen, which is slow, has low taxonomic resolution, and has few expert practitioners. One alternative is pollen DNA barcoding, which could overcome these issues. Recent studies demonstrate that both chloroplast and nuclear barcoding markers can be amplified from pollen. These recent validations of pollen metabarcoding indicate that now is the time for researchers in various fields to consider applying these methods to their research programs. In this paper, we review the nascent field of pollen DNA barcoding and discuss potential new applications of this technology, highlighting existing limitations and future research developments that will improve its utility in a wide range of applications. </p><h1>Large-Scale Monitoring of Plants through Environmental DNA Metabarcoding of Soil: Recovery, Resolution, and Annotation of Four DNA Markers</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0157505">doi:10.1371/Journal.Pone.0157505</a></p>

<p><a href=None>None</a></p>
<p>In a rapidly changing world we need methods to efficiently assess biodiversity in order to monitor ecosystem trends. Ecological monitoring often uses plant community composition to infer quality of sites but conventional aboveground surveys only capture a snapshot of the actively growing plant diversity. Environmental DNA (eDNA) extracted from soil samples, however, can include taxa represented by both active and dormant tissues, seeds, pollen, and detritus. Analysis of this eDNA through DNA metabarcoding provides a more comprehensive view of plant diversity at a site from a single assessment but it is not clear which DNA markers are best used to capture this diversity. Sequence recovery, annotation, and sequence resolution among taxa were evaluated for four established DNA markers (matK, rbcL, ITS2, and the trnL P6 loop) in silico using database sequences and in situ using high throughput sequencing of 35 soil samples from a remote boreal wetland. Overall, ITS2 and rbcL are recommended for DNA metabarcoding of vascular plants from eDNA when not using customized or geographically restricted reference databases. We describe a new framework for evaluating DNA metabarcodes and, contrary to existing assumptions, we found that full length DNA barcode regions could outperform shorter markers for surveying plant diversity from soil samples. By using current DNA barcoding markers rbcL and ITS2 for plant metabarcoding, we can take advantage of existing resources such as the growing DNA barcode database. Our work establishes the value of standard DNA barcodes for soil plant eDNA analysis in ecological investigations and biomonitoring programs and supports the collaborative development of DNA barcoding and metabarcoding. </p><h1>Resilience of Freshwater Communities of Small Microbial Eukaryotes Undergoing Severe Drought Events</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2016.00812">doi:10.3389/Fmicb.2016.00812</a></p>

<p><a href=None>None</a></p>
<p>Small and shallow aquatic ecosystems such as ponds and streams constitute a significant proportion of continental surface waters, especially in temperate zones. In comparison with bigger lakes and rivers, they harbor higher biodiversity but they also exhibit reduced buffering capacity face to environmental shifts, such that climate global change can affect them in a more drastic way. For instance, many temperate areas are predicted to undergo droughts with increasing frequency in the near future, which may lead to the temporal desiccation of streams and ponds. In this work, we monitored temporal dynamics of planktonic communities of microbial eukaryotes (cell size range: 0.2-5 μm) in one brook and one pond that experienced recurrent droughts from 1 to 5 consecutive months during a temporal survey carried out monthly for 2 years based on high-throughput 18S rDNA metabarcoding. During drought-induced desiccation events, protist communities present in the remaining dry sediment, though highly diverse, differed radically from their planktonic counterparts. However, after water refill, the aquatic protist assemblages recovered their original structure within a month. This rapid recovery indicates that these eukaryotic communities are resilient to droughts, most likely via the entrance in dormancy. This property is essential for the long-term survival and functional stability of small freshwater ecosystems. </p><h1>Genetic assessment of meiobenthic community composition and spatial distribution in coastal sediments along northern Gulf of Mexico</h1>
<p><a href="https://doi.org/10.1016/J.Marenvres.2016.05.011">doi:10.1016/J.Marenvres.2016.05.011</a></p>

<p><a href=None>None</a></p>
<p>Meiobenthic (meiofauna and micro-eukaryotes) organisms are important contributors to ecosystem functioning in aquatic environments through their roles in nutrient transport, sediment stability, and food web interactions. Despite their ecological importance, information pertaining to variation of these communities at various spatial and temporal scales is not widely known. Many studies in the Gulf of Mexico (GOM) have focused either on deep sea or continental shelf areas, while little attention has been paid to bays and coastal regions. Herein, we take a holistic approach by using high-throughput sequencing approaches to examine spatial variation in meiobenthic communities within Alabama bays and the coastal northern GOM region. Sediment samples were collected along three transects (Mississippi Sound: MS, FOCAL: FT, and Orange Beach: OB) from September 2010 to April 2012 and community composition was determined by metabarcoding the V9 hypervariable region of the nuclear18S rRNA gene. Results showed that Stramenopiles (diatoms), annelids, arthropods (copepods), and nematodes were the dominate groups within samples, while there was presence of other phyla throughout the dataset. Location played a larger role than time sampled in community composition. However, samples were collected over a short temporal scale. Samples clustered in reference to transect, with the most eastern transect (OB) having a distinct community composition in comparison to the other two transects (MS and FT). Communities also differed in reference to region (Bay versus Shelf). Bulk density and percent inorganic carbon were the only measured environmental factors that were correlated with community composition. </p><h1>Vector soup: high-throughput identification of Neotropical phlebotomine sand flies using metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12556">doi:10.1111/1755-0998.12556</a></p>

<p><a href=None>None</a></p>
<p>Phlebotomine sand flies are haematophagous dipterans of primary medical importance. They represent the only proven vectors of leishmaniasis worldwide and are involved in the transmission of various other pathogens. Studying the ecology of sand flies is crucial to understand the epidemiology of leishmaniasis and further control this disease. A major limitation in this regard is that traditional morphological-based methods for sand fly species identifications are time-consuming and require taxonomic expertise. DNA metabarcoding holds great promise in overcoming this issue by allowing the identification of multiple species from a single bulk sample. Here, we assessed the reliability of a short insect metabarcode located in the mitochondrial 16S rRNA for the identification of Neotropical sand flies, and constructed a reference database for 40 species found in French Guiana. Then, we conducted a metabarcoding experiment on sand flies mixtures of known content and showed that the method allows an accurate identification of specimens in pools. Finally, we applied metabarcoding to field samples caught in a 1-ha forest plot in French Guiana. Besides providing reliable molecular data for species-level assignations of phlebotomine sand flies, our study proves the efficiency of metabarcoding based on the mitochondrial 16S rRNA for studying sand fly diversity from bulk samples. The application of this high-throughput identification procedure to field samples can provide great opportunities for vector monitoring and eco-epidemiological studies.</p><h1>Metabarcoding reveals environmental factors influencing spatio-temporal variation in pelagic micro-eukaryotes</h1>
<p><a href="https://doi.org/10.1111/Mec.13709">doi:10.1111/Mec.13709</a></p>

<p><a href=None>None</a></p>
<p>Marine environments harbour a vast diversity of micro-eukaryotic organisms (protists and other small eukaryotes) that play important roles in structuring marine ecosystems. However, micro-eukaryote diversity is not well understood. Likewise, knowledge is limited regarding micro-eukaryote spatial and seasonal distribution, especially over long temporal scales. Given the importance of this group for mobilizing energy from lower trophic levels near the base of the food chain to larger organisms, assessing community stability, diversity and resilience is important to understand ecosystem health. Herein, we use a metabarcoding approach to examine pelagic micro-eukaryote communities over a 2.5-year time series. Bimonthly surface sampling (July 2009 to December 2011) was conducted at four locations within Mobile Bay (Bay) and along the Alabama continental shelf (Shelf). Alpha-diversity only showed significant differences in Shelf sites, with the greatest differences observed between summer and winter. Beta-diversity showed significant differences in community composition in relation to season and the Bay was dominated by diatoms, while the Shelf was characterized by dinoflagellates and copepods. The northern Gulf of Mexico is heavily influenced by the Mobile River Basin, which brings low-salinity nutrient-rich water mostly during winter and spring. Community composition was correlated with salinity, temperature and dissolved silicate. However, species interactions (e.g. predation and parasitism) may also contribute to the observed variation, especially on the Shelf, which warrants further exploration. Metabarcoding revealed clear patterns in surface pelagic micro-eukaryote communities that were consistent over multiple years, demonstrating how these techniques could be greatly beneficial to ecological monitoring and management over temporal scales.</p><h1>Metabarcoding improves detection of eukaryotes from early biofouling communities: implications for pest monitoring and pathway management</h1>
<p><a href="https://doi.org/10.1080/08927014.2016.1186165">doi:10.1080/08927014.2016.1186165</a></p>

<p><a href=None>None</a></p>
<p>In this experimental study the patterns in early marine biofouling communities and possible implications for surveillance and environmental management were explored using metabarcoding, viz. 18S ribosomal RNA gene barcoding in combination with high-throughput sequencing. The community structure of eukaryotic assemblages and the patterns of initial succession were assessed from settlement plates deployed in a busy port for one, five and 15 days. The metabarcoding results were verified with traditional morphological identification of taxa from selected experimental plates. Metabarcoding analysis identified > 400 taxa at a comparatively low taxonomic level and morphological analysis resulted in the detection of 25 taxa at varying levels of resolution. Despite the differences in resolution, data from both methods were consistent at high taxonomic levels and similar patterns in community shifts were observed. A high percentage of sequences belonging to genera known to contain non-indigenous species (NIS) were detected after exposure for only one day.</p><h1>Advances in DNA metabarcoding for food and wildlife forensic species identification</h1>
<p><a href="https://doi.org/10.1007/S00216-016-9595-8">doi:10.1007/S00216-016-9595-8</a></p>

<p><a href=None>None</a></p>
<p>Species identification using DNA barcodes has been widely adopted by forensic scientists as an effective molecular tool for tracking adulterations in food and for analysing samples from alleged wildlife crime incidents. DNA barcoding is an approach that involves sequencing of short DNA sequences from standardized regions and comparison to a reference database as a molecular diagnostic tool in species identification. In recent years, remarkable progress has been made towards developing DNA metabarcoding strategies, which involves next-generation sequencing of DNA barcodes for the simultaneous detection of multiple species in complex samples. Metabarcoding strategies can be used in processed materials containing highly degraded DNA e.g. for the identification of endangered and hazardous species in traditional medicine. This review aims to provide insight into advances of plant and animal DNA barcoding and highlights current practices and recent developments for DNA metabarcoding of food and wildlife forensic samples from a practical point of view. Special emphasis is placed on new developments for identifying species listed in the Convention on International Trade of Endangered Species (CITES) appendices for which reliable methods for species identification may signal and/or prevent illegal trade. Current technological developments and challenges of DNA metabarcoding for forensic scientists will be assessed in the light of stakeholders' needs. </p><h1>Fungi Sailing the Arctic Ocean: Speciose Communities in North Atlantic Driftwood as Revealed by High-Throughput Amplicon Sequencing</h1>
<p><a href="https://doi.org/10.1007/S00248-016-0778-9">doi:10.1007/S00248-016-0778-9</a></p>

<p><a href=None>None</a></p>
<p>High amounts of driftwood sail across the oceans and provide habitat for organisms tolerating the rough and saline environment. Fungi have adapted to the extremely cold and saline conditions which driftwood faces in the high north. For the first time, we applied high-throughput sequencing to fungi residing in driftwood to reveal their taxonomic richness, community composition, and ecology in the North Atlantic. Using pyrosequencing of ITS2 amplicons obtained from 49 marine logs, we found 807 fungal operational taxonomic units (OTUs) based on clustering at 97 % sequence similarity cut-off level. The phylum Ascomycota comprised 74 % of the OTUs and 20 % belonged to Basidiomycota. The richness of basidiomycetes decreased with prolonged submersion in the sea, supporting the general view of ascomycetes being more extremotolerant. However, more than one fourth of the fungal OTUs remained unassigned to any fungal class, emphasising the need for better DNA reference data from the marine habitat. Different fungal communities were detected in coniferous and deciduous logs. Our results highlight that driftwood hosts a considerably higher fungal diversity than currently known. The driftwood fungal community is not a terrestrial relic but a speciose assemblage of fungi adapted to the stressful marine environment and different kinds of wooden substrates found in it.</p><h1>Habitat conditions and phenological tree traits overrule the influence of tree genotype in the needle mycobiome-Picea glauca system at an arctic treeline ecotone</h1>
<p><a href="https://doi.org/10.1111/Nph.13988">doi:10.1111/Nph.13988</a></p>

<p><a href=None>None</a></p>
<p>Plant-associated mycobiomes in extreme habitats are understudied and poorly understood. We analysed Illumina-generated ITS1 sequences from the needle mycobiome of white spruce (Picea glauca) at the northern treeline in Alaska (USA). Sequences were obtained from the same DNA that was used for tree genotyping. In the present study, fungal metabarcoding and tree microsatellite data were compared for the first time. In general, neighbouring trees shared more fungal taxa with each other than trees growing in further distance. Mycobiomes correlated strongly with phenological host traits and local habitat characteristics contrasting a dense forest stand with an open treeline site. Genetic similarity between trees did not influence fungal composition and no significant correlation existed between needle mycobiome and tree genotype. Our results suggest the pronounced influence of local habitat conditions and phenotypic tree traits on needle-inhabiting fungi. By contrast, the tree genetic identity cannot be benchmarked as a dominant driver for needle-inhabiting mycobiomes, at least not for white spruce in this extreme environment.</p><h1>Testing the potential of a ribosomal 16S marker for DNA metabarcoding of insects</h1>
<p><a href="https://doi.org/10.7717/Peerj.1966">doi:10.7717/Peerj.1966</a></p>

<p><a href=None>None</a></p>
<p>Cytochrome c oxidase I (COI) is a powerful marker for DNA barcoding of animals, with good taxonomic resolution and a large reference database. However, when used for DNA metabarcoding, estimation of taxa abundances and species detection are limited due to primer bias caused by highly variable primer binding sites across the COI gene. Therefore, we explored the ability of the 16S ribosomal DNA gene as an alternative metabarcoding marker for species level assessments. Ten bulk samples, each containing equal amounts of tissue from 52 freshwater invertebrate taxa, were sequenced with the Illumina NextSeq 500 system. The 16S primers amplified three more insect species than the Folmer COI primers and amplified more equally, probably due to decreased primer bias. Estimation of biomass might be less biased with 16S than with COI, although variation in read abundances of two orders of magnitudes is still observed. According to these results, the marker choice depends on the scientific question. If the goal is to obtain a taxonomic identification at the species level, then COI is more appropriate due to established reference databases and known taxonomic resolution of this marker, knowing that a greater proportion of insects will be missed using COI Folmer primers. If the goal is to obtain a more comprehensive survey the 16S marker, which requires building a local reference database, or optimised degenerated COI primers could be more appropriate. </p><h1>Protist metabarcoding and environmental biomonitoring: Time for change</h1>
<p><a href="https://doi.org/S0932-4739(16)30003-7">doi:S0932-4739(16)30003-7</a></p>

<p><a href=None>None</a></p>
<p>High-throughput amplicon sequencing of environmental DNA and/or RNA proved to be a powerful tool to describe protist diversity. This new approach called also the metabarcoding has totally transformed our view of protist diversity, revealing a large number of novel lineages and expanding the range of protist phylogenetic diversity at almost every taxonomic level. However, until now the objectives of the vast majority of metabarcoding studies were purely academic. Practical applications of protist metabarcoding are surprisingly scarce, despite the fact that several groups of protists are commonly used as bioindicators of environmental impacts in freshwater or marine ecosystems. Here, we are reviewing studies that examine the ecological applications of metabarcoding for two groups of well-known protist bioindicators: diatoms and foraminifera. The results of these studies show that despite some biological and technical biases, molecular data quite faithfully reflect the morphology-based biotic indices and provide a similar assessment of ecosystem status. In view of these results, protist metabarcoding appears as a rapid and accurate tool for the evaluation of the quality of aquatic ecosystems. Hence, we plead for integration of protist metabarcoding in future biomonitoring projects as a complement of traditional methods and a source of new biosensors for environmental impact assessment. </p><h1>In silico assessment of primers for eDNA studies using PrimerTree and application to characterize the biodiversity surrounding the Cuyahoga River</h1>
<p><a href="https://doi.org/10.1038/Srep22908">doi:10.1038/Srep22908</a></p>

<p><a href=None>None</a></p>
<p>Analysis of environmental DNA (eDNA) enables the detection of species of interest from water and soil samples, typically using species-specific PCR. Here, we describe a method to characterize the biodiversity of a given environment by amplifying eDNA using primer pairs targeting a wide range of taxa and high-throughput sequencing for species identification. We tested this approach on 91 water samples of 40 mL collected along the Cuyahoga River (Ohio, USA). We amplified eDNA using 12 primer pairs targeting mammals, fish, amphibians, birds, bryophytes, arthropods, copepods, plants and several microorganism taxa and sequenced all PCR products simultaneously by high-throughput sequencing. Overall, we identified DNA sequences from 15 species of fish, 17 species of mammals, 8 species of birds, 15 species of arthropods, one turtle and one salamander. Interestingly, in addition to aquatic and semi-aquatic animals, we identified DNA from terrestrial species that live near the Cuyahoga River. We also identified DNA from one Asian carp species invasive to the Great Lakes but that had not been previously reported in the Cuyahoga River. Our study shows that analysis of eDNA extracted from small water samples using wide-range PCR amplification combined with high-throughput sequencing can provide a broad perspective on biological diversity. </p><h1>A metabarcoding framework for facilitated survey of endolithic phototrophs with tufA</h1>
<p><a href="https://doi.org/10.1186/S12898-016-0068-X">doi:10.1186/S12898-016-0068-X</a></p>

<p><a href=None>None</a></p>
<p>In spite of their ecological importance as primary producers and microbioeroders of marine calcium carbonate (CaCO3) substrata, endolithic phototrophs spanning both prokaryotic (the cyanobacteria) and eukaryotic algae lack established molecular resources for their facilitated survey with high throughput sequencing. Here, the development of a metabarcoding framework for the elongation factor EF-Ttu (tufA) was tested on four Illumina-sequenced marine CaCO3 microfloras for the characterization of their endolithic phototrophs, especially the abundant bioeroding Ostreobium spp. (Ulvophyceae). The framework consists of novel tufA degenerate primers and a comprehensive database enabling Operational Taxonomic Unit (OTU) identification at multiple taxonomic ranks with percent identity thresholds determined herein.</p><h1>Unveiling the Biodiversity of Deep-Sea Nematodes through Metabarcoding: Are We Ready to Bypass the Classical Taxonomy?</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0144928">doi:10.1371/Journal.Pone.0144928</a></p>

<p><a href=None>None</a></p>
<p>Nematodes inhabiting benthic deep-sea ecosystems account for >90% of the total metazoan abundances and they have been hypothesised to be hyper-diverse, but their biodiversity is still largely unknown. Metabarcoding could facilitate the census of biodiversity, especially for those tiny metazoans for which morphological identification is difficult. We compared, for the first time, different DNA extraction procedures based on the use of two commercial kits and a previously published laboratory protocol and tested their suitability for sequencing analyses of 18S rDNA of marine nematodes. We also investigated the reliability of Roche 454 sequencing analyses for assessing the biodiversity of deep-sea nematode assemblages previously morphologically identified. Finally, intra-genomic variation in 18S rRNA gene repeats was investigated by Illumina MiSeq in different deep-sea nematode morphospecies to assess the influence of polymorphisms on nematode biodiversity estimates. Our results indicate that the two commercial kits should be preferred for the molecular analysis of biodiversity of deep-sea nematodes since they consistently provide amplifiable DNA suitable for sequencing. We report that the morphological identification of deep-sea nematodes matches the results obtained by metabarcoding analysis only at the order-family level and that a large portion of Operational Clustered Taxonomic Units (OCTUs) was not assigned. We also show that independently from the cut-off criteria and bioinformatic pipelines used, the number of OCTUs largely exceeds the number of individuals and that 18S rRNA gene of different morpho-species of nematodes displayed intra-genomic polymorphisms. Our results indicate that metabarcoding is an important tool to explore the diversity of deep-sea nematodes, but still fails in identifying most of the species due to limited number of sequences deposited in the public databases, and in providing quantitative data on the species encountered. These aspects should be carefully taken into account before using metabarcoding in quantitative ecological research and monitoring programmes of marine biodiversity. </p><h1>MiFish, a set of universal PCR primers for metabarcoding environmental DNA from fishes: detection of more than 230 subtropical marine species</h1>
<p><a href="https://doi.org/10.1098/Rsos.150088">doi:10.1098/Rsos.150088</a></p>

<p><a href=None>None</a></p>
<p>We developed a set of universal PCR primers (MiFish-U/E) for metabarcoding environmental DNA (eDNA) from fishes. Primers were designed using aligned whole mitochondrial genome (mitogenome) sequences from 880 species, supplemented by partial mitogenome sequences from 160 elasmobranchs (sharks and rays). The primers target a hypervariable region of the 12S rRNA gene (163-185 bp), which contains sufficient information to identify fishes to taxonomic family, genus and species except for some closely related congeners. To test versatility of the primers across a diverse range of fishes, we sampled eDNA from four tanks in the Okinawa Churaumi Aquarium with known species compositions, prepared dual-indexed libraries and performed paired-end sequencing of the region using high-throughput next-generation sequencing technologies. Out of the 180 marine fish species contained in the four tanks with reference sequences in a custom database, we detected 168 species (93.3%) distributed across 59 families and 123 genera. These fishes are not only taxonomically diverse, ranging from sharks and rays to higher teleosts, but are also greatly varied in their ecology, including both pelagic and benthic species living in shallow coastal to deep waters. We also sampled natural seawaters around coral reefs near the aquarium and detected 93 fish species using this approach. Of the 93 species, 64 were not detected in the four aquarium tanks, rendering the total number of species detected to 232 (from 70 families and 152 genera). The metabarcoding approach presented here is non-invasive, more efficient, more cost-effective and more sensitive than the traditional survey methods. It has the potential to serve as an alternative (or complementary) tool for biodiversity monitoring that revolutionizes natural resource management and ecological studies of fish communities on larger spatial and temporal scales. </p><h1>Targeted gene enrichment and high-throughput sequencing for environmental biomonitoring: a case study using freshwater macroinvertebrates</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12488">doi:10.1111/1755-0998.12488</a></p>

<p><a href=None>None</a></p>
<p>Recent studies have advocated biomonitoring using DNA techniques. In this study, two high-throughput sequencing (HTS)-based methods were evaluated: amplicon metabarcoding of the cytochrome C oxidase subunit I (COI) mitochondrial gene and gene enrichment using MYbaits (targeting nine different genes including COI). The gene-enrichment method does not require PCR amplification and thus avoids biases associated with universal primers. Macroinvertebrate samples were collected from 12 New Zealand rivers. Macroinvertebrates were morphologically identified and enumerated, and their biomass determined. DNA was extracted from all macroinvertebrate samples and HTS undertaken using the illumina miseq platform. Macroinvertebrate communities were characterized from sequence data using either six genes (three of the original nine were not used) or just the COI gene in isolation. The gene-enrichment method (all genes) detected the highest number of taxa and obtained the strongest Spearman rank correlations between the number of sequence reads, abundance and biomass in 67% of the samples. Median detection rates across rare (<1% of the total abundance or biomass), moderately abundant (1-5%) and highly abundant (>5%) taxa were highest using the gene-enrichment method (all genes). Our data indicated primer biases occurred during amplicon metabarcoding with greater than 80% of sequence reads originating from one taxon in several samples. The accuracy and sensitivity of both HTS methods would be improved with more comprehensive reference sequence databases. The data from this study illustrate the challenges of using PCR amplification-based methods for biomonitoring and highlight the potential benefits of using approaches, such as gene enrichment, which circumvent the need for an initial PCR step. </p><h1>Molecular diversity and distribution of marine fungi across 130 European environmental samples</h1>
<p><a href="https://doi.org/10.1098/Rspb.2015.2243">doi:10.1098/Rspb.2015.2243</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA and culture-based analyses have suggested that fungi are present in low diversity and in low abundance in many marine environments, especially in the upper water column. Here, we use a dual approach involving high-throughput diversity tag sequencing from both DNA and RNA templates and fluorescent cell counts to evaluate the diversity and relative abundance of fungi across marine samples taken from six European near-shore sites. We removed very rare fungal operational taxonomic units (OTUs) selecting only OTUs recovered from multiple samples for a detailed analysis. This approach identified a set of 71 fungal 'OTU clusters' that account for 66% of all the sequences assigned to the Fungi. Phylogenetic analyses demonstrated that this diversity includes a significant number of chytrid-like lineages that had not been previously described, indicating that the marine environment encompasses a number of zoosporic fungi that are new to taxonomic inventories. Using the sequence datasets, we identified cases where fungal OTUs were sampled across multiple geographical sites and between different sampling depths. This was especially clear in one relatively abundant and diverse phylogroup tentatively named Novel Chytrid-Like-Clade 1 (NCLC1). For comparison, a subset of the water column samples was also investigated using fluorescent microscopy to examine the abundance of eukaryotes with chitin cell walls. Comparisons of relative abundance of RNA-derived fungal tag sequences and chitin cell-wall counts demonstrate that fungi constitute a low fraction of the eukaryotic community in these water column samples. Taken together, these results demonstrate the phylogenetic position and environmental distribution of 71 lineages, improving our understanding of the diversity and abundance of fungi in marine environments. </p><h1>Metabarcoding approach for nonindigenous species surveillance in marine coastal waters</h1>
<p><a href="https://doi.org/S0025-326X(15)30045-X">doi:S0025-326X(15)30045-X</a></p>

<p><a href=None>None</a></p>
<p>In this study, high-throughput sequencing (HTS) metabarcoding was applied for the surveillance of plankton communities within the southeastern (SE) Baltic Sea coastal zone. These results were compared with those from routine monitoring survey and morphological analyses. Four of five nonindigenous species found in the samples were identified exclusively by metabarcoding. All of them are considered as invasive in the Baltic Sea with reported impact on the ecosystem and biodiversity. This study indicates that, despite some current limitations, HTS metabarcoding can provide information on the presence of exotic species and advantageously complement conventional approaches, only requiring the same monitoring effort as before. Even in the currently immature status of HTS, this combination of HTS metabarcoding and observational records is recommended in the early detection of marine pests and delivery of the environmental status metrics of nonindigenous species. </p><h1>High-throughput sequencing and morphology perform equally well for benthic monitoring of marine ecosystems</h1>
<p><a href="https://doi.org/10.1038/Srep13932">doi:10.1038/Srep13932</a></p>

<p><a href=None>None</a></p>
<p>Environmental diversity surveys are crucial for the bioassessment of anthropogenic impacts on marine ecosystems. Traditional benthic monitoring relying on morphotaxonomic inventories of macrofaunal communities is expensive, time-consuming and expertise-demanding. High-throughput sequencing of environmental DNA barcodes (metabarcoding) offers an alternative to describe biological communities. However, whether the metabarcoding approach meets the quality standards of benthic monitoring remains to be tested. Here, we compared morphological and eDNA/RNA-based inventories of metazoans from samples collected at 10 stations around a fish farm in Scotland, including near-cage and distant zones. For each of 5 replicate samples per station, we sequenced the V4 region of the 18S rRNA gene using the Illumina technology. After filtering, we obtained 841,766 metazoan sequences clustered in 163 Operational Taxonomic Units (OTUs). We assigned the OTUs by combining local BLAST searches with phylogenetic analyses. We calculated two commonly used indices: the Infaunal Trophic Index and the AZTI Marine Biotic Index. We found that the molecular data faithfully reflect the morphology-based indices and provides an equivalent assessment of the impact associated with fish farms activities. We advocate that future benthic monitoring should integrate metabarcoding as a rapid and accurate tool for the evaluation of the quality of marine benthic ecosystems. </p><h1>DNA metabarcoding of insects and allies: an evaluation of primers and pipelines</h1>
<p><a href="https://doi.org/10.1017/S0007485315000681">doi:10.1017/S0007485315000681</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding, the coupling of DNA-based species identification and high-throughput sequencing, offers enormous promise for arthropod biodiversity studies but factors such as cost, speed and ease-of-use of bioinformatic pipelines, crucial for making the leapt from demonstration studies to a real-world application, have not yet been adequately addressed. Here, four published and one newly designed primer sets were tested across a diverse set of 80 arthropod species, representing 11 orders, to establish optimal protocols for Illumina-based metabarcoding of tropical Malaise trap samples. Two primer sets which showed the highest amplification success with individual specimen polymerase chain reaction (PCR, 98%) were used for bulk PCR and Illumina MiSeq sequencing. The sequencing outputs were subjected to both manual and simple metagenomics quality control and filtering pipelines. We obtained acceptable detection rates after bulk PCR and high-throughput sequencing (80-90% of input species) but analyses were complicated by putative heteroplasmic sequences and contamination. The manual pipeline produced similar or better outputs to the simple metagenomics pipeline (1.4 compared with 0.5 expected:unexpected Operational Taxonomic Units). Our study suggests that metabarcoding is slowly becoming as cheap, fast and easy as conventional DNA barcoding, and that Malaise trap metabarcoding may soon fulfill its potential, providing a thermometer for biodiversity. </p><h1>Accurate assessment of the impact of salmon farming on benthic sediment enrichment using foraminiferal metabarcoding</h1>
<p><a href="https://doi.org/S0025-326X(15)00526-3">doi:S0025-326X(15)00526-3</a></p>

<p><a href=None>None</a></p>
<p>Assessing the environmental impact of salmon farms on benthic systems is traditionally undertaken using biotic indices derived from microscopic analyses of macrobenthic infaunal (MI) communities. In this study, we tested the applicability of using foraminiferal-specific high-throughput sequencing (HTS) metabarcoding for monitoring these habitats. Sediment samples and physico-chemical data were collected along an enrichment gradient radiating out from three Chinook salmon (Oncorhynchus tshawytscha) farms in New Zealand. HTS of environmental DNA and RNA (eDNA/eRNA) resulted in 1,875,300 sequences that clustered into 349 Operational Taxonomic Units. Strong correlations were observed among various biotic indices calculated from MI data and normalized fourth-root transformed HTS data. Correlations were stronger using eRNA compared to eDNA data. Quantile regression spline analyses identified 12 key foraminiferal taxa that have potential to be used as bioindicator species. This study demonstrates the huge potential for using this method for biomonitoring of fish-farming and other marine industrial activities. </p><h1>The rise and fall of arbuscular mycorrhizal fungal diversity during ecosystem retrogression</h1>
<p><a href="https://doi.org/10.1111/Mec.13363">doi:10.1111/Mec.13363</a></p>

<p><a href=None>None</a></p>
<p>Ecosystem retrogression following long-term pedogenesis is attributed to phosphorus (P) limitation of primary productivity. Arbuscular mycorrhizal fungi (AMF) enhance P acquisition for most terrestrial plants, but it has been suggested that this strategy becomes less effective in strongly weathered soils with extremely low P availability. Using next generation sequencing of the large subunit ribosomal RNA gene in roots and soil, we compared the composition and diversity of AMF communities in three contrasting stages of a retrogressive >2-million-year dune chronosequence in a global biodiversity hotspot. This chronosequence shows a ~60-fold decline in total soil P concentration, with the oldest stage representing some of the most severely P-impoverished soils found in any terrestrial ecosystem. The richness of AMF operational taxonomic units was low on young (1000's of years), moderately P-rich soils, greatest on relatively old (~120 000 years) low-P soils, and low again on the oldest (>2 000 000 years) soils that were lowest in P availability. A similar decline in AMF phylogenetic diversity on the oldest soils occurred, despite invariant host plant diversity and only small declines in host cover along the chronosequence. Differences in AMF community composition were greatest between the youngest and the two oldest soils, and this was best explained by differences in soil P concentrations. Our results point to a threshold in soil P availability during ecosystem regression below which AMF diversity declines, suggesting environmental filtering of AMF insufficiently adapted to extremely low P availability. </p><h1>High-throughput monitoring of wild bee diversity and abundance via mitogenomics</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Bee populations and other pollinators face multiple, synergistically acting threats, which have led to population declines, loss of local species richness and pollination services, and extinctions. However, our understanding of the degree, distribution and causes of declines is patchy, in part due to inadequate monitoring systems, with the challenge of taxonomic identification posing a major logistical barrier. Pollinator conservation would benefit from a high-throughput identification pipeline.We show that the metagenomic mining and resequencing of mitochondrial genomes (mitogenomics) can be applied successfully to bulk samples of wild bees. We assembled the mitogenomes of 48 UK bee species and then shotgun-sequenced total DNA extracted from 204 whole bees that had been collected in 10 pan-trap samples from farms in England and been identified morphologically to 33 species. Each sample data set was mapped against the 48 reference mitogenomes.The morphological and mitogenomic data sets were highly congruent. Out of 63 total species detections in the morphological data set, the mitogenomic data set made 59 correct detections (93·7% detection rate) and detected six more species (putative false positives). Direct inspection and an analysis with species-specific primers suggested that these putative false positives were most likely due to incorrect morphological IDs. Read frequency significantly predicted species biomass frequency (<i>R</i><sup>2</sup> = 24·9%). Species lists, biomass frequencies, extrapolated species richness and community structure were recovered with less error than in a metabarcoding pipeline.Mitogenomics automates the onerous task of taxonomic identification, even for cryptic species, allowing the tracking of changes in species richness and distributions. A mitogenomic pipeline should thus be able to contain costs, maintain consistently high-quality data over long time series, incorporate retrospective taxonomic revisions and provide an auditable evidence trail. Mitogenomic data sets also provide estimates of species counts within samples and thus have potential for tracking population trajectories.</p><h1>Comparative study of the validity of three regions of the 18S-rRNA gene for massively parallel sequencing-based monitoring of the planktonic eukaryote community</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12459">doi:10.1111/1755-0998.12459</a></p>

<p><a href=None>None</a></p>
<p>The nuclear 18S-rRNA gene has been used as a metabarcoding marker in massively parallel sequencing (MPS)-based environmental surveys for plankton biodiversity research. However, different hypervariable regions have been used in different studies, and their utility has been debated among researchers. In this study, detailed investigations into 18S-rRNA were carried out; we investigated the effective number of sequences deposited in international nucleotide sequence databases (INSDs), the amplification bias, and the amplicon sequence variability among the three variable regions, V1-3, V4-5 and V7-9, using in silico polymerase chain reaction (PCR) amplification based on INSDs. We also examined the primer universality and the taxonomic identification power, using MPS-based environmental surveys in the Sea of Okhotsk, to determine which region is more useful for MPS-based monitoring. The primer universality was not significantly different among the three regions, but the number of sequences deposited in INSDs was markedly larger for the V4-5 region than for the other two regions. The sequence variability was significantly different, with the highest variability in the V1-3 region, followed by the V7-9 region, and the lowest variability in the V4-5 region. The results of the MPS-based environmental surveys showed significantly higher identification power in the V1-3 and V7-9 regions than in the V4-5 region, but no significant difference was detected between the V1-3 and V7-9 regions. We therefore conclude that the V1-3 region will be the most suitable for future MPS-based monitoring of natural eukaryote communities, as the number of sequences deposited in INSDs increases.</p><h1>Standardizing metadata and taxonomic identification in metabarcoding studies</h1>
<p><a href="https://doi.org/10.1186/S13742-015-0074-5">doi:10.1186/S13742-015-0074-5</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing-based metabarcoding studies produce vast amounts of ecological data, but a lack of consensus on standardization of metadata and how to refer to the species recovered severely hampers reanalysis and comparisons among studies. Here we propose an automated workflow covering data submission, compression, storage and public access to allow easy data retrieval and inter-study communication. Such standardized and readily accessible datasets facilitate data management, taxonomic comparisons and compilation of global metastudies. </p><h1>Environmental DNA COI barcoding for quantitative analysis of protists communities: A test using the Nebela collaris complex (Amoebozoa; Arcellinida; Hyalospheniidae)</h1>
<p><a href="https://doi.org/10.1016/J.Ejop.2015.06.005">doi:10.1016/J.Ejop.2015.06.005</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA surveys are used for screening eukaryotic diversity. However, it is unclear how quantitative this approach is and to what extent results from environmental DNA studies can be used for ecological studies requiring quantitative data. Mitochondrial cytochrome oxidase (COI) is used for species-level taxonomic studies of testate amoebae and should allow assessing the community composition from environmental samples, thus bypassing biases due to morphological identification. We tested this using a COI clone library approach and focusing on the Nebela collaris complex. Comparisons with direct microscopy counts showed that the COI clone library diversity data matched the morphologically identified taxa, and that community composition estimates using the two approaches were similar. However, this correlation was improved when microscopy counts were corrected for biovolume. Higher correlation with biovolume-corrected community data suggests that COI clone library data matches the ratio of mitochondria and that within closely-related taxa the density of mitochondria per unit biovolume is approximately constant. Further developments of this metabarcoding approach including quantifying the mitochondrial density among closely-related taxa, experiments on other taxonomic groups and using high throughput sequencing should make if possible to quantitatively estimate community composition of different groups, which would be invaluable for microbial food webs studies. </p><h1>Detecting nuisance species using NGST: Methodology shortcomings and possible application in ballast water monitoring</h1>
<p><a href="https://doi.org/10.1016/J.Marenvres.2015.07.002">doi:10.1016/J.Marenvres.2015.07.002</a></p>

<p><a href=None>None</a></p>
<p>Detecting the presence of potential invasive species in ballast water is a priority for preventing their spread into new environments. Next generation sequencing technologies are being increasingly used for exploring and assessing biodiversity from environmental samples. Here we apply high throughput sequencing from DNA extracted from ballast water (BW) samples employing two different platforms, Ion Torrent and 454, and compare the putative species catalogues from the resulting Operational Taxonomic Units (OTU). Water samples were taken from the RV Polastern ballast tank in five different days between the second and the twentieth navigation day. Pronounced decrease of oxygen concentration and increase of temperature occurred in the BW during this time, coincident with a progressively higher proportion of unassigned OTU and short reads indicating DNA degradation. Discrepancy between platforms for species catalogues was consistent with previously published bias in AT-rich sequences for Ion Torrent platform. Some putative species detected from the two platforms increased in frequency during the Polarstern travel, which suggests they were alive and therefore tolerant to adverse conditions. OTU assigned to the highly invasive red alga Polysiphonia have been detected at low but increasing frequency from the two platforms. Although in this moment NGST could not replace current methods of sampling, sorting and individual taxonomic identification of BW biota, it has potential as an exploratory methodology especially for detecting scarce species. </p><h1>Can DNA-Based Ecosystem Assessments Quantify Species Abundance? Testing Primer Bias and Biomass--Sequence Relationships with an Innovative Metabarcoding Protocol</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0130324">doi:10.1371/Journal.Pone.0130324</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding is an emerging genetic tool to rapidly assess biodiversity in ecosystems. It involves high-throughput sequencing of a standard gene from an environmental sample and comparison to a reference database. However, no consensus has emerged regarding laboratory pipelines to screen species diversity and infer species abundances from environmental samples. In particular, the effect of primer bias and the detection limit for specimens with a low biomass has not been systematically examined, when processing samples in bulk. We developed and tested a DNA metabarcoding protocol that utilises the standard cytochrome c oxidase subunit I (COI) barcoding fragment to detect freshwater macroinvertebrate taxa. DNA was extracted in bulk, amplified in a single PCR step, and purified, and the libraries were directly sequenced in two independent MiSeq runs (300-bp paired-end reads). Specifically, we assessed the influence of specimen biomass on sequence read abundance by sequencing 31 specimens of a stonefly species with known haplotypes spanning three orders of magnitude in biomass (experiment I). Then, we tested the recovery of 52 different freshwater invertebrate taxa of similar biomass using the same standard barcoding primers (experiment II). Each experiment was replicated ten times to maximise statistical power. The results of both experiments were consistent across replicates. We found a distinct positive correlation between species biomass and resulting numbers of MiSeq reads. Furthermore, we reliably recovered 83% of the 52 taxa used to test primer bias. However, sequence abundance varied by four orders of magnitudes between taxa despite the use of similar amounts of biomass. Our metabarcoding approach yielded reliable results for high-throughput assessments. However, the results indicated that primer efficiency is highly species-specific, which would prevent straightforward assessments of species abundance and biomass in a sample. Thus, PCR-based metabarcoding assessments of biodiversity should rely on presence-absence metrics. </p><h1>Toward accurate molecular identification of species in complex environmental samples: testing the performance of sequence filtering and clustering methods</h1>
<p><a href="https://doi.org/10.1002/Ece3.1497">doi:10.1002/Ece3.1497</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding has the potential to become a rapid, sensitive, and effective approach for identifying species in complex environmental samples. Accurate molecular identification of species depends on the ability to generate operational taxonomic units (OTUs) that correspond to biological species. Due to the sometimes enormous estimates of biodiversity using this method, there is a great need to test the efficacy of data analysis methods used to derive OTUs. Here, we evaluate the performance of various methods for clustering length variable 18S amplicons from complex samples into OTUs using a mock community and a natural community of zooplankton species. We compare analytic procedures consisting of a combination of (1) stringent and relaxed data filtering, (2) singleton sequences included and removed, (3) three commonly used clustering algorithms (mothur, UCLUST, and UPARSE), and (4) three methods of treating alignment gaps when calculating sequence divergence. Depending on the combination of methods used, the number of OTUs varied by nearly two orders of magnitude for the mock community (60-5068 OTUs) and three orders of magnitude for the natural community (22-22191 OTUs). The use of relaxed filtering and the inclusion of singletons greatly inflated OTU numbers without increasing the ability to recover species. Our results also suggest that the method used to treat gaps when calculating sequence divergence can have a great impact on the number of OTUs. Our findings are particularly relevant to studies that cover taxonomically diverse species and employ markers such as rRNA genes in which length variation is extensive. </p><h1>Divergence thresholds and divergent biodiversity estimates: can metabarcoding reliably describe zooplankton communities?</h1>
<p><a href="https://doi.org/10.1002/Ece3.1485">doi:10.1002/Ece3.1485</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is a promising method for describing communities and estimating biodiversity. This approach uses high-throughput sequencing of targeted markers to identify species in a complex sample. By convention, sequences are clustered at a predefined sequence divergence threshold (often 3%) into operational taxonomic units (OTUs) that serve as a proxy for species. However, variable levels of interspecific marker variation across taxonomic groups make clustering sequences from a phylogenetically diverse dataset into OTUs at a uniform threshold problematic. In this study, we use mock zooplankton communities to evaluate the accuracy of species richness estimates when following conventional protocols to cluster hypervariable sequences of the V4 region of the small subunit ribosomal RNA gene (18S) into OTUs. By including individually tagged single specimens and "populations" of various species in our communities, we examine the impact of intra- and interspecific diversity on OTU clustering. Communities consisting of single individuals per species generated a correspondence of 59-84% between OTU number and species richness at a 3% divergence threshold. However, when multiple individuals per species were included, the correspondence between OTU number and species richness dropped to 31-63%. Our results suggest that intraspecific variation in this marker can often exceed 3%, such that a single species does not always correspond to one OTU. We advocate the need to apply group-specific divergence thresholds when analyzing complex and taxonomically diverse communities, but also encourage the development of additional filtering steps that allow identification of artifactual rRNA gene sequences or pseudogenes that may generate spurious OTUs. </p><h1>Environmental Monitoring: Inferring the Diatom Index from Next-Generation Sequencing Data</h1>
<p><a href="https://doi.org/10.1021/Es506158M">doi:10.1021/Es506158M</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are widely used as bioindicators for the assessment of water quality in rivers and streams. Classically, the diatom biotic indices are based on the relative abundance of morphologically identified species weighted by their autoecological value. Obtaining such indices is time-consuming, costly, and requires excellent taxonomic expertise, which is not always available. Here we tested the possibility to overcome these limitations using a next-generation sequencing (NGS) approach to identify and quantify diatoms found in environmental DNA and RNA samples. We analyzed 27 river sites in the Geneva area (Switzerland), in order to compare the values of the Swiss Diatom Index (DI-CH) computed either by microscopic quantification of diatom species or directly from NGS data. Despite gaps in the reference database and variations in relative abundance of analyzed species, the diatom index shows a significant correlation between morphological and molecular data indicating similar biological quality status for the majority of sites. This proof-of-concept study demonstrates the potential of the NGS approach for identification and quantification of diatoms in environmental samples, opening new avenues toward the routine application of genetic tools for bioassessment and biomonitoring of aquatic ecosystems. </p><h1>Microbial metagenomics in the Baltic Sea: Recent advancements and prospects for environmental monitoring</h1>
<p><a href="https://doi.org/10.1007/S13280-015-0663-7">doi:10.1007/S13280-015-0663-7</a></p>

<p><a href=None>None</a></p>
<p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed. </p><h1>Not all are free-living: high-throughput DNA metabarcoding reveals a diverse community of protists parasitizing soil metazoa</h1>
<p><a href="https://doi.org/10.1111/Mec.13238">doi:10.1111/Mec.13238</a></p>

<p><a href=None>None</a></p>
<p>Protists, the most diverse eukaryotes, are largely considered to be free-living bacterivores, but vast numbers of taxa are known to parasitize plants or animals. High-throughput sequencing (HTS) approaches now commonly replace cultivation-based approaches in studying soil protists, but insights into common biases associated with this method are limited to aquatic taxa and samples. We created a mock community of common free-living soil protists (amoebae, flagellates, ciliates), extracted DNA and amplified it in the presence of metazoan DNA using 454 HTS. We aimed at evaluating whether HTS quantitatively reveals true relative abundances of soil protists and at investigating whether the expected protist community structure is altered by the co-amplification of metazoan-associated protist taxa. Indeed, HTS revealed fundamentally different protist communities from those expected. Ciliate sequences were highly over-represented, while those of most amoebae and flagellates were under-represented or totally absent. These results underpin the biases introduced by HTS that prevent reliable quantitative estimations of free-living protist communities. Furthermore, we detected a wide range of nonadded protist taxa probably introduced along with metazoan DNA, which altered the protist community structure. Among those, 20 taxa most closely resembled parasitic, often pathogenic taxa. Therewith, we provide the first HTS data in support of classical observational studies that showed that potential protist parasites are hosted by soil metazoa. Taken together, profound differences in amplification success between protist taxa and an inevitable co-extraction of protist taxa parasitizing soil metazoa obscure the true diversity of free-living soil protist communities. </p><h1>obitools: a unix-inspired software package for DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12428">doi:10.1111/1755-0998.12428</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding offers new perspectives in biodiversity research. This recently developed approach to ecosystem study relies heavily on the use of next-generation sequencing (NGS) and thus calls upon the ability to deal with huge sequence data sets. The obitools package satisfies this requirement thanks to a set of programs specifically designed for analysing NGS data in a DNA metabarcoding context. Their capacity to filter and edit sequences while taking into account taxonomic annotation helps to set up tailor-made analysis pipelines for a broad range of DNA metabarcoding applications, including biodiversity surveys or diet analyses. The obitools package is distributed as an open source software available on the following website: http://metabarcoding.org/obitools. A Galaxy wrapper is available on the GenOuest core facility toolshed: http://toolshed.genouest.org. </p><h1>Metabarcoding of benthic eukaryote communities predicts the ecological condition of estuaries</h1>
<p><a href="https://doi.org/S0269-7491(15)00177-3">doi:S0269-7491(15)00177-3</a></p>

<p><a href=None>None</a></p>
<p>DNA-derived measurements of biological composition have the potential to produce data covering all of life, and provide a tantalizing proposition for researchers and managers. We used metabarcoding to compare benthic eukaryote composition from five estuaries of varying condition. In contrast to traditional studies, we found biotic richness was greatest in the most disturbed estuary, with this being due to the large volume of extraneous material (i.e. run-off from aquaculture, agriculture and other catchment activities) being deposited in the system. In addition, we found strong correlations between composition and a number of environmental variables, including nutrients, pH and turbidity. A wide range of taxa responded to these environmental gradients, providing new insights into their sensitivities to natural and anthropogenic stressors. Metabarcoding has the capacity to bolster current monitoring techniques, enabling the decisions regarding ecological condition to be based on a more holistic view of biodiversity. </p><h1>The use of genus-specific amplicon pyrosequencing to assess phytophthora species diversity using eDNA from soil and water in Northern Spain</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0119311">doi:10.1371/Journal.Pone.0119311</a></p>

<p><a href=None>None</a></p>
<p>Phytophthora is one of the most important and aggressive plant pathogenic genera in agriculture and forestry. Early detection and identification of its pathways of infection and spread are of high importance to minimize the threat they pose to natural ecosystems. eDNA was extracted from soil and water from forests and plantations in the north of Spain. Phytophthora-specific primers were adapted for use in high-throughput Sequencing (HTS). Primers were tested in a control reaction containing eight Phytophthora species and applied to water and soil eDNA samples from northern Spain. Different score coverage threshold values were tested for optimal Phytophthora species separation in a custom-curated database and in the control reaction. Clustering at 99% was the optimal criteria to separate most of the Phytophthora species. Multiple Molecular Operational Taxonomic Units (MOTUs) corresponding to 36 distinct Phytophthora species were amplified in the environmental samples. Pyrosequencing of amplicons from soil samples revealed low Phytophthora diversity (13 species) in comparison with the 35 species detected in water samples. Thirteen of the MOTUs detected in rivers and streams showed no close match to sequences in international sequence databases, revealing that eDNA pyrosequencing is a useful strategy to assess Phytophthora species diversity in natural ecosystems. </p><h1>PhytoREF: a reference database of the plastidial 16S rRNA gene of photosynthetic eukaryotes with curated taxonomy</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12401">doi:10.1111/1755-0998.12401</a></p>

<p><a href=None>None</a></p>
<p>Photosynthetic eukaryotes have a critical role as the main producers in most ecosystems of the biosphere. The ongoing environmental metabarcoding revolution opens the perspective for holistic ecosystems biological studies of these organisms, in particular the unicellular microalgae that often lack distinctive morphological characters and have complex life cycles. To interpret environmental sequences, metabarcoding necessarily relies on taxonomically curated databases containing reference sequences of the targeted gene (or barcode) from identified organisms. To date, no such reference framework exists for photosynthetic eukaryotes. In this study, we built the PhytoREF database that contains 6490 plastidial 16S rDNA reference sequences that originate from a large diversity of eukaryotes representing all known major photosynthetic lineages. We compiled 3333 amplicon sequences available from public databases and 879 sequences extracted from plastidial genomes, and generated 411 novel sequences from cultured marine microalgal strains belonging to different eukaryotic lineages. A total of 1867 environmental Sanger 16S rDNA sequences were also included in the database. Stringent quality filtering and a phylogeny-based taxonomic classification were applied for each 16S rDNA sequence. The database mainly focuses on marine microalgae, but sequences from land plants (representing half of the PhytoREF sequences) and freshwater taxa were also included to broaden the applicability of PhytoREF to different aquatic and terrestrial habitats. PhytoREF, accessible via a web interface (http://phytoref.fr), is a new resource in molecular ecology to foster the discovery, assessment and monitoring of the diversity of photosynthetic eukaryotes using high-throughput sequencing. </p><h1>Reconstructing long-term human impacts on plant communities: an ecological approach based on lake sediment DNA</h1>
<p><a href="https://doi.org/10.1111/Mec.13136">doi:10.1111/Mec.13136</a></p>

<p><a href=None>None</a></p>
<p>Paleoenvironmental studies are essential to understand biodiversity changes over long timescales and to assess the relative importance of anthropogenic and environmental factors. Sedimentary ancient DNA (sedaDNA) is an emerging tool in the field of paleoecology and has proven to be a complementary approach to the use of pollen and macroremains for investigating past community changes. SedaDNA-based reconstructions of ancient environments often rely on indicator taxa or expert knowledge, but quantitative ecological analyses might provide more objective information. Here, we analysed sedaDNA to investigate plant community trajectories in the catchment of a high-elevation lake in the Alps over the last 6400 years. We combined data on past and present plant species assemblages along with sedimentological and geochemical records to assess the relative impact of human activities through pastoralism, and abiotic factors (temperature and soil evolution). Over the last 6400 years, we identified significant variation in plant communities, mostly related to soil evolution and pastoral activities. An abrupt vegetational change corresponding to the establishment of an agropastoral landscape was detected during the Late Holocene, approximately 4500 years ago, with the replacement of mountain forests and tall-herb communities by heathlands and grazed lands. Our results highlight the importance of anthropogenic activities in mountain areas for the long-term evolution of local plant assemblages. SedaDNA data, associated with other paleoenvironmental proxies and present plant assemblages, appear to be a relevant tool for reconstruction of plant cover history. Their integration, in conjunction with classical tools, offers interesting perspectives for a better understanding of long-term ecosystem dynamics under the influence of human-induced and environmental drivers. </p><h1>Environmental genes and genomes: understanding the differences and challenges in the approaches and software for their analyses</h1>
<p><a href="https://doi.org/10.1093/Bib/Bbv001">doi:10.1093/Bib/Bbv001</a></p>

<p><a href=None>None</a></p>
<p>DNA-based taxonomic and functional profiling is widely used for the characterization of organismal communities across a rapidly increasing array of research areas that include the role of microbiomes in health and disease, biomonitoring, and estimation of both microbial and metazoan species richness. Two principal approaches are currently used to assign taxonomy to DNA sequences: DNA metabarcoding and metagenomics. When initially developed, each of these approaches mandated their own particular methods for data analysis; however, with the development of high-throughput sequencing (HTS) techniques they have begun to share many aspects in data set generation and processing. In this review we aim to define the current characteristics, goals and boundaries of each field, and describe the different software used for their analysis. We argue that an appreciation of the potential and limitations of each method can help underscore the improvements required by each field so as to better exploit the richness of current HTS-based data sets. </p><h1>Metabarcoding approach for the ballast water surveillance--an advantageous solution or an awkward challenge?</h1>
<p><a href="https://doi.org/S0025-326X(15)00024-7">doi:S0025-326X(15)00024-7</a></p>

<p><a href=None>None</a></p>
<p>Transfer of organisms with ships' ballast water is recognized as a major pathway of non-indigenous species introduction and addressed in a few recent legislative initiatives. Among other they imply scientific and technical research and monitoring to be conducted in a efficient and reliable way. The recent development of DNA barcoding and metabarcoding technologies opens new opportunities for biodiversity and biosecurity surveillance. In the current study, the performance of metabarcoding approach was assessed in comparison to the conventional (visual) observations, during the en route experimental ballast water survey. Opportunities and limitations of the molecular method were identified from taxonomical datasets rendered by two molecular markers of different degree of universality - the universal cytochrome oxydase sub-unit I gene and a fragment of RuBisCO gene. The cost-efficacy and possible improvements of these methods are discussed for the further successful development and implementation of the approach in ballast water control and NIS surveillance. </p><h1>Deciphering diversity and ecological function from marine metagenomes</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Metagenomic sequencing now represents a common, powerful approach for investigating diversity and functional relationships in marine ecosystems. High-throughput datasets generated from random fragments of environmental DNA can provide a less biased view of organismal abundance (versus PCR-based amplicon sequencing) and enable novel exploration of microbial genomes by recovering genome assemblies from uncultured species, identifying ecological functions, and reconstructing metabolic pathways. This review highlights the current state of knowledge in marine metagenomics, focusing on biological insights gained from recent environmental studies and detailing commonly employed methods for data collection and analysis. </p><h1>Next-generation environmental diversity surveys of foraminifera: preparing the future</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Foraminifera are commonly defined as marine testate protists, and their diversity is mainly assessed on the basis of the morphology of their agglutinated or mineralized tests. Diversity surveys based on environmental DNA (eDNA) have dramatically changed this view by revealing an unexpected diversity of naked and organic-walled lineages as well as detecting foraminiferal lineages in soil and freshwater environments. Moreover, single-cell analyses have allowed discrimination among genetically distinctive types within almost every described morphospecies. In view of these studies, the foraminiferal diversity appeared to be largely underestimated, but its accurate estimation was impeded by the low speed and coverage of a cloning-based eDNA approach. With the advent of high-throughput sequencing (HTS) technologies, these limitations disappeared in favor of exhaustive descriptions of foraminiferal diversity in numerous samples. Yet, the biases and errors identified in early HTS studies raised some questions about the accuracy of HTS data and their biological interpretation. Among the most controversial issues affecting the reliability of HTS diversity estimates are (1) the impact of technical and biological biases, (2) the sensitivity and specificity of taxonomic sequence assignment, (3) the ability to distinguish rare species, and (4) the quantitative interpretation of HTS data. Here, we document the lessons learned from previous HTS surveys and present the current advances and applications focusing on foraminiferal eDNA. We discuss the problems associated with HTS approaches and predict the future trends and avenues that hold promises for surveying foraminiferal diversity accurately and efficiently. </p><h1>Multiplex sequencing of pooled mitochondrial genomes-a crucial step toward biodiversity analysis using mito-metagenomics</h1>
<p><a href="https://doi.org/10.1093/Nar/Gku917">doi:10.1093/Nar/Gku917</a></p>

<p><a href=None>None</a></p>
<p>The advent in high-throughput-sequencing (HTS) technologies has revolutionized conventional biodiversity research by enabling parallel capture of DNA sequences possessing species-level diagnosis. However, polymerase chain reaction (PCR)-based implementation is biased by the efficiency of primer binding across lineages of organisms. A PCR-free HTS approach will alleviate this artefact and significantly improve upon the multi-locus method utilizing full mitogenomes. Here we developed a novel multiplex sequencing and assembly pipeline allowing for simultaneous acquisition of full mitogenomes from pooled animals without DNA enrichment or amplification. By concatenating assemblies from three de novo assemblers, we obtained high-quality mitogenomes for all 49 pooled taxa, with 36 species >15 kb and the remaining >10 kb, including 20 complete mitogenomes and nearly all protein coding genes (99.6%). The assembly quality was carefully validated with Sanger sequences, reference genomes and conservativeness of protein coding genes across taxa. The new method was effective even for closely related taxa, e.g. three Drosophila spp., demonstrating its broad utility for biodiversity research and mito-phylogenomics. Finally, the in silico simulation showed that by recruiting multiple mito-loci, taxon detection was improved at a fixed sequencing depth. Combined, these results demonstrate the plausibility of a multi-locus mito-metagenomics approach as the next phase of the current single-locus metabarcoding method. </p><h1>Intracellular diversity of the V4 and V9 regions of the 18S rRNA in marine protists (radiolarians) assessed by high-throughput sequencing</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0104297">doi:10.1371/Journal.Pone.0104297</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding is a powerful tool for exploring microbial diversity in the environment, but its accurate interpretation is impeded by diverse technical (e.g. PCR and sequencing errors) and biological biases (e.g. intra-individual polymorphism) that remain poorly understood. To help interpret environmental metabarcoding datasets, we investigated the intracellular diversity of the V4 and V9 regions of the 18S rRNA gene from Acantharia and Nassellaria (radiolarians) using 454 pyrosequencing. Individual cells of radiolarians were isolated, and PCRs were performed with generalist primers to amplify the V4 and V9 regions. Different denoising procedures were employed to filter the pyrosequenced raw amplicons (Acacia, AmpliconNoise, Linkage method). For each of the six isolated cells, an average of 541 V4 and 562 V9 amplicons assigned to radiolarians were obtained, from which one numerically dominant sequence and several minor variants were found. At the 97% identity, a diversity metrics commonly used in environmental surveys, up to 5 distinct OTUs were detected in a single cell. However, most amplicons grouped within a single OTU whereas other OTUs contained very few amplicons. Different analytical methods provided evidence that most minor variants forming different OTUs correspond to PCR and sequencing artifacts. Duplicate PCR and sequencing from the same DNA extract of a single cell had only 9 to 16% of unique amplicons in common, and alignment visualization of V4 and V9 amplicons showed that most minor variants contained substitutions in highly-conserved regions. We conclude that intracellular variability of the 18S rRNA in radiolarians is very limited despite its multi-copy nature and the existence of multiple nuclei in these protists. Our study recommends some technical guidelines to conservatively discard artificial amplicons from metabarcoding datasets, and thus properly assess the diversity and richness of protists in the environment. </p><h1>An Illumina metabarcoding pipeline for fungi</h1>
<p><a href="https://doi.org/10.1002/Ece3.1107">doi:10.1002/Ece3.1107</a></p>

<p><a href=None>None</a></p>
<p>High-throughput metabarcoding studies on fungi and other eukaryotic microorganisms are rapidly becoming more frequent and more complex, requiring researchers to handle ever increasing amounts of raw sequence data. Here, we provide a flexible pipeline for pruning and analyzing fungal barcode (ITS rDNA) data generated as paired-end reads on Illumina MiSeq sequencers. The pipeline presented includes specific steps fine-tuned for ITS, that are mostly missing from pipelines developed for prokaryotes. It (1) employs state of the art programs and follows best practices in fungal high-throughput metabarcoding; (2) consists of modules and scripts easily modifiable by the user to ensure maximum flexibility with regard to specific needs of a project or future methodological developments; and (3) is straightforward to use, also in classroom settings. We provide detailed descriptions and revision techniques for each step, thus giving the user maximum control over data treatment and avoiding a black-box approach. Employing this pipeline will improve and speed up the tedious and error-prone process of cleaning fungal Illumina metabarcoding data. </p><h1>Efficient and sensitive identification and quantification of airborne pollen using next-generation DNA sequencing</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12288">doi:10.1111/1755-0998.12288</a></p>

<p><a href=None>None</a></p>
<p>Pollen monitoring is an important and widely used tool in allergy research and creation of awareness in pollen-allergic patients. Current pollen monitoring methods are microscope-based, labour intensive and cannot identify pollen to the genus level in some relevant allergenic plant groups. Therefore, a more efficient, cost-effective and sensitive method is needed. Here, we present a method for identification and quantification of airborne pollen using DNA sequencing. Pollen is collected from ambient air using standard techniques. DNA is extracted from the collected pollen, and a fragment of the chloroplast gene trnL is amplified using PCR. The PCR product is subsequently sequenced on a next-generation sequencing platform (Ion Torrent). Amplicon molecules are sequenced individually, allowing identification of different sequences from a mixed sample. We show that this method provides an accurate qualitative and quantitative view of the species composition of samples of airborne pollen grains. We also show that it correctly identifies the individual grass genera present in a mixed sample of grass pollen, which cannot be achieved using microscopic pollen identification. We conclude that our method is more efficient and sensitive than current pollen monitoring techniques and therefore has the potential to increase the throughput of pollen monitoring. </p><h1>Next generation restoration genetics: applications and opportunities</h1>
<p><a href="https://doi.org/10.1016/J.Tplants.2014.03.011">doi:10.1016/J.Tplants.2014.03.011</a></p>

<p><a href=None>None</a></p>
<p>Restoration ecology is a young scientific discipline underpinning improvements in the rapid global expansion of ecological restoration. The application of molecular tools over the past 20 years has made an important contribution to understanding genetic factors influencing ecological restoration success. Here we illustrate how recent advances in next generation sequencing (NGS) methods are revolutionising the practical contribution of genetics to restoration. Novel applications include a dramatically enhanced capacity to measure adaptive variation for optimal seed sourcing, high-throughput assessment and monitoring of natural and restored biological communities aboveground and belowground, and gene expression analysis as a measure of genetic resilience of restored populations. Challenges remain in data generation, handling and analysis, and how best to apply NGS for practical outcomes in restoration. </p><h1>Environmental monitoring through protist next-generation sequencing metabarcoding: assessing the impact of fish farming on benthic foraminifera communities</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12261">doi:10.1111/1755-0998.12261</a></p>

<p><a href=None>None</a></p>
<p>The measurement of species diversity represents a powerful tool for assessing the impacts of human activities on marine ecosystems. Traditionally, the impact of fish farming on the coastal environment is evaluated by monitoring the dynamics of macrobenthic infaunal populations. However, taxonomic sorting and morphology-based identification of the macrobenthos demand highly trained specialists and are extremely time-consuming and costly, making it unsuitable for large-scale biomonitoring efforts involving numerous samples. Here, we propose to alleviate this laborious task by developing protist metabarcoding tools based on next-generation sequencing (NGS) of environmental DNA and RNA extracted from sediment samples. In this study, we analysed the response of benthic foraminiferal communities to the variation of environmental gradients associated with salmon farms in Scotland. We investigated the foraminiferal diversity based on ribosomal minibarcode sequences generated by the Illumina NGS technology. We compared the molecular data with morphospecies counts and with environmental gradients, including distance to cages and redox used as a proxy for sediment oxygenation. Our study revealed high variations between foraminiferal communities collected in the vicinity of fish farms and at distant locations. We found evidence for species richness decrease in impacted sites, especially visible in the RNA data. We also detected some candidate bioindicator foraminiferal species. Based on this proof-of-concept study, we conclude that NGS metabarcoding using foraminifera and other protists has potential to become a new tool for surveying the impact of aquaculture and other industrial activities in the marine environment.</p><h1>Ancient DNA sheds new light on the Svalbard foraminiferal fossil record of the last millennium</h1>
<p><a href="https://doi.org/10.1111/Gbi.12087">doi:10.1111/Gbi.12087</a></p>

<p><a href=None>None</a></p>
<p>Recent palaeogenetic studies have demonstrated the occurrence of preserved ancient DNA (aDNA) in various types of fossilised material. Environmental aDNA sequences assigned to modern species have been recovered from marine sediments dating to the Pleistocene. However, the match between the aDNA and the fossil record still needs to be evaluated for the environmental DNA approaches to be fully exploited. Here, we focus on foraminifera in sediments up to one thousand years old retrieved from the Hornsund fjord (Svalbard). We compared the diversity of foraminiferal microfossil assemblages with the diversity of aDNA sequenced from subsurface sediment samples using both cloning and high-throughput sequencing (HTS). Our study shows that 57% of the species archived in the fossil record were also detected in the aDNA data. However, the relative abundance of aDNA sequence reads and fossil specimens differed considerably. We also found a limited match between the stratigraphic occurrence of some fossil species and their aDNA sequences, especially in the case of rare taxa. The aDNA data comprised a high proportion of non-fossilised monothalamous species, which are known to dominate in modern foraminiferal communities of the Svalbard region. Our results confirm the relevance of HTS for studying past micro-eukaryotic diversity and provide insight into its ability to reflect fossil assemblages. Palaeogenetic studies including aDNA analyses of non-fossilised groups expand the range of palaeoceanographical proxies and therefore may increase the accuracy of palaeoenvironmental reconstructions. </p><h1>Profiling the diversity of microbial communities with single-strand conformation polymorphism (SSCP)</h1>
<p><a href="https://doi.org/10.1007/978-1-62703-712-9_6">doi:10.1007/978-1-62703-712-9_6</a></p>

<p><a href=None>None</a></p>
<p>Genetic fingerprinting techniques for microbial community analysis have evolved over the last decade into standard applications for efficient and fast differentiation of microbial communities based on their diversity. These techniques commonly analyze the diversity of PCR products amplified from extracted environmental DNA usually utilizing primers hybridizing to suspected conserved regions of the targeted genes. In comparison to the more commonly applied terminal restriction fragment length polymorphism (TRFLP) or denaturing gradient gel electrophoresis (DGGE) techniques, the here-described single-strand conformation polymorphism (SSCP) fingerprinting technique features some advantageous key characteristics. (1) Primers for the polymerase chain reaction (PCR) do only need minimal 5'-end alterations; (2) SSCP is adaptable to high throughput applications in automated sequencers; and (3) a second dimension in the SSCP gel electrophoresis can be implemented to obtain high resolution 2D gels. One central key requirement for SSCP gel electrophoresis is a tight temperature control. Gels that run at different temperatures will produce entirely different fingerprints. This can be exploited for an improved analysis of highly diverse communities by running the same template at different temperatures or by 2D-SSCP gel electrophoresis. </p><h1>Fifty thousand years of Arctic vegetation and megafaunal diet</h1>
<p><a href="https://doi.org/10.1038/Nature12921">doi:10.1038/Nature12921</a></p>

<p><a href=None>None</a></p>
<p>Although it is generally agreed that the Arctic flora is among the youngest and least diverse on Earth, the processes that shaped it are poorly understood. Here we present 50 thousand years (kyr) of Arctic vegetation history, derived from the first large-scale ancient DNA metabarcoding study of circumpolar plant diversity. For this interval we also explore nematode diversity as a proxy for modelling vegetation cover and soil quality, and diets of herbivorous megafaunal mammals, many of which became extinct around 10 kyr bp (before present). For much of the period investigated, Arctic vegetation consisted of dry steppe-tundra dominated by forbs (non-graminoid herbaceous vascular plants). During the Last Glacial Maximum (25-15 kyr bp), diversity declined markedly, although forbs remained dominant. Much changed after 10 kyr bp, with the appearance of moist tundra dominated by woody plants and graminoids. Our analyses indicate that both graminoids and forbs would have featured in megafaunal diets. As such, our findings question the predominance of a Late Quaternary graminoid-dominated Arctic mammoth steppe. </p><h1>Using environmental DNA to census marine fishes in a large mesocosm</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0086175">doi:10.1371/Journal.Pone.0086175</a></p>

<p><a href=None>None</a></p>
<p>The ocean is a soup of its resident species' genetic material, cast off in the forms of metabolic waste, shed skin cells, or damaged tissue. Sampling this environmental DNA (eDNA) is a potentially powerful means of assessing whole biological communities, a significant advance over the manual methods of environmental sampling that have historically dominated marine ecology and related fields. Here, we estimate the vertebrate fauna in a 4.5-million-liter mesocosm aquarium tank at the Monterey Bay Aquarium of known species composition by sequencing the eDNA from its constituent seawater. We find that it is generally possible to detect mitochondrial DNA of bony fishes sufficient to identify organisms to taxonomic family- or genus-level using a 106 bp fragment of the 12S ribosomal gene. Within bony fishes, we observe a low false-negative detection rate, although we did not detect the cartilaginous fishes or sea turtles present with this fragment. We find that the rank abundance of recovered eDNA sequences correlates with the abundance of corresponding species' biomass in the mesocosm, but the data in hand do not allow us to develop a quantitative relationship between biomass and eDNA abundance. Finally, we find a low false-positive rate for detection of exogenous eDNA, and we were able to diagnose non-native species' tissue in the food used to maintain the mesocosm, underscoring the sensitivity of eDNA as a technique for community-level ecological surveys. We conclude that eDNA has substantial potential to become a core tool for environmental monitoring, but that a variety of challenges remain before reliable quantitative assessments of ecological communities in the field become possible. </p><h1>Improved inference of taxonomic richness from environmental DNA</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0071974">doi:10.1371/Journal.Pone.0071974</a></p>

<p><a href=None>None</a></p>
<p>Accurate estimation of biological diversity in environmental DNA samples using high-throughput amplicon pyrosequencing must account for errors generated by PCR and sequencing. We describe a novel approach to distinguish the underlying sequence diversity in environmental DNA samples from errors that uses information on the abundance distribution of similar sequences across independent samples, as well as the frequency and diversity of sequences within individual samples. We have further refined this approach into a bioinformatics pipeline, Amplicon Pyrosequence Denoising Program (APDP) that is able to process raw sequence datasets into a set of validated sequences in formats compatible with commonly used downstream analyses packages. We demonstrate, by sequencing complex environmental samples and mock communities, that APDP is effective for removing errors from deeply sequenced datasets comprising biological and technical replicates, and can efficiently denoise single-sample datasets. APDP provides more conservative diversity estimates for complex datasets than other approaches; however, for some applications this may provide a more accurate and appropriate level of resolution, and result in greater confidence that returned sequences reflect the diversity of the underlying sample. </p><h1>Quantifying sequence proportions in a DNA-based diet study using Ion Torrent amplicon sequencing: which counts count?</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12103">doi:10.1111/1755-0998.12103</a></p>

<p><a href=None>None</a></p>
<p>A goal of many environmental DNA barcoding studies is to infer quantitative information about relative abundances of different taxa based on sequence read proportions generated by high-throughput sequencing. However, potential biases associated with this approach are only beginning to be examined. We sequenced DNA amplified from faeces (scats) of captive harbour seals (Phoca vitulina) to investigate whether sequence counts could be used to quantify the seals' diet. Seals were fed fish in fixed proportions, a chordate-specific mitochondrial 16S marker was amplified from scat DNA and amplicons sequenced using an Ion Torrent PGM™. For a given set of bioinformatic parameters, there was generally low variability between scat samples in proportions of prey species sequences recovered. However, proportions varied substantially depending on sequencing direction, level of quality filtering (due to differences in sequence quality between species) and minimum read length considered. Short primer tags used to identify individual samples also influenced species proportions. In addition, there were complex interactions between factors; for example, the effect of quality filtering was influenced by the primer tag and sequencing direction. Resequencing of a subset of samples revealed some, but not all, biases were consistent between runs. Less stringent data filtering (based on quality scores or read length) generally produced more consistent proportional data, but overall proportions of sequences were very different than dietary mass proportions, indicating additional technical or biological biases are present. Our findings highlight that quantitative interpretations of sequence proportions generated via high-throughput sequencing will require careful experimental design and thoughtful data analysis.</p><h1>Unveiling the diet of elusive rainforest herbivores in next generation sequencing era? The tapir as a case study</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0060799">doi:10.1371/Journal.Pone.0060799</a></p>

<p><a href=None>None</a></p>
<p>Characterizing the trophic relationships between large herbivores and the outstanding plant diversity in rainforest is a major challenge because of their elusiveness. This is crucial to understand the role of these herbivores in the functioning of the rainforest ecosystems. We tested a non-invasive approach based on the high-throughput sequencing of environmental samples using small plant plastid sequences (the trnL P6 loop) and ribosomal ITS1 primers, referred to as DNA metabarcoding, to investigate the diet of the largest neotropical herbivore, the lowland tapir. Sequencing was performed on plant DNA extracted from tapir faeces collected at the Nouragues station, a protected area of French Guiana. In spite of a limited sampling, our approach reliably provided information about the lowland tapir's diet at this site. Indeed, 95.1% and 74.4% of the plant families and genera identified thanks to the trnL P6 loop, respectively, matched with taxa already known to be consumed by tapirs. With this approach we were able to show that two families and eight new genera are also consumed by the lowland tapir. The taxonomic resolution of this method is limited to the plant family and genera. Complementary barcodes, such as a small portion of ITS1, can be used to efficiently narrow identifications down to the species in some problematic families. We will discuss the remaining limitations of this approach and how useful it is at this stage to unravel the diet of elusive rainforest herbivores and better understand their role as engineers of the ecosystem.</p><h1>High-throughput analysis of ammonia oxidiser community composition via a novel, amoA-based functional gene array</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0051542">doi:10.1371/Journal.Pone.0051542</a></p>

<p><a href=None>None</a></p>
<p>Advances in microbial ecology research are more often than not limited by the capabilities of available methodologies. Aerobic autotrophic nitrification is one of the most important and well studied microbiological processes in terrestrial and aquatic ecosystems. We have developed and validated a microbial diagnostic microarray based on the ammonia-monooxygenase subunit A (amoA) gene, enabling the in-depth analysis of the community structure of bacterial and archaeal ammonia oxidisers. The amoA microarray has been successfully applied to analyse nitrifier diversity in marine, estuarine, soil and wastewater treatment plant environments. The microarray has moderate costs for labour and consumables and enables the analysis of hundreds of environmental DNA or RNA samples per week per person. The array has been thoroughly validated with a range of individual and complex targets (amoA clones and environmental samples, respectively), combined with parallel analysis using traditional sequencing methods. The moderate cost and high throughput of the microarray makes it possible to adequately address broader questions of the ecology of microbial ammonia oxidation requiring high sample numbers and high resolution of the community composition.</p><h1>Testing three pipelines for 18S rDNA-based metabarcoding of soil faunal diversity</h1>
<p><a href="https://doi.org/10.1007/S11427-012-4423-7">doi:10.1007/S11427-012-4423-7</a></p>

<p><a href=None>None</a></p>
<p>A number of basic and applied questions in ecology and environmental management require the characterization of soil and leaf litter faunal diversity. Recent advances in high-throughput sequencing of barcode-gene amplicons ('metabarcoding') have made it possible to survey biodiversity in a robust and efficient way. However, one obstacle to the widespread adoption of this technique is the need to choose amongst many candidates for bioinformatic processing of the raw sequencing data. We compare three candidate pipelines for the processing of 18S small subunit rDNA metabarcode data from solid substrates: (i) USEARCH/CROP, (ii) Denoiser/UCLUST, and (iii) OCTUPUS. The three pipelines produced reassuringly similar and highly correlated assessments of community composition that are dominated by taxa known to characterize the sampled environments. However, OCTUPUS appears to inflate phylogenetic diversity, because of higher sequence noise. We therefore recommend either the USEARCH/CROP or Denoiser/UCLUST pipelines, both of which can be run within the QIIME (Quantitative Insights Into Microbial Ecology) environment.</p><h1>Incorporating 16S gene copy number information improves estimates of microbial diversity and abundance</h1>
<p><a href="https://doi.org/10.1371/Journal.Pcbi.1002743">doi:10.1371/Journal.Pcbi.1002743</a></p>

<p><a href=None>None</a></p>
<p>The abundance of different SSU rRNA ("16S") gene sequences in environmental samples is widely used in studies of microbial ecology as a measure of microbial community structure and diversity. However, the genomic copy number of the 16S gene varies greatly - from one in many species to up to 15 in some bacteria and to hundreds in some microbial eukaryotes. As a result of this variation the relative abundance of 16S genes in environmental samples can be attributed both to variation in the relative abundance of different organisms, and to variation in genomic 16S copy number among those organisms. Despite this fact, many studies assume that the abundance of 16S gene sequences is a surrogate measure of the relative abundance of the organisms containing those sequences. Here we present a method that uses data on sequences and genomic copy number of 16S genes along with phylogenetic placement and ancestral state estimation to estimate organismal abundances from environmental DNA sequence data. We use theory and simulations to demonstrate that 16S genomic copy number can be accurately estimated from the short reads typically obtained from high-throughput environmental sequencing of the 16S gene, and that organismal abundances in microbial communities are more strongly correlated with estimated abundances obtained from our method than with gene abundances. We re-analyze several published empirical data sets and demonstrate that the use of gene abundance versus estimated organismal abundance can lead to different inferences about community diversity and structure and the identity of the dominant taxa in microbial communities. Our approach will allow microbial ecologists to make more accurate inferences about microbial diversity and abundance based on 16S sequence data.</p><h1>DNA from soil mirrors plant taxonomic and growth form diversity</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05545.X">doi:10.1111/J.1365-294X.2012.05545.X</a></p>

<p><a href=None>None</a></p>
<p>Ecosystems across the globe are threatened by climate change and human activities. New rapid survey approaches for monitoring biodiversity would greatly advance assessment and understanding of these threats. Taking advantage of next-generation DNA sequencing, we tested an approach we call metabarcoding: high-throughput and simultaneous taxa identification based on a very short (usually <100 base pairs) but informative DNA fragment. Short DNA fragments allow the use of degraded DNA from environmental samples. All analyses included amplification using plant-specific versatile primers, sequencing and estimation of taxonomic diversity. We tested in three steps whether degraded DNA from dead material in soil has the potential of efficiently assessing biodiversity in different biomes. First, soil DNA from eight boreal plant communities located in two different vegetation types (meadow and heath) was amplified. Plant diversity detected from boreal soil was highly consistent with plant taxonomic and growth form diversity estimated from conventional above-ground surveys. Second, we assessed DNA persistence using samples from formerly cultivated soils in temperate environments. We found that the number of crop DNA sequences retrieved strongly varied with years since last cultivation, and crop sequences were absent from nearby, uncultivated plots. Third, we assessed the universal applicability of DNA metabarcoding using soil samples from tropical environments: a large proportion of species and families from the study site were efficiently recovered. The results open unprecedented opportunities for large-scale DNA-based biodiversity studies across a range of taxonomic groups using standardized metabarcoding approaches.</p><h1>Towards next-generation biodiversity assessment using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05470.X">doi:10.1111/J.1365-294X.2012.05470.X</a></p>

<p><a href=None>None</a></p>
<p>Virtually all empirical ecological studies require species identification during data collection. DNA metabarcoding refers to the automated identification of multiple species from a single bulk sample containing entire organisms or from a single environmental sample containing degraded DNA (soil, water, faeces, etc.). It can be implemented for both modern and ancient environmental samples. The availability of next-generation sequencing platforms and the ecologists' need for high-throughput taxon identification have facilitated the emergence of DNA metabarcoding. The potential power of DNA metabarcoding as it is implemented today is limited mainly by its dependency on PCR and by the considerable investment needed to build comprehensive taxonomic reference libraries. Further developments associated with the impressive progress in DNA sequencing will eliminate the currently required DNA amplification step, and comprehensive taxonomic reference libraries composed of whole organellar genomes and repetitive ribosomal nuclear DNA can be built based on the well-curated DNA extract collections maintained by standardized barcoding initiatives. The near-term future of DNA metabarcoding has an enormous potential to boost data acquisition in biodiversity research.</p><h1>Next-generation sequencing technologies for environmental DNA research</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05538.X">doi:10.1111/J.1365-294X.2012.05538.X</a></p>

<p><a href=None>None</a></p>
<p>Since 2005, advances in next-generation sequencing technologies have revolutionized biological science. The analysis of environmental DNA through the use of specific gene markers such as species-specific DNA barcodes has been a key application of next-generation sequencing technologies in ecological and environmental research. Access to parallel, massive amounts of sequencing data, as well as subsequent improvements in read length and throughput of different sequencing platforms, is leading to a better representation of sample diversity at a reasonable cost. New technologies are being developed rapidly and have the potential to dramatically accelerate ecological and environmental research. The fast pace of development and improvements in next-generation sequencing technologies can reflect on broader and more robust applications in environmental DNA research. Here, we review the advantages and limitations of current next-generation sequencing technologies in regard to their application for environmental DNA analysis.</p><h1>Monitoring endangered freshwater biodiversity using environmental DNA</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2011.05418.X">doi:10.1111/J.1365-294X.2011.05418.X</a></p>

<p><a href=None>None</a></p>
<p>Freshwater ecosystems are among the most endangered habitats on Earth, with thousands of animal species known to be threatened or already extinct. Reliable monitoring of threatened organisms is crucial for data-driven conservation actions but remains a challenge owing to nonstandardized methods that depend on practical and taxonomic expertise, which is rapidly declining. Here, we show that a diversity of rare and threatened freshwater animals--representing amphibians, fish, mammals, insects and crustaceans--can be detected and quantified based on DNA obtained directly from small water samples of lakes, ponds and streams. We successfully validate our findings in a controlled mesocosm experiment and show that DNA becomes undetectable within 2 weeks after removal of animals, indicating that DNA traces are near contemporary with presence of the species. We further demonstrate that entire faunas of amphibians and fish can be detected by high-throughput sequencing of DNA extracted from pond water. Our findings underpin the ubiquitous nature of DNA traces in the environment and establish environmental DNA as a tool for monitoring rare and threatened species across a wide range of taxonomic groups.</p><h1>Eukaryotic richness in the abyss: insights from pyrotag sequencing</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0018169">doi:10.1371/Journal.Pone.0018169</a></p>

<p><a href=None>None</a></p>
<p>The deep sea floor is considered one of the most diverse ecosystems on Earth. Recent environmental DNA surveys based on clone libraries of rRNA genes confirm this observation and reveal a high diversity of eukaryotes present in deep-sea sediment samples. However, environmental clone-library surveys yield only a modest number of sequences with which to evaluate the diversity of abyssal eukaryotes.</p><h1>Multiple marker parallel tag environmental DNA sequencing reveals a highly complex eukaryotic community in marine anoxic water</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2009.04480.X">doi:10.1111/J.1365-294X.2009.04480.X</a></p>

<p><a href=None>None</a></p>
<p>Sequencing of ribosomal DNA clone libraries amplified from environmental DNA has revolutionized our understanding of microbial eukaryote diversity and ecology. The results of these analyses have shown that protist groups are far more genetically heterogeneous than their morphological diversity suggests. However, the clone library approach is labour-intensive, relatively expensive, and methodologically biased. Therefore, even the most intensive rDNA library analyses have recovered only small samples of much larger assemblages, indicating that global environments harbour a vast array of unexplored biodiversity. High-throughput parallel tag 454 sequencing offers an unprecedented scale of sampling for molecular detection of microbial diversity. Here, we report a 454 protocol for sampling and characterizing assemblages of eukaryote microbes. We use this approach to sequence two SSU rDNA diversity markers-the variable V4 and V9 regions-from 10 L of anoxic Norwegian fjord water. We identified 38 116 V4 and 15 156 V9 unique sequences. Both markers detect a wide range of taxonomic groups but in both cases the diversity detected was dominated by dinoflagellates and close relatives. Long-tailed rank abundance curves suggest that the 454 sequencing approach provides improved access to rare genotypes. Most tags detected represent genotypes not currently in GenBank, although many are similar to database sequences. We suggest that current understanding of the ecological complexity of protist communities, genetic diversity, and global species richness are severely limited by the sequence data hitherto available, and we discuss the biological significance of this high amplicon diversity.</p><h1>Accurate quantification of microorganisms in PCR-inhibiting environmental DNA extracts by a novel internal amplification control approach using Biotrove OpenArrays</h1>
<p><a href="https://doi.org/10.1128/Aem.00796-09">doi:10.1128/Aem.00796-09</a></p>

<p><a href=None>None</a></p>
<p>PCR-based detection assays are prone to inhibition by substances present in environmental samples, thereby potentially leading to inaccurate target quantification or false-negative results. Internal amplification controls (IACs) have been developed to help alleviate this problem but are generally applied in a single concentration, thereby yielding less-than-optimal results across the wide range of microbial gene target concentrations possible in environmental samples (J. Hoorfar, B. Malorny, A. Abdulmawjood, N. Cook, M. Wagner, and P. Fach, J. Clin. Microbiol. 42:1863-1868, 2004). Increasing the number of IACs for each quantitative PCR (qPCR) sample individually, however, typically reduces sensitivity and, more importantly, the reliability of quantification. Fortunately, current advances in high-throughput qPCR platforms offer the possibility of multiple reactions for a single sample simultaneously, thereby allowing the implementation of more than one IAC concentration per sample. Here, we describe the development of a novel IAC approach that is specifically designed for the state-of-the-art Biotrove OpenArray platform. Different IAC targets were applied at a range of concentrations, yielding a calibration IAC curve for each individual DNA sample. The developed IACs were optimized, tested, and validated by using more than 5,000 unique qPCR amplifications, allowing accurate quantification of microorganisms when applied to soil DNA extracts containing various levels of PCR-inhibiting compounds. To our knowledge, this is the first study using a suite of IACs at different target concentrations to monitor PCR inhibition across a wide target range, thereby allowing reliable and accurate quantification of microorganisms in PCR-inhibiting DNA extracts. The developed IAC is ideally suited for high-throughput screenings of, for example, ecological and agricultural samples on next-generation qPCR platforms.</p><h1>The ITS region as a target for characterization of fungal communities using emerging sequencing technologies</h1>
<p><a href="https://doi.org/10.1111/J.1574-6968.2009.01618.X">doi:10.1111/J.1574-6968.2009.01618.X</a></p>

<p><a href=None>None</a></p>
<p>The advent of new high-throughput DNA-sequencing technologies promises to redefine the way in which fungi and fungal communities--as well as other groups of organisms--are studied in their natural environment. With read lengths of some few hundred base pairs, massively parallel sequencing (pyrosequencing) stands out among the new technologies as the most apt for large-scale species identification in environmental samples. Although parallel pyrosequencing can generate hundreds of thousands of sequences at an exceptional speed, the limited length of the reads may pose a problem to the species identification process. This study explores whether the discrepancy in read length between parallel pyrosequencing and traditional (Sanger) sequencing will have an impact on the perceived taxonomic affiliation of the underlying species. Based on all 39,200 publicly available fungal environmental DNA sequences representing the nuclear ribosomal internal transcribed spacer (ITS) region, the results show that the two approaches give rise to quite different views of the diversity of the underlying samples. Standardization of which subregion from the ITS region should be sequenced, as well as a recognition that the composition of fungal communities as depicted through different sequencing methods need not be directly comparable, appear crucial to the integration of the new sequencing technologies with current mycological praxis.</p><h1>Biodiversity, chemical diversity and drug discovery</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Drugs developed from microbial natural products are in the fundaments of modern pharmaceutical companies. Despite decades of research, all evidences suggest that there must remain many interesting natural molecules with potential therapeutic application yet to be discovered. Any efforts to successfully exploit the chemical diversity of microbial secondary metabolites need to rely heavily on a good understanding of microbial diversity, being the working hypothesis that maximizing biological diversity is the key strategy to maximizing chemical diversity. This chapter presents an overview of diverse topics related with this basic principle, always in relation with the discovery of novel secondary metabolites. The types of microorganisms more frequently used for natural products discovery are briefly reviewed, as well as the differences between terrestrial and marine habitats as sources of bioactive secondary metabolite producers. The concepts about microbial diversity as applied to prokaryotes have evolved in the last years, but recent data suggest the existence of true biogeographic patterns of bacterial diversity, which are also discussed. Special attention is dedicated to the existing strategies to exploit the microbial diversity that is not easy to tackle by conventional approaches. This refers explicitly to the current attempts to isolate and cultivate the previously uncultured bacteria, including the application of high throughput techniques. Likewise, the advances of microbial molecular biology has allowed the development of metagenomic approaches, i.e., the expression of biosynthetic pathways directly obtained from environmental DNA and cloned in a suitable host, as another way of accessing microbial genetic resources. Also, approaches relying on the genomics of metabolite producers are reviewed.</p><h1>Development and evaluation of a meat mitochondrial metagenomic (3MG) method for composition determination of meat from fifteen mammalian and avian species</h1>
<p><a href="https://doi.org/10.1186/s12864-021-08263-0">doi:10.1186/s12864-021-08263-0</a></p>

<p><a href=None>None</a></p>
<p>Background: Bioassessment and biomonitoring of meat products are aimed at identifying and quantifying adulterants and contaminants, such as meat from unexpected sources and microbes. Several methods for determining the biological composition of mixed samples have been used, including metabarcoding, metagenomics and mitochondrial metagenomics. In this study, we aimed to develop a method based on next-generation DNA sequencing to estimate samples that might contain meat from 15 mammalian and avian species that are commonly related to meat bioassessment and biomonitoring. Results: In this project, we found the meat composition from 15 species could not be identified with the metabarcoding approach because of the lack of universal primers or insufficient discrimination power. Consequently, we developed and evaluated a meat mitochondrial metagenomics (3MG) method. The 3MG method has four steps: (1) extraction of sequencing reads from mitochondrial genomes (mitogenomes); (2) assembly of mitogenomes; (3) mapping of mitochondrial reads to the assembled mitogenomes; and (4) biomass estimation based on the number of uniquely mapped reads. The method was implemented in a python script called 3MG. The analysis of simulated datasets showed that the method can determine contaminant composition at a proportion of 2% and the relative error was < 5%. To evaluate the performance of 3MG, we constructed and analysed mixed samples derived from 15 animal species in equal mass. Then, we constructed and analysed mixed samples derived from two animal species (pork and chicken) in different ratios. DNAs were extracted and used in constructing 21 libraries for next-generation sequencing. The analysis of the 15 species mix with the method showed the successful identification of 12 of the 15 (80%) animal species tested. The analysis of the mixed samples of the two species revealed correlation coefficients of 0.98 for pork and 0.98 for chicken between the number of uniquely mapped reads and the mass proportion. Conclusion: To the best of our knowledge, this study is the first to demonstrate the potential of the non-targeted 3MG method as a tool for accurately estimating biomass in meat mix samples. The method has potential broad applications in meat product safety.</p><h1>Freeze-drying can replace cold-chains for transport and storage of fecal microbiome samples</h1>
<p><a href="https://doi.org/10.7717/peerj.13095">doi:10.7717/peerj.13095</a></p>

<p><a href=None>None</a></p>
<p>Background. The transport and storage of samples in temperatures of minus 80 °C is commonly considered as the gold standard for microbiome studies. However, studies conducting sample collection at remote sites without a reliable cold-chain would benefit from a sample preservation method that allows transport and storage at ambient temperature. Methods. In this study we compare alpha diversity and 16S microbiome composition of 20 fecal sample replicates from Damaraland mole-rats (Fukomys damarensis) preserved in a minus 80 °C freezer and transported on dry ice to freeze-dried samples that were stored and transported in ambient temperature until DNA extraction. Results. We found strong correlations between relative abundances of Amplicon Sequence Variants (ASVs) between preservation treatments of the sample, no differences in alpha diversity measures between the two preservation treatments and minor effects of the preservation treatment on beta diversity measures. Our results show that freezedrying samples can be a useful method for cost-effective transportation and storage of microbiome samples that yields quantitatively almost indistinguishable results in 16S microbiome analyses as those stored in minus 80 °C.</p><h1>Compatibility of Diatom Valve Records With Sedimentary Ancient DNA Amplicon Data: A Case Study in a Brackish, Alkaline Tibetan Lake</h1>
<p><a href="https://doi.org/10.3389/feart.2022.824656">doi:10.3389/feart.2022.824656</a></p>

<p><a href=None>None</a></p>
<p>Lake sediments represent valuable and widely used archives for tracking environmental and biotic changes over time. Past aquatic communities are traditionally studied via morphological identification of the remains of organisms. However, molecular identification tools, such as DNA metabarcoding, have revolutionized the field of biomonitoring by enabling high-throughput and fast identification of organisms from environmental samples (e.g., sediments and soil). Sedimentary ancient DNA (sedaDNA) metabarcoding, an approach to track the biodiversity of target organisms from sediment cores, spanning thousands of years, has been successfully applied in many studies. However, researchers seldom explore how well the signals from sedaDNA data correlate with the fossil records of target organisms. This information is essential to infer past environmental conditions and community changes of bioindicators when the increasingly popular molecular identification method, metabarcoding, is desired instead of a morphological identification approach. In this study, we explore the correlations of diatom valve records across the last ∼940 years with the diatom sedaDNA metabarcoding data from the same sediment core from lake Nam Co (Tibetan Plateau). Overall, the results from valve vs. sedaDNA data revealed concordant diatom richness as well as community patterns. However, several mismatches in the diatom taxonomic composition existed between the data sets. In general, sedaDNA data harbored much higher diatom diversity, but due to the lack of reference sequences in public databases, many molecular units (amplicon sequence variants) remained unclassified to lower taxonomic levels. As our study lake, Nam Co, is characterized by brackish water and alkaline pH, some likely cases for the observed taxonomic composition mismatches may be due to a valve dissolution issue. Nevertheless, significant drivers for the diatom richness and community structure largely corresponded between data sets. Both valve and sedaDNA data demonstrated similar breakpoints for historical diatom community shifts. A particularly strong shift in the diatom community structure occurred after ∼1950 CE, which may be associated with abrupt environmental changes on the Tibetan Plateau. Altogether, our study indicates that environmentally driven signals reflected by the diatom communities are successfully recovered via microfossil as well as molecular identification methods.</p><h1>Assessment of ITS1, ITS2, 5<sup>′</sup>-ETS, and trnL-F DNA Barcodes for Metabarcoding of Poaceae Pollen</h1>
<p><a href="https://doi.org/10.3390/d14030191">doi:10.3390/d14030191</a></p>

<p><a href=None>None</a></p>
<p>Grass pollen is one of the major causes of allergy. Aerobiological monitoring is a necessary element of the complex of anti-allergic measures, but the similar pollen morphology of Poaceae species makes it challenging to discriminate species in airborne pollen mixes, which impairs the quality of aerobiological monitoring. One of the solutions to this problem is the metabarcoding approach employing DNA barcodes for taxonomical identification of species in a mix by high-throughput sequencing of the pollen DNA. A diverse set of 14 grass species of different genera were selected to create a local reference database of nuclear ITS1, ITS2, 5′-ETS, and plastome trnL-F DNA barcodes. Sequences for the database were Sanger sequenced from live field and herbarium specimens and collected from GenBank. New Poaceae-specific primers for 5′-ETS were designed and tested to obtain a 5′-ETS region less than 600 bp long, suitable for high-throughput sequencing. The DNA extraction method for single-species pollen samples and mixes was optimized to increase the yield for amplification and sequencing of pollen DNA. Barcode sequences were analyzed and compared by the barcoding gap and intra-and interspecific distances. Their capability to correctly identify grass pollen was tested on artificial pollen mixes of various complexity. Metabarcoding analysis of the artificial pollen mixes showed that nuclear DNA barcodes ITS1, ITS2, and 5′-ETS proved to be more efficient than the plastome barcode in both amplification from pollen DNA and identification of grass species. Although the metabarcoding results were qualitatively congruent with the actual composition of the pollen mixes in most cases, the quantitative results based on read-counts did not match the actual ratio of pollen grains in the mixes.</p><h1>An assessment of minimum sequence copy thresholds for identifying and reducing the prevalence of artefacts in dietary metabarcoding data</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13780">doi:10.1111/2041-210X.13780</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding provides a powerful tool for investigating biodiversity and trophic interactions, but the high sensitivity of this methodology makes it vulnerable to errors, resulting in artefacts in the final data. Metabarcoding studies thus often utilise minimum sequence copy thresholds (MSCTs) to remove artefacts that remain in datasets; however, there is no consensus on best practice for the use of MSCTs. To mitigate erroneous reporting of results and inconsistencies, this study discusses and provides guidance for best-practice filtering of metabarcoding data for the ascertainment of conservative and accurate data. Several of the most commonly used MSCTs were applied to example datasets of Eurasian otter Lutra lutra and cereal crop spider (Araneae: Linyphiidae and Lycosidae) diets. Changes in both the method and threshold value considerably affected the resultant data. Of the MSCTs tested, it was concluded that the optimal method for the examples given combined a sample-based threshold with removal of maximum taxon contamination, providing stringent filtering of artefacts while retaining target data. Choice of threshold value differed between datasets due to variation in artefact abundance and sequencing depth, thus studies should employ controls (mock communities, negative controls with no DNA and unused MID tag combinations) to select threshold values appropriate for each individual study.</p><h1>Biodiversity exploration in autumn using environmental DNA in the South China sea</h1>
<p><a href="https://doi.org/10.1016/j.envres.2021.112357">doi:10.1016/j.envres.2021.112357</a></p>

<p><a href=None>None</a></p>
<p>The South China Sea (SCS) is an important part of the Indo-Pacific convergence zone, with high biodiversity and abundant marine resources. Traditional methods are primarily used to monitor biodiversity. However, a few studies have used environmental DNA (eDNA) metabarcoding to research the assemblage structure of the SCS. This study used eDNA metabarcoding to survey the SCS assemblage and its relationship with environmental factors over a month-long time-series (August 30th to September 30th, 2020) of seawater samples from the central part of the SCS (9°–20°86′ N, 113°–118°47′ E). 32 stations were divided into six groups (A, B, C, D, E, F) according to longitude. We collected water samples, extracted eDNA, and amplified 18S rRNA gene V4 region (18S V4), 18S rRNA gene V9 region (18S V9), and 12S rRNA gene (12S). Krona diagrams were used to show species composition. We identified 192 phytoplankton, 104 invertebrate, and 61 fish species from 18S V4, 18S V9, and 12S, respectively. Generally, the three assemblage structures exhibited an increase in species diversity with increasing longitude. Group E had the highest fish diversity. Groups F and C had the highest phytoplankton and invertebrate diversity, respectively. Canonical correspondence analysis showed that four factors (chlorophyll a, depth, salinity, and temperature) were correlated with assemblage structure. Chlorophyll a was the main environmental factor that affected fish, phytoplankton, and invertebrate assemblage structures; salinity was strongly correlated with fish and invertebrate assemblage structures; temperature was a key factor that impacted fish and invertebrate assemblage structures; and depth was strongly correlated with invertebrate assemblage structure. Our results revealed that eDNA metabarcoding is a powerful tool for improving detection rate and using multiple markers is an effective approach for monitoring biodiversity. This study provided information that can be used to enhance biodiversity protection efforts in the SCS.</p><h1>The use of environmental DNA metabarcoding and quantitative PCR for molecular detection of marine invasive non-native species associated with artificial structures</h1>
<p><a href="https://doi.org/10.1007/s10530-021-02672-8">doi:10.1007/s10530-021-02672-8</a></p>

<p><a href=None>None</a></p>
<p>Artificial coastal structures associated with coastal defences, energy generation, ports, marinas and other developments, are known to support lower levels of biodiversity than natural coastal environments and tend to be hotspots of invasive non-native species (INNS). In the present study, we attempted to detect INNS through both quantitative (q)PCR and metabarcoding of environmental (e)DNA from seawater samples. A mitochondrial COI based species-specific qPCR assay was developed and deployed to detect Didemnum vexillum, a colonial tunicate that has successfully become established at coastal sites across Europe. Our targeted qPCR assay was able to detect D. vexillum in eDNA seawater samples from all sampled sites where it is currently found in Ireland and Wales. Through metabarcoding of the same eDNA samples, we detected an established INNS at all sites but not D. vexillum even in locations where it is present. We conclude that our qPCR approach is effective for sensitive and targeted screening for specific INNS at coastal sites including those with artificial structures, and while metabarcoding is a less sensitive approach it is a valuable tool to detect a broad taxonomic range of native and non-native species.</p><h1>Dataset complexity impacts both MOTU delimitation and biodiversity estimates in eukaryotic 18S rRNA metabarcoding studies</h1>
<p><a href="https://doi.org/10.1002/edn3.255">doi:10.1002/edn3.255</a></p>

<p><a href=None>None</a></p>
<p>How does the evolution of bioinformatics tools impact the biological interpretation of high-throughput sequencing datasets? For eukaryotic metabarcoding studies, in particular, researchers often rely on tools originally developed for the analysis of 16S ribosomal RNA (rRNA) datasets. Such tools do not adequately account for the complexity of eukaryotic genomes, the ubiquity of intragenomic variation in eukaryotic metabarcoding loci, or the differential evolutionary rates observed across eukaryotic genes and taxa. Recently, metabarcoding workflows have shifted away from the use of operational taxonomic units (OTUs) toward delimitation of amplicon sequence variants (ASVs). We assessed how the choice of bioinformatics algorithm impacts the downstream biological conclusions that are drawn from eukaryotic 18S rRNA metabarcoding studies. We focused on four workflows including UCLUST and VSearch algorithms for OTU clustering, and DADA2 and Deblur algorithms for ASV delimitation. We used two 18S rRNA datasets to further evaluate whether dataset complexity had a major impact on the statistical trends and ecological metrics: a “high complexity” (HC) environmental dataset generated from community DNA in Arctic marine sediments, and a “low complexity” (LC) dataset representing individually barcoded nematodes. Our results indicate that ASV algorithms produce more biologically realistic metabarcoding outputs, with DADA2 being the most consistent and accurate pipeline regardless of dataset complexity. In contrast, OTU clustering algorithms inflate the metabarcoding-derived estimates of biodiversity, consistently returning a high proportion of “rare” molecular operational taxonomic units (MOTUs) that appear to represent computational artifacts and sequencing errors. However, species-specific MOTUs with high relative abundance are often recovered regardless of the bioinformatics approach. We also found high concordance across pipelines for downstream ecological analysis based on beta-diversity and alpha-diversity comparisons that utilize taxonomic assignment information. Analyses of LC datasets and rare MOTUs are especially sensitive to the choice of algorithms and better software tools may be needed to address these scenarios.</p><h1>Diversity and seasonal variation of marine phytoplankton in Jiaozhou Bay, China revealed by morphological observation and metabarcoding</h1>
<p><a href="https://doi.org/10.1007/s00343-021-0457-7">doi:10.1007/s00343-021-0457-7</a></p>

<p><a href=None>None</a></p>
<p>Phytoplankton are central components of marine environments, and are major players in the production and respiration budgeting. However, their diversity and distribution patterns are still poorly understood due largely to their small sizes and inconspicuous morphology that have been determined via the application of traditional morphology methods over the past two decades. To better understand the composition and diversity of phytoplankton in Jiaozhou Bay, China, seasonal sampling was carried out in 2019 and samples were analyzed with morphological observations and high-throughput sequencing, from which obvious seasonal variations in phytoplankton composition and proportional abundances were uncovered. Metabarcoding revealed far more diversity and species richness of phytoplankton than morphological observations, especially with respect to dinoflagellates. Diatoms were the most dominant phytoplankton group throughout the year, of which Thalassionema and Skeletonema were co-dominant in the bay. Parasitic dinoflagellates (e.g. Amoebophrya), which is often overlooked in the morphological observations, were in dominance and high diversity in the metabarcoding dataset, thus more attention should be paid to exploring the potential role of parasitic dinoflagellates. Temperature, chlorophyll a, and nutrient levels were the main influential factors on the distribution of phytoplankton. This study provided a comprehensive morphological and molecular description of phytoplankton and clearly demonstrated the importance of molecular technology in exploring phytoplankton communities. More-widespread use of molecular technology will facilitate deeper understanding of the ecological importance of the different species.</p><h1>Application and progress in high-throughput sequencing technology for meat adulteration detection</h1>
<p><a href="https://doi.org/10.13345/j.cjb.210113">doi:10.13345/j.cjb.210113</a></p>

<p><a href=None>None</a></p>
<p>Adulteration in meat products is a widespread issue that could lead to serious threats to public health and religious violations. Technology that offers rapid, sensitive, accurate and reliable detection of meat species is the key to an effectual monitoring and control against meat adulteration. In recent years, high-throughput sequencing-based DNA metabarcoding technology has developed rapidly. With the characteristics of being high-throughput, highly precise and high-speed, this technology can simultaneously identify multiple species in complex samples, thus offering pronounced advantages in the surveillance of adulteration in meat and meat products. Starting with an introduction of the major developments in the high-throughput sequencing technology in the past two decades, this review provides an overview of the technical characteristics and research methods of DNA metabarcoding, summarizes the application of DNA metabarcoding technology in meat adulteration detection over the last few years, discusses the challenges of using DNA metabarcoding technology in the detection of meat adulteration, and provides future prospects on the development of this technology.</p><h1>16S rRNA metabarcoding unearths responses of rare gut microbiome of fathead minnows exposed to benzo[a]pyrene</h1>
<p><a href="https://doi.org/10.1016/j.scitotenv.2021.151060">doi:10.1016/j.scitotenv.2021.151060</a></p>

<p><a href=None>None</a></p>
<p>Activities of gut microbiomes are often overlooked in assessments of ecotoxicological effects of environmental contaminants. Effects of the polycyclic aromatic hydrocarbon, benzo[a]pyrene (BaP) on active gut microbiomes of juvenile fathead minnows (Pimephales promelas) were investigated. Fish were exposed for two weeks, to concentrations of 0, 1, 10, 100, or 1000 μg BaP g−1 in the diet. The active gut microbiome was characterized using 16S rRNA metabarcoding to determine its response to dietary exposure of BaP. BaP reduced alpha-diversity at the greatest exposure concentrations. Additionally, exposure to BaP altered community composition of active microbiome and resulted in differential proportion of taxa associated with hydrocarbon degradation and fish health. Neighborhood selection networks of active microbiomes were not reduced with greater concentrations of BaP, which suggests ecological resistance and/or resilience of gut microbiota. The active gut microbiome had a similar overall biodiversity as that of the genomic gut microbiota, but had a distinct composition from that of the 16S rDNA profile. Responses of alpha- and beta-diversities of the active microbiome to BaP exposure were consistent with that of genomic microbiomes. Normalized activity of microbiome via the ratio of rRNA to rDNA abundance revealed rare taxa that became active or dormant due to exposure to BaP. These differences highlight the need to assess both 16S rDNA and rRNA metabarcoding to fully derive bacterial compositional changes resulting from exposure to contaminants.</p><h1>Data Filtering Algorithm for DNA Metabarcoding Studies of Ecological Communities</h1>
<p><a href="https://doi.org/10.1134/S1022795422020089">doi:10.1134/S1022795422020089</a></p>

<p><a href=None>None</a></p>
<p>Abstract: Determination of taxonomic diversity of communities is an important stage in many ecological and monitoring studies of ecosystems. As a rule, these studies involve amplicon metagenomics techniques (metabarcoding). In particular, the metagenomic approach is widely used in the analysis of microbial communities. Thus, the number (coverage) of decoded nucleotide sequences determines the representativeness of a particular taxon in a metagenomic sample. Despite the large sizes of samples with decoded nucleotide sequences obtained on the next-generation sequencing platforms, the question of the statistical significance of the estimated indicators of taxonomic representativeness remains open. This mainly concerns the minor component of the investigated communities. To solve the problem of estimating the statistical representativeness of the samples with decoded DNA sequences for metabarcoding research, we developed an algorithm based on the bootstrap method. This algorithm can estimate the confidence interval and absolute and relative errors of the indicators of taxonomic representativeness in communities. The original code of the algorithm, examples of work and working files are available at: https://github.com/Krasnopeev/otu_error_filtering.</p><h1>New Sets of Primers for DNA Identification of Non-Indigenous Fish Species in the Volga-Kama Basin (European Russia)</h1>
<p><a href="https://doi.org/10.3390/w14030437">doi:10.3390/w14030437</a></p>

<p><a href=None>None</a></p>
<p>Adequate species’ identification is critical for the detection and monitoring of biological invasions. In this study, we proposed and assessed the efficiency of newly created primer sets for the genetic identification of non-indigenous species (NIS) of fishes in the Volga basin based on: (a) a “long” fragment of cytochrome c oxidase subunit one of the mitochondrial gene (COI) (0.7 kb), used in “classical” DNA barcoding; (b) a short 3’-fragment (0.3 kb) of COI, suitable for use in high-throughput sequencing systems (i.e., for dietary analysis); (c) fragment of 16S mitochondrial rRNA, including those designed to fill the library of reference sequences for work on the metabarcoding of communities and eDNA studies; (d) a fragment of 18S nuclear rRNA, including two hypervariable regions V1-V2, valuable for animal phylogeny. All four sets of primers demonstrated a high amplification efficiency and high specificity for freshwater fish. Also, we proposed the protocols for the cost-effective isolation of total DNA and purification of the PCR product without the use of commercial kits. We propose an algorithm to carry out extremely cheap studies on the assessment of biological diversity without expensive equipment. We also present original data on the genetic polymorphism of all mass NIS fish species in the Volga-Kama region. The high efficiency of DNA identification based on our primers is shown relative to the traditional monitoring of biological invasions.</p><h1>Unearthing soil arthropod diversity through DNA metabarcoding</h1>
<p><a href="https://doi.org/10.7717/peerj.12845">doi:10.7717/peerj.12845</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding has the potential to greatly advance understanding of soil biodiversity, but this approach has seen limited application for the most abundant and species-rich group of soil fauna-the arthropods. This study begins to address this gap by comparing information on species composition recovered from metabarcoding two types of bulk samples (specimens, soil) from a temperate zone site and from bulk soil samples collected at eight sites in the Arctic. Analysis of 22 samples (3 specimen, 19 soil) revealed 410 arthropod OTUs belonging to 112 families, 25 orders, and nine classes. Studies at the temperate zone site revealed little overlap in species composition between soil and specimen samples, but more overlap at higher taxonomic levels (families, orders) and congruent patterns of α- and β-diversity. Expansion of soil analyses to the Arctic revealed locally rich, highly dissimilar, and spatially structured assemblages compatible with dispersal limited and environmentally driven assembly. The current study demonstrates that DNA metabarcoding of bulk soil enables rapid, large-scale assessments of soil arthropod diversity. However, deep sequence coverage is required to adequately capture the species present in these samples, and expansion of the DNA barcode reference library is necessary to improve taxonomic resolution of the sequences recovered through this approach.</p><h1>Combined Use of eDNA Metabarcoding and Bottom Trawling for the Assessment of Fish Biodiversity in the Zhoushan Sea</h1>
<p><a href="https://doi.org/10.3389/fmars.2021.809703">doi:10.3389/fmars.2021.809703</a></p>

<p><a href=None>None</a></p>
<p>With the increase in fishing intensity and the intensification of marine pollution, the fishery resources in the Zhoushan Sea are seriously degraded, and the difficulty of censusing fish diversity hampers effective management in marine fishes. Environmental DNA metabarcoding and bottom trawl methods were used to determine the ability of the methods to distinguish fish assemblages in the Zhoushan Sea. The species composition and diversity of the Zhoushan Sea were assessed via high-throughput sequencing analysis of eDNA coupled with bottom trawl fishery survey data, after which the two methods were compared. eDNA screening identified 38.2% more fish species than bottom trawls. Combining these two methods, 33 orders, 65 families, and 130 species of fishes were identified. Perciformes and Clupeiformes, the most abundant orders in the catch, represented 31.5 and 10.0% of the total fish abundance, respectively. The results of ANOSIM and redundancy analyses indicated that the fish community structure varied significantly between summer and winter, however depth and temperature being the main environmental factors influencing fish distribution. The biodiversity index was higher in summer than in winter. Thus, our work provides more detailed seasonal data on biodiversity in the Zhoushan Sea, which is essential for the long-term management and conservation of coastal biodiversity. Compared with traditional survey methods, eDNA determination is highly sensitive, accurate, cost-efficient, and suitable for fish diversity studies in relevant sea areas. Although this approach cannot completely replace traditional methods, our findings demonstrate that it provides a reliable complementary method for assessing fish diversity in marine ecosystems.</p><h1>APPLICATIONS OF ENVIRONMENTAL DNA IN LAKE BIODIVERSITY</h1>
<p><a href="https://doi.org/10.7541/2021.2020.237">doi:10.7541/2021.2020.237</a></p>

<p><a href=None>None</a></p>
<p>Lake is indispensable part of inland water ecosystem, and China harbors numerous lakes with rich biodiversity. In recent years, the degradation of lakes severely lead to the decreasing of level of biodiversity. Therefore, lake biodiversity has always been a research hot-spot in limnology. All efforts to research on lake biodiversity essentially depend on monitoring species composition, population size and distribution. Such studies traditionally rely on morphological identification through biological specimen surveys, until to Environmental DNA (eDNA) arose. eDNA exists in the environmental samples such as water, soil and sediments. This feature makes eDNA to be a good indicator to monitor past and current biodiversity. Using molecular biology methods to monitor lake biodiversity is conducive to understanding the dynamic changes of lake ecosystems and is of great significance to the development of lake ecological protection. Compared to traditional investigation methods, eDNA is an non-invasive, efficient and easy to be standardized research approach. It especially doesn’t rely on experts’ experience and professional level of morphological classification. With the development of high-throughput sequencing (HTS) technology and emergence of eDNA metabarcoding, eDNA can be used as a supplement or alternative to traditional investigations. It is currently the most economical and effective method for lake biodiversity research. Lakes are easier to be enriched DNA owing to static water environment, therefore more suitable for applying eDNA methods to carry out their biodiversity researches. This article reviewed the application of eDNA in studies of the lake biodiversity. We also summarized the related experimental design, and analyzed challenges and prospects of this method. The application of eDNA in the lake biodiversity indicated characters of research objects from low-level organisms to high-level organisms. The samples are mainly water and sediments. By extracting ancient DNA from lake sediments, the lake history and the evolution of biodiversity have been studied; eDNA extracted from water samples can be used to understand the current aquatic biodiversity in the lake. The experimental technology of eDNA includes sampling and preservation, DNA extraction and detection. The three main aspects to determine quality of eDNA samples include suitable amount of samples, sampling methods and storage ways. eDNA can be used to detect specific taxa or multi-taxa based on PCR/qPCR with specific primer or meta-barcoding technology. The application of eDNA in lake biodiversity faces challenges such as lack of optimal experimental protocols, contamination, errors and imperfect taxa DNA database. In the future, the application of eDNA method needs to improve relevant experimental techniques and develop DNA database to solve the corresponding dilemmas. We recommend that human contamination and cross-contamination between samples should be avoided as much as possible during sampling. For the lack of database, we suggest establishing a regional eDNA database, conducting targeted data management and strengthening data sharing among various laboratories.</p><h1>Global soil microbiomes: A new frontline of biome-ecology research</h1>
<p><a href="https://doi.org/10.1111/geb.13487">doi:10.1111/geb.13487</a></p>

<p><a href=None>None</a></p>
<p>Aim: Organisms on our planet form spatially congruent and functionally distinct communities, which at large geographical scales are called “biomes”. Understanding their pattern and function is vital for sustainable use and protection of biodiversity. Current global terrestrial biome classifications are based primarily on climate characteristics and functional aspects of plant community assembly. These and other existing biome schemes do not take account of soil organisms, including highly diverse and functionally important microbial groups. We aimed to define large-scale structure in the diversity of soil microbes (soil microbiomes), pinpoint the environmental drivers shaping it and identify resemblance and mismatch with existing terrestrial biome schemes. Location: Global. Time period: Current. Major taxa studied: Soil eukaryotes and prokaryotes. Methods: We collected soil samples from natural environments world-wide, incorporating most known terrestrial biomes. We used high-throughput sequencing to characterize soil biotic communities and k-means clustering to define soil microbiomes describing the diversity of microbial eukaryotic and prokaryotic groups. We used climatic data and soil variables measured in the field to identify the environmental variables shaping soil microbiome structure. Results: We recorded strong correlations among fungal, bacterial, archaeal, plant and animal communities, defined a system of global soil microbiomes (producing seven biome types for microbial eukaryotes and six biome types for prokaryotes) and showed that these are typically structured by pH alongside temperature. None of the soil microbiomes are directly paralleled by any current terrestrial biome scheme, with mismatch most substantial for prokaryotes and for microbial eukaryotes in cold climates; nor do they consistently distinguish grassland and forest ecosystems. Main conclusions: Existing terrestrial biome classifications represent a limited surrogate for the large-scale diversity patterns of microbial soil organisms. We show that empirically defined soil microbiomes are attainable using metabarcoding and statistical clustering approaches and suggest that they can have wide application in theoretical and applied biodiversity research.</p><h1>Application of Environmental DNA (eDNA) Metabarcoding Method to Identify Threatened Sulawesi Mammal Based on 12S rRNA Gene</h1>
<p><a href="https://doi.org/10.4308/HJB.29.1.114-121">doi:10.4308/HJB.29.1.114-121</a></p>

<p><a href=None>None</a></p>
<p>Species detection and identification is a crucial steps in biodiversity assessment. Traditional methods are often invasive and resource intensive. The number of studies demonstrating successful of eDNA metabarcoding approach in species identification has increased rapidly in recent years. Some of large terrestrial mammals have reportedly utilize natural salt licks as a source of minerals in the diet and its genetic material left in the environment can be used to identify species from this site. An eDNA metabarcoding protocol had been carried out to identify Sulawesi mammals from Adudu natural salt-licks, Nantu Wildlife Reserve, Gorontalo. Environmental DNA were extracted from water samples, Amplicon libraries were prepared by PCR amplification and Illumina MiSeq high throughput sequencing. Reads processing and taxonomic assignment carried out in two bioinformatics packages, PipeCraft-1.0 and OBITools-2.11. Two endangered Sulawesi mammals species had been identified, i.e. lowland anoa (Bubalus depressicornis) and babirusa (Babyrousa babyrussa). The accuracy of mammal species identification using eDNA metabarcoding is affected by rigorous experimental procedures, DNA marker reliability, and availability of reference sequence database.</p><h1>High-frequency and long-term observations of eDNA from imperiled salmonids in a coastal stream: Temporal dynamics, relationships with environmental factors, and comparisons with conventional observations</h1>
<p><a href="https://doi.org/10.1002/edn3.293">doi:10.1002/edn3.293</a></p>

<p><a href=None>None</a></p>
<p>A greater understanding of eDNA behavior in the environment is needed before it can be employed for ecosystem monitoring applications. The objectives of this study were to use autonomous sampling to conduct long-term, high-frequency monitoring of the eDNA of native salmonid species in a Californian coastal stream, describe temporal variation of eDNA on multiple scales and identify environmental factors that drive this variation, and evaluate the ability of the eDNA datasets to detect rare species and represent organismal abundance. Using high-throughput autonomous environmental sample processors (ESPs) and qPCR, we enumerated eDNA concentrations from 674 water samples collected at subdaily intervals over 360 days at a single site. We detected eDNA from two imperiled salmonids (coho salmon Oncorhynchus kisutch and steelhead/rainbow trout O. mykiss) in most samples; O. kisutch eDNA was generally in lower concentration and more variable than O. mykiss eDNA. High-frequency (i.e., subdaily and daily) variability in salmonid eDNA concentrations showed occasional patchiness (i.e., large differences between consecutive samples), while seasonal differences were observed consistent with the ecology of the species at this site. Salmonid eDNA concentrations were significantly associated with creek discharge, photoperiod, and whether the creek mouth was open or closed by a seasonal sandbar. The release of hatchery-origin O. kisutch parr into the stream was associated with a significant increase in eDNA concentration for the remainder of the study. We compared eDNA signals with fish abundance data collected from traps located at the site. Fish were detected more often by eDNA than from trapping. Significant positive associations between fish abundance and eDNA concentrations were observed for O. mykiss; however, no such associations were observed for O. kisutch. This study adds to our knowledge on the occurrence and behavior of fish eDNA in lotic systems and informs future biomonitoring efforts using automated sampling technology.</p><h1>Does revegetation cause soil microbiota recovery? Evidence from revisiting a revegetation chronosequence 6 years after initial sampling</h1>
<p><a href="https://doi.org/10.1111/rec.13635">doi:10.1111/rec.13635</a></p>

<p><a href=None>None</a></p>
<p>The global biodiversity and land degradation crises have brought about an urgent need and great demand for restoration actions. However, restoration outcomes are often less than ideal, indicating a need for improved restoration practices. Soil microbiota are extremely diverse and functionally important and should be further considered in restoration. However, despite their importance, there remains a gap in understanding of how soil microbiota respond following native plant revegetation. Several studies have used cross-sectional study designs of restoration chronosequences to infer that revegetation causes the recovery of soil microbiota, but it is near-impossible to determine cause and effect relationships with cross-sectional study designs. Here we used high-throughput amplicon sequencing of the bacterial 16s rRNA gene from soil samples collected at two timepoints, 6 years apart, at a revegetation chronosequence in South Australia. Our results show some indications of recovery but not the additional recovery in bacterial community composition toward the reference sites as expected after this 6-year period—a result that appears at odds to the expected patterns of revegetation causing recovery of soil microbiota. Spatially dependent factors (e.g. soil chemistry), biotic and abiotic barriers, seasonal differences in sampling, and variability among the ecological reference sites could each help explain this apparent lack of additional microbial recovery. More detailed longitudinal and/or experimental manipulation work is required to further examine the cause-effect relationships. Our study contributes important new information and highlights knowledge gaps in how soil microbiota respond to revegetation, and we urge caution when attempting to infer causation from cross-sectional chronosequence studies.</p><h1>Trophic and symbiotic links between obligate-glacier water bears (Tardigrada) and cryoconite microorganisms</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0262039">doi:10.1371/journal.pone.0262039</a></p>

<p><a href=None>None</a></p>
<p>Insights into biodiversity and trophic webs are important for understanding ecosystem functions. Although the surfaces of glaciers are one of the most productive and biologically diverse parts of the cryosphere, the links between top consumers, their diet and microbial communities are poorly understood. In this study, for the first time we investigated the relationships between bacteria, fungi and other microeukaryotes as they relate to tardigrades, microscopic metazoans that are top consumers in cryoconite, a biologically rich and productive biogenic sediment found on glacier surfaces. Using metabarcoding (16S rDNA for bacteria, ITS1 for fungi, and 18S rDNA for other microeukaryotes), we analyzed the microbial community structures of cryoconite and compared them with the community found in both fully fed and starved tardigrades. The community structure of each microbial group (bacteria, fungi, microeukaryotes) were similar within each host group (cryoconite, fully fed tardigrades and starved tardigrades), and differed significantly between groups, as indicated by redundancy analyses. The relative number of operational taxonomic units (ZOTUs, OTUs) and the Shannon index differed significantly between cryoconite and tardigrades. Species indicator analysis highlighted a group of microbial taxa typical of both fully fed and starved tardigrades (potential commensals), like the bacteria of the genera Staphylococcus and Stenotrophomonas, as well as a group of taxa typical of both cryoconite and fully fed tardigrades (likely part of the tardigrade diet; bacteria Flavobacterium sp., fungi Preussia sp., algae Trebouxiophyceae sp.). Tardigrades are consumers of bacteria, fungi and other microeukaryotes in cryoconite and, being hosts for diverse microbes, their presence can enrich the microbiome of glaciers.</p><h1>Unveiling biogeographical patterns of the ichthyofauna in the Tuichi basin, a biodiversity hotspot in the Bolivian Amazon, using environmental DNA</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0262357">doi:10.1371/journal.pone.0262357</a></p>

<p><a href=None>None</a></p>
<p>To date, more than 2400 valid fish species have been recorded in the Amazon basin. However, some regions remain poorly documented. This is the case in the Beni basin and in particular in one of its main sub-basins, the Tuichi, an Andean foothills rivers flowing through the Madidi National Park in the Bolivian Amazonia. The knowledge of its ichthyological diversity is, however, essential for the management and protection of aquatic ecosystems, which are threatened by the development of infrastructures (dams, factories and cities), mining and deforestation. Environmental DNA (eDNA) has been relatively little used so far in the Amazon basin. We sampled eDNA from water in 34 sites in lakes and rivers in the Beni basin including 22 sites in the Tuichi sub-basin, during the dry season. To assess the biogeographical patterns of the amazonian ichthyofauna, we implemented a metabarcoding approach using two pairs of specific primers designed and developed in our laboratory to amplify two partially overlapping CO1 fragments, one of 185bp and another of 285bp. We detected 252 fish taxa (207 at species level) among which 57 are newly identified for the Beni watershed. Species compositions are significantly different between lakes and rivers but also between rivers according to their hydrographic rank and altitude. Furthermore, the diversity patterns are related to the different hydro-ecoregions through which the Tuichi flows. The eDNA approach makes it possible to identify and complete the inventory of the ichthyofauna in this still poorly documented Amazon basin. However, taxonomic identification remains constrained by the lack of reference barcodes in public databases and does not allow the assignment of all OTUs. Our results can be taken into account in conservation and management strategies and could serve as a baseline for future studies, including on other Andean tributaries.</p><h1>Coupling ecological network analysis with high-throughput sequencing-based surveys: Lessons from the next-generation biomonitoring project</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2021.10.007">doi:10.1016/bs.aecr.2021.10.007</a></p>

<p><a href=None>None</a></p>
<p>Biomonitoring ecosystems is necessary in order to evaluate risks and to efficiently manage ecosystems and their associated services. Agrosystems are the target of multiple stressors that can affect many species through effects cascading along food webs. However, classic biomonitoring, focused on species diversity or indicator species, might be a poor predictor of the risk of such whole-ecosystem perturbations. Thanks to high-throughput sequencing methods, however, it might be possible to obtain sufficient information about entire ecological communities to infer the functioning of their associated interaction networks, and thus monitor more closely the risk of the collapse of entire food webs due to external stressors. In the course of the ‘next-generation biomonitoring’ project, we collectively sought to experiment with this idea of inferring ecological networks on the basis of metabarcoding information gathered on different systems. We here give an overview of issues and preliminary results associated with this endeavour and highlight the main difficulties that such next-generation biomonitoring is still facing. Going from sampling protocols up to methods for comparing inferred networks, through biomolecular, bioinformatic, and network inference, we review all steps of the process, with a view towards generality and transferability towards other systems.</p><h1>Multispecies site occupancy modelling and study design for spatially replicated environmental DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13732">doi:10.1111/2041-210X.13732</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding has become widely applied to gauge biodiversity in a non-invasive and cost-efficient manner. The detection of species using eDNA metabarcoding is, however, imperfect owing to various factors that can cause false negatives in the inherent multistage workflow. Imperfect detection in the multistage workflow of eDNA metabarcoding also raises an issue of study design, namely, how available resources should be allocated among the different stages to optimize survey efficiency. Here, we propose a variant of the multispecies site occupancy model for eDNA metabarcoding studies where samples are collected at multiple sites within a region of interest. Unlike traditional site occupancy models that deal with binary detection data, this model describes the variation in sequence reads, the output of the high-throughput sequencers. It explicitly accounts for the hierarchical workflow of eDNA metabarcoding and interspecific heterogeneity and allows the analysis of the sources of variation in the detectability of species throughout the different stages of the workflow. We also introduced a Bayesian decision analysis framework to identify the study design that optimizes the effectiveness of species detection with a limited budget. An application of the model to freshwater fish communities in the Lake Kasumigaura watershed, in Japan, highlighted a remarkable inhomogeneity in the detectability of species, indicating a potential risk of the biased detection of specific species. Species with lower site occupancy probabilities tended to be difficult to detect as they had lower capture probabilities and fewer sequence reads. The expected abundance of sequence reads was predicted to vary by up to 23.5 times among species. An analysis of the study design suggested that ensuring multiple within-site replications of the environmental samples is preferred to achieve higher species detection effectiveness, provided that tens of thousands of sequence reads were secured per replicate. The proposed framework makes the application of eDNA metabarcoding more error-tolerant, allowing ecologists to monitor ecological communities more efficiently.</p><h1>Out of the dark: Using energy flux to connect above- and belowground communities and ecosystem functioning</h1>
<p><a href="https://doi.org/10.1111/ejss.13154">doi:10.1111/ejss.13154</a></p>

<p><a href=None>None</a></p>
<p>Soil ecosystems are both particularly important to humans and vulnerable to human-made global change. Here, we identify some key aspects of soil community and ecosystem research that need to be more widely studied to understand soil responses to global change and enable us to efficiently protect them. This perspective integrates multiple taxa and trophic levels, combines structural community variables with ecosystem processes, and considers multiple energy channels rather than focusing on only bacterial, fungal, or plant-derived resources. Moreover, it enables implementing the claim that terrestrial ecosystem research should more widely adopt an integrative view of above–belowground processes. Having identified these key areas requiring higher attention, we suggest a wider application of the food-web energetics approach to calculating energy flux as a suitable and very powerful tool to simultaneously integrate these aspects. The approach combines food-web and metabolic theory to quantify energy flux through trophic networks as a universal currency of multitrophic ecosystem functioning. In addition to whole-community metrics, it allows for quantifying various important processes by summing up the flux out of autotrophs, detritus, or animals to their respective consumers. This includes the assessment of processes that are otherwise hard to quantify, such as belowground herbivory or predation. The calculation requires data on the focal community and its metabolic demand, trophic interactions, feeding preferences and assimilation efficiency, some of which can be measured, whereas other components can be inferred from readily available literature resources. We outline how novel, high-throughput methods, such as metabarcoding, can be combined with the energy-flux approach to improve our understanding of soil ecosystem structure and functioning. We hope that our article motivates fellow soil researchers to adopt energy-flux approaches for their community data and support the further development of this promising approach for soil community and ecosystem science. Highlights: Multitrophic interactions bridge the structure and functioning of soil ecosystems. An integrated ecosystem perspective requires the quantification of the main energy channels in soil. Linking above- and belowground compartments provides deeper insights into whole-ecosystem processes. Quantifying energy flux through above–belowground systems enables those insights.</p><h1>Yeast and Filamentous Fungi Microbial Communities in Organic Red Grape Juice: Effect of Vintage, Maturity Stage, SO<inf>2</inf>, and Bioprotection</h1>
<p><a href="https://doi.org/10.3389/fmicb.2021.748416">doi:10.3389/fmicb.2021.748416</a></p>

<p><a href=None>None</a></p>
<p>Changes are currently being made to winemaking processes to reduce chemical inputs [particularly sulfur dioxide (SO2)] and adapt to consumer demand. In this study, yeast growth and fungal diversity were investigated in merlot during the prefermentary stages of a winemaking process without addition of SO2. Different factors were considered, in a two-year study: vintage, maturity level and bioprotection by the adding yeast as an alternative to SO2. The population of the target species was monitored by quantitative-PCR, and yeast and filamentous fungi diversity was determined by 18S rDNA metabarcoding. A gradual decrease of the α-diversity during the maceration process was highlighted. Maturity level played a significant role in yeast and fungal abundance, which was lower at advanced maturity, while vintage had a strong impact on Hanseniaspora spp. population level and abundance. The presence of SO2 altered the abundance of yeast and filamentous fungi, but not their nature. The absence of sulfiting led to an unexpected reduction in diversity compared to the presence of SO2, which might result from the occupation of the niche by certain dominant species, namely Hanseniaspora spp. Inoculation of the grape juice with non-Saccharomyces yeast resulted in a decrease in the abundance of filamentous fungi generally associated with a decline in grape must quality. Lower abundance and niche occupation by bioprotection agents were observed at the overripened stage, thus suggesting that doses applied should be reconsidered at advanced maturity. Our study confirmed the bioprotective role of Metschnikowia pulcherrima and Torulaspora delbrueckii in a context of vinification without sulfites.</p><h1>Periphyton diversity in two different Antarctic lakes assessed using metabarcoding</h1>
<p><a href="https://doi.org/10.1017/S0954102021000316">doi:10.1017/S0954102021000316</a></p>

<p><a href=None>None</a></p>
<p>Antarctic lakes have generally simple periphyton communities when compared with those of lower latitudes. To date, assessment of microbial diversity in Antarctica has relied heavily on traditional direct observation and cultivation methods. In this study, sterilized cotton baits were left submerged for two years in two lakes on King George Island and Deception Island, South Shetland Islands (Maritime Antarctic), followed by assessment of diversity by metabarcoding using high-throughput sequencing. DNA sequences of 44 taxa belonging to four kingdoms and seven phyla were found. Thirty-six taxa were detected in Hennequin Lake on King George Island and 20 taxa were detected in Soto Lake on Deception Island. However, no significant difference in species composition was detected between the two assemblages (Shannon index). Our data suggest that metabarcoding provides a suitable method for the assessment of periphyton biodiversity in oligotrophic Antarctic lakes.</p><h1>Spatio-temporal dynamics of bacterial communities in the shoreline of Laurentian great Lake Erie and Lake St. Clair’s large freshwater ecosystems</h1>
<p><a href="https://doi.org/10.1186/s12866-021-02306-y">doi:10.1186/s12866-021-02306-y</a></p>

<p><a href=None>None</a></p>
<p>Background: Long-term trends in freshwater bacterial community composition (BCC) and dynamics are not yet well characterized, particularly in large lake ecosystems. We addressed this gap by temporally (15 months) and spatially (6 sampling locations) characterizing BCC variation in lakes Erie and St. Clair; two connected ecosystems in the Laurentian Great Lakes. Results: We found a spatial variation of the BCC between the two lakes and among the sampling locations (significant changes in the relative abundance of 16% of the identified OTUs at the sampling location level). We observed five distinct temporal clusters (UPGMA broad-scale temporal variation) corresponding to seasonal variation over the 15 months of sampling. Temporal variation among months was high, with significant variation in the relative abundance of 69% of the OTUs. We identified significant differences in taxonomic composition between summer months of 2016 and 2017, with a corresponding significant reduction in the diversity of BCC in summer 2017. Conclusions: As bacteria play a key role in biogeochemical cycling, and hence in healthy ecosystem function our study defines the scope for temporal and spatial variation in large lake ecosystems. Our data also show that freshwater BCC could serve as an effective proxy and monitoring tool to access large lake health.</p><h1>Potential of high-throughput eDNA sequencing of soil fungi and bacteria for monitoring ecological restoration in ultramafic substrates: The case study of the New Caledonian biodiversity hotspot</h1>
<p><a href="https://doi.org/10.1016/j.ecoleng.2021.106416">doi:10.1016/j.ecoleng.2021.106416</a></p>

<p><a href=None>None</a></p>
<p>Due to their central role in ecosystems functioning and their ability to rapidly respond to environmental changes, soil microorganisms could potentially be used for monitoring ecosystems recovery in the context of degraded land restoration. However, these belowground organisms have been, to date, largely neglected. Here, we investigated fungal and bacterial community diversity, composition, and structure from ultramafic soils in New Caledonia, an archipelago in the southwest Pacific recognized as a priority for conservation and restoration. The emerging approach of high-throughput amplicon sequencing of environmental DNA (eDNA) – metabarcoding of eDNA – was used to compare soil microbial communities from four different native vegetation types, representing different stages of a chronosequence and defined as reference ecosystems, to five distinct post-mining sites revegetated several years ago. Our results clearly revealed changes in soil microbial phyla and functional groups along the reference chronosequence and variable responses at the different revegetated sites, with two of the five sites showing a good trajectory of recovery. We thus propose three ratios as metrics for monitoring the restoration trajectory of soil microorganisms: the Ascomycota:Basidiomycota and Saprotrophic:Ectomycorrhizal ratios for fungi, and the Cyanobacteria:Chloroflexi ratios for bacteria. Our study, combined with recent works undertaken in other geographical areas, underpins the great promise that could represent soil microbial eDNA metabarcoding for monitoring restoration progress and success. With the emergence of these new cost-effective and scalable sequencing technologies, soil microbes could, in the near future, be included in guidelines for restoration operations in complement to more conventional approaches.</p><h1>Mitochondrial Genetic Diversity, Population Structure and Detection of Antillean and Amazonian Manatees in Colombia: New Areas and New Techniques</h1>
<p><a href="https://doi.org/10.3389/fgene.2021.726916">doi:10.3389/fgene.2021.726916</a></p>

<p><a href=None>None</a></p>
<p>The Antillean manatee (Trichechus manatus) and the Amazonian manatee (Trichechus inunguis) are distributed in rivers in the Caribbean and Amazonian region of Colombia respectively. For 30 years, genetic information has been obtained from these populations in order to inform conservation programs for these endangered species and decide on the location to release them back to the wild. However, in previous studies, samples from rivers in some areas of the country were not included, given the difficulties to access these regions due to either logistic or safety issues. In this study, we analyzed mitochondrial DNA (mtDNA) control region (CR) sequences of from samples of T. manatus (n = 37) and T. inunguis (n = 4) (410 and 361 bp, respectively), obtained in new and previously unexplored rivers and bays in the country, including Santa Marta, Urabá Gulf, Ayapel Marsh (San Jorge River Basin), Meta River and Magdalena Medio and the low Magdalena River (Cesar Province and Canal del Dique) as well as additional samples from Puerto Nariño in the Colombian Amazon. Our results included the discovery of two newly described mtDNA CR haplotypes for T. manatus. In addition, we confirmed significant population differentiation at the mitochondrial level between the Magdalena and Sinú rivers and differentiation among areas of the same river, including the middle and low Magdalena River. This differentiation may be related to anthropic changes in the river since construction of the Canal del Dique in the XVI century. We also tested environmental DNA sampling and analyses techniques to evaluate its potential use for manatee detection and monitoring in bodies of water in Colombia, in order to evaluate new areas for future manatee conservation initiatives. We emphasize the need to continue using genetic information to provide evidence on the potential best locations to undertake animal release to prevent outbreeding depression.</p><h1>Monitoring newt communities in urban area using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.7717/peerj.12357">doi:10.7717/peerj.12357</a></p>

<p><a href=None>None</a></p>
<p>Newts are amphibians commonly present in small ponds or garden pools in urban areas. They are protected in many countries and their presence is monitored through visual observation and/or trapping. However, newts are not easy to spot as they are small, elusive and often hidden at the bottom of water bodies. In recent years, environmental DNA (eDNA) has become a popular tool for detecting newts, with a focus on individual species using qPCR assays. Here, we assess the effectiveness of eDNA metabarcoding compared to conventional visual surveys of newt diversity in 45 ponds within urban areas of Geneva canton, Switzerland. We designed newt-specific mitochondrial 16S rRNA primers, which assign the majority of amplicons to newts, and were able to detect four species known to be present in the region, including the invasive subspecies Lissotriton vulgaris meridionalis, native to the Italian peninsula, that has been introduced in the Geneva area recently. The obtained eDNA results were congruent overall with conventional surveys, confirming the morphological observations in the majority of cases (67%). In 25% of cases, a species was only detected genetically, while in 8% of cases, the observations were not supported by eDNA metabarcoding. Our study confirms the usefulness of eDNA metabarcoding as a tool for the effective and non-invasive monitoring of newt community and suggests its broader use for the survey of newt diversity in urban area at larger scales.</p><h1>Environmental DNA metabarcoding for monitoring metazoan biodiversity in Antarctic nearshore ecosystems</h1>
<p><a href="https://doi.org/10.7717/peerj.12458">doi:10.7717/peerj.12458</a></p>

<p><a href=None>None</a></p>
<p>Antarctic benthic ecosystems support high biodiversity but their characterization is limited to a few well-studied areas, due to the extreme environment and remoteness making access and sampling difficult. Our aim was to compare water and sediment as sources of environmental DNA (eDNA) to better characterise Antarctic benthic communities and further develop practical approaches for DNA-based biodiversity assessment in remote environments. We used a cytochrome c oxidase subunit I (COI) metabarcoding approach to characterise metazoan communities in 26 nearshore sites across 12 locations in the Vestfold Hills (East Antarctica) based on DNA extracted from either sediment cores or filtered seawater. We detected a total of 99 metazoan species from 12 phyla across 26 sites, with similar numbers of species detected in sediment and water eDNA samples. However, significantly different communities were detected in the two sample types at sites where both were collected (i.e., where paired samples were available). For example, nematodes and echinoderms were more likely to be detected exclusively in sediment and water eDNA samples, respectively. eDNA from water and sediment core samples are complementary sample types, with epifauna more likely to be detected in water column samples and infauna in sediment. More reference DNA sequences are needed for infauna/meiofauna to increase the proportion of sequences and number of taxa that can be identified. Developing a better understanding of the temporal and spatial dynamics of eDNA at low temperatures would also aid interpretation of eDNA signals from polar environments. Our results provide a preliminary scan of benthic metazoan communities in the Vestfold Hills, with additional markers required to provide a comprehensive biodiversity survey. However, our study demonstrates the choice of sample type for eDNA studies of benthic ecosystems (sediment, water or both) needs to be carefully considered in light of the research or monitoring question of interest.</p><h1>DNA metabarcoding marker choice skews perception of marine eukaryotic biodiversity</h1>
<p><a href="https://doi.org/10.1002/edn3.245">doi:10.1002/edn3.245</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is an increasingly popular technique to investigate biodiversity; however, many methodological unknowns remain, especially concerning the biases resulting from marker choice. Regions of the cytochrome c oxidase subunit I (COI) and 18S rDNA (18S) genes are commonly employed “universal” markers for eukaryotes, but the extent of taxonomic biases introduced by these markers and how such biases may impact metabarcoding performance is not well quantified. Here, focusing on macroeukaryotes, we use standardized sampling from autonomous reef monitoring structures (ARMS) deployed in the world's most biodiverse marine ecosystem, the Coral Triangle, to compare the performance of COI and 18S markers. We then compared metabarcoding data to image-based annotations of ARMS plates. Although both markers provided similar estimates of taxonomic richness and total sequence reads, marker choice skewed estimates of eukaryotic diversity. The COI marker recovered relative abundances of the dominant sessile phyla consistent with image annotations. Both COI and the image annotations provided higher relative abundance estimates of Bryozoa and Porifera and lower estimates of Chordata as compared to 18S, but 18S recovered 25% more phyla than COI. Thus, while COI more reliably reflects the occurrence of dominant sessile phyla, 18S provides a more holistic representation of overall taxonomic diversity. Ideal marker choice is, therefore, contingent on study system and research question, especially in relation to desired taxonomic resolution, and a multimarker approach provides the greatest application across a broad range of research objectives. As metabarcoding becomes an essential tool to monitor biodiversity in our changing world, it is critical to evaluate biases associated with marker choice.</p><h1>Ecological quality in freshwater streams is reflected across all three domains of life</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2021.108059">doi:10.1016/j.ecolind.2021.108059</a></p>

<p><a href=None>None</a></p>
<p>Assessment of ecological quality in streaming surface water is often based on different Biological Quality Elements (BQE) such as plants, fish or invertebrates. Conventional stream-water quality assessment based on invertebrates as BQE relies on taxonomic expertise, which is costly and time consuming. Next-generation sequencing approaches for high-throughput analyses of diverse ecosystems are increasingly used for environmental monitoring and holds a great potential for application in stream-water quality assessments. This approach is to some extent hampered by the currently available reference databases representing freshwater invertebrates. In the present study we apply metabarcoding simultaneously targeting the 16S (prokaryotes) and 18S (eukaryotes) rRNA genes to capture a snapshot of the ecosystem composition across the three domains of life. Results based on the analysis of 50 selected Danish streams showed that the combined, as well as the domain-specific profiles can separate the samples into their respective ecological quality categories as reflected by the parallel conventional assessment based on macroinvertebrates as BQE. Furthermore, it was possible to suggest potential indicator organisms, from all three domains, which correlated specifically to the conventional data e.g. organisms with a strong correlation to ecological status across all categories. The results clearly showed that community structure in all three domains of life reflect the ecological status of the sample location. Hence, when applying a molecular approach for water-quality assessment we are not limited to the composition of visible BQE, such as macroinvertebrates. The microbial community composition in the streams may often capture an even better and more comprehensive and sensitive snapshot of the ecological quality of stream waters.</p><h1>Molecular dietary analyses of western capercaillies (Tetrao urogallus) reveal a diverse diet</h1>
<p><a href="https://doi.org/10.1002/edn3.237">doi:10.1002/edn3.237</a></p>

<p><a href=None>None</a></p>
<p>Conservation strategies centered around species habitat protection rely on species’ dietary information. One species at the focal point of conservation efforts is the herbivorous grouse, the western capercaillie (Tetrao urogallus), which is an indicator species for forest biodiversity conservation. Non-molecular means used to study their diet are time-consuming and at low taxonomic resolution. This delays the implementation of conservation strategies including resource protection due to uncertainty about its diet. Thus, limited knowledge on diet is hampering conservation efforts. Here, we use non-invasive environmental DNA (eDNA) metabarcoding on DNA extracted from faces to present the first large-scale molecular dietary analysis of capercaillies. Facal samples were collected from seven populations located in Norway (Finnmark, Troms, Trøndelag, Innlandet) and France (Vosges, Jura, Pyrenees) (n = 172). We detected 122 plant taxa belonging to 46 plant families of which 37.7% of the detected taxa could be identified at species level. The average dietary richness of each sample was 7 ± 5 SD taxa. The most frequently occurring plant groups with the highest relative read abundance (RRA) were trees and dwarf shrubs, in particular, Pinus and Vaccinium myrtillus, respectively. There was a difference in dietary composition (RRA) between samples collected from the different locations (adonis pseudo F5,86 = 11.01, r2 = 0.17, p = 0.001) and seasons (adonis pseudo F2,03 = 0.64, r2 = 0.01, p = 0.036). Dietary composition also differed between sexes at each location (adonis pseudo F1,47 = 2.77, r2 = 0.04, p = 0.024), although not significant for all data combined. In total, 35 taxa (36.8% of taxa recorded) were new capercaillie food items compared with existing knowledge from non-molecular means. The non-invasive molecular dietary analysis applied in this study provides new ecological information of capercaillies’ diet, improving our understanding of adequate habitat required for their conservation.</p><h1>Editorial: Stressors Acting on Aquatic Ecosystems: High-Throughput Sequencing Approaches to Shed Light on Human-Nature Interactions</h1>
<p><a href="https://doi.org/10.3389/fevo.2021.763442">doi:10.3389/fevo.2021.763442</a></p>

<p><a href=None>None</a></p>
<p>None</p><h1>Standardized high-throughput biomonitoring using DNA metabarcoding: Strategies for the adoption of automated liquid handlers</h1>
<p><a href="https://doi.org/10.1016/j.ese.2021.100122">doi:10.1016/j.ese.2021.100122</a></p>

<p><a href=None>None</a></p>
<p>Reliable and comprehensive monitoring data are required to trace and counteract biodiversity loss. High-throughput metabarcoding using DNA extracted from community samples (bulk) or from water or sediment (environmental DNA) has revolutionized biomonitoring, given the capability to assess biodiversity across the tree of life rapidly with feasible effort and at a modest price. DNA metabarcoding can be upscaled to process hundreds of samples in parallel. However, while automated high-throughput analysis workflows are well-established in the medical sector, manual sample processing still predominates in biomonitoring laboratory workflows limiting the upscaling and standardization for routine monitoring applications. Here we present an automated, scalable, and reproducible metabarcoding workflow to extract DNA from bulk samples, perform PCR and library preparation on a liquid handler. Key features are the independent sample replication throughout the workflow and the use of many negative controls for quality assurance and quality control. We generated two datasets: i) a validation dataset consisting of 42 individual arthropod specimens of different species, and ii) a routine monitoring dataset consisting of 60 stream macroinvertebrate bulk samples. As a marker, we used the mitochondrial COI gene. Our results show that the developed single-deck workflow is free of laboratory-derived contamination and produces highly consistent results. Minor deviations between replicates are mostly due to stochastic differences for low abundant OTUs. Thus, we successfully demonstrated that robotic liquid handling can be used reliably from DNA extraction to final library preparation on a single deck, thereby substantially increasing throughput, reducing costs, and increasing data robustness for biodiversity assessments and monitoring.</p><h1>Non-invasive surveys of mammalian viruses using environmental DNA</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13661">doi:10.1111/2041-210X.13661</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) and invertebrate-derived DNA (iDNA) are used to survey biodiversity non-invasively to mitigate difficulties in obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA were used to monitor known wildlife pathogens; however, most wildlife pathogens are unknown and often evolutionarily divergent. To detect and identify known and novel mammalian viruses from eDNA/iDNA, we used a curated set of RNA oligonucleotides as viral baits in a hybridization capture system coupled with high-throughput sequencing. We detected multiple known and novel mammalian RNA and DNA viruses from multiple viral families from both waterhole eDNA and leech-derived iDNA. Congruence was found between detected hosts and viruses identified in leeches and waterholes. Our results demonstrate that eDNA/iDNA samples represent an effective non-invasive resource for studying wildlife viral diversity and for detecting novel potentially zoonotic viruses prior to their emergence.</p><h1>Multiple mitochondrial haplotypes within individual specimens may interfere with species identification and biodiversity estimation by DNA barcoding and metabarcoding in fig wasps</h1>
<p><a href="https://doi.org/10.1111/syen.12500">doi:10.1111/syen.12500</a></p>

<p><a href=None>None</a></p>
<p>Accurate assessment of biodiversity is important for scientists, conservationists and policy makers. With the development of DNA barcoding and metabarcoding, many cryptic species have been identified based on the sequence diversity of a ‘barcode’ region in the mitochondrial genome. These molecular identifications have increased the estimated number of species on Earth ten-fold. Mitochondrial DNA heteroplasmy, in which multiple mitochondrial haplotypes coexist in a single organism, can interfere with the accuracy of these estimates. However, this phenomenon is still infrequently documented. The main reason is that it is difficult to distinguish heteroplasmic sequences from nuclear mitochondrial segments (Numts). We investigated the prevalence of multiple haplotypes in DNA barcoding sequences in fig wasps (Insect: Hymenoptera) by performing high-throughput sequencing on individual specimens. DNA metabarcoding was artificially simulated by combining all the haplotype sequences. Using conventional DNA barcoding by Sanger sequencing, the 28 morphological species we studied were divided into 38 taxa, suggesting the existence of ten cryptic species. However, the haplotypes originating from a single specimen clustered into different cryptic species clades, which conflicted with the conventional barcoding results. Furthermore, when we artificially simulated DNA metabarcoding by mixing all haplotype sequences, the number of molecular operational taxonomic units (or putative species) was 3.2× the number of morphological species. These results indicate that the prevalence of multiple haplotypes within individual specimens may have an important impact on DNA barcode-based species identification and biodiversity assessment more broadly.</p><h1>Environmental DNA of preservative ethanol performed better than water samples in detecting macroinvertebrate diversity using metabarcoding</h1>
<p><a href="https://doi.org/10.1111/ddi.13284">doi:10.1111/ddi.13284</a></p>

<p><a href=None>None</a></p>
<p>Aim: High-throughput pipelines supported by eDNA metabarcoding have been applied in various freshwater ecosystems. Both eDNA in ethanol (EtOH) samples (ES-eDNA) and in water samples (WS-eDNA) can provide comprehensive classification lists with good taxonomic resolution and coverage for determining freshwater biodiversity and biomonitoring. But, the advantages of ES-eDNA metabarcoding over WS-eDNA metabarcoding remain unclear for routine assessments of diversity of benthic macroinvertebrates in streams. Location: Qiantang River Basin, China. Methods: Here, we compared ES-eDNA and WS-eDNA metabarcoding to evaluate the performance of two eDNA workflows in determining biodiversity and recovery of damaged macroinvertebrate communities. All eDNA samples from the environment and bulk specimen of macroinvertebrates were processed into available molecular operational taxonomic units (MOTUs) and identified to the level of genus. Results: WS-eDNA detected more exact sequence variants (ESVs) (formerly referred to as operational taxonomic units; OTUs), than did ES-eDNA (2,866 vs. 2,406), but fewer macroinvertebrate ESVs (381 vs. 481). Among sampling sites, the two eDNA workflows exhibited relatively large dissimilarity on inferred community composition (p <.001). Furthermore, ES-eDNA metabarcoding exhibited more consistent with morphological identification approaches than did WS-eDNA metabarcoding (24.24% vs. 17.63%, p =.002), especially for species identified by traditional morphology (morphotaxa). Main conclusions: Based on the attributes of ES-eDNA and WS-eDNA, it is suggested that ES-eDNA metabarcoding performs better than does WS-eDNA metabarcoding in detecting local biodiversity and was consistent with morphological results, while WS-eDNA was more suitable for exploring biodiversity patterns on a broad scale, as it is the easiest and most convenient way to collect samples. Results of this study suggest ES-eDNA metabarcoding could be an option in building molecular measurement biomonitoring programme based on EtOH sample used for preserving biological samples.</p><h1>Comparing eDNA metabarcoding with morphological analyses: Fungal species richness and community composition of differently managed stages along a forest conversion of Norway spruce towards European beech in Germany</h1>
<p><a href="https://doi.org/10.1016/j.foreco.2021.119429">doi:10.1016/j.foreco.2021.119429</a></p>

<p><a href=None>None</a></p>
<p>Analyzing fungal diversity and community composition through environmental DNA (eDNA) metabarcoding and high-throughput sequencing relies on sequence databases and their taxonomic coverage which are often doubted in regards of data accuracy. To assess the potential of eDNA metabarcoding to distinguish differently managed forest conversion stages, we compared an extant morphological dataset created through sporocarp surveys with a metabarcoding dataset from the same study sites. The study was conducted along a spruce forest conversion project of Norway spruce towards European beech in the Eifel National Park in Germany. Using the UNITE ITS reference database, a total of 198 fungal operational taxonomic units (OTUs) were assigned up to the species level. Comparing the morphological and metabarcoding dataset, a low species overlap was observed with 27 shared fungi. The metabarcoding dataset revealed that all investigated forest conversion management stages shared beech-associated fungi (even the spruce forests), while within the morphological dataset, only the beech-inhabiting forest conversion management stages showed beech-associated fungi. The metabarcoding dataset could not show the same fungal community response patterns on the spruce forest conversion, compared to the morphological dataset, but revealed the genetic refugium of the soil fungal community. We conclude that fungal eDNA metabarcoding should always be evaluated by taxonomic experts to identify potential sequence database errors. eDNA metabarcoding cannot be used interchangeably for morphological community analyses to identify response patterns of fungal communities on forest management strategies. However, both approaches performed well in combination and showed that beech-associated fungal communities with high functional redundancy can develop after a spruce forest conversion by restoring natural European beech forests with an appropriate close-to-nature management strategy.</p><h1>Development of droplet digital Polymerase Chain Reaction assays for the detection of long-finned (Anguilla dieffenbachii) and short-finned (Anguilla australis) eels in environmental samples</h1>
<p><a href="https://doi.org/10.7717/peerj.12157">doi:10.7717/peerj.12157</a></p>

<p><a href=None>None</a></p>
<p>Freshwater eels are ecologically, and culturally important worldwide. The New Zealand long-finned eel (Anguilla dieffenbachii) and short-finned eel (Anguilla australis) are apex predators, playing an important role in ecosystem functioning of rivers and lakes. Recently, there has been a national decline in their populations due to habitat destruction and commercial harvest. The emergence of targeted environmental DNA detection methodologies provides an opportunity to enhance information about their past and present distributions. In this study we successfully developed species-specific droplet digital Polymerase Chain Reaction (ddPCR) assays to detect A. dieffenbachii and A. australis DNA in water and sediment samples. Assays utilized primers and probes designed for regions of the mitochondrial cytochrome b and 16S ribosomal RNA genes in A. dieffenbachii and A. australis, respectively. River water samples (n D 27) were analyzed using metabarcoding of fish taxa and were compared with the ddPCR assays. The presence of A. dieffenbachii and A. australis DNA was detected in a greater number of water samples using ddPCR in comparison to metabarcoding. There was a strong and positive correlation between gene copies (ddPCR analyses) and relative eel sequence reads (metabarcoding analyses) when compared to eel biomass. These ddPCR assays provide a new method for assessing spatial distributions of A. dieffenbachii and A. australis in a range of environments and sample types.</p><h1>Environmental dna gives comparable results to morphology-based indices of macroinvertebrates in a large-scale ecological assessment</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0257510">doi:10.1371/journal.pone.0257510</a></p>

<p><a href=None>None</a></p>
<p>Anthropogenic activities are changing the state of ecosystems worldwide, affecting community composition and often resulting in loss of biodiversity. Rivers are among the most impacted ecosystems. Recording their current state with regular biomonitoring is important to assess the future trajectory of biodiversity. Traditional monitoring methods for ecological assessments are costly and time-intensive. Here, we compared monitoring of macroinvertebrates based on environmental DNA (eDNA) sampling with monitoring based on traditional kick-net sampling to assess biodiversity patterns at 92 river sites covering all major Swiss river catchments. From the kick-net community data, a biotic index (IBCH) based on 145 indicator taxa had been established. The index was matched by the taxonomically annotated eDNA data by using a machine learning approach. Our comparison of diversity patterns only uses the zero-radius Operational Taxonomic Units assigned to the indicator taxa. Overall, we found a strong congruence between both methods for the assessment of the total indicator community composition (gamma diversity). However, when assessing biodiversity at the site level (alpha diversity), the methods were less consistent and gave complementary data on composition. Specifically, environmental DNA retrieved significantly fewer indicator taxa per site than the kick-net approach. Importantly, however, the subsequent ecological classification of rivers based on the detected indicators resulted in similar biotic index scores for the kick-net and the eDNA data that was classified using a random forest approach. The majority of the predictions (72%) from the random forest classification resulted in the same river status categories as the kick-net approach. Thus, environmental DNA validly detected indicator communities and, combined with machine learning, provided reliable classifications of the ecological state of rivers. Overall, while environmental DNA gives complementary data on the macroinvertebrate community composition compared to the kick-net approach, the subsequently calculated indices for the ecological classification of river sites are nevertheless directly comparable and consistent.</p><h1>Harnessing the power of metabarcoding in the ecological interpretation of plant—pollinator DNA data: Strategies and consequences of filtering approaches</h1>
<p><a href="https://doi.org/10.3390/d13090437">doi:10.3390/d13090437</a></p>

<p><a href=None>None</a></p>
<p>Although DNA metabarcoding of pollen mixtures has been increasingly used in the field of pollination biology, methodological and interpretation issues arise due to its high sensitivity. Filtering or maintaining false positives, contaminants, and rare taxa or molecular features could lead to different ecological results. Here, we reviewed how this choice has been addressed in 43 studies featuring pollen DNA metabarcoding, which highlighted a very high heterogeneity of filtering methods. We assessed how these strategies shaped pollen assemblage composition, species richness, and interaction networks. To do so, we compared four processing methods: unfiltering, filtering with a proportional 1% of sample reads, a fixed threshold of 100 reads, and the ROC approach (Receiver Operator Characteristic). The results indicated that filtering impacted species composition and reduced species richness, with ROC emerging as a conservative approach. Moreover, in contrast to unfiltered networks, filtering decreased network Connectance and Entropy, and it increased Modularity and Connectivity, indicating that using cut-off thresholds better describes interactions. Overall, unfiltering might compromise reliable ecological interpretations, unless a study targets rare species. We discuss the suitability of each filtering type,plead for justifying filtering strategies on biological or methodological bases and for developing shared approaches to make future studies more comparable.</p><h1>eDNA metabarcoding outperforms traditional fisheries sampling and reveals fine-scale heterogeneity in a temperate freshwater lake</h1>
<p><a href="https://doi.org/10.1002/edn3.197">doi:10.1002/edn3.197</a></p>

<p><a href=None>None</a></p>
<p>Understanding biodiversity in aquatic systems is critical to ecological research and conservation efforts, but accurately measuring species richness using traditional methods can be challenging. Environmental DNA (eDNA) metabarcoding, which uses high-throughput sequencing and universal primers to amplify DNA from multiple species present in an environmental sample, has shown great promise for augmenting results from traditional sampling to characterize fish communities in aquatic systems. Few studies, however, have compared exhaustive traditional sampling with eDNA metabarcoding of corresponding water samples at a small spatial scale. We intensively sampled Boardman Lake (1.4 km2) in Michigan, USA, from May to June in 2019 using gill and fyke nets and paired each net set with lake water samples collected in triplicate. We analyzed water samples using eDNA metabarcoding with 12S and 16S fish-specific primers and compared estimates of fish diversity among methods. In total, we set 60 nets and analyzed 180 1 L lake water samples. We captured a total of 12 fish species in our traditional gear and detected 40 taxa in the eDNA water samples, which included all the species observed in nets. The 12S and 16S assays detected a comparable number of taxa, but taxonomic resolution varied between the two genes. In our traditional gear, there was a clear difference in the species selectivity between the two net types, and there were several species commonly detected in the eDNA samples that were not captured in nets. Finally, we detected spatial heterogeneity in fish community composition across relatively small scales in Boardman Lake with eDNA metabarcoding, but not with traditional sampling. Our results demonstrated that eDNA metabarcoding was substantially more efficient than traditional gear for estimating community composition, highlighting the utility of eDNA metabarcoding for assessing species diversity and informing management and conservation.</p><h1>Combining Multiple Markers in Environmental DNA Metabarcoding to Assess Deep-Sea Benthic Biodiversity</h1>
<p><a href="https://doi.org/10.3389/fmars.2021.684955">doi:10.3389/fmars.2021.684955</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is an emerging tool to estimate diversity by combining DNA from the environmental samples and the high-throughput sequencing. Despite its wide use in estimating eukaryotic diversity, many factors may bias the results. Maker choice and reference databases are among the key issues in metabarcoding analyses. In the present study, we compared the performance of a novel 28S rRNA gene marker designed in this study and two commonly used 18S rRNA gene markers (V1-2 and V9) in estimating the eukaryotic diversity in the deep-sea sediments. The metabarcoding analyses based on the sediment surveys of the Okinawa Trough found that more eukaryotic taxa were discovered by 18S V9 than 28S and 18S V1-2, and that 18S V9 also performed better in metazoan recovery than the other two markers. Although a broad range of taxa were detected by the three metabarcoding markers, only a small proportion of taxa were shared between them even at the phylum level. The non-metric multidimensional scaling (NMDS) analysis also supported that communities detected by the three markers were distinct from each other. In addition, different communities were resolved by different reference databases (NCBI nt vs. SILVA) for the two 18S markers. Combining the three markers, annelids were found to be the most abundant (44.9%) and diverse [179 operational taxonomic units (OTUs)] metazoan group in the sediments of the Okinawa Trough. Therefore, multiple independent markers are recommended to be used in metabarcoding analyses during marine diversity surveys, especially for the poorly understood deep-sea sediments.</p><h1>Environmental biomonitoring of reef fish community structure with eDNA metabarcoding in the Coral Triangle</h1>
<p><a href="https://doi.org/10.1007/s10641-021-01118-3">doi:10.1007/s10641-021-01118-3</a></p>

<p><a href=None>None</a></p>
<p>Coral reef fishes perform a range of vital ecosystem functions, and can serve as indicators of ecological stress and resilience. However, many species are not observed when using Underwater Visual Census (UVC) during biomonitoring, and therefore overall biodiversity is often underestimated. Environmental DNA (eDNA) is proposed as an advanced and non-invasive next-generation biomonitoring method for determining the presence of aquatic organisms such as fish. Therefore, this study aimed to assess the community structure of coral fish from three different marine protected area reef zones (utility zone, open access zone, core zone) around Lombok Island using eDNA metabarcoding. Biological community composition, richness, and diversity were evaluated based on reads from mid-column water and sediment samples. A total of 58 species were identified from the eDNA samples using the Multiplex Barcode Research And Visualization Environment (mBRAVE) pipeline. The Shannon–Wiener index (H') showed significantly higher species diversity in the core zone than the utility and open access zones. There was no significant between-zone difference in community structure (ANOSIM, R = 0.11 < 0.25). NMDS analysis using the Bray–Curtis test showed significant between-zone differences in species diversity and abundance (PERMANOVA Adonis Pr (> F) = 0.001, p < 0.05). Based on the high number of fish species detected in this study, eDNA can be recommended as an alternative tool or as a complement to visual survey methods for biological monitoring and diversity assessment of remote reefs, with less stringent requirements in terms of field conditions (e.g. visibility) and taxonomic expertise.</p><h1>Combining Traditional Taxonomy and Metabarcoding: Assemblage Structure of Nematodes in the Shelf Sediments of the Eastern Antarctic Peninsula</h1>
<p><a href="https://doi.org/10.3389/fmars.2021.629706">doi:10.3389/fmars.2021.629706</a></p>

<p><a href=None>None</a></p>
<p>This study provides a snapshot of the largely understudied meiobenthic and nematode communities in the Prince Gustav Channel (PGC) and Duse Bay (DB). We compared five stations sampled at different water depths along the shelf and investigated their meiobenthic community structure. We approached nematode biodiversity combining traditional taxonomic identification and high throughput sequencing (HTS), with the use of Amplicon Sequence Variants (ASVs). Additionally, we characterized the environment by primary production proxies, grain size and seasonal ice conditions. Our results suggest that the availability of organic matter and its freshness are responsible for the high densities found at all depths. However, potential factors influencing the high local and regional variability of meiofauna density and biodiversity are less clear. A bathymetric transect consisting of three stations in DB (200, 500, and 1,000 m depth) showed increasing pigment concentrations in the first centimeters of the sediment vertical profile with increasing water depth, whereas the meiofauna densities showed the opposite trend. The deepest station of DB seems to function as a sink for fine material as supported by the higher silt fraction and higher organic matter concentrations. When comparing the two basins in the PGC (1,000 and 1,250 m) and the one in DB (1,000 m), differences in terms of environmental variables, meiofaunal densities, and composition were observed. The deepest basin in PGC is located further South (closer to the highly unstable Larsen area), and marked differences with the other basins suggest that it might be experiencing different conditions as a result of its presence near the summer ice margin and its more elongated topography. Both, the shallowest and the deepest stations showed the highest number of unique sequences, suggesting a more biodiverse nematode assemblage. The morphological identification did not show significant differences in the biodiversity of all stations, differently from the ASVs approach. However, the lack of reference sequences in online databases and the thickness of nematode’s cuticule are still important issues to consider as they potentially lead to underestimations of biodiversity and functional traits.</p><h1>Land-use types shape soil microbial compositions under rapid urbanization in the Xiong'an New Area, China</h1>
<p><a href="https://doi.org/10.1016/j.scitotenv.2021.145976">doi:10.1016/j.scitotenv.2021.145976</a></p>

<p><a href=None>None</a></p>
<p>For urban planning and infrastructural projects, considerable attention has been paid to the relationship between soil biota, especially protists, and edaphic conditions in various land-use types having different plant species in the Xiong'an New Area of China. To elucidate this relationship, we assessed edaphic variables and soil biota compositions and compared them among 5 habitat types: human-made forests, crop cultivations, arid rivers, Baiyangdian (BYD) Lake, and around oil wells. In all, 12 experimental plots from terrestrial and aquatic ecosystems were assessed using high-throughput sequencing of environmental DNA, targeting the V3–V4 region of the 16S rRNA gene, internal transcribed spacer 1, and V4 region of the 18S rRNA gene for bacteria, fungi, and protists, respectively. The abundance of bacterial and protist communities was higher than fungi, possibly because fungi prefer acidic soil conditions and likely have greater susceptibility to anthropogenic activities. Across all experimental plots, land-use types contributed the most to the β-diversity of soil biota, followed by soil moisture. Diversity and richness were significantly higher at aquatic habitats than at terrestrial habitats. Predictive metagenomic analysis of trophic groups predicted relatively high frequency of functional genes from bacterial metabolism pathways (carbohydrate and amino acid); contrary to expectation, phototrophic protists, but not fungal symbionts and protistan consumers, were the dominant group at the BYD Lake. Geographical coordinates showed significant (P < 0.05) relationships with all microbiome taxa (nodes at network) from all land-use types. Moreover, soil–microbiome relationships were more complex and more intense at crop habitats. Links between protist and fungal taxa were the highest at the petroleum-contaminated sampling sites, indicating the importance of these two soil microbiomes in polluted soil. Thus, our findings suggest that human manipulation and land-use types are crucial factors for soil biota structure and composition across our sampling sites.</p><h1>Universal primers screening and verification for environmental DNA metabarcoding of freshwater mussels in the Lake Poyang Basin</h1>
<p><a href="https://doi.org/10.18307/2021.0425">doi:10.18307/2021.0425</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) technology is a non-invasive, highly sensitive, efficient and non-destructive to the environment, no damage to the organism investigation tool. To select suitable metabarcoding primers for freshwater mussel's environmental DNA biodiversity research, 11 pairs of primers (with 9 pairs of universal primers designed and 2 pairs cited from relevant literature) were screened by general amplification and high-throughput sequencing of genomic DNA of 24 common mussels in the Lake Poyang Basin. The results showed that the primer cyt b and 16S rRNA had good amplification effect and high discrimination. The two pairs of primers were further verified with environmental samples (n=6) and traditional sampling. It was found that 8 mussels species belonging to 6 genera were annotated by 16S rRNA which accounted for 26.69% of the total sequences, and 5 mussels species belonging to 4 genera were annotated by cyt b which accounted for 6.60% of the total sequences. The primer 16S rRNA was more suitable used for metabarcoding primer for the research of environmental DNA biodiversity in freshwater mussels. Environmental DNA technology can be used as an effective supplement to traditional methods in biodiversity monitoring. The reliability and detection rate can be increased by using multiple pairs of primers at the same time.</p><h1>The future of dna barcoding: Reflections from early career researchers</h1>
<p><a href="https://doi.org/10.3390/d13070313">doi:10.3390/d13070313</a></p>

<p><a href=None>None</a></p>
<p>Over the last two decades, the use of DNA barcodes has transformed our ability to identify and assess life on our planet. Both strengths and weaknesses of the method have been exemplified through thousands of peer-reviewed scientific articles. Given the novel sequencing approaches, currently capable of generating millions of reads at low cost, we reflect on the questions: What will the future bring for DNA barcoding? Will identification of species using short, standardized fragments of DNA stand the test of time? We present reflected opinions of early career biodiversity researchers in the form of a SWOT analysis and discuss answers to these questions.</p><h1>Antifungal agents, yeast abundance and diversity in surface water: Potential risks to water users</h1>
<p><a href="https://doi.org/10.1016/j.chemosphere.2021.129718">doi:10.1016/j.chemosphere.2021.129718</a></p>

<p><a href=None>None</a></p>
<p>South African surface waters are subject to various forms of pollution. Recent findings in aquatic systems suggest an association exists between yeast diversity, chemical pollutants and land coverage, which are important water quality determinants. Yeast abundance and diversity, as well as antifungal agents in two river systems in South Africa, were investigated and related to the existing land coverage. Yeast abundance and diversity were determined from environmental DNA by quantitative polymerase chain reaction and next-generation sequencing, respectively, of the 26S ribosomal ribonucleic acid (rRNA) gene. Antifungal agents were qualitatively and/or quantitatively detected by ultra-high-pressure liquid chromatography-mass spectrometry. Analyses of 2 031 714 high-quality 26S rRNA sequences yielded 5554 amplicon sequence variants (ASVs)/species. ASV richness and Shannon-Wiener index of diversity reflected the southward flow of the river with higher values observed downstream compared to the upstream. Fluconazole concentrations were quantifiable in only two samples; 178 and 271 ng L−1. Taxonomically, at least 20 yeast species were detected, including the dominant Candida tropicalis, Cryptococcus spp. as well as the lesser dominant Bensingtonia bomiensis, Fereydounia khargensis, Hericium erinaceus, Kondoa changbaiensi, Pseudozyma spp. and Sphacelotheca pamparum. The two dominant species are known opportunistic pathogens which had antifungal resistant traits in previous studies from the same rivers and therefore is a public health threat. The present study provides further evidence that yeasts should be included as part of water quality parameters, especially in developing countries where much of the population are economically disadvantaged, and also immunocompromised due to age and disease.</p><h1>Conversion of a High-Altitude Temperate Forest for Agriculture Reduced Alpha and Beta Diversity of the Soil Fungal Communities as Revealed by a Metabarcoding Analysis</h1>
<p><a href="https://doi.org/10.3389/fmicb.2021.667566">doi:10.3389/fmicb.2021.667566</a></p>

<p><a href=None>None</a></p>
<p>Land-use change is one of the most important drivers of change in biodiversity. Deforestation for grazing or agriculture has transformed large areas of temperate forest in the central highlands of Mexico, but its impact on soil fungal communities is still largely unknown. In this study, we determined how deforestation of a high-altitude temperate forest for cultivation of maize (Zea mays L.) or husbandry altered the taxonomic, phylogenetic, functional, and beta diversity of soil fungal communities using a 18S rRNA metabarcoding analysis. The true taxonomic and phylogenetic diversity at order q = 1, i.e., considering frequent operational taxonomic units, decreased significantly in the arable, but not in the pasture soil. The beta diversity decreased in the order forest > pasture > arable soil. The ordination analysis showed a clear effect of intensity of land-use as the forest soil clustered closer to pasture than to the arable soil. The most abundant fungal phyla in the studied soils were Ascomycota, Basidiomycota, and Mucoromycota. Deforestation more than halved the relative abundance of Basidiomycota; mostly Agaricomycetes, such as Lactarius and Inocybe. The relative abundance of Glomeromycota decreased in the order pasture > forest > arable soil. Symbiotrophs, especially ectomycorrhizal fungi, were negatively affected by deforestation while pathotrophs, especially animal pathogens, were enriched in the pasture and arable soil. Ectomycorrhizal fungi were more abundant in the forest soil as they are usually associated with conifers. Arbuscular mycorrhizal fungi were more abundant in the pasture than in the arable soil as the higher plant diversity provided more suitable hosts. Changes in fungal communities resulting from land-use change can provide important information for soil management and the assessment of the environmental impact of deforestation and conversion of vulnerable ecosystems such as high-altitude temperate forests.</p><h1>Grab what you can—an evaluation of spatial replication to decrease heterogeneity in sediment eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.7717/peerj.11619">doi:10.7717/peerj.11619</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA methods such as metabarcoding have been suggested as possible alternatives or complements to the current practice of morphology-based diversity assessment for characterizing benthic communities in marine sediment. However, the source volume used in sediment eDNA studies is several magnitudes lower than that used in morphological identification. Here, we used data from a North Sea benthic sampling station to investigate to what extent metabarcoding data is affected by sampling bias and spatial heterogeneity. Using three grab parallels, we sampled five separate sediment samples from each grab. We then made five DNA extraction replicates from each sediment sample. Each extract was amplified targeting both the 18S SSU rRNA V1–V2 region for total eukaryotic composition, and the cytochrome c oxidase subunit I (COI) gene for metazoans only. In both datasets, extract replicates from the same sediment sample were significantly more similar than different samples from the same grab. Further, samples from different grabs were less similar than those from the same grab for 18S. Interestingly, this was not true for COI metabarcoding, where the differences within the same grab were similar to the differences between grabs. We also investigated how much of the total identified richness could be covered by extract replicates, individual sediment samples and all sediment samples from a single grab, as well as the variability of Shannon diversity and, for COI, macrofaunal biotic indices indicating environmental status. These results were largely consistent with the beta diversity findings, and show that total eukaryotic diversity can be well represented using 18S metabarcoding with a manageable number of biological replicates. Based on these results, we strongly recommend the combination of different parts of the surface of single grabs for eDNA extraction as well as several grab replicates, or alternatively box cores or similar. This will dilute the effects of dominating species and increase the coverage of alpha diversity. COI-based metabarcoding consistency was found to be lower compared to 18S, but COI macrofauna-based indices were more consistent than direct COI alpha diversity measures.</p><h1>Don't catch me if you can – Using cabled observatories as multidisciplinary platforms for marine fish community monitoring: An in situ case study combining Underwater Video and environmental DNA data</h1>
<p><a href="https://doi.org/10.1016/j.scitotenv.2021.145351">doi:10.1016/j.scitotenv.2021.145351</a></p>

<p><a href=None>None</a></p>
<p>Cabled observatories are marine infrastructures equipped with biogeochemical and oceanographic sensors as well as High-Definition video and audio equipment, hence providing unprecedented opportunities to study marine biotic and abiotic components. Additionally, non-invasive monitoring approaches such as environmental DNA (eDNA) metabarcoding have further enhanced the ability to characterize marine life. Although the use of non-invasive tools beholds great potential for the sustainable monitoring of biodiversity and declining natural resources, such techniques are rarely used in parallel and understanding their limitations is challenging. Thus, this study combined Underwater Video (UV) with eDNA metabarcoding data to produce marine fish community profiles over a 2 months period in situ at a cabled observatory in the northeast Atlantic (SmartBay Ireland). By combining both approaches, an increased number of fish could be identified to the species level (total of 22 species), including ecologically and economically important species such as Atlantic cod, whiting, mackerel and monkfish. The eDNA approach alone successfully identified a higher number of species (59%) compared to the UV approach (18%), whereby 23% of species were detected by both methods. The parallel implementation of point collection eDNA and time series UV data not only confirmed expectations of the corroborative effect of using multiple disciplines in fish community composition, but also enabled the assessment of limitations intrinsic to each technique including the identification of false-negative detections in one sampling technology relative to the other. This work showcased the usefulness of cabled observatories as key platforms for in situ empirical assessment of both challenges and prospects of novel technologies in aid to future monitoring of marine life.</p><h1>Using optimal environmental dna method to improve the fish diversity survey—from laboratory to aquatic life reserve</h1>
<p><a href="https://doi.org/10.3390/w13111468">doi:10.3390/w13111468</a></p>

<p><a href=None>None</a></p>
<p>Conserving aquatic ecosystems requires efficient tools to accurately assess the biodiversity of aquatic species. However, existing knowledge is insufficient in terms of the reliability and the comparability of methods measuring fish diversity. Environmental DNA (eDNA), as a promising method, was used to detect fish taxa in this study. We optimized the eDNA method in the laboratory, and applied the optimal eDNA method to survey fish diversity in a natural aquatic life reserve. We simulated necessary steps of the eDNA method in the lab to increase the confidence of the field survey. Specifically, we compared different eDNA sampling, extraction, and sequencing strategies for accurately capturing fish species of the target area. We found that 1L water samples were sufficient for sampling eDNA information of the majority taxa. The filtration was more effective than the centrifugal precipitation for the eDNA extraction. The cloning sequencing was better than the high-throughput sequencing. The field survey showed that the Shannon–Wiener diversity index of fish taxa was the highest in Huairou Reservoir. The diversity index also showed seasonal changes. The accuracy rate of detecting fish taxa was positively correlated with the eDNA concentration. This study provides a scientific reference for an application of the eDNA method in terms of surveying and estimating the biodiversity of aquatic species.</p><h1>Cyprus’ diatom diversity and the association of environmental and anthropogenic influences for ecological assessment of rivers using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1016/j.chemosphere.2021.129814">doi:10.1016/j.chemosphere.2021.129814</a></p>

<p><a href=None>None</a></p>
<p>Human activities are the leading cause of environmental impairments. Appropriate biomonitoring of ecosystems is needed to assess these activities effectively. In freshwater ecosystems, periphytic and epilithic biofilms have diatom assemblages. These assemblages respond rapidly to environmental changes, making diatoms valuable bioindicators. For this reason, freshwater biomonitoring programs are currently using diatoms (e.g., Water Framework Directive). In the past ten years, DNA metabarcoding coupled with next-generation sequencing and bioinformatics represents a complementary approach for diatom biomonitoring. In this study, this approach is used for the first time in Cyprus by considering the association of environmental and anthropogenic pressures to diatom assemblages. Statistical analysis was then applied to identify the environmental (i.e., river types, geo-morphological) and anthropogenic (i.e., physicochemical, human land-use pressures) variables’ role in the observed diatom diversity. Results indicate differences in diatom assemblages between intermittent and perennial rivers. Achnanthidium minutissimum was more abundant in intermittent rivers; whereas Amphora pediculus and Planothidium caputium in perennial ones. Additionally, we could demonstrate the correlation between nutrients (e.g., nitrogen, phosphorus), stations’ local characteristics (e.g., elevation), and land use activities on the observed differences in diatom diversity. Finally, we conclude that multi-stressors and anthropogenic pressures together as multiple stressors have a significant statistical relationship to the observed diatom diversity and play a pivotal role in determining Cyprus’ rivers’ ecological status.</p><h1>Epiphytic diatom-based biomonitoring in mediterranean ponds: Traditional microscopy versus metabarcoding approaches</h1>
<p><a href="https://doi.org/10.3390/w13101351">doi:10.3390/w13101351</a></p>

<p><a href=None>None</a></p>
<p>Benthic diatoms have traditionally been used as bioindicators of aquatic ecosystems. Because diatom-based monitoring of water quality is required by European legislation, molecular-based methods had emerged as useful alternatives to classical methods based on morphological identification using light microscopy. The aim of this study was to test the reliability of DNA metabarcoding combined with High-Throughput Sequencing (HTS) techniques in the bioassessment of the trophic status of 22 Mediterranean shallow ponds in NW Spain. For each pond, the Trophic Diatom Index (TDI) was calculated from inventories obtained by identification using light microscopy (LM) followed by high-throughput sequencing (HTS) at the molecular level. Ponds were subsequently classified into five water quality classes. The results showed a good correspondence between both methods, especially after applying a correction factor that depended on the biovolume of the cells. This correspondence led to the assignment to the same quality class in 59% of the ponds. The determination and quantification of valves or DNA sequences was one of the main pitfalls, which mainly included those related to the variability in the relative abundances of some species. Accordingly, ponds with similar relative abundances for the dominant species were assigned to the same quality class. Moreover, other difficulties leading the discrepancies were the misidentification of some species due to the presence of semi-cryptic taxa, the incompleteness of the reference database and the bioinformatic protocol. Thus, the validation of DNA-based methods for the identification of freshwater diatoms represents an important goal, as an alternative to using traditional methods in Mediterranean shallow ponds.</p><h1>Seeking life in sedimented waters: Environmental DNA from diverse habitat types reveals ecologically significant species in a tropical marine environment</h1>
<p><a href="https://doi.org/10.1002/edn3.162">doi:10.1002/edn3.162</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) with metabarcoding or metagenomics will likely become a major biomonitoring tool in the 21st century, perhaps even more so in the face of increased coastal urbanization and its associated effects such as pollution, land reclamation, and seabed dredging. Together, these impacts and the consequent high turbidity pose severe challenges to traditional survey techniques that rely heavily on visual observations. We here demonstrate that eDNA can be used for biomonitoring in turbid waters, using Singapore as a case study. The cytochrome c oxidase subunit I (COI) locus was used to detect 525 metazoan molecular operational taxonomic units (MOTUs) based on eDNA obtained from 52 L of filtered seawater. Of these, 130 MOTUs (24.7%) were identifiable to species, including ecologically significant species that in some cases were invasive or rarely observed. Metazoan signals also enabled discrimination of discrete, but connected, environments from intertidal and subtidal zones. Taxa with known habitat preferences were found to have left trace eDNA at sites that matched putatively suitable habitats. Moreover, no coast-specific signals were detected in open water samples, which suggest that intermixing of water was limited. The study confirms that eDNA metabarcoding is a viable biomonitoring tool for coastal areas with highly sedimented waters.</p><h1>A comparison of DNA metabarcoding and microscopy methodologies for the study of aquatic microbial eukaryotes</h1>
<p><a href="https://doi.org/10.3390/d13050180">doi:10.3390/d13050180</a></p>

<p><a href=None>None</a></p>
<p>The procedures and methodologies employed to study microbial eukaryotic plankton have been thoroughly discussed. Two main schools exist-one insisting on classic microscopy methodologies and the other supporting modern high-throughput sequencing (DNA metabarcoding). However, few studies have attempted to combine both these approaches; most studies implement one method while ignoring the other. This work aims to contribute to this discussion and examine the advantages and disadvantages of each methodology by comparing marine plankton community results from microscopy and DNA metabarcoding. The results obtained by the two methodologies do not vary significantly for Bacillariophyta, although they do for Dinoflagellata and Ciliophora. The lower the taxonomic level, the higher the inconsistency between the two methodologies for all the studied groups. Considering the different characteristics of microscopy-based identification and DNA metabarcoding, this work underlines that each method should be chosen depending on the aims of the study. DNA metabarcoding provides a better estimate of the taxonomic richness of an ecosystem while microscopy provides more accurate quantitative results regarding abundance and biomass. In any case, the combined use of the two methods, if properly standardized, can provide much more reliable and accurate results for the study of marine microbial eukaryotes.</p><h1>Nuisance species in lake constance revealed through eDNA</h1>
<p><a href="https://doi.org/10.1007/s10530-021-02462-2">doi:10.1007/s10530-021-02462-2</a></p>

<p><a href=None>None</a></p>
<p>Biological invasions are a global threat to biodiversity especially for aquatic resources. The distribution of alien species is associated with human activities; therefore, exotic species tend to accumulate near big urban areas through different invasion vectors such as ballast water, hull fouling, aquarium and pet releases. The Rhine River region is one of the most economically important in Europe. Around 60 million people live in the river basin that is connected with other large European rivers via the Rhine–Main–Danube shipping canal. The Alpine Rhine flows to Lake Constance, which is the second largest subalpine lake in Europe. Here, eDNA metabarcoding was employed to inventory aquatic species from water samples in six riverine and four lake localities within Lake Constance region. A 313 bp fragment within cytochrome c oxidase subunit I gene was PCR amplified using generalist primers for metazoan and sequenced with MiSeq High-Throughput Sequencing platform. Seven invertebrate invasive species and the invasive fish Oncorhynchus mykiss were detected from eDNA. Species-specific primers were employed to confirm metabarcoded species. Most of the invasive species detected in this study correspond to samples from areas around lake ports, followed by other lake and degraded downstream river areas. Samples taken upstream of Lake Constance were free of invertebrate aliens. To establish common regulation and management actions regarding aquatic invasions in the three countries that share Lake Constance is recommended.</p><h1>Global Trends of Benthic Bacterial Diversity and Community Composition Along Organic Enrichment Gradients of Salmon Farms</h1>
<p><a href="https://doi.org/10.3389/fmicb.2021.637811">doi:10.3389/fmicb.2021.637811</a></p>

<p><a href=None>None</a></p>
<p>The analysis of benthic bacterial community structure has emerged as a powerful alternative to traditional microscopy-based taxonomic approaches to monitor aquaculture disturbance in coastal environments. However, local bacterial diversity and community composition vary with season, biogeographic region, hydrology, sediment texture, and aquafarm-specific parameters. Therefore, without an understanding of the inherent variation contained within community complexes, bacterial diversity surveys conducted at individual farms, countries, or specific seasons may not be able to infer global universal pictures of bacterial community diversity and composition at different degrees of aquaculture disturbance. We have analyzed environmental DNA (eDNA) metabarcodes (V3–V4 region of the hypervariable SSU rRNA gene) of 138 samples of different farms located in different major salmon-producing countries. For these samples, we identified universal bacterial core taxa that indicate high, moderate, and low aquaculture impact, regardless of sampling season, sampled country, seafloor substrate type, or local farming and environmental conditions. We also discuss bacterial taxon groups that are specific for individual local conditions. We then link the metabolic properties of the identified bacterial taxon groups to benthic processes, which provides a better understanding of universal benthic ecosystem function(ing) of coastal aquaculture sites. Our results may further guide the continuing development of a practical and generic bacterial eDNA-based environmental monitoring approach.</p><h1>Metabarcoding analysis of the bacterial succession during vermicomposting of municipal solid waste employing the earthworm Eisenia fetida</h1>
<p><a href="https://doi.org/10.1016/j.scitotenv.2020.144389">doi:10.1016/j.scitotenv.2020.144389</a></p>

<p><a href=None>None</a></p>
<p>A culture-independent DNA metabarcoding analysis of the bacterial communities was carried out throughout a complete vermicomposting cycle of municipal solid waste material using the earthworm Eisenia fetida. 16S rRNA amplicons from the initial material (0 days), an intermediate (42 days), and a final stage (84 days) were sequenced in an Illumina NGS platform and compared. A steady increase in community diversity was observed corresponding to a 2.5-fold higher taxa richness and correspondingly risen values of the Shannon and Simpson ecological indexes and the evenness parameter. A total of 49,665 operational taxonomic units (OTUs) were counted. From the qualitative standpoint, a clear successional shift was observed with an initial community dominated by putatively plant-associated groups belonging to the Rhizobiales order within the Alphaproteobacteria class, regressively leaving the scores of relative abundance (RA) to the Firmicutes phylum and in particular to the Bacilli. Vermistabilization of municipal solid waste (MSW) increased (p < 0.001) the TKN and total P content in the final vermicompost, while pH, TOC, and C/N ratio declined (p < 0.001) in the process. Likewise, a progressive decrease was noticed in β-glucosidase, acid phosphatase, and urease activity while protease and dehydrogenase showed a slight increase, followed by a steep fall. A strong positive correlation was observed among the canonical functions of physico-chemical attributes and enzyme activities. The canonical correspondence analysis (CCA) revealed that significant families did not change on the temporal scale; however, their abundance was influenced by the abiotic environmental factors. In comparison to prior studies on vermicomposting that used different earthworm species (Eisenia andrei) and different substrates, results reflect a considerable degree of substrate specificity for the earthworm species used. The results offer clues to optimize the vermistabilization of MSW along with its potential use in agriculture, to foster improved levels of the circular economy.</p><h1>The Study of the Diversity of Hydrobionts from Listvennichny Bay of Lake Baikal by DNA Metabarcoding</h1>
<p><a href="https://doi.org/10.1134/S1022795421040050">doi:10.1134/S1022795421040050</a></p>

<p><a href=None>None</a></p>
<p>Abstract: The taxonomic diversity of hydrobionts from Listvennichny Bay was studied by DNA metabarcoding. The sequencing of a gene fragment (313 bp) of mitochondrial DNA encoding the first subunit of cytochrome c oxidase (COI) was performed using an Illumina MiSeq after amplification from internal primers (mlCOIintF, jgHCO2198). Bioinformatic analysis of COI reads revealed 187 OTUs (operational taxonomic units), among which Arthropoda, Annelida, Mollusca, and Plathelminthes prevail, accounting for 52, 30, 10, and 4%, respectively. In general, the species diversity of the bay’s fauna by the Shannon index (3.2 bits) is comparable to other areas of Lake Baikal. The most diverse composition of taxa (22 OTUs) of the species rank falls on the larvae of chironomids, which confirms the effectiveness of the method for identifying juvenile stages of invertebrates. The research results are the first step toward improving the system of hydrobiological monitoring of Lake Baikal based on the synthesis of molecular genetic diagnostics of invertebrates and classical morphological taxonomy.</p><h1>metabaR: An r package for the evaluation and improvement of DNA metabarcoding data quality</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13552">doi:10.1111/2041-210X.13552</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is becoming the tool of choice for biodiversity assessment across taxa and environments. Yet, the artefacts present in metabarcoding datasets often preclude a proper interpretation of ecological patterns. Bioinformatic pipelines to remove experimental noise exist. However, these often only partially target produced artefacts, or are marker specific. In addition, assessments of data curation quality and chosen filtering thresholds are seldom available in existing pipelines, partly due to the lack of appropriate visualisation tools. Here, we present metabaR, an r package that provides a comprehensive suite of tools to effectively curate DNA metabarcoding data after basic bioinformatic analyses. In particular, metabaR uses experimental negative or positive controls to identify different types of artefactual sequences, that is, contaminants and tag-jumps. It also flags potentially dysfunctional PCRs based on PCR replicate similarities when those are available. Finally, metabaR provides tools to visualise DNA metabarcoding data characteristics in their experimental context as well as their distribution, and facilitates assessment of the appropriateness of data curation filtering thresholds. metabaR is applicable to any DNA metabarcoding experimental design but is most powerful when the design includes experimental controls and replicates. More generally, the simplicity and flexibility of the package makes it applicable any DNA marker, and data generated with any sequencing platform, and pre-analysed with any bioinformatic pipeline. Its outputs are easily usable for downstream analyses with any ecological r package. metabaR complements existing bioinformatics pipelines by providing scientists with a variety of functions to effectively clean DNA metabarcoding data and avoid serious misinterpretations. It thus offers a promising platform for automatised data quality assessments of DNA metabarcoding data for environmental research and biomonitoring.</p><h1>eDNAir: Proof of concept that animal DNA can be collected from air sampling</h1>
<p><a href="https://doi.org/10.7717/peerj.11030">doi:10.7717/peerj.11030</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is one of the fastest developing tools for species biomonitoring and ecological research. However, despite substantial interest from research, commercial and regulatory sectors, it has remained primarily a tool for aquatic systems with a small amount of work in substances such as soil, snow and rain. Here we demonstrate that eDNA can be collected from air and used to identify mammals. Our proof of concept successfully demonstrated that eDNA sampled from air contained mixed templates which reflect the species known to be present within a confined space and that this material can be accessed using existing sampling methods. We anticipate this demonstration will initiate a much larger research programme in terrestrial airDNA sampling and that this may rapidly advance biomonitoring approaches. Lastly, we outline these and potential related applications we expect to benefit from this development.</p><h1>DNA metabarcoding of zooplankton communities: Species diversity and seasonal variation revealed by 18S rRNA and COI</h1>
<p><a href="https://doi.org/10.7717/peerj.11057">doi:10.7717/peerj.11057</a></p>

<p><a href=None>None</a></p>
<p>Background. Zooplankton is an important component of aquatic organisms and has important biological and economical significance in freshwater ecosystems. However, traditional methods that rely on morphology to classify zooplankton require expert taxonomic skills. Moreover, traditional classification methods are time-consuming and labor-intensive, which is not practical for the design of conservation measures and ecological management tools based on zooplankton diversity assessment. Methods. We used DNA metabarcoding technology with two different markers: The nuclear small subunit ribosomal RNA (18S rRNA) and mitochondrial cytochrome c oxidase (COI), to analyze 72 zooplankton samples collected in 4 seasons and 9 locations from the Sanmenxia Reservoir. We investigated seasonal changes in the zooplankton community and their relationship with water environmental factors. Results. A total of 190 species of zooplankton were found, belonging to 12 phyla, 24 classes, 61 orders, 111 families, and 174 genera. Protozoa, especially ciliates, were the most diverse taxa. Richness and relative abundance of zooplankton showed significant seasonal changes. Both alpha and beta diversity showed seasonal trends: The diversity in summer and autumn was higher than that in winter and spring. The zooplankton diversity was most similar in winter and spring. By correlating metabarcoding data and water environmental factors, we proved that water temperature, chemical oxygen demand, total nitrogen and ammoniacal nitrogen were the main environmental factors driving the seasonal changes in zooplankton in the Sanmenxia Reservoir. Water temperature, followed by total nitrogen, were the most influential factors. This study highlights the advantages and some limitations of zooplankton molecular biodiversity assessment using two molecular markers. Subjects Biodiversity, Ecology, Molecular Biology, Zoology, Freshwater Biology</p><h1>Development of an environmental DNA metabarcoding assay for aquatic vascular plant communities</h1>
<p><a href="https://doi.org/10.1002/edn3.120">doi:10.1002/edn3.120</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcodes allow for the simultaneous detection of multiple taxa if the barcode regions meet several key requirements including conserved primer-binding sites, interspecific variability that exceeds intraspecific variability, and relatively short amplicons. Currently, there are no established metabarcoding assays for aquatic vascular plants, which could limit biodiversity assessments and the early detection of alien species. We used a combination of novel and pre-existing primers to generate eDNA metabarcodes from three gene regions that are commonly used for plant barcoding: two regions of chloroplast DNA (rbcL and matK) plus a segment of an internal transcribed spacer (ITS2). We optimized the assay on a mock community of 25 known species and then applied it to wild samples collected from two waterbodies in southern Ontario, Canada (Black River and Seymour Lake). Our novel rbcL primers, which amplify a fragment of ~220 bp, provided the most comprehensive description of the mock community, identifying all but one of the taxa to species or genus. In addition, after pooling data from five sites within each sampled waterbody, metabarcodes generated by this same primer pair identified more taxa than all other primer pairs; more specifically, 20 and 26 taxa were identified from Black River and Seymour Lake, respectively, to species or genus. Across the two sites, nine of the identified taxa are alien invasive aquatic plants. Five of these invasive species have no previous reports from our sites, and in some cases have no known established Ontario populations; our data therefore suggest an urgent need to increase surveillance for these aliens. Overall, our study showed that eDNA metabarcoding with a novel rbcL primer pair provides a cost-effective method for simultaneously detecting multiple aquatic vascular plant taxa and is a valuable tool for the early detection of invasive species.</p><h1>Species detection from aquatic eDNA: Assessing the importance of capture methods</h1>
<p><a href="https://doi.org/10.1002/edn3.130">doi:10.1002/edn3.130</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is increasingly used for biodiversity monitoring, particularly in aquatic systems. However, each step, from sample collection to bioinformatic analysis, can introduce biases and influence the reliability of results. While much effort has been put into the optimization of laboratory methods, less attention has been devoted to estimate the impacts of eDNA capture methods. To address this issue, water samples were collected at nine small ponds and puddles where up to 10 amphibian species occur, using precipitation, disc filters, and capsules. We focused on targeted detection of an amphibian species, Salamandra salamandra, and on the composition of the whole amphibian community. Species detection was performed using a novel qPCR assay for S. salamandra and high-throughput sequencing, combined with stringent versus relaxed PCR replication thresholds. Filtration techniques (disc filters and capsules) outperformed precipitation, generating a higher number of detections of S. salamandra and higher amounts of captured eDNA, while species detection was identical between disc filters and capsules. There were no significant differences between capture methods regarding amphibian community composition. The variation in detection success associated with capture methods was far higher than that associated with PCR replication, regardless of the detection method used. Our results highlight the importance of choosing a suitable capture method for eDNA studies and suggest that the choice of capture method outweighs the choice of detection method used. To the best of our knowledge, this is the first study to compare high-capacity capsules with common eDNA methods for water samples, such as precipitation and standard disc filters.</p><h1>A Facile High-Throughput Model of Surface-Independent Staphylococcus aureus Biofilms by Spontaneous Aggregation</h1>
<p><a href="https://doi.org/10.1128/mSphere.00186-21">doi:10.1128/mSphere.00186-21</a></p>

<p><a href=None>None</a></p>
<p>Many microbes in their natural habitats are found in biofilm ecosystems attached to surfaces and not as free-floating (planktonic) organisms. Furthermore, it is estimated that nearly 80% of human infections are associated with biofilms. Biofilms are traditionally defined as three-dimensional, structured microbial communities that are attached to a surface and encased in a matrix of exopolymeric material. While this view of biofilm largely arises from in vitro studies under static or flow conditions, in vivo observations have indicated that this view of biofilms is essentially true only for foreign-body infections on catheters or implants where biofilms are attached to the biomaterial. In mucosal infections such as chronic wounds or cystic fibrosis or joint infections, biofilms can be found unattached to a surface and as three-dimensional aggregates. In this work, we describe a high-throughput model of aggregate biofilms of methicillin-resistant Staphylococcus aureus (MRSA) using 96-well plate hanging-drop technology. We show that MRSA forms surface-independent biofilms, distinct from surface-attached biofilms, that are rich in exopolymeric proteins, polysaccharides, and extracellular DNA (eDNA), express biofilm-related genes, and exhibit heightened antibiotic resistance. We also show that the surface-independent biofilms of clinical isolates of MRSA from cystic fibrosis and central catheter-related infections demonstrate morphological differences. Overall, our results show that biofilms can form by spontaneous aggregation without attachment to a surface, and this new in vitro system can model surface-independent biofilms that may more closely mimic the corresponding physiological niche during infection. IMPORTANCE The canonical model of biofilm formation begins with the attachment and growth of microbial cells on a surface. While these in vitro models reasonably mimic biofilms formed on foreign bodies such as catheters and implants, this is not the case for biofilms formed in cystic fibrosis and chronic wound infections, which appear to present as aggregates not attached to a surface. The hanging-drop model of biofilms of methicillin-resistant Staphylococcus aureus (MRSA), the major causative organism of skin and soft tissue infections, shows that these biofilms display morphological and antibiotic response patterns that are distinct from those of their surface-attached counterparts, and biofilm growth is consistent with their in vivo location. The simplicity and throughput of this model enable adoption to investigate other single or polymicrobial biofilms in a physiologically relevant setting.</p><h1>Composition and distribution of fish environmental DNA in an Adirondack watershed</h1>
<p><a href="https://doi.org/10.7717/peerj.10539">doi:10.7717/peerj.10539</a></p>

<p><a href=None>None</a></p>
<p>Background: Environmental DNA (eDNA) surveys are appealing options for monitoring aquatic biodiversity. While factors affecting eDNA persistence, capture and amplification have been heavily studied, watershed-scale surveys of fish communities and our confidence in such need further exploration. Methods: We characterized fish eDNA compositions using rapid, low-volume filtering with replicate and control samples scaled for a single Illumina MiSeq flow cell, using the mitochondrial 12S ribosomal RNA locus for taxonomic profiling. Our goals were to determine: (1) spatiotemporal variation in eDNA abundance, (2) the filtrate needed to achieve strong sequencing libraries, (3) the taxonomic resolution of 12S ribosomal sequences in the study environment, (4) the portion of the expected fish community detectable by 12S sequencing, (5) biases in species recovery, (6) correlations between eDNA compositions and catch per unit effort (CPUE) and (7) the extent that eDNA profiles reflect major watershed features. Our bioinformatic approach included (1) estimation of sequencing error from unambiguous mappings and simulation of taxonomic assignment error under various mapping criteria; (2) binning of species based on inferred assignment error rather than by taxonomic rank; and (3) visualization of mismatch distributions to facilitate discovery of distinct haplotypes attributed to the same reference. Our approach was implemented within the St. Regis River, NY, USA, which supports tribal and recreational fisheries and has been a target of restoration activities. We used a large record of St. Regis-specific observations to validate our assignments. Results: We found that 300 mL drawn through 25-mm cellulose nitrate filters yielded greater than 5 ng/μL DNA at most sites in summer, which was an approximate threshold for generating strong sequencing libraries in our hands. Using inferred sequence error rates, we binned 12S references for 110 species on a state checklist into 85 single-species bins and seven multispecies bins. Of 48 bins observed by capture survey in the St. Regis, we detected eDNA consistent with 40, with an additional four detections flagged as potential contaminants. Sixteen unobserved species detected by eDNA ranged from plausible to implausible based on distributional data, whereas six observed species had no 12S reference sequence. Summed log-ratio compositions of eDNA-detected taxa correlated with log(CPUE) (Pearson's R = 0.655, P < 0.001). Shifts in eDNA composition of several taxa and a genotypic shift in channel catfish (Ictalurus punctatus) coincided with the Hogansburg Dam, NY, USA. In summary, a simple filtering apparatus operated by field crews without prior expertise gave useful summaries of eDNA composition with minimal evidence of field contamination. 12S sequencing achieved useful taxonomic resolution despite the short marker length, and data exploration with standard bioinformatic tools clarified taxonomic uncertainty and sources of error.</p><h1>Integration of DNA-based approaches in aquatic ecological assessment using benthic macroinvertebrates</h1>
<p><a href="https://doi.org/10.3390/w13030331">doi:10.3390/w13030331</a></p>

<p><a href=None>None</a></p>
<p>Benthic macroinvertebrates are among the most used biological quality elements for assessing the condition of all types of aquatic ecosystems worldwide (i.e., fresh water, transitional, and marine). Current morphology-based assessments have several limitations that may be circumvented by using DNA-based approaches. Here, we present a comprehensive review of 90 publications on the use of DNA metabarcoding of benthic macroinvertebrates in aquatic ecosystems bioassessments. Metabarcoding of bulk macrozoobenthos has been preferentially used in fresh waters, whereas in marine waters, environmental DNA (eDNA) from sediment and bulk communities from deployed artificial structures has been favored. DNA extraction has been done predominantly through commercial kits, and cytochrome c oxidase subunit I (COI) has been, by far, the most used marker, occasionally combined with others, namely, the 18S rRNA gene. Current limitations include the lack of standardized protocols and broad-coverage primers, the incompleteness of reference libraries, and the inability to reliably extrapolate abundance data. In addition, morphology versus DNA benchmarking of ecological status and biotic indexes are required to allow general worldwide implementation and higher end-user confidence. The increased sensitivity, high throughput, and faster execution of DNA metabarcoding can provide much higher spatial and temporal data resolution on aquatic ecological status, thereby being more responsive to immediate management needs.</p><h1>Metabarcoding of zooplankton diversity within the Chukchi Borderland, Arctic Ocean: improved resolution from multi-gene markers and region-specific DNA databases</h1>
<p><a href="https://doi.org/10.1007/s12526-020-01136-x">doi:10.1007/s12526-020-01136-x</a></p>

<p><a href=None>None</a></p>
<p>DNA-based analyses have become powerful tools for characterizing the metazoan biodiversity of diverse marine ecosystems. Metabarcoding (i.e., large-scale taxonomic identification of complex samples via high-throughput sequencing of a DNA barcode region) frequently uses hypervariable regions of the nuclear eukaryotic 18S ribosomal RNA (rRNA) gene. However, species-level taxonomic identification is hampered by the conservative nature of the 18S gene in comparison to the mitochondrial cytochrome oxidase I (COI) barcode gene. Additionally, metabarcoding relies on reference DNA sequence databases for classification of millions of unknown sequence reads and molecular operational taxonomic units (OTUs); databases that are at present depauperate for marine zooplankton taxa. Here, we characterized the mesozooplankton community for the Chukchi Borderland (CBL) region, western Arctic Ocean, through metabarcoding analysis of the V4 and V9 hypervariable regions of 18S rRNA and a portion of COI. Characterization of zooplankton diversity for the epipelagic and upper mesopelagic layers (0–500 m) was based upon 17 metazoan taxonomic categories encompassing 24 orders in 14 classes. Taxonomic classification using the V4 and V9 markers was most reliable for orders, with copepods dominating OTU counts. To increase taxonomic resolution and allow detection of species, V4, V9, and COI OTUs were classified against DNA sequence databases for the Arctic Ocean for the subclass Copepoda. The geographic region-specific databases for 18S rRNA and COI resulted in the detection and identification of 6 genera and 49 species of copepods representing 23 families, a marked increase in the taxonomic classification of the 18S rRNA markers. The greatest copepod species diversity was captured with V4 (34 species) followed by COI (28 species) with the least copepod diversity detected by V9 (5 species). Our results demonstrate the power of using multiple gene markers, with DNA reference databases that are specific to the geographic region of interest, providing more accurate metabarcoding biodiversity measures for the copepod assemblages compared to universal metazoan sequence reference databases. Results from this study highlight the need for continued DNA barcode sequencing to increase species representation in the reference sequence databases that are crucial for accurate characterization of mesozooplankton communities.</p><h1>Soil fungal diversity and functionality are driven by plant species used in phytoremediation</h1>
<p><a href="https://doi.org/10.1016/j.soilbio.2020.108102">doi:10.1016/j.soilbio.2020.108102</a></p>

<p><a href=None>None</a></p>
<p>Soil biodiversity loss due to pollution may affect ecosystem services negatively. This environmental problem may be solved by phytoremediation, which is an effective strategy to manage and remediate contaminated areas. During this remediation process, the establishment of plant communities may improve soil fungal community structure and, in particular, may favour mycorrhizal symbiotic associations. As a consequence, afforestation of degraded lands will have different outcomes on fungal diversity and functionality, which will depend on the selected tree and shrub species. We analysed soil fungal diversity and functional guilds by high-throughput sequencing of environmental DNA in a trace element contaminated area, part of a large scale phytoremediation project running for 20 years. We selected five habitats for comparison purposes: three under the canopy of selected tree species (wild olive, white poplar and stone pine), adjacent treeless areas (grassland) and non-remediated areas (bare soil). Soil fungal diversity and richness seemed to be enhanced by phytoremediation. White poplar soil had the highest diversity and richness compared to wild olive and stone pine. Fungal communities were especially different between stone pine, with soils rich in organic C and high C:N ratio, and grassland soils. We identified 9,428 fungal OTUs from which 1,283 were assigned to a unique functional guild; the most abundant belonging to saprotrophic, plant pathogenic and ectomycorrhizal functional guilds. Ectomycorrhizal fungi were more abundant in soils under ectomycorrhizal host trees. Saprotrophs were abundant in grassland and wild olive soils, while plant pathogens were abundant in non-remediated soils. The remediation of soils (clean-up and amendment addition) allowed the natural establishment of grassland habitats throughout the study area, increasing fungal diversity, richness, taxonomy and functionality, when compared to non-remediated soils. Tree afforestation allowed the establishment of a forest type community bringing a further recruitment of fungal taxa, mainly the ectomycorrhizal fungal guild. Afforestation with different tree species showed species-specific effects on soil N, organic C, Ca and C:N ratio which led to increased spatial heterogeneity in areas with potential to recruit a wider diversity of fungi.</p><h1>Identifying prey of the Javan mongoose (Urva javanica) in Java from fecal samples using next-generation sequencing</h1>
<p><a href="https://doi.org/10.1007/s42991-020-00086-y">doi:10.1007/s42991-020-00086-y</a></p>

<p><a href=None>None</a></p>
<p>The Javan mongoose (Urva javanica) is a mesopredator native to Java island that has been rarely studied. The limited information raises questions as to its ecological roles, particularly in terms of its effects on agriculture and biodiversity. These questions are critical as its congeneric species, the small Indian mongoose, has been listed among the 100 worst invasive species. To disclose its ecological function, a new approach that is capable of revealing the diet of the Javan mongoose at the species level is required. We conducted this study to identify the prey species of the Javan mongoose using next-generation sequencing methodology. Our study was conducted in a forested area of the southern mountains of Java, Indonesia. We collected fecal samples during the dry season of March–October 2018, extracting DNA using a QIAamp Fast DNA Stool Mini Kit. We performed Illumina sequencing using a 133 bp mini-barcode of the Cytochrome Oxidase I gene. Our rigorous analysis found 17 species, 14 genera, and 12 families of vertebrates and invertebrates in the mongoose feces. The greater bandicoot rat (Bandicota indica) and the domestic chicken (Gallus gallus) made up the primary diet of the Javan mongoose. Cricket, ant, moth, and fly species were also found in the feces. In effect, our results suggested that this species has a relatively diverse diet, one that includes mammals, birds, amphibians, snakes, fish, and many invertebrates. It confirms that the Javan mongoose has an opportunistic diet, which is most likely related to the availability of putative prey.</p><h1>Meiofauna as a valuable bioindicator of climate change in the polar regions</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2020.107133">doi:10.1016/j.ecolind.2020.107133</a></p>

<p><a href=None>None</a></p>
<p>Establishing robust estimates of polar marine biodiversity is important for interpreting future changes in the Arctic; however, despite a recent increase in scientific expeditions, this region remains relatively underexplored. Particularly overlooked in biodiversity assessments are small species, such as protists, fungi, and many small invertebrates that are collectively known as meiofauna. These species contribute to the foundation of food webs and are crucial for the survival of larger species that are economically and socially important. The application of high-throughput sequencing methodologies has proven effective for biomonitoring small metazoan species but has sparingly been applied in the Arctic. We used a metabarcoding approach to assess the diversity of sea ice and sediment-associated metazoans from Utqiaġvik (Barrow), Alaska. Sea ice and sediment samples were collected six times over eight months (January through August) encompassing three seasons (winter, spring, and summer) from polar night to ice-out in August. Biodiversity was assessed as both richness and community composition by incorporating incidence data and phylogenetic distance. Environmental conditions associated with ice, sediment, water, and snow were measured and tested for possible correlations with biodiversity estimates. We found a high number of taxa distributed locally, suggesting that metabarcoding can be effectively applied to Arctic biomonitoring programs. In addition, these results show that season and habitat are significant predictors of meiofaunal biodiversity, supporting hypotheses that meiofauna can be used as a valuable indicator of climate change.</p><h1>Medicinal Plants and Herbal Products From Brazil: How Can We Improve Quality?</h1>
<p><a href="https://doi.org/10.3389/fphar.2020.606623">doi:10.3389/fphar.2020.606623</a></p>

<p><a href=None>None</a></p>
<p>None</p><h1>Regional Diversity of Maritime Antarctic Soil Fungi and Predicted Responses of Guilds and Growth Forms to Climate Change</h1>
<p><a href="https://doi.org/10.3389/fmicb.2020.615659">doi:10.3389/fmicb.2020.615659</a></p>

<p><a href=None>None</a></p>
<p>We report a metabarcoding study documenting the fungal taxa in 29 barren fellfield soils sampled from along a 1,650 km transect encompassing almost the entire maritime Antarctic (60–72°S) and the environmental factors structuring the richness, relative abundance, and taxonomic composition of three guilds and growth forms. The richness of the lichenised fungal guild, which accounted for 19% of the total fungal community, was positively associated with mean annual surface air temperature (MASAT), with an increase of 1.7 operational taxonomic units (OTUs) of lichenised fungi per degree Celsius rise in air temperature. Soil Mn concentration, MASAT, C:N ratio, and pH value determined the taxonomic composition of the lichenised guild, and the relative abundance of the guild was best predicted by soil Mn concentration. There was a 3% decrease in the relative abundance of the saprotrophic fungal guild in the total community for each degree Celsius rise in air temperature, and the OTU richness of the guild, which accounted for 39% of the community, was negatively associated with Mn concentration. The taxonomic composition of the saprotrophic guild varied with MASAT, pH value, and Mn, NH4+-N, and SO42− concentrations. The richness of the yeast community, which comprised 3% of the total fungal community, was positively associated with soil K concentration, with its composition being determined by C:N ratio. In contrast with a similar study in the Arctic, the relative abundance and richness of lichenised fungi declined between 60°S and 69°S, with those of saprotrophic Agaricales also declining sharply in soils beyond 63°S. Basidiomycota, which accounted for 4% of reads, were much less frequent than in vegetated soils at lower latitudes, with the Ascomycota (70% of reads) being the dominant phylum. We conclude that the richness, relative abundance, and taxonomic composition of guilds and growth forms of maritime Antarctic soil fungi are influenced by air temperature and edaphic factors, with implications for the soils of the region as its climate changes during the 21st century.</p><h1>Secondary metabolite biosynthetic diversity in Arctic Ocean metagenomes</h1>
<p><a href="https://doi.org/10.1099/MGEN.0.000731">doi:10.1099/MGEN.0.000731</a></p>

<p><a href=None>None</a></p>
<p>Polyketide synthases (PKSs) and non-ribosomal peptide synthetases (NRPSs) are mega enzymes responsible for the biosynthesis of a large fraction of natural products (NPs). Molecular markers for biosynthetic genes, such as the ketosynthase (KS) domain of PKSs, have been used to assess the diversity and distribution of biosynthetic genes in complex microbial communities. More recently, metagenomic studies have complemented and enhanced this approach by allowing the recovery of complete biosynthetic gene clusters (BGCs) from environmental DNA. In this study, the distribution and diversity of biosynthetic genes and clusters from Arctic Ocean samples (NICE-2015 expedition), was assessed using PCR-based strategies coupled with high-throughput sequencing and metagenomic analysis. In total, 149 KS domain OTU sequences were recovered, 36 % of which could not be assigned to any known BGC. In addition, 74 bacterial metagenome-assembled genomes were recovered, from which 179 BGCs were extracted. A network analysis identified potential new NP families, including non-ribosomal peptides and polyketides. Complete or near-complete BGCs were recovered, which will enable future heterologous expression efforts to uncover the respective NPs. Our study represents the first report of biosynthetic diversity assessed for Arctic Ocean metagenomes and highlights the potential of Arctic Ocean planktonic microbiomes for the discovery of novel secondary metabolites. The strategy employed in this study will enable future bioprospection, by identifying promising samples for bacterial isolation efforts, while providing also full-length BGCs for heterologous expression.</p><h1>Phytool, a ShinyApp to homogenise taxonomy of freshwater microalgae from DNA barcodes and microscopic observations</h1>
<p><a href="https://doi.org/10.3897/mbmg.5.74096">doi:10.3897/mbmg.5.74096</a></p>

<p><a href=None>None</a></p>
<p>Methods for biomonitoring of freshwater phytoplankton are evolving rapidly with eDNA-based methods, offering great complementarity with microscopy. Metabarcoding approaches have been more commonly used over the last years, with a continuous increase in the amount of data generated. Depending on the researchers and the way they assigned barcodes to species (bioinformatic pipelines and molecular reference databases), the taxonomic assignment obtained for HTS DNA reads might vary. This is also true for traditional taxonomic studies by microscopy with regular adjustments of the classification and taxonomy. For those reasons (leading to non-homogeneous taxonomies), gap-analyses and comparisons between studies become even more challenging and the curation processes to find potential consensus names are time-consuming. Here, we present a web-based application (Phytool), developed with ShinyApp (Rstudio), that aims to make the harmonisation of taxonomy easier and in a more efficient way, using a complete and up-to-date taxonomy reference database for freshwater microalgae. Phytool allows users to homogenise and update freshwater phytoplankton taxonomical names from sequence files and data tables directly uploaded in the application. It also gathers barcodes from curated references in a user-friendly way in which it is possible to search for specific organisms. All the data provided are downloadable with the possibility to apply filters in order to select only the required taxa and fields (e.g. specific taxonomic ranks). The main goal is to make accessible to a broad range of users the connection between microscopy and molecular biology and taxonomy through different ready-to-use functions. This study estimates that only 25% of species of freshwater phytoplankton in Phytobs are associated with a barcode. We plead for an increased effort to enrich reference databases by coupling taxonomy and molecular methods. Phytool should make this crucial work more efficient.</p><h1>Metabarcoding hyperdiverse kelp holdfast communities on temperate reefs: An experimental approach to inform future studies</h1>
<p><a href="https://doi.org/10.1002/edn3.265">doi:10.1002/edn3.265</a></p>

<p><a href=None>None</a></p>
<p>Classical taxonomic approaches to quantifying biodiversity can be notoriously laborious and restrictive. Instead, molecular metabarcoding is emerging as a rapid, high-throughput, and cost-effective tool to catalog biodiversity. Despite the appeal of metabarcoding, methodological and procedural biases must be understood before robust biodiversity inferences can be made. Here, we use CO1 metabarcoding to characterize marine eukaryote communities associated with Ecklonia radiata, the dominant eco-engineering kelp of temperate Australasia. To establish a standardized and reproducible community metabarcoding protocol, we examined the influence of different sample preparation, laboratory, and bioinformatic steps on inferences of species richness and composition of communities associated with E. radiata holdfasts (the root-like structure anchoring the kelp to the substratum) sampled from northeastern New Zealand. Specifically, we examined the effect of sieving the community into different size fractions and the replicability of results across DNA extractions, polymerase chain reactions and sequencing. Overall, we found that sieving the community into two size fractions before DNA extraction enabled detection of a greater diversity of taxa than not sieving samples. When compared with traditional morphology-based inventories of kelp holdfast biodiversity, we found that although the taxonomic precision of our metabarcoding approach at the species and genus level was limited by the availability of reference sequences in public repositories, we recovered ~40% more taxa and a greater taxonomic breadth of organisms than morphological surveys (e.g., 18 phyla as compared with 14 phyla). On the basis of our findings, we provide methodological guidelines for the use of metabarcoding as a tool for surveying and monitoring the hyperdiverse species assemblages associated with kelp holdfasts.</p><h1>Plankton biodiversity and species co-occurrence based on environmental DNA – a multiple marker study</h1>
<p><a href="https://doi.org/10.3897/MBMG.5.72371">doi:10.3897/MBMG.5.72371</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding in combination with high-throughput sequencing (HTS) allows simultaneous detection of multiple taxa by targeting single or several taxonomically informative gene regions from environmental DNA samples. In this study, a multiple-marker HTS approach was applied to investigate the plankton diversity and seasonal succession in the Baltic Sea from winter to autumn. Four different markers targeting the 16S, 18S, and 28S ribosomal RNA genes were employed, including a marker for more efficient dinoflagellate detection. Typical seasonal changes were observed in phyto- and bacterioplankton communities. In phytoplankton, the appearance patterns of selected common, dominant, or harmful species followed the patterns also confirmed based on 20 years of phytoplankton monitoring data. In the case of zooplankton, both macro- and microzooplankton species were detected. However, no seasonal patterns were detected in their appearance. In total, 15 and 2 new zoo- and phytoplankton species were detected from the Baltic Sea. HTS approach was especially useful for detecting microzooplankton species as well as for investigating the co-occurrence and potential interactions of different taxa. The results of this study further exemplify the efficiency of metabarcoding for biodiversity monitoring and the advantage of employing multiple markers through the detection of species not identifiable based on a single marker survey and/or by traditional morphology-based methods.</p><h1>Metabarcoding advances for ecology and biogeography of neotropical protists: What do we know, where do we go?</h1>
<p><a href="https://doi.org/10.1590/1676-0611-BN-2021-1214">doi:10.1590/1676-0611-BN-2021-1214</a></p>

<p><a href=None>None</a></p>
<p>The Neotropics is one of the most diverse regions of the globe in terms of plants and animal species. Regarding the microbial world, however, little is known about the diversity and biogeography patterns of microorganisms in the Neotropics. The biogeography of several microbial taxonomic groups is still missing and/or incomplete, such as the protists. Despite the hard taxonomic identification of protists, the advance of molecular techniques (e.g., metabarcoding) have allowed to better explore the distribution of several protistan groups. Our goal here was to summarize the available information of Neotropical protists, focusing on metabarcoding studies, to explore what these data evidence on their ecology and biogeography. For this, we reviewed the findings from all articles that focused on or included the terrestrial protists using a metabarcoding approach and identified the gaps and future perspectives in this research field. We found that Neotropical protistan diversity patterns seem to be, at least in part, congruent with that of macro-organisms and, different than plants and bacteria, just weakly explained by environmental variables. We argue that studies with standardized protocols including different ecoregions are necessary, such as temperate forests, grasslands, and savannas from Southern of South America and Northern Atlantic Forest, to fully characterize the ecology and biogeography on Neotropical protists. Furthermore, dismembering evolutionary lineages and functional guilds of protists are important to better understand the relationship between diversity, dispersal abilities, and functionality of particular taxa of protists in their habitats.</p><h1>Advances in the macrozoobenthos biodiversity monitoring and ecosystem assessment using environmental DNA metabarcoding</h1>
<p><a href="https://doi.org/10.5846/stxb202009162411">doi:10.5846/stxb202009162411</a></p>

<p><a href=None>None</a></p>
<p>Macrozoobenthos are the most diversified organisms in freshwater ecosystem, playing a vital role in maintaining the structural and functional integrity of freshwater ecosystem. They are one of the most significant bioindicators that have been widely used in water quality monitoring and assessment. Nowadays, anthropogenic disturbance, pollution and global warming lead to a remarkable decline of the macrozoobenthos biodiversity. Therefore, how to provide substantial data for macrozoobenthos biodiversity monitoring rapidly, accurately, and reliably has become a major issue in water environment management. However, the traditional morphology-based biomonitoring survey method can hardly fulfill current biomonitoring requirements due to being costly, time consuming, and labor intensive. Moreover, such method is highly dependent on experienced taxonomists and often results in low precision and unverifiable taxonomic data. Environmental DNA metabarcoding (eDNA metabarcoding), a newly emerged approach, is considered as a promising survey tool for aquatic biodiversity monitoring. Compared with traditional biomonitoring survey methods, eDNA metabarcoding remarkably outperforms morphology-based method by its low cost, straightforward workflow, and high sensitivity in species detection of macrozoobenthos. Particularly, it circumvents the taxonomy literature and expertise, providing rapid, accurate, and high-throughput identification simultaneously to a large variety of taxa regardless of the organism conditions and environmental disturbances, thus permitting the easy access of large scale biomonitoring programs for freshwater ecosystem. Currently, the application of eDNA metabarcoding in macrozoobenthos biodiversity monitoring is largely affected by several technical factors, such as the sampling protocols, primer selection, and the completeness of reference database. Particularly, different methods and procedures often lead to contrasting results. Hence much attention is drawn to the field workflow and laboratory methodologies to be standardized to allow results to be compared across studies. In this paper, we reviewed the recent published studies referring to macrozoobenthos biodiversity monitoring using eDNA metabarcoding to discuss its definition, advantages, applications and its methodologies used in macrozoobenthos biodiversity biomonitoring and assessment. Especially we discussed the vital technical factors that influences eDNA metabarcoding, including sample collection protocol, gene marker selection, primer design, PCR bias, the completeness of reference database, and their corresponding recommendations to be optimized. Based on the summary above, we give several helpful suggestions for improving the efficiency and accuracy of macrozoobenthos monitoring using eDNA metabarcoding. We expect that this review will provide reliable guidelines for developing a sound macrozoobenthos biomonitoring design using eDNA metabarcoding. At last, we gave perspectives regarding several emerging applications and innovative methodologies of eDNA metabarcoding can be used in ecology.</p><h1>Advanced molecular-based surveillance of quagga and zebra mussels: A review of environmental DNA/RNA (eDNA/eRNA) studies and considerations for future directions</h1>
<p><a href="https://doi.org/10.3897/neobiota.66.60751">doi:10.3897/neobiota.66.60751</a></p>

<p><a href=None>None</a></p>
<p>Sensitive methods, capable of rapidly and accurately detecting aquatic invasive species, are in demand. Molecular-based approaches, such as environmental DNA (eDNA) surveys, satisfy these requirements and have grown in popularity. As such, eDNA surveys could aid the effort to combat the colonisation and spread of two notoriously invasive freshwater mussel species, the quagga mussel (Dreissena rostriformis bu-gensis) and zebra mussel (D. polymorpha), through improved surveillance ability. Here, we provide a review of dreissenid eDNA literature (both grey and published), summarising efforts involved in the development of various assays for use in multiple different technologies (e.g. quantitative PCR, high-throughput sequencing and loop-mediated isothermal amplification) and sampling scenarios. We discuss important discoveries made along the way, including novel revelations involving environmental RNA (eRNA), as well as the advantages and limitations of available methods and instrumentation. In closing, we highlight critical remaining gaps, where further investigation could lead to advancements in dreissenid monitoring capacity.</p><h1>How can integrated morphotaxonomy-and metabarcoding-based diatom assemblage analyses best contribute to the ecological assessment of streams?</h1>
<p><a href="https://doi.org/10.3897/MBMG.5.68438">doi:10.3897/MBMG.5.68438</a></p>

<p><a href=None>None</a></p>
<p>Environmental conditions, such as nutrient concentrations, salinity, elevation etc., shape diatom assemblages of periphytic biofilms. These assemblages respond rapidly to environmental changes, a fact which makes diatoms valuable bioindicators. Hence, freshwater biomonitoring programmes currently use diatom indices (e.g. EU Water Framework Directive - WFD). To date, microscopy-based assessments require high taxonomic expertise for diatom identification at the species level. High-throughput technologies now provide cost-effective identification approaches that are promising, complementary or alternative tools for bioassessment. The suitability of the metabarcoding method is evaluated for the first time in the Cyprus streams WFD monitoring network, an eastern Mediterranean country with many endemic species and results are compared to the results acquired from the morphotaxonomic analysis. Morphotaxonomic identification was conducted microscopically, using the most updated taxonomic concepts, literature and online resources. At the same time, DNA metabarcoding involved the use of the rbcL 312 bp barcode, high-throughput sequencing and bioinformatic analysis. The ecological status was calculated using the IPS Index. Results show a positive correlation between morpho-taxonomic and molecular IPS scores. Discrepancies between the two methodologies are related to the limitations of both techniques. This study confirmed that Fistulifera saprophila can have a crucial role in key differences observed, as it negatively influences IPS scores and microscopy methods frequently overlook it. Importantly, gaps in the DNA barcoding reference databases lead to a positive overestimation in IPS scores. Overall, we conclude that DNA metabarcoding offsets the morphotaxonomic methodology for the ecological quality assessment of freshwaters.</p><h1>Ciliate SSU-rDNA reference alignments and trees for phylogenetic placements of metabarcoding data</h1>
<p><a href="https://doi.org/10.3897/MBMG.5.69602">doi:10.3897/MBMG.5.69602</a></p>

<p><a href=None>None</a></p>
<p>Although ciliates are one of the most dominant microbial eukaryotic groups in many environments, there is a lack of updated global ciliate alignments and reference trees that can be used for phylogenetic placement methods to analyze environmental metabarcoding data. Here we fill this gap by providing reference alignments and trees for those ciliates taxa with available SSU-rDNA sequences derived from identified species. Each alignment contains 478 ciliate and six outgroup taxa, and they were made using different masking strategies for alignment positions (unmasked, masked and masked except the hypervariable V4 region). We constrained the monophyly of the major ciliate groups based on the recently updated classification of protists and based on phylogenomic data. Taxa of uncertain phylogenetic position were kept unconstrained, except for Mesodinium species that we constrained to form a clade with the Litostomatea. These ciliate reference alignments and trees can be used to perform taxonomic assignments of metabarcoding data, discover novel ciliate clades, estimate species richness, and overlay measured ecological parameters onto the phylogenetic placements.</p><h1>Application and Evaluation of Environmental Dna Technology in Fish Diversity Research In Xiangshan Bay</h1>
<p><a href="https://doi.org/10.12264/JFSC2020-0219">doi:10.12264/JFSC2020-0219</a></p>

<p><a href=None>None</a></p>
<p>Fish are one of the key groups of organisms in marine ecosystems and are also the main object of marine fisheries. Knowledge of their diversity and distribution not only provides basic data for the sustaina-ble development and utilization of marine ecosystems but also set up a theoretical basis for evaluating the influence of human disturbance factors (such as marine development and fishing) on these ecosystems. With the rapid development of biotechnology in recent years, environmental DNA (eDNA) technology has been gradually applied in the field of ecology owing to its advantages in sensitivity, accuracy, and cost, but there have been few local surveys of marine fish resources and their diversity using eDNA, especially in the local marine areas and fishing grounds. Xiangshan Bay is a semi-closed narrow harbor that is located on the north coast of the central Zhejiang Province, China. For years, overfishing, environmental pollution, and large-scale marine development activities have caused the ecological environment to deteriorate, and fishery resources have declined. Biological resource studies in Xiangshan Bay are still insufficient, and research on fish community structure and diversity are lacking. However, owing to the existence of many artificial and natural reefs, fish trawling surveys cannot be carried out in the area, which also limits the routine detection of fish biodiversity (especially for reef-dwelling fish) in the area. Therefore, the application of eDNA technol-ogy is of great significance to the assessment of fish biodiversity in Xiangshan Bay. In the present study, by eDNA sample collection and high-throughput sequencing analysis, combined with survey data of fishery re-sources in the same region, the fish species composition and diversity characteristics of the main fish com-munities in Xiangshan Bay were expounded. The prospect of applying eDNA technology in fish diversity studies of typical sea areas was also discussed. The results showed that 26 common fish species (belonging to seven orders of Actinopterygii and from 21 genera) were detected from the aquatic eDNA samples from Xiangshan Bay. Two orders with the highest abundance were Clupeiformes and Perciformes, whose relative abundance accounted for 92.5% of the total fish species abundance. Among all the fish species identified from eDNA analysis, Konosirus punctatus and Acanthopagrus schlegelii had the highest abundance, which accounted for 45.85% and 17.69% of the total fish abundance, respectively. Both species also contributed the most to fisheries resources in Xiangshan Bay. The next most abundant species were Tridentiger barbatus and Lateolabrax japonicus. The sequence abundances of such taxa were similar to the trends in the fisheries catch data. Compared with traditional survey methods, the eDNA analysis had high sensitivity, high accuracy, and low cost and was suitable for studying the fish diversity of relevant sea areas. The present research not only enriched the structural and functional information of the Xiangshan Bay aquatic ecosystem but also provided basic information to support the management and restoration of marine ecosystems in this marine area.</p><h1>Metabarcoding assays for the detection of freshwater mussels (Unionida) with environmental DNA</h1>
<p><a href="https://doi.org/10.1002/edn3.166">doi:10.1002/edn3.166</a></p>

<p><a href=None>None</a></p>
<p>Freshwater mussels of the order Unionida are a widely distributed taxon that are important in maintaining freshwater ecosystems and are also highly imperiled throughout the world. Monitoring of mussel populations with environmental DNA (eDNA) is an attractive alternative to traditional methods because it is noninvasive and requires less labor and taxonomic knowledge from field personnel. We developed eDNA metabarcoding assays specific to freshwater mussels and tested them at six sites in the Clinch River, located in the southeastern United States. Our objective was to determine the utility of eDNA metabarcoding for future monitoring of mussel populations and restoration efforts in this watershed. Two metabarcoding assays that target the mitochondrial DNA regions of the cytochrome c oxidase subunit I (COI) and NADH dehydrogenase subunit (ND1) genes were developed and tested. Our assays appear to be order specific, amplifying members from the two families found in North America, Unionidae and Margaritiferidae, while not amplifying nontarget fish or other bivalve species. From the field collected samples, our assays together detected 19 species, eight of which are listed as federally endangered. The assays also detected 42%, 58%, and 54% of the species identified by recent quantitative visual mussel surveys at three sampling sites. Increased sampling effort by processing a greater water volume or number of samples will likely increase species detections. These eDNA metabarcoding assays may enable enhanced monitoring of freshwater mussel assemblages and subsequently inform conservation efforts.</p><h1>eDNA-based biomonitoring at an experimental German vineyard to characterize how management regimes shape ecosystem diversity</h1>
<p><a href="https://doi.org/10.1002/edn3.131">doi:10.1002/edn3.131</a></p>

<p><a href=None>None</a></p>
<p>There is growing interest in the application of sustainable agricultural methods to minimize the environmental impact of farming and thus aiding quantification of the actual benefit that such approaches may confer. We applied DNA metabarcoding with the aim of exploring how the diversity of fungi and arthropods were affected by different agricultural management systems (integrated, organic, biodynamic) at the experimental vineyard of Geisenheim (Rheingau, Germany). Data were generated for the bloom and harvest periods in 2017, using environmental DNA (eDNA) metabarcoding analysis of both soil and vane trap samples. Our data revealed four principal results. (a) Overall richness of vane trap samples was unaffected by the management systems, likely due to the relatively small scale of the plots compared to the ranges of taxa such as the arthropods caught. In contrast, however, the richness of soil-living taxa appeared to be negatively affected by conventional treatments, especially at harvest. (b) Analysis of similarity revealed that the species composition was significantly differentiated by management systems for both fungal and other taxa in both sample types. (c) Taxonomic analysis of fungi revealed that the management system drove differentiation in the abundance patterns for wine-related fungi. Overall, our study reiterates the potential of eDNA techniques as a tool for assessing how biodiversity is affected by different agricultural management regimes, and we hope such approaches will be adopted in future research aimed at guiding vineyard management decisions.</p><h1>Biodiversity Soup II: A bulk-sample metabarcoding pipeline emphasizing error reduction</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13602">doi:10.1111/2041-210X.13602</a></p>

<p><a href=None>None</a></p>
<p>Despite widespread recognition of its great promise to aid decision-making in environmental management, the applied use of metabarcoding requires improvements to reduce the multiple errors that arise during PCR amplification, sequencing and library generation. We present a co-designed wet-lab and bioinformatic workflow for metabarcoding bulk samples that removes both false-positive (tag jumps, chimeras, erroneous sequences) and false-negative (‘dropout’) errors. However, we find that it is not possible to recover relative-abundance information from amplicon data, due to persistent species-specific biases. To present and validate our workflow, we created eight mock arthropod soups, all containing the same 248 arthropod morphospecies but differing in absolute and relative DNA concentrations, and we ran them under five different PCR conditions. Our pipeline includes qPCR-optimized PCR annealing temperature and cycle number, twin-tagging, multiple independent PCR replicates per sample, and negative and positive controls. In the bioinformatic portion, we introduce Begum, which is a new version of DAMe (Zepeda-Mendoza et al., 2016. BMC Res. Notes 9:255) that ignores heterogeneity spacers, allows primer mismatches when demultiplexing samples and is more efficient. Like DAMe, Begum removes tag-jumped reads and removes sequence errors by keeping only sequences that appear in more than one PCR above a minimum copy number per PCR. The filtering thresholds are user-configurable. We report that OTU dropout frequency and taxonomic amplification bias are both reduced by using a PCR annealing temperature and cycle number on the low ends of the ranges currently used for the Leray-FolDegenRev primers. We also report that tag jumps and erroneous sequences can be nearly eliminated with Begum filtering, at the cost of only a small rise in dropouts. We replicate published findings that uneven size distribution of input biomasses leads to greater dropout frequency and that OTU size is a poor predictor of species input biomass. Finally, we find no evidence for ‘tag-biased’ PCR amplification. To aid learning, reproducibility, and the design and testing of alternative metabarcoding pipelines, we provide our Illumina and input-species sequence datasets, scripts, a spreadsheet for designing primer tags and a tutorial.</p><h1>Sharks and rays (Chondrichthyes) around Banggai Island, Banggai MPA, Indonesia: Biodiversity data from an environmental DNA pilot study</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>The Banggai Archipelago is one of six major island groups around Sulawesi Island in Indonesia, within the Wallacea region. Initiated after two district level marine protected areas (MPAs) in the Banggai Archipelago became obsolete under the revised regional autonomy Act (UU 23/2014) before they had become operational, the more extensive provincial level Banggai MPA was formally established under national legislation in 2019. Comprehensive and up-to-date biodiversity data for this MPA are needed; however taxonomic expertise and funding are limited. Furthermore, many taxa are likely to be missed using visual census methods. This study applied molecular biology methods to evaluate biodiversity of the Class Chondrichthyes (sharks and rays) at four sites around Banggai Island, in Banggai Laut District, within the Banggai MPA. Environmental DNA (eDNA) seawater samples were collected in October 2018 (3 replicates per site). The eDNA was extracted at Bionesia in Bali. Metabarcoding (using both standard MiFish 12S primer pairs) and sequence library preparation were conducted at the Barber Lab, University of California Los Angeles (UCLA). High-throughput sequencing was performed on a Nextseq. Generated sequences were processed using the Anacapa Toolkit; elasmobranch sequences were analysed by site and aggregated into amplicon sequence variants (ASVs) using the 60% Bayesian confidence score. Across all sites we identified 11 ASVs belonging to 2 Orders, 2 Families, and 7 genera. Nine ASVs were resolved to species level, while over two fifths of sequences were only assigned to genus level. BLAST search of the NCBI GenBank database and phylogenetic analysis of the ASVs in MEGA 10 produced similar results. Taxa identified included endangered species protected under Indonesian law. The results can inform elasmobranch management in the Banggai MPA. They also highlight a need for further efforts to barcode elasmobranchs in Wallacea.</p><h1>High-throughput sequencing view on the magnitude of global fungal diversity</h1>
<p><a href="https://doi.org/10.1007/s13225-021-00472-y">doi:10.1007/s13225-021-00472-y</a></p>

<p><a href=None>None</a></p>
<p>High-throughput DNA sequencing has dramatically transformed several areas of biodiversity research including mycology. Despite limitations, high-throughput sequencing is nowadays a predominant method to characterize the alpha and beta diversity of fungal communities. Across the papers utilizing high-throughput sequencing approaches to study natural habitats in terrestrial ecosystems worldwide, > 200 studies published until 2019 have generated over 250 million sequences of the primary mycological metabarcoding marker, the nuclear ribosomal internal transcribed spacer 2 (ITS2). Here we show that at a 97% sequence similarity threshold, the total richness of non-singleton fungal taxa across the studies published so far is 1.08 million, mostly Ascomycota (56.8% of the taxa) and Basidiomycota (36.7% of the taxa). The Chao-1 estimate of the total extant fungal diversity based on this dataset is 6.28 million taxa, representing a conservative estimate of global fungal species richness. Soil and litter represent the habitats with the highest alpha diversity of fungi followed by air, plant shoots, plant roots and deadwood with Chao-1 predictions, for samples containing 5000 sequences, of 1219, 569, 392, 228, 215 and 140 molecular species, respectively. Based on the high-throughput sequencing data, the highest proportion of unknown fungal species is associated with samples of lichen and plant tissues. When considering the use of high-throughput sequencing for the estimation of global fungal diversity, the limitations of the method have to be taken into account, some of which are sequencing platform-specific while others are inherent to the metabarcoding approaches of species representation. In this respect, high-throughput sequencing data can complement fungal diversity predictions based on methods of traditional mycology and increase our understanding of fungal biodiversity.</p><h1>Detecting the pest fish, Gambusia affinis from environmental DNA in New Zealand: a comparison of methods</h1>
<p><a href="https://doi.org/10.1080/03014223.2020.1858880">doi:10.1080/03014223.2020.1858880</a></p>

<p><a href=None>None</a></p>
<p>We assessed the usefulness of environmental DNA (eDNA) for monitoring the introduced pest fish Gambusia affinis by filtering water samples from streams in the Nelson and Tasman regions, South Island, New Zealand, known to contain G. affinis, and from streams where G. affinis were absent. We used the Smith Root DNA sampler backpack with two types of filters (1.2 and 5 µm pores), to filter water and capture eDNA, and used quantitative polymerase chain reactions (qPCR), and digital droplet PCR (ddPCR) to measure the amount of G. affinis DNA collected by the two filter sizes. We also used high throughput (Illumina MiSeq) sequencing (HTS) to detect G. affinis. Results from the two PCR methods and the high throughput sequencing were compared to hand net counts of G. affinis. We found that all three methods were equally successful at detecting G. affinis when four replicates were taken from each site, but that sensitivity over all replicates was ddPCR > qPCR > HTS. We conclude that the use of environmental DNA to detect the presence of G. affinis is a useful tool to assist in mapping the distribution of G. affinis and will aid in the control of this invasive species.</p><h1>Revealing hidden plant diversity in arid environments</h1>
<p><a href="https://doi.org/10.1111/ecog.05100">doi:10.1111/ecog.05100</a></p>

<p><a href=None>None</a></p>
<p>Estimating total plant diversity in extreme or hyperarid environments can be challenging, as adaptations to pronounced climate variability include evading prolonged stress periods through seeds or specialized underground organs. Short-term surveys of these ecosystems are thus likely poor estimators of actual diversity. Here we develop a multimethod strategy to obtain a more complete understanding of plant diversity from a community in the Atacama Desert. We explicitly test environmental DNA-based techniques (eDNA) to see if they can reveal the observed and ‘hidden' (dormant or locally rare) species. To estimate total plant diversity, we performed long-term traditional surveys during eight consecutive years, including El Niño and La Niña events, we then analyzed eDNA from soil samples using high-throughput sequencing. We further used soil pollen analysis and soil seed bank germination assays to identify ‘hidden' species. Each approach offers different subsets of current biodiversity at different taxonomic, spatial and temporal resolution, with a total of 92 taxa identified along the transect. Traditional field surveys identified 77 plant species over eight consecutive years. Observed community composition greatly varies interannually, with only 22 species seen every year. eDNA analysis revealed 37 taxa, eight of which were ‘hidden' in our field surveys. Soil samples contain a viable seed bank of 21 taxa. Soil pollen (27 taxa) and eDNA analysis show affinities with vegetation at the landscape scale but a weak relationship to local plot diversity. Multimethod approaches (including eDNA) in deserts are valuable tools that add to a comprehensive assessment of biodiversity in such extreme environments, where using a single method or observations over a few years is insufficient. Our results can also explain the resilience of Atacama plant communities as ‘hidden' taxa may have been active in the recent past or could even emerge in the future as accelerated global environmental change continues unabated.</p><h1>Cora timucua (Hygrophoraceae), a new and potentially extinct, previously misidentified basidiolichen of Florida inland scrub documented from historical collections</h1>
<p><a href="https://doi.org/10.1639/0007-2745-123.4.657">doi:10.1639/0007-2745-123.4.657</a></p>

<p><a href=None>None</a></p>
<p>The known collections of the genus Cora in continental North America north of Mexico, all restricted to Florida, are shown to belong to a single species, representing a previously unrecognized taxon formally described herein as C. timucua. Based on data of the fungal ITS barcoding marker, obtained through Sanger and Illumina sequencing from two historical collections, the new species is phylogenetically most closely related to C. casanarensis from Colombia and C. itabaiana from Brazil, although it is morphologically most similar to the only distantly related C. hymenocarpa from Costa Rica. Based on data from the Consortium of North American Lichen Herbaria (CNALH) and from the Global Biodiversity Information Facility (GBIF), most of the collections of C. timucua originate from around the turn of the 19th century, while a few were made in the second half of the 20th century, all between 1968 and 1985. Almost all collections originate from Florida sand pine scrub, apparently the preferred habitat of this taxon. Neither modern collections nor extant localities are known. Based on these findings and the substantial degree of land use change in Florida in the past decades, we assessed the conservation status of C. timucua using the IUCN Red List criteria and found that it should be classified as critically endangered (CR), in line with the status of another Florida endemic, Cladonia perforata, which was the first federally red-listed lichen in the United States. The most likely location where C. timucua may still be extant is Ocala National Forest in the north-central portion of the Florida peninsula, although recent macrolichen surveys in that area did not encounter this species.</p><h1>Application of next generation sequencing for detection of protozoan pathogens in shellfish</h1>
<p><a href="https://doi.org/10.1016/j.fawpar.2020.e00096">doi:10.1016/j.fawpar.2020.e00096</a></p>

<p><a href=None>None</a></p>
<p>Food and waterborne protozoan pathogens can cause serious disease in people. Three common species Cryptosporidium parvum, Giardia enterica and Toxoplasma gondii can contaminate diverse shellfish species, including commercial oysters. Current methods of protozoan detection in shellfish are not standardized, and few are able to simultaneously identify multiple species. Here, we present a novel metabarcoding assay targeting the 18S rRNA gene followed by next generation sequencing (NGS) for simultaneous detection of Cryptosporidium spp., Giardia spp. and T. gondii spiked into oyster samples. We further developed a bioinformatic pipeline to process and analyze 18S rRNA data for protozoa classification. The ability of the NGS assay to detect protozoa was later compared with conventional PCR. Results demonstrated that background amplification of oyster and other eukaryotic DNA competed with that of protozoa for obtained sequence reads. Sequences of target protozoans were obtained across all spiking levels; however, low numbers of target sequences in negative controls imply that a threshold for true positives must be defined for assay interpretation. While this study focused on three target parasites, the ability of this approach to detect numerous known and potentially unknown protozoan pathogens make it a promising screening tool for monitoring protozoan contamination in food and water.</p><h1>Imprinting statistically sound conclusions for gut microbiota in comparative animal studies: A case study with diet and teleost fishes</h1>
<p><a href="https://doi.org/10.1016/j.cbd.2020.100738">doi:10.1016/j.cbd.2020.100738</a></p>

<p><a href=None>None</a></p>
<p>Despite the technical progress in high-throughput sequencing technologies, defining the sample size which is capable of yielding representative inferences in metabarcoding analysis still remains debatable. The present study addresses the influence of individual variability in assessing dietary effects on fish gut microbiota parameters and estimates the biological sample size that is sufficient to imprint a statistically secure outcome. European sea bass (Dicentrarchus labrax) and gilthead sea bream (Sparus aurata) were fed three alternative animal protein diets and a fishmeal control diet. Gut microbiota data from 12 individuals per diet, derived from Illumina sequencing of the V3–V4 region of the 16S rRNA gene, were randomized in all possible combinations of n-1 individuals. Results in this study showcased that increasing the sample size can limit the prevalence of individuals with high microbial load on the outcome and can ensure the statistical confidence required for an accurate validation of dietary-induced microbe shifts. Inter-individual variability was evident in the four dietary treatments where consequently misleading inferences arose from insufficient biological replication. These findings have critical implications for the design of future metabarcoding studies and highlight the urgency in selecting an adequate sample size able to safely elucidate the dietary effects on fish gut microbial communities.</p><h1>Environmental DNA metabarcoding reveals estuarine benthic community response to nutrient enrichment – Evidence from an in-situ experiment</h1>
<p><a href="https://doi.org/10.1016/j.envpol.2020.115472">doi:10.1016/j.envpol.2020.115472</a></p>

<p><a href=None>None</a></p>
<p>Nutrient loading is a major threat to estuaries and coastal environments worldwide, therefore, it is critical that we have good monitoring tools to detect early signs of degradation in these ecologically important and vulnerable ecosystems. Traditionally, bottom-dwelling macroinvertebrates have been used for ecological health assessment but recent advances in environmental genomics mean we can now characterize less visible forms of biodiversity, offering a more holistic view of the ecosystem and potentially providing early warning signals of disturbance. We carried out a manipulative nutrient enrichment experiment (0, 150 and 600 g N fertilizer m−2) in two estuaries in New Zealand to assess the effects of nutrient loading on benthic communities. After seven months of enrichment, environmental DNA (eDNA) metabarcoding was used to examine the response of eukaryotic (18S rRNA), diatom only (rbcL) and bacterial (16S rRNA) communities. Multivariate analyses demonstrated changes in eukaryotic, diatom and bacterial communities in response to nutrient enrichment at both sites, despite differing environmental conditions. These patterns aligned with changes in macrofaunal communities identified using traditional morphological techniques, confirming concordance between disturbance indicators detected by eDNA and current monitoring approaches. Clear shifts in eukaryotic and bacterial indicator taxa were seen in response to nutrient loading while changes in diatom only communities were more subtle. Community changes were discernible between 0 and 150 g N m−2 treatments, suggesting that estuary health assessment tools could be developed to detect early signs of degradation. Increasing variation in community structure associated with nutrient loading could also be used as an indicator of stress or approaching tipping points. This work represents a first step towards the development of molecular-based estuary monitoring tools, which could provide a more holistic and standardized approach to ecosystem health assessment with faster turn-around times and lower costs.</p><h1>Bioavailable metal(loid)s and physicochemical features co-mediating microbial communities at combined metal(loid) pollution sites</h1>
<p><a href="https://doi.org/10.1016/j.chemosphere.2020.127619">doi:10.1016/j.chemosphere.2020.127619</a></p>

<p><a href=None>None</a></p>
<p>Heavy metal contamination poses considerable threats to various ecosystems, yet little is known about the assembly and adaptation of microbial communities at sites with combined heavy metal(loid) pollution. Here, we examined metal(loid) pollutants and bacterial communities in three zones (Zones Ⅰ, Ⅱ, and Ⅲ) of an abandoned sewage reservoir with different usage years. The contamination level of multiple metal(loid)s was higher in Zone Ⅰ than in the other zones, and arsenic (As), zinc (Zn), selenium (Se), copper (Cu), tin (Sn), molybdenum (Mo), antimony (Sb), cadmium (Cd), lead (Pb), thallium (Tl), and nickel (Ni) were the major contaminants (pollution load index > 1). Bioavailable forms of titanium (Ti), chromium (Cr), Sn, and cobalt (Co) played essential roles in shaping the microbial structure, and physicochemical properties, especially organic matter (OM) and pH, also mediated the microbial diversity and composition in the metal(loid) contaminated zones. Metal–microbe interactions and heatmap analysis revealed that the bioavailability of metal(loid)s promoted the niche partitioning of microbial species. Metal-resistant species were abundant in Zone Ⅰ that had the highest metal-contaminated level, whereas metal-sensitive species prevailed in Zone Ⅲ that had the lowest pollution level. The bioavailable metal(loid)s rather than physicochemical and spatial variables explained a larger portion of the variance in the microbial community, and the homogeneous selection was the dominant ecological process driving the assembly of the microbial community. Overall, our study highlighted the importance of metal(loid) bioavailability in shaping microbial structure, future bioremediation, and environmental management of metal(loid) contaminated sites.</p><h1>Diversity hotspot and unique community structure of foraminifera in the world’s deepest marine blue hole – Sansha Yongle Blue Hole</h1>
<p><a href="https://doi.org/10.1038/s41598-020-67221-0">doi:10.1038/s41598-020-67221-0</a></p>

<p><a href=None>None</a></p>
<p>Marine blue holes are precious geological heritages with high scientific research values. Their physical and chemical characteristics are unique because of the steep-walled structure and isolated water column which create isolated ecosystems in geographically restricted areas. The Sansha Yongle Blue Hole (SYBH) is the world’s deepest marine blue hole. Here, we generated the first DNA metabarcoding dataset from SYBH sediment focusing on foraminifera, a group of protists that have colonized various marine environments. We collected sediment samples from SYBH along a depth gradient to characterize the foraminiferal diversity and compared them with the foraminiferal diversity of the costal Jiaozhou Bay (JZB) and the abyssal Northwest Pacific Ocean (NWP). We amplified the SSU rDNA of foraminifera and sequenced them with high-throughput sequencing. The results showed that the foraminiferal assemblages in SYBH were vertically structured in response to the abiotic gradients and diversity was higher than in JZB and NWP. This study illustrates the capacity of foraminifera to colonize hostile environments and shows that blue holes are natural laboratories to explore physiological innovation associated with anoxia.</p><h1>Environmental DNA reveals seasonal shifts and potential interactions in a marine community</h1>
<p><a href="https://doi.org/10.1038/s41467-019-14105-1">doi:10.1038/s41467-019-14105-1</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis allows the simultaneous examination of organisms across multiple trophic levels and domains of life, providing critical information about the complex biotic interactions related to ecosystem change. Here we used multilocus amplicon sequencing of eDNA to survey biodiversity from an eighteen-month (2015–2016) time-series of seawater samples from Monterey Bay, California. The resulting dataset encompasses 663 taxonomic groups (at Family or higher taxonomic rank) ranging from microorganisms to mammals. We inferred changes in the composition of communities, revealing putative interactions among taxa and identifying correlations between these communities and environmental properties over time. Community network analysis provided evidence of expected predator-prey relationships, trophic linkages, and seasonal shifts across all domains of life. We conclude that eDNA-based analyses can provide detailed information about marine ecosystem dynamics and identify sensitive biological indicators that can suggest ecosystem changes and inform conservation strategies.</p><h1>Metabarcoding From Microbes to Mammals: Comprehensive Bioassessment on a Global Scale</h1>
<p><a href="https://doi.org/10.3389/fevo.2020.581835">doi:10.3389/fevo.2020.581835</a></p>

<p><a href=None>None</a></p>
<p>Global biodiversity loss is unprecedented, and threats to existing biodiversity are growing. Given pervasive global change, a major challenge facing resource managers is a lack of scalable tools to rapidly and consistently measure Earth's biodiversity. Environmental genomic tools provide some hope in the face of this crisis, and DNA metabarcoding, in particular, is a powerful approach for biodiversity assessment at large spatial scales. However, metabarcoding studies are variable in their taxonomic, temporal, or spatial scope, investigating individual species, specific taxonomic groups, or targeted communities at local or regional scales. With the advent of modern, ultra-high throughput sequencing platforms, conducting deep sequencing metabarcoding surveys with multiple DNA markers will enhance the breadth of biodiversity coverage, enabling comprehensive, rapid bioassessment of all the organisms in a sample. Here, we report on a systematic literature review of 1,563 articles published about DNA metabarcoding and summarize how this approach is rapidly revolutionizing global bioassessment efforts. Specifically, we quantify the stakeholders using DNA metabarcoding, the dominant applications of this technology, and the taxonomic groups assessed in these studies. We show that while DNA metabarcoding has reached global coverage, few studies deliver on its promise of near-comprehensive biodiversity assessment. We then outline how DNA metabarcoding can help us move toward real-time, global bioassessment, illustrating how different stakeholders could benefit from DNA metabarcoding. Next, we address barriers to widespread adoption of DNA metabarcoding, highlighting the need for standardized sampling protocols, experts and computational resources to handle the deluge of genomic data, and standardized, open-source bioinformatic pipelines. Finally, we explore how technological and scientific advances will realize the promise of total biodiversity assessment in a sample—from microbes to mammals—and unlock the rich information genomics exposes, opening new possibilities for merging whole-system DNA metabarcoding with (1) abundance and biomass quantification, (2) advanced modeling, such as species occupancy models, to improve species detection, (3) population genetics, (4) phylogenetics, and (5) food web and functional gene analysis. While many challenges need to be addressed to facilitate widespread adoption of environmental genomic approaches, concurrent scientific and technological advances will usher in methods to supplement existing bioassessment tools reliant on morphological and abiotic data. This expanded toolbox will help ensure that the best tool is used for the job and enable exciting integrative techniques that capitalize on multiple tools. Collectively, these new approaches will aid in addressing the global biodiversity crisis we now face.</p><h1>The Response of Estuarine Ammonia-Oxidizing Communities to Constant and Fluctuating Salinity Regimes</h1>
<p><a href="https://doi.org/10.3389/fmicb.2020.574815">doi:10.3389/fmicb.2020.574815</a></p>

<p><a href=None>None</a></p>
<p>Aerobic nitrification is a fundamental nitrogen biogeochemical process that links the oxidation of ammonia to the removal of fixed nitrogen in eutrophicated water bodies. However, in estuarine environments there is an enormous variability of water physicochemical parameters that can affect the ammonia oxidation biological process. For instance, it is known that salinity can affect nitrification performance, yet there is still a lack of information on the ammonia-oxidizing communities behavior facing daily salinity fluctuations. In this work, laboratory experiments using upstream and downstream estuarine sediments were performed to address this missing gap by comparing the effect of daily salinity fluctuations with constant salinity on the activity and diversity of ammonia-oxidizing microorganisms (AOM). Activity and composition of AOM were assessed, respectively by using nitrogen stable isotope technique and 16S rRNA gene metabarcoding analysis. Nitrification activity was negatively affected by daily salinity fluctuations in upstream sediments while no effect was observed in downstream sediments. Constant salinity regime showed clearly higher rates of nitrification in upstream sediments while a similar nitrification performance between the two salinity regimes was registered in the downstream sediments. Results also indicated that daily salinity fluctuation regime had a negative effect on both ammonia-oxidizing bacteria (AOB) and ammonia-oxidizing archaea (AOA) community’s diversity. Phylogenetically, the estuarine downstream AOM were dominated by AOA (0.92–2.09%) followed by NOB (0.99–2%), and then AOB (0.2–0.32%); whereas NOB dominated estuarine upstream sediment samples (1.4–9.5%), followed by AOA (0.27–0.51%) and AOB (0.01–0.23%). Analysis of variance identified the spatial difference between samples (downstream and upstream) as the main drivers of AOA and AOB diversity. Our study indicates that benthic AOM inhabiting different estuarine sites presented distinct plasticity toward the salinity regimes tested. These findings help to improve our understanding in the dynamics of the nitrogen cycle of estuarine systems by showing the resilience and consequently the impact of different salinity regimes on the diversity and activity of ammonia oxidizer communities.</p><h1>DNA metabarcoding captures subtle differences in forest beetle communities following disturbance</h1>
<p><a href="https://doi.org/10.1111/rec.13236">doi:10.1111/rec.13236</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is an emerging approach for monitoring biodiversity, but uncertainties remain about its capacity to detect subtle differences in invertebrate community composition comparable to those achievable based on conventional morphological identification. In this study, DNA metabarcoding and morphology-based approaches were compared as tools for investigating whether logging history impacted beetle communities in Tasmanian wet eucalypt forests. We compared 12 unlogged mature forest sites with 12 neighboring regeneration sites that had been logged approximately 55 years previously. The number of species identified based on morphology (173) was close to the number of zero-radius operational taxonomic units (ZOTUs) identified by DNA metabarcoding of cytochrome c oxidase subunit I (COI, 176) and 16S ribosomal RNA (16S, 156) markers. Subtle but significant differences in beetle species composition between regeneration and unlogged mature forests were captured by both morphology-based and COI DNA metabarcoding approaches, but not by 16S DNA metabarcoding. Our results support the suitability of mitochondrial COI for studying invertebrate biodiversity. A slight loss of signal compared to the morphology-based approach may be resolved by developing more comprehensive DNA reference databases. While confirming forest recovery of 48–58 years did not fully restore mature forest beetle communities, we suggest that DNA metabarcoding can be used for monitoring biodiversity and probing subtle differences in community composition.</p><h1>MiFish metabarcoding: a high-throughput approach for simultaneous detection of multiple fish species from environmental DNA and other samples</h1>
<p><a href="https://doi.org/10.1007/s12562-020-01461-x">doi:10.1007/s12562-020-01461-x</a></p>

<p><a href=None>None</a></p>
<p>We reviewed the current methodology and practices of the DNA metabarcoding approach using a universal PCR primer pair MiFish, which co-amplifies a short fragment of fish DNA (approx. 170 bp from the mitochondrial 12S rRNA gene) across a wide variety of taxa. This method has mostly been applied to biodiversity monitoring using environmental DNA (eDNA) shed from fish and, coupled with next-generation sequencing technologies, has enabled massively parallel sequencing of several hundred eDNA samples simultaneously. Since the publication of its technical outline in 2015, this method has been widely used in various aquatic environments in and around the six continents, and MiFish primers have demonstrably outperformed other competing primers. Here, we outline the technical progress in this method over the last 5 years and highlight some case studies on marine, freshwater, and estuarine fish communities. Additionally, we discuss various applications of MiFish metabarcoding to non-fish organisms, single-species detection systems, quantitative biodiversity monitoring, and bulk DNA samples other than eDNA. By recognizing the MiFish eDNA metabarcoding strengths and limitations, we argue that this method is useful for ecosystem conservation strategies and the sustainable use of fishery resources in “ecosystem-based fishery management” through continuous biodiversity monitoring at multiple sites.</p><h1>Characteristics of long-term changes in microbial communities from contaminated sediments along the west coast of South Korea: Ecological assessment with eDNA and physicochemical analyses</h1>
<p><a href="https://doi.org/10.1016/j.marpolbul.2020.111592">doi:10.1016/j.marpolbul.2020.111592</a></p>

<p><a href=None>None</a></p>
<p>The environmental DNA (eDNA) metabarcoding was applied to assess benthic ecological health in the west coast of South Korea by investigating a long-term microbial community change (2015–17). The ecological interaction among microorganisms, from phylum to family level, and their associations to environmental variables across the five regions were highlighted. As part of the study, the available chemistry and toxicological data in the regions during the monitoring periods were incorporated into an integrated sediment triad assessment. The bacterial communities were dominated by Proteobacteria (34.2%), Bacteroidetes (13.8%), and Firmicutes (10.8%). Proteobacteria and Bacteroidetes dominated consistently across regions and years, while Firmicutes and Cyanobacteria significantly varied by region and years (p < 0.05). The abundance of this phylum declined over time with the increasing abundance of Cyanobacteria, indicating their independent interactions to certain environmental changes. Planctomycetes and Gemmatimonadetes linked to some contaminants (ΣPAHs and Cu), implying indicator taxa. Overall, eDNA-based microbial community analysis combined with exposures of contaminants and responses of microorganisms is a promising strategy for the assessment of benthic ecological health in contaminated sediments from coastal waters.</p><h1>Stem-inhabiting fungal communities differ between intact and snapped trees after hurricane Maria in a Puerto Rican tropical dry forest</h1>
<p><a href="https://doi.org/10.1016/j.foreco.2020.118350">doi:10.1016/j.foreco.2020.118350</a></p>

<p><a href=None>None</a></p>
<p>Hurricanes impact forests by damaging trees and altering multiple ecosystem functions. As such, predicting which individuals are likely to be most affected has crucial economic importance as well as conservation value. Tree stem-inhabiting fungal communities, notably rot-causing agents, have been mentioned as a potential factor of tree predisposition to hurricane damage, but this assumption remains poorly explored. To examine this relationship, we sampled the stem wood of intact and damaged trees shortly after Hurricane Maria in a Puerto Rican dry tropical forest in 2017. We categorized samples depending on two types: trees with intact stems and trees in which stems were snapped. We extracted fungal environmental DNA of wood from 40 samples consisting of four different tree species. Fungal community taxonomic and functional richness and composition was assessed using high-throughput DNA metabarcoding. We found that snapped trees harbored significantly higher fungal operational taxonomic unit (OTU) richness than the intact trees and that the composition of the stem-inhabiting fungal communities diverged consistently between intact and snapped trees. On average, snapped trees’ fungal communities were relatively enriched in “other saprotrophs” guild category and depleted in endophytes. Conversely, intact trees had high relative abundances of Clonostachys, a mycoparasitic endophyte, suggesting that endophytic fungi might act as biocontrols in tree stems. Overall, our results support the hypothesis that stem-inhabiting fungal communities could represent a predisposition factor of tree damage caused by hurricanes in tropical dry forests.</p><h1>Development of a novel metric for evaluating diatom assemblages in rivers using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2020.106725">doi:10.1016/j.ecolind.2020.106725</a></p>

<p><a href=None>None</a></p>
<p>Fundamental differences in the nature of diatom assemblage composition data generated using light microscopy and molecular barcoding create problems when applying current paradigms and metrics developed for ecological assessment. We therefore describe the development of a new metric designed specifically for diatom rbcL barcode data gathered using high throughput sequencing (HTS). Although the structure of datasets collected using HTS is similar to that collected using light microscopy (LM), differences in the proportions of key species between the two methods mean that the use of metrics designed for LM on HTS data gives biased results. We therefore recalibrated the Trophic Diatom Index in order to produce a version that is sensitive to nutrient pressures in rivers but that can be used with HTS data. Correlation between the LM and HTS metrics is good (r = 0.86 on a cross-validated model); however, 30% of sites will change class when the current Water Framework Directive classification approach is applied. Although less than 15% of diatom taxa recorded from UK and Ireland are included in the rbcL barcode reference database, gaps in this database are not a major source of variation between the HTS and LM models. We argue that use of metrics calibrated using HTS data is a more realistic option than applying correction factors to enable HTS data to be used with existing indices. We also stress the importance of starting the process of integrating HTS into ecological assessments with relatively conservative approaches. This enables the data collected by HTS to be related to those generated by established approaches, both now and during long-term monitoring, making it possible for scientists, regulators and stakeholders to have an informed conversation about the benefits and challenges of HTS. Overall, the study demonstrates that it is possible to translate the legal requirements of an ecological assessment framework from LM to HTS, though differences in these two approaches mean that there is unlikely to be perfect agreement between their outputs.</p><h1>Comparing sediment DNA extraction methods for assessing organic enrichment associated with marine aquaculture</h1>
<p><a href="https://doi.org/10.7717/peerj.10231">doi:10.7717/peerj.10231</a></p>

<p><a href=None>None</a></p>
<p>Marine sediments contain a high diversity of micro- and macro-organisms which are important in the functioning of biogeochemical cycles. Traditionally, anthropogenic perturbation has been investigated by identifying macro-organism responses along gradients. Environmental DNA (eDNA) analyses have recently been advocated as a rapid and cost-effective approach to measuring ecological impacts and efforts are underway to incorporate eDNA tools into monitoring. Before these methods can replace or complement existing methods, robustness and repeatability of each analytical step has to be demonstrated. One area that requires further investigation is the selection of sediment DNA extraction method. Environmental DNA sediment samples were obtained along a disturbance gradient adjacent to a Chinook (Oncorhynchus tshawytscha) salmon farm in Otanerau Bay, New Zealand. DNA was extracted using four extraction kits (Qiagen DNeasy PowerSoil, Qiagen DNeasy PowerSoil Pro, Qiagen RNeasy PowerSoil Total RNA/DNA extraction/elution and Favorgen FavorPrep Soil DNA Isolation Midi Kit) and three sediment volumes (0.25, 2, and 5 g). Prokaryotic and eukaryotic communities were amplified using primers targeting the 16S and 18S ribosomal RNA genes, respectively, and were sequenced on an Illumina MiSeq. Diversity and community composition estimates were obtained from each extraction kit, as well as their relative performance in established metabarcoding biotic indices. Differences were observed in the quality and quantity of the extracted DNA amongst kits with the two Qiagen DNeasy PowerSoil kits performing best. Significant differences were observed in both prokaryotes and eukaryotes (p < 0.001) richness among kits. A small proportion of amplicon sequence variants (ASVs) were shared amongst the kits (~3%) although these shared ASVs accounted for the majority of sequence reads (prokaryotes: 59.9%, eukaryotes: 67.2%). Differences were observed in the richness and relative abundance of taxonomic classes revealed with each kit. Multivariate analysis showed that there was a significant interaction between "distance" from the farm and "kit" in explaining the composition of the communities, with the distance from the farm being a stronger determinant of community composition. Comparison of the kits against the bacterial and eukaryotic metabarcoding biotic index suggested that all kits showed similar patterns along the environmental gradient. Overall, we advocate for the use of Qiagen DNeasy PowerSoil kits for use when characterizing prokaryotic and eukaryotic eDNA from marine farm sediments. We base this conclusion on the higher DNA quality values and richness achieved with these kits compared to the other kits/amounts investigated in this study. The additional advantage of the PowerSoil Kits is that DNA extractions can be performed using an extractor robot, offering additional standardization and reproducibility of results.</p><h1>Ontogeny, species identity, and environment dominate microbiome dynamics in wild populations of kissing bugs (Triatominae)</h1>
<p><a href="https://doi.org/10.1186/s40168-020-00921-x">doi:10.1186/s40168-020-00921-x</a></p>

<p><a href=None>None</a></p>
<p>Background: Kissing bugs (Triatominae) are blood-feeding insects best known as the vectors of Trypanosoma cruzi, the causative agent of Chagas' disease. Considering the high epidemiological relevance of these vectors, their biology and bacterial symbiosis remains surprisingly understudied. While previous investigations revealed generally low individual complexity but high among-individual variability of the triatomine microbiomes, any consistent microbiome determinants have not yet been identified across multiple Triatominae species. Methods: To obtain a more comprehensive view of triatomine microbiomes, we investigated the host-microbiome relationship of five Triatoma species sampled from white-throated woodrat (Neotoma albigula) nests in multiple locations across the USA. We applied optimised 16S rRNA gene metabarcoding with a novel 18S rRNA gene blocking primer to a set of 170 T. cruzi-negative individuals across all six instars. Results: Triatomine gut microbiome composition is strongly influenced by three principal factors: Ontogeny, species identity, and the environment. The microbiomes are characterised by significant loss in bacterial diversity throughout ontogenetic development. First instars possess the highest bacterial diversity while adult microbiomes are routinely dominated by a single taxon. Primarily, the bacterial genus Dietzia dominates late-stage nymphs and adults of T. rubida, T. protracta, and T. lecticularia but is not present in the phylogenetically more distant T. gerstaeckeri and T. sanguisuga. Species-specific microbiome composition, particularly pronounced in early instars, is further modulated by locality-specific effects. In addition, pathogenic bacteria of the genus Bartonella, acquired from the vertebrate hosts, are an abundant component of Triatoma microbiomes. Conclusion: Our study is the first to demonstrate deterministic patterns in microbiome composition among all life stages and multiple Triatoma species. We hypothesise that triatomine microbiome assemblages are produced by species- A nd life stage-dependent uptake of environmental bacteria and multiple indirect transmission strategies that promote bacterial transfer between individuals. Altogether, our study highlights the complexity of Triatominae symbiosis with bacteria and warrant further investigation to understand microbiome function in these important vectors. [MediaObject not available: See fulltext.]</p><h1>Novel universal primers for metabarcoding environmental DNA surveys of marine mammals and other marine vertebrates</h1>
<p><a href="https://doi.org/10.1002/edn3.72">doi:10.1002/edn3.72</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding studies using environmental DNA (eDNA) and high-throughput sequencing (HTS) are rapidly becoming an important tool for assessing and monitoring marine biodiversity, detecting invasive species, and supporting basic ecological research. Several barcode loci targeting teleost fish and elasmobranchs have previously been developed, but to date primer sets focusing on other marine megafauna, such as marine mammals, have received less attention. Similarly, there have been few attempts to identify potentially “universal” barcode loci which may be informative across multiple marine vertebrate orders. Here we describe the design and validation of two new sets of primers targeting hypervariable regions of the vertebrate mitochondrial 12S and 16S rRNA genes, which have conserved priming sites across virtually all cetaceans, pinnipeds, elasmobranchs, boney fish, sea turtles, and birds, and amplify fragments with consistently high levels of taxonomically diagnostic sequence variation. “In silico” validation using the OBITOOLS software showed our new barcode loci outperformed most existing vertebrate barcode loci for taxon detection and resolution. We also evaluated sequence diversity and taxonomic resolution of the new barcode loci in 680 complete marine mammal mitochondrial genomes demonstrating that they are effective at resolving amplicons for most taxa to the species level. Finally, we evaluated the performance of the primer sets with eDNA samples from aquarium communities with known species composition. These new primers will potentially allow surveys of complete marine vertebrate communities in single HTS metabarcoding assessments, simplifying workflows, reducing costs, and increasing accessibility to a wider range of investigators.</p><h1>Macroinvertebrate community diversity and habitat quality relationships along a large river from targeted eDNA metabarcode assays</h1>
<p><a href="https://doi.org/10.1002/edn3.90">doi:10.1002/edn3.90</a></p>

<p><a href=None>None</a></p>
<p>Species compositions and diversity levels of aquatic macroinvertebrate communities provide important indicators of ecosystem health. However, such community analyses typically are limited by time and effort of sampling, sorting, and identifications, as well as morphological character uncertainty for some taxa, especially at early life stages. Our objective was to evaluate a suite of targeted metabarcode high-throughput sequencing assays to characterize the macroinvertebrate communities (specifically targeting Annelida, Bryozoa, Crustacea, Insecta, and Mollusca) from environmental (e)DNA water samples along 160 km of the Maumee River, OH (a major Lake Erie, Laurentian Great Lakes tributary). Multiple alpha (richness, Shannon Index, and Simpson's Index) and beta (Bray–Curtis and Sørensen dissimilarities) diversity metrics from the metabarcode assays were compared with an Invertebrate Community Index (ICI) metric calculated from traditional morphological sampling surveys conducted by the Ohio Environmental Protection Agency. The 15 Maumee River sites varied in their ICI scores (ranging from 4 to 46), with seven sites rated as “poor” or “very poor” and eight scoring “fair” or “good.” Metabarcoding assays yielded greater gamma richness, delineating 181 Operational Taxonomic Units versus 172 taxa from morphology (the latter often limited to family or genus-level identifications). Both datasets supported similar river-wide trends, with comparable gamma, alpha, and beta diversity patterns and community compositions across habitat types and habitat quality scores. Metabarcode assays revealed similar detection of important bioindicator Insecta, but missed most Trichoptera (caddisflies). eDNA identified eight aquatic invasive species on the GLANSIS (Great Lakes Aquatic Nonindigenous Species Information System) list, including three missed by the morphological surveys (Branchiura sowerbyi, Potamothrix bedoti, and Skistodiaptomus pallidus; with Lophopodella carteri, Faxonius rusticus, Corbicula fluminea, Dreissena polymorpha, and Valvata piscinalis in both datasets). Findings illustrate the utility of eDNA sampling and targeted metabarcode assays to enhance and complement environmental assessments of aquatic ecosystems.</p><h1>The impact of intragenomic rRNA variation on metabarcoding-derived diversity estimates: A case study from marine nematodes</h1>
<p><a href="https://doi.org/10.1002/edn3.77">doi:10.1002/edn3.77</a></p>

<p><a href=None>None</a></p>
<p>Intragenomic rRNA variation is a critical concern for eukaryotic metabarcoding studies, due to its potentially confounding effects on species delimitation and biodiversity estimates derived from -Omics data. In the present study, we assessed patterns associated with 18S rRNA metabarcoding loci in marine nematodes, including characterization of intragenomic rRNA gene variants (number of variants and abundance profiles) and aspects of datasets that can obscure biological signals (e.g., amplification of nontarget DNA, ambiguous taxonomy assignments). We estimated amplicon sequence variants (ASVs) using DADA2 from an 18S rRNA metabarcoding dataset (Illumina MiSeq) generated from individual marine nematodes. Illumina data were analyzed in conjunction with nematode morphological identifications and nearly full-length 18S reference sequences (~1,600 bp Sanger barcodes) generated for a subset of the same specimens. Our results indicated that levels of intragenomic rRNA variation appeared to vary widely across nematode taxa (irrespective of phylogenetic clades or ecological feeding groups) and that coamplification of nontarget DNA was common (relic DNA, gut contents, etc.). The DADA2 pipeline appeared to produce a biologically accurate profile of intragenomic rRNA variants in nematodes that was consistent with “Head-Tail” patterns (of dominant vs. minor rRNA gene variants) identified in previous studies. Although intragenomic rRNA variation appears to be ubiquitous in marine nematodes, nematode identifications were highly congruent across our three methods for species delimitation (traditional morphological taxonomy, Sanger DNA barcoding, and high-throughput metabarcoding). In spite of pervasive intragenomic variation and high copy number of rRNA genes, the most abundant ASVs in metabarcoding datasets are likely to represent true species barcodes and thus confer an accurate view of extant biodiversity. However, our findings also emphasize the importance of applying bioinformatic filtering techniques and developing well-curated reference databases in order to better link rRNA molecules with specimen-level data and alleviate the confounding effects of intragenomic gene variants in studies of microbial eukaryotes.</p><h1>An amplicon sequencing protocol for attacker identification from DNA traces left on artificial prey</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13459">doi:10.1111/2041-210X.13459</a></p>

<p><a href=None>None</a></p>
<p>Clay model studies are a popular tool to identify predator–prey interactions that are challenging to observe directly in the field. But despite its wide use, the method's applicability is limited by its low taxonomic resolution. Attack marks on clay models are usually identified visually, which only allows classification into higher taxonomic levels of predators. Thus, the method is often biased, lacks proof and, above all, standardization. Here, we tested whether precise identification of attackers can be provided by amplification and sequencing of mitochondrial DNA left in bite marks on clay models. We validated our approach in a controlled laboratory study as well as in a field experiment using clay models of a common European amphibian, the European fire salamander Salamandra salamandra. DNA-based taxonomic assignments were additionally compared to visual assessments of bite marks. We show that trace DNA of attackers can be routinely isolated and sequenced from bite marks, providing accurate species-level classification. In contrast, visual identification alone yielded a high number of unassigned predator taxa. We also highlight the sensitivity of the method and show likely sources of contamination as well as probable cases of secondary and indirect predation. Our standardized approach for species-level attacker identification opens up new possibilities far beyond the standard use of clay models to date, including food web studies at unprecedented detail, invasive species monitoring as well as biodiversity inventories.</p><h1>Is a Central Sediment Sample Sufficient? Exploring Spatial and Temporal Microbial Diversity in a Small Lake</h1>
<p><a href="https://doi.org/10.3390/toxins12090580">doi:10.3390/toxins12090580</a></p>

<p><a href=None>None</a></p>
<p>(1) Background: Paleolimnological studies use sediment cores to explore long-term changes in lake ecology, including occurrences of harmful cyanobacterial blooms. Most studies are based on single cores, assuming this is representative of the whole lake, but data on small-scale spatial variability of microbial communities in lake sediment are scarce. (2) Methods: Surface sediments (top 0.5 cm) from 12 sites (n = 36) and two sediment cores were collected in Lake Rotorua (New Zealand). Bacterial community (16S rRNA metabarcoding), Microcystis specific 16S rRNA, microcystin synthetase gene E (mcyE) and microcystins (MCs) were assessed. Radionuclide measurements (210Pb, 137Cs) were used to date sediments. (3) Results: Bacterial community, based on relative abundances, differed significantly between surface sediment sites (p < 0.001) but the majority of bacterial amplicon sequence variants (88.8%) were shared. Despite intense MC producing Microcystis blooms in the past, no Microcystis specific 16S rRNA, mcyE and MCs were found in surface sediments but occurred deeper in sediment cores (approximately 19500s). 210Pb measurements showed a disturbed profile, similar to patterns previously observed, as a result of earthquakes. (4) Conclusions: A single sediment core can capture dominant microbial communities. Toxin producing Microcystis blooms are a recent phenomenon in Lake Rotorua. We posit that the absence of Microcystis from the surface sediments is a consequence of the Kaikoura earthquake two years prior to our sampling.</p><h1>Revealing cryptic changes of cyanobacterial community structure in two eutrophic lakes using EDNA sequencing</h1>
<p><a href="https://doi.org/10.3390/ijerph17176356">doi:10.3390/ijerph17176356</a></p>

<p><a href=None>None</a></p>
<p>Harmful cyanobacterial blooms pose a risk to human health worldwide. To enhance understanding on the bloom-forming mechanism, the spatiotemporal changes in cyanobacterial diversity and composition in two eutrophic lakes (Erhai Lake and Lushui Reservoir) of China were investigated from 2010 to 2011 by high-throughput sequencing of environmental DNA. For each sample, 118 to 260 cpcBA-IGS operational taxonomic units (OTUs) were obtained. Fifty-two abundant OTUs were identified, which made up 95.2% of the total sequences and were clustered into nine cyanobacterial groups. Although the cyanobacterial communities of both lakes were mainly dominated by Microcystis, Erhai Lake had a higher cyanobacterial diversity. The abundance of mixed Nostocales species was lower than that of Microcystis, whereas Phormidium and Synechococcus were opportunistically dominant. The correlation between the occurrence frequency and relative abundance of OTUs was poorly fitted by the Sloan neutral model. Deterministic processes such as phosphorus availability were shown to have significant effects on the cyanobacterial community structure in Erhai Lake. In summary, the Microcystis-dominated cyanobacterial community was mainly affected by the deterministic process. Opportunistically dominant species have the potential to replace Microcystis and form blooms in eutrophic lakes, indicating the necessity to monitor these species for drinking water safety.</p><h1>Metabarcoding mites: Three years of elevated CO<inf>2</inf> has no effect on oribatid assemblages in a Eucalyptus woodland</h1>
<p><a href="https://doi.org/10.1016/j.pedobi.2020.150667">doi:10.1016/j.pedobi.2020.150667</a></p>

<p><a href=None>None</a></p>
<p>Elevated atmospheric carbon dioxide concentrations (eCO2) are known to significantly impact aboveground vegetation, with cascading effects on belowground systems. However, the effects of altered biogeochemical and hydrological cycles on belowground faunal assemblages remain unclear. Oribatid mites are abundant and hyperdiverse litter decomposers, with the potential to act as bioindicators of ecosystem disturbance. Here, we assessed oribatid mite assemblages under ambient and eCO2 (ambient +150 ppm) at the Eucalyptus Free-Air CO2 Enrichment (EucFACE) facility using high-throughput sequencing of the mitochondrial cytochrome oxidase 1 (COI) gene. Our aims were to i) quantify the effect of eCO2 on mite abundance and diversity, and ii) evaluate metabarcoding as a reliable approach for the rapid assessment of oribatid mite assemblages. Three years of eCO2 exposure resulted in lower soil pH but had no significant impact on mite abundances or oribatid mite assemblage structure. Instead, mite densities were found to correlate negatively with soil NH4+, while oribatid species richness using both morphological and metabarcoding datasets were positively correlated with total soil nitrogen content. Additionally, metabarcoding showed that oribatid mite assemblage structure was correlated with total soil carbon and nitrogen content, and soil pH. However, dissimilarity matrices based on the two approaches were not correlated, indicating that metabarcoding can be used as a proxy for shifts in assemblage structure but cannot be directly compared with morphological datasets. This study demonstrates how metabarcoding can rapidly assess belowground assemblages, providing a useful monitoring tool under current and future climate scenarios.</p><h1>Detection of the rare Australian endemic blind cave eel (Ophisternon candidum) with environmental DNA: implications for threatened species management in subterranean environments</h1>
<p><a href="https://doi.org/10.1007/s10750-020-04304-z">doi:10.1007/s10750-020-04304-z</a></p>

<p><a href=None>None</a></p>
<p>The blind cave eel, Ophisternon candidum (Mees in J R Soc West Aust 45: 24–32,1962), is a rare groundwater inhabitant found in geographically isolated populations of north-west Australia. The species is listed as vulnerable under Commonwealth legislation and is a priority consideration when environmental disturbance by resource companies is proposed. Detection of this species for Environmental Impact Assessment (EIA) and monitoring is difficult because individuals are naturally rare or traditional sampling techniques are ineffective. To properly manage the species, information on population distribution and connectivity is critical. We sought to examine whether environmental DNA (eDNA) of O. candidum could be detected and whether positive detection was correlated with previous locations where the species had been physically caught. We developed new eDNA species-specific PCR assays to screen groundwater sampling points and we detected O. candidum DNA in three boreholes where the species has previously been collected and five additional groundwater sampling points. Our results demonstrated that the newly designed assays were effective for detecting this rare and vulnerable subterranean species. This work sets a benchmark for the application of eDNA species-specific PCR assays for EIA and monitoring, and has potential for these assays to be expanded more broadly to high-throughput eDNA metabarcoding for subterranean groundwater communities in the future.</p><h1>Soil bacterial community differences along a coastal restoration chronosequence</h1>
<p><a href="https://doi.org/10.1007/s11258-019-00979-0">doi:10.1007/s11258-019-00979-0</a></p>

<p><a href=None>None</a></p>
<p>Restoration interventions such as revegetation are globally-important to combat biodiversity declines and land degradation. However, restoration projects are generally poorly monitored because current approaches to monitoring are limited in their ability to assess important components of biodiversity, such as belowground microbial diversity. Since soil bacterial communities mediate many belowground ecosystems processes and represent substantial biodiversity in their own right, bacteria are important components to monitor during ecosystem restoration. High-throughput amplicon sequencing (DNA metabarcoding) has been put forward as a potential cost-effective, scalable and easy-to-standardise partial solution to restoration’s monitoring problem. However, its application to restoration projects has to date been limited. Here, we used DNA metabarcoding of bacterial 16S rRNA gene from soil DNA to explore community differences across a 16-year restoration chronosequence. The bacterial composition in the oldest revegetation sites was comparable to the remnant sites. Proteobacteria and Acidobacteria were significantly higher in relative sequence abundance, while Actinobacteria was significantly lower, with time since revegetation. Classes Alphaproteobacteria and Acidobacteria were indicative of remnant and the oldest revegetation sites, while Deltaproteobacteria and Rubrobacteria were characteristic of younger revegetation sites. Changes in the soil physical and chemical characteristics associated with revegetation appear to shape bacterial community structure and composition. These findings provide evidence that revegetation can have positive effects on belowground microbial communities, and help demonstrate that the soil bacterial community can be restored towards its native state by revegetation, which may be useful in restoration monitoring.</p><h1>Calibrating Environmental DNA Metabarcoding to Conventional Surveys for Measuring Fish Species Richness</h1>
<p><a href="https://doi.org/10.3389/fevo.2020.00276">doi:10.3389/fevo.2020.00276</a></p>

<p><a href=None>None</a></p>
<p>The ability to properly identify species present in a landscape is foundational to ecology and essential for natural resource management and conservation. However, many species are often unaccounted for due to ineffective direct capture and visual surveys, especially in aquatic environments. Environmental DNA metabarcoding is an approach that overcomes low detection probabilities and should consequently enhance estimates of biodiversity and its proxy, species richness. Here, we synthesize 37 studies in natural aquatic systems to compare species richness estimates for bony fish between eDNA metabarcoding and conventional methods, such as nets, visual census, and electrofishing. In freshwater systems with fewer than 100 species, we found eDNA metabarcoding detected more species than conventional methods. Using multiple genetic markers further increased species richness estimates with eDNA metabarcoding. For more diverse freshwater systems and across marine systems, eDNA metabarcoding reported similar values of species richness to conventional methods; however, more studies are needed in these environments to better evaluate relative performance. In systems with greater biodiversity, eDNA metabarcoding will require more populated reference databases, increased sampling effort, and multi-marker assays to ensure robust species richness estimates to further validate the approach. eDNA metabarcoding is reliable and provides a path for broader biodiversity assessments that can outperform conventional methods for estimating species richness.</p><h1>High-throughput sequencing of litter and moss eDNA reveals a positive correlation between the diversity of Apicomplexa and their invertebrate hosts across alpine habitats</h1>
<p><a href="https://doi.org/10.1016/j.soilbio.2020.107837">doi:10.1016/j.soilbio.2020.107837</a></p>

<p><a href=None>None</a></p>
<p>A high diversity of Apicomplexa was recently found in tropical soils presumably reflecting the diversity of their invertebrate hosts, but such patterns have not been explored in colder regions. We analysed the diversity of Apicomplexa and their potential metazoan hosts in litter and mosses collected in 11 different alpine habitats using an eDNA metabarcoding approach. The abundance and diversity of Apicomplexa phylotypes and of their potential invertebrate hosts were positively correlated. This confirms that eDNA metabarcoding is a useful tool to explore the unknown biodiversity of free-living eukaryotes, as well as potential host-parasite interactions. Future studies should aim at describing this diversity using a combination of morphological and molecular approaches.</p><h1>DNA metabarcoding reveals the responses of prokaryotes and eukaryotes microbiota to warming: Are the patterns similar between taxonomic and trophic groups?</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2020.106452">doi:10.1016/j.ecolind.2020.106452</a></p>

<p><a href=None>None</a></p>
<p>The aquatic microbiota is involved in crucial ecosystem functions. Thus, investigating the effects of global warming on these organisms is highly relevant, especially given the numerous climatic changes expected by the end of the century. In this study, we used an experimental approach and high-throughput sequencing to evaluate the shortterm effect of warming predicted by different future scenarios in the composition of the planktonic freshwater bacteria and microeukaryotes, and to verify if the same effects occur for each trophic level separately (autotrophic, heterotrophic and mixotrophic). Our experiment demonstrated that the composition for eukaryotes and prokaryotes based on DNA metabarcoding is affected by the increase in temperature and these have a similar pattern of response to warming. This highlights the temperature importance in structuring the communities of different groups. Modifications in the communities were observed through the substitution of specific taxa, which occurred mainly in warmer levels. Changes in community composition were also identified when trophic levels were assessed separately. Mixotrophic eukaryotes organisms are more sensitive to warming, modifying the patterns of composition with an increase in temperature of 2 °C. Microeukaryotes and heterotrophic bacteria were more resistant, with alterations in the communities composition visualized only in higher warming levels. The composition of autotrophic organisms was not affected by the increase in water temperature in any of the biological classifications evaluated, although the richness of eukaryotic autotrophic has decreased with warming. Our results contribute to predict how different biological levels and trophic groups of the aquatic microbiota respond to global warming. This approach is relevant because warming leads to changes in community composition and affects ecosystem processes essentials to the aquatic environment.</p><h1>Environmental DNA and RNA as records of human exposome, including biotic/abiotic exposures and its implications in the assessment of the role of environment in chronic diseases</h1>
<p><a href="https://doi.org/10.3390/ijms21144879">doi:10.3390/ijms21144879</a></p>

<p><a href=None>None</a></p>
<p>Most of environment-related diseases often result from multiple exposures of abiotic and/or biotic stressors across various life stages. The application of environmental DNA/RNA (eDNA/eRNA) to advance ecological understanding has been very successfully used. However, the eminent extension of eDNA/eRNA-based approaches to estimate human exposure to biotic and/or abiotic environmental stressors to understand the environmental causes of chronic diseases has yet to start. Here, we introduce the potential of eDNA/eRNA for bio-monitoring of human exposome and health effects in the real environmental or occupational settings. This review is the first of its kind to discuss how eDNA/eRNA-based approaches can be applied for assessing the human exposome. eDNA-based exposome assessment is expected to rely on our ability to capture the genome-and epigenome-wide signatures left behind by individuals in the indoor and outdoor physical spaces through shedding, excreting, etc. Records of eDNA/eRNA exposome may reflect the early appearance, persistence, and presence of biotic and/or abiotic-exposure-mediated modifications in these nucleic acid molecules. Functional genome-and epigenome-wide mapping of eDNA offer great promise to help elucidate the human exposome. Assessment of longitudinal exposure to physical, biological, and chemical agents present in the environment through eDNA/eRNA may enable the building of an integrative causal dynamic stochastic model to estimate environmental causes of human health deficits. This model is expected to incorporate key biological pathways and gene networks linking individuals, their geographic locations, and random multi-hits of environmental factors. Development and validation of monitoring of eDNA/eRNA exposome should seriously be considered to introduce into safety and risk assessment and as surrogates of chronic exposure to environmental stressors. Here we highlight that eDNA/eRNA reflecting longitudinal exposure of both biotic and abiotic environmental stressors may serve as records of human exposome and discuss its application as molecular tools for understanding the toxicogenomics basis of environment-related health deficits.</p><h1>Contrasting patterns of functional diversity in coffee root fungal communities associated with organic and conventionally managed fields</h1>
<p><a href="https://doi.org/10.1128/AEM.00052-20">doi:10.1128/AEM.00052-20</a></p>

<p><a href=None>None</a></p>
<p>The structure and function of fungal communities in the coffee rhizosphere are influenced by crop environment. Because coffee can be grown along a management continuum from conventional application of pesticides and fertilizers in full sun to organic management in a shaded understory, we used coffee fields to hold host constant while comparing rhizosphere fungal communities under markedly different environmental conditions with regard to shade and inputs. We characterized the shade and soil environment in 25 fields under conventional, organic, or transitional management in two regions of Costa Rica. We amplified the internal transcribed spacer 2 (ITS2) region of fungal DNA from coffee roots in these fields and characterized the rhizosphere fungal community via high-throughput sequencing. Sequences were assigned to guilds to determine differences in functional diversity and trophic structure among coffee field environments. Organic fields had more shade, a greater richness of shade tree species, and more leaf litter and were less acidic, with lower soil nitrate availability and higher soil copper, calcium, and magnesium availability than conventionally managed fields, although differences between organic and conventionally managed fields in shade and calcium and magnesium availability depended on region. Differences in richness and community composition of rhizosphere fungi between organic and conventionally managed fields were also correlated with shade, soil acidity, and nitrate and copper availability. Trophic structure differed with coffee field management. Saprotrophs, plant pathogens, and mycoparasites were more diverse, and plant pathogens were more abundant, in organic than in conventionally managed fields, while saprotroph-plant pathogens were more abundant in conventionally managed fields. These differences reflected environmental differences and depended on region.</p><h1>Applications of environmental DNA (eDNA) in ecology and conservation: opportunities, challenges and prospects</h1>
<p><a href="https://doi.org/10.1007/s10531-020-01980-0">doi:10.1007/s10531-020-01980-0</a></p>

<p><a href=None>None</a></p>
<p>Conserving biodiversity in the face of ever-increasing human pressure is hampered by our lack of basic information on species occurrence, distribution, abundance, habitat requirements, and threats. Obtaining this information requires efficient and sensitive methods capable of detecting and quantifying true occurrence and diversity, including rare, cryptic and elusive species. Environmental DNA (eDNA) is an emerging technique that can increase our ability to detect and quantify biodiversity, by overcoming some of the challenges of labor-intensive traditional surveys. The application of eDNA in ecology and conservation has grown enormously in recent years, but without a concurrent growth in appreciation of its strengths and limitations. In many situations, eDNA may either not work, or it may work but not provide the information needed. Problems with (1) imperfect detection, (2) abundance quantification, (3) taxonomic assignment, (4) eDNA spatial and temporal dynamics, (5) data analysis and interpretation, and (6) assessing ecological status have all been significant. The technique has often been used without a careful evaluation of the technical challenges and complexities involved, and a determination made that eDNA is the appropriate method for the species or environment of interest. It is therefore important to evaluate the scope and relevance of eDNA-based studies, and to identify critical considerations that need to be taken into account before using the approach. We review and synthesize eDNA studies published to date to highlight the opportunities and limitations of utilizing eDNA in ecology and conservation. We identify potential ways of reducing limitations in eDNA analysis, and demonstrate how eDNA and traditional surveys can complement each other.</p><h1>Nitrifying niche differentiation in biofilms from full-scale chloraminated drinking water distribution system</h1>
<p><a href="https://doi.org/10.1016/j.watres.2020.115738">doi:10.1016/j.watres.2020.115738</a></p>

<p><a href=None>None</a></p>
<p>Tropical conditions favour the auto-decomposition of monochloramine (MCA) leading to disinfectant decay and free ammonia in drinking water distribution systems (DWDS); thus, they promote the growth of nitrifiers and the development of biofilms on the inner-pipe surface. Biofilms can adversely impact the provision of safe and biologically stable water. Moreover, there is a general lack of understanding of the role of microbial communities in DWDS in regions with warm temperatures and no distinct seasons. Here, we report a survey on biofilms from full-scale monochloraminated DWDS in a highly urbanised metropolis using next generation sequencing tools. The monitoring campaign consisted of sampling biofilms and bulk waters from 21 in-service pipes. We characterized the microbial community with emphasis on nitrifying bacteria and archaea using 16S rRNA gene amplicon sequencing and potential nitrification activity. Samples grouped into two clusters, characterized by their low (Cluster LD) and high (Cluster HD) α-diversity. Both clusters harbour microorganisms related to nitrification: i) Nitrosomonas (24.9–68.8%), an ammonia oxidising bacterium (AOB) that dominated Cluster LD, and ii) a co-aggregation of genus Nitrospira (9.8–32.5%), a nitrite oxidising bacterium (NOB), and Thaumarchaeota (1.4–10.9%), chemolithotrophic ammonia oxidising (AOA) archaea that were among the most abundant OTUs in Cluster HD. Activity tests performed with fresh biofilm samples confirmed that these two clusters represent distinctive biofilm niches performing different stages of the nitrification process. Cluster LD correlated with a high concentration of MCA, which caused dysbiosis and resulted in high unevenness of the cluster. In cluster HD, with more biomass, chemical reactions involving nitrite increased the MCA demand, releasing ammonia and allowing more nitrifiers to grow, like AOA and NOB. From this study, we conclude that an MCA residual gradient along the DWDS drives and shapes the microbial community assembly and should be considered when designing effective disinfection strategies.</p><h1>A practical guide to DNA metabarcoding for entomological ecologists</h1>
<p><a href="https://doi.org/10.1111/een.12831">doi:10.1111/een.12831</a></p>

<p><a href=None>None</a></p>
<p>1. DNA metabarcoding is a cost-effective species identification approach with great potential to assist entomological ecologists. This review presents a practical guide to help entomological ecologists design their own DNA metabarcoding studies and ensure that sound ecological conclusions can be obtained. 2. The review considers approaches to field sampling, laboratory work, and bioinformatic analyses, with the aim of providing the background knowledge needed to make decisions at each step of a DNA metabarcoding workflow. 3. Although most conventional sampling methods can be adapted to DNA metabarcoding, this review highlights techniques that will ensure suitable DNA preservation during field sampling and laboratory storage. The review also calls for a greater understanding of the occurrence, transportation, and deposition of environmental DNA when applying DNA metabarcoding approaches for different ecosystems. 4. Accurate species detection with DNA metabarcoding needs to consider biases introduced during DNA extraction and PCR amplification, cross-contamination resulting from inappropriate amplicon library preparation, and downstream bioinformatic analyses. Quantifying species abundance with DNA metabarcoding is in its infancy, yet recent studies demonstrate promise for estimating relative species abundance from DNA sequencing reads. 5. Given that bioinformatics is one of the biggest hurdles for researchers new to DNA metabarcoding, several useful graphical user interface programs are recommended for sequence data processing, and the application of emerging sequencing technologies is discussed.</p><h1>Lake-depth related pattern of genetic and morphological diatom diversity in boreal Lake Bolshoe Toko, Eastern Siberia</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0230284">doi:10.1371/journal.pone.0230284</a></p>

<p><a href=None>None</a></p>
<p>Large, old and heterogenous lake systems are valuable sources of biodiversity. The analysis of current spatial variability within such lakes increases our understanding of the origin and establishment of biodiversity. The environmental sensitivity and the high taxonomic richness of diatoms make them ideal organisms to investigate intra-lake variability. We investigated modern intra-lake diatom diversity in the large and old sub-arctic Lake Bolshoe Toko in Siberia. Our study uses diatom-specific metabarcoding, applying a short rbcL marker combined with next-generation sequencing and morphological identification to analyse the diatom diversity in modern sediment samples of 17 intra-lake sites. We analysed abundance-based compositional taxonomic diversity and generic phylogenetic diversity to investigate the relationship of diatom diversity changes with water depth. The two approaches show differences in taxonomic identification and alpha diversity, revealing a generally higher diversity with the genetic approach. With respect to beta diversity and ordination analyses, both approaches result in similar patterns. Water depth or related lake environmental conditions are significant factors influencing intra-lake diatom patterns, showing many significant negative correlations between alpha and beta diversity and water depth. Further, one near-shore and two lagoon lake sites characterized by low (0-10m) and medium (10-30m) water depth are unusual with unique taxonomic compositions. At deeper (>30m) water sites we identified strongest phylogenetic clustering in Aulacoseira, but generally much less in Staurosira, which supports that water depth is a strong environmental filter on the Aulacoseira communities. Our study demonstrates the utility of combining analyses of genetic and morphological as well as phylogenetic diversity to decipher compositional and generic phylogenetic patterns, which are relevant in understanding intra-lake heterogeneity as a source of biodiversity in the sub-arctic glacial Lake Bolshoe Toko.</p><h1>Genetic tools in the management of invasive mammals: recent trends and future perspectives</h1>
<p><a href="https://doi.org/10.1111/mam.12189">doi:10.1111/mam.12189</a></p>

<p><a href=None>None</a></p>
<p>Invasive non-native species are now considered to be one of the greatest threats to biodiversity worldwide. Therefore, efficient and cost-effective management of species invasions requires robust knowledge of their demography, ecology and impacts, and genetic-based techniques are becoming more widely adopted in acquiring such knowledge. We focus on the use of genetic tools in the applied management of mammalian invasions globally, as well as on their inherent advantages and disadvantages. We cover tools that are used in: 1) detecting and monitoring mammalian invaders; 2) identifying origins and invasive pathways; 3) assessing and quantifying the negative impacts of invaders; and 4) population management and potential eradication of invasive mammals. We highlight changes in sequencing technologies, including how the use of techniques such as Sanger sequencing and microsatellite genotyping, for monitoring and tracing invasive pathways respectively, are now giving way to the use of high-throughput sequencing methods. These include the emergence of environmental DNA (eDNA) metabarcoding for the early detection of invasive mammals, and single nucleotide polymorphisms or whole genomes to trace the sources of invasive populations. We are now moving towards trials of genome-editing techniques and gene drives to control or eradicate invasive rodents. Genetic tools can provide vital information that may not be accessible with non-genetic methods, for the implementation of conservation policies (e.g. early detection using systematic eDNA surveillance, the identification of novel pathogens). However, the lack of clear communication of novel genetic methods and results (including transparency and reproducibility) to relevant stakeholders can be prohibitive in translating these findings to appropriate management actions. Geneticists should engage early with stakeholders to co-design experiments in relation to management goals for invasive mammals.</p><h1>Integrated assessment of west coast of South Korea by use of benthic bacterial community structure as determined by eDNA, concentrations of contaminants, and in vitro bioassays</h1>
<p><a href="https://doi.org/10.1016/j.envint.2020.105569">doi:10.1016/j.envint.2020.105569</a></p>

<p><a href=None>None</a></p>
<p>During the past few decades, contamination of sediments by persistent toxic substances (PTSs) has been observed in estuarine and coastal areas on the west coast of South Korea. The contaminants are suspected to cause toxicities in aquatic biota, but little is known about their ecological effects, particularly on benthic microbial communities. In this study, an eDNA-based assessment was applied along with classic assessments of exposure, such as chemistry and in vitro bioassays, to evaluate condition of benthic bacterial communities subjected to PTSs. Two strategies were adopted for the study. One was to conduct a comprehensive assessment in space (by comparing seawater and freshwater sites at five coastal regions) and in time (by following change over a 5-y period). Although we found that bacterial composition varied among and within years, some phyla, such as Proteobacteria (28.7%), Actinobacteria (13.1%), Firmicutes (12.7%), and Chloroflexi (12.5%) were consistently dominated across the study regions. Certain bacterial groups, such as Firmicutes and Verrucomicrobia have been linked to contamination at some sites in the study area and at specific points in time. Bacterial communities were not significantly correlated with salinity or AhR- and ER-mediated potencies, whereas concentrations of PAHs, APs, and certain metals (Cd and Hg) exhibited significant associations to the structure of bacterial communities at the phylum level. In fact, the relative abundance of microbes in the phylum Planctomycetes was significantly and negatively correlated with concentrations of PAHs and metals. Thus, the relative abundance of Planctomycetes could be used as an indicator of sedimentary contamination by PAHs and/or metals. Based on our correlation analyses, Cd and ER-mediated potencies were associated more with bacterial abundances at the class taxonomic level than were other PTSs and metals. Overall, the eDNA-based assessment was useful by augmenting more traditional measures of exposure and responses in a sediment triad approach and has potential as a more rapid screening tool.</p><h1>Simultaneous metabarcoding of eukaryotes and prokaryotes to elucidate the community structures within tardigrade microhabitats</h1>
<p><a href="https://doi.org/10.3390/d12030110">doi:10.3390/d12030110</a></p>

<p><a href=None>None</a></p>
<p>Tardigrades are microscopic invertebrates that can withstand complete desiccation, but their interspecies interactions with prokaryotes and eukaryotes within their microhabitat remain relatively unexplored. Here, I utilized combined metabarcoding of eukaryotes and prokaryotes to simultaneously identify entire community structures within xeric and mesic mosses that harbor tardigrades. The populations of organisms within the microecosystems were successfully determined in 45 xeric moss samples and 47 mesic moss samples. Organismal composition was largely consistent regardless of the moss/lichen substrate, but significantly varied in the two tested locations, possibly because of the differences in environmental humidity. Xeric mosses containing xerophilic tardigrades and other anhydrobiotic invertebrates tended to have significantly limited biological diversity and prokaryotic population dominated by cyanobacteria, suggesting a selection due to extreme desiccation. A combined metabarcoding approach to identify both eukaryotes and prokaryotes can successfully elucidate community structures within microscopic ecosystems, and this can be a potential approach to study the microecology of meiofauna, including tardigrades.</p><h1>Changes in the Phylogenetic Structure of the Metabolically Active Prokaryotic Soil Complex Induced by Oil Pollution</h1>
<p><a href="https://doi.org/10.1134/S0026261720020083">doi:10.1134/S0026261720020083</a></p>

<p><a href=None>None</a></p>
<p>Abstract—: Molecular genetic techniques (FISH and metabarcoding) were used to investigate comparative biodiversity in the prokaryotic complex of soil microcosms of gray forest, chestnut, and chernozem soils before and after oil pollution. At the level of high-rank taxonomic units, the structure of prokaryotic communities from different soil types was similar. In oil-polluted microcosms microbial diversity decreased, and the metbolically active dominants of the Bacteria and Archaea domains changed compared to the control samples. A specific bacterial complex was found to emerge in experimental samples of all soil types, with predominance of the Gammaproteobacteria and Actinobacteria, as well as of archaea, among which Thaumarchaeota and Crenarchaeota prevailed. Members of the soil prokaryotic complex active and inactive in respect to oil pollution were determined. Our results indicate similar succession responses of microbial communities from different soil type to oil pollution.</p><h1>Sampling strategies to assess microbial diversity of Antarctic cryptoendolithic communities</h1>
<p><a href="https://doi.org/10.1007/s00300-020-02625-2">doi:10.1007/s00300-020-02625-2</a></p>

<p><a href=None>None</a></p>
<p>Describing the total biodiversity of an environmental metacommunity is challenging due to the presence of cryptic and rare species and incompletely described taxonomy. How many samples to collect is a common issue that faces ecologists when designing fieldwork sampling. Nowadays, high-throughput sequencing allows examination of large numbers of samples enabling comprehensive biodiversity assessments. In this study, we sought to estimate how the scale of sampling affects accuracy of community diversity description in the McMurdo Dry Valleys (Southern Victoria Land) in Antarctica accounted as the closest Martian analogue on Earth, exhibiting extreme conditions such as low temperatures, wide thermal fluctuations, low nutrient availability and high UV radiation. We found that sampling effort, based on accumulation curves and statistical analysis, had a considerable impact on assessing species richness and composition in these ecosystems, confirming that a sampling as large as nine rock specimens was necessary to detect almost all fungal species present, but was not sufficient to capture whole bacterial assemblage. The sampling would require approximately four times more effort (~ 40 samples) for a comprehensive description of bacterial diversity. Our findings will be helpful to develop strategies to exhaustively describe the microbial diversity of Antarctic cryptoendolithic communities.</p><h1>Relevance of plant and fungal DNA metabarcoding in aerobiology</h1>
<p><a href="https://doi.org/10.1007/s10453-019-09574-2">doi:10.1007/s10453-019-09574-2</a></p>

<p><a href=None>None</a></p>
<p>Aerobiology studies biological particles present in the air of both indoor and outdoor environments. In the past few years, the application of molecular techniques has considerably improved the knowledge of organismal diversity in air samples and highlighted its ecological and clinical importance. DNA metabarcoding in particular has been recently applied worldwide to describe the biological composition of airborne bacteria, fungi and plants. Here, we provide a comprehensive overview of those aerobiology studies which, in the past few years, have applied DNA metabarcoding techniques. We focus specifically on researches including plants and fungi and review the multiplicity of experimental plans, sampling procedures, laboratory processing methods, the choice of the DNA barcode(s), the selection of sequencing platforms and bioinformatics pipelines for the comparability of data obtained from multiple origins.</p><h1>Diatom metabarcoding applied to large scale monitoring networks: Optimization of bioinformatics strategies using Mothur software</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2019.105775">doi:10.1016/j.ecolind.2019.105775</a></p>

<p><a href=None>None</a></p>
<p>Benthic diatoms are routinely used as ecological indicators in rivers. A standardized methodology is based on biofilm sampling, species identification, and counting under microscope. DNA-metabarcoding is an alternative methodology that can identify species and assess their proportion based on high-throughput DNA sequencing. Sequence data is analyzed with bioinformatics tools, and several strategies can be chosen. The strategy choice can affect communities composition and structure, and therefore the resulting ecological assessment. We wanted to optimize the bioinformatics strategy to obtain the closest results to microscopy. This was done in the framework of the Mothur pipeline. Here, 447 samples from French rivers were analyzed in the monitoring context of the European Water Framework Directive. Samples were analyzed both with DNA metabarcoding and microscopy. A usual bioinformatics strategy in Mothur includes clustering DNA-sequences into Operational Taxonomic Units (OTUs). Different algorithms exist for this. From a subsample of 142 samples, we showed that some strategies (Furthest neighbor) gave closer results to microscopy than others (Opticlust) in terms of community structure and diatom index values. However, we showed that OTU clustering was not necessary for ecological monitoring: Direct taxonomic assignment of individual sequence units (ISU) gave similar results to those obtained in microscopy. Interestingly, direct assignment enabled the detection of more species 2 to 3 times faster in terms of computation time compared to the OTU strategy. However, it remained important to remove low quality and chimeric sequences; if not, biomonitoring results differed greatly from microscopy. We showed that it was preferable to have a loose taxonomical identification threshold instead of a stringent one. This allowed detecting more species, which could participate in the index calculation and increased its performance. Indeed, in diatoms, phylogenetically neighbor species often have similar ecologies, and this explains why it is preferable, in a biomonitoring framework, to identify more species with less stringency instead of identifying few species with stringency. Finally, the best strategy (direct assignment of filtered ISU with a loose taxonomical threshold of 60%) was applied to the 447 samples covering a large diversity of ecological qualities. These data were then used to produce quality index values, using a quantification correction factor taking into account species biovolumes. Compared to microscopy, the DNA-based method assigned the same quality class for 66% of the samples, and 72% of the samples had an index value (ranging from 0 to 20) with less than one point difference from microscopy.</p><h1>Dissecting the factors shaping fish skin microbiomes in a heterogeneous inland water system</h1>
<p><a href="https://doi.org/10.1186/s40168-020-0784-5">doi:10.1186/s40168-020-0784-5</a></p>

<p><a href=None>None</a></p>
<p>Background: Fish skin microbiomes are rarely studied in inland water systems, in spite of their importance for fish health and ecology. This is mainly because fish species distribution often covaries with other biotic and abiotic factors, complicating the study design. We tackled this issue in the northern part of the Jordan River system, in which a few fish species geographically overlap, across steep gradients of water temperature and salinity. Results: Using 16S rRNA metabarcoding, we studied the water properties that shape the skin bacterial communities, and their interaction with fish taxonomy. To better characterise the indigenous skin community, we excluded bacteria that were equally abundant in the skin samples and in the water samples, from our analysis of the skin samples. With this in mind, we found alpha diversity of the skin communities to be stable across sites, but higher in benthic loaches, compared to other fish. Beta diversity was found to be different among sites and to weakly covary with the dissolved oxygen, when treated skin communities were considered. In contrast, water temperature and conductivity were strong factors explaining beta diversity in the untreated skin communities. Beta diversity differences between co-occurring fish species emerged only for the treated skin communities. Metagenomics predictions highlighted the microbiome functional implications of excluding the water community contamination from the fish skin communities. Finally, we found that human-induced eutrophication promotes dysbiosis of the fish skin community, with signatures relating to fish health. Conclusions: Consideration of the background water microbiome when studying fish skin microbiomes, across varying fish species and water properties, exposes patterns otherwise undetected and highlight among-fish-species differences. We suggest that sporadic nutrient pollution events, otherwise undetected, drive fish skin communities to dysbiosis. This finding is in line with a recent study, showing that biofilms capture sporadic pollution events, undetectable by interspersed water monitoring.</p><h1>Checklist of the planktonic marine dinoflagellates of New Zealand</h1>
<p><a href="https://doi.org/10.1080/00288330.2019.1626746">doi:10.1080/00288330.2019.1626746</a></p>

<p><a href=None>None</a></p>
<p>Planktonic dinoflagellate records for New Zealand are substantial due to intense monitoring programmes that have taken place on behalf of New Zealand’s biotoxin regulators and the shellfish industry since 1993. At that time a Karenia bloom caused human illnesses and shellfish harvesting ceased until monitoring was instigated. Phytoplankton records are based on morphological identification using light microscopy but are backed up by government funded research programmes which implement techniques such as electron microscopy, DNA sequencing, molecular detection assays and high throughput sequencing of environmental DNA. This checklist will support management of aquaculture industries, recreational shellfish harvests and environmental health initiatives. Some genera are considered benthic or epiphytic, but have lengthy planktonic life stages. Forty-five genera in the class Dinophyceae Fritsch are reported: Akashiwo, Alexandrium, Amphidinium, Amylax, Azadinium, Biecheleria, Bysmatrum, Cachonina, Cochlodinium (synonym: Margalefidinium), Coolia, Dicroerisma, Dinophysis, Diploneis, Diplopsalis, Fragilidium, Glenodinium, Gonyaulax, Gymnodinium, Gyrodinium, Heterocapsa, Karenia, Karlodinium, Lepidodinium, Lingulodinium, Margalefidinium, Ostreopsis, Oxyphysis, Pelagodinium, Pentapharsodinium, Phalacroma, Podolampas, Polarella, Polykrikos, Prorocentrum, Protodinium, Protoodinium, Protoperidinium, Pseliodinium, Scrippsiella, Takayama, Togula, Torodinium, Tripos, Vulcanodinium, Wolosynska. Other genera belonging to the Infraphylum Dinoflagellata also occur in New Zealand waters. They are not in the list but include Noctiluca, Pronoctiluca and Spatulodinium (Class: Noctilucophyceae) and Oxyrrhis (Class: Oxyrrhidophyceae).</p><h1>Next-generation sequencing technologies for environmental dna as an efficient bio indicator for bacterial biodiversity in tigris river, iraq</h1>
<p><a href="https://doi.org/10.31838/srp.2020.11.158">doi:10.31838/srp.2020.11.158</a></p>

<p><a href=None>None</a></p>
<p>Access to multiple, large sequence data volumes, as well as further improvements in read length and sequence platform efficiency, lead to better sample diversity. In order to assess the ability of Next Generation Sequence (NGS) approach to establish and evaluate the environmental molecular diagnostics, directly monitor bulk bacterial communities, the Illumina system of bacterial 16S rRNA gene amplicon was investigated in total bacterial communities in water samples. The dominant bacterial species can be identified through this method. NGS data will also allow the identification of drivers of the composition of bacterial communities during water treatment and distribution in combination with metadata, demonstrating the power of this approach to track bacterial regrowth and contamination in technical systems.</p><h1>Assessing the responses of Sphagnum micro-eukaryotes to climate changes using high throughput sequencing</h1>
<p><a href="https://doi.org/10.7717/peerj.9821">doi:10.7717/peerj.9821</a></p>

<p><a href=None>None</a></p>
<p>Current projections suggest that climate warming will be accompanied by more frequent and severe drought events. Peatlands store ca. one third of the world's soil organic carbon. Warming and drought may cause peatlands to become carbon sources through stimulation of microbial activity increasing ecosystem respiration, with positive feedback effect on global warming. Micro-eukaryotes play a key role in the carbon cycle through food web interactions and therefore, alterations in their community structure and diversity may affect ecosystem functioning and could reflect these changes. We assessed the diversity and community composition of Sphagnum-associated eukaryotic microorganisms inhabiting peatlands and their response to experimental drought and warming using high throughput sequencing of environmental DNA. Under drier conditions, micro-eukaryotic diversity decreased, the relative abundance of autotrophs increased and that of osmotrophs (including Fungi and Peronosporomycetes) decreased. Furthermore, we identified climate change indicators that could be used as early indicators of change in peatland microbial communities and ecosystem functioning. The changes we observed indicate a shift towards a more "terrestrial"community in response to drought, in line with observed changes in the functioning of the ecosystem.</p><h1>Assessment of fish biodiversity in four Korean rivers using environmental DNA metabarcoding</h1>
<p><a href="https://doi.org/10.7717/peerj.9508">doi:10.7717/peerj.9508</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is a cost-effective novel approach to estimate biodiversity in an ecosystem. In this study, the MiFish pipeline was employed to test if the system methodology is sufficiently reliable to estimate fish biodiversity in Korean rivers. A total of 125 unique haplotypes and 73 species were identified at the species level from 16 water samples collected from a single survey in four Korean rivers (Hyeongsan, Taehwa, Seomjin, and Nakdong). Among the four rivers, the highest species richness was recorded in the Seomjin River (52 species), followed by the Taehwa (42 species) and Hyeongsan (40 species) rivers. The Nakdong River (26 species) presented the lowest species richness and number of endemic species, presumably due to its metropolitan location and anthropogenic impacts, such as dams or weirs. We were also able to detect that five exotic species (Carassius cuvieri, Cyprinus carpio, Cyprinus megalophthalmus, Lepomis macrochirus, and Micropterus salmoides) are widely distributed in all surveyed rivers, a situation that might be problematic in terms of conservation. Our findings indicate that the eDNA metabarcoding technique is one of the most cost-effective scientific tools available for the management and conservation of the freshwater fish resources available in Korea. However, the low number of 12S sequences of endemic species in the database and low resolution of the MiFish region for differentiating several taxa should be upgraded for their wide use.</p><h1>Culicidae-centric metabarcoding through targeted use of D2 ribosomal DNA primers</h1>
<p><a href="https://doi.org/10.7717/peerj.9057">doi:10.7717/peerj.9057</a></p>

<p><a href=None>None</a></p>
<p>A practical limitation to many metabarcoding initiatives is that sampling methods tend to collect many non-target taxa, which become ‘‘amplicon noise’’ that can saturate Next Generation Sequencing results and lead to both financial and resource inefficiencies. An available molecular tool that can significantly decrease these non-target amplicons and decrease the need for pre-DNA-extraction sorting of bycatch is the design of PCR primers tailored to the taxa under investigation. We assessed whether the D2 extension segment of the 28S ribosomal operon can limit this shortcoming within the context of mosquito (Culicidae) monitoring. We designed PCR primers that are fully conserved across mosquitos and exclude from amplification most other taxa likely to be collected with current sampling apparatuses. We show that, given enough sequencing depth, D2 is an effective marker for the detection of mosquito sequences within mock genomic DNA pools. As few as 3,050 quality-filtered Illumina reads were able to recover all 17 species in a bulk pool containing as little as 0.2% of constituent DNA from single taxa. We also mixed these mosquito DNA pools with high concentrations of non-Culicidae bycatch DNA and show that the component mosquito species are generally still recoverable and faithful to their original relative frequencies. Finally, we show that there is little loss of fidelity in abundance parameters when pools from degraded DNA samples were sequenced using the D2 primers.</p><h1>Validating metabarcoding-based biodiversity assessments with multi-species occupancy models: A case study using coastal marine eDNA</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0224119">doi:10.1371/journal.pone.0224119</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is an increasingly popular method for rapid biodiversity assessment. As with any ecological survey, false negatives can arise during sampling and, if unaccounted for, lead to biased results and potentially misdiagnosed environmental assessments. We developed a multi-scale, multi-species occupancy model for the analysis of community biodiversity data resulting from eDNA metabarcoding; this model accounts for imperfect detection and additional sources of environmental and experimental variation. We present methods for model assessment and model comparison and demonstrate how these tools improve the inferential power of eDNA metabarcoding data using a case study in a coastal, marine environment. Using occupancy models to account for factors often overlooked in the analysis of eDNA metabarcoding data will dramatically improve ecological inference, sampling design, and methodologies, empowering practitioners with an approach to wield the high-resolution biodiversity data of next-generation sequencing platforms.</p><h1>Advances in the use of molecular tools in ecological and biodiversity assessment of aquatic ecosystems</h1>
<p><a href="https://doi.org/10.23818/limn.39.27">doi:10.23818/limn.39.27</a></p>

<p><a href=None>None</a></p>
<p>Conservation and sustainable management of aquatic ecosystems is a priority in environmental programs worldwide. However, these aims are highly dependent on the efficiency, accuracy and cost of existent methods for the detection of keystone species and monitoring of biological communities. Rapid advances in eDNA, barcoding and metabarcoding promoted by high-throughput sequencing technologies are generating millions of sequences in a fast way, with a promising cost reduction, and overcoming some difficulties of the traditional taxonomic approaches. This paper provides an updated broad perspective of the current developments in this dynamic field presented in the special session (SS) “The use of molecular tools in ecological and biodiversity assessment of aquatic ecosystems” of the XIX Congress of the Iberian Association of Limnology (AIL2018), held in Coimbra, Portugal. Developments presented are mainly focused on the Iberian Peninsula (Portugal and Spain, including Atlantic Macaronesian islands) but include studies in France, Germany, Finland, Russia (Siberia) and South America. The networks within which these researchers are involved are yet even broader, profiting from existing molecular facilities, and traditional taxonomic expertise, which can be viewed as a characteristic of this new research area. It was evident in the SS that the use of molecular tools is widespread, being used to study a diversity of aquatic systems, from rivers’ headwaters to estuaries and coastal lagoons, and volcanic, mountain and frozen lakes to hot springs. The organisms targeted are likewise varied and include fish, macroinvertebrates, meiofauna, microalgae such as diatoms and dinoflagellates, other protists, fungi, and bacteria (cyanobacteria and other). Some studies address the whole biodiversity (i.e., all species present independently of the taxonomic group) from environmental samples of water, biofilms and preservative solution from field samples (e.g., ethanol from macroinvertebrate samples). Great advances were acknowledged in the special session, namely in the use of metabarcoding for detecting hidden biodiversity, juvenile stages, low-abundance species, non-indigenous species and toxicity potential, and ultimately for ecological monitoring of diatoms and invertebrates. Yet, several drawbacks were highlighted and need further work, which include: taxonomic gaps in the reference databases (including gaps at species level and on intraspecific variability) or absence of public databases (e.g. for meiofauna), still high sequencing costs, the need of a substantial bioinformatics effort, difficulties in establishing the amount of environmental sample necessary for a good DNA extraction and the need for testing different genetic markers to obtain accurate results.</p><h1>Towards an iberian dna barcode reference library of freshwater macroinvertebrates and fishes</h1>
<p><a href="https://doi.org/10.23818/limn.39.06">doi:10.23818/limn.39.06</a></p>

<p><a href=None>None</a></p>
<p>Freshwater macroinvertebrates and fishes are focal groups in major ecosystem biomonitoring programs in the Iberian Peninsula. Yet, their use as bioindicators is sometimes constrained by the time and cost needed for sorting macroinvertebrates specimens and their challenging taxonomic identification, and the huge sampling procedures for capturing rare or incipient fish species, respectively. Given the increasing cost-effectiveness of metazoan identification based on metabarcoding [i.e., high-throughput sequencing (HTS) of DNA barcodes] and reliability of species-level identification and the high number of samples that can be processed, its use in biomonitoring of freshwater communities can provide an alternative to morphology-based approaches. However, the accuracy of species assignment in metabarcoding approaches relies on the availability of a comprehensive DNA barcode reference library. Because of the high level of endemicity in the Iberian Peninsula, current public repositories for DNA barcodes may not be informative enough to identify the Iberian fauna to species level. Here, we compiled the Iberian freshwater macroinvertebrates and fishes taxonomic list (including indigenous and non-indigenous species) and the available molecular data for the cytochrome oxidase I DNA barcode (cox1, COI-5P) in public repositories to assess the extent of DNA barcode coverage. The DNA barcode coverage was reported for DNA fragments within the Folmer region (658 bp). Given that HTS platforms provide DNA sequence in the range of 50-400 bp in length, we also reported the second half of the DNA barcode (313 bp, Leray region) and the first part of the Leray region (285 bp, Leray-285), which are short DNA barcodes useful to assign metabarcoding cox1 data. For macroinvertebrates, the final taxonomic checklist comprises 3348 species including Mollusca (65 species), Crustacea (101 species) and Insecta (3182 species). We present an initial DNA barcode reference library, with an overall coverage of ~ 35 % of the Iberian taxa. Exploring this data, we find a strong taxonomic bias. Based on Leray-285, Odonata (43 of 79 species barcoded, 54.43 %) and Hemiptera (44 of 81 species barcoded, 54.32 %) were the best represented lineages. In contrast, Diptera (393 of 1693 species barcoded, 23.21 %), and Plecoptera (42 of 135 species barcoded, 31.11 %) were underrepresented. For fishes, the available DNA barcode data covered 98.11 % of the indigenous (76) and non-indigenous (30) species. By revealing and quantifying current gaps on the available data (~ 65 %), we aim to improve efficiency in designing the next steps towards the ambitious yet necessary goal of compiling a complete DNA barcode reference library for Iberian macroinvertebrates and fishes.</p><h1>Advances and prospects of environmental DNA in neotropical rainforests</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2020.01.001">doi:10.1016/bs.aecr.2020.01.001</a></p>

<p><a href=None>None</a></p>
<p>The rainforests of the Neotropics shelter a vast diversity of plant, animal and microscopic species that provide critical ecosystem goods and services for both local and worldwide populations. These environments face a major crisis due to increased deforestation, pollution, and climate change, emphasizing the need for more effective conservation efforts. The adequate monitoring of these ecosystems has proven a complex and time consuming endeavour, which depends on ever dwindling taxonomic expertise. To date, many species remain undiscovered, let alone described, with otherwise limited information regarding known species population distributions and densities. Overcoming these knowledge shortfalls and practical limitations is becoming increasingly possible through techniques based on environmental DNA (eDNA), i.e., DNA that can be obtained from environmental samples (e.g. tissues, soil, sediment, water, etc.). When coupled with high-throughput sequencing, these techniques now enable realistic, cost-effective, and standardisable biodiversity assessments. This opens up enormous opportunities for advancing our understanding of complex and species-rich tropical communities, but also in facilitating large-scale biomonitoring programs in the neotropics. In this review, we provide a brief introduction to eDNA methods, and an overview of their current and potential uses in both terrestrial and aquatic ecosystems of neotropical rainforests. We also discuss the limits and challenges of these methods for our understanding and monitoring of biodiversity, as well as future research and applied perspectives of these techniques in neotropical rainforests, and beyond.</p><h1>Molecular biogeography of planktonic and benthic diatoms in the Yangtze River</h1>
<p><a href="https://doi.org/10.1186/s40168-019-0771-x">doi:10.1186/s40168-019-0771-x</a></p>

<p><a href=None>None</a></p>
<p>Background: Diatoms are of great significance to primary productivity in oceans, yet little is known about their biogeographic distribution in oligotrophic rivers. Results: With the help of metabarcoding analysis of 279 samples from the Yangtze River, we provided the first integral biogeographic pattern of planktonic and benthic diatoms over a 6030 km continuum along the world's third largest river. Our study revealed spatial dissimilarity of diatoms under varying landforms, including plateau, mountain, foothill, basin, foothill-mountain, and plain regions, from the river source to the estuary. Environmental drivers of diatom communities were interpreted in terms of photosynthetically active radiation, temperature, channel slope and nutrients, and human interference. Typical benthic diatoms, such as Pinnularia, Paralia, and Aulacoseira, experienced considerable reduction in relative abundance downstream of the Three Gorges Dam and the Xiluodu Dam, two of the world's largest dams. Conclusions: Our study revealed that benthic diatoms are of particular significance in characterizing motile guild in riverine environments, which provides insights into diatom biogeography and biogeochemical cycles in large river ecosystems.</p><h1>Watered-down biodiversity? A comparison of metabarcoding results from DNA extracted from matched water and bulk tissue biomonitoring samples</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0225409">doi:10.1371/journal.pone.0225409</a></p>

<p><a href=None>None</a></p>
<p>Biomonitoring programs have evolved beyond the sole use of morphological identification to determine the composition of invertebrate species assemblages in an array of ecosystems. The application of DNA metabarcoding in freshwater systems for assessing benthic invertebrate communities is now being employed to generate biological information for environmental monitoring and assessment. A possible shift from the extraction of DNA from net-collected bulk benthic samples to its extraction directly from water samples for metabarcoding has generated considerable interest based on the assumption that taxon detectability is comparable when using either method. To test this, we studied paired water and benthos samples from a taxon-rich wetland complex, to investigate differences in the detection of arthropod taxa from each sample type. We demonstrate that metabarcoding of DNA extracted directly from water samples is a poor surrogate for DNA extracted from bulk benthic samples, focusing on key bioindicator groups. Our results continue to support the use of bulk benthic samples as a basis for metabarcoding-based biomonitoring, with nearly three times greater total richness in benthic samples compared to water samples. We also demonstrated that few arthropod taxa are shared between collection methods, with a notable lack of key bioindicator EPTO taxa in the water samples. Although species coverage in water could likely be improved through increased sample replication and/or increased sequencing depth, benthic samples remain the most representative, cost-effective method of generating aquatic compositional information via metabarcoding.</p><h1>Meiofaunal diversity in the Atlantic Forest soil: A quest for nematodes in a native reserve using eukaryotic metabarcoding analysis</h1>
<p><a href="https://doi.org/10.1016/j.foreco.2019.117591">doi:10.1016/j.foreco.2019.117591</a></p>

<p><a href=None>None</a></p>
<p>The assessment of environmental communities may lead to the identification of the novel or not-yet-classified organisms. Few studies have focused on the meiofauna of poorly known habitats such as native soils in subtropical forests. We surveyed eukaryotic communities inhabiting soil in an unexplored area of a native Atlantic Forest in southern Brazil using high-throughput sequencing. We analyzed a total of 281,400 sequences using V4 and V9 hypervariable regions of 18S rRNA gene. Opisthokonta was the most abundant supergroup, with Fungi and Metazoa representing an average of 40.6% and 15.8% of the sequences, respectively. Among the metazoan, Nematoda was the second most abundant phylum (4.8%). Isolation and morphological assessment 1745 specimens of nematodes, classified by functional feeding groups as plant parasites (35%), bacterial feeders (30%), omnivores (7%) and predators (5%). Specimens belonged to the class Chromadorea (89.6%) and Enoplea (10.4%).Approximately 65% of them could be morphologically classified only to order level. Results identified a considerable number of newly or still uncharacterized nematodes in soil biota in Atlantic Forest soils. Furthermore, this is the first study in this subtropical area to show that both high throughput sequencing and morphology can give complementary indications of the diversity of nematodes in soil samples.</p><h1>Studying Ecosystems With DNA Metabarcoding: Lessons From Biomonitoring of Aquatic Macroinvertebrates</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00434">doi:10.3389/fevo.2019.00434</a></p>

<p><a href=None>None</a></p>
<p>An ongoing challenge for ecological studies has been the collection of data with high precision and accuracy at a suitable scale to detect and manage critical global change processes. A major hurdle has been the time-consuming and challenging process of sorting and identification of organisms, but the rapid development of DNA metabarcoding as a biodiversity observation tool provides a potential solution. As high-throughput sequencing becomes more rapid and cost-effective, a “big data” revolution is anticipated, based on higher and more accurate taxonomic resolution, more efficient detection, and greater sample processing capacity. These advances have the potential to amplify the power of ecological studies to detect change and diagnose its cause, through a methodology termed “Biomonitoring 2.0.” Despite its promise, the unfamiliar terminology and pace of development in high-throughput sequencing technologies has contributed to a growing concern that an unproven technology is supplanting tried and tested approaches, lowering trust among potential users, and reducing uptake by ecologists and environmental management practitioners. While it is reasonable to exercise caution, we argue that any criticism of new methods must also acknowledge the shortcomings and lower capacity of current observation methods. Broader understanding of the statistical properties of metabarcoding data will help ecologists to design, test and review evidence for new hypotheses. We highlight the uncertainties and challenges underlying DNA metabarcoding and traditional methods for compositional analysis, specifically comparing the interpretation of otherwise identical bulk-community samples of freshwater benthic invertebrates. We explore how taxonomic resolution, sample similarity, taxon misidentification, and taxon abundance affect the statistical properties of these samples, but recognize these issues are relevant to applications across all ecosystem types. In conclusion, metabarcoding has the capacity to improve the quality and utility of ecological data, and consequently the quality of new research and efficacy of management responses.</p><h1>Assessing pollution of aquatic environments with diatoms' DNA metabarcoding: Experience and developments from France Water Framework Directive networks</h1>
<p><a href="https://doi.org/10.3897/mbmg.3.39646">doi:10.3897/mbmg.3.39646</a></p>

<p><a href=None>None</a></p>
<p>Ecological status assessment of watercourses is based on the calculation of quality indices using pollution sensitivity of targeted biological groups, including diatoms. The determination and quantification of diatom species is generally based on microscopic morphological identification, which requires expertise and is time-consuming and costly. In Europe, this morphological approach is legally imposed by standards and regulatory decrees by the Water Framework Directive (WFD). Over the past decade, a DNA-based molecular biology approach has newly been developed to identify species based on genetic criteria rather than morphological ones (i.e. DNA metabarcoding). In combination with high throughput sequencing technologies, metabarcoding makes it possible both to identify all species present in an environmental sample and to process several hundred samples in parallel. This article presents the results of two recent studies carried out on the WFD networks of rivers of Mayotte (2013-2018) and metropolitan France (2016-2018). These studies aimed at testing the potential application of metabarcoding for biomonitoring in the context of the WFD. We discuss the various methodological developments and optimisations that have been made to make the taxonomic inventories of diatoms produced by metabarcoding more reliable, particularly in terms of species quantification. We present the results of the application of this DNA approach on more than 500 river sites, comparing them with those obtained using the standardised morphological method. Finally, we discuss the potential of metabarcoding for routine application, its limits of application and propose some recommendations for future implementation in WFD.</p><h1>Vertebrate diversity revealed by metabarcoding of bulk arthropod samples from tropical forests</h1>
<p><a href="https://doi.org/10.1002/edn3.34">doi:10.1002/edn3.34</a></p>

<p><a href=None>None</a></p>
<p>Background: Thousands of bulk arthropod samples are collected globally every year for monitoring programs, conservation efforts, and ecosystem assessments. The taxonomic contents of these samples can be assessed either morphologically or molecularly using DNA metabarcoding coupled with high-throughput sequencing, the latter of which has gained popularity in recent years. In a related field, only vertebrateingesting invertebrates, such as carrion flies and blood-feeding leeches, are targeted for collection, and metabarcoding is carried out on the vertebrate DNA in their gut contents to provide information on vertebrate diversity (invertebrate-derived DNA, iDNA). Aims: Here, we show that the two approaches can be combined, that is, that vertebrate DNA can be detected through metabarcoding of bulk arthropod samples. Materials and Methods: Two metabarcoding primer sets were used to PCR amplify mammal and vertebrate DNA in DNA extracted from bulk arthropod samples collected with pitfall and Malaise traps in tropical forests in Brazil and Tanzania. Results: In total, 32 vertebrate taxa were detected representing mammals, amphibians, and birds. Detected taxa were within, or close to, their known geographical distributions. Conclusion: This study demonstrates that with a relatively small additional investment, information on vertebrate diversity can be obtained from bulk arthropod samples. This is of particular interest in projects where bulk arthropod samples are collected and extracted with the aim to use metabarcoding to assess arthropod taxa. In such studies, the additional information on vertebrates can further inform ecological assessments and monitoring programs and function as a supplement to traditional survey methods of vertebrates.</p><h1>Primer and Database Choice Affect Fungal Functional but Not Biological Diversity Findings in a National Soil Survey</h1>
<p><a href="https://doi.org/10.3389/fenvs.2019.00173">doi:10.3389/fenvs.2019.00173</a></p>

<p><a href=None>None</a></p>
<p>The internal transcribed spacer (ITS) region is the accepted DNA barcode of fungi. Its use has led to a step-change in the assessment and characterisation of fungal communities from environmental samples by precluding the need to isolate, culture, and identify individuals. However, certain functionally important groups, such as the arbuscular mycorrhizas (Glomeromycetes), are better characterised by alternative markers such as the 18S rRNA region. Previous use of an ITS primer set in a nationwide metabarcoding soil biodiversity survey revealed that fungal richness declined along a gradient of productivity and management intensity. Here, we wanted to discern whether this trend was also present in data generated from universal 18S primers. Furthermore, we wanted to extend this comparison to include measures of functional diversity and establish trends with soil types and soil organic matter (SOM) content. Over the 413 individual sites examined (arable, grassland, woodland, moorland, heathland), we found congruent trends of total fungal richness and β-diversity across land uses, SOM class, and soil type with both ITS and 18S primer sets. A total of 24 fungal classes were shared between datasets, in addition to 15 unique to ITS1 and 12 unique to 18S. However, using FUNGUILD, divergent trends of functional group richness became apparent, especially for symbiotrophic fungi, likely driven by an increased detection rate of Glomeromycetes in the 18S dataset. The disparate trends were also apparent when richness and β-diversity were compared to soil properties. Additionally, we found SOM class to be a more meaningful variable than soil type biodiversity for predicting biodiversity analyses because organic matter was calculated for each sample whereas soil type was assigned from a national soil map. We advocate that a combination of fungal primers should be used in large-scale soil biodiversity surveys to capture important groups that can be underrepresented by universal barcodes. Utilising such an approach can prevent the oversight of ubiquitous but poorly described species as well as critically important functional groups.</p><h1>Relative performance of MinION (Oxford Nanopore Technologies) versus Sequel (Pacific Biosciences) thirdgeneration sequencing instruments in identification of agricultural and forest fungal pathogens</h1>
<p><a href="https://doi.org/10.1128/AEM.01368-19">doi:10.1128/AEM.01368-19</a></p>

<p><a href=None>None</a></p>
<p>Culture-based molecular identification methods have revolutionized detection of pathogens, yet these methods are slow and may yield inconclusive results from environmental materials. The second-generation sequencing tools have muchimproved precision and sensitivity of detection, but these analyses are costly and may take several days to months. Of the third-generation sequencing techniques, the portable MinION device (Oxford Nanopore Technologies) has received much attention because of its small size and possibility of rapid analysis at reasonable cost. Here, we compare the relative performances of two third-generation sequencing instruments, MinION and Sequel (Pacific Biosciences), in identification and diagnostics of fungal and oomycete pathogens from conifer (Pinaceae) needles and potato (Solanum tuberosum) leaves and tubers. We demonstrate that the Sequel instrument is efficient for metabarcoding of complex samples, whereas MinION is not suited for this purpose due to a high error rate and multiple biases. However, we find that MinION can be utilized for rapid and accurate identification of dominant pathogenic organisms and other associated organisms from plant tissues following both ampliconbased and PCR-free metagenomics approaches. Using the metagenomics approach with shortened DNA extraction and incubation times, we performed the entire MinION workflow, from sample preparation through DNA extraction, sequencing, bioinformatics, and interpretation, in 2.5 h. We advocate the use of MinION for rapid diagnostics of pathogens and potentially other organisms, but care needs to be taken to control or account for multiple potential technical biases.</p><h1>Testing different (e)DNA metabarcoding approaches to assess aquatic oligochaete diversity and the biological quality of sediments</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2019.105453">doi:10.1016/j.ecolind.2019.105453</a></p>

<p><a href=None>None</a></p>
<p>Aquatic oligochaetes are important bioindicators of sediment quality in watercourses and lakes, but their morphological identification to the species level is challenging and sometimes impossible. The use of DNA barcoding and metabarcoding could greatly facilitate the identifications of specimens and improve ecological diagnoses based on oligochaete communities. The aim of this study was to test how well metabarcoding approaches based on high throughout sequencing (HTS) could perform in assessing oligochaete species diversity and the biological quality of sediments. We analysed oligochaete communities at several sites in Swiss rivers, comparing morphological data to metabarcoding of mixed specimen, sieved sediment and total sediment samples. We amplified the cytochrome c oxidase (COI) marker using universal primers and/or specific primers to metazoans. Our results showed that metabarcoding analysis of mixed specimens allowed a more representative assessment of the biodiversity than the metabarcoding of sieved or total sediments. Although community structures obtained with the morphological and metabarcoding analyses were different in term of presence/absence of species and species abundances, the ecological diagnoses based on these two approaches largely agreed.</p><h1>Applicability of DNA metabarcoding approach in the bioassessment of Portuguese rivers using diatoms</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2019.105470">doi:10.1016/j.ecolind.2019.105470</a></p>

<p><a href=None>None</a></p>
<p>Benthic diatom communities are used in the ecological assessment of Portuguese rivers through the calculation of an autecological index, the IPS (Indice de Polluosensibilité Spécifique), officially adopted for Portugal. This index requires a high level of taxonomic expertise for morphological identification of individuals at species level. Advances in genomics, such as the DNA metabarcoding combined with high-throughput sequencing techniques offer a promising alternative to this classical approach, decreasing expertise requirement and therefore reducing time and costs. The aim of this study was to test the potential of DNA metabarcoding of diatoms in the bioassessment of Portuguese rivers, by comparing the classifications of the IPS obtained with morphological and molecular approaches. A total of 88 samples were collected in four river types and a quality gradient in central Portugal in the spring of 2017, following the Water Framework Directive (WFD) standards. The morphological approach comprised taxonomic identification of at least 400 valves, under the light microscope. Molecular approach included eDNA extraction followed by PCR amplification and DNA high-throughput sequencing (Illumina MiSeq) using a diatom-specific 312 bp rbcL DNA barcode. Sequences were analysed with Mothur software, producing Operational Taxonomic Units (OTUs) that were taxonomically assigned using the R-Syst::diatom reference library. We also tested the effect of a correction factor (CF) based on biovolume for the quantification of molecular data. When comparing the number of species identified, morphological, molecular without and with CF inventories revealed 306, 125 and 111 species, respectively. Molecular data showed 67% of unassigned OTUs. These results were mainly due to the incompleteness of the reference library, which currently represents the major difficulty in taxonomic assignment of DNA sequences in metabarcoding. Regarding IPS values, the results indicated a good correlation between morphological and molecular methods, particularly when applying the CF. Furthermore, Principal Coordinates analysis plot based on species abundances revealed a quality gradient in all 3 methodologies (morphological, molecular without and with CF). These results support the hypothesis that DNA metabarcoding is a valid approach for ecological quality assessment of Portuguese rivers. Yet, there is still work to be done on this new methodology to enable a smooth transition between the traditional and this new approach without misspend accumulated knowledge from the last decades on water quality assessment.</p><h1>Diatom DNA Metabarcoding for Biomonitoring: Strategies to Avoid Major Taxonomical and Bioinformatical Biases Limiting Molecular Indices Capacities</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00409">doi:10.3389/fevo.2019.00409</a></p>

<p><a href=None>None</a></p>
<p>Recent years provided intense progression in the implementation of molecular techniques in a wide variety of research fields in ecology. Biomonitoring and bioassessment can greatly benefit from DNA metabarcoding and High-Throughput Sequencing (HTS) methods that potentially provide reliable, high quantity and quality standardized data in a cost- and time-efficient way. However, DNA metabarcoding has its drawbacks, introducing biases at all the steps of the process, particularly during bioinformatics treatments used to prepare HTS data for ecological analyses. The high diversity of bioinformatics methods (e.g., OTU clustering, chimera detection, taxonomic assignment) and parameters (e.g., percentage similarity threshold used to define OTUs) make inter-studies comparison difficult, limiting the development of standardized and easy-accessible bioassessment procedures for routine freshwater monitoring. In order to study and overcome these drawbacks, we constructed four de novo indices to assess river ecological status based on the same biological samples of diatoms analyzed with morphological and molecular methods. The biological inventories produced are (i) morphospecies identified by microscopy, (ii) OTUs provided via metabarcoding and hierarchical clustering of sequences using a 95% similarity threshold, (iii) individual sequence units (ISUs) via metabarcoding and only minimal bioinformatical quality filtering, and (iv) exact sequence variants (ESVs) using DADA2 denoising algorithm. The indices based on molecular data operated directly with ecological values estimated for OTUs/ ISUs/ ESVs. Our study used an approach of bypassing taxonomic assignment, so bias related to unclassified sequences missing from reference libraries could be handled and no information on ecology of sequences is lost. Additionally, we showed that the indices based on ISUs and ESVs were equivalent, outperforming the OTU-based one in terms of predictive power and accuracy by revealing the hidden ecological information of sequences that are otherwise clustered in the same OTU (intra-species/intra-population variability). Furthermore, ISUs, ESVs, and morphospecies indices provided similar estimation of site ecological status, validating that ISUs with limited bioinformatics treatments may be used for DNA freshwater monitoring. Our study is a proof of concept where taxonomy- and clustering-free approach is presented, that we believe is a step forward a standardized and comparable DNA bioassessment, complementary to morphological methods.</p><h1>Insight into picophytoplankton diversity of the subarctic White Sea—The first recording of Pedinophyceae in environmental DNA</h1>
<p><a href="https://doi.org/10.1002/mbo3.892">doi:10.1002/mbo3.892</a></p>

<p><a href=None>None</a></p>
<p>Operational taxonomic units 94%–95% similar to the known Pedinophyceae were found as a result of high-through sequencing of 18S rDNA V4 amplicons of environmental DNA from the summer picophytoplankton samples from the White Sea. Partial sequence of a ribosomal operon (the 5,298 bp includes partial 18S and 28S rDNA, complete 5.8S rDNA, ITS1, and ITS2 sequences) and a partial 2,112 bp chloroplast 23S rDNA sequence White Sea Pedinophyceae was amplified from metagenomic DNA by specific primers and sequenced. A new phylotype was designated as uncultured Pedinophyceae WS. On Chlorophyta phylogenetic trees the discovered phylotype occupies a basal position in the Marsupiomonadales clade. The synapomorphic base substitutions in rRNA hairpins confirm the relationship of Pedinophyceae WS to Marsupiomonadales and its difference from known genera of the order. The obtained results extend knowledge of picophytoplankton diversity in subarctic waters.</p><h1>Current practice in plankton metabarcoding: Optimization and error management</h1>
<p><a href="https://doi.org/10.1093/plankt/fbz041">doi:10.1093/plankt/fbz041</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of a targeted genetic marker is being widely used to analyze biodiversity across taxa and environments. Amid a multitude of exciting findings, scientists have also identified and addressed technical and biological limitations. Improved study designs and alternative sampling, lab and bioinformatic procedures have progressively enhanced data quality, but some problems persist. This article provides a framework to recognize and bypass the main types of errors that can affect metabarcoding data: false negatives, false positives, artifactual variants, disproportions and incomplete or incorrect taxonomic identifications. It is crucial to discern potential error impacts on different ecological parameters (e.g. taxon distribution, community structure, alpha and beta-diversity), as error management implies compromises and is thus directed by the research question. Synthesis of multiple plankton metabarcoding evaluations (mock sample sequencing or microscope comparisons) shows that high-quality data for qualitative and some semiquantitative goals can be achieved by implementing three checkpoints: first, rigorous protocol optimization; second, error minimization; and third, downstream analysis that considers potentially remaining biases. Conclusions inform us about the reliability of metabarcoding for plankton studies and, because plankton provides unique chances to compare genotypes and phenotypes, the robustness of this method in general.</p><h1>Gaps in DNA-Based Biomonitoring Across the Globe</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00337">doi:10.3389/fevo.2019.00337</a></p>

<p><a href=None>None</a></p>
<p>DNA-based methodology has proven to be a vital tool for ecosystem assessment and monitoring. Increasingly, high-throughput approaches such as DNA metabarcoding are being used to address more complex questions, including ecological network analyses through machine learning. Despite the technological advances which allow for such questions to be posed, there remains inherent limitations in studies utilizing DNA metabarcoding, referring to environmental sample type targeted, geographical coverage and lack of standardized field and laboratory procedures. Additionally, DNA reference databases are lacking information from taxa, resulting in unidentified sequences, and underrepresentation of some taxa. These issues need to be addressed to enable a more representative approach to ecosystem monitoring to allow for detection and monitoring of global ecosystem change.</p><h1>Casting a broader net: Using microfluidic metagenomics to capture aquatic biodiversity data from diverse taxonomic targets</h1>
<p><a href="https://doi.org/10.1002/edn3.26">doi:10.1002/edn3.26</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) assays for single- and multi-species detection show promise for providing standardized assessment methods for diverse taxa, but techniques for evaluating multiple taxonomically divergent assemblages are in their infancy. We evaluated whether microfluidic multiplex metabarcoding on the Fluidigm Access Array™ platform and high-throughput sequencing could identify diverse stream and riparian assemblages from 48 taxon-general and taxon-specific metabarcode primers. eDNA screening was paired with electrofishing along a stream continuum to evaluate congruence between methods. A fish hatchery located midway along the stream continuum provided a dispersal barrier, and a point source for non-native White Sturgeon (Acipencer transmontanus). Microfluidic metabarcoding had 87% accuracy with respect to electrofishing and detected all 13 species electrofishing observed. Taxon-specific barcoding primers were more successful than taxon-general universal metabarcoding primers at classifying sequences to species. Both types of markers detected a transition from downstream sites dominated by multiple fish species, to upstream sites dominated by a single species; however, we failed to detect a complementary transition in amphibian occupancy. White Sturgeon was only detected at the hatchery outflow, indicating eDNA transport was not detectable ~2.4 km from its source. Overall, we identified 878 predicted taxa. Most sequences (50.1%) derived from fish (Actinopteri, Petromyzontidae), oomycetes (21.3%), arthropoda (classes Insecta, Decapoda; 16.5%), and apicomplexan parasites (3.8%). Taxa accounting for ~1% or less of sequences included freshwater red algae, diatoms, amphibians, and beaver. Our work shows that microfluidic metabarcoding can survey multiple phyla per assay, providing fine discrimination required to resolve closely related species, and enable data-driven prioritization for multiple forest health objectives.</p><h1>Establishment and Optimization of The Edna Preservation Method for Zhoushan Coastal Waters</h1>
<p><a href="https://doi.org/10.11693/hyhz20190200034">doi:10.11693/hyhz20190200034</a></p>

<p><a href=None>None</a></p>
<p>In recent years, environmental DNA (eDNA) has been widely used in the research of aquatic biodiversity. We use absolute quantification and high-throughput sequencing to establish and optimize the preservation methods of eDNA for Zhoushan coastal waters. Three eDNA preservation methods were tested and compared, including freeze at -20°C, in alcohol at room temperature, and in alcohol at -20°C. The results show that the yield of preservation in alcohol at -20°C was 0.78-1.06 (7d), 0.89-1.80 (15d), 1.05-2.75 (30d), and 2.69 (60d) times higher than that of preservation in alcohol at room temperature. However, both alcohol methods have problems of eDNA enrichment material leaking and filter membrane adhesion. The yield of freeze at -20°C in short term was 1.25-1.59 (7d) and 1.07-1.20 (15d) times higher than that in alcohol. The degradation rate of eDNA in alcohol was lower than that in freeze. The yield of samples preserved in alcohol was 1.99-2.10 times (30d) and 2.84-7.64 (60d) times higher than that of samples preserved in freeze. In addition, the yield of eDNA was significantly increased by secondary filtration. The concentration of enriched group increased the yield to 1.60-4.95 (alcohol at -20°C) and 1.21-2.04 (alcohol at room temperature) times higher than that of unenriched group. We find that samples preserved in cold alcohol were better than those preserved with freeze in many aspects, including the total tags of high-throughput sequencing, the abundance of fish species, and the diversity index. The eDNA degradation rate of a micro-organism is slower than that of macro-organism. The optimal preservation methods of animals of a different kingdom might be different. The optimal preservation for eDNA collected from Zhoushan coastal waters was established first time, which provided a reference for eDNA preservation in similar water region.</p><h1>DNA metabarcoding in diet studies: Unveiling ecological aspects in aquatic and terrestrial ecosystems</h1>
<p><a href="https://doi.org/10.1002/edn3.27">doi:10.1002/edn3.27</a></p>

<p><a href=None>None</a></p>
<p>Effective conservation of species and ecosystems requires the understanding of important ecological traits, such as dietary habits, food webs, and trophic niches. In diet studies, the visual identification of partially digested prey has been enhanced with the recent more powerful and accurate technique, DNA barcoding. Here, we summarize the contribution of this recent methodology to the investigation of both terrestrial and aquatic taxa diet, and compare the level of novelty uncovered through the use of this technique regarding species' ecology. From a total of 150 studies analyzed, focusing on more than 250 vertebrate wild species, seven domesticated taxa, and humans, we suggest that barcoding has led to more significant findings for aquatic taxa and ecosystems, where direct observations of feeding events and consequent trophic niche understanding are typically limited. Finally, we introduce the term dietary DNA (dDNA) to describe environmental approaches that use DNA extracted from gut, stomach, or fecal contents, aiming to assess both species dietary habits and describe local biodiversity. Particularly, we highlight the complementarity of environmental DNA (eDNA) and dDNA as a new tool for biodiversity assessments in remote areas, including most of the aquatic realm.</p><h1>A review on the applications and recent advances in environmental DNA (eDNA) metagenomics</h1>
<p><a href="https://doi.org/10.1007/s11157-019-09501-4">doi:10.1007/s11157-019-09501-4</a></p>

<p><a href=None>None</a></p>
<p>Progressions accomplished in the field of molecular biology amid the most recent decade has changed the way in which we contemplate biodiversity on earth. Together with the accessibility of new molecular techniques like high-throughput sequencing (HTS), a prompt metamorphosis was attained in understanding the behaviour and biodiversity patterns of biological systems at a level never before possible. The DNA gathered from different environmental samples (named as environmental DNA or eDNA) when coupled with HTS offers a powerful tool by empowering the census of individual species on a global scale in real time. The applications of eDNA are transpiring in different domains, for example, trophic and community ecology (functional diversity, ecosystem dynamics and prey–predator interactions), biomonitoring, conservation biology (single and multi species detection, abundance estimates), invasion biology (early species detection, passive surveillance) and environmental assessment (detection of anthropogenic contamination, microbial source tracking). However, more empirical data is required to standardize the specific sampling procedures to achieve in the best possible way. Although the application of eDNA is intensifying swiftly at a global scale, there are still some knowledge gaps, especially with methods and applications. These procedures require some refinements and validations to diminish the burden of false positives/negatives. Considering these impediments, we mainly concentrated in pooling together the most recent outcome of research articles (2008–2019) available in eDNA analysis that converse about diverse ecosystems (freshwater, marine and terrestrial habitats). We likewise discussed developments and limitations that are generally concerned with eDNA exercise in the present review.</p><h1>Anacapa Toolkit: An environmental DNA toolkit for processing multilocus metabarcode datasets</h1>
<p><a href="https://doi.org/10.1111/2041-210X.13214">doi:10.1111/2041-210X.13214</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is a promising method to monitor species and community diversity that is rapid, affordable and non-invasive. The longstanding needs of the eDNA community are modular informatics tools, comprehensive and customizable reference databases, flexibility across high-throughput sequencing platforms, fast multilocus metabarcode processing and accurate taxonomic assignment. Improvements in bioinformatics tools make addressing each of these demands within a single toolkit a reality. The new modular metabarcode sequence toolkit Anacapa (https://github.com/limey-bean/Anacapa/) addresses the above needs, allowing users to build comprehensive reference databases and assign taxonomy to raw multilocus metabarcode sequence data. A novel aspect of Anacapa is its database building module, “Creating Reference libraries Using eXisting tools” (CRUX), which generates comprehensive reference databases for specific user-defined metabarcoding loci. The Quality Control and ASV Parsing module sorts and processes multiple metabarcoding loci and processes merged, unmerged and unpaired reads maximizing recovered diversity. DADA2 then detects amplicon sequence variants (ASVs) and the Anacapa Classifier module aligns these ASVs to CRUX-generated reference databases using Bowtie2. Lastly, taxonomy is assigned to ASVs with confidence scores using a Bayesian Lowest Common Ancestor (BLCA) method. The Anacapa Toolkit also includes an r package, ranacapa, for automated results exploration through standard biodiversity statistical analysis. Benchmarking tests verify that the Anacapa Toolkit effectively and efficiently generates comprehensive reference databases that capture taxonomic diversity, and can assign taxonomy to both MiSeq and HiSeq-length sequence data. We demonstrate the value of the Anacapa Toolkit in assigning taxonomy to seawater eDNA samples collected in southern California. The Anacapa Toolkit improves the functionality of eDNA and streamlines biodiversity assessment and management by generating metabarcode specific databases, processing multilocus data, retaining a larger proportion of sequencing reads and expanding non-traditional eDNA targets. All the components of the Anacapa Toolkit are open and available in a virtual container to ease installation.</p><h1>Comparison and validation of Oomycetes metabarcoding primers for Phytophthora high throughput sequencing</h1>
<p><a href="https://doi.org/10.1007/s42161-019-00276-9">doi:10.1007/s42161-019-00276-9</a></p>

<p><a href=None>None</a></p>
<p>Oomycetes are eukaryotic plant pathogens that require health monitoring. High-throughput sequencing (HTS) methods replace progressively cultivation-based approaches in soil surveys of Oomycetes, but very little control has been done from synthetic communities. Indeed, several potential biases do exist and need to be assessed for Oomycetes communities. We created a mock community by mixing DNA from 24 Phytophthora species. We amplified two barcode regions with Oomycete-specific primers before HTS. With this aim, we used three primer sets in nested PCR amplification, targeting the ITS-1 region or the RAS gene region. The three nested PCR strategies proved to be a reliable qualitative approach, identifying approximately 95% of the species after Illumina Miseq sequencing and bioinformatic analysis. However, quantitative proportions of each species showed distortions compared to the original mixture of the mock. In addition, we compared the two ITS primer sets on soil environmental DNA sampled from temperate forests. The ‘oom18S-ITS7/18ph2f-5.8S-1R’ primer set, more specific to Phytophthora, was able to detect seven Phytophthora species, confirming what was expected for temperate forests. Using the ‘DC6-ITS7/oom18S-ITS7’ primer set that covers the broader Peronosporaceans, we detected only one Phytophthora species among the dominance of Pythium and Phytopythium species. We concluded that ‘oom18S-ITS7/18ph2f-5.8S-1R’ primer set is a reliable tool for the qualitative description of environmental Phytophthora communities.</p><h1>RpoB, a promising marker for analyzing the diversity of bacterial communities by amplicon sequencing</h1>
<p><a href="https://doi.org/10.1186/s12866-019-1546-z">doi:10.1186/s12866-019-1546-z</a></p>

<p><a href=None>None</a></p>
<p>Background: Microbiome composition is frequently studied by the amplification and high-throughput sequencing of specific molecular markers (metabarcoding). Various hypervariable regions of the 16S rRNA gene are classically used to estimate bacterial diversity, but other universal bacterial markers with a finer taxonomic resolution could be employed. We compared specificity and sensitivity between a portion of the rpoB gene and the V3 V4 hypervariable region of the 16S rRNA gene. Results: We first designed universal primers for rpoB suitable for use with Illumina sequencing-based technology and constructed a reference rpoB database of 45,000 sequences. The rpoB and V3 V4 markers were amplified and sequenced from (i) a mock community of 19 bacterial strains from both Gram-negative and Gram-positive lineages; (ii) bacterial assemblages associated with entomopathogenic nematodes. In metabarcoding analyses of mock communities with two analytical pipelines (FROGS and DADA2), the estimated diversity captured with the rpoB marker resembled the expected composition of these mock communities more closely than that captured with V3 V4. The rpoB marker had a higher level of taxonomic affiliation, a higher sensitivity (detection of all the species present in the mock communities), and a higher specificity (low rates of spurious OTU detection) than V3 V4. We compared the performance of the rpoB and V3 V4 markers in an animal ecosystem model, the infective juveniles of the entomopathogenic nematode Steinernema glaseri carrying the symbiotic bacteria Xenorhabdus poinarii. Both markers showed the bacterial community associated with this nematode to be of low diversity (< 50 OTUs), but only rpoB reliably detected the symbiotic bacterium X. poinarii. Conclusions: Our results confirm that different microbiota composition data may be obtained with different markers. We found that rpoB was a highly appropriate marker for assessing the taxonomic structure of mock communities and the nematode microbiota. Further studies on other ecosystems should be considered to evaluate the universal usefulness of the rpoB marker. Our data highlight two crucial elements that should be taken into account to ensure more reliable and accurate descriptions of microbial diversity in high-throughput amplicon sequencing analyses: i) the need to include mock communities as controls; ii) the advantages of using a multigenic approach including at least one housekeeping gene (rpoB is a good candidate) and one variable region of the 16S rRNA gene. This study will be useful to the growing scientific community describing bacterial communities by metabarcoding in diverse ecosystems.</p><h1>Diversity of endosymbiotic Symbiodiniaceae in giant clams at Dongsha Atoll, northern South China Sea</h1>
<p><a href="https://doi.org/10.1007/s13199-019-00615-5">doi:10.1007/s13199-019-00615-5</a></p>

<p><a href=None>None</a></p>
<p>Despite the importance of endosymbiotic dinoflagellates in the ecology of giant clams, the diversity and distribution of Symbiodiniaceae in different tridacnine species remain relatively poorly studied. Using a DNA metabarcoding approach based on the nuclear ribosomal ITS2 marker, this study examined the patterns of Symbiodiniaceae diversity and composition in two giant clam species, Tridacna maxima (n = 32) and Tridacna noae (n = 41) found at Dongsha Atoll, the largest atoll in the northern South China Sea. Both species of giant clams hosted Symbiodiniaceae from genera Symbiodinium (formerly Clade A), Cladocopium (formerly Clade C) and Durusdinium (formerly Clade D). Tridacna maxima harboured Cladocopium preferentially, followed by Symbiodinium and Durusdinium, while T. noae hosted Durusdinium most abundantly, followed by Symbiodinium and Cladocopium. Endosymbiont diversity also varied between host species—T. maxima contained 11 species while T. noae had 13 species. Among the endosymbionts, Cladocopium goreaui (ITS2 type C1) was most common in both host species. Further analyses revealed that endosymbiont species richness was influenced primarily by depth, size, and, to some extent, geographic locality of giant clams. Endosymbiont community structure was significantly different between host species and this variation was primarily driven by depth. Even though both tridacnine species share similar habitats on coral reefs, the contrasting diversity and composition of Symbiodiniaceae present in each species may underlie the host’s adaptability to micro- and macro-environmental changes. These results not only provide a baseline of the various endosymbionts occurring in giant clams on an isolated reef ecosystem, they provide useful information for predicting impacts on these host species that could arise due to climate-related environmental stressors.</p><h1>Global molecular variation of Paramecium jenningsi complex (Ciliophora, Protista): a starting point for further, detailed biogeography surveys</h1>
<p><a href="https://doi.org/10.1080/14772000.2019.1643424">doi:10.1080/14772000.2019.1643424</a></p>

<p><a href=None>None</a></p>
<p>The world’s second stand of P. primjenningsi (Paramecium jenningsi complex, Ciliophora, Protista), heretofore known only from India, has been revealed in Ethiopia (Africa). This finding has enlarged the range of this cryptic species and was a trigger to re-analyse the distribution of all members of the complex (known from ∼20 tropical locations). The current survey is an initial one, where, based on haplotype networks, a detailed analysis of the relationship within the P. jenningsi complex has been performed. Although the V4 hypervariable fragment of the SSU rDNA gene is widely used as a first-step barcode marker for microbial HTS analyses, it has provided inconclusive results based on the dataset investigated. However, the ITS1-5.8S-ITS2-5’rDNA and COI mtDNA fragments indicate the possibility of the delimitation of the cryptic species of P. jenningsi (which is crucial from the point of view of metabarcoding surveys). We suppose that future sampling of unexplored, tropical regions will certainly change our knowledge about Paramecium biodiversity and biogeography. This sampling will probably rely on the integration of metabarcodes from environmental DNA studies, with molecular data obtained from identified representatives of particular cryptic species.</p><h1>High-resolution biomonitoring of plant pathogens and plant species using metabarcoding of pollen pellet contents collected from a honey bee hive</h1>
<p><a href="https://doi.org/10.1002/edn3.17">doi:10.1002/edn3.17</a></p>

<p><a href=None>None</a></p>
<p>The Canadian beekeeping industry is spread across the country, with the greatest proportion of managed honey bee colonies occurring in the Prairie Provinces. Nationally, the number of beekeepers has recently been trending upwards. Simultaneously, agronomic and environmental plant pest incidents are increasing due to a number of factors, including the introduction of exotic organisms through international trade, which is a major pathway for the introduction of potentially invasive alien species and quarantine pests. Therefore, regulatory agencies are interested in developing high-throughput tools to achieve earlier detection of unwanted species in order to expedite application of mitigating measures to limit the impacts of their introduction. This study evaluates the potential of pollen pellet contents collected by honey bees to monitor plant pests using metabarcoding, a high-throughput sequencing (HTS) approach for monitoring complex environmental samples. The study used the ITS1 intergenic region to target oomycetes and fungi, the ATP9-NAD9 spacer to specifically target Phytophthora species, and the ITS2 region to target plant species. From the HTS results, a number of plants that were detected corresponded to known hosts of certain pathogens or species closely related to potentially invasive plant species. Genera including phytopathogenic species found in the pollen samples comprised Fusarium sp., Ophiostoma sp., Peronospora sp., Phytophthora sp., and Pythium sp. Correlations, high entropy, and co-occurrences between certain plants and oomycetes or fungi were observed. The potential for using honey bee-collected pollen pellets to study phytopathogens in a given environment is demonstrated here, and this concept could represent a promising complementary tool for the surveillance of phytopathogens or unwanted plants with previously described air and insect sampling methods if the protocol was applied with additional genetic markers.</p><h1>Shifting up a gear with iDNA: From mammal detection events to standardised surveys</h1>
<p><a href="https://doi.org/10.1111/1365-2664.13411">doi:10.1111/1365-2664.13411</a></p>

<p><a href=None>None</a></p>
<p>Invertebrate-derived DNA (iDNA), in combination with high throughput sequencing, has been proposed as a cost-efficient and powerful tool to survey vertebrate species. Previous studies, however, have only provided evidence that vertebrates can be detected using iDNA, but have not taken the next step of placing these detection events within a statistical framework that allows for robust biodiversity assessments. Here, we compare concurrent iDNA and camera-trap surveys. Leeches were repeatedly collected in close vicinity to 64 camera-trap stations in Sabah, Malaysian Borneo. We analyse iDNA-derived mammalian detection events in a modern occupancy model that accounts for imperfect detection and compare the results with those from occupancy models parameterised with camera-trap-derived detection events. We also combine leech-iDNA and camera-trap data in a single occupancy model. We found consistent estimates of occupancy probabilities produced by our camera-trap and leech datasets. This indicates that the metabarcoding of leech-iDNA method provides reasonable estimates of occupancy and may be a suitable method for studying and monitoring mammal species in tropical rainforests. However, we also show that a more extensive collection of leeches would be needed to assess mammal biodiversity with a robustness similar to that of camera traps. As certain taxa were only detected in leeches, we see great potential in complementing camera-trap studies with the iDNA approach, as long as the collection of leeches follows a robust and standardised sampling scheme. Synthesis and applications. Here, we describe an approach to analyse detection records of mammals derived from leech samples using an occupancy framework that accounts for leech-specific factors influencing the detection probability. We further combined camera trap and leech data, which lead to increased confidence in occupancy estimates. Our approach is not restricted to the processing of leech samples, but can be used for the analysis of other invertebrate DNA and environmental DNA data. Our study is the first step to shift the application of invertebrate DNA studies from opportunistic ad-hoc collections to the systematic surveys required for long-term management of wildlife populations.</p><h1>Substrate and spatial variables are major determinants of fungal community in karst caves in Southwest China</h1>
<p><a href="https://doi.org/10.1111/jbi.13594">doi:10.1111/jbi.13594</a></p>

<p><a href=None>None</a></p>
<p>Aim: Previous studies primarily based on the culture method have revealed that caves host a high fungal diversity and that exogenous carbon sources influence fungal communities. Here, we assess the fungal diversity and distribution patterns in karst caves in Southwest China and reveal the major factors that shape the fungal communities. Location: Southwest China. Methods: Samples of air, rock, sediment and water were collected from eight caves in Southwest China. Using high-throughput sequencing of the internal transcribed spacer 1 region of fungal rDNA, we examined the cave fungal communities in Southwest China and studied their correlations with potentially influential factors, such as substrate, cave, location and environmental variables. Results: Approximately 10,000 fungal operational taxonomic units (OTUs) spanning 15 phyla were obtained. The highest diversity genus was Penicillium, while Mortierella had the highest relative abundance. Compared with air and rock samples, sediment and water samples harboured higher diversity and distinct communities. Substrate and cave were revealed to be the key determinants of the fungal communities in caves. Sample location and sampling distance from the entrance had more significant impacts on rock and sediment communities than on air and water assemblages. The fungal assemblages in air and water samples from the cave and outside cave environments were similar, while those of rock and sediment samples were significantly different. Main conclusions: Our results indicate that sediment and water harboured higher fungal diversity than air and rock, and substrate and spatial variables are key determinants of fungal communities in caves.</p><h1>Protist Biodiversity and Biogeography in Lakes From Four Brazilian River–Floodplain Systems</h1>
<p><a href="https://doi.org/10.1111/jeu.12703">doi:10.1111/jeu.12703</a></p>

<p><a href=None>None</a></p>
<p>The biodiversity and biogeography of protists inhabiting many ecosystems have been intensely studied using different sequencing approaches, but tropical ecosystems are relatively under-studied. Here, we sampled planktonic waters from 32 lakes associated with four different river–floodplains systems in Brazil, and sequenced the DNA using a metabarcoding approach with general eukaryotic primers. The lakes were dominated by the largely free-living Discoba (mostly the Euglenida), Ciliophora, and Ochrophyta. There was low community similarity between lakes even within the same river–floodplain. The protists inhabiting these floodplain systems comprise part of the large and relatively undiscovered diversity in the tropics.</p><h1>Molecular versus morphological data for benthic diatoms biomonitoring in Northern Europe freshwater and consequences for ecological status</h1>
<p><a href="https://doi.org/10.3897/mbmg.3.34002">doi:10.3897/mbmg.3.34002</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are known to be efficient bioindicators for water quality assessment because of their rapid response to environmental pressures and their omnipresence in water bodies. The identification of benthic diatoms communities in the biofilm, coupled with quality indices such as the Indice de polluosensibilité spécifique (IPS) can be used for biomonitoring purposes in freshwater. However, the morphological identification and counting of diatoms species under the microscope is time-consuming and requires extensive expertise to deal with a constantly evolving taxonomy. In response, a molecular-based and potentially more cost-effective method has been developed, coupling high-throughput sequencing and DNA metabarcoding. The method has already been tested for water quality assessment with diatoms in Central Europe. In this study, we applied both the traditional and molecular methods on 180 biofilms samples from Northern Europe (rivers and lakes of Fennoscandia and Iceland). The DNA metabarcoding data were obtained on two different DNA markers, the 18S-V4 and rbcL barcodes, with the NucleoSpin Soil kit for DNA extraction and sequenced on an Ion Torrent PGM platform. We assessed the ability of the molecular method to produce species inventories, IPS scores and ecological status class comparable to the ones generated by the traditional morphology-based approach. The two methods generated correlated but significantly different IPS scores and ecological status assessment. The observed deviations are explained by presence/absence and abundance discrepancies in the species inventories, mainly due to the incompleteness of the barcodes reference databases, primer bias and strictness of the bioinformatic pipeline. Abundance discrepancies are less common than presence/absence discrepancies but have a greater effect on the ecological assessment. Missing species in the reference databases are mostly acidophilic benthic diatoms species, typical of the low pH waters of Northern Europe. The two different DNA markers also generated significantly different ecological status assessments. The use of the 18S-V4 marker generates more species inventories discrepancies, but achieves an ecological assessment more similar to the traditional morphology-based method. Further development of the metabarcoding method is needed for its use in environmental assessment. For its application in Northern Europe, completion and curation of reference databases are necessary, as well as evaluation of the currently available bioinformatics pipelines. New indices, fitted for environmental biomonitoring, should also be developed directly from molecular data.</p><h1>Genomic and microscopic analysis of ballast water in the great lakes region</h1>
<p><a href="https://doi.org/10.3390/app9122441">doi:10.3390/app9122441</a></p>

<p><a href=None>None</a></p>
<p>Invasive aquatic species can have damaging effects on fisheries and aquaculture through significant, and irreversible, effects on biodiversity. Human health may also be affected. To combat this threat the International Maritime Organization (IMO) Convention for BallastWater and Sediments (BWMC) came into force in September 2017. U.S. Federal and IMO ballast water standards for discharged organisms stipulate discharge limits for different size classes of organisms. Several studies including recent trials aboard Great Lakes freighters have shown that many phytoplankton found in ballast water do not fall into the regulated 10-50 μM size class. Such issues illustrate the need for new methods of assessing microorganism populations that will supersede laborious microscopy requiring rare technical expertise. Recent progress has been made in the use of DNA (deoxyribose nucleic acid)-based methods as a means of identifying the appearance of invasive species in aquatic environments. A significant advance has been the development of high throughput sequencing (HTS), which has expanded DNA barcoding, relating to an individual organism, into second generation sequencing (metabarcoding), capable of mapping whole populations of organisms in an environmental sample. Several recent studies of HTS in ships' ballast water, have shown that the technique has the capacity for detecting potentially harmful taxonomic groups and is capable of differentiating among water from different sources. The current study was undertaken to investigate the suitability (or otherwise) of HTS as a tool for ballast water management. Possible applications include improved risk assessment relating to invasive species. Feasibility for indicative testing for ballast water treatment effcacy was also addressed. However, pending analysis of treated samples, the current study was confined to a comparison of HTS and microscope counts in untreated samples. A correlation of visual and molecular taxonomic assignments of microorganisms found in the ballast water from different ports and during different seasons indicated that such a comparison was best conducted at Family level, although Principal Components Analysis showed that the two methods differed qualitatively among major taxonomic groups.</p><h1>Comparison of DNA metabarcoding and morphological identification for stream macroinvertebrate biodiversity assessment and monitoring</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2019.02.008">doi:10.1016/j.ecolind.2019.02.008</a></p>

<p><a href=None>None</a></p>
<p>Conventional morphology-based identification is commonly used for routine assessment of freshwater ecosystems. However, cost and time efficient techniques such as high-throughput sequencing (HTS) based approaches may resolve the constraints encountered in conducting morphology-based surveys. Here, we characterized stream macroinvertebrate species diversity and community composition via metabarcoding and morphological analysis from environmental samples collected from the Shigenobu River Basin in Ehime Prefecture, Japan. We compared diversity metrics and assessed both approaches’ ability to evaluate the relationship between macroinvertebrate community and environmental variables. In total, we morphologically identified 45 taxa (3 families, six subfamilies, 31 genera, and five species) from 8,276 collected individuals from ten study sites. We detected 44 species by metabarcoding, with 35 species collapsed into 11 groups matching the morphologically identified taxa. A significant positive correlation between logged depth (number of HTS reads) and abundance of morphological taxa was observed, which implied that quantitative data could be used for subsequent analyses. We recovered a considerably high rate of relative abundance detection of the morphologically identified samples. Recovery of samples by incidence or presence/absence count were considerably low, with a high rate of false-negative detection specifically for species with scarce representation in the community sample. Given the low taxonomic resolution of the morphological assignment in this study, we report that metabarcoding does not reflect the majority of the species naturally occurring in our site, which could further be proven by performing refined morphological assessment of the samples. However, abundance-based detection proved to be efficient with 92% of the individuals correctly demonstrated. We conclude that DNA metabarcoding provides a practical and cost-effective approach especially for rapid biological monitoring of freshwater macroinvertebrate communities, but further improvements in the detection of scarce samples should be considered to increase the sensitivity of detecting most, if not all, of the species present in the environment.</p><h1>Spatial variations in community structure of haptophytes across the Kuroshio front in the Tokara Strait</h1>
<p><a href="https://doi.org/10.1002/9781119428428.ch13">doi:10.1002/9781119428428.ch13</a></p>

<p><a href=None>None</a></p>
<p>Haptophytes are important members of the marine microbial community with high diversities in taxonomy and ecological function. However, little is known about the distribution and diversity of this algal group in and around the Kuroshio. Here, we present a fine-scale community structure and diversity of haptophytes in the surface and deep chlorophyll maximum (DCM) layers across the Kuroshio in the Tokara Strait. Phytoplankton pigment analysis revealed that the haptophytes were the most dominant phytoplankton group in the current axis and offshore side of the Kuroshio. Water samples were divided into surface and DCM layers on the basis of the community structure of haptophytes, which was estimated from high-throughput 18S rDNA metabarcoding. The redundancy analysis suggests that haptophyte community composition was constrained mainly by depth at both operational taxonomic unit (=97% sequence similarity) and genus levels. Haptophyte diversity was greater in the DCM waters than in the surface waters. Additionally, diversity, richness, and evenness indices tended to increase around the Kuroshio axis, possibly due to the advection of haptophyte taxa from the upstream region. Our results underscore the importance of physical properties such as light intensity and advection and diffusion of water mass on regulating haptophyte community structure in Kuroshio ecosystems.</p><h1>An efficient and robust laboratory workflow and tetrapod database for larger scale environmental DNA studies</h1>
<p><a href="https://doi.org/10.1093/gigascience/giz029">doi:10.1093/gigascience/giz029</a></p>

<p><a href=None>None</a></p>
<p>The use of environmental DNA for species detection via metabarcoding is growing rapidly. We present a co-designed lab workflow and bioinformatic pipeline to mitigate the 2 most important risks of environmental DNA use: sample contamination and taxonomic misassignment. These risks arise from the need for polymerase chain reaction (PCR) amplification to detect the trace amounts of DNA combined with the necessity of using short target regions due to DNA degradation. Findings: Our high-throughput workflow minimizes these risks via a 4-step strategy: (i) technical replication with 2 PCR replicates and 2 extraction replicates; (ii) using multi-markers (12S,16S,CytB); (iii) a "twin-tagging," 2-step PCR protocol; and (iv) use of the probabilistic taxonomic assignment method PROTAX, which can account for incomplete reference databases. Because annotation errors in the reference sequences can result in taxonomic misassignment, we supply a protocol for curating sequence datasets. For some taxonomic groups and some markers, curation resulted in >50% of sequences being deleted from public reference databases, owing to (i) limited overlap between our target amplicon and reference sequences, (ii) mislabelling of reference sequences, and (iii) redundancy. Finally, we provide a bioinformatic pipeline to process amplicons and conduct PROTAX assignment and tested it on an invertebrate-derived DNA dataset from 1,532 leeches from Sabah, Malaysia. Twin-tagging allowed us to detect and exclude sequences with non-matching tags. The smallest DNA fragment (16S) amplified most frequently for all samples but was less powerful for discriminating at species rank. Using a stringent and lax acceptance criterion we found 162 (stringent) and 190 (lax) vertebrate detections of 95 (stringent) and 109 (lax) leech samples. Conclusions: Our metabarcoding workflow should help research groups increase the robustness of their results and therefore facilitate wider use of environmental and invertebrate-derived DNA, which is turning into a valuable source of ecological and conservation information on tetrapods.</p><h1>Influence of environmental factors on spatial–temporal distribution patterns of dinoflagellate cyst communities in the South China Sea</h1>
<p><a href="https://doi.org/10.1007/s12526-018-0850-4">doi:10.1007/s12526-018-0850-4</a></p>

<p><a href=None>None</a></p>
<p>The spatial–temporal distribution of dinoflagellate cyst (i.e., dinocyst) communities is crucial for understanding the detailed mechanisms of recurrence and spread of harmful algae blooms in marine ecosystems. Here, we employed the newly developed high-throughput sequencing-based metabarcoding to characterize dinocyst communities collected from the South China Sea. Further, we clarified the spatial–temporal distribution patterns and analyzed the correlation between environmental factors and the observed patterns to investigate how they are influenced by environmental factors. Our results showed that the spatial distribution of dinocyst species richness and abundance varied greatly between sampling sites in different seasons. Both redundancy and Pearson analyses showed that the chemical oxygen femand, which could explained 35.0% of the total community variation, had positive correlations with heterotrophic dinocyst richness and negative correlations with autotrophic dinocysts richness. We did not find significant correlations between heavy metals and any features of dinocyst species richness. No environmental factor showed significant effects on dinocyst abundance based on forward selection after excluding colinearity; however, Pearson’s correlation analyses showed that the abundance of heterotrophic dinocysts presented a significant positive correlation with Mn (P < 0.05). Our results showed that the influence of environmental factors on spatial–temporal distribution of dinocyst species could be region- and/or environment-specific. Consequently, we suggest that detailed investigations should be performed to clarify the influence of varied environmental factors on dinocyst community characteristics in different regions and/or seasons.</p><h1>Is targeted community DNA metabarcoding suitable for biodiversity inventories of lichen-forming fungi?</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2018.11.061">doi:10.1016/j.ecolind.2018.11.061</a></p>

<p><a href=None>None</a></p>
<p>Comprehensive lichen inventories at biomonitoring reference sites provide valuable ecological information but are notoriously difficult to obtain. Due to the limited number of specialists and challenges with generating objective species inventories, investigating alternative or complementary strategies for compiling lichen inventories is paramount to improve biomonitoring strategies. The aim of this study was to determine if targeted community DNA metabarcoding provides an efficient, objective and reliable approach for characterizing lichen-forming fungal diversity in biomonitoring research. To evaluate if reliable biodiversity data could be obtained efficiently and objectively, metacommunity samples were collected by trained field technicians and a professional lichenologist at a previously established biomonitoring reference site in east-central Nevada, USA. Using amplicon-based high-throughput sequencing of a portion of the fungal DNA barcoding marker, the internal transcribed spacer region, molecular operational taxonomic units (OTUs) were inferred and used as a proxy for species diversity. OTUs from individual sampling efforts and two DNA extraction methods were compared to data from the original, voucher-based inventory at the same site. Treating OTUs as a proxy for species diversity, a 1.6-fold increase in diversity of lichen-forming fungi was observed when comparing the targeted community DNA metabarcoding to the initial, voucher-based inventories – 72 OTUs vs. 43 species. Strikingly, substantial overlap in captured OTU diversity was observed between minimally trained technicians and a professional lichenologist metacommunity samples. Based on these results, targeted community DNA metabarcoding is a promising strategy for creating objective lichen inventories. However, metabarcoding cannot discriminate the presence of established thalli from that of spores or propagules (in absence of established thalli) present in the sample/area. Therefore, targeted community DNA metabarcoding for assessing lichen biodiversity offers a complementary, rather than a substitutive sampling strategy.</p><h1>Scaling up DNA metabarcoding for freshwater macrozoobenthos monitoring</h1>
<p><a href="https://doi.org/10.1111/fwb.13220">doi:10.1111/fwb.13220</a></p>

<p><a href=None>None</a></p>
<p>The viability of DNA metabarcoding for assessment of freshwater macrozoobenthos has been demonstrated over the recent years. The method has matured to a stage where it can be applied to monitoring at a large scale, keeping pace with increased high-throughput sequencing capacity. However, workflows and sample tagging need to be optimised to accommodate for hundreds of samples within a single sequencing run. Here, we conceptualise a streamlined metabarcoding workflow, in which samples are processed in 96-well plates. Each sample is replicated starting with tissue extraction. Negative and positive controls are included to ensure data reliability. With our newly developed fusion primer sets for the BF2 + BR2 primer pair up to three 96-well plates (288 wells) can be uniquely tagged for a single Illumina sequencing run. By including Illumina indices, tagging can be extended to thousands of samples. We hope that our metabarcoding workflow will be used as a practical guide for future large-scale biodiversity assessments involving freshwater invertebrates. However, as this is just one possible metabarcoding approach, we hope this article will stimulate discussion and publication of alternatives and extensions to this method.</p><h1>The critical importance of rhodoliths in the life cycle completion of both macro- and microalgae, and as holobionts for the establishment and maintenance of marine biodiversity</h1>
<p><a href="https://doi.org/10.3389/fmars.2018.00502">doi:10.3389/fmars.2018.00502</a></p>

<p><a href=None>None</a></p>
<p>Rhodoliths are the main hard substrata for the attachment of benthic macroalgae in the NW Gulf of Mexico rubble habitats that are associated with salt domes, unique deep bank habitats at ~50-90 m depth on the continental shelf offshore Louisiana and Texas. With the advent of additional sequencing technologies, methodologies for biodiversity assessments are now rapidly shifting to DNA metabarcoding, i.e., High Throughput Sequencing (HTS) of environmental DNA mixtures with standardized molecular markers, such as 16S V4, for rapid, cost-effective biodiversity measurement. We newly tested 16S V4 metabarcoding on endolithic portions of mesophtic rhodoliths exhibiting low phototroph colonization that revealed a hidden, cryptic algal diversity targeting spores, propagules, and unsuspected life history stages. We explored cryo-SEM as a potentially more informative method than regular SEM to minimize artifacts of sample preparation in the study of endolithic cell inclusions which brought to light a suite of microalgal stages. We were able to differentiate floridean starch from cellular inclusions. We associated the effect of anatomical growth pattern on presence or absence of cellular inclusions in biogenic rhodoliths. Analyses of combined 16S V4 metabarcodes and 16S Sanger sequences of two red algal orders, the Halymeniales and Bonnemaisoniales, increased the established record of diversity in the region. We view rhodoliths as marine biodiversity hotspots that may function as seedbanks, temporary reservoirs for life history stages of ecologically important eukaryotic microalgae, and macroalgae or as refugia for ecosystem resilience following environmental stress.</p><h1>Advances in sampling strategies and analysis of phytoplankton</h1>
<p><a href="https://doi.org/10.1016/B978-0-12-817497-5.00031-8">doi:10.1016/B978-0-12-817497-5.00031-8</a></p>

<p><a href=None>None</a></p>
<p>Phytoplankton are microscopic, unicellular organisms that play a pivotal role as the base of food webs in aquatic systems. Depending on the econiche inhabited, the sampling strategies and procedures employed for analysis will differ. In this chapter, we discuss the sampling strategies used for water and sediment, in diverse coastal and oceanic environments. Analyses of phytoplankton with regard to taxonomy, community structure, benthic diatoms, dinoflagellate cysts, fouling diatoms, epibiotic phytoplankton, and picophytoplankton are elaborated. The application of FlowCAM and flow cytometry techniques in analysis of phytoplankton and picophytoplankton are also discussed. Analysis of phytoplankton pigments through high performance liquid chromatography; phytoplankton toxins through solid-phase adsorption toxin tracking approach; and determination of photosynthetic efficiency using the fluorescence induction and relaxation technique are also detailed. In addition, the determination of primary productivity through various isotope techniques and fast repetition rate fluorometry is examined. The application of remote sensing to determination of primary productivity and monitoring of harmful algal blooms is elucidated. The application of high throughput sequencing-based methods like DNA metabarcoding to phylogenetic analysis of phytoplankton and its implications in phytoplankton taxonomy is also highlighted.</p><h1>Metabarcoding on planktonic larval stages: An efficient approach for detecting and investigating life cycle dynamics of benthic aliens</h1>
<p><a href="https://doi.org/10.3391/mbi.2019.10.4.06">doi:10.3391/mbi.2019.10.4.06</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing (HTS) technologies offer new promise to support surveillance programs targeting marine non-indigenous species (NIS). Metabarcoding might surpass traditional monitoring methods, for example through its ability to detect rare species, a key feature in early detection of NIS. Another interest of this approach is the identification of organisms difficult to identify based on morphology only (e.g., early developmental stages), making it relevant in the context of management programs. Because many marine benthic NIS have a biphasic bentho-pelagic life cycle, targeting their pelagic larval stages in zooplankton may allow early detection and assessment of their establishment and potential spread. We illustrate this approach with an analysis of bulk-DNA retrieved from a time-series of zooplankton samples collected over 22 months in one bay in Brittany (France). Using HTS of amplicons obtained with two markers (COI and 18S) and a metabarcoding approach, 12 NIS were identified and their temporal larval dynamics were monitored. Importantly, we chose to focus on a closed list of species, from four metazoan classes encompassing 52 NIS reported within the study area or nearby seas, with molecular references available or obtained locally for 42 of them. The use of a custom-designed database allowed the detection of three NIS otherwise not detected when using public databases. Interestingly, NIS known to have a short-lived larval stage were detected (e.g., the bryozoan Bugula neritina or the tunicate Corella eumyota). For two molluscs Ruditapes philippinarum and Crepidula fornicata, metabarcoding results were compared to those obtained using traditional methods (i.e., barcoding of individual larvae and morphology, respectively) to show the reliability of the approach in detecting and assessing the extent of their reproductive periods. Our results also revealed that the Pacific oyster Crassostrea gigas, a notorious invasive species, failed to reproduce in the study bay, showing that metabarcoding on larval stages also provides information regarding the establishment success (or failure) of NIS. While metabarcoding has its limitations and biases, this study demonstrates its effectiveness for surveillance of targeted NIS, notably to support management strategies like the European Marine Strategy Framework Directive (MSFD).</p><h1>Fast surveys and molecular diet analysis of carnivores based on fecal DNA and metabarcoding</h1>
<p><a href="https://doi.org/10.17520/biods.2018214">doi:10.17520/biods.2018214</a></p>

<p><a href=None>None</a></p>
<p>Large carnivores play an important role in the regulation of food-web structure and ecosystem functioning. However, large carnivores face serious threats that have caused declines in their populations and geographic ranges due to habitat loss and degradation, hunting, human disturbance and pathogen transmission. Conservation of large carnivore species richness and population size has become a pressing issue and an important research focus of conservation biology. The western Sichuan Plateau, located at the intersection of the mountains of southwest China and the eastern margin of the Tibetan Plateau, is a global biodiversity hotspot and has high carnivore species richness. However, increasing human activities may exacerbate the destruction of local flora and fauna, thereby threatening the survival of wild carnivores. Information on species composition and dietary habits can improve our understanding of the structure and function of the ecosystem and food-web relationships in the study area. In addition, species composition and dietary habits are of great significance for understanding multi-species coexistence mechanisms and preserving biodiversity. This study collected carnivore fecal samples from Xinlong and Shiqu counties in the Ganzi Tibetan Autonomous Prefecture, Sichuan Province. DNA was then extracted from the samples and the species was identified based on DNA sequences and DNA barcoding techniques. Seven carnivores were identified, including five large carnivores (Canis lupus, Ursus arctos, Panthera pardus, P. uncia and Canis lupus familiaris) and two medium and small-sized carnivores (Prionailurus bengalensis and Vulpes vulpes). Using fecal DNA, high-throughput sequencing and metabarcoding, we conducted diet analysis for the seven carnivores and found 28 different food molecular operational taxonomic units (MOTUs), including 19 mammals, eight birds and one fish species. The predominant prey categories of wolves, dogs and brown bears were ungulates. The domestic yak (Bos grunniens) was the most frequently identified prey species. Small mammals such as rodents and lagomorphs accounted for a significant proportion in the diets of leopard cats and red foxes, The most frequent prey of this category of carnivore were the Chinese scrub vole (Neodon irene) and plateau pika (Ochotona curzoniae). In addition, leopards and snow leopards mainly fed on the Chinese goral (Naemorhedus griseus) and blue sheep (Pseudois nayaur), respectively. Our study highlights the utility of fecal DNA and metabarcoding technique in fast carnivore surveys and high-throughput diet analysis, and provides a technical reference and guidance for future biodiversity surveys and food-web studies.</p><h1>Environmental DNA (eDNA)-metabarcoding-based early monitoring and warning for invasive species in aquatic ecosystems</h1>
<p><a href="https://doi.org/10.17520/biods.2018233">doi:10.17520/biods.2018233</a></p>

<p><a href=None>None</a></p>
<p>Biological invasion is a major threat to multiple ecosystems across the globe, causing severe damages to ecological integrity, loss of biodiversity, economic and social development and even human health. With the rapid development in aquaculture, shipping and aquarium and ornamental trades in the past several decades, China has become one of the countries most influenced by invasive species. Studies have clearly shown that the development and application of robust early monitoring and warning is one of the most effective ways to prevent and possibly control invasive species in aquatic ecosystems. Compared to terrestrial ecosystems, there remain several technical difficulties for developing early monitoring and warning in aquatic habitats. The technical challenges are mainly due to several features of aquatic biological communities such as high biodiversity and complex structure, a large number of microscopic species, extremely low population density and lack of available taxonomic keys for species identification. With the rapid development of high-throughput sequencing techniques, environmental DNA (eDNA)-metabarcoding has become the top priority method for developing the early monitoring and warning programs in aquatic ecosystems. In this review, we aim to synthesize the research progress on eDNA-metabarcoding and its application to early monitoring and warning of invasive species in aquatic ecosystems. In addition, we briefly discuss the technological advantages of eDNA-metabarcoding for the early monitoring and warning programs. Finally, we propose research perspectives for solutions to technical issues for false positive and false negative errors in the eDNA-metabarcoding process.</p><h1>DNA barcoding and emerging reference construction and data analysis technologies</h1>
<p><a href="https://doi.org/10.17520/biods.201820">doi:10.17520/biods.201820</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding has been growing exponentially in terms of the number of barcode generated as well as its applications, e.g. as conservation tools in: species identification for damaged specimens, diet analysis from gut content and feces, biodiversity assessment from environmental DNA (eDNA), bulk arthropod samples or invertebrate-derived DNA (iDNA). These applications often require coupling with high throughput sequencing (HTS) technologies, and when done so are referred to as metabarcoding. Here, we discuss the methods used to generate reference barcodes using cost-efficient HTS platforms, and introduce several rules-of-thumb and some widely-used tools to conduct data quality control, denoising, and Operational Taxonomic Units (OTUs) clustering. We hope this review will help readers better understand how these emerging technologies can be implemented alongside existing technologies to accelerate biodiversity assessments in an accurate and efficient way.</p><h1>Assessing the diversity and distribution of apicomplexans in host and free-living environments using high-throughput amplicon data and a phylogenetically informed reference framework</h1>
<p><a href="https://doi.org/10.3389/fmicb.2019.02373">doi:10.3389/fmicb.2019.02373</a></p>

<p><a href=None>None</a></p>
<p>Apicomplexans are a group of microbial eukaryotes that contain some of the most well-studied parasites, including the causing agents of toxoplasmosis and malaria, and emergent diseases like cryptosporidiosis or babesiosis. Decades of research have illuminated the pathogenic mechanisms, molecular biology, and genomics of model apicomplexans, but we know little about their diversity and distribution in natural environments. In this study we analyze the distribution of apicomplexans across a range of both host-associated and free-living environments. Using publicly available small subunit (SSU) rRNA gene databases, high-throughput environmental sequencing (HTES) surveys, and our own generated HTES data, we developed an apicomplexan reference database, which includes the largest apicomplexan SSU rRNA tree available to date and encompasses comprehensive sampling of this group and their closest relatives. This tree allowed us to identify and correct incongruences in the molecular identification of apicomplexan sequences. Analyzing the diversity and distribution of apicomplexans in HTES studies with this curated reference database also showed a widespread, and quantitatively important, presence of apicomplexans across a variety of free-living environments. These data allow us to describe a remarkable molecular diversity of this group compared with our current knowledge, especially when compared with that identified from described apicomplexan species. This is most striking in marine environments, where potentially the most diverse apicomplexans apparently exist, but have not yet been formally recognized. The new database will be useful for microbial ecology and epidemiological studies, and provide valuable reference for medical and veterinary diagnosis especially in cases of emerging, zoonotic, and cryptic infections.</p><h1>Global distribution and diversity of Chaetoceros (Bacillariophyta, Mediophyceae): Integration of classical and novel strategies</h1>
<p><a href="https://doi.org/10.7717/peerj.7410">doi:10.7717/peerj.7410</a></p>

<p><a href=None>None</a></p>
<p>Information on taxa distribution is a prerequisite for many research fields, and biological records are a major source of data contributing to biogeographic studies. The Global Biodiversity Information Facility (GBIF) and the Ocean Biogeographic Information System (OBIS) are important infrastructures facilitating free and open access to classical biological data from several sources in both temporal and spatial scales. Over the last ten years, high throughput sequencing (HTS) metabarcoding data have become available, which constitute a great source of detailed occurrence data. Among the global sampling projects that have contributed to such data are Tara Oceans and the Ocean Sampling Day (OSD). Integration of classical and metabarcoding data may aid a more comprehensive assessment of the geographic range of species, especially of microscopic ones such as protists. Rare, small and cryptic species are often ignored in surveys or mis-assigned with the classical approaches. Here we show how integration of data from various sources can contribute to insight in the biogeography and diversity at the genus- and species-level using Chaetoceros as study system, one of the most diverse and abundant genera among marine planktonic diatoms. Chaetoceros records were extracted from GBIF and OBIS and literature data were collected by means of a Google Scholar search. Chaetoceros references barcodes where mapped against the metabarcode datasets of Tara Oceans (210 sites) and OSD (144 sites). We compared the resolution of different data sources in determining the global distribution of the genus and provided examples, at the species level, of detection of cryptic species, endemism and cosmopolitan or restricted distributions. Our results highlighted at genus level a comparable picture from the different sources but a more complete assessment when data were integrated. Both the importance of the integration but also the challenges related to it were illustrated. Chaetoceros data collected in this study are organised and available in the form of tables and maps, providing a powerful tool and a baseline for further research in e.g., ecology, conservation and evolutionary biology.</p><h1>Interactome-based abiotic and biotic impacts on biodiversity of plankton communities in disturbed wetlands</h1>
<p><a href="https://doi.org/10.1111/ddi.12949">doi:10.1111/ddi.12949</a></p>

<p><a href=None>None</a></p>
<p>Aim: Despite great efforts for conservation, biodiversity in wetland ecosystems is still losing at an alarming rate. Thus, it is crucial to deeply understand ecological processes and mechanisms that potentially affect the loss of biodiversity. Location: Sanjiang wetland in Sanjiang floodplain, north-east China. Methods: Biological samples and environmental data were collected from five approximate habitats under different anthropogenic disturbance levels from Sanjiang wetland. Biodiversity of plankton communities (i.e., bacterioplankton and zooplankton) was characterized by high-throughput sequencing-based metabarcoding. Multidisciplinary methods including interactome-based construction of scale-free networks were employed to examine the complex biotic interactions and abiotic influence on plankton biodiversity. Results: Both environmental variables and community composition of plankton communities varied significantly among five habitats. Environmental filtering was identified as the dominant ecological process that drove the distinct community composition and geographical distribution of biodiversity. However, the significantly influential factors were different for bacterioplankton (e.g., pH and As) and zooplankton communities (e.g., NH4 and TP). In total, environmental factors could explain approximately 30% of community variation for both types of plankton communities. Based on interactome networks, more organism-associated edges were identified than environment-associated edges. Such a pattern suggests that, in addition to environmental influence, biotic interactions also play a crucial role to shape biodiversity variation. Main conclusions: Both abiotic factors and biotic interactions largely contributed to the distinct composition and geographical distribution of bacterioplankton and zooplankton communities in Sanjiang wetland. Therefore, the interactome-based investigation is recommended to examine synthesized impacts of abiotic and biotic variables, and such a comprehensive survey can largely facilitate our understanding of ecological processes and associated mechanisms that drive the loss of wetland biodiversity. Results from comprehensive surveys can help identify conservation priorities and improve the predictability of conservation programmes.</p><h1>Field evaluation of DNA based biodiversity monitoring of Caribbean mosquitoes</h1>
<p><a href="https://doi.org/10.3389/fevo.2019.00240">doi:10.3389/fevo.2019.00240</a></p>

<p><a href=None>None</a></p>
<p>Mosquito borne diseases pose a threat to human health worldwide. Disease risk is primarily determined by presence and abundance of vector species. A better understanding of mosquito diversity and abundance can direct improved vector control, but this requires a combination of monitoring techniques that yield both rapid and reliable information. Particularly improved larval detection is pivotal to move toward more targeted management with less environmental impact. Current detection methods rely strongly on manual labor and taxonomic expertise, which greatly limits the extent to which these methodologies can be employed. As such, insight in the efficiency of novel, high-throughput vs. traditional sampling techniques is required. We compared the effectiveness of a recently developed environmental DNA (eDNA) approach on water and sediment samples with other commonly used sampling techniques ("dipping" for larvae and adult trapping) in a field study on three Caribbean islands. All sampling methods were employed across a range of ecologically contrasting sites. Species identification was performed both morphologically and molecularly using an in-house developed reference database supplemented with sequences from BOLD and GenBank. Our analysis of water samples from 39 sites shows that eDNA sampling can be more reliable than dipping, yields a higher within-sample richness and produces a subset of the adult community in all sampled water types. Furthermore, for both adults and larvae, our identifications showed complete overlap between morphological and molecular approaches in 133 out of 134 samples. Overall, results from this study provide evidence that both our eDNA-based detection of larvae and our DNA-based identification of larvae and adults present methods that are, although more expensive, as reliable, and for some species even more reliable than the currently used methods. Additionally, our results highlight that a DNA approach can be used to identify larvae of early developmental stages, which generally lack important morphological characteristics. As such it allows for development of efficient disease control strategies, verification of management effectiveness and monitoring of population dynamics.</p><h1>Taxonomic and functional characterization of the microbial community during spontaneous in vitro fermentation of riesling must</h1>
<p><a href="https://doi.org/10.3389/fmicb.2019.00697">doi:10.3389/fmicb.2019.00697</a></p>

<p><a href=None>None</a></p>
<p>Although there is an extensive tradition of research into the microbes that underlie the winemaking process, much remains to be learnt. We combined the high-throughput sequencing (HTS) tools of metabarcoding and metagenomics, to characterize how microbial communities of Riesling musts sampled at four different vineyards, and their subsequent spontaneously fermented derivatives, vary. We specifically explored community variation relating to three points: (i) how microbial communities vary by vineyard; (ii) how community biodiversity changes during alcoholic fermentation; and (iii) how microbial community varies between musts that successfully complete alcoholic fermentation and those that become 'stuck' in the process. Our metabarcoding data showed a general influence of microbial composition at the vineyard level. Two of the vineyards (4 and 5) had strikingly a change in the differential abundance of Metschnikowia. We therefore additionally performed shotgun metagenomic sequencing on a subset of the samples to provide preliminary insights into the potential relevance of this observation, and used the data to both investigate functional potential and reconstruct draft genomes (bins). At these two vineyards, we also observed an increase in non-Saccharomycetaceae fungal functions, and a decrease in bacterial functions during the early fermentation stage. The binning results yielded 11 coherent bins, with both vineyards sharing the yeast bins Hanseniaspora and Saccharomyces. Read recruitment and functional analysis of this data revealed that during fermentation, a high abundance of Metschnikowia might serve as a biocontrol agent against bacteria, via a putative iron depletion pathway, and this in turn could help Saccharomyces dominate the fermentation. During alcoholic fermentation, we observed a general decrease in biodiversity in both the metabarcoding and metagenomic data. Unexpected Micrococcus behavior was observed in vineyard 4 according to metagenomic analyses based on reference-based read mapping. Analysis of open reading frames using these data showed an increase of functions assigned to class Actinobacteria in the end of fermentation. Therefore, we hypothesize that bacteria might sit-and-wait until Saccharomyces activity slows down. Complementary approaches to annotation instead of relying a single database provide more coherent information true species. Lastly, our metabarcoding data enabled us to identify a relationship between stuck fermentations and Starmerella abundance. Given that robust chemical analysis indicated that although the stuck samples contained residual glucose, all fructose had been consumed, we hypothesize that this was because fructophilic Starmerella, rather than Saccharomyces, dominated these fermentations. Overall, our results showcase the different ways in which metagenomic analyses can improve our understanding of the wine alcoholic fermentation process.</p><h1>Evaluate the pollution of aquatic environments with diatom DNA: Were do we stand?</h1>
<p><a href="https://doi.org/10.1051/tsm/201905053">doi:10.1051/tsm/201905053</a></p>

<p><a href=None>None</a></p>
<p>The assessment of the ecological status of streams relies on the calculation of quality indices based on the sensitivity of certain biological groups to pollution, including diatoms. The determination and quantification of diatom species is generally performed using morphological identification methods under microscope, which are complex, time-consuming and rapidly expensive. In the last decade, a new DNA-based method has been developed which uses genetic criteria to identify species rather than morphological criteria: metabarcoding. In combination with high-throughput sequencing technologies, DNA metabarcoding can identify all the species present in an environmental sample and process several hundred samples in parallel. These advantages make metabarcoding an interesting biomonitoring tool for assessing the ecological status of the thousands of streams sites monitored every year in France as part of the monitoring program of the Water Framework Directive. This paper presents the results of two recent studies carried out on the main island of the French overseas department of Mayotte (2013-2018) and Mainland France (2016-2018) streams to evaluate the potential of DNA metabarcoding for biomonitoring. We discuss the different methodological developments and optimizations performed to improve reliability of the taxonomic inventories of diatoms produced in metabarcoding, particularly regarding species quantification based on the relative abundance of DNA sequences. We present the results of application of the molecular approach for the assessment of the ecological status of more than 500 national streams sites, comparing them with the results obtained with the classical morphological approach. Finally, we discuss the potential application of metabarcoding in routine at the scale of stream monitoring networks, its application limits and some recommendations for a future implementation along with the morphological approach.</p><h1>Biodiversity assessments in the 21st century: The potential of insect traps to complement environmental samples for estimating eukaryotic and prokaryotic diversity using high-throughput DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1139/gen-2018-0096">doi:10.1139/gen-2018-0096</a></p>

<p><a href=None>None</a></p>
<p>The rapid loss of biodiversity, coupled with difficulties in species identification, call for innovative approaches to assess biodiversity. Insects make up a substantial proportion of extant diversity and play fundamental roles in any given ecosystem. To complement morphological species identification, new techniques such as metabarcoding make it possible to quantify insect diversity and insect-ecosystem interactions through DNA sequencing. Here we examine the potential of bulk insect samples (i.e., containing many non-sorted specimens) to assess prokaryote and eukaryote biodiversity and to complement the taxonomic coverage of soil samples. We sampled 25 sites on three continents and in various ecosystems, collecting insects with SLAM traps (Brazil) and Malaise traps (South Africa and Sweden). We then compared our diversity estimates with the results obtained with biodiversity data from soil samples from the same localities. We found a largely different taxonomic composition between the soil and insect samples, testifying to the potential of bulk insect samples to complement soil samples. Finally, we found that non-destructive DNA extraction protocols, which preserve insect specimens for morphological studies, constitute a promising choice for cost-effective biodiversity assessments. We propose that the sampling and sequencing of insect samples should become a standard complement for biodiversity studies based on environmental DNA.</p><h1>Environmental DNA and metabarcoding for the study of amphibians and reptiles: species distribution, the microbiome, and much more</h1>
<p><a href="https://doi.org/10.1163/15685381-20191194">doi:10.1163/15685381-20191194</a></p>

<p><a href=None>None</a></p>
<p>In the last decade, eDNA and metabarcoding have opened new avenues to biodiversity studies; amphibians and reptiles are animals for which these new approaches have allowed great leaps forward. Here we review different approaches through which eDNA can be used to study amphibians, reptiles and many more organisms. eDNA is often used to evaluate the presence of target species in freshwaters; it has been particularly useful to detect invasive alien amphibians and secretive or rare species, but the metabarcoding approach is increasingly used as a cost-effective approach to assess entire communities. There is growing evidence that eDNA can be also useful to study terrestrial organisms, to evaluate the relative abundance of species, and to detect reptiles. Metabarcoding has also revolutionized studies on the microbiome associated to skin and gut, clarifying the complex relationships between pathogens, microbial diversity and environmental variation. We also identify additional aspects that have received limited attention so far, but can greatly benefit from innovative applications of eDNA, such as the study of past biodiversity, diet analysis and the reconstruction of trophic interactions. Despite impressive potential, eDNA and metabarcoding also bear substantial technical and analytical complexity; we identify laboratory and analytical strategies that can improve the robustness of results. Collaboration among field biologists, ecologist, molecular biologists, and bioinformaticians is allowing fast technical and conceptual advances; multidisciplinary studies involving eDNA analyses will greatly improve our understanding of the complex relationships between organisms, and our effectiveness in assessing and preventing the impact of human activities.</p><h1>Application of DNA metabarcoding in species identification of Chinese herbal medicines</h1>
<p><a href="https://doi.org/10.19540/j.cnki.cjcmm.2019.0001">doi:10.19540/j.cnki.cjcmm.2019.0001</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding, one rapid and robust method using specific standard DNA fragments, has been widely used for rapid species identification of a bulk sample through high-throughput sequencing technologies. While it has been widely used in the studies of metagenomics, animal and plant biodiversity, it has gradually come to be used as a profitable method in species identification of mixed Chinese herbal medicines. In this paper, we mainly summarize the current studies of the application of DNA metabarcoding in species identification of mixed Chinese herbal medicines. Moreover, high-throughput sequencing technologies adopted in those studies, such as Sanger, the next-generation, and third-generation sequencing technologies, are discussed. It is conducted to provide a theoretical guidance for the application of DNA metabarcoding in species identification of mixed Chinese herbal medicines and in more other biodiversity studies.</p><h1>Monitoring riverine fish communities through eDNA metabarcoding: Determining optimal sampling strategies along an altitudinal and biodiversity gradient</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.30457">doi:10.3897/mbmg.2.30457</a></p>

<p><a href=None>None</a></p>
<p>Monitoring aquatic biodiversity through DNA extracted from environmental samples (eDNA) combined with high-throughput sequencing, commonly referred to as eDNA metabarcoding, is increasing in popularity within the scientific community. However, sampling strategies, laboratory protocols and analytical pipelines can influence the results of eDNA metabarcoding surveys. While the impact of laboratory protocols and analytical pipelines have been extensively studied, the importance of sampling strategies on eDNA metabarcoding surveys has not received the same attention. To avoid underestimating local biodiversity, adequate sampling strategies (i.e. sampling intensity and spatial sampling replication) need to be implemented. This study evaluated the impact of sampling strategies along an altitudinal and biodiversity gradient in the upper section of the Murrumbidgee River (Murray-Darling Basin, Australia). An eDNA metabarcoding survey was used to determine the local fish biodiversity and evaluate the influence of sampling intensity and spatial sampling replication on the biodiversity estimates. The results show that optimal eDNA sampling strategies varied between sites and indicate that river morphology, species richness and species abundance affect the optimal sampling intensity and spatial sampling replication needed to accurately assess the fish biodiversity. While the generality of the patterns will need to be confirmed through future studies, these findings provide a basis to guide future eDNA metabarcoding surveys in river systems.</p><h1>Seasonality drives microbial community structure, shaping both eukaryotic and prokaryotic host–viral relationships in an arctic marine ecosystem</h1>
<p><a href="https://doi.org/10.3390/v10120715">doi:10.3390/v10120715</a></p>

<p><a href=None>None</a></p>
<p>The Arctic marine environment experiences dramatic seasonal changes in light and nutrient availability. To investigate the influence of seasonality on Arctic marine virus communities, five research cruises to the west and north of Svalbard were conducted across one calendar year, collecting water from the surface to 1000 m in depth. We employed metabarcoding analysis of major capsid protein g23 and mcp genes in order to investigate T4-like myoviruses and large dsDNA viruses infecting prokaryotic and eukaryotic picophytoplankton, respectively. Microbial abundances were assessed using flow cytometry. Metabarcoding results demonstrated that seasonality was the key mediator shaping virus communities, whereas depth exerted a diversifying effect within seasonal virus assemblages. Viral diversity and virus-to-prokaryote ratios (VPRs) dropped sharply at the commencement of the spring bloom but increased across the season, ultimately achieving the highest levels during the winter season. These findings suggest that viral lysis may be an important process during the polar winter, when productivity is low. Furthermore, winter viral communities consisted of Operational Taxonomic Units (OTUs) distinct from those present during the spring-summer season. Our data provided a first insight into the diversity of viruses in a hitherto undescribed marine habitat characterized by extremes in light and productivity.</p><h1>Optimising the detection of marine taxonomic richness using environmental DNA metabarcoding: The effects of filter material, pore size and extraction method</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.28963">doi:10.3897/mbmg.2.28963</a></p>

<p><a href=None>None</a></p>
<p>The analysis of environmental DNA (eDNA) using metabarcoding has increased in use as a method for tracking biodiversity of ecosystems. Little is known about eDNA in marine human-modified environments, such as commercial ports, which are key sites to monitor for anthropogenic impacts on coastal ecosystems. To optimise an eDNA metabarcoding protocol in these environments, seawater samples were collected in a commercial port and methodologies for concentrating and purifying eDNA were tested for their effect on eukaryotic DNA yield and subsequent richness of Operational Taxonomic Units (OTUs). Different filter materials [Cellulose Nitrate (CN) and Glass Fibre (GF)], with different pore sizes (0.5 μm, 0.7 μm and 1.2 μm) and three previously published liquid phase extraction methods were tested. The number of eukaryotic OTUs detected differed by a factor of three amongst the method combinations. The combination of CN filters with phenol-chloroform-isoamyl alcohol extractions recovered a higher amount of eukaryotic DNA and OTUs compared to GF filters and the chloroform-isoamyl alcohol extraction method. Pore size was not independent of filter material but did affect the yield of eukaryotic DNA. For the OTUs assigned to a highly successful non-indigenous species, Styela clava, the two extraction methods with phenol significantly outperformed the extraction method without phenol; other experimental treatments did not contribute significantly to detection. These results highlight that careful consideration of methods is warranted because choice of filter material and extraction method create false negative detections of marine eukaryotic OTUs and underestimate taxonomic richness from environmental samples.</p><h1>Biodiversity of phytoplankton cyst assemblages in surface sediments of the Black Sea based on metabarcoding</h1>
<p><a href="https://doi.org/10.1080/13102818.2018.1532816">doi:10.1080/13102818.2018.1532816</a></p>

<p><a href=None>None</a></p>
<p>Resting stages are common for the life cycle of some phytoplankton species, including blooming and potentially toxic species. The “seed bank” accumulated in the sediments can initiate blooms in the water column and could be an early warning signal of harmful algal blooms (HABs). In order to identify the phytoplankton cyst assemblages, thirteen surface sediment samples were collected from different sites in the Black Sea. The diversity of the resting stages was assessed using high-throughput sequencing metabarcoding (V7-9 hypervariable region of the 18S rDNA). One hundred and eighty microalgal species were identified with high level of similarity to the reference sequences. Dinoflagellates were dominated by Biecheleria, Gymnodinium and Karlodinium. Within diatoms, Skeletonema, Chaetoceros and Thalassiosira were the most abundant genera. Sixteen of the detected operational taxonomic units (OTUs) were assigned to harmful microalgae (12 dinoflagellates species, 1 diatom, 1 haptophyte and 2 raphidophytes). No pattern of microalgal sequences depth distribution was discriminated. The results show that DNA metabarcoding has a great potential for assessment of the phytoplankton diversity in environmental sediments.</p><h1>Testing clustering strategies for metabarcoding-based investigation of community–environment interactions</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12922">doi:10.1111/1755-0998.12922</a></p>

<p><a href=None>None</a></p>
<p>The degradation of freshwater ecosystems has become a common ecological and environmental problem globally. Owing to the complexity of biological communities, there remain tremendous technical challenges for investigating influence of environmental stressors (e.g., chemical pollution) on biological communities. High-throughput sequencing-based metabarcoding provides a powerful tool to reveal complex interactions between environments and biological communities. Among many technical issues, the clustering strategies for operational taxonomic units (OTUs) which are crucial for assessing biodiversity of communities, may affect final conclusions. Here, we used zooplankton communities along an environmental pollution gradient in the Chaobai River in Northern China to test different clustering strategies, including nonclustering and clustering with varied thresholds. Our results showed that though the number of OTUs estimated by nonclustering strategies and clustering strategies with divergence thresholds of 99%–97% largely varied, they were able to identify the same set of significant environmental and spatial variables responsible for geographical distributions of zooplankton communities. In addition, the ecological conclusions obtained by clustering thresholds of 99%–97% were consistent with nonclustering strategies, where for all eight clustering scenarios we detected that species sorting predicted by environmental variables overrode dispersal as the dominant factor in structuring zooplankton communities. However, clustering with the divergence thresholds of <95% affected the environmental and spatial variables identified. We conclude that both newly developed nonclustering methods and traditional clustering methods with divergence thresholds ≥97% were reliable to reveal mechanisms of complex community–environment interactions, although different clustering strategies could lead to largely varied biodiversity estimates such as those for α-diversity.</p><h1>Debugging diversity – a pan-continental exploration of the potential of terrestrial blood-feeding leeches as a vertebrate monitoring tool</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12912">doi:10.1111/1755-0998.12912</a></p>

<p><a href=None>None</a></p>
<p>The use of environmental DNA (eDNA) has become an applicable noninvasive tool with which to obtain information about biodiversity. A subdiscipline of eDNA is iDNA (invertebrate-derived DNA), where genetic material ingested by invertebrates is used to characterize the biodiversity of the species that served as hosts. While promising, these techniques are still in their infancy, as they have only been explored on limited numbers of samples from only a single or a few different locations. In this study, we investigate the suitability of iDNA extracted from more than 3,000 haematophagous terrestrial leeches as a tool for detecting a wide range of terrestrial vertebrates across five different geographical regions on three different continents. These regions cover almost the full geographical range of haematophagous terrestrial leeches, thus representing all parts of the world where this method might apply. We identify host taxa through metabarcoding coupled with high-throughput sequencing on Illumina and IonTorrent sequencing platforms to decrease economic costs and workload and thereby make the approach attractive for practitioners in conservation management. We identified hosts in four different taxonomic vertebrate classes: mammals, birds, reptiles and amphibians, belonging to at least 42 different taxonomic families. We find that vertebrate blood ingested by haematophagous terrestrial leeches throughout their distribution is a viable source of DNA with which to examine a wide range of vertebrates. Thus, this study provides encouraging support for the potential of haematophagous terrestrial leeches as a tool for detecting and monitoring terrestrial vertebrate biodiversity.</p><h1>Core bacterial community of soy-daddawa: Insights from high-throughput DNA metabarcoding</h1>
<p><a href="https://doi.org/10.1016/j.lwt.2018.06.039">doi:10.1016/j.lwt.2018.06.039</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequence analysis of microbial communities in food matrices provides unprecedented insights into species richness and potential safety of foods. Here, the bacterial community of soy-daddawa, a fermented indigenous food condiment, was investigated for the first time using high-throughput DNA metabarcoding (16S rRNA gene). Analyses of 1 246 692 high-quality 16S rRNA gene sequences obtained from seven soy-daddawa samples/producers yielded 1520 operational taxonomic units (OTUs). Beta-diversity analysis of soy-daddawa bacterial communities suggested differences between samples, and in some cases, clustering according to geographical proximity of producers. Sixty-five OTUs (taxonomically assigned to 13 phylotypes) comprised the core (100% threshold) bacterial community of soy-daddawa and included Bacillus amyloliquefaciens, B. lichenformis, B. mojavensis, B. pumilus, B. subtilis, B. subtilis subsp. spizizenii, B. tequilensis, Exiguobacterium sp., Staphylococcus epidermidis and S. pseudintermedius. However, genera with known pathogenic species such as Atopostipes, Enterococcus, Granulicatella, Nosocomicoccus, Proteus, Pseudomonas and Vagoccoccus constituted at least 1% (relative abundance) in at least one sample. Insights gained from this study are useful for the development of a starter culture technology for the controlled fermentation of soy-daddawa. Also, the presence of potentially pathogenic species corroborates established global consensus on the microbiological risks associated with spontaneously fermented indigenous foods.</p><h1>A meta-barcoding census of freshwater planktonic protists in Appalachia - Natural Tunnel State Park, Virginia, USA</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.26939">doi:10.3897/mbmg.2.26939</a></p>

<p><a href=None>None</a></p>
<p>The purpose of our study was to survey the freshwater planktonic protists within an inland natural preserve in the Ridge and Valley physiographic province of the Appalachian Region using metabarcoding. Microbial eukaryotes are essential primary producers and predators in small freshwater ecosystems, yet they are often overlooked due to the difficulty of identification. This has been remedied, in part, by the cost reduction of high throughput DNA sequencing and the growth of barcode databases, making the identification and analysis of microorganisms by way of metabarcoding surveys in complex ecosystems increasingly feasible. Water samples were collected from five sites at the Natural Tunnel State Park in Scott County, VA (USA), representing three common bodies of water found in this region. Samples were initially collected during a Bioblitz event in April 2016 and then seven and fourteen weeks afterwards. Metabarcode analysis of the 23S and 18S genes identified 3663 OTUs representing 213 family level and 332 genus level taxa. This study provides an initial barcode census within a region that has a reputation as a temperate biodiversity "hotspot". The overall protist diversity was comparably high to other temperate systems, but not unusually high; the microalgal diversity, however, was higher than that reported for other temperate regions. The three types of water bodies had their own distinctive protist biomes despite close proximity.</p><h1>Microbiome of a revegetated iron-mining site and pristine ecosystems from the Brazilian Cerrado</h1>
<p><a href="https://doi.org/10.1016/j.apsoil.2018.07.011">doi:10.1016/j.apsoil.2018.07.011</a></p>

<p><a href=None>None</a></p>
<p>The Cerrado domain is composed of different phytophysiognomies and considered a hot spot of global diversity. However, few studies have assessed the soil microbiome of this domain using metabarcoding approach. This study aimed to determine the soil microbiome in a mining site under revegetation and compare with communities of pristine ecosystems. Soil samples were collected in the dry and rainy seasons in an iron-mining site under revegetation (RA), semi-decidual forest (FL), canga (NG) and Cerrado stricto sensu (CE). Soil community was accessed by the amplification of the 16S and 18S ribosomal gene fragments using high-throughput sequencing. The prokaryotic operational taxonomic units (OTUs) were assigned to 18 phyla, 48 classes, 73 orders, 123 families and 168 genera. Acidobacteria and Proteobacteria were the most abundant phylum for FL, CE and RA while Actinobacteria and Acidobacteria were dominant in NG. Relative abundance of families within each phylum varied mainly among sites, but not seasons. For Eukarya, OTUs were assigned to 15 phyla, 35 classes, 94 orders, 168 families and 247 genera. Opisthokonta, SAR (Stramenopiles, Alveolates, and Rhizaria), and Archaeplastida were the most abundant groups for all phytophysiognomies and Fungi accounted for 51–71% of total sequences. Principal coordinate analysis indicates that soil community composition was not influenced by season within each site. We concluded that soil microbiome was dissimilar among sites at the family and genus level but sites shared the same dominant phyla, and that differentiation of soil microbiome was driven mainly by plant community composition.</p><h1>Toward an ecoregion scale evaluation of eDNA metabarcoding primers: A case study for the freshwater fish biodiversity of the Murray–Darling Basin (Australia)</h1>
<p><a href="https://doi.org/10.1002/ece3.4387">doi:10.1002/ece3.4387</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing of environmental DNA (i.e., eDNA metabarcoding) has become an increasingly popular method for monitoring aquatic biodiversity. At present, such analyses require target-specific primers to amplify DNA barcodes from co-occurring species, and this initial amplification can introduce biases. Understanding the performance of different primers is thus recommended prior to undertaking any metabarcoding initiative. While multiple software programs are available to evaluate metabarcoding primers, all programs have their own strengths and weaknesses. Therefore, a robust in silico workflow for the evaluation of metabarcoding primers will benefit from the use of multiple programs. Furthermore, geographic differences in species biodiversity are likely to influence the performance of metabarcoding primers and further complicate the evaluation process. Here, an in silico workflow is presented that can be used to evaluate the performance of metabarcoding primers on an ecoregion scale. This workflow was used to evaluate the performance of published and newly developed eDNA metabarcoding primers for the freshwater fish biodiversity of the Murray–Darling Basin (Australia). To validate the in silico workflow, a subset of the primers, including one newly designed primer pair, were used in metabarcoding analyses of an artificial DNA community and eDNA samples. The results show that the in silico workflow allows for a robust evaluation of metabarcoding primers and can reveal important trade-offs that need to be considered when selecting the most suitable primer. Additionally, a new primer pair was described and validated that allows for more robust taxonomic assignments and is less influenced by primer biases compared to commonly used fish metabarcoding primers.</p><h1>BBI: An R package for the computation of Benthic Biotic Indices from composition data</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.25649">doi:10.3897/mbmg.2.25649</a></p>

<p><a href=None>None</a></p>
<p>The monitoring of impacts of anthropic activities in marine environments, such as aquaculture, oil-drilling platforms or deep-sea mining, relies on Benthic Biotic Indices (BBI). Several indices have been formalised to reduce the multivariate composition data into a single continuous value that is ascribed to a discrete ecological quality status. Such composition data is traditionally obtained from macrofaunal inventories, which is time-consuming and expertise-demanding. Important efforts are ongoing towards using High-Throughput Sequencing of environmental DNA (eDNA metabarcoding) to replace or complement morpho-taxonomic surveys for routine biomonitoring. The computation of BBI from such composition data is usually being undertaken by practitioners with excel spreadsheets or through custom script. Hence, the updating of reference morpho-taxonomic tables and cross studies comparison could be hampered. Here we introduce the R package BBI for the computation of BBI from composition data, either obtained from traditional morpho-taxonomic inventories or from metabarcoding data. Its aim is to provide an open-source, transparent and centralised method to compute BBI for routine biomonitoring.</p><h1>Needle in a haystack? A comparison of eDNA metabarcoding and targeted qPCR for detection of the great crested newt (Triturus cristatus)</h1>
<p><a href="https://doi.org/10.1002/ece3.4013">doi:10.1002/ece3.4013</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) analysis is a rapid, cost-effective, non-invasive biodiversity monitoring tool which utilises DNA left behind in the environment by organisms for species detection. The method is used as a species-specific survey tool for rare or invasive species across a broad range of ecosystems. Recently, eDNA and “metabarcoding” have been combined to describe whole communities rather than focusing on single target species. However, whether metabarcoding is as sensitive as targeted approaches for rare species detection remains to be evaluated. The great crested newt Triturus cristatus is a flagship pond species of international conservation concern and the first UK species to be routinely monitored using eDNA. We evaluate whether eDNA metabarcoding has comparable sensitivity to targeted real-time quantitative PCR (qPCR) for T. cristatus detection. Extracted eDNA samples (N = 532) were screened for T. cristatus by qPCR and analysed for all vertebrate species using high-throughput sequencing technology. With qPCR and a detection threshold of 1 of 12 positive qPCR replicates, newts were detected in 50% of ponds. Detection decreased to 32% when the threshold was increased to 4 of 12 positive qPCR replicates. With metabarcoding, newts were detected in 34% of ponds without a detection threshold, and in 28% of ponds when a threshold (0.028%) was applied. Therefore, qPCR provided greater detection than metabarcoding but metabarcoding detection with no threshold was equivalent to qPCR with a stringent detection threshold. The proportion of T. cristatus sequences in each sample was positively associated with the number of positive qPCR replicates (qPCR score) suggesting eDNA metabarcoding may be indicative of eDNA concentration. eDNA metabarcoding holds enormous potential for holistic biodiversity assessment and routine freshwater monitoring. We advocate this community approach to freshwater monitoring to guide management and conservation, whereby entire communities can be initially surveyed to best inform use of funding and time for species-specific surveys.</p><h1>Assessment of passive traps combined with high-throughput sequencing to study airborne fungal communities</h1>
<p><a href="https://doi.org/10.1128/AEM.02637-17">doi:10.1128/AEM.02637-17</a></p>

<p><a href=None>None</a></p>
<p>Techniques based on high-throughput sequencing (HTS) of environmental DNA have provided a new way of studying fungal diversity. However, these techniques suffer from a number of methodological biases which may appear at any of the steps involved in a metabarcoding study. Air is one of the most important environments where fungi can be found, because it is the primary medium of dispersal for many species. Looking ahead to future developments, it was decided to test 20 protocols, including different passive spore traps, spore recovery procedures, DNA extraction kits, and barcode loci. HTS was performed with the Illumina MiSeq platform targeting two subloci of the fungal internal transcribed spacer. Multivariate analysis and generalized linear models showed that the type of passive spore trap, the spore recovery procedure, and the barcode all impact the description of fungal communities in terms of richness and diversity when assessed by HTS metabarcoding. In contrast, DNA extraction kits did not significantly impact these results. Although passive traps may be used to describe airborne fungal communities, a study using specific real-time PCR and a mock community showed that these kinds of traps are affected by environmental conditions that may induce losses of biological material, impacting diversity and community composition results.</p><h1>ANDe<sup>™</sup>: A fully integrated environmental DNA sampling system</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12994">doi:10.1111/2041-210X.12994</a></p>

<p><a href=None>None</a></p>
<p>Species monitoring using environmental DNA (eDNA) is a powerful new technique for natural resource scientists and the number of research groups employing eDNA detection is growing rapidly. However, current eDNA sampling technologies consist mainly of do-it-yourself solutions, and the lack of purpose-built sampling equipment is limiting the efficiency and standardization of eDNA studies. Here, we describe the first fully integrated sampling system (ANDe™) designed by a team of molecular ecologists and engineers for high-throughput eDNA sample collection. It consists of a backpack portable pump that integrates sensor feedback, a pole extension with remote pump controller, custom-made filter housings in single-use packets for each sampling site and onboard sample storage. The system is optimized for sampling speed and replicability, while minimizing risk of contamination. We present an example pilot study designed to identify optimal ANDe™ system parameter values (i.e. pump pressure, flow rate, filter pore size, sample volume) in a new sampling environment. We identified a peak in filtration efficiency at a flow rate threshold of 1.0 L/m, and found that 5 μm filters captured significantly more target eDNA than 1 μm filters. Results also suggest that high filtration pressures may reduce eDNA retention, which implies that pressure should be standardized to avoid biasing detection data. Similar to the technological evolution of backpack electrofishers, eDNA sampling technology is in the process of transitioning from a nascent phase to professionally engineered research tools. Such innovations will be essential as eDNA monitoring becomes one of the industry standard methods used for species detection and management.</p><h1>Qualitative and quantitative molecular construction of plant–pollinator network: Application and prospective</h1>
<p><a href="https://doi.org/10.17520/biods.2018058">doi:10.17520/biods.2018058</a></p>

<p><a href=None>None</a></p>
<p>Pollinators serve key ecological functions, ensuring stable ecosystems and high agricultural yields. Hence, assessing ecosystem health and effects of agricultural management would benefit from understanding and monitoring pollination networks, which involves identifications of pollinators and pollinated plants. Classic approaches of morphology-based identification of plants and pollinators can be time-consuming, labor-intensive and costly, and require highly specialized taxonomic expertise. In comparison, DNA barcoding and high-throughput sequencing technologies can provide efficient and accurate identifications of plants and their pollinators, which may facilitate construction of pollination networks. Here we propose using sequencing technologies with a PCR-free genome-skimming work frame, using “super DNA barcode” as a new method to assess plant-pollinator networks. We expect this technique to improve resolution and accuracy of taxonomic identification to help gain quantitative information for bulk samples of pollinators or pollens. Al-though there are technical challenges to be resolved, the robustness of the new methodology has been validated in relevant biodiversity studies, suggesting promise in constructing pollination networks.</p><h1>Characterising planktonic dinoflagellate diversity in Singapore using DNA metabarcoding</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.25136">doi:10.3897/mbmg.2.25136</a></p>

<p><a href=None>None</a></p>
<p>Dinoflagellates are traditionally identified morphologically using microscopy, which is a time-consuming and labour-intensive process. Hence, we explored DNA metabarcoding using high-throughput sequencing as a more efficient way to study planktonic dinoflagellate diversity in Singapore's waters. From 29 minimally pre-sorted water samples collected at four locations in western Singapore, DNA was extracted, amplified and sequenced for a 313-bp fragment of the V4-V5 region in the 18S ribosomal RNA gene. Two sequencing runs generated 2,847,170 assembled paired-end reads, corresponding to 573,176 unique sequences. Sequences were clustered at 97% similarity and analysed with stringent thresholds (≥150 bp, ≥20 reads, ≥95% match to dinoflagellates), recovering 28 dinoflagellate taxa. Dinoflagellate diversity captured includes parasitic and symbiotic groups which are difficult to identify morphologically. Richness is similar between the inner and outer West Johor Strait, but variations in community structure are apparent, likely driven by environmental differences. None of the taxa detected in a recent phytoplankton bloom along the West Johor Strait have been recovered in our samples, suggesting that background communities are distinct from bloom communities. The voluminous data obtained in this study contribute baseline information for Singapore's phytoplankton communities and prompt future research and monitoring to adopt the approach established here.</p><h1>Estimating fish alpha- And beta-diversity along a small stream with environmental DNA metabarcoding</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.24262">doi:10.3897/mbmg.2.24262</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding has been increasingly applied to biodiversity surveys in stream ecosystems. In stream networks, the accuracy of eDNA-based biodiversity assessment depends on whether the upstream eDNA influx affects downstream detection. Biodiversity assessment in low-discharge streams should be less influenced by eDNA transport than in high-discharge streams. We estimated α- and β-diversity of the fish community from eDNA samples collected in a small Michigan (USA) stream from its headwaters to its confluence with a larger river. We found that α-diversity increased from upstream to downstream and, as predicted, we found a significant positive correlation between β-diversity and physical distance (stream length) between locations indicating species turnover along the longitudinal stream gradient. Sample replicates and different genetic markers showed similar species composition, supporting the consistency of the eDNA metabarcoding approach to estimate α- and β-diversity of fishes in low-discharge streams.</p><h1>Fungal community profiles in agricultural soils of a long-term field trial under different tillage, fertilization and crop rotation conditions analyzed by high-throughput ITS-amplicon sequencing</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0195345">doi:10.1371/journal.pone.0195345</a></p>

<p><a href=None>None</a></p>
<p>Fungal communities in agricultural soils are assumed to be affected by soil and crop management. Our intention was to investigate the impact of different tillage and fertilization practices on fungal communities in a long-term crop rotation field trial established in 1992 in Central Germany. Two winter wheat fields in replicated strip-tillage design, comprising conventional vs. conservation tillage, intensive vs. extensive fertilization and different pre-crops (maize vs. rapeseed) were analyzed by a metabarcoding approach applying Illumina paired-end sequencing of amplicons generated by two recently developed primer pairs targeting the two fungal Internal Transcribed Spacer regions (ITS1, ITS2). Analysis of 5.1 million high-quality sequence reads uncovered a diverse fungal community in the two fields, composed of 296 fungal genera including 3,398 Operational Taxonomic Units (OTUs) at the 97% sequence similarity threshold. Both primer pairs detected the same fungal phyla (Basidio-, Asco-, Zygo-, Glomero- and Chytridiomycota), but in different relative abundances. OTU richness was higher in the ITS1 dataset, while ITS2 data were more diverse and of higher evenness. Effects of farming practice on fungal community structures were revealed. Almost two-thirds of the fungal genera were represented in all different soil treatments, whereas the remaining genera clearly responded to farming practice. Principal Component Analysis revealed four distinct clusters according to tillage practice and pre-crop. Analysis of Variance (ANOVA) substantiated the results and proved significant influences of tillage and pre-crop, while fertilization had the smallest and non-significant effect. In-depth analysis of putative phytopathogenic and plant beneficial fungal groups indicated distinct responses; for example Fusarium was significantly enriched in the intensively fertilized conservation tillage variants with the pre-crop maize, while Phoma displayed significant association with conventional tillage and pre-crop rapeseed. Many putative plant beneficial fungi also reacted differentially to farming practice with the most distinct responders identified among the Glomeromycota (arbuscular mycorrhizal fungi, AMF).</p><h1>Avoiding quantification bias in metabarcoding: Application of a cell biovolume correction factor in diatom molecular biomonitoring</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12960">doi:10.1111/2041-210X.12960</a></p>

<p><a href=None>None</a></p>
<p>In recent years, remarkable progress has been made in developing environmental DNA metabarcoding. However, its ability to quantify species relative abundance remains uncertain, limiting its application for biomonitoring. In diatoms, although the rbcL gene appears to be a suitable barcode for diatoms, providing relevant qualitative data to describe taxonomic composition, improvement of species quantification is still required. Here, we hypothesized that rbcL copy number is correlated with diatom cell biovolume (as previously described for the 18S gene) and that a correction factor (CF) based on cell biovolume should be applied to improve taxa quantification. We carried out a laboratory experiment using pure cultures of eight diatom species with contrasted cell biovolumes in order to (1) verify the relationship between rbcL copy numbers (estimated by qPCR) and diatom cell biovolumes and (2) define a potential CF. In order to evaluate CF efficiency, five mock communities were created by mixing different amounts of DNA from the eight species, and were sequenced using HTS and targeting the same rbcL barcode. As expected, the correction of DNA reads proportions by the CF improved the congruence between morphological and molecular inventories. Final validation of the CF was obtained on environmental samples (metabarcoding data from 80 benthic biofilms) for which the application of CF allowed differences between molecular and morphological water quality indices to be reduced by 47%. Overall, our results highlight the usefulness of applying a CF factor, which is effective in reducing over-estimation of high biovolume species, correcting quantitative biases in diatom metabarcoding studies and improving final water quality assessment.</p><h1>The need for standardisation: Exemplified by a description of the diversity, community structure and ecological indices of soil nematodes</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2017.12.002">doi:10.1016/j.ecolind.2017.12.002</a></p>

<p><a href=None>None</a></p>
<p>Molecular approaches are offering a supplement to, or even the possibility of replacing morphological identification of soil fauna, because of advantages for throughput, coverage and objectivity. We determined ecological indices of nematode community data from four sets of duplicate soil cores, based on morphological identification of nematodes after elutriation from 200 g soil and high throughput sequencing (HTS) targeting nematodes both after being elutriated from soils and DNA extracted directly from 10 g soil. HTS (at genus and species level) increased the taxonomic resolution compared to morphology (at family level). DNA extracted from elutriated nematodes identified more nematode taxa than when extracted from soil, due to an enrichment in nematode sequences. Each method also gave a different ecological footprint for the nematode community. Standardisation to previously determined indices based on morphological identification is needed in order to provide more meaningful information about soil quality and for ecological monitoring.</p><h1>Aquatic bacterial communities associated with land use and environmental factors in agricultural landscapes using a metabarcoding approach</h1>
<p><a href="https://doi.org/10.3389/fmicb.2018.02301">doi:10.3389/fmicb.2018.02301</a></p>

<p><a href=None>None</a></p>
<p>This study applied a 16S rRNA gene metabarcoding approach to characterize bacterial community compositional and functional attributes for surface water samples collected within, primarily, agriculturally dominated watersheds in Ontario and Québec, Canada. Compositional heterogeneity was best explained by stream order, season, and watercourse discharge. Generally, community diversity was higher at agriculturally dominated lower order streams, compared to larger stream order systems such as small to large rivers. However, during times of lower relative water flow and cumulative 2-day rainfall, modestly higher relative diversity was found in the larger watercourses. Bacterial community assemblages were more sensitive to environmental/land use changes in the smaller watercourses, relative to small-to-large river systems, where the proximity of the sampled water column to bacteria reservoirs in the sediments and adjacent terrestrial environment was greater. Stream discharge was the environmental variable most significantly correlated (all positive) with bacterial functional groups, such as C/N cycling and plant pathogens. Comparison of the community structural similarity via network analyses helped to discriminate sources of bacteria in freshwater derived from, for example, wastewater treatment plant effluent and intensity and type of agricultural land uses (e.g., intensive swine production vs. dairy dominated cash/livestock cropping systems). When using metabarcoding approaches, bacterial community composition and coexisting pattern rather than individual taxonomic lineages, were better indicators of environmental/land use conditions (e.g., upstream land use) and bacterial sources in watershed settings. Overall, monitoring changes and differences in aquatic microbial communities at regional and local watershed scales has promise for enhancing environmental footprinting and for better understanding nutrient cycling and ecological function of aquatic systems impacted by a multitude of stressors and land uses.</p><h1>Quantitative monitoring of multispecies fish environmental DNA using high-throughput sequencing</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.23297">doi:10.3897/mbmg.2.23297</a></p>

<p><a href=None>None</a></p>
<p>Effective ecosystem conservation and resource management require quantitative monitoring of biodiversity, including accurate descriptions of species composition and temporal variations of species abundance. Accordingly, quantitative monitoring of biodiversity has been performed for many ecosystems, but it is often time- and effort- consuming and costly. Recent studies have shown that environmental DNA (eDNA), which is released to the environment from macro-organisms living in a habitat, contains information about species identity and abundance. Thus, analysing eDNA would be a promising approach for more efficient biodiversity monitoring. In the present study, internal standard DNAs (i.e. known amounts of short DNA fragments from fish species that have never been observed in a sampling area) were added to eDNA samples, which were collected weekly from a coastal marine ecosystem in Maizuru Bay, Japan (from April 2015 to March 2016) and metabarcoding analysis was performed using Illumina MiSeq to simultaneously identify fish species and quantify fish eDNA copy numbers. A correction equation was obtained for each sample using the relationship between the number of sequence reads and the added amount of the standard DNAs and this equation was used to estimate the copy numbers from the sequence reads of non-standard fish eDNA. The calculated copy numbers showed significant positive correlation with those determined by quantitative PCR, suggesting that eDNA metabarcoding with standard DNA enabled useful quantifications of eDNA. Furthermore, for samples that show a high level of PCR inhibition, this method might allow more accurate quantification than qPCR because the correction equations generated using internal standard DNAs would include the effect of PCR inhibition. A single run of Illumina MiSeq produced >70 quantitative fish eDNA time series in this study, showing that this method could contribute to more efficient and quantitative monitoring of biodiversity.</p><h1>Development and preliminary validation of a multi-trophic metabarcoding biotic index for monitoring benthic organic enrichment</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2017.11.014">doi:10.1016/j.ecolind.2017.11.014</a></p>

<p><a href=None>None</a></p>
<p>Sea-based fish farms are associated with strong, benthic enrichment gradients and routine monitoring is usually required by regulation. This has traditionally been based on benthic macrofaunal communities, which is reliant on taxonomic expertise, and turn-around times can be slow, limiting opportunities for adaptive management. Environmental metabarcoding is a powerful high-throughput sequencing-based technique that can identify and quantify benthic assemblages. Previous studies have demonstrated relationships between specific taxonomic groups (e.g., bacteria, foraminifera) and anthropogenic effects. However, the absence of fixed categorical scales makes the use of data challenging for routine biomonitoring. In this study, we analysed 105 sediment samples collected over three years from three salmon farms spanning two separate bioregions. Environmental DNA and RNA (eDNA/eRNA) metabarcoding of three taxonomic groups (foraminifera, bacteria, and general eukaryotes) was undertaken in parallel with traditional macrofaunal and biochemical analyses, which were used to calculate an Enrichment Stage index for each sample. The most abundant 200–250 Operational Taxonomic Units in each taxonomic group were assessed and assigned to Eco-Groups. These bioindicator taxa were then used to develop a Metabarcoding Biotic Index for each taxonomic group individually and in combination (multi-trophic Metabarcoding Biotic Index). The weakest relationship was between the foraminiferal Metabarcoding Biotic Index and Enrichment Stage (eDNA/eRNA, R2 = 0.731–0.850), whereas strong correspondence was obtained between the multi-trophic Metabarcoding Biotic Index and Enrichment Stage, or when just bacteria and eukaryotes data were combined (eDNA/RNA, R2 > 0.900). The strong relationships obtained between the multi-trophic Metabarcoding Biotic Index and Enrichment Stage confirms that this method has the potential to complement or even replace current fish farm biomonitoring techniques in the near future.</p><h1>Can high-throughput sequencing detect macroinvertebrate diversity for routine monitoring of an urban river?</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2017.11.002">doi:10.1016/j.ecolind.2017.11.002</a></p>

<p><a href=None>None</a></p>
<p>Macroinvertebrate families identified through morphological examination have traditionally been used in routine assessment of freshwater ecosystems. However, high throughput DNA sequencing (HTS) promises to improve routine assessment by providing rapid and cost-effective identification of macroinvertebrate species. In freshwater ecosystems in urbanised areas where family diversity is often low, new insights into ecosystem condition and impacting factors are likely through species-level assessments. Here we compare morphological identification to HTS based identification of macroinvertebrate families by considering 12 sites in an urban river system. Some taxa detected morphologically were not detected by HTS and vice versa. However, this had only a small impact on computed family-level metrics of ecological condition. We detected multiple species using HTS in the Chironomidae, Coenagrionidae, Hydrobiidae, Leptoceridae, Ceratopogonidae, Corixidae, Veliidae, Oligochaeta and Acarina. The highest species diversity was found in the Chironomidae, and for many of these species we had prior knowledge of their likely pollution sensitivity. In the Chironomidae, we showed that species level data was congruent with expectations based on measured levels of pollutants at sites and other family level metrics. Importantly, we also identified many species in the same family that differed in their distribution and likely pollution sensitivity in this urban river system. Therefore, HTS provided similar levels of information to traditional methods at the family level, but also generated new information for more accurate condition monitoring at the species level.</p><h1>Metabarcoding of lake benthic diatoms: from structure assemblages to ecological assessment</h1>
<p><a href="https://doi.org/10.1007/s10750-017-3381-2">doi:10.1007/s10750-017-3381-2</a></p>

<p><a href=None>None</a></p>
<p>Benthic diatoms are relevant indicators of the ecological status of the littoral zone of lakes. Their use as bio-indicators is based on their morphological identification at species level using microscopy which is time consuming, requires taxonomic expertise, and is consequently expensive. To overcome these limitations, a molecular approach for diatom identification has been tested with success in rivers. DNA metabarcoding enables species identification from a standardized DNA barcode and high-throughput sequencing (HTS), using DNA reference library. The suitability of the morphological and molecular approaches to assess the diatom community structure and the ecological status of the littoral zone of the largest deep lake in France (Lake Bourget) was compared. 66 sites were sampled in August 2015 along the shoreline, all around the lake. The composition of diatom assemblages was similar with both morphological and molecular approaches, and diatom assemblages were structured by the same environmental factors. However, the ecological status of Lake Bourget differed significantly among approaches since floristic inventories to species level also differed significantly. The main source of this difference was the incompleteness of the DNA reference library. Nevertheless, in a near future, when this constraint will be solved, the use of DNA metabarcoding for biomonitoring purposes seems promising.</p><h1>Development of environmental DNA (eDNA) methods for detecting high-risk freshwater fishes in live trade in Canada</h1>
<p><a href="https://doi.org/10.1007/s10530-017-1532-z">doi:10.1007/s10530-017-1532-z</a></p>

<p><a href=None>None</a></p>
<p>Preventing the arrival, establishment, and spread of aquatic invasive species is an important step in protecting our aquatic environments. The use of detection tools, like DNA barcoding technologies, high-throughput sequencing and environmental DNA (eDNA) monitoring, is becoming increasingly important in preventing the introduction of potential invasive species. The combination of eDNA with realtime PCR (qPCR) provide the opportunity to have a rapid and specific detection. In this study, we developed a DNA sequence library that has sufficient depth and species coverage such that high-risk species can be confidently discriminated from legitimately imported and native species. A total of 12 species-specific qPCR assays were developed for the detection of 13 potential invasive species (pAIS) in bulk water samples. Detection of these species was also compared with a HTS approach. We have demonstrated the high sensitivity of qPCR assays using eDNA at very low densities, suggesting we could detect a low number of individuals mixed with non-target species in a simulated live shipment. For the detection of a targeted list of species, qPCR is advantageous. The mini-barcodes developed in this project offered a good sensitivity of detection, and HTS is a discovery tool that can be desirable when unlisted or numerous species need to be identified.</p><h1>High-throughput techniques as support for knowledge-based spatial conservation prioritization in mangrove ecosystems</h1>
<p><a href="https://doi.org/10.1007/978-3-319-73016-5_24">doi:10.1007/978-3-319-73016-5_24</a></p>

<p><a href=None>None</a></p>
<p>The conservation of functioning ecosystems worldwide is warranted by the need for reliable and sustainable provision of ecosystem services locally, regionally and globally. Mangroves provide numerous ecosystem services both to local human communities, e.g., coastal protection or food security, and to mankind worldwide, e.g., climate change-mitigation. Nonetheless they still lack protection in many places of occurrence. Here we base spatial prioritization and planning of mangrove conservation on functional biodiversity and service-relevant ecosystem processes, being studied through cutting-edge genetic and chemical analyses of sediments to unravel the links between biodiversity, biotic interactions, ecosystem processes and ecosystem services. We nonetheless recommend multidisciplinary approaches when planning protected area networks for the sustainable use and provision of ecosystem services and pledge for (i) considering and prioritizing societal, biological and economic values of mangroves, (ii) integrating adjacent ecosystems to maintain connectivity, and (iii) taking into account the spatial and temporal dynamics of mangrove ecosystems and their community composition under global change, i.e. changes in the spatial distribution of species and services over time. Beyond the example of mangroves and the turnover of organic matter in mangrove sediments described herein, our approach to spatial conservation prioritization and planning is applicable to any other ecosystems and their services.</p><h1>Molecular approaches for an operational marine biodiversity observation network</h1>
<p><a href="https://doi.org/10.1016/B978-0-12-805052-1.00032-2">doi:10.1016/B978-0-12-805052-1.00032-2</a></p>

<p><a href=None>None</a></p>
<p>Biological health underpins ecosystem services, including the production of food from the sea. Therefore, many policy frameworks rely on biological observations to guide the sustainable use of marine resources. For example, a marine biodiversity observation network can provide evidence of ecosystem degradation, remediation success, and progress toward goals of sustainable development. The utility of biodiversity observations is limited, however, if measurements are not standardized to allow comparisons across space and time or if they are too difficult or expensive to make routinely or at large scale. Molecular approaches (e.g., metagenomics, metabarcoding, eDNA, metatranscriptomics) offer efficiencies in sample collection and processing and provide analysis across trophic levels compared to traditional methods. Increased affordability and throughput of nucleic acid sequencing allow integration of molecular biological techniques into operational monitoring with the potential benefit of expanding the spatial and temporal scale of biological observations.</p><h1>Linking DNA Metabarcoding and Text Mining to Create Network-Based Biomonitoring Tools: A Case Study on Boreal Wetland Macroinvertebrate Communities</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2018.09.001">doi:10.1016/bs.aecr.2018.09.001</a></p>

<p><a href=None>None</a></p>
<p>Ecological networks are powerful tools for visualizing biodiversity data and assessing ecosystem health and function. Constructing these networks requires considerable empirical efforts, and this remains highly challenging due to sampling limitations and the laborious and notoriously limited, error-prone process of traditional taxonomic identification. Recent advancements in high-throughput gene sequencing and high-performance computing provide new ways to address these challenges. DNA metabarcoding, a method of bulk taxonomic identification from DNA extracted from environmental samples, can generate detailed biodiversity information through a standardizable analytical pipeline for species detection. When this biodiversity information is annotated with prior knowledge on taxon interactions, body size, and trophic position, it is possible to generate trait-based networks, which we call “heuristic food webs”. Although curating trait matrices for constructing heuristic food webs is a laborious, often intractable process using manual literature surveys, it can be greatly accelerated via text mining, allowing knowledge of relevant traits to be gathered across large databases. To explore this possibility, we employed a General Architecture for Text Engineering (GATE) system to create a hybrid text-mining pipeline combining rule-based and machine-learning modules. This pipeline was then used to query online repositories of published papers for missing data on a key trait, body size, that could not be gathered from existing trophic link libraries of freshwater benthic macroinvertebrates. Combining text-mined body size information with feeding information from existing sources allowed us to generate a database of over 20,000 pairwise trophic interactions. Next, we developed a pipeline that uses taxa lists generated from DNA metabarcoding and annotates this matrix with trophic information from existing databases and text-mined body size data. In this way, we generated heuristic food webs for wetland sites within a large delta complex formed by the confluence of the Peace and Athabasca rivers in northern Alberta: the Peace–Athabasca delta. Finally, we used these putative food webs and their network properties to resolve spatial and temporal differences between the benthic subwebs of wetlands in the Peace and Athabasca sectors of the delta complex. Specifically, we asked two questions. (1) How do food web properties (e.g. number of links, linkage density, trophic height) differ between the wetlands of the Peace and Athabasca deltas? (2) How do food web properties change temporally in wetlands of the two deltas? We discuss using DNA-generated, trait-based food webs as a powerful tool for rapid bioassessment, assess the limitations of our current approach, and outline a path forward to make this powerful tool more widely available for land managers and conservation biologists.</p><h1>Tackling critical parameters in metazoan meta-barcoding experiments: A preliminary study based on coxI DNA barcode</h1>
<p><a href="https://doi.org/10.7717/peerj.4845">doi:10.7717/peerj.4845</a></p>

<p><a href=None>None</a></p>
<p>Nowadays DNA meta-barcoding is a powerful instrument capable of quickly discovering the biodiversity of an environmental sample by integrating the DNA barcoding approach with High Throughput Sequencing technologies. It mainly consists of the parallel reading of informative genomic fragment/s able to discriminate living entities. Although this approach has been widely studied, it still needs optimization in some necessary steps requested in its advanced accomplishment. A fundamental element concerns the standardization of bioinformatic analyses pipelines. The aim of the present study was to underline a number of critical parameters of laboratory material preparation and taxonomic assignment pipelines in DNA meta-barcoding experiments using the cytochrome oxidase subunit-I (coxI) barcode region, known as a suitable molecular marker for animal species identification. We compared nine taxonomic assignment pipelines, including a custom in-house method, based on Hidden Markov Models. Moreover, we evaluated the potential influence of universal primers amplification bias in qPCR, as well as the correlation between GC content with taxonomic assignment results. The pipelines were tested on a community of known terrestrial invertebrates collected by pitfall traps from a chestnut forest in Italy. Although the present analysis was not exhaustive and needs additional investigation, our results suggest some potential improvements in laboratory material preparation and the introduction of additional parameters in taxonomic assignment pipelines. These include the correct setup of OTU clustering threshold, the calibration of GC content affecting sequencing quality and taxonomic classification, as well as the evaluation of PCR primers amplification bias on the final biodiversity pattern. Thus, careful attention and further validation/optimization of the above-mentioned variables would be required in a DNA meta-barcoding experimental routine.</p><h1>Why We Need Sustainable Networks Bridging Countries, Disciplines, Cultures and Generations for Aquatic Biomonitoring 2.0: A Perspective Derived From the DNAqua-Net COST Action</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2018.01.001">doi:10.1016/bs.aecr.2018.01.001</a></p>

<p><a href=None>None</a></p>
<p>Aquatic biomonitoring has become an essential task in Europe and many other regions as a consequence of strong anthropogenic pressures affecting the health of lakes, rivers, oceans and groundwater. A typical assessment of the environmental quality status, such as it is required by European but also North American and other legislation, relies on matching the composition of assemblages of organisms identified using morphological criteria present in aquatic ecosystems to those expected in the absence of anthropogenic pressures. Through decade-long and difficult intercalibration exercises among networks of regulators and scientists in European countries, a pragmatic biomonitoring approach was developed and adopted, which now produces invaluable information. Nonetheless, this approach is based on several hundred different protocols, making it susceptible to issues with comparability, scale and resolution. Furthermore, data acquisition is often slow due to a lack of taxonomic experts for many taxa and regions and time-consuming morphological identification of organisms. High-throughput genetic screening methods such as (e)DNA metabarcoding have been proposed as a possible solution to these shortcomings. Such “next-generation biomonitoring”, also termed “biomonitoring 2.0”, has many advantages over the traditional approach in terms of speed, comparability and costs. It also creates the potential to include new bioindicators and thereby further improves the assessment of aquatic ecosystem health. However, several major conceptual and technological challenges still hinder its implementation into legal and regulatory frameworks. Academic scientists sometimes tend to overlook legal or socioeconomic constraints, which regulators have to consider on a regular basis. Moreover, quantification of species abundance or biomass remains a significant bottleneck to releasing the full potential of these approaches. Here, we highlight the main challenges for next-generation aquatic biomonitoring and outline principles and good practices to address these with an emphasis on bridging traditional disciplinary boundaries between academics, regulators, stakeholders and industry.</p><h1>Environmental DNA: For biodiversity research and monitoring</h1>
<p><a href="https://doi.org/10.1093/oso/9780198767220.001.0001">doi:10.1093/oso/9780198767220.001.0001</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA), i.e. DNA released in the environment by any living form, represents a formidable opportunity to gather high-throughput and standard information on the distribution or feeding habits of species. It has therefore great potential for applications in ecology and biodiversity management. However, this research field is fast-moving, involves different areas of expertise and currently lacks standard approaches, which calls for an up-to-date and comprehensive synthesis. Environmental DNA for biodiversity research and monitoring covers current methods based on eDNA, with a particular focus on “eDNA metabarcoding”. Intended for scientists and managers, it provides the background information to allow the design of sound experiments. It revisits all steps necessary to produce high-quality metabarcoding data such as sampling, metabarcode design, optimization of PCR and sequencing protocols, as well as analysis of large sequencing datasets. All these different steps are presented by discussing the potential and current challenges of eDNA-based approaches to infer parameters on biodiversity or ecological processes. The last chapters of this book review how DNA metabarcoding has been used so far to unravel novel patterns of diversity in space and time, to detect particular species, and to answer new ecological questions in various ecosystems and for various organisms. Environmental DNA for biodiversity research and monitoring constitutes an essential reading for all graduate students, researchers and practitioners who do not have a strong background in molecular genetics and who are willing to use eDNA approaches in ecology and biomonitoring.</p><h1>Identifying north American freshwater invertebrates using DNA barcodes: Are existing COI sequence libraries fit for purpose?</h1>
<p><a href="https://doi.org/10.1086/696613">doi:10.1086/696613</a></p>

<p><a href=None>None</a></p>
<p>We reviewed the availability of cytochrome c oxidase subunit I (COI) sequences for 2534 North American freshwater invertebrate genera in public databases (GenBank and Barcode of Life Data Systems) and assessed representation of genera commonly encountered in the Canadian Aquatic Biomonitoring Network (CABIN) database. COI sequence records were available for 61.2% of North American genera and 72.4% of Insecta genera in public databases. Mollusca (73.9%) and Nematoda (15.4%) were the best and worst represented groups, respectively. In CABIN, 85.4% of genera had COI sequence records, and 95.2% of genera occurring in >1% of samples were represented. Genera absent from CABIN tended to be uncommon or members of groups not routinely used for biomonitoring purposes. On average, 94.1% of genera in well-identified samples had associated sequence data. To leverage the full potential of genomics approaches, we must expand DNA-barcode reference libraries for poorly described components of freshwater food webs. Some genera appear to be well represented (e.g., Eukiefferiella), but deposited sequences represent few sampling localities or few species and lead to underestimation of sequence diversity at the genus level and reduced confidence in identifications. Public COI libraries are sufficiently populated to permit routine application of genomics tools in biomonitoring, and ongoing quality assurance/quality control should include re-evaluation as new COI reference sequences are added or taxonomic hierarchies change. Next, we must understand whether and how established biomonitoring approaches can capitalize on high-throughput sequencing tools. Biomonitoring approaches that use genomics data to facilitate structural and functional assessments are fertile ground for future investigation and will benefit from continued improvement of publicly available sequence libraries.</p><h1>The potential of High–Throughput Sequencing (HTS) of natural samples as a source of primary taxonomic information for reference libraries of diatom barcodes</h1>
<p><a href="https://doi.org/10.5507/fot.2017.013">doi:10.5507/fot.2017.013</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are used routinely to assess pollution level in rivers and lakes. Current methods are based on identification by light microscopy, which is laborious. An alternative is to identify species based on short DNA fragments and High–Throughput Sequencing (HTS). However a potential limitation is the incomplete coverage of species in reference barcode libraries. Usually these libraries are compiled by isolating cells, before culturing and sequencing them, which is tedious and often unsuccessful. Here we propose the use of rbcL sequences from environmental samples analysed by HTS. We set several criteria to ensure good sequence quality and correspondence with the target species observed in microscopy: the sequence needed to be abundant in the sample, and with no insertions nor deletions or stop codon, phylogenetic neighbour taxa had to correspond to neighbour taxonomic taxa expected from morphological observations. Four species from tropical rivers are given as examples, including one that is new to science.</p><h1>Metabarcoding analysis of eukaryotic microbiota in the gut of HIV-infected patients</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0191913">doi:10.1371/journal.pone.0191913</a></p>

<p><a href=None>None</a></p>
<p>Research on the relationship between changes in the gut microbiota and human disease, including AIDS, is a growing field. However, studies on the eukaryotic component of the intestinal microbiota have just begun and have not yet been conducted in HIV-infected patients. Moreover, eukaryotic community profiling is influenced by the use of different methodologies at each step of culture-independent techniques. Herein, initially, four DNA extraction protocols were compared to test the efficiency of each method in recovering eukaryotic DNA from fecal samples. Our results revealed that recovering eukaryotic components from fecal samples differs significantly among DNA extraction methods. Subsequently, the composition of the intestinal eukaryotic microbiota was evaluated in HIV-infected patients and healthy volunteers through clone sequencing, high-throughput sequencing of nuclear ribosomal internal transcribed spacers 1 (ITS1) and 2 (ITS2) ampli-cons and real-time PCRs. Our results revealed that not only richness (Chao-1 index) and alpha diversity (Shannon diversity) differ between HIV-infected patients and healthy volunteers, depending on the molecular strategy used, but also the global eukaryotic community composition, with little overlapping taxa found between techniques. Moreover, our results based on cloning libraries and ITS1/ITS2 metabarcoding sequencing showed significant differences in fungal composition between HIV-infected patients and healthy volunteers, but without distinct clusters separating the two groups. Malassezia restricta was significantly more prevalent in fecal samples of HIV-infected patients, according to cloning libraries, whereas operational taxonomic units (OTUs) belonging to Candida albicans and Candida tropicalis were significantly more abundant in fecal samples of HIV-infected patients compared to healthy subjects in both ITS subregions. Finally, real-time PCR showed the presence of Microsporidia, Giardia lamblia, Blastocystis and Hymenolepis diminuta in different proportions in fecal samples from HIV patients as compared to healthy individuals. Our work revealed that the use of different sequencing approaches can impact the perceived eukaryotic diversity results of the human gut. We also provide a more comprehensive view of the eukaryotic community in the gut of HIV-infected patients through the complementarity of the different molecular techniques used. Combining these various methodologies may provide a gold standard for a more complete characterization of the eukaryotic microbiome in future studies.</p><h1>High-throughput eDNA monitoring of fungi to track functional recovery in ecological restoration</h1>
<p><a href="https://doi.org/10.1016/j.biocon.2017.10.035">doi:10.1016/j.biocon.2017.10.035</a></p>

<p><a href=None>None</a></p>
<p>Fungi are key functional components of ecosystems (e.g. decomposers, symbionts), but are rarely included in restoration monitoring programs. Many fungi occur belowground, making them difficult to observe directly, but are observable with environmental DNA (eDNA) methods. Although eDNA approaches have been proposed as ecological monitoring tools for microbial diversity, their application to restoration projects is very limited. We used eDNA metabarcoding of fungal ITS barcodes on soil collected across a 10-year restoration chronosequence to explore fungal responses to restoration. We observed a dramatic shift in the fungal community towards that of the natural fungal community after just 10 years of active native plant revegetation. Agaricomycetes and other Basidiomycota – involved in wood decay and ectomycorrhizal symbiosis – increased in rarefied sequence abundance in older restored sites. Ascomycota dominated the fungal community, but decreased in rarefied sequence abundance across the restoration chronosequence. Our results highlight eDNA metabarcoding as a useful restoration monitoring tool that allows quantification of changes in important fungal indicator groups linked with functional recovery and, being underground, are normally omitted in restoration monitoring.</p><h1>Scrutinizing key steps for reliable metabarcoding of environmental samples</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12849">doi:10.1111/2041-210X.12849</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding of environmental samples has many challenges and limitations that require carefully considered laboratory and analysis workflows to ensure reliable results. We explore how decisions regarding study design, laboratory set-up, and bioinformatic processing affect the final results, and provide guidelines for reliable study of environmental samples. We evaluate the performance of four primer sets targeting COI and 16S regions characterizing arthropod diversity in bat faecal samples, and investigate how metabarcoding results are affected by parameters including: (1) number of PCR replicates per sample, (2) sequencing depth, (3) PCR replicate processing strategy (i.e. either additively, by combining the sequences obtained from the PCR replicates, or restrictively, by only retaining sequences that occur in multiple PCR replicates for each sample), (4) minimum copy number for sequences to be retained, (5) chimera removal, and (6) similarity thresholds for Operational Taxonomic Unit (OTU) clustering. Lastly, we measure within- and between-taxa dissimilarities when using sequences from public databases to determine the most appropriate thresholds for OTU clustering and taxonomy assignment. Our results show that the use of multiple primer sets reduces taxonomic biases and increases taxonomic coverage. Taxonomic profiles resulting from each primer set are principally affected by how many PCR replicates are carried out per sample and how sequences are filtered across them, the sequence copy number threshold and the OTU clustering threshold. We also report considerable diversity differences between PCR replicates from each sample. Sequencing depth increases the dissimilarity between PCR replicates unless the bioinformatic strategies to remove allegedly artefactual sequences are adjusted according to the number of analysed sequences. Finally, we show that the appropriate identity thresholds for OTU clustering and taxonomy assignment differ between markers. Metabarcoding of complex environmental samples ideally requires (1) investigation of whether more than one primer sets targeting the same taxonomic group is needed to offset primer biases, (2) more than one PCR replicate per sample, (3) bioinformatic processing of sequences that balance diversity detection with removal of artefactual sequences, and (4) empirical selection of OTU clustering and taxonomy assignment thresholds tailored to each marker and the obtained taxa.</p><h1>Combining NCBI and BOLD databases for OTU assignment in metabarcoding and metagenomic datasets: The BOLD-NCBI-Merger</h1>
<p><a href="https://doi.org/10.3897/mbmg.1.22262">doi:10.3897/mbmg.1.22262</a></p>

<p><a href=None>None</a></p>
<p>Background Metabarcoding and metagenomic approaches are becoming routine techniquesfor use in biodiversity assessment and in ecological studies. The assignment of taxonomic information to millions of sequences obtained via high-throughput sequencing is challenging, as many DNA reference libraries are lacking information on certain taxonomic groups and can contain erroneous sequences. Combining different reference databases is therefore a promising approach for maximising taxonomic coverage and reliability of results. New information The "BOLD-NCBI-Merger" bash script is introduced, which combines sequence data obtained from the National Centre for Biotechnology Information (NCBI) GenBank and the Barcode of Life Database (BOLD) and prepares it for taxonomic assignment with the software MEGAN.</p><h1>Performance of DNA metabarcoding, standard barcoding, and morphological approach in the identification of hostparasitoid interactions</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0187803">doi:10.1371/journal.pone.0187803</a></p>

<p><a href=None>None</a></p>
<p>Understanding interactions between herbivores and parasitoids is essential for successful biodiversity protection and monitoring and for biological pest control. Morphological identifications employ insect rearing and are complicated by insects' high diversity and crypsis. DNA barcoding has been successfully used in studies of host-parasitoid interactions as it can substantially increase the recovered real host-parasitoid diversity distorted by overlooked species complexes, or by species with slight morphological differences. However, this approach does not allow the simultaneous detection and identification of host(s) and parasitoid(s). Recently, high-throughput sequencing has shown high potential for surveying ecological communities and trophic interactions. Using mock samples comprising insect larvae and their parasitoids, we tested the potential of DNA metabarcoding for identifying individuals involved in host-parasitoid interactions to different taxonomic levels, and compared it to standard DNA barcoding and morphological approaches. For DNA metabarcoding, we targeted the standard barcoding marker cytochrome oxidase subunit I using highly degenerate primers, 2∗300 bp sequencing on a MiSeq platform, and RTAX classification using paired-end reads. Additionally, using a large host-parasitoid dataset from a Central European floodplain forest, we assess the completeness and usability of a local reference library by confronting the number of Barcoding Index Numbers obtained by standard barcoding with the number of morphotypes. Overall, metabarcoding recovery was high, identifying 92.8% of the taxa present in mock samples, and identification success within individual taxonomic levels did not significantly differ among metabarcoding, standard barcoding, and morphology. Based on the current local reference library, 39.4% parasitoid and 90.7% host taxa were identified to the species level. DNA barcoding estimated higher parasitoid diversity than morphotyping, especially in groups with high level of crypsis. This study suggests the potential of metabarcoding for effectively recovering host-parasitoid diversity, together with more accurate identifications obtained from building reliable and comprehensive reference libraries, especially for parasitoids.</p><h1>To what extent is the DNA of microbial eukaryotes modified during burying into lake sediments? A repeat-coring approach on annually laminated sediments</h1>
<p><a href="https://doi.org/10.1007/s10933-017-0005-9">doi:10.1007/s10933-017-0005-9</a></p>

<p><a href=None>None</a></p>
<p>Paleogenetics provides a powerful framework to reconstruct the long-term temporal dynamics of various biological groups from aquatic sediments. However, validations are still required to ensure the authenticity of the molecular signal obtained from sedimentary DNA. Here, we investigated the effects of early diagenesis on the DNA signal from micro-eukaryotes preserved in sediments by comparing metabarcoding inventories obtained for two sediment cores sampled in 2007 and 2013 respectively. High-throughput sequencing (Illumina MiSeq) of sedimentary DNA was utilized to reconstruct the composition of microbial eukaryotic communities by targeting the V7 region of the 18S rDNA gene. No significant difference was detected between the molecular inventories obtained for the two cores both for total richness and diversity indices. Moreover, community structures obtained for the two cores were congruent as revealed by procrustean analysis. Though most of the eukaryotic groups showed no significant difference in terms of richness and relative proportion according to the core, the group of fungi was found to differ both in terms of richness and relative proportion (possibly due to their spatial heterogeneity and potential activity in sediments). Considering the OTUs level (i.e. Operational Taxonomic Units as a proxy of ecological species), our results showed that, for the older analyzed strata (age: 15–40 years), the composition and structure of communities were very similar for the two cores (except for fungi) and the DNA signal was considered stable. However, for the uppermost strata (age < 15 years), changes of moderate magnitude were detected in the relative abundance of few OTUs. Overall, this study points out that, in Nylandssjön sediments, early diagenesis did not induce marked modifications in the micro-eukaryotic DNA signal, thus opening new perspectives based on the analysis of eukaryotic sedimentary DNA to address scientific issues both in the domains of paleolimnology and microbial ecology. Because this study site is ideal for DNA preservation in sediment (quick sedimentation processes, no sediment resuspension, anoxic conditions at sediment–water interface), the generalization of our conclusions, in particular for less favorable sites, must be considered cautiously.</p><h1>Invasions Toolkit: Current Methods for Tracking the Spread and Impact of Invasive Species</h1>
<p><a href="https://doi.org/10.1016/bs.aecr.2016.10.009">doi:10.1016/bs.aecr.2016.10.009</a></p>

<p><a href=None>None</a></p>
<p>Biological invasions exert multiple pervasive effects on ecosystems, potentially disrupting species interactions and global ecological processes. Our ability to successfully predict and manage the ecosystem-level impacts of biological invasions is strongly dependent on our capacity to empirically characterize complex biological interactions and their spatiotemporal dynamics. In this chapter, we argue that the comprehensive integration of multiple complementary tools within the explicit context of ecological networks is essential for providing mechanistic insight into invasion processes and their impact across organizational levels. We provide an overview of traditional (stable isotopes, populations genetics) and emerging (metabarcoding, citizen science) techniques and methods, and their practical implementation in the context of biological invasions. We also present several currently available models and machine-learning approaches that could be used for predicting novel or undocumented interactions, thus allowing a more robust and cost-effective forecast of network and ecosystem stability. Finally, we discuss the importance of methodological advancements on the emergence of scientific and societal challenges for investigating local and global species histories with several skill sets.</p><h1>No single driver of biodiversity: Divergent responses of multiple taxa across land use types</h1>
<p><a href="https://doi.org/10.1002/ecs2.1997">doi:10.1002/ecs2.1997</a></p>

<p><a href=None>None</a></p>
<p>Understanding the responses of biodiversity to different land use regimes is critical for managing biodiversity in the face of future land use change. However, there is still significant uncertainty around how consistent the responses of different taxonomic groups to land use change are. Here, we use a combination of high-throughput environmental DNA sequencing and traditional field-based survey methods to examine how patterns of richness and community composition correlate among four domains/kingdoms (bacteria, fungi, plants, and metazoans) and the four most-abundant animal taxonomic groups (arachnids, Collembola, insects, and nematodes) across five different land use types (natural forest, planted forest, unimproved grassland, improved grassland, and vineyards). Richness for each taxonomic group varied between land use types, yet different taxa showed inconsistent responses to land use, and their richness was rarely correlated. This contrasted with community composition of taxonomic groups, for which there was relatively good discrimination of land use types and there was strong correlation between group responses. We found little evidence for consistent drivers of taxonomic richness, yet identified several significant drivers of community composition that were shared across many groups. Drivers of composition were not the same as the drivers of diversity, suggesting diversity and composition are independently controlled. While land use intensification has been viewed as having generally negative effects on biodiversity, our results provide evidence that different taxa respond divergently across different land uses. Further, our study demonstrates the power of high-throughput sequencing of environmental DNA as a tool for addressing broad ecological patterns relating to landscape biodiversity.</p><h1>Wanted dead or alive? Using metabarcoding of environmental DNA and RNA to distinguish living assemblages for biosecurity applications</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0187636">doi:10.1371/journal.pone.0187636</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing metabarcoding studies in marine biosecurity have largely focused on targeting environmental DNA (eDNA). DNA can persist extracellularly in the environment, making discrimination of living organisms difficult. In this study, bilge water samples (i.e., water accumulating on-board a vessel during transit) were collected from 15 small recreational and commercial vessels. eDNA and eRNA molecules were co-extracted and the V4 region of the 18S ribosomal RNA gene targeted for metabarcoding. In total, 62.7% of the Operational Taxonomic Units (OTUs) were identified at least once in the corresponding eDNA and eRNA reads, with 19.5% unique to eDNA and 17.7% to eRNA. There were substantial differences in diversity between molecular compartments; 57% of sequences from eDNA-only OTUs belonged to fungi, likely originating from legacy DNA. In contrast, there was a higher percentage of metazoan (50.2%) and ciliate (31.7%) sequences in the eRNA-only OTUs. Our data suggest that the presence of eRNA-only OTUs could be due to increased cellular activities of some rare taxa that were not identified in the eDNA datasets, unusually high numbers of rRNA transcripts in ciliates, and/or artefacts produced during the reverse transcriptase, PCR and sequencing steps. The proportions of eDNA/eRNA shared and unshared OTUs were highly heterogeneous within individual bilge water samples. Multiple factors including boat type and the activities performed on-board, such as washing of scientific equipment, may play a major role in contributing to this variability. For some marine biosecurity applications analysis, eDNA-only data may be sufficient, however there are an increasing number of instances where distinguishing the living portion of a community is essential. For these circumstances, we suggest only including OTUs that are present in both eDNA and eRNA data. OTUs found only in the eRNA data need to be interpreted with caution until further research provides conclusive evidence for their origin.</p><h1>Environmental DNA characterization of amphibian communities in the Brazilian Atlantic forest: Potential application for conservation of a rich and threatened fauna</h1>
<p><a href="https://doi.org/10.1016/j.biocon.2017.09.015">doi:10.1016/j.biocon.2017.09.015</a></p>

<p><a href=None>None</a></p>
<p>The Brazilian Atlantic forest is a highly threatened biodiversity hotspot, harboring one of the highest levels of amphibian species richness in the world. Amphibian conservation in Neotropical biomes is critical because freshwater ecosystems typically experience sharp declines in biodiversity before much is known about species that depend on those environments. Environmental DNA (eDNA) analysis is a new approach for monitoring aquatic organisms and provides valuable information on species occurrence in freshwater ecosystems. Here, we assess community diversity in Neotropical streams using eDNA metabarcoding. We compare data from a five-year traditional field survey with results from a short-term eDNA analysis in four streams of the Atlantic forest of southeastern Brazil. We recorded 19 species over 5 years using visual-acoustic methods, of which 10 species are associated with the streams during at least one life stage (egg, tadpole or post-metamorphic). We were able to detect nine of the latter species using eDNA metabarcoding from water samples collected over 4 days. Amphibian community composition as measured by both methods showed high similarity in three streams, but was not perfect, as eDNA failed to detect known species in a fourth stream. Furthermore, in one stream we detected through eDNA metabarcoding a species (Aplastodiscus eugenioi) found only once during the 5-year traditional survey in that stream. Also, three species (Cycloramphus boraceiensis, Hylodes asper, and Hylodes phyllodes) with the highest dependence on aquatic habitat showed the highest number of positive PCR detections on eDNA samples. Our results showed that eDNA metabarcoding can be a useful tool to assess community diversity in tropical streams, and although not perfect in detection, this method can potentially improve our ability to conserve Neotropical amphibians.</p><h1>Urban habitat restoration provides a human health benefit through microbiome rewilding: the Microbiome Rewilding Hypothesis</h1>
<p><a href="https://doi.org/10.1111/rec.12610">doi:10.1111/rec.12610</a></p>

<p><a href=None>None</a></p>
<p>Restoration aims to return ecosystem services, including the human health benefits of exposure to green space. The loss of such exposure with urbanization and industrialization has arguably contributed to an increase in human immune dysregulation. The Biodiversity and Old Friends hypotheses have described the possible mechanisms of this relationship, and suggest that reduced exposure to diverse, beneficial microorganisms can result in negative health consequences. However, it is unclear whether restoration of biodiverse habitat can reverse this effect, and what role the environmental microbiome might have in such recovery. Here, we propose the Microbiome Rewilding Hypothesis, which specifically outlines that restoring biodiverse habitats in urban green spaces can rewild the environmental microbiome to a state that enhances primary prevention of human disease. We support our hypothesis with examples from allied fields, including a case study of active restoration that reversed the degradation of the soil bacterial microbiome of a former pasture. This case study used high-throughput amplicon sequencing of environmental DNA to assess the quality of a restoration intervention in restoring the soil bacterial microbiome. The method is rapid, scalable, and standardizable, and has great potential as a monitoring tool to assess functional outcomes of green-space restoration. Evidence for the Microbiome Rewilding Hypothesis will help motivate health professionals, urban planners, and restoration practitioners to collaborate and achieve co-benefits. Co-benefits include improved human health outcomes and investment opportunities for biodiversity conservation and restoration.</p><h1>Assessing ecological status with diatoms DNA metabarcoding: Scaling-up on a WFD monitoring network (Mayotte island, France)</h1>
<p><a href="https://doi.org/10.1016/j.ecolind.2017.06.024">doi:10.1016/j.ecolind.2017.06.024</a></p>

<p><a href=None>None</a></p>
<p>Diatoms are excellent ecological indicators of water quality because they are broadly distributed, they show high species diversity and they respond rapidly to human pressures. In Europe, the Water Framework Directive (WFD) gives the legal basis for the use of this indicator for water quality assessment and its management. Several quality indices, like the Specific Polluosensitivity Index (SPI), were developed to assess the ecological quality status of rivers based on diatom communities. It is based on morphological identifications and count of diatom species present in natural biofilms using a microscope. This methodology requires high taxonomic skills and several hours of analysis per sample as 400 individuals must be identified to species level. Since several years, a molecular approach based on DNA metabarcoding combined to High-Throughput Sequencing (HTS) is developed to characterize species assemblages in environmental samples which is potentially faster and cheaper. The ability of this approach to provide reliable diatom inventories has been demonstrated and its application to water quality assessment is currently being improved. Despite optimization of the DNA metabarcoding process with diatoms, few studies had yet extended it at the scale of a freshwater monitoring network and evaluated the reliability of its quality assessment compared to the classical morphological approach. In the present study we applied DNA metabarcoding to the river monitoring network of the tropical Island Mayotte. This island is a French département since 2011 and the WFD has to be applied. This offered the opportunity to scale up the comparison of molecular and morphological approaches and their ability to produce comparable community inventories and water quality assessments. Benthic diatoms were sampled following WFD standards in 45 river sites in 2014 and 2015 (80 samples). All samples were submitted in parallel to the molecular and the morphological approaches. DNA metabarcoding was carried out using Genelute DNA extraction method, rbcL DNA barcode and PGM sequencing, while microscopic counts were carried out for the classical methodology. Diatom community structures in terms of molecular (OTUs) and of morphological (species) were significantly correlated. However, only 13% of the species was shared by both approaches, with qualitative and quantitative variation due to i) the incompleteness of the reference library (82% of morphological species are not represented in the database), ii) limits in taxonomic knowledge and iii) biases in the estimation of relative abundances linked to diatom cell biovolume. However, ecological quality status assessed with the molecular and morphological SPI values were congruent, and little affected by sequencing depth. DNA metabarcoding of diatom communities allowed a reliable estimation of the quality status for most of the rivers at the scale of the full biomonitoring network of Mayotte Island.</p><h1>Harmful non-indigenous dinoflagellate cysts in China: A review</h1>
<p><a href="https://doi.org/10.1080/14634988.2017.1403269">doi:10.1080/14634988.2017.1403269</a></p>

<p><a href=None>None</a></p>
<p>Harmful dinoflagellate cysts (e.g. dinocysts) represent a great threat to marine and coastal ecosystems and even human health. As dinocysts have resistant cell walls that allow dinoflagellate species to successfully survive harsh environments, they can easily pass through multiple barriers during biological invasions and successfully colonize new locations. In introduced ranges, dinocysts often serve as seeds for harmful algal blooms, which can result in large-scale environmental disasters and economic losses. Correct identification of dinocysts represents the first step for both ecological studies and management programs. However, traditional methods, which are based on morphological observations, pose great technical challenges for dinocysts. Recent advances in molecular techniques, such as high-throughput sequencing-based metabarcoding, have tremendously revolutionized our abilities to detect marine biodiversity. Here, we summarize species composition of non-indigenous/cryptogenic dinocysts and their identification methods commonly used in China, discuss major problems in traditional morphological identification and technical issues of molecular identification, and finally propose future perspectives for the efficient identification of non-indigenous dinocysts using high-throughput sequencing-based metabarcoding. We also discuss technical issues that should be addressed before high-throughput sequencing-based metabarcoding can be widely used for dinocyst identification at large geographical scales.</p><h1>Assessment of the metabarcoding approach for community analysis of benthic-epiphytic dinoflagellates using mock communities</h1>
<p><a href="https://doi.org/10.1080/00288330.2017.1298632">doi:10.1080/00288330.2017.1298632</a></p>

<p><a href=None>None</a></p>
<p>In this study, we assessed the use of DNA metabarcoding as a method for biodiversity assessment of benthic-epiphytic dinoflagellate communities and for detecting rare, toxin-producing taxa. Mock communities and three primer pairs were used to establish the recovery of species signal and quantitative representation of species in the samples, as well as to determine primer biases, bioinformatic filtering steps, and threshold levels. Samples were analysed using high-throughput sequencing Illumina™ MiSeq technology. We did not find a relationship between read number and cell abundance for all treatments. However, the method was extremely sensitive, with two of the primer pairs detecting a single cell representing less than 0.001% of the cells in the sample. Benthic and epiphytic dinoflagellate communities were also collected from the Bay of Islands (Northland, New Zealand). Dinophyceae species richness was much higher when samples were analysed using metabarcoding than when analysed by microscopy, and we detected several new taxonomic records for New Zealand.</p><h1>Assessing strengths and weaknesses of DNA metabarcoding-based macroinvertebrate identification for routine stream monitoring</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12789">doi:10.1111/2041-210X.12789</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding holds great promise for the assessment of macroinvertebrates in stream ecosystems. However, few large-scale studies have compared the performance of DNA metabarcoding with that of routine morphological identification. We performed metabarcoding using four primer sets on macroinvertebrate samples from 18 stream sites across Finland. The samples were collected in 2013 and identified based on morphology as part of a Finnish stream monitoring program. Specimens were morphologically classified, following standardised protocols, to the lowest taxonomic level for which identification was feasible in the routine national monitoring. DNA metabarcoding identified more than twice the number of taxa than the morphology-based protocol, and also yielded a higher taxonomic resolution. For each sample, we detected more taxa by metabarcoding than by the morphological method, and all four primer sets exhibited comparably good performance. Sequence read abundance and the number of specimens per taxon (a proxy for biomass) were significantly correlated in each sample, although the adjusted R2 values were low. With a few exceptions, the ecological status assessment metrics calculated from morphological and DNA metabarcoding datasets were similar. Given the recent reduction in sequencing costs, metabarcoding is currently approximately as expensive as morphology-based identification. Using samples obtained in the field, we demonstrated that DNA metabarcoding can achieve comparable assessment results to current protocols relying on morphological identification. Thus, metabarcoding represents a feasible and reliable method to identify macroinvertebrates in stream bioassessment, and offers powerful advantage over morphological identification in providing identification for taxonomic groups that are unfeasible to identify in routine protocols. To unlock the full potential of DNA metabarcoding for ecosystem assessment, however, it will be necessary to address key problems with current laboratory protocols and reference databases.</p><h1>Metabarcoding techniques for assessing biodiversity of marine animal forests</h1>
<p><a href="https://doi.org/10.1007/978-3-319-21012-4_53">doi:10.1007/978-3-319-21012-4_53</a></p>

<p><a href=None>None</a></p>
<p>The "marine animal forests" are among the most diverse ecosystems in the Biosphere. However, exhaustive biodiversity assessment of these communities has been so far elusive. The real extent of biodiversity and its temporal and spatial variability patterns remain unknown for most animal forests, mainly due to the inability of traditional taxonomy methods to cope with such degree of diversity and structural complexity. The development of metabarcoding techniques has revolutionized biomonitoring. Using this approach, thousands of species present in any environmental sample can be detected by high-throughput DNA sequencing and identified using public databases. Though initially limited to homogeneous substrates such as plankton or sediments, the applications of metabarcoding have been recently extended to communities on heterogeneous complex hard bottom substrates. Here we present novel metabarcoding protocols, based on the use of short fragments of 18S rRNA or cytochrome c oxidase I genes as genetic markers. We aim to develop methods for robust, reproducible eukaryotic biodiversity assessment of structurally complex communities such as marine animal forests, allowing characterization of communities living on hard-bottom substrates or other marine benthic ecosystems. We propose some guidelines focusing on sampling techniques, sample preprocessing, DNA extraction, selection of genetic markers, and bioinformatic pipelines, including steps such as sequence filtering (removal of low quality reads), clustering algorithms for delimiting molecular operational taxonomic units, and automated taxonomic assignment using reference databases. We expect these recommendations will help marine ecologists to become familiar with the paradigm shift that metabarcoding represents in the way marine ecosystems will be monitored and managed in the next future.</p><h1>DNA in a bottle - Rapid metabarcoding survey for early alerts of invasive species in ports</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0183347">doi:10.1371/journal.pone.0183347</a></p>

<p><a href=None>None</a></p>
<p>Biota monitoring in ports is increasingly needed for biosecurity reasons and safeguarding marine biodiversity from biological invasion. Present and future international biosecurity directives can be accomplished only if the biota acquired by maritime traffic in ports is controlled. Methodologies for biota inventory are diverse and now rely principally on extensive and labor-intensive sampling along with taxonomic identification by experts. In this study, we employed an extremely simplified environmental DNA (eDNA) sampling methodology from only three 1-L bottles of water per port, followed by metabarcoding (high-throughput sequencing and DNA-based species identification) using 18S rDNA and Cytochrome oxidase I as genetic barcodes. Eight Bay of Biscay ports with available inventory of fouling invertebrates were employed as a case study. Despite minimal sampling efforts, three invasive invertebrates were detected: the barnacle Austrominius modestus, the tubeworm Ficopomatus enigmaticus and the polychaete Polydora triglanda. The same species have been previously found from visual and DNA barcoding (genetic identification of individuals) surveys in the same ports. The current costs of visual surveys, conventional DNA barcoding and this simplified metabarcoding protocol were compared. The results encourage the use of metabarcoding for early biosecurity alerts.</p><h1>Distribution patterns of soil microbial eukaryotes suggests widespread algivory by phagotrophic protists as an alternative pathway for nutrient cycling</h1>
<p><a href="https://doi.org/10.1016/j.soilbio.2017.05.002">doi:10.1016/j.soilbio.2017.05.002</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing (HTS) of soil environmental DNA (eDNA) allows assessing the full diversity of soil micro-eukaryotes. The resulting operational taxonomic units (OTUs) can be assigned to potential taxonomic and functional identities using increasingly complete reference databases. HTS of soil eDNA is revealing a high diversity and abundance of potential eukaryovorous protists, thus challenging the paradigm of the predominantly bacterivorous function of soil phagotrophic protists (i.e. microbial loop). Using Illumina sequencing of soil eDNA and targeting the V9 region of the SSU rRNA gene, we investigated the taxonomic and functional diversities, distribution and co-occurrence patterns of soil micro-eukaryotes in three land-use categories: forests, meadows and croplands located in Switzerland. Each OTU was assigned to a broad functional category (phototrophs, phagotrophs, osmotrophs, or parasites). Total OTU richness was similar in the three land-use categories, but community composition differed significantly between forests and other land-uses. The proportion of fungal sequences (especially Basidiomycota) was highest, and phototroph (i.e. soil microalgae) sequences least abundant in forests. Seven OTUs representing phagotrophic protists, together accounting for >25% of all phagotroph sequences, were significantly correlated to the total number of phototroph sequences, thus suggesting algivory. At least three of these OTUs corresponded to known algal predators. These results suggest that beyond plants, soil microalgae represent a functionally significant but rarely considered input of carbon in soils that should be taken into account when modelling soil nutrient cycling.</p><h1>Metabarcoding reveals seasonal and temperature-dependent succession of zooplankton communities in the red sea</h1>
<p><a href="https://doi.org/10.3389/fmars.2017.00241">doi:10.3389/fmars.2017.00241</a></p>

<p><a href=None>None</a></p>
<p>Very little is known about the composition and the annual cycle of zooplankton assemblages in the Red Sea, a confined water body characterized by a high biodiversity and endemism but at the same time one of the most understudied areas in the world in terms of marine biodiversity. This high diversity together with the lack of references for several of the groups poses a problem in obtaining basic information on zooplankton seasonal patterns. In the present work, we used high throughput sequencing to examine the temporal and spatial distribution of the zooplankton communities inhabiting the epipelagic zone in the central Red Sea. The analysis of zooplankton assemblages collected at two sites-coastal and offshore-twice a month at several depth strata by using MANTA, Bongo and WP2 nets provides baseline information of the seasonal patterns of the zooplankton community over 1 year. We show that the seasonal fluctuation of zooplankton communities living in the upper 100 m of the water column is driven mainly by the annual changes in seawater temperature. The 18S rRNA gene was used for metabarcoding of zooplankton assemblages revealing 630 metazoan OTUs (97% similarity) in five phyla, highlighting the richness of the Red Sea community. During colder months, communities were characterized by lower richness and higher biomass than communities found during the hot season. Throughout the year the zooplankton communities were dominated by the class Maxillopoda, mainly represented by copepods and class Hydrozoa. The rise in the water temperature favors the appearance of classes Malacostraca, Cephalopoda, Gastropoda, and Saggitoidea. The present study provides essential baseline information for future monitoring and improves our knowledge of the marine ecosystem in the Red Sea while reporting the main environmental variable structuring zooplankton assemblages in this region.</p><h1>Haptophyte Diversity and Vertical Distribution Explored by 18S and 28S Ribosomal RNA Gene Metabarcoding and Scanning Electron Microscopy</h1>
<p><a href="https://doi.org/10.1111/jeu.12388">doi:10.1111/jeu.12388</a></p>

<p><a href=None>None</a></p>
<p>Haptophyta encompasses more than 300 species of mostly marine pico- and nanoplanktonic flagellates. Our aims were to investigate the Oslofjorden haptophyte diversity and vertical distribution by metabarcoding, and to improve the approach to study haptophyte community composition, richness and proportional abundance by comparing two rRNA markers and scanning electron microscopy (SEM). Samples were collected in August 2013 at the Outer Oslofjorden, Norway. Total RNA/cDNA was amplified by haptophyte-specific primers targeting the V4 region of the 18S, and the D1-D2 region of the 28S rRNA. Taxonomy was assigned using curated haptophyte reference databases and phylogenetic analyses. Both marker genes showed Chrysochromulinaceae and Prymnesiaceae to be the families with highest number of Operational Taxonomic Units (OTUs), as well as proportional abundance. The 18S rRNA data set also contained OTUs assigned to eight supported and defined clades consisting of environmental sequences only, possibly representing novel lineages from family to class. We also recorded new species for the area. Comparing coccolithophores by SEM with metabarcoding shows a good correspondence with the 18S rRNA gene proportional abundances. Our results contribute to link morphological and molecular data and 28S to 18S rRNA gene sequences of haptophytes without cultured representatives, and to improve metabarcoding methodology.</p><h1>Freshwater biomonitoring in the Information Age</h1>
<p><a href="https://doi.org/10.1002/fee.1490">doi:10.1002/fee.1490</a></p>

<p><a href=None>None</a></p>
<p>Freshwaters worldwide face serious threats, making their protection increasingly important. Freshwater monitoring has historically produced valuable data and continues to develop. Rapid improvements to biomolecular techniques are revolutionizing the way scientists describe biological communities and are bringing about major changes in biomonitoring. Combined with high-throughput sequencing, DNA metabarcoding is fast and cost-effective, generating massive amounts of data. In a world with numerous ecological threats, “big data” constitute a tremendous opportunity to improve the efficiency of biological monitoring. These fundamental changes in biomonitoring will require freshwater ecologists and environmental managers to reconsider how they handle large amounts of data.</p><h1>DNA sequencing as a tool to monitor marine ecological status</h1>
<p><a href="https://doi.org/10.3389/fmars.2017.00107">doi:10.3389/fmars.2017.00107</a></p>

<p><a href=None>None</a></p>
<p>Many ocean policies mandate integrated, ecosystem-based approaches to marine monitoring, driving a global need for efficient, low-cost bioindicators of marine ecological quality. Most traditional methods to assess biological quality rely on specialized expertise to provide visual identification of a limited set of specific taxonomic groups, a time-consuming process that can provide a narrow view of ecological status. In addition, microbial assemblages drive food webs but are not amenable to visual inspection and thus are largely excluded from detailed inventory. Molecular-based assessments of biodiversity and ecosystem function offer advantages over traditional methods and are increasingly being generated for a suite of taxa using a "microbes to mammals" or "barcodes to biomes" approach. Progress in these efforts coupled with continued improvements in high-throughput sequencing and bioinformatics pave the way for sequence data to be employed in formal integrated ecosystem evaluation, including food web assessments, as called for in the European Union Marine Strategy Framework Directive. DNA sequencing of bioindicators, both traditional (e.g., benthic macroinvertebrates, ichthyoplankton) and emerging (e.g., microbial assemblages, fish via eDNA), promises to improve assessment of marine biological quality by increasing the breadth, depth, and throughput of information and by reducing costs and reliance on specialized taxonomic expertise.</p><h1>PrimerMiner: an r package for development and in silico validation of DNA metabarcoding primers</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12687">doi:10.1111/2041-210X.12687</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding is a powerful tool to assess biodiversity by amplifying and sequencing a standardized gene marker region. Its success is often limited due to variable binding sites that introduce amplification biases. Thus, the development of optimized primers for communities or taxa under study in a certain geographic region and/or ecosystems is of critical importance. However, no tool for obtaining and processing of reference sequence data in bulk that can serve as a backbone for primer design is currently available. We developed the r package PrimerMiner, which batch downloads DNA barcode gene sequences from BOLD and NCBI data bases for specified target taxonomic groups and then applies sequence clustering into operational taxonomic units (OTUs) to reduce biases introduced by the different number of available sequences per species. Additionally, PrimerMiner offers functionalities to evaluate primers in silico, which are in our opinion more realistic than the strategy employed in another available software for that purpose, ecoPCR. We used PrimerMiner to download cytochrome c oxidase subunit I (COI) sequences for 15 important freshwater invertebrate groups, relevant for ecosystem assessment. By processing COI markers from both data bases, we were able to increase the amount of reference data 249-fold on average, compared to using complete mitochondrial genomes alone. Furthermore, we visualized the generated OTU sequence alignments and describe how to evaluate primers in silico using PrimerMiner. With PrimerMiner, we provide a useful tool to obtain relevant sequence data for targeted primer development and evaluation. The OTU-based reference alignments generated with PrimerMiner can be used for manual primer design or processed with bioinformatic tools for primer development.</p><h1>Application of high-throughput sequencing (HTS) metabarcoding to diatom biomonitoring: Do DNA extraction methods matter?</h1>
<p><a href="https://doi.org/10.1086/690649">doi:10.1086/690649</a></p>

<p><a href=None>None</a></p>
<p>Current freshwater biomonitoring with diatoms is based on microscopic examination of the morphology of their silica skeleton. This standardized approach is time consuming and requires a high degree of taxonomic expertise. Metabarcoding combined with high-throughput sequencing (HTS) has great potential for next-generation biomonitoring applications but requires standardization.Molecular inventories are strongly influenced by the DNA extraction method used, but the effect of extraction protocols has not been tested to enable selection of the best DNA extraction method for HTS metabarcoding. We used 5 DNA extraction methods combining various types of cell lysis and DNA purification to extract DNA from 8 pure diatom cultures and 8 samples from streams and lakes with differing water quality. We compared the methods based on: 1) quality and purity of the extracted DNA, 2) community inventories obtained from HTS targeting the ribulose-1, 5-bisphosphate carboxylase (rbcL) barcode, and 3) similarity between molecular and microscopy-based inventories of community composition and the Specific Pollutionsensitivity Index [SPI]. A method based on GenElute-LPA had higher extraction efficiency than the 4 commercial kits but had the highest polymerase chain reaction inhibition level. All 5 methods were efficient for HTS, and method did not affect operational taxonomic unit richness. We observed variations in the relative abundance of some taxa within Nitzschia, Amphora, Encyonema, Gomphonema, and Navicula between 2 of the 5 methods, but method did not affect global diatom community composition or SPI values. SPI values calculated from microscopy-based inventories and molecular inventories based on all 5 extraction methods were strongly correlated. For convenience purposes (high DNA quantity and low cost), we encourage standardization of HTS diatom biomonitoring based on the SA-Gen method.</p><h1>A primer to metabarcoding surveys of Antarctic terrestrial biodiversity</h1>
<p><a href="https://doi.org/10.1017/S0954102016000389">doi:10.1017/S0954102016000389</a></p>

<p><a href=None>None</a></p>
<p>Ice-free regions of Antarctica are concentrated along the coastal margins but are scarce throughout the continental interior. Environmental changes, including the introduction of non-indigenous species, increasingly threaten these unique habitats. At the same time, the unique biotic communities subsisting in isolation across the continent are difficult to survey due to logistical constraints, sampling challenges and problems related to the identification of small and cryptic taxa. Baseline biodiversity data from remote Antarctic habitats are still missing for many parts of the continent but are critical to the detection of community changes over time, including newly introduced species. Here we review the potential of standardized (non-specialist) sampling in the field (e.g. from soil, vegetation or water) combined with high-throughput sequencing (HTS) of bulk DNA as a possible solution to overcome some of these problems. In particular, HTS metabarcoding approaches benefit from being able to process many samples in parallel, while workflow and data structure can stay highly uniform. Such approaches have quickly gained recognition and we show that HTS metabarcoding surveys are likely to play an important role in continent-wide biomonitoring of all Antarctic terrestrial habitats.</p><h1>Distribution and diversity of Phytophthora across Australia</h1>
<p><a href="https://doi.org/10.1071/PC16032">doi:10.1071/PC16032</a></p>

<p><a href=None>None</a></p>
<p>The introduction and subsequent impact of Phytophthora cinnamomi within native vegetation is one of the major conservation issues for biodiversity in Australia. Recently, many new Phytophthora species have been described from Australia's native ecosystems; however, their distribution, origin, and potential impact remain unknown. Historical bias in Phytophthora detection has been towards sites showing symptoms of disease, and traditional isolation methods show variable effectiveness of detecting different Phytophthora species. However, we now have at our disposal new techniques based on the sampling of environmental DNA and metabarcoding through the use of high-throughput sequencing. Here, we report on the diversity and distribution of Phytophthora in Australia using metabarcoding of 640 soil samples and we compare the diversity detected using this technique with that available in curated databases. Phytophthora was detected in 65% of sites, and phylogenetic analysis revealed 68 distinct Phytophthora phylotypes. Of these, 21 were identified as potentially unique taxa and 25 were new detections in natural areas and/or new introductions to Australia. There are 66Phytophthora taxa listed in Australian databases, 43 of which were also detected in this metabarcoding study. This study revealed high Phytophthora richness within native vegetation and the additional records provide a valuable baseline resource for future studies. Many of the Phytophthora species now uncovered in Australia's native ecosystems are newly described and until more is known we need to be cautious with regard to the spread and conservation management of these new species in Australia's unique ecosystems.</p><h1>Using DNA metabarcoding to assess New Zealand’s terrestrial biodiversity</h1>
<p><a href="https://doi.org/10.20417/nzjecol.41.28">doi:10.20417/nzjecol.41.28</a></p>

<p><a href=None>None</a></p>
<p>High throughput DNA sequencing technology has enabled entire biological communities to be characterised from DNA derived from pools of organisms, such as bulk-collected invertebrates, or DNA extracted from environmental samples (e.g. soil). These DNA-based techniques have the potential to revolutionise biodiversity monitoring. One approach in particular, DNA metabarcoding, can provide unprecedented taxonomic breadth at a scale not practically achievable through the morphological identification of individual organisms. Here, we assess the current strengths and weaknesses of DNA metabarcoding techniques for biodiversity assessment. We argue that it is essential to integrate conventional monitoring methods with novel DNA methods, to validate methods, and to better use and interpret data. We present a conceptual framework for how this might be done, explore potential applications within national biodiversity assessment frameworks, Maori biodiversity monitoring and the primary sector, and highlight areas of current uncertainty and future research directions. Rapid developments in DNA sequencing technology and bioinformatics will make DNA-based community data increasingly accessible to ecologists, and there needs to be a corresponding shift in research focus from DNA metabarcoding method development and evaluation to real-world applications that provide rich information for a range of purposes, including conservation planning and land management decisions.</p><h1>Evaluation of methods for the concentration and extraction of viruses from sewage in the context of metagenomic sequencing</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0170199">doi:10.1371/journal.pone.0170199</a></p>

<p><a href=None>None</a></p>
<p>Viral sewage metagenomics is a novel field of study used for surveillance, epidemiological studies, and evaluation of waste water treatment efficiency. In raw sewage human waste is mixed with household, industrial and drainage water, and virus particles are, therefore, only found in low concentrations. This necessitates a step of sample concentration to allow for sensitive virus detection. Additionally, viruses harbor a large diversity of both surface and genome structures, which makes universal viral genomic extraction difficult. Current studies have tackled these challenges in many different ways employing a wide range of viral concentration and extraction procedures. However, there is limited knowledge of the efficacy and inherent biases associated with these methods in respect to viral sewage metagenomics, hampering the development of this field. By the use of next generation sequencing this study aimed to evaluate the efficiency of four commonly applied viral concentrations techniques (precipitation with polyethylene glycol, organic flocculation with skim milk, monolithic adsorption filtration and glass wool filtration) and extraction methods (Nucleospin RNA XS, QIAamp Viral RNA Mini Kit, NucliSENS® miniMAG®, or PowerViral® Environmental RNA/DNA Isolation Kit) to determine the viriome in a sewage sample. We found a significant influence of concentration and extraction protocols on the detected viriome. The viral richness was largest in samples extracted with QIAamp Viral RNA Mini Kit or PowerViral® Environmental RNA/DNA Isolation Kit. Highest viral specificity were found in samples concentrated by precipitation with polyethylene glycol or extracted with Nucleospin RNA XS. Detection of viral pathogens depended on the method used. These results contribute to the understanding of method associated biases, within the field of viral sewage metagenomics, making evaluation of the current literature easier and helping with the design of future studies.</p><h1>Clay-to-Carbon Ratio Controls the Effect of Herbicide Application on Soil Bacterial Richness and Diversity in a Loamy Field</h1>
<p><a href="https://doi.org/10.1007/s11270-016-3175-6">doi:10.1007/s11270-016-3175-6</a></p>

<p><a href=None>None</a></p>
<p>Soil texture and soil organic carbon (OC) influence the bacterial microenvironment and also control herbicide sorption. A field-scale exploratory study was conducted to investigate the potential interaction between soil texture parameters, herbicides, and soil bacterial richness and diversity. Glyphosate and bentazon were used to evaluate the herbicidal effect on bacterial community under different conditions created by clay and OC gradients in a loamy field. Metabarcoding by high-throughput sequencing of bacterial rDNA was used to estimate bacterial richness and diversity using OTUs, abundance-based coverage (ACE), Shannon diversity index, and phylogenetic diversity. In general, bacterial richness and diversity increased after bentazon application and decreased after glyphosate application. There was no significant effect for field locations with Dexter n (the ratio between clay and OC) values below 4.04 (the median of the values in the field study). The correlation coefficient (r) between bacterial richness and clay decreased after bentazon application, but increased after glyphosate application. Correlations between Dexter n and bacterial indices followed the same pattern, decreasing after bentazon application and increasing after glyphosate application. This indicated that the specific chemical nature of individual herbicides affected bacterial communities. This study reinforced the importance of including soil physical and chemical characteristics to explain the influence of pesticides on the variation in soil bacterial communities in agroecosystems.</p><h1>Early detection of aquatic invaders using metabarcoding reveals a high number of non-indigenous species in Canadian ports</h1>
<p><a href="https://doi.org/10.1111/ddi.12465">doi:10.1111/ddi.12465</a></p>

<p><a href=None>None</a></p>
<p>Aim: Invasive species represent one of the greatest threats to biodiversity. The ability to detect non-indigenous species (NIS), particularly those present at low abundance, is limited by difficulties in performing exhaustive sampling and in identifying species. Here we sample zooplankton from 16 major Canadian ports and apply a metabarcoding approach to detect NIS. Location: Marine and freshwater ports along Canadian coastlines (Pacific, Arctic, Atlantic) and the Great Lakes. Methods: We amplified the V4 region of the small subunit ribosomal DNA (18S) and used two distinct analytic protocols to identify species present at low abundance. Taxonomic assignment was conducted using BLAST searches against a local 18S sequence database of either (i) individual reads (totalling 7,733,541 reads) or (ii) operational taxonomic units (OTUs) generated by sequence clustering. Phylogenetic analyses were performed to confirm the identity of reads with ambiguous taxonomic assignment. Results: Taxonomic assignment of individual reads identified 379 zooplankton species at a minimum sequence identity of 97%. Of these, 24 species were identified as NIS, 11 of which were detected in previously unreported locations. When reads were clustered into OTUs prior to taxonomic assignment, six NIS were no longer detected and an additional NIS was falsely identified. Phylogenetic analyses revealed that sequences belonging to closely related species clustered together into shared OTUs as a result of low interspecific variation. NIS can thus be misidentified when their sequences join the OTUs of more abundant native species. Main conclusions: Our results reveal the power of the metabarcoding approach, whilst also highlighting the need to account for potentially low levels of genetic diversity when processing data, to use barcode markers that allow differentiation of closely related species and to continue building comprehensive sequence databases that allow reliable and fine-scale taxonomic designation.</p><h1>A new way to contemplate darwin’s tangled bank: How DNA barcodes are reconnecting biodiversity science and biomonitoring</h1>
<p><a href="https://doi.org/10.1098/rstb.2015.0330">doi:10.1098/rstb.2015.0330</a></p>

<p><a href=None>None</a></p>
<p>Encompassing the breadth of biodiversity in biomonitoring programmes has been frustrated by an inability to simultaneously identify large numbers of species accurately and in a timely fashion. Biomonitoring infers the state of an ecosystem from samples collected and identified using the best available taxonomic knowledge. The advent of DNA barcoding has now given way to the extraction of bulk DNA from mixed samples of organisms in environmental samples through the development of high-throughput sequencing (HTS). This DNA metabarcoding approach allows an unprecedented view of the true breadth and depth of biodiversity, but its adoption poses two important challenges. First, bioinformatics techniques must simultaneously perform complex analyses of large datasets and translate the results of these analyses to a range of users. Second, the insights gained from HTS need to be amalgamated with concepts such as Linnaean taxonomy and indicator species, which are less comprehensive but more intuitive. It is clear that we are moving beyond proof-of-concept studies to address the challenge of implementation of this new approach for environmental monitoring and regulation. Interpreting Darwin’s ‘tangled bank’ through a DNA lens is now a reality, but the question remains: how can this information be generated and used reliably, and how does it relate to accepted norms in ecosystem study?.</p><h1>Freshwater vertebrate metabarcoding on Illumina platforms using double-indexed primers of the mitochondrial 16S rRNA gene</h1>
<p><a href="https://doi.org/10.1007/s12686-016-0550-y">doi:10.1007/s12686-016-0550-y</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding is a promising tool for biodiversity inventories and other applications in conservation genetics. We developed a new pair of primers for efficient and affordable high-throughput analysis of a 250 base pair stretch of DNA from the mitochondrial 16S rRNA gene of vertebrates, especially amphibians and fishes. By adapting a double-indexed protocol for Illumina platforms, our approach allows pooling of hundreds of samples in a single sequencing run. We obtained high detection rates of 82–93 % for fish in two German streams, 70 % for mock mixes of DNA from amphibians and fishes, and could distinguish multiple gene copies in amphibians, probably caused by nuclear-mitochondrial transposed DNA or heteroplasmy.</p><h1>Metabarcoding and mitochondrial metagenomics of endogean arthropods to unveil the mesofauna of the soil</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12557">doi:10.1111/2041-210X.12557</a></p>

<p><a href=None>None</a></p>
<p>Biological communities inhabiting the soil are among the most diversified, complex and yet most poorly studied terrestrial ecosystems. The greatest knowledge gaps apply to the arthropod mesofauna (0·1–2 mm body size) because conventional morphological and molecular approaches are in many cases insufficient for the characterisation of these complex communities. The development of high-throughput sequencing (HTS) methodologies is required to solve current impediments and to further advance our understanding of below-ground biodiversity. We propose a flotation–Berlese–flotation (FBF) protocol for sampling and specimen processing to obtain ‘clean’ DNA extractions of arthropod mesofauna from the soil. In addition, we developed and tested HTS protocols for the characterisation of arthropod communities from these bulk DNA extractions using cox1 metabarcoding and shotgun metagenomic sequencing on the MiSeq Illumina platform. The FBF protocol provided DNA of soil arthropods from sufficiently large volumes of soil and free from contaminating bacteria and inhibitors. Metabarcoding and metagenomic sequencing on two deep soil samples from Iberian grasslands revealed >100 species of Acari and Collembola from 28 families. Genome assembly straight from shotgun sequencing of bulk specimens produced partial and full mitogenomes for 54 species with average length of >6000 bp. Metabarcoding and metagenomic sequencing resulted in closely congruent OTUs, but species numbers were highest with metabarcoding, while ∼73% of species were confirmed by matching shotgun sequence reads and ∼48% by contig assembly from those shotgun reads. In combination, the FBF protocol together with the PCR-based and shotgun sequencing pipelines addressed most of the challenges of studying soil arthropod mesofauna on the MiSeq Illumina platform. They are powerful, cost-efficient tools for characterising soil diversity in a phylogenetic and community ecology context. These methodological developments of HTS approaches for the study of mesofauna will accelerate ecological and evolutionary studies, biomonitoring of soil arthropods, and progress in both theoretical and applied soil science.</p><h1>Metabarcoding of marine zooplankton: Prospects, progress and pitfalls</h1>
<p><a href="https://doi.org/10.1093/plankt/fbw023">doi:10.1093/plankt/fbw023</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding (large-scale taxonomic identification of complex samples via analysis of one or few orthologous DNA regions, called barcodes) is revolutionizing analysis of biodiversity of marine zooplankton assemblages. Metabarcoding relies on high-throughput DNA sequencing (HTS) technologies, which yield millions of DNA sequences in parallel and allow large-scale analysis of environmental samples. Metabarcoding studies of marine zooplankton have used various regions of nuclear small-(18S) and large-subunit (28S) rRNA, which allow accurate classification of novel sequences and reliable amplification with consensus primers, but-due to their relatively conserved nature-may underestimate species diversity in a community. To discriminate species, more variable genes are needed. A limited number of metabarcoding studies have used mitochondrial cytochrome oxidase I (COI), which ensures detection of species-level diversity, but may require group-specific primers and thus result in inconsistent amplification success rates. Reference databases with sequences for accurately-identified species are critically needed to allow taxonomic designation of molecular operational taxonomic units (MOTU) and comparison with previous studies of zooplankton diversity. Potential and promising applications of metabarcoding include rapid detection of impacts of climate change, monitoring and assessment of ecosystem health, calculation of biotic indices, characterization of food webs and detection of introduced, non-indigenous species.</p><h1>DNA-based identification of aquatic invertebrates - Useful in the South African context?</h1>
<p><a href="https://doi.org/10.17159/sajs.2016/20150444">doi:10.17159/sajs.2016/20150444</a></p>

<p><a href=None>None</a></p>
<p>The concept of using specific regions of DNA to identify organisms - processes such as DNA barcoding - is not new to South African biologists. The African Centre for DNA Barcoding reports that 12 548 plant species and 1493 animal species had been barcoded in South Africa by July 2013, while the Barcode of Life Database (BOLD) contains 62 926 records for South Africa, 11 392 of which had species names (representing 4541 species). In light of this, it is surprising that aquatic macroinvertebrates of South Africa have not received much attention as potential barcoding projects thus far - barcoding of aquatic species has tended to focus on invasive species and fishes. Perusal of the BOLD records for South Africa indicates a noticeable absence of aquatic macroinvertebrates, including families used for biomonitoring strategies such as the South African Scoring System. Meanwhile, the approach of collecting specimens and isolating their DNA individually in order to identify them (as in the case of DNA barcoding), has been shifting towards making use of the DNA which organisms naturally shed into their environments (eDNA). Coupling environmental and bulk sample DNA with high-throughput sequencing technology has given rise to metabarcoding, which has the potential to characterise the whole community of organisms present in an environment. Harnessing barcoding and metabarcoding approaches with environmental DNA (eDNA) potentially offers a non-invasive means of measuring the biodiversity in an environment and has great potential for biomonitoring. Aquatic ecosystems are well suited to these approaches - but could they be useful in a South African context?</p><h1>Metabarcoding reveals strong spatial structure and temporal turnover of zooplankton communities among marine and freshwater ports</h1>
<p><a href="https://doi.org/10.1111/ddi.12427">doi:10.1111/ddi.12427</a></p>

<p><a href=None>None</a></p>
<p>Aim: The urgent need for large-scale spatio-temporal assessments of biodiversity in the face of rapid environmental change prompts technological advancements in species identification and biomonitoring such as metabarcoding. The high-throughput DNA sequencing of bulk samples offers many advantages over traditional morphological identification for describing community composition. Our objective was to evaluate the applicability of metabarcoding to identify species in taxonomically complex samples, evaluate biodiversity trends across broad geographical and temporal scales and facilitate cross-study comparisons. Location: Marine and freshwater ports along Canadian coastlines (Pacific, Arctic and Atlantic) and the Great Lakes. Methods: We used metabarcoding of bulk zooplankton samples to identify species and profile biodiversity across habitats and seasons in busy commercial ports. A taxonomic assignment approach circumventing sequence clustering was implemented to provide increased resolution and accuracy compared to pre-clustering. Results: Taxonomic classification of over seven million sequences identified organisms spanning around 400 metazoan families and complements previous surveys based on morphological identification. Metabarcoding revealed over 30 orders that were previously not reported, while certain taxonomic groups were underrepresented because of depauperate reference databases. Despite the limitations of assigning metabarcoding data to the species level, zooplankton communities were distinct among coastlines and significantly divergent among marine, freshwater and estuarine habitats even at the family level. Furthermore, biodiversity varied substantially across two seasons reaching a beta diversity of 0.9 in a sub-Arctic port exposed to high vessel traffic. Main Conclusions: Metabarcoding offers a powerful and sensitive approach to conduct large-scale biodiversity surveys and allows comparability across studies when rooted in taxonomy. We highlight ways of overcoming current limitations of metabarcoding for identifying species and assessing biodiversity, which has important implications for detecting organisms at low abundance such as endangered species and early invaders. Our study conveys pertinent and timely considerations for future large-scale monitoring surveys in relationship to environmental change.</p><h1>Extracellular DNA extraction is a fast, cheap and reliable alternative for multi-taxa surveys based on soil DNA</h1>
<p><a href="https://doi.org/10.1016/j.soilbio.2016.01.008">doi:10.1016/j.soilbio.2016.01.008</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding on soil samples is increasingly used for large-scale and multi-taxa biodiversity studies. However, DNA extraction may be a major bottleneck for such wide uses. It should be cost/time effective and allow dealing with large sample volumes so as to maximise the representativeness of both micro- and macro-organisms diversity. Here, we compared the performances of a fast and cheap extracellular DNA extraction protocol with a total DNA extraction method in retrieving bacterial, eukaryotic and plant diversity from tropical soil samples of ca. 10 g. The total DNA extraction protocol yielded more high-quality DNA. Yet, the extracellular DNA protocol provided similar diversity assessments although it presented some differences in clades relative abundance and undersampling biases. We argue that extracellular DNA is a good compromise between cost, labor, and accuracy for high-throughput DNA metabarcoding studies of soil biodiversity.</p><h1>A cost-effective and efficient strategy for Illumina sequencing of fungal communities: A case study of beech endophytes identified elevation as main explanatory factor for diversity and community composition</h1>
<p><a href="https://doi.org/10.1016/j.funeco.2015.12.009">doi:10.1016/j.funeco.2015.12.009</a></p>

<p><a href=None>None</a></p>
<p>We describe an accurate and efficient workflow for highly multiplexed paired-end Illumina sequencing of fungal full-length ITS amplicons. The impact of habitat and substratum conditions on leaf-inhabiting fungal communities was analysed. Fully vital and clearly senescent leaves of European beech (Fagus sylvatica) were sampled along an elevation gradient of about 1000 m in the Bavarian Alps, Germany, in autumn 2013. Surface-sterilised leaves were used for genomic DNA extraction, tagging-by-amplification and high-throughput sequencing. Significant correlation of community composition with elevation was observed. The mycobiome was little affected by the physiological state of the leaves, because only a partial shift of taxonomic composition was observed from vital towards clearly senescent leaves.</p><h1>Antarctic eukaryotic soil diversity of the Prince Charles Mountains revealed by high-throughput sequencing</h1>
<p><a href="https://doi.org/10.1016/j.soilbio.2015.12.013">doi:10.1016/j.soilbio.2015.12.013</a></p>

<p><a href=None>None</a></p>
<p>Studies of Antarctic eukaryotes have been hampered by their morphological conservatism, small size and the logistical constraints of remote field work, resulting in a deficiency of baseline biodiversity information about Antarctic terrestrial environments. The application of high throughput sequencing (HTS) in metataxonomic approaches is a promising alternative. Here, we apply such HTS approaches to the hitherto largely unsurveyed micro-eukaryote fauna of the Prince Charles Mountains, East Antarctica. We sequenced 18S rDNA amplicons of twelve Antarctic bulk-soil DNA extracts, retrieved from three sampling regions (four bulk-soil extracts per sampling region). After isolating eukaryotic phylotypes with a stringent filtering approach and initial network visualization, we firstly used rarefied data to compare four α diversity metrics between the three regions. Weighted and unweighted inter-sample UniFrac distances were then used for β diversity comparisons among rarefied data. Furthermore, we analysed the distribution of the most abundant phylotypes and phylotype groups. Lastly, we checked the validity of species-level taxonomic assignments using different sets of reference data in conjunction with two different taxonomy assignment approaches. Phylotype numbers in un-rarefied data compared across regions were lowest for Mount Menzies (73°S; 3330 m), intermediate at Mawson Escarpment (73°S; 807 m) and highest at Lake Terrasovoje (70°S; 173 m), likely due to low biological load at the higher latitude and elevation inland sites. Analysis of rarefied data was difficult due to low sequence coverage particularly from Mount Menzies, but indicated differences in Shannon diversity between Mawson Escarpment and Lake Terrasovoje. PCoA of weighted UniFrac distances between samples from Mawson Escarpment and Lake Terrasovoje indicated changes in community composition in relation to elevation of the sampling locations. The most widespread phylotypes were fungal, followed by non-algal protists. Species-level assignments included known Antarctic taxa in all sampling regions. We show that HTS can provide a rapid survey of the micro-eukaryote fauna to provide baseline biodiversity information for remote, harsh, and hitherto largely unsurveyed environments in the Prince Charles Mountains.</p><h1>Can theory improve the scope of quantitative metazoan metabarcoding?</h1>
<p><a href="https://doi.org/10.3390/d8010001">doi:10.3390/d8010001</a></p>

<p><a href=None>None</a></p>
<p>Using high-throughput sequencing approaches to quantify biodiversity has a number of hurdles, in particular that the number of reads for a given taxon may not be proportional to the number of individuals of that taxon in a sample. Here, we consider whether summary statistics generated in the course of population genetic analyses (such as estimates of haplotype diversity and mutation rate) may be useful in reverse inference of the number of individuals input to an assay. Although our results show that these statistics-combined with the observed number of segregating sites and number of haplotypes in the assay-may be informative, there remain significant concerns about the ability to "metabarcode" a sample and infer relative species abundance.</p><h1>Culture-independent discovery of natural products from soil metagenomes</h1>
<p><a href="https://doi.org/10.1007/s10295-015-1706-6">doi:10.1007/s10295-015-1706-6</a></p>

<p><a href=None>None</a></p>
<p>Bacterial natural products have proven to be invaluable starting points in the development of many currently used therapeutic agents. Unfortunately, traditional culture-based methods for natural product discovery have been deemphasized by pharmaceutical companies due in large part to high rediscovery rates. Culture-independent, or “metagenomic,” methods, which rely on the heterologous expression of DNA extracted directly from environmental samples (eDNA), have the potential to provide access to metabolites encoded by a large fraction of the earth’s microbial biosynthetic diversity. As soil is both ubiquitous and rich in bacterial diversity, it is an appealing starting point for culture-independent natural product discovery efforts. This review provides an overview of the history of soil metagenome-driven natural product discovery studies and elaborates on the recent development of new tools for sequence-based, high-throughput profiling of environmental samples used in discovering novel natural product biosynthetic gene clusters. We conclude with several examples of these new tools being employed to facilitate the recovery of novel secondary metabolite encoding gene clusters from soil metagenomes and the subsequent heterologous expression of these clusters to produce bioactive small molecules.</p><h1>Benchmarking DNA metabarcoding for biodiversity-based monitoring and assessment</h1>
<p><a href="https://doi.org/10.3389/fmars.2016.00096">doi:10.3389/fmars.2016.00096</a></p>

<p><a href=None>None</a></p>
<p>Characterization of biodiversity has been extensively used to confidently monitor and assess environmental status. Yet, visual morphology, traditionally and widely used for species identification in coastal and marine ecosystem communities, is tedious and entails limitations. Metabarcoding coupled with high-throughput sequencing (HTS) represents an alternative to rapidly, accurately, and cost-effectively analyze thousands of environmental samples simultaneously, and this method is increasingly used to characterize the metazoan taxonomic composition of a wide variety of environments. However, a comprehensive study benchmarking visual and metabarcoding-based taxonomic inferences that validates this technique for environmental monitoring is still lacking. Here, we compare taxonomic inferences of benthic macroinvertebrate samples of known taxonomic composition obtained using alternative metabarcoding protocols based on a combination of different DNA sources, barcodes of the mitochondrial cytochrome oxidase I gene and amplification conditions. Our results highlight the influence of the metabarcoding protocol in the obtained taxonomic composition and suggest the better performance of an alternative 313 bp length barcode to the traditionally 658 bp length one used for metazoan metabarcoding. Additionally, we show that a biotic index inferred from the list of macroinvertebrate taxa obtained using DNA-based taxonomic assignments is comparable to that inferred using morphological identification. Thus, our analyses prove metabarcoding valid for environmental status assessment and will contribute to accelerating the implementation of this technique to regular monitoring programs.</p><h1>The effects of parameter choice on defining molecular operational taxonomic units and resulting ecological analyses of metabarcoding data<sup>1</sup></h1>
<p><a href="https://doi.org/10.1139/gen-2015-0184">doi:10.1139/gen-2015-0184</a></p>

<p><a href=None>None</a></p>
<p>The combination of DNA barcoding and high-throughput (next-generation) sequencing (metabarcoding) provides many promises but also serious challenges. Generating a reliable comparable estimate of biodiversity remains a central challenge to the application of the technology. Many approaches have been used to turn millions of sequences into distinct taxonomic units. However, the extent to which these methods impact the outcome of simple ecological analyses is not well understood. Here we performed a simple analysis of dietary overlap by skinks and shrews on Ile Aux Aigrettes, Mauritius. We used a combination of filtering thresholds and clustering algorithms on a COI metabarcoding dataset and demonstrate that all bioinformatics parameters will have interacting effects on molecular operational taxonomic unit (MOTU) recovery rates. These effects generated estimates covering two orders of magnitude. However, the effect on a simple ecological analysis was not large and, despite the wide variation in estimates of niche overlap, the same ecological conclusion was drawn in most cases. We advise that a conservative clustering programme coupled with larger sequence divergences to define a cluster, the removal of singletons, rigorous length filtering, and stringent match criteria for Molecular Identifier tags are preferable to avoid MOTU inflation and that the same parameters be used in all comparative analyses.</p><h1>Validation of picogram- and femtogram-input DNA libraries for microscale metagenomics</h1>
<p><a href="https://doi.org/10.7717/peerj.2486">doi:10.7717/peerj.2486</a></p>

<p><a href=None>None</a></p>
<p>High-throughput sequencing libraries are typically limited by the requirement for nanograms to micrograms of input DNA. This bottleneck impedes the microscale analysis of ecosystems and the exploration of low biomass samples. Current methods for amplifying environmental DNA to bypass this bottleneck introduce considerable bias into metagenomic profiles. Here we describe and validate a simple modification of the Illumina Nextera XT DNA library preparation kit which allows creation of shotgun libraries from sub-nanogram amounts of input DNA. Community composition was reproducible down to 100 fg of input DNA based on analysis of a mock community comprising 54 phylogenetically diverse Bacteria and Archaea. The main technical issues with the low input libraries were a greater potential for contamination, limited DNA complexity which has a direct effect on assembly and binning, and an associated higher percentage of read duplicates. We recommend a lower limit of 1 pg (~100-1,000 microbial cells) to ensure community composition fidelity, and the inclusion of negative controls to identify reagent-specific contaminants. Applying the approach to marine surface water, pronounced differences were observed between bacterial community profiles of microliter volume samples, which we attribute to biological variation. This result is consistent with expected microscale patchiness in marine communities. We thus envision that our benchmarked, slightly modified low input DNA protocol will be beneficial for microscale and low biomass metagenomics.</p><h1>Barcoding the food chain: From Sanger to high-throughput sequencing<sup>1</sup></h1>
<p><a href="https://doi.org/10.1139/gen-2016-0028">doi:10.1139/gen-2016-0028</a></p>

<p><a href=None>None</a></p>
<p>Society faces the complex challenge of supporting biodiversity and ecosystem functioning, while ensuring food security by providing safe traceable food through an ever-more-complex global food chain. The increase in human mobility brings the added threat of pests, parasites, and invaders that further complicate our agro-industrial efforts. DNA barcoding technologies allow researchers to identify both individual species, and, when combined with universal primers and high-throughput sequencing techniques, the diversity within mixed samples (metabarcoding). These tools are already being employed to detect market substitutions, trace pests through the forensic evaluation of trace "environmental DNA", and to track parasitic infections in livestock. The potential of DNA barcoding to contribute to increased security of the food chain is clear, but challenges remain in regulation and the need for validation of experimental analysis. Here, we present an overview of the current uses and challenges of applied DNA barcoding in agriculture, from agro-ecosystems within farmland to the kitchen table.</p><h1>Application of environmental DNA metabarcoding in ecology</h1>
<p><a href="https://doi.org/10.5846/stxb201501150125">doi:10.5846/stxb201501150125</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA metabarcoding is defined as high throughput DNA sequencing used for simultaneous molecular identification of multiple taxa in a complex environmental sample (i.e., soil, water, feces, etc.). In recent years, this method has aroused widespread concern among scientists and has been increasingly used in the field of ecology, such as biodiversity research, monitoring of aquatic, rare, or invasive species. This article describes the significance and the methods of environmental DNA metabarcoding, and highlights its application in the fields of species monitoring, detection of rare and invasive species, biodiversity, and diet analysis. Finally, this article summarizes the challenges in application of environmental DNA metabarcoding in ecological research and discusses the future prospects of its development.</p><h1>The application of DNA metabarcoding in the study of soil animal diversity in Taibai Mountain</h1>
<p><a href="https://doi.org/10.5846/stxb201412122475">doi:10.5846/stxb201412122475</a></p>

<p><a href=None>None</a></p>
<p>DNA metabarcoding which couples DNA barcoding with high-throughput sequencing technology enables quick and easy identification of species in a multiple sample, and has become a reliable method for surveying species composition and richness of a community. Taibai Mountain is the main peak of the Qinling Mountains. It is a well-preserved natural ecological system, and is a key area of biodiversity protection in China. Surveys on the diversity of soil meso-microanimals in Taibai Mountain will enrich the data of soil fauna composition and provide scientific basis for biodiversity protection in China. In this paper, we used the metabarcoding approach to analyze the diversity of soil meso-microanimals in 5 different habitats of Taibai Mountain in the Qinling Mountains. These habitats include alpine meadow, coniferous forest, deciduous small-leaved forest, deciduous broad-leaved forest, and farmland. We set 3 plots of 10 m × 10 m in each habitat, and 3 soil samples were collected randomly from each plot, the sampling depth was 0-15cm under ground. A Tullgren funnel was used to separate soil animals from soil samples. Soil animals from the same plot were transferred to 1.5-mL centrifuge tubes and the total DNA was extracted. The universal primers for the fraction of 18S rRNA and COI genes were used to amplify specific barcoding sequences. Sequencing of PCR amplicons was performed on a MiSeq Illumina sequencing platform. Raw data was analyzed using the Qiime and Mothur software to obtain the OTUs list and species list. Ecological analysis was performed using software R. A total of 199 families from 28 orders, 9 classes, and 3 phyla for soil animals were observed. Community composition analysis showed that habitat changes have some effect on the soil animal community composition. Alpha diversit y analysis showed that the highest community richness index for soil animals is the coniferous forest and the lowest is the farmland; in addition, the highest community diversity index for soil animals is the coniferous forest and the lowest is the deciduous small-leaved forest. The community similarity analysis showed that the soil animal community composition in alpine meadow, coniferous forest, and farmland has a high similarity. The soil animal community composition in deciduous small-leaved forest and deciduous broad-leaved forest greatly differed from that in alpine meadow, coniferous forest, and farmland. The difference between the soil animal community composition in deciduous small-leaved forest and deciduous broad-leaved forest also differed greatly. The results of cluster analysis was in conformity with the community similarity analysis.</p><h1>Exploring the environmental diversity of kinetoplastid flagellates in the high-throughput DNA sequencing era</h1>
<p><a href="https://doi.org/10.1590/0074-02760150253">doi:10.1590/0074-02760150253</a></p>

<p><a href=None>None</a></p>
<p>The class Kinetoplastea encompasses both free-living and parasitic species from a wide range of hosts. Several representatives of this group are responsible for severe human diseases and for economic losses in agriculture and livestock. While this group encompasses over 30 genera, most of the available information has been derived from the vertebrate pathogenic genera Leishmania and Trypanosoma. Recent studies of the previously neglected groups of Kinetoplastea indicated that the actual diversity is much higher than previously thought. This article discusses the known segment of kinetoplastid diversity and how gene-directed Sanger sequencing and next-generation sequencing methods can help to deepen our knowledge of these interesting protists</p><h1>Environmental DNA evidence of transfer of North Sea molluscs across tropical waters through ballast water</h1>
<p><a href="https://doi.org/10.1093/mollus/eyv022">doi:10.1093/mollus/eyv022</a></p>

<p><a href=None>None</a></p>
<p>Maritime transport, in particular of ballast water, is considered to be one of the most important pathways of marine biological invasions worldwide. Here we provide the first molecular evidence of potential survival of the European mudsnail, Peringia ulvae, in ballast water on cross-latitudinal voyages. Ballast water from the RV Polarstern was sampled at its departure from the North Sea and again in tropical latitudes; DNA was extracted and amplicon-sequenced employing high-throughput sequencing methodology. Mollusc species were detected by cytochrome oxidase subunit I DNA barcode sequences. The increasing proportion of operational taxonomic units that were identified as P. ulvae after 2 weeks of navigation suggests that this species withstands the harsh conditions in the ballast tank. As such, P. ulvae has the potential to reach very distant, new marine areas where it eventually might establish itself as a nonindigenous species. We also discuss the potential of environmental DNA analysis for en-route biodiversity screening and species-specific risk assessments, as well as some current limitations of the approach.</p><h1>Response to Comment on "Global diversity and geography of soil fungi"</h1>
<p><a href="https://doi.org/10.1126/science.aaa5594">doi:10.1126/science.aaa5594</a></p>

<p><a href=None>None</a></p>
<p>Schadt and Rosling (Technical Comment, 26 June 2015, p. 1438) argue that primer-template mismatches neglected the fungal class Archaeorhizomycetes in a global soil survey. Amplicon-based metabarcoding of nine barcode-primer pair combinations and polymerase chain reaction (PCR)-free shotgun metagenomics revealed that barcode and primer choice and PCR bias drive the diversity and composition of microorganisms in general, but the Archaeorhizomycetes were little affected in the global study. We urge that careful choice of DNA markers and primers is essential for ecological studies using high-throughput sequencing for identification.</p><h1>Can DNA-based ecosystem assessments quantify species abundance? Testing primer bias and biomass-sequence relationships with an innovative metabarcoding protocol</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0130324">doi:10.1371/journal.pone.0130324</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding is an emerging genetic tool to rapidly assess biodiversity in ecosystems. It involves high-throughput sequencing of a standard gene from an environmental sample and comparison to a reference database. However, no consensus has emerged regarding laboratory pipelines to screen species diversity and infer species abundances from environmental samples. In particular, the effect of primer bias and the detection limit for specimens with a low biomass has not been systematically examined, when processing samples in bulk. We developed and tested a DNA metabarcoding protocol that utilises the standard cytochrome c oxidase subunit I (COI) barcoding fragment to detect freshwater macroinvertebrate taxa. DNA was extracted in bulk, amplified in a single PCR step, and purified, and the libraries were directly sequenced in two independent MiSeq runs (300-bp paired-end reads). Specifically, we assessed the influence of specimen biomass on sequence read abundance by sequencing 31 specimens of a stonefly species with known haplotypes spanning three orders of magnitude in biomass (experiment I). Then, we tested the recovery of 52 different freshwater invertebrate taxa of similar biomass using the same standard barcoding primers (experiment II). Each experiment was replicated ten times to maximise statistical power. The results of both experiments were consistent across replicates. We found a distinct positive correlation between species biomass and resulting numbers of MiSeq reads. Furthermore, we reliably recovered 83% of the 52 taxa used to test primer bias. However, sequence abundance varied by four orders of magnitudes between taxa despite the use of similar amounts of biomass. Our metabarcoding approach yielded reliable results for high-throughput assessments. However, the results indicated that primer efficiency is highly species-specific, which would prevent straightforward assessments of species abundance and biomass in a sample. Thus, PCR-based metabarcoding assessments of biodiversity should rely on presence-absence metrics.</p><h1>How will the 'molecular revolution' contribute to biological recording?</h1>
<p><a href="https://doi.org/10.1111/bij.12516">doi:10.1111/bij.12516</a></p>

<p><a href=None>None</a></p>
<p>Soaring throughput, plummeting costs, and increased sensitivity for assaying degraded or low-concentration DNA are driving a revolution in the way that we monitor biodiversity. Arguably the biggest 'game-changer' is environmental DNA (eDNA), which refers to free-floating DNA released by organisms into their environment. Rare or elusive species can be detected with greater sensitivity and accuracy using eDNA than by most conventional methods, and we have the capability to screen and describe whole communities, as well as perform targeted monitoring of single species. This paper discusses the basic approaches for molecular monitoring of biodiversity, provides case studies to demonstrate the effectiveness of the techniques, and considers any challenges and limitations that could impact molecular biological recording. It is argued that eDNA surveys offer exciting new opportunities to engage the public in biological recording and that molecular approaches will complement conventional surveys, enabling unprecedented insights into species distributions. Finally, with the number of eDNA studies increasing at a rapid pace, it is argued that there is a need to rapidly establish ways of managing molecular records. Integrating molecular records into existing biological records databases would enhance our understanding of species distributions and may be something that the Biological Records Centre should be considering to mark its landmark anniversary.</p><h1>Rare biosphere exploration using high-throughput sequencing: research progress and perspectives</h1>
<p><a href="https://doi.org/10.1007/s10592-014-0678-9">doi:10.1007/s10592-014-0678-9</a></p>

<p><a href=None>None</a></p>
<p>Identification of rare species and mapping their distributions is crucial for understanding natural species distributions and causes and consequences of accelerating species declines. However, detection of rare species in both terrestrial and especially aquatic communities typically dominated by numerous microscopic species (i.e. rare biosphere) represents a formidable technical challenge. Rapid advances in high-throughput sequencing (HTS) technologies have revolutionized biodiversity studies in the rare biosphere, and also stimulated associated debates. Here we summarize research progress, discuss debates and problems, and propose possible solutions and future studies to address these issues. In addition, we provide take-home messages for experimental design and data interpretation when utilizing HTS techniques for rare biosphere exploration in ecology and conservation biology.</p><h1>Next generation sequencing for characterizing biodiversity: promises and challenges</h1>
<p><a href="https://doi.org/10.1007/s10709-015-9816-7">doi:10.1007/s10709-015-9816-7</a></p>

<p><a href=None>None</a></p>
<p>DNA barcoding approaches are used to describe biodiversity by analysing specimens or environmental samples in taxonomic, phylogenetic and ecological studies. While sharing data among these disciplines would be highly valuable, this remains difficult because of contradictory requirements. The properties making a DNA barcode efficient for specimen identification or species delimitation are hardly reconcilable with those required for a powerful analysis of degraded DNA from environmental samples. The use of next generation sequencing methods open up the way towards the development of new markers (e.g., multilocus barcodes) that would overcome such limitations. However, several challenges should be taken up for coordinating actions at the interface between taxonomy, ecology, molecular biology and bioinformatics in order to develop methods and protocols compatible with both taxonomic and ecological studies.</p><h1>Metabarcoding approach for the ballast water surveillance - An advantageous solution or an awkward challenge?</h1>
<p><a href="https://doi.org/10.1016/j.marpolbul.2015.01.008">doi:10.1016/j.marpolbul.2015.01.008</a></p>

<p><a href=None>None</a></p>
<p>Transfer of organisms with ships' ballast water is recognized as a major pathway of non-indigenous species introduction and addressed in a few recent legislative initiatives. Among other they imply scientific and technical research and monitoring to be conducted in a efficient and reliable way. The recent development of DNA barcoding and metabarcoding technologies opens new opportunities for biodiversity and biosecurity surveillance. In the current study, the performance of metabarcoding approach was assessed in comparison to the conventional (visual) observations, during the en route experimental ballast water survey. Opportunities and limitations of the molecular method were identified from taxonomical datasets rendered by two molecular markers of different degree of universality - the universal cytochrome oxydase sub-unit I gene and a fragment of RuBisCO gene. The cost-efficacy and possible improvements of these methods are discussed for the further successful development and implementation of the approach in ballast water control and NIS surveillance.</p><h1>Metagenome of a microbial community inhabiting a metal-rich tropical stream sediment</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0119465">doi:10.1371/journal.pone.0119465</a></p>

<p><a href=None>None</a></p>
<p>Here, we describe the metagenome and functional composition of a microbial community in a historically metal-contaminated tropical freshwater stream sediment. The sediment was collected from the Mina Stream located in the Iron Quadrangle (Brazil), one of the world's largest mining regions. Environmental DNA was extracted and was sequenced using SOLiD technology, and a total of 7.9 Gbp was produced. A taxonomic profile that was obtained by comparison to the Greengenes database revealed a complex microbial community with a dominance of Proteobacteriaand Parvarcheota. Contigs were recruited by bacterial and archaeal genomes, especially Candidatus Nitrospira defluviiand Nitrosopumilus maritimus, and their presence implicated them in the process of N cycling in the Mina Stream sediment (MSS). Functional reconstruction revealed a large, diverse set of genes for ammonium assimilation and ammonification. These processes have been implicated in the maintenance of the N cycle and the health of the sediment. SEED subsystems functional annotation unveiled a high degree of diversity of metal resistance genes, suggesting that the prokaryotic community is adapted to metal contamination. Furthermore, a high metabolic diversity was detected in the MSS, suggesting that the historical arsenic contamination is no longer affecting the prokaryotic community. These results expand the current knowledge of the microbial taxonomic and functional composition of tropical metal-contaminated freshwater sediments.</p><h1>DNA (meta)barcoding of biological invasions: a powerful tool to elucidate invasion processes and help managing aliens</h1>
<p><a href="https://doi.org/10.1007/s10530-015-0854-y">doi:10.1007/s10530-015-0854-y</a></p>

<p><a href=None>None</a></p>
<p>Biological invasions are a major threat to the world’s biodiversity with consequences on ecosystem structure and functioning, species evolution, and human well-being (through ecosystem services). Conservation of biological diversity and management of biological resources require multi-level management strategies on non-native species, in order to (1) prevent biological introductions, (2) detect non-native species at an early stage of the introduction, and (3) eradicate or maintain at a low level of population density non-native species that were successfully introduced. A pre-requisite to any control measures on non-native species is the ability to rapidly and accurately identify the putative threatening alien species. DNA barcoding, and its recent extension, DNA metabarcoding are complementary tools that have proved their value in the identification of living beings. Here we review their use in the identification of non-native species at several steps of the introduction processes, and how they can be applied in the control and management of biological introductions. Through examples covering various taxa and ecosystems (terrestrial, freshwater, marine), we highlight the strengths and weaknesses of approaches that we foresee as crucial in the implementation of early warning strategies.</p><h1>DNA barcoding and metabarcoding of standardized samples reveal patterns of marine benthic diversity</h1>
<p><a href="https://doi.org/10.1073/pnas.1424997112">doi:10.1073/pnas.1424997112</a></p>

<p><a href=None>None</a></p>
<p>Documenting the diversity of marine life is challenging because many species are cryptic, small, and rare, and belong to poorly known groups. New sequencing technologies, especially when combined with standardized sampling, promise to make comprehensive biodiversity assessments and monitoring feasible on a large scale. We used this approach to characterize patterns of diversity on oyster reefs across a range of geographic scales comprising a temperate location [Virginia (VA)] and a subtropical location [Florida (FL)]. Eukaryotic organisms that colonized multilayered settlement surfaces (autonomous reef monitoring structures) over a 6-mo period were identified by cytochrome c oxidase subunit I barcoding (>2-mm mobile organisms) and metabarcoding (sessile and smaller mobile organisms). In a total area of ∼15.64 m2 and volume of ∼0.09 m3, 2,179 operational taxonomic units (OTUs) were recorded from 983,056 sequences. However, only 10.9% could be matched to reference barcodes in public databases, with only 8.2% matching barcodes with both genus and species names. Taxonomic coverage was broad, particularly for animals (22 phyla recorded), but 35.6% of OTUs detected via metabarcoding could not be confidently assigned to a taxonomic group. The smallest size fraction (500 to 106 μm) was the most diverse (more than two-thirds of OTUs). There was little taxonomic overlap between VA and FL, and samples separated by ∼2 m were significantly more similar than samples separated by ∼100 m. Ground-truthing with independent assessments of taxonomic composition indicated that both presence-absence information and relative abundance information are captured by metabarcoding data, suggesting considerable potential for ecological studies and environmental monitoring.</p><h1>Scraping the bottom of the barrel: Are rare high throughput sequences artifacts?</h1>
<p><a href="https://doi.org/10.1016/j.funeco.2014.08.006">doi:10.1016/j.funeco.2014.08.006</a></p>

<p><a href=None>None</a></p>
<p>Metabarcoding data generated using next-generation sequencing (NGS) technologies are overwhelmed with rare taxa and skewed in Operational Taxonomic Unit (OTU) frequencies comprised of few dominant taxa. Low frequency OTUs comprise a rare biosphere of singleton and doubleton OTUs, which may include many artifacts. We present an in-depth analysis of global singletons across sixteen NGS libraries representing different ribosomal RNA gene regions, NGS technologies and chemistries. Our data indicate that many singletons (average of 38% across gene regions) are likely artifacts or potential artifacts, but a large fraction can be assigned to lower taxonomic levels with very high bootstrap support (~32% of sequences to genus with ≥90% bootstrap cutoff). Further, many singletons clustered into rare OTUs from other datasets highlighting their overlap across datasets or the poor performance of clustering algorithms. These data emphasize a need for caution when discarding rare sequence data en masse: such practices may result in throwing the baby out with the bathwater, and underestimating the biodiversity. Yet, the rare sequences are unlikely to greatly affect ecological metrics. As a result, it may be prudent to err on the side of caution and omit rare OTUs prior to downstream analyses.</p><h1>Multiplex sequencing of pooled mitochondrial genomes - A crucial step toward biodiversity analysis using mito-metagenomics</h1>
<p><a href="https://doi.org/10.1093/nar/gku917">doi:10.1093/nar/gku917</a></p>

<p><a href=None>None</a></p>
<p>The advent in high-throughput-sequencing (HTS) technologies has revolutionized conventional biodiversity research by enabling parallel capture of DNA sequences possessing species-level diagnosis. However, polymerase chain reaction (PCR)-based implementation is biased by the efficiency of primer binding across lineages of organisms. A PCR-free HTS approach will alleviate this artefact and significantly improve upon the multi-locus method utilizing full mitogenomes. Here we developed a novel multiplex sequencing and assembly pipeline allowing for simultaneous acquisition of full mitogenomes from pooled animals without DNA enrichment or amplification. By concatenating assemblies from three de novo assemblers, we obtained high-quality mitogenomes for all 49 pooled taxa, with 36 species >15 kb and the remaining >10 kb, including 20 complete mitogenomes and nearly all protein coding genes (99.6%). The assembly quality was carefully validated with Sanger sequences, reference genomes and conservativeness of protein coding genes across taxa. The new method was effective even for closely related taxa, e.g. three Drosophila spp., demonstrating its broad utility for biodiversity research and mito-phylogenomics. Finally, the in silicosimulation showed that by recruiting multiple mito-loci, taxon detection was improved at a fixed sequencing depth. Combined, these results demonstrate the plausibility of a multi-locus mito-metagenomics approach as the next phase of the current single-locus metabarcoding method.</p><h1>Patchiness of deep-sea benthic Foraminifera across the Southern Ocean: Insights from high-throughput DNA sequencing</h1>
<p><a href="https://doi.org/10.1016/j.dsr2.2014.07.018">doi:10.1016/j.dsr2.2014.07.018</a></p>

<p><a href=None>None</a></p>
<p>Spatial patchiness is a natural feature that strongly influences the level of species richness we perceive in surface sediments sampled in the deep-sea. Recent environmental DNA (eDNA) surveys of benthic micro- and meiofauna confirmed this exceptional richness. However, it is unknown to which extent the results of these studies, based usually on few grams of sediment, are affected by spatial patchiness of deep-sea benthos. Here, we analyse the eDNA diversity of Foraminifera in 42 deep-sea sediment samples collected across different scales in the Southern Ocean. At three stations, we deployed at least twice the multicorer and from each multicorer cast, we subsampled 3 sediment replicates per core for 2 cores. Using high-throughput sequencing (HTS), we generated over 2.35 million high-quality sequences that we clustered into 451 operational taxonomic units (OTUs). The majority of OTUs were assigned to the monothalamous (single-chambered) taxa and environmental clades. On average, a one-gram sediment sample captures 57.9% of the overall OTU diversity found in a single core, while three replicates cover at most 61.9% of the diversity found in a station. The OTUs found in all the replicates of each core gather up to 87.9% of the total sequenced reads, but only represent from 12.2% to 30% of the OTUs found in one core. These OTUs represent the most abundant species, among which dominate environmental lineages. The majority of the OTUs are represented by few sequences comprising several well-known deep-sea morphospecies or remaining unassigned. It is crucial to study wider arrays of sample and PCR replicates as well as RNA together with DNA in order to overcome biases stemming from deep-sea patchiness and molecular methods.</p><h1>The contribution of DNA metabarcoding to fungal conservation: Diversity assessment, habitat partitioning and mapping red-listed fungi in protected coastal Salix repens communities in the Netherlands</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0099852">doi:10.1371/journal.pone.0099852</a></p>

<p><a href=None>None</a></p>
<p>Western European coastal sand dunes are highly important for nature conservation. Communities of the creeping willow (Salix repens) represent one of the most characteristic and diverse vegetation types in the dunes. We report here the results of the first kingdom-wide fungal diversity assessment in S. repens coastal dune vegetation. We carried out massively parallel pyrosequencing of ITS rDNA from soil samples taken at ten sites in an extended area of joined nature reserves located along the North Sea coast of the Netherlands, representing habitats with varying soil pH and moisture levels. Fungal communities in Salix repens beds are highly diverse and we detected 1211 non-singleton fungal 97% sequence similarity OTUs after analyzing 688,434 ITS2 rDNA sequences. Our comparison along a north-south transect indicated strong correlation between soil pH and fungal community composition. The total fungal richness and the number OTUs of most fungal taxonomic groups negatively correlated with higher soil pH, with some exceptions. With regard to ecological groups, dark-septate endophytic fungi were more diverse in acidic soils, ectomycorrhizal fungi were represented by more OTUs in calcareous sites, while detected arbuscular mycorrhizal genera fungi showed opposing trends regarding pH. Furthermore, we detected numerous red listed species in our samples often from previously unknown locations, indicating that some of the fungal species currently considered rare may be more abundant in Dutch S. repens communities than previously thought. © 2014 Geml et al.</p><h1>A molecular-based approach for examining responses of eukaryotes in microcosms to contaminant-spiked estuarine sediments</h1>
<p><a href="https://doi.org/10.1002/etc.2450">doi:10.1002/etc.2450</a></p>

<p><a href=None>None</a></p>
<p>Ecotoxicological information for most contaminants is limited to a small number of taxa, and these are generally restricted to comparatively hardy organisms that are readily extractable from test media and easily identifiable. Advances in DNA sequencing can now provide a comprehensive view of benthic invertebrate diversity. The authors applied 454 pyrosequencing to examine the responses of benthic communities in microcosms exposed to sediments with elevated concentrations of triclosan, the endpoint being eukaryl communities that have successfully vertically migrated through the manipulated sediments. The biological communities associated with the 3 treatments (control triclosan, low triclosan [14mg/kg], and high triclosan [180mg/kg]) clustered into 3 groups: control/low (n=6 controls and 4 low), moderate (n=2 low), and high (n=5 high). One sample was discarded as an outlier. The most pronounced change as a response to triclosan was the loss of number of metazoan operational taxonomic units (OTUs), indicative of the control/low and moderate groups, with this being most evident in the range of taxa associated with the classes Chromadorea and Bivalvia and the phylum Kinorhyncha. The authors also describe a range of other taxa that aided discrimination between the groups; compare findings with traditionally obtained meio- and macrofaunal communities obtained from the same experiment; and illustrate some of the advantages and limitations associated with both the molecular and traditional approaches. The described approach illustrates the capacity for amplicon sequencing to provide ecologically relevant information that can be used to strengthen an understanding of how sedimentary communities respond to a range of environmental stressors. © 2014 SETAC.</p><h1>Meta-genomic surveillance of invasive species in the bait trade</h1>
<p><a href="https://doi.org/10.1007/s12686-014-0213-9">doi:10.1007/s12686-014-0213-9</a></p>

<p><a href=None>None</a></p>
<p>There are a number of different pathways by which invasive species can enter aquatic ecosystems, including the relatively unstudied live bait trade. Through contaminated stocks, the bait trade vector has the potential to distribute species widely, and unknowingly, across a wide geographic area. These introductions can have large implications for the conservation of native biodiversity and habitats. Reliable techniques for monitoring for invasive species remains challenging, particularly due to a lack of taxonomic expertise by those using live bait. Here, we show that non-target species (i.e., rare; not intended to be purchased) can be detected based on environmental DNA (eDNA) collected in water samples from commercial bait vendors. Utilizing high-throughput DNA sequencing, we analyzed water samples collected from six different commercial bait shops, screening the resulting sequence data for presence of non-target, and potentially invasive species of fish. Our findings show that DNA from multiple non-target species was present in the collected samples, including DNA from at least one potentially harmful invasive species. Additionally, this work supports the use of eDNA surveillance for screening the bait shop vector for rare and potentially harmful aquatic invasive species. © 2014 Springer Science+Business Media Dordrecht.</p><h1>SOAPBarcode: Revealing arthropod biodiversity through assembly of Illumina shotgun sequences of PCR amplicons</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12120">doi:10.1111/2041-210X.12120</a></p>

<p><a href=None>None</a></p>
<p>Summary: Metabarcoding of mixed arthropod samples for biodiversity assessment has mostly been carried out on the 454 GS FLX sequencer (Roche, Branford, Connecticut, USA), due to its ability to produce long reads (≥400 bp) that are believed to allow higher taxonomic resolution. The Illumina sequencing platforms, with their much higher throughputs, could potentially reduce sequencing costs and improve sequence quality, but the associated shorter read length (typically <150 bp) has deterred their usage in next-generation-sequencing (NGS)-based analyses of eukaryotic biodiversity, which often utilize standard barcode markers (e.g. COI, rbcL, matK, ITS) that are hundreds of nucleotides long. We present a new Illumina-based pipeline to recover full-length COI barcodes from mixed arthropod samples. Our new assembly program, SOAPBarcode, a variant of the genome assembly program SOAPdenovo, uses paired-end reads of the standard COI barcode region as anchors to extract the correct pathways (sequences) out of otherwise chaotic 'de Bruijn graphs', which are caused by the presence of large numbers of COI homologs of high sequence similarity. Two bulk insect samples of known species composition have been analysed in a recently published 454 metabarcoding study (Yu et al. 2012) and are re-analysed by our analysis pipeline. Compared to the results of Roche 454 (c. 400-bp reads), our pipeline recovered full-length COI barcodes (658 bp) and 17-31% more species-level operational taxonomic units (OTUs) from bulk insect samples, with fewer untraceable (novel) OTUs. On the other hand, our PCR-based pipeline also revealed higher rates of contamination across samples, due to the Illumina's increased sequencing depth. On balance, the assembled full-length barcodes and increased OTU recovery rates resulted in more resolved taxonomic assignments and more accurate beta diversity estimation. The HiSeq 2000 and the SOAPBarcode pipeline together can achieve more accurate biodiversity assessment at a much reduced sequencing cost in metabarcoding analyses. However, greater precaution is needed to prevent cross-sample contamination during field preparation and laboratory operation because of greater ability to detect non-target DNA amplicons present in low-copy numbers. © 2013 The Authors. Methods in Ecology and Evolution published by John Wiley & Sons Ltd on behalf of British Ecological Society.</p><h1>A DNA-based approach for assessing and monitoring benthic communities</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>Compositional changes in benthic invertebrate communities are commonly used to monitor and assess sedimentary marine environments. For the most part, sampling is restricted to a relatively small number of benthic organisms (<50 taxa), including taxa such as polychaetes, crustaceans and molluscs. The selection of taxa is predominately determined by size, ease of extraction from the sediment and regional taxonomic expertise. However, there are several significant limitations associated with this approach: 1) a large number of samples are required to produce informative data; 2) benthic programs are costly; 3) there is often a turnaround time of 6-12 months; and 4) assumptions about an ecosystem's health are being derived from a minute fraction of the total biodiversity. Recent innovations in DNA sequencing and computation now make it viable to obtain biodiversity information on potentially all organisms residing within a sediment sample (>1000 taxa in some studies). This can include traditionally examined organisms (e.g. polychaetes and bivalves), as well as micro- and meio-faunal taxa such as fungi, diatoms and bacteria. This ecogenomic approach not only produces comprehensive ecological information, but also, requires a small amount sediment; has a fairly rapid turnaround time; is cost competitive with traditional approaches; and can produce ecological data from poorly examined systems or those where taxonomic knowledge is limited. Here we provide an overview of the fundamentals behind a DNA-based monitoring program for marine sediments and present relevant examples of where this approach has been applied. In addition, we will discuss some of the advantages associated with the approach, as well as the limitations. Also described are some new technologies that have the potential to vastly bolster the ecological data obtained from DNA based ecological monitoring programs.</p><h1>Biodiversity soup: Metabarcoding of arthropods for rapid biodiversity assessment and biomonitoring</h1>
<p><a href="https://doi.org/10.1111/j.2041-210X.2012.00198.x">doi:10.1111/j.2041-210X.2012.00198.x</a></p>

<p><a href=None>None</a></p>
<p>1. Traditional biodiversity assessment is costly in time, money and taxonomic expertise. Moreover, data are frequently collected in ways (e.g. visual bird lists) that are unsuitable for auditing by neutral parties, which is necessary for dispute resolution. 2. We present protocols for the extraction of ecological, taxonomic and phylogenetic information from bulk samples of arthropods. The protocols combine mass trapping of arthropods, mass-PCR amplification of the COI barcode gene, pyrosequencing and bioinformatic analysis, which together we call 'metabarcoding'. 3. We construct seven communities of arthropods (mostly insects) and show that it is possible to recover a substantial proportion of the original taxonomic information. We further demonstrate, for the first time, that metabarcoding allows for the precise estimation of pairwise community dissimilarity (beta diversity) and within-community phylogenetic diversity (alpha diversity), despite the inevitable loss of taxonomic information inherent to metabarcoding. 4. Alpha and beta diversity metrics are the raw materials of ecology and the environmental sciences, facilitating assessment of the state of the environment with a broad and efficient measure of biodiversity. © 2012 The Authors.</p><h1>Short and informative DNA products to indirectly measure vascular plant biodiversity</h1>
<p><a href="https://doi.org/10.1111/j.1365-294X.2012.05633.x">doi:10.1111/j.1365-294X.2012.05633.x</a></p>

<p><a href=None>None</a></p>
<p>Anthropogenic activities are having a deleterious effect on biodiversity. To understand the magnitude of this issue, and ultimately temper its pace, requires reproducible biodiversity measurements at suitable spatio-temporal scales. Procuring such data solely by existing approaches is unachievable because of the costs, time and the taxonomic expertise required. High-throughput molecular biodiversity analysis shows great promise, increasing the breadth of biota sampled and accelerating the rate of data collection. In this issue of Molecular Ecology, Yoccoz et al. (2012) use short informative DNA product 'meta-barcodes' to provide an insight into above-ground vascular plant diversity from boreal, temperate and tropical environments. Interestingly, their molecular analysis was derived from the soils and not the plants themselves, with the molecular signatures of the soils not only strongly reflecting current diversity, but also traces of crops not sown for up to one hundred years. Importantly, the research examines the complexities associated with deriving biomass estimates from molecular data and the need to consider biomass turnover. The use of soil-derived meta-barcodes extends beyond estimating vascular plant diversity, with the approach being suited to the range of ecological applications, especially scenarios where DNA may be degraded. © 2012 Blackwell Publishing Ltd.</p><h1>Accounting for uncertainty in species delineation during the analysis of environmental DNA sequence data</h1>
<p><a href="https://doi.org/10.1111/j.2041-210X.2011.00122.x">doi:10.1111/j.2041-210X.2011.00122.x</a></p>

<p><a href=None>None</a></p>
<p>Defining species boundaries represents a significant challenge in biodiversity studies, especially as these studies increasingly rely on high-throughput DNA sequencing technologies. A promising approach for delineating species in environmental sequence data combines phylogenetics and coalescence theory to estimate species boundaries from distributions of lineage birth rates within multispecies coalescent trees. Existing methods for interpreting these models utilize hypothetico-deductive reasoning to identify thresholds associated with a mixed speciation-coalescent model that fits the data better than a null model. Here, I describe an alternative approach that ranks and assigns weights to models based on their fit to the data using information criteria and then uses model averaging to estimate parameters and species probabilities. This approach is applied to data from two independent studies that address (i) patterns of cospeciation in an aphid-bacterial symbiosis and (ii) diversity of bacterial communities associated with the human gut. In both of these cases, accounting for uncertainty during model selection allowed greater flexibility to detect variable (with respect to time) speciation-coalescent thresholds among lineages. The precision of the predicted species boundaries varied among the studies, and the variance-to-mean ratio for richness estimates ranged from 0.023 to 0.079. Sample-based estimates of gut bacteria richness revealed that accounting for uncertainty during species delineation increased the variance in the estimates of population means (by individual from which the samples were taken or by sex of the individuals) by up to 7.5%. In ecological and evolutionary studies, conclusions are highly dependent on the classification system that is adopted; given the uncertainty in species boundaries observed here, ignoring this source of error (as is common practice) likely results in inflated type I error rates. The approach described here represents an objective, theory-based method for predicting species boundaries and explicitly incorporates uncertainty in the classification system into biodiversity estimation, thus allowing researchers to better address the causes and consequences of biodiversity. © 2011 The Author. Methods in Ecology and Evolution © 2011 British Ecological Society.</p><h1>Rare metabolic conversions-harvesting diversity through nature</h1>
<p><a href="https://doi.org/None">doi:None</a></p>

<p><a href=None>None</a></p>
<p>At the fundamental level the cellular metabolism is mediated through the enzymes that catalyze biotransformation reactions. e specic reactions transforming the particular substrate in an experimental assay (or in an in vivo system) may be or may not be of a practical interest. However, in case the research focuses on biotransformations-conversions and their biotechnological applications, in most cases a knowledge of a whole genetic circuit, particular pathway, and individual enzymes comprising the pathway are needed. Traditionally, the screening for a specic biocatalytic reaction was performed through the growth of a particular microorganism and the disappearance of the starting compound (reaction substrate) and detection of its disappearance, or/and occurrence and accumulation of corresponding metabolites [1,2]. e structures of metabolites and the stoichiometry of their formation (determined both, experimentally or in silico) give some important hints to the specicity of an enzyme and its probable usefulness for the application [3]. Whatever the case, the reactions of an apparent new pathway are routinely elucidated via molecular biological methods, typically through molecular cloning and expression of the protein in a surrogate host, such as Escherichia coli, and consequent characterization of its enzymatic activity (Table 5.1). To obtain in a desired biocatalyst a known pathway or reaction might be taken. e goal is then to obtain an ideal enzyme which is highly active with a given substrate or under specic set of conditions [4]. In this case, the screening method is important, as it might be necessary to look at hundreds or thousands of enzymes in the rst rounds of screening. e description of a wide range of screening methods can be seen in numerous publications, the most relevant being those by the groups of Omura and Cutler [5,6]. Moreover, fast, ecient, and high-throughput screening methods that could test vast numbers of enzymes for specic applications have been discovered in the last decades, bringing made to order enzymes a step closer [7]. Recently, a screening system utilizing the "uorescence activated cell sorting," or FACS, a technology that enables the identication of biological activity within a single cell, has been developed by Diversa Co. (CA). is system incorporates a laser with multiple wavelength capabilities and the ability to screen up to 50,000 clones per second, or over 1 billion clones per day. Another approach is to test existing isolates of microorganisms to determine if they have an enzyme which show a high activity with the substrate of interest. e pure cultures of microorganisms have passed the test of time well: those have successfully been used for over 100 years. However, will the advent of the genome-wide high-throughput techniques of enzymatic screening and in silico predictions [8], make the cultivation-based enzyme discovery excessive? Based on genome sequencing data it is currently possible to make a metabolic reconstruction in which all possible known biochemical conversions could be identied. us, a comprehensive metabolic reconstruction on aspergilli revealed up to 14,400 ORFs, 335 reactions, and 284 metabolites distributed over the intra-and extracellular compartments [9]. Moreover, the genomes of extremophilic bacteria and archaea could be even more sophisticated: those from ermotoga, Sulfolobus, Picrophilus spp., containing the ORF numbers in the range of 1,500-4,000, may comprise more complex biochemistries in terms of unique proteins, metabolites, and enzymes under extreme physical-chemical conditions existing in their natural environment [10-12]. Even though the valuable genomic information of individual organisms represents a good starting point, the cultivation of microbes oen represents a bottleneck. e biological diversity on our planet is mostly comprised of microorganisms, in terms of both absolute numbers of cells and in terms of the biomass and the numbers of species [13]. Recently it has been reported that up to 25,000 dierent genotypes in a millilitre of the seawater sample in some marine ecosystems have been found [14]. However, the majority of microbes in nature will never be cultured, and to access this great inexhaustable genetic and metabolic diversity the new "metagenomic" approach is used [15-21]. is approach uses harvesting DNA from an environmental sample (or from an enrichment), its archiving in the metagenomic libraries in appropriate hosts, and screening these libraries for a gene of interest or expressing the DNA and screening for the enzymatic activities of interest [17]. Alternatively, these libraries are subjected to a high throughput shotgun sequencing and automated annotation, which oen produces thousands of new genes whose functions oen remain unknown. is has been shown recently by the group of Venter [22], whose shotgun sequencing of environmental DNA from Sargasso Sea yielded 1,045 billion base pairs derived from 1,800 genomic species and deduced more than 1.2 million of previously unknown genes. However, the work reects diculties on genome assembly and other challenges in interpretation of large data sets. Metagenomics may result in the development of a number of new products and to provide the tools for the resolution of rare conversions which are not amenable to the existing biocatalysts. Below we focus on the issues, what are current challenges for enzymatic conversions and how the natural biodiversity may help to deal with these challenges.</p><h1>Strategies for accessing soil metagenome for desired applications</h1>
<p><a href="https://doi.org/10.1016/j.biotechadv.2008.08.002">doi:10.1016/j.biotechadv.2008.08.002</a></p>

<p><a href=None>None</a></p>
<p>Most of the microorganisms in nature are inaccessible as they are uncultivable in the laboratory. Metagenomic approaches promise the accessibility of the genetic resources and their potential applications. Genetic resources from terrestrial environments can be accessed by exploring the soil metagenome. Soil metagenomic analyses are usually initiated by the isolation of environmental DNAs. Several methods have been described for the direct isolation of environmental DNAs from soil and sediments. Application of metagenomics largely depends on the construction of genomic DNA libraries and subsequent high-throughput sequencing or library screening. Thus, obtaining large quantities of pure cloneable DNA from the environment is a prerequisite. This review discusses the recent developments related to efficient extraction and purification of soil metagenome highlighting the considerations for various metagenomic applications. © 2008 Elsevier Inc. All rights reserved.</p><h1>Genetically Modified Bacterial Strains and Novel Bacterial Artificial Chromosome Shuttle Vectors for Constructing Environmental Libraries and Detecting Heterologous Natural Products in Multiple Expression Hosts</h1>
<p><a href="https://doi.org/10.1128/AEM.70.4.2452-2463.2004">doi:10.1128/AEM.70.4.2452-2463.2004</a></p>

<p><a href=None>None</a></p>
<p>The enormous diversity of uncultured microorganisms in soil and other environments provides a potentially rich source of novel natural products, which is critically important for drug discovery efforts. Our investigators reported previously on the creation and screening of an Escherichia coli library containing soil DNA cloned and expressed in a bacterial artificial chromosome (BAC) vector. In that initial study, our group identified novel enzyme activities and a family of antibacterial small molecules encoded by soil DNA cloned and expressed in E. coli. To continue our pilot study of the utility and feasibility of this approach to natural product drug discovery, we have expanded our technology to include Streptomyces lividans and Pseudomonas putida as additional hosts with different expression capabilities, and herein we describe the tools we developed for transferring environmental libraries into all three expression hosts and screening for novel activities. These tools include derivatives of S. lividans that contain complete and unmarked deletions of the act and red endogenous pigment gene clusters, a derivative of P. putida that can accept environmental DNA vectors and integrate the heterologous DNA into the chromosome, and new BAC shuttle vectors for transferring large fragments of environmental DNA from E. coli to both S. lividans and P. putida by high-throughput conjugation. Finally, we used these tools to confirm that the three hosts have different expression capabilities for some known gene clusters.</p><h1>Marine Genomics: Methods and Protocols</h1>
<p>No DOI</p>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/240551>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/240551</a></p>
<p>This volume presents the latest protocols for both laboratory and bioinformatics based analyses in the field of marine genomics. The chapters presented in the book cover a wide range of topics, including the sampling and genomics of bacterial communities, DNA extraction in marine organisms, high-throughput sequencing of whole mitochondrial genomes, phylogenomics, SNP discovery, SNP-arrays for species identification, digital PCR-based quantification methods, environment DNA for invasive species surveillance and monitoring, microarrays for the detection of waterborne pathogens, DNA barcoding of marine biodiversity, metabarcoding protocols for marine eukaryotes, analytical protocols for the visualization of eukaryotic diversity, and applications for genomic data to benthic indices for environmental monitoring. Written in the highly successful Methods in Molecular Biology series format, chapters include introduction to their respective topics, lists of the necessary materials and reagents, step-by-step, readily reproducible laboratory protocols, and tips on troubleshooting and avoiding known pitfalls. Cutting-edge and thorough, Marine Genomics: Methods and Protocols is a valuable resource for researchers, students, and policy makers in the field of marine biology.</p><h1>Habitat conditions and phenological tree traits overrule the influence of tree genotype in the needle mycobiome–Picea glauca system at an arctic treeline ecotone</h1>
<p>No DOI</p>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/236528>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/236528</a></p>
<p>Plant-associated mycobiomes in extreme habitats are understudied and poorly understood. We analysed Illumina-generated ITS1 sequences from the needle mycobiome of white spruce (Picea glauca) at the northern treeline in Alaska (USA). Sequences were obtained from the same DNA that was used for tree genotyping. In the present study, fungal metabarcoding and tree microsatellite data were compared for the first time. In general, neighbouring trees shared more fungal taxa with each other than trees growing in further distance. Mycobiomes correlated strongly with phenological host traits and local habitat characteristics contrasting a dense forest stand with an open treeline site. Genetic similarity between trees did not influence fungal composition and no significant correlation existed between needle mycobiome and tree genotype. Our results suggest the pronounced influence of local habitat conditions and phenotypic tree traits on needle-inhabiting fungi. By contrast, the tree genetic identity cannot be benchmarked as a dominant driver for needle-inhabiting mycobiomes, at least not for white spruce in this extreme environment.</p><h1>The future of dna barcoding : Reflections from early career researchers</h1>
<p>No DOI</p>
<p><a href=http://swepub.kb.se/bib/swepub:oai:lup.lub.lu.se:ab9c5837-8948-45f6-91e6-2920c2ebcbd7>http://swepub.kb.se/bib/swepub:oai:lup.lub.lu.se:ab9c5837-8948-45f6-91e6-2920c2ebcbd7</a></p>
<p>Over the last two decades, the use of DNA barcodes has transformed our ability to identify and assess life on our planet. Both strengths and weaknesses of the method have been exemplified through thousands of peer-reviewed scientific articles. Given the novel sequencing approaches, currently capable of generating millions of reads at low cost, we reflect on the questions: What will the future bring for DNA barcoding? Will identification of species using short, standardized fragments of DNA stand the test of time? We present reflected opinions of early career biodiversity researchers in the form of a SWOT analysis and discuss answers to these questions.</p><h1>The Ecological Role of Roadside Stormwater Ponds - Potential to Support Biodiversity</h1>
<p>No DOI</p>
<p><a href=http://swepub.kb.se/bib/swepub:oai:research.chalmers.se:4e046b16-b69b-4c4b-8914-aeb14d9f96be>http://swepub.kb.se/bib/swepub:oai:research.chalmers.se:4e046b16-b69b-4c4b-8914-aeb14d9f96be</a></p>
<p>The increased recognition that roads may impair the aquatic environment and ecosystems has led to a shift from conventional drainage systems toward blue-green solutions such as stormwater ponds. Research on blue-green stormwater solutions has until now mainly focused on water quantity and quality. The aim of this study was to explore the ability of highway stormwater ponds to provide suitable habitats and support, especially macroinvertebrate biodiversity, and to identify the key environmental variables that affect biological community composition and the number of taxa, utilizing data at regional and global scales. Ultimately, this research informs design recommendations for stormwater systems that simultaneously provide multiple ecosystem services. The results of this thesis indicate that larger ponds are better for supporting aquatic biodiversity due to a more heterogeneous environment and the ability to dilute pollutants. Also, the presence of other ponds in the vicinity of the stormwater ponds can facilitate the movement of invertebrates between ponds through increased connectivity. An apparent negative effect of pollution levels on the macroinvertebrate community composition was observed, but not on the biodiversity measured as the number of taxa or Shannon index. The analyses based on the datasets identified using both morphology and DNA metabarcoding demonstrated that DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing more accurate information regarding the way the ponds function and how they are affected by management. Subsequently, the differences in the macroinvertebrate community composition between different types of ponds were compared at the regional and global scale. The results indicated that environmental characteristics, especially conductivity and pH, were different between different types of ponds. Alpha and gamma diversity were similar or even higher in manmade ponds compared to natural ponds due to very different macroinvertebrate communities. Moreover, generally ponds exhibited high levels of spatial heterogeneity, which subsequently enhances gamma diversity. In summary, stormwater ponds have the potential to provide suitable habitats to foster biodiversity. When such systems are created, larger ponds should be built to provide more heterogeneous habitat and dilute harmful pollutants. Additional ponds should also be created in the vicinity of the ponds, thereby promoting aquatic biodiversity through higher connectivity. Although stormwater ponds accumulate pollutants due to their primary functions, this pollution retention process creates a unique environment within the stormwater ponds, which are more suitable for taxa that are moderately to strongly tolerant to pollutants and that may not be found in natural ponds. In this way, stormwater ponds constitute an option in the areas along the highway so that they could combine water treatment properties with providing a suitable habitat for aquatic organisms.</p><h1>Response to Comment on “Global diversity and geography of soil fungi”</h1>
<p>No DOI</p>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/221440>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/221440</a></p>
<p>Schadt and Rosling (Technical Comment, 26 June 2015, p. 1438) argue that primer-template mismatches neglected the fungal class Archaeorhizomycetes in a global soil survey. Amplicon-based metabarcoding of nine barcode-primer pair combinations and polymerase chain reaction (PCR)–free shotgun metagenomics revealed that barcode and primer choice and PCR bias drive the diversity and composition of microorganisms in general, but the Archaeorhizomycetes were little affected in the global study. We urge that careful choice of DNA markers and primers is essential for ecological studies using high-throughput sequencing for identification.</p><h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/v4716jt6s49cgn5c>http://libris.kb.se/bib/v4716jt6s49cgn5c</a></p>
<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p><h1>Application of high-throughput sequencing (HTS) metabarcoding to diatom biomonitoring: do DNA extraction methods matter? [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/20847965>http://libris.kb.se/bib/20847965</a></p>
<p>Vasselon, Valentin and Domaizon, Isabelle and Rimet, Frederic and Kahlert, Maria and Bouchez, Agnes (2017). Application of high-throughput sequencing (HTS) metabarcoding to diatom biomonitoring: do DNA extraction methods matter? Freshwater Science. 36:1, 162-177 [Journal article]</p><h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/6k52v43q4t6mfq5q>http://libris.kb.se/bib/6k52v43q4t6mfq5q</a></p>
<p>Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates &lt; 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%–6.9%), when compared to low OC sediments (1.2%–2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode  Sabatieria  and foraminifera  Elphidium  in high OC, and  Cryptocaryon ‐like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p><h1>Lake Sedimentary DNA Research on Past Terrestrial and Aquatic Biodiversity [Elektronisk resurs] Overview and Recommendations</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/1f02zsxxzg9jzjlf>http://libris.kb.se/bib/1f02zsxxzg9jzjlf</a></p>
<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p><h1>Fungal communities of West African ectomycorrhizal woodlands [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/4hl9mv0l2kqm111r>http://libris.kb.se/bib/4hl9mv0l2kqm111r</a></p>
<p>Ectomycorrhizal symbiosis is a widespread mutualist relationship between fungi and plants, often trees. It is most well-known in temperate and boreal forests, but also occurs in a variety of tropical ecosystems, including Sudanian savanna woodlands and gallery forests in West Africa. In addition to their important role in nutrient cycling, many ectomycorrhizal fungi also produce edible mushrooms as their fruit-bodies. In this thesis, I explore spatial patterns of above-ground and below-ground diversity in West African ectomycorrhizal communities, as well as the use of wild edible fungi by the local human population. In Papers I and II , I used soil DNA metabarcoding to investigate fungal communities in their vegetative state. Paper I used measurements of the scale of spatial autocorrelation within the fungal community in Sudanian woodlands in Benin as a test to compare different high-throughput DNA sequencing strategies, including short (≈350 bp) amplicon barcoding using the Illumina, Ion Torrent, and PacBio sequencing platforms, as well as long (≈1550 bp) amplicon barcoding using PacBio. There were some differences in species richness and community composition recovered by the two amplicon lengths, attributable to known primer biases for the short amplicons, but these did not lead to different ecological results. Additionally, Paper I introduced new software packages for analysis of long-amplicon metabarcoding data and integrating phylogenetic information into sequence-based taxonomic identification. In Paper II we sampled both Sudanian woodland and gallery forest sites in five countries across West Africa using long-amplicon metabarcoding with PacBio. We found significant differences in fungal community composition between Sudanian woodland and gallery forest sites, but not between different tree species within each vegetation type. Papers III and IV focused above-ground to study the natural production and human use of the mushrooms themselves. In Paper III we exhaustively collected mushrooms from nine Sudanian woodland plots in Benin during three consecutive rainy seasons. We measured the total biomass produced of each morphospecies, as well as environmental variables related to microclimate, host tree availability, and soil chemistry. Mushroom production and diversity were negatively correlated with soil nitrogen levels, and positively correlated with soil phosphorus levels. Although there were no clear differences in the fungal communities associated with the four host trees present in our plots, greater host tree diversity was associated with greater fungal diversity and productivity. Finally, Paper IV combined interviews with local people from four ethnic groups in five villages near the study sites from Paper III about their knowledge and preferences for edible mushrooms with DNA barcoding of specimens. Knowledge and preferences for different mushrooms varied between ethnic groups, but people living in a village where their ethnic group is a minority tended to absorb knowledge about mushrooms from the majority group in the village. Women over age 35 were the most knowledgeable about mushrooms, but the degree of gender difference varied between groups. We compiled a list of the most choice edible mushrooms in the area, which can inform the possibility for commercial trade in wild mushrooms to supplement the income of rural people and encourage sustainable forest management.</p><p>Diss. (sammanfattning) Uppsala : Uppsala universitet, 2020</p><p></p><h1>Microbial DNA Sequencing in Environmental Studies [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/pxjkrs51m8nq3t87>http://libris.kb.se/bib/pxjkrs51m8nq3t87</a></p>
<p>The field of microbial ecology has just entered a new era of rapid technological development and generation of big data. The high-throughput sequencing techniques presently available provide an opportunity to extensively inventorize the blueprints of life. Now, millions of microbes of natural microbial communities can be studied simultaneously without prior cultivation. New species and new functions (genes) can be discovered just by mining sequencing data. However, there is still a tremendous number of microorganisms not yet examined, nor are the ecosystem functions these carry out. The modern genomic technologies can contribute to solve environmental problems and help us understand ecosystems, but to most efficiently do so, methods need to be continuously optimised.   During my Ph. D. studies, I developed a method to survey eukaryotic microbial diversity with a higher accuracy, and applied various sequencing-based approaches in an attempt to answer questions of importance in environmental research and ecology. In PAPER-I , we developed a set of 18S rRNA gene PCR primers with high taxonomic coverage, meeting the requirements of currently popular sequencing technologies and matching the richness of 18S rRNA reference sequences accumulated so far. In PAPER-II , we conducted the first sequencing-based spatial survey on the combined eukaryotic and bacterial planktonic community in the Baltic Sea to uncover the relationship of microbial diversity and environmental conditions. Here, the 18S primers designed in PAPER-I and a pair of broad-coverage 16S primers were employed to target the rRNA genes of protists and bacterioplankton for amplicon sequencing. In PAPER-III , we integrated metagenomic, metabarcoding, and metatranscriptomic data in an effort to scrutinise the protein synthesis potential (i.e., activity) of microbes in the sediment at a depth of 460 m in the Baltic Sea and, thus, disclosing microbial diversity and their possible ecological functions within such an extreme environment. Lastly, in PAPER-IV , we compared the performance of E. coli culturing, high-throughput sequencing, and portable real-time sequencing in tracking wastewater contamination in an urban stormwater system. From the aspects of cost, mobility and accuracy, we evaluated the usage of sequencing-based approaches in civil engineering, and for the first time, validated the real-time sequencing device in use within water quality monitoring.   In summary, these studies demonstrate how DNA sequencing of microbial communities can be applied in environmental monitoring and ecological research.</p><p>Diss. (sammanfattning) Stockholm : Kungliga Tekniska högskolan, 2017</p><p></p><h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/frvn2nftcjhdbl6d>http://libris.kb.se/bib/frvn2nftcjhdbl6d</a></p>
<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p><h1>What do we learn from cultures in the omics age? High-throughput sequencing and cultivation of leaf-inhabiting endophytes from beech (Fagus sylvatica L.) revealed complementary community composition but similar correlations with local habitat conditions [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/9qb56fsz7hz4zhlf>http://libris.kb.se/bib/9qb56fsz7hz4zhlf</a></p>
<p>Comparative simultaneous studies of environmental high-throughput sequencing (HTS) and cultivation of plant-associated fungi have rarely been conducted in the past years. For the present contribution, HTS and extinction culturing were applied for the same leaf samples of European beech (Fagus sylvatica) in order to trace both “real” environmental drivers as well as method-dependent signals of the observed mycobiomes. Both approaches resulted in non-overlapping community composition and pronounced differences in taxonomic classification and trophic stages. However, both methods revealed similar correlations of the fungal communities with local environmental conditions. Our results indicate undeniable advantages of HTS over cultivation in terms of revealing a good representation of the major functional guilds, rare taxa and biodiversity signals of leaf-inhabiting fungi. On the other hand our results demonstrate that the immense body of literature about cultivable endophytic fungi can and should be used for the interpretation of community signals and environmental correlations obtained from HTS studies and that cultivation studies should be continued at the highest standards, e.g. when sequencing facilities are not available or if such surveys are expanded into functional aspects with experiments on living isolates.</p><h1>Environmental DNA : for biodiversity research and monitoring</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/22521916>http://libris.kb.se/bib/22521916</a></p>
<p>Environmental DNA (eDNA), i.e. DNA released in the environment by any living form, represents a formidable opportunity to gather high-throughput and standard information on the distribution or feeding habits of species. It has therefore great potential for applications in ecology and biodiversity management. However, this research field is fast-moving, involves different areas of expertise and currently lacks standard approaches, which calls for an up-to-date and comprehensive synthesis. Environmental DNA for biodiversity research and monitoring covers current methods based on eDNA, with a particular focus on “eDNA metabarcoding”. Intended for scientists and managers, it provides the background information to allow the design of sound experiments. It revisits all steps necessary to produce high-quality metabarcoding data such as sampling, metabarcode design, optimization of PCR and sequencing protocols, as well as analysis of large sequencing datasets. All these different steps are presented by discussing the potential and current challenges of eDNA-based approaches to infer parameters on biodiversity or ecological processes. The last chapters of this book review how DNA metabarcoding has been used so far to unravel novel patterns of diversity in space and time, to detect particular species, and to answer new ecological questions in various ecosystems and for various organisms. Environmental DNA for biodiversity research and monitoring constitutes an essential reading for all graduate students, researchers and practitioners who do not have a strong background in molecular genetics and who are willing to use eDNA approaches in ecology and biomonitoring.</p><p>Specialized.</p><p>Includes bibliographical references and index.</p><p>1 Introduction to environmental DNA (eDNA) -- 2 DNA metabarcode choice and design -- 3 Reference databases -- 4 Sampling -- 5 DNA extraction -- 6 DNA amplification and multiplexing -- 7 DNA sequencing -- 8 DNA metabarcoding data analysis -- 9 Single-species detection -- 10 Environmental DNA for functional diversity -- 11 Some early landmark studies -- 12 Freshwater ecosystems -- 13 Marine environments -- 14 Terrestrial ecosystems -- 15 Paleoenvironments -- 16 Host-associated microbiota -- 17 Diet analysis -- 18 Analysis of bulk samples -- 19 The future of eDNA metabarcoding</p><p>Description based on online resource; title from home page (viewed on February 19, 2018).</p><p></p><h1>The Ecological Role of Roadside Stormwater Ponds - Potential to Support Biodiversity [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/drm91w3qbgpjqgnx>http://libris.kb.se/bib/drm91w3qbgpjqgnx</a></p>
<p>The increased recognition that roads may impair the aquatic environment and ecosystems has led to a shift from conventional drainage systems toward blue-green solutions such as stormwater ponds. Research on blue-green stormwater solutions has until now mainly focused on water quantity and quality. The aim of this study was to explore the ability of highway stormwater ponds to provide suitable habitats and support, especially macroinvertebrate biodiversity, and to identify the key environmental variables that affect biological community composition and the number of taxa, utilizing data at regional and global scales. Ultimately, this research informs design recommendations for stormwater systems that simultaneously provide multiple ecosystem services. The results of this thesis indicate that larger ponds are better for supporting aquatic biodiversity due to a more heterogeneous environment and the ability to dilute pollutants. Also, the presence of other ponds in the vicinity of the stormwater ponds can facilitate the movement of invertebrates between ponds through increased connectivity. An apparent negative effect of pollution levels on the macroinvertebrate community composition was observed, but not on the biodiversity measured as the number of taxa or Shannon index. The analyses based on the datasets identified using both morphology and DNA metabarcoding demonstrated that DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing more accurate information regarding the way the ponds function and how they are affected by management. Subsequently, the differences in the macroinvertebrate community composition between different types of ponds were compared at the regional and global scale. The results indicated that environmental characteristics, especially conductivity and pH, were different between different types of ponds. Alpha and gamma diversity were similar or even higher in manmade ponds compared to natural ponds due to very different macroinvertebrate communities. Moreover, generally ponds exhibited high levels of spatial heterogeneity, which subsequently enhances gamma diversity. In summary, stormwater ponds have the potential to provide suitable habitats to foster biodiversity. When such systems are created, larger ponds should be built to provide more heterogeneous habitat and dilute harmful pollutants. Additional ponds should also be created in the vicinity of the ponds, thereby promoting aquatic biodiversity through higher connectivity. Although stormwater ponds accumulate pollutants due to their primary functions, this pollution retention process creates a unique environment within the stormwater ponds, which are more suitable for taxa that are moderately to strongly tolerant to pollutants and that may not be found in natural ponds. In this way, stormwater ponds constitute an option in the areas along the highway so that they could combine water treatment properties with providing a suitable habitat for aquatic organisms.</p><h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy) [Elektronisk resurs]</h1>
<p>No DOI</p>
<p><a href=http://libris.kb.se/bib/2g044ft508hpppc2>http://libris.kb.se/bib/2g044ft508hpppc2</a></p>
<p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p><h1>eDNA in environmental monitoring</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>Einsatz von eDNA im Amphibien-Monitoring</h1>
<p><a href="https://doi.org/10.5167/uzh-142598">doi:10.5167/uzh-142598</a></p>

<p><a href=None>None</a></p>
<p>Monitoring ist ein unverzichtbarer Teil des Naturschutzes, denn Monitoring informiert über den Zustand der Biodiversität und liefert so die notwendigen Grundlagen für die Evaluation der praktischen Naturschutzarbeit und für die zu ergreifenden Schutzmassnahmen. Während klassische Monitorings mittels Nachweis von Arten oder Artengruppen direkt im Feld durchgeführt werden, sind kürzlich Methoden entwickelt worden, welche Umwelt-DNA (eDNA) für den Artnachweis verwenden. Im vorliegenden Artikel geben wir einen Überblick über eDNA-Methoden und deren Eignung für Monitorings, mit speziellem Fokus auf den Nachweis von Amphibien in Gewässern.</p><h1>Aquatic eDNA for monitoring French Guiana biodiversity</h1>
<p><a href="https://doi.org/10.3897/BDJ.7.e37518">doi:10.3897/BDJ.7.e37518</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA [eDNA] metabarcoding has recently emerged as a non-destructive alternative to traditional sampling for characterising species assemblages. We here provide a consistent dataset synthetising all eDNA sampling sites in French Guiana to date. Field collections have been initiated in 2014 and have continued until 2019. This dataset is however a work in progress and will be updated after each collecting campaign. We also provide a taxon by site matrix for fishes presence / absence as inferred from eDNA. Our aim is to allow a transparent communication to the stakeholders and provide the foundation for a monitoring programme based on eDNA. The lastest version of the dataset is publicly and freely accessible through the CEBA geoportal (http://vmcebagn-dev.ird.fr) or through the French Guiana geographic portal (https://www.geoguyane.fr).</p><h1>Perspectives on eDNA monitoring in Alpine waters</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>eDNA and metabarcoding for rewilding projects monitoring, a dietary approach</h1>
<p><a href="https://doi.org/10.1007/s42991-020-00032-y">doi:10.1007/s42991-020-00032-y</a></p>

<p><a href=None>None</a></p>
<p>There is an increased interest in the possibility to use large animals in the restoration of degraded ecosystems and to increase the capacity of natural areas to sustain richer biodiversity. We quantify the dietary differences and similarities between five moose and 22 red deer introduced in a 2100 ha fenced area under restoration (Lille Vildmose, Denmark) and their potential in the restoration process. Moose and red deer were selected based on their biogeographic affinity and existing knowledge of their diet and feeding behaviour, and are expected to counteract encroachment on open habitats. Using eDNA metabarcoding of dung collected in the field and species identification through saliva recovered from browsed twigs, we investigated the species’ diets and diet overlap after the moose were released into the area, and assessed how their diets match the expected ecosystem function. Despite the partial overlap of some dietary items, such as Betulaceae and Salicaceae, we report a differential use in height of tree species and distinctive use of other food resources between the two species, with red deer eating more herbaceous plants and the moose diet also containing aquatic plants. This study exemplifies a monitoring approach to restoration projects where large herbivores are expected to provide a key ecosystem function through their browsing and grazing.</p><h1>A roadmap for integrating eDNA in Australian marine park monitoring</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65257">doi:10.3897/aca.4.e65257</a></p>

<p><a href=None>None</a></p>
<p>Australia has one of the world's largest marine park estates. At 3.3 million km2, it spans an area three times larger than Germany, France, and the UK combined. Managing and monitoring such a vast and often remote area is logistically challenging and expensive. Current monitoring of Australian parks is decentralised and depends on traditional survey methods. As a result, real-time data on the state of Australia's marine parks is incomplete, hampering effective management. Environmental DNA has been suggested as a potential solution to some of these challenges, but practical large-scale applications remain largely lacking in Australia. To overcome this, we are developing a roadmap towards integrating eDNA methods in marine park monitoring. We present an overview of the current state of marine monitoring in Australia marine, identify the aspects of bio-monitoring that eDNA can best contribute to, and suggest pathways towards best practice use of eDNA for resource managers in Australia and globally.</p><h1>eDNA-based monitoring of parasitic plant (Sapria himalayana)</h1>
<p><a href="https://doi.org/10.1038/s41598-019-45647-5">doi:10.1038/s41598-019-45647-5</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Monitoring freshwater fish communities with environmental DNA (eDNA) metabarcoding</h1>
<p><a href="https://doi.org/10.13140/rg.2.2.32976.15364">doi:10.13140/rg.2.2.32976.15364</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Sediment eDNA Metabarcoding for monitoring impacts from offshore oil extraction</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65034">doi:10.3897/aca.4.e65034</a></p>

<p><a href=None>None</a></p>
<p>Routine biological monitoring of the areas affected by offshore oil drilling and extraction is critical for ensuring proper environmental management. In addition to sufficient knowledge of the ecosystem affected, formalised e.g. as biotic indices of indicator species, adequate temporal and spatial resolution is also required, to provide accurate information. As already demonstrated in several types of environments, environmental DNA (eDNA) metabarcoding offers an attractive alternative to current morphology-based assessments, including for impacts of oil extraction or spills.We have recently studied the influence of different experimental strategies on the accuracy of marine sediment metabarcoding, suggesting minimum criteria for technical and spatial replication (Hestetun et al. 2020). Here, we aim to evaluate the predictive power of this strategy, through agreement with assessments based on physicochemical measurements and current bioindicators. To this end, we targeted the metazoan, and total eukaryotic benthic communities, using COI and 18S V1-V2 markers, respectively. Sampled sites ranged from high to low impacts. The studied areas were located near active production installations and reference sites on the Norwegian continental shelf, in the North Sea and Barents Sea. As a proxy for accumulated impact, we developed a simple physicochemical pressure index (PI) based on total hydrocarbons, PAH16, barium and copper, all of which agreed well with impact reported from recent routine monitoring. Alpha diversity of both molecular datasets, as well as of morphology data, correlated strongly with this PI. However, the correlation was stronger yet with the macroinvertebrate-based Norwegian Sensitivity Index (NSI) derived from COI metabarcoding data, which also agreed well with NSI values derived from morphology-based monitoring. We also identified a set of bioindicator taxa from each of the two metabarcoding datasets, used to develop two novel metabarcoding-based biotic indeces. Using cross-validation, we demonstrated that predictions based on these indeces agreed well with PI. Predictive performance was better, and similar to NSI, for the COI-based index, but also high for the 18S-based version. In conclusion, this study demonstrates how de novo biotic indices can be developed, that perform comparably to existing biotic indices. We are confident that, using a larger set of samples, performance can be improved beyond that of current monitoring practices. Thanks to the reduced costs of eDNA analysis in comparison to morphological identifiation, this would also pave the way for improved spatial and temporal resolution employed in routine environmental monitoring. In doing so, it can also provide valuable raw data for improving our understanding of benthic ecology, biodiversity and its sensitivity to anthropogenic pressures.</p><h1>eDNA Metabarcoding Benchmarked towards Conventional Survey Methods in Amphibian Monitoring</h1>
<p><a href="https://doi.org/10.3390/ani12060763">doi:10.3390/ani12060763</a></p>

<p><a href=None>None</a></p>
<p>A keystone in protection work is accurate and thorough the monitoring of amphibian species, and the currently applied conventional survey methods are invasive, time-consuming, and dependent on expert knowledge. Research suggests that eDNA metabarcoding is a precise and cost-efficient method that could supplement the currently applied methods. The present study assessed the efficiency of conventional survey methods and eDNA metabarcoding in terms of species richness, the average number of detected species per site, the relative frequency of species occurrence, and the similarity of applied methods. The study found eDNA metabarcoding surveys to detect Lissotriton vulgaris (smooth newt), Triturus cristatus (great crested newt), Rana arvalis (moor frog), Rana temporaria (common frog), and Bufo bufo (common toad), as well as an average of 0.9 species per site, reflecting the species composition at the time of sampling in mid-July 2020. In addition to the species mentioned above, the conventional survey detected Epidalea calamita (natterjack toad) and an average of 1.7 species per site, reflecting the species composition at the time of sampling in early June 2020. The similarity between the methods applied in the present study was 27%, thus indicating a large number of unique observations of both eDNA metabarcoding and conventional surveys. The differences in detection can most likely be explained by the time of sampling, which was conducted a month apart. eDNA metabarcoding was efficient in detecting multiple amphibian species and produced unique observations that were not detected using conventional survey methods. Applying eDNA techniques as a supplement will most likely produce important knowledge on species distribution and presence, as well as enable more frequent monitoring due to cost efficiency and disturbance.</p><h1>101 lakes - lentic fish community monitoring using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65438">doi:10.3897/aca.4.e65438</a></p>

<p><a href=None>None</a></p>
<p>We have recently developed and deployed methods for environmental DNA (eDNA) based monitoring of lake fish communities in the UK. This approach combines eDNA with modern High-Throughput-Sequencing technology, so-called eDNA metabarcoding. This non-invasive method has proven to be more effective at detecting elusive species than established invasive surveying techniques such as electro fishing or fyke netting and can provide meaningful semi-quantitative abundance estimates. The UK Environment Agencies have funded the collection of an eDNA meta-barcoding data set of vertebrates from 101 UK lakes covering a broad spectrum of environmental conditions Fig. 1. This dataset is based on analysing 20 water samples per lake and has successfully been used to develop a WFD compatible water quality assessment tool. In its current form this tool is suitable for reporting the status of fish in water bodies where eutrophication is the dominant pressure.DNA is not homogeneously distributed in lentic environments and hence the detection of species relies on the collection of an adequate number of samples from a water body to capture the eDNA signal. Previous analyses on a subset of lakes have indicated that the number of samples used for the 101 lake fish data set is more than sufficient to reliably estimate species richness of lakes, but it is unclear how exactly reduced sampling effort affects other biodiversity estimates and inferences made about water quality. As the number of samples determines the cost of monitoring programmes it is essential that the sampling effort is optimised for a specific monitoring objective. The objective of this study was to explore the effect a reduced sampling effort would have on various biological inferences using algorithmic and statistical resampling techniques. with a much lower number of samples. For example, almost 90% of lakes achieved a sample coverage of 95% with only 10 samples. However, rare species are more often missed with fewer samples, with implications for monitoring programs of invasive or endangered species. Estimates of community composition and the ecological quality ratio (EQR) responded slowly to decreasing sampling effort. For example, subsets of 10 samples were in most cases much more closely related to each other than to sample sets from other lakes and showed very similar Ecological Quality Ratios. These results are able to inform the design of eDNA sampling strategies, so that these can be optimised to achieve specific monitoring goals.</p><h1>Monitoring spawning migrations of potamodromous fish species via eDNA</h1>
<p><a href="https://doi.org/10.6084/m9.figshare.9868193">doi:10.6084/m9.figshare.9868193</a></p>

<p><a href=None>None</a></p>
<p>Dataset for the titled publication. The file includes data of environmental variables, eDNA concentrations obtained from filtered water samples and fish counts, all obtained during the 2017 spawning season of Alburnus mento and Vimba vimba in the river Zeller Ache. Environmental variables and eDNA samples were measured / taken at two points in the rivers (data sheets ZA1 and ZA3), fish were counted in seven river sections, and daily discharge was provided by an automatic measurement station at the most upstream river section. See the "readme" tab for detailed information contained in the two tabs named "ZA1" and "ZA3".</p><h1>Monitoring Biodiversity and Water Pollution via High-Throughput eDNA Metabarcoding</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>Sampling Designs for Landscape‐level eDNA Monitoring Programs</h1>
<p><a href="https://doi.org/10.1002/ieam.4155">doi:10.1002/ieam.4155</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Technical guidelines for eDNA monitoring in Alpine waters</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>Applications of eDNA Monitoring for the Indian Rhinoceros (Rhinoceros unicornis)</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>The use of environmental DNA (eDNA) monitoring has shown great promise in the context of aquatic, marine, and recently semi-aquatic species, but little research has examined its potential use in terrestrial or semi-terrestrial mammals, such as the Indian rhinoceros (Rhinoceros unicornis). This rhinoceratid has experienced significant habitat fragmentation, pressure from poaching, and subsequent population decline in its range of India and Nepal. As such, effective conservation management techniques that are based on accurate distribution assessments—ideally, free from potential observer error, less influenced by elusive behavior, and of minimal disturbance to the individuals being observed—are crucial to continue to ensure the survival of this species. The objective of this study was to evaluate the potential use of eDNA monitoring for the Indian rhinoceros by collecting water samples from several locations along a lake system, at the beginning of which Indian rhinos are maintained in a semi-free ranging context. Quantitative PCR successfully amplified eDNA up to the farthest site approximately 2800m from the beginning of the lake system, suggesting that eDNA could potentially be a feasible monitoring technique with terrestrial mammals in general and the Indian rhinoceros specifically.</p><h1>Monitoring ESA-protected species in the City using eDNA</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>A method that extracts DNA from aquatic habitats has been used to explore fish population structure in City monitoring efforts. In such sampling, genetic material shed by organisms – environmental DNA (eDNA) – is collected by filtering streamflow, and analyzed for species-specific DNA arrangements using polymerase chain reaction (PCR) sequencing. BES has been using this passive method to sample reaches throughout its watersheds, often where and when conventional methods are not feasible. Here I will briefly present recent salmon, trout, and lamprey results, focusing on the Crystal Springs Creek reach of the Johnson Creek watershed.</p><h1>Developing Cost Effective Monitoring for Rainbow Smelt Using eDNA</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) tools developed at the University of Maine were successfully deployed in four coastal streams in Casco Bay, Maine in spring 2018 to detect the presence of anadromous rainbow smelt (Osmerus mordax), the first full application of this emerging technique. Field methods were refined and tested at sites with documented high (2) and low (2) spawning productivity. Samples were collected below known spawning areas immediately upstream of estuarine tidal influence 2-3 times each week during the spawning season. Three replicate samples were collected in the field at each site, as well as a contamination control, and all samples were filtered and preserved for laboratory analysis. Extracted eDNA samples and controls were run on three replicate qPCR assays. Initial efforts to extract eDNA from samples were hampered by the presence of environmental inhibitors. Use of a Zymo OneStep PCR Inhibitor Removal Kits appears to have overcome this problem and field collected eDNA samples were amplified successfully using quantitative polymerase chain reaction (qPCR). In partnership with a qualified lab, these tools can now provide a low-cost, userfriendly, and reliable method for monitoring the presence of rainbow smelt.</p><h1>Alternatives for trap monitoring in large rivers and lakes : Camera monitoring and eDNA sampling as alternative for conventional trap monitoring</h1>
<p><a href="https://doi.org/10.18174/503595">doi:10.18174/503595</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Peer Review #2 of "Monitoring newt communities in urban area using eDNA metabarcoding (v0.1)"</h1>
<p><a href="https://doi.org/10.7287/peerj.12357v0.1/reviews/2">doi:10.7287/peerj.12357v0.1/reviews/2</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Monitoring Chrysaora hysoscella (Cnidaria, Scyphozoa) in the Belgian part of the North Sea using eDNA</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>The use of Environmental DNA (eDNA) in monitoring ecosystems is now gaining attention in the field of research. The technique has shown a snapshot of the organisms present in the ecosystem being studied. Recent studies have demonstrated that the distribution and biomass of aquatic organisms can be estimated through detection and quantification of eDNA samples in the studied ecosystem. The approach is very rarely used in studying the detection and distribution of jellyfish in marine ecosystem. To investigate the technique’s applicability in detecting and monitoring marine gelatinous zooplankton, eDNA was used to monitory the compass jellyfish ( Chrysaora hysoscella ) in the southern North Sea. eDNA samples were collected from the surface water of the nine studied stations in the Belgian part of the North Sea (BPNS) from 2014 to 2016. In this study, eDNA samples from October 2014, March, May, August and October 2015, January, March, May and August 2016 were extracted using CTAB. eDNA extracts were run in a qPCR for C. hysoscella eDNA detection and quantification.This study detected C. hysoscella eDNA in the BPNS across the sampling months with a reduction in the frequency of detection in 2016. The target eDNA was found to be more common in Oostende then in Nieuwpoort and least in Zeebrugge stations. C. hysoscella eDNA was common and abundant in offshore stations and least in the shoreline stations. Peaks of eDNA abundance were recorded in spring, summer and autumn periods (October 2014, March, May and August 2016, March and August 2016). The recorded eDNA abundance was found to be not correlated with temperature (p = 0.4254). The results also revealed that the abundance of C. hysoscella eDNA somehow exhibited temporal and spatial variations. The results of this study imply that eDNA approach can be used to study the presence, patterns of distribution and the estimates of C. hysoscella biomass in the BPNS. This study confirms the broad potential of eDNA method in surveying ecosystems. The eDNA protocol used in the present study can be developed further to monitor jellyfish population in the BPNS obtaining a more detailed estimates of jellyfish abundance and distribution.</p><h1>Subsurface automated samplers for eDNA (SASe) for biological monitoring and research</h1>
<p><a href="https://doi.org/10.1016/j.ohx.2021.e00239">doi:10.1016/j.ohx.2021.e00239</a></p>

<p><a href=None>None</a></p>
<p></p><h1>The use of eDNA and DNA metabarcoding in monitoring the ecological condition of Norwegian lakes</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65309">doi:10.3897/aca.4.e65309</a></p>

<p><a href=None>None</a></p>
<p>Biodiversity assessments using molecular identification of organisms through high-throughput sequencing techniques have been a game changer in ecosystem monitoring, providing increased taxonomic resolution, more objective identifications, potential cost reductions, and reduced processing times. The use of DNA metabarcoding of bulk samples and environmental DNA (eDNA) is now widespread but is not yet universally implemented in national monitoring programs. While bulk sample metabarcoding involves extraction of DNA from organisms in a sample, eDNA analysis involves obtaining DNA directly from environmental samples, which can include microorganisms, meiofauna-size taxa and macrofauna traces such as larval stages, skin and hair cells, gametes, faeces and free DNA bound to particles.In Norway, freshwater biomonitoring in compliance with the EU Water Framework Directive (WFD) is conducted on several administrative levels, including national monitoring programs for running water, small and large lakes. These programs typically focus on a fraction of the actual biodiversity present in the monitored habitats (Weigand 2019). DNA metabarcoding of both bulk samples and eDNA samples are relevant tools for future freshwater biomonitoring in Norway.The aim of this PhD project is to develop assessment protocols based on DNA-metabarcoding and eDNA of benthic invertebrates, microcrustaceans and fish that can be used as standard biomonitoring tools to assess the ecological condition of lakes. The main topics addressed will be:- Development of protocols throughout the eDNA-metabarcoding workflow (i.e. sampling, filtration, preservation, extraction, amplification and sequencing) suitable to execute biodiversity assessments and determine the ecological status of lakes.- Comparison of the results obtained using molecular tools and traditional morphology-based approaches in order to assess the feasibility of such techniques to be incorporated as standard biomonitoring tools, such as the ones implemented under the provisions of the WFD.- Evaluate the effect of improved taxonomic resolution from molecular techniques on determining the ecological status of lakes, both by broadening the number of taxa analyzed and by identifying more taxa to species level.- Assess the feasibility of using eDNA extracted from water samples, taken at different depths and fish densities, to measure fish abundance/biomass as a proxy to calculate the ecological quality indices regulated in the WFD.- Analyze the coverage and resolution provided by reference libraries for certain taxa, such as crustacea, in order to assess the reliability and precision of taxonomic assignments.</p><h1>Data from: Monitoring a Norwegian freshwater crayfish tragedy - eDNA snapshots of invasion, infection and extinction</h1>
<p><a href="https://doi.org/10.5061/dryad.vf86jb2">doi:10.5061/dryad.vf86jb2</a></p>

<p><a href=None>None</a></p>
<p>1.The European Noble crayfish (Astacus astacus) is threatened by crayfish plague caused by the oomycete Aphanomyces astaci, which is spread by the invasive North American crayfish (e.g. signal crayfish, Pacifastacus leniusculus). Surveillance of crayfish plague status in Norway has traditionally relied on the monitoring survival of cage‐held noble crayfish, a method of ethical concern. Additionally, trapping is used in crayfish population surveillance. Here we test whether environmental DNA (eDNA) monitoring could provide a suitable alternative to the cage‐method, and a supplement to trapping. 2.We took advantage of an emerging crayfish plague outbreak in a Norwegian watercourse following illegal introduction of disease‐carrying signal crayfish, and initiated simultaneous eDNA‐monitoring and cage‐based surveillance, supplemented with trapping. A total of 304 water samples were filtered from several sampling stations over a four year period. eDNA data (species‐specific qPCR) for the presence of A. astaci, noble and signal crayfish within the water samples were compared to cage mortality and trapping. 3.This is the first study comparing eDNA‐monitoring and cage‐surveillance during a natural crayfish plague outbreak. We show that eDNA‐monitoring corresponds well with the biological status measured in terms of crayfish mortality and trapping results. eDNA analysis also reveals the presence of A. astaci in the water up to 2.5 weeks in advance of the cage‐method. eDNA estimates of A. astaci concentration and noble crayfish numbers increased markedly during mortality, and vanished quickly thereafter. eDNA provides a snapshot of the presence, absence or disappearance of crayfish regardless of season, and constitutes a valuable supplement to the trapping‐method that relies on season and legislation. 4.Synthesis and applications. Simultaneous eDNA‐monitoring of Aphanomyces astaci (crayfish plague) and relevant native and invasive freshwater crayfish species is well‐suited for early‐warning of invasion or infection, risk assessments, habitat evaluation and surveillance regarding pathogen and invasive/native crayfish status. This non‐invasive, animal‐welfare friendly method excludes the need for cage‐held susceptible crayfish in disease‐monitoring. Further, eDNA‐monitoring is less likely to spread A. astaci than traditional methods. This study resulted in the implementation of eDNA‐monitoring for Norwegian crayfish plague and crayfish surveillance programmes, and we believe other countries could improve management strategies for freshwater crayfish using a similar approach. ; eDNA results 2014-2017eDNA results from 2014-2017 in Haldenvassdraget watercourse.eDNA results 2015 - Statistical analysiseDNA results from 2015 used in statistical analysisR_script_CorrelationBetweenShortTimeSeries_eDNA_SnaphotR-script for statistical analysis</p><h1>The need for robust qPCR‐based eDNA detection assays in environmental monitoring and species inventories</h1>
<p><a href="https://doi.org/10.1002/edn3.164">doi:10.1002/edn3.164</a></p>

<p><a href=None>None</a></p>
<p></p><h1>A Review on Environmental DNA (eDNA) Metabarcoding Markers for Wildlife Monitoring Research</h1>
<p><a href="https://doi.org/10.1088/1755-1315/736/1/012054">doi:10.1088/1755-1315/736/1/012054</a></p>

<p><a href=None>None</a></p>
<p>Abstract Environmental DNA or eDNA utilizes traceable genetic materials in the environment for monitoring the presence of organisms in a given area and it is now gaining popularity as an alternative for traditional monitoring methods. Thus, the selection of genetic markers is crucial for identification of species in wildlife monitoring. This paper aims to review several DNA markers which are appropriate and reliable for detection of organisms from the environmental samples. We performed systematic literature search from SCOPUS database to review all molecular markers of eDNA. This study focuses on the importance of markers selection which can be utilized by next-generation sequencing (NGS) for biodiversity monitoring. Cytochrome C oxidase Subunit I (COI) are noted as the most widely used marker in metabarcoding research for detection of targeted species.</p><h1>Comparison of eDNA and conventional techniques for monitoring species diversity of boulder reefs in Danish waters</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>Development of a multi-assay approach for monitoring coral diversity using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.1007/s00338-019-01875-9">doi:10.1007/s00338-019-01875-9</a></p>

<p><a href=None>None</a></p>
<p></p><h1>eDNA: review of applicability for monitoring and detecting biotic populations of the Murray-Darling Basin</h1>
<p><a href="https://doi.org/10.26181/5d1ad38d652fe">doi:10.26181/5d1ad38d652fe</a></p>

<p><a href=None>None</a></p>
<p>MMCP Collaboration (MMCP) is a project supported by the Joint State Governments and the Murray-Darling Basin Authority to generate and adopt freshwater ecological knowledge through collaboration, to maintain research capability and contribute supporting science to underpin the Basin-Wide Watering Strategy. This synthesis paper is a response to a question poised by the Murray-Darling Basin Officials Committee (BOC), in relation to the value of eDNA. The objectives of this theme within the MMCP is to help the BOC address specific questions as they arise. These question will have relevance to the on-going management of Basin Assets.</p><h1>New eDNA based tool applied to the specific detection and monitoring</h1>
<p><a href="https://doi.org/10.1016/j.biocon.2020.108750">doi:10.1016/j.biocon.2020.108750</a></p>

<p><a href=None>None</a></p>
<p>This study was funded by the Government of Principado de Asturias, Spain (contract FUO-18-106 and FC-GRUPINIDI/2018/000201)</p><h1>Validation of a multi-primer assay for monitoring coral diversity using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.5281/zenodo.3413038">doi:10.5281/zenodo.3413038</a></p>

<p><a href=None>None</a></p>
<p>Raw data output from Scleractinia eDNA metabarcoding survey completed using multi-assay approach.</p><h1>A Systematic Review of Sources of Variability and Uncertainty in eDNA Data for Environmental Monitoring</h1>
<p><a href="https://doi.org/10.3389/fevo.2020.00135">doi:10.3389/fevo.2020.00135</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is becoming a standard tool in environmental monitoring that aims to quantify spatiotemporal variation for the measurement and prediction of ecosystem change. eDNA surveys have complex workflows encompassing multiple decision-making steps in which uncertainties can accumulate due to field sampling design, molecular biology lab work, and bioinformatics analyses. We conducted a quantitative review of studies published prior to December 2017 (n = 431) that had sampled eDNA from a variety of ecosystems and that had explicitly accounted for variability and uncertainty associated with eDNA workflows, either in their study design (e.g., replication) or data analysis (e.g., statistically modeling the spatiotemporal variation). We recorded differences among research studies in their spatial and temporal study design, the detected scales of natural variation in the study taxa, and how researchers measured and addressed the multiple sources of variability and uncertainty associated with the eDNA workflow. We show that relatively few studies used eDNA to understand temporal variation in biodiversity compared to spatial variation, and fewer described how uncertainties were addressed. We recommend increasing the number of temporal studies and to account for both natural variation and sources of uncertainty, such as imperfect detection, when undertaking eDNA surveys. Of studies that quantified spatiotemporal variation, this review identified gaps in the scales over which researchers have observed these patterns. Increasing the number of long-term and broad-scale eDNA studies will improve understanding of how useful eDNA is at scales relevant for monitoring the effects of environmental changes such as climatic shifts or land use change. Even where sources of spatiotemporal variation and uncertainty were accounted for, the effort in quantifying this variation differed among the different steps in the eDNA process, from field, to laboratory and bioinformatics procedures, depending on the type of community studied (micro- vs. macro-organism communities). We recommend more consistent experimental and modeling methods, accounting for spatiotemporal variation, and uncertainty in eDNA collection, and analysis, and incorporation of prior knowledge of sources of variability via Bayesian modeling approaches to account for uncertainties such as imperfect detection, to generate robust diversity estimates and increase the comparability of eDNA datasets for environmental monitoring across space and time.</p><h1>Monitoring vertebrate biodiversity of a protected coastal wetland using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.1002/edn3.200">doi:10.1002/edn3.200</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Data_Sheet_1_A National Scale “BioBlitz” Using Citizen Science and eDNA Metabarcoding for Monitoring Coastal Marine Fish.pdf</h1>
<p><a href="https://doi.org/10.3389/fmars.2022.824100.s001">doi:10.3389/fmars.2022.824100.s001</a></p>

<p><a href=None>None</a></p>
<p>Marine biodiversity is threatened by human activities. To understand the changes happening in aquatic ecosystems and to inform management, detailed, synoptic monitoring of biodiversity across large spatial extents is needed. Such monitoring is challenging due to the time, cost, and specialized skills that this typically requires. In an unprecedented study, we combined citizen science with eDNA metabarcoding to map coastal fish biodiversity at a national scale. We engaged 360 citizen scientists to collect filtered seawater samples from 100 sites across Denmark over two seasons (1 p.m. on September 29th 2019 and May 10th 2020), and by sampling at nearly the exact same time across all 100 sites, we obtained an overview of fish biodiversity largely unaffected by temporal variation. This would have been logistically impossible for the involved scientists without the help of volunteers. We obtained a high return rate of 94% of the samples, and a total richness of 52 fish species, representing approximately 80% of coastal Danish fish species and approximately 25% of all Danish marine fish species. We retrieved distribution patterns matching known occurrence for both invasive, endangered, and cryptic species, and detected seasonal variation in accordance with known phenology. Dissimilarity of eDNA community compositions increased with distance between sites. Importantly, comparing our eDNA data with National Fish Atlas data (the latter compiled from a century of observations) we found positive correlation between species richness values and a congruent pattern of community compositions. These findings support the use of eDNA-based citizen science to detect patterns in biodiversity, and our approach is readily scalable to other countries, or even regional and global scales. We argue that future large-scale biomonitoring will benefit from using citizen science combined with emerging eDNA technology, and that such an approach will be important for data-driven biodiversity management and conservation.</p><h1>Monitoring a Norwegian freshwater crayfish tragedy: eDNA snapshots of invasion, infection and extinction</h1>
<p><a href="https://doi.org/10.1111/1365-2664.13404">doi:10.1111/1365-2664.13404</a></p>

<p><a href=None>None</a></p>
<p>1. The European noble crayfish Astacus astacus is threatened by crayfish plague caused by the oomycete Aphanomyces astaci, which is spread by the invasive North American crayfish (e.g. signal crayfish Pacifastacus leniusculus). Surveillance of crayfish plague status in Norway has traditionally relied on the monitoring survival of cage‐held noble crayfish, a method of ethical concern. Additionally, trapping is used in crayfish population surveillance. Here, we test whether environmental DNA (eDNA) monitoring could provide a suitable alternative to the cage method, and a supplement to trapping. 2. We took advantage of an emerging crayfish plague outbreak in a Norwegian watercourse following illegal introduction of disease‐carrying signal crayfish, and initiated simultaneous eDNA monitoring and cage‐based surveillance, supplemented with trapping. A total of 304 water samples were filtered from several sampling stations over a 4‐year period. eDNA data (species‐specific quantitative real‐time PCR [qPCR]) for the presence of A. astaci, noble and signal crayfish within the water samples were compared to cage mortality and trapping. 3. This is the first study comparing eDNA monitoring and cage surveillance during a natural crayfish plague outbreak. We show that eDNA monitoring corresponds well with the biological status measured in terms of crayfish mortality and trapping results. eDNA analysis also reveals the presence of A. astaci in the water up to 2.5 weeks in advance of the cage method. Estimates of A. astaci and noble crayfish eDNA concentrations increased markedly during mortality and vanished quickly thereafter. eDNA provides a snapshot of the presence, absence or disappearance of crayfish regardless of season, and constitutes a valuable supplement to the trapping method that relies on season and legislation. 4. Synthesis and applications. Simultaneous eDNA monitoring of Aphanomyces astaci (crayfish plague) and relevant native and invasive freshwater crayfish species is well‐suited for early warning of invasion or infection, risk assessments, habitat evaluation and surveillance regarding pathogen and invasive/native crayfish status. This non‐invasive, animal welfare friendly method excludes the need for cage‐held susceptible crayfish in disease monitoring. Furthermore, eDNA monitoring is less likely to spread A. astaci than traditional methods. This study resulted in the implementation of eDNA monitoring for Norwegian crayfish plague and crayfish surveillance programmes, and we believe other countries could improve management strategies for freshwater crayfish using a similar approach. ; publishedVersion</p><h1>The application of eDNA for monitoring of the Great Crested Newt in the UK</h1>
<p><a href="https://doi.org/10.1002/ece3.1272">doi:10.1002/ece3.1272</a></p>

<p><a href=None>None</a></p>
<p>Current ecological surveys for great crested newts are time-consuming and expensive and can only be carried out within a short survey window. Additional survey methods which would facilitate the detection of rare or protected species such as the great crested newt (Triturus cristatus) would be extremely advantageous. Environmental DNA (eDNA) analysis has been utilized for the detection of great crested newts in Denmark. Here, the same methodology has been applied to water samples taken from UK ponds concurrently with conventional field surveying techniques. Our eDNA analysis exhibited an 84% success rate with a kappa coefficient of agreement between field and eDNA surveys of 0.86. One pond determined to be negative for great crested newt by field survey was positive by eDNA analysis, revealing the potential for improved detection rates using this methodology. Analysis of water samples collected in late summer indicates that eDNA analysis could be used to detect great crested newt after the optimal survey window for current field techniques had passed. Consequently, eDNA analysis could augment currently stipulated techniques for great crested newt surveying as a relatively quick and inexpensive tool for collecting great crested newt presence and distribution data within the UK instead of or prior to full field surveys.</p><h1>Phytoplankton assessment and Non-Indigenous species monitoring through eDNA metabarcoding in marine ecosystems</h1>
<p><a href="https://doi.org/10.3897/aca.4.e64983">doi:10.3897/aca.4.e64983</a></p>

<p><a href=None>None</a></p>
<p>Invasive Alien Species (IAS) are one of the most contributing constituents of biodiversity loss with the distribution range of an increasing number of IAS being artificially re-shaped by human activities. Rapid and effective tools are therefore needed in order to monitor changes in biodiversity and the dispersal of IAS. Molecular tools such as environmental DNA metabarcoding are being progressively included in the tool kit for IAS monitoring, however, the literature is largely focused on metazoans markers targeting regions such as COI and 18S and scarce studies include a marker for plants or algae. Here, we have tested a marker in ribulose-1,5-bisphosphate carboxylase-oxygenase chloroplast region to assess phytoplankton through environmental DNA metabarcoding. Several sample types (low and high-volume water filtration, sediment grabs, net tows) were deployed at twelve study sites along the Irish coast to determine which technique would perform better for the targeted group. A total of eleven NIS were detected during the study. Low-volume surface water filtering exceeded in NIS detection and recovered the highest diversity with 123 species detected. High volume samplers were the ones recovering the lowest number of OTUs (N= 53). Western sites (Galway Bay) had greater diversity and the highest number of total NIS compared to eastern sites in Carlingford, Dublin and Waterford. These results have shown the need for a specific IAS surveillance and has served as a basis to implement the use of eDNA metabarcoding and algae markers in future monitoring programs.</p><h1>Can eDNA metabarcoding offer a catchment-based approach for biodiversity monitoring?</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65651">doi:10.3897/aca.4.e65651</a></p>

<p><a href=None>None</a></p>
<p>In previous studies eDNA metabarcoding has been demonstrated as a viable tool for catchment-level biodiversity sampling in rivers (Deiner et al. 2016). However, questions still remain over the appropriate sampling protocol for large spatial scale sampling. River reaches are composed of multiple habitats with species composition varying from one to the next (Costa and Melo 2007). Therefore, how many spatial replicates are needed to reliably represent the river network? Is the previously used approach to sample at every river confluence (Deiner et al. 2016) sufficient or is more needed? These questions were addressed using a case study in the headwaters of the Cound Brook, a tributary to the River Severn in Shropshire, UK.Two sub-catchments of the Cound Brook were used. One sub-catchment had a sample taken at the most downstream point before the confluence. Additionally, a sample at the upstream extent of the same sub-catchment was taken to estimate any correlation between the species found at the beginning of the river reach and at the end. Another sub-catchment also had the same up- and downstream sample design. However, in between was a systemic sampling regime every 500 m. This is to test if increasing the spatial resolution gave significantly different results to the sparser sampling regime.At each sample location, a 1 L water sample was sequentially filtered through membranes of three different mesh sizes: 5µm, 0.45µm and 0.2µm. Sequential filtering was performed because DNA resides in two forms in the environment (Civade et al. 2017), within whole cells (cellular DNA) and outside of cells (extracellular). The theory is that the coarser filters predominantly collect cellular DNA and the finer filters collect predominantly extracellular DNA of increasingly smaller fragment lengths. Consequently, sequential filtering could represent DNA degradation in the environment (Fig. 1). Also, Turner et al. 2014 suggested that the larger particles could determine very recent or local organisms. Therefore, we hypothesised that the DNA collected by the coarser filters would represent local diversity and the DNA collected by the finer filters would reflect biodiversity further upstream.Initial results suggest sequential filtering through the 5µm and 0.45µm filters caught detectable levels of eDNA where the 0.2µm did not catch enough to show up through gel electrophoresis. The relevance of the initial finding suggests that if we only used a 5µm filter the data collected at 0.45µm could have been discarded. Further investigations of any differences in species compositions between filters and the relationships to other sampling locations is still to be determined. This ongoing research is intended to determine the appropriate sampling protocol for a large-scale biodiversity assessment regime combining eDNA metabarcoding and species distribution modelling.</p><h1>Utility of eDNA and occupancy models for monitoring an endangered fish across diverse riverine habitats</h1>
<p><a href="https://doi.org/10.1007/s10750-018-3723-8">doi:10.1007/s10750-018-3723-8</a></p>

<p><a href=None>None</a></p>
<p></p><h1>The use of environmental DNA (eDNA) in biological monitoring of the Natterjack toad (Epidalea calamita)</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) is a promising new tool in the field of conservation management, which can contribute to non-destructive species determination as well as improved data on species occurrence. Still, it is important to gain knowledge on how eDNA works compared to conventional monitoring methods in order to evaluate the detection efficiency. In this study, I have investigated the use of eDNA as a compliment to the traditional survey methods used in Sweden’s national biogeographical monitoring of the red listed Natterjack toad (Epidalea calamita). E. calamita occurs in 4 different counties, Västra Götaland having a large viable population distributed on small islands in Kattegat. In Skåne, Blekinge and Halland, where the species is not considered common, it occurs in a much wider range of habitats such as coastal rock-pools, heath meadows and gravel pits. The difference in population status and county, have resulted in two slightly different survey and sampling methodologies. The study demonstrates that eDNA proved to be a useful tool for species identification and species occurrence. The assurance of species identification and low amount of potential false positives and negatives, provide justification for the usefulness of eDNA in the national monitoring of E. calamita. However, difference in results between counties indicates that there is a need for site-specific protocols to assure the most reliable results for all counties. ; Can a water sample tell us if toads are present or not? In Sweden and other countries in Europe, many toads and frogs are declining or at risk of declining. But how can we know that? This is because their numbers are monitored and reported to EU every 6th year. Monitoring of amphibians is usually done by counting number of calling males or the number of tadpoles in wetlands. However, for some species this may a bit tricky because they are not always calling or it may be difficult to count the tadpoles in the water if they cannot be seen. The tadpoles may also be difficult to identify to species level. I have investigated if small water samples from wetlands contained DNA from two toad species in Sweden. Today, environmental DNA also called eDNA can possibly be used as a compliment when searching for different species. eDNA is DNA that originates from, for example, skin cells that the animal shed into the environment. A sample from the environment the species lives in therefore contains DNA if the species is present, and this DNA can be extracted in a laboratory. The endangered Natterjack toad (Epidalea calamita) breeds in small and shallow ponds during late spring and summer months, a water sample from these types of ponds can thus potentially reveal the presence of the species, even if the species is not there for the time of the monitoring. In Sweden the Natterjack toad exist in 4 counties: Västra Götaland, Skåne, Blekinge and Halland. A total of 114 wetlands have been monitored in these counties during 2016 and 2017 by estimating the number of tadpoles. Additionally, a water sample was taken from each of the 114 wetlands in order to extract DNA from the Natterjack toad and the Common toad. I have compared the success between the traditional monitoring (counting tadpoles and calling males) with the detection success by eDNA, both in terms of the number of wetlands in which the species were found to be present, but also if both monitoring methods gave the same results. The results from Västra Götaland was different from the other counties In Västra Götaland conventional monitoring methods showed that the Natterjack toad occurred in 53 % of the visited wetlands whereas eDNA only showed that the toad occurred in 37 %. This is different from the other counties, where both monitoring methods showed that the toad occurred in 33 % of the visited wetlands. In Västra Götaland traditional monitoring gave different results compared to eDNA analyses. This means that even if the species had been observed during traditional monitoring, eDNA analysis did not necessarily confirm the observation. There are many potential reasons behind these regional differences between methods and patterns; one could be that the sites in Västra Götaland, are known to be affected by salt water from Kattegat - the DNA might be to degraded or diluted to be detected. Overall, eDNA proved to be a useful tool in species identification, as field observations of both species of tadpoles was both confirmed and affirmed. Most importantly, eDNA also indicated the presence of the Natterjack toad at sites, where conventional methods would conclude it was absent. Even though the results are promising, site-specific protocols for collecting water samples for DNA extraction need to be developed. Master’s Degree Project in Biology 30 credits 2018 Department of Biology, Lund University Advisor: Per Nyström and Marika Stenberg Advisors company Ekoll AB</p><h1>Development of species-specific Cichla species eDNA primers for alien invasive species (AIS) monitoring in Malaysia</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65449">doi:10.3897/aca.4.e65449</a></p>

<p><a href=None>None</a></p>
<p>Peacock bass or the cichlids are known locally as top predator fishes which are invasive in Malaysia freshwater system. Detection probabilities for these fishes are typically low, especially using conventional capture-survey method due to the fish’s behaviour of hiding beneath the water’s surface. Hence, the environmental DNA (eDNA) monitoring is a relatively new approach that can be used to assess the distribution of these invasive fishes. Here, we report the strategy to develop small fragment (280- 400 bp) specific-specific primers for three selected invasive Cichla species namely, C. ocellaris, C. monoculus, and C. kelberi based on mitochondrial DNA (mtDNA) sequences. Current research showed that the developed species-specific primers from cytochrome oxidase I (COI) gene has high resolution at species level. Species-specific amplification tests also proved the specificity of the developed primers, securing the high- level species identification potential which may help in controlling the spread of alien invasive fish species.</p><h1>Peer Review #1 of "Monitoring newt communities in urban area using eDNA metabarcoding (v0.1)"</h1>
<p><a href="https://doi.org/10.7287/peerj.12357v0.1/reviews/1">doi:10.7287/peerj.12357v0.1/reviews/1</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Little samplers, big fleet: eDNA metabarcoding from commercial trawlers enhances ocean monitoring</h1>
<p><a href="https://doi.org/10.1016/j.fishres.2022.106259">doi:10.1016/j.fishres.2022.106259</a></p>

<p><a href=None>None</a></p>
<p></p><h1>eDNA‐based monitoring: Advancement in management and conservation of critically endangered killifish species</h1>
<p><a href="https://doi.org/10.1002/edn3.92">doi:10.1002/edn3.92</a></p>

<p><a href=None>None</a></p>
<p>Abstract Ecosystems are currently changing at unprecedented rates due to anthropogenic influences. Application of appropriate management regimes and mitigation measures requires knowledge of ecological community composition and monitoring of any changes that occur. Environmental DNA‐based monitoring is becoming increasingly common and offers substantial potential as a noninvasive method associated with highly repeatable and reliable results. In this study, we monitored river systems in Western Greece that have been strongly impacted by anthropogenic activities and the spread of an alien invasive fish species, the Eastern mosquitofish (Gambusia holbrooki). This invasive species has been credited as the major cause for the drastic decline of two endemic killifish species (Valencia letourneuxi and Valencia robertae). Here, we investigated the efficacy of an environmental DNA (eDNA‐based) method of detection for all three species, as an alternative to conventional monitoring methods. Initially, a mesocosm experiment provided material for the design and validation of the sampling protocol. This was followed by two sampling periods in the field conducted in autumn 2017 and 2018, comparing the novel eDNA assays with the conventional surveying methods in six and 20 systems, respectively. eDNA detection consistently outperformed the traditional monitoring methods for both V. letourneuxi and V. robertae and was comparable for the invasive G. holbrooki. This supports the now increasing body of literature, highlighting the benefits of species‐specific, targeted eDNA assays for the assessment of threatened and/or invasive species, one which can be utilized by conservation organizations and government bodies alike. However, we note that care should always be taken when designing such tools and strict validation steps should be adhered to, particularly with respect to minimizing the probability of false positives and negatives.</p><h1>Meta‐analysis supports further refinement of eDNA for monitoring aquatic species‐specific abundance in nature</h1>
<p><a href="https://doi.org/10.1002/edn3.7">doi:10.1002/edn3.7</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Prospects and challenges of environmental DNA (eDNA) monitoring in freshwater ponds</h1>
<p><a href="https://doi.org/10.1007/s10750-018-3750-5">doi:10.1007/s10750-018-3750-5</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Early detection of an emerging invasive species: eDNA monitoring of a parthenogenetic crayfish in freshwater systems</h1>
<p><a href="https://doi.org/10.3391/mbi.2019.10.3.04">doi:10.3391/mbi.2019.10.3.04</a></p>

<p><a href=None>None</a></p>
<p>Procambarus virginalis, also known as the Marmorkrebs is a highly invasive crayfish species characterized by parthenogenetic reproduction. As conservation management plans rely on the accuracy of the presence and distribution information of invasive species, a reliable method is needed for detecting such species in aquatic systems. We developed and validated a qPCR-based assay for monitoring P. virginalis at low abundance, by detecting their eDNA traces left in freshwater systems. We were able to implement this new assay in-situ at two separate lakes in Germany, where the crayfish were known to be present. Furthermore, we did not detect the pathogenic fungus Aphanomyces astaci in the locations where the Marmorkrebs were detected. We conclude that the use of eDNA is therefore a reliable tool for the early detection of this “perfect invader”. ; NA</p><h1>Correction to: eDNA and metabarcoding for rewilding projects monitoring, a dietary approach</h1>
<p><a href="https://doi.org/10.1007/s42991-020-00061-7">doi:10.1007/s42991-020-00061-7</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Evaluating eDNA-based monitoring of fire salamander larvae under field conditions</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65291">doi:10.3897/aca.4.e65291</a></p>

<p><a href=None>None</a></p>
<p>Amphibians globally belong to the most threatened animal groups and monitoring their populations is of critical importance for their conservation. The fire salamander (Salamandra salamandra) is one of the European amphibian species which has been experiencing drastic regional population declines due to the spread of the invasive chytrid fungus Batrachochytrium salamandrivorans, making it a key species for monitoring efforts. Here, we evaluated how the sampling and analysis of eDNA can aid the monitoring of larval salamander populations in small streams under field conditions. Nine small streams with known adult and larval fire salamander populations were investigated in Tirol (Austria). Per steam a stretch of 30 m downstream from its source was divided into 10 m sections where salamander larvae were counted. Water samples were taken at the end of each section and filtered on site. The DNA extracted from these filters was tested by a new PCR assay developed for the detection of mitochondrial DNA of S. salamandra. This assay combines endpoint PCR with capillary electrophoresis, allowing to relatively quantify the amount of fire salamander eDNA present in the water samples. In two of the nine streams no eDNA of S. Salamandra could be detected. The outcomes of an analysis of how larval densities, discharge and volume of filtered water affected the detection of salamander eDNA will be presented. Finally, we will conclude on the practical implications of the current findings for eDNA-based monitoring of fire salamander populations.</p><h1>Strategy for Successful Integration of eDNA-based Methods in Aquatic Monitoring</h1>
<p><a href="https://doi.org/10.3897/aca.4.e65125">doi:10.3897/aca.4.e65125</a></p>

<p><a href=None>None</a></p>
<p>Recent developments in the use of environmental DNA are opening up new horizons for the assessment of the quality of aquatic environments. These rapid and cost-effective methods, in very swift progress, will potentially offer the opportunity to identify all the taxa present in an environmental sample (water or biota) by the use of complementary markers. The produced inventories can then be used for the assessment of biodiversity and ecological quality. However, the inclusion of these new DNA-based methods in monitoring practices is not straightforward and requires harmonised actions in the coming years at national and international levels.In order to foresee and stimulate such a harmonised implementation, the European network DNAqua-Net (COST Action CA15219) brought together some of its members, experts of ECOSTAT and other environmental biomonitoring stakeholders from different European countries. Through workshops, bringing together 51 participants in 7 sub-groups in April 2020, an implementation roadmap was designed. The coordinated actions to be taken in the different countries, and the possible collaborations and steps to be taken at the EU level were identified.This presentation will give an overview of all discussions (Lefrançois et al. 2020) reflecting the diversity of situations in Europe, as well as common views. We will highlight important actions required for a successful implementation of DNA-based biomonitoring of aquatic ecosystems to the horizon of 2030.</p><h1>An eDNA-based method for monitoring a salmonid infectious disease: Development and application</h1>
<p><a href="https://doi.org/10.3897/aca.4.e64797">doi:10.3897/aca.4.e64797</a></p>

<p><a href=None>None</a></p>
<p>In the current context of global change, freshwater species are increasingly exposed to emerging infectious diseases (Okamura and Feist 2011). As an example, the Proliferative Kidney Disease (PKD) has emerged in salmonid fish during the last two decades, both in Europe and North America, causing important losses in aquaculture and worrying declines of several wild salmonid populations (Sudhagar et al. 2019). It is caused by Tetracapsuloides bryosalmonae, a myxozoan parasite with a complex life cycle involving two hosts: salmonids (intermediate host) and bryozoans (primary host). As PKD development strongly depends upon water temperature and quality, it is expected that global change could lead to more outbreaks (Okamura et al. 2011). Current monitoring of fish parasite load and infection status relies on histological observation or T. bryosalmonae DNA amplification out of kidney samples, involving fish euthanasia, and thus relatively small sample sizes when inferring infection prevalence. As large-scale screening of this parasite infections are required to better understand PKD dynamics, we have developed a non-lethal method for T. bryosalmonae detection in fish host based on the biological fact that T. bryosalmonae spores can be excreted from infected fish into the water through urine (Hedrick et al. 2004). This novel approach based on the detection of T. bryosalmonae DNA in fish urine was developed on wild brown trout (Salmo trutta), a species known to be an intermediate host of T. bryosalmonae and for releasing infective spores (only towards bryozoan host) through urine (Okamura et al. 2011). Applying this method, we have been able to map wild brown trout infection prevalence across 50 sites at the foothill of French Pyrenees and to identify the main environmental drivers of this disease.</p><h1>Monitoring a Norwegian freshwater crayfish tragedy: :eDNA snapshots of invasion, infection and extinction</h1>
<p><a href="https://doi.org/10.1111/1365-2664.13404">doi:10.1111/1365-2664.13404</a></p>

<p><a href=None>None</a></p>
<p>The European noble crayfish Astacus astacus is threatened by crayfish plague caused by the oomycete Aphanomyces astaci, which is spread by the invasive North American crayfish (e.g. signal crayfish Pacifastacus leniusculus). Surveillance of crayfish plague status in Norway has traditionally relied on the monitoring survival of cage-held noble crayfish, a method of ethical concern. Additionally, trapping is used in crayfish population surveillance. Here, we test whether environmental DNA (eDNA) monitoring could provide a suitable alternative to the cage method, and a supplement to trapping. We took advantage of an emerging crayfish plague outbreak in a Norwegian watercourse following illegal introduction of disease-carrying signal crayfish, and initiated simultaneous eDNA monitoring and cage-based surveillance, supplemented with trapping. A total of 304 water samples were filtered from several sampling stations over a 4-year period. eDNA data (species-specific quantitative real-time PCR [qPCR]) for the presence of A. astaci, noble and signal crayfish within the water samples were compared to cage mortality and trapping. This is the first study comparing eDNA monitoring and cage surveillance during a natural crayfish plague outbreak. We show that eDNA monitoring corresponds well with the biological status measured in terms of crayfish mortality and trapping results. eDNA analysis also reveals the presence of A. astaci in the water up to 2.5 weeks in advance of the cage method. Estimates of A. astaci and noble crayfish eDNA concentrations increased markedly during mortality and vanished quickly thereafter. eDNA provides a snapshot of the presence, absence or disappearance of crayfish regardless of season, and constitutes a valuable supplement to the trapping method that relies on season and legislation. Synthesis and applications. Simultaneous eDNA monitoring of Aphanomyces astaci (crayfish plague) and relevant native and invasive freshwater crayfish species is well-suited for early warning of invasion or infection, risk assessments, habitat evaluation and surveillance regarding pathogen and invasive/native crayfish status. This non-invasive, animal welfare friendly method excludes the need for cage-held susceptible crayfish in disease monitoring. Furthermore, eDNA monitoring is less likely to spread A. astaci than traditional methods. This study resulted in the implementation of eDNA monitoring for Norwegian crayfish plague and crayfish surveillance programmes, and we believe other countries could improve management strategies for freshwater crayfish using a similar approach.</p><h1>A novel trap design for non-lethal monitoring of dung beetles using eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.1007/s10841-021-00329-4">doi:10.1007/s10841-021-00329-4</a></p>

<p><a href=None>None</a></p>
<p>International audience</p><h1>Development of Genetic Markers for Environmental DNA (eDNA) Monitoring of Sturgeon</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>All living organisms shed DNA into the environment. Recent studies have demonstrated that environmental water samples can be effectively assayed for the DNA of target organisms. The DNA contained in these samples is commonly referred to as environmental DNA or eDNA. Because DNA assays can be sensitive to very small amounts of genetic material, eDNA surveys have the potential to detect sites where target organisms occur at very low abundances, potentially even when organisms are too rare for conventional survey methods to effectively detect them. We designed and tested twelve new eDNA markers for aquatic eDNA surveys of North American sturgeon. Eight of the markers are general for all North American sturgeon (i.e., not genus- or species-specific), two are specific to Scaphirhynchus sturgeon, and two are specific to white sturgeon (Acipenser transmontanus). The Scaphirhynchus marker could identify sites where endangered species like the pallid (S. albus) or Alabama (S. suttkusi) sturgeon might occur. Positive detections with the general sturgeon marker combined with the absence of positive detections with the white sturgeon marker could be used to discern if green sturgeon (A. medirostris) might occur at a location. All sturgeon markers were tested for specificity against a battery of 32 non-target fish species common to the Mississippi and Illinois River watersheds and the sensitivity or limit of detection for each marker was determined with assays of increasingly dilute solutions of target DNA. Four of the general sturgeon markers were used to assay 88 water samples from the Lower Mississippi River for sturgeon eDNA and resulted in 3 positive detections. While additional optimization of sturgeonspecific sampling protocols and marker assays is advised prior to use of these markers for sturgeon eDNA surveys, the newly designed markers represent a significant step forward in the use of eDNA in sturgeon monitoring and management.</p><h1>Development of species-specific eDNA-based test systems for monitoring of freshwater crayfish</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>This MONIS 5 report focuses on freshwater crayfish and was funded by the Danish Environmental Protection Agency. The aim of the present study was to develop species-specific systems for tracking environmental DNA (eDNA) from nine species of freshwater crayfish in water samples. Among these nine species, only one (Astacus astacus) can be considered indigenous in Scandinavia, and among the other eight non-indigenous species; two (Astacus leptodactylus and Pacifastacus leniusculus) are frequently encountered in Scandinavian freshwater systems. The remaining seven non-indigenous species are rarely encountered in Scandinavia but have been recorded in the past from other places in Europe. It is our aim that the nine species-specific eDNA assays presented here will allow for continuous monitoring of both the one indigenous species, the two non-indigenous species more frequently encountered and help produce early warnings of the seven non-indigenous species that might disperse to Denmark.</p><h1>Investigating the Potential Use of Environmental DNA (eDNA) for Genetic Monitoring of Marine Mammals</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0041781">doi:10.1371/journal.pone.0041781</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Increasing eDNA capabilities with CRISPR technology for real‐time monitoring of ecosystem biodiversity</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13084">doi:10.1111/1755-0998.13084</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Monthly eDNA Monitoring of an Invasive Bryozoan, Bugulina californica , in Seawater Using Species-Specific Markers</h1>
<p><a href="https://doi.org/10.3390/ani11071966">doi:10.3390/ani11071966</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) method used by many ecologists as effective investigation tool can detect endangered species, rare species, and invasive species. In case of invasive species, eDNA method help to monitor the target species when the species was hard to detect through the traditional survey such as the early stage of invasion, low abundance, and larva or juvenile stage. The bryozoan, Bugulina californica , was known as a marine fouling invasive species in Korea since its first reported in 1978. This species expanded nationwide, and damages to ascidian aquaculture through attached on the ship hulls and artificial facilities. To monitor the distribution and biomass of invasive bryozoan, B. californica , the qPCR analysis of environmental DNA was performed on seawater samples from 12 harbors. In this study, we designed species-specific markers which can calculate the detected DNA copies of B. californica , and the presence and monitoring of this species can be more accurately estimated by environmental DNA analysis than by traditional survey, in which it is difficult to identify the species. Real-time PCR analysis using environmental DNA is an effective monitoring method that can determine both the distribution and the monthly change in biomass of B. californica in Korea.</p><h1>Improving the containment of a freshwater invader using environmental DNA (eDNA) based monitoring</h1>
<p><a href="https://doi.org/10.1007/s10530-016-1203-5">doi:10.1007/s10530-016-1203-5</a></p>

<p><a href=None>None</a></p>
<p></p><h1>A National Scale “BioBlitz” Using Citizen Science and eDNA Metabarcoding for Monitoring Coastal Marine Fish</h1>
<p><a href="https://doi.org/10.3389/fmars.2022.824100">doi:10.3389/fmars.2022.824100</a></p>

<p><a href=None>None</a></p>
<p>Marine biodiversity is threatened by human activities. To understand the changes happening in aquatic ecosystems and to inform management, detailed, synoptic monitoring of biodiversity across large spatial extents is needed. Such monitoring is challenging due to the time, cost, and specialized skills that this typically requires. In an unprecedented study, we combined citizen science with eDNA metabarcoding to map coastal fish biodiversity at a national scale. We engaged 360 citizen scientists to collect filtered seawater samples from 100 sites across Denmark over two seasons (1 p.m. on September 29th 2019 and May 10th 2020), and by sampling at nearly the exact same time across all 100 sites, we obtained an overview of fish biodiversity largely unaffected by temporal variation. This would have been logistically impossible for the involved scientists without the help of volunteers. We obtained a high return rate of 94% of the samples, and a total richness of 52 fish species, representing approximately 80% of coastal Danish fish species and approximately 25% of all Danish marine fish species. We retrieved distribution patterns matching known occurrence for both invasive, endangered, and cryptic species, and detected seasonal variation in accordance with known phenology. Dissimilarity of eDNA community compositions increased with distance between sites. Importantly, comparing our eDNA data with National Fish Atlas data (the latter compiled from a century of observations) we found positive correlation between species richness values and a congruent pattern of community compositions. These findings support the use of eDNA-based citizen science to detect patterns in biodiversity, and our approach is readily scalable to other countries, or even regional and global scales. We argue that future large-scale biomonitoring will benefit from using citizen science combined with emerging eDNA technology, and that such an approach will be important for data-driven biodiversity management and conservation.</p><h1>Monitoring the eradication of the highly invasive topmouth gudgeon ( Pseudorasbora parva ) using a novel eDNA assay</h1>
<p><a href="https://doi.org/10.1002/edn3.12">doi:10.1002/edn3.12</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Early detection and spatial monitoring of an emerging biological invasion by population genetics and eDNA metabarcoding</h1>
<p><a href="https://doi.org/10.1111/csp2.86">doi:10.1111/csp2.86</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Past, present, and future perspectives of environmental DNA (eDNA) metabarcoding: A systematic review in methods, monitoring, and applications of global eDNA</h1>
<p><a href="https://doi.org/10.1016/j.gecco.2019.e00547">doi:10.1016/j.gecco.2019.e00547</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding is a novel method of assessing biodiversity wherein samples are taken from the environment via water, sediment or air from which DNA is extracted, and then amplified using general or universal primers in polymerase chain reaction and sequenced using next-generation sequencing to generate thousands to millions of reads. From this data, species presence can be determined, and overall biodiversity assessed. It is an interdisciplinary method that brings together traditional field-based ecology with in-depth molecular methods and advanced computational tools. As an emerging monitoring method, there are many pitfalls and roadblocks to be considered and avoided, but the method may still have the ability to revolutionize modern biodiversity surveys for the molecular era. In this paper, we review the basic methodology, benefits, and concerns of eDNA metabarcoding, and systematically cover the applications of the method in global ecology thus far, including biodiversity monitoring across all habitats and taxonomic groups, ancient ecosystem reconstruction, plant-pollinator interactions, diet analysis, invasive species detection, pollution responses, and air quality monitoring. We also discuss the future applications of the method as well as expected technological advances and how they may impact the way that eDNA metabarcoding may used in the future. eDNA metabarcoding is a unique method still in development and will likely remain in flux for some time as technology advances and procedures become standardized. However, as metabarcoding is optimized and its use becomes more widespread, it is likely to become an essential tool for ecological monitoring and global conservation study. Keywords: Environmental DNA, Metabarcoding, Methods, Monitoring, Applications, Systemic review</p><h1>Using eDNA to improve environmental monitoring for water bodies effected by hydropower in Sweden ; Användning av eDNA för att förbättra miljöövervakning av vattendrag påverkade av vattenkraft i Sverige</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>The aim of this report is to contribute to the base of knowledge on environmental monitoring by increasing understanding of how eDNA, electrofishing and sampling fishing may be used to examine fish biodiversity. It also aims at understanding if fish indexes developed within the Water Framework Directive reflect biodiversity, as well as the potential of eDNA data to serve as input to these indexes. This was done by using three different approaches. Firstly, in order to establish which of the methods eDNA, electrofishing and sampling fishing is more suitable to measure the different dimensions of biodiversity (species richness, species evenness and genetic diversity), a literature review comparing the different methods was carried out. It was found that eDNA yields a more detailed results for species richness, electrofishing yields better results for species evenness and sampling fishing is outperformed by eDNA and electrofishing alike. Both electrofishing and sampling fishing may collect data for genetic diversity analysis, however electrofishing outperforms sampling fishing with regards to amount of species caught, making electrofishing a more suitable data collection method. Secondly, in order to gain insight on practical usage of eDNA, a case study of Spjutmo (Dalarna county) was reviewed. It was established that eDNA generated more detailed information of species richness in the case of Spjutmo (as compared to electrofishing). The relative abundance data generated by the eDNA study might be seen as a measure of species evenness. However, electrofishing yielded data which may serve as input to species evenness indices. To the best knowledge of the author, none of the methods generated data on genetic diversity in this specific case. Officials from the energy company Fortum and the county board of Dalarna were also interviewed in order to get insight on what potential they see for eDNA to contribute to environmental monitoring. Both officials point at the ability to estimate abundance as a desired feature, hence a better understanding of what the relative abundance results indicates is wanted. The two interviews indicate that this understanding is an important feature to develop in order to make metabarcoding studies effective in current environmental monitoring. Thirdly, in order to understand if fish indexes developed within the Water Framework Directive reflect biodiversity, a literature review was performed. It was found that, all but one of the compared indexes incorporates or somewhat incorporates species richness. However, only five indexes are indicative or somewhat indicative of species richness. Species evenness is incorporated or somewhat incorporated by two indexes, which are also indicative or somewhat indicative of species evenness. None of the indexes incorporate or indicate genetic diversity. Within the third literature review, the potential of eDNA data to serve as input to current fish-based indexes developed within the Water Framework Directive, was studied. It was found that eDNA data may serve as input to only one index in its present form. However, five indexes also use proportional information (e.g. proportion of tolerant species), which possibly could be provided by eDNA data. The index where usage of eDNA data is currently possible uses presence-absence information. ; Målet med denna rapport är att bidra till kunskapsläget kring miljöövervakning genom att öka förståelsen för hur eDNA, elfiske och provfiske kan användas för att undersöka fisk biodiversitet. Målet är också att förstå om fisk-index utvecklade inom ramen för det Europeiska vattendirektivet reflekterar biodiversitet samt om data från eDNA kan utgöra input till dessa index. För att uppfylla dessa mål användes tre metoder. För att etablera vilken av metoderna eDNA, elfiske och provfiske är mer lämpad att mäta de olika dimensionerna av biodiversitet (artrikedom, distribution av arter och genetisk diversitet), genomfördes en litteraturstudie. Slutsatsen kunde dras att eDNA mäter artrikedom med högst noggrannhet, elfiske mäter distribution av arter mer detaljerat och att provfiske överträffas av både eDNA och elfiske i alla dimensioner. Både elfiske och provfiske kan samla data för analys av genetisk diversitet, men elfiske överträffar provfiske gällande hur många arter som fångas, vilket gör elfiske mer lämpligt som metod att samla in data för genetisk analys. För att få praktisk insikt i ett fall där eDNA använts, granskades en fallstudie från Spjutmo (i Dalarnas län). eDNA genererade mer detaljerad information om artrikedom än elfiske i detta fall. Datan genererad av eDNA kring relativ abundans mellan arter skulle kunna tolkas som ett mått på distribution av arter. Data genererad av elfiske kan å andra sidan användas som input till olika index för distribution av arter. Författaren veterligen, genererade varken eDNA eller elfiske mått på genetisk diversitet i detta specifika fall. Två personer, en från Fortum och en från länsstyrelsen Dalarna intervjuades också för att få insikt i deras syn på potentialen av att använda eDNA som ett miljöövervakningsverktyg. Båda intervjupersonerna pekade på att en bättre förståelse av de relativa abundansvärdena indikerar är önskad. Båda intervjupersonerna pekade på att det är en viktig aspekt för att metabarcoding studier ska vara effektiva i nuvarande miljöövervakning. För att förstå om fisk-index utvecklade för EU’s vattendirektiv reflekterar biodiversitet, genomfördes en komparativ litteraturstudie av index. Alla index förutom ett inkorporerar eller delvis inkorporerar artrikedom. Bara fem indikerar eller delvis indikerar artrikedom. Distribution av arter inkorporeras eller delvis inkorporeras av två index som också indikerar eller delvis indikerar distribution av arter. Inom den komparativa litteraturstudien av index, studerades även potential att fungera som input av data genererad av eDNA till indexen. Data genererad av eDNA kan i dagsläget fungera som input till ett av indexen. Fem index använder någon form av proportionell data (t.ex. proportion av toleranta arter), som möjligen skulle kunna ges av eDNA. Indexet till vilket det är möjligt att använda eDNA data använder närvarande-ej närvarande information som input.</p><h1>Development of species-specific eDNA-based test systems for monitoring of freshwater crayfish ; Udvikling af arts-specifikke eDNA-baserede testsystemer til overvågning af ferskvandskrebs</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Project manager Jesper Harbo Andersen ; This MONIS 5 report focuses on freshwater crayfish and was funded by the Danish Environmental Protection Agency. The aim of the present study was to develop species-specific systems for tracking environmental DNA (eDNA) from nine species of freshwater crayfish in water samples. Among these nine species, only one (Astacus astacus) can be considered indigenous in Scandinavia, and among the other eight non-indigenous species; two (Astacus leptodactylus and Pacifastacus leniusculus) are frequently encountered in Scandinavian freshwater systems. The remaining seven non-indigenous species are rarely encountered in Scandinavia but have been recorded in the past from other places in Europe. It is our aim that the nine species-specific eDNA assays presented here will allow for continuous monitoring of both the one indigenous species, the two non-indigenous species more frequently encountered and help produce early warnings of the seven non-indigenous species that might disperse to Denmark. ; Danish Environmental Protection Agency ; publishedVersion</p><h1>Importance of sampling design using an eDNA monitoring approach for pond-living amphibians ; Vikten av provtagningskontroll vid eDNA analyser av dammlevande amphibier</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>The eDNA approach for monitoring aquatic species is already widely used. However, there are still uncertainties, for instance regarding the distribution of target DNA in ponds. These uncertainties need to be addressed when using eDNA in management. The current study uses individual capture and eDNA monitoring methods based on bottle trapping and 3x15 ml water sampling for Triturus cristatus in different ponds to show the importance of a carefully considered and performed study and in particular sampling design for achieving reliable results. Most of the results were received from only one pond. While here the bottle trapping resulted only in 15 catches, the eDNA method showed better results in confirming presence of the target. Examining two different primer combinations, the one with the bigger amplicon was more reliable than the one with the shorter amplicon. However, the data clearly indicated an influence of the sampling location on the eDNA results. This is also true for weather factors and / or sample time within the season. Therefore the sampling design has to be reconsidered, with respect to DNA distribution and persistence, seasonal habitation differences of the target and the primer specifics.</p><h1>eDNA-based crayfish plague monitoring is superior to conventional trap-based assessments in year-round detection probability</h1>
<p><a href="https://doi.org/10.1007/s10750-017-3408-8">doi:10.1007/s10750-017-3408-8</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Can environmental DNA (eDNA) be used for detection and monitoring of introduced crab species in the Baltic Sea?</h1>
<p><a href="https://doi.org/10.1016/j.marpolbul.2016.05.054">doi:10.1016/j.marpolbul.2016.05.054</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Stones in the road prevents effective implementation of eDNA-based freshwater quality monitoring in the Iberian Peninsula</h1>
<p><a href="https://doi.org/10.3897/aca.4.e64974">doi:10.3897/aca.4.e64974</a></p>

<p><a href=None>None</a></p>
<p>The Water Framework Directive (WFD) monitoring requires the use of stressor-specific Multimetric Indices, intercalibrated and validated at river basin level, such as the IBMWP, which is one of the most widely used indices for the biomonitoring of rivers in the Iberian Peninsula. This qualitative index is based on the identification mainly at family level of 125 groups of macroinvertebrates. Currently, molecular biomonitoring systems (biomonitoring 2.0) is widely considered as a good alternative to conduct water quality assessments. The problem comes with the requirement of having complete DNA sequences databases for the targeted species within the indicator groups. In order to know the percentage of sequence coverage of the different taxonomic groups, is needed to know the number of freshwater species in each group. The fact that currently it is only necessary to identify them morphologically at family level to calculate the index means that there are no exhaustive lists of lower taxonomic ranges. In this study, we have studied the available taxonomic databases at species level for river macroinvertebrates in the Iberian Peninsula. For this purpose, species geographically referred to the Iberian Peninsula were extracted from three sources: two European (Freshwaterecology.info database and Weigand et al. (2019)) and one Iberian database/list (Múrria et al. 2020). The resultant species list was compared with GBIF (Global Biodiversity Information Facility) using its tool "Species Matching" to find synonyms and fuzzy names. The similarities and differences between databases were studied (considering the synonyms but not the fuzzy names) and a new list of 3586 species was constructed. Our analysis revealed that from 125 taxonomic groups covered by the IBMWP, there are no species collected for 11 of them (9%) and for another 34 groups (27%) there are less than 5 species collected in the databases. Then, using the BAGS software, all sequences of the mitochondrial gene COX1 were mined from BOLD database. From the 3586 species in the list, only 1900 (53%) have at least one sequence in BOLD. Therefore, conducting biomonitoring 2.0 in Iberian freshwater ecosystems is far from effective today. We need to overcome two big obstacles to reach the goal of using molecular biomonitoring in the Iberian Peninsula: a complete taxonomic list of quality indicators species by groups, and to sequence DNA barcodes from all relevant species.</p><h1>Using eDNA to develop a national citizen science-based monitoring programme for the great crested newt (Triturus cristatus)</h1>
<p><a href="https://doi.org/10.1016/j.biocon.2014.11.029">doi:10.1016/j.biocon.2014.11.029</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Fishing for mammals: landscape-level monitoring of terrestrial and semi-aquatic communities using eDNA from lotic ecosystems</h1>
<p><a href="https://doi.org/10.5061/dryad.d51c59zzf">doi:10.5061/dryad.d51c59zzf</a></p>

<p><a href=None>None</a></p>
<p>1. Environmental DNA (eDNA) metabarcoding has revolutionised biomonitoring in both marine and freshwater ecosystems. However, for semi-aquatic and terrestrial animals, the application of this technique remains relatively untested. 2. We first assess the efficiency of eDNA metabarcoding in detecting semi-aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, using occupancy modelling we compared the detection efficiency of eDNA metabarcoding to multiple conventional non-invasive survey methods (latrine surveys and camera trapping). 3. eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA sampling in areas where authenticated records do not currently exist, but potential false positives were also identified. 4. Water-based eDNA metabarcoding provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole, and red deer) using occupancy models. The comparison between survey 'effort' to reach a detection probability of ≥0.95 revealed that 3-6 water replicates would be equivalent to 3-5 latrine surveys and 5-30 weeks of single camera deployment, depending on the species. 5. Synthesis and Applications. eDNA metabarcoding can be used to generate an initial 'distribution map' of mammalian diversity at the landscape level. If conducted during times of peak abundance, carefully chosen sampling points along multiple river courses provide a reliable snapshot of the species that are present in a catchment area. In order to fully capture solitary, rare and invasive species, we would currently recommend the use of eDNA metabarcoding alongside other non-invasive surveying methods (i.e. camera traps) to maximize monitoring efforts.</p><h1>Holistic pelagic biodiversity monitoring of the Black Sea via eDNA metabarcoding approach: From bacteria to marine mammals</h1>
<p><a href="https://doi.org/10.1016/j.envint.2019.105307">doi:10.1016/j.envint.2019.105307</a></p>

<p><a href=None>None</a></p>
<p>As the largest semi-closed marine ecosystem in the world, the Black Sea has been heavily affected by human activities for a long time. Describing the biodiversity of multi-trophic biota in pelagic zone of the Black Sea and identifying the dominant environmental factors are prerequisites for protecting the sustainability of ecosystems. However, up to now, the taxonomic and distributional information about the Black Sea biota is not clear. Here, we employed a Tree-of-Life metabarcoding to analyze the biodiversity of eight communities in the Black Sea, investigated their biogeographical distribution, and further analyzed the influence of biological and abiotic factors on biota on large scales. We found that, (1) Over 8900 OTUs were detected in the Black Sea, of which 630 species were identified, covering the holistic biota from single-celled (bacteria 5620 OTUs 141 species; algae 1096 OTUs 185 species; protozoa 546 OTUs 146 species) to multicellular organisms (invertebrate metazoans 150 OTUs 34 species; fishes 1369 OTUs 76 species; large marine mammals 39 OTUs 5 species). (2) Higher trophic organisms (fishes and large mammals) distributed more evenly in space than the lower (microorganisms, protozoa and invertebrates). For lower trophic organisms, the vertical stratification was more obvious than the horizontal stratification (vertical p < 0.02, horizontal p < 0.05). (3) The bottom trophic organisms (bacteria and algae) of the food web significantly affected the distribution and composition of the others through biological interactions (Mantel p < 0.05). (4) At the level of abiotic factors, the effect of local species sorting on the composition of communities was 15% higher than that of mass dispersal effect. For the first time, this study monitored and profiled the holistic biodiversity in the pelagic zone of the Black Sea, and provided technological advances and preliminary knowledge for the ongoing Black Sea ecosystem protection efforts. Keywords: Black Sea, Multi-trophic biodiversity, eDNA metabarcoding, Interactome, Dispersal, Species sorting</p><h1>Fishing for mammals: Landscape-level monitoring of terrestrial and semi-aquatic communities using eDNA from riverine systems</h1>
<p><a href="https://doi.org/10.1111/1365-2664.13592">doi:10.1111/1365-2664.13592</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) metabarcoding has revolutionized biomonitoring in both marine and freshwater ecosystems. However, for semi-aquatic and terrestrial animals, the application of this technique remains relatively untested. We first assess the efficiency of eDNA metabarcoding in detecting semi-aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, using occupancy modelling we compared the detection efficiency of eDNA metabarcoding to multiple conventional non-invasive survey methods (latrine surveys and camera trapping). eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA sampling in areas where authenticated records do not currently exist, but potential false positives were also identified. Water-based eDNA metabarcoding provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole and red deer) using occupancy models. The comparison between survey ‘effort’ to reach a detection probability of ≥.95 revealed that 3–6 water replicates would be equivalent to 3–5 latrine surveys and 5–30 weeks of single camera deployment, depending on the species. Synthesis and applications. eDNA metabarcoding can be used to generate an initial ‘distribution map’ of mammalian diversity at the landscape level. If conducted during times of peak abundance, carefully chosen sampling points along multiple river courses provide a reliable snapshot of the species that are present in a catchment area. In order to fully capture solitary, rare and invasive species, we would currently recommend the use of eDNA metabarcoding alongside other non-invasive surveying methods (i.e. camera traps) to maximize monitoring efforts. © 2020 British Ecological Society</p><h1>Monitoring the silver carp invasion in Africa: a case study using environmental DNA (eDNA) in dangerous watersheds</h1>
<p><a href="https://doi.org/10.3897/neobiota.56.47475">doi:10.3897/neobiota.56.47475</a></p>

<p><a href=None>None</a></p>
<p>Volume: 56 ; Start Page: 31-47</p><h1>DNA metabarcoding in insect biodiversity monitoring - an application of eDNA for non-lethal detection of wild bees</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>High-throughput eDNA surveys for benthic monitoring of salmon farms in Norway: a validation study. FHF project 901092</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p></p><h1>Isothermal amplification of environmental DNA (eDNA) for direct field-based monitoring and laboratory confirmation of Dreissena sp</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0186462">doi:10.1371/journal.pone.0186462</a></p>

<p><a href=None>None</a></p>
<p>Loop-mediated isothermal amplification (LAMP) of aquatic invasive species environmental DNA (AIS eDNA) was used for rapid, sensitive, and specific detection of Dreissena sp. relevant to the Great Lakes (USA) basin. The method was validated for two uses including i) direct amplification of eDNA using a hand filtration system and ii) confirmation of the results after DNA extraction using a conventional thermal cycler run at isothermal temperatures. Direct amplification eliminated the need for DNA extraction and purification and allowed detection of target invasive species in grab or concentrated surface water samples, containing both free DNA as well as larger cells and particulates, such as veligers, eggs, or seeds. The direct amplification method validation was conducted using Dreissena polymorpha and Dreissena bugensis and uses up to 1 L grab water samples for high target abundance (e.g., greater than 10 veligers (larval mussels) per L for Dreissena sp.) or 20 L samples concentrated through 35 μm nylon screens for low target abundance, at less than 10 veligers per liter water. Surface water concentrate samples were collected over a period of three years, mostly from inland lakes in Michigan with the help of a network of volunteers. Field samples collected from 318 surface water locations included i) filtered concentrate for direct amplification validation and ii) 1 L grab water sample for eDNA extraction and confirmation. Though the extraction-based protocol was more sensitive (resulting in more positive detections than direct amplification), direct amplification could be used for rapid screening, allowing for quicker action times. For samples collected between May and August, results of eDNA direct amplification were consistent with known presence/absence of selected invasive species. A cross-platform smartphone application was also developed to disseminate the analyzed results to volunteers. Field tests of the direct amplification protocol using a portable device (Gene-Z) showed the method could be used in the field to obtain results within one hr (from sample to result). Overall, the direct amplification has the potential to simplify the eDNA-based monitoring of multiple aquatic invasive species. Additional studies are warranted to establish quantitative correlation between eDNA copy number, veliger, biomass or organismal abundance in the field.</p><h1>eDNA detection of native and invasive crayfish species allows for year-round monitoring and large-scale screening of lotic systems</h1>
<p><a href="https://doi.org/10.3389/fenvs.2021.639380">doi:10.3389/fenvs.2021.639380</a></p>

<p><a href=None>None</a></p>
<p>Effective management of both endangered native and invasive alien crayfishes requires knowledge about distribution, monitoring of existing and early detection of newly established populations. Complementary to traditional survey methods, eDNA sampling has recently emerged as a highly sensitive non-invasive detection method to monitor crayfish populations. To advance the use of eDNA as detection tool for crayfish we used a twofold approach: 1) we designed a novel set of specific eDNA-assays for all native (Austropotamobius torrentium, Austropotamobius pallipes, Astacus astacus) and the most relevant invasive crayfish species (Pacifastacus leniusculus, Faxonius limosus, Faxonius immunis) in Central Europe. To ensure specificity each primer pair was tested in silico, in vitro, and in situ; 2) we assessed the influence of spatio-temporal variables (distance to upstream population, season, stream size) on eDNA detection in seven streams using two different detection methods (qualitative endpoint PCR and quantitative droplet digital PCR, ddPCR). The newly developed eDNA assays successfully detected all crayfish species across different lotic and lentic habitats. eDNA detection rate (endpoint PCR) and eDNA-concentration (ddPCR) were significantly influenced by distance and season. eDNA detection was successful up to 7 km downstream of the source population and across all seasons, although detectability was lowest in winter. eDNA detection rate further decreased with increasing stream size. Finally, eDNA-concentration correlated positively with estimated upstream population size. Overall, we provide near operational eDNA assays for six crayfish species, enabling year-round detection, which represents a clear benefit over conventional methods. Due to its high sensitivity, eDNA detection is also suitable for the targeted search of as-yet unrecorded or newly emerging populations. Using quantitative ddPCR might further allow for a rough estimation of population size, provided that the identified spatio-temporal factors are accounted for. We therefore recommend implementing eDNA-detection as a complementary survey tool, particularly for a large-scale screening of data-deficient catchments or a year-round monitoring.</p><h1>Fishing for mammals: Landscape‐level monitoring of terrestrial and semi‐aquatic communities using eDNA from riverine systems</h1>
<p><a href="https://doi.org/10.1111/1365-2664.13592">doi:10.1111/1365-2664.13592</a></p>

<p><a href=None>None</a></p>
<p>This is the peer reviewed version of the following article: Sales NG, McKenzie, Drake, Harper, Browett SS, Coscia I, Wangensteen Fuentes OS, Baillie C, Bryce E, Dawson DA, Ochu, Hänfling B, Lawson Handley L, Mariani S, Lambin X, Sutherland, McDevitt AD. Fishing for mammals: Landscape‐level monitoring of terrestrial and semi‐aquatic communities using eDNA from riverine systems. Journal of Applied Ecology, 57 , 707-716, which has been published in final form at https://doi.org/10.1111/1365-2664.13592. This article may be used for non-commercial purposes in accordance with Wiley Terms and Conditions for Use of Self-Archived Versions. ; 1: Environmental DNA (eDNA) metabarcoding has revolutionized biomonitoring in both marine and freshwater ecosystems. However, for semi‐aquatic and terrestrial animals, the application of this technique remains relatively untested. 2: We first assess the efficiency of eDNA metabarcoding in detecting semi‐aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, using occupancy modelling we compared the detection efficiency of eDNA metabarcoding to multiple conventional non‐invasive survey methods (latrine surveys and camera trapping). 3: eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA sampling in areas where authenticated records do not currently exist, but potential false positives were also identified. 4: Water‐based eDNA metabarcoding provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole and red deer) using occupancy models. The comparison between survey ‘effort’ to reach a detection probability of ≥.95 revealed that 3–6 water replicates would be equivalent to 3–5 latrine surveys and 5–30 weeks of single camera deployment, depending on the species. 5: Synthesis and applications : eDNA metabarcoding can be used to generate an initial ‘distribution map’ of mammalian diversity at the landscape level. If conducted during times of peak abundance, carefully chosen sampling points along multiple river courses provide a reliable snapshot of the species that are present in a catchment area. In order to fully capture solitary, rare and invasive species, we would currently recommend the use of eDNA metabarcoding alongside other non‐invasive surveying methods (i.e. camera traps) to maximize monitoring efforts.</p><h1>Environmental DNA (eDNA) detects the pool frog (Pelophylax lessonae) at times when traditional monitoring methods are insensitive</h1>
<p><a href="https://doi.org/10.1038/s41598-018-23740-5">doi:10.1038/s41598-018-23740-5</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Evaluation of eDNA for groundwater invertebrate detection and monitoring: a case study with endangered Stygobromus (Amphipoda: Crangonyctidae)</h1>
<p><a href="https://doi.org/10.1007/s12686-017-0785-2">doi:10.1007/s12686-017-0785-2</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Monitoring the silver carp invasion in Africa : a case study using environmental DNA (eDNA) in dangerous watersheds</h1>
<p><a href="https://doi.org/10.3897/neobiota.56.47475">doi:10.3897/neobiota.56.47475</a></p>

<p><a href=None>None</a></p>
<p>CITATION: Crookes, S., et al. 2020.Monitoring the silver carp invasion in Africa : a case study using environmental DNA (eDNA) in dangerous watersheds. NeoBiota 56: 31-47, doi:10.3897/neobiota.56.47475. ; The original publication is available at https://neobiota.pensoft.net ; Biodiverse habitats are increasingly subject to an intensification of anthropogenic stressors that may severely diminish species richness. Invasive species pose a dominant threat to biodiversity and biosecurity, particularly in biodiversity hotspots like Kruger National Park, South Africa. The invasive silver carp, Hypophthalmichthys molitrix, was introduced into the Olifants River and may experience range spread owing to favorable environmental conditions. Intensive monitoring protocols are necessary to effectively manage invasions of species like silver carp. Unfortunately, tropical and sub-tropical aquatic systems are difficult to monitor using conventional methods (e.g., netting, electrofishing and snorkeling) owing to a range of factors including the presence of dangerous megafauna. Conservation of such systems may be advanced by the adoption of novel methods, including environmental DNA (eDNA) detection. Here, we explore the utility of environmental DNA (eDNA) to conduct safe, reliable and repeatable surveys in dangerous watersheds using silver carp as a case study. We conducted eDNA surveys at 12 sites in two neighbouring watersheds, and determined that the species has expanded its range within the Olifants River and to the south in the Sabie River. Expansion in the former is consistent with the presence of suitable spawning conditions. We discuss the implications of this survey for biodiversity monitoring in similar aquatic systems in the tropics and advocate an integrative approach to biomonitoring in these ecosystems. ; https://neobiota.pensoft.net/article/47475/ ; Publisher's version</p><h1>Development of benthic monitoring approaches for salmon aquaculture sites using machine learning, hydroacoustic data and bacterial eDNA</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Intensive caged salmon production can lead to localized perturbations of the seafloor environment where organic waste (flocculent matter) accumulates and disrupts ecological processes. As the aquaculture industry expands, the development of tools to rapidly detect changes in seafloor condition is critical. Here, we examine whether applying machine learning to two types of monitoring data could improve environmental assessments at aquaculture sites in Newfoundland. First, we apply machine learning to single beam echosounder data to detect flocculent matter at aquaculture sites over larger areas than currently achieved used drop camera imaging. Then, we use machine learning to categorize sediments by levels of disturbance based on bacterial tetranucleotide frequency distributions generated from environmental DNA. While echosounder data can detect flocculent matter with moderate success in this region, bacterial tetranucleotide frequencies are highly effective classifiers of benthic disturbance; this simplified environmental DNA-based approach could be implemented within novel aquaculture benthic monitoring pipelines.</p><h1>Fishing for mammals : landscape-level monitoring of terrestrial and semi-aquatic communities using eDNA from riverine systems</h1>
<p><a href="https://doi.org/10.1111/1365-2664.13592">doi:10.1111/1365-2664.13592</a></p>

<p><a href=None>None</a></p>
<p>1. Environmental DNA (eDNA) metabarcoding has revolutionised biomonitoring in both marine and freshwater ecosystems. However, for semi-aquatic and terrestrial animals, the application of this technique remains relatively untested. 2. We first assess the efficiency of eDNA metabarcoding in detecting semi-aquatic and terrestrial mammals in natural lotic ecosystems in the UK by comparing sequence data recovered from water and sediment samples to the mammalian communities expected from historical data. Secondly, using occupancy modelling we compared the detection efficiency of eDNA metabarcoding to multiple conventional non-invasive survey methods (latrine surveys and camera trapping). 3. eDNA metabarcoding detected a large proportion of the expected mammalian community within each area. Common species in the areas were detected at the majority of sites. Several key species of conservation concern in the UK were detected by eDNA sampling in areas where authenticated records do not currently exist, but potential false positives were also identified. 4. Water-based eDNA metabarcoding provided comparable results to conventional survey methods in per unit of survey effort for three species (water vole, field vole, and red deer) using occupancy models. The comparison between survey ‘effort’ to reach a detection probability of ≥0.95 revealed that 3-6 water replicates would be equivalent to 3-5 latrine surveys and 5-30 weeks of single camera deployment, depending on the species. 5. Synthesis and Applications. eDNA metabarcoding can be used to generate an initial ‘distribution map’ of mammalian diversity at the landscape level. If conducted during times of peak abundance, carefully chosen sampling points along multiple river courses provide a reliable snapshot of the species that are present in a catchment area. In order to fully capture solitary, rare and invasive species, we would currently recommend the use of eDNA metabarcoding alongside other non-invasive surveying methods (i.e. camera traps) to maximize monitoring efforts.</p><h1>An environmental DNA (eDNA) based method for monitoring spawning activity: a case study using the endangered Macquarie perch (Macquaria australasica)</h1>
<p><a href="https://doi.org/10.13140/rg.2.2.11602.76481">doi:10.13140/rg.2.2.11602.76481</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Roadmap for implementing environmental DNA (eDNA) and other molecular monitoring methods in Finland – Vision and action plan for 2022–2025</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Roadmap for implementing environmental DNA (eDNA) and other molecular monitoring methods in Finland – Vision and Action Plan for 2022-2025 Technological development in molecular methodology has been extremely fast in the past two decades, and groundbreaking new approaches have been introduced. It is now possible to detect and quantify DNA or RNA of target species or even map the whole species community in environmental samples of water, sediment, soil, air or assemblages of whole organisms. Moreover, the costs of high-throughput sequencing and other advanced molecular methods have decreased and methodological pipelines from sampling to data analysis developed sufficiently to allow large-scale, routine application of the new methods in environmental monitoring. This presents a huge opportunity to improve the coverage, accuracy and cost-efficiency of monitoring, enabling a much more complete picture of biodiversity and the state of the environment and their trends. As the new European Biodiversity Strategy for 2030 and other international policies to halt biodiversity loss and the degradation of habitats are translated into concrete measures, the quality of the monitoring data will play a crucial role in determining their success or failure. In this roadmap commissioned by the Finnish Ministry of the Environment, we assess the state-of-the-art in molecular monitoring methods in Finland within the international context, identify challenges and development areas that remain to be addressed and propose an action plan for promoting the coordinated implementation of molecular methods in national monitoring programs. Apart from the most recent scientific literature, our analysis is based on survey results, direct enquiries and interviews. Participation of the national community of experts from different sectors was enabled and invited at several stages of the roadmap preparation. Internationally, molecular monitoring methods are being actively developed and are routinely implemented in monitoring across different taxa and ecosystems. In Finland, molecular monitoring methods have been tested and piloted by all major institutions responsible for environmental monitoring, and the methods are already applied routinely in the monitoring of individual game species such as the wolf and European and Canadian beaver. However, other areas such as the monitoring of biodiversity, threatened species, non-mammalian invasive species or emerging plant or animal pests remain less developed, and national efforts and expertise are scattered across different organizations. Funding and know-how are perceived as the most important factors limiting molecular monitoring method implementation. We estimate that extensive, routine implementation of a wide range of molecular monitoring methods is conceivable in Finland before 2030. As the primary development areas for reaching this goal, we identify (i) international coordination and standard development, (ii) networking across sectors, (iii) education, (iv) infrastructure, (v) reference sequence libraries and the mapping of whole genomes, and (vi) modelling and analysis tool development. For concrete actions in 2022–2025, we propose (1) a cross-governmental funding instrument, (2) a permanent working group responsible for national and international coordination, (3) a national network and (4) an online platform to enhance interaction and knowledge transfer, as well as (5) a national data management system with collectively agreed data and metadata formats and standards. --- Kansallinen tiekartta ympäristö-DNA:n ja muiden molekyylibiologisten seurantamenetelmien käyttöönotolle – visio ja toimenpidesuunnitelma vuosille 2022-2025 Molekyylibiologisten menetelmien teknologinen kehitys on ollut ennennäkemättömän nopeaa kahden viime vuosikymmenen aikana. Uudet menetelmät mahdollistavat kohdelajien DNA:n tai RNA:n havaitsemisen ja runsausmäärityksen tai koko eliöyhteisön kartoittamisen esimerkiksi vesi-, sedimentti-, maaperä- tai ilmanäytteistä tai kokonaisia yksilöitä sisältävistä kokoomanäytteistä. Massiivisen rinnakkaissekvensoinnin ja muiden menetelmien kustannukset ovat merkittävästi laskeneet ja menetelmäketjut näytteenotosta tulosten tulkintaan kehittyneet asteelle, joka mahdollistaa niiden laajamittaisen, rutiininomaisen käytön ympäristön seurannassa. Uusien menetelmien avulla voimme parantaa seurannan kattavuutta, tarkkuutta ja kustannustehokkuutta ja siten täydentää seurannan kautta muodostuvaa kuvaa luonnon monimuotoisuudesta ja sen muutoksista. Tälle tiedolle on suuri tarve – laadukas seuranta on keskeinen edellytys sille, että EU:n uuden biodiversiteettistrategian ja muiden luontokadon ja elinympäristöjen tilan huonontumisen pysäyttämiseen tähtäävien kansainvälisten sitoumusten toimeenpano onnistuu. Tässä ympäristöministeriön tilaamassa tiekartassa arvioimme molekyylibiologisten seurantamenetelmien nykytilaa Suomessa osana laajempaa kansainvälistä kenttää, tunnistamme huomiota vaativia haasteita ja kehityskohteita ja ehdotamme konkreettisia toimenpiteitä molekyylibiologisten seurantamenetelmien koordinoidun käyttöönoton edistämiseksi lähivuosien aikana. Selvityksemme perustuu uusimman tieteellisen kirjallisuuden lisäksi kyselytutkimukseen sekä suoriin tiedusteluihin ja haastatteluihin. Yhteiskunnan eri sektoreita edustava kansallinen asiantuntijayhteisö osallistui tiekartan valmisteluun työn eri vaiheissa. Molekyylibiologisia seurantamenetelmiä kehitetään parhaillaan aktiivisesti ympäri maailmaa eri eliöryhmille ja ekosysteemeille, ja yksittäisiä menetelmiä on useissa maissa otettu myös rutiininomaiseen käyttöön. Suomessa menetelmiä on kehitetty ja pilotoitu kaikissa keskeisissä ympäristön seurantaa koordinoivissa laitoksissa, ja yksittäisten riistaeläinten kuten suden ja kanadan- ja euroopanmajavan seurannassa ne ovat jo rutiinikäytössä. Biodiversiteetin, uhanalaisten lajien, vieraslajien (nisäkkäitä lukuun ottamatta) ja muiden haitallisten lajien kansallisessa seurannassa molekyylibiologisten menetelmien käyttö on kuitenkin vielä kokeiluasteella, ja kehittämishankkeiden ja asiantuntijuuden kenttä on hajanainen. Riittämätöntä rahoitusta ja osaamista pidetään alan asiantuntijoiden keskuudessa tärkeimpinä menetelmien käyttöönottoa rajoittavina tekijöinä. Arviomme mukaan laaja kirjo molekyylibiologisia seurantamenetelmiä olisi mahdollista ottaa laajamittaiseen rutiininomaiseen käyttöön vuoteen 2030 mennessä. Tärkeimmiksi kehityskohteiksi nousivat (i) kansainvälinen koordinaatio ja menetelmien standardointi, (ii) organisaatioiden ja sektoreiden välinen verkostoituminen, (iii) koulutus, (iv) infrastruktuuri, (v) referenssisekvenssikirjastot ja kokonaisten genomien kartoittaminen sekä (vi) malli- ja analyysityökalujen kehittäminen. Konkreettisiksi toimenpiteiksi vuosille 2022-2025 esitämme (1) poikkihallinnollista rahoitusohjelmaa molekyylibiologisten seurantamenetelmien käyttöönottoa edistäville tutkimus- ja kehityshankkeille, (2) pysyvää työryhmää kansallisen ja kansainvälisen koordinaation edistämiseksi, (3) olemassa olevan kansallisen asiantuntijaverkoston laajentamista, (4) internet-pohjaista alustaa vuorovaikutuksen ja tiedonjaon tehostamiseksi sekä (5) kansallista, yhdessä sovittuja data- ja metadatastandardeja noudattavaa molekyylibiologisten seuranta-aineistojen tiedonhallintajärjestelmää.</p><h1>Determining an effective sampling method for eDNA metabarcoding: a case study for fish biodiversity monitoring in a small, natural river</h1>
<p><a href="https://doi.org/10.1007/s10201-020-00645-9">doi:10.1007/s10201-020-00645-9</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Combining traditional and environmental DNA (eDNA) based monitoring to improve the management of native and invasive fish species</h1>
<p><a href="https://doi.org/10.13140/rg.2.1.1569.2400">doi:10.13140/rg.2.1.1569.2400</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Development and validation of an eDNA protocol for monitoring endemic Asian spiny frogs in the Himalayan region of Pakistan</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Wildlife monitoring programs are instrumental for the assessment of species, habitat status, and for the management of factors affecting them. This is particularly important for species found in freshwater ecosystems, such as amphibians, as they have higher estimated extinction rates than terrestrial species. We developed and validated two species-specific environmental DNA (eDNA) protocols and applied them in the field to detect the Hazara Torrent Frog (Allopaa hazarensis) and Murree Hills Frog (Nanorana vicina). Additionally, we compared eDNA surveys with visual encounter surveys and estimated site occupancy. eDNA surveys resulted in higher occurrence probabilities for both A. hazarensis and N. vicina than for visual encounter surveys. Detection probability using eDNA was greater for both species, particularly for A. hazarensis. The top-ranked detection model for visual encounter surveys included effects of both year and temperature on both species, and the top-ranked occupancy model included effects of elevation and year. The top-ranked detection model for eDNA data was the null model, and the top-ranked occupancy model included effects of elevation, year, and wetland type. To our knowledge, this is the first time an eDNA survey has been used to monitor amphibian species in the Himalayan region.</p><h1>ong8181/eDNA-qmiseq: Source scripts for Ushio et al. "Quantitative monitoring of multispecies fish environmental DNA using high-throughput sequencing"</h1>
<p><a href="https://doi.org/10.5281/zenodo.1173828">doi:10.5281/zenodo.1173828</a></p>

<p><a href=None>None</a></p>
<p>Analysis codes for "Quantitative monitoring of multispecies fish environmental DNA using high-throughput sequencing"</p><h1>Overvåking og kartlegging av fremmed ferskvannsfisk ved bruk av miljø-DNA ; Monitoring of invasive freshwater fish using environmental DNA (eDNA)</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Prosjektleder Anette Engesmo ; Dette prosjektet er gjennomført på oppdrag fra Miljødirektoratet og målsetningen var å utvikle og teste ut et system for bruk av miljø-DNA som supplerende verktøy i overvåkning og kartlegging av fremmed ferskvannsfisk for forvaltningen. Vi har testet ulike eksisterende metodikk for feltarbeid, prøvetaking og molekylære protokoller og forenklet disse der dette har vært mulig. I denne rapporten sammenfatter vi erfaringene gjort i alle deler av prosessen, fra felt til ferdig data, og gir anbefalinger om prioriteringer og veien videre for perfeksjonering av dette potensielt viktige verktøyet for miljøforvaltningen. Det er ikke gjennomført omfattende feltprøvetakning i dette prosjektet. Eksisterende prøvetakningsprotokoll for bruk av Sterivex-filter er blitt forenklet ved at filtreringen gjøres i felt og filteret fikseres på ATL-buffer og oppbevares i romtemperatur. Denne behandlingen av filtrene ser ut til å medføre et visst tap av DNA, men er samtidig en betydelig forenkling for prøvetaker ved at man kan unngå å ta med tørris i felt. Videre har vi benyttet en ny qPCR polymerase, noe som ser ut til å minimere påvirkningen av inhibitorer i miljø-DNA. Eksisterende qPCR markører for gjedde, abbor, ørekyt, karuss, mort, karpe, regnbueørret og pukkellaks er blitt validert og testet på miljø-DNA. Det ble gjennomført en modifisering av markøren for karpe for å eliminere «støy» i qPCR reaksjonen. Vi kunne ikke påvise ørekyt fra noen lokaliteter, men vi mener dette var reelle negative prøver og ikke reflekterer svakheter ved markøren. Tidligere benyttet markør for bekkerøye viste seg ikke å fungere, og det foreligger derfor ikke resultater for denne arten. Resultatene for pukkellaks var særs gode, og vi hadde tydelig deteksjon av arten fra alle prøvetatte elver i Finnmark. Det har blitt utviklet nye qPCR markører for sørv, suter og solabbor. Ved videre utvikling av metodikk bør det legges vekt på prøvetaking på optimal tid på året (for eksempel gytetid), og det bør også undersøkes hvorvidt prøvetaking bør skje i habitater knyttet til arten som skal kartlegges. Videre vil innsjøens hydrologi påvirke hvor robuste resultater som kan ventes. Våre resultater har understreket viktigheten av at alle protokoller for nye arter må utvikles og optimaliseres individuelt, der det tas hensyn til målartens biologi og økologi. Artsspesifikke markører har i hovedsak blitt testet på miljø-DNA fra lokaliteter med kjent forekomst av målartene. Dette ble gjort for å validere metodikken for, og for å kunne bruke de innsamlede miljø-DNA prøvene som en slags «fasit» for hvilke arter vi bør detektere. Likevel har undersøkelsene påvist sørv fra fire hittil ukjente lokaliteter: Høgstlidammen, Mortensplasstjenn, Sollidammen og Tuftdammen. ; Miljødirektoratet ; publishedVersion</p><h1>High-throughput eDNA monitoring of fungi to track functional recovery in ecological restoration: OTU raw data matrix</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>We generated a total of 4,993,144 ITS fungal raw reads (118,884 ± 42,210 SD per replicate) across the 42 replicates. A total of 4,955,680 fungal sequences (117,430 ± 42,164 SD per replicate) remained for further analysis after quality filtering</p><h1>Data from: Meta‐analysis supports further refinement of eDNA for monitoring aquatic species‐specific abundance in nature</h1>
<p><a href="https://doi.org/10.5061/dryad.pm66g75">doi:10.5061/dryad.pm66g75</a></p>

<p><a href=None>None</a></p>
<p>The use of eDNA to detect the presence/absence of rare or invasive species is well documented and its use in biodiversity monitoring is expanding. Preliminary laboratory research has also shown a positive correlation between the concentration of species‐specific eDNA particles and the density/biomass of a species in a given environment. However, the extent to which these results can be extended to natural environments has yet to be formally quantified. We collated data from experiments that examined the correlation between eDNA and two metrics of abundance (biomass and density) and, using mixed‐effects meta‐analysis, quantified the strength of that correlation across laboratory and natural environments. We found that eDNA particle concentration was more strongly correlated with abundance in laboratory environments compared to natural environments, accounting for approximately 82% and 57% of the observed variation in abundance, respectively. We found some evidence of potential publication bias that may have impacted the estimation of the correlation in natural environments; after smaller studies were removed from the dataset, eDNA particle concentration accounted for approximately 50% of the observed variation in abundance in natural environments. No effect of abundance metric was found on the strength of correlation between eDNA particle concentration and abundance. Despite a weaker general correlation in natural environments, eDNA concentration often still explained substantial variation in abundance. eDNA research is still an emergent field of study; with only moderate improvements in technology or technique, it could represent a powerful new tool for quantifying abundance. ; Correlation Coefficients and Variance/Covariance MatrixCorrelation coefficients from published studies on eDNA particle concentration and abundance, including variance/covariance matrix used to account for non-independence of biomass/density correlations obtained from the same experimental subject groups.Yates_et_al_2019.xlsxFunding provided by: National Science FoundationAward Number: NA</p><h1>An eDNA protocol for monitoring and preserving biodiversity from genes to species: a case study of Alpine amphibians from Trentino</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>In the last decade, environmental DNA (eDNA) and metabarcoding have provided new opportunities for the study of biodiversity in space and time. The eDNA metabarcoding approach is increasingly used as a cost effective tool for species detection and community characterization, particularly for elusive species. However, technical challenges and case-specific limitations exist and need to be explored. In this study, we tested if eDNA metabarcoding of water samples can be used for developing an effective protocol for the characterization and monitoring of Alpine amphibian communities. By PCR amplification of a 151 bp COI barcode region using primers specific for amphibians, we aimed at assessing the species composition of several selected wetlands and ponds in an Alpine area, part of the MAB UNESCO ‘Alpi Ledrensi and Judicaria’ Biosphere Reserve (Trentino, south-eastern Alps). Choosing a widespread and abundant anuran, the common frog (Rana temporaria) as a model species, we tested for differences in detection success among three developmental stages and various freshwater environments. Lastly, by selecting a more specific COI fragment, we also aimed at assessing mtDNA genetic diversity in the focal species, which previous studies have found to be high in the study region and correlated with nuclear diversity. Preliminary findings will be presented here, together with the discussion of technical challenges and potential applications. Amphibians are threatened worldwide by climate change and habitat alterations: the development of a standardized, cost-effective protocol for monitoring amphibian biodiversity at species and genetic levels may represent an important resource for conservation managers.</p><h1>Refinement of<scp>eDNA</scp>as an early monitoring tool at the landscape‐level: study design considerations</h1>
<p><a href="https://doi.org/10.1002/eap.1951">doi:10.1002/eap.1951</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Development of species-specific eDNA-based test systems for monitoring of non-indigenous species in Danish marine waters</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Project manager Jesper H. Andersen. ; The MONIS 3 project reports the development of 22 species-specific eDNA-based test systems for monitoring and mapping of the occurrence of non-indigenous species in Danish marine waters. The development has been undertaken by the MONIS partnership and the species have been selected based on the outcome of the MONIS 2 project. It should be noted that the test systems are considered operational cf. the MONIS definition (see Chapter 1). It should also be noted that the 22 test systems have been applied in the MONIS 4 project, where monitoring of 16 Danish harbours have been carried out to map occurrence of nonindigenous marine species using both conventional methods and molecular methods (eDNA). Development of additional test systems is being planned and expected to be carried out in 2019 pending funding. The MONIS partnership encourages interested institutions, especially in neighbouring countries (Germany, Norway, Poland and Sweden) to make use of the test systems. ; publishedVersion</p><h1>Developing and testing the specificity and sensitivity of eDNA protocols for Shortfin Mako shark (Isurus oxyrinchus) monitoring</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Effective conservation relies on knowledge of species biology and ecology of a target species. For marine pelagics, this can be especially difficult to obtain due to the difficulties of working in a marine environment, and the nature of large pelagics species to be evasive, to rarely visit near shore habitats, and to persist as solitary animals or in small groups. In Australia, there is a high demand for biological and ecological information on the Shortfin Mako shark Isurus oxyrinchus, due to recreational fishing importance and its recent formal listing as an Endangered species. This new listing was due to information stating that the I. oxyrinchus global population has decreased by 37% over the last 15 years, with further declines being imminent in the absence of effective conservation and fisheries management. In this study I developed an Environmental DNA (eDNA) assay to assist the future collection of data necessary for guiding management. eDNA assays are an emerging tool for the detection of species presence through the detection of extracellular genetic material in environmental samples (e.g. soil, water, air) which are shed from animals via excretions (e.g. hair, blood, sweat, skin, faeces etc). The technology is revolutionising the field of wildlife monitoring and biosecurity, providing a cost-effective, non-invasive and sensitive alternative to traditional methods aimed at determining species presence/absence. The first aim of this study was to create the technology needed for eDNA surveys of I. oxyrinchus, via the development of a polymerase chain reaction (PCR) assay, and subsequent testing of its sensitivity and species specificity. The second goal was to undertake a laboratory experiment which would provide insights into the temporal decay rates of Mako eDNA (i.e. how long does eDNA persist and remain detectable in the environment). Understanding rates of eDNA decay is critical for determining if positive eDNA detections in the field indicate recent species presence in close proximity to the sample location. 3 Using bioinformatic software, a TaqMan probe (consisting of a primer pair and fluorescently labelled probe) was designed to amplify a 20 base pair region of the mitochondrial cytochrome oxidase 1 (CO1) gene. Subsequent testing of the probe demonstrated effective amplification of I. oxyrinchus DNA, and a high level of species specificity based on tests for cross-amplification against a range of Australian endemic and closely related shark species. Next, an eDNA decay trial was conducted by submerging an I. oxyrinchus specimen in seawater for 20 hours, separating the water into triplicate tubs and monitoring each tubs DNA concentration via PCR under controlled laboratory conditions for 10 days. The trial indicated rapid decay of I. oxyrinchus eDNA over the first 12-24 hours (approximately 85% reduction in detectable DNA material), plateauing at 60-84 hours when 99.8% of the material had deteriorated. These findings are highly consistent with other studies of a similar nature in different marine species. I discuss the implications of these lab-condition based findings and underline the need for future testing to determine the temporal and spatial sensitivity of the assay under field conditions. Overall, this study has provided a valuable new tool for monitoring I. oxyrinchus populations in Australian and international waters. While follow up research is needed to validate the spatio-temporal sensitivity of the assay under field conditions, it is envisaged that the adoption of the eDNA assay by researchers and managers will help to gain information on I. oxyrinchus habitat affiliations, movement patterns and general distribution, to inform future management geared toward conserving this important marine apex predator.</p><h1>Effects of sampling strategies and DNA extraction methods on eDNA metabarcoding: A case study of estuarine fish diversity monitoring</h1>
<p><a href="https://doi.org/10.24272/j.issn.2095-8137.2021.331">doi:10.24272/j.issn.2095-8137.2021.331</a></p>

<p><a href=None>None</a></p>
<p>Environmental DNA (eDNA) integrated with metabarcoding is a promising and powerful tool for species composition and biodiversity assessment in aquatic ecosystems and is increasingly applied to evaluate fish diversity. To date, however, no standardized eDNA-based protocol has been established to monitor fish diversity. In this study, we investigated and compared two filtration methods and three DNA extraction methods using three filtration water volumes to determine a suitable approach for eDNA-based fish diversity monitoring in the Pearl River Estuary (PRE), a highly anthropogenically disturbed estuarine ecosystem. Compared to filtration-based precipitation, direct filtration was a more suitable method for eDNA metabarcoding in the PRE. The combined use of DNeasy Blood and Tissue Kit (BT) and traditional phenol/chloroform (PC) extraction produced higher DNA yields, amplicon sequence variants (ASVs), and Shannon diversity indices, and generated more homogeneous and consistent community composition among replicates. Compared to the other combined protocols, the PC and BT methods obtained better species detection, higher fish diversity, and greater consistency for the filtration water volumes of 1 000 and 2 000 mL, respectively. All eDNA metabarcoding protocols were more sensitive than bottom trawling in the PRE fish surveys and combining two techniques yielded greater taxonomic diversity. Furthermore, combining traditional methods with eDNA analysis enhanced accuracy. These results indicate that methodological decisions related to eDNA metabarcoding should be made with caution for fish community monitoring in estuarine ecosystems.</p><h1>Supervised machine learning is superior to indicator value inference in monitoring the environmental impacts of salmon aquaculture using eDNA metabarcodes</h1>
<p><a href="https://doi.org/10.1111/mec.15434">doi:10.1111/mec.15434</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Fish community surveys in eelgrass beds using both eDNA metabarcoding and seining: implications for biodiversity monitoring in the coastal zone</h1>
<p><a href="https://doi.org/10.1139/cjfas-2021-0215">doi:10.1139/cjfas-2021-0215</a></p>

<p><a href=None>None</a></p>
<p>Marine Protected Areas (MPAs) have been adopted globally as a tool to combat biodiversity loss and restore marine ecosystems. Successful application of MPAs will be predicated on the ability to monitor biodiversity in a synoptic and non-invasive manner. Environmental DNA (eDNA) methods have important advantages over traditional biodiversity survey methods for monitoring conservation areas. To evaluate the efficacy of eDNA metabarcoding for fish biodiversity monitoring, we sampled 19 coastal eelgrass beds in Canada, as eelgrass beds are known for high biodiversity and significant conservation value. At each site, beach seines were used to survey fish and water samples were collected contemporaneously for eDNA metabarcoding. In total, beach seining caught 32,672 individuals across 59 fish taxa and eDNA detected 129 fish taxa. eDNA captured site-level variation and detected higher species richness at both site and regional levels compared to seining. eDNA abundance had a positive association with capture abundance. Collectively these results highlight how eDNA metabarcoding offers an efficient approach for monitoring fish biodiversity in coastal eelgrass beds, thus providing a valuable and non-invasive tool for MPA planning and coastal monitoring.</p><h1>Fish diversity monitoring in Maninjau Lake, West Sumatra using the eDNA with the next generation sequencing (NGS) technique</h1>
<p><a href="https://doi.org/10.1088/1755-1315/819/1/012045">doi:10.1088/1755-1315/819/1/012045</a></p>

<p><a href=None>None</a></p>
<p>Abstract Various phenomena in nature and human activities have resulted in the loss of biodiversity, such as freshwater fish in Maninjau lake, West Sumatra, which has declined from year to year. A new method to monitor biodiversity quickly and efficiently is the NGS technique on environment DNA (eDNA). The study aimed to know the NGS technique’s ability and effectiveness to detect multiple species at one time from the water samples of Maninjau Lake. Water was taken from the surface as much as one L with two replications. The sequencing was used to identify species as shotgun metagenomic with universal primers. The results showed that 92 individuals were identified with 56 species from 24 genera, 16 families, and 12 orders. As much as 25% of fishes still cannot be grouped into the valid taxa (unclassified), which is allegedly related to the lack of available information database (NCBI) compared to the sequences obtained. NGS on the eDNA method detected two families (Cyprinidae and Cichlidae), three genera ( Oreochromis, Cyprinus , and Xiphophorus ), and two species of fish ( Oreochromis niloticus and Cyprinus carpio ) in Maninjau Lake, which were also previously reported using the conventional method. The native species were not detected in Maninjau lake by the eDNA method allegedly due to not being captured DNA from the collected water samples or the low DNA concentration, which cannot be continued to the PCR process. Thus, several efforts are needed to reduce the limitations in monitoring using the eDNA method to obtain the maximal results. The eDNA method can be a useful tool for monitoring biodiversity. The result can determine the conservation strategies, especially for the fishes in Maninjau Lake, West Sumatra.</p><h1>Environmental DNA as a monitoring tool for the endangered freshwater pearl mussel (Margaritifera margaritifera L.): a substitute for classical monitoring approaches? ; EDNA in mussel surveys</h1>
<p><a href="https://doi.org/10.1002/aqc.2611">doi:10.1002/aqc.2611</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Development of species-specific eDNA-based test systems for monitoring of non-indigenous Decapoda in Danish marine waters ; Udvikling af artsspecifikke eDNA-baserede testsystemer til overvågning af ikke-hjemmehørende krabber i de danske farvande</h1>
<p>No DOI</p>
<p><a href=None>None</a></p>
<p>Project manager Jesper H. Andersen ; We report the development of seven eDNA-based species-specific test systems for monitoring of marine Decapoda in Danish marine waters. The seven species are 1) Callinectes sapidus (blå svømmekrabbe), 2) Eriocheir sinensis (kinesisk uldhåndskrabbe), 3) Hemigrapsus sanguineus (stribet klippekrabbe), 4) Hemigrapsus takanoi (pensel-klippekrabbe), 5) Homarus americanus (amerikansk hummer), 6) Paralithodes camtschaticus (Kamchatka-krabbe) and 7) Rhithropanopeus harrisii (østamerikansk brakvandskrabbe). The following three are new developments: Callinectes sapidus, Hemigrapsus sanguineus and Hemigrapsus takanoi, whilst the remaining four previously developed under earlier phases of the MONIS projects have been tested again. The additional tests on the four previously developed assays were performed with a broader diversity of Decapoda known from Danish seas, to re-evaluate the specificity of these assays. This additional testing revealed that two previously published assays were unspecific, and to accommodate this, two new assays are presented replacing those previously published. We recommend that the species-specific eDNA assays presented here will allow for continuous monitoring of these species as part of the NOVANA monitoring programme. ; publishedVersion</p><h1>Data_Sheet_2_Environmental DNA (eDNA) Monitoring of Noble Crayfish Astacus astacus in Lentic Environments Offers Reliable Presence-Absence Surveillance – But Fails to Predict Population Density.xlsx</h1>
<p><a href="https://doi.org/10.3389/fenvs.2020.612253.s002">doi:10.3389/fenvs.2020.612253.s002</a></p>

<p><a href=None>None</a></p>
<p>Noble crayfish is the most widespread native freshwater crayfish species in Europe. It is threatened in its entire distribution range and listed on the International Union for Concervation Nature- and national red lists. Reliable monitoring data is a prerequisite for implementing conservation measures, and population trends are traditionally obtained from catch per unit effort (CPUE) data. Recently developed environmental DNA (eDNA) tools can potentially improve the effort. In the past decade, eDNA monitoring has emerged as a promising tool for species surveillance, and some studies have established that eDNA methods yield adequate presence-absence data for crayfish. There are also high expectations that eDNA concentrations in the water can predict biomass or relative density. However, eDNA studies for crayfish have not yet been able to establish a convincing relationship between eDNA concentrations and crayfish density. This study compared eDNA and CPUE data obtained the same day and with high sampling effort, and evaluated whether eDNA concentrations can predict relative density of crayfish. We also compared two analytical methods [Quantitative real-time PCR (qPCR) and digital droplet PCR (ddPCR)], and estimated the detection probability for eDNA monitoring compared to trapping using occupancy modeling. In all lakes investigated, we detected eDNA from noble crayfish, even in lakes with very low densities. The eDNA method is reliable for presence-absence monitoring of noble crayfish, and the probability of detecting noble crayfish from eDNA samples increased with increasing relative crayfish densities. However, the crayfish eDNA concentrations were consistently low and mostly below the limit of quantification, even in lakes with very high crayfish densities. The hypothesis that eDNA concentrations can predict relative crayfish density was consequently not supported. Our study underlines the importance of intensified sampling effort for successful detection of very low-density populations, and for substantiating presumed absence, inferred from negative results. Surprisingly, we found a higher likelihood of eDNA detection using qPCR compared to ddPCR. We conclude that eDNA monitoring cannot substitute CPUE data, but is a reliable supplement for rapid presence-absence overviews. Combined with eDNA analyses of alien crayfish species and diseases such as crayfish plague, this is a cost-efficient supplement offering a more holistic monitoring approach for aquatic environments and native crayfish conservation.</p><h1>Presentation: Monitoring Diversity Of Fish In Maninjau Lake, West Sumatra Using The EDNA Method With The Next Generation Sequencing (NGS) Technique</h1>
<p><a href="https://doi.org/10.5281/zenodo.4276021">doi:10.5281/zenodo.4276021</a></p>

<p><a href=None>None</a></p>
<p>This material presented in 2nd ICESE 2020</p><h1>Development and testing of an environmental DNA (eDNA) assay for endangered Atlantic sturgeon to assess its potential as a monitoring and management tool</h1>
<p><a href="https://doi.org/10.1002/edn3.186">doi:10.1002/edn3.186</a></p>

<p><a href=None>None</a></p>
<p></p><h1>eDNA as a tool for non-invasive monitoring of the fauna of a turbid, well-mixed system, the Elbe estuary in Germany</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0250452">doi:10.1371/journal.pone.0250452</a></p>

<p><a href=None>None</a></p>
<p>The Elbe is one of the longest European rivers and features a large, turbid and well-mixed estuary, which runs through the inner city of Hamburg. The Elbe has been closely monitored using classical catch techniques in the past. Here we tested a COI-based eDNA approach for assessing the biodiversity within the Elbe. We sampled three stations in the Elbe, included low and high tide events, as well as two adjoining lakes to compare the recovered faunas. To analyze the data, we employed two different pipelines: the automated mBRAVE pipeline utilizing the BOLD database and one including NCBI BLAST. The number of OTUs with species or higher-level identifications were similar between both approaches with 352 OTUs and 355 OTUs for BLAST and mBRAVE, respectively, however, BLAST searches recovered another 942 unidentified metazoan OTUs. Many taxa were well represented; however, fish species were poorly represented, especially in the Elbe estuary samples. This could be a result of the universal COI primers, which also yielded high read numbers for non-metazoan OTUs, and small-bodies taxa like Rotifera, which might have been sampled together with the eDNA. Our results show a strong tidal influence on the recovered taxa. During low tide, downstream stations resembled sites further upstream, but the former showed a very different OTU composition during high tide and early tide. Such differences might be due to varying impacts of upstream-originating eDNA during tide cycles. Such factors need to be considered when routinely employing eDNA for monitoring programs.</p><h1>Environmental DNA (eDNA) applications for the conservation of imperiled crayfish (Decapoda: Astacidea) through monitoring of invasive species barriers and relocated populations</h1>
<p><a href="https://doi.org/10.1093/jcbiol/ruy007">doi:10.1093/jcbiol/ruy007</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Environmental DNA (eDNA) Monitoring of Noble Crayfish Astacus astacus in Lentic Environments Offers Reliable Presence-Absence Surveillance – But Fails to Predict Population Density</h1>
<p><a href="https://doi.org/10.3389/fenvs.2020.612253">doi:10.3389/fenvs.2020.612253</a></p>

<p><a href=None>None</a></p>
<p>Noble crayfish is the most widespread native freshwater crayfish species in Europe. It is threatened in its entire distribution range and listed on the International Union for Concervation Nature- and national red lists. Reliable monitoring data is a prerequisite for implementing conservation measures, and population trends are traditionally obtained from catch per unit effort (CPUE) data. Recently developed environmental DNA (eDNA) tools can potentially improve the effort. In the past decade, eDNA monitoring has emerged as a promising tool for species surveillance, and some studies have established that eDNA methods yield adequate presence-absence data for crayfish. There are also high expectations that eDNA concentrations in the water can predict biomass or relative density. However, eDNA studies for crayfish have not yet been able to establish a convincing relationship between eDNA concentrations and crayfish density. This study compared eDNA and CPUE data obtained the same day and with high sampling effort, and evaluated whether eDNA concentrations can predict relative density of crayfish. We also compared two analytical methods [Quantitative real-time PCR (qPCR) and digital droplet PCR (ddPCR)], and estimated the detection probability for eDNA monitoring compared to trapping using occupancy modeling. In all lakes investigated, we detected eDNA from noble crayfish, even in lakes with very low densities. The eDNA method is reliable for presence-absence monitoring of noble crayfish, and the probability of detecting noble crayfish from eDNA samples increased with increasing relative crayfish densities. However, the crayfish eDNA concentrations were consistently low and mostly below the limit of quantification, even in lakes with very high crayfish densities. The hypothesis that eDNA concentrations can predict relative crayfish density was consequently not supported. Our study underlines the importance of intensified sampling effort for successful detection of very low-density populations, and for substantiating presumed absence, inferred from negative results. Surprisingly, we found a higher likelihood of eDNA detection using qPCR compared to ddPCR. We conclude that eDNA monitoring cannot substitute CPUE data, but is a reliable supplement for rapid presence-absence overviews. Combined with eDNA analyses of alien crayfish species and diseases such as crayfish plague, this is a cost-efficient supplement offering a more holistic monitoring approach for aquatic environments and native crayfish conservation. ; publishedVersion</p><h1>The development of early monitoring tools to detect aquatic invasive species: eDNA assay development and the case of the armored catfish Hypostomus robinii</h1>
<p><a href="https://doi.org/10.1002/edn3.260">doi:10.1002/edn3.260</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Table_1_Environmental DNA (eDNA) Monitoring of Noble Crayfish Astacus astacus in Lentic Environments Offers Reliable Presence-Absence Surveillance – But Fails to Predict Population Density.DOCX</h1>
<p><a href="https://doi.org/10.3389/fenvs.2020.612253.s004">doi:10.3389/fenvs.2020.612253.s004</a></p>

<p><a href=None>None</a></p>
<p>Noble crayfish is the most widespread native freshwater crayfish species in Europe. It is threatened in its entire distribution range and listed on the International Union for Concervation Nature- and national red lists. Reliable monitoring data is a prerequisite for implementing conservation measures, and population trends are traditionally obtained from catch per unit effort (CPUE) data. Recently developed environmental DNA (eDNA) tools can potentially improve the effort. In the past decade, eDNA monitoring has emerged as a promising tool for species surveillance, and some studies have established that eDNA methods yield adequate presence-absence data for crayfish. There are also high expectations that eDNA concentrations in the water can predict biomass or relative density. However, eDNA studies for crayfish have not yet been able to establish a convincing relationship between eDNA concentrations and crayfish density. This study compared eDNA and CPUE data obtained the same day and with high sampling effort, and evaluated whether eDNA concentrations can predict relative density of crayfish. We also compared two analytical methods [Quantitative real-time PCR (qPCR) and digital droplet PCR (ddPCR)], and estimated the detection probability for eDNA monitoring compared to trapping using occupancy modeling. In all lakes investigated, we detected eDNA from noble crayfish, even in lakes with very low densities. The eDNA method is reliable for presence-absence monitoring of noble crayfish, and the probability of detecting noble crayfish from eDNA samples increased with increasing relative crayfish densities. However, the crayfish eDNA concentrations were consistently low and mostly below the limit of quantification, even in lakes with very high crayfish densities. The hypothesis that eDNA concentrations can predict relative crayfish density was consequently not supported. Our study underlines the importance of intensified sampling effort for successful detection of very low-density populations, and for substantiating presumed absence, inferred from negative results. Surprisingly, we found a higher likelihood of eDNA detection using qPCR compared to ddPCR. We conclude that eDNA monitoring cannot substitute CPUE data, but is a reliable supplement for rapid presence-absence overviews. Combined with eDNA analyses of alien crayfish species and diseases such as crayfish plague, this is a cost-efficient supplement offering a more holistic monitoring approach for aquatic environments and native crayfish conservation.</p><h1>Data from: Development and testing of an environmental DNA (eDNA) assay for endangered Atlantic sturgeon to assess its potential as a monitoring and management tool</h1>
<p><a href="https://doi.org/10.5061/dryad.q83bk3jgs">doi:10.5061/dryad.q83bk3jgs</a></p>

<p><a href=None>None</a></p>
<p>Significant declines in Atlantic sturgeon (Acipenser oxyrhynchus oxyrhynchus) abundances along the US east coast have spurred major research efforts and management actions over the last 50 years, yet information on spawning stock abundances and habitat use is still lacking for many river systems, including in the Chesapeake Bay, USA. Here, we developed and tested a new quantitative PCR (qPCR) assay to detect Atlantic sturgeon environmental DNA (eDNA) in water samples with the goal of providing an alternative method to monitor presence and relative abundance. We also examined Atlantic sturgeon eDNA shedding rates in laboratory experiments. A qPCR-probe assay targeting Cytochrome-B was developed and showed no amplification of other related and co-occurring fishes. Pond trials at a density of ~0.2 g/L sturgeon produced relatively strong eDNA detections (~1,000-25,000 copies/L) in all seven water samples assayed. Water samples taken from two river systems in the Chesapeake Bay produced zero eDNA detections in the summer, while fall sampling during sturgeon spawning produced positive eDNA detections in 26% of samples, though at much lower concentrations (400-1,800 copies/L) compared with the pond (mesocosm) detections. Acoustic detections of sturgeon near river sampling sites were positively associated with eDNA detections. However, the eDNA assay failed to detect the presence of sturgeon in some samples when abundances were very low or when fish were in deep water. Finally, Atlantic sturgeon eDNA shedding rates were estimated to be on the order of estimates for other fish species, which suggests that relatively weak detections in the field are not necessarily driven by low rates of eDNA shedding. Overall, eDNA analysis represents a promising new monitoring tool for Atlantic sturgeon. Applying these methods in other rivers along the US east coast is an important next step in documenting Atlantic sturgeon distribution for management and conservation purposes. ; The ATSDecay_Feb2021_R_Final.csv dataset contains eDNA abundance data over time from two replicate mesocosms (tanks ATT1 and ATT2, with 2 replicate water samples, A and B, at each time point) from the shedding rate experiment. The ATS_Env_Detections_Final.fasta file contains the trimmed Sanger sequences produced via the re-amplification and sequencing of all positive eDNA detections of Atlantic sturgeon in ponds and environmental (river) samples. Note that previous versions of these files (datasets) contained errors so only the latest versions (February 2021) are accurate. Funding provided by: Maryland Department of Natural ResourcesCrossref Funder Registry ID: http://dx.doi.org/10.13039/100015849Award Number: KOOB7400109Funding provided by: Virginia Department of Wildlife Resources*Crossref Funder Registry ID: Award Number: Funding provided by: Deerbrook Charitable Trust*Crossref Funder Registry ID: Award Number: DCT‐19‐34AFunding provided by: NOAA Species Recovery Grants to States (Section 6 Program)*Crossref Funder Registry ID: Award Number: A19NMF4720101 ; Water samples were collected from two rivers in the Chesapeake Bay (Nanticoke River, Pamunkey River), pond mesocosms, and indoor mesocosms (both with Atlantic sturgeon), and processed to detect Atlantic sturgeon eDNA using a quantitative PCR (qPCR) assay.</p><h1>Monitoring the ecological status of rivers with diatom eDNA metabarcoding: A comparison of taxonomic markers and analytical approaches for the inference of a molecular diatom index</h1>
<p><a href="https://doi.org/10.1111/mec.15646">doi:10.1111/mec.15646</a></p>

<p><a href=None>None</a></p>
<p>Recently, several studies demonstrated the usefulness of diatom eDNA metabarcoding as an alternative to assess the ecological quality of rivers and streams. However, the choice of the taxonomic marker as well as the methodology for data analysis differ between these studies, hampering the comparison of their results and effectiveness. The aim of this study was to compare two taxonomic markers commonly used in diatom metabarcoding and three distinct analytical approaches to infer a molecular diatom index. We used the values of classical morphological diatom index as a benchmark for this comparison. We amplified and sequenced both a fragment of the rbcL gene and the V4 region of the 18S rRNA gene for 112 epilithic samples from Swiss and French rivers. We inferred index values using three analytical approaches: by computing it directly from taxonomically assigned sequences, by calibrating de novo the ecovalues of all metabarcodes, and by using a supervised machine learning algorithm to train predictive models. In general, the values of index obtained using the two “taxonomy‐free” approaches, encompassing molecular assignment and machine learning, were closer correlated to the values of the morphological index than the values based on taxonomically assigned sequences. The correlations of the three analytical approaches were higher in the case of rbcL compared to the 18S marker, highlighting the importance of the reference database which is more complete for the rbcL marker. Our study confirms the effectiveness of diatom metabarcoding as an operational tool for rivers ecological quality assessment and shows that the analytical approaches by‐passing the taxonomic assignments are particularly efficient when reference databases are incomplete.</p><h1>Detecting spawning of threatened chum salmon Oncorhynchus keta over a large spatial extent using eDNA sampling: Opportunities and considerations for monitoring recovery</h1>
<p><a href="https://doi.org/10.1002/edn3.159">doi:10.1002/edn3.159</a></p>

<p><a href=None>None</a></p>
<p></p><h1>Supplementary material 1 from: Crookes S, Heer T, Castañeda RA, Mandrak NE, Heath DD, Weyl OLF, MacIsaac HJ, Foxcroft LC (2020) Monitoring the silver carp invasion in Africa: a case study using environmental DNA (eDNA) in dangerous watersheds. NeoBiota 56: 31-47. https://doi.org/10.3897/neobiota.56.47475</h1>
<p><a href="https://doi.org/10.3897/neobiota.56.47475.suppl1">doi:10.3897/neobiota.56.47475.suppl1</a></p>

<p><a href=None>None</a></p>
<p>Table 1</p><h1>Supplementary material 2 from: Bylemans J, Gleeson DM, Lintermans M, Hardy CM, Beitzel M, Gilligan DM, Furlan EM (2018) Monitoring riverine fish communities through eDNA metabarcoding: determining optimal sampling strategies along an altitudinal and biodiversity gradient. Metabarcoding and Metagenomics 2: e30457. https://doi.org/10.3897/mbmg.2.30457</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.30457.suppl2">doi:10.3897/mbmg.2.30457.suppl2</a></p>

<p><a href=None>None</a></p>
<p>: Data type: statistical data</p><h1>Supplementary material 1 from: Bylemans J, Gleeson DM, Lintermans M, Hardy CM, Beitzel M, Gilligan DM, Furlan EM (2018) Monitoring riverine fish communities through eDNA metabarcoding: determining optimal sampling strategies along an altitudinal and biodiversity gradient. Metabarcoding and Metagenomics 2: e30457. https://doi.org/10.3897/mbmg.2.30457</h1>
<p><a href="https://doi.org/10.3897/mbmg.2.30457.suppl1">doi:10.3897/mbmg.2.30457.suppl1</a></p>

<p><a href=None>None</a></p>
<p>: Data type: Microsoft Word Document (.docx)</p>