<h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/v4716jt6s49cgn5c>http://libris.kb.se/bib/v4716jt6s49cgn5c</a></p>
E-article<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p>
<h1>Application of high-throughput sequencing (HTS) metabarcoding to diatom biomonitoring: do DNA extraction methods matter? [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/20847965>http://libris.kb.se/bib/20847965</a></p>
E-book<p>Vasselon, Valentin and Domaizon, Isabelle and Rimet, Frederic and Kahlert, Maria and Bouchez, Agnes (2017). Application of high-throughput sequencing (HTS) metabarcoding to diatom biomonitoring: do DNA extraction methods matter? Freshwater Science. 36:1, 162-177 [Journal article]</p>
<h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/6k52v43q4t6mfq5q>http://libris.kb.se/bib/6k52v43q4t6mfq5q</a></p>
E-article<p>Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates &lt; 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%–6.9%), when compared to low OC sediments (1.2%–2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode  Sabatieria  and foraminifera  Elphidium  in high OC, and  Cryptocaryon ‐like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p>
<h1>Lake Sedimentary DNA Research on Past Terrestrial and Aquatic Biodiversity [Elektronisk resurs] Overview and Recommendations</h1>
<p><a href=http://libris.kb.se/bib/1f02zsxxzg9jzjlf>http://libris.kb.se/bib/1f02zsxxzg9jzjlf</a></p>
E-article<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p>
<h1>Fungal communities of West African ectomycorrhizal woodlands [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/4hl9mv0l2kqm111r>http://libris.kb.se/bib/4hl9mv0l2kqm111r</a></p>
E-book<p>Ectomycorrhizal symbiosis is a widespread mutualist relationship between fungi and plants, often trees. It is most well-known in temperate and boreal forests, but also occurs in a variety of tropical ecosystems, including Sudanian savanna woodlands and gallery forests in West Africa. In addition to their important role in nutrient cycling, many ectomycorrhizal fungi also produce edible mushrooms as their fruit-bodies. In this thesis, I explore spatial patterns of above-ground and below-ground diversity in West African ectomycorrhizal communities, as well as the use of wild edible fungi by the local human population. In Papers I and II , I used soil DNA metabarcoding to investigate fungal communities in their vegetative state. Paper I used measurements of the scale of spatial autocorrelation within the fungal community in Sudanian woodlands in Benin as a test to compare different high-throughput DNA sequencing strategies, including short (≈350 bp) amplicon barcoding using the Illumina, Ion Torrent, and PacBio sequencing platforms, as well as long (≈1550 bp) amplicon barcoding using PacBio. There were some differences in species richness and community composition recovered by the two amplicon lengths, attributable to known primer biases for the short amplicons, but these did not lead to different ecological results. Additionally, Paper I introduced new software packages for analysis of long-amplicon metabarcoding data and integrating phylogenetic information into sequence-based taxonomic identification. In Paper II we sampled both Sudanian woodland and gallery forest sites in five countries across West Africa using long-amplicon metabarcoding with PacBio. We found significant differences in fungal community composition between Sudanian woodland and gallery forest sites, but not between different tree species within each vegetation type. Papers III and IV focused above-ground to study the natural production and human use of the mushrooms themselves. In Paper III we exhaustively collected mushrooms from nine Sudanian woodland plots in Benin during three consecutive rainy seasons. We measured the total biomass produced of each morphospecies, as well as environmental variables related to microclimate, host tree availability, and soil chemistry. Mushroom production and diversity were negatively correlated with soil nitrogen levels, and positively correlated with soil phosphorus levels. Although there were no clear differences in the fungal communities associated with the four host trees present in our plots, greater host tree diversity was associated with greater fungal diversity and productivity. Finally, Paper IV combined interviews with local people from four ethnic groups in five villages near the study sites from Paper III about their knowledge and preferences for edible mushrooms with DNA barcoding of specimens. Knowledge and preferences for different mushrooms varied between ethnic groups, but people living in a village where their ethnic group is a minority tended to absorb knowledge about mushrooms from the majority group in the village. Women over age 35 were the most knowledgeable about mushrooms, but the degree of gender difference varied between groups. We compiled a list of the most choice edible mushrooms in the area, which can inform the possibility for commercial trade in wild mushrooms to supplement the income of rural people and encourage sustainable forest management.</p><p>Diss. (sammanfattning) Uppsala : Uppsala universitet, 2020</p><p></p>
<h1>Microbial DNA Sequencing in Environmental Studies [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/pxjkrs51m8nq3t87>http://libris.kb.se/bib/pxjkrs51m8nq3t87</a></p>
E-book<p>The field of microbial ecology has just entered a new era of rapid technological development and generation of big data. The high-throughput sequencing techniques presently available provide an opportunity to extensively inventorize the blueprints of life. Now, millions of microbes of natural microbial communities can be studied simultaneously without prior cultivation. New species and new functions (genes) can be discovered just by mining sequencing data. However, there is still a tremendous number of microorganisms not yet examined, nor are the ecosystem functions these carry out. The modern genomic technologies can contribute to solve environmental problems and help us understand ecosystems, but to most efficiently do so, methods need to be continuously optimised.   During my Ph. D. studies, I developed a method to survey eukaryotic microbial diversity with a higher accuracy, and applied various sequencing-based approaches in an attempt to answer questions of importance in environmental research and ecology. In PAPER-I , we developed a set of 18S rRNA gene PCR primers with high taxonomic coverage, meeting the requirements of currently popular sequencing technologies and matching the richness of 18S rRNA reference sequences accumulated so far. In PAPER-II , we conducted the first sequencing-based spatial survey on the combined eukaryotic and bacterial planktonic community in the Baltic Sea to uncover the relationship of microbial diversity and environmental conditions. Here, the 18S primers designed in PAPER-I and a pair of broad-coverage 16S primers were employed to target the rRNA genes of protists and bacterioplankton for amplicon sequencing. In PAPER-III , we integrated metagenomic, metabarcoding, and metatranscriptomic data in an effort to scrutinise the protein synthesis potential (i.e., activity) of microbes in the sediment at a depth of 460 m in the Baltic Sea and, thus, disclosing microbial diversity and their possible ecological functions within such an extreme environment. Lastly, in PAPER-IV , we compared the performance of E. coli culturing, high-throughput sequencing, and portable real-time sequencing in tracking wastewater contamination in an urban stormwater system. From the aspects of cost, mobility and accuracy, we evaluated the usage of sequencing-based approaches in civil engineering, and for the first time, validated the real-time sequencing device in use within water quality monitoring.   In summary, these studies demonstrate how DNA sequencing of microbial communities can be applied in environmental monitoring and ecological research.</p><p>Diss. (sammanfattning) Stockholm : Kungliga Tekniska högskolan, 2017</p><p></p>
<h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/frvn2nftcjhdbl6d>http://libris.kb.se/bib/frvn2nftcjhdbl6d</a></p>
E-article<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p>
<h1>What do we learn from cultures in the omics age? High-throughput sequencing and cultivation of leaf-inhabiting endophytes from beech (Fagus sylvatica L.) revealed complementary community composition but similar correlations with local habitat conditions [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/9qb56fsz7hz4zhlf>http://libris.kb.se/bib/9qb56fsz7hz4zhlf</a></p>
E-article<p>Comparative simultaneous studies of environmental high-throughput sequencing (HTS) and cultivation of plant-associated fungi have rarely been conducted in the past years. For the present contribution, HTS and extinction culturing were applied for the same leaf samples of European beech (Fagus sylvatica) in order to trace both “real” environmental drivers as well as method-dependent signals of the observed mycobiomes. Both approaches resulted in non-overlapping community composition and pronounced differences in taxonomic classification and trophic stages. However, both methods revealed similar correlations of the fungal communities with local environmental conditions. Our results indicate undeniable advantages of HTS over cultivation in terms of revealing a good representation of the major functional guilds, rare taxa and biodiversity signals of leaf-inhabiting fungi. On the other hand our results demonstrate that the immense body of literature about cultivable endophytic fungi can and should be used for the interpretation of community signals and environmental correlations obtained from HTS studies and that cultivation studies should be continued at the highest standards, e.g. when sequencing facilities are not available or if such surveys are expanded into functional aspects with experiments on living isolates.</p>
<h1>Environmental DNA : for biodiversity research and monitoring</h1>
<p><a href=http://libris.kb.se/bib/22521916>http://libris.kb.se/bib/22521916</a></p>
E-book<p>Environmental DNA (eDNA), i.e. DNA released in the environment by any living form, represents a formidable opportunity to gather high-throughput and standard information on the distribution or feeding habits of species. It has therefore great potential for applications in ecology and biodiversity management. However, this research field is fast-moving, involves different areas of expertise and currently lacks standard approaches, which calls for an up-to-date and comprehensive synthesis. Environmental DNA for biodiversity research and monitoring covers current methods based on eDNA, with a particular focus on “eDNA metabarcoding”. Intended for scientists and managers, it provides the background information to allow the design of sound experiments. It revisits all steps necessary to produce high-quality metabarcoding data such as sampling, metabarcode design, optimization of PCR and sequencing protocols, as well as analysis of large sequencing datasets. All these different steps are presented by discussing the potential and current challenges of eDNA-based approaches to infer parameters on biodiversity or ecological processes. The last chapters of this book review how DNA metabarcoding has been used so far to unravel novel patterns of diversity in space and time, to detect particular species, and to answer new ecological questions in various ecosystems and for various organisms. Environmental DNA for biodiversity research and monitoring constitutes an essential reading for all graduate students, researchers and practitioners who do not have a strong background in molecular genetics and who are willing to use eDNA approaches in ecology and biomonitoring.</p><p>Specialized.</p><p>Includes bibliographical references and index.</p><p>1 Introduction to environmental DNA (eDNA) -- 2 DNA metabarcode choice and design -- 3 Reference databases -- 4 Sampling -- 5 DNA extraction -- 6 DNA amplification and multiplexing -- 7 DNA sequencing -- 8 DNA metabarcoding data analysis -- 9 Single-species detection -- 10 Environmental DNA for functional diversity -- 11 Some early landmark studies -- 12 Freshwater ecosystems -- 13 Marine environments -- 14 Terrestrial ecosystems -- 15 Paleoenvironments -- 16 Host-associated microbiota -- 17 Diet analysis -- 18 Analysis of bulk samples -- 19 The future of eDNA metabarcoding</p><p>Description based on online resource; title from home page (viewed on February 19, 2018).</p><p></p>
<h1>The Ecological Role of Roadside Stormwater Ponds - Potential to Support Biodiversity [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/drm91w3qbgpjqgnx>http://libris.kb.se/bib/drm91w3qbgpjqgnx</a></p>
E-book<p>The increased recognition that roads may impair the aquatic environment and ecosystems has led to a shift from conventional drainage systems toward blue-green solutions such as stormwater ponds. Research on blue-green stormwater solutions has until now mainly focused on water quantity and quality. The aim of this study was to explore the ability of highway stormwater ponds to provide suitable habitats and support, especially macroinvertebrate biodiversity, and to identify the key environmental variables that affect biological community composition and the number of taxa, utilizing data at regional and global scales. Ultimately, this research informs design recommendations for stormwater systems that simultaneously provide multiple ecosystem services. The results of this thesis indicate that larger ponds are better for supporting aquatic biodiversity due to a more heterogeneous environment and the ability to dilute pollutants. Also, the presence of other ponds in the vicinity of the stormwater ponds can facilitate the movement of invertebrates between ponds through increased connectivity. An apparent negative effect of pollution levels on the macroinvertebrate community composition was observed, but not on the biodiversity measured as the number of taxa or Shannon index. The analyses based on the datasets identified using both morphology and DNA metabarcoding demonstrated that DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing more accurate information regarding the way the ponds function and how they are affected by management. Subsequently, the differences in the macroinvertebrate community composition between different types of ponds were compared at the regional and global scale. The results indicated that environmental characteristics, especially conductivity and pH, were different between different types of ponds. Alpha and gamma diversity were similar or even higher in manmade ponds compared to natural ponds due to very different macroinvertebrate communities. Moreover, generally ponds exhibited high levels of spatial heterogeneity, which subsequently enhances gamma diversity. In summary, stormwater ponds have the potential to provide suitable habitats to foster biodiversity. When such systems are created, larger ponds should be built to provide more heterogeneous habitat and dilute harmful pollutants. Additional ponds should also be created in the vicinity of the ponds, thereby promoting aquatic biodiversity through higher connectivity. Although stormwater ponds accumulate pollutants due to their primary functions, this pollution retention process creates a unique environment within the stormwater ponds, which are more suitable for taxa that are moderately to strongly tolerant to pollutants and that may not be found in natural ponds. In this way, stormwater ponds constitute an option in the areas along the highway so that they could combine water treatment properties with providing a suitable habitat for aquatic organisms.</p>
<h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy) [Elektronisk resurs]</h1>
<p><a href=http://libris.kb.se/bib/2g044ft508hpppc2>http://libris.kb.se/bib/2g044ft508hpppc2</a></p>
E-article<p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p>
