<h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/271353>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/271353</a></p>
article<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p>
<h1>Biodiversity between sand grains: Meiofauna composition across southern and western Sweden assessed by metabarcoding</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:nrm-3885>http://swepub.kb.se/bib/swepub:oai:DiVA.org:nrm-3885</a></p>
article<p>The meiofauna is an important part of the marine ecosystem, but its composition and distribution patterns are relatively unexplored. Here we assessed the biodiversity and community structure of meiofauna from five locations on the Swedish western and southern coasts using a high-throughput DNA sequencing (metabarcoding) approach. The mitochondrial cytochrome oxidase 1 (COI) mini-barcode and nuclear 18S small ribosomal subunit (18S) V1-V2 region were amplified and sequenced using Illumina MiSeq technology. Our analyses revealed a higher number of species than previously found in other areas: thirteen samples comprising 6.5 dm3 sediment revealed 708 COI and 1,639 18S metazoan OTUs. Across all sites, the majority of the metazoan biodiversity was assigned to Arthropoda, Nematoda and Platyhelminthes. Alpha and beta diversity measurements showed that community composition differed significantly amongst sites. OTUs initially assigned to Acoela, Gastrotricha and the two Platyhelminthes sub-groups Macrostomorpha and Rhabdocoela were further investigated and assigned to species using a phylogeny-based taxonomy approach. Our results demonstrate that there is great potential for discovery of new meiofauna species even in some of the most extensively studied locations.</p>
<h1>Marine Genomics: Methods and Protocols</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/240551>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/240551</a></p>
book<p>This volume presents the latest protocols for both laboratory and bioinformatics based analyses in the field of marine genomics. The chapters presented in the book cover a wide range of topics, including the sampling and genomics of bacterial communities, DNA extraction in marine organisms, high-throughput sequencing of whole mitochondrial genomes, phylogenomics, SNP discovery, SNP-arrays for species identification, digital PCR-based quantification methods, environment DNA for invasive species surveillance and monitoring, microarrays for the detection of waterborne pathogens, DNA barcoding of marine biodiversity, metabarcoding protocols for marine eukaryotes, analytical protocols for the visualization of eukaryotic diversity, and applications for genomic data to benthic indices for environmental monitoring. Written in the highly successful Methods in Molecular Biology series format, chapters include introduction to their respective topics, lists of the necessary materials and reagents, step-by-step, readily reproducible laboratory protocols, and tips on troubleshooting and avoiding known pitfalls. Cutting-edge and thorough, Marine Genomics: Methods and Protocols is a valuable resource for researchers, students, and policy makers in the field of marine biology.</p>
<h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/295858>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/295858</a></p>
article<p>© 2020 The Authors. Molecular Ecology published by John Wiley  &  Sons Ltd Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates < 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%–6.9%), when compared to low OC sediments (1.2%–2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode Sabatieria and foraminifera Elphidium in high OC, and Cryptocaryon-like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p>
<h1>Lake Sedimentary DNA Research on Past Terrestrial and Aquatic Biodiversity : Overview and Recommendations</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-440931>http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-440931</a></p>
article<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p>
<h1>Habitat conditions and phenological tree traits overrule the influence of tree genotype in the needle mycobiome–Picea glauca system at an arctic treeline ecotone</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/236528>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/236528</a></p>
article<p>Plant-associated mycobiomes in extreme habitats are understudied and poorly understood. We analysed Illumina-generated ITS1 sequences from the needle mycobiome of white spruce (Picea glauca) at the northern treeline in Alaska (USA). Sequences were obtained from the same DNA that was used for tree genotyping. In the present study, fungal metabarcoding and tree microsatellite data were compared for the first time. In general, neighbouring trees shared more fungal taxa with each other than trees growing in further distance. Mycobiomes correlated strongly with phenological host traits and local habitat characteristics contrasting a dense forest stand with an open treeline site. Genetic similarity between trees did not influence fungal composition and no significant correlation existed between needle mycobiome and tree genotype. Our results suggest the pronounced influence of local habitat conditions and phenotypic tree traits on needle-inhabiting fungi. By contrast, the tree genetic identity cannot be benchmarked as a dominant driver for needle-inhabiting mycobiomes, at least not for white spruce in this extreme environment.</p>
<h1>Fungal communities of West African ectomycorrhizal woodlands</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-425732>http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-425732</a></p>
book<p>Ectomycorrhizal symbiosis is a widespread mutualist relationship between fungi and plants, often trees. It is most well-known in temperate and boreal forests, but also occurs in a variety of tropical ecosystems, including Sudanian savanna woodlands and gallery forests in West Africa. In addition to their important role in nutrient cycling, many ectomycorrhizal fungi also produce edible mushrooms as their fruit-bodies. In this thesis, I explore spatial patterns of above-ground and below-ground diversity in West African ectomycorrhizal communities, as well as the use of wild edible fungi by the local human population. In Papers I and II, I used soil DNA metabarcoding to investigate fungal communities in their vegetative state. Paper I used measurements of the scale of spatial autocorrelation within the fungal community in Sudanian woodlands in Benin as a test to compare different high-throughput DNA sequencing strategies, including short (≈350 bp) amplicon barcoding using the Illumina, Ion Torrent, and PacBio sequencing platforms, as well as long (≈1550 bp) amplicon barcoding using PacBio. There were some differences in species richness and community composition recovered by the two amplicon lengths, attributable to known primer biases for the short amplicons, but these did not lead to different ecological results. Additionally, Paper I introduced new software packages for analysis of long-amplicon metabarcoding data and integrating phylogenetic information into sequence-based taxonomic identification. In Paper II we sampled both Sudanian woodland and gallery forest sites in five countries across West Africa using long-amplicon metabarcoding with PacBio. We found significant differences in fungal community composition between Sudanian woodland and gallery forest sites, but not between different tree species within each vegetation type. Papers III and IV focused above-ground to study the natural production and human use of the mushrooms themselves. In Paper III we exhaustively collected mushrooms from nine Sudanian woodland plots in Benin during three consecutive rainy seasons. We measured the total biomass produced of each morphospecies, as well as environmental variables related to microclimate, host tree availability, and soil chemistry. Mushroom production and diversity were negatively correlated with soil nitrogen levels, and positively correlated with soil phosphorus levels. Although there were no clear differences in the fungal communities associated with the four host trees present in our plots, greater host tree diversity was associated with greater fungal diversity and productivity. Finally, Paper IV combined interviews with local people from four ethnic groups in five villages near the study sites from Paper III about their knowledge and preferences for edible mushrooms with DNA barcoding of specimens. Knowledge and preferences for different mushrooms varied between ethnic groups, but people living in a village where their ethnic group is a minority tended to absorb knowledge about mushrooms from the majority group in the village. Women over age 35 were the most knowledgeable about mushrooms, but the degree of gender difference varied between groups. We compiled a list of the most choice edible mushrooms in the area, which can inform the possibility for commercial trade in wild mushrooms to supplement the income of rural people and encourage sustainable forest management.</p>
<h1>The future of dna barcoding : Reflections from early career researchers</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:lup.lub.lu.se:ab9c5837-8948-45f6-91e6-2920c2ebcbd7>http://swepub.kb.se/bib/swepub:oai:lup.lub.lu.se:ab9c5837-8948-45f6-91e6-2920c2ebcbd7</a></p>
article<p>Over the last two decades, the use of DNA barcodes has transformed our ability to identify and assess life on our planet. Both strengths and weaknesses of the method have been exemplified through thousands of peer-reviewed scientific articles. Given the novel sequencing approaches, currently capable of generating millions of reads at low cost, we reflect on the questions: What will the future bring for DNA barcoding? Will identification of species using short, standardized fragments of DNA stand the test of time? We present reflected opinions of early career biodiversity researchers in the form of a SWOT analysis and discuss answers to these questions.</p>
<h1>A method for the reintroduction of entire benthic invertebrate communities in formerly degraded streams</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:umu-165413>http://swepub.kb.se/bib/swepub:oai:DiVA.org:umu-165413</a></p>
article<p>Reintroduction is a commonly used tool in wildlife conservation. Although freshwater ecosystems are among the most threatened worldwide, reintroductions of entire benthic invertebrate communities have never been reported before. Here, for the first time, we developed a comprehensive method for the reintroduction of entire communities in formerly degraded streams that describes the selection of donor and recipient streams, the transfer of specimens, mortality rates, and a monitoring scheme. The stream selection procedure of the new method is based on widely available environmental variables and richness metrics that are regularly monitored by water authorities to fulfill the requirements of water policy frameworks (e.g. EU WFD). The sampling and transfer of specimens is also based on common and simply applicable methods (i.e. kick sampling). For each recipient stream, three donor streams were selected with a complementary set of reference species missing at the recipient stream. The sampling at each donor stream was done six times a year to cover different seasons and life stages. We applied our method to one stream (10â¯km2) and one river (1608â¯km2) in central Germany. In total, we transferred 325,197 and 401,233 specimens from three different donor streams, including 45 (28,682 specimens) and 50 (47,716 specimens) missing reference taxa into each recipient stream and river, respectively. Average mortality rates varied between 0.97% for all specimens and 1.72% for reference specimens. We recommend a post-reintroduction monitoring that includes new methods, i.e. environmental DNA and high throughput sequencing. Our proposed reintroduction method is based on commonly used variables, metrics and sampling techniques. Accordingly, this method requires only minor modifications to become applicable in other countries. The simultaneous reintroduction of a large number of reference taxa will increase the likelihood to reestablish near-natural benthic invertebrate communities in formerly degraded streams. This may increase the public and political acceptance of costly river restoration projects that so far, do not often reach a good ecological status even after decades, due to a large-scale depletion of potential source populations. Our method will help in the reintroduction of reference species in restored streams where natural recolonization is not possible.</p>
<h1>Microbial DNA Sequencing in Environmental Studies</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:kth-204897>http://swepub.kb.se/bib/swepub:oai:DiVA.org:kth-204897</a></p>
book<p>The field of microbial ecology has just entered a new era of rapid technological development and generation of big data. The high-throughput sequencing techniques presently available provide an opportunity to extensively inventorize the blueprints of life. Now, millions of microbes of natural microbial communities can be studied simultaneously without prior cultivation. New species and new functions (genes) can be discovered just by mining sequencing data. However, there is still a tremendous number of microorganisms not yet examined, nor are the ecosystem functions these carry out. The modern genomic technologies can contribute to solve environmental problems and help us understand ecosystems, but to most efficiently do so, methods need to be continuously optimised. During my Ph. D. studies, I developed a method to survey eukaryotic microbial diversity with a higher accuracy, and applied various sequencing-based approaches in an attempt to answer questions of importance in environmental research and ecology. In PAPER-I, we developed a set of 18S rRNA gene PCR primers with high taxonomic coverage, meeting the requirements of currently popular sequencing technologies and matching the richness of 18S rRNA reference sequences accumulated so far. In PAPER-II, we conducted the first sequencing-based spatial survey on the combined eukaryotic and bacterial planktonic community in the Baltic Sea to uncover the relationship of microbial diversity and environmental conditions. Here, the 18S primers designed in PAPER-I and a pair of broad-coverage 16S primers were employed to target the rRNA genes of protists and bacterioplankton for amplicon sequencing. In PAPER-III, we integrated metagenomic, metabarcoding, and metatranscriptomic data in an effort to scrutinise the protein synthesis potential (i.e., activity) of microbes in the sediment at a depth of 460 m in the Baltic Sea and, thus, disclosing microbial diversity and their possible ecological functions within such an extreme environment. Lastly, in PAPER-IV, we compared the performance of E. coli culturing, high-throughput sequencing, and portable real-time sequencing in tracking wastewater contamination in an urban stormwater system. From the aspects of cost, mobility and accuracy, we evaluated the usage of sequencing-based approaches in civil engineering, and for the first time, validated the real-time sequencing device in use within water quality monitoring. In summary, these studies demonstrate how DNA sequencing of microbial communities can be applied in environmental monitoring and ecological research.</p>
<h1>Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:kth-188057>http://swepub.kb.se/bib/swepub:oai:DiVA.org:kth-188057</a></p>
article<p>Microbial plankton form the productive base of both marine and freshwater ecosystems and are key drivers of global biogeochemical cycles of carbon and nutrients. Plankton diversity is immense with representations from all major phyla within the three domains of life. So far, plankton monitoring has mainly been based on microscopic identification, which has limited sensitivity and reproducibility, not least because of the numerical majority of plankton being unidentifiable under the light microscope. High-throughput sequencing of taxonomic marker genes offers a means to identify taxa inaccessible by traditional methods; thus, recent studies have unveiled an extensive previously unknown diversity of plankton. Here, we conducted ultra-deep Illumina sequencing (average 105 sequences/sample) of rRNA gene amplicons of surface water eukaryotic and bacterial plankton communities sampled in summer along a 2000 km transect following the salinity gradient of the Baltic Sea. Community composition was strongly correlated with salinity for both bacterial and eukaryotic plankton assemblages, highlighting the importance of salinity for structuring the biodiversity within this ecosystem. In contrast, no clear trends in alpha-diversity for bacterial or eukaryotic communities could be detected along the transect. The distribution of major planktonic taxa followed expected patterns as observed in monitoring programs, but groups novel to the Baltic Sea were also identified, such as relatives to the coccolithophore Erniliana huxleyi detected in the northern Baltic Sea. This study provides the first ultra-deep sequencing-based survey on eukaryotic and bacterial plankton biogeography in the Baltic Sea.</p>
<h1>Microbial metagenomics in the Baltic Sea : Recent advancements and prospects for environmental monitoring</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:lnu-56706>http://swepub.kb.se/bib/swepub:oai:DiVA.org:lnu-56706</a></p>
article<p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed.</p>
<h1>Application of high-throughput sequencing (HTS) metabarcoding to diatom biomonitoring: Do DNA extraction methods matter?</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:80689>http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:80689</a></p>
article<p>Current freshwater biomonitoring with diatoms is based on microscopic examination of the morphology of their silica skeleton. This standardized approach is time consuming and requires a high degree of taxonomic expertise. Metabarcoding combined with high-throughput sequencing (HTS) has great potential for next-generation biomonitoring applications but requires standardization. Molecular inventories are strongly influenced by the DNA extraction method used, but the effect of extraction protocols has not been tested to enable selection of the best DNA extraction method for HTS metabarcoding. We used 5 DNA extraction methods combining various types of cell lysis and DNA purification to extract DNA from 8 pure diatom cultures and 8 samples from streams and lakes with differing water quality. We compared the methods based on: 1) quality and purity of the extracted DNA, 2) community inventories obtained from HTS targeting the ribulose-1, 5-bisphosphate carboxylase (rbcL) barcode, and 3) similarity between molecular and microscopy-based inventories of community composition and the Specific Pollution sensitivity Index [SPI]. A method based on GenEluteTM-LPA had higher extraction efficiency than the 4 commercial kits but had the highest polymerase chain reaction inhibition level. All 5 methods were efficient for HTS, and method did not affect operational taxonomic unit richness. We observed variations in the relative abundance of some taxa within Nitzschia, Amphora, Encyonerna, Gomphonema, and Navicula between 2 of the 5 methods, but method did not affect global diatom community composition or SPI values. SPI values calculated from microscopy-based inventories and molecular inventories based on all 5 extraction methods were strongly correlated. For convenience purposes (high DNA quantity and low cost), we encourage standardization of FITS diatom biomonitoring based on the SA-Gen method.</p>
<h1>Molecular versus morphological data for benthic diatoms biomonitoring in Northern Europe freshwater and consequences for ecological status</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:101554>http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:101554</a></p>
article<p>Diatoms are known to be efficient bioindicators for water quality assessment because of their rapid response to environmental pressures and their omnipresence in water bodies. The identification of benthic diatoms communities in the biofilm, coupled with quality indices such as the Indice de polluosensibilité spécifique (IPS) can be used for biomonitoring purposes in freshwater. However, the morphological identification and counting of diatoms species under the microscope is time-consuming and requires extensive expertise to deal with a constantly evolving taxonomy. In response, a molecular-based and potentially more cost-effective method has been developed, coupling high-throughput sequencing and DNA metabarcoding. The method has already been tested for water quality assessment with diatoms in Central Europe. In this study, we applied both the traditional and molecular methods on 180 biofilms samples from Northern Europe (rivers and lakes of Fennoscandia and Iceland). The DNA metabarcoding data were obtained on two different DNA markers, the 18S-V4 and rbcL barcodes, with the NucleoSpin Soil kit for DNA extraction and sequenced on an Ion Torrent PGM platform. We assessed the ability of the molecular method to produce species inventories, IPS scores and ecological status class comparable to the ones generated by the traditional morphology-based approach. The two methods generated correlated but significantly different IPS scores and ecological status assessment. The observed deviations are explained by presence/absence and abundance discrepancies in the species inventories, mainly due to the incompleteness of the barcodes reference databases, primer bias and strictness of the bioinformatic pipeline. Abundance discrepancies are less common than presence/absence discrepancies but have a greater effect on the ecological assessment. Missing species in the reference databases are mostly acidophilic benthic diatoms species, typical of the low pH waters of Northern Europe. The two different DNA markers also generated significantly different ecological status assessments. The use of the 18S-V4 marker generates more species inventories discrepancies, but achieves an ecological assessment more similar to the traditional morphology-based method. Further development of the metabarcoding method is needed for its use in environmental assessment. For its application in Northern Europe, completion and curation of reference databases are necessary, as well as evaluation of the currently available bioinformatics pipelines. New indices, fitted for environmental biomonitoring, should also be developed directly from molecular data.</p>
<h1>Why We Need Sustainable Networks Bridging Countries, Disciplines, Cultures and Generations for Aquatic Biomonitoring 2.0: A Perspective Derived From the DNAqua-Net COST Action</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:94484>http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:94484</a></p>
article<p>Aquatic biomonitoring has become an essential task in Europe and many other regions as a consequence of strong anthropogenic pressures affecting the health of lakes, rivers, oceans and groundwater. A typical assessment of the environmental quality status, such as it is required by European but also North American and other legislation, relies on matching the composition of assemblages of organisms identified using morphological criteria present in aquatic ecosystems to those expected in the absence of anthropogenic pressures. Through decade-long and difficult intercalibration exercises among networks of regulators and scientists in European countries, a pragmatic biomonitoring approach was developed and adopted, which now produces invaluable information. Nonetheless, this approach is based on several hundred different protocols, making it susceptible to issues with comparability, scale and resolution. Furthermore, data acquisition is often slow due to a lack of taxonomic experts for many taxa and regions and time-consuming morphological identification of organisms. High-throughput genetic screening methods such as (e)DNA metabarcoding have been proposed as a possible solution to these shortcomings. Such "next-generation biomonitoring", also termed "biomonitoring 2.0", has many advantages over the traditional approach in terms of speed, comparability and costs. It also creates the potential to include new bioindicators and thereby further improves the assessment of aquatic ecosystem health. However, several major conceptual and technological challenges still hinder its implementation into legal and regulatory frameworks. Academic scientists sometimes tend to overlook legal or socioeconomic constraints, which regulators have to consider on a regular basis. Moreover, quantification of species abundance or biomass remains a significant bottleneck to releasing the full potential of these approaches. Here, we highlight the main challenges for next-generation aquatic biomonitoring and outline principles and good practices to address these with an emphasis on bridging traditional disciplinary boundaries between academics, regulators, stakeholders and industry.</p>
<h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:umu-168962>http://swepub.kb.se/bib/swepub:oai:DiVA.org:umu-168962</a></p>
article<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p>
<h1>Benthic Diatom Communities in an Alpine River Impacted by Waste Water Treatment Effluents as Revealed Using DNA Metabarcoding</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:99661>http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:99661</a></p>
article<p>Freshwater ecosystems are continuously affected by anthropogenic pressure. One of the main sources of contamination comes from wastewater treatment plant (WWTP) effluents that contain wide range of micro- and macropollutants. Chemical composition, toxicity levels and impact of treated effluents (TEs) on the recipient aquatic ecosystems may strongly differ depending on the wastewater origin. Compared to urban TEs, hospital ones may contain more active pharmaceutical substances. Benthic diatoms are relevant ecological indicators because of their high species and ecological diversity and rapid response to human pressure. They are routinely used for water quality monitoring. However, there is a knowledge gap on diatom communities' development and behavior in treated wastewater in relation to prevailing micro- and macropollutants. In this study, we aim to (1) investigate the response of diatom communities to urban and hospital TEs, and (2) evaluate TEs effect on communities in the recipient river. Environmental biofilms were colonized in TEs and the recipient river up- and downstream from the WWTP output to study benthic diatoms using DNA metabarcoding combined with high-throughput sequencing (HTS). In parallel, concentrations of nutrients, pharmaceuticals and seasonal conditions were recorded. Diatom metabarcoding showed that benthic communities differed strongly in their diversity and structure depending on the habitat. TE sites were generally dominated by few genera with polysaprobic preferences belonging to the motile guild, while river sites favored diverse communities from oligotrophic and oligosaprobic groups. Seasonal changes were visible to lower extent. To categorize parameters important for diatom changes we performed redundancy analysis which suggested that communities within TE sites were associated to higher concentrations of beta-blockers and non-steroidal anti-inflammatory drugs in urban effluents vs. antibiotics and orthophosphate in hospital effluents. Furthermore, indicator species analysis showed that 27% of OTUs detected in river downstream communities were indicator for urban or hospital TE sites and were absent in the river upstream. Finally, biological diatom index (BDI) calculated to evaluate the ecological status of the recipient river suggested water quality decrease linked to the release of TEs. Thus, in-depth assessment of diatom community composition using DNA metabarcoding is proposed as a promising technique to highlight the disturbing effect of pollutants in Alpine rivers.</p>
<h1>Diatom DNA Metabarcoding for Biomonitoring: Strategies to Avoid Major Taxonomical and Bioinformatical Biases Limiting Molecular Indices Capacities</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:102945>http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:102945</a></p>
article<p>Recent years provided intense progression in the implementation of molecular techniques in a wide variety of research fields in ecology. Biomonitoring and bioassessment can greatly benefit from DNA metabarcoding and High-Throughput Sequencing (HTS) methods that potentially provide reliable, high quantity and quality standardized data in a cost- and time-efficient way. However, DNA metabarcoding has its drawbacks, introducing biases at all the steps of the process, particularly during bioinformatics treatments used to prepare HTS data for ecological analyses. The high diversity of bioinformatics methods (e.g., OTU clustering, chimera detection, taxonomic assignment) and parameters (e.g., percentage similarity threshold used to define OTUs) make inter-studies comparison difficult, limiting the development of standardized and easy-accessible bioassessment procedures for routine freshwater monitoring. In order to study and overcome these drawbacks, we constructed four de novo indices to assess river ecological status based on the same biological samples of diatoms analyzed with morphological and molecular methods. The biological inventories produced are (i) morphospecies identified by microscopy, (ii) OTUs provided via metabarcoding and hierarchical clustering of sequences using a 95% similarity threshold, (iii) individual sequence units (ISUs) via metabarcoding and only minimal bioinformatical quality filtering, and (iv) exact sequence variants (ESVs) using DADA2 denoising algorithm. The indices based on molecular data operated directly with ecological values estimated for OTUs/ISUs/ESVs. Our study used an approach of bypassing taxonomic assignment, so bias related to unclassified sequences missing from reference libraries could be handled and no information on ecology of sequences is lost. Additionally, we showed that the indices based on ISUs and ESVs were equivalent, outperforming the OTU-based one in terms of predictive power and accuracy by revealing the hidden ecological information of sequences that are otherwise clustered in the same OTU (intra-species/intra-population variability). Furthermore, ISUs, ESVs, and morphospecies indices provided similar estimation of site ecological status, validating that ISUs with limited bioinformatics treatments may be used for DNA freshwater monitoring. Our study is a proof of concept where taxonomy- and clustering-free approach is presented, that we believe is a step forward a standardized and comparable DNA bioassessment, complementary to morphological methods.</p>
<h1>Freshwater biomonitoring in the Information Age</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:85017>http://swepub.kb.se/bib/swepub:oai:slubar.slu.se:85017</a></p>
article<p>Freshwaters worldwide face serious threats, making their protection increasingly important. Freshwater monitoring has historically produced valuable data and continues to develop. Rapid improvements to biomolecular techniques are revolutionizing the way scientists describe biological communities and are bringing about major changes in biomonitoring. Combined with high-throughput sequencing, DNA metabarcoding is fast and cost-effective, generating massive amounts of data. In a world with numerous ecological threats, "big data" constitute a tremendous opportunity to improve the efficiency of biological monitoring. These fundamental changes in biomonitoring will require freshwater ecologists and environmental managers to reconsider how they handle large amounts of data.</p>
<h1>Protist Biodiversity and Biogeography in Lakes From Four Brazilian River-Floodplain Systems</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/282251>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/282251</a></p>
article<p>The biodiversity and biogeography of protists inhabiting many ecosystems have been intensely studied using different sequencing approaches, but tropical ecosystems are relatively under-studied. Here, we sampled planktonic waters from 32 lakes associated with four different river-floodplains systems in Brazil, and sequenced the DNA using a metabarcoding approach with general eukaryotic primers. The lakes were dominated by the largely free-living Discoba (mostly the Euglenida), Ciliophora, and Ochrophyta. There was low community similarity between lakes even within the same river-floodplain. The protists inhabiting these floodplain systems comprise part of the large and relatively undiscovered diversity in the tropics.</p>
<h1>Preparation of Amplicon Libraries for Metabarcoding of Marine Eukaryotes Using Illumina MiSeq: The Adapter Ligation Method</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/240563>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/240563</a></p>
article<p>Amplicon-based studies of marine microscopic eukaryotes, also referred to as metabarcoding studies, can be performed to analyze patterns of biodiversity or predator–prey interactions targeting the mitochondrial cytochrome oxidase 1 (CO1) or the small ribosomal subunit (SSU) markers. Because high-throughput sequencing (HTS) Illumina platforms provide millions of reads per run, hundreds of samples may be sequenced simultaneously. This protocol details the preparation of multiplexed amplicon libraries for Illumina MiSeq sequencing. We describe a strategy for sample multiplexing using a combination of tailed PCR primers and ligation of indexed adapters.</p>
<h1>Debugging metabarcoding for insect biodiversity studies</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:su-180293>http://swepub.kb.se/bib/swepub:oai:DiVA.org:su-180293</a></p>
book<p>Insects are one of the most abundant and diverse animal groups, and they include many valuable ecological indicator species, but taxonomic discovery projects and biodiversity surveys targeting this group are often challenging. While mass trapping devices allow the collection of insects in great numbers, the task of identifying the species present is a painstaking and resource-demanding process. Metabarcoding, that is, high throughput sequencing of PCR-amplified species-specific genetic markers in environmental samples, promises to solve this problem. However, metabarcoding is still in its infancy. In this thesis, I optimized metabarcoding methods for inventorying and accelerating species discovery of terrestrial insects. In paper I, we designed new PCR primers for mitochondrial markers and evaluated them against existing ones using in silico methods. We showed that the best marker for metabarcoding of insects is 16S because of its broad taxonomic coverage and low amplification bias. However, there is significantly more reference data for COI, and its taxonomic coverage is reasonable when using sufficiently degenerate primers (mixes of primer sequences). In paper II, we applied 16S and COI metabarcoding to different types of samples of the same insect communities: Malaise trap samples (preservative ethanol or homogenized samples) and soil samples. The results show that the two-marker strategy increases biodiversity detection over single-marker analyses. They also show that 16S is better than COI for metabarcoding of eDNA samples because the less degenerate 16S primers do not amplify as many off-target organisms. Finally, the results show that analyses of tissue homogenate and preservative ethanol yield strikingly different results. Large and heavily sclerotized insects do not leak DNA into preservative ethanol like small and weakly sclerotized ones do, but their DNA tends to swamp the DNA of the latter in homogenized samples.  In paper III we evaluated the performance of various non-destructive mild lysis treatments and DNA purification methods. We subjected mock community samples to incubation in either a milder or a more aggressive digestion buffer for a short or a long period of incubation. The DNA was then extracted using either a manual or an automated purification protocol. We found that the milder digestion buffer and the shorter incubation time preserved the morphology of the insect best while at the same time giving the most accurate DNA metabarcoding results; the purification protocol had little or no effect on metabarcoding results. Finally, in paper IV, we explored the received wisdom that high concentrations of ethanol, although optimal for preservation of the DNA, make insects fragile and difficult to work with from a morphological point of view. We preserved insects in different ethanol concentrations and subjected them to damaging processes, such as shaking or transporting. We verified that high concentrations of ethanol induce brittleness, although the effect is less pronounced in robust insects. Our results also indicate that shipping by mail is safe for samples preserved at intermediate concentrations (70 or 80 %). In summary, this thesis represents a significant step forward in the development of methods for preserving and analyzing samples of terrestrial insects for biodiversity surveys, monitoring programs, and taxonomic research projects.</p>
<h1>Exploring the taxonomic composition of two fungal communities on the Swedish west coast through metabarcoding</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/283548>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/283548</a></p>
article<p>Background Fungi are heterotrophic, unicellular or filamentous organisms that exhibit a wide range of different lifestyles as, e.g., symbionts, parasites, and saprotrophs. Mycologists have traditionally considered fungi to be a nearly exclusively terrestrial group of organisms, but it is now known that fungi have a significant presence in aquatic environments as well. We know little about most fungi in limnic and marine systems, including aspects of their taxonomy, ecology, and geographic distribution. The present study seeks to improve our knowledge of fungi in the marine environment. The fungal communities of two coastal marine environments of the Kattegat sea, Sweden, were explored with metabarcoding techniques using the nuclear ribosomal internal transcribed spacer 2 (ITS2) metabarcode. Our data add new information to the current picture of fungal community composition in benthic and coastal habitats in Northern Europe. New information The dataset describes the number of operational taxonomic units (OTUs) and their taxonomic affiliations in two littoral gradients sampled on the Swedish west coast, Gothenburg municipality. Our data include basic diversity indices as well as chemical and edaphic sediment/soil parameters of the sampling sites. From the sites, 3470 and 4315 fungal OTUs, respectively, were recovered. The number of reads were 673,711 and 779,899, respectively, after quality filtering. Within the benthic sites, more than 80% of the sequences could not be classified taxonomically. The phylum composition of the classifiable sequences was dominated in both localities by Dikarya, which made up around 33% of the OTUs. Within Dikarya, Ascomycota was the dominant phylum. Guild assignment failed for more than half of the classifiable OTUs, with undefined saprotrophs being the most common resolved guild. This guild classification was slightly more common in the ocean sediment samples than in the terrestrial ones. Our metadata indicated that ocean sites contain organisms at a lower trophic level and that there are predominantly endophytic, parasitic, and pathogenic fungi in the marine environments. This hints at the presence of interesting and currently poorly understood fungus-driven ecological processes. It is also clear from our results that a very large number of marine fungi are in urgent need of taxonomic study and formal description.</p>
<h1>Advancing biodiversity assessments with environmental DNA: Long-read technologies help reveal the drivers of Amazonian fungal diversity</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/295161>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/295161</a></p>
article<p>Fungi are a key component of tropical biodiversity. However, due to their inconspicuous and largely subterranean nature, they are usually neglected in biodiversity inventories. The goal of this study was to identify the key determinants of fungal richness, community composition, and turnover in tropical rainforests. We tested specifically for the effect of soil properties, habitat, and locality in Amazonia. For these analyses, we used high-throughput sequencing data of short and long reads of fungal DNA present in soil and organic litter samples, combining existing and novel genomic data. Habitat type (phytophysiognomy) emerges as the strongest factor explaining fungal community composition. Naturally open areas-campinas-are the richest habitat overall. Soil properties have different effects depending on the soil layer (litter or mineral soil) and the choice of genetic marker. We suggest that campinas could be a neglected hotspot of fungal diversity. An underlying cause for their rich diversity may be the overall low soil fertility, which increases the reliance on biotic interactions essential for nutrient absorption in these environments, notably ectomycorrhizal fungi-plant associations. Our results highlight the advantages of using both short and long DNA reads produced through high-throughput sequencing to characterize fungal diversity. While short reads can suffice for diversity and community comparison, long reads add taxonomic precision and have the potential to reveal population diversity.</p>
<h1>Biodiversity assessments in the 21st century: The potential of insect traps to complement environmental samples for estimating eukaryotic and prokaryotic diversity using high-throughput DNA metabarcoding</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/277068>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/277068</a></p>
article<p>The rapid loss of biodiversity, coupled with difficulties in species identification, call for innovative approaches to assess biodiversity. Insects make up a substantial proportion of extant diversity and play fundamental roles in any given ecosystem. To complement morphological species identification, new techniques such as metabarcoding make it possible to quantify insect diversity and insect-ecosystem interactions through DNA sequencing. Here we examine the potential of bulk insect samples (i.e., containing many non-sorted specimens) to assess prokaryote and eukaryote biodiversity and to complement the taxonomic coverage of soil samples. We sampled 25 sites on three continents and in various ecosystems, collecting insects with Slam-traps (Brazil) and Malaise-traps (South Africa and Sweden). We then compared our diversity estimates with the results obtained with biodiversity data from soil samples from the same localities. We found a largely different taxonomic composition between the soil and insect samples, testifying to the potential of bulk insect samples to complement soil samples. Finally, we found that non-destructive DNA extraction protocols, which preserve insect specimens for morphological studies, constitute a promising choice for cost-effective biodiversity assessments. We propose that the sampling and sequencing of insect samples should become a standard complement for biodiversity studies based on environmental DNA.</p>
<h1>High-throughput metabarcoding reveals the effect of physicochemical soil properties on soil and litter biodiversity and community turnover across Amazonia</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/271602>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/271602</a></p>
article<p>Knowledge on the globally outstanding Amazonian biodiversity and its environmental determinants stems almost exclusively from aboveground organisms, notably plants. In contrast, the environmental factors and habitat preferences that drive diversity patterns for micro-organisms in the ground remain elusive, despite the fact that micro-organisms constitute the overwhelming majority of life forms in any given location, in terms of both diversity and abundance. Here we address how the diversity and community turnover of operational taxonomic units (OTU) of organisms in soil and litter respond to soil physicochemical properties; whether OTU diversities and community composition in soil and litter are correlated with each other; and whether they respond in a similar way to soil properties.We used recently inferred OTUs from high-throughput metabarcoding of the 16S (prokaryotes) and 18S (eukaryotes) genes to estimate OTU diversity (OTU richness and effective number of OTUs) and community composition for prokaryotes and eukaryotes in soil and litter across four localities in Brazilian Amazonia. All analyses were run separately for prokaryote and eukaryote OTUs, and for each group using both presence-absence and abundance data. Combining these with novel data on soil chemical and physical properties, we identify abiotic correlates of soil and litter organism diversity and community structure using regression, ordination, and variance partitioning analysis.Soil organic carbon content was the strongest factor explaining OTU diversity (negative correlation) and pH was the strongest factor explaining community turnover for prokaryotes and eukaryotes in both soil and litter. We found significant effects also for other soil variables, including both chemical and physical properties. The correlation between OTU diversity in litter and in soil was non-significant for eukaryotes and weak for prokaryotes. The community compositions of both prokaryotes and eukaryotes were more separated among habitat types (terra-firme, várzea, igapó and campina) than between substrates (soil and litter).In spite of the limited sampling (four localities, 39 plots), our results provide a broad-scale view of the physical and chemical correlations of soil and litter biodiversity in a longitudinal transect across the world's largest rainforest. Our methods help to understand links between soil properties, OTU diversity patterns, and community composition and turnover. The lack of strong correlation between OTU diversity in litter and in soil suggests independence of diversity drives of these substrates and highlights the importance of including both measures in biodiversity assessments. Massive sequencing of soil and litter samples holds the potential to complement traditional biological inventories in advancing our understanding of the factors affecting tropical diversity.</p>
<h1>Perspectives from Ten Years of Protist Studies by High-Throughput Metabarcoding</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-439315>http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-439315</a></p>
article<p>During the last decade, high-throughput metabarcoding became routine for analyzing protistan diversity and distributions in nature. Amid a multitude of exciting findings, scientists have also identified and addressed technical and biological limitations, although problems still exist for inference of meaningful taxonomic and ecological knowledge based on shortDNAsequences. Given the extensive use of this approach, it is critical to settle our understanding on its strengths and weaknesses and to synthesize up-to-date methodological and conceptual trends. This article summarizes key scientific and technical findings, and identifies current and future directions in protist research that uses metabarcoding.</p>
<h1>What do we learn from cultures in the omics age? High-throughput sequencing and cultivation of leaf-inhabiting endophytes from beech (Fagus sylvatica L.) revealed complementary community composition but similar correlations with local habitat conditions</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:umu-191778>http://swepub.kb.se/bib/swepub:oai:DiVA.org:umu-191778</a></p>
article<p>Comparative simultaneous studies of environmental high-throughput sequencing (HTS) and cultivation of plant-associated fungi have rarely been conducted in the past years. For the present contribution, HTS and extinction culturing were applied for the same leaf samples of European beech (Fagus sylvatica) in order to trace both “real” environmental drivers as well as method-dependent signals of the observed mycobiomes. Both approaches resulted in non-overlapping community composition and pronounced differences in taxonomic classification and trophic stages. However, both methods revealed similar correlations of the fungal communities with local environmental conditions. Our results indicate undeniable advantages of HTS over cultivation in terms of revealing a good representation of the major functional guilds, rare taxa and biodiversity signals of leaf-inhabiting fungi. On the other hand our results demonstrate that the immense body of literature about cultivable endophytic fungi can and should be used for the interpretation of community signals and environmental correlations obtained from HTS studies and that cultivation studies should be continued at the highest standards, e.g. when sequencing facilities are not available or if such surveys are expanded into functional aspects with experiments on living isolates.</p>
<h1>DNA metabarcoding adds valuable information for management of biodiversity in roadside stormwater ponds</h1>
<p><a href=http://libris.kb.se/bib/oai:research.chalmers.se:b5321427-50d7-4f0f-ac8a-5646e02816d6>http://libris.kb.se/bib/oai:research.chalmers.se:b5321427-50d7-4f0f-ac8a-5646e02816d6</a></p>
article<p>Stormwater ponds are used to compensate for the adverse effects that road runoff might have on the natural environment. Depending on their design and placement, stormwater ponds can act as both refugia and traps for local biodiversity. To evaluate the impact of stormwater ponds on biodiversity, it is critical to use effective and precise methods for identification of life associated with the water body. DNA metabarcoding has recently become a promising tool for identification and assessment of freshwater biodiversity. Using both morphology and DNA metabarcoding, we analyze species richness and biological composition of samples from 12 stormwater ponds and investigate the impact of pond size and pollution levels in the sediments and water column on the macroinvertebrate community structure. DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. The (dis)similarity of macroinvertebrate community composition in different ponds showed that the ponds appear better separated in the results obtained by DNA metabarcoding, but that the explained variation is higher for the results obtained by morphologically identification, since it provides abundance data. The reliance on morphological methods has limited our perception of the aquatic biodiversity in response to anthropogenic stressors, thereby providing inaccurate information for appropriate design and management of stormwater ponds; these drawbacks can be overcome by DNA metabarcoding. Synthesis and applications. The results indicate that DNA metabarcoding is a useful tool in identifying species, especially Diptera, which are difficult to determine. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing a more accurate information regarding the way the ponds function and how they are affected by management. OPEN PRACTICES: This article has earned an Open Data Badge for making publicly available the digitally-shareable data necessary to reproduce the reported results. The data is available at https://www.ebi.ac.uk/ena/data/view/PRJEB30841.</p>
<h1>The Ecological Role of Roadside Stormwater Ponds - Potential to Support Biodiversity</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:research.chalmers.se:4e046b16-b69b-4c4b-8914-aeb14d9f96be>http://swepub.kb.se/bib/swepub:oai:research.chalmers.se:4e046b16-b69b-4c4b-8914-aeb14d9f96be</a></p>
book<p>The increased recognition that roads may impair the aquatic environment and ecosystems has led to a shift from conventional drainage systems toward blue-green solutions such as stormwater ponds. Research on blue-green stormwater solutions has until now mainly focused on water quantity and quality. The aim of this study was to explore the ability of highway stormwater ponds to provide suitable habitats and support, especially macroinvertebrate biodiversity, and to identify the key environmental variables that affect biological community composition and the number of taxa, utilizing data at regional and global scales. Ultimately, this research informs design recommendations for stormwater systems that simultaneously provide multiple ecosystem services. The results of this thesis indicate that larger ponds are better for supporting aquatic biodiversity due to a more heterogeneous environment and the ability to dilute pollutants. Also, the presence of other ponds in the vicinity of the stormwater ponds can facilitate the movement of invertebrates between ponds through increased connectivity. An apparent negative effect of pollution levels on the macroinvertebrate community composition was observed, but not on the biodiversity measured as the number of taxa or Shannon index. The analyses based on the datasets identified using both morphology and DNA metabarcoding demonstrated that DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing more accurate information regarding the way the ponds function and how they are affected by management. Subsequently, the differences in the macroinvertebrate community composition between different types of ponds were compared at the regional and global scale. The results indicated that environmental characteristics, especially conductivity and pH, were different between different types of ponds. Alpha and gamma diversity were similar or even higher in manmade ponds compared to natural ponds due to very different macroinvertebrate communities. Moreover, generally ponds exhibited high levels of spatial heterogeneity, which subsequently enhances gamma diversity. In summary, stormwater ponds have the potential to provide suitable habitats to foster biodiversity. When such systems are created, larger ponds should be built to provide more heterogeneous habitat and dilute harmful pollutants. Additional ponds should also be created in the vicinity of the ponds, thereby promoting aquatic biodiversity through higher connectivity. Although stormwater ponds accumulate pollutants due to their primary functions, this pollution retention process creates a unique environment within the stormwater ponds, which are more suitable for taxa that are moderately to strongly tolerant to pollutants and that may not be found in natural ponds. In this way, stormwater ponds constitute an option in the areas along the highway so that they could combine water treatment properties with providing a suitable habitat for aquatic organisms.</p>
<h1>Response to Comment on “Global diversity and geography of soil fungi”</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/221440>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/221440</a></p>
article<p>Schadt and Rosling (Technical Comment, 26 June 2015, p. 1438) argue that primer-template mismatches neglected the fungal class Archaeorhizomycetes in a global soil survey. Amplicon-based metabarcoding of nine barcode-primer pair combinations and polymerase chain reaction (PCR)–free shotgun metagenomics revealed that barcode and primer choice and PCR bias drive the diversity and composition of microorganisms in general, but the Archaeorhizomycetes were little affected in the global study. We urge that careful choice of DNA markers and primers is essential for ecological studies using high-throughput sequencing for identification.</p>
<h1>Mycobiomes of sympatric Amorphophallus albispathus (Araceae) and Camellia sinensis (Theaceae) – a case study reveals clear tissue preferences and differences in diversity and composition</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/264112>http://swepub.kb.se/bib/swepub:oai:gup.ub.gu.se/264112</a></p>
article<p>Multiple biotic and abiotic parameters influence the dynamics of individual fungal species and entire communities. Major drivers for tropical plant endophytes are undoubtedly seasonality, local habitat conditions and biogeography. However, host specialization and tissue preferences also contribute to the structuring of endophytic mycobiomes. To elucidate such specializations and preferences, we sampled two commercially important, unrelated plant species, Amorphophallus albispathus and Camellia sinensis (tea plant) simultaneously at close proximity. The mycobiomes of different tissue types were assessed with high-throughput amplicon sequencing of the internal transcribed spacer DNA region. Both plants hosted different fungal communities and varied in α- and β-diversity, despite their neighboring occurrence. However, the fungal assemblages of Amorphophallus leaflets shared taxa with the mycobiomes of tea leaves, thereby suggesting common driving forces for leaf-inhabiting fungi irrespective of host plant identity. The mycobiome composition and diversity of tea leaves was clearly driven by leaf age. We suggest that the very youngest tea leaves are colonized by stochastic processes, while mycobiomes of old leaves are rather similar as the result of progressive succession. The biodiversity of fungi associated with A. albispathus was characterized by a large number of unclassified OTUs (at genus and species level) and by tissue-specific composition.This study is the first cultivation-independent high-throughput assessment of fungal biodiversity of an Amorphophallus species, and additionally expands the knowledge base on fungi associated with tea plants.</p>
<h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy)</h1>
<p><a href=http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-440098>http://swepub.kb.se/bib/swepub:oai:DiVA.org:uu-440098</a></p>
article<p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p>
