<h1>A method for the reintroduction of entire benthic invertebrate communities in formerly degraded streams</h1>
<p><a href="https://doi.org/10.1016/j.limno.2019.125689">doi:10.1016/j.limno.2019.125689</a></p>
article-journal<p>Reintroduction is a commonly used tool in wildlife conservation. Although freshwater ecosystems are among the most threatened worldwide, reintroductions of entire benthic invertebrate communities have never been reported before. Here, for the first time, we developed a comprehensive method for the reintroduction of entire communities in formerly degraded streams that describes the selection of donor and recipient streams, the transfer of specimens, mortality rates, and a monitoring scheme. The stream selection procedure of the new method is based on widely available environmental variables and richness metrics that are regularly monitored by water authorities to fulfill the requirements of water policy frameworks (e.g. EU WFD). The sampling and transfer of specimens is also based on common and simply applicable methods (i.e. kick sampling). For each recipient stream, three donor streams were selected with a complementary set of reference species missing at the recipient stream. The sampling at each donor stream was done six times a year to cover different seasons and life stages. We applied our method to one stream (10â¯km2) and one river (1608â¯km2) in central Germany. In total, we transferred 325,197 and 401,233 specimens from three different donor streams, including 45 (28,682 specimens) and 50 (47,716 specimens) missing reference taxa into each recipient stream and river, respectively. Average mortality rates varied between 0.97% for all specimens and 1.72% for reference specimens. We recommend a post-reintroduction monitoring that includes new methods, i.e. environmental DNA and high throughput sequencing. Our proposed reintroduction method is based on commonly used variables, metrics and sampling techniques. Accordingly, this method requires only minor modifications to become applicable in other countries. The simultaneous reintroduction of a large number of reference taxa will increase the likelihood to reestablish near-natural benthic invertebrate communities in formerly degraded streams. This may increase the public and political acceptance of costly river restoration projects that so far, do not often reach a good ecological status even after decades, due to a large-scale depletion of potential source populations. Our method will help in the reintroduction of reference species in restored streams where natural recolonization is not possible.</p>
<h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy)</h1>
<p><a href="https://doi.org/10.1038/s41598-020-79738-5">doi:10.1038/s41598-020-79738-5</a></p>
article-journal<p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p>
<h1>A Synthesis of Marine Monitoring Methods With the Potential to Enhance the Status Assessment of the Baltic Sea</h1>
<p><a href="https://doi.org/10.3389/fmars.2020.552047">doi:10.3389/fmars.2020.552047</a></p>
article-journal<p>A multitude of anthropogenic pressures deteriorate the Baltic Sea, resulting in theneed to protect and restore its marine ecosystem. For an efficient conservation,comprehensive monitoring and assessment of all ecosystem elements is of fundamentalimportance. The Baltic Marine Environment Protection Commission HELCOMcoordinates conservation measures regulated by several European directives. However,this holistic assessment is hindered by gaps within the current monitoring schemes.Here, twenty-two novel methods with the potential to fill some of these gaps andimprove the monitoring of the Baltic marine environment are examined. We asked keystakeholders to point out methods likely to improve current Baltic Sea monitoring. Wethen described these methods in a comparable way and evaluated them based ontheir costs and applicability potential (i.e., possibility to make them operational). Twelvemethods require low to very low costs, while five require moderate and two high costs.Seventeen methods were rated with a high to very high applicability, whereas fourmethods had moderate and one low applicability for Baltic Sea monitoring. Methodswith both low costs and a high applicability include the Manta Trawl, Rocket, SedimentCorer, Argo Float, Artificial Substrates, Citizen Observation, Earth Observation, theHydroFIARpH system, DNA Metabarcoding and Stable Isotope Analysis. © 2020 Mack, Attila, Aylagas, Beermann, Borja, Hering, Kahlert, Leese, Lenz, Lehtiniemi, Liess, Lips, Mattila, Meissner, Pyhälahti, Setälä, Strehse, Uusitalo, Willstrand Wranne and Birk.</p>
<h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden</h1>
<p><a href="https://doi.org/10.7717/peerj.8424">doi:10.7717/peerj.8424</a></p>
article-journal<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p>
<h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden</h1>
<p><a href="https://doi.org/10.7717/peerj.8424">doi:10.7717/peerj.8424</a></p>
article-journal<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p>
<h1>An urban Blitz with a twist : rapid biodiversity assessment using aquatic environmental DNA</h1>
<p><a href="https://doi.org/10.1002/edn3.152">doi:10.1002/edn3.152</a></p>
article-journal<p>As global biodiversity declines, there is an increasing need to create an educated and engaged society. Having people of all ages participate in measuring biodiversity where they live helps to create awareness. Recently, the use of environmental DNA (eDNA) for biodiversity surveys has gained momentum. Here, we explore whether sampling eDNA and sequencing it can be used as a means of rapidly surveying urban biodiversity for educational purposes. We sampled 2 × 1 L of water from each of 15 locations in the city of Trondheim, Norway, including a variety of freshwater, marine, and brackish habitats. DNA was extracted, amplified in triplicate targeting the barcoding fragment of COI gene, and sequenced. The obtained data were analyzed on the novel mBRAVE platform, an online open‐access software and computing resource. The water samples were collected in 2 days by two people, and the laboratory analysis was completed in 5 days by one person. Overall, we detected the presence of 506 BINs identified as belonging to 435 taxa, representing at least 265 putative species. On average, only 5.4% of the taxa were shared among six replicates per site. Based on the observed diversity, three distinct clusters were detected and related to the geographic distribution of sites. There were some taxa shared between the habitats, with a substantial presence of terrestrial biota. Here we propose a new form of BioBlitz, where with noninvasive sampling effort combined with swift processing and straightforward online analyses, hundreds of species can be detected. Thus, using eDNA analysis of water is useful for rapid biodiversity surveys and valuable for educational purposes. We show that rapid eDNA surveys, combined with openly available services and software, can be used as an educational tool to raise awareness about the importance of biodiversity.</p>
<h1>Analytics for Management : En modell som beskriver framtagandet av ett beslutsunderlag där rätt mätetal visualiseras på rätt sätt utifrån en formulerad strategi</h1>
<p>No DOI</p>thesis<p>Hur strategiarbete och processmätning bör gå till behandlas i befintliga teorier var för sig även om vikten av att sammankoppla dessa också beskrivs. Dock anses det saknas en enhetlig modell som förenar strategiarbete och processmätningar fullt ut och beskriver hur genomförandet praktiskt ska gå till. Med bakgrund i detta syftar examensarbetet Analytics for Management (AFM) till att utveckla en modell som beskriver hur mätetal (Key Performance Indicators, KPI:er) tas fram utifrån ett företags strategi och visualiseras på rätt sätt för att kunna beskriva enskilda processers kapacitet och förmåga. Detta resulterar i ett beslutsunderlag på ledningsnivå där strategiarbete och processmätning kopplas samman.Framtagandet av AFM-modellen inleddes med teoretiska studier inom ämnet vilket resulterade i en konceptuell modell. Därefter testades den konceptuella modellen genom fallstudier på tre företag med olika struktur och storlek. Genom att kombinera den konceptuella modellen med resultaten från fallstudierna färdigställdes sedan den slutgiltiga AFM-modellen.AFM-modellen består av tre faser där den första fasen involverar strategikonkretisering och målnedbrytning. Fasen innefattar att strategin bryts ner till strategiska mål och en strategikarta, formulering av kritiska framgångsfaktorer (KFF:er) och slutligen framtagning av taktiska mål. I den andra fasen sker KPI-framtagning genom att först generera preliminära KPI:er utifrån de taktiska målen och sedan kartlägga dem för att identifiera två typer av KPI:er. Dessa är utfallsmått (Key Performance Outcomes, KPO:er), som beskriver processprestandan och påverkansmått (Key Performance Drivers, KPD:er), som påverkar utfallen som KPO:er illustrerar. Genom att KPD:erna förbättras kan även KPO:erna förbättras och därför är det viktigt att styrning sker med hjälp av KPD:erna. Efter att KPI:er har kartlagts väljs vilka KPI:er som ska användas för mätning.I den sista fasen sker mätningsförberedelser genom att göra en datainsamlingsplan. Därefter sker mätningar och slutligen sammanställs mätdata för att visualisera KPI:erna på ett bra sätt. Hur KPI:er ska visualiseras beror på KPI-typ eftersom KPO:er och KPD:er ska användas för olika syften. KPO:erna ska visualiseras med enkla diagram i ett resultatkort som ledningen ska använda sig utav i ett första steg för att följa upp verksamheten. För att sedan finna orsaken till KPO:erna och kunna förbättra dem, ska ett styrkort användas. I styrkortet sammanställs KPD:erna med hjälp av styrdiagram, vilka visar variation över tid i processerna, för att tidigt upptäcka förändringar och därmed styra processer. AFM-modellen resulterar med hjälp av resultatkortet och styrkortet i ett beslutsunderlag där rätt KPI:er visualiseras på rätt sätt.Resultaten från fallstudieföretagen har visat att AFM-modellen har hög generaliserbarhet. Vidare kan AFM-modellen användas av olika typer av organisationer då strategiarbete och processmätning är aktuellt och viktigt för det dagliga arbetet oavsett bransch. Modellen ska genom beslutsunderlaget underlätta för organisationer att agera handlingskraftigt och arbeta mer proaktivt med hjälp av beslut baserat på data framtaget utifrån företagets strategi. AFM-modellen möjliggör därmed en enhetlig användning av mätetal i organisationen, där uppföljningsarbetet i hela verksamheten genomsyras av strategin.</p>
<h1>Anthropogenic impact on the historical phytoplankton community of Lake Constance reconstructed by multimarker analysis of sediment-core environmental DNA</h1>
<p><a href="https://doi.org/10.1111/mec.15696">doi:10.1111/mec.15696</a></p>
article-journal<p>During the 20th century, many lakes in the Northern Hemisphere were affected by increasing human population and urbanization along their shorelines and catchment, resulting in aquatic eutrophication. Ecosystem monitoring commenced only after the changes became apparent, precluding any examination of timing and dynamics of initial community change in the past and comparison of pre- and postimpact communities. Peri-Alpine Lake Constance (Germany) underwent a mid-century period of eutrophication followed by re-oligotrophication since the 1980s and is now experiencing warm temperatures. We extended the period for which monitoring data of indicator organisms exist by analysing historical environmental DNA (eDNA) from a sediment core dating back some 110 years. Using three metabarcoding markers-for microbial eukaryotes, diatoms and cyanobacteria-we revealed two major breakpoints of community change, in the 1930s and the mid-1990s. In our core, the latest response was exhibited by diatoms, which are classically used as palaeo-bioindicators for the trophic state of lakes. Following re-oligotrophication, overall diversity values reverted to similar ones of the early 20th century, but multivariate analysis indicated that the present community is substantially dissimilar. Community changes of all three groups were strongly correlated to phosphorus concentration changes, whereas significant relationships to temperature were only observed when we did not account for temporal autocorrelation. Our results indicate that each microbial group analysed exhibited a unique response, highlighting the particular strength of multimarker analysis of eDNA, which is not limited to organisms with visible remains and can therefore discover yet unknown responses and abiotic-biotic relationships.</p>
<h1>Archaea in boreal Swedish lakes are diverse, dominated by Woesearchaeota and follow deterministic community assembly</h1>
<p><a href="https://doi.org/10.1111/1462-2920.15058">doi:10.1111/1462-2920.15058</a></p>
article-journal<p>Despite their key role in biogeochemical processes, particularly the methane cycle, archaea are widely underrepresented in molecular surveys because of their lower abundance compared with bacteria and eukaryotes. Here, we use parallel high-resolution small subunit rRNA gene sequencing to explore archaeal diversity in 109 Swedish lakes and correlate archaeal community assembly mechanisms to large-scale latitudinal, climatic (nemoral to arctic) and nutrient (oligotrophic to eutrophic) gradients. Sequencing with universal primers showed the contribution of archaea was on average 0.8% but increased up to 1.5% of the three domains in forest lakes. Archaea-specific sequencing revealed that freshwater archaeal diversity could be partly explained by lake variables associated with nutrient status. Combined with deterministic co-occurrence patterns this finding suggests that ecological drift is overridden by environmental sorting, as well as other deterministic processes such as biogeographic and evolutionary history, leading to lake-specific archaeal biodiversity. Acetoclastic, hydrogenotrophic and methylotrophic methanogens as well as ammonia-oxidizing archaea were frequently detected across the lakes. Archaea-specific sequencing also revealed representatives of Woesearchaeota and other phyla of the DPANN superphylum. This study adds to our understanding of the ecological range of key archaea in freshwaters and links these taxa to hypotheses about processes governing biogeochemical cycles in lakes.</p>
<h1>Archaeal community changes in Lateglacial lake sediments : Evidence from ancient DNA</h1>
<p><a href="https://doi.org/10.1016/j.quascirev.2017.11.037">doi:10.1016/j.quascirev.2017.11.037</a></p>
article-journal<p>The Lateglacial/early Holocene sediments from the ancient lake at Hasseldala Port, southern Sweden provide an important archive for the environmental and climatic shifts at the end of the last ice age and the transition into the present Interglacial. The existing multi-proxy data set highlights the complex interplay of physical and ecological changes in response to climatic shifts and lake status changes. Yet, it remains unclear how microorganisms, such as Archaea, which do not leave microscopic features in the sedimentary record, were affected by these climatic shifts. Here we present the metagenomic data set of Hasseldala Port with a special focus on the abundance and biodiversity of Archaea. This allows reconstructing for the first time the temporal succession of major Archaea groups between 13.9 and 10.8 ka BP by using ancient environmental DNA metagenomics and fossil archaeal cell membrane lipids. We then evaluate to which extent these findings reflect physical changes of the lake system, due to changes in lake-water summer temperature and seasonal lake-ice cover. We show that variations in archaeal composition and diversity were related to a variety of factors (e.g., changes in lake water temperature, duration of lake ice cover, rapid sediment infilling), which influenced bottom water conditions and the sediment-water interface. Methanogenic Archaea dominated during the Allerod and Younger Dryas pollen zones, when the ancient lake was likely stratified and anoxic for large parts of the year. The increase in archaeal diversity at the Younger Dryas/Holocene transition is explained by sediment infilling and formation of a mire/peatbog. (C) 2017 Elsevier Ltd. All rights reserved.</p>
<h1>Archaeal community changes in Lateglacial lake sediments : Evidence from ancient DNA</h1>
<p><a href="https://doi.org/10.1016/j.quascirev.2017.11.037">doi:10.1016/j.quascirev.2017.11.037</a></p>
article-journal<p>The Lateglacial/early Holocene sediments from the ancient lake at Hasseldala Port, southern Sweden provide an important archive for the environmental and climatic shifts at the end of the last ice age and the transition into the present Interglacial. The existing multi-proxy data set highlights the complex interplay of physical and ecological changes in response to climatic shifts and lake status changes. Yet, it remains unclear how microorganisms, such as Archaea, which do not leave microscopic features in the sedimentary record, were affected by these climatic shifts. Here we present the metagenomic data set of Hasseldala Port with a special focus on the abundance and biodiversity of Archaea. This allows reconstructing for the first time the temporal succession of major Archaea groups between 13.9 and 10.8 ka BP by using ancient environmental DNA metagenomics and fossil archaeal cell membrane lipids. We then evaluate to which extent these findings reflect physical changes of the lake system, due to changes in lake-water summer temperature and seasonal lake-ice cover. We show that variations in archaeal composition and diversity were related to a variety of factors (e.g., changes in lake water temperature, duration of lake ice cover, rapid sediment infilling), which influenced bottom water conditions and the sediment-water interface. Methanogenic Archaea dominated during the Allerod and Younger Dryas pollen zones, when the ancient lake was likely stratified and anoxic for large parts of the year. The increase in archaeal diversity at the Younger Dryas/Holocene transition is explained by sediment infilling and formation of a mire/peatbog.</p>
<h1>Belowground fungal community diversity and composition associated with Norway spruce along an altitudinal gradient</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0208493">doi:10.1371/journal.pone.0208493</a></p>
article-journal<p>Altitudinal gradients provide valuable information about the effects of environmental variables on changes in species richness and composition as well as the distribution of below ground fungal communities. Since most knowledge in this respect has been gathered on aboveground communities, we focused our study towards the characterization of belowground fungal communities associated with two different ages of Norway spruce (Picea abies) trees along an altitudinal gradient. By sequencing the internal transcribed spacer (ITS) region on the Illumina platform, we investigated the fungal communities in a floristically and geologically relatively well explored forest on the slope of Mt. Iseler of the Bavarian Alps. From fine roots and rhizosphere of a total of 90 of Norway spruce trees from 18 plots we detected 1285 taxa, with a range of 167 to 506 (average 377) taxa per plot. Fungal taxa are distributed over 96 different orders belonging to the phyla Ascomycota, Basidiomycota, Chrytridiomycota, Glomeromycota, and Mucoromycota. Overall the Agaricales (438 taxa) and Tremellales (81 taxa) belonging to the Basidiomycota and the Hypocreales (65 spp.) and Helotiales (61 taxa) belonging to the Ascomycota represented the taxon richest orders. The evaluation of our multivariate generalized mixed models indicate that the altitude has a significant influence on the composition of the fungal communities (p &lt; 0.003) and that tree age determines community diversity (p &lt; 0.05). A total of 47 ecological guilds were detected, of which the ectomycorrhizal and saprophytic guilds were the most taxon-rich. Our ITS amplicon Illumina sequencing approach allowed us to characterize a high fungal community diversity that would not be possible to capture with fruiting body surveys alone. We conclude that it is an invaluable tool for diverse monitoring tasks and inventorying biodiversity, especially in the detection of microorganisms developing very ephemeral and/or inconspicuous fruiting bodies or lacking them all together. Results suggest that the altitude mainly influences the community composition, whereas fungal diversity becomes higher in mature/older trees. Finally, we demonstrate that novel techniques from bacterial microbiome analyses are also useful for studying fungal diversity and community structure in a DNA metabarcoding approach, but that incomplete reference sequence databases so far limit effective identification.</p>
<h1>Benthic-pelagic coupling in a changing world : Structural and functional responses of microbenthic communities to organic matter settling</h1>
<p>No DOI</p>thesis<p>Marine soft sediments form the second largest habitat on the planet. Organisms residing in this environment represent a vast reservoir of biodiversity, and play key roles in ecosystem processes. Most benthic organisms depend on organic matter (OM) inputs from phytoplankton in the overlying water column as food supply, but human impacts such as eutrophication and climate change are profoundly altering natural ecosystem dynamics. The consequences of changes in benthic-pelagic coupling for the biodiversity and functioning of soft-sediment communities have yet to be resolved. The aim of this thesis is to assess the role of OM settling on soft-sediments microeukaryotic (small organisms &lt; 1 mm) and bacterial communities. The intents are two-fold, to investigate impacts on (1) community structure and diversity (<b>chapters I</b>, <b>II</b> and <b>IV</b>); and (2) ecosystem functioning, notably in relation to nitrogen (N) cycling (<b>chapters I</b> and <b>III</b>). Our results show that settling OM quantity and quality both had a significant impact on microeukaryotic alpha-diversity. We observed a decrease in alpha-diversity following settling of diatom-derived spring bloom OM, possibly as a result of competitive exclusion, while cyanobacteria-derived summer bloom OM did not affect alpha-diversity (<b>chapters I</b> and <b>IV</b>). We also found that high biomass of diatoms and others fast sinking phytoplankton groups in the water column led to lower microeukaryotic alpha diversity after this material settled on the seafloor (<b>chapter IV</b>). Presumably, following this large sedimentation event, sediment oxygen (O<sub>2</sub>) demand was strongly stimulated, excluding O<sub>2</sub>-sensitive taxa. Overall, we propose that the assembly of microeukaryotic communities was primarily mediated by OM settling quantity (<b>chapter IV</b>), while differences in OM quality led to significant but more subtle changes, occurring at fine taxonomic level (<b>chapter I</b>). The response of bacterial communities to OM settling was less pronounced, and probably restricted to the uppermost sediment layer (<b>chapters I</b> and <b>IV</b>). We did, however, observe a significant effect of OM quality on bacterial communities assembly at the sediment-water interface, with taxa favored either by diatom- or by cyanobacteria-derived OM (<b>chapter II</b>). This study also showed that feedback mechanisms from nutrient recycling in the sediment could play a role in this response. Finally, our results indicated a substantial influence of OM quality on N cycling at the sediment-water interface. We found that settling of fresh OM (i.e. low C:N ratio) stimulated denitrification activity (<b>chapters I</b> and <b>III</b>), while simultaneously promoting more N recycling to the water column than settling of degraded OM (i.e. high C:N ratio) did (<b>chapter III</b>).  Altogether, our results indicate that current changes in OM settling dynamics in marine systems will likely impact microeukaryotic and, to some extent, bacterial biodiversity in soft sediments. Alterations in settling OM quality, in particular, may also affect crucial microbial processes involved in N cycling. This thesis highlights the importance of considering benthic-pelagic coupling mechanisms to better understand likely future changes in marine ecosystems.</p>
<h1>Biodiversity between sand grains: Meiofauna composition across southern and western Sweden assessed by metabarcoding</h1>
<p><a href="https://doi.org/10.3897/BDJ.8.e51813">doi:10.3897/BDJ.8.e51813</a></p>
article-journal<p>The meiofauna is an important part of the marine ecosystem, but its composition and distribution patterns are relatively unexplored. Here we assessed the biodiversity and community structure of meiofauna from five locations on the Swedish western and southern coasts using a high-throughput DNA sequencing (metabarcoding) approach. The mitochondrial cytochrome oxidase 1 (COI) mini-barcode and nuclear 18S small ribosomal subunit (18S) V1-V2 region were amplified and sequenced using Illumina MiSeq technology. Our analyses revealed a higher number of species than previously found in other areas: thirteen samples comprising 6.5 dm3 sediment revealed 708 COI and 1,639 18S metazoan OTUs. Across all sites, the majority of the metazoan biodiversity was assigned to <i>Arthropoda</i>, <i>Nematoda</i> and <i>Platyhelminthes</i>. Alpha and beta diversity measurements showed that community composition differed significantly amongst sites. OTUs initially assigned to <i>Acoela</i>, <i>Gastrotricha</i> and the two <i>Platyhelminthes</i> sub-groups <i>Macrostomorpha</i> and <i>Rhabdocoela</i> were further investigated and assigned to species using a phylogeny-based taxonomy approach. Our results demonstrate that there is great potential for discovery of new meiofauna species even in some of the most extensively studied locations.</p>
<h1>Biodiversity estimates and ecological interpretations of meiofaunal communities are biased by the taxonomic approach</h1>
<p><a href="https://doi.org/10.1038/s42003-018-0119-2">doi:10.1038/s42003-018-0119-2</a></p>
article-journal<p>No Abstract</p><h1>Comments Concerning the National Swedish Contaminant Monitoring Programme in Marine Biota, 2015</h1>
<p>No DOI</p>report<p>The environmental toxicants examined in this report can be classified into five groups – heavy metals, chlorinated compounds, brominated flame retardants, polyaromatic hydrocarbons and perfluorinated compounds. Each of these contaminants have been examined from various sites for up to six different fish species, in blue mussels, and in guillemot eggs, for varying lengths of time. The following summary examines overall trends, spatial and temporal, for the five groups.Condition and Fat ContentCondition and fat content in different species tended to follow the same pattern at the same sites, with a few exceptions. Most of the fish species generally displayed a decreasing trend in both condition and fat content at most sites examined. Exceptions to this were increases in condition factor seen in cod liver at Fladen, perch muscle at Kvädöfjärden, and for herring at Ängskärsklubb in spring. Also, an increase in fat content was seen during the most recent ten years for herring at Ängskärsklubb in spring. There were also some sites where no log linear trends were seen.Heavy MetalsDue to a change in methods for metal analysis (not mercury) in 2004, values between 2003 and 2007 should be interpreted with care. From 2009 metals are analyzed at ACES, Stockholm University.Generally, higher mercury concentrations are found in the Bothnian Bay, but also from one station in the Northern parts of Baltic Proper, compared to other parts of the Swedish coastline. The time series show varying concentrations over the study period. The longer time series in guillemot egg and spring-caught herring from the southern Bothnian Sea and southern Baltic Proper show significant decreases of mercury. On the other hand, increasing concentrations are seen in e.g., cod muscle, but the concentrations are fairly low compared to measured concentrations in perch from fresh water and coastal sites. In most cases, the mercury concentrations are above the EQSbiota of 20 ng/g wet weight.Lead is generally decreasing over the study period (in time series of sufficient length), supposedly due to the elimination of lead in gasoline. The highest concentrations are seen in the southern part of the Baltic Sea. Elevated lead concentrations between 2003 and 2007 (e.g. Harufjärden) should be viewed with caution (see above regarding change in analysis methods). Lead concentrations are below the suggested target level at all stations.Cadmium concentrations show varying non-linear trends over the monitored period. It is worth noting that despite several measures taken to reduce discharges of cadmium, generally the most recent concentrations are similar to concentrations measured 30 yearsago in the longer time series. Cadmium concentrations in herring and perch are all below the suggested target level of 160 μg/kg wet weight.The reported nickel concentrations show no consistent decreasing trends. Some series begin with two elevated values that exert a strong leverage effect on the regression line and may give a false impression of decreasing trends. Chromium generally shows decreasing concentrations, possibly explained by a shift in analytical method. The essential trace metals, copper and zinc, show no consistent trends during the monitored period.Generally higher concentrations of arsenic and silver are found along the west coast compared to other parts of the Sweadish coast line. However for silver a few stations in the Bothnian Sea and Bothnian Bay show comparable concentrations to the west coast stations.Chlorinated CompoundsGenerally, a decreasing concentrations were observed for all compounds (DDT’s, PCB’s, HCH’s, HCB) in all species examined, with a few exceptions, such as no change in TCDD-equivalents being seen in herring muscle (except at Änskärsklubb where very high concentrations at the beginning of the sampling period were seen and also at the west coast station Fladen). The longer time-series in guillemot also show a marked decrease in TCDD-equivalents from the start in the late 1960s until about 1985 from where no change occurred for many years, however, during the most recent ten years a decrease in the concentration is seen. Concentrations of DDE and CB-118 are for some species and sites still above their respective target levels.The chlorinated compounds generally show higher concentrations in the Bothnian Sea and/or Baltic Proper when compared to the Bothnian Bay and the Swedish west coast.Brominated Flame RetardantsElevated levels of HBCDD are seen in sites from the Baltic Proper, while the investigated PBDEs show higher concentrations in the Bothnian Bay. In addition, lower concentrations of all investigated PBDEs and HBCDD are seen on the Swedish west coast compared to the east coast. Temporally, significant increases in BDE-47, -99 and -100 have been seen in guillemot eggs since the late 1960s until the early 1990s, where concentrations then began to show decreases. Also, the concentration of HBCDD in guillemot eggs shows a decrease during the most recent ten years. For fish and blue mussels, BDE-47, -99, and -153 decreased at some sites and showed no trend at other sites. The concentration of HBCDD in fish and blue mussels showed inconsistent trends. The concentration of HBCDD is below the EQSbiota of 167 μg/kg wet weight for all fish species from all areas, while the concentration of BDE-47 alone is above the EQSbiota for sumPBDE of 0.0085 ng/g wet weight.PAHsOnly blue mussels have been examined for spatial differences in PAH concentrations. Concentration of ΣPAH was found to be higher from Kvädöfjärden in the Baltic Proper compared to stations at the West coast, but individual PAHs showed varying spatial patterns. Over time, acenaphthalene was rarely found above the detection limit. Significant decreasing trends were observed for ΣPAH, chrysene, fluoranthene and pyrene at Fjällbacka; for naphthalene at Kvädöfjärden; and for pyrene at Fladen.All time series where concentrations of various PAHs were compared with the target value based on OSPAR Ecological Assessment Criteria, or EC Environmental Quality Standards were below the target value.PFASsPFHxS and PFOS show a similar spatial pattern, but PFOS concentrations were approximately 25 times higher than PFHxS levels. The distribution of PFOS is quite homogenous along the Swedish coast but with somewhat higher concentrations in the Baltic Proper. PFOS concentrations in guillemot eggs are about 100-200 times higher than in herring liver. An overall increasing concentration of PFOS in guillemot eggs has been observed throughout the whole time period, however, during the most recent ten years, a change of direction is detected. The longer herring time series from Harufjärden, Landsort, and Utlängan show increasing concentrations for PFOS and most carboxylates. For FOSA, on the other hand, decreasing concentrations are seen during the most recent ten years.Organotin compoundsThe majority of the analysed tinorganic compounds showed concentrations below LOQ. However TBT and DPhT showed concentrations above LOQ at all stations with highest reported concentrations in fish from Örefjärden in the northern part of Bothnian Sea.</p>
<h1>Comparison of capture and storage methods for aqueous macrobial eDNA using an optimized extraction protocol : advantage of enclosed filter</h1>
<p><a href="https://doi.org/10.1111/2041-210X.12683">doi:10.1111/2041-210X.12683</a></p>
article-journal<p>Aqueous environmental DNA (eDNA) is an emerging efficient non-invasive tool for species inventory studies. To maximize performance of downstream quantitative PCR (qPCR) and next-generation sequencing (NGS) applications, quality and quantity of the starting material is crucial, calling for optimized capture, storage and extraction techniques of eDNA. Previous comparative studies for eDNA capture/storage have tested precipitation and open' filters. However, practical enclosed' filters which reduce unnecessary handling have not been included. Here, we fill this gap by comparing a filter capsule (Sterivex-GP polyethersulfone, pore size 022m, hereafter called SX) with commonly used methods. Our experimental set-up, covering altogether 41 treatments combining capture by precipitation or filtration with different preservation techniques and storage times, sampled one single lake (and a fish-free control pond). We selected documented capture methods that have successfully targeted a wide range of fauna. The eDNA was extracted using an optimized protocol modified from the DNeasy((R)) Blood &amp; Tissue kit (Qiagen). We measured total eDNA concentrations and Cq-values (cycles used for DNA quantification by qPCR) to target specific mtDNA cytochrome b (cyt b) sequences in two local keystone fish species. SX yielded higher amounts of total eDNA along with lower Cq-values than polycarbonate track-etched filters (PCTE), glass fibre filters (GF) or ethanol precipitation (EP). SX also generated lower Cq-values than cellulose nitrate filters (CN) for one of the target species. DNA integrity of SX samples did not decrease significantly after 2weeks of storage in contrast to GF and PCTE. Adding preservative before storage improved SX results. In conclusion, we recommend SX filters (originally designed for filtering micro-organisms) as an efficient capture method for sampling macrobial eDNA. Ethanol or Longmire's buffer preservation of SX immediately after filtration is recommended. Preserved SX capsules may be stored at room temperature for at least 2weeks without significant degradation. Reduced handling and less exposure to outside stress compared with other filters may contribute to better eDNA results. SX capsules are easily transported and enable eDNA sampling in remote and harsh field conditions as samples can be filtered/preserved on site.</p>
<h1>Complementary methods assessing short and long-term prey of a marine top predator ‒ Application to the grey seal-fishery conflict in the Baltic Sea</h1>
<p><a href="https://doi.org/10.1371/journal.pone.0208694">doi:10.1371/journal.pone.0208694</a></p>
article-journal<p>The growing grey seal (<i>Halichoerus grypus</i>) population in the Baltic Sea has created conflicts with local fisheries, comparable to similar emerging problems worldwide. Adequate information on the foraging habits is a requirement for responsible management of the seal population. We investigated the applicability of available dietary assessment methods by comparing morphological analysis and DNA metabarcoding of gut contents (short-term diet; n = 129/125 seals, respectively), and tissue chemical markers <i>i</i>.<i>e</i>. fatty acid (FA) profiles of blubber and stable isotopes (SIs) of liver and muscle (mid- or long-term diet; n = 108 seals for the FA and SI markers). The methods provided complementary information. Short-term methods indicated prey species and revealed dietary differences between age groups and areas but for limited time period. In the central Baltic, herring was the main prey, while in the Gulf of Finland percid and cyprinid species together comprised the largest part of the diet. Perch was also an important prey in the western Baltic Proper. The DNA analysis provided firm identification of many prey species, which were neglected or identified only at species group level by morphological analysis. Liver SIs distinguished spatial foraging patterns and identified potentially migrated individuals, whereas blubber FAs distinguished individuals frequently utilizing certain types of prey. Tissue chemical markers of adult males suggested specialized feeding to certain areas and prey, which suggest that these individuals are especially prone to cause economic losses for fisheries. We recommend combined analyses of gut contents and tissue chemical markers as dietary monitoring methodology of aquatic top predators to support an optimal ecosystem-based management.</p>
<h1>Continuous-cover forestry maintains soil fungal communities in Norway spruce dominated boreal forests</h1>
<p><a href="https://doi.org/10.1016/j.foreco.2020.118659">doi:10.1016/j.foreco.2020.118659</a></p>
article-journal<p>Traditional clear-fell forestry greatly alters community structure and ecosystem function within boreal forests and alternative management practices may reduce these impacts. Continuous-cover forestry can maintain similar invertebrate and plant communities to unmanaged forest, but whether this extends to soil fungal communities remains unclear. Within four sites across the mid-boreal zone of Sweden, we conducted a comprehensive study to assess the impact of continuous-cover and clear-felling on soil fungi and chemical properties within Norway spruce dominated forests, using unmanaged forest as a control. We sampled soils for chemical properties (pH, carbon, nitrogen, C/N and Organic matter) and used both surveys of fungal fruiting bodies and state of the DNA metabarcoding techniques to assess treatment effects on soil fungal communities. We found that forest management practices had significant effects soil pH, C and C/N ratio and that continuous-cover forestry had more similar soil properties to unmanaged forest. Furthermore, the biodiversity of fruiting bodies, as expressed by species richness and Shannon’s diversity index, was higher in continuous-cover forestry and unmanaged forest compared to clear-felled areas. However, the opposite was true for the diversity of soil fungal communities, which was probably due to the high level of disturbance in clear-felled areas, and thus, ample habitat for early successional colonisers and some remnants of mature forest communities. However, in agreement with predictions we found that the composition of both fruiting body and soil fungal communities broadly similar in continuous-cover and unmanaged forest, but fundamentally different to clear-felled areas. Consequently, our findings highlight that continuous-cover forestry is an alternative to conventional practise, maintaining communities associated with unmanaged forest and mimicking natural disturbance regimes.</p>
<h1>Contribution of soft-bodied meiofaunal taxa to Italian marine biodiversity</h1>
<p><a href="https://doi.org/10.1080/24750263.2020.1786607">doi:10.1080/24750263.2020.1786607</a></p>
article-journal<p>Meiofauna includes an astonishing diversity of organisms, whose census is far from being complete. Most classic ecological studies have focused on hard-bodied Ecdysozoan taxa (notably Copepoda and Nematoda), whose cuticle allows determination at species-level after fixation, rather than soft-bodied, Spiralian taxa, which most often lose any diagnostic feature in fixed samples. Yet, metabarcoding studies have recently revealed a species-richness of soft-bodied taxa comparable, and in cases superior, to that of Copepoda and Nematoda together. However, given objective difficulties inherent to their study, which necessarily has to be performed on living individuals, and their limited utilisation for ecological and applicative research, taxonomic expertise on soft-bodied organisms has declined over the years, and diversity of these phyla in most areas of the world is presently completely unknown. Here we present an expert-based survey of current knowledge on the composition and distribution of soft-bodied meiofaunal taxa in Italy, with special references to the predominantly or exclusively meiobenthic phyla Gastrotricha, Gnathostomulida, Platyhelminthes, Rotifera, Xenacoelomorpha, and macrofaunal taxa with conspicuous meiofaunal representatives (Annelida, Mollusca and Nemertea). A total of 638 described species have been reported from Italian coasts; furthermore, the existence of a large number of undescribed species is mentioned. Knowledge of Annelida, Gastrotricha, and Rotifera appears particularly detailed, placing Italy among the best-known country worldwide. In contrast, knowledge of Platyhelminthes and Xenacoelomorpha appears patchy, and limited to few areas. Sampling effort has been uneven, with most species recorded from the Tyrrhenian Sea, while large sections of the Adriatic and Ionian seas have been poorly explored. Results highlight the role that Marine Biological Stations, notably the Zoological Station “Anton Dohrn” in Naples, have had in promoting the study of soft-bodied taxa in Italy.</p>
<h1>Debugging metabarcoding for insect biodiversity studies</h1>
<p>No DOI</p>thesis<p>Insects are one of the most abundant and diverse animal groups, and they include many valuable ecological indicator species, but taxonomic discovery projects and biodiversity surveys targeting this group are often challenging. While mass trapping devices allow the collection of insects in great numbers, the task of identifying the species present is a painstaking and resource-demanding process. Metabarcoding, that is, high throughput sequencing of PCR-amplified species-specific genetic markers in environmental samples, promises to solve this problem. However, metabarcoding is still in its infancy. In this thesis, I optimized metabarcoding methods for inventorying and accelerating species discovery of terrestrial insects. In <b>paper I,</b> we designed new PCR primers for mitochondrial markers and evaluated them against existing ones using <i>in silico</i> methods. We showed that the best marker for metabarcoding of insects is 16S because of its broad taxonomic coverage and low amplification bias. However, there is significantly more reference data for COI, and its taxonomic coverage is reasonable when using sufficiently degenerate primers (mixes of primer sequences). In <b>paper II,</b> we applied 16S and COI metabarcoding to different types of samples of the same insect communities: Malaise trap samples (preservative ethanol or homogenized samples) and soil samples. The results show that the two-marker strategy increases biodiversity detection over single-marker analyses. They also show that 16S is better than COI for metabarcoding of eDNA samples because the less degenerate 16S primers do not amplify as many off-target organisms. Finally, the results show that analyses of tissue homogenate and preservative ethanol yield strikingly different results. Large and heavily sclerotized insects do not leak DNA into preservative ethanol like small and weakly sclerotized ones do, but their DNA tends to swamp the DNA of the latter in homogenized samples.  In <b>paper III</b> we evaluated the performance of various non-destructive mild lysis treatments and DNA purification methods. We subjected mock community samples to incubation in either a milder or a more aggressive digestion buffer for a short or a long period of incubation. The DNA was then extracted using either a manual or an automated purification protocol. We found that the milder digestion buffer and the shorter incubation time preserved the morphology of the insect best while at the same time giving the most accurate DNA metabarcoding results; the purification protocol had little or no effect on metabarcoding results. Finally, in <b>paper IV,</b> we explored the received wisdom that high concentrations of ethanol, although optimal for preservation of the DNA, make insects fragile and difficult to work with from a morphological point of view. We preserved insects in different ethanol concentrations and subjected them to damaging processes, such as shaking or transporting. We verified that high concentrations of ethanol induce brittleness, although the effect is less pronounced in robust insects. Our results also indicate that shipping by mail is safe for samples preserved at intermediate concentrations (70 or 80 %). In summary, this thesis represents a significant step forward in the development of methods for preserving and analyzing samples of terrestrial insects for biodiversity surveys, monitoring programs, and taxonomic research projects.</p>
<h1>Design and development of a graphical user interface for the monitoring process of an automated guided vehicle fleet</h1>
<p>No DOI</p>thesis<p>Many different autonomously driving mobile robots are used for industrial transports of materials or  oods in the context of internal logistic processes because of different use cases. The problem for the users that need to monitor the robots is that each manufacturer provides its own graphical user interface (GUI) wi th different operating modes and visual designs, which requires different trainings and constant swi tching between software. Therefore, this paper shows the design and development process of a graphical user interface in the form of a web application for the monitoring process of a fleet of automated guided vehicles from different manufacturers and answers the following question: <i>"What are the main criteria when designing a graphical user interface with high usability for the monitoring process of manufacturer-independent automa ted guided vehicle f leets?” </i>To answer the question, existing graphical user interfaces from different manufacturers were analyzed and interviews with developers and end-users of the GUIs were conducted. Requirements were then derived, on whose basis sketching, wireframing and high-fidelity prototyping have been performed. Usability testing and a heuristic evaluation were chosen to improve the application and its usability continually. As a result, the following six main criteria could be derived that summarize the most essential points to consider when designing such a GUI: administrabi lity, adaptiveness, observability, analyzability, robot and job awareness, and intervention.</p>
<h1>Distinct Oceanic Microbiomes From Viruses to Protists Located Near the Antarctic Circumpolar Current</h1>
<p><a href="https://doi.org/10.3389/fmicb.2018.01474">doi:10.3389/fmicb.2018.01474</a></p>
article-journal<p>Microbes occupy diverse ecological niches and only through recent advances in next generation sequencing technologies have the true microbial diversity been revealed. Furthermore, lack of perceivable marine barriers to genetic dispersal (i.e., mountains or islands) has allowed the speculation that organisms that can be easily transported by currents and therefore proliferate everywhere. That said, ocean currents are now commonly being recognized as barriers for microbial dispersal. Here we analyzed samples collected from a total of six stations, four located in the Indian Ocean, and two in the Southern Ocean. Amplicon sequencing was used to characterize both prokaryotic and eukaryotic plankton communities, while shotgun sequencing was used for the combined environmental DNA (eDNA), microbial eDNA (meDNA), and viral fractions. We found that Cyanobacteria dominated the prokaryotic component in the South-West Indian Ocean, while gamma-Proteobacteria dominated the South-East Indian Ocean. A combination of gamma- and alpha-Proteobacteria dominated the Southern Ocean. Alveolates dominated almost exclusively the eukaryotic component, with variation in the ratio of Protoalveolata and Dinoflagellata depending on station. However, an increase in haptophyte relative abundance was observed in the Southern Ocean. Similarly, the viral fraction was dominated by members of the order Caudovirales across all stations; however, a higher presence of nucleocytoplasmic large DNA viruses (mainly chloroviruses and mimiviruses) was observed in the Southern Ocean. To our knowledge, this is the first that a statistical difference in the microbiome (from viruses to protists) between the subtropical Indian and Southern Oceans. We also show that not all phylotypes can be found everywhere, and that meDNA is not a suitable resource for monitoring aquatic microbial diversity.</p>
<h1>Distribution, occurrence and biotoxin composition of the main shellfish toxin producing microalgae within European waters : A comparison of methods of analysis</h1>
<p><a href="https://doi.org/10.1016/j.hal.2016.02.008">doi:10.1016/j.hal.2016.02.008</a></p>
book<p>Harmful algal blooms (HABs) are a natural global phenomena emerging in severity and extent. Incidents have many economic, ecological and human health impacts. Monitoring and providing early warning of toxic HABs are critical for protecting public health. Current monitoring programmes include measuring the number of toxic phytoplankton cells in the water and biotoxin levels in shellfish tissue. As these efforts are demanding and labour intensive, methods which improve the efficiency are essential. This study compares the utilisation of a multitoxin surface plasmon resonance (multitoxin SPR) biosensor with enzyme-linked immunosorbent assay (ELISA) and analytical methods such as high performance liquid chromatography with fluorescence detection (HPLC-FLD) and liquid chromatography-tandem mass spectrometry (LC-MS/MS) for toxic HAB monitoring efforts in Europe. Seawater samples (n = 256) from European waters, collected 2009-2011, were analysed for biotoxins: saxitoxin and analogues, okadaic acid and dinophysistoxins 1/2 (VDU /DTX2) and domoic acid responsible for paralytic shellfish poisoning (PSP), diarrheic shellfish poisoning (DSP) and amnesic shellfish poisoning (ASP), respectively. Biotoxins were detected mainly in samples from Spain and Ireland. France and Norway appeared to have the lowest number of toxic samples. Both the multitoxin SPR biosensor and the RNA microarray were more sensitive at detecting toxic HABs than standard light microscopy phytoplankton monitoring. Correlations between each of the detection methods were performed with the overall agreement, based on statistical 2 x 2 comparison tables, between each testing platform ranging between 32% and 74% for all three toxin families illustrating that one individual testing method may not be an ideal solution. An efficient early warning monitoring system for the detection of toxic HABs could therefore be achieved by combining both the multitoxin SPR biosensor and RNA microarray. (C) 2016 Elsevier B.V. All rights reserved.</p>
<h1>Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea</h1>
<p><a href="https://doi.org/10.3389/fmicb.2016.00679">doi:10.3389/fmicb.2016.00679</a></p>
book<p>Microbial plankton form the productive base of both marine and freshwater ecosystems and are key drivers of global biogeochemical cycles of carbon and nutrients. Plankton diversity is immense with representations from all major phyla within the three domains of life. So far, plankton monitoring has mainly been based on microscopic identification, which has limited sensitivity and reproducibility, not least because of the numerical majority of plankton being unidentifiable under the light microscope. High-throughput sequencing of taxonomic marker genes offers a means to identify taxa inaccessible by traditional methods; thus, recent studies have unveiled an extensive previously unknown diversity of plankton. Here, we conducted ultra-deep Illumina sequencing (average 105 sequences/sample) of rRNA gene amplicons of surface water eukaryotic and bacterial plankton communities sampled in summer along a 2000 km transect following the salinity gradient of the Baltic Sea. Community composition was strongly correlated with salinity for both bacterial and eukaryotic plankton assemblages, highlighting the importance of salinity for structuring the biodiversity within this ecosystem. In contrast, no clear trends in alpha-diversity for bacterial or eukaryotic communities could be detected along the transect. The distribution of major planktonic taxa followed expected patterns as observed in monitoring programs, but groups novel to the Baltic Sea were also identified, such as relatives to the coccolithophore Erniliana huxleyi detected in the northern Baltic Sea. This study provides the first ultra-deep sequencing-based survey on eukaryotic and bacterial plankton biogeography in the Baltic Sea.</p>
<h1>Effects of operational taxonomic unit inference methods on soil microeukaryote community analysis using long‐read metabarcoding</h1>
<p><a href="https://doi.org/10.1002/ece3.8676">doi:10.1002/ece3.8676</a></p>
article-journal<p>Long amplicon metabarcoding has opened the door for phylogenetic analysis of the largely unknown communities of microeukaryotes in soil. Here, we amplified and sequenced the ITS and LSU regions of the rDNA operon (around 1500 bp) from grassland soils using PacBio SMRT sequencing. We tested how three different methods for generation of operational taxonomic units (OTUs) effected estimated richness and identified taxa, and how well large-scale ecological patterns associated with shifting environmental conditions were recovered in data from the three methods. The field site at Kungsängen Nature Reserve has drawn frequent visitors since Linnaeus's time, and its species rich vegetation includes the largest population of <i>Fritillaria meleagris</i> in Sweden. To test the effect of different OTU generation methods, we sampled soils across an abrupt moisture transition that divides the meadow community into a <i>Carex acuta</i> dominated plant community with low species richness in the wetter part, which is visually distinct from the mesic-dry part that has a species rich grass-dominated plant community including a high frequency of <i>F</i>. <i>meleagris</i>. We used the moisture and plant community transition as a framework to investigate how detected belowground microeukaryotic community composition was influenced by OTU generation methods. Soil communities in both moisture regimes were dominated by protists, a large fraction of which were taxonomically assigned to Ciliophora (Alveolata) while 30%–40% of all reads were assigned to kingdom Fungi. Ecological patterns were consistently recovered irrespective of OTU generation method used. However, different methods strongly affect richness estimates and the taxonomic and phylogenetic resolution of the characterized community with implications for how well members of the microeukaryotic communities can be recognized in the data.</p>
<h1>Effects of predation stress and food ration on perch gut microbiota</h1>
<p><a href="https://doi.org/10.1186/s40168-018-0400-0">doi:10.1186/s40168-018-0400-0</a></p>
article-journal<p>Background: Gut microbiota provide functions of importance to influence hosts' food digestion, metabolism, and protection against pathogens. Factors that affect the composition and functions of gut microbial communities are well studied in humans and other animals; however, we have limited knowledge of how natural food web factors such as stress from predators and food resource rations could affect hosts' gut microbiota and how it interacts with host sex. In this study, we designed a two-factorial experiment exposing perch (Perca fluviatilis) to a predator (pike, Esox lucius), and different food ratios, to examine the compositional and functional changes of perch gut microbiota based on 16S rRNA amplicon sequencing. We also investigated if those changes are host sex dependent.Results: We showed that overall gut microbiota composition among individual perch significantly responded to food ration and predator presence. We found that species richness decreased with predator presence, and we identified 23 taxa from a diverse set of phyla that were over-represented when a predator was present. For example, Fusobacteria increased both at the lowest food ration and at predation stress conditions, suggesting that Fusobacteria are favored by stressful situations for the host. In concordance, both food ration and predation stress seemed to influence the metabolic repertoire of the gut microbiota, such as biosynthesis of other secondary metabolites, metabolism of cofactors, and vitamins. In addition, the identified interaction between food ration and sex emphasizes sex-specific responses to diet quantity in gut microbiota.Conclusions: Collectively, our findings emphasize an alternative state in gut microbiota with responses to changes in natural food webs depending on host sex. The obtained knowledge from this study provided us with an important perspective on gut microbiota in a food web context.</p>
<h1>Effects of Recreational Boating on Microbial and Meiofauna Diversity in Coastal Shallow Ecosystems of the Baltic Sea</h1>
<p><a href="https://doi.org/10.1128/mSphere.00127-21">doi:10.1128/mSphere.00127-21</a></p>
article-journal<p>Recreational boating can impact benthic ecosystems in coastal waters. Reduced height and cover of aquatic vegetation in shallow Baltic Sea inlets with high boat traffic have raised concerns about cascading effects on benthic communities in these ecosystems. Here, we characterized the diversity and composition of sediment-associated microbial and meiofaunal communities across five bays subjected to low and high degrees of boating activity and examined the community-environment relationships and association with bay morphometry. We found that recreational boating activity altered meiofauna alpha diversity and the composition of both micro- and meiobenthic communities, and there were strong correlations between community structure and morphometric variables like topographic openness, wave exposure, water surface area, and total phosphorous concentrations. Inlets with high boat traffic showed an increase of bacterial taxa like <i>Hydrogenophilaceae</i> and <i>Burkholderiaceae</i>. Several meiofauna taxa previously reported to respond positively to high levels of suspended organic matter were found in higher relative abundances in the bays with high boat traffic. Overall, our results show that morphometric characteristics of inlets are the strongest drivers of benthic diversity in shallow coastal environments. However, while the effects were small, we found significant effects of recreational boating on benthic community structure that should be considered when evaluating the new mooring projects.<b>IMPORTANCE</b> With the increase of recreational boating activity and development of boating infrastructure in shallow, wave-protected areas, there is growing concern for their impact on coastal ecosystems. In order to properly assess the effects and consider the potential for recovery, it is important to investigate microbial and meiofaunal communities that underpin the functioning of these ecosystems. Here, we present the first study that uses DNA metabarcoding to assess how benthic biodiversity in shallow coastal areas is impacted by recreational boating. Our study shows a relatively small, but significant, effect of recreational boating both on meiofauna alpha diversity and meiofauna and bacterial community composition. However, both meiofauna and bacterial community composition in shallow benthic habitats is mediated to a higher degree by abiotic variables, such as topographic openness, area or size of the inlets, and wave exposure. Despite the fact that the effects were small, such impacts on benthic biodiversity should be considered in the management of coastal shallow habitats.</p>
<h1>Efficient traffic monitoring in 5G Core Network</h1>
<p>No DOI</p>thesis<p>5G is an enabler to several new use cases. To support all of them, the network infrastructure must be flexible and it should adapt to the different situations. This feature is powered by SDN, NFV, and Automation, three of the main pillars on which the 5G network is built.Traditional network management approaches may not be suitable for the 5G Core Network User Plane, which holds strict requirements in terms of latency and throughput. Therefore, Artificial Intelligence agents have been proposed to manage the 5G in a more efficient manner, delivering a more optimized allocation of the resources. This approach requires real-time monitoring of the data passing by the Core Network, a feature not standardized by the current protocols. In this thesis, the design of a monitoring protocol for the 5G Core Network User Plane has been studied, focusing on precise measurement of latencies. Then, a In-band Network Telemetry (INT) framework has been implemented on top of a User Plane Function prototype. The prototype is built on top of a novel User Plane implementation, based on chaining of atomic functions called micro-UPFs (µUPFs).While the main focus of this work has been latency measurement, packet counters, byte counters and Inter Packet Gap values can be collected from the framework, proving the main KPIs of a 5G User Plane. The INT framework has been implemented through two new µUPFs, one for updating the INT metadata and one for collecting them. These metadata are attached to the user packets as GTP-U extended header, maintaining compatibility with the standard protocol. Moreover, the implemented framework allows high flexibility through dynamic tuning of the parameters, providing mechanisms to reduce the amount of telemetry data generated and, thus, the system overhead.The framework has been tested on a physical setup of four server machines, abstracting a Core Network User Plane, connected with 10 Gbps NICs. In all the tests performed, the performances of the User Plane are affected by the new functionalities only when INT metadata are inserted very frequently. The results show that is possible to monitor the three main KPIs of a 5G User Plane without heavily limiting the system performances.</p>
<h1>Environmental DNA : For Biodiversity Research and Monitoring</h1>
<p><a href="https://doi.org/10.1177/0959683619877297">doi:10.1177/0959683619877297</a></p>
review-book<p>No Abstract</p><h1>Environmental DNA time series in ecology</h1>
<p><a href="https://doi.org/10.1016/j.tree.2018.09.003">doi:10.1016/j.tree.2018.09.003</a></p>
review<p>Ecological communities change in time and space, but long-term dynamics at the century-to-millennia scale are poorly documented due to lack of relevant data sets. Nevertheless, understanding long-term dynamics is important for explaining present-day biodiversity patterns and placing conservation goals in a historical context. Here, we use recent examples and new perspectives to highlight how environmental DNA (eDNA) is starting to provide a powerful new source of temporal data for research questions that have so far been overlooked, by helping to resolve the ecological dynamics of populations, communities, and ecosystems over hundreds to thousands of years. We give examples of hypotheses that may be addressed by temporal eDNA biodiversity data, discuss possible research directions, and outline related challenges.</p>
<h1>Establishment of a taxonomic and molecular reference collection to support the identification of species regulated by the Western Australian Prevention List for Introduced Marine Pests</h1>
<p><a href="https://doi.org/10.3391/mbi.2017.8.2.09">doi:10.3391/mbi.2017.8.2.09</a></p>
article-journal<p>Introduced Marine Pests (IMP, = non-indigenous marine species) prevention, early detection and risk-based management strategies have become the priority for biosecurity operations worldwide, in recognition of the fact that, once established, the effective management of marine pests can rapidly become cost prohibitive or impractical. In Western Australia (WA), biosecurity management is guided by the Western Australian Prevention List for Introduced Marine Pests which is a policy tool that details species or genera as being of high risk to the region. This list forms the basis of management efforts to prevent introduction of these species, monitoring efforts to detect them at an early stage, and rapid response should they be detected. It is therefore essential that the species listed can be rapid and confidently identified and discriminated from native species by a range of government and industry stakeholders. Recognising that identification of these species requires very specialist expertise which may be in short supply and not readily accessible in a regulatory environment, and the fact that much publicly available data is not verifiable or suitable for regulatory enforcement, the WA government commissioned the current project to collate a reference collection of these marine pest specimens. In this work, we thus established collaboration with researchers worldwide in order to source representative specimens of the species listed. Our main objective was to build a reference collection of taxonomically vouchered specimens and subsequently to generate species-specific DNA barcodes suited to supporting their future identification. To date, we were able to obtain specimens of 75 species (representative of all but four of the pests listed) which have been identified by experts and placed with the WA Government Department of Fisheries and, where possible, in accessible museums and institutions in Australasia. The reference collection supports the fast and reliable taxonomic and molecular identification of marine pests in WA and constitutes a valuable resource for training of stakeholders with interest in IMP recognition in Australia. The reference collection is also useful in supporting the development of a variety of DNA-based detection strategies such as real-time PCR and metabarcoding of complex environmental samples (e.g. biofouling communities). The Prevention List is under regular review to ensure its continued relevance and that it remains evidence and risk-based. Similarly, its associated reference collection also remains to some extent a work in progress. In recognition of this fact, this report seeks to provide details of this continually evolving information repository publicly available to the biosecurity management community worldwide.</p>
<h1>Fish consumption of harbour seals (Phoca vitulina) in north western Iceland assessed by DNA metabarcoding and morphological analysis</h1>
<p>No DOI</p>article-journal<p>No Abstract</p><h1>Fish consumption of harbour seals (Phoca vitulina) in north western Iceland assessed by DNA metabarcoding and morphological analysis</h1>
<p><a href="https://doi.org/DOI: 10.1007/s00300-018-2354-x">doi:DOI: 10.1007/s00300-018-2354-x</a></p>
article-journal<p>No Abstract</p><h1>Fungal communities of West African ectomycorrhizal woodlands</h1>
<p>No DOI</p>thesis<p>Ectomycorrhizal symbiosis is a widespread mutualist relationship between fungi and plants, often trees. It is most well-known in temperate and boreal forests, but also occurs in a variety of tropical ecosystems, including Sudanian savanna woodlands and gallery forests in West Africa. In addition to their important role in nutrient cycling, many ectomycorrhizal fungi also produce edible mushrooms as their fruit-bodies. In this thesis, I explore spatial patterns of above-ground and below-ground diversity in West African ectomycorrhizal communities, as well as the use of wild edible fungi by the local human population. In <b>Papers I</b> and <b>II</b>, I used soil DNA metabarcoding to investigate fungal communities in their vegetative state. <b>Paper I</b> used measurements of the scale of spatial autocorrelation within the fungal community in Sudanian woodlands in Benin as a test to compare different high-throughput DNA sequencing strategies, including short (≈350 bp) amplicon barcoding using the Illumina, Ion Torrent, and PacBio sequencing platforms, as well as long (≈1550 bp) amplicon barcoding using PacBio. There were some differences in species richness and community composition recovered by the two amplicon lengths, attributable to known primer biases for the short amplicons, but these did not lead to different ecological results. Additionally, <b>Paper I</b> introduced new software packages for analysis of long-amplicon metabarcoding data and integrating phylogenetic information into sequence-based taxonomic identification. In <b>Paper II</b> we sampled both Sudanian woodland and gallery forest sites in five countries across West Africa using long-amplicon metabarcoding with PacBio. We found significant differences in fungal community composition between Sudanian woodland and gallery forest sites, but not between different tree species within each vegetation type. <b>Papers III</b> and <b>IV</b> focused above-ground to study the natural production and human use of the mushrooms themselves. In <b>Paper III</b> we exhaustively collected mushrooms from nine Sudanian woodland plots in Benin during three consecutive rainy seasons. We measured the total biomass produced of each morphospecies, as well as environmental variables related to microclimate, host tree availability, and soil chemistry. Mushroom production and diversity were negatively correlated with soil nitrogen levels, and positively correlated with soil phosphorus levels. Although there were no clear differences in the fungal communities associated with the four host trees present in our plots, greater host tree diversity was associated with greater fungal diversity and productivity. Finally, <b>Paper IV</b> combined interviews with local people from four ethnic groups in five villages near the study sites from <b>Paper III</b> about their knowledge and preferences for edible mushrooms with DNA barcoding of specimens. Knowledge and preferences for different mushrooms varied between ethnic groups, but people living in a village where their ethnic group is a minority tended to absorb knowledge about mushrooms from the majority group in the village. Women over age 35 were the most knowledgeable about mushrooms, but the degree of gender difference varied between groups. We compiled a list of the most choice edible mushrooms in the area, which can inform the possibility for commercial trade in wild mushrooms to supplement the income of rural people and encourage sustainable forest management.</p>
<h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding</h1>
<p><a href="https://doi.org/10.3897/mycokeys.39.28109">doi:10.3897/mycokeys.39.28109</a></p>
article-journal<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p>
<h1>Habitat segregation of plate phenotypes in a rapidly expanding population of three-spined stickleback</h1>
<p><a href="https://doi.org/10.1002/ecs2.3561">doi:10.1002/ecs2.3561</a></p>
article-journal<p>Declines of large predatory fish due to overexploitation are restructuring food webs across the globe. It is now becoming evident that restoring these altered food webs requires addressing not only ecological processes, but evolutionary ones as well, because human-induced rapid evolution may in turn affect ecological dynamics. We studied the potential for niche differentiation between different plate armor phenotypes in a rapidly expanding population of a small prey fish, the three-spined stickleback (Gasterosteus aculeatus). In the central Baltic Sea, three-spined stickleback abundance has increased dramatically during the past decades. The increase in this typical mesopredator has restructured near-shore food webs, increased filamentous algal blooms, and threatens coastal biodiversity. Time-series data covering 22 years show that the increase coincides with a decline in the number of juvenile perch (Perca fluviatilis), the most abundant predator of stickleback along the coast. We investigated the distribution of different stickleback plate armor phenotypes depending on latitude, environmental conditions, predator and prey abundances, nutrients, and benthic production; and described the stomach content of the stickleback phenotypes using metabarcoding. We found two distinct lateral armor plate phenotypes of stickleback, incompletely and completely plated. The proportion of incompletely plated individuals increased with increasing benthic production and decreasing abundances of adult perch. Metabarcoding showed that the stomach content of the completely plated individuals more often contained invertebrate herbivores (amphipods) than the incompletely plated ones. Since armor plates are defense structures favored by natural selection in the presence of fish predators, the phenotype distribution suggests that a novel low-predation regime favors stickleback with less armor. Our results suggest that morphological differentiation of the three-spined stickleback has the potential to affect food web dynamics and influence the persistence and resilience of the stickleback take-over in the Baltic Sea.</p>
<h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient</h1>
<p><a href="https://doi.org/10.1111/mec.15561">doi:10.1111/mec.15561</a></p>
article-journal<p>Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates &lt; 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%–6.9%), when compared to low OC sediments (1.2%–2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode <i>Sabatieria</i> and foraminifera <i>Elphidium</i> in high OC, and <i>Cryptocaryon</i>‐like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p>
<h1>High-throughput sequencing of African <i>chikanda</i> cake highlights conservation challenges in orchids</h1>
<p><a href="https://doi.org/10.1007/s10531-017-1343-7">doi:10.1007/s10531-017-1343-7</a></p>
article-journal<p><i>Chikanda</i> is a traditional dish made with wild-harvested ground orchid tubers belonging to three orchidioid genera, <i>Disa</i>, <i>Satyrium</i> and <i>Habenaria</i>, all of which are CITES appendix II-listed. Identification of collected orchid tubers is very difficult and documentation of constituent species in prepared <i>chikanda</i> has hitherto been impossible. Here amplicon metabarcoding was used in samples of six prepared <i>chikanda</i> cakes to study genetic sequence diversity and species diversity in this product. Molecular operational taxonomic unit identification using similarity-matching reveals that species of all three genera were present in the <i>chikanda</i> samples studied. <i>Disa</i> was present in all of the samples, <i>Satyrium</i> in five out of six and <i>Habenaria</i> in one of the samples, as well as a number of other plants. The fact that each sample contained orchids and the presence of a wide variety of species from all genera in this traditional dish raise serious concerns about the sustainability of this trade and the future of wild orchid populations in the main harvest areas. This proof-of-concept study shows that Ion-Torrent PGM is a cost-effective scalable platform for metabarcoding using the relatively long nrITS1 and nrITS2 regions. Furthermore, nrITS metabarcoding can be successfully used for the detection of specific ingredients in a highly-processed food product at genus level, and this makes it a useful tool in the detection of possible conservation issues arising from commercialized trade or processed plant products.</p>
<h1>Historical biome distribution and recent human disturbance shape the diversity of arbuscular mycorrhizal fungi</h1>
<p><a href="https://doi.org/10.1111/nph.14695">doi:10.1111/nph.14695</a></p>
article-journal<p>The availability of global microbial diversity data, collected using standardized metabarcoding techniques, makes microorganisms promising models for investigating the role of regional and local factors in driving biodiversity. Here we modelled the global diversity of symbiotic arbuscular mycorrhizal (AM) fungi using currently available data on AM fungal molecular diversity (small subunit (SSU) ribosomal RNA (rRNA) gene sequences) in field samples. To differentiate between regional and local effects, we estimated species pools (sets of potentially suitable taxa) for each site, which are expected to reflect regional processes. We then calculated community completeness, an index showing the fraction of the species pool present, which is expected to reflect local processes. We found significant spatial variation, globally in species pool size, as well as in local and dark diversity (absent members of the species pool). Species pool size was larger close to areas containing tropical grasslands during the last glacial maximum, which are possible centres of diversification. Community completeness was greater in regions of high wilderness (remoteness from human disturbance). Local diversity was correlated with wilderness and current connectivity to mountain grasslands. Applying the species pool concept to symbiotic fungi facilitated a better understanding of how biodiversity can be jointly shaped by large-scale historical processes and recent human disturbance.</p>
<h1>Human access impacts biodiversity of microscopic animals in sandy beaches</h1>
<p><a href="https://doi.org/10.1038/s42003-020-0912-6">doi:10.1038/s42003-020-0912-6</a></p>
article-journal<p>Whereas most work to understand impacts of humans on biodiversity on coastal areas has focused on large, conspicuous organisms, we highlight effects of tourist access on the diversity of microscopic marine animals (meiofauna). We used a DNA metabarcoding approach with an iterative and phylogeny-based approach for the taxonomic assignment of meiofauna and relate diversity patterns to the numbers of tourists accessing sandy beaches on an otherwise un-impacted island National Park. Tourist frequentation, independently of differences in sediment granulometry, beach length, and other potential confounding factors, affected meiofaunal diversity in the shallow “swash” zone right at the mean water mark; the impacts declined with water depth (up to 2 m). The indicated negative effect on meiofauna may have a consequence on all the biota including the higher trophic levels. Thus, we claim that it is important to consider restricting access to beaches in touristic areas, in order to preserve biodiversity.</p>
<h1>Hållbar köttproduktion och köttkonsumtion i Dalarna</h1>
<p>No DOI</p>thesis<p>The release of green house gases and the environmental impacts humans have on the environment are huge and might cause irreversible changes to our earth. Meat is considered to have large impacts on the environment, especially beef meat and therefore there have lately been considerable discussions on the sustainable level of meat consumption and production. The main goal with this thesis is to study sustainable meat production and meat consumption in the county of Dalarna in Sweden. Indicators for sustainable meat production and meat consumption were developed and applied on three cases. Interviews were carried out with a municipality in the region that represents the consumer and three beef farmers that represents the producers. The three farmers in this study got many positive remarks regarding sustainability. There are still things to improve but on the whole many positive things were observed, e.g. use of natural grazing areas, maintenance of biodiversity, minor use of fertilizers and pesticides, use of land (pasture/leys) that otherwise would not be used for production of human food for direct consumption (e.g. grain) since the land is not suitable for that. To sum up the results from the consumer group, what can be seen is that the municipality is trying to improve their consumption patterns and that they have good intentions. They try for example to decrease their portions of meat and reduce their waste, they further try to implement days with only vegetarian food. On the other hand are the emissions from each pupil still too large if IPPC’s climate goal of a total of 1-2 ton CO<sub>2</sub>e emissions per person per year is to be reached. The consumers in Dalarna have the possibility, looking at things from a sustainable point of view, to choose a fine beef meat product (depending on how it is produced) from the neighbourhood. This is due to the fact that the producers in the study got many positive remarks regarding the sustainable indicators. It can therefore be considered positive if the municipality would support this production a thereby procure beef meat from the neighbourhood. The municipality must however change their consumer patterns and buy an amount of beef meat that can be considered sustainable.</p>
<h1>Identification of sawflies and horntails (Hymenoptera, ‘Symphyta’) through DNA barcodes: successes and caveats</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12614">doi:10.1111/1755-0998.12614</a></p>
article-journal<p>The ‘Symphyta’ is a paraphyletic assemblage at the base of the order Hymenoptera, comprising 14 families and about 8750 species. All have phytophagous larvae, except for the Orussidae, which are parasitoids. This study presents and evaluates the results of DNA barcoding of approximately 5360 specimens of ‘Symphyta’, mainly adults, and 4362 sequences covering 1037 species were deemed of suitable quality for inclusion in the analysis. All extant families are represented, except for the Anaxyelidae. The majority of species and specimens are from Europe, but approximately 38% of the species and 13% of the specimens are of non-European origin. The utility of barcoding for species identification and taxonomy of ‘Symphyta’ is discussed on the basis of examples from each of the included families. A significant level of cryptic species diversity was apparent in many groups. Other attractive applications include the identification of immature stages without the need to rear them, community analyses based on metabarcoding of bulk samples and association of the sexes of adults.</p>
<h1>Integrating DNA metabarcoding and biomonitoring data reveals trophic pathways of primary production in a flux-balance marine food web</h1>
<p>No DOI</p>manuscript<p>Estimating energy fluxes, or weight of trophic interactions, in food webs allows linking food web structure to properties of ecosystem functioning. Due to limitations in traditional methods, resolved food-web models are often binary or based on body size. Species-specific feeding traits and weights of interactions are typically not included, particularly at the base of the pelagic food web. Consequently, models are prone to underestimate the trophic diversity of species interactions.Here, we focus on trophic pathways of primary production in the Baltic Sea using a bioenergetic model that includes several trophic levels from primary producers to fish. For the first time, dietary DNA metabarcoding data of zooplankton are combined with accumulated biomass data from long-term pelagic monitoring and metabolic theory in a network model to describe energy pathways, including the diversity of planktonic organisms.We show that picocyanobacteria and filamentous cyanobacteria are the main contributors to secondary production in the Baltic Sea and that the latter experience high predation pressure from zooplankton. In contrast, the combination of high biomasses and low predation pressure on dinoflagellates and diatoms suggests that a significant fraction of the spring bloom is not utilized in the pelagic food web, explaining the high export of biological material to the sea seafloor.This study can be used to address ecosystem management objectives in the Baltic Sea under changing environmental conditions. Furthermore, the novel framework presented in this study, integrating DNA metabarcoding and biomonitoring data to assess energy fluxes, can be extended to dynamic food web modeling in other ecosystems.</p>
<h1>Integration of data for nowcasting of harmful algal blooms.</h1>
<p>No DOI</p>paper-conference<p>Harmful algal blooms (HABs) are a significant and potentially expanding problem around the world. Resource management and public health protection require sufficient information to reduce the impacts of HABs by response strategies and through warnings and advisories. To be effective, these programs can best be served by an integration of improved detection methods with both evolving monitoring systems and new communications capabilities. Data sets are typically collected from a variety of sources, these can be considered as several types: point data, such as water samples; transects, such as from shipboard continuous sampling; and synoptic, such as from satellite imagery. Generation of a field of the HAB distribution requires all of these sampling approaches. This means that the data sets need to be interpreted and analyzed with each other to create the field or distribution of the HAB. The HAB field is also a necessary input into models that forecast blooms. Several systems have developed strategies that demonstrate these approaches. These range from data sets collected at key sites, such as swimming beaches, to automated collection systems, to integration of interpreted satellite data. Improved data collection, particularly in speed and cost, will be one of the advances of the next few years. Methods to improve creation of the HAB field from the variety of data types will be necessary for routine nowcasting and forecasting of HABs.</p>
<h1>Internal quality control in forensic DNA analysis</h1>
<p><a href="https://doi.org/10.1007/s00769-013-0968-9">doi:10.1007/s00769-013-0968-9</a></p>
article-journal<p>The trail from initial evidence examination to a DNA profile reported to match a suspect is long and complex. The different nature and great variability in the biological and DNA evidence to be recovered and analyzed, add to this complexity. Internal quality controls play an important role in maintaining a high-quality performance in daily forensic biology and DNA profiling practice. In many cases are empirical rather than analytical approaches adopted. Obviously, despite the fact of being necessary, the internal quality controls performed still need to be kept rational at a limited, yet acceptable level. Quality control from a forensic biology and DNA profiling horizon has a wider context and does not only concern obvious fit-for-purpose verifications of analytical processes, chemicals, or reagents in daily routine practice. It also includes control on computerized laboratory management and expert systems, laboratory environmental DNA monitoring, and the use of elimination DNA databases. In addition, a structured recording and handling of non-conformances and “near failures” is essential. Proper management of the non-conformances supports continuous quality improvements by learning from the errors occurring in daily practice. High transparency of non-conformances is important not only for internal improvements, but also for the criminal justice system as well as to maintain public confidence and trust. Together the quality controls used aim at maintaining evidence and DNA sample integrity and to accomplish correct results and interpretations by verifying that methods used data transfers and interpretations made are correct and performed according to validated and accredited conditions.</p>
<h1>Lake sedimentary dna research on past terrestrial and aquatic biodiversity: Overview and recommendations</h1>
<p><a href="https://doi.org/10.3390/quat4010006">doi:10.3390/quat4010006</a></p>
review<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p>
<h1>Lake Sedimentary DNA Research on Past Terrestrial and Aquatic Biodiversity : Overview and Recommendations</h1>
<p><a href="https://doi.org/10.3390/quat4010006">doi:10.3390/quat4010006</a></p>
review<p>The use of lake sedimentary DNA to track the long-term changes in both terrestrial and aquatic biota is a rapidly advancing field in paleoecological research. Although largely applied nowadays, knowledge gaps remain in this field and there is therefore still research to be conducted to ensure the reliability of the sedimentary DNA signal. Building on the most recent literature and seven original case studies, we synthesize the state-of-the-art analytical procedures for effective sampling, extraction, amplification, quantification and/or generation of DNA inventories from sedimentary ancient DNA (sedaDNA) via high-throughput sequencing technologies. We provide recommendations based on current knowledge and best practises.</p>
<h1>Metabarcoding for use in Nordic routine aquatic biomonitoring : a validation study​</h1>
<p><a href="https://doi.org/10.6027/temanord2020-538">doi:10.6027/temanord2020-538</a></p>
book<p>Over 75% of all lakes and 40% of all rivers in the EU are found in the Nordic countries. Biomonitoring according to the EU Water Framework Directive (WFD) is adopted by all Nordic countries and forms an integral part of management efforts to preserve and restore the ecological quality of freshwaters and their ecosystem services. Organisms used in WFD biomonitoring are identified by experts; this is time consuming and error prone. Molecular identification methods could be used but have not been tested for monitoring. We compared molecular to expert identification for samples from 297 Nordic lakes and streams. The DNA-based results were highly similar to expert identifications. We suggest that Nordic or European efforts towards implementation and standardization of DNA-based methodology should be undertaken to swiftly ensure the use of this promising tool in future WFD monitoring. </p>
<h1>Metabarcoding of marine nematodes – evaluation of reference datasets used in tree-based taxonomy assignment approach</h1>
<p><a href="https://doi.org/10.3897/BDJ.4.e10021">doi:10.3897/BDJ.4.e10021</a></p>
article-journal<p>No Abstract</p><h1>Metabarcoding of marine nematodes – evaluation of similarity scores used in alignment-based taxonomy assignment approach</h1>
<p><a href="https://doi.org/10.3897/BDJ.4.e10647">doi:10.3897/BDJ.4.e10647</a></p>
article-journal<p>No Abstract</p><h1>Metabarcoding-based dietary analysis of hen harrier (Circus cyaneus) in Great Britain using buccal swabs from chicks</h1>
<p><a href="https://doi.org/10.1007/s10592-019-01215-y">doi:10.1007/s10592-019-01215-y</a></p>
article-journal<p>The hen harrier is a heavily persecuted bird of prey in Great Britain since its diet includes Red grouse, a game bird shot in driven and walked-up grouse shooting. Unlike walked-up shooting where shooters walk up moors and flush grouse with dogs, in driven grouse shooting the grouse are driven by beaters towards static shooters. Driven grouse moors (DGMs) are increasingly being managed to sustain high densities of Red grouse intensifying a long-standing conservation conflict between conservationists and grouse moor keepers. A metabarcoding approach was used with degenerate universal cytochrome oxidase I and cytochrome b primers along with hen harrier blocking primers. A novel sampling method was used to detect prey in buccal swab samples from chicks from broods across Great Britain from both managed and unmanaged moorland habitats. This resulted in detection of 62 species of prey across 51 broods with Meadow pipit, Red grouse, Wren, Skylark, and voles being most frequently detected. Frequency of occurrence data and species accumulation curves reveal high incidence of Red grouse and low prey species richness in the diet of hen harriers in DGMs but low incidence of Red grouse and high prey species richness in walked-up and unmanaged moors. Waders were only detected within walked-up and unmanaged moors and not within DGMs where they have been reported to occur at high densities. Regional species detected included endemic species such as the Orkney vole seen only in Orkney. This study represents the first metabarcoding-based dietary analysis in a raptor using buccal swabs.</p>
<h1>Microbial DNA Sequencing in Environmental Studies</h1>
<p>No DOI</p>thesis<p>The field of microbial ecology has just entered a new era of rapid technological development and generation of big data. The high-throughput sequencing techniques presently available provide an opportunity to extensively inventorize the blueprints of life. Now, millions of microbes of natural microbial communities can be studied simultaneously without prior cultivation. New species and new functions (genes) can be discovered just by mining sequencing data. However, there is still a tremendous number of microorganisms not yet examined, nor are the ecosystem functions these carry out. The modern genomic technologies can contribute to solve environmental problems and help us understand ecosystems, but to most efficiently do so, methods need to be continuously optimised.<b> </b>During my Ph. D. studies, I developed a method to survey eukaryotic microbial diversity with a higher accuracy, and applied various sequencing-based approaches in an attempt to answer questions of importance in environmental research and ecology. In <b>PAPER-I</b>, we developed a set of 18S rRNA gene PCR primers with high taxonomic coverage, meeting the requirements of currently popular sequencing technologies and matching the richness of 18S rRNA reference sequences accumulated so far. In <b>PAPER-II</b>, we conducted the first sequencing-based spatial survey on the combined eukaryotic and bacterial planktonic community in the Baltic Sea to uncover the relationship of microbial diversity and environmental conditions. Here, the 18S primers designed in <b>PAPER-I</b> and a pair of broad-coverage 16S primers were employed to target the rRNA genes of protists and bacterioplankton for amplicon sequencing. In <b>PAPER-III</b>, we integrated metagenomic, metabarcoding, and metatranscriptomic data in an effort to scrutinise the protein synthesis potential (i.e., activity) of microbes in the sediment at a depth of 460 m in the Baltic Sea and, thus, disclosing microbial diversity and their possible ecological functions within such an extreme environment. Lastly, in <b>PAPER-IV</b>, we compared the performance of <i>E. coli</i> culturing, high-throughput sequencing, and portable real-time sequencing in tracking wastewater contamination in an urban stormwater system. From the aspects of cost, mobility and accuracy, we evaluated the usage of sequencing-based approaches in civil engineering, and for the first time, validated the real-time sequencing device in use within water quality monitoring. In summary, these studies demonstrate how DNA sequencing of microbial communities can be applied in environmental monitoring and ecological research.</p>
<h1>Microbial metagenomics in the Baltic Sea : Recent advancements and prospects for environmental monitoring</h1>
<p><a href="https://doi.org/10.1007/s13280-015-0663-7">doi:10.1007/s13280-015-0663-7</a></p>
article-journal<p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed.</p>
<h1>Microbial metagenomics in the Baltic Sea : Recent advancements and prospects for environmental monitoring</h1>
<p><a href="https://doi.org/10.1007/s13280-015-0663-7">doi:10.1007/s13280-015-0663-7</a></p>
article-journal<p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed.</p>
<h1>MIDTAL (Microarrays for the Detection of Toxic Algae)</h1>
<p><a href="https://doi.org/10.11646/phytotaxa.127.1.19">doi:10.11646/phytotaxa.127.1.19</a></p>
article-journal<p>Microalgae in marine and brackish waters of Europe regularly cause harmful effects, considered from the human perspective, in that they cause economic damage to fisheries and tourism. Cyanobacteria cause similar problems in freshwaters. These episodes encompass a broad range of phenomena collectively referred to as harmful algal blooms (HABs). For adequate management of these phenomena, monitoring of microalgae is required. However, present day monitoring is time consuming and based on morphology as determined by light microscopy, which may be insufficient to give definitive species and toxin attribution. In the European Union (EU) FP7 project MIDTAL (microarrays for the detection of toxic algae), we will first target rapid species identification using rRNA genes. The variable regions of the rRNA genes can be used for probe design to recognize species or even strains. Second, a toxin based microarray will be developed that includes antibody reactions to specific toxins produced by these microalgae because even when cell numbers are low, toxins can be present and can accumulate in the shellfish. Microarrays are the state of the art technology in molecular biology for the processing of bulk samples for detection of target RNA/DNA sequence. Existing rRNA probes and antibodies for toxic algal species/strains and their toxins will be adapted and optimized for microarray use. The purpose of MIDTAL is to support the common fisheries policy and to aid the national monitoring agencies by providing new rapid tools for the identification of toxic algae and their toxins so they can comply with EU directive 91/1491/CEE to monitor for toxic algae, and reduce the need for the mouse bioassay.</p>
<h1>Miljöövervakning av växtplankton i Kattegatt och Östersjön med rDNA-barcoding och mikroskopi : En jämförelse av molekylärbiologisk metodik och mikroskopi</h1>
<p>No DOI</p>report<p>Växtplankton utgör en viktig del av den marina näringsväven. Därför pågår nationell och regional övervakning av växtplanktons utbredning i tid och rum, antal, biomassa och biodiversitet i haven runt Sverige. EU-direktiv och mellanstatliga konventioner influerar övervakningens utformning. Vissa algblomningar kan vara skadliga, därför finns ett visst fokus på alger som producerar gifter eller är skadliga på annat sätt. För att utföra övervakningen på ett kostnadseffektivt sätt behöver flera olika metoder kombineras. I den här rapporten presenteras resultat från en jämförelse av mikroskopbaserad analys av växtplanktonprover med resultat baserad på rDNA-metabarcoding. Provtagning skedde med hjälp av ett så kallat FerryBox-system i juli 2013 på arton platser längs en salthalts-gradient (ca 3-24 promille) från Bottenviken via Bottenhavet och Egentliga Östersjön till Kattegatt. Resultat från rDNA metabarcoding visade på en mycket större biodiversitet jämfört med data från mikroskopi. Totalt 89 olika organismer identifierades med mikroskop (Utermöhl-metoden). Bland dessa ingår samlingsgrupper som ”oidentifierade flagellater” och ”oidentifierade encelliga organismer”. Baserat på rDNA data noterades totalt ca 2000 olika organismer (exklusive heterotrofa bakterier) vilket innebär att mer än 95% av biodiversiteten förbisågs med Utermöhl-metoden. För prokaryoter (16S rDNA) noterades 36 olika Operational Taxonomic Units (OTUs) för cyanobakterier och för eukaryoter (18S) noterades 1860 olika OTU. Flera organismer som noterades med mikroskopi saknades helt i rDNA data. Det tyder på att referensdatabaser för 16S och 18S rDNA saknar sekvenser för vanligt förekommande arter i haven runt Sverige. Det kan också bero på att just sekvensdelen 18S är identisk för vissa arter/släkten, varför klassificeringen hamnade på en högre taxonomisk nivå. rDNA-barcoding ger inte samma typ av data som mikroskopi, det saknas bl.a. motsvarigheter till cellantal och biomassa. En alternativ molekylärbiologisk metod är så kallad kvantitativ PCR (qPCR). Med qPCR kan mängden DNA för enskilda organismer bestämmas, mängden DNA är relaterat till cellantal. Idag är det endast möjligt att utföra qPCR på ett fåtal organismer i ett prov.  Författarna föreslår att rDNA-barcoding av plankton införs i marina övervakningsprogram i Sverige som komplement till annan metodik. Förutom att rDNA-data ger högupplöst information om biologisk mångfald, så har rDNA barcoding ett lågt pris per prov om många prover analyseras och är inte beroende av mikroskopisters skicklighet att identifiera organismer. Införandet bör innehålla följande delmoment: 1. Pilotstudie – rDNA-data för ett helt år jämförs med mikroskopi och flödescytometri; 2: Sekvensering av vanligt förekommande arter som saknas i referensdatabaser; 3: Standardisering av provtagningsprotokoll; 4: Standardisering av metodik för sekvensering; 5: Utveckling av qPCR för utvalda skadliga arter; 6: Standardisering av dataflöden och jämförelser med internationella referensdatabaser; 7: Uppbyggnad av system för datahantering hos nationell datavärd och 8: Utveckling av bedömning av miljöstatus gällande t.ex. biodiversitet och invasiva arter baserat på rDNA data.</p>
<h1>Molecular- and pollen-based vegetation analysis in lake sediments from central Scandinavia</h1>
<p><a href="https://doi.org/10.1111/mec.12298">doi:10.1111/mec.12298</a></p>
article-journal<p>Plant and animal biodiversity can be studied by obtaining DNA directly from the environment. This new approach in combination with the use of generic barcoding primers (metabarcoding) has been suggested as complementary or alternative to traditional biodiversity monitoring in ancient soil sediments. However, the extent to which metabarcoding truly reflects plant composition remains unclear, as does its power to identify species with no pollen or macrofossil evidence. Here, we compared pollen-based and metabarcoding approaches to explore the Holocene plant composition around two lakes in central Scandinavia. At one site, we also compared barcoding results with those obtained in earlier studies with species-specific primers. The pollen analyses revealed a larger number of taxa (46), of which the majority (78%) was not identified by metabarcoding. The metabarcoding identified 14 taxa (MTUs), but allowed identification to a lower taxonomical level. The combined analyses identified 52 taxa. The barcoding primers may favour amplification of certain taxa, as they did not detect taxa previously identified with species-specific primers. Taphonomy and selectiveness of the primers are likely the major factors influencing these results. We conclude that metabarcoding from lake sediments provides a complementary, but not an alternative, tool to pollen analysis for investigating past flora. In the absence of other fossil evidence, metabarcoding gives a local and important signal from the vegetation, but the resulting assemblages show limited capacity to detect all taxa, regardless of their abundance around the lake. We suggest that metabarcoding is followed by pollen analysis and the use of species-specific primers to provide the most comprehensive signal from the environment.</p>
<h1>Mycobiomes of sympatric Amorphophallus albispathus (Araceae) and Camellia sinensis (Theaceae) – a case study reveals clear tissue preferences and differences in diversity and composition</h1>
<p><a href="https://doi.org/10.1007/s11557-018-1375-8">doi:10.1007/s11557-018-1375-8</a></p>
article-journal<p>Multiple biotic and abiotic parameters influence the dynamics of individual fungal species and entire communities. Major drivers for tropical plant endophytes are undoubtedly seasonality, local habitat conditions and biogeography. However, host specialization and tissue preferences also contribute to the structuring of endophytic mycobiomes. To elucidate such specializations and preferences, we sampled two commercially important, unrelated plant species, Amorphophallus albispathus and Camellia sinensis (tea plant) simultaneously at close proximity. The mycobiomes of different tissue types were assessed with high-throughput amplicon sequencing of the internal transcribed spacer DNA region. Both plants hosted different fungal communities and varied in α- and β-diversity, despite their neighboring occurrence. However, the fungal assemblages of Amorphophallus leaflets shared taxa with the mycobiomes of tea leaves, thereby suggesting common driving forces for leaf-inhabiting fungi irrespective of host plant identity. The mycobiome composition and diversity of tea leaves was clearly driven by leaf age. We suggest that the very youngest tea leaves are colonized by stochastic processes, while mycobiomes of old leaves are rather similar as the result of progressive succession. The biodiversity of fungi associated with A. albispathus was characterized by a large number of unclassified OTUs (at genus and species level) and by tissue-specific composition.This study is the first cultivation-independent high-throughput assessment of fungal biodiversity of an Amorphophallus species, and additionally expands the knowledge base on fungi associated with tea plants.</p>
<h1>NGS-based biodiversity and community structure analysis of meiofaunal eukaryotes in shell sand from Hållö island, Smögen, and soft mud from Gullmarn Fjord, Sweden</h1>
<p><a href="https://doi.org/10.3897/BDJ.5.e12731">doi:10.3897/BDJ.5.e12731</a></p>
article-journal<p>No Abstract</p><h1>Ocean urea fertilization for carbon credits poses high ecological risks</h1>
<p><a href="https://doi.org/10.1016/j.marpolbul.2008.03.010">doi:10.1016/j.marpolbul.2008.03.010</a></p>
article-journal<p>The proposed plan for enrichment of the Sulu Sea, Philippines, a region of rich marine biodiversity, with thousands of tonnes of urea in order to stimulate algal blooms and sequester carbon is flawed for multiple reasons. Urea is preferentially used as a nitrogen source by some cyanobacteria and dinoflagellates, many of which are neutrally or positively buoyant. Biological pumps to the deep sea are classically leaky, and the inefficient burial of new biomass makes the estimation of a net loss of carbon from the atmosphere questionable at best. The potential for growth of toxic dinoflagellates is also high, as many grow well on urea and some even increase their toxicity when grown on urea. Many toxic dinoflagellates form cysts which can settle to the sediment and germinate in subsequent years, forming new blooms even without further fertilization. If large-scale blooms do occur, it is likely that they will contribute to hypoxia in the bottom waters upon decomposition. Lastly, urea production requires fossil fuel usage, further limiting the potential for net carbon sequestration. The environmental and economic impacts are potentially great and need to be rigorously assessed.</p>
<h1>Perspectives from Ten Years of Protist Studies by High-Throughput Metabarcoding</h1>
<p><a href="https://doi.org/10.1111/jeu.12813">doi:10.1111/jeu.12813</a></p>
article-journal<p>During the last decade, high-throughput metabarcoding became routine for analyzing protistan diversity and distributions in nature. Amid a multitude of exciting findings, scientists have also identified and addressed technical and biological limitations, although problems still exist for inference of meaningful taxonomic and ecological knowledge based on shortDNAsequences. Given the extensive use of this approach, it is critical to settle our understanding on its strengths and weaknesses and to synthesize up-to-date methodological and conceptual trends. This article summarizes key scientific and technical findings, and identifies current and future directions in protist research that uses metabarcoding.</p>
<h1>Phytoplankton settling quality has a subtle but significant effect on sediment microeukaryotic and bacterial communities</h1>
<p><a href="https://doi.org/10.1038/s41598-021-03303-x">doi:10.1038/s41598-021-03303-x</a></p>
article-journal<p>In coastal aphotic sediments, organic matter (OM) input from phytoplankton is the primary food resource for benthic organisms. Current observations from temperate ecosystems like the Baltic Sea report a decline in spring bloom diatoms, while summer cyanobacteria blooms are becoming more frequent and intense. These climate-driven changes in phytoplankton communities may in turn have important consequences for benthic biodiversity and ecosystem functions, but such questions are not yet sufficiently explored experimentally. Here, in a 4-week experiment, we investigated the response of microeukaryotic and bacterial communities to different types of OM inputs comprising five ratios of two common phytoplankton species in the Baltic Sea, the diatom <i>Skeletonema marinoi</i> and filamentous cyanobacterium <i>Nodularia spumigena</i>. Metabarcoding analyses on 16S and 18S ribosomal RNA (rRNA) at the experiment termination revealed subtle but significant changes in diversity and community composition of microeukaryotes in response to settling OM quality. Sediment bacteria were less affected, although we observed a clear effect on denitrification gene expression (<i>nirS</i> and <i>nosZ</i>), which was positively correlated with increasing proportions of cyanobacteria. Altogether, these results suggest that future changes in OM input to the seafloor may have important effects on both the composition and function of microbenthic communities.</p>
<h1>Phytoplankton settling quality has a subtle but significant effect on sediment microeukaryotic and bacterial communities</h1>
<p><a href="https://doi.org/10.1038/s41598-021-03303-x">doi:10.1038/s41598-021-03303-x</a></p>
article-journal<p>In coastal aphotic sediments, organic matter (OM) input from phytoplankton is the primary food resource for benthic organisms. Current observations from temperate ecosystems like the Baltic Sea report a decline in spring bloom diatoms, while summer cyanobacteria blooms are becoming more frequent and intense. These climate-driven changes in phytoplankton communities may in turn have important consequences for benthic biodiversity and ecosystem functions, but such questions are not yet sufficiently explored experimentally. Here, in a 4-week experiment, we investigated the response of microeukaryotic and bacterial communities to different types of OM inputs comprising five ratios of two common phytoplankton species in the Baltic Sea, the diatom Skeletonema marinoi and filamentous cyanobacterium Nodularia spumigena. Metabarcoding analyses on 16S and 18S ribosomal RNA (rRNA) at the experiment termination revealed subtle but significant changes in diversity and community composition of microeukaryotes in response to settling OM quality. Sediment bacteria were less affected, although we observed a clear effect on denitrification gene expression (nirS and nosZ), which was positively correlated with increasing proportions of cyanobacteria. Altogether, these results suggest that future changes in OM input to the seafloor may have important effects on both the composition and function of microbenthic communities.</p>
<h1>Prestationsmätningssystem: Styrning, Planering och Uppföljning i tjänsteföretag : En utvärderingsstudie av IKEA Hemmas PMS</h1>
<p>No DOI</p>thesis<p><b>Examensarbete, Civilekonomprogrammet, Ekonomihögskolan vid Linnéuniversitetet, Controller, 4FE12E, VT 2014</b><b>Författare: </b>Madeleine Olsson och Julia Henriksson<b></b><b>Examinator och Handledare: </b>Fredrik Karlsson<b></b><b> </b><b>Titel: </b>Prestationsmätningssystem: Planering, Styrning och Uppföljning i tjänsteföretag – en utvärdering av IKEA Hemmas PMS<b></b><b> </b><b>Nyckelord: </b>Prestationsmätningssystem, KPI, nyckeltal, utvärdering, utvärderingsstudie, tjänsteföretag, informationsflöde<b> </b><b>Bakgrund:</b> Det är vitalt att organisationer löpande förvaltar sitt prestationsmätningssystem (PMS) i syfte att säkerställa att rätt saker mäts. Om inte måtten följs upp kontinuerligt och utvärderas, kan prestationsmätning bli en kostsam process som inte leder till några förbättringar för organisationen. Önskemål från IKEA Hemma var att utvärdera deras PMS.<b> </b><b>Syfte: </b>Syftet med denna studie är att utvärdera funktionaliteten i IKEA Hemmas prestationsmätningssystem. Systemets uppbyggnad samt effekterna av PMS:et analyseras för att identifiera prestationsmätningssystemets utvecklingsområden. Ett vidare syfte är att föreslå förändringar som IKEA Hemma kan vidta för att utveckla sitt PMS.<b> </b><b>Metod: </b>Det angrepssätt som tillämpats i studien är utvärderingsstudie. Det primära syftet med en utvärderingsstudie är att tillhandahålla kunskap huruvida de mål som satts upp uppnås eller inte samt även att undersöka vilka effekter detta projekt haft. Intervjuer samt enkäter har legat till grund för empiriinsamlingen.<b></b><b> </b><b>Slutsats: </b>KPI:erna är utformade i syfte att vägleda och motivera handläggarna att tillfredsställa de kunder som kontaktar IKEA Hemma. Systemet är allmänt accepterat av medarbetarna och målen uppnås till stor del på gruppnivå. Förbättringsområden har identifierats, där bland annat nya KPI samt mätning bytts ut för att säkerställa att måttens resultat är påverkbara.  </p>
<h1>Shifts among Eukaryota, Bacteria, and Archaea define the vertical organization of a lake sediment</h1>
<p><a href="https://doi.org/10.1186/s40168-017-0255-9">doi:10.1186/s40168-017-0255-9</a></p>
article-journal<p>Background: Lake sediments harbor diverse microbial communities that cycle carbon and nutrients while being constantly colonized and potentially buried by organic matter sinking from the water column. The interaction of activity and burial remained largely unexplored in aquatic sediments. We aimed to relate taxonomic composition to sediment biogeochemical parameters, test whether community turnover with depth resulted from taxonomic replacement or from richness effects, and to provide a basic model for the vertical community structure in sediments.Methods: We analyzed four replicate sediment cores taken from 30-m depth in oligo-mesotrophic Lake Stechlin in northern Germany. Each 30-cm core spanned ca. 170 years of sediment accumulation according to 137Cs dating and was sectioned into layers 1–4 cm thick. We examined a full suite of biogeochemical parameters and used DNA metabarcoding to examine community composition of microbial Archaea, Bacteria, and Eukaryota.Results: Community β-diversity indicated nearly complete turnover within the uppermost 30 cm. We observed a pronounced shift from Eukaryota- and Bacteria-dominated upper layers (&lt;5 cm) to Bacteria-dominated intermediate layers (5–14 cm) and to deep layers (&gt;14 cm) dominated by enigmatic Archaea that typically occur in deep-sea sediments. Taxonomic replacement was the prevalent mechanism in structuring the community composition and was linked to parameters indicative of microbial activity (e.g., CO2 and CH4 concentration, bacterial protein production). Richness loss played a lesser role but was linked to conservative parameters (e.g., C, N, P) indicative of past conditions.Conclusions: By including all three domains, we were able to directly link the exponential decay of eukaryotes with the active sediment microbial community. The dominance of Archaea in deeper layers confirms earlier findings from marine systems and establishes freshwater sediments as a potential low-energy environment, similar to deep sea sediments. We propose a general model of sediment structure and function based on microbial characteristics and burial processes. An upper “replacement horizon” is dominated by rapid taxonomic turnover with depth, high microbial activity, and biotic interactions. A lower “depauperate horizon” is characterized by low taxonomic richness, more stable “low-energy” conditions, and a dominance of enigmatic Archaea.</p>
<h1>Soil fungal communities of ectomycorrhizal dominated woodlands across West Africa</h1>
<p><a href="https://doi.org/10.3897/mycokeys.81.66249">doi:10.3897/mycokeys.81.66249</a></p>
article-journal<p>Forests and woodlands in the West African Guineo-Sudanian transition zone contain many tree species that form symbiotic interactions with ectomycorrhizal (ECM) fungi. These fungi facilitate plant growth by increasing nutrient and water uptake and include many fruiting body-forming fungi, including some edible mushrooms. Despite their importance for ecosystem functioning and anthropogenic use, diversity and distribution of ECM fungi is severely under-documented in West Africa. We conducted a broad regional sampling across five West African countries using soil eDNA to characterize the ECM as well as the total soil fungal community in gallery forests and savanna woodlands dominated by ECM host tree species. We subsequently sequenced the entire ITS region and much of the LSU region to infer a phylogeny for all detected soil fungal species. Utilizing a long read sequencing approach allows for higher taxonomic resolution by using the full ITS region, while the highly conserved LSU gene allows for a more accurate higher-level assignment of species hypotheses, including species without ITS-based taxonomy assignments. We detect no overall difference in species richness between gallery forests and woodlands. However, additional gallery forest plots and more samples per plot would have been needed to firmly conclude this pattern. Based on both abundance and richness, species from the families Russulaceae and Inocybaceae dominate the ECM fungal soil communities across both vegetation types. The community structure of both total soil fungi and ECM fungi was significantly influenced by vegetation types and showed strong correlation within plots. However, we found no significant difference in fungal community structure between samples collected adjacent to different host tree species within each plot. We conclude that within plots, the fungal community is structured more by the overall ECM host plant community than by the species of the individual host tree that each sample was collected from.</p>
<h1>Systematic comparison of the relative accuracy of vegetation surveys and soil DNA metabarcoding : Assessing plant biodiversity at different spatial scales</h1>
<p>No DOI</p>thesis<p>Analysis of soil-derived DNA has been shown to minimize problems seen during traditional vegetation surveys, e.g. by matching the eDNA to a reference database for taxonomic identification rather than relying solely on taxonomic expertise. However, it has been debated to what extent and how accurately eDNA acts as a proxy for biodiversity. The reliability on eDNA and the awareness on influencing factors is also important for palaeoenvironmental reconstructions where above-ground vegetation cannot be used. This study aimed to investigate how well modern soil-derived eDNA reflects the contemporary vascular vegetation communities in a subarctic environment, and if the efficiency of the taxonomic identification differed between spatial scales. Near-surface soil samples at altitudinal gradients along numerous transects were collected in combination with vegetation surveys. The eDNA was amplified through metabarcoding using the P6 loop region of the chloroplast trnL intron, followed by a high-throughput sequencing. No difference in the number of identified taxa between eDNA and vegetation survey was seen at landscape scale. In contrast, the number of identified taxa was consistently higher in the vegetation survey at smaller spatial scales. The efficiency of identified taxa per scale remained stable for the vegetation survey, whereas for eDNA, a decreasing trend was seen. This study highlights the variations on taxa identification between both methods and which factors might cause it. Combining the methods allows for a more precise modern biodiversity estimation, as well as to minimize wrongful conclusions. This allows for a more accurate palaeoenvironmental reconstructions, which in turn can improve future species conservation decisions.</p>
<h1>Taxonomy assignment approach determines the efficiency of identification of OTUs in marine nematodes</h1>
<p><a href="https://doi.org/10.1098/rsos.170315">doi:10.1098/rsos.170315</a></p>
article-journal<p>No Abstract</p><h1>The Taxon Hypothesis Paradigm - On the Unambiguous Detection and Communication of Taxa</h1>
<p><a href="https://doi.org/10.3390/microorganisms8121910">doi:10.3390/microorganisms8121910</a></p>
article-journal<p>Here, we describe the taxon hypothesis (TH) paradigm, which covers the construction, identification, and communication of taxa as datasets. Defining taxa as datasets of individuals and their traits will make taxon identification and most importantly communication of taxa precise and reproducible. This will allow datasets with standardized and atomized traits to be used digitally in identification pipelines and communicated through persistent identifiers. Such datasets are particularly useful in the context of formally undescribed or even physically undiscovered species if data such as sequences from samples of environmental DNA (eDNA) are available. Implementing the TH paradigm will to some extent remove the impediment to hastily discover and formally describe all extant species in that the TH paradigm allows discovery and communication of new species and other taxa also in the absence of formal descriptions. The TH datasets can be connected to a taxonomic backbone providing access to the vast information associated with the tree of life. In parallel to the description of the TH paradigm, we demonstrate how it is implemented in the UNITE digital taxon communication system. UNITE TH datasets include rich data on individuals and their rDNA ITS sequences. These datasets are equipped with digital object identifiers (DOI) that serve to fix their identity in our communication. All datasets are also connected to a GBIF taxonomic backbone. Researchers processing their eDNA samples using UNITE datasets will, thus, be able to publish their findings as taxon occurrences in the GBIF data portal. UNITE species hypothesis (species level THs) datasets are increasingly utilized in taxon identification pipelines and even formally undescribed species can be identified and communicated by using UNITE. The TH paradigm seeks to achieve unambiguous, unique, and traceable communication of taxa and their properties at any level of the tree of life. It offers a rapid way to discover and communicate undescribed species in identification pipelines and data portals before they are lost to the sixth mass extinction.</p>
<h1>The wolf reference genome sequence (Canis lupus lupus) and its implications for Canis spp. population genomics</h1>
<p><a href="https://doi.org/10.1186/s12864-017-3883-3">doi:10.1186/s12864-017-3883-3</a></p>
article-journal<p>No Abstract</p><h1>Three years after the Chrysochromulina polylepis bloom in Scandinavian waters in 1988 : some conclusions of recent research and monitoring</h1>
<p>No DOI</p>paper-conference<p>No Abstract</p><h1>Tools for Evaluating Energy Efficiency of Steel Production : Lessons from Sweden and Europe</h1>
<p>No DOI</p>thesis<p>The European Union faces challenges related to climate change, security of energy supply, and competitiveness of European industries. Energy efficiency indicators are required for monitoring and controlling the effectiveness of policies such as the recently endorsed Energy Efficiency Directive. This thesis aims at assessing whether traditionally used energy efficiency indicators capture the development of energy efficiency in the iron and steel sector. The study is based on results from two statistical methods: a top-down, i.e. <i>Malmquist productivity index,</i> and a bottom-up, i.e. <i>partial least squares regression</i>.The <i>specific energy consumption </i>(the indicator representing the sector within the <i>Odyssee energy efficiency index</i>) was scrutinised together with associated indicators based on economic production using the aforementioned statistical methods. The results demonstrated the <i>specific energy consumption</i> does not capture the characteristics of the value chain of steel products. Therefore, it is not sufficient for capturing the energy efficiency of iron and steel industries. Previous studies suggest using indicators based on economic production (e.g. <i>value added</i>) since they represent the value chain to larger degree. However, the value creation process of companies belonging to larger international groups cannot be estimated reliably. Furthermore, the trends of both types of indicators tend to be highly influenced by structural changes, veiling the actual efficiency development.Energy use statistics published by international organisations were also compared for the Swedish case. The results demonstrated that international organisations use different methodologies for allocating energy use statistics between consumption and transformation sectors. The method has significant implications on the trends observed, if based on openly available statistics.This thesis complements previous research by reviewing implications of traditional energy efficiency indicators based on company data, national statistics or openly available statistics and contributes with insights essential for future efforts towards improving energy efficiency indicators for the steel industry.</p>
<h1>Tree species identity and diversity drive fungal richness and community composition along an elevational gradient in a Mediterranean ecosystem</h1>
<p><a href="https://doi.org/10.1007/s00572-017-0806-8">doi:10.1007/s00572-017-0806-8</a></p>
article-journal<p>Ecological and taxonomic knowledge is important for conservation and utilization of biodiversity. Biodiversity and ecology of fungi in Mediterranean ecosystems is poorly understood. Here, we examined the diversity and spatial distribution of fungi along an elevational gradient in a Mediterranean ecosystem, using DNA metabarcoding. This study provides novel information about diversity of all ecological and taxonomic groups of fungi along an elevational gradient in a Mediterranean ecosystem. Our analyses revealed that among all biotic and abiotic variables tested, host species identity is the main driver of the fungal richness and fungal community composition. Fungal richness was strongly associated with tree richness and peaked in Quercus-dominated habitats and Cistus-dominated habitats. The highest taxonomic richness of ectomycorrhizal fungi was observed under Quercus ilex, whereas the highest taxonomic richness of saprotrophs was found under Pinus. Our results suggest that the effect of plant diversity on fungal richness and community composition may override that of abiotic variables across environmental gradients.</p>
<h1>Using airborne eDNA to study ecosystem dynamics</h1>
<p>No DOI</p>thesis<p>In this era of global biodiversity crisis, the need to monitor ecological communities over space and time is more pressing than ever to effectively direct biodiversity conservation and management efforts. To understand the natural dynamics of an ecosystem, and the impact of anthropogenic activities related to environmental and climate change, long time series are needed to accurately link such processes to ecosystem change. This thesis uses a unique resource of archived air filters collected in Sweden originally intended for radioactive particle measurements to reconstruct historical eDNA abundance in the most extensive time-series of airborne eDNA abundance to date - spanning across four decades. By using metabarcoding and metagenomic analysis, it is evident that airborne eDNA from a very large diversity of species is present in air, representing all major branches of the tree of life, including bacteria, fungi, plants, metazoans, and viruses, from a wide range of terrestrial and aquatic sources. These have seasonal as well as long term trends that in part can be explained by temporal variation in climate and regional differences. The data generated in this thesis comprise an extensive resource for analysis of trends related to climate and environmental change and will also allow deeper studies on phenology, phenological change, functional genomics, and potentially antibiotic resistance. The results presented here show the potential of using airborne eDNA to monitor species in the local ecosystem over time and the methods provides an efficient tool for assessment of broad scale biodiversity, in a non-invasive and cost-effective way.</p>
<h1>Using eDNA to improve environmental monitoring for water bodies effected by hydropower in Sweden</h1>
<p>No DOI</p>thesis<p>The aim of this report is to contribute to the base of knowledge on environmental monitoring by increasing understanding of how eDNA, electrofishing and sampling fishing may be used to examine fish biodiversity. It also aims at understanding if fish indexes developed within the Water Framework Directive reflect biodiversity, as well as the potential of eDNA data to serve as input to these indexes. This was done by using three different approaches.Firstly, in order to establish which of the methods eDNA, electrofishing and sampling fishing is more suitable to measure the different dimensions of biodiversity (species richness, species evenness and genetic diversity), a literature review comparing the different methods was carried out. It was found that eDNA yields a more detailed results for species richness, electrofishing yields better results for species evenness and sampling fishing is outperformed by eDNA and electrofishing alike. Both electrofishing and sampling fishing may collect data for genetic diversity analysis, however electrofishing outperforms sampling fishing with regards to amount of species caught, making electrofishing a more suitable data collection method.Secondly, in order to gain insight on practical usage of eDNA, a case study of Spjutmo (Dalarna county) was reviewed. It was established that eDNA generated more detailed information of species richness in the case of Spjutmo (as compared to electrofishing). The relative abundance data generated by the eDNA study might be seen as a measure of species evenness. However, electrofishing yielded data which may serve as input to species evenness indices. To the best knowledge of the author, none of the methods generated data on genetic diversity in this specific case.Officials from the energy company Fortum and the county board of Dalarna were also interviewed in order to get insight on what potential they see for eDNA to contribute to environmental monitoring. Both officials point at the ability to estimate abundance as a desired feature, hence a better understanding of what the relative abundance results indicates is wanted. The two interviews indicate that this understanding is an important feature to develop in order to make metabarcoding studies effective in current environmental monitoring.Thirdly, in order to understand if fish indexes developed within the Water Framework Directive reflect biodiversity, a literature review was performed. It was found that, all but one of the compared indexes incorporates or somewhat incorporates species richness. However, only five indexes are indicative or somewhat indicative of species richness. Species evenness is incorporated or somewhat incorporated by two indexes, which are also indicative or somewhat indicative of species evenness. None of the indexes incorporate or indicate genetic diversity.Within the third literature review, the potential of eDNA data to serve as input to current fish-based indexes developed within the Water Framework Directive, was studied. It was found that eDNA data may serve as input to only one index in its present form. However, five indexes also use proportional information (e.g. proportion of tolerant species), which possibly could be provided by eDNA data. The index where usage of eDNA data is currently possible uses presence-absence information.</p>
<h1>Utvärdering av ny övervakning av främmande arter : Metodjämförelse mellan traditionell och DNA-baserad identifiering</h1>
<p>No DOI</p>report<p>Denna rapport redovisar resultat från en studie där vi undersökt artförekomst och speciellt förekomst av invasiva främmande arter i kustmiljö. Syftet har varit att utvärdera konventionell undersökningsmetodik för att identifiera arter genom taxonomisk artbestämning genetisk identifiering genom DNAmetabarkodning, i hamnmiljöer.Genom att övervaka miljön med nya genetiska metoder får vi möjligheter att upptäcka spår av organismers förekomst, något som annars är svårt med sortering av organismer från vatten-/planktonprov, bottenhugg eller fiskeredskap. Utvecklingen av DNA-tekniker går fort och det ser lovande ut att tillämpa dessa i jakten på invasiva främmande arter och patogener som är skadliga för den biologiska mångfalden, människors och djurs hälsa samt orsakar stora samhällskostnader, alltså organismer som vi anser är invasiva och främmande.Det finns idag inga miljöövervakningsprogram som baseras på DNA. Samtidigt blir det allt svårare att hitta experter, så kallade taxonomer som kan identifiera arter och som oftast är specialiserade på en eller några artgrupper. Dessutom kommer många arter hit som larver eller juveniler som kanske inte alls liknar det vuxna djuret, eller algen, och därmed är svåra att artbestämma. Detta innebär att många arter kan komma hit utan att upptäckas förrän de är etablerade, och då har vi vanligtvis nått en situation då den invasiva främmande arten i fråga etablerat sig i en ny miljö, och därmed inte går att åtgärda.Såväl EU:s förordning om invasiva främmande arter som havsmiljödirektivet ställer krav på nationell övervakning av främmande arter. Och inom havsmiljöförvaltningen finns även behoven för att fullfölja arbetet med främmande arter inom Helcoms och Ospar. För att snabbt kunna utrota eller begränsa nyetableringar av invasiva främmande arter behöver vi utveckla metoder som är effektiva på att tidigt upptäcka nya introduktioner. Övervakning med hjälp av DNA-baserade metoder som metabarkodning är en lämplig väg att gå. Eftersom det är nya metoder för övervakning så bör även nya indikatorer tas fram som lämpar sig för genetiska metoder. Genetiska metoder bör ses som ett komplement till de traditionella – inte minst under en övergångsfas och om så möjligt ersätta traditionella.</p>
<h1>What do we learn from cultures in the omics age? High-throughput sequencing and cultivation of leaf-inhabiting endophytes from beech (Fagus sylvatica L.) revealed complementary community composition but similar correlations with local habitat conditions</h1>
<p><a href="https://doi.org/10.3897/mycokeys.20.11265">doi:10.3897/mycokeys.20.11265</a></p>
article-journal<p>Comparative simultaneous studies of environmental high-throughput sequencing (HTS) and cultivation of plant-associated fungi have rarely been conducted in the past years. For the present contribution, HTS and extinction culturing were applied for the same leaf samples of European beech (Fagus sylvatica) in order to trace both “real” environmental drivers as well as method-dependent signals of the observed mycobiomes. Both approaches resulted in non-overlapping community composition and pronounced differences in taxonomic classification and trophic stages. However, both methods revealed similar correlations of the fungal communities with local environmental conditions. Our results indicate undeniable advantages of HTS over cultivation in terms of revealing a good representation of the major functional guilds, rare taxa and biodiversity signals of leaf-inhabiting fungi. On the other hand our results demonstrate that the immense body of literature about cultivable endophytic fungi can and should be used for the interpretation of community signals and environmental correlations obtained from HTS studies and that cultivation studies should be continued at the highest standards, e.g. when sequencing facilities are not available or if such surveys are expanded into functional aspects with experiments on living isolates.</p>
<h1>Why are so many Northern European aquatic invertebrates missing in red-listing and how can we improve assessments for those?</h1>
<p>No DOI</p>paper-conference<p>The biodiversity crisis is advancing rapidly. One tool to measure extinction risk is the Red List of Threatened Species which follows the IUCN evaluation criteria (International Union for Conservation of Nature). Many aquatic invertebrates in Northern Europe are completely missing a red listing process and are evaluated as Data Deficient (DD) or Not Evaluated (NE). In our project, we focus on marine crustaceans and freshwater molluscs (Bivalvia). A systematic survey of more than 440 crustacean and 44 molluscan species in 12 Northern European countries shows that while many freshwater bivalve molluscs and marine crustaceans have existing molecular barcodes as well as digital occurrence records in databases (e.g. in GBIF, the Global Biodiversity Information Facility), there exists no evaluation process or regular monitoring for those species and their population status. With such a high level of non-evaluation of species status, species action plans (for single species or multi-taxon approaches) are far away from reality.In general, traditional monitoring methods based on observational surveys are known to be inefficient, costly and time consuming. e-DNA allows us to detect species with a high level of sensitivity as long as those assays are well validated. Molecular occurrence records can be used to detect rare species and to collect population information. In our Swedish project, we are metabarcoding sediment and plankton samples using metazoan and taxon-specific primers to detect threatened aquatic species. During 2019 and 2020, we collected samples at 15 localities in two marine protected areas for marine crustaceans and at 15 different localities for freshwater molluscs at the Swedish west coast. At each location plankton, sediment and traditional aquatic monitoring samples were taken. The idea is to compare how the methods perform in finding rare species, which could improve the data for those groups so they can be evaluated in the next round of red listing (2025) in Sweden. During the entire project, there is an on-going dialogue with stakeholders and experts from the Swedish Species Information Centre, responsible for the red listing process in the country.</p>
<h1>Åtgärdsprogram för temporära sötvatten : med fokus på organismer i naturbetesmarker, hällkar och alvarmiljöer</h1>
<p>No DOI</p>report<p>Det här åtgärdsprogrammet fokuserar på temporära sötvatten och de organismer som lever i sådana miljöer. Med temporära vatten menas små stillastående sötvattensamlingar som utmärker sig genom att periodvis torka ut. Dessa saknar tillflöde i form av rinnande vatten och är således helt grundvatten- eller nederbördsförsörjda. Temporära vatten bildas i sänkor i landskapet eller i fördjupningar i berghällar. Det är en heterogen grupp av naturtyper, dit allt från små regnpölar till större vattensamlingar i betesmarker räknas in. Den främsta anledningen till att temporära vatten, och många av deras organismer drabbats negativt, är att de har försvunnit vid de omfattande utdikningar som gjorts under många år till fördel för åkermark.</p>
