<h1>Short- and long-read metabarcoding of the eukaryotic rRNA operon: evaluation of primers and comparison to shotgun metagenomics sequencing.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13623">doi:10.1111/1755-0998.13623</a></p>
['PacBio long-read sequencing', 'Primer design', 'marine plankton', 'microbial eukaryotes', 'rRNA operon']<p>High-throughput sequencing for analysis of microbial diversity has evolved vastly over the last decade. Currently the go-to method for studying microbial eukaryotes is short-read metabarcoding of variable regions of the 18S rRNA gene with <500 bp amplicons. However, there is a growing interest in applying long-read sequencing of amplicons covering the rRNA operon for improving taxonomic resolution. For both methods, the choice of primers is crucial. It determines if community members are covered, if they can be identified at a satisfactory taxonomic level, and if the obtained community profile is representative. Here, we designed new primers targeting 18S and 28S rRNA based on 177,934 and 21,072 database sequences, respectively. The primers were evaluated in silico along with published primers on reference sequence databases and marine metagenomics datasets. We further evaluated a subset of the primers for short- and long-read sequencing on environmental samples in vitro and compared the obtained community profile with primer-unbiased metagenomic sequencing. Of the short-read pairs, a new V6-V8 pair and the V4_Balzano pair used with a simplified PCR protocol provided good results in silico and in vitro. Fewer differences were observed between the long-read primer pairs. The long-read amplicons and ITS1 alone provided higher taxonomic resolution than V4. Together, our results represent a reference and guide for selection of robust primers for research on and environmental monitoring of microbial eukaryotes.</p>
<h1>DNA sequence and taxonomic gap analyses to quantify the coverage of aquatic cyanobacteria and eukaryotic microalgae in reference databases: Results of a survey in the Alpine region.</h1>
<p><a href="https://doi.org/S0048-9697(22)02268-9">doi:S0048-9697(22)02268-9</a></p>
['Amplicon sequence variants', 'Gap analysis', 'Interreg Eco-AlpsWater', 'Metabarcoding', 'Taxonomic coverage', 'Taxonomic databases']<p>The taxonomic identification of organisms based on the amplification of specific genetic markers (metabarcoding) implicitly requires adequate discriminatory information and taxonomic coverage of environmental DNA sequences in taxonomic databases. These requirements were quantitatively examined by comparing the determination of cyanobacteria and microalgae obtained by metabarcoding and light microscopy. We used planktic and biofilm samples collected in 37 lakes and 22 rivers across the Alpine region. We focused on two of the most used and best represented genetic markers in the reference databases, namely the 16S rRNA and 18S rRNA genes. A sequence gap analysis using blastn showed that, in the identity range of 99-100%, approximately 30% (plankton) and 60% (biofilm) of the sequences did not find any close counterpart in the reference databases (NCBI GenBank). Similarly, a taxonomic gap analysis showed that approximately 50% of the cyanobacterial and eukaryotic microalgal species identified by light microscopy were not represented in the reference databases. In both cases, the magnitude of the gaps differed between the major taxonomic groups. Even considering the species determined under the microscope and represented in the reference databases, 22% and 26% were still not included in the results obtained by the blastn at percentage levels of identity ≥95% and ≥97%, respectively. The main causes were the absence of matching sequences due to amplification and/or sequencing failure and potential misidentification in the microscopy step. Our results quantitatively demonstrated that in metabarcoding the main obstacles in the classification of 16S rRNA and 18S rRNA sequences and interpretation of high-throughput sequencing biomonitoring data were due to the existence of important gaps in the taxonomic completeness of the reference databases and the short length of reads. The study focused on the Alpine region, but the extent of the gaps could be much greater in other less investigated geographic areas.</p>
<h1>Best practices in metabarcoding of fungi: From experimental design to results.</h1>
<p><a href="https://doi.org/10.1111/Mec.16460">doi:10.1111/Mec.16460</a></p>
['biodiversity', 'bioinformatics', 'community ecology', 'experiment setup', 'molecular identification', 'statistical analyses']<p>The development of high-throughput sequencing (HTS) technologies has greatly improved our capacity to identify fungi and unveil their ecological roles across a variety of ecosystems. Here we provide an overview of current best practices in metabarcoding analysis of fungal communities, from experimental design through molecular and computational analyses. By reanalysing published data sets, we demonstrate that operational taxonomic units (OTUs) outperform amplified sequence variants (ASVs) in recovering fungal diversity, a finding that is particularly evident for long markers. Additionally, analysis of the full-length ITS region allows more accurate taxonomic placement of fungi and other eukaryotes compared to the ITS2 subregion. Finally, we show that specific methods for compositional data analyses provide more reliable estimates of shifts in community structure. We conclude that metabarcoding analyses of fungi are especially promising for integrating fungi into the full microbiome and broader ecosystem functioning context, recovery of novel fungal lineages and ancient organisms as well as barcoding of old specimens including type material.</p>
<h1>Detection and population genomics of sea turtle species via noninvasive environmental DNA analysis of nesting beach sand tracks and oceanic water.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13617">doi:10.1111/1755-0998.13617</a></p>
['ChHV5', 'endangered species', 'environmental DNA (eDNA)', 'pathogens', 'population genetics/genomics', 'population monitoring', 'sea turtles']<p>Elusive aquatic wildlife, such as endangered sea turtles, are difficult to monitor and conserve. As novel molecular and genetic technologies develop, it is possible to adapt and optimize them for wildlife conservation. One such technology is environmental (e)DNA - the detection of DNA shed from organisms into their surrounding environments. We developed species-specific green (Chelonia mydas) and loggerhead (Caretta caretta) sea turtle probe-based qPCR assays, which can detect and quantify sea turtle eDNA in controlled (captive tank water and sand samples) and free ranging (oceanic water samples and nesting beach sand) settings. eDNA detection complemented traditional in-water sea turtle monitoring by enabling detection even when turtles were not visually observed. Furthermore, we report that high throughput shotgun sequencing of eDNA sand samples enabled sea turtle population genetic studies and pathogen monitoring, demonstrating that noninvasive eDNA techniques are viable and efficient alternatives to biological sampling (e.g., biopsies and blood draws). Genetic information was obtained from sand many hours after nesting events, without having to observe or interact with the target individual. This greatly reduces the sampling stress experienced by nesting mothers and emerging hatchlings, and avoids sacrificing viable eggs for genetic analysis. The detection of pathogens from sand indicates significant potential for increased wildlife disease monitoring capacity and viral variant surveillance. Together, these results demonstrate the potential of eDNA approaches to ultimately help understand and conserve threatened species such as sea turtles.</p>
<h1>LANDMark: an ensemble approach to the supervised selection of biomarkers in high-throughput sequencing data.</h1>
<p><a href="https://doi.org/10.1186/S12859-022-04631-Z">doi:10.1186/S12859-022-04631-Z</a></p>
['Biomarker selection', 'Biomonitoring', 'Ecological assessment', 'Machine learning', 'Metabarcoding', 'Metagenomics']<p>Identification of biomarkers, which are measurable characteristics of biological datasets, can be challenging. Although amplicon sequence variants (ASVs) can be considered potential biomarkers, identifying important ASVs in high-throughput sequencing datasets is challenging. Noise, algorithmic failures to account for specific distributional properties, and feature interactions can complicate the discovery of ASV biomarkers. In addition, these issues can impact the replicability of various models and elevate false-discovery rates. Contemporary machine learning approaches can be leveraged to address these issues. Ensembles of decision trees are particularly effective at classifying the types of data commonly generated in high-throughput sequencing (HTS) studies due to their robustness when the number of features in the training data is orders of magnitude larger than the number of samples. In addition, when combined with appropriate model introspection algorithms, machine learning algorithms can also be used to discover and select potential biomarkers. However, the construction of these models could introduce various biases which potentially obfuscate feature discovery.</p>
<h1>[Application and progress in high-throughput sequencing technology for meat adulteration detection].</h1>
<p><a href="https://doi.org/10.13345/J.Cjb.210113">doi:10.13345/J.Cjb.210113</a></p>
['DNA barcoding', 'food safety', 'high-throughput sequencing', 'meat adulteration', 'metabarcoding technology', 'species identification']<p>Adulteration in meat products is a widespread issue that could lead to serious threats to public health and religious violations. Technology that offers rapid, sensitive, accurate and reliable detection of meat species is the key to an effectual monitoring and control against meat adulteration. In recent years, high-throughput sequencing-based DNA metabarcoding technology has developed rapidly. With the characteristics of being high-throughput, highly precise and high-speed, this technology can simultaneously identify multiple species in complex samples, thus offering pronounced advantages in the surveillance of adulteration in meat and meat products. Starting with an introduction of the major developments in the high-throughput sequencing technology in the past two decades, this review provides an overview of the technical characteristics and research methods of DNA metabarcoding, summarizes the application of DNA metabarcoding technology in meat adulteration detection over the last few years, discusses the challenges of using DNA metabarcoding technology in the detection of meat adulteration, and provides future prospects on the development of this technology.</p>
<h1>Disentangling bias for non-destructive insect metabarcoding.</h1>
<p><a href="https://doi.org/10.7717/Peerj.12981">doi:10.7717/Peerj.12981</a></p>
['Barcoding', 'Biodiversity', 'Biosecurity', 'Entomology', 'High throughput sequencing', 'Primer bias']<p>A fast and reliable method for obtaining a species-level identification is a fundamental requirement for a wide range of activities, from plant protection and invasive species management to biodiversity assessments and ecological studies. For insects, novel molecular techniques such as DNA metabarcoding have emerged as a rapid alternative to traditional morphological identification, reducing the dependence on limited taxonomic experts. Until recently, molecular techniques have required a destructive DNA extraction, precluding the possibility of preserving voucher specimens for future studies, or species descriptions. Here we paired insect metabarcoding with two recent non-destructive DNA extraction protocols, to obtain a rapid and high-throughput taxonomic identification of diverse insect taxa while retaining a physical voucher specimen. The aim of this work was to explore how non-destructive extraction protocols impact the semi-quantitative nature of metabarcoding, which alongside species presence/absence also provides a quantitative, but biased, representation of their relative abundances. By using a series of mock communities representing each stage of a typical metabarcoding workflow we were able to determine how different morphological (<i>i.e</i>., insect biomass and exoskeleton hardness) and molecular traits (<i>i.e</i>., primer mismatch and amplicon GC%), interact with different protocol steps to introduce quantitative bias into non-destructive metabarcoding results. We discuss the relevance of taxonomic bias to metabarcoding identification of insects and potential approaches to account for it.</p>
<h1>Next generation restoration metrics: Using soil eDNA bacterial community data to measure trajectories towards rehabilitation targets.</h1>
<p><a href="https://doi.org/S0301-4797(22)00321-8">doi:S0301-4797(22)00321-8</a></p>
['Mine closure assessment', 'Rehabilitation trajectory', 'Restoration genomics', 'Soil microbiota', 'Spatial autocorrelation', 'eDNA']<p>In post-mining rehabilitation, successful mine closure planning requires specific, measurable, achievable, relevant and time-bound (SMART) completion criteria, such as returning ecological communities to match a target level of similarity to reference sites. Soil microbiota are fundamentally linked to the restoration of degraded ecosystems, helping to underpin ecological functions and plant communities. High-throughput sequencing of soil eDNA to characterise these communities offers promise to help monitor and predict ecological progress towards reference states. Here we demonstrate a novel methodology for monitoring and evaluating ecological restoration using three long-term (>25 year) case study post-mining rehabilitation soil eDNA-based bacterial community datasets. Specifically, we developed rehabilitation trajectory assessments based on similarity to reference data from restoration chronosequence datasets. Recognising that numerous alternative options for microbiota data processing have potential to influence these assessments, we comprehensively examined the influence of standard versus compositional data analyses, different ecological distance measures, sequence grouping approaches, eliminating rare taxa, and the potential for excessive spatial autocorrelation to impact on results. Our approach reduces the complexity of information that often overwhelms ecologically-relevant patterns in microbiota studies, and enables prediction of recovery time, with explicit inclusion of uncertainty in assessments. We offer a step change in the development of quantitative microbiota-based SMART metrics for measuring rehabilitation success. Our approach may also have wider applications where restorative processes facilitate the shift of microbiota towards reference states.</p>
<h1>Exploring the microbial community inhabiting the phosphogypsum stacks of Huelva (SW SPAIN) by a high throughput 16S/18S rDNA sequencing approach.</h1>
<p><a href="https://doi.org/S0166-445X(22)00030-3">doi:S0166-445X(22)00030-3</a></p>
['16S/18S rRNA', 'Acid mine drainage', 'High throughput sequencing', 'Metabarcoding', 'Phosphogypsum', 'Pollutants', 'Radionuclides']<p>Around 100 Mt of phosphogypsum (PG) have been deposited in large stacks on the salt marshes of the Tinto River estuary in Huelva (SW Spain), covering about 1000 ha. These stacks contain extremely acidic water (pH < 2) with high concentrations of pollutants which can cause emissions into their surroundings, generating important environmental concerns. Despite many chemical, geological or hydrological studies have been conducted to characterize the PG stacks of Huelva, the microbial community inhabiting this extreme environment remains unexplored. Using a 16S/18S-rRNA-high throughput sequencing approach, we have uncovered the main taxonomic groups able to live in the acidic metal-contaminated water, which is in direct contact with the PG, demonstrating for the first time the existence of a huge diversity of microbial species in these extreme conditions. In addition, the physicochemical characteristics of the water sampled have been analyzed. These studies have revealed that the most abundant bacteria found in two different leachate samples of the PG stacks belong to the genera Acidiphilium, Pseudomonas, Leptosprillum, Acidithrix, or Acidithiobacillus, typically found in acid mine drainage (AMD) environments, which in total represent around 50% of the total bacterial community. Biodiversity of eukaryotes in PG water is lower than that of prokaryotes, especially in the water collected from the perimeter channel that surrounds the PG stacks, where the pH reaches a value of 1.5 and the activity concentrations exceed 300 Bq L<sup>-1</sup> for <sup>238</sup>U or 20 Bq L<sup>-1</sup> for <sup>210</sup>Po, values which are from four to five orders of magnitude higher than those usually found in unperturbed surface waters. Even so, an unexpected diversity of algae, fungi, and ciliates have been found in the PG stacks of Huelva, where chlorophyte microalgae and basidiomycetes fungi are the most abundant eukaryotes. Additional bioinformatics tools have been used to perform a functional analysis and predict the most common metabolic pathways in the PG microbiota. The obtained data indicate that the extreme conditions of these PG stacks hide an unexpected microbial diversity, which can play an important role in the dynamics of the contaminating compounds of the PG and provide new strains with unique biotechnological applications.</p>
<h1>Metabarcoding reveals massive species diversity of Diptera in a subtropical ecosystem.</h1>
<p><a href="https://doi.org/10.1002/Ece3.8535">doi:10.1002/Ece3.8535</a></p>
['COI gene', 'Malaise trap', 'community structure', 'high‐throughput sequencing', 'two‐winged flies']<p>Diptera is often considered to be the richest insect group due to its great species diversity and broad ecological versatility. However, data on dipteran diversity from subtropical ecosystems have hitherto been scarce, due to the lack of studies conducted at an appropriate large scale. We investigated the diversity and composition of Diptera communities on Tianmu Mountain, Zhejiang, China, using DNA metabarcoding technology, and evaluated their dynamic responses to the effects of slope aspect, season, and altitudinal zone. A total of 5,092 operational taxonomic units (OTUs) were discovered and tentatively assigned to 72 dipteran families, including 2 family records new for China and 30 family records new for the locality. Cecidomyiidae, Sciaridae, and Phoridae were the predominant families, representing 53.6% of total OTUs, while 52 families include >95% unidentified and presumed undescribed species. We found that the community structure of Diptera was significantly affected by aspect, seasonality (month) and elevation, with richer diversity harbored in north-facing than south-facing slopes, and seasonality a more profound driver of community structure and diversity than elevation. Overall, massive species diversity of Diptera communities was discovered in this subtropical ecosystem of east China. The huge diversity of potentially undescribed species only revealed by metabarcoding now requires more detailed taxonomic study, as a step toward an evolutionary integration that accumulates information on species' geographic ranges, ecological traits, functional roles, and species interactions, and thus places the local communities in the context of the growing knowledge base of global biodiversity and its response to environmental change.</p>
<h1>Ichthyoplankton DNA metabarcoding: Challenges and perspectives.</h1>
<p><a href="https://doi.org/10.1111/Mec.16387">doi:10.1111/Mec.16387</a></p>
['DNA barcode', 'biomonitoring 2.0', 'environmental genomics', 'high-throughput DNA sequencing', 'neotropical fish']<p>DNA metabarcoding has been widely used to access and monitor species. However, several challenges remain open for its mainstream application in ecological studies, particularly when dealing with a quantitative approach. In a From the Cover article in this issue of Molecular Ecology, Mariac et al. (2021) report species-level ichthyoplankton dynamics for 97 fish species from two Amazon river basins using a clever quantitative metabarcoding approach employing a probe capture method. They clearly show that most species spawned during the floods, although ~20% also spawned mainly during the receding period and some other year-round, but interestingly, species from the same genus reproduced in distinct periods (i.e., inverse phenology). Opportunistically, Mariac et al. (2021) reported that during an intense hydrological anomaly, several species had a sharp reduction in spawning activity, demonstrating a quick response to environmental cues. This is an interesting result since the speed at which fish species can react to environmental changes, during the spawning period, is largely unknown. Thus, this study brings remarkable insights into basic life history information that is imperative for proposing strategies that could lead to a realistic framework for sustainable fisheries management practices and conservation, fundamental for an understudied and threatened realm, such as the Amazon River basin.</p>
<h1>Metabarcoding of feces and intestinal contents to determine carnivorous diets in red-crowned cranes in eastern Hokkaido, Japan.</h1>
<p><a href="https://doi.org/10.1292/Jvms.21-0516">doi:10.1292/Jvms.21-0516</a></p>
['Grus japonensis', 'Japan', 'cytochrome c oxidase subunit I', 'high throughput sequencing', 'scatology']<p>The red-crowned crane Grus japonensis in Hokkaido, Japan forms a closed population as a residence that is independent of the mainland population. Based on observations of a limited number of individuals as well as cranes in captivity, red-crowned cranes are omnivores and eat fish, worms, insects and plants in their own territories except in winter, when they are fed with dent corn that is supplied in eastern Hokkaido. DNA metabarcoding based on high throughput sequencing was carried out using universal primer sets for cytochrome oxidase subunit I gene. Feces from 27 chicks collected in June and July in the period from 2016 to 2018 and intestinal contents from 33 adult and subadult cranes that were found dead almost throughout year in 2006-2013 in the field in eastern Hokkaido were used. Although compositions varied considerably in the cranes, both insects and fish were found in adults and subadults to the same extents, while insects were predominant in chicks. Both insects and fish were detected in all seasons for adults and subadults. Horse flies, scarab beetles and weevils accounted for the most of the insects regardless of the life stage. Dace, stickleback, flatfish and sculpin were the major fish species in adults, while chicks ate almost only stickleback. The results provide the first comprehensive data on carnivorous diets in wild red-crowned cranes in eastern Hokkaido as basis for conservation of red-crowned cranes, for which the life style and area continue to change.</p>
<h1>Airborne environmental DNA for terrestrial vertebrate community monitoring.</h1>
<p><a href="https://doi.org/S0960-9822(21)01690-0">doi:S0960-9822(21)01690-0</a></p>
['air filtration', 'air samplers', 'amplicon sequencing', 'bioaerosol', 'biodiversity', 'biomonitoring', 'conservation', 'eDNA', 'high-throughput sequencing', 'metabarcoding']<p>Biodiversity monitoring at the community scale is a critical element of assessing and studying species distributions, ecology, diversity, and movements, and it is key to understanding and tracking environmental and anthropogenic effects on natural ecosystems.<sup>1-4</sup> Vertebrates in terrestrial ecosystems are experiencing extinctions and declines in both population numbers and sizes due to increasing threats from human activities and environmental change.<sup>5-8</sup> Terrestrial vertebrate monitoring using existing methods is generally costly and laborious, and although environmental DNA (eDNA) is becoming the tool of choice to assess biodiversity, few sample types effectively capture terrestrial vertebrate diversity. We hypothesized that eDNA captured from air could allow straightforward collection and characterization of terrestrial vertebrate communities. We filtered air at three localities in the Copenhagen Zoo: a stable, outside between the outdoor enclosures, and in the Rainforest House. Through metabarcoding of airborne eDNA, we detected 49 vertebrate species spanning 26 orders and 37 families: 30 mammal, 13 bird, 4 fish, 1 amphibian, and 1 reptile species. These spanned animals kept at the zoo, species occurring in the zoo surroundings, and species used as feed in the zoo. The detected species comprise a range of taxonomic orders and families, sizes, behaviors, and abundances. We found shorter distance to the air sampling device and higher animal biomass to increase the probability of detection. We hereby show that airborne eDNA can offer a fundamentally new way of studying and monitoring terrestrial communities.</p>
<h1>Soil Fungal Diversity of the Aguarongo Andean Forest (Ecuador).</h1>
<p><a href="https://doi.org/1289">doi:1289</a></p>
['Andes', 'environmental DNA', 'forest ecosystems tropical mycobiota vulnerable species', 'fungal biodiversity', 'metabarcoding', 'natural reserve']<p>Fungi represent an essential component of ecosystems, functioning as decomposers and biotrophs, and they are one of the most diverse groups of Eukarya. In the tropics, many species are unknown. In this work, high-throughput DNA sequencing was used to discover the biodiversity of soil fungi in the Aguarongo forest reserve, one of the richest biodiversity hotspots in Ecuador. The rDNA metabarcoding analysis revealed the presence of seven phyla: <i>Ascomycota</i>, <i>Basidiomycota</i>, <i>Mortierellomycota</i>, <i>Mucoromycota</i>, <i>Glomeromycota</i>, <i>Chytridiomycota</i>, and <i>Monoblepharomycota</i>. A total of 440 identified species were recorded. They mainly belonged to <i>Ascomycota</i> (263) and <i>Basidiomycota</i> (127). In <i>Mortierellomycota</i>, 12 species were recorded, among which <i>Podila verticillata</i> is extremely frequent and represents the dominant species in the entire mycobiota of Aguarongo. The present research provides the first account of the entire soil mycobiota in the Aguarongo forest, where many fungal species exist that have strong application potential in agriculture, bioremediation, chemical, and the food industry. The Aguarongo forest hides a huge number of unknown fungal species that could be assessed, and its protection is of the utmost importance.</p>
<h1>Zooplankton diversity monitoring strategy for the urban coastal region using metabarcoding analysis.</h1>
<p><a href="https://doi.org/10.1038/S41598-021-03656-3">doi:10.1038/S41598-021-03656-3</a></p>
<p>No keywords</p><p>Marine ecosystems in urban coastal areas are exposed to many risks due to human activity. Thus, long-term and continuous monitoring of zooplankton diversity is necessary. High-throughput DNA metabarcoding has gained recognition as an efficient and highly sensitive approach to accurately describing the species diversity of marine zooplankton assemblages. In this study, we collected 30 zooplankton samples at about 2-week intervals for 1 year. Zooplankton diversity showing a typical four season pattern. Of the "total" and "common" zooplankton, we assigned 267 and 64 taxa. The cluster structure and seasonal diversity pattern were rough when only the "common" zooplankton was used. Our study examined how to maximize the benefits of metabarcoding for monitoring zooplankton diversity in urban coastal areas. The results suggest that to take full advantage of metabarcoding when monitoring a zooplankton community, it is necessary to carefully investigate potential ecosystem threats (non-indigenous species) through sufficient curation rather than disregarding low-abundance operational taxonomic units.</p>
<h1>A high-throughput amplicon sequencing approach for population-wide species diversity and composition survey.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13576">doi:10.1111/1755-0998.13576</a></p>
['\nBemisia\n', 'African cassava whitefly', 'Aphelinidae', 'metabarcoding', 'multiprimer molecular diagnostics', 'parasitoids']<p>Management of agricultural pests requires an understanding of pest species diversity, their interactions with beneficial insects and spatial-temporal patterns of pest abundance. Invasive and agriculturally important insect pests can build up very high populations, especially in cropping landscapes. Traditionally, sampling effort for species identification involves small sample sizes and is labour intensive. Here, we describe a multiprimer high throughput sequencing (HTS) metabarcoding method and associated analytical workflow for a rapid, intensive, high-volume survey of pest species compositions. We demonstrate our method using the taxonomically challenging Bemisia pest cryptic species complex as examples. The whiteflies Bemisia including the"tabaci" species are agriculturally important capable of vectoring diverse plant viruses that cause diseases and crop losses. Our multiprimer metabarcoding HTS amplicon approach simultaneously process high volumes of whitefly individuals, with efficiency to detect rare (i.e., 1%) test-species, while our improved whitefly primers for metabarcoding also detected beneficial hymenopteran parasitoid species from whitefly nymphs. Field-testing our redesigned Bemisia metabarcoding primer sets across the Tanzania, Uganda and Malawi cassava cultivation landscapes, we identified the sub-Saharan Africa 1 Bemisia putative species as the dominant pest species, with other cryptic Bemisia species being detected at various abundances. We also provide evidence that Bemisia species compositions can be affected by host crops and sampling techniques that target either nymphs or adults. Our multiprimer HTS metabarcoding method incorporated two overlapping amplicons of 472 bp and 518 bp that spanned the entire 657 bp 3' barcoding region for Bemisia, and is particularly suitable to molecular diagnostic surveys of this highly cryptic insect pest species complex that also typically exhibited high population densities in heavy crop infestation episodes. Our approach can be adopted to understand species biodiversity across landscapes, with broad implications for improving transboundary biosecurity preparedness, thus contributing to molecular ecological knowledge and the development of control strategies for high-density, cryptic, pest-species complexes.</p>
<h1>Analysis of Serological Biomarkers of SARS-CoV-2 Infection in Convalescent Samples From Severe, Moderate and Mild COVID-19 Cases.</h1>
<p><a href="https://doi.org/10.3389/Fimmu.2021.748291">doi:10.3389/Fimmu.2021.748291</a></p>
['Antibodies', 'COVID-19', 'COVID-19 immune response', 'Correlates of Protection', 'SARS-CoV-2', 'Serological biomarkers', 'WHO International Standard']<p>Precision monitoring of antibody responses during the COVID-19 pandemic is increasingly important during large scale vaccine rollout and rise in prevalence of <i>Severe Acute Respiratory Syndrome-related Coronavirus-2</i> (SARS-CoV-2) variants of concern (VOC). Equally important is defining Correlates of Protection (CoP) for SARS-CoV-2 infection and COVID-19 disease. Data from epidemiological studies and vaccine trials identified virus neutralising antibodies (Nab) and SARS-CoV-2 antigen-specific (notably RBD and S) binding antibodies as candidate CoP. In this study, we used the World Health Organisation (WHO) international standard to benchmark neutralising antibody responses and a large panel of binding antibody assays to compare convalescent sera obtained from: a) COVID-19 patients; b) SARS-CoV-2 seropositive healthcare workers (HCW) and c) seronegative HCW. The ultimate aim of this study is to identify biomarkers of humoral immunity that could be used to differentiate severe from mild or asymptomatic SARS-CoV-2 infections. Some of these biomarkers could be used to define CoP in further serological studies using samples from vaccination breakthrough and/or re-infection cases. Whenever suitable, the antibody levels of the samples studied were expressed in International Units (IU) for virus neutralisation assays or in Binding Antibody Units (BAU) for ELISA tests. In this work we used commercial and non-commercial antibody binding assays; a lateral flow test for detection of SARS-CoV-2-specific IgG/IgM; a high throughput multiplexed particle flow cytometry assay for SARS-CoV-2 Spike (S), Nucleocapsid (N) and Receptor Binding Domain (RBD) proteins); a multiplex antigen semi-automated immuno-blotting assay measuring IgM, IgA and IgG; a pseudotyped microneutralisation test (pMN) and an electroporation-dependent neutralisation assay (EDNA). Our results indicate that overall, severe COVID-19 patients showed statistically significantly higher levels of SARS-CoV-2-specific neutralising antibodies (average 1029 IU/ml) than those observed in seropositive HCW with mild or asymptomatic infections (379 IU/ml) and that clinical severity scoring, based on WHO guidelines was tightly correlated with neutralisation and RBD/S antibodies. In addition, there was a positive correlation between severity, N-antibody assays and intracellular virus neutralisation.</p>
<h1>Benefits of merging paired-end reads before pre-processing environmental metagenomics data.</h1>
<p><a href="https://doi.org/S1874-7787(21)00080-5">doi:S1874-7787(21)00080-5</a></p>
['Environmental DNA', 'High throughput sequencing', 'Metagenomics', 'Quality processing', 'Sequence alignment', 'Trimmomatic']<p>High throughput sequencing of environmental DNA has applications in biodiversity monitoring, taxa abundance estimation, understanding the dynamics of community ecology, and marine species studies and conservation. Environmental DNA, especially, marine eDNA, has a fast degradation rate. Aside from the good quality reads, the data could have a significant number of reads that fall slightly below the default PHRED quality threshold of 30 on sequencing. For quality control, trimming methods are employed, which generally precede the merging of the read pairs. However, in the case of eDNA, a significant percentage of reads within the acceptable quality score range are also dropped.</p>
<h1>Metabarcoding analysis of trophic sources and linkages in the plankton community of the Kuroshio and neighboring waters.</h1>
<p><a href="https://doi.org/10.1038/S41598-021-02083-8">doi:10.1038/S41598-021-02083-8</a></p>
<p>No keywords</p><p>Trophic sources and pathways supporting early life stages are crucial for survival of forage fishes recruiting around the oligotrophic and unproductive Kuroshio. However, information is limited for the Kuroshio planktonic food web and its trophodynamics because of its high biodiversity. Here, we explore trophic sources and linkages in the Kuroshio plankton community using metabarcoding analysis of gut-content DNA for 22 mesozooplankton groups. The major prey was dinoflagellates and calanoids for omnivorous groups, and calanoids and gelatinous organisms for carnivorous groups. Larvaceans and hydrozoans were the most frequently appeared prey for both omnivores and carnivores, whereas they were minor constituents of the available prey in water samples. Although calanoids overlapped as major prey items for both omnivores and carnivores because they were the most available, contributions from phytoplankton and gelatinous prey differed among taxonomic groups. Further analysis of the metabarcoding data showed that in addition to omnivorous copepods like calanoids, gelatinous groups like larvaceans and hydrozoans were important hubs in the planktonic food web with their multiple trophic linkages to many components. These findings suggest that gelatinous organisms are important as supplementary prey and provide evidence of niche segregation on trophic sources among mesozooplankton groups in the Kuroshio.</p>
<h1>High-Throughput Sequencing of Environmental DNA as a Tool for Monitoring Eukaryotic Communities and Potential Pathogens in a Coastal Upwelling Ecosystem.</h1>
<p><a href="https://doi.org/10.3389/Fvets.2021.765606">doi:10.3389/Fvets.2021.765606</a></p>
['eDNA', 'environment', 'eukaryote', 'high-throughput sequencing', 'metabarcoding', 'pathogens']<p>The marine environment includes diverse microeukaryotic organisms that play important functional roles in the ecosystem. With molecular approaches, eukaryotic taxonomy has been improved, complementing classical analysis. In this study, DNA metabarcoding was performed to describe putative pathogenic eukaryotic microorganisms in sediment and marine water fractions collected in Galicia (NW Spain) from 2016 to 2018. The composition of eukaryotic communities was distinct between sediment and water fractions. Protists were the most diverse group, with the clade TSAR (Stramenopiles, Alveolata, Rhizaria, and Telonemida) as the primary representative organisms in the environment. Harmful algae and invasive species were frequently detected. Potential pathogens, invasive pathogenic organisms as well as the causative agents of harmful phytoplanktonic blooms were identified in this marine ecosystem. Most of the identified pathogens have a crucial impact on the aquacultural sector or affect to relevant species in the marine ecosystem, such as diatoms. Moreover, pathogens with medical and veterinary importance worldwide were also found, as well as pathogens that affect diatoms. The evaluation of the health of a marine ecosystem that directly affects the aquacultural sector with a zoonotic concern was performed with the metabarcoding assay.</p>
<h1>High-throughput sequencing of faeces provides evidence for dispersal of parasites and pathogens by migratory waterbirds.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13548">doi:10.1111/1755-0998.13548</a></p>
['Coccidia', 'anti-metazoan', 'endozoochory', 'metabarcoding', 'metagenomics', 'trichomonads']<p>Examination of faecal material has demonstrated how a broad range of organisms are distributed by bird movements. Such research has largely focused on dispersal of plant seeds by frugivores and of freshwater organisms by waterbirds. However, with few exceptions (e.g. avian influenza, Ebola virus), there is a dearth of evidence for transport of parasites and pathogens. High-throughput sequencing methods now provide a powerful means of addressing this knowledge gap by elucidating faecal contents in unprecedented detail. We collected faeces excreted by a range of migratory waterbirds in south-west Spain and pooled faecal DNA to create libraries reflective of feeding behavior. We created sets of libraries using high-throughput metagenomic and amplicon sequencing. For the latter we employed two sets of primers to broadly target the V4 region of the 18S rRNA gene (one set amplifying the region across all eukaryotes, the other excluding amplification of metazoans). Libraries revealed a wide diversity of eukaryotes, including parasites of the faecal producers themselves, parasites of food items, or those incidentally ingested. We also detected novel microbial eukaryotic taxa and found that parasite assemblage profiles were relatively distinct. Comparing the performance of the methods used supports their joint use for future studies of diversity and abundance. Because viable stages of many parasites are likely to be present in faeces, our results suggest significant levels of bird-mediated dispersal of parasites (both from avian and other hosts). Our methods revealed much hidden biodiversity, and allowed identification of the individuals who produced the faecal samples to species level, facilitating the study of interaction networks.</p>
<h1>Preservation methods of honey bee-collected pollen are not a source of bias in ITS2 metabarcoding.</h1>
<p><a href="https://doi.org/10.1007/S10661-021-09563-4">doi:10.1007/S10661-021-09563-4</a></p>
['Citizen science', 'DNA metabarcoding', 'Pollen DNA barcoding', 'Preservation bias', 'Silica gel preservation']<p>Pollen metabarcoding is emerging as a powerful tool for ecological research and offers unprecedented scale in citizen science projects for environmental monitoring via honey bees. Biases in metabarcoding can be introduced at any stage of sample processing and preservation is at the forefront of the pipeline. While in metabarcoding studies pollen has been preserved at - 20 °C (FRZ), this is not the best method for citizen scientists. Herein, we compared this method with ethanol (EtOH), silica gel (SG) and room temperature (RT) for preservation of pollen collected from hives in Austria and Denmark. After ~ 4 months of storage, DNAs were extracted with a food kit, and their quality and concentration measured. Most DNA extracts exhibited 260/280 absorbance ratios close to the optimal 1.8, with RT samples from Austria performing slightly worse than FRZ and SG samples (P < 0.027). Statistical differences were also detected for DNA concentration, with EtOH samples producing lower yields than RT and FRZ samples in both countries and SG in Austria (P < 0.042). Yet, qualitative and quantitative assessments of floral composition obtained using high-throughput sequencing with the ITS2 barcode gave non-significant effects of preservation methods on richness, relative abundance and Shannon diversity, in both countries. While freezing and ethanol are commonly employed for archiving tissue for molecular applications, desiccation is cheaper and easier to use regarding both storage and transportation. Since SG is less dependent on ambient humidity and less prone to contamination than RT, we recommend SG for preserving pollen for metabarcoding. SG is straightforward for laymen to use and hence robust for widespread application in citizen science studies.</p>
<h1>Can moderate heavy metal soil contaminations due to cement production influence the surrounding soil bacterial communities?</h1>
<p><a href="https://doi.org/10.1007/S10646-021-02494-3">doi:10.1007/S10646-021-02494-3</a></p>
['Contamination', 'Environmental genomics', 'Heavy metals', 'Metabarcoding', 'Soil']<p>Events of soil contamination by heavy metals are mostly related to human activities that release these metals into the environment as emissions or effluents. Among the industrial activities related to heavy metal pollution, cement production plants are considered one of the most common sources. In this work we applied the High-throughput sequencing approach called 16 S rDNA metabarcoding to perform the taxonomic characterization of the prokaryotic communities of the soil surrounding three cement plants as well as two areas outside the influence of the cement plants that represented agricultural production environments free of heavy metal contamination (control areas). We applied the environmental genomics approaches known as "structural community metrics" (α- and β-diversity metrics) and "functional community metrics" (PICRUSt2 approach) to verify whether or not the effects of heavy metal contamination in the study area generated impacts on soil bacterial communities. We found that the impact related to the elevation of heavy metal concentration due to the operation of cement plants in the surrounding soil can be considered smooth according to globally recognized indices such as Igeo. However, we identified that both the taxonomic and functional structures of the communities surrounding cement plants were different from those found in the control areas. We consider that our findings contribute significantly to the general understanding of the effects of heavy metals on the soil ecosystem by showing that light contamination can disturb the dynamics of ecosystem services provided by soil, specifically those associated with microbial metabolism.</p>
<h1>RNA metabarcoding helps reveal zooplankton community response to environmental stressors.</h1>
<p><a href="https://doi.org/S0269-7491(21)02028-5">doi:S0269-7491(21)02028-5</a></p>
['Boreal lake', 'COI', 'Community RNA', 'Ecogenomics', 'Ecological indicator', 'Freshwater ecosystem']<p>DNA metabarcoding can provide a high-throughput and rapid method for characterizing responses of communities to environmental stressors. However, within bulk samples, DNA metabarcoding hardly distinguishes live from the dead organisms. Here, both DNA and RNA metabarcoding were applied and compared in experimental freshwater mesocosms conducted for assessment of ecotoxicological responses of zooplankton communities to remediation treatment until 38 days post oil-spill. Furthermore, a novel indicator of normalized vitality (NV), sequence counts of RNA metabarcoding normalized by that of DNA metabarcoding, was developed for assessment of ecological responses. DNA and RNA metabarcoding detected similar taxa richness and rank of relative abundances. Both DNA and RNA metabarcoding demonstrated slight shifts in measured α-diversities in response to treatments. NV presented relatively greater magnitudes of differential responses of community compositions to treatments compared to DNA or RNA metabarcoding. NV declined from the start of the experiment (3 days pre-spill) to the end (38 days post-spill). NV also differed between Rotifer and Arthropoda, possibly due to differential life histories and sizes of organisms. NV could be a useful indicator for characterizing ecological responses to anthropogenic influence; however, the biology of target organisms and subsequent RNA production need to be considered.</p>
<h1>Diversity and ecology of protists revealed by metabarcoding.</h1>
<p><a href="https://doi.org/S0960-9822(21)01056-3">doi:S0960-9822(21)01056-3</a></p>
<p>No keywords</p><p>Protists are the dominant eukaryotes in the biosphere where they play key functional roles. While protists have been studied for over a century, it is the high-throughput sequencing of molecular markers from environmental samples - the approach of metabarcoding - that has revealed just how diverse, and abundant, these small organisms are. Metabarcoding is now routine to survey environmental diversity, so data have rapidly accumulated from a multitude of environments and at different sampling scales. This mass of data has provided unprecedented opportunities to study the taxonomic and functional diversity of protists, and how this diversity is organised in space and time. Here, we use metabarcoding as a common thread to discuss the state of knowledge in protist diversity research, from technical considerations of the approach to important insights gained on diversity patterns and the processes that might have structured this diversity. In addition to these insights, we conclude that metabarcoding is on the verge of an exciting added dimension thanks to the maturation of high-throughput long-read sequencing, so that a robust eco-evolutionary framework of protist diversity is within reach.</p>
<h1>Strategies for sample labelling and library preparation in DNA metabarcoding studies.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13512">doi:10.1111/1755-0998.13512</a></p>
['Illumina sequencing', 'amplicon sequencing', 'biodiversity assessment', 'eDNA', 'environmental DNA', 'high-throughput sequencing', 'library preparation']<p>Metabarcoding of DNA extracted from environmental or bulk specimen samples is increasingly used to profile biota in basic and applied biodiversity research because of its targeted nature that allows sequencing of genetic markers from many samples in parallel. To achieve this, PCR amplification is carried out with primers designed to target a taxonomically informative marker within a taxonomic group, and sample-specific nucleotide identifiers are added to the amplicons prior to sequencing. The latter enables assignment of the sequences back to the samples they originated from. Nucleotide identifiers can be added during the metabarcoding PCR and during "library preparation", that is, when amplicons are prepared for sequencing. Different strategies to achieve this labelling exist. All have advantages, challenges and limitations, some of which can lead to misleading results, and in the worst case compromise the fidelity of the metabarcoding data. Given the range of questions addressed using metabarcoding, ensuring that data generation is robust and fit for the chosen purpose is critically important for practitioners seeking to employ metabarcoding for biodiversity assessments. Here, we present an overview of the three main workflows for sample-specific labelling and library preparation in metabarcoding studies on Illumina sequencing platforms; one-step PCR, two-step PCR, and tagged PCR. Further, we distill the key considerations for researchers seeking to select an appropriate metabarcoding strategy for their specific study. Ultimately, by gaining insights into the consequences of different metabarcoding workflows, we hope to further consolidate the power of metabarcoding as a tool to assess biodiversity across a range of applications.</p>
<h1>A Standardized Procedure for Monitoring Harmful Algal Blooms in Chile by Metabarcoding Analysis.</h1>
<p><a href="https://doi.org/10.3791/62967">doi:10.3791/62967</a></p>
<p>No keywords</p><p>Harmful algae blooms (HABs) monitoring has been implemented worldwide, and Chile, a country famous for its fisheries and aquaculture, has intensively used microscopic and toxin analyses for decades for this purpose. Molecular biological methods, such as high-throughput DNA sequencing and bacterial assemblage-based approaches, are just beginning to be introduced in Chilean HAB monitoring, and the procedures have not yet been standardized. Here, 16S rRNA and 18S rRNA metabarcoding analyses for monitoring Chilean HABs are introduced stepwise. According to a recent hypothesis, algal-bacterial mutualistic association plays a critical synergetic or antagonistic relationship accounting for bloom initiation, maintenance, and regression. Thus, monitoring HAB from algal-bacterial perspectives may provide a broader understanding of HAB mechanisms and the basis for early warning. Metabarcoding analysis is one of the best suited molecular-based tools for this purpose because it can detect massive algal-bacterial taxonomic information in a sample. The visual procedures of sampling to metabarcoding analysis herein provide specific instructions, aiming to reduce errors and collection of reliable data.</p>
<h1>Coming of age for COI metabarcoding of whole organism community DNA: Towards bioinformatic harmonisation.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13502">doi:10.1111/1755-0998.13502</a></p>
['COI barcode', 'animal communities', 'bioinformatics', 'community ecology', 'high-throughput sequencing', 'metabarcoding']<p>Metabarcoding of DNA extracted from community samples of whole organisms (whole organism community DNA, wocDNA) is increasingly being applied to terrestrial, marine and freshwater metazoan communities to provide rapid, accurate and high resolution data for novel molecular ecology research. The growth of this field has been accompanied by considerable development that builds on microbial metabarcoding methods to develop appropriate and efficient sampling and laboratory protocols for whole organism metazoan communities. However, considerably less attention has focused on ensuring bioinformatic methods are adapted and applied comprehensively in wocDNA metabarcoding. In this study we examined over 600 papers and identified 111 studies that performed COI metabarcoding of wocDNA. We then systematically reviewed the bioinformatic methods employed by these papers to identify the state-of-the-art. Our results show that the increasing use of wocDNA COI metabarcoding for metazoan diversity is characterised by a clear absence of bioinformatic harmonisation, and the temporal trends show little change in this situation. The reviewed literature showed (i) high heterogeneity across pipelines, tasks and tools used, (ii) limited or no adaptation of bioinformatic procedures to the nature of the COI fragment, and (iii) a worrying underreporting of tasks, software and parameters. Based upon these findings we propose a set of recommendations that we think the metabarcoding community should consider to ensure that bioinformatic methods are appropriate, comprehensive and comparable. We believe that adhering to these recommendations will improve the long-term integrative potential of wocDNA COI metabarcoding for biodiversity science.</p>
<h1>Seeing through sedimented waters: environmental DNA reduces the phantom diversity of sharks and rays in turbid marine habitats.</h1>
<p><a href="https://doi.org/10.1186/S12862-021-01895-6">doi:10.1186/S12862-021-01895-6</a></p>
['12S ribosomal RNA', 'Chondrichthyes', 'Cytochrome c oxidase subunit I', 'Dark diversity', 'Southeast Asia', 'Urban coastlines']<p>Sharks and rays are some of the most threatened marine taxa due to the high levels of bycatch and significant demand for meat and fin-related products in many Asian communities. At least 25% of shark and ray species are considered to be threatened with extinction. In particular, the density of reef sharks in the Pacific has declined to 3-10% of pre-human levels. Elasmobranchs are thought to be sparse in highly urbanised and turbid environments. Low visibility coupled with the highly elusive behaviour of sharks and rays pose a challenge to diversity estimation and biomonitoring efforts as sightings are limited to chance encounters or from carcasses ensnared in nets. Here we utilised an eDNA metabarcoding approach to enhance the precision of elasmobranch diversity estimates in urbanised marine environments.</p>
<h1>Environmental DNA sampling protocols for the surveillance of marine non-indigenous species in Irish coastal waters.</h1>
<p><a href="https://doi.org/S0025-326X(21)00927-9">doi:S0025-326X(21)00927-9</a></p>
['COI', 'High throughput sequencing', 'Metabarcoding', 'Non-indigenous species', 'Rapid detection of species', 'eDNA']<p>Understanding the spread and distribution of Non-Indigenous Species (NIS) is key when implementing legislation to maintain good ecosystem health. Environmental DNA (eDNA) has shown great potential to detect aquatic organisms in a rapid and cost-effective way, however their applicability to new environments must be validated prior to their implementation. Here, we tested different field sampling methods in combination with eDNA metabarcoding to develop a tool to detect NIS. Large and small volumes of seawater were filtered, in addition to the collection of sediment and horizontal tow net samples at 12 locations across four distinct geographic areas in Ireland. The biggest dissimilarity in the species recovered was found between sediment and town net samples. Tow nets showed to be the most efficient. A total of 357 taxa were identified, including 16 NIS. Fine mesh tow nets were identified as the most cost-efficient for large-scale monitoring and surveillance of NIS.</p>
<h1>Dung beetles as samplers of mammals in Malaysian Borneo-a test of high throughput metabarcoding of iDNA.</h1>
<p><a href="https://doi.org/10.7717/Peerj.11897">doi:10.7717/Peerj.11897</a></p>
['Biodiversity surveys', 'Invertebrate-derived DNA', 'Scarabaeoidea', 'Southeast Asia', 'Tropical forests', 'Vertebrates']<p>Invertebrate-derived DNA (iDNA) sampling in biodiversity surveys is becoming increasingly widespread, with most terrestrial studies relying on DNA derived from the gut contents of blood-feeding invertebrates, such as leeches and mosquitoes. Dung beetles (superfamily Scarabaeoidea) primarily feed on the faecal matter of terrestrial vertebrates and offer several potential benefits over blood-feeding invertebrates as samplers of vertebrate DNA. Importantly, these beetles can be easily captured in large numbers using simple, inexpensive baited traps, are globally distributed, and occur in a wide range of habitats. To build on the few existing studies demonstrating the potential of dung beetles as sources of mammalian DNA, we subjected the large-bodied, Bornean dung beetle (<i>Catharsius renaudpauliani</i>) to a controlled feeding experiment. We analysed DNA from gut contents at different times after feeding using qPCR techniques. Here, we first describe the window of DNA persistence within a dung beetle digestive tract. We found that the ability to successfully amplify cattle DNA decayed over relatively short time periods, with DNA copy number decreasing by two orders of magnitude in just 6 h. In addition, we sampled communities of dung beetles from a lowland tropical rainforest in Sabah, Malaysia, in order to test whether it is possible to identify vertebrate sequences from dung beetle iDNA. We sequenced both the gut contents from large dung beetle species, as well as whole communities of smaller beetles. We successfully identified six mammalian species from our samples, including the bearded pig (<i>Sus barbatus</i>) and the sambar deer (<i>Rusa unicolor</i>)-both vulnerable species on the IUCN red list. Our results represent the first use of dung beetle iDNA to sample Southeast Asian vertebrate fauna, and highlight the potential for dung beetle iDNA to be used in future biodiversity monitoring surveys.</p>
<h1>Inventory of the benthic eukaryotic diversity in the oldest European lake.</h1>
<p><a href="https://doi.org/10.1002/Ece3.7907">doi:10.1002/Ece3.7907</a></p>
['ancient lake', 'benthic eukaryotes', 'eDNA', 'functional diversity']<p>We have profound knowledge on biodiversity on Earth including plants and animals. In the recent decade, we have also increased our understanding on microorganisms in different hosts and the environment. However, biodiversity is not equally well studied among different biodiversity groups and Earth's systems with eukaryotes in freshwater sediments being among the least known. In this study, we used high-throughput sequencing of the 18S rRNA gene to investigate the entire diversity of benthic eukaryotes in three distinct habitats (littoral sediment and hard substrate, profundal sediment) of Lake Ohrid, the oldest European lake. Eukaryotic sequences were dominated by annelid and arthropod animals (54% of all eukaryotic reads) and protists (Ochrophyta and Ciliophora; together 40% of all reads). Eukaryotic diversity was 15% higher in the deep profundal than on either near-surface hard substrates or littoral sediments. The three habitats differed in their taxonomic and functional community composition. Specifically, heterotrophic organisms accounted for 92% of the reads in the profundal, whereas phototrophs accounted for 43% on the littoral hard substrate. The profundal community was the most homogeneous, and its network was the most complex, suggesting its highest stability among the sampled habitats.</p>
<h1>Localized reshaping of the fungal community in response to a forest fungal pathogen reveals resilience of Mediterranean mycobiota.</h1>
<p><a href="https://doi.org/S0048-9697(21)04657-X">doi:S0048-9697(21)04657-X</a></p>
['Castanea sativa', 'Ink disease', 'Metabarcoding', 'Phytophthora cambivora']<p>Mediterranean forests are facing the impact of pests such as the soilborne Phytophthora cambivora, the causal agent of Ink disease, and this impact is made more severe by global changes. The status and resilience of the soil microbial ecosystem in areas with such a disturbance are little known; however, the assessment of the microbial community is fundamental to preserve the ecosystem functioning under emerging challenges. We profile soil fungal communities in a chestnut stand affected by ink disease in Italy using metabarcoding, and couple high-throughput sequencing with physico-chemical parameters and dendrometric measurements. Since the site also includes an area where the disease symptoms seem to be suppressed, we performed several analyses to search for determinants that may contribute to such difference. We demonstrate that neither pathogen presence nor trees decline associate with the reduction of the residing community diversity and functions, but rather with microbial network reshaping through substitutions and new interactions, despite a conservation of core taxa. We predict interactions between taxa and parameters such as soil pH and C/N ratio, and suggest that disease incidence may also relate with disappearance of pathogen antagonists, including ericoid- and ectomycorrhizal (ECM) fungi. By combining metabarcoding and field studies, we infer the resilient status of the fungal community towards a biotic stressor, and provide a benchmark for the study of other threatened ecosystems.</p>
<h1>Characterizing parasitic nematode faunas in faeces and soil using DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1186/S13071-021-04935-8">doi:10.1186/S13071-021-04935-8</a></p>
['Amplicon sequencing', 'Biomonitoring', 'Gastrointestinal parasitic nematode', 'Metabarcoding', 'NC1–NC2 primers', 'Parasite', 'Reindeer']<p>Gastrointestinal parasitic nematodes can impact fecundity, development, behaviour, and survival in wild vertebrate populations. Conventional monitoring of gastrointestinal parasitic nematodes in wild populations involves morphological identification of eggs, larvae, and adults from faeces or intestinal samples. Adult worms are typically required for species-level identification, meaning intestinal material from dead animals is needed to characterize the nematode community with high taxonomic resolution. DNA metabarcoding of environmental samples is increasingly used for time- and cost-effective, high-throughput biodiversity monitoring of small-bodied organisms, including parasite communities. Here, we evaluate the potential of DNA metabarcoding of faeces and soil samples for non-invasive monitoring of gastrointestinal parasitic nematode communities in a wild ruminant population.</p>
<h1>Towards reproducible metabarcoding data: Lessons from an international cross-laboratory experiment.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13485">doi:10.1111/1755-0998.13485</a></p>
['18S ribosomal rRNA (18S rRNA)', 'high-throughput sequencing', 'metabarcoding', 'mitochondrial cytochrome c oxidase subunit 1 (COI)', 'reproducibility', 'standardization']<p>Advances in high-throughput sequencing (HTS) are revolutionizing monitoring in marine environments by enabling rapid, accurate and holistic detection of species within complex biological samples. Research institutions worldwide increasingly employ HTS methods for biodiversity assessments. However, variance in laboratory procedures, analytical workflows and bioinformatic pipelines impede the transferability and comparability of results across research groups. An international experiment was conducted to assess the consistency of metabarcoding results derived from identical samples and primer sets using varying laboratory procedures. Homogenized biofouling samples collected from four coastal locations (Australia, Canada, New Zealand and the USA) were distributed to 12 independent laboratories. Participants were asked to follow one of two HTS library preparation workflows. While DNA extraction, primers and bioinformatic analyses were purposefully standardized to allow comparison, many other technical variables were allowed to vary among laboratories (amplification protocols, type of instrument used, etc.). Despite substantial variation observed in raw results, the primary signal in the data was consistent, with the samples grouping strongly by geographical origin for all data sets. Simple post hoc data clean-up by removing low-quality samples gave the best improvement in sample classification for nuclear 18S rRNA gene data, with an overall 92.81% correct group attribution. For mitochondrial COI gene data, the best classification result (95.58%) was achieved after correction for contamination errors. The identified critical methodological factors that introduced the greatest variability (preservation buffer, sample defrosting, template concentration, DNA polymerase, PCR enhancer) should be of great assistance in standardizing future biodiversity studies using metabarcoding.</p>
<h1>Occurrence and distribution of microbial pollutants in coastal areas of the Adriatic Sea influenced by river discharge.</h1>
<p><a href="https://doi.org/S0269-7491(21)01254-9">doi:S0269-7491(21)01254-9</a></p>
['Adriatic sea', 'Coastal rivers', 'Metabarcoding', 'Microbial pollution', 'Source tracking']<p>The transport of a variety of pollutants from agricultural, industrial and urbanised areas makes rivers major contributors to the contamination of coastal marine environments. Too little is known of their role in carrying pathogens to the coast. We used DNA-based metabarcoding data to describe the microbial community composition in seawater and sediment collected in front of the estuary of the Tronto, the Chienti and the Esino, three Italian rivers with different pollution levels that empty into the north-central Adriatic Sea, and to detect and measure within these communities the relative abundance of microbial pollutants, including traditional faecal indicators and alternative faecal and sewage-associated pollutants. We then applied the FORENSIC algorithm to distinguish human from non-human sources of microbial pollution and FAPROTAX to map prokaryotic clades to established metabolic or other ecologically relevant functions. Finally, we searched the dataset for other common pathogenic taxa. Seawater and sediment contained numerous potentially pathogenic bacteria, mainly faecal and sewage-associated. The samples collected in front of the Tronto estuary showed the highest level of contamination, likely sewage-associated. The pathogenic signature showed a weak but positive correlation with some nutrients and strong correlations with some polycyclic aromatic hydrocarbons. This study confirms that rivers transport pathogenic bacteria to the coastal sea and highlights the value of expanding the use of HTS data, source tracking and functional identification tools to detect microbial pollutants and identify their sources with a view to gaining a better understanding of the pathways of sewage-associated discharges to the sea.</p>
<h1>Next-Generation Sequencing and Its Impacts on Entomological Research in Ecology and Evolution.</h1>
<p><a href="https://doi.org/10.1007/S13744-021-00895-X">doi:10.1007/S13744-021-00895-X</a></p>
['Agroecosystems', 'NGS', 'conservative biological control', 'ecosystem services', 'food webs', 'insect-vectored diseases', 'pollination']<p>The advent of NGS-based methods has been profoundly transforming entomological research. Through continual development and improvement of different methods and sequencing platforms, NGS has promoted mass elucidation of partial or whole genetic materials associated with beneficial insects, pests (of agriculture, forestry and animal, and human health), and species of conservation concern, helping to unravel ecological and evolutionary mechanisms and characterizing survival, trophic interactions, and dispersal. It is shifting the scale of biodiversity and environmental analyses from individuals and biodiversity indicator species to the large-scale study of communities and ecosystems using bulk samples of species or a mixed "soup" of environmental DNA. As the NGS-based methods have become more affordable, complexity demystified, and specificity and sensitivity proven, their use in entomological research has spread widely. This article presents several examples on how NGS-based methods have been used in entomology to provide incentives to apply them when appropriate and to open our minds to the expected advances in entomology that are yet to come.</p>
<h1>Environmental DNA Metabarcoding: A Novel Method for Biodiversity Monitoring of Marine Fish Communities.</h1>
<p><a href="https://doi.org/10.1146/Annurev-Marine-041421-082251">doi:10.1146/Annurev-Marine-041421-082251</a></p>
['Sustainable Development Goals', 'biodiversity', 'eDNA', 'fish ecology', 'fisheries management', 'marine conservation']<p>Environmental DNA (eDNA) is genetic material that has been shed from macroorganisms. It has received increased attention as an indirect marker for biodiversity monitoring. This article reviews the current status of eDNA metabarcoding (simultaneous detection of multiple species) as a noninvasive and cost-effective approach for monitoring marine fish communities and discusses the prospects for this growing field. eDNA metabarcoding coamplifies short fragments of fish eDNA across a wide variety of taxa and, coupled with high-throughput sequencing technologies, allows massively parallel sequencing to be performed simultaneously for dozens to hundreds of samples. It can predict species richness in a given area, detect habitat segregation and biogeographic patterns from small to large spatial scales, and monitor the spatiotemporal dynamics of fish communities. In addition, it can detect an anthropogenic impact on fish communities through evaluation of their functional diversity. Recognizing the strengths and limitations of eDNA metabarcoding will help ensure that continuous biodiversity monitoring at multiple sites will be useful for ecosystem conservation and sustainable use of fishery resources, possibly contributing to achieving the targets of the United Nations' Sustainable Development Goal 14 for 2030.</p>
<h1>Illumina iSeq 100 and MiSeq exhibit similar performance in freshwater fish environmental DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1038/S41598-021-95360-5">doi:10.1038/S41598-021-95360-5</a></p>
<p>No keywords</p><p>Environmental DNA (eDNA) analysis is a method of detecting DNA from environmental samples and is used as a biomonitoring tool. In recent studies, Illumina MiSeq has been the most extensively used tool for eDNA metabarcoding. The Illumina iSeq 100 (hereafter, iSeq), one of the high-throughput sequencers (HTS), has a relatively simple workflow and is potentially more affordable than other HTS. However, its utility in eDNA metabarcoding has still not been investigated. In the present study, we applied fish eDNA metabarcoding to 40 water samples from river and lake ecosystems to assess the difference in species detectability and composition between iSeq and MiSeq. To check differences in sequence quality and errors, we also assessed differences in read changes between the two HTS. There were similar sequence qualities between iSeq and MiSeq. Significant difference was observed in the number of species between two HTS, but no difference was observed in species composition between the two HTS. Additionally, the species compositions in common with the conventional method were the same between the two HTS. According to the results, using the same amplicon library for sequencing, two HTS would exhibit a similar performance of fish species detection using eDNA metabarcoding.</p>
<h1>A molecular method for biomonitoring of an exotic plant-pest: Leafmining for environmental DNA.</h1>
<p><a href="https://doi.org/10.1111/Mec.16092">doi:10.1111/Mec.16092</a></p>
['agromyzid', 'biomonitoring', 'environmental DNA', 'invasive species', 'species-specific detection']<p>Understanding how invasive species respond to novel environments is limited by a lack of sensitivity and throughput in conventional biomonitoring methods. Arthropods in particular are often difficult to monitor due to their small size, rapid lifecycles, and/or visual similarities with co-occurring species. This is true for the agromyzid leafminer fly, Liriomyza sativae, a global pest of vegetable and nursery industries that has recently established in Australia. A robust method based on environmental DNA (eDNA) was developed exploiting traces of DNA left inside "empty" leaf mines, which are straightforward to collect and persist longer in the environment than the fly. This extends the window of possible diagnosis to at least 28 days after a leaf mine becomes empty. The test allowed for visually indistinguishable leafmining damage caused by L. sativae to be genetically differentiated from that of other flies. Field application resulted in the identification of new local plant hosts for L. sativae, including widely distributed weeds and common garden crops, which has important implications for the pest's ability to spread. Moreover, the test confirmed the presence of a previously unknown population of L. sativae on an island in the Torres Strait. The developed eDNA method is likely to become an important tool for L. sativae and other leafmining species of biosecurity significance, which, historically, have been difficult to detect, diagnose and monitor. More generally, eDNA is emerging as a highly sensitive and labour-efficient surveillance tool for difficult to survey species to improve outcomes for agricultural industries, global health, and the environment.</p>
<h1>Description of Microbial Communities of Phosphate Mine Wastes in Morocco, a Semi-Arid Climate, Using High-Throughput Sequencing and Functional Prediction.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2021.666936">doi:10.3389/Fmicb.2021.666936</a></p>
['PICRUSt prediction', 'biodiversity', 'metabarcoding', 'microbial communities', 'phosphate mine wastes']<p>Soil microbiota are vital for successful revegetation, as they play a critical role in nutrient cycles, soil functions, and plant growth and health. A rehabilitation scenario of the abandoned Kettara mine (Morocco) includes covering acidic tailings with alkaline phosphate mine wastes to limit water infiltration and hence acid mine drainage. Revegetation of phosphate wastes is the final step to this rehabilitation plan. However, revegetation is hard on this type of waste in semi-arid areas and only a few plants managed to grow naturally after 5 years on the store-and-release cover. As we know that belowground biodiversity is a key component for aboveground functioning, we sought to know if any structural problem in phosphate waste communities could explain the almost absence of plants. To test this hypothesis, bacterial and archaeal communities present in these wastes were assessed by 16S rRNA metabarcoding. Exploration of taxonomic composition revealed a quite diversified community assigned to 19 Bacterial and two Archaeal phyla, similar to other studies, that do not appear to raise any particular issues of structural problems. The dominant sequences belonged to <i>Proteobacteria</i>, <i>Bacteroidetes</i>, <i>Actinobacteria</i>, and <i>Gemmatimonadetes</i> and to the genera <i>Massilia</i>, <i>Sphingomonas</i>, and <i>Adhaeribacter</i>. LEfSe analysis identified 19 key genera, and metagenomic functional prediction revealed a broader phylogenetic range of taxa than expected, with all identified genera possessing at least one plant growth-promoting trait. Around 47% of the sequences were also related to genera possessing strains that facilitate plant development under biotic and environmental stress conditions, such as drought and heat.</p>
<h1>Just keep it simple? Benchmarking the accuracy of taxonomy assignment software in metabarcoding studies.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13473">doi:10.1111/1755-0998.13473</a></p>
['COI', 'fish', 'insects', 'metabarcoding', 'species identification', 'zooplankton']<p>How do you put a name on an unknown piece of DNA? From microbes to mammals, high-throughput metabarcoding studies provide a more objective view of natural communities, overcoming many of the inherent limitations of traditional field surveys and microscopy-based observations (Deiner et al., 2017). Taxonomy assignment is one of the most critical aspects of any metabarcoding study, yet this important bioinformatics task is routinely overlooked. Biodiversity surveys and conservation efforts often depend on formal species inventories: the presence (or absence) of species, and the number of individuals reported across space and time. However, computational workflows applied in eukaryotic metabarcoding studies were originally developed for use with bacterial/archaeal data sets, where microbial researchers rely on one conserved locus (nuclear 16S rRNA) and have access to vast databases with good coverage across most prokaryotic lineages - a situation not mirrored in most multicellular taxa. In this issue of Molecular Ecology Resources, Hleap et al. (2021) carry out an extensive benchmarking exercise focused on taxonomy assignment strategies for eukaryotic metabarcoding studies utilizing the mitochondrial Cytochrome C oxidase I marker gene (COI). They assess the performance and accuracy of software tools representing diverse methodological approaches: from "simple" strategies based on sequence similarity and composition, to model-based phylogenetic and probabilistic classification tools. Contrary to popular assumptions, less complex approaches (BLAST and the QIIME2 feature classifier) consistently outperformed more sophisticated mathematical algorithms and were highly accurate for assigning taxonomy at higher levels (e.g. family). Lower-level assignments at the genus and species level still pose significant challenge for most existing algorithms, and sparse eukaryotic reference databases further limit software performance. This study illuminates current best practices for metabarcoding taxonomy assignments, and underscores the need for community-driven efforts to expand taxonomic and geographic representation in reference DNA barcode databases.</p>
<h1>Relative species abundance estimation in artificial mixtures of insects using mito-metagenomics and a correction factor for the mitochondrial DNA copy number.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13464">doi:10.1111/1755-0998.13464</a></p>
['Metazoa', 'PCR-free', 'mitochondrial genomes', 'mitogenome skimming', 'mock sample', 'next-generation sequencing']<p>Mito-metagenomics (MMG) is becoming an alternative to amplicon metabarcoding for the assessment of biodiversity in complex biological samples using high-throughput sequencing. Whereas MMG overcomes the biases introduced by the PCR step in the generation of amplicons, it is not yet a technique free of shortcomings. First, as the reads are obtained from shotgun sequencing, a very low proportion of reads map into the mitogenomes, so a high sequencing effort is needed. Second, as the number of mitogenomes per cell can vary among species, the relative species abundance (RSA) in a mixture could be wrongly estimated. Here, we challenge the MMG method to estimate the RSA using artificial libraries of 17 insect species whose complete genomes are available on public repositories. With fresh specimens of these species, we created single-species libraries to calibrate the bioinformatic pipeline and mixed-species libraries to estimate the RSA. Our results showed that the MMG approach confidently recovers the species list of the mixtures, even when they contain congeneric species. The method was also able to estimate the abundance of a species across different samples (within-species estimation) but failed to estimate the RSA within a single sample (across-species estimation) unless a correction factor accounting for the variable number of mitogenomes per cell was used. To estimate this correction factor, we used the proportion of reads mapping into mitogenomes in the single-species libraries and the lengths of the whole genomes and mitogenomes.</p>
<h1>Dumpster diving for diatom plastid 16S rRNA genes.</h1>
<p><a href="https://doi.org/10.7717/Peerj.11576">doi:10.7717/Peerj.11576</a></p>
['16S rRNA', 'Algae', 'Bacillariophyta', 'Biofilm', 'Bioindicator', 'Cyanobacteria', 'Diatom', 'Metabarcoding']<p>High throughput sequencing is improving the efficiency of monitoring diatoms, which inhabit and support aquatic ecosystems across the globe. In this study, we explored the potential of a standard V4 515F-806RB primer pair in recovering diatom plastid 16S rRNA sequences. We used PhytoREF to classify the 16S reads from our freshwater biofilm field sampling from three stream segments across two streams in south-eastern Australia and retrieved diatom community data from other, publicly deposited, Australian 16S amplicon datasets. When these diatom operational taxonomic units (OTUs) were traced using the default RDPII and NCBI databases, 68% were characterized as uncultured cyanobacteria. We analysed the 16S rRNA sequences from 72 stream biofilm samples, separated the chloroplast OTUs, and classified them using the PhytoREF database. After filtering the reads attributed to Bacillariophyta (relative abundance >1%), 71 diatom OTUs comprising more than 90% of the diatom reads in each stream biofilm sample were identified. Beta-diversity analyses demonstrated significantly different diatom assemblages and discrimination among river segments. To further test the approach, the diatom OTUs from our biofilm sampling were used as reference sequences to identify diatom reads from other Australian 16S rRNA datasets in the NCBI-SRA database. Across the three selected public datasets, 67 of our 71 diatom OTUs were detected in other Australian ecosystems. Our results show that diatom plastid 16S rRNA genes are readily amplified with existing 515F-806RB primer sets. Therefore, the volume of existing 16S rRNA amplicon datasets initially generated for microbial community profiling can also be used to detect, characterize, and map diatom distribution to inform phylogeny and ecological health assessments, and can be extended into a range of ecological and industrial applications. To our knowledge, this study represents the first attempt to classify freshwater samples using this approach and the first application of PhytoREF in Australia.</p>
<h1>eDNA metabarcoding revealed differential structures of aquatic communities in a dynamic freshwater ecosystem shaped by habitat heterogeneity.</h1>
<p><a href="https://doi.org/S0013-9351(21)00896-3">doi:S0013-9351(21)00896-3</a></p>
['Community composition', 'Daqing river basin', 'Environmental variables', 'Habitat heterogeneity', 'eDNA metabarcoding']<p>Freshwater ecosystems have been threatened by complicated disturbances from both natural and anthropogenic variables, especially in dynamic and complex river basins. The environmental DNA (eDNA)-based approach provides a broader spectrum and higher throughput way of biomonitoring for biodiversity assessment compared with traditional morphological survey. Most eDNA metabarcoding studies have been limited to a few specific taxa/groups and habitat scopes. Here we applied the eDNA metabarcoding to characterize the structures and spatial variations of zooplankton and fish communities among different habitat types in a highly dynamic and complex freshwater ecosystem of the Daqing River basin (DRB). The results showed that varied species spectra of zooplankton and fish communities were identified and unique dominant species occurred across habitats. Additionally, markedly spatial distributions of biotic community structures were observed in areas with different habitat characteristics. Natural variables, including geographic distances and gradient ratio, as well as anthropogenic factors of chemical oxygen demand (COD) and organic chemicals demonstrated significant effects but different outcomes on the structures of zooplankton and fish communities. Moreover, the relative abundances of specific aquatic taxa were associated with the gradient of particular environmental variables. This case study verified the distribution patterns and differentiation mechanisms of biotic communities under habitat heterogeneity could be captured by application of eDNA biomonitoring. And habitat-specific and even species-specific environmental stressors would be diagnosed for improving management of complex river basins.</p>
<h1>Phospholipid fatty acid (PLFA) analysis as a tool to estimate absolute abundances from compositional 16S rRNA bacterial metabarcoding data.</h1>
<p><a href="https://doi.org/S0167-7012(21)00139-1">doi:S0167-7012(21)00139-1</a></p>
['Biomarker', 'Environmental monitoring', 'Environmental substrates', 'Microbial biomass', 'PLFA', 'eDNA']<p>Microbial biodiversity monitoring through the analysis of DNA extracted from environmental samples is increasingly popular because it is perceived as being rapid, cost-effective, and flexible concerning the sample types studied. DNA can be extracted from diverse media before high-throughput sequencing of the prokaryotic 16S rRNA gene is used to characterize the taxonomic diversity and composition of the sample (known as metabarcoding). While sources of bias in metabarcoding methodologies are widely acknowledged, previous studies have focused mainly on the effects of these biases within a single substrate type, and relatively little is known of how these vary across substrates. We investigated the effect of substrate type (water, microbial mats, lake sediments, stream sediments, soil and a mock microbial community) on the relative performance of DNA metabarcoding in parallel with phospholipid fatty acid (PLFA) analysis. Quantitative estimates of the biomass of different taxonomic groups in samples were made through the analysis of PLFAs, and these were compared to the relative abundances of microbial taxa estimated from metabarcoding. Furthermore, we used the PLFA-based quantitative estimates of the biomass to adjust relative abundances of microbial groups determined by metabarcoding to provide insight into how the biomass of microbial taxa from PLFA analysis can improve understanding of microbial communities from environmental DNA samples. We used two sets of PLFA biomarkers that differed in their number of PLFAs to evaluate how PLFA biomarker selection influences biomass estimates. Metabarcoding and PLFA analysis provided significantly different views of bacterial composition, and these differences varied among substrates. We observed the most notable differences for the Gram-negative bacteria, which were overrepresented by metabarcoding in comparison to PLFA analysis. In contrast, the relative biomass and relative sequence abundances aligned reasonably well for Cyanobacteria across the tested freshwater substrates. Adjusting relative abundances of microbial taxa estimated by metabarcoding with PLFA-based quantification estimates of the microbial biomass led to significant changes in the microbial community compositions in all substrates. We recommend including independent estimates of the biomass of microbial groups to increase comparability among metabarcoding libraries from environmental samples, especially when comparing communities associated with different substrates.</p>
<h1>On the phenology of protists: recurrent patterns reveal seasonal variation of protistan (Rhizaria: Cercozoa and Endomyxa) communities in tree canopies.</h1>
<p><a href="https://doi.org/Fiab081">doi:Fiab081</a></p>
['forest ecosystems', 'metabarcoding', 'microhabitats', 'plant microbiome', 'plant pathogens', 'unicellular eukaryotes']<p>Tree canopies are colonized by billions of highly specialized microorganisms that are well adapted to the highly variable microclimatic conditions, caused by diurnal fluctuations and seasonal changes. In this study, we investigated seasonality patterns of protists in the tree canopies of a temperate floodplain forest via high-throughput sequencing with group-specific primers for the phyla Cercozoa and Endomyxa. We observed consistent seasonality, and identified divergent spring and autumn taxa. Tree crowns were characterized by a dominance of bacterivores and omnivores, while eukaryvores gained a distinctly larger share in litter and soil communities on the ground. In the canopy seasonality was largest among communities detected on the foliar surface: In spring, higher variance within alpha diversity of foliar samples indicated greater heterogeneity during initial colonization. However, communities underwent compositional changes during the aging of leaves in autumn, highly reflecting recurring phenological changes during protistan colonization. Surprisingly, endomyxan root pathogens appeared to be exceptionally abundant across tree canopies during autumn, demonstrating a potential role of the canopy surface as a physical filter for air-dispersed propagules. Overall, about 80% of detected OTUs could not be assigned to known species-representing dozens of microeukaryotic taxa whose canopy inhabitants are waiting to be discovered.</p>
<h1>Consistent declines in aquatic biodiversity across diverse domains of life in rivers impacted by surface coal mining.</h1>
<p><a href="https://doi.org/10.1002/Eap.2389">doi:10.1002/Eap.2389</a></p>
['biodiversity loss', 'biomonitoring', 'community thresholds', 'environmental DNA', 'freshwater', 'indicator species', 'mountaintop mining', 'stream specific conductivity', 'surface coal mining', 'water quality criteria']<p>The rivers of Appalachia (United States) are among the most biologically diverse freshwater ecosystems in the temperate zone and are home to numerous endemic aquatic organisms. Throughout the Central Appalachian ecoregion, extensive surface coal mines generate alkaline mine drainage that raises the pH, salinity, and trace element concentrations in downstream waters. Previous regional assessments have found significant declines in stream macroinvertebrate and fish communities after draining these mined areas. Here, we expand these assessments with a more comprehensive evaluation across a broad range of organisms (bacteria, algae, macroinvertebrates, all eukaryotes, and fish) using high-throughput amplicon sequencing of environmental DNA (eDNA). We collected water samples from 93 streams in Central Appalachia (West Virginia, United States) spanning a gradient of mountaintop coal mining intensity and legacy to assess how this land use alters downstream water chemistry and affects aquatic biodiversity. For each group of organisms, we identified the sensitive and tolerant taxa along the gradient and calculated stream specific conductivity thresholds in which large synchronous declines in diversity were observed. Streams below mining operations had steep declines in diversity (-18 to -41%) and substantial shifts in community composition that were consistent across multiple taxonomic groups. Overall, large synchronous declines in bacterial, algal, and macroinvertebrate communities occurred even at low levels of mining impact at stream specific conductivity thresholds of 150-200 µS/cm that are substantially below the current U.S. Environmental Protection Agency aquatic life benchmark of 300 µS/cm for Central Appalachian streams. We show that extensive coal surface mining activities led to the extirpation of 40% of biodiversity from impacted rivers throughout the region and that current water quality criteria are likely not protective for many groups of aquatic organisms.</p>
<h1>Assessing the in situ bacterial diversity and composition at anthropogenically active sites using the environmental DNA (eDNA).</h1>
<p><a href="https://doi.org/S0025-326X(21)00627-5">doi:S0025-326X(21)00627-5</a></p>
['Bacteria', 'High-throughput sequencing', 'Metagenomics', 'Sediment and seawater', 'V3, V4', 'Vibrio', 'eDNA']<p>In this study, we identified the in situ bacterial groups and their community structure in coastal waters influenced by anthropogenic inputs. The use of environmental DNA (eDNA) and high throughput sequencing (HTS) were employed to derive accurate and reliable information on bacterial abundance. The V3 and V4 hypervariable regions of the 16S rRNA gene were amplified and the sequences were clustered into operational taxonomic units to analyze the site-specific variations in community composition. The percentage composition within the bacterial orders varied significantly among nearshore anthropogenic hotspots and offshore (5 km) samples. The microbial network constructed taking the bacterial abundance as nodes displayed strong positive and negative correlations within the bacterial families. Overall, the use of eDNA coupled with HTS is an incredible means for monitoring and assessing the abundance of bacterial communities and also serves as a biomonitoring tool to understand the degree of anthropogenic contamination in coastal waters.</p>
<h1>On the phenology of protists: recurrent patterns reveal seasonal variation of protistan (Rhizaria: Cercozoa and Endomyxa) communities in tree canopies.</h1>
<p><a href="https://doi.org/Fiab081">doi:Fiab081</a></p>
['forest ecosystems', 'metabarcoding', 'microhabitats', 'plant microbiome', 'plant pathogens', 'unicellular eukaryotes']<p>Tree canopies are colonized by billions of highly specialized microorganisms that are well adapted to the highly variable microclimatic conditions, caused by diurnal fluctuations and seasonal changes. In this study, we investigated seasonality patterns of protists in the tree canopies of a temperate floodplain forest via high-throughput sequencing with group-specific primers for the phyla Cercozoa and Endomyxa. We observed consistent seasonality, and identified divergent spring and autumn taxa. Tree crowns were characterized by a dominance of bacterivores and omnivores, while eukaryvores gained a distinctly larger share in litter and soil communities on the ground. In the canopy seasonality was largest among communities detected on the foliar surface: In spring, higher variance within alpha diversity of foliar samples indicated greater heterogeneity during initial colonization. However, communities underwent compositional changes during the aging of leaves in autumn, highly reflecting recurring phenological changes during protistan colonization. Surprisingly, endomyxan root pathogens appeared to be exceptionally abundant across tree canopies during autumn, demonstrating a potential role of the canopy surface as a physical filter for air-dispersed propagules. Overall, about 80% of detected OTUs could not be assigned to known species-representing dozens of microeukaryotic taxa whose canopy inhabitants are waiting to be discovered.</p>
<h1>The largest moss carpet transplant in Antarctica and its bryosphere cryptic biodiversity.</h1>
<p><a href="https://doi.org/10.1007/S00792-021-01235-Y">doi:10.1007/S00792-021-01235-Y</a></p>
['Antarctica', 'Diversity', 'HTS', 'Metabarcode', 'Moss carpet', 'Sanionia']<p>As part of the reconstruction of the Brazilian Antarctic Station on King George Island, three areas of moss carpet were transplanted to minimize the impact of the new facilities on the local biodiversity. A total of 650 m<sup>2</sup> of moss carpet was transplanted to neighboring but previously uncolonized locations and has subsequently survived for the last 3 years. Antarctic moss carpets typically comprise low moss species diversity and are often monospecific. We investigated the cryptic biodiversity that was transplanted along with the carpets using a metabarcoding approach through high throughput sequencing. We targeted 16S rRNA for Bacteria and Archaea, ITS for Fungi and Viridiplantae and Cox1 for Metazoa. We detected DNA representing 263 taxa from five Kingdoms (Chromista, Fungi, Metazoa, Protista and Viridiplantae), two Domains (Archaea and Bacteria) and 33 Phyla associated with the carpet. This diversity included one Archaea, 189 Bacteria, 24 Chromista, 19 Fungi, eight Metazoa, seven Protista and 16 Viridiplantae. Bacteria was the most abundant, rich and diverse group, with Chromista second in diversity and richness. Metazoa was less diverse but second highest in dominance. This is the first study to attempt transplanting a significant area of moss carpet to minimize anthropogenic environmental damage in Antarctica and to use metabarcoding as a proxy to assess diversity associated with Antarctic moss carpets, further highlighting the importance of such habitats for other organisms and their importance for conservation.</p>
<h1>Assessing the ecological quality status of the highly polluted Bagnoli area (Tyrrhenian Sea, Italy) using foraminiferal eDNA metabarcoding.</h1>
<p><a href="https://doi.org/S0048-9697(21)02942-9">doi:S0048-9697(21)02942-9</a></p>
['Biomonitoring', 'Ecological indices', 'High-throughput sequencing', 'Marine and coastal ecosystems', 'SIN', 'Sediment']<p>Morphology-based benthic foraminifera indices are increasingly used worldwide for biomonitoring the ecological quality of marine sediments. The recent development of foraminiferal eDNA metabarcoding offers a reliable, time-, and cost-effective alternative to morphology-based foraminiferal biomonitoring. However, the practical applications of these new tools are still highly limited. In the present study, we evaluate the response of benthic foraminifera and define the ecological quality status (EcoQS) in the Bagnoli area (Tyrrhenian Sea, Italy) based on a traditional morphology-based approach and eDNA metabarcoding. The geochemical data show that several sites in front of the former industrial plant contain higher concentrations of potentially toxic elements than the effect range median and are characterized by the highest total organic carbon (TOC) content, whereas the distantly located sites can be considered relatively low- to unpolluted. Significant differences (i.e., diversity and assemblage composition) in both morphological and molecular datasets were found between the relatively low- to unpolluted and the most polluted areas. Similarly, the selected ecological indices of both morphological and molecular datasets strikingly and congruently resulted in a clear separation following the environmental stress gradient. The molecular indices (i.e., g-exp(H'<sub>bc</sub>), g-Foram AMBI, and g-Foram AMBI-MOTUs) reliably identified poor-to-bad EcoQS in the polluted area in front of the former industrial plant. On the other hand, the Foram-AMBI based on morphology well identified an overall trend but seemed to overestimate the EcoQS if the traditional class boundaries were considered. The congruent and complementary trends between morphological and metabarcoding data observed in the case of the Bagnoli site further support the application of foraminiferal metabarcoding in routine biomonitoring to assess the environmental impacts of heavily polluted marine areas.</p>
<h1>Winogradsky columns as a strategy to study typically rare microbial eukaryotes.</h1>
<p><a href="https://doi.org/S0932-4739(21)00042-0">doi:S0932-4739(21)00042-0</a></p>
['Breviatea', 'DNA metabarcoding', 'Enrichment cultures', 'Soil protists', 'Vampyrellida', 'Winogradsky column']<p>Winogradsky columns have been widely used to study soil microbial communities, but the vast majority of those investigations have focused on the ecology and diversity of bacteria. In contrast, microbial eukaryotes (ME) have been regularly overlooked in studies based on experimental soil columns. Despite the recognized ecological relevance of ME in soil communities, investigations focused on ME diversity and the abundance of certain groups of interest are still scarce. In the present study, we used DNA metabarcoding (high-throughput sequencing of the V4 region of the 18S rRNA locus) to survey the ME diversity and abundance in an experimental Winogradsky soil column. Consistent with previous surveys in natural soils, our survey identified members of Cercozoa (Rhizaria; 31.2%), Apicomplexa and Ciliophora (Alveolata; 12.5%) as the predominant ME groups, but at particular depths we also detected the abundant presence of ME lineages that are typically rare in natural environments, such as members of the Vampyrellida (Rhizaria) and Breviatea (Amorphea). Our survey demonstrates that experimental soil columns are an efficient enrichment-culture approach that can enhance investigations about the diversity and ecology of ME in soils.</p>
<h1>To the canopy and beyond: Air dispersal as a mechanism of ubiquitous protistan pathogen assembly in tree canopies.</h1>
<p><a href="https://doi.org/S0932-4739(21)00040-7">doi:S0932-4739(21)00040-7</a></p>
['Airborne microorganisms', 'Canopies', 'Forest ecosystems', 'Pathogens', 'Protists']<p>Cercozoa and Oomycota contain a huge biodiversity and important pathogens of forest trees and other vegetation. We analyzed air dispersal of these protistan phyla with an air sampler near-ground (~2 m) and in tree crowns (~25 m) of three tree species (oak, linden and ash) in a temperate floodplain forest in March (before leafing) and May (after leaf unfolding) 2019 with a cultivation-independent high-throughput metabarcoding approach. We found a high diversity of Cercozoa and Oomycota in air samples with 122 and 81 OTUs, respectively. Especially oomycetes showed a significant difference in community composition between both sampling dates. Differences in community composition between air samples in tree canopies and close to the ground were however negligible, and also tree species identity did not affect communities in air samples, indicating that the distribution of protistan propagules through the air was not spatially restricted in the forest ecosystem. OTUs of plant pathogens, whose host species did not occur in the forest, demonstrate dispersal of propagules from outside the forest biome. Overall, our results lead to a better understanding of the stochastic processes of air dispersal of protists and protistan pathogens, a prerequisite to understand the mechanisms of their community assembly in forest ecosystems.</p>
<h1>DNA metabarcoding reveals the significant influence of anthropogenic effects on microeukaryotic communities in urban waterbodies.</h1>
<p><a href="https://doi.org/S0269-7491(21)00918-0">doi:S0269-7491(21)00918-0</a></p>
['Environmental DNA', 'Environmental heterogeneity', 'Indicator species', 'Multiple approaches', 'Urban black-odor water', 'Water quality']<p>Biological monitoring and assessment are the first and most fundamental steps towards diagnosing ecological or environmental quality. Increasing anthropogenic impact on urban ecosystems has prompted the development of less expensive and more efficient bioassessment approaches. Generally, a morphospecies based approach is effective for plants and large organisms but challenging for the microbial biosphere. To overcome this challenge, we used high-throughput DNA sequencing for predicting anthropogenic effects on microeukaryotic communities in urban waterbodies along a pollution gradient in Wuhan City, central China in summer 2019. Our results indicated that microeukaryotic community structure was distinct between non-urban polluted reservoir and urban polluted waterbodies. The heterogeneity of environmental condition significantly affected the microeukaryotic diversity, community structure, and species interactions. Integrated co-occurring network analysis revealed that the pollution gradient has a significant adverse impact on network complexity and network dissimilarity. These results revealed that the significant variation in anthropogenically-driven environmental condition shaped microeukaryotic communities in urban freshwater ecosystems. Furthermore, we observed that the relative abundance of indicative OTUs were significantly and negatively correlated with pollution level and these indicative OTUs could be used to predict the water quality status with up to 77% success. Thus, our multiple approaches combining 18S rDNA amplicon sequencing, co-occurring network and indicator species analyses suggest that this study gives a novel approach based on microeukaryotic communities to assess and predict the water quality status of urban aquatic environments.</p>
<h1>The diversity of soil mesofauna declines after bamboo invasion in subtropical China.</h1>
<p><a href="https://doi.org/S0048-9697(21)03053-9">doi:S0048-9697(21)03053-9</a></p>
['Ecosystem function', 'High-throughput sequencing', 'Metabarcoding', 'Plant invasion', 'Soil fauna', 'Trophic structure']<p>Plant invasions often act as ecosystem 'simplifiers' to simplify diversity and community structure of soil biota. However, inconsistent relationships between plant invasion and soil fauna have been found and few studies have addressed how soil fauna communities change upon plant invasions across taxa and feeding guilds. Here, we investigated the effects of moso bamboo (Phyllostachys edulis) invasion in subtropical China on soil mesofauna communities using novel high-throughput sequencing (HTS). Specifically, we analyzed the spatio-temporal dynamics of fauna diversity and feeding guilds in the litter and soil layers for three stages of moso bamboo invasion, i.e., uninvaded (secondary broadleaved forest), moderately invaded (mixed bamboo forest) and completely invaded (P. edulis forest). Overall, we found that the completely invaded bamboo forest decreased species richness and diversity of total fauna, herbivores, and microbivores consistently across different soil layers, but less so detritivores and predators. Although we did not find any interaction effects of bamboo invasion and soil layers on soil fauna diversity indices, significant interaction effects were found on the community composition, for total fauna and their feeding guilds. Specifically, the detrimental effects of bamboo invasion on the trophic structure of soil fauna communities were more profound in the litter layer than in the soil layer, suggesting that a litter layer with more diverse taxa does not mean higher resistance to plant invasion in maintaining the soil food web structure. Taken together, our findings suggest that different responses within fauna feeding guilds to plant invasion were pervasive, and a deeper soil layer may better alleviate the negative effects of pant invasion on fauna community structure. These shifts in soil biodiversity may further degrade ecosystem functioning.</p>
<h1>High-throughput sequencing on preservative ethanol is effective at jointly examining infraspecific and taxonomic diversity, although bioinformatics pipelines do not perform equally.</h1>
<p><a href="https://doi.org/10.1002/Ece3.7453">doi:10.1002/Ece3.7453</a></p>
['biodiversity', 'bioinformatics', 'bulkDNA', 'ethanol‐based DNA', 'haplotype diversity', 'high‐throughput sequencing', 'metabarcoding', 'tunicate']<p>High-throughput sequencing of amplicons (HTSA) has been proposed as an effective approach to evaluate taxonomic and genetic diversity at the same time. However, there are still uncertainties as to how the results produced by different bioinformatics treatments impact the conclusions drawn on biodiversity and population genetics indices.We evaluated the ability of six bioinformatics pipelines to recover taxonomic and genetic diversity from HTSA data obtained from controlled assemblages. To that end, 20 assemblages were produced using 354 colonies of <i>Botrylloides</i> spp., sampled in the wild in ten marinas around Brittany (France). We used DNA extracted from preservative ethanol (ebDNA) after various time of storage (3, 6, and 12 months), and from a bulk of preserved specimens (bulkDNA). DNA was amplified with primers designed for targeting this ascidian genus. Results obtained from HTSA data were compared with Sanger sequencing on individual zooids (i.e., individual barcoding).Species identification and relative abundance determined with HTSA data from either ebDNA or bulkDNA were similar to those obtained with traditional individual barcoding. However, after 12 months of storage, the correlation between HTSA and individual-based data was lower than after shorter durations. The six bioinformatics pipelines were able to depict accurately the genetic diversity using standard population genetics indices (H<sub>S</sub> and F<sub>ST</sub>), despite producing false positives and missing rare haplotypes. However, they did not perform equally and dada2 was the only pipeline able to retrieve all expected haplotypes.This study showed that ebDNA is a nondestructive alternative for both species identification and haplotype recovery, providing storage does not last more than 6 months before DNA extraction. Choosing the bioinformatics pipeline is a matter of compromise, aiming to retrieve all true haplotypes while avoiding false positives. We here recommend to process HTSA data using dada2, including a chimera-removal step. Even if the possibility to use multiplexed primer sets deserves further investigation to expand the taxonomic coverage in future similar studies, we showed that primers targeting a particular genus allowed to reliably analyze this genus within a complex community.</p>
<h1>Sampling forest soils to describe fungal diversity and composition. Which is the optimal sampling size in mediterranean pure and mixed pine oak forests?</h1>
<p><a href="https://doi.org/S1878-6146(21)00012-X">doi:S1878-6146(21)00012-X</a></p>
['Beta-diversity', 'DNA metabarcoding', 'Fungal communities', 'Mediterranean forest', 'Mixed forests', 'Sample pooling']<p>Soil sampling is a critical step affecting perceived fungal diversity, however sampling optimization for high-throughput-DNA sequencing studies have never been tested in Mediterranean forest ecosystems. We identified the minimum number of pooled samples needed to obtain a reliable description of fungal communities in terms of diversity and composition in three different Mediterranean forests (pine, oak, and mixed-pine-oak). Twenty soil samples were randomly selected in each of the three plots per type. Samples obtained in 100 m<sup>2</sup> plots were pooled to obtain mixtures of 3, 6, 10, 15, 20 samples, and sequenced using Illumina MiSeq of fungal ITS2 amplicons. Pooling three soil samples in Pinus and Quercus stands provided consistent richness estimations, while at least six samples were needed in mixed-stands. β-diversity decreased with increasing sample pools in monospecific-stands, while there was no effect of sample pool size on mixed-stands. Soil sample pooling had no effect over species composition. We estimate that three samples would be already optimal to describe fungal richness and composition in Mediterranean pure stands, while at least six samples would be needed in mixed stands.</p>
<h1>Using zooplankton metabarcoding to assess the efficacy of different techniques to clean-up an oil-spill in a boreal lake.</h1>
<p><a href="https://doi.org/S0166-445X(21)00106-5">doi:S0166-445X(21)00106-5</a></p>
['Canada', 'Ecogenomics', 'Ecotoxicology', 'Freshwater', 'Mitochondrial COI', 'Next-generation sequencing']<p>Regulators require adequate information to select best practices with less ecosystem impacts for remediation of freshwater ecosystems after oil spills. Zooplankton are valuable indicators of aquatic ecosystem health as they play pivotal roles in biochemical cycles while stabilizing food webs. Compared with morphological identification, metabarcoding holds promise for cost-effective, high-throughput, and benchmarkable biomonitoring of zooplankton communities. The objective of this study was to apply DNA and RNA metabarcoding of zooplankton for ecotoxicological assessment and compare it with traditional morphological identification in experimental shoreline enclosures in a boreal lake. These identification methods were also applied in context of assessing response of the zooplankton community exposed to simulated spills of diluted bitumen (dilbit), with experimental remediation practices (enhanced monitored natural recovery and shoreline cleaner application). Metabarcoding detected boreal zooplankton taxa up to the genus level, with a total of 24 shared genera, and while metabarcoding-based relative abundance served as an acceptable proxy for biomass inferred by morphological identification (ρ ≥ 0.52). Morphological identification determined zooplankton community composition changes due to treatments at 11 days post-spill (PERMANOVA, p = 0.0143) while metabarcoding methods indicated changes in zooplankton richness and communities at 38 days post-spill (T-test, p < 0.05; PERMANOVA, p ≤ 0.0429). Shoreline cleaner application overall seemed to have the largest impact on zooplankton communities relative to enhanced monitored natural recovery, regardless of zooplankton identification method. Both metabarcoding and morphological identification were able to discern the differences between the two experimental remediation practices. Metabarcoding of zooplankton could provide informative results for ecotoxicological assessment of the remediation practices of dilbit, advancing our knowledge of best practices for remediating oil-impacted aquatic ecosystems while serving to accelerate the assessment of at-risk freshwater ecosystems.</p>
<h1>Profile hidden Markov model sequence analysis can help remove putative pseudogenes from DNA barcoding and metabarcoding datasets.</h1>
<p><a href="https://doi.org/10.1186/S12859-021-04180-X">doi:10.1186/S12859-021-04180-X</a></p>
['Bioinformatics', 'COI mtDNA', 'DNA barcode', 'Hidden Markov model', 'Metabarcode', 'NuMT', 'Nuclear encoded mitochondrial sequences', 'Pseudogene']<p>Pseudogenes are non-functional copies of protein coding genes that typically follow a different molecular evolutionary path as compared to functional genes. The inclusion of pseudogene sequences in DNA barcoding and metabarcoding analysis can lead to misleading results. None of the most widely used bioinformatic pipelines used to process marker gene (metabarcode) high throughput sequencing data specifically accounts for the presence of pseudogenes in protein-coding marker genes. The purpose of this study is to develop a method to screen for nuclear mitochondrial DNA segments (nuMTs) in large COI datasets. We do this by: (1) describing gene and nuMT characteristics from an artificial COI barcode dataset, (2) show the impact of two different pseudogene removal methods on perturbed community datasets with simulated nuMTs, and (3) incorporate a pseudogene filtering step in a bioinformatic pipeline that can be used to process Illumina paired-end COI metabarcode sequences. Open reading frame length and sequence bit scores from hidden Markov model (HMM) profile analysis were used to detect pseudogenes.</p>
<h1>Potential applications of CRISPR/Cas for next-generation biomonitoring of harmful algae blooms: A review.</h1>
<p><a href="https://doi.org/S1568-9883(21)00056-1">doi:S1568-9883(21)00056-1</a></p>
['Biomonitoring', 'CRISPR/Cas systems', 'Environmental DNA', 'Environmental RNA', 'HABs']<p>Research on harmful algal and cyanobacterial blooms (HABs and CHABs) has risen dramatically due to their increasing global distribution, frequency, and intensity. These blooms jeopardize public health, ecosystem function, sustainability and can have negative economic impacts. Numerous monitoring programs have been established using light microscopy, liquid chromatography coupled to mass spectrometry (LC-MS), ELISA, and spectrophotometry to monitor HABs/CHABs outbreaks. Recently, DNA/RNA-based molecular methods have been integrated into these programs to replace or complement traditional methods through analyzing environmental DNA and RNA (eDNA/eRNA) with techniques such as quantitative polymerase chain reaction (qPCR), fluorescent in situ hybridization (FISH), sandwich hybridization assay (SHA), isothermal amplification methods, and microarrays. These have enabled the detection of rare or cryptic species, enhanced sample throughput, and reduced costs and the need for visual taxonomic expertise. However, these methods have limitations, such as the need for high capital investment in equipment or detection uncertainties, including determining whether organisms are viable. In this review, we discuss the potential of newly developed molecular diagnosis technology based on Clustered Regularly Interspaced Short Palindromic Repeats/Cas proteins (CRISPR/Cas), which utilizes the prokaryotic adaptative immune systems of bacteria and archaea. Cas12 and Cas13-based platforms can detect both DNA and RNA with attomolar sensitivity within an hour. CRISPR/Cas diagnostic is a rapid, inexpensive, specific, and ultrasensitive technology that, with some further development, will provide many new platforms that can be used for HABs/CHABs biomonitoring and research.</p>
<h1>DNA metabarcoding and morphological methods show complementary patterns in the metacommunity organization of lentic epiphytic diatoms.</h1>
<p><a href="https://doi.org/S0048-9697(21)02481-5">doi:S0048-9697(21)02481-5</a></p>
['DNA barcoding', 'Diatoms', 'High-Throughput Sequencing', 'Metacommunity', 'rbcL']<p>Diatoms are important organisms in freshwater ecosystems due to their position as primary producers and therefore, analyzing their assemblages provides relevant information on ecosystem functioning. Diatoms have historically been identified based on morphological traits, which is time-consuming and requires well-trained specialists. Nevertheless, DNA barcoding offers an alternative approach to overcome some limitations of the morphological method. Here, we assess if both approaches are comparable methods to study patterns and mechanisms (including environmental filtering and dispersal limitation) of epiphytic diatom metacommunities using a comprehensive dataset from 22 Mediterranean ponds at different taxonomic resolutions. We used a fragment of rbcL barcode gene combined with High-Throughput Sequencing to infer diatom community composition. The overall degree of correspondence between both approaches was assessed by Procrustean rotation analysis and Procrustean randomization tests, whereas the role of local environmental variables and geographical distances was studied using a comprehensive combination of BIOENV, Mantel tests and distance-based redundancy analysis. Our results showed a relatively poor correspondence in the compositional variation of diatom metacommunity between both approaches. We speculate that the incompleteness of the reference database and the bioinformatics processing are the biases most likely affecting the molecular approach, whereas the limited counting effort and the presence of cryptic species are presumably the major biases related with the morphological method. On the other hand, variation in diatom community composition detected with both approaches was strongly related to the environmental template, which may be related with the narrow community-environment relationships in diatoms. Nevertheless, we found no significant relationship between compositional variation and geographical distances. Overall, our work shows the complementary nature of both approaches and highlights the importance of DNA metabarcoding to address empirical research questions of community ecology in freshwaters, especially once the reference databases include most genotypes of occurring taxa and bioinformatics biases are overcome.</p>
<h1>Using next generation sequencing of alpine plants to improve fecal metabarcoding diet analysis for Dall's sheep.</h1>
<p><a href="https://doi.org/10.1186/S13104-021-05590-Z">doi:10.1186/S13104-021-05590-Z</a></p>
['Alpine', 'Arctic', 'Boreal', 'Chloroplast', 'Dall’s sheep', 'Diet', 'Fecal', 'Metabarcoding', 'Plant']<p>Dall's sheep (Ovis dalli dalli) are important herbivores in the mountainous ecosystems of northwestern North America, and recent declines in some populations have sparked concern. Our aim was to improve capabilities for fecal metabarcoding diet analysis of Dall's sheep and other herbivores by contributing new sequence data for arctic and alpine plants. This expanded reference library will provide critical reference sequence data that will facilitate metabarcoding diet analysis of Dall's sheep and thus improve understanding of plant-animal interactions in a region undergoing rapid climate change.</p>
<h1>The aerobiome uncovered: Multi-marker metabarcoding reveals potential drivers of turn-over in the full microbial community in the air.</h1>
<p><a href="https://doi.org/S0160-4120(21)00176-8">doi:S0160-4120(21)00176-8</a></p>
['Aerobiome monitoring', 'Co-dispersal', 'Early-warning system', 'Meteorology', 'Microbial dispersal', 'Microbiome', 'Potential pathogens']<p>Air is a major conduit for the dispersal of organisms at the local and the global scale. Most research has focused on the dispersal of plants, vertebrates and human disease agents. However, the air represents a key dispersal medium also for bacteria, fungi and protists. Many of those represent potential pathogens of animals and plants and have until now gone largely unrecorded. Here we studied the turnover in composition of the entire aerobiome, the collective diversity of airborne microorganisms. For that we performed daily analyses of all prokaryotes and eukaryotes (including plants) using multi-marker high-throughput sequencing for a total of three weeks. We linked the resulting communities to local weather conditions, to assess determinants of aerobiome composition and distribution. We observed hundreds of microbial taxa, mostly belonging to spore-forming organisms including fungi, but also protists. Additionally, we detected many potential human- and plant-pathogens. Community composition fluctuated on a daily basis and was linked to concurrent weather conditions, particularly air pressure and temperature. Using network analyses, we identified taxonomically diverse groups of organisms with correlated temporal dynamics. In part, this was due to co-variation with environmental conditions, while we could also detect specific host-parasite interactions. This study provides the first full inventory of the aerobiome and identifies putative drivers of its dynamics in terms of taxon composition. This knowledge can help develop early warning systems against pathogens and improve our understanding of microbial dispersal.</p>
<h1>DNA Metabarcoding Reveals Broad Presence of Plant Pathogenic Oomycetes in Soil From Internationally Traded Plants.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2021.637068">doi:10.3389/Fmicb.2021.637068</a></p>
['Globisporangium', 'Phytophthora', 'Pythium', 'baiting', 'environmental DNA', 'plant pathogens']<p>Plants with roots and soil clumps transported over long distances in plant trading can harbor plant pathogenic oomycetes, facilitating disease outbreaks that threaten ecosystems, biodiversity, and food security. Tools to detect the presence of such oomycetes with a sufficiently high throughput and broad scope are currently not part of international phytosanitary testing regimes. In this work, DNA metabarcoding targeting the internal transcribed spacer (ITS) region was employed to broadly detect and identify oomycetes present in soil from internationally shipped plants. This method was compared to traditional isolation-based detection and identification after an enrichment step. DNA metabarcoding showed widespread presence of potentially plant pathogenic <i>Phytophthora</i> and <i>Pythium</i> species in internationally transported rhizospheric soil with <i>Pythium</i> being the overall most abundant genus observed. Baiting, a commonly employed enrichment method for <i>Phytophthora</i> species, led to an increase of golden-brown algae in the soil samples, but did not increase the relative or absolute abundance of potentially plant pathogenic oomycetes. Metabarcoding of rhizospheric soil yielded DNA sequences corresponding to oomycete isolates obtained after enrichment and identified them correctly but did not always detect the isolated oomycetes in the same samples. This work provides a proof of concept and outlines necessary improvements for the use of environmental DNA (eDNA) and metabarcoding as a standalone phytosanitary assessment tool for broad detection and identification of plant pathogenic oomycetes.</p>
<h1>Diversity of fungal DNA in lake sediments on Vega Island, north-east Antarctic Peninsula assessed using DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1007/S00792-021-01226-Z">doi:10.1007/S00792-021-01226-Z</a></p>
['Antarctica', 'Extremophile', 'Fungi', 'High-throughput sequencing', 'Taxonomy']<p>We assessed the diversity of fungal DNA present in sediments of three lakes on Vega Island, north-east Antarctic Peninsula using metabarcoding through high-throughput sequencing (HTS). A total of 640,902 fungal DNA reads were detected, which were assigned to 224 taxa of the phyla Ascomycota, Rozellomycota, Basidiomycota, Chytridiomycota and Mortierellomycota, in rank order of abundance. The most abundant genera were Pseudogymnoascus, Penicillium and Mortierella. However, a majority (423,508, 66%) of the reads, representing by 43 ASVs, could only be assigned at higher taxonomic levels and may represent taxa not currently included in the sequence databases used or be new or previously unreported taxa present in Antarctic lakes. The three lakes were characterized by high sequence diversity, richness, and moderate dominance indices. The ASVs were dominated by psychrotolerant and cosmopolitan cold-adapted Ascomycota, Basidiomycota and Mortierellomycota commonly reported in Antarctic environments. However, other taxa detected included unidentified members of Rozellomycota and Chytridiomycota species not previously reported in Antarctic lakes. The assigned diversity was composed mainly of taxa recognized as decomposers and pathogens of plants and invertebrates.</p>
<h1>Mosquito Identification From Bulk Samples Using DNA Metabarcoding: a Protocol to Support Mosquito-Borne Disease Surveillance in Canada.</h1>
<p><a href="https://doi.org/10.1093/Jme/Tjab046">doi:10.1093/Jme/Tjab046</a></p>
['DNA metabarcoding', 'biodiversity', 'mosquito', 'taxonomy', 'vector-borne disease']<p>Approximately 80 species of mosquitoes (Diptera: Culicidae) have been documented in Canada. Exotic species such as Aedes albopictus (Skuse) (Diptera: Culicidae) are becoming established. Recently occurring endemic mosquito-borne diseases (MBD) in Canada including West-Nile virus (WNV) and Eastern Equine Encephalitis (EEE) are having significant public health impacts. Here we explore the use of DNA metabarcoding to identify mosquitoes from CDC light-trap collections from two locations in eastern Canada. Two primer pairs (BF2-BR2 and F230) were used to amplify regions of the cytochrome c oxidase subunit I (CO1) gene. High throughput sequencing was conducted using an Illumina MiSeq platform and GenBank-based species identification was applied using a QIIME 1.9 bioinformatics pipeline. From a site in southeastern Ontario, Canada, 26 CDC light trap collections of 72 to >300 individual mosquitoes were used to explore the capacity of DNA metabarcoding to identify and quantify captured mosquitoes. The DNA metabarcoding method identified 33 species overall while 24 species were identified by key. Using replicates from each trap, the dried biomass needed to identify the majority of species was determined to be 76 mg (equivalent to approximately 72 mosquitoes), and at least two replicates from the dried biomass would be needed to reliably detect the majority of species in collections of 144-215 mosquitoes and three replicates would be advised for collections with >215 mosquitoes. This study supports the use of DNA metabarcoding as a mosquito surveillance tool in Canada which can help identify the emergence of new mosquito-borne disease potential threats.</p>
<h1>Microbial community and abiotic effects on aquatic bacterial communities in north temperate lakes.</h1>
<p><a href="https://doi.org/S0048-9697(21)01839-8">doi:S0048-9697(21)01839-8</a></p>
['Bacterial community', 'Deterministic', 'Highthroughput sequencing', 'Metabarcoding', 'Microeukaryotes', 'Stochastic']<p>The aquatic bacterial community (BC) plays a vital role in determining the nature and rate of ecosystem function. However, the biotic and abiotic factors influencing BC structure and function are largely unknown. Hence, the current study characterizes the impact of biotic and abiotic factors on aquatic bacterial biodiversity to determine whether the dominant effects are biotic or abiotic by partitioning their relative effects across temperate Canadian lakes. We collected water samples from sixty southern Ontario lakes and characterized their BC and microbial eukaryotic community (MEC) compositions using high throughput metabarcode sequencing of 16S and 18S rRNA gene fragments. The diversity and richness of aquatic BCs differed considerably among our study lakes, and those differences were explained by environmental, spatial, and biotic (MEC) factors (31%, 23%, and 23% of variance explained, respectively). The relatively large contribution from biotic and abiotic factors (54%), relative to spatial effects, shows deterministic processes prevail in shaping BC assembly in freshwater lakes. However, spatial effects also contributed significantly, highlighting the role of stochastic processes (ecological drift and coupled with limited dispersal) in shaping BC structure. Furthermore, our co-occurrence network analysis showed strong positive and negative interactions within and between the BCs and MECs, indicating mutualistic or antagonistic co-occurrence patterns relationships play important roles in driving the variation in BC composition among our sampled lakes. Considered together, our community analyses show that deterministic and stochastic processes combined contribute to determining the aquatic BC composition, and hence likely function as well, across a broad array of temperate freshwater lakes.</p>
<h1>Culture-Dependent and Amplicon Sequencing Approaches Reveal Diversity and Distribution of Black Fungi in Antarctic Cryptoendolithic Communities.</h1>
<p><a href="https://doi.org/213">doi:213</a></p>
['Antarctica', 'black fungi', 'cryptoendolithic communities', 'extremophiles', 'metabarcoding']<p>In the harshest environmental conditions of the Antarctic desert, normally incompatible with active life, microbes are adapted to exploit the cryptoendolithic habitat (i.e., pore spaces of rocks) and represent the predominant life-forms. In the rocky niche, microbes take advantage of the thermal buffering, physical stability, protection against UV radiation, excessive solar radiation, and water retention-of paramount importance in one of the driest environments on Earth. In this work, high-throughput sequencing and culture-dependent approaches have been combined, for the first time, to untangle the diversity and distribution of black fungi in the Antarctic cryptoendolithic microbial communities, hosting some of the most extreme-tolerant microorganisms. Rock samples were collected in a vast area, along an altitudinal gradient and opposite sun exposure-known to influence microbial diversity-with the aim to compare and integrate results gained with the two approaches. Among black fungi, <i>Friedmanniomyces endolithicus</i> was confirmed as the most abundant taxon. Despite the much stronger power of the high-throughput sequencing, several species were not retrieved with DNA sequencing and were detectable by cultivation only. We conclude that both culture-dependent and -independent analyses are needed for a complete overview of black fungi diversity. The reason why some species remain undetectable with molecular methods are speculated upon. The effect of environmental parameters such as sun exposure on relative abundance was clearer if based on the wider biodiversity detected with the molecular approach.</p>
<h1>eDNA metabarcoding of small plankton samples to detect fish larvae and their preys from Atlantic and Pacific waters.</h1>
<p><a href="https://doi.org/10.1038/S41598-021-86731-Z">doi:10.1038/S41598-021-86731-Z</a></p>
<p>No keywords</p><p>Zooplankton community inventories are the basis of fisheries management for containing fish larvae and their preys; however, the visual identification of early-stage larvae (the "missing biomass") is difficult and laborious. Here, eDNA metabarcoding was employed to detect zooplankton species of interest for fisheries from open and coastal waters. High-Throughput sequencing (HTS) from environmental samples using small water volumes has been proposed to detect species of interest whose DNA is the most abundant. We analyzed 6-L water samples taken from subtropical and tropical waters using Cytochrome oxidase I (COI) gene as metabarcode. In the open ocean, several commercial fish larvae and invertebrate species important in fish diet were found from metabarcodes and confirmed from individual barcoding. Comparing Atlantic, Mediterranean, Red Sea, and Pacific samples we found a lower taxonomic depth of OTU assignments in samples from tropical waters than in those from temperate ones, suggesting large gaps in reference databases for those areas; thus a higher effort of zooplankton barcoding in tropical oceans is highly recommended. This and similar simplified sampling protocols could be applied in early detection of species important for fisheries.</p>
<h1>Metabarcoding Reveals Changes in Benthic Eukaryote and Prokaryote Community Composition along a Tropical Marine Sediment Nickel Gradient.</h1>
<p><a href="https://doi.org/10.1002/Etc.5039">doi:10.1002/Etc.5039</a></p>
['Marine sediments', 'Metals', 'New Caledonia', 'Nickel mining', 'Tropical ecotoxicology', 'Vavouto Bay']<p>The Southeast Asia and Melanesia region has extensive nickel (Ni)-rich lateritic regoliths formed from the tropical weathering of ultramafic rocks. As the global demand for Ni continues to rise, these lateritic regoliths are increasingly being exploited for their economic benefit. Mining of these regoliths contributes to the enrichment of coastal sediments in trace metals, especially Ni. The present study used high-throughput sequencing (metabarcoding) to determine changes in eukaryote (18s v7 recombinant DNA [rDNA] and diatom-specific subregion of the 18s v4 rDNA) and prokaryote (16s v4 rDNA) community compositions along a sediment Ni concentration gradient offshore from a large lateritized ultramafic regolith in New Caledonia (Vavouto Bay). Significant changes in the eukaryote, diatom, and prokaryote community compositions were found along the Ni concentration gradient. These changes correlated most with the dilute-acid extractable concentration of Ni in the sediments, which explained 26, 23, and 19% of the variation for eukaryote, diatom, and prokaryote community compositions, respectively. Univariate analyses showed that there was no consistent change in indices of biodiversity, evenness, or richness. Diatom richness and diversity did, however, decrease as sediment acid extractable-Ni concentrations increased. Threshold indicator taxa analysis was conducted separately for each of the 3 targeted genes to detect changes in taxa whose occurrences decreased or increased along the acid extractable-Ni concentration gradient. Based on these data, 46 mg acid extractable-Ni/kg was determined as a threshold value where sensitive species began to disappear. In the case of the estuarine sediments offshore from lateritized ultramafic regolith in New Caledonia, this is recommended as an interim threshold value until further lines of evidence can contribute to a region-specific Ni sediment quality guideline value. Environ Toxicol Chem 2021;40:1894-1907. © 2021 SETAC.</p>
<h1>Debar: A sequence-by-sequence denoiser for COI-5P DNA barcode data.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13384">doi:10.1111/1755-0998.13384</a></p>
['COI', 'DNA barcode', 'Markov model', 'biodiversity', 'denoising', 'metabarcode']<p>DNA barcoding and metabarcoding are now widely used to advance species discovery and biodiversity assessments. High-throughput sequencing (HTS) has expanded the volume and scope of these analyses, but elevated error rates introduce noise into sequence records that can inflate estimates of biodiversity. Denoising -the separation of biological signal from instrument (technical) noise-of barcode and metabarcode data currently employs abundance-based methods which do not capitalize on the highly conserved structure of the cytochrome c oxidase subunit I (COI) region employed as the animal barcode. This manuscript introduces debar, an R package that utilizes a profile hidden Markov model to denoise indel errors in COI sequences introduced by instrument error. In silico studies demonstrated that debar recognized 95% of artificially introduced indels in COI sequences. When applied to real-world data, debar reduced indel errors in circular consensus sequences obtained with the Sequel platform by 75%, and those generated on the Ion Torrent S5 by 94%. The false correction rate was less than 0.1%, indicating that debar is receptive to the majority of true COI variation in the animal kingdom. In conclusion, the debar package improves DNA barcode and metabarcode workflows by aiding the generation of more accurate sequences aiding the characterization of species diversity.</p>
<h1>DNA Metabarcoding for the Characterization of Terrestrial Microbiota-Pitfalls and Solutions.</h1>
<p><a href="https://doi.org/361">doi:361</a></p>
['DNA metabarcoding workflow', 'archaea', 'bacteria', 'fungi', 'high-throughput sequencing', 'protists', 'soil and plant-associated microorganisms', 'terrestrial ecosystem']<p>Soil-borne microbes are major ecological players in terrestrial environments since they cycle organic matter, channel nutrients across trophic levels and influence plant growth and health. Therefore, the identification, taxonomic characterization and determination of the ecological role of members of soil microbial communities have become major topics of interest. The development and continuous improvement of high-throughput sequencing platforms have further stimulated the study of complex microbiota in soils and plants. The most frequently used approach to study microbiota composition, diversity and dynamics is polymerase chain reaction (PCR), amplifying specific taxonomically informative gene markers with the subsequent sequencing of the amplicons. This methodological approach is called DNA metabarcoding. Over the last decade, DNA metabarcoding has rapidly emerged as a powerful and cost-effective method for the description of microbiota in environmental samples. However, this approach involves several processing steps, each of which might introduce significant biases that can considerably compromise the reliability of the metabarcoding output. The aim of this review is to provide state-of-the-art background knowledge needed to make appropriate decisions at each step of a DNA metabarcoding workflow, highlighting crucial steps that, if considered, ensures an accurate and standardized characterization of microbiota in environmental studies.</p>
<h1>Simultaneous absolute quantification and sequencing of fish environmental DNA in a mesocosm by quantitative sequencing technique.</h1>
<p><a href="https://doi.org/10.1038/S41598-021-83318-6">doi:10.1038/S41598-021-83318-6</a></p>
<p>No keywords</p><p>The combination of high-throughput sequencing technology and environmental DNA (eDNA) analysis has the potential to be a powerful tool for comprehensive, non-invasive monitoring of species in the environment. To understand the correlation between the abundance of eDNA and that of species in natural environments, we have to obtain quantitative eDNA data, usually via individual assays for each species. The recently developed quantitative sequencing (qSeq) technique enables simultaneous phylogenetic identification and quantification of individual species by counting random tags added to the 5' end of the target sequence during the first DNA synthesis. Here, we applied qSeq to eDNA analysis to test its effectiveness in biodiversity monitoring. eDNA was extracted from water samples taken over 4 days from aquaria containing five fish species (Hemigrammocypris neglectus, Candidia temminckii, Oryzias latipes, Rhinogobius flumineus, and Misgurnus anguillicaudatus), and quantified by qSeq and microfluidic digital PCR (dPCR) using a TaqMan probe. The eDNA abundance quantified by qSeq was consistent with that quantified by dPCR for each fish species at each sampling time. The correlation coefficients between qSeq and dPCR were 0.643, 0.859, and 0.786 for H. neglectus, O. latipes, and M. anguillicaudatus, respectively, indicating that qSeq accurately quantifies fish eDNA.</p>
<h1>eDNAFlow, an automated, reproducible and scalable workflow for analysis of environmental DNA sequences exploiting Nextflow and Singularity.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13356">doi:10.1111/1755-0998.13356</a></p>
['Nextflow', 'Singularity', 'environmental DNA', 'metabarcoding']<p>Metabarcoding of environmental DNA (eDNA) when coupled with high throughput sequencing is revolutionising the way biodiversity can be monitored across a wide range of applications. However, the large number of tools deployed in downstream bioinformatic analyses often places a challenge in configuration and maintenance of a workflow, and consequently limits the research reproducibility. Furthermore, scalability needs to be considered to handle the growing amount of data due to increase in sequence output and the scale of project. Here, we describe eDNAFlow, a fully automated workflow that employs a number of state-of-the-art applications to process eDNA data from raw sequences (single-end or paired-end) to generation of curated and noncurated zero-radius operational taxonomic units (ZOTUs) and their abundance tables. This pipeline is based on Nextflow and Singularity which enable a scalable, portable and reproducible workflow using software containers on a local computer, clouds and high-performance computing (HPC) clusters. Finally, we present an in-house Python script to assign taxonomy to ZOTUs based on user specified thresholds for assigning lowest common ancestor (LCA). We demonstrate the utility and efficiency of the pipeline using an example of a published coral diversity biomonitoring study. Our results were congruent with the aforementioned study. The scalability of the pipeline is also demonstrated through analysis of a large data set containing 154 samples. To our knowledge, this is the first automated bioinformatic pipeline for eDNA analysis using two powerful tools: Nextflow and Singularity. This pipeline addresses two major challenges in the analysis of eDNA data; scalability and reproducibility.</p>
<h1>Temporal and Spatial Blood Feeding Patterns of Urban Mosquitoes in the San Juan Metropolitan Area, Puerto Rico.</h1>
<p><a href="https://doi.org/129">doi:129</a></p>
['Aedes aegypti', 'Culex quinquefasciatus', 'bird', 'high-throughput sequencing', 'iDNA', 'mammal', 'metabarcoding', 'species diversity', 'vector-borne diseases']<p>Urban ecosystems are a patchwork of habitats that host a broad diversity of animal species. Insects comprise a large portion of urban biodiversity which includes many pest species, including those that transmit pathogens. Mosquitoes (Diptera: Culicidae) inhabit urban environments and rely on sympatric vertebrate species to complete their life cycles, and in this process transmit pathogens to animals and humans. Given that mosquitoes feed upon vertebrates, they can also act as efficient samplers that facilitate detection of vertebrate species that utilize urban ecosystems. In this study, we analyzed DNA extracted from mosquito blood meals collected temporally in multiple neighborhoods of the San Juan Metropolitan Area, Puerto Rico to evaluate the presence of vertebrate fauna. DNA was collected from 604 individual mosquitoes that represented two common urban species, <i>Culex quinquefasciatus</i> (<i>n</i> = 586) and <i>Aedes aegypti</i> (<i>n</i> = 18). <i>Culex quinquefasciatus</i> fed on 17 avian taxa (81.2% of blood meals), seven mammalian taxa (17.9%), and one reptilian taxon (0.85%). Domestic chickens dominated these blood meals both temporally and spatially, and no statistically significant shift from birds to mammals was detected. <i>Aedes aegypti</i> blood meals were from a less diverse group, with two avian taxa (11.1%) and three mammalian taxa (88.9%) identified. The blood meals we identified provided a snapshot of the vertebrate community in the San Juan Metropolitan Area and have potential implications for vector-borne pathogen transmission.</p>
<h1>The influence of intraspecific sequence variation during DNA metabarcoding: A case study of eleven fungal species.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13329">doi:10.1111/1755-0998.13329</a></p>
['DNA metabarcoding', 'ITS', 'community ecology', 'fungi', 'haplotypes']<p>DNA metabarcoding has become a powerful approach for analysing complex communities from environmental samples, but there are still methodological challenges limiting its full potential. While conserved DNA markers, like 16S and 18S, often are not able to discriminate among closely related species, other more variable markers - like the fungal ITS region, may include considerable intraspecific variation, which can lead to oversplitting of species during DNA metabarcoding analyses. Here we assessed the effects of intraspecific sequence variation in DNA metabarcoding by analysing local populations of eleven fungal species. We investigated the allelic diversity of ITS2 haplotypes using both Sanger sequencing and high throughput sequencing (HTS) coupled with error correction with the software dada2. All the eleven species, except one, included some level of intraspecific variation in the ITS2 region. Overall, we observed a high correspondence between haplotypes generated by Sanger sequencing and HTS, with the exception of a few additional haplotypes detected using either approach. These extra haplotypes, typically occurring in low frequencies, were probably due to PCR and sequencing errors or intragenomic variation in the rDNA region. The presence of intraspecific (and possibly intragenomic) variation in ITS2 suggest that haplotypes (or ASVs) should not be used as basic units in ITS-based fungal community analyses, but an extra clustering step is needed to approach species-level resolution.</p>
<h1>A pilot study of eDNA metabarcoding to estimate plant biodiversity by an alpine glacier core (Adamello glacier, North Italy).</h1>
<p><a href="https://doi.org/10.1038/S41598-020-79738-5">doi:10.1038/S41598-020-79738-5</a></p>
<p>No keywords</p><p>Current biodiversity loss is a major concern and thus biodiversity assessment of modern ecosystems is compelling and needs to be contextualized on a longer timescale. High Throughput Sequencing (HTS) is progressively becoming a major source of data on biodiversity time series. In this multi proxy study, we tested, for the first time, the potential of HTS to estimate plant biodiversity archived in the surface layers of a temperate alpine glacier, amplifying the trnL barcode for vascular plants from eDNA of firn samples. A 573 cm long core was drilled by the Adamello glacier and cut into sections; produced samples were analyzed for physical properties, stable isotope ratio, and plant biodiversity by eDNA metabarcoding and conventional light microscopy analysis. Results highlighted the presence of pollen and plant remains within the distinct layers of snow, firn and ice. While stable isotope ratio showed a scarcely informative pattern, DNA metabarcoding described distinct plant species composition among the different samples, with a broad taxonomic representation of the biodiversity of the catchment area and a high-ranking resolution. New knowledge on climate and plant biodiversity changes of large catchment areas can be obtained by this novel approach, relevant for future estimates of climate change effects.</p>
<h1>A multi-kingdom metabarcoding study on cattle grazing Alpine pastures discloses intra-seasonal shifts in plant selection and faecal microbiota.</h1>
<p><a href="https://doi.org/10.1038/S41598-020-79474-W">doi:10.1038/S41598-020-79474-W</a></p>
<p>No keywords</p><p>Diet selection by grazing livestock may affect animal performance as well as the biodiversity of grazed areas. Recent DNA barcoding techniques allow to assess dietary plant composition in faecal samples, which may be additionally integrated by the description of gut microbiota. In this high throughput metabarcoding study, we investigated the diversity of plant, fungal and bacterial taxa in faecal samples of lactating cows of two breeds grazing an Alpine semi-natural grassland during summer. The estimated plant composition of the diet comprised 67 genera and 39 species, which varied remarkably during summer, suggesting a decline of the diet forage value with the advancing of the vegetative season. The fungal community included Neocallimastigomycota gut symbionts, but also Ascomycota and Basidiomycota plant parasite and coprophilous taxa, likely ingested during grazing. The proportion of ingested fungi was remarkably higher than in other studies, and varied during summer, although less than that observed for plants. Some variation related to breed was also detected. The gut bacterial taxa remained stable through the summer but displayed a breed-specific composition. The study provided insights in the reciprocal organisms' interactions affecting, and being affected by, the foraging behaviour: plants showed a high temporal variation, fungi a smaller one, while bacteria had practically none; conversely, the same kingdoms showed the opposite gradient of variation as respect to the animal host breed, as bacteria revealed to be the group mostly characterized by host-specificity.</p>
<h1>Connecting high-throughput biodiversity inventories: Opportunities for a site-based genomic framework for global integration and synthesis.</h1>
<p><a href="https://doi.org/10.1111/Mec.15797">doi:10.1111/Mec.15797</a></p>
['DNA metabarcoding', 'Genomic Observatories', 'biodiversity assessment', 'harmonized data generation', 'high-throughput sequencing']<p>High-throughput sequencing (HTS) is increasingly being used for the characterization and monitoring of biodiversity. If applied in a structured way, across broad geographical scales, it offers the potential for a much deeper understanding of global biodiversity through the integration of massive quantities of molecular inventory data generated independently at local, regional and global scales. The universality, reliability and efficiency of HTS data can potentially facilitate the seamless linking of data among species assemblages from different sites, at different hierarchical levels of diversity, for any taxonomic group and regardless of prior taxonomic knowledge. However, collective international efforts are required to optimally exploit the potential of site-based HTS data for global integration and synthesis, efforts that at present are limited to the microbial domain. To contribute to the development of an analogous strategy for the nonmicrobial terrestrial domain, an international symposium entitled "Next Generation Biodiversity Monitoring" was held in November 2019 in Nicosia (Cyprus). The symposium brought together evolutionary geneticists, ecologists and biodiversity scientists involved in diverse regional and global initiatives using HTS as a core tool for biodiversity assessment. In this review, we summarize the consensus that emerged from the 3-day symposium. We converged on the opinion that an effective terrestrial Genomic Observatories network for global biodiversity integration and synthesis should be spatially led and strategically united under the umbrella of the metabarcoding approach. Subsequently, we outline an HTS-based strategy to collectively build an integrative framework for site-based biodiversity data generation.</p>
<h1>Genotype by sequencing: An alternative new method to amplicon metabarcoding and shotgun metagenomics for the assessment of eukaryote biodiversity.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13320">doi:10.1111/1755-0998.13320</a></p>
<p>No keywords</p><p>The use of high-throughput DNA sequencing (HTS) has revolutionized the assessment of biodiversity in plant and animal communities. There are two main approaches to estimate the identity and the relative species abundance (RSA) in complex mixtures using HTS: amplicon metabarcoding and shotgun metagenomics. While amplicon metabarcoding targets one or a few genomic regions, shotgun metagenomics randomly explores the genome of the species. In this issue of Molecular Ecology Resources, Wagemaker et al. (2021) present a new method, multi-species Genotyping by Sequencing (msGBS), as an alternative middle ground between metabarcoding and metagenomics. They apply the technique to mixtures of plant roots and report the remarkable capacity of msGBS to estimate the RSA. If validated in other laboratories and biological communities, msGBS might become a third method to explore the biodiversity of biological communities, especially of plants, where current techniques are struggling to get sufficient taxonomic resolution.</p>
<h1>Assessment of fungal diversity present in lakes of Maritime Antarctica using DNA metabarcoding: a temporal microcosm experiment.</h1>
<p><a href="https://doi.org/10.1007/S00792-020-01212-X">doi:10.1007/S00792-020-01212-X</a></p>
['Antarctica', 'Freshwater', 'Fungi', 'Ingoldian', 'Pounds']<p>We evaluated the fungal diversity in two lakes on the South Shetland Islands, using DNA metabarcoding through high-throughput sequencing (HTS). A microcosm experiment was deployed for two consecutive years in lakes on Deception and King George islands to capture potential decomposer freshwater fungi. Analyses of the baits revealed 258,326 DNA reads distributed in 34 fungal taxa of the phyla Ascomycota, Basidiomycota, Mortierellomycota, Chytridiomycota and Rozellomycota. Tetracladium marchalianum, Tetracladium sp., Rozellomycota sp., Fungal sp. 1 and Fungal sp. 2 were the most common taxa detected. However, the majority of the communities comprised intermediate and rare taxa. Both fungal communities displayed moderate indices of diversity, richness and dominance. Only six taxa were detected in both lakes, including the most dominant T. marchalianum and Tetracladium sp. The high numbers of reads of the known aquatic saprotrophic hyphomycetes T. marchalianum and Tetracladium sp. in the baits suggest that these fungi may digest organic material in Antarctic lakes, releasing available carbon and nutrients to the other aquatic organisms present in the complex lake food web. Our data confirm that the use of cotton baits together with HTS approaches can be appropriate to study the diversity of resident freshwater fungi present in Antarctic lakes.</p>
<h1>From Forest Soil to the Canopy: Increased Habitat Diversity Does Not Increase Species Richness of Cercozoa and Oomycota in Tree Canopies.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2020.592189">doi:10.3389/Fmicb.2020.592189</a></p>
['canopies', 'forest ecosystems', 'habitat filtering', 'illumina', 'metabarcoding', 'protists']<p>Tree canopies provide habitats for diverse and until now, still poorly characterized communities of microbial eukaryotes. One of the most general patterns in community ecology is the increase in species richness with increasing habitat diversity. Thus, environmental heterogeneity of tree canopies should be an important factor governing community structure and diversity in this subsystem of forest ecosystems. Nevertheless, it is unknown if similar patterns are reflected at the microbial scale within unicellular eukaryotes (protists). In this study, high-throughput sequencing of two prominent protistan taxa, Cercozoa (Rhizaria) and Oomycota (Stramenopiles), was performed. Group specific primers were used to comprehensively analyze their diversity in various microhabitats of a floodplain forest from the forest floor to the canopy region. Beta diversity indicated highly dissimilar protistan communities in the investigated microhabitats. However, the majority of operational taxonomic units (OTUs) was present in all samples, and therefore differences in beta diversity were mainly related to species performance (i.e., relative abundance). Accordingly, habitat diversity strongly favored distinct protistan taxa in terms of abundance, but due to their almost ubiquitous distribution the effect of species richness on community composition was negligible.</p>
<h1>Progress in the use of DNA barcodes in the identification and classification of medicinal plants.</h1>
<p><a href="https://doi.org/S0147-6513(20)31528-1">doi:S0147-6513(20)31528-1</a></p>
['Bar-HRM', 'Chloroplast genome', 'DNA barcodes', 'Medicinal plants', 'Metabarcoding']<p>DNA barcoding is an emerging molecular identification and classification technology that has been applied to medicinal plants since 2008. The application of this technique has greatly ensured the safety and effectiveness of medicinal materials. In this paper, we review the application of DNA barcoding and some related technologies over the past 10 years with respect to improving our knowledge of medicinal plant identification and authentication. From single locus-based DNA barcodes to combined markers to genome-scale levels, DNA barcodes contribute more and more genetic information. At the same time, other technologies, such as high-resolution melting (HRM), have been combined with DNA barcoding. With the development of next-generation sequencing (NGS), metabarcoding technology has also been shown to identify species in mixed samples successfully. As a widely used and effective tool, DNA barcoding will become more useful over time in the field of medicinal plants.</p>
<h1>Vertebrate-Aedes aegypti and Culex quinquefasciatus (Diptera)-arbovirus transmission networks: Non-human feeding revealed by meta-barcoding and next-generation sequencing.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pntd.0008867">doi:10.1371/Journal.Pntd.0008867</a></p>
<p>No keywords</p><p>Aedes aegypti mosquito-borne viruses including Zika (ZIKV), dengue (DENV), yellow fever (YFV), and chikungunya (CHIKV) have emerged and re-emerged globally, resulting in an elevated burden of human disease. Aedes aegypti is found worldwide in tropical, sub-tropical, and temperate areas. The characterization of mosquito blood meals is essential to understand the transmission dynamics of mosquito-vectored pathogens.</p>
<h1>Metabarcoding on both environmental DNA and RNA highlights differences between fungal communities sampled in different habitats.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0244682">doi:10.1371/Journal.Pone.0244682</a></p>
<p>No keywords</p><p>In recent years, metabarcoding has become a key tool to describe microbial communities from natural and artificial environments. Thanks to its high throughput nature, metabarcoding efficiently explores microbial biodiversity under different conditions. It can be performed on environmental (e)DNA to describe so-called total microbial community, or from environmental (e)RNA to describe active microbial community. As opposed to total microbial communities, active ones exclude dead or dormant organisms. For what concerns Fungi, which are mostly filamentous microorganisms, the relationship between DNA-based (total) and RNA-based (active) communities is unclear. In the present study, we evaluated the consequences of performing metabarcoding on both soil and wood-extracted eDNA and eRNA to delineate molecular operational taxonomic units (MOTUs) and differentiate fungal communities according to the environment they originate from. DNA and RNA-based communities differed not only in their taxonomic composition, but also in the relative abundances of several functional guilds. From a taxonomic perspective, we showed that several higher taxa are globally more represented in either "active" or "total" microbial communities. We also observed that delineation of MOTUs based on their co-occurrence among DNA and RNA sequences highlighted differences between the studied habitats that were overlooked when all MOTUs were considered, including those identified exclusively by eDNA sequences. We conclude that metabarcoding on eRNA provides original functional information on the specific roles of several taxonomic or functional groups that would not have been revealed using eDNA alone.</p>
<h1>Changes in soil microbial communities in post mine ecological restoration: Implications for monitoring using high throughput DNA sequencing.</h1>
<p><a href="https://doi.org/S0048-9697(20)35791-0">doi:S0048-9697(20)35791-0</a></p>
['Ecological restoration', 'Land degradation', 'Restoration genomics', 'Soil microbial communities', 'eDNA']<p>The ecological restoration of ecosystem services and biodiversity is a key intervention used to reverse the impacts of anthropogenic activities such as mining. Assessment of the performance of restoration against completion criteria relies on biodiversity monitoring. However, monitoring usually overlooks soil microbial communities (SMC), despite increased awareness of their pivotal role in many ecological functions. Recent advances in cost, scalability and technology has led to DNA sequencing being considered as a cost-effective biological monitoring tool, particularly for otherwise difficult to survey groups such as microbes. However, such approaches for monitoring complex restoration sites such as post-mined landscapes have not yet been tested. Here we examine bacterial and fungal communities across chronosequences of mine site restoration at three locations in Western Australia to determine if there are consistent changes in SMC diversity, community composition and functional capacity. Although we detected directional changes in community composition indicative of microbial recovery, these were inconsistent between locations and microbial taxa (bacteria or fungi). Assessing functional diversity provided greater understanding of changes in site conditions and microbial recovery than could be determined through assessment of community composition alone. These results demonstrate that high-throughput amplicon sequencing of environmental DNA (eDNA) is an effective approach for monitoring the complex changes in SMC following restoration. Future monitoring of mine site restoration using eDNA should consider archiving samples to provide improved understanding of changes in communities over time. Expansion to include other biological groups (e.g. soil fauna) and substrates would also provide a more holistic understanding of biodiversity recovery.</p>
<h1>Monitoring and Surveillance of Aerial Mycobiota of Rice Paddy through DNA Metabarcoding and qPCR.</h1>
<p><a href="https://doi.org/E372">doi:E372</a></p>
['Bipolaris', 'DNA metabarcoding', 'ITS', 'Pyricularia', 'airborne', 'rice', 'spore trap']<p>The airborne mycobiota has been understudied in comparison with the mycobiota present in other agricultural environments. Traditional, culture-based methods allow the study of a small fraction of the organisms present in the atmosphere, thus missing important information. In this study, the aerial mycobiota in a rice paddy has been examined during the cropping season (from June to September 2016) using qPCRs for two important rice pathogens (<i>Pyricularia oryzae</i> and <i>Bipolaris oryzae</i>) and by using DNA metabarcoding of the fungal ITS region. The metabarcoding results demonstrated a higher alpha diversity (Shannon-Wiener diversity index H' and total number of observed species) at the beginning of the trial (June), suggesting a higher level of community complexity, compared with the end of the season. The main taxa identified by HTS analysis showed a shift in their relative abundance that drove the cluster separation as a function of time and temperature. The most abundant OTUs corresponded to genera such as <i>Cladosporium</i>, <i>Alternaria, Myrothecium</i>, or <i>Pyricularia</i>. Changes in the mycobiota composition were clearly dependent on the average air temperature with a potential impact on disease development in rice. In parallel, oligotyping analysis was performed to obtain a sub-OTU identification which revealed the presence of several oligotypes of <i>Pyricularia</i> and <i>Bipolaris</i> with relative abundance changing during monitoring.</p>
<h1>Taxonomic Diversity of Pico-/Nanoeukaryotes Is Related to Dissolved Oxygen and Productivity, but Functional Composition Is Shaped by Limiting Nutrients in Eutrophic Coastal Oceans.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2020.601037">doi:10.3389/Fmicb.2020.601037</a></p>
['functional redundancy', 'metabarcoding', 'mixotrophy', 'nutrient limitation', 'productivity']<p>Pico-/nanoeukaryotes (P/NEs) comprise both primary producers and bacterial predators, playing important biogeochemical and ecological roles in the marine microbial loop. Besides the difference in size, these small-sized fractions can be distinguished from microplankton by certain functional and ecological traits. Nevertheless, little information is available regarding patterns of their taxonomic and functional diversity and community composition along environmental gradients in coastal marine ecosystems. In this study, we applied high-throughput sequencing of 18S rRNA gene to assess the taxonomic species richness and community composition of P/NEs in surface waters of Bohai Sea and North Yellow Sea, northern China spanning a 600-km distance during summer and winter of 2011. The richness of operational taxonomic units (OTUs) formed a U-shaped relationship with concentration of chlorophyll <i>a</i> (Chl-<i>a</i>, a proxy of primary productivity), but a stronger, negative relationship with concentration of dissolved oxygen (DO). These two factors also significantly co-varied with the OTU-based community composition of P/NEs. The effect of geographic distance on community composition of P/NEs was negligible. Among the three functional groups defined by trophic traits, heterotrophs had the highest OTU richness, which exhibited a U-shaped relationship with both DO and Chl-<i>a</i>. The community of P/NEs was dominated by heterotrophs and mixotrophs in terms of read numbers, which showed a trade-off along the gradient of phosphate, but no significant changes along DO and Chl-<i>a</i> gradients, indicating functional redundancy. Similarly, the proportion of phototrophs was significantly and positively correlated with the concentration of silicate. Our results indicate that taxonomic and functional composition of P/NEs are decoupled on a regional scale, and limiting nutrients are important factors in modulating functional composition of these microorganisms in the studied area. These findings contribute toward gaining a better understanding of how diversity of small eukaryotes and their functions are structured in coastal oceans and the effect of environmental changes on the structuring process.</p>
<h1>Food Resource Sharing of Alder Leaf Beetle Specialists (Coleoptera: Chrysomelidae) as Potential Insect-Plant Interface for Horizontal Transmission of Endosymbionts.</h1>
<p><a href="https://doi.org/10.1093/Ee/Nvaa111">doi:10.1093/Ee/Nvaa111</a></p>
['\n          Rickettsia\n        ', '\n          Wolbachia\n        ', 'biodiversity', 'natural transmission', 'trophic network']<p>Recent studies suggest that endosymbionts of herbivore insects can be horizontally transferred to other herbivores feeding on the same host plants, whereby the plant acts as an intermediate stage in the chain of transmission. If this mechanism operates, it is also expected that insect communities sharing the same host plant will have higher chances to share their endosymbionts. In this study, we use a high-throughput 16S rRNA metabarcoding approach to investigate the presence, diversity, and potential sharing of endosymbionts in several species of leaf beetles (Coleoptera: Chrysomelidae) of a local community specialized on an alder diet in North America. Rickettsia and Wolbachia were predominant in the sample, with strong evidence for each species having their own dominant infection, of either or both types of bacteria. However, all species shared a much lower proportion of a particular Wolbachia type, compatible with the same strain dominant in one of the species of leaf beetles. Crucially, the same 16S rRNA haplotype of Wolbachia was found on alder leaf extracts. The combined evidence and the absence of this strain in a syntopic species of leaf beetle feeding on a different host plant support the hypothesis that at least the initial stages of the mechanism that would allow horizontal transmission of endosymbionts across species feeding on the same plant is possible. The accessibility and characteristics of endosymbiont associations of this system make it suitable for deeper analyses of their diversity and transmission in natural conditions.</p>
<h1>The pico-sized Mamiellophyceae and a novel Bathycoccus clade from the summer plankton of Russian Arctic Seas and adjacent waters.</h1>
<p><a href="https://doi.org/Fiaa251">doi:Fiaa251</a></p>
['\n          Bathycoccus\n        ', '\n          Micromonas\n        ', '\n          Ostreococcus\n        ', 'photosynthetic picoeukaryotes', 'the Barents Sea', 'the Kara Sea', 'the Laptev Sea', 'the Norwegian Sea']<p>Global climate changes and anthropogenic activity greatly impact Arctic marine biodiversity including phytoplankton which contribute greatly to atmospheric oxygen production. Thus the study of microalgae has rising topicality. Class Mamiellophyceae is an important component of phototrophic picoplankton. To gain more knowledge about Mamiellophyceae distribution and diversity special studies were performed in such remote areas as the Russian Arctic seas. A metabarcoding of pico-sized Mamiellophyceae was undertaken by high-throughput sequencing of the 18S rRNA gene sequence V4 region from samples collected in July-September 2017 in the Barents, Kara and Laptev seas, and in the adjacent waters of the Norwegian Sea. Our study is the first to show that Mamiellophyceae among the summer picoplankton of Russian Arctic seas are diverse and represented by 16 algae species/phylotypes. We discovered a new candidate species of Bathycoccus assigned to a new Bathycoccus clade A-uncultured Bathycoccus Kara 2017. It was found that several Micromonas species can co-exist, with Micromonas polaris dominating north of 72°N. The presence of Ostreococcus tauri, Ostreococcus lucimarinus and Ostreococcus mediterraneus at high latitudes beyond 65°N was documented for the first time, similar to findings for some other taxa. Our results will be important for obtaining a global view of Mamiellophyceae community dynamics.</p>
<h1>DNA diet profiles with high-resolution animal tracking data reveal levels of prey selection relative to habitat choice in a crepuscular insectivorous bird.</h1>
<p><a href="https://doi.org/10.1002/Ece3.6893">doi:10.1002/Ece3.6893</a></p>
['DNA metabarcoding', 'food availability', 'foraging ecology', 'high‐throughput sequencing', 'lepidoptera']<p>Given the global decline of many invertebrate food resources, it is fundamental to understand the dietary requirements of insectivores. We give new insights into the functional relationship between the spatial habitat use, food availability, and diet of a crepuscular aerial insectivore, the European Nightjar (<i>Caprimulgus europaeus</i>) by relating spatial use data with high-throughput sequencing (HTS) combined with DNA metabarcoding. Our study supports the predictions that nightjars collect a substantial part of their daily nourishment from foraging locations, sometimes at considerable distance from nesting sites. Lepidopterans comprise 65% of nightjars' food source. Nightjars tend to select larger species of <i>Lepidoptera</i> (>19 mm) which suggests that nightjars optimize the efficiency of foraging trips by selecting the most energetically favorable-larger-prey items. We anticipate that our findings may shed additional light on the interactions between invertebrate communities and higher trophic levels, which is required to understand the repercussions of changing food resources on individual- and population-level processes.</p>
<h1>Efficient Monitoring of Adult and Immature Mosquitoes Through Metabarcoding of Bulk Samples: A Case Study for Non-Model Culicids With Unique Ecologies.</h1>
<p><a href="https://doi.org/10.1093/Jme/Tjaa267">doi:10.1093/Jme/Tjaa267</a></p>
['\n          Mansonia spp', 'mass-sampling', 'metabarcoding', 'mosquito monitoring', 'ribosomal DNA']<p>The rapid and economical monitoring of mosquitos is imperative to understanding the dynamics of both disease vectors and nuisance species. In light of technological advances in mosquito sampling and DNA sequencing, health agencies can now utilize the full potential of metabarcoding pipelines for rapid and standardizable surveillance. Here, we describe mosquito spatial and temporal variation, with particular focus on Mansonia Blanchard species, in the Madeira (Rondônia State) and the Ribeira (São Paulo) watersheds, Brazil using metabarcoding of the D2 rDNA marker. Sampling and molecular pipelines were used to evaluate the taxonomic contribution of mosquitos in pools of culicids collected en masse from macrophyte-roots (immatures) and from Mosquito Magnet traps and protected human landings (adults). Results for adult captures are comparable to morphological diagnoses and clarify previously unknown temporal and spatial species turnover. Metabarcoding of immature stages also confirmed the extent of the geographical distribution of some species and each taxon's association with macrophyte species. Given the benefits of metabarcoding, such as taxonomic acuity, high throughput processing, and objectivity, we suggest such techniques should be more fully incorporated into culicid monitoring schemes. The metabarcoding protocol described herein paired with standardized field sampling schemes, when used by mosquito monitoring professionals, offers substantial improvements in terms of practicality, speed and cost.</p>
<h1>Monitoring of spatiotemporal occupancy patterns of fish and amphibian species in a lentic aquatic system using environmental DNA.</h1>
<p><a href="https://doi.org/10.1111/Mec.15742">doi:10.1111/Mec.15742</a></p>
['eDNA', 'high-throughput DNA sequencing', 'metabarcoding', 'monitoring', 'relative abundance', 'water sampling']<p>To effectively monitor, manage and protect aquatic species and understand their interactions, knowledge of their spatiotemporal distribution is needed. In this study, we used a fine-scale spatiotemporal water sampling design, followed by environmental DNA (eDNA) 12S metabarcoding, to investigate occupancy patterns of a natural community of fish and amphibian species in a lentic system. In the same system, we experimentally estimated the spatial and temporal dispersion of eDNA by placing a community of different fish and amphibian species in cages at one side of the pond, creating a controlled point of eDNA emission. Analyses of this cage community revealed a sharp spatial decline in detection rates and relative eDNA quantities at a distance of 5-10 m from the source, depending on the species and its abundance. In addition, none of the caged species could be detected 1 week after removal from the system. This indicates high eDNA decay rates and limited spatial eDNA dispersal, facilitating high local resolution for monitoring spatial occupancy patterns of aquatic species. Remarkably, for seven of the nine cage species, the presence of a single individual could be detected by pooling water of subsamples taken across the whole water body, illustrating the high sensitivity of the eDNA sampling and detection method applied. Finally, our work demonstrated that a fine-scale sampling design in combination with eDNA metabarcoding can cover total biodiversity very precisely and allows the construction of consistent spatiotemporal patterns of relative abundance and local distribution of free-living fish and amphibian species in a lentic ecosystem.</p>
<h1>Protist taxonomic and functional diversity in soil, freshwater and marine ecosystems.</h1>
<p><a href="https://doi.org/S0160-4120(20)32217-0">doi:S0160-4120(20)32217-0</a></p>
['Metabarcoding', 'Microbial eukaryotes', 'Ocean', 'Taxonomic and functional diversity', 'Terrestrial systems']<p>Protists dominate eukaryotic diversity and play key functional roles in all ecosystems, particularly by catalyzing carbon and nutrient cycling. To date, however, a comparative analysis of their taxonomic and functional diversity that compares the major ecosystems on Earth (soil, freshwater and marine systems) is missing. Here, we present a comparison of protist diversity based on standardized high throughput 18S rRNA gene sequencing of soil, freshwater and marine environmental DNA. Soil and freshwater protist communities were more similar to each other than to marine protist communities, with virtually no overlap of Operational Taxonomic Units (OTUs) between terrestrial and marine habitats. Soil protists showed higher γ diversity than aquatic samples. Differences in taxonomic composition of the communities led to changes in a functional diversity among ecosystems, as expressed in relative abundance of consumers, phototrophs and parasites. Phototrophs (eukaryotic algae) dominated freshwater systems (49% of the sequences) and consumers soil and marine ecosystems (59% and 48%, respectively). The individual functional groups were composed of ecosystem- specific taxonomic groups. Parasites were equally common in all ecosystems, yet, terrestrial systems hosted more OTUs assigned to parasites of macro-organisms while aquatic systems contained mostly microbial parasitoids. Together, we show biogeographic patterns of protist diversity across major ecosystems on Earth, preparing the way for more focused studies that will help understanding the multiple roles of protists in the biosphere.</p>
<h1>Freshwater diatom biomonitoring through benthic kick-net metabarcoding.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0242143">doi:10.1371/Journal.Pone.0242143</a></p>
<p>No keywords</p><p>Biomonitoring is an essential tool for assessing ecological conditions and informing management strategies. The application of DNA metabarcoding and high throughput sequencing has improved data quantity and resolution for biomonitoring of taxa such as macroinvertebrates, yet, there remains the need to optimise these methods for other taxonomic groups. Diatoms have a longstanding history in freshwater biomonitoring as bioindicators of water quality status. However, multi-substrate periphyton collection, a common diatom sampling practice, is time-consuming and thus costly in terms of labour. This study examined whether the benthic kick-net technique used for macroinvertebrate biomonitoring could be applied to bulk-sample diatoms for metabarcoding. To test this approach, we collected samples using both conventional multi-substrate microhabitat periphyton collections and bulk-tissue kick-net methodologies in parallel from replicated sites with different habitat status (good/fair). We found there was no significant difference in community assemblages between conventional periphyton collection and kick-net methodologies or site status, but there was significant difference between diatom communities depending on site (P = 0.042). These results show the diatom taxonomic coverage achieved through DNA metabarcoding of kick-net is suitable for ecological biomonitoring applications. The shift to a more robust sampling approach and capturing diatoms as well as macroinvertebrates in a single sampling event has the potential to significantly improve efficiency of biomonitoring programmes that currently only use the kick-net technique to sample macroinvertebrates.</p>
<h1>Natrix: a Snakemake-based workflow for processing, clustering, and taxonomically assigning amplicon sequencing reads.</h1>
<p><a href="https://doi.org/10.1186/S12859-020-03852-4">doi:10.1186/S12859-020-03852-4</a></p>
['Amplicon Sequence Variants', 'Amplicon sequencing', 'Illumina', 'Operational Taxonomic Units', 'Pipline', 'Snakemake']<p>Sequencing of marker genes amplified from environmental samples, known as amplicon sequencing, allows us to resolve some of the hidden diversity and elucidate evolutionary relationships and ecological processes among complex microbial communities. The analysis of large numbers of samples at high sequencing depths generated by high throughput sequencing technologies requires efficient, flexible, and reproducible bioinformatics pipelines. Only a few existing workflows can be run in a user-friendly, scalable, and reproducible manner on different computing devices using an efficient workflow management system.</p>
<h1>Decision-making and best practices for taxonomy-free environmental DNA metabarcoding in biomonitoring using Hill numbers.</h1>
<p><a href="https://doi.org/10.1111/Mec.15725">doi:10.1111/Mec.15725</a></p>
['\nCOI\n', 'biodiversity monitoring', 'diversity measures', 'environmental DNA', 'freshwater', 'high-throughput sequencing']<p>Environmental DNA (eDNA) metabarcoding is raising expectations for biomonitoring of organisms that have hitherto been neglected. To bypass current limitations in taxonomic assignments due to incomplete or erroneous reference databases, taxonomy-free approaches are proposed for biomonitoring at the level of operational taxonomic units (OTUs). This is challenging, because OTUs cannot be annotated and directly compared against classically derived taxonomic data. The application of good stringency treatments to infer the validity of OTUs and clear understanding of the consequences of such treatments is especially relevant for biodiversity assessments. We investigated how common practices of stringency filtering affect eDNA diversity estimates in the statistical framework of Hill numbers. We collected water eDNA samples at 61 sites across a 740-km<sup>2</sup> river catchment, reflecting a spatially realistic scenario in biomonitoring. After bioinformatic processing of the data, we studied how different stringency treatments affect conclusions with respect to biodiversity at the catchment and site levels. The applied stringency treatments were based on the consistent appearance of OTUs across filter replicates, a relative abundance cut-off and rarefaction. We detected large differences in diversity estimates when accounting for presence/absence only, such that detected diversity at the catchment scale differed by an order of magnitude between the treatments. These differences disappeared when using stringency treatments with increasing weighting of the OTU abundances. Our study demonstrated the usefulness of Hill numbers for biodiversity analyses and comparisons of eDNA data sets that strongly differ in diversity. We recommend best practice for data stringency filtering for biomonitoring using eDNA.</p>
<h1>Assessment of microbial plankton diversity as an ecological indicator in the NW Mediterranean coast.</h1>
<p><a href="https://doi.org/S0025-326X(20)30809-2">doi:S0025-326X(20)30809-2</a></p>
['Anthropogenic pressures', 'Coastal marine monitoring', 'Ecological indicators', 'Eutrophication', 'High-throughput sequencing', 'Plankton diversity']<p>High-throughput sequencing of microbial assemblages has been proposed as an alternative methodology to the traditional ones used in marine monitoring and environmental assessment. Here, we evaluated pico- and nanoplankton diversity as ecological indicators in NW Mediterranean coastal waters by comparing their diversity in samples subjected to varying degrees of continental pressures. Using metabarcoding of the 16S and 18S rRNA genes, we explored whether alphadiversity indices, abundance of Operational Taxonomic Units and taxonomic groups (and their ratios) provide information on the ecological quality of coastal waters. Our results revealed that only eukaryotic diversity metrics and a limited number of prokaryotic and eukaryotic taxa displayed potential in assessing continental influences in our surveyed area, resulting thus in a restrained potential of microbial plankton diversity as an ecological indicator. Therefore, incorporating microbial plankton diversity in environmental assessment could not always result in a significant improvement of current marine monitoring strategies.</p>
<h1>Massively parallel sequencing is unlocking the potential of environmental trace evidence.</h1>
<p><a href="https://doi.org/S1872-4973(20)30165-4">doi:S1872-4973(20)30165-4</a></p>
['Botany', 'DNA', 'Forensic', 'MPS', 'Metabarcoding', 'Microbial', 'Pollen', 'Soil', 'Trace evidence']<p>Massively parallel sequencing (MPS) has revolutionised the field of genomics enabling substantial advances in human DNA profiling. Further, the advent of MPS now allows biological signatures to be obtained from complex DNA mixtures and trace amounts of low biomass samples. Environmental samples serve as ideal forms of contact trace evidence as detection at a scene can establish a link between a suspect, location and victim. Many studies have applied MPS technology to characterise the biodiversity within high biomass environmental samples (such as soil and water) to address questions related to ecology, conservation, climate change and human health. However, translation of these tools to forensic science remains in its infancy, due in part to the merging of traditional forensic ecology practices with unfamiliar DNA technologies and complex datasets. In addition, people and objects also carry low biomass environmental signals which have recently been shown to reflect a specific individual or location. The sensitivity, and reducing cost, of MPS is now unlocking the power of both high and low biomass environmental DNA (eDNA) samples as useful sources of genetic information in forensic science. This paper discusses the potential of eDNA to forensic science by reviewing the most explored applications that are leading the integration of this technology into the field. We introduce novel areas of forensic ecology that could also benefit from these tools with a focus on linking a suspect to a scene or establishing provenance of an unknown sample and discuss the current limitations and validation recommendations to achieve translation of eDNA into casework.</p>
<h1>BIOCOM-PIPE: a new user-friendly metabarcoding pipeline for the characterization of microbial diversity from 16S, 18S and 23S rRNA gene amplicons.</h1>
<p><a href="https://doi.org/10.1186/S12859-020-03829-3">doi:10.1186/S12859-020-03829-3</a></p>
['Archaeal', 'Bacterial', 'Ecology', 'France', 'Fungal', 'Land-use', 'Metabarcoding', 'Photosynthetic microeukaryotes', 'ReClustOR']<p>The ability to compare samples or studies easily using metabarcoding so as to better interpret microbial ecology results is an upcoming challenge. A growing number of metabarcoding pipelines are available, each with its own benefits and limitations. However, very few have been developed to offer the opportunity to characterize various microbial communities (e.g., archaea, bacteria, fungi, photosynthetic microeukaryotes) with the same tool.</p>
<h1>Assessment of littoral algal diversity from the northern Gulf of Mexico using environmental DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/Jpy.13087">doi:10.1111/Jpy.13087</a></p>
['alien species', 'biodiversity', 'bioinvasions', 'eDNA', 'intertidal algae', 'introduced species', 'metabarcoding', 'nonindigenous algae', 'species detection', 'species richness']<p>Traditional methods for algal biodiversity monitoring are costly and time inefficient because they rely on high-level taxonomic expertise to address species identity problems involving phenotypic plasticity and morphological convergence. These problems are exacerbated in regions such as the Gulf of Mexico, that has a limited history of phycological exploration, but that are economically important or threatened by numerous anthropogenic stressors. Given the high pace of disturbance to natural systems, there is a critical need for expedient and cost-effective tools for the study of benthic algal communities. Here we document the use of environmental DNA metabarcoding, using the partial LSU rDNA and 23S rDNA plastid molecular markers, to elucidate littoral algal diversity in the Northern Gulf of Mexico. We assigned 73.7% of algal OTUs to genus and 59.6% to species ranks. Our current study detected molecular signals for 35 algal/protist species with no previous reports in the Gulf of Mexico, thus providing an important, molecular-validated, baseline of species richness for this region. We also make several bioinformatic recommendations for the efficient use of high-throughput sequence data to assess biological communities.</p>
<h1>Environmental drivers affecting the bacterial community of intertidal sediments in the Yellow Sea.</h1>
<p><a href="https://doi.org/S0048-9697(20)36255-0">doi:S0048-9697(20)36255-0</a></p>
['Bacterial community', 'Biodiversity', 'Indicator species analysis', 'Machine learning', 'Rhizosphere', 'Tidal marsh']<p>Intertidal flats, as transition zones where terrestrial and marine ecosystems meet, provide unique environments and play an important role in marine ecosystems. In particular, the environmental characteristics of tidal marshes show are different than those of bare flats, especially in the rhizosphere. However, unlike the rhizosphere in terrestrial ecosystems, the rhizosphere of plants in tidal marsh areas and the associated microbial community have been the focus of very little research. Thus, this study investigated the diversity and variation in bacterial communities in the rhizosphere of a Phragmites australis and Suaeda japonica and along the sediment depths. High-throughput sequencing was performed by amplifying the 16S rRNA gene of environmental DNA extracted from sediment cores, and indicator species were identified with respect to the vegetation type and sediment depth. The most abundant phylum was Proteobacteria, followed by Chloroflexi, Bacteroidetes, Acidobacteria, and Firmicutes. In general, the results indicated that not only vegetation type and sediment depth themselves but also their interaction resulted in significant differences among the bacterial communities. The envfit results revealed that the environmental variables of sediment, such as mud content, organic matter, total organic carbon, and total nitrogen, had significant effects on the bacterial community structure. The indicator species varied depending on the vegetation type and sediment depth, showing significant correlations with certain selected environmental variables, but were fundamentally related to the rhizosphere. Overall, this study revealed the key factors that determine the bacterial community structure in tidal marshes and the indicator species according to vegetation type in the little studied rhizosphere of the intertidal ecosystem.</p>
<h1>High-resolution multi-marker DNA metabarcoding reveals sexual dietary differentiation in a bird with minor dimorphism.</h1>
<p><a href="https://doi.org/10.1002/Ece3.6687">doi:10.1002/Ece3.6687</a></p>
['Oenanthe leucura', 'bird', 'diet', 'high‐throughput sequencing', 'multi‐marker', 'resource differentiation']<p>Although sexual dietary differentiation is well known in birds, it is usually linked with significant morphological dimorphism between males and females, with lower differentiation reported in sexually monomorphic or only slightly dimorphic species. However, this may be an artifact of poor taxonomic resolution achieved in most conventional dietary studies, which may be unable to detect subtle intraspecific differentiation in prey consumption. Here, we show the power of multi-marker metabarcoding to address these issues, focusing on a slightly dimorphic generalist passerine, the black wheatear <i>Oenanthe leucura</i>. Using markers from four genomic regions (18S, 16S, COI, and <i>trn</i>L), we analyzed fecal droppings collected from 93 adult black wheatears during the breeding season. We found that sexes were rather similar in bill and body features, though males had a slightly thicker bill and longer wings and tail than females. Diet was dominated in both sexes by a very wide range of arthropod species and a few fleshy fruits, but the overall diet diversity was higher for males than females, and there was a much higher frequency of occurrence of ants in female (58%) than male (29%) diets. We hypothesize that the observed sexual differentiation was likely related to females foraging closer to their offspring on abundant prey, while males consumed a wider variety of prey while foraging more widely. Overall, our results suggest that dietary sexual differentiation in birds may be more widespread than recognized at present and that multi-marker DNA metabarcoding is a particularly powerful tool to unveiling such differences.</p>
<h1>Fish and crustacean biodiversity in an outer maritime estuary of the Pearl River Delta revealed by environmental DNA.</h1>
<p><a href="https://doi.org/S0025-326X(20)30825-0">doi:S0025-326X(20)30825-0</a></p>
['Cytochrome oxidase I (COI)', 'High-throughput sequencing', 'Marine', 'Metabarcoding', 'Subtropical']<p>Understanding the faunal community structure in the estuary would be crucial in assessing the health of the ecosystem. The poor visibility in the estuarine area due to the outflow from the Pearl River hinders the conventional visual census in assessing the megafaunal biodiversity. In this study, the fish and crustacean biodiversity of Hong Kong's western waters, i.e. the outer maritime estuary of the PRD, were studied through the metabarcoding of environmental DNA (eDNA). eDNA from the seawater and sediment samples was extracted from five sites in the region. After testing the performance of two genetic markers, amplicons of the cytochrome oxidase I, amplified by polymerase chain reaction, were subjected to Illumina high-throughput sequencing (MiSeq) analysis. A total of 22 fish species from 17 families and 34 crustacean species from 27 families were identified by blasting the sequences against the NCBI GenBank database, demonstrating segregation between samples from different sites. This study provides insight on the detail distribution of fish assembly in PRD, when compared with a previous eDNA study in the inner brackish PRD.</p>
<h1>Bathymetric gradient shapes the community composition rather than the species richness of deep-sea benthic ciliates.</h1>
<p><a href="https://doi.org/S0048-9697(20)36152-0">doi:S0048-9697(20)36152-0</a></p>
['Abyssal plain', 'Benthic community', 'Protist', 'Seamount', 'Source-sink dynamic', 'Trench']<p>The bathymetric gradient is one of the most important factors that regulate the distribution of life. However, community variations of benthic ciliates along bathymetric gradients in the deep sea remain rather unexplored. In this study, we hypothesize that in the deep sea, the bathymetric gradient shapes the benthic ciliate community composition rather than the species richness. Here, we evaluated the distribution patterns and drivers of benthic ciliate communities of an abyssal plain, a seamount, and a trench with water depths ranging from 800 m down to 6600 m by high throughput eDNA sequencing and statistical analyses. We observed no significant correlation between ciliate operated taxonomic unit (OTU) richness and water depth. A meta-analysis, which combined our previously published data from the neritic habitats, supports the notion that water depth exceeding 800 m has little effect on the richness of benthic ciliate species. In contrast, the composition of deep-sea ciliate communities was significantly distinct in different habitats along the bathymetric gradients. A SourceTracker analysis revealed extremely low connectivity among ciliate communities along the bathymetric gradients. More than 95% of the community dissimilarity in the deep-sea floor was attributed to species replacement, which might be caused by environmental sorting or historical constraints. Furthermore, the observed community variations could be ascribed more to water depth than to geographic distance. The findings imply that the strong force of environmental sorting along the bathymetric gradients and the low connectivity among the ciliate communities might lead to an isolated evolution. This could shape the community composition rather than the species richness, which is mainly determined by the limited nutrient availability and the extreme environmental conditions in the deep sea.</p>
<h1>Seabed mining could come at a high price for a unique fauna.</h1>
<p><a href="https://doi.org/10.1111/Mec.15678">doi:10.1111/Mec.15678</a></p>
['abyss', 'environmental DNA', 'metazoan', 'polymetallic nodules', 'seamount']<p>The deep seafloor is teeming with life, most of which remains poorly known to science. It also constitutes an important reserve of natural resources, particularly minerals, that mining companies will start harvesting in the next few years (Nat Rev Earth Environ, 1, 2020, 158). In this context, broad biodiversity assessments of deep-sea ecosystems are urgently needed to establish a baseline prior to mining. However, significant gaps in our taxonomic knowledge and the high cost of sampling in the deep sea limit the effectiveness of conventional morphology-based surveys. In this issue of Molecular Ecology, Laroche et al. (Mol Ecol, 2020) capitalize on high throughput molecular methods to conduct one of the most detailed and rigorous surveys of the composition and biogeography of deep-seafloor metazoan communities to date. The authors show that deep seamounts in the Clarion Clipperton Zone are inhabited by rich metazoan communities that are distinct from those of the surrounding abyssal plains. These results have important conservation implications: if communities on deep seamounts were to persist after large-scale industrial mining operations on the surrounding plains, the seamounts would not serve as appropriate reservoirs to repopulate impacted areas.</p>
<h1>Comparing fish prey diversity for a critically endangered aquatic mammal in a reserve and the wild using eDNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1038/S41598-020-73648-2">doi:10.1038/S41598-020-73648-2</a></p>
<p>No keywords</p><p>Using environmental DNA (eDNA) metabarcoding, we compared fish diversity in two distinct water bodies within the Yangtze River Basin with known populations of the critically endangered Yangtze finless porpoise (Neophocaena asiaeorientalis; YFP): the Tian-e-Zhou Reserve and Poyang Lake. We aimed to create a fish surveying tool for use in the Yangtze River Basin, while also gaining a better understanding of the prey distribution and diversity within two of the remaining strongholds of YFP. 16S rRNA universal primers were developed to amplify fish eDNA. After high-throughput sequencing and stringent data filtering, we identified a total of 75 fish species (6 orders, 9 families, 57 genera) across seasons and regions. Nine of the 75 fish species were among the 28 known YFP prey species, three of which were detected in all water samples. Our eDNA metabarcoding identified many species that had been previously captured using traditional netting practices, but also numerous species not previously collected in these water bodies. Fish diversity was higher in Poyang Lake than in Tian-e-Zhou Reserve, as well as higher in the spring than in summer. These methods provide a broadly applicable tool to quantify fish diversity and distributions throughout the Yangtze River Basin, and to inform conservation strategies of YFP.</p>
<h1>Assessing the responses of <i>Sphagnum</i> micro-eukaryotes to climate changes using high throughput sequencing.</h1>
<p><a href="https://doi.org/10.7717/Peerj.9821">doi:10.7717/Peerj.9821</a></p>
['Biodiversity', 'Community structure', 'Food-web', 'Protists', 'Warming', 'Water table manipulation', 'Wetlands']<p>Current projections suggest that climate warming will be accompanied by more frequent and severe drought events. Peatlands store ca. one third of the world's soil organic carbon. Warming and drought may cause peatlands to become carbon sources through stimulation of microbial activity increasing ecosystem respiration, with positive feedback effect on global warming. Micro-eukaryotes play a key role in the carbon cycle through food web interactions and therefore, alterations in their community structure and diversity may affect ecosystem functioning and could reflect these changes. We assessed the diversity and community composition of <i>Sphagnum</i>-associated eukaryotic microorganisms inhabiting peatlands and their response to experimental drought and warming using high throughput sequencing of environmental DNA. Under drier conditions, micro-eukaryotic diversity decreased, the relative abundance of autotrophs increased and that of osmotrophs (including Fungi and Peronosporomycetes) decreased. Furthermore, we identified climate change indicators that could be used as early indicators of change in peatland microbial communities and ecosystem functioning. The changes we observed indicate a shift towards a more "terrestrial" community in response to drought, in line with observed changes in the functioning of the ecosystem.</p>
<h1>Space-time dynamics in monitoring neotropical fish communities using eDNA metabarcoding.</h1>
<p><a href="https://doi.org/S0048-9697(20)35625-4">doi:S0048-9697(20)35625-4</a></p>
['Biodiversity assessment', 'Brazil', 'Fish', 'Freshwater', 'River', 'eDNA']<p>The biodiverse Neotropical ecoregion remains insufficiently assessed, poorly managed, and threatened by unregulated human activities. Novel, rapid and cost-effective DNA-based approaches are valuable to improve understanding of the biological communities and for biomonitoring in remote areas. Here, we evaluate the potential of environmental DNA (eDNA) metabarcoding for assessing the structure and distribution of fish communities by analysing water and sediment from 11 locations along the Jequitinhonha River catchment (Brazil). Each site was sampled twice, before and after a major rain event in a five-week period and fish diversity was estimated using high-throughput sequencing of 12S rRNA amplicons. In total, 252 Molecular Operational Taxonomic Units (MOTUs) and 34 fish species were recovered, including endemic, introduced, and previously unrecorded species for this basin. Spatio-temporal variation of eDNA from fish assemblages was observed and species richness was nearly twice as high before the major rain event compared to afterwards. Yet, peaks of diversity were primarily associated with only four of the locations. No correlation between β-diversity and longitudinal distance or presence of dams was detected, but low species richness observed at sites located near dams might that these anthropogenic barriers may have an impact on local fish diversity. Unexpectedly high α-diversity levels recorded at the river mouth suggest that these sections should be further evaluated as putative "eDNA reservoirs" for rapid monitoring. By uncovering spatio-temporal changes, unrecorded biodiversity components, and putative anthropogenic impacts on fish assemblages, we further strengthen the potential of eDNA metabarcoding as a biomonitoring tool, especially in regions often neglected or difficult to access.</p>
<h1>Environmental DNA metabarcoding reveals estuarine benthic community response to nutrient enrichment - Evidence from an in-situ experiment.</h1>
<p><a href="https://doi.org/S0269-7491(20)36160-1">doi:S0269-7491(20)36160-1</a></p>
['Bacteria 16S', 'Diatoms rbcL', 'Environmental monitoring', 'Eukaryotes 18S', 'High-throughput sequencing']<p>Nutrient loading is a major threat to estuaries and coastal environments worldwide, therefore, it is critical that we have good monitoring tools to detect early signs of degradation in these ecologically important and vulnerable ecosystems. Traditionally, bottom-dwelling macroinvertebrates have been used for ecological health assessment but recent advances in environmental genomics mean we can now characterize less visible forms of biodiversity, offering a more holistic view of the ecosystem and potentially providing early warning signals of disturbance. We carried out a manipulative nutrient enrichment experiment (0, 150 and 600 g N fertilizer m<sup>-2</sup>) in two estuaries in New Zealand to assess the effects of nutrient loading on benthic communities. After seven months of enrichment, environmental DNA (eDNA) metabarcoding was used to examine the response of eukaryotic (18S rRNA), diatom only (rbcL) and bacterial (16S rRNA) communities. Multivariate analyses demonstrated changes in eukaryotic, diatom and bacterial communities in response to nutrient enrichment at both sites, despite differing environmental conditions. These patterns aligned with changes in macrofaunal communities identified using traditional morphological techniques, confirming concordance between disturbance indicators detected by eDNA and current monitoring approaches. Clear shifts in eukaryotic and bacterial indicator taxa were seen in response to nutrient loading while changes in diatom only communities were more subtle. Community changes were discernible between 0 and 150 g N m<sup>-2</sup> treatments, suggesting that estuary health assessment tools could be developed to detect early signs of degradation. Increasing variation in community structure associated with nutrient loading could also be used as an indicator of stress or approaching tipping points. This work represents a first step towards the development of molecular-based estuary monitoring tools, which could provide a more holistic and standardized approach to ecosystem health assessment with faster turn-around times and lower costs.</p>
<h1>Root-Associated Mycobiome Differentiate between Habitats Supporting Production of Different Truffle Species in Serbian Riparian Forests.</h1>
<p><a href="https://doi.org/E1331">doi:E1331</a></p>
['ITS2', 'Tuber sp.', 'core-mycobiome', 'ectomycorrhiza', 'fungal metabarcoding']<p>Balkan lowlands bordering with the Pannonia region are inhabited by diverse riparian forests that support production of different truffle species, predominantly the most prized white truffle of Piedmont (<i>Tuber magnatum</i> Pico), but also other commercial species (<i>T.</i><i>macrosporum</i> Vitt., <i>T</i>. <i>aestivum</i> Vitt.). Surprisingly, little is known about the native root-associated mycobiome (RAM) of these lowland truffle-producing forests. Therefore, in this study we aim at exploring and comparing the RAMs of three different truffle-producing forests from Kolubara river plane in Serbia. Molecular methods based on next generation sequencing (NGS) were used to evaluate the diversity of root-associated fungal communities and to elucidate the influence of environmental factors on their differentiation. To our knowledge, this is the first study from such habitats with a particular focus on comparative analysis of the RAM in different truffle-producing habitats using a high-throughput sequencing approach. Our results indicated that the alpha diversity of investigated fungal communities was not significantly different between different truffle-producing forests and within a specific forest type, while the seasonal differences in the alpha diversity were only observed in the white truffle-producing forests. Taxonomic profiling at phylum level indicated the dominance of fungal OTUs belonging to phylum Ascomycota and Basidiomycota, with very minor presence of other phyla. Distinct community structures of root-associated mycobiomes were observed for white, mixed, and black truffle-producing forests. The core mycobiome analysis indicated a fair share of fungal genera present exclusively in white and black truffle-producing forest, while the core genera of mixed truffle-producing forests were shared with both white and black truffle-producing forests. The majority of detected fungal OTUs in all three forest types were symbiotrophs, with ectomycorrhizal fungi being a dominant functional guild. Apart from assumed vegetation factor, differentiation of fungal communities was driven by factors connected to the distance from the river and exposure to fluvial activities, soil age, structure, and pH. Overall, Pannonian riparian forests appear to host diverse root-associated fungal communities that are strongly shaped by variation in soil conditions.</p>
<h1>Modelling technical and biological biases in macroinvertebrate community assessment from bulk preservative using multiple metabarcoding markers.</h1>
<p><a href="https://doi.org/10.1111/Mec.15620">doi:10.1111/Mec.15620</a></p>
['biomonitoring', 'detection probability', 'ecological indicators', 'high-throughput sequencing', 'multimarker metabarcoding']<p>DNA metabarcoding from the ethanol used to store macroinvertebrate bulk samples is a convenient methodological option in molecular biodiversity assessment and biomonitoring of aquatic ecosystems, as it preserves specimens and reduces problems associated with sample sorting. However, this method may be affected by errors and biases, which need to be thoroughly quantified before it can be mainstreamed into biomonitoring programmes. Here, we used 80 unsorted macroinvertebrate samples collected in Portugal under a Water Framework Directive monitoring programme, to compare community diversity and taxonomic composition metrics estimated through morphotaxonomy versus metabarcoding from storage ethanol using three markers (COI-M19BR2, 16S-Inse01 and 18S-Euka02) and a multimarker approach. A preliminary in silico analysis showed that the three markers were adequate for the target taxa, with detection failures related primarily to the lack of adequate barcodes in public databases. Metabarcoding of ethanol samples retrieved far less taxa per site (alpha diversity) than morphotaxonomy, albeit with smaller differences for COI-M19BR2 and the multimarker approach, while estimates of taxa turnover (beta diversity) among sites were similar across methods. Using generalized linear mixed models, we found that after controlling for differences in read coverage across samples, the probability of detection of a taxon was positively related to its proportional abundance, and negatively so to the presence of heavily sclerotized exoskeleton (e.g., Coleoptera). Overall, using our experimental protocol with different template dilutions, the COI marker showed the best performance, but we recommend the use of a multimarker approach to detect a wider range of taxa in freshwater macroinvertebrate samples. Further methodological development and optimization efforts are needed to reduce biases associated with body armouring and rarity in some macroinvertebrate taxa.</p>
<h1>Diversity and Ecology of Chlorophyta (Viridiplantae) Assemblages in Protected and Non-protected Sites in Deception Island (Antarctica, South Shetland Islands) Assessed Using an NGS Approach.</h1>
<p><a href="https://doi.org/10.1007/S00248-020-01584-9">doi:10.1007/S00248-020-01584-9</a></p>
['Antarctica', 'Green algae', 'High-throughput sequencing', 'South Shetland Islands']<p>Assessment of the diversity of algal assemblages in Antarctica has until now largely relied on traditional microbiological culture approaches. Here we used DNA metabarcoding through high-throughput sequencing (HTS) to assess the uncultured algal diversity at two sites on Deception Island, Antarctica. The first was a relatively undisturbed site within an Antarctic Specially Protected Area (ASPA 140), and the second was a site heavily impacted by human visitation, the Whalers Bay historic site. We detected 65 distinct algal taxa, 50 from within ASPA 140 and 61 from Whalers Bay. Of these taxa, 46 were common to both sites, and 19 only occurred at one site. Algal richness was about six times greater than reported in previous studies using culture methods. A high proportion of DNA reads obtained was assigned to the highly invasive species Caulerpa webbiana at Whalers Bay, and the potentially pathogenic genus Desmodesmus was found at both sites. Our data demonstrate that important differences exist between these two protected and human-impacted sites on Deception Island in terms of algal diversity, richness, and abundance. The South Shetland Islands have experienced considerable effects of climate change in recent decades, while warming through geothermal activity on Deception Island itself makes this island one of the most vulnerable to colonization by non-native species. The detection of DNA of non-native taxa highlights concerns about how human impacts, which take place primarily through tourism and national research operations, may influence future biological colonization processes in Antarctica.</p>
<h1>Metabarcoding quantifies differences in accumulation of ballast water borne biodiversity among three port systems in the United States.</h1>
<p><a href="https://doi.org/S0048-9697(20)34985-8">doi:S0048-9697(20)34985-8</a></p>
['Ballast water', 'Biodiversity', 'Biosecurity', 'High throughput sequencing', 'Metabarcoding', 'Zooplankton']<p>Characterizing biodiversity conveyed in ships' ballast water (BW), a global driver of biological invasions, is critically important for understanding risks posed by this key vector and establishing baselines to evaluate changes associated with BW management. Here we employ high throughput sequence (HTS) metabarcoding of the 18S small subunit rRNA to test for and quantify differences in the accumulation of BW-borne biodiversity among three distinct recipient port systems in the United States. These systems were located on three different coasts (Pacific, Gulf, and Atlantic) and chosen to reflect distinct trade patterns and source port biogeography. Extensive sampling of BW tanks (n = 116) allowed detailed exploration of molecular diversity accumulation. Our results indicate that saturation of introduced zooplankton diversity may be achieved quickly, with fewer than 25 tanks needed to achieve 95% of the total extrapolated diversity, if source biogeography is relatively limited. However, as predicted, port systems with much broader source geographies require more extensive sampling to estimate diversity, which continues to accumulate after sampling >100 discharges. The ability to identify BW sources using molecular indicators was also found to depend on the breadth of source biogeography and the extent to which sources had been sampled. These findings have implications both for the effort required to fully understand introduced diversity and for projecting risks associated with future changes to maritime traffic that may increase source biogeography for many recipient ports. Our data also suggest that molecular diversity may not decline significantly with BW age, indicating either that some organisms survive longer than recognized in previous studies or that nucleic acids from dead organisms persist in BW tanks. We present evidence for detection of potentially invasive species in arriving BW but discuss important caveats that preclude strong inferences regarding the presence of living representatives of these species in BW tanks.</p>
<h1>Environmental DNA assessment of airborne plant and fungal seasonal diversity.</h1>
<p><a href="https://doi.org/S0048-9697(20)33770-0">doi:S0048-9697(20)33770-0</a></p>
['Aerobiology', 'High throughput sequencing', 'Italy', 'PERMANOVA', 'PLANiTS', 'Pollen', 'Spore']<p>Environmental DNA (eDNA) metabarcoding and metagenomics analyses can improve taxonomic resolution in biodiversity studies. Only recently, these techniques have been applied in aerobiology, to target bacteria, fungi and plants in airborne samples. Here, we present a nine-month aerobiological study applying eDNA metabarcoding in which we analyzed simultaneously airborne diversity and variation of fungi and plants across five locations in North and Central Italy. We correlated species composition with the ecological characteristics of the sites and the seasons. The most abundant taxa among all sites and seasons were the fungal genera Cladosporium, Alternaria, and Epicoccum and the plant genera Brassica, Corylus, Cupressus and Linum, the latter being much more variable among sites. PERMANOVA and indicator species analyses showed that the plant diversity from air samples is significantly correlated with seasons, while that of fungi varied according to the interaction between seasons and sites. The results consolidate the performance of a new eDNA metabarcoding pipeline for the simultaneous amplification and analysis of airborne plant and fungal particles. They also highlight the promising complementarity of this approach with more traditional biomonitoring frameworks and routine reports of air quality provided by environmental agencies.</p>
<h1>Tracking seasonal changes in diversity of pollen allergen exposure: Targeted metabarcoding of a subtropical aerobiome.</h1>
<p><a href="https://doi.org/S0048-9697(20)34718-5">doi:S0048-9697(20)34718-5</a></p>
['Aerobiome', 'Allergic rhinitis', 'Metabarcoding', 'Next generation sequencing', 'Pollen-monitoring', 'Subtropics']<p>The importance of grass pollen to the global burden of allergic respiratory disease is well established but exposure to subtropical and temperate pollens is difficult to discern. Current monitoring of airborne pollen relies on light microscopy, limiting identification of taxa to family level. This informs seasonal fluctuations in pollen aerobiology but restricts analysis of aerobiological composition. We aimed to test the utility of DNA metabarcoding to identify specific taxa contributing to the aerobiome of environmental air samples, using routine pollen and spore monitoring equipment, as well as assess temporal variation of Poaceae pollen across an entire season. Airborne pollen concentrations were determined by light microscopy over two pollen seasons in the subtropical city of Brisbane (27°32'S, 153°00E), Australia. Thirty daily pollen samples were subjected to high throughput sequencing of the plastid rbcL amplicon. Amplicons corresponded to plants observed in the local biogeographical region with up to 3238 different operational taxonomic units (OTU) detected. The aerobiome sequencing data frequently identified pollen to genus levels with significant quantitative differences in aerobiome diversity between the months and seasons detected. Moreover, multiple peaks of Chloridoideae and Panicoideae pollen were evident over the collection period confirming these grasses as the dominant Poaceae pollen source across the season. Targeted high throughput sequencing of routinely collected airborne pollen samples appears to offer utility to track temporal changes in the aerobiome and shifts in pollen exposure. Precise identification of the composition and temporal distributions of airborne pollen is important for tracking biodiversity and for management of allergic respiratory disease.</p>
<h1>The limited spatial scale of dispersal in soil arthropods revealed with whole-community haplotype-level metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/Mec.15591">doi:10.1111/Mec.15591</a></p>
['cMBC', 'dispersal', 'distance decay', 'endemism', 'haplotype', 'soil mesofauna', 'speciation scale']<p>Soil arthropod communities are highly diverse and critical for ecosystem functioning. However, our knowledge of spatial structure and the underlying processes of community assembly are scarce, hampered by limited empirical data on species diversity and turnover. We implement a high-throughput sequencing approach to generate comparative data for thousands of arthropods at three hierarchical levels: genetic, species and supra-specific lineages. A joint analysis of the spatial arrangement across these levels can reveal the predominant processes driving the variation in biological assemblages at the local scale. This multihierarchical approach was performed using haplotype-level COI metabarcoding of entire communities of mites, springtails and beetles from three Iberian mountain regions. Tens of thousands of specimens were extracted from deep and superficial soil layers and produced comparative phylogeographic data for >1,000 codistributed species and nearly 3,000 haplotypes. Local assemblage composition differed greatly between grasslands and forests and, within each habitat, showed strong spatial structure and high endemicity. Distance decay was high at all levels, even at the scale of a few kilometres or less. The local distance decay patterns were self-similar for the haplotypes and higher hierarchical entities, and this fractal structure was similar in all regions, suggesting that uniform processes of limited dispersal determine local-scale community assembly. Our results from whole-community metabarcoding provide insight into how dispersal limitations constrain mesofauna community structure within local spatial settings over evolutionary timescales. If generalized across wider areas, the high turnover and endemicity in the soil locally may indicate extremely high richness globally, challenging our current estimations of total arthropod diversity on Earth.</p>
<h1>Advancing biodiversity assessments with environmental DNA: Long-read technologies help reveal the drivers of Amazonian fungal diversity.</h1>
<p><a href="https://doi.org/10.1002/Ece3.6477">doi:10.1002/Ece3.6477</a></p>
['PacBio', 'environmental DNA', 'high‐throughput sequencing', 'metabarcoding', 'neotropical biodiversity', 'third‐generation sequencing']<p>Fungi are a key component of tropical biodiversity. However, due to their inconspicuous and largely subterranean nature, they are usually neglected in biodiversity inventories. The goal of this study was to identify the key determinants of fungal richness, community composition, and turnover in tropical rainforests. We tested specifically for the effect of soil properties, habitat, and locality in Amazonia. For these analyses, we used high-throughput sequencing data of short and long reads of fungal DNA present in soil and organic litter samples, combining existing and novel genomic data. Habitat type (phytophysiognomy) emerges as the strongest factor explaining fungal community composition. Naturally open areas-campinas-are the richest habitat overall. Soil properties have different effects depending on the soil layer (litter or mineral soil) and the choice of genetic marker. We suggest that campinas could be a neglected hotspot of fungal diversity. An underlying cause for their rich diversity may be the overall low soil fertility, which increases the reliance on biotic interactions essential for nutrient absorption in these environments, notably ectomycorrhizal fungi-plant associations. Our results highlight the advantages of using both short and long DNA reads produced through high-throughput sequencing to characterize fungal diversity. While short reads can suffice for diversity and community comparison, long reads add taxonomic precision and have the potential to reveal population diversity.</p>
<h1>Fine sediment and flow velocity impact bacterial community and functional profile more than nutrient enrichment.</h1>
<p><a href="https://doi.org/10.1002/Eap.2212">doi:10.1002/Eap.2212</a></p>
['16S rRNA', 'China', 'Exstream system', 'METAGENassist', 'mesocosms', 'microbiome', 'multiple stressors']<p>Freshwater ecosystems face many simultaneous pressures due to human activities. Consequently, there has been a rapid loss of freshwater biodiversity and an increase in biomonitoring programs. Our study assessed the potential of benthic stream bacterial communities as indicators of multiple-stressor impacts associated with urbanization and agricultural intensification. We conducted a fully crossed four-factor experiment in 64 flow-through mesocosms fed by a pristine montane stream (21 d of colonization, 21 d of manipulations) and investigated the effects of nutrient enrichment, flow-velocity reduction and added fine sediment after 2 and 3 weeks of stressor exposure. We used high-throughput sequencing and metabarcoding techniques (16S rRNA genes), as well as curated biological databases (METAGENassit, MetaCyc), to identify changes in bacterial relative abundances and predicted metabolic functional profile. Sediment addition and flow-velocity reduction were the most pervasive stressors. They both increased α-diversity and had strong taxon-specific effects on community composition and predicted functions. Sediment and flow velocity also interacted frequently, with 88% of all bacterial response variables showing two-way interactions and 33% showing three-way interactions including nutrient enrichment. Changes in relative abundances of common taxa were associated with shifts in dominant predicted functions, which can be extrapolated to underlaying stream-wide mechanisms such as carbon use and bacterial energy production pathways. Observed changes were largely stable over time and occurred after just 2 weeks of exposure, demonstrating that bacterial communities can be well-suited for early detection of multiple stressors. Overall, added sediment and reduced flow velocity impacted both bacterial community structure and predicted function more than nutrient enrichment. In future research and stream management, a holistic approach to studying multiple-stressor impacts should include multiple trophic levels with their functional responses, to enhance our mechanistic understanding of complex stressor effects and promote establishment of more efficient biomonitoring programs.</p>
<h1>Meta-omics highlights the diversity, activity and adaptations of fungi in deep oceanic crust.</h1>
<p><a href="https://doi.org/10.1111/1462-2920.15181">doi:10.1111/1462-2920.15181</a></p>
<p>No keywords</p><p>The lithified oceanic crust, lower crust gabbros in particular, has remained largely unexplored by microbiologists. Recently, evidence for heterogeneously distributed viable and transcriptionally active autotrophic and heterotrophic microbial populations within low-biomass communities was found down to 750 m below the seafloor at the Atlantis Bank Gabbro Massif, Indian Ocean. Here, we report on the diversity, activity and adaptations of fungal communities in the deep oceanic crust from ~10 to 780 mbsf by combining metabarcoding analyses with mid/high-throughput culturing approaches. Metabarcoding along with culturing indicate a low diversity of viable fungi, mostly affiliated to ubiquitous (terrestrial and aquatic environments) taxa. Ecophysiological analyses coupled with metatranscriptomics point to viable and transcriptionally active fungal populations engaged in cell division, translation, protein modifications and other vital cellular processes. Transcript data suggest possible adaptations for surviving in the nutrient-poor, lithified deep biosphere that include the recycling of organic matter. These active communities appear strongly influenced by the presence of cracks and veins in the rocks where fluids and resulting rock alteration create micro-niches.</p>
<h1>High throughput shotgun sequencing of eRNA reveals taxonomic and derived functional shifts across a benthic productivity gradient.</h1>
<p><a href="https://doi.org/10.1111/Mec.15561">doi:10.1111/Mec.15561</a></p>
['biodiversity', 'community ecology', 'environmental RNA', 'organic carbon', 'sediment', 'shotgun sequencing']<p>Benthic macrofauna is regularly used in monitoring programmes, however the vast majority of benthic eukaryotic biodiversity lies mostly in microscopic organisms, such as meiofauna (invertebrates < 1 mm) and protists, that rapidly responds to environmental change. These communities have traditionally been hard to sample and handle in the laboratory, but DNA sequencing has made such work less time consuming. While DNA sequencing captures both alive and dead organisms, environmental RNA (eRNA) better targets living organisms or organisms of recent origin in the environment. Here, we assessed the biodiversity of three known bioindicator microeukaryote groups (nematodes, foraminifera, and ciliates) in sediment samples collected at seven coastal sites along an organic carbon (OC) gradient. We aimed to investigate if eRNA shotgun sequencing can be used to simultaneously detect differences in (i) biodiversity of multiple microeukaryotic communities; and (ii) functional feeding traits of nematodes. Results showed that biodiversity was lower for nematodes and foraminifera in high OC (6.2%-6.9%), when compared to low OC sediments (1.2%-2.8%). Dissimilarity in community composition increased for all three groups between Low OC and High OC, as well as the classified feeding type of nematode genera (with more nonselective deposit feeders in high OC sediment). High relative abundant genera included nematode Sabatieria and foraminifera Elphidium in high OC, and Cryptocaryon-like ciliates in low OC sediments. Considering that future sequencing technologies are likely to decrease in cost, the use of eRNA shotgun sequencing to assess biodiversity of benthic microeukaryotes could be a powerful tool in recurring monitoring programmes.</p>
<h1>Nematodes as bioindicators of polluted sediments using metabarcoding and microscopic taxonomy.</h1>
<p><a href="https://doi.org/S0160-4120(20)31877-8">doi:S0160-4120(20)31877-8</a></p>
['Biomonitoring', 'High throughput sequencing', 'Innovative monitoring', 'NemaSPEAR[%]', 'Sediment quality']<p>The use of bioindicator species is a widely applied approach to evaluate ecological conditions, and several indices have been designed for this purpose. To assess the impact of pollution, especially in sediments, a pollution-sensitive index based on nematodes, one of the most abundant and species-rich groups of metazoa, was developed. The NemaSPEAR[%] index in its original form relies on the morphological inspection of nematode species. The application of a morphologically based NemaSPEAR[%] at the genus-level was previously validated. The present study evaluated a NemaSPEAR[%] index based on metabarcoding of nematode communities and tested the potential of fragments from the 28S rDNA, 18S rDNA and cytochrome c oxidase subunit I (COI) genes. In general, molecular-based results tended to show a poorer condition than morphology-based results for the investigated sites. At the genus level, NemaSPEAR[%] values based on morphological data strongly correlated with those based on molecular data for both the 28S rDNA and the 18S rDNA gene fragments (R<sup>2</sup> = 0.86 and R<sup>2</sup> = 0.74, respectively). Within the dominant genera (>3%) identified by morphology, 68% were detected by at least one of the two ribosomal markers. At the species level, however, concordance was less pronounced, as there were several deviations of the molecular from the morphological data. These differences could mostly be attributed to shortcomings in the reference database used in the molecular-based assignments. Our pilot study shows that a molecularly based, genus-level NemaSPEAR[%] can be successfully applied to evaluate polluted sediment. Future studies need to validate this approach further, e.g. with bulk extractions of whole meiofaunal communities in order to circumvent time-consuming nematode isolation. Further database curation with abundant NemaSPEAR[%] species will also increase the applicability of this approach.</p>
<h1>Tagsteady: A metabarcoding library preparation protocol to avoid false assignment of sequences to samples.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13227">doi:10.1111/1755-0998.13227</a></p>
['amplicon sequencing', 'environmental DNA', 'high-throughput sequencing', 'tag-jumps']<p>Metabarcoding of environmental DNA (eDNA) and DNA extracted from bulk specimen samples is a powerful tool in studies of biodiversity, diet and ecological interactions as its inherent labelling of amplicons allows sequencing of taxonomically informative genetic markers from many samples in parallel. However, the occurrence of so-called 'tag-jumps' can cause incorrect assignment of sequences to samples and artificially inflate diversity. Two steps during library preparation of pools of 5' nucleotide-tagged amplicons have been suggested to cause tag-jumps: (a) T4 DNA polymerase blunt-ending in the end-repair step and (b) postligation PCR amplification of amplicon libraries. The discovery of tag-jumps has led to recommendations to only carry out metabarcoding PCR amplifications with primers carrying twin-tags to ensure that tag-jumps cannot result in false assignments of sequences to samples. As this increases both cost and workload, a metabarcoding library preparation protocol which circumvents the two steps that causes tag-jumps is needed. Here, we demonstrate Tagsteady, a PCR-free metabarcoding Illumina library preparation protocol for pools of nucleotide-tagged amplicons that enables efficient and cost-effective generation of metabarcoding data with virtually no tag-jumps. We use pools of twin-tagged amplicons to investigate the effect of T4 DNA polymerase blunt-ending and postligation PCR on the occurrence of tag-jumps and demonstrate that both blunt-ending and postligation PCR, alone or together, can result in detrimental amounts of tag-jumps (here, up to ca. 49% of total sequences), while leaving both steps out (the Tagsteady protocol) results in amounts of sequences carrying new combinations of used tags (tag-jumps) comparable to background contamination.</p>
<h1>Diel Dynamics of Freshwater Bacterial Communities at Beaches in Lake Erie and Lake St. Clair, Windsor, Ontario.</h1>
<p><a href="https://doi.org/10.1007/S00248-020-01539-0">doi:10.1007/S00248-020-01539-0</a></p>
['Diel cycle', 'Great Lakes', 'Metabarcoding', 'Microbial ecology', 'Water safety']<p>Bacteria play a key role in freshwater biogeochemical cycling as well as water safety, but short-term trends in freshwater bacterial community composition and dynamics are not yet well characterized. We sampled four public beaches in southern Ontario, Canada; in June, July, and August (2016) over a 24-h (diel) cycle at 2-h intervals. Using high-throughput sequencing of 16S rRNA gene, we found substantial bi-hourly and day/night variation in the bacterial communities with considerable fluctuation in the relative abundance of Actinobacteria and Proteobacteria phyla. Moreover, relative abundance of Enterobacteriaceae (associated with potential health risk) was significantly high at night in some dial cycles. Diversity was significantly high at night across most of the diel sampling events. qPCR assays showed a substantial bi-hourly variation of Escherichia coli levels with a significant high level of E. coli at night hours in comparison with day hours and the lowest levels at noon and during the afternoon hours. Taken together, these findings highlighted a considerable short-term temporal variation of bacterial communities which helps better understanding of freshwater bacterial dynamics and their ecology. E. coli monitoring showed that multiple samples in different hours will provide more accurate picture of freshwater safety and human health risk. Graphical abstract.</p>
<h1>Diversity hotspot and unique community structure of foraminifera in the world's deepest marine blue hole - Sansha Yongle Blue Hole.</h1>
<p><a href="https://doi.org/10.1038/S41598-020-67221-0">doi:10.1038/S41598-020-67221-0</a></p>
<p>No keywords</p><p>Marine blue holes are precious geological heritages with high scientific research values. Their physical and chemical characteristics are unique because of the steep-walled structure and isolated water column which create isolated ecosystems in geographically restricted areas. The Sansha Yongle Blue Hole (SYBH) is the world's deepest marine blue hole. Here, we generated the first DNA metabarcoding dataset from SYBH sediment focusing on foraminifera, a group of protists that have colonized various marine environments. We collected sediment samples from SYBH along a depth gradient to characterize the foraminiferal diversity and compared them with the foraminiferal diversity of the costal Jiaozhou Bay (JZB) and the abyssal Northwest Pacific Ocean (NWP). We amplified the SSU rDNA of foraminifera and sequenced them with high-throughput sequencing. The results showed that the foraminiferal assemblages in SYBH were vertically structured in response to the abiotic gradients and diversity was higher than in JZB and NWP. This study illustrates the capacity of foraminifera to colonize hostile environments and shows that blue holes are natural laboratories to explore physiological innovation associated with anoxia.</p>
<h1>Environmental DNA from plastic and textile marine litter detects exotic and nuisance species nearby ports.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0228811">doi:10.1371/Journal.Pone.0228811</a></p>
<p>No keywords</p><p>Marine debris is currently a significant source of environmental and economic problems. Floating litter can be employed by marine organisms as a surface to attach to and use as spreading vector. Human activities are promoting the expansion of potentially harmful species into novel ecosystems, endangering autochthonous communities. In this project, more than 1,000 litter items were collected and classified from five beaches eastwards the port of Gijon, in Asturias, Spain. Next generation sequencing was employed to study biofouling communities attached to items of different materials. A dominance of DNA from Florideophyceae, Dinophyceae and Arthropoda was found, and four non-indigenous species (NIS) were identified. Results showed a clear preference of Florideophyceae and Bryozoa to attach on textile surfaces versus plastic ones. Considering that these taxa contain several highly invasive species described to date, these data emphasize the potential of textile marine debris as a vector for dispersal of NIS. Moreover, the closest beaches to the port contained a more similar biota profile than the farther ones, confirming that both plastic and textile marine litter can be vectors for species dispersal from ports.</p>
<h1>A new method of metabarcoding Microsporidia and their hosts reveals high levels of microsporidian infections in mosquitoes (Culicidae).</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13205">doi:10.1111/1755-0998.13205</a></p>
['DNA barcoding', 'Nosema spp.', 'co-infection', 'molecular diagnostics', 'molecular phylogeny', 'rDNA']<p>DNA metabarcoding offers new perspectives, especially with regard to the high-throughput identification and diagnostics of pathogens. Microsporidia are an example of widely distributed, opportunistic and pathogenic microorganisms in which molecular identification is important for both environmental research and clinical diagnostics. We have developed a method for parallel detection of both microsporidian infection and the host species. We designed new primer sets: one specific for the classical Microsporidia (targeting the hypervariable V5 region of small subunit [ssu] rDNA), and a second one targeting a shortened fragment of the COI gene (standard metazoan DNA-barcode); both markers are well suited for next generation sequencing. Analysis of the ssu rDNA data set representing 607 microsporidian species (120 genera) indicated that the V5 region enables identification of >98% species in the data set (596/607). To test the method, we used microsporidians that infect mosquitoes in natural populations. Using mini-COI data, all field-collected mosquitoes were unambiguously assigned to seven species; among them almost 60% of specimens were positive for at least 11 different microsporidian species, including a new microsporidian ssu rDNA sequence (Microsporidium sp. PL01). Phylogenetic analysis showed that this species belongs to one of the two main clades in the Terresporidia. We found a high rate of microsporidian co-infections (9.4%). The numbers of sequence reads for the operational taxonomic units suggest that the occurrence of Nosema spp. in co-infections could benefit them; however, this observation should be retested using a more intensive host sampling. Our results show that DNA barcoding is a rapid and cost-effective method for deciphering sample diversity in greater resolution, including the hidden biodiversity that may be overlooked using classical methodology.</p>
<h1>Executing multi-taxa eDNA ecological assessment via traditional metrics and interactive networks.</h1>
<p><a href="https://doi.org/S0048-9697(20)32318-4">doi:S0048-9697(20)32318-4</a></p>
['18S', 'Biomonitoring', 'COI', 'Diatom', 'High throughput sequencing', 'Macroinvertebrates', 'rbcL']<p>Current approaches to ecological assessment are limited by the traditional morpho-taxonomic methods presently employed and the inability to meet increasing demands for rapid assessments. Advancements in high throughput sequencing now enable rapid high-resolution ecological assessment using environmental DNA (eDNA). Here we test the ability of using eDNA-based ecological assessment methods against traditional assessment of two key indicator groups (diatoms and macroinvertebrates) and show how eDNA across multiple gene regions (COI, rbcL, 12S and 18S) can be used to infer interactive networks that link to ecological assessment criteria. We compared results between taxonomic and eDNA based assessments and found significant positive associations between macroinvertebrate (p < 0.001 R<sup>2</sup> = 0.645) and diatom (p = 0.015, R<sup>2</sup> = 0.222) assessment metrics. We further assessed the ability of eDNA based assessment to identify environmentally sensitive genera and found an order of magnitude greater potential for 18S, versus COI or rbcL, to determine environmental filtering of ecologically assessed communities. Lastly, we compared the ability of traditional metrics against co-occurrence network properties of our combined 18S, COI and rbcL indicator genera to infer habitat quality measures currently used by managers. We found that transitivity (network connectivity), linkage density and cohesion were significantly associated with habitat modification scores (HMS), whereas network properties were inconsistent with linking to the habitat quality score (HQS) metric. The incorporation of multi-marker eDNA network assessment opens up a means for finer scale ecological assessment, currently limited using traditional methods. While utilization of eDNA-based assessment is recommended, direct comparisons with traditional approaches are difficult as the methods are intrinsically different and should be treated as such with regards to future research. Overall, our findings show that eDNA can be used for effective ecological assessment while offering a wider range of scope and application compared to traditional assessment methods.</p>
<h1>Perspectives from Ten Years of Protist Studies by High-Throughput Metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/Jeu.12813">doi:10.1111/Jeu.12813</a></p>
['Biodiversity', 'biogeography', 'biological interactions', 'ecological patterns', 'function', 'phylogenetics']<p>During the last decade, high-throughput metabarcoding became routine for analyzing protistan diversity and distributions in nature. Amid a multitude of exciting findings, scientists have also identified and addressed technical and biological limitations, although problems still exist for inference of meaningful taxonomic and ecological knowledge based on short DNA sequences. Given the extensive use of this approach, it is critical to settle our understanding on its strengths and weaknesses and to synthesize up-to-date methodological and conceptual trends. This article summarizes key scientific and technical findings, and identifies current and future directions in protist research that uses metabarcoding.</p>
<h1>Rapid range shifts and megafaunal extinctions associated with late Pleistocene climate change.</h1>
<p><a href="https://doi.org/10.1038/S41467-020-16502-3">doi:10.1038/S41467-020-16502-3</a></p>
<p>No keywords</p><p>Large-scale changes in global climate at the end of the Pleistocene significantly impacted ecosystems across North America. However, the pace and scale of biotic turnover in response to both the Younger Dryas cold period and subsequent Holocene rapid warming have been challenging to assess because of the scarcity of well dated fossil and pollen records that covers this period. Here we present an ancient DNA record from Hall's Cave, Texas, that documents 100 vertebrate and 45 plant taxa from bulk fossils and sediment. We show that local plant and animal diversity dropped markedly during Younger Dryas cooling, but while plant diversity recovered in the early Holocene, animal diversity did not. Instead, five extant and nine extinct large bodied animals disappeared from the region at the end of the Pleistocene. Our findings suggest that climate change affected the local ecosystem in Texas over the Pleistocene-Holocene boundary, but climate change on its own may not explain the disappearance of the megafauna at the end of the Pleistocene.</p>
<h1>Comparing diversity levels in environmental samples: DNA sequence capture and metabarcoding approaches using 18S and COI genes.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13201">doi:10.1111/1755-0998.13201</a></p>
['18S and COI amplicons', 'ASVs', 'DNA sequence capture', 'OTUs', 'dada2', 'eDNA', 'eukaryotic diversity', 'metabarcoding']<p>Environmental DNA studies targeting multiple taxa using metabarcoding provide remarkable insights into levels of species diversity in any habitat. The main drawbacks are the presence of primer bias and difficulty in identifying rare species. We tested a DNA sequence-capture method in parallel with the metabarcoding approach to reveal possible advantages of one method over the other. Both approaches were performed using the same eDNA samples and the same 18S and COI regions, followed by high throughput sequencing. Metabarcoded eDNA libraries were PCR amplified with one primer pair from 18S and COI genes. DNA sequence-capture libraries were enriched with 3,639 baits targeting the same gene regions. We tested amplicon sequence variants (ASVs) and operational taxonomic units (OTUs) in silico approaches for both markers and methods, using for this purpose the metabarcoding data set. ASVs methods uncovered more species for the COI gene, whereas the opposite occurred for the 18S gene, suggesting that clustering reads into OTUs could bias diversity richness especially using 18S with relaxed thresholds. Additionally, metabarcoding and DNA sequence-capture recovered 80%-90% of the control sample species. DNA sequence-capture was 8x more expensive, nonetheless it identified 1.5x more species for COI and 13x more genera for 18S than metabarcoding. Both approaches offer reliable results, sharing ca. 40% species and 72% families and retrieve more taxa when nuclear and mitochondrial markers are combined. eDNA metabarcoding is quite well established and low-cost, whereas DNA-sequence capture for biodiversity assessment is still in its infancy, is more time-consuming but provides more taxonomic assignments.</p>
<h1>Quantitative evaluation of intraspecific genetic diversity in a natural fish population using environmental DNA analysis.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13200">doi:10.1111/1755-0998.13200</a></p>
['environmental DNA', 'high-throughput sequencing', 'internal standard method', 'intraspecific genetic diversity', 'qMiSeq', 'quantitative estimation']<p>Recent advances in environmental DNA (eDNA) analysis using high-throughput sequencing (HTS) enable evaluation of intraspecific genetic diversity in a population. As the intraspecific genetic diversity provides invaluable information for wildlife conservation and management, there is an increasing demand to apply eDNA analysis to population genetics and the phylogeography by quantitative evaluation of intraspecific diversity. However, quantitative evaluations of intraspecific genetic diversity using eDNA is not straightforward because the number of eDNA sequence reads obtained by HTS may not be an index of the quantity of eDNA. In this study, to quantitatively evaluate genetic diversity using eDNA analysis, we applied a quantitative eDNA metabarcoding method using the internal standard DNAs. We targeted Ayu (Plecoglossus altivelis altivelis) and added internal standard DNAs with known copy numbers to each eDNA sample obtained from three rivers during the library preparation process. The sequence reads of each Ayu haplotype were successfully converted to DNA copy numbers based on the relationship between the copy numbers and sequence reads of the internal standard DNAs. In all rivers, the calculated copy number of each haplotype showed a significant positive correlation with the haplotype frequency estimated by a capture-based survey. Furthermore, estimates of genetic indicators such as nucleotide diversity based on the eDNA copy numbers were comparable with those estimated based on a capture-based study. Our results demonstrate that eDNA analysis with internal standard DNAs enables reasonable quantification of intraspecific genetic diversity, and this method could thus be a promising tool in the field of population genetics and phylogeography.</p>
<h1>Surprising low diversity of the plant pathogen Phytophthora in Amazonian forests.</h1>
<p><a href="https://doi.org/10.1111/1462-2920.15099">doi:10.1111/1462-2920.15099</a></p>
<p>No keywords</p><p>The genus Phytophthora represents a group of plant pathogens with broad global distribution. The majority of them cause the collar and root-rot of diverse plant species. Little is known about Phytophthora communities in forest ecosystems, especially in the Neotropical forests where natural enemies could maintain the huge plant diversity via negative density dependence. We characterized the diversity of soil-borne Phytophthora communities in the North French Guiana rainforest and investigated how they are structured by host identity and environmental factors. In this little-explored habitat, 250 soil cores were sampled from 10 plots hosting 10 different plant families across three forest environments (Terra Firme, Seasonally Flooded and White Sand). Phytophthora diversity was studied using a baiting approach and metabarcoding (High-Throughput Sequencing) on environmental DNA extracted from both soil samples and baiting-leaves. These three approaches revealed very similar communities, characterized by an unexpected low diversity of Phytophthora species, with the dominance of two cryptic species close to Phytophthora heveae. As expected, the Phytophthora community composition of the French Guiana rainforest was significantly impacted by the host plant family and environment. However, these plant pathogen communities are very small and are dominated by generalist species, questioning their potential roles as drivers of plant diversity in these Amazonian forests.</p>
<h1>Ecosystems monitoring powered by environmental genomics: A review of current strategies with an implementation roadmap.</h1>
<p><a href="https://doi.org/10.1111/Mec.15472">doi:10.1111/Mec.15472</a></p>
['biodiversity', 'biomonitoring', 'ecosystem management', 'environmental DNA', 'implementation strategy', 'metabarcoding']<p>A decade after environmental scientists integrated high-throughput sequencing technologies in their toolbox, the genomics-based monitoring of anthropogenic impacts on the biodiversity and functioning of ecosystems is yet to be implemented by regulatory frameworks. Despite the broadly acknowledged potential of environmental genomics to this end, technical limitations and conceptual issues still stand in the way of its broad application by end-users. In addition, the multiplicity of potential implementation strategies may contribute to a perception that the routine application of this methodology is premature or "in development", hence restraining regulators from binding these tools into legal frameworks. Here, we review recent implementations of environmental genomics-based methods, applied to the biomonitoring of ecosystems. By taking a general overview, without narrowing our perspective to particular habitats or groups of organisms, this paper aims to compare, review and discuss the strengths and limitations of four general implementation strategies of environmental genomics for monitoring: (a) Taxonomy-based analyses focused on identification of known bioindicators or described taxa; (b) De novo bioindicator analyses; (c) Structural community metrics including inferred ecological networks; and (d) Functional community metrics (metagenomics or metatranscriptomics). We emphasise the utility of the three latter strategies to integrate meiofauna and microorganisms that are not traditionally utilised in biomonitoring because of difficult taxonomic identification. Finally, we propose a roadmap for the implementation of environmental genomics into routine monitoring programmes that leverage recent analytical advancements, while pointing out current limitations and future research needs.</p>
<h1>Assessment of Ciguatera and Other Phycotoxin-Related Risks in Anaho Bay (Nuku Hiva Island, French Polynesia): Molecular, Toxicological, and Chemical Analyses of Passive Samplers.</h1>
<p><a href="https://doi.org/E321">doi:E321</a></p>
['CBA-N2a', 'Gambierdiscus', 'HP20 resin', 'HTS metabarcoding', 'LC-MS/MS', 'SPATT passive samplers', 'WS artificial substrate', 'ciguatera monitoring', 'ciguatoxins', 'qPCR']<p>Ciguatera poisoning is a foodborne illness caused by the consumption of seafood contaminated with ciguatoxins (CTXs) produced by dinoflagellates from the genera <i>Gambierdiscus</i> and <i>Fukuyoa</i>. The suitability of Solid Phase Adsorption Toxin Tracking (SPATT) technology for the monitoring of dissolved CTXs in the marine environment has recently been demonstrated. To refine the use of this passive monitoring tool in ciguateric areas, the effects of deployment time and sampler format on the adsorption of CTXs by HP20 resin were assessed in Anaho Bay (Nuku Hiva Island, French Polynesia), a well-known ciguatera hotspot. Toxicity data assessed by means of the mouse neuroblastoma cell-based assay (CBA-N2a) showed that a 24 h deployment of 2.5 g of resin allowed concentrating quantifiable amounts of CTXs on SPATT samplers. The CTX levels varied with increasing deployment time, resin load, and surface area. In addition to CTXs, okadaic acid (OA) and dinophysistoxin-1 (DTX1) were also detected in SPATT extracts using liquid chromatography coupled to tandem mass spectrometry (LC-MS/MS), consistent with the presence of <i>Gambierdiscus</i> and <i>Prorocentrum</i> species in the environment, as assessed by quantitative polymerase chain reaction (qPCR) and high-throughput sequencing (HTS) metabarcoding analyses conducted on passive window screen (WS) artificial substrate samples. Although these preliminary findings await further confirmation in follow-up studies, they highlight the usefulness of SPATT samplers in the routine surveillance of CP risk on a temporal scale, and the monitoring of other phycotoxin-related risks in ciguatera-prone areas.</p>
<h1>DNA Metabarcoding as a Tool for Disentangling Food Webs in Agroecosystems.</h1>
<p><a href="https://doi.org/E294">doi:E294</a></p>
['DNA sequencing', 'arthropod diets', 'conservation biological control', 'feces analysis', 'intra-guild predation', 'millet-based agroecosystem', 'trophic network']<p>Better knowledge of food webs and related ecological processes is fundamental to understanding the functional role of biodiversity in ecosystems. This is particularly true for pest regulation by natural enemies in agroecosystems. However, it is generally difficult to decipher the impact of predators, as they often leave no direct evidence of their activity. Metabarcoding via high-throughput sequencing (HTS) offers new opportunities for unraveling trophic linkages between generalist predators and their prey, and ultimately identifying key ecological drivers of natural pest regulation. Here, this approach proved effective in deciphering the diet composition of key predatory arthropods (nine species.; 27 prey taxa), insectivorous birds (one species, 13 prey taxa) and bats (one species; 103 prey taxa) sampled in a millet-based agroecosystem in Senegal. Such information makes it possible to identify the diet breadth and preferences of predators (e.g., mainly moths for bats), to design a qualitative trophic network, and to identify patterns of intraguild predation across arthropod predators, insectivorous vertebrates and parasitoids. Appropriateness and limitations of the proposed molecular-based approach for assessing the diet of crop pest predators and trophic linkages are discussed.</p>
<h1>Biodiversity between sand grains: Meiofauna composition across southern and western Sweden assessed by metabarcoding.</h1>
<p><a href="https://doi.org/10.3897/Bdj.8.E51813">doi:10.3897/Bdj.8.E51813</a></p>
[' Acoela ', ' Gastrotricha ', ' Macrostomorpha ', ' Platyhelminthes ', ' Rhabdocoela ', 'High Throughput Sequencing', 'Illumina Mi-Seq', 'Species Identification']<p>The meiofauna is an important part of the marine ecosystem, but its composition and distribution patterns are relatively unexplored. Here we assessed the biodiversity and community structure of meiofauna from five locations on the Swedish western and southern coasts using a high-throughput DNA sequencing (metabarcoding) approach. The mitochondrial cytochrome oxidase 1 (COI) mini-barcode and nuclear 18S small ribosomal subunit (18S) V1-V2 region were amplified and sequenced using Illumina MiSeq technology. Our analyses revealed a higher number of species than previously found in other areas: thirteen samples comprising 6.5 dm<sup>3</sup> sediment revealed 708 COI and 1,639 18S metazoan OTUs. Across all sites, the majority of the metazoan biodiversity was assigned to Arthropoda, Nematoda and Platyhelminthes. Alpha and beta diversity measurements showed that community composition differed significantly amongst sites. OTUs initially assigned to Acoela, Gastrotricha and the two Platyhelminthes sub-groups Macrostomorpha and Rhabdocoela were further investigated and assigned to species using a phylogeny-based taxonomy approach. Our results demonstrate that there is great potential for discovery of new meiofauna species even in some of the most extensively studied locations.</p>
<h1>Vegetation type determines spore deposition within a forest-agricultural mosaic landscape.</h1>
<p><a href="https://doi.org/Fiaa082">doi:Fiaa082</a></p>
['community assembly', 'dispersal limitations', 'fungal communities', 'high-throughput sequencing', 'spore traps']<p>Predicting fungal community assembly is partly limited by our understanding of the factors driving the composition of deposited spores. We studied the relative contribution of vegetation, geographical distance, seasonality and weather to fungal spore deposition across three vegetation types. Active and passive spore traps were established in agricultural fields, deciduous forests and coniferous forests across a geographic gradient of ∼600 km. Active traps captured the spore community suspended in air, reflecting the potential deposition, whereas passive traps reflected realized deposition. Fungal species were identified by metabarcoding of the ITS2 region. The composition of spore communities captured by passive traps differed more between vegetation types than across regions separated by >100 km, indicating that vegetation type was the strongest driver of composition of deposited spores. By contrast, vegetation contributed less to potential deposition, which followed a seasonal pattern. Within the same site, the spore communities captured by active traps differed from those captured by passive traps. Realized deposition tended to be dominated by spores of species related to vegetation. Temperature was negatively correlated with the fungal species richness of both potential and realized deposition. Our results indicate that vegetation may be able to maintain similar fungal communities across distances, and likely be the driving factor of fungal spore deposition at landscape level.</p>
<h1>Evaluation and sensitivity analysis of diatom DNA metabarcoding for WFD bioassessment of Mediterranean rivers.</h1>
<p><a href="https://doi.org/S0048-9697(20)31958-6">doi:S0048-9697(20)31958-6</a></p>
['Benthic diatoms', 'Catalan rivers', 'Environmental DNA', 'High-throughput sequencing', 'Water Framework Directive', 'rbcL']<p>Our study of 164 diatom samples from Catalonia (NE Spain) is the first to evaluate the applicability of DNA metabarcoding, based on high throughput sequencing (HTS) using a 312-bp rbcL marker, for biomonitoring Mediterranean rivers. For this, we compared the values of a biotic index (IPS) and the ecological status classes derived from them, between light microscope-based (LM) and HTS methods. Very good correspondence between methods gives encouraging results concerning the applicability of DNA metabarcoding for Catalan rivers for the EU Water Framework Directive (WFD). However, in 10 sites, the ecological status class was downgraded from "Good"/"High" obtained by LM to "Moderate"/"Poor"/"Bad" by HTS; these "critical" sites are especially important, because the WFD requires remedial action by water managers for any river with Moderate or lower status. We investigated the contribution of each species to the IPS using a "leave-one-out" sensitivity analysis, paying special attention to critical sites. Discrepancies in IPS between LM and HTS were mainly due to the misidentification and overlooking in LM of a few species, which were better recovered by HTS. This bias was particularly important in the case of Fistulifera saprophila, whose clear underrepresentation in LM was important for explaining 8 out of the 10 critical sites and probably reflected destruction of weakly-silicified frustules during sample preparation. Differences between species in the rbcL copy number per cell affected the relative abundance obtained by HTS for Achnanthidium minutissimum, Nitzschia inconspicua and Ulnaria ulna, which were also identified by the sensitivity analysis as important for the WFD. Only minor IPS discrepancies were attributed to the incompleteness of the reference library, as most of the abundant and influential species (to the IPS) were well represented there. Finally, we propose that leave-one-out analysis is a good method for identifying priority species for isolation and barcoding.</p>
<h1>Local factors drive bacterial and microeukaryotic community composition in lake surface sediment collected across an altitudinal gradient.</h1>
<p><a href="https://doi.org/Fiaa070">doi:Fiaa070</a></p>
['16S rRNA gene', '18S rRNA gene', 'freshwater microbiology', 'high throughput sequencing', 'metabarcoding']<p>Lake surface sediments are dominated by microorganisms that play significant roles in biogeochemical cycling within lakes. There is limited knowledge on the relative importance of local environmental factors and altitude on bacterial and microeukaryotic community richness and composition in lake sediments. In the present study, surface sediment samples were collected from 40 lakes along an altitude gradient (2-1215 m). Microbial communities were characterized using 16S (bacteria) and 18S (microeukaryotes) rRNA gene metabarcoding. Bacterial and microeukaryotic richness were not correlated with altitude but instead to environmental variables (e.g. area of water in the catchment (bacteria: R = -0.43). For both bacteria and microeukaryotes, dissimilarity in the community structure had a higher correlation to combined environmental variables (without altitude) (bacteria: R = 0.53; microeukaryotes: R = 0.55) than altitude alone (bacteria: R = 0.34; microeukaryotes: R = 0.47). Sediment sulfur and productive grassland were important variables in determining the relative abundance of sulfate reducing bacteria. Nitrospira, was positively related to altitude but negatively to water column total organic carbon and the proportion of productive grassland in the catchment. Little overlap in amplicon sequence variants was shown amongst lakes. This has important considerations for management decisions, suggesting that to protect biodiversity, conservation of numerous lakes and lake types is required.</p>
<h1>Study on the correlation between soil microbial diversity and ambient environmental factors influencing the safflower distribution in Xinjiang.</h1>
<p><a href="https://doi.org/10.1002/Jobm.201900626">doi:10.1002/Jobm.201900626</a></p>
['distance-based redundancy analysis (dbRDA)', 'ecological factor', 'environmental DNA (eDNA)', 'high-throughput sequencing', 'safflower (Carthamus tinctorius L.)', 'soil microorganism']<p>The effects of soil microbial properties and physiographical factors on safflower distributions in the main safflower plantations of Xinjiang province in China were studied. This study may help determine the basis of the environmental factors for evaluating the geoherbalism of this medicinal plant. The soil microbial biodiversity in the bulk soil and rhizosphere of safflower at different growth stages and from different sampling plots were characterized by analyzing the environmental DNAs in the samples. With general primers targeting the 16S ribosomal DNA for bacteria and the internal transcribed spacer 1 gene for fungi, the study was performed using marker gene amplification coupled with Illumina HiSeq high-throughput sequencing technologies. Correlation analysis and a distance-based redundancy analysis were performed to determine the dominant factors affecting the distribution of the microorganism in safflower soils. A total of 16517 bacterial operational taxonomic units (OTUs) were obtained from all the 108 soil samples of nine safflower sampling plots. At the phylum level, 48 phyla have been identified with Actinobacteria (32.9%) and proteobacteria (28.7%) being predominant. For fungi, 8746 OTUs were obtained, which belonged to seven phyla with Ascomycota overwhelmingly superior in relative abundance. A significant positive correlation was found between soil microbe quantity and ASL (above sea level). Safflower was sensitive to changes in elevation, growing more abundantly in the mountainous regions at heights of around 1,200 m above sea level. It is concluded that the dominant factors affecting the distribution of microorganisms in safflower soils were soil moisture, available N, and ASL.</p>
<h1>Environmental DNA analysis shows high potential as a tool for estimating intraspecific genetic diversity in a wild fish population.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13165">doi:10.1111/1755-0998.13165</a></p>
['environmental DNA', 'field water', 'intraspecific genetic diversity', 'mitochondrial haplotype', 'wild fish population']<p>Environmental DNA (eDNA) analysis has recently been used as a new tool for estimating intraspecific diversity. However, whether known haplotypes contained in a sample can be detected correctly using eDNA-based methods has been examined only by an aquarium experiment. Here, we tested whether the haplotypes of Ayu fish (Plecoglossus altivelis altivelis) detected in a capture survey could also be detected from an eDNA sample derived from the field that contained various haplotypes with low concentrations and foreign substances. A water sample and Ayu specimens collected from a river on the same day were analysed by eDNA analysis and Sanger sequencing, respectively. The 10 L water sample was divided into 20 filters for each of which 15 PCR replications were performed. After high-throughput sequencing, denoising was performed using two of the most widely used denoising packages, unoise3 and dada2. Of the 42 haplotypes obtained from the Sanger sequencing of 96 specimens, 38 (unoise3) and 41 (dada2) haplotypes were detected by eDNA analysis. When dada2 was used, except for one haplotype, haplotypes owned by at least two specimens were detected from all the filter replications. Accordingly, although it is important to note that eDNA-based method has some limitations and some risk of false positive and false negative, this study showed that the eDNA analysis for evaluating intraspecific genetic diversity provides comparable results for large-scale capture-based conventional methods. Our results suggest that eDNA-based methods could become a more efficient survey method for investigating intraspecific genetic diversity in the field.</p>
<h1>A taxonomy-free approach based on machine learning to assess the quality of rivers with diatoms.</h1>
<p><a href="https://doi.org/S0048-9697(20)31413-3">doi:S0048-9697(20)31413-3</a></p>
['Bioassessment', 'HYDRA', 'Machine learning', 'Metabarcoding', 'OTUs', 'Rivers']<p>Diatoms are a compulsory biological quality element in the ecological assessment of rivers according to the Water Framework Directive. The application of current official indices requires the identification of individuals to species or lower rank under a microscope based on the valve morphology. This is a highly time-consuming task, often susceptible of disagreements among analysts. In alternative, the use of DNA metabarcoding combined with High-Throughput Sequencing (HTS) has been proposed. The sequences obtained from environmental DNA are clustered into Operational Taxonomic Units (OTUs), which can be assigned to a taxon using reference databases, and from there calculate biotic indices. However, there is still a high percentage of unassigned OTUs to species due to the incompleteness of reference libraries. Alternatively, we tested a new taxonomy-free approach based on diatom community samples to assess rivers. A combination of three machine learning techniques is used to build models that predict diatom OTUs expected in test sites, under reference conditions, from environmental data. The Observed/Expected OTUs ratio indicates the deviation from reference condition and is converted into a quality class. This approach was never used with diatoms neither with OTUs data. To evaluate its efficiency, we built a model based on OTUs lists (HYDGEN) and another based on taxa lists from morphological identification (HYDMORPH), and also calculated a biotic index (IPS). The models were trained and tested with data from 81 sites (44 reference sites) from central Portugal. Both models were considered accurate (linear regression for Observed and Expected richness: R<sup>2</sup> ≈ 0.7, interception ≈ 0.8) and sensitive to global anthropogenic disturbance (Rs<sup>2</sup> > 0.30 p < 0.006 for global disturbance). Yet, the HYDGEN model based on molecular data was sensitive to more types of pressures (such as, changes in land use and habitat quality), which gives promising insights to its use for bioassessment of rivers.</p>
<h1>A Review and Perspective of eDNA Application to Eutrophication and HAB Control in Freshwater and Marine Ecosystems.</h1>
<p><a href="https://doi.org/E417">doi:E417</a></p>
['eDNA', 'eutrophication', 'freshwater ecosystem', 'harmful algae blooms', 'marine ecosystem']<p>Changing ecological communities in response to anthropogenic activities and climate change has become a worldwide problem. The eutrophication of waterbodies in freshwater and seawater caused by the effects of human activities and nutrient inputs could result in harmful algae blooms (HABs), decreases water quality, reductions in biodiversity and threats to human health. Rapid and accurate monitoring and assessment of aquatic ecosystems are imperative. Environmental DNA (eDNA) analysis using high-throughput sequencing has been demonstrated to be an effective and sensitive assay for detecting and monitoring single or multiple species in different samples. In this study, we review the potential applications of eDNA approaches in controlling and mitigating eutrophication and HABs in freshwater and marine ecosystems. We use recent studies to highlight how eDNA methods have been shown to be a useful tool for providing comprehensive data in studies of eutrophic freshwater and marine environments. We also provide perspectives on using eDNA techniques to reveal molecular mechanisms in biological processes and mitigate eutrophication and HABs in aquatic ecosystems. Finally, we discuss the feasible applications of eDNA for monitoring biodiversity, surveying species communities and providing instructions for the conservation and management of the environment by integration with traditional methods and other advanced techniques.</p>
<h1>Assessing insect biodiversity with automatic light traps in Brazil: Pearls and pitfalls of metabarcoding samples in preservative ethanol.</h1>
<p><a href="https://doi.org/10.1002/Ece3.6042">doi:10.1002/Ece3.6042</a></p>
['Brazilian biomes', 'eDNA metabarcoding', 'insect biodiversity', 'light traps']<p>Automated species identification based on data produced with metabarcoding offers an alternative for assessing biodiversity of bulk insect samples obtained with traps. We used a standard two-step PCR approach to amplify a 313 bp fragment of the barcoding region of the mitochondrial COI gene. The PCR products were sequenced on an Illumina MiSeq platform, and the OTUs production and taxonomic identifications were performed with a customized pipeline and database. The DNA used in the PCR procedures was extracted directly from the preservative ethanol of bulk insect samples obtained with automatic light traps in 12 sampling areas located in different biomes of Brazil, during wet and dry seasons. Agricultural field and forest edge habitats were collected for all sampling areas. A total of 119 insect OTUs and nine additional OTUs assigned to other arthropod taxa were obtained at a ≥97% sequence similarity level. The alpha and beta diversity analyses comparing biomes, habitats, and seasons were mostly inconclusive, except for a significant difference in beta diversity between biomes. In this study, we were able to metabarcode and HTS adult insects from their preservative medium. Notwithstanding, our results underrepresent the true magnitude of insect diversity expected from samples obtained with automatic light traps in Brazil. Although biological and technical factors might have impacted our results, measures to optimize and standardize eDNA HTS should be in place to improve taxonomic coverage of samples of unknown diversity and stored in suboptimal conditions, which is the case of most eDNA samples.</p>
<h1>Standards for Methods Utilizing Environmental DNA for Detection of Fish Species.</h1>
<p><a href="https://doi.org/E296">doi:E296</a></p>
['detection error', 'eDNA capture', 'eDNA detection', 'eDNA extraction', 'environmental DNA', 'genetic marker', 'water sampling']<p>Environmental DNA (eDNA) techniques are gaining attention as cost-effective, non-invasive strategies for acquiring information on fish and other aquatic organisms from water samples. Currently, eDNA approaches are used to detect specific fish species and determine fish community diversity. Various protocols used with eDNA methods for aquatic organism detection have been reported in different eDNA studies, but there are no general recommendations for fish detection. Herein, we reviewed 168 papers to supplement and highlight the key criteria for each step of eDNA technology in fish detection and provide general suggestions for eliminating detection errors. Although there is no unified recommendation for the application of diverse eDNA in detecting fish species, in most cases, 1 or 2 L surface water collection and eDNA capture on 0.7-μm glass fiber filters followed by extraction with a DNeasy Blood and Tissue Kit or PowerWater DNA Isolation Kit are useful for obtaining high-quality eDNA. Subsequently, species-specific quantitative polymerase chain reaction (qPCR) assays based on mitochondrial cytochrome b gene markers or eDNA metabarcoding based on both <i>12S</i> and <i>16S rRNA</i> markers via high-throughput sequencing can effectively detect target DNA or estimate species richness. Furthermore, detection errors can be minimized by mitigating contamination, negative control, PCR replication, and using multiple genetic markers. Our aim is to provide a useful strategy for fish eDNA technology that can be applied by researchers, advisors, and managers.</p>
<h1>Towards conserving natural diversity: A biotic inventory by observations, specimens, DNA barcoding and high-throughput sequencing methods.</h1>
<p><a href="https://doi.org/10.3897/Bdj.8.E50124">doi:10.3897/Bdj.8.E50124</a></p>
['biomonitoring', 'birds', 'earthworms', 'metabarcoding', 'terrestrial invertebrates', 'vegetation']<p>The Kenai National Wildlife Refuge has been given a broad conservation mandate to conserve natural diversity. A prerequisite for fulfilling this purpose is to be able to identify the species and communities that make up that biodiversity. We tested a set of varied methods for inventory and monitoring of plants, birds and terrestrial invertebrates on a grid of 40 sites in a 938 ha study area in the Slikok Creek watershed, Kenai Peninsula, Alaska. We sampled plants and lichens through observation and specimen-based methods. We surveyed birds using bird call surveys on variable circular plots. We sampled terrestrial arthropods by sweep net sampling, processing samples with High Throughput Sequencing methods. We surveyed for earthworms, using the hot mustard extraction method and identified worm specimens by morphology and DNA barcoding. We examined community membership using clustering methods and Nonmetric Multidimensional Scaling. We documented a total of 4,764 occurrences of 984 species and molecular operational taxonomic units: 87 vascular plants, 51 mosses, 12 liverworts, 111 lichens, 43 vertebrates, 663 arthropods, 9 molluscs and 8 annelid worms. Amongst these records, 102 of the arthropod species appeared to be new records for Alaska. We found three non-native species: <i>Deroceras agreste</i> (Linnaeus, 1758) (Stylommatophora: Agriolimacidae), <i>Dendrobaena octaedra</i> (Savigny, 1826) (Crassiclitellata: Lumbricidae) and <i>Heterarthrus nemoratus</i> (Fallén, 1808) (Hymenoptera: Tenthredinidae). Both <i>D. octaedra</i> and <i>H. nemoratus</i> were found at sites distant from obvious human disturbance. The 40 sites were grouped into five community groups: upland mixed forest, black spruce forest, open deciduous forest, shrub-sedge bog and willow. We demonstrated that, at least for a subset of species that could be detected using these methods, we were able to document current species distributions and assemblages in a way that could be efficiently repeated for the purposes of biomonitoring. While our methods could be improved and additional methods and groups could be added, our combination of techniques yielded a substantial portion of the data necessary for fulfilling Kenai National Wildlife Refuge's broad conservation purposes.</p>
<h1>Species detection and delineation in the marine planktonic diatoms Chaetoceros and Bacteriastrum through metabarcoding: making biological sense of haplotype diversity.</h1>
<p><a href="https://doi.org/10.1111/1462-2920.14984">doi:10.1111/1462-2920.14984</a></p>
<p>No keywords</p><p>High-throughput sequencing (HTS) metabarcoding is commonly applied to assess phytoplankton diversity. Usually, haplotypes are grouped into operational taxonomic units (OTUs) through clustering, whereby the resulting number of OTUs depends on chosen similarity thresholds. We applied, instead, a phylogenetic approach to infer taxa among 18S rDNA V4-metabarcode haplotypes gathered from 48 time-series samples using the marine planktonic diatoms Chaetoceros and Bacteriastrum as test case. The 73 recovered taxa comprised both solitary haplotypes and polytomies, the latter composed each of a highly abundant, dominant haplotype and one to several minor, peripheral haplotypes. The solitary and dominant haplotypes usually matched reference sequences, enabling species assignation of taxa. We hypothesise that the super-abundance of reads in dominant haplotypes results from the homogenization effect of concerted evolution. Reads of populous peripheral haplotypes and dominant haplotypes show comparable distribution patterns over the sample dates, suggesting that they are part of the same population. Many taxa revealed marked seasonality, with closely related ones generally showing distinct periodicity, whereas others occur year-round. Phylogenies inferred from metabarcode haplotypes enable delineation of biologically meaningful taxa, whereas OTUs resulting from clustering algorithms often deviate markedly from such taxa.</p>
<h1>Land use as a driver for protist community structure in soils under agricultural use across Europe.</h1>
<p><a href="https://doi.org/S0048-9697(20)30738-5">doi:S0048-9697(20)30738-5</a></p>
['Arable', 'Grassland', 'Land use intensity', 'Metagenomics', 'Protist microbiomes', 'V4 region of 18S rRNA']<p>Soil biodiversity is threatened by intensification of land use. The consequences of different land use on belowground biodiversity remain insufficiently explored for soil protists. Alongside being abundant and extremely diverse in soil, protists provide many ecosystem services: key players in the microbial loop, turnover of organic matter and stimulation of plant growth-promoting rhizobacteria. However, we lack knowledge of effects of site, land use intensity and management on diversity of soil protists. Here we assessed protist communities in four European arable sites with contrasting land use intensities at each site: Lusignan, France; Moskanjci, Slovenia; Castro Verde, Portugal and Scheyern, Germany as well as two grassland sites: Hainich, Germany and Lancaster, UK. Each site has consistent agricultural management history of low and high land use intensities quantified in terms of land use index (LUI). We employed high-throughput sequencing of environmental DNA, targeting the V4 region of the 18S rRNA gene. By assigning the protist composition to trophic groups, we inspected for effects of management, and other biotic and abiotic variables. While overall protist richness was unaffected by LUI within sites, specific trophic groups such as plant pathogens and saprotrophs were affected. Effects on protist biome across land uses and sites were also observed. LUI sensitive taxa were taxonomically diverse in each plot, and their trophic groups responded in specific patterns to specific practices. The most abundant trophic group was phagotrophs (73%), followed by photoautotrophs (16%), plant pathogens (4%), animal parasites (2%) and saprotrophs (1%). Community compositions and factors affecting the structure of individual trophic groups differed between land uses and management systems. The agricultural management selected for distinct protist populations as well as specific functional traits, and the protist community and diversity were indeed affected by site, LUI and management, which indicates the ecological significance of protists in the soil food web.</p>
<h1>High-throughput sequencing for community analysis: the promise of DNA barcoding to uncover diversity, relatedness, abundances and interactions in spider communities.</h1>
<p><a href="https://doi.org/10.1007/S00427-020-00652-X">doi:10.1007/S00427-020-00652-X</a></p>
['Community assembly', 'Gut content analysis', 'Metabarcoding', 'Portable sequencing', 'Third generation sequencing']<p>Large-scale studies on community ecology are highly desirable but often difficult to accomplish due to the considerable investment of time, labor and, money required to characterize richness, abundance, relatedness, and interactions. Nonetheless, such large-scale perspectives are necessary for understanding the composition, dynamics, and resilience of biological communities. Small invertebrates play a central role in ecosystems, occupying critical positions in the food web and performing a broad variety of ecological functions. However, it has been particularly difficult to adequately characterize communities of these animals because of their exceptionally high diversity and abundance. Spiders in particular fulfill key roles as both predator and prey in terrestrial food webs and are hence an important focus of ecological studies. In recent years, large-scale community analyses have benefitted tremendously from advances in DNA barcoding technology. High-throughput sequencing (HTS), particularly DNA metabarcoding, enables community-wide analyses of diversity and interactions at unprecedented scales and at a fraction of the cost that was previously possible. Here, we review the current state of the application of these technologies to the analysis of spider communities. We discuss amplicon-based DNA barcoding and metabarcoding for the analysis of community diversity and molecular gut content analysis for assessing predator-prey relationships. We also highlight applications of the third generation sequencing technology for long read and portable DNA barcoding. We then address the development of theoretical frameworks for community-level studies, and finally highlight critical gaps and future directions for DNA analysis of spider communities.</p>
<h1>High-throughput DNA barcoding of oligochaetes for abundance-based indices to assess the biological quality of sediments in streams and lakes.</h1>
<p><a href="https://doi.org/10.1038/S41598-020-58703-2">doi:10.1038/S41598-020-58703-2</a></p>
<p>No keywords</p><p>Aquatic oligochaete communities are valuable indicators of the biological quality of sediments in streams and lakes, but identification of specimens to the species level based on morphological features requires solid expertise in taxonomy and is possible only for a fraction of specimens present in a sample. The identification of aquatic oligochaetes using DNA barcodes would facilitate their use in biomonitoring and allow a wider use of this taxonomic group for ecological diagnoses. Previous approaches based on DNA metabarcoding of samples composed of total sediments or pools of specimens have been proposed for assessing the biological quality of ecosystems, but such methods do not provide precise information on species abundance, which limits the value of resulting ecological diagnoses. Here, we tested how a DNA barcoding approach based on high-throughput sequencing of sorted and genetically tagged specimens performed to assess oligochaete species diversity and abundance and the biological quality of sediments in streams and lakes. We applied both molecular and morphological approaches at 13 sites in Swiss streams and at 7 sites in Lake Geneva. We genetically identified 33 or 66 specimens per site. For both approaches, we used the same index calculations. We found that the ecological diagnoses derived from the genetic approach matched well with those of the morphological approach and that the genetic identification of only 33 specimens per site provided enough ecological information for correctly estimating the biological quality of sediments in streams and lakes.</p>
<h1>Airborne microbial biodiversity and seasonality in Northern and Southern Sweden.</h1>
<p><a href="https://doi.org/10.7717/Peerj.8424">doi:10.7717/Peerj.8424</a></p>
[' High-throughput sequencing', ' Metabarcoding', ' Microbial seasonality', ' eDNA', 'Airborne biodiversity']<p>Microorganisms are essential constituents of ecosystems. To improve our understanding of how various factors shape microbial diversity and composition in nature it is important to study how microorganisms vary in space and time. Factors shaping microbial communities in ground level air have been surveyed in a limited number of studies, indicating that geographic location, season and local climate influence the microbial communities. However, few have surveyed more than one location, at high latitude or continuously over more than a year. We surveyed the airborne microbial communities over two full consecutive years in Kiruna, in the Arctic boreal zone, and Ljungbyhed, in the Southern nemoral zone of Sweden, by using a unique collection of archived air filters. We mapped both geographic and seasonal differences in bacterial and fungal communities and evaluated environmental factors that may contribute to these differences and found that location, season and weather influence the airborne communities. Location had stronger influence on the bacterial community composition compared to season, while location and season had equal influence on the fungal community composition. However, the airborne bacterial and fungal diversity showed overall the same trend over the seasons, regardless of location, with a peak during the warmer parts of the year, except for the fungal seasonal trend in Ljungbyhed, which fluctuated more within season. Interestingly, the diversity and evenness of the airborne communities were generally lower in Ljungbyhed. In addition, both bacterial and fungal communities varied significantly within and between locations, where orders like Rhizobiales, Rhodospirillales and Agaricales dominated in Kiruna, whereas Bacillales, Clostridiales and Sordariales dominated in Ljungbyhed. These differences are a likely reflection of the landscape surrounding the sampling sites where the landscape in Ljungbyhed is more homogenous and predominantly characterized by artificial and agricultural surroundings. Our results further indicate that local landscape, as well as seasonal variation, shapes microbial communities in air.</p>
<h1>Diversity and microhabitat associations of Labyrinthula spp. in the Indian River Lagoon System.</h1>
<p><a href="https://doi.org/10.3354/Dao03431">doi:10.3354/Dao03431</a></p>
['Biodiversity', 'Biodiversity-disease relationship', 'Host specificity', 'Labyrinthula', 'Metabarcode', 'Seagrass']<p>Seagrasses create foundational habitats in coastal ecosystems. One contributing factor to their global decline is disease, primarily caused by parasites in the genus Labyrinthula. To explore the relationship between seagrass and Labyrinthula spp. diversity in coastal waters, we examined the diversity and microhabitat association of Labyrinthula spp. in 2 inlets on Florida's Atlantic Coast, the Indian River Lagoon (IRL) and Banana River. We used amplicon-based high throughput sequencing with 2 newly designed primers to amplify Labyrinthula spp. from 5 seagrass species, water, and sediments to determine their spatial distribution and microhabitat associations. The SSU primer set identified 12 Labyrinthula zero-radius operational taxonomic units (ZOTUs), corresponding to at least 8 putative species. The ITS1 primer set identified 2 ZOTUs, corresponding to at least 2 putative species. Based on our phylogenetic analyses, which include sequences from previous studies that assigned seagrass-related pathogenicity to Labyrinthula clades, all but one of the ZOTUs that we recovered with the SSU primers were from non-pathogenic species, while the 2 ZOTUs recovered with the ITS1 primers were from pathogenic species. Some of the ZOTUs were widespread across the sampling sites and microhabitats (e.g. SSU ZOTU_10), and most were present in more than one site. Our results demonstrate that targeted metabarcoding is a useful tool for examining the relationships between seagrass and Labyrinthula diversity in coastal waters.</p>
<h1>Absolute quantitation of microbes using 16S rRNA gene metabarcoding: A rapid normalization of relative abundances by quantitative PCR targeting a 16S rRNA gene spike-in standard.</h1>
<p><a href="https://doi.org/10.1002/Mbo3.977">doi:10.1002/Mbo3.977</a></p>
['16S rRNA gene', 'absolute count data', 'metabarcoding', 'microbiome', 'normalization', 'spike-in']<p>Metabarcoding of the 16S rRNA gene is commonly used to characterize microbial communities, by estimating the relative abundance of microbes. Here, we present a method to retrieve the concentrations of the 16S rRNA gene per gram of any environmental sample using a synthetic standard in minuscule amounts (100 ppm to 1% of the 16S rRNA sequences) that is added to the sample before DNA extraction and quantified by two quantitative polymerase chain reaction (qPCR) reactions. This allows normalizing by the initial microbial density, taking into account the DNA recovery yield. We quantified the internal standard and the total load of 16S rRNA genes by qPCR. The qPCR for the latter uses the exact same primers as those used for Illumina sequencing of the V3-V4 hypervariable regions of the 16S rRNA gene to increase accuracy. We are able to calculate the absolute concentration of the species per gram of sample, taking into account the DNA recovery yield. This is crucial for an accurate estimate as the yield varied between 40% and 84%. This method avoids sacrificing a high proportion of the sequencing effort to quantify the internal standard. If sacrificing a part of the sequencing effort to the internal standard is acceptable, we however recommend that the internal standard accounts for 30% of the environmental 16S rRNA genes to avoid the PCR bias associated with rare phylotypes. The method proposed here was tested on a feces sample but can be applied more broadly on any environmental sample. This method offers a real improvement of metabarcoding of microbial communities since it makes the method quantitative with limited efforts.</p>
<h1>Exploring the taxonomic composition of two fungal communities on the Swedish west coast through metabarcoding.</h1>
<p><a href="https://doi.org/10.3897/Bdj.7.E35332">doi:10.3897/Bdj.7.E35332</a></p>
['Biodiversity', 'Ecology', 'Fungal diversity', 'High-throughput sequencing', 'Marine fungi', 'Metabarcoding']<p>Fungi are heterotrophic, unicellular or filamentous organisms that exhibit a wide range of different lifestyles as, e.g., symbionts, parasites, and saprotrophs. Mycologists have traditionally considered fungi to be a nearly exclusively terrestrial group of organisms, but it is now known that fungi have a significant presence in aquatic environments as well. We know little about most fungi in limnic and marine systems, including aspects of their taxonomy, ecology, and geographic distribution. The present study seeks to improve our knowledge of fungi in the marine environment. The fungal communities of two coastal marine environments of the Kattegat sea, Sweden, were explored with metabarcoding techniques using the nuclear ribosomal internal transcribed spacer 2 (ITS2) metabarcode. Our data add new information to the current picture of fungal community composition in benthic and coastal habitats in Northern Europe.</p>
<h1>Host tree organ is the primary driver of endophytic fungal community structure in a hemiboreal forest.</h1>
<p><a href="https://doi.org/Fiz199">doi:Fiz199</a></p>
['Capnodiales', 'Dothideomycetes', 'Pleosporales', 'core microbiome', 'metabarcoding', 'tree organ specificity']<p>Despite numerous studies on plant endophytes, little is known about fungal communities associated with different aboveground tissues of living trees. We used high-throughput sequencing to compare the diversity and community structure of fungi inhabiting leaves, branches and trunks of Alnus incana and Corylus avellana growing at three hemiboreal forest sites. Our analysis revealed that tree organs are the main determinants of the structure of fungal communities, whereas the effects of host species and locality remained secondary and negligible, respectively. The structure of fungal communities in trunks was the most distinct compared to that in leaves and branches. The foliar fungal communities were more similar within than between individual trees, implying that certain fungi may grow through parts of the tree crown. The weak effect of locality compared to host organs and species identity suggests that the structural variation of fungal communities in the aboveground parts of trees depends mainly on deterministic factors rather than dispersal limitation.</p>
<h1>Long-read metabarcoding of the eukaryotic rDNA operon to phylogenetically and taxonomically resolve environmental diversity.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13117">doi:10.1111/1755-0998.13117</a></p>
['PacBio', 'metabarcoding', 'phylogeny', 'protists', 'rDNA operon', 'taxonomy']<p>High-throughput DNA metabarcoding of amplicon sizes below 500 bp has revolutionized the analysis of environmental microbial diversity. However, these short regions contain limited phylogenetic signal, which makes it impractical to use environmental DNA in full phylogenetic inferences. This lesser phylogenetic resolution of short amplicons may be overcome by new long-read sequencing technologies. To test this idea, we amplified soil DNA and used PacBio Circular Consensus Sequencing (CCS) to obtain an ~4500-bp region spanning most of the eukaryotic small subunit (18S) and large subunit (28S) ribosomal DNA genes. We first treated the CCS reads with a novel curation workflow, generating 650 high-quality operational taxonomic units (OTUs) containing the physically linked 18S and 28S regions. To assign taxonomy to these OTUs, we developed a phylogeny-aware approach based on the 18S region that showed greater accuracy and sensitivity than similarity-based methods. The taxonomically annotated OTUs were then combined with available 18S and 28S reference sequences to infer a well-resolved phylogeny spanning all major groups of eukaryotes, allowing us to accurately derive the evolutionary origin of environmental diversity. A total of 1,019 sequences were included, of which a majority (58%) corresponded to the new long environmental OTUs. The long reads also allowed us to directly investigate the relationships among environmental sequences themselves, which represents a key advantage over the placement of short reads on a reference phylogeny. Together, our results show that long amplicons can be treated in a full phylogenetic framework to provide greater taxonomic resolution and a robust evolutionary perspective to environmental DNA.</p>
<h1>eDNA metabarcoding in zooplankton improves the ecological status assessment of aquatic ecosystems.</h1>
<p><a href="https://doi.org/S0160-4120(19)32222-6">doi:S0160-4120(19)32222-6</a></p>
['DNA barcoding', 'Ecological assessment', 'High-throughput sequencing', 'Mitochondrial cytochrome c oxidase I', 'Water quality', 'Zooplankton integrity']<p>Aquatic ecosystems are monitored worldwide using a range of biological quality elements that are morphologically identified. The environmental DNA (eDNA)-based approach has unprecedented advantages (e.g., high throughput, high efficiency and low cost) for biodiversity surveys in both freshwater and marine ecosystems compared with traditional sampling and image recognition. The use of eDNA has been mostly limited to biodiversity estimation, how to apply the eDNA approach in assessing the ecological health status is largely unexplored. Here, using zooplankton as an example, we examined the application of eDNA monitoring for ecological status assessment in an aquatic ecosystem. The results showed that eDNA monitoring reflected the spatial and temporal variations in zooplankton structure. Both species composition and bio-interactions varied significantly between sampling seasons (dry, normal and wet). A total of 60 different zooplankton indices were calculated based on eDNA monitoring and most of these indices were highly correlated with the level of water pollution, which was indicated by the water quality index in one or all three seasons. Both qualitative and quantitative eDNA-based biological indices were correlated with water quality. The season-dependent eDNA zooplankton integrity index (IZI) reflected the ecological status, and this method improves the timeliness of bioassessment.</p>
<h1>Foraging strategies are maintained despite workforce reduction: A multidisciplinary survey on the pollen collected by a social pollinator.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0224037">doi:10.1371/Journal.Pone.0224037</a></p>
<p>No keywords</p><p>The way pollinators gather resources may play a key role for buffering their population declines. Social pollinators like bumblebees could adjust their foraging after significant workforce reductions to keep provisions to the colony optimal, especially in terms of pollen diversity and quantity. To test what effects a workforce reduction causes on the foraging for pollen, commercially-acquired colonies of the bumblebee Bombus terrestris were allowed to forage in the field and they were experimentally manipulated by removing half the number of workers. For each bumblebee, the pollen pellets were taxonomically identified with DNA metabarcoding of the ITS2 region followed by a statistical filtering based on ROC curves to filter out underrepresented OTUs. Video cameras and network analyses were employed to investigate changes in foraging strategies and behaviour. After filtering out the false-positives, HTS metabarcoding yielded a high plant diversity in the pollen pellets; for plant identity and pollen quantity traits no differences emerged between samples from treated and from control colonies, suggesting that plant choice was influenced mainly by external factors such as the plant phenology. The colonies responded to the removal of 50% of their workers by increasing the foraging activity of the remaining workers, while only negligible changes were found in diet breadth and indices describing the structure of the pollen transport network. Therefore, a consistency in the bumblebees' feeding strategies emerges in the short term despite the lowered workforce.</p>
<h1>Genetic Biomonitoring and Biodiversity Assessment Using Portable Sequencing Technologies: Current Uses and Future Directions.</h1>
<p><a href="https://doi.org/E858">doi:E858</a></p>
['portable sequencing, nanopore sequencing, biodiversity monitoring, local capacity building, MinION, DNA barcoding, long read sequencing']<p>We live in an era of unprecedented biodiversity loss, affecting the taxonomic composition of ecosystems worldwide. The immense task of quantifying human imprints on global ecosystems has been greatly simplified by developments in high-throughput DNA sequencing technology (HTS). Approaches like DNA metabarcoding enable the study of biological communities at unparalleled detail. However, current protocols for HTS-based biodiversity exploration have several drawbacks. They are usually based on short sequences, with limited taxonomic and phylogenetic information content. Access to expensive HTS technology is often restricted in developing countries. Ecosystems of particular conservation priority are often remote and hard to access, requiring extensive time from field collection to laboratory processing of specimens. The advent of inexpensive mobile laboratory and DNA sequencing technologies show great promise to facilitate monitoring projects in biodiversity hot-spots around the world. Recent attention has been given to portable DNA sequencing studies related to infectious organisms, such as bacteria and viruses, yet relatively few studies have focused on applying these tools to Eukaryotes, such as plants and animals. Here, we outline the current state of genetic biodiversity monitoring of higher Eukaryotes using Oxford Nanopore Technology's MinION portable sequencing platform, as well as summarize areas of recent development.</p>
<h1>Microbiota Associated with Dromedary Camel Milk from Algerian Sahara.</h1>
<p><a href="https://doi.org/10.1007/S00284-019-01788-4">doi:10.1007/S00284-019-01788-4</a></p>
<p>No keywords</p><p>Despite the fact that camel milk represents a valuable food source, the fungal diversity of raw camel milk has been poorly studied so far. Here, we investigated the fungal and bacterial communities found in dromedary camel milk from Ghardaia, a representative region of Algerian Sahara. The application of both culture-dependent and independent molecular techniques, based on dHPLC analysis and metabarcoding of ITS region, provided a complementary biodiversity assessment of camel milk fungi which was composed of 15 different taxa. Yeast species belonged to Filobasidium, Naganishia, Malassezia, Mrakia, Rhodotorula, and Yarrowia genera; and mold species belonged to Fusarium, Cladosporium, and Penicillium genera. All three techniques revealed that the fungal community was dominated by species belonging to the former genus Cryptococcus (Filobasidium and Naganishia) although none of them was able to encompass the entire fungal diversity alone. In addition, massive parallel 16S rRNA tag sequencing was applied to gain an insight into the diversity of bacterial communities which were dominated by Pseudomonas spp. Our results provide an initial insight about fungal and bacterial population found in dromedary camel milk from Algerian Sahara.</p>
<h1>Latent Dirichlet Allocation reveals spatial and taxonomic structure in a DNA-based census of soil biodiversity from a tropical forest.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13109">doi:10.1111/1755-0998.13109</a></p>
['OTU presence-absence', 'community ecology', 'environmental DNA', 'metabarcoding', 'soil microbiome', 'topic modelling']<p>High-throughput sequencing of amplicons from environmental DNA samples permits rapid, standardized and comprehensive biodiversity assessments. However, retrieving and interpreting the structure of such data sets requires efficient methods for dimensionality reduction. Latent Dirichlet Allocation (LDA) can be used to decompose environmental DNA samples into overlapping assemblages of co-occurring taxa. It is a flexible model-based method adapted to uneven sample sizes and to large and sparse data sets. Here, we compare LDA performance on abundance and occurrence data, and we quantify the robustness of the LDA decomposition by measuring its stability with respect to the algorithm's initialization. We then apply LDA to a survey of 1,131 soil DNA samples that were collected in a 12-ha plot of primary tropical forest and amplified using standard primers for bacteria, protists, fungi and metazoans. The analysis reveals that bacteria, protists and fungi exhibit a strong spatial structure, which matches the topographical features of the plot, while metazoans do not, confirming that microbial diversity is primarily controlled by environmental variation at the studied scale. We conclude that LDA is a sensitive, robust and computationally efficient method to detect and interpret the structure of large DNA-based biodiversity data sets. We finally discuss the possible future applications of this approach for the study of biodiversity.</p>
<h1>GenBank is a reliable resource for 21st century biodiversity research.</h1>
<p><a href="https://doi.org/10.1073/Pnas.1911714116">doi:10.1073/Pnas.1911714116</a></p>
['environmental DNA', 'metabarcoding', 'taxonomic assignments']<p>Traditional methods of characterizing biodiversity are increasingly being supplemented and replaced by approaches based on DNA sequencing alone. These approaches commonly involve extraction and high-throughput sequencing of bulk samples from biologically complex communities or samples of environmental DNA (eDNA). In such cases, vouchers for individual organisms are rarely obtained, often unidentifiable, or unavailable. Thus, identifying these sequences typically relies on comparisons with sequences from genetic databases, particularly GenBank. While concerns have been raised about biases and inaccuracies in laboratory and analytical methods, comparatively little attention has been paid to the taxonomic reliability of GenBank itself. Here we analyze the metazoan mitochondrial sequences of GenBank using a combination of distance-based clustering and phylogenetic analysis. Because of their comparatively rapid evolutionary rates and consequent high taxonomic resolution, mitochondrial sequences represent an invaluable resource for the detection of the many small and often undescribed organisms that represent the bulk of animal diversity. We show that metazoan identifications in GenBank are surprisingly accurate, even at low taxonomic levels (likely <1% error rate at the genus level). This stands in contrast to previously voiced concerns based on limited analyses of particular groups and the fact that individual researchers currently submit annotated sequences to GenBank without significant external taxonomic validation. Our encouraging results suggest that the rapid uptake of DNA-based approaches is supported by a bioinformatic infrastructure capable of assessing both the losses to biodiversity caused by global change and the effectiveness of conservation efforts aimed at slowing or reversing these losses.</p>
<h1>Stream Algal Biofilm Community Diversity Along An Acid Mine Drainage Recovery Gradient Using Multimarker Metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/Jpy.12935">doi:10.1111/Jpy.12935</a></p>
['algae', 'biofilm', 'cyanobacteria', 'diatom', 'freshwater', 'metabarcoding']<p>In southeastern Ohio, active remediation of streams affected by Acid Mine Drainage (AMD) has proven to be successful for some streams, while others have not recovered based on macroinvertebrate assessment. In this study, biofilms were collected from three Moderately Impaired, three Recovered, and two Unimpaired streams. The biodiversity was characterized by metabarcoding using two universal barcode markers (16S and 18S) along with two algal specific markers (UPA and rbcL) and high-throughput amplicon sequencing. For each marker, the ordination of Bray-Curtis Index calculated from the total Amplicon Sequence Variants (ASVs) present in each stream showed the Unimpaired and Recovered streams clustered, while Moderately Impaired streams were more distant. Focusing on the algal ASVs, the Shannon index for the rbcL, and UPA markers showed significantly lower alpha diversity in Moderately Impaired streams compared to Unimpaired streams, but the Recovered streams were not significantly different from the other two stream categories. The two universal markers together captured all algal phyla providing an outline of the diversity, but the two algal specific markers produced a greater number of ASVs and taxonomic depth for algal taxa. Further examination of the UPA marker revealed a drastic decrease in relative abundance of diatoms in Moderately Impaired streams compared to Recovered and Unimpaired streams. Likewise, diatom genera identified in the rbcL data and indicative of stream water quality showed marked differences in relative abundance among stream categories. Although all markers were useful, the algal-specific UPA and rbcL contributed more insights into algal community differences among stream categories.</p>
<h1>Genetic diversity of primate strongylid nematodes: Do sympatric nonhuman primates and humans share their strongylid worms?</h1>
<p><a href="https://doi.org/10.1111/Mec.15257">doi:10.1111/Mec.15257</a></p>
['metabarcoding', 'primate', 'strongylid nematode', 'sympatric']<p>The close phylogenetic relationship between humans and nonhuman primates (NHPs) can result in a high potential for pathogen exchange. In recent decades, NHP and human interactions have become more frequent due to increasing habitat encroachment and ecotourism. Strongylid communities, which include members of several genera, are typically found in NHPs. Using optimized high-throughput sequencing for strain-level identification of primate strongylids, we studied the structure of strongylid communities in NHPs and humans co-habiting a tropical forest ecosystem in the Central African Republic. General taxonomic assignment of 85 ITS-2 haplotypes indicated that the studied primates harbour at least nine genera of strongylid nematodes, with Oesophagostomum and Necator being the most prevalent. We detected both host-specific and shared strongylid haplotypes. Skin-penetrating Necator gorillaehaplotypes were shared between humans and gorillas but Necator americanus were much more restricted to humans. Strongylid communities of local hunter-gatherers employed as trackers were more similar to those of gorillas compared to their relatives, who spent more time in villages. This was due to lower abundance of human-origin N. americanus in both gorillas and trackers. Habituated gorillas or those under habituation did not show larger overlap of strongylids with humans compared to unhabituated. We concluded that the occurrence of the human-specific strongylids in gorillas does not increase with direct contact between gorillas and humans due to the habituation. Overall, our results indicate that the degree of habitat sharing between hosts, together with mode of parasite transmission, are important factors for parasite spillover among primates.</p>
<h1>DNA metabarcoding adds valuable information for management of biodiversity in roadside stormwater ponds.</h1>
<p><a href="https://doi.org/10.1002/Ece3.5503">doi:10.1002/Ece3.5503</a></p>
['DNA metabarcoding', 'high throughput sequencing', 'macroinvertebrates', 'pollutants', 'road runoff', 'stormwater ponds']<p>Stormwater ponds are used to compensate for the adverse effects that road runoff might have on the natural environment. Depending on their design and placement, stormwater ponds can act as both refugia and traps for local biodiversity. To evaluate the impact of stormwater ponds on biodiversity, it is critical to use effective and precise methods for identification of life associated with the water body. DNA metabarcoding has recently become a promising tool for identification and assessment of freshwater biodiversity.Using both morphology and DNA metabarcoding, we analyze species richness and biological composition of samples from 12 stormwater ponds and investigate the impact of pond size and pollution levels in the sediments and water column on the macroinvertebrate community structure.DNA metabarcoding captured and identified more than twice the number of taxa compared to morphological identification. The (dis)similarity of macroinvertebrate community composition in different ponds showed that the ponds appear better separated in the results obtained by DNA metabarcoding, but that the explained variation is higher for the results obtained by morphologically identification, since it provides abundance data.The reliance on morphological methods has limited our perception of the aquatic biodiversity in response to anthropogenic stressors, thereby providing inaccurate information for appropriate design and management of stormwater ponds; these drawbacks can be overcome by DNA metabarcoding. <i>Synthesis and applications</i>. The results indicate that DNA metabarcoding is a useful tool in identifying species, especially Diptera, which are difficult to determine. Application of DNA metabarcoding greatly increases the number of species identified at each sampling site, thereby providing a more accurate information regarding the way the ponds function and how they are affected by management.</p>
<h1>COI metabarcoding primer choice affects richness and recovery of indicator taxa in freshwater systems.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0220953">doi:10.1371/Journal.Pone.0220953</a></p>
<p>No keywords</p><p>Mixed community or environmental DNA marker gene sequencing has become a commonly used technique for biodiversity analyses in freshwater systems. Many cytochrome c oxidase subunit I (COI) primer sets are now available for such work. The purpose of this study is to test whether COI primer choice affects the recovery of arthropod richness, beta diversity, and recovery of target assemblages in the benthos kick-net samples typically used in freshwater biomonitoring. We examine six commonly used COI primer sets on samples collected from six freshwater sites. Biodiversity analyses show that richness is sensitive to primer choice and the combined use of multiple COI amplicons recovers higher richness. Thus, to recover maximum richness, multiple primer sets should be used with COI metabarcoding. In ordination analyses based on community dissimilarity, samples consistently cluster by site regardless of amplicon choice or PCR replicate. Thus, for broadscale community analyses, overall beta diversity patterns are robust to COI marker choice. Recovery of traditional freshwater bioindicator assemblages such as Ephemeroptera, Trichoptera, Plectoptera, and Chironomidae as well as Arthropoda site indicators were differentially detected by each amplicon tested. This work will help future biodiversity and biomonitoring studies develop not just standardized, but optimized workflows that either maximize taxon-detection or the selection of amplicons for water quality or Arthropoda site indicators.</p>
<h1>Global distribution and diversity of <i>Chaetoceros</i> (Bacillariophyta, Mediophyceae): integration of classical and novel strategies.</h1>
<p><a href="https://doi.org/10.7717/Peerj.7410">doi:10.7717/Peerj.7410</a></p>
['18S rDNA', 'Biodiversity', 'Biogeography', 'Biological records', 'Chaetoceros', 'Global distribution', 'Marine diatoms', 'Metabarcoding', 'OSD', 'TARA']<p>Information on taxa distribution is a prerequisite for many research fields, and biological records are a major source of data contributing to biogeographic studies. The Global Biodiversity Information Facility (GBIF) and the Ocean Biogeographic Information System (OBIS) are important infrastructures facilitating free and open access to classical biological data from several sources in both temporal and spatial scales. Over the last ten years, high throughput sequencing (HTS) metabarcoding data have become available, which constitute a great source of detailed occurrence data. Among the global sampling projects that have contributed to such data are Tara Oceans and the Ocean Sampling Day (OSD). Integration of classical and metabarcoding data may aid a more comprehensive assessment of the geographic range of species, especially of microscopic ones such as protists. Rare, small and cryptic species are often ignored in surveys or mis-assigned with the classical approaches. Here we show how integration of data from various sources can contribute to insight in the biogeography and diversity at the genus- and species-level using <i>Chaetoceros</i> as study system<i>,</i> one of the most diverse and abundant genera among marine planktonic diatoms. <i>Chaetoceros</i> records were extracted from GBIF and OBIS and literature data were collected by means of a Google Scholar search. <i>Chaetoceros</i> references barcodes where mapped against the metabarcode datasets of Tara Oceans (210 sites) and OSD (144 sites). We compared the resolution of different data sources in determining the global distribution of the genus and provided examples, at the species level, of detection of cryptic species, endemism and cosmopolitan or restricted distributions. Our results highlighted at genus level a comparable picture from the different sources but a more complete assessment when data were integrated. Both the importance of the integration but also the challenges related to it were illustrated. <i>Chaetoceros</i> data collected in this study are organised and available in the form of tables and maps, providing a powerful tool and a baseline for further research in e.g., ecology, conservation and evolutionary biology.</p>
<h1>A novel metabarcoding diagnostic tool to explore protozoan haemoparasite diversity in mammals: a proof-of-concept study using canines from the tropics.</h1>
<p><a href="https://doi.org/10.1038/S41598-019-49118-9">doi:10.1038/S41598-019-49118-9</a></p>
<p>No keywords</p><p>Haemoparasites are responsible for some of the most prevalent and debilitating canine illnesses across the globe, whilst also posing a significant zoonotic risk to humankind. Nowhere are the effects of such parasites more pronounced than in developing countries in the tropics where the abundance and diversity of ectoparasites that transmit these pathogens reaches its zenith. Here we describe the use of a novel next-generation sequencing (NGS) metabarcoding based approach to screen for a range of blood-borne apicomplexan and kinetoplastid parasites from populations of temple dogs in Bangkok, Thailand. Our methodology elucidated high rates of Hepatozoon canis and Babesia vogeli infection, whilst also being able to characterise co-infections. In addition, our approach was confirmed to be more sensitive than conventional endpoint PCR diagnostic methods. Two kinetoplastid infections were also detected, including one by Trypanosoma evansi, a pathogen that is rarely screened for in dogs and another by Parabodo caudatus, a poorly documented organism that has been previously reported inhabiting the urinary tract of a dog with haematuria. Such results demonstrate the power of NGS methodologies to unearth rare and unusual pathogens, especially in regions of the world where limited information on canine vector-borne haemoparasites exist.</p>
<h1>Current insights into fungal species diversity and perspective on naming the environmental DNA sequences of fungi.</h1>
<p><a href="https://doi.org/10.1080/21501203.2019.1614106">doi:10.1080/21501203.2019.1614106</a></p>
['Fungi', 'biodiversity', 'fungal phylogeny', 'molecular methods', 'numbers of fungi']<p>The global bio-diversity of fungi has been extensively investigated and their species number has been estimated. Notably, the development of molecular phylogeny has revealed an unexpected fungal diversity and utilisation of culture-independent approaches including high-throughput amplicon sequencing has dramatically increased number of fungal operational taxonomic units. A number of novel taxa including new divisions, classes, orders and new families have been established in last decade. Many cryptic species were identified by molecular phylogeny. Based on recently generated data from culture-dependent and -independent survey on same samples, the fungal species on the earth were estimated to be 12 (11.7-13.2) million compared to 2.2-3.8 million species recently estimated by a variety of the estimation techniques. Moreover, it has been speculated that the current use of high-throughput sequencing techniques would reveal an even higher diversity than our current estimation. Recently, the formal classification of environmental sequences and permission of DNA sequence data as fungal names' type were proposed but strongly objected by the mycologist community. Surveys on fungi in unusual niches have indicated that many previously regarded "unculturable fungi" could be cultured on certain substrates under specific conditions. Moreover, the high-throughput amplicon sequencing, shotgun metagenomics and a single-cell genomics could be a powerful means to detect novel taxa. Here, we propose to separate the fungal types into physical type based on specimen, genome DNA (gDNA) type based on complete genome sequence of culturable and uncluturable fungal specimen and digital type based on environmental DNA sequence data. The physical and gDNA type should have priority, while the digital type can be temporal supplementary before the physical type and gDNA type being available. The fungal name based on the "digital type" could be assigned as the "clade" name + species name. The "clade" name could be the name of genus, family or order, etc. which the sequence of digital type affiliates to. Facilitating future cultivation efforts should be encouraged. Also, with the advancement in knowledge of fungi inhabiting various environments mostly because of rapid development of new detection technologies, more information should be expected for fungal diversity on our planet.</p>
<h1>Relative Performance of MinION (Oxford Nanopore Technologies) versus Sequel (Pacific Biosciences) Third-Generation Sequencing Instruments in Identification of Agricultural and Forest Fungal Pathogens.</h1>
<p><a href="https://doi.org/E01368-19">doi:E01368-19</a></p>
['Oxford Nanopore MinION', 'PacBio Sequel', 'Solanum tuberosum\n', 'fungal plant pathogens', 'metabarcoding', 'metagenomics', 'molecular diagnostics', 'needle pathogens', 'oomycetes', 'plant pathogens', 'potato']<p>Culture-based molecular identification methods have revolutionized detection of pathogens, yet these methods are slow and may yield inconclusive results from environmental materials. The second-generation sequencing tools have much-improved precision and sensitivity of detection, but these analyses are costly and may take several days to months. Of the third-generation sequencing techniques, the portable MinION device (Oxford Nanopore Technologies) has received much attention because of its small size and possibility of rapid analysis at reasonable cost. Here, we compare the relative performances of two third-generation sequencing instruments, MinION and Sequel (Pacific Biosciences), in identification and diagnostics of fungal and oomycete pathogens from conifer (Pinaceae) needles and potato (<i>Solanum tuberosum</i>) leaves and tubers. We demonstrate that the Sequel instrument is efficient for metabarcoding of complex samples, whereas MinION is not suited for this purpose due to a high error rate and multiple biases. However, we find that MinION can be utilized for rapid and accurate identification of dominant pathogenic organisms and other associated organisms from plant tissues following both amplicon-based and PCR-free metagenomics approaches. Using the metagenomics approach with shortened DNA extraction and incubation times, we performed the entire MinION workflow, from sample preparation through DNA extraction, sequencing, bioinformatics, and interpretation, in 2.5 h. We advocate the use of MinION for rapid diagnostics of pathogens and potentially other organisms, but care needs to be taken to control or account for multiple potential technical biases.<b>IMPORTANCE</b> Microbial pathogens cause enormous losses to agriculture and forestry, but current combined culturing- and molecular identification-based detection methods are too slow for rapid identification and application of countermeasures. Here, we develop new and rapid protocols for Oxford Nanopore MinION-based third-generation diagnostics of plant pathogens that greatly improve the speed of diagnostics. However, due to high error rate and technical biases in MinION, the Pacific BioSciences Sequel platform is more useful for in-depth amplicon-based biodiversity monitoring (metabarcoding) from complex environmental samples.</p>
<h1>Altitude and fungal diversity influence the structure of Antarctic cryptoendolithic Bacteria communities.</h1>
<p><a href="https://doi.org/10.1111/1758-2229.12788">doi:10.1111/1758-2229.12788</a></p>
<p>No keywords</p><p>Endolithic growth within rocks is a critical adaptation of microbes living in harsh environments where exposure to extreme temperature, radiation, and desiccation limits the predominant life forms, such as in the ice-free regions of Continental Antarctica. The microbial diversity of the endolithic communities in these areas has been sparsely examined. In this work, diversity and composition of bacterial assemblages in the cryptoendolithic lichen-dominated communities of Victoria Land (Continental Antarctica) were explored using a high-throughput metabarcoding approach, targeting the V4 region of 16S rDNA. Rocks were collected in 12 different localities (from 14 different sites), along a gradient ranging from 1000 to 3300 m a.s.l. and at a sea distance ranging from 29 to 96 km. The results indicate Actinobacteria and Proteobacteria are the dominant taxa in all samples and defined a 'core' group of bacterial taxa across all sites. The structure of bacteria communities is correlated with the fungal counterpart and among the environmental parameters considered, altitude was found to influence bacterial biodiversity, while distance from sea had no evident influence.</p>
<h1>The effects of spatial and temporal replicate sampling on eDNA metabarcoding.</h1>
<p><a href="https://doi.org/10.7717/Peerj.7335">doi:10.7717/Peerj.7335</a></p>
['Environmental DNA', 'Freshwater', 'High-throughput sequencing', 'Metabarcoding', 'Replicates', 'Sampling strategy', 'Spatiotemporal variation']<p>The heterogeneous nature of environmental DNA (eDNA) and its effects on species detection and community composition estimates has been highlighted in several studies in the past decades. Mostly in the context of spatial distribution over large areas, in fewer occasions looking at spatial distribution within a single body of water. Temporal variation of eDNA, similarly, has mostly been studied as seasonality, observing changes over large periods of time, and often only for small groups of organisms such as fish and amphibians.</p>
<h1>Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance.</h1>
<p><a href="https://doi.org/Giz092">doi:Giz092</a></p>
['alien species', 'bioinformatics', 'biosecurity', 'biosurveillance', 'controls', 'early detection', 'non-destructive', 'quality assurance', 'reference database', 'validation']<p>Trap-based surveillance strategies are widely used for monitoring of invasive insect species, aiming to detect newly arrived exotic taxa as well as track the population levels of established or endemic pests. Where these surveillance traps have low specificity and capture non-target endemic species in excess of the target pests, the need for extensive specimen sorting and identification creates a major diagnostic bottleneck. While the recent development of standardized molecular diagnostics has partly alleviated this requirement, the single specimen per reaction nature of these methods does not readily scale to the sheer number of insects trapped in surveillance programmes. Consequently, target lists are often restricted to a few high-priority pests, allowing unanticipated species to avoid detection and potentially establish populations. DNA metabarcoding has recently emerged as a method for conducting simultaneous, multi-species identification of complex mixed communities and may lend itself ideally to rapid diagnostics of bulk insect trap samples. Moreover, the high-throughput nature of recent sequencing platforms could enable the multiplexing of hundreds of diverse trap samples on a single flow cell, thereby providing the means to dramatically scale up insect surveillance in terms of both the quantity of traps that can be processed concurrently and number of pest species that can be targeted. In this review of the metabarcoding literature, we explore how DNA metabarcoding could be tailored to the detection of invasive insects in a surveillance context and highlight the unique technical and regulatory challenges that must be considered when implementing high-throughput sequencing technologies into sensitive diagnostic applications.</p>
<h1>Improving eDNA-based protist diversity assessments using networks of amplicon sequence variants.</h1>
<p><a href="https://doi.org/10.1111/1462-2920.14764">doi:10.1111/1462-2920.14764</a></p>
<p>No keywords</p><p>Effective and precise grouping of highly similar sequences remains a major bottleneck in the evaluation of high-throughput sequencing datasets. Amplicon sequence variants (ASVs) offer a promising alternative that may supersede the widely used operational taxonomic units (OTUs) in environmental sequencing studies. We compared the performance of a recently developed pipeline based on the algorithm DADA2 for obtaining ASVs against a pipeline based on the algorithm SWARM for obtaining OTUs. Illumina-sequencing of 29 individual ciliate species resulted in up to 11 ASVs per species, while SWARM produced up to 19 OTUs per species. To improve the congruency between species diversity and molecular diversity, we applied sequence similarity networks (SSNs) for second-level sequence grouping into network sequence clusters (NSCs). At 100% sequence similarity in SWARM-SSNs, NSC numbers decreased from 7.9-fold overestimation without abundance filter, to 4.5-fold overestimation when an abundance filter was applied. For the DADA2-SSN approach, NSC numbers decreased from 3.5-fold to 3-fold overestimation. Rand index cluster analyses predicted best binning results between 97% and 94% sequence similarity for both DADA2-SSNs and SWARM-SSNs. Depending on the ecological questions addressed in an environmental sequencing study with protists we recommend ASVs as replacement for OTUs, best in combination with SSNs.</p>
<h1>Advancing the integration of multi-marker metabarcoding data in dietary analysis of trophic generalists.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13060">doi:10.1111/1755-0998.13060</a></p>
['bird', 'diet', 'metabarcoding', 'morphological identification', 'overlapping markers', 'secondary predation']<p>The application of DNA metabarcoding to dietary analysis of trophic generalists requires using multiple markers in order to overcome problems of primer specificity and bias. However, limited attention has been given to the integration of information from multiple markers, particularly when they partly overlap in the taxa amplified, and vary in taxonomic resolution and biases. Here, we test the use of a mix of universal and specific markers, provide criteria to integrate multi-marker metabarcoding data and a python script to implement such criteria and produce a single list of taxa ingested per sample. We then compare the results of dietary analysis based on morphological methods, single markers, and the proposed combination of multiple markers. The study was based on the analysis of 115 faeces from a small passerine, the Black Wheatears (Oenanthe leucura). Morphological analysis detected far fewer plant taxa (12) than either a universal 18S marker (57) or the plant trnL marker (124). This may partly reflect the detection of secondary ingestion by molecular methods. Morphological identification also detected far fewer taxa (23) than when using 18S (91) or the arthropod markers IN16STK (244) and ZBJ (231), though each method missed or underestimated some prey items. Integration of multi-marker data provided far more detailed dietary information than any single marker and estimated higher frequencies of occurrence of all taxa. Overall, our results show the value of integrating data from multiple, taxonomically overlapping markers in an example dietary data set.</p>
<h1>Land use is a determinant of plant pathogen alpha- but not beta-diversity.</h1>
<p><a href="https://doi.org/10.1111/Mec.15177">doi:10.1111/Mec.15177</a></p>
['environmental DNA', 'high-throughput sequencing', 'illumina', 'metabarcoding', 'plant pathogen communities']<p>Little is known about the diversity patterns of plant pathogens and how they change with land use at a broad scale. We employed DNA metabarcoding to describe the diversity and composition of putative plant pathogen communities in three substrates (soil, roots, and leaves) across five major land uses at a national scale. Almost all plant pathogen communities (fungi, oomycetes, and bacteria) showed strong responses to land use and substrate type. Land use category could explain up to 24% of the variance in composition between communities. Alpha-diversity (richness) of plant pathogens was consistently lower in natural forests than in agricultural systems. In planted forests, there was also generally low pathogen alpha-diversity in soil and roots, but alpha-diversity in leaves was high compared with most other land uses. In contrast to alpha-diversity, differences in within-land use beta-diversity of plant pathogens (the predictability of plant pathogen communities within land use) were subtle. Our results show that large-scale patterns and distributions of putative plant pathogens can be determined using metabarcoding, allowing some of the first landscape level insights into these critically important communities.</p>
<h1>An optimised eDNA protocol for detecting fish in lentic and lotic freshwaters using a small water volume.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0219218">doi:10.1371/Journal.Pone.0219218</a></p>
<p>No keywords</p><p>Environmental DNA is increasingly being used for assessing the presence and relative abundance of fish in freshwater, but existing protocols typically rely on filtering large volumes of water which is not always practical. We compared the effects of water volume, filtration type and eDNA extraction procedures in the detection of fish in three freshwater bodies (pond, lake and river) using a short fragment of the 12s rRNA mtDNA gene. Quantification of eDNA capture efficiency after DNA extraction, as well as amplification efficiency, were evaluated by conventional PCR and quantitative PCR. No significant differences on eDNA capture yield were found among freshwater bodies, but increasing water volume had a positive effect on eDNA capture and amplification efficiency. Although highest eDNA capture rates were obtained using 2 L of filtered water, 100 mL syringe filtration in combination with ethanol- sodium acetate precipitation proved to be more practical and increased quantitative PCR amplification efficiency by 6.4%. Our results indicate that such method may be optimal to detect fish species effectively across both lotic and lentic freshwater environments.</p>
<h1>Global distribution of Trebouxiophyceae diversity explored by high-throughput sequencing and phylogenetic approaches.</h1>
<p><a href="https://doi.org/10.1111/1462-2920.14738">doi:10.1111/1462-2920.14738</a></p>
<p>No keywords</p><p>Trebouxiophyceae are a ubiquitous class of Chlorophyta encountered in aquatic and terrestrial environments. Most taxa are photosynthetic, and many acts as photobionts in symbiotic relationships, while others are free-living. Trebouxiophyceae have also been widely investigated for their use for biotechnological applications. In this work, we aimed at obtaining a comprehensive image of their diversity by compiling the information of 435 freshwater, soil and marine environmental DNA samples surveyed with Illumina sequencing technology in order to search for the most relevant environments for bioprospecting. Freshwater and soil were most diverse and shared more than half of all operational taxonomic units (OTUs), however, their communities were significantly distinct. Oceans hosted the highest genetic novelty, and did not share any OTUs with the other environments; also, marine samples host more diversity in warm waters. Symbiotic genera usually found in lichens such as Trebouxia, Myrmecia and Symbiochloris were also abundantly detected in the ocean, suggesting either free-living lifestyles or unknown symbiotic relationships with marine planktonic organisms. Altogether, our study opens the way to new prospection for trebouxiophycean strains, especially in understudied environments like the ocean.</p>
<h1>The menu varies with metabarcoding practices: A case study with the bat Plecotus auritus.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0219135">doi:10.1371/Journal.Pone.0219135</a></p>
<p>No keywords</p><p>Metabarcoding of feces has revolutionized the knowledge of animal diets by providing unprecedented resolution of consumed resources. However, it is still unclear how different methodological approaches influence the ecological conclusions that can be drawn from such data. Here, we propose a critical evaluation of several data treatments on the inferred diet of the bat Plecotus auritus using guano regularly collected from various colonies throughout the entire active season. First and unlike previous claims, our data indicates that DNA extracted from large amounts of fecal material issued from guano accumulates yield broader taxonomic diversity of prey than smaller numbers of pellets would do, provided that extraction buffer volumes are adapted to such increased amounts of material. Second, trophic niche analyses based on prey occurrence data uncover strong seasonality in the bat's diet and major differences among neighboring maternity colonies. Third, while the removal of rare prey items is not always warranted as it introduces biases affecting particularly samples with greater prey species richness. Fourth, examination of distinct taxonomic depths in diet analyses highlights different aspects of food consumption providing a better understanding of the consumer's diet. Finally, the biologically meaningful patterns recovered with presence-absence approaches are virtually lost when attempting to quantify prey consumed using relative read abundances. Even in an ideal situation where reference barcodes are available for most potential prey species, inferring realistic patterns of prey consumption remains relatively challenging. Although best practice in metabarcoding analyses will depend on the aims of the study, several previous methodological recommendations seem unwarranted for studying such diverse diets as that of brown long-eared bats.</p>
<h1>High-throughput environmental DNA analysis informs a biological assessment of an urban stream.</h1>
<p><a href="https://doi.org/10.1016/J.Ecolind.2019.04.088">doi:10.1016/J.Ecolind.2019.04.088</a></p>
['Aquatic life use', 'Bioassessment', 'Biological impairment', 'Taxonomy', 'eDNA']<p>There is growing interest in the use of DNA barcoding and metabarcoding approaches to aid biological assessments and monitoring of waterbodies. While biodiversity measured by morphology and by DNA often has been found correlated, few studies have compared DNA data to established measures of impairment such as multimetric pollution tolerance indices used by many bioassessment programs. We incorporated environmental DNA (eDNA) metabarcoding of seston into a rigorous watershed-scale biological assessment of an urban stream to examine the extent to which eDNA richness and diversity patterns were correlated with multimetric indices and ecological impairment status designations. We also evaluated different filtering approaches and taxonomic classifications to identify best practices for environmental assessments. Seston eDNA revealed a wide diversity of eukaryotic taxa but was dominated by diatoms (36%). Differentiation among sites in alpha and beta diversity was greater when operational taxonomic units (OTUs) were classified taxonomically, but coarse resolution taxonomy (kingdom) was more informative than finer resolution taxonomy (family, genus). Correlations of DNA richness and diversity with multimetric indices for fish and macroinvertebrates were generally weak, possibly because Metazoa were not highly represented in our DNA dataset. Nonetheless, sites could be differentiated based on ecological impairment status, with more impaired sites having lower eDNA diversity as measured by the Shannon index, but higher taxonomic richness. Significant environmental drivers of community structure, as inferred from constrained ordination analyses, differed among kingdoms within the eDNA dataset, as well as from fish and macrobenthos, suggesting that eDNA provides novel environmental information. These results suggest that even a simple seston eDNA filtering protocol can provide biodiversity information of value to stream bioassessment programs. The approach bears further investigation as a potentially useful rapid assessment protocol to supplement more intensive field sampling efforts.</p>
<h1>Single Fragment or Bulk Soil DNA Metabarcoding: Which is Better for Characterizing Biological Taxa Found in Surface Soils for Sample Separation?</h1>
<p><a href="https://doi.org/E431">doi:E431</a></p>
['DNA barcoding', 'DNA metabarcoding', 'animals', 'forensic-type soils', 'plants']<p>In forensic geology casework, sample size typically limits routine characterization of material using bulk approaches. To address this, DNA-based characterization of biological taxa has received attention, as the taxa present can be useful for sample-to-sample comparisons and source attribution. In our initial work, low biodiversity was captured when DNA barcodes were Sanger-sequenced from plant and insect fragments isolated from 10 forensic-type surface soils. Considering some forensic laboratories now have access to massively parallel sequencing platforms, we assessed whether biological taxa present in the same surface soils could be better characterized using DNA metabarcoding. To achieve this, plant and animal barcodes were amplified and sequenced on an Illumina MiniSeq for three different DNA sample types (<i>n</i> = 50): individual fragments used in our initial study, and 250 and 100 mg of bulk soil (from the 10 sites used in the initial study). A total of 572 unique target barcode sequences passed quality filtering and were used in downstream statistical analyses: 54, 321, and 285 for individual fragments, 100 mg, and 250 mg bulk soil samples, respectively. Plant barcodes permitted some spatial separation of sample sites in non-metric multidimensional scaling plots; better separation was obtained for samples prepared from bulk soil. This study confirmed that bulk soil DNA metabarcoding is a better approach for characterizing biological taxa present in surface soils, which could supplement traditional geologic examinations.</p>
<h1>Towards an in-depth characterization of Symbiodiniaceae in tropical giant clams via metabarcoding of pooled multi-gene amplicons.</h1>
<p><a href="https://doi.org/10.7717/Peerj.6898">doi:10.7717/Peerj.6898</a></p>
['Biodiversity', 'High-throughput sequencing', 'Marine ecology', 'Multi-gene metabarcoding', 'South Pacific Ocean', 'Symbiosis', 'Tridacna']<p>High-throughput sequencing is revolutionizing our ability to comprehensively characterize free-living and symbiotic Symbiodiniaceae, a diverse dinoflagellate group that plays a critical role in coral reef ecosystems. Most studies however, focus on a single marker for metabarcoding Symbiodiniaceae, potentially missing important ecological traits that a combination of markers may capture. In this proof-of-concept study, we used a small set of symbiotic giant clam (<i>Tridacna maxima</i>) samples obtained from nine French Polynesian locations and tested a dual-index sequence library preparation method that pools and simultaneously sequences multiple Symbiodiniaceae gene amplicons per sample for in-depth biodiversity assessments. The rationale for this approach was to allow the metabarcoding of multiple genes without extra costs associated with additional single amplicon dual indexing and library preparations. Our results showed that the technique effectively recovered very similar proportions of sequence reads and dominant Symbiodiniaceae clades among the three pooled gene amplicons investigated per sample, and captured varying levels of phylogenetic resolution enabling a more comprehensive assessment of the diversity present. The pooled Symbiodiniaceae multi-gene metabarcoding approach described here is readily scalable, offering considerable analytical cost savings while providing sufficient phylogenetic information and sequence coverage.</p>
<h1>Taxonomic and Functional Characterization of the Microbial Community During Spontaneous <i>in vitro</i> Fermentation of Riesling Must.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2019.00697">doi:10.3389/Fmicb.2019.00697</a></p>
['alcoholic fermentation', 'metabarcoding', 'microbial diversity', 'shotgun sequencing', 'wine']<p>Although there is an extensive tradition of research into the microbes that underlie the winemaking process, much remains to be learnt. We combined the high-throughput sequencing (HTS) tools of metabarcoding and metagenomics, to characterize how microbial communities of Riesling musts sampled at four different vineyards, and their subsequent spontaneously fermented derivatives, vary. We specifically explored community variation relating to three points: (i) how microbial communities vary by vineyard; (ii) how community biodiversity changes during alcoholic fermentation; and (iii) how microbial community varies between musts that successfully complete alcoholic fermentation and those that become 'stuck' in the process. Our metabarcoding data showed a general influence of microbial composition at the vineyard level. Two of the vineyards (4 and 5) had strikingly a change in the differential abundance of <i>Metschnikowia</i>. We therefore additionally performed shotgun metagenomic sequencing on a subset of the samples to provide preliminary insights into the potential relevance of this observation, and used the data to both investigate functional potential and reconstruct draft genomes (bins). At these two vineyards, we also observed an increase in non-Saccharomycetaceae fungal functions, and a decrease in bacterial functions during the early fermentation stage. The binning results yielded 11 coherent bins, with both vineyards sharing the yeast bins <i>Hanseniaspora</i> and <i>Saccharomyces</i>. Read recruitment and functional analysis of this data revealed that during fermentation, a high abundance of <i>Metschnikowia</i> might serve as a biocontrol agent against bacteria, via a putative iron depletion pathway, and this in turn could help <i>Saccharomyces</i> dominate the fermentation. During alcoholic fermentation, we observed a general decrease in biodiversity in both the metabarcoding and metagenomic data. Unexpected <i>Micrococcus</i> behavior was observed in vineyard 4 according to metagenomic analyses based on reference-based read mapping. Analysis of open reading frames using these data showed an increase of functions assigned to class Actinobacteria in the end of fermentation. Therefore, we hypothesize that bacteria might sit-and-wait until <i>Saccharomyces</i> activity slows down. Complementary approaches to annotation instead of relying a single database provide more coherent information true species. Lastly, our metabarcoding data enabled us to identify a relationship between stuck fermentations and <i>Starmerella</i> abundance. Given that robust chemical analysis indicated that although the stuck samples contained residual glucose, all fructose had been consumed, we hypothesize that this was because fructophilic <i>Starmerella</i>, rather than <i>Saccharomyces</i>, dominated these fermentations. Overall, our results showcase the different ways in which metagenomic analyses can improve our understanding of the wine alcoholic fermentation process.</p>
<h1>Benthic Diatom Communities in an Alpine River Impacted by Waste Water Treatment Effluents as Revealed Using DNA Metabarcoding.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2019.00653">doi:10.3389/Fmicb.2019.00653</a></p>
['DNA metabarcoding', 'WWTP effluents', 'diatom communities', 'functional traits', 'indicator species analysis', 'pharmaceuticals', 'water quality index']<p>Freshwater ecosystems are continuously affected by anthropogenic pressure. One of the main sources of contamination comes from wastewater treatment plant (WWTP) effluents that contain wide range of micro- and macropollutants. Chemical composition, toxicity levels and impact of treated effluents (TEs) on the recipient aquatic ecosystems may strongly differ depending on the wastewater origin. Compared to urban TEs, hospital ones may contain more active pharmaceutical substances. Benthic diatoms are relevant ecological indicators because of their high species and ecological diversity and rapid response to human pressure. They are routinely used for water quality monitoring. However, there is a knowledge gap on diatom communities' development and behavior in treated wastewater in relation to prevailing micro- and macropollutants. In this study, we aim to (1) investigate the response of diatom communities to urban and hospital TEs, and (2) evaluate TEs effect on communities in the recipient river. Environmental biofilms were colonized in TEs and the recipient river up- and downstream from the WWTP output to study benthic diatoms using DNA metabarcoding combined with high-throughput sequencing (HTS). In parallel, concentrations of nutrients, pharmaceuticals and seasonal conditions were recorded. Diatom metabarcoding showed that benthic communities differed strongly in their diversity and structure depending on the habitat. TE sites were generally dominated by few genera with polysaprobic preferences belonging to the motile guild, while river sites favored diverse communities from oligotrophic and oligosaprobic groups. Seasonal changes were visible to lower extent. To categorize parameters important for diatom changes we performed redundancy analysis which suggested that communities within TE sites were associated to higher concentrations of beta-blockers and non-steroidal anti-inflammatory drugs in urban effluents vs. antibiotics and orthophosphate in hospital effluents. Furthermore, indicator species analysis showed that 27% of OTUs detected in river downstream communities were indicator for urban or hospital TE sites and were absent in the river upstream. Finally, biological diatom index (BDI) calculated to evaluate the ecological status of the recipient river suggested water quality decrease linked to the release of TEs. Thus, in-depth assessment of diatom community composition using DNA metabarcoding is proposed as a promising technique to highlight the disturbing effect of pollutants in Alpine rivers.</p>
<h1>Using eDNA to biomonitor the fish community in a tropical oligotrophic lake.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0215505">doi:10.1371/Journal.Pone.0215505</a></p>
<p>No keywords</p><p>Environmental DNA (eDNA) is an effective approach for detecting vertebrates and plants, especially in aquatic ecosystems, but prior studies have largely examined eDNA in cool temperate settings. By contrast, this study employs eDNA to survey the fish fauna in tropical Lake Bacalar (Mexico) with the additional goal of assessing the possible presence of invasive fishes, such as Amazon sailfin catfish and tilapia. Sediment and water samples were collected from eight stations in Lake Bacalar on three occasions over a 4-month interval. Each sample was stored in the presence or absence of lysis buffer to compare eDNA recovery. Short fragments (184-187 bp) of the cytochrome c oxidase I (COI) gene were amplified using fusion primers and then sequenced on Ion Torrent PGM or S5 before their source species were determined using a custom reference sequence database constructed on BOLD. In total, eDNA sequences were recovered from 75 species of vertebrates including 47 fishes, 15 birds, 7 mammals, 5 reptiles, and 1 amphibian. Although all species are known from this region, six fish species represent new records for the study area, while two require verification. Sequences for five species (2 birds, 2 mammals, 1 reptile) were only detected from sediments, while sequences from 52 species were only recovered from water. Because DNA from the Amazon sailfin catfish was not detected, we used a mock eDNA experiment to confirm our methods would enable its detection. In summary, we developed protocols that recovered eDNA from tropical oligotrophic aquatic ecosystems and confirmed their effectiveness in detecting fishes and diverse species of vertebrates.</p>
<h1>Distribution patterns of dinoflagellate communities along the Songhua River.</h1>
<p><a href="https://doi.org/10.7717/Peerj.6733">doi:10.7717/Peerj.6733</a></p>
['Dinoflagellate', 'Dispersal', 'Metacommunity metabarcoding', 'RDA', 'Species sorting']<p>Dinoflagellates have the potential to pose severe ecological and economic damages to aquatic ecosystems. It is therefore largely needed to understand the causes and consequences of distribution patterns of dinoflagellate communities in order to manage potential environmental problems. However, a majority of studies have focused on marine ecosystems, while the geographical distribution patterns of dinoflagellate communities and associated determinants in freshwater ecosystems remain unexplored, particularly in running water ecosystems such as rivers and streams.</p>
<h1>Assessing the influence of sample tagging and library preparation on DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13018">doi:10.1111/1755-0998.13018</a></p>
['Illumina', 'fusion primers', 'index', 'one-step', 'tag switching', 'two-step']<p>Metabarcoding is increasingly used to assess species diversity by high-throughput sequencing where millions of sequences can be generated in parallel and multiple samples can be analysed in one sequencing run. Generating amplified fragments with a unique sequence identifier ('tag') for each sample is crucial, as it allows assigning sequences to the original samples. The tagging through so-called fusion primers is a fast and cheap alternative to commercially produced ligation-based kits. However, little is known about potential bias and inconsistencies introduced by the long nucleotide tail attached to those primers, which could lead to deficient reports of community composition in metabarcoding studies. We therefore tested the consistency and taxa detection efficiency of fusion primers in (1) a one-step and (2) two-step PCR protocol as well as (3) a commercially manufactured Illumina kit using mock communities of known composition. The Illumina kit delivered the most consistent results and detected the highest number of taxa. However, success of the two-step PCR approach was only marginally lower compared to the kit with the additional advantage of a much more competitive price per library. While most taxa were also detected with the one-step PCR approach, the consistency between replicates including read abundance was substantially lower. Our results highlight that method choice depends on the precision needed for analysis as well as on economic considerations and recommend the Illumina kit to obtain most accurate results and the two-step PCR approach as a much cheaper yet very robust alternative.</p>
<h1>Evaluating next-generation sequencing (NGS) methods for routine monitoring of wild bees: Metabarcoding, mitogenomics or NGS barcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.13013">doi:10.1111/1755-0998.13013</a></p>
['DNA barcoding', 'conservation biology', 'insects', 'molecular identification', 'pollinators', 'survey']<p>Implementing cost-effective monitoring programs for wild bees remains challenging due to the high costs of sampling and specimen identification. To reduce costs, next-generation sequencing (NGS)-based methods have lately been suggested as alternatives to morphology-based identifications. To provide a comprehensive presentation of the advantages and weaknesses of different NGS-based identification methods, we assessed three of the most promising ones, namely metabarcoding, mitogenomics and NGS barcoding. Using a regular monitoring data set (723 specimens identified using morphology), we found that NGS barcoding performed best for both species presence/absence and abundance data, producing only few false positives (3.4%) and no false negatives. In contrast, the proportion of false positives and false negatives was higher using metabarcoding and mitogenomics. Although strong correlations were found between biomass and read numbers, abundance estimates significantly skewed the communities' composition in these two techniques. NGS barcoding recovered the same ecological patterns as morphology. Ecological conclusions based on metabarcoding and mitogenomics were similar to those based on morphology when using presence/absence data, but different when using abundance data. In terms of workload and cost, we show that metabarcoding and NGS barcoding can compete with morphology, but not mitogenomics which was consistently more expensive. Based on these results, we advocate that NGS barcoding is currently the seemliest NGS method for monitoring of wild bees. Furthermore, this method has the advantage of potentially linking DNA sequences with preserved voucher specimens, which enable morphological re-examination and will thus produce verifiable records which can be fed into faunistic databases.</p>
<h1>Diversity of photosynthetic picoeukaryotes in eutrophic shallow lakes as assessed by combining flow cytometry cell-sorting and high throughput sequencing.</h1>
<p><a href="https://doi.org/Fiz038">doi:Fiz038</a></p>
['eutrophic shallow lakes', 'flow cytometry', 'metabarcoding', 'picoplankton']<p>Photosynthetic picoeukaryotes (PPE) are key components of primary production in marine and freshwater ecosystems. In contrast with those of marine environments, freshwater PPE groups have received little attention. In this work, we used flow cytometry cell sorting, microscopy and metabarcoding to investigate the composition of small photosynthetic eukaryote communities from six eutrophic shallow lakes in South America, Argentina. We compared the total molecular diversity obtained from PPE sorted populations as well as from filtered total plankton samples (FTP). Most reads obtained from sorted populations belonged to the classes: Trebouxiophyceae, Chlorophyceae and Bacillariophyceae. We retrieved sequences from non-photosynthetic groups, such as Chytridiomycetes and Ichthyosporea which contain a number of described parasites, indicating that these organisms were probably in association with the autotrophic cells sorted. Dominant groups among sorted PPEs were poorly represented in FTP and their richness was on average lower than in the sorted samples. A significant number of operational taxonomic units (OTUs) were exclusively found in sorting samples, emphasizing that sequences from FTP underestimate the diversity of PPE. Moreover, 22% of the OTUs found among the dominant groups had a low similarity (<95%) with reported sequences in public databases, demonstrating a high potential for novel diversity in these lakes.</p>
<h1>Targeted Amplification and Sequencing of Ancient Environmental and Sedimentary DNA.</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-9176-1_16">doi:10.1007/978-1-4939-9176-1_16</a></p>
['Amplicon sequencing', 'Ancient DNA', 'Environmental DNA', 'Metabarcoding', 'Universal primers', 'sedaDNA']<p>All organisms release their DNA into the environment through processes such as excretion and the senescence of tissues and limbs. This DNA, often referred to as environmental DNA (eDNA) or sedimentary ancient DNA (sedaDNA), can be recovered from both present-day and ancient soils, fecal samples, bodies of water and lake cores, and even air. While eDNA is a potentially useful record of past and present biodiversity, several challenges complicate data generation and interpretation of results. Most importantly, eDNA samples tend to be highly taxonomically mixed, and the target organism or group of organisms may be present at very low abundance within this mixture. To overcome this challenge, enrichment approaches are often used to target specific taxa of interest. Here, we describe a protocol to amplify metabarcodes or short, variable loci that identify lineages within broad taxonomic groups (e.g., plants, mammals), using the polymerase chain reaction (PCR) with established generic "barcode" primers. We also provide a catalog of animal and plant barcode primers that, because they target relatively short fragments of DNA, are potentially suitable for use with degraded DNA.</p>
<h1>[Application of DNA metabarcoding in species identification of Chinese herbal medicines].</h1>
<p><a href="https://doi.org/10.19540/J.Cnki.Cjcmm.2019.0001">doi:10.19540/J.Cnki.Cjcmm.2019.0001</a></p>
['DNA metabarcoding', 'mixed Chinese herbal medicines', 'species identification']<p>DNA metabarcoding,one rapid and robust method using specific standard DNA fragments,has been widely used for rapid species identification of a bulk sample through high-throughput sequencing technologies.While it has been widely used in the studies of metagenomics,animal and plant biodiversity,it has gradually come to be used as a profitable method in species identification of mixed Chinese herbal medicines.In this paper,we mainly summarize the current studies of the application of DNA metabarcoding in species identification of mixed Chinese herbal medicines.Moreover,high-throughput sequencing technologies adopted in those studies,such as Sanger,the next-generation,and third-generation sequencing technologies,are discussed.It is conducted to provide a theoretical guidance for the application of DNA metabarcoding in species identification of mixed Chinese herbal medicines and in more other biodiversity studies.</p>
<h1>Evaluation of detection probabilities at the water-filtering and initial PCR steps in environmental DNA metabarcoding using a multispecies site occupancy model.</h1>
<p><a href="https://doi.org/10.1038/S41598-019-40233-1">doi:10.1038/S41598-019-40233-1</a></p>
<p>No keywords</p><p>Environmental DNA (eDNA) metabarcoding is a recently developed method to assess biodiversity based on a high-throughput parallel DNA sequencing applied to DNA present in the ecosystem. Although eDNA metabarcoding enables a rapid assessment of biodiversity, it is prone to species detection errors that may occur at sequential steps in field sampling, laboratory experiments, and bioinformatics. In this study, we illustrate how the error rates in the eDNA metabarcoding-based species detection can be accounted for by applying the multispecies occupancy modelling framework. We report a case study with the eDNA sample from an aquarium tank in which the detection probabilities of species in the two major steps of eDNA metabarcoding, filtration and PCR, across a range of PCR annealing temperatures, were examined. We also show that the results can be used to examine the efficiency of species detection under a given experimental design and setting, in terms of the efficiency of species detection, highlighting the usefulness of the multispecies site occupancy modelling framework to study the optimum conditions for molecular experiments.</p>
<h1>Nanopore sequencing of long ribosomal DNA amplicons enables portable and simple biodiversity assessments with high phylogenetic resolution across broad taxonomic scale.</h1>
<p><a href="https://doi.org/Giz006">doi:Giz006</a></p>
['MinION', 'Oxford Nanopore Technologies', 'biodiversity', 'eukaryotes', 'long DNA barcodes', 'metabarcoding', 'ribosomal']<p>In light of the current biodiversity crisis, DNA barcoding is developing into an essential tool to quantify state shifts in global ecosystems. Current barcoding protocols often rely on short amplicon sequences, which yield accurate identification of biological entities in a community but provide limited phylogenetic resolution across broad taxonomic scales. However, the phylogenetic structure of communities is an essential component of biodiversity. Consequently, a barcoding approach is required that unites robust taxonomic assignment power and high phylogenetic utility. A possible solution is offered by sequencing long ribosomal DNA (rDNA) amplicons on the MinION platform (Oxford Nanopore Technologies).</p>
<h1>Changes in Trophic Groups of Protists With Conversion of Rainforest Into Rubber and Oil Palm Plantations.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2019.00240">doi:10.3389/Fmicb.2019.00240</a></p>
['Indonesia', 'V4 region of 18S rRNA', 'community composition', 'high-throughput sequencing', 'rainforest transformation', 'soil protist diversity']<p>Protists, abundant but enigmatic single-celled eukaryotes, are important soil microbiota providing numerous ecosystem functions. We employed high-throughput sequencing of environmental DNA, targeting the V4 region of the 18S rRNA gene, to characterize changes in their abundance, species richness, and community structure with conversion of lowland rainforest into rubber agroforest (jungle rubber), and rubber and oil palm plantations; typical agricultural systems in Sumatra, Indonesia. We identified 5,204 operational taxonomic units (OTUs) at 97% identity threshold of protists from 32 sites. Protists species richness was similar in rainforest, jungle rubber and oil palm plantations but significantly lower in rubber plantations. After standardization, 4,219 OTUs were assigned to five trophic groups, and inspected for effects of land-use change, and potential biotic and abiotic driving factors. The most abundant trophic group was phagotrophs (52%), followed by animal parasites (29%), photoautotrophs (12%), plant parasites (1%), and symbionts (<1%). However, the relative abundance and OTU richness of phagotrophs and photoautotrophs increased significantly with increasing land-use intensity. This was similar, but less pronounced, for the relative abundance of symbionts. Animal and plant parasites decreased significantly in abundance and species richness with increasing land-use intensity. Community compositions and factors affecting the structure of individual trophic groups differed between land-use systems. Parasites were presumably mainly driven by the abundance and species richness of their hosts, while phagotrophs by changes in soil pH and increase in Gram-positive bacteria, and photoautotrophs by light availability. Overall, the results show that relative species richness, relative abundance, and community composition of individual trophic groups of protists in tropical lowland rainforest significantly differ from that in converted ecosystems. This is likely associated with changes in ecosystem functioning. The study provides novel insight into protist communities and their changes with land-use intensity in tropical lowland ecosystems. We show, that trophic groups of protists are powerful indicators reflecting changes in the functioning of ecosystems with conversion of rainforest into monoculture plantations.</p>
<h1>Metabarcoding free-living marine nematodes using curated 18S and CO1 reference sequence databases for species-level taxonomic assignments.</h1>
<p><a href="https://doi.org/10.1002/Ece3.4814">doi:10.1002/Ece3.4814</a></p>
['Nematoda', 'UClust', 'USearch9', 'metabarcoding', 'mock community', 'reference sequence database']<p>High-throughput sequencing has the potential to describe biological communities with high efficiency yet comprehensive assessment of diversity with species-level resolution remains one of the most challenging aspects of metabarcoding studies. We investigated the utility of curated ribosomal and mitochondrial nematode reference sequence databases for determining phylum-specific species-level clustering thresholds. We compiled 438 ribosomal and 290 mitochondrial sequences which identified 99% and 94% as the species delineation clustering threshold, respectively. These thresholds were evaluated in HTS data from mock communities containing 39 nematode species as well as environmental samples from Vietnam. We compared the taxonomic description of the mocks generated by two read-merging and two clustering algorithms and the cluster-free Dada2 pipeline. Taxonomic assignment with the RDP classifier was assessed under different training sets. Our results showed that 36/39 mock nematode species were identified across the molecular markers (18S: 32, JB2: 19, JB3: 21) in UClust_ref OTUs at their respective clustering thresholds, outperforming UParse_denovo and the commonly used 97% similarity. Dada2 generated the most realistic number of ASVs (18S: 83, JB2: 75, JB3: 82), collectively identifying 30/39 mock species. The ribosomal marker outperformed the mitochondrial markers in terms of species and genus-level detections for both OTUs and ASVs. The number of taxonomic assignments of OTUs/ASVs was highest when the smallest reference database containing only nematode sequences was used and when sequences were truncated to the respective amplicon length. Overall, OTUs generated more species-level detections, which were, however, associated with higher error rates compared to ASVs. Genus-level assignments using ASVs exhibited higher accuracy and lower error rates compared to species-level assignments, suggesting that this is the most reliable pipeline for rapid assessment of alpha diversity from environmental samples.</p>
<h1>SLIM: a flexible web application for the reproducible processing of environmental DNA metabarcoding data.</h1>
<p><a href="https://doi.org/10.1186/S12859-019-2663-2">doi:10.1186/S12859-019-2663-2</a></p>
['Amplicon sequencing', 'High-throughput sequencing', 'Molecular ecology', 'Pipeline', 'Reproducibility', 'eDNA metabarcoding']<p>High-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) has become a routine tool for biodiversity survey and ecological studies. By including sample-specific tags in the primers prior PCR amplification, it is possible to multiplex hundreds of samples in a single sequencing run. The analysis of millions of sequences spread into hundreds to thousands of samples prompts for efficient, automated yet flexible analysis pipelines. Various algorithms and software have been developed to perform one or multiple processing steps, such as paired-end reads assembly, chimera filtering, Operational Taxonomic Unit (OTU) clustering and taxonomic assignment. Some of these software are now well established and widely used by scientists as part of their workflow. Wrappers that are capable to process metabarcoding data from raw sequencing data to annotated OTU-to-sample matrix were also developed to facilitate the analysis for non-specialist users. Yet, most of them require basic bioinformatic or command-line knowledge, which can limit the accessibility to such integrative toolkits. Furthermore, for flexibility reasons, these tools have adopted a step-by-step approach, which can prevent an easy automation of the workflow, and hence hamper the analysis reproducibility.</p>
<h1>Unmasking the identity of toxigenic cyanobacteria driving a multi-toxin bloom by high-throughput sequencing of cyanotoxins genes and 16S rRNA metabarcoding.</h1>
<p><a href="https://doi.org/S0048-9697(19)30564-9">doi:S0048-9697(19)30564-9</a></p>
['Anatoxin-a', 'Cuspidothrix', 'High-throughput sequencing', 'Microcystin', 'Planktothrix', 'Saxitoxin']<p>Cyanobacterial harmful algal blooms (CyanoHABs) are complex communities that include coexisting toxic and non-toxic strains only distinguishable by genetic methods. This study shows a water-management oriented use of next generation sequencing (NGS) to specifically pinpoint toxigenic cyanobacteria within a bloom simultaneously containing three of the most widespread cyanotoxins (the hepatotoxins microcystins, MCs; and the neurotoxins anatoxin-a, ATX, and saxitoxins, STXs). The 2013 summer bloom in Rosarito reservoir (Spain) comprised 33 cyanobacterial OTUs based on 16S rRNA metabarcoding, 7 of which accounted for as much as 96.6% of the community. Cyanotoxins and their respective biosynthesis genes were concurrently present throughout the entire bloom event including: MCs and mcyE gene; ATX and anaF gene; and STXs and sxtI gene. NGS applied to amplicons of cyanotoxin-biosynthesis genes unveiled 6 toxigenic OTUs, comprising 3 involved in MCs production (Planktothrix agardhii and 2 Microcystis spp.), 2 in ATX production (Cuspidothrix issatschenkoi and Phormidium/Tychonema spp.) and 1 in STXs production (Aphanizomenon gracile). These toxigenic taxa were also present in 16S rRNA OTUs list and their relative abundance was positively correlated with the respective toxin concentrations. Our results point at MC-producing P. agardhii and ATX-producing C. issatschenkoi as the main contributors to the moderate toxin concentrations observed, and suggest that their distribution in Southern Europe is broader than previously thought. Our findings also stress the need for monitoring low-abundance cyanobacteria (<1% relative abundance) in cyanotoxicity studies, and provide novel data on the presence of picocyanobacteria and potentially ATX-producing benthic taxa (e.g., Phormidium) in deep thermally-stratified water bodies. This study showcases a straightforward use of amplicon metagenomics of cyanotoxin biosynthesis genes in a multi-toxin bloom thus illustrating the broad applicability of NGS for water management in risk-oriented monitoring of CyanoHABs.</p>
<h1>High-Throughput Sequencing to Investigate Phytopathogenic Fungal Propagules Caught in Baited Insect Traps.</h1>
<p><a href="https://doi.org/E15">doi:E15</a></p>
['HTS', 'forest', 'fungi', 'insects', 'metagenomics', 'oomycete', 'vectors']<p>Studying the means of dispersal of plant pathogens is crucial to better understand the dynamic interactions involved in plant infections. On one hand, entomologists rely mostly on both traditional molecular methods and morphological characteristics, to identify pests. On the other hand, high-throughput sequencing (HTS) is becoming the go-to avenue for scientists studying phytopathogens. These organisms sometimes infect plants, together with insects. Considering the growing number of exotic insect introductions in Canada, forest pest-management efforts would benefit from the development of a high-throughput strategy to investigate the phytopathogenic fungal and oomycete species interacting with wood-boring insects. We recycled formerly discarded preservative fluids from the Canadian Food Inspection Agency annual survey using insect traps and analysed more than one hundred samples originating from across Canada. Using the Ion Torrent Personal Genome Machine (PGM) HTS technology and fusion primers, we performed metabarcoding to screen unwanted fungi and oomycetes species, including <i>Phytophthora</i> spp. Community profiling was conducted on the four different wood-boring, insect-attracting semiochemicals; although the preservative (contained ethanol) also attracted other insects. Phytopathogenic fungi (e.g., <i>Leptographium</i> spp<i>.</i> and <i>Meria</i> <i>laricis</i> in the pine sawyer semiochemical) and oomycetes (mainly <i>Peronospora</i> spp. and <i>Pythium</i> aff. <i>hypogynum</i> in the General Longhorn semiochemical), solely associated with one of the four types of semiochemicals, were detected. This project demonstrated that the insect traps' semiochemical microbiome represents a new and powerful matrix for screening phytopathogens. Compared to traditional diagnostic techniques, the fluids allowed for a faster and higher throughput assessment of the biodiversity contained within. Additionally, minimal modifications to this approach would allow it to be used in other phytopathology fields.</p>
<h1>Assessing high-throughput environmental DNA extraction methods for meta-barcode characterization of aquatic microbial communities.</h1>
<p><a href="https://doi.org/10.2166/Wh.2018.108">doi:10.2166/Wh.2018.108</a></p>
<p>No keywords</p><p>The characterization of microbial community dynamics using genomic methods is rapidly expanding, impacting many fields including medical, ecological, and environmental research and applications. One of the biggest challenges for such studies is the isolation of environmental DNA (eDNA) from a variety of samples, diverse microbes, and widely variable community compositions. The current study developed environmentally friendly, user safe, economical, and high throughput eDNA extraction methods for mixed aquatic microbial communities and tested them using 16 s rRNA gene meta-barcoding. Five different lysis buffers including (1) cetyltrimethylammonium bromide (CTAB), (2) digestion buffer (DB), (3) guanidinium isothiocyanate (GITC), (4) sucrose lysis (SL), and (5) SL-CTAB, coupled with four different purification methods: (1) phenol-chloroform-isoamyl alcohol (PCI), (2) magnetic Bead-Robotic, (3) magnetic Bead-Manual, and (4) membrane-filtration were tested for their efficacy in extracting eDNA from recreational freshwater samples. Results indicated that the CTAB-PCI and SL-Bead-Robotic methods yielded the highest genomic eDNA concentrations and succeeded in detecting the core microbial community including the rare microbes. However, our study recommends the SL-Bead-Robotic eDNA extraction protocol because this method is safe, environmentally friendly, rapid, high-throughput and inexpensive.</p>
<h1>Broadening the taxonomic scope of coral reef palaeoecological studies using ancient DNA.</h1>
<p><a href="https://doi.org/10.1111/Mec.15038">doi:10.1111/Mec.15038</a></p>
['ancient DNA', 'coral reef', 'environmental DNA', 'macroalgae', 'marine palaeoecology', 'metabarcoding']<p>Marine environments face acute pressures from human impacts, often resulting in substantial changes in community structure. On the inshore Great Barrier Reef (GBR), palaeoecological studies show the collapse of the previously dominant coral Acropora from the impacts of degraded water quality associated with European colonization. Even more dramatic impacts can result in the replacement of corals by fleshy macroalgae on modern reefs, but their past distribution is unknown because they leave no fossil record. Here, we apply DNA metabarcoding and high-throughput sequencing of the 18S rDNA gene on palaeoenvironmental DNA (aeDNA) derived from sediment cores at two sites on Pandora Reef (GBR), to enhance palaeoecological studies by incorporating key soft-bodied taxa, including macroalgae. We compared temporal trends in this aeDNA record with those of coral genera derived from macrofossils. Multivariate analysis of 12 eukaryotic groups from the aeDNA community showed wide variability over the past 750 years. The occurrence of brown macroalgae was negatively correlated only with the dominant coral at both sites. The occurrence of coralline and green macroalgae was positively correlated with only the dominant coral at one of the sites, where we also observed a significant association between the whole coral community and the occurrence of each of the three macroalgae groups. Our results demonstrate that reef sediments can provide a valuable archive for understanding the past distribution and occurrence of important soft-bodied reef dwellers. Combining information from fossils and aeDNA provides an enhanced understanding of temporal changes of reefs ecosystems at decadal to millennial timescales.</p>
<h1>Environmental DNA metabarcoding for fish community analysis in backwater lakes: A comparison of capture methods.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0210357">doi:10.1371/Journal.Pone.0210357</a></p>
<p>No keywords</p><p>The use of environmental DNA (eDNA) methods for community analysis has recently been developed. High-throughput parallel DNA sequencing (HTS), called eDNA metabarcoding, has been increasingly used in eDNA studies to examine multiple species. However, eDNA metabarcoding methodology requires validation based on traditional methods in all natural ecosystems before a reliable method can be established. To date, relatively few studies have performed eDNA metabarcoding of fishes in aquatic environments where fish communities were intensively surveyed using multiple traditional methods. Here, we have compared fish communities' data from eDNA metabarcoding with seven conventional multiple capture methods in 31 backwater lakes in Hokkaido, Japan. We found that capture and field surveys of fishes were often interrupted by macrophytes and muddy sediments in the 31 lakes. We sampled 1 L of the surface water and analyzed eDNA using HTS. We also surveyed the fish communities using seven different capture methods, including various types of nets and electrofishing. At some sites, we could not detect any eDNA, presumably because of the polymerase chain reaction (PCR) inhibition. We also detected the marine fish species as sewage-derived eDNA. Comparisons of eDNA metabarcoding and capture methods showed that the detected fish communities were similar between the two methods, with an overlap of 70%. Thus, our study suggests that to detect fish communities in backwater lakes, the performance of eDNA metabarcoding with the use of 1 L surface water sampling is similar to that of capturing methods. Therefore, eDNA metabarcoding can be used for fish community analysis but environmental factors that can cause PCR inhibition, should be considered in eDNA applications.</p>
<h1>Measuring global fish species richness with eDNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12929">doi:10.1111/1755-0998.12929</a></p>
['biodiversity', 'environmental DNA', 'freshwater', 'marine']<p>Despite mounting threats to global freshwater and marine biodiversity, including climate change, habitat alteration, overharvesting and pollution, we struggle to know which species are present below the water's surface that are suffering from these stressors. However, the idea that a water sample containing environmental DNA (eDNA) can be screened using high-throughput sequencing and bioinformatics to reveal the identity of aquatic species is a revolutionary advance for studying the patterns of species extirpation, invasive species establishment and the dynamics of species richness. To date, many of the critical tests of fisheries diversity using this metabarcoding approach have been conducted in lower diversity systems (<40 fish species), but in this issue of Molecular Ecology Resources, Cilleros et al. (2018) described their eDNA application in the species-rich French Guiana fishery (>200 fish species) and showed the greater potential and some limitations of using eDNA in species-rich environments.</p>
<h1>Are drivers of root-associated fungal community structure context specific?</h1>
<p><a href="https://doi.org/10.1038/S41396-019-0350-Y">doi:10.1038/S41396-019-0350-Y</a></p>
<p>No keywords</p><p>The composition and structure of plant-root-associated fungal communities are determined by local abiotic and biotic conditions. However, the relative influence and identity of relationships to abiotic and biotic factors may differ across environmental and ecological contexts, and fungal functional groups. Thus, understanding which aspects of root-associated fungal community ecology generalise across contexts is the first step towards a more predictive framework. We investigated how the relative importance of biotic and abiotic factors scale across environmental and ecological contexts using high-throughput sequencing (ca. 55 M Illumina metabarcoding sequences) of >260 plant-root-associated fungal communities from six UK salt marshes across two geographic regions (South-East and North-West England) in winter and summer. Levels of root-associated fungal diversity were comparable with forests and temperate grasslands, quadrupling previous estimates of salt-marsh fungal diversity. Whilst abiotic variables were generally most important, a range of site- and spatial scale-specific abiotic and biotic drivers of diversity and community composition were observed. Consequently, predictive models of diversity trained on one site, extrapolated poorly to others. Fungal taxa from the same functional groups responded similarly to the specific drivers of diversity and composition. Thus site, spatial scale and functional group are key factors that, if accounted for, may lead to a more predictive understanding of fungal community ecology.</p>
<h1>From feces to data: A metabarcoding method for analyzing consumed and available prey in a bird-insect food web.</h1>
<p><a href="https://doi.org/10.1002/Ece3.4787">doi:10.1002/Ece3.4787</a></p>
['DNA barcoding', 'Lepidoptera', 'dietary ecology', 'fecal DNA', 'frass', 'insectivorous birds', 'metagenomics']<p>Diets play a key role in understanding trophic interactions. Knowing the actual structure of food webs contributes greatly to our understanding of biodiversity and ecosystem functioning. The research of prey preferences of different predators requires knowledge not only of the prey consumed, but also of what is available. In this study, we applied DNA metabarcoding to analyze the diet of 4 bird species (willow tits <i>Poecile montanus</i>, Siberian tits <i>Poecile cinctus</i>, great tits <i>Parus major</i> and blue tits <i>Cyanistes caeruleus</i>) by using the feces of nestlings. The availability of their assumed prey (Lepidoptera) was determined from feces of larvae (frass) collected from the main foraging habitat, birch (<i>Betula</i> spp.) canopy. We identified 53 prey species from the nestling feces, of which 11 (21%) were also detected from the frass samples (eight lepidopterans). Approximately 80% of identified prey species in the nestling feces represented lepidopterans, which is in line with the earlier studies on the parids' diet. A subsequent laboratory experiment showed a threshold for fecal sample size and the barcoding success, suggesting that the smallest frass samples do not contain enough larval DNA to be detected by high-throughput sequencing. To summarize, we apply metabarcoding for the first time in a combined approach to identify available prey (through frass) and consumed prey (via nestling feces), expanding the scope and precision for future dietary studies on insectivorous birds.</p>
<h1>Biodiversity assessments in the 21st century: the potential of insect traps to complement environmental samples for estimating eukaryotic and prokaryotic diversity using high-throughput DNA metabarcoding <sup>1</sup>.</h1>
<p><a href="https://doi.org/10.1139/Gen-2018-0096">doi:10.1139/Gen-2018-0096</a></p>
['16S rDNA', '18S rDNA', 'ADN environnemental', 'ADNr 16S', 'ADNr 18S', 'COI ADNmt', 'COI mtDNA', 'environmental DNA', 'metabarcoding', 'métacodage à barres', 'non-destructive DNA extraction', 'protocole d’extraction d’ADN non-destructif']<p>The rapid loss of biodiversity, coupled with difficulties in species identification, call for innovative approaches to assess biodiversity. Insects make up a substantial proportion of extant diversity and play fundamental roles in any given ecosystem. To complement morphological species identification, new techniques such as metabarcoding make it possible to quantify insect diversity and insect-ecosystem interactions through DNA sequencing. Here we examine the potential of bulk insect samples (i.e., containing many non-sorted specimens) to assess prokaryote and eukaryote biodiversity and to complement the taxonomic coverage of soil samples. We sampled 25 sites on three continents and in various ecosystems, collecting insects with SLAM traps (Brazil) and Malaise traps (South Africa and Sweden). We then compared our diversity estimates with the results obtained with biodiversity data from soil samples from the same localities. We found a largely different taxonomic composition between the soil and insect samples, testifying to the potential of bulk insect samples to complement soil samples. Finally, we found that non-destructive DNA extraction protocols, which preserve insect specimens for morphological studies, constitute a promising choice for cost-effective biodiversity assessments. We propose that the sampling and sequencing of insect samples should become a standard complement for biodiversity studies based on environmental DNA.</p>
<h1>Clay beads as artificial trapping matrices for monitoring bacterial distribution among urban stormwater infiltration systems and their connected aquifers.</h1>
<p><a href="https://doi.org/10.1007/S10661-019-7190-0">doi:10.1007/S10661-019-7190-0</a></p>
['16S rRNA', 'Environmental monitoring', 'Infiltration basins', 'Metabarcoding', 'Microbial biomass and activity', 'Next generation sequencing (NGS)']<p>Stormwater infiltration systems (SIS) have been developed to limit surface runoff and flooding in urban areas. The impacts of such practices on the ecological and biological quality of groundwater ecosystems remain poorly studied due to the lack of efficient methodologies to assess microbiological quality of aquifers. In the present study, a monitoring method based on the incubation of artificial matrices (clay beads) is presented to evaluate microbial biomass, microbial activities, and bacterial community structure. Four microbial variables (biomass, dehydrogenase and hydrolytic activities, bacterial community structures) were measured on clay beads incubated in three urban water types (stormwater surface runoffs, SIS-impacted and non-impacted groundwaters) for six SIS. Analyses based on next-generation sequencing (NGS) of partial rrs (16S rRNA) PCR products (V5-V6) were used to compare bacterial community structures of biofilms on clay beads after 10 days of incubation with those of waters collected from the same sampling points at three occasions. Biofilm biomass and activities on clay beads were indicative of nutrient transfers from surface to SIS-impacted groundwaters. Biofilms allowed impacts of SIS on groundwater bacterial community structures to be determined. Although bacterial communities on clay beads did not perfectly match those of waters, clay beads captured the most abundant bacterial taxa. They also captured bacterial taxa that were not detected in waters collected at three occasions during the incubation, demonstrating the integrative character of this approach. Monitoring biofilms on clay beads also allowed the tracking of bacterial genera containing species representing health concerns.</p>
<h1>Annotated 18S and 28S rDNA reference sequences of taxa in the planktonic diatom family Chaetocerotaceae.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0208929">doi:10.1371/Journal.Pone.0208929</a></p>
<p>No keywords</p><p>The species-rich diatom family Chaetocerotaceae is common in the coastal marine phytoplankton worldwide where it is responsible for a substantial part of the primary production. Despite its relevance for the global cycling of carbon and silica, many species are still described only morphologically, and numerous specimens do not fit any described taxa. Nowadays, studies to assess plankton biodiversity deploy high throughput sequencing metabarcoding of the 18S rDNA V4 region, but to translate the gathered metabarcodes into biologically meaningful taxa, there is a need for reference barcodes. However, 18S reference barcodes for this important family are still relatively scarce. We provide 18S rDNA and partial 28S rDNA reference sequences of 443 morphologically characterized chaetocerotacean strains. We gathered 164 of the 216 18S sequences and 244 of the 413 28S sequences of strains from the Gulf of Naples, Atlantic France, and Chile. Inferred phylogenies showed 84 terminal taxa in seven principal clades. Two of these clades included terminal taxa whose rDNA sequences contained spliceosomal and Group IC1 introns. Regarding the commonly used metabarcode markers in planktonic diversity studies, all terminal taxa can be discriminated with the 18S V4 hypervariable region; its primers fit their targets in all but two species, and the V4-tree topology is similar to that of the 18S. Hence V4-metabarcodes of unknown Chaetocerotaceae are assignable to the family. Regarding the V9 hypervariable region, most terminal taxa can be discriminated, but several contain introns in their primer targets. Moreover, poor phylogenetic resolution of the V9 region affects placement of metabarcodes of putative but unknown chaetocerotacean taxa, and hence, uncertainty in taxonomic assignment, even of higher taxa.</p>
<h1>Diatom diversity through HTS-metabarcoding in coastal European seas.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-36345-9">doi:10.1038/S41598-018-36345-9</a></p>
<p>No keywords</p><p>Diatoms constitute a diverse lineage of unicellular organisms abundant and ecologically important in aquatic ecosystems. Compared to other protists, their biology and taxonomy are well-studied, offering the opportunity to combine traditional approaches and new technologies. We examined a dataset of diatom 18S rRNA- and rDNA- (V4 region) reads from different plankton size-fractions and sediments from six European coastal marine sites, with the aim of identifying peculiarities and commonalities with respect to the whole protistan community. Almost all metabarcodes (99.6%) were assigned to known genera (121) and species (236), the most abundant of which were those already known from classic studies and coincided with those seen in light microscopy. rDNA and rRNA showed comparable patterns for the dominant taxa, but rRNA revealed a much higher diversity particularly in the sediment communities. Peculiar to diatoms is a tight bentho-pelagic coupling, with many benthic or planktonic species colonizing both water column and sediments and the dominance of planktonic species in both habitats. Overall metabarcoding results reflected the marked specificity of diatoms compared to other protistan groups in terms of morphological and ecological characteristics, at the same time confirming their great potential in the description of protist communities.</p>
<h1>Belowground fungal community diversity and composition associated with Norway spruce along an altitudinal gradient.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0208493">doi:10.1371/Journal.Pone.0208493</a></p>
<p>No keywords</p><p>Altitudinal gradients provide valuable information about the effects of environmental variables on changes in species richness and composition as well as the distribution of below ground fungal communities. Since most knowledge in this respect has been gathered on aboveground communities, we focused our study towards the characterization of belowground fungal communities associated with two different ages of Norway spruce (Picea abies) trees along an altitudinal gradient. By sequencing the internal transcribed spacer (ITS) region on the Illumina platform, we investigated the fungal communities in a floristically and geologically relatively well explored forest on the slope of Mt. Iseler of the Bavarian Alps. From fine roots and rhizosphere of a total of 90 of Norway spruce trees from 18 plots we detected 1285 taxa, with a range of 167 to 506 (average 377) taxa per plot. Fungal taxa are distributed over 96 different orders belonging to the phyla Ascomycota, Basidiomycota, Chrytridiomycota, Glomeromycota, and Mucoromycota. Overall the Agaricales (438 taxa) and Tremellales (81 taxa) belonging to the Basidiomycota and the Hypocreales (65 spp.) and Helotiales (61 taxa) belonging to the Ascomycota represented the taxon richest orders. The evaluation of our multivariate generalized mixed models indicate that the altitude has a significant influence on the composition of the fungal communities (p < 0.003) and that tree age determines community diversity (p < 0.05). A total of 47 ecological guilds were detected, of which the ectomycorrhizal and saprophytic guilds were the most taxon-rich. Our ITS amplicon Illumina sequencing approach allowed us to characterize a high fungal community diversity that would not be possible to capture with fruiting body surveys alone. We conclude that it is an invaluable tool for diverse monitoring tasks and inventorying biodiversity, especially in the detection of microorganisms developing very ephemeral and/or inconspicuous fruiting bodies or lacking them all together. Results suggest that the altitude mainly influences the community composition, whereas fungal diversity becomes higher in mature/older trees. Finally, we demonstrate that novel techniques from bacterial microbiome analyses are also useful for studying fungal diversity and community structure in a DNA metabarcoding approach, but that incomplete reference sequence databases so far limit effective identification.</p>
<h1>Evaluating the efficacy of sample collection approaches and DNA metabarcoding for identifying the diversity of plants utilized by nectivorous bats.</h1>
<p><a href="https://doi.org/10.1139/Gen-2018-0102">doi:10.1139/Gen-2018-0102</a></p>
['DNA metabarcoding', 'chauves-souris nectarivores', 'diet', 'diversity', 'diversité', 'diète', 'guano', 'métacodage à barres de l’ADN', 'nectivorous bats']<p>In this study, we evaluated the efficacy of sample collection approaches and DNA metabarcoding to identify plants utilized by nectivorous bats. Samples included guano collected from beneath bat roosts and pollen-swabs from bat fur, both of which were subjected to DNA metabarcoding and visual identification of pollen (microscopy) to measure plant diversity. Our objectives were to determine whether DNA metabarcoding could detect likely food plants of nectivorous bats, whether sample types would produce different estimates of plant diversity, and to compare results of DNA metabarcoding to visual identification. Visual identification found that 99% of pollen was from Agave, which is thought to be the bats' main food source. The dominant taxon found by metabarcoding was also Agavoideae, but a broader diversity of plant species was also detected, many of which are likely "by-catch" from the broader environment. Metabarcoding outcomes differed between sample types, likely because pollen-swabs measured the plant species visited by bats and guano samples measured all items consumed in the bat's diet, even those that were not pollen or nectar. Overall, metabarcoding is a powerful, high-throughput tool to understand bat ecology and species interactions, but careful analysis of results is necessary to derive accurate ecological conclusions.</p>
<h1>Protist Biodiversity and Biogeography in Lakes From Four Brazilian River-Floodplain Systems.</h1>
<p><a href="https://doi.org/10.1111/Jeu.12703">doi:10.1111/Jeu.12703</a></p>
['Ciliophora', 'Discoba', 'Euglenida', 'Neotropical', 'Ochrophyta', 'high-throughput sequencing', 'metabarcoding']<p>The biodiversity and biogeography of protists inhabiting many ecosystems have been intensely studied using different sequencing approaches, but tropical ecosystems are relatively under-studied. Here, we sampled planktonic waters from 32 lakes associated with four different river-floodplains systems in Brazil, and sequenced the DNA using a metabarcoding approach with general eukaryotic primers. The lakes were dominated by the largely free-living Discoba (mostly the Euglenida), Ciliophora, and Ochrophyta. There was low community similarity between lakes even within the same river-floodplain. The protists inhabiting these floodplain systems comprise part of the large and relatively undiscovered diversity in the tropics.</p>
<h1>Metabarcoding using multiplexed markers increases species detection in complex zooplankton communities.</h1>
<p><a href="https://doi.org/10.1111/Eva.12694">doi:10.1111/Eva.12694</a></p>
['18S', 'cytochrome c oxidase subunit I', 'metabarcoding', 'multigene', 'multiple primer pairs', 'zooplankton']<p>Metabarcoding combines DNA barcoding with high-throughput sequencing, often using one genetic marker to understand complex and taxonomically diverse samples. However, species-level identification depends heavily on the choice of marker and the selected primer pair, often with a trade-off between successful species amplification and taxonomic resolution. We present a versatile metabarcoding protocol for biomonitoring that involves the use of two barcode markers (COI and 18S) and four primer pairs in a single high-throughput sequencing run, via sample multiplexing. We validate the protocol using a series of 24 mock zooplanktonic communities incorporating various levels of genetic variation. With the use of a single marker and single primer pair, the highest species recovery was 77%. With all three COI fragments, we detected 62%-83% of species across the mock communities, while the use of the 18S fragment alone resulted in the detection of 73%-75% of species. The species detection level was significantly improved to 89%-93% when both markers were used. Furthermore, multiplexing did not have a negative impact on the proportion of reads assigned to each species and the total number of species detected was similar to when markers were sequenced alone. Overall, our metabarcoding approach utilizing two barcode markers and multiple primer pairs per barcode improved species detection rates over a single marker/primer pair by 14% to 35%, making it an attractive and relatively cost-effective method for biomonitoring natural zooplankton communities. We strongly recommend combining evolutionary independent markers and, when necessary, multiple primer pairs per marker to increase species detection (i.e., reduce false negatives) in metabarcoding studies.</p>
<h1>Metabarcoding successfully tracks temporal changes in eukaryotic communities in coastal sediments.</h1>
<p><a href="https://doi.org/10.1093/Femsec/Fiy226">doi:10.1093/Femsec/Fiy226</a></p>
<p>No keywords</p><p>Metabarcoding is a method that combines high-throughput DNA sequencing and DNA-based identification. Previously, this method has been successfully used to target spatial variation of eukaryote communities in marine sediments, however, the temporal changes in these communities remain understudied. Here, we follow the temporal changes of the eukaryote communities in Baltic Sea surface sediments collected from two coastal localities during three seasons of two consecutive years. Our study reveals that the structure of the sediment eukaryotic ecosystem was primarily driven by annual and seasonal changes in prevailing environmental conditions, whereas spatial variation was a less significant factor in explaining the variance in eukaryotic communities over time. Therefore, our data suggests that shifts in regional climate regime or large-scale changes in the environment are the overdriving factors in shaping the coastal eukaryotic sediment ecosystems rather than small-scale changes in local environmental conditions or heterogeneity in ecosystem structure. More studies targeting temporal changes are needed to further understand the long-term trends in ecosystem stability and response to climate change. Furthermore, this work contributes to the recent efforts in developing metabarcoding applications for environmental biomonitoring, proving a comprehensive option for traditional monitoring approaches.</p>
<h1>Author Correction: Comparing Sanger sequencing and high-throughput metabarcoding for inferring photobiont diversity in lichens.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-35358-8">doi:10.1038/S41598-018-35358-8</a></p>
<p>No keywords</p><p>A correction to this article has been published and is linked from the HTML and PDF versions of this paper. The error has been fixed in the paper.</p>
<h1>Approaches to integrating genetic data into ecological networks.</h1>
<p><a href="https://doi.org/10.1111/Mec.14941">doi:10.1111/Mec.14941</a></p>
['DNA barcoding', 'bats', 'food webs', 'high-throughput sequencing', 'interaction networks', 'metabarcoding']<p>As molecular tools for assessing trophic interactions become common, research is increasingly focused on the construction of interaction networks. Here, we demonstrate three key methods for incorporating DNA data into network ecology and discuss analytical considerations using a model consisting of plants, insects, bats and their parasites from the Costa Rica dry forest. The simplest method involves the use of Sanger sequencing to acquire long sequences to validate or refine field identifications, for example of bats and their parasites, where one specimen yields one sequence and one identification. This method can be fully quantified and resolved and these data resemble traditional ecological networks. For more complex taxonomic identifications, we target multiple DNA loci, for example from a seed or fruit pulp sample in faeces. These networks are also well resolved but gene targets vary in resolution and quantification is difficult. Finally, for mixed templates such as faecal contents of insectivorous bats, we use DNA metabarcoding targeting two sequence lengths (157 and 407 bp) of one gene region and a MOTU, BLAST and BIN association approach to resolve nodes. This network type is complex to generate and analyse, and we discuss the implications of this type of resolution on network analysis. Using these data, we construct the first molecular-based network of networks containing 3,304 interactions between 762 nodes of eight trophic functions and involving parasitic, mutualistic and predatory interactions. We provide a comparison of the relative strengths and weaknesses of these data types in network ecology.</p>
<h1>Protist Diversity and Seasonal Dynamics in Skagerrak Plankton Communities as Revealed by Metabarcoding and Microscopy.</h1>
<p><a href="https://doi.org/10.1111/Jeu.12700">doi:10.1111/Jeu.12700</a></p>
['Biovolume', 'Oslofjorden', 'high-throughput sequencing', 'richness']<p>Protist community composition and seasonal dynamics are of major importance for the production of higher trophic levels, such as zooplankton and fish. Our aim was to reveal how the protist community in the Skagerrak changes through the seasons by combining high-throughput sequencing and microscopy of plankton collected monthly over two years. The V4 region of the 18S rRNA gene was amplified by eukaryote universal primers from the total RNA/cDNA. We found a strong seasonal variation in protist composition and proportional abundances, and a difference between two depths within the euphotic zone. Highest protist richness was found in late summer-early autumn, and lowest in winter. Temperature was the abiotic factor explaining most of the variation in diversity. Dinoflagellates was the most abundant and diverse group followed by ciliates and diatoms. We found about 70 new taxa recorded for the first time in the Skagerrak. The seasonal pattern in relative read abundance of major phytoplankton groups was well in accordance with microscopical biovolumes. This is the first metabarcoding study of the protist plankton community of all taxonomic groups and through seasons in the Skagerrak, which may serve as a baseline for future surveys to reveal effects of climate and environmental changes.</p>
<h1>Shotgun metagenomics of honey DNA: Evaluation of a methodological approach to describe a multi-kingdom honey bee derived environmental DNA signature.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0205575">doi:10.1371/Journal.Pone.0205575</a></p>
<p>No keywords</p><p>Honey bees are considered large-scale monitoring tools due to their environmental exploration and foraging activities. Traces of these activities can be recovered in the honey that also may reflect the hive ecological micro-conditions in which it has been produced. This study applied a next generation sequencing platform (Ion Torrent) for shotgun metagenomic analysis of honey environmental DNA (eDNA). The study tested a methodological framework to interpret DNA sequence information useful to describe the complex ecosystems of the honey bee colony superorganism, its pathosphere and the heterogeneity of the agroecological environments and environmental sources that left DNA marks in the honey. Analysis of two honeys reported sequence reads from five main organism groups (kingdoms or phyla): arthropods (that mainly included reads from Apis mellifera, several other members of the Hymenotpera, in addition to members of the Diptera, Coleoptera and Lepidoptera, as well as aphids and mites), plants (that clearly confirmed the botanical origin of the two honeys, i.e. orange tree blossom and eucalyptus tree blossom honeys), fungi and bacteria (including common hive and honey bee gut microorganisms, honey bee pathogens and plant pathogens), and viruses (which accounted for the largest number of reads in both honeys, mainly assigned to Apis mellifera filamentous virus). The shotgun metagenomic approach that was used in this study can be applied in large scale experiments that might have multiple objectives according to the multi-kingdom derived eDNA that is contained in the honey.</p>
<h1>The endangered northern bettong, Bettongia tropica, performs a unique and potentially irreplaceable dispersal function for ectomycorrhizal truffle fungi.</h1>
<p><a href="https://doi.org/10.1111/Mec.14916">doi:10.1111/Mec.14916</a></p>
['\nBettongia tropica\n', 'dispersal', 'food web', 'functional redundancy', 'metabarcoding', 'truffle fungi']<p>Organisms that are highly connected in food webs often perform unique and vital functions within ecosystems. Understanding the unique ecological roles played by highly connected organisms and the consequences of their loss requires a comprehensive understanding of the functional redundancy amongst organisms. One important, yet poorly understood, food web is that between truffle-forming ectomycorrhizal fungi and their mammalian consumers and dispersers. Mammalian fungal specialists rely on fungi as a food source, and they consume and disperse a higher diversity and abundance of fungi than do mycophagous mammals with generalist diets. Therefore, we hypothesize that mammalian fungal specialists are functionally distinct because they disperse a set of fungal taxa not fully nested within the set consumed by the combined generalist mammalian community (i.e., functional redundancy of fungal dispersal is limited). Using high-throughput sequencing, we compared the fungal composition of 93 scats from the endangered fungal specialist northern bettong (Bettongia tropica) and 120 scats from nine co-occurring generalist mammal species across three sites and three seasons. Compared with other generalist mammals, B. tropica consumed a more diverse fungal diet with more unique taxa. This aligns with our hypothesis that B. tropica performs a unique dispersal function for ectomycorrhizal truffle fungi. Additionally, modelling of mammalian extinctions predicted rapid loss of food web connections which could result in loss of gene flow for truffle taxa. Our results suggest that this system is sensitive to the extinction of highly connected specialist species like B. tropica and their loss could have consequences for ectomycorrhizal truffle fungal diversity. This suggests that the conservation of fungal specialists is imperative to maintaining ectomycorrhizal fungal diversity and healthy plant-mycorrhizal relationships.</p>
<h1>High-throughput identification and diagnostics of pathogens and pests: Overview and practical recommendations.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12959">doi:10.1111/1755-0998.12959</a></p>
['community ecology', 'crop disease', 'forest pathology', 'high-throughput sequencing', 'metabarcoding', 'metagenomics', 'molecular diagnostics', 'plant pathogens', 'sentinel plantations']<p>High-throughput identification technologies provide efficient tools for understanding the ecology and functioning of microorganisms. Yet, these methods have been only rarely used for monitoring and testing ecological hypotheses in plant pathogens and pests in spite of their immense importance in agriculture, forestry and plant community dynamics. The main objectives of this manuscript are the following: (a) to provide a comprehensive overview about the state-of-the-art high-throughput quantification and molecular identification methods used to address population dynamics, community ecology and host associations of microorganisms, with a specific focus on antagonists such as pathogens, viruses and pests; (b) to compile available information and provide recommendations about specific protocols and workable primers for bacteria, fungi, oomycetes and insect pests; and (c) to provide examples of novel methods used in other microbiological disciplines that are of great potential use for testing specific biological hypotheses related to pathology. Finally, we evaluate the overall perspectives of the state-of-the-art and still evolving methods for diagnostics and population- and community-level ecological research of pathogens and pests.</p>
<h1>Metabarcoding of shrimp stomach content: Harnessing a natural sampler for fish biodiversity monitoring.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12956">doi:10.1111/1755-0998.12956</a></p>
['\nCrangon crangon\n', 'DNA', 'biodiversity monitoring', 'environmental assessment', 'fish communities', 'trophic metabarcoding']<p>Given their positioning and biological productivity, estuaries have long represented key providers of ecosystem services and consequently remain under remarkable pressure from numerous forms of anthropogenic impact. The monitoring of fish communities in space and time is one of the most widespread and established approaches to assess the ecological status of estuaries and other coastal habitats, but traditional fish surveys are invasive, costly, labour intensive and highly selective. Recently, the application of metabarcoding techniques, on either sediment or aqueous environmental DNA, has rapidly gained popularity. Here, we evaluate the application of a novel, high-throughput DNA-based monitoring tool to assess fish diversity, based on the analysis of the gut contents of a generalist predator/scavenger, the European brown shrimp, Crangon crangon. Sediment and shrimp samples were collected from eight European estuaries, and DNA metabarcoding (using both 12S and COI markers) was carried out to infer fish assemblage composition. We detected 32 teleost species (16 and 20, for 12S and COI, respectively). Twice as many species were recovered using metabarcoding than by traditional net surveys. By comparing and interweaving trophic, environmental DNA and traditional survey-based techniques, we show that the DNA-assisted gut content analysis of a ubiquitous, easily accessible, generalist species may serve as a powerful, rapid and cost-effective tool for large-scale, routine estuarine biodiversity monitoring.</p>
<h1>Shifts in community composition and co-occurrence patterns of phyllosphere fungi inhabiting <i>Mussaenda shikokiana</i> along an elevation gradient.</h1>
<p><a href="https://doi.org/10.7717/Peerj.5767">doi:10.7717/Peerj.5767</a></p>
['Biodiversity', 'Elevation gradient', 'High-throughput sequencing', 'Mussaenda shikokiana', 'Phyllosphere fungi']<p>The altitudinal effects on the distributions of phyllosphere fungal assemblages in conspecific plants remain poorly elucidated. To address this, phyllosphere fungal communities associated with <i>Mussaenda shikokiana</i> were investigated at four sites across a 350 m elevation gradient in a subtropical forest by employing Illumina metabarcoding of the fungal internal transcribed spacer 2 (ITS2) region. Our results demonstrated that phyllosphere fungal assemblages with a single host possessed high taxonomic diversity and multiple trophic guilds. OTU richness was significantly influenced by elevation. The elevation gradient also entailed distinct shifts in the community composition of phyllosphere fungi, which was significantly related to geographical distance and mean annual temperature (MAT). Additionally, comparison of phyllosphere fungal networks showed reduced connectivity with increasing elevation. Our data provide insights on the distribution and interactions of the phyllosphere fungal community associated with a single host along a short elevation gradient.</p>
<h1>Development and testing of an 18S rRNA phylogenetic microarray for marine sediments.</h1>
<p><a href="https://doi.org/S0167-7012(18)30829-7">doi:S0167-7012(18)30829-7</a></p>
['Benthic', 'Biodiversity', 'Ecosystem', 'Environment', 'Genomic DNA', 'Microarray', 'Monitoring', 'Probe']<p>There is increasing interest in finding new, more efficient methods for routine monitoring of anthropogenic effects on benthic biodiversity and ecosystems. A range of molecular methods have been developed for assessing biodiversity the last decades. Particularly interesting are microarrays targeting phylogenetic marker genes, such as the small subunit of ribosomal RNA in eukaryotes (18S rRNA). This method can detect a large number of taxonomic groups in several samples simultaneously within a relatively short time and has the potential for incorporation in automated remote sensing pipelines. In this study we developed and tested a microarray for eukaryotes in marine sediments. The probes were designed to target 18S rRNA OTUs obtained through metabarcoding of marine sediments. The resulting microarray was tested using both a spiked sample consisting of 50 plasmid-clones and further, samples of genomic DNA extracted from marine sediments. We developed a filtration pipeline to eliminate noise and reduce the number of false positives, making it possible to detect and quantify most of the OTUs with ≥ 0.1% abundance in the spiked sample. Our data indicated that the microarray was specific at higher taxonomic levels. However, the specificity decreased with increasing sequence similarity suggesting cross-hybridization between closely related OTUs. When using genomic DNA isolated from marine sediment there was a positive correlation between hybridization intensity signals and abundance of sequencing reads, suggesting a quantitative behavior of the microarray. Overall, the data suggest a potential for microarrays as a tool for high throughput sediment monitoring.</p>
<h1>Metabarcoding of marine environmental DNA based on mitochondrial and nuclear genes.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-32917-X">doi:10.1038/S41598-018-32917-X</a></p>
<p>No keywords</p><p>We establish the new approach of environmental DNA (eDNA) analyses for the North Sea. Our study uses a multigene approach, including the mitochondrial cytochrome-c-oxidase subunit I (COI) gene for analyzing species composition and the nuclear hypervariable region V8 of 18S rDNA for analyzing supraspecific biodiversity. A new minibarcode primer (124 bp) was created on the basis of a metazoan COI barcode library with 506 species and tested in silico, in vitro, and in situ. We applied high throughput sequencing to filtrates of 23 near-bottom water samples taken at three seasons from 14 stations. The set of COI primers allowed amplification of mitochondrial minibarcodes for diverse metazoan phyla and the differentiation at the species level for more than 99% of the specimens in the dataset. Our results revealed that the number of sequences is not consistent with proportions in the given DNA mixture. Altogether, environmental sequences could be assigned to 114 species and to 12 metazoan phyla. A spatial distribution of taxa recovered by eDNA was congruent with known distributions. Finally, the successful detection of species and biodiversity depends on a comprehensive sequence reference database. Our study offers a powerful tool for future biodiversity research, including the detection of nonnative species.</p>
<h1>High-throughput metabarcoding reveals the effect of physicochemical soil properties on soil and litter biodiversity and community turnover across Amazonia.</h1>
<p><a href="https://doi.org/10.7717/Peerj.5661">doi:10.7717/Peerj.5661</a></p>
['Brazil', 'Eukaryotes', 'Operational Taxonomic Units (OTUs)', 'Prokaryotes', 'Rainforest', 'Soil microorganisms']<p>Knowledge on the globally outstanding Amazonian biodiversity and its environmental determinants stems almost exclusively from aboveground organisms, notably plants. In contrast, the environmental factors and habitat preferences that drive diversity patterns for micro-organisms in the ground remain elusive, despite the fact that micro-organisms constitute the overwhelming majority of life forms in any given location, in terms of both diversity and abundance. Here we address how the diversity and community turnover of operational taxonomic units (OTU) of organisms in soil and litter respond to soil physicochemical properties; whether OTU diversities and community composition in soil and litter are correlated with each other; and whether they respond in a similar way to soil properties.</p>
<h1>DNA metabarcoding unveils multiscale trophic variation in a widespread coastal opportunist.</h1>
<p><a href="https://doi.org/10.1111/Mec.14886">doi:10.1111/Mec.14886</a></p>
['community ecology', 'crustaceans', 'diet analysis', 'environmental DNA', 'food webs', 'metabarcoding']<p>A thorough understanding of ecological networks relies on comprehensive information on trophic relationships among species. Since unpicking the diet of many organisms is unattainable using traditional morphology-based approaches, the application of high-throughput sequencing methods represents a rapid and powerful way forward. Here, we assessed the application of DNA metabarcoding with nearly universal primers for the mitochondrial marker cytochrome c oxidase I in defining the trophic ecology of adult brown shrimp, Crangon crangon, in six European estuaries. The exact trophic role of this abundant and widespread coastal benthic species is somewhat controversial, while information on geographical variation remains scant. Results revealed a highly opportunistic behaviour. Shrimp stomach contents contained hundreds of taxa (>1,000 molecular operational taxonomic units), of which 291 were identified as distinct species, belonging to 35 phyla. Only twenty ascertained species had a mean relative abundance of more than 0.5%. Predominant species included other abundant coastal and estuarine taxa, including the shore crab Carcinus maenas and the amphipod Corophium volutator. Jacobs' selectivity index estimates based on DNA extracted from both shrimp stomachs and sediment samples were used to assess the shrimp's trophic niche indicating a generalist diet, dominated by crustaceans, polychaetes and fish. Spatial variation in diet composition, at regional and local scales, confirmed the highly flexible nature of this trophic opportunist. Furthermore, the detection of a prevalent, possibly endoparasitic fungus (Purpureocillium lilacinum) in the shrimp's stomach demonstrates the wide range of questions that can be addressed using metabarcoding, towards a more robust reconstruction of ecological networks.</p>
<h1>Biodiversity estimates and ecological interpretations of meiofaunal communities are biased by the taxonomic approach.</h1>
<p><a href="https://doi.org/10.1038/S42003-018-0119-2">doi:10.1038/S42003-018-0119-2</a></p>
<p>No keywords</p><p>Accurate assessments of biodiversity are crucial to advising ecosystem-monitoring programs and understanding ecosystem function. Nevertheless, a standard operating procedure to assess biodiversity accurately and consistently has not been established. This is especially true for meiofauna, a diverse community (>20 phyla) of small benthic invertebrates that have fundamental ecological roles. Recent studies show that metabarcoding is a cost-effective and time-effective method to estimate meiofauna biodiversity, in contrast to morphological-based taxonomy. Here, we compare biodiversity assessments of a diverse meiofaunal community derived by applying multiple taxonomic methods based on comparative morphology, molecular phylogenetic analysis, DNA barcoding of individual specimens, and metabarcoding of environmental DNA. We show that biodiversity estimates are strongly biased across taxonomic methods and phyla. Such biases affect understanding of community structures and ecological interpretations. This study supports the urgency of improving aspects of environmental high-throughput sequencing and the value of taxonomists in correctly understanding biodiversity estimates.</p>
<h1>Toward an ecoregion scale evaluation of eDNA metabarcoding primers: A case study for the freshwater fish biodiversity of the Murray-Darling Basin (Australia).</h1>
<p><a href="https://doi.org/10.1002/Ece3.4387">doi:10.1002/Ece3.4387</a></p>
['environmental DNA', 'high‐throughput sequencing', 'in silico', 'metabarcoding', 'primers']<p>High-throughput sequencing of environmental DNA (i.e., eDNA metabarcoding) has become an increasingly popular method for monitoring aquatic biodiversity. At present, such analyses require target-specific primers to amplify DNA barcodes from co-occurring species, and this initial amplification can introduce biases. Understanding the performance of different primers is thus recommended prior to undertaking any metabarcoding initiative. While multiple software programs are available to evaluate metabarcoding primers, all programs have their own strengths and weaknesses. Therefore, a robust in silico workflow for the evaluation of metabarcoding primers will benefit from the use of multiple programs. Furthermore, geographic differences in species biodiversity are likely to influence the performance of metabarcoding primers and further complicate the evaluation process. Here, an in silico workflow is presented that can be used to evaluate the performance of metabarcoding primers on an ecoregion scale. This workflow was used to evaluate the performance of published and newly developed eDNA metabarcoding primers for the freshwater fish biodiversity of the Murray-Darling Basin (Australia). To validate the in silico workflow, a subset of the primers, including one newly designed primer pair, were used in metabarcoding analyses of an artificial DNA community and eDNA samples. The results show that the in silico workflow allows for a robust evaluation of metabarcoding primers and can reveal important trade-offs that need to be considered when selecting the most suitable primer. Additionally, a new primer pair was described and validated that allows for more robust taxonomic assignments and is less influenced by primer biases compared to commonly used fish metabarcoding primers.</p>
<h1>Great differences in performance and outcome of high-throughput sequencing data analysis platforms for fungal metabarcoding.</h1>
<p><a href="https://doi.org/10.3897/Mycokeys.39.28109">doi:10.3897/Mycokeys.39.28109</a></p>
['Microbial communities', 'amplicon sequencing', 'fungal biodiversity', 'metagenomics', 'microbiome', 'mycobiome']<p>Along with recent developments in high-throughput sequencing (HTS) technologies and thus fast accumulation of HTS data, there has been a growing need and interest for developing tools for HTS data processing and communication. In particular, a number of bioinformatics tools have been designed for analysing metabarcoding data, each with specific features, assumptions and outputs. To evaluate the potential effect of the application of different bioinformatics workflow on the results, we compared the performance of different analysis platforms on two contrasting high-throughput sequencing data sets. Our analysis revealed that the computation time, quality of error filtering and hence output of specific bioinformatics process largely depends on the platform used. Our results show that none of the bioinformatics workflows appears to perfectly filter out the accumulated errors and generate Operational Taxonomic Units, although PipeCraft, LotuS and PIPITS perform better than QIIME2 and Galaxy for the tested fungal amplicon dataset. We conclude that the output of each platform requires manual validation of the OTUs by examining the taxonomy assignment values.</p>
<h1>Using noninvasive metagenomics to characterize viral communities from wildlife.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12946">doi:10.1111/1755-0998.12946</a></p>
['\nDesmodus rotundus\n', 'microbial community', 'shotgun metagenomics', 'virome']<p>Microbial communities play an important role in organismal and ecosystem health. While high-throughput metabarcoding has revolutionized the study of bacterial communities, generating comparable viral communities has proven elusive, particularly in wildlife samples where the diversity of viruses and limited quantities of viral nucleic acid present distinctive challenges. Metagenomic sequencing is a promising solution for studying viral communities, but the lack of standardized methods currently precludes comparisons across host taxa or localities. Here, we developed an untargeted shotgun metagenomic sequencing protocol to generate comparable viral communities from noninvasively collected faecal and oropharyngeal swabs. Using samples from common vampire bats (Desmodus rotundus), a key species for virus transmission to humans and domestic animals, we tested how different storage media, nucleic acid extraction procedures and enrichment steps affect viral community detection. Based on finding viral contamination in foetal bovine serum, we recommend storing swabs in RNAlater or another nonbiological medium. We recommend extracting nucleic acid directly from swabs rather than from supernatant or pelleted material, which had undetectable levels of viral RNA. Results from a low-input RNA library preparation protocol suggest that ribosomal RNA depletion and light DNase treatment reduce host and bacterial nucleic acid, and improve virus detection. Finally, applying our approach to twelve pooled samples from seven localities in Peru, we showed that detected viral communities saturated at the attained sequencing depth, allowing unbiased comparisons of viral community composition. Future studies using the methods outlined here will elucidate the determinants of viral communities across host species, environments and time.</p>
<h1>Using metabarcoding to compare the suitability of two blood-feeding leech species for sampling mammalian diversity in North Borneo.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12943">doi:10.1111/1755-0998.12943</a></p>
['Borneo', 'Haemadipsidae', 'biodiversity', 'invertebrate-derived DNA', 'mammals', 'metabarcoding']<p>The application of high-throughput sequencing (HTS) for metabarcoding of mixed samples offers new opportunities in conservation biology. Recently, the successful detection of prey DNA from the guts of leeches has raised the possibility that these, and other blood-feeding invertebrates, might serve as useful samplers of mammals. Yet little is known about whether sympatric leech species differ in their feeding preferences, and whether this has a bearing on their relative suitability for monitoring local mammalian diversity. To address these questions, we collected spatially matched samples of two congeneric leech species Haemadipsa picta and Haemadipsa sumatrana from lowland rainforest in Borneo. For each species, we pooled ~500 leeches into batches of 10 individuals, performed PCR to target a section of the mammalian 16S rRNA locus and undertook sequencing of amplicon libraries using an Illumina MiSeq. In total, we identified sequences from 14 mammalian genera, spanning nine families and five orders. We found greater numbers of detections, and higher diversity of OTUs, in H. picta compared with H. sumatrana, with rodents only present in the former leech species. However, comparison of samples from across the landscape revealed no significant difference in mammal community composition between the leech species. We therefore suggest that H. picta is the more suitable iDNA sampler in this degraded Bornean forest. We conclude that the choice of invertebrate sampler can influence the detectability of different mammal groups and that this should be accounted for when designing iDNA studies.</p>
<h1>Metabarcoding by capture using a single COI probe (MCSP) to identify and quantify fish species in ichthyoplankton swarms.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0202976">doi:10.1371/Journal.Pone.0202976</a></p>
<p>No keywords</p><p>The ability to determine the composition and relative frequencies of fish species in large ichthyoplankton swarms could have extremely important ecological applications However, this task is currently hampered by methodological limitations. We proposed a new method for Amazonian species based on hybridization capture of the COI gene DNA from a distant species (Danio rerio), absent from our study area (the Amazon basin). The COI sequence of this species is approximately equidistant from all COI of Amazonian species available. By using this sequence as probe we successfully facilitated the simultaneous identification of fish larvae belonging to the order Siluriformes and to the Characiformes represented in our ichthyoplankton samples. Species relative frequencies, estimated by the number of reads, showed almost perfect correlations with true frequencies estimated by a Sanger approach, allowing the development of a quantitative approach. We also proposed a further improvement to a previous protocol, which enables lowering the sequencing effort by 40 times. This new Metabarcoding by Capture using a Single Probe (MCSP) methodology could have important implications for ecology, fisheries management and conservation in fish biodiversity hotspots worldwide. Our approach could easily be extended to other plant and animal taxa.</p>
<h1>Over 2.5 million COI sequences in GenBank and growing.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0200177">doi:10.1371/Journal.Pone.0200177</a></p>
<p>No keywords</p><p>The increasing popularity of cytochrome c oxidase subunit 1 (COI) DNA metabarcoding warrants a careful look at the underlying reference databases used to make high-throughput taxonomic assignments. The objectives of this study are to document trends and assess the future usability of COI records for metabarcode identification. The number of COI records deposited to the NCBI nucleotide database has increased by a geometric average of 51% per year, from 8,137 records deposited in 2003 to a cumulative total of ~ 2.5 million by the end of 2017. About half of these records are fully identified to the species rank, 92% are at least 500 bp in length, 74% have a country annotation, and 51% have latitude-longitude annotations. To ensure the future usability of COI records in GenBank we suggest: 1) Improving the geographic representation of COI records, 2) Improving the cross-referencing of COI records in the Barcode of Life Data System and GenBank to facilitate consolidation and incorporation into existing bioinformatic pipelines, 3) Adherence to the minimum information about a marker gene sequence guidelines, and 4) Integrating metabarcodes from eDNA and mixed community studies with existing reference sequences. The growth of COI reference records over the past 15 years has been substantial and is likely to be a resource across many fields for years to come.</p>
<h1>Protocols for Investigating the Leaf Mycobiome Using High-Throughput DNA Sequencing.</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-8724-5_4">doi:10.1007/978-1-4939-8724-5_4</a></p>
['Amplicons', 'Fungal leaf endophytes', 'Metabarcoding', 'Mock community', 'Mycobiome', 'Next-generation sequencing', 'Phyllosphere', 'Phytobiome', 'Primer bias', 'Sample bias']<p>High-throughput sequencing of taxon-specific loci, or DNA metabarcoding, has become an invaluable tool for investigating the composition of plant-associated fungal communities and for elucidating plant-fungal interactions. While sequencing fungal communities has become routine, there remain numerous potential sources of systematic error that can introduce biases and compromise metabarcoding data. This chapter presents a protocol for DNA metabarcoding of the leaf mycobiome based on current best practices to minimize errors through careful laboratory practices and validation.</p>
<h1>Barcoding blood meals: New vertebrate-specific primer sets for assigning taxonomic identities to host DNA from mosquito blood meals.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pntd.0006767">doi:10.1371/Journal.Pntd.0006767</a></p>
<p>No keywords</p><p>The transmission dynamics of mosquito-vectored pathogens are, in part, mediated by mosquito host-feeding patterns. These patterns are elucidated using blood meal analysis, a collection of serological and molecular techniques that determine the taxonomic identities of the host animals from which blood meals are derived. Modern blood meal analyses rely on polymerase chain reaction (PCR), DNA sequencing, and bioinformatic comparisons of blood meal DNA sequences to reference databases. Ideally, primers used in blood meal analysis PCRs amplify templates from a taxonomically diverse range of vertebrates, produce a short amplicon, and avoid co-amplification of non-target templates. Few primer sets that fit these requirements are available for the cytochrome c oxidase subunit I (COI) gene, the species identification marker with the highest taxonomic coverage in reference databases. Here, we present new primer sets designed to amplify fragments of the DNA barcoding region of the vertebrate COI gene, while avoiding co-amplification of mosquito templates, without multiplexed or nested PCR. Primers were validated using host vertebrate DNA templates from mosquito blood meals of known origin, representing all terrestrial vertebrate classes, and field-collected mosquito blood meals of unknown origin. We found that the primers were generally effective in amplifying vertebrate host, but not mosquito DNA templates. Applied to the sample of unknown mosquito blood meals, > 98% (60/61) of blood meals samples were reliably identified, demonstrating the feasibility of identifying mosquito hosts with the new primers. These primers are beneficial in that they can be used to amplify COI templates from a diverse range of vertebrate hosts using standard PCR, thereby streamlining the process of identifying the hosts of mosquitoes, and could be applied to next generation DNA sequencing and metabarcoding approaches.</p>
<h1>Finding the pond through the weeds: eDNA reveals underestimated diversity of pondweeds.</h1>
<p><a href="https://doi.org/10.1002/Aps3.1155">doi:10.1002/Aps3.1155</a></p>
['ITS2', 'Ontario', 'Potamogetonaceae', 'aquatic plants', 'atpB‐rbcL', 'eDNA metabarcoding']<p>The detection of environmental DNA (eDNA) using high-throughput sequencing has rapidly emerged as a method to detect organisms from environmental samples. However, eDNA studies of aquatic biomes have focused on surveillance of animal species with less emphasis on plants. Pondweeds are important bioindicators of freshwater ecosystems, although their diversity is underestimated due to difficulties in morphological identification and monitoring.</p>
<h1>Why the COI barcode should be the community DNA metabarcode for the metazoa.</h1>
<p><a href="https://doi.org/10.1111/Mec.14844">doi:10.1111/Mec.14844</a></p>
['barcoding', 'eDNA', 'high-throughput sequencing', 'metabarcoding', 'next-generation sequencing']<p>Metabarcoding of complex metazoan communities is increasingly being used to measure biodiversity in terrestrial, freshwater and marine ecosystems, revolutionizing our ability to observe patterns and infer processes regarding the origin and conservation of biodiversity. A fundamentally important question is which genetic marker to amplify, and although the mitochondrial cytochrome oxidase subunit I (COI) gene is one of the more widely used markers in metabarcoding for the Metazoa, doubts have recently been raised about its suitability. We argue that (a) the extensive coverage of reference sequence databases for COI; (b) the variation it presents; (c) the comparative advantages for denoising protein-coding genes; and (d) recent advances in DNA sequencing protocols argue in favour of standardizing for the use of COI for metazoan community samples. We also highlight where research efforts should focus to maximize the utility of metabarcoding.</p>
<h1>Multi-marker metabarcoding approach to study mesozooplankton at basin scale.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-30157-7">doi:10.1038/S41598-018-30157-7</a></p>
<p>No keywords</p><p>Zooplankton plays a pivotal role in marine ecosystems and the characterisation of its biodiversity still represents a challenge for marine ecologists. In this study, mesozooplankton composition from 46 samples collected in summer along the western Adriatic Sea, was retrieved by DNA metabarcoding analysis. For the first time, the highly variable fragments of the mtDNA COI and the V9 region of 18S rRNA genes were used in a combined matrix to compile an inventory of mesozooplankton at basin scale. The number of sequences retrieved after quality filtering were 824,148 and 223,273 for COI and 18S (V9), respectively. The taxonomical assignment against reference sequences, using 95% (for COI) and 97% (for 18S) similarity thresholds, recovered 234 taxa. NMDS plots and cluster analysis divided coastal from offshore samples and the most representative species of these clusters were distributed according to the dominant surface current pattern of the Adriatic for the summer period. For selected sampling sites, mesozooplankton species were also identified under a stereo microscope providing insights on the strength and weakness of the two approaches. In addition, DNA metabarcoding was shown to be helpful for the monitoring of non-indigenous marine metazoans and spawning areas of commercial fish species. We defined pros and cons of applying this approach at basin scale and the benefits of combining the datasets from two genetic markers.</p>
<h1>Evaluating freshwater macroinvertebrates from eDNA metabarcoding: A river Nalón case study.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0201741">doi:10.1371/Journal.Pone.0201741</a></p>
<p>No keywords</p><p>Rivers are a vital resource for human wellbeing. To reduce human impact on water bodies, the European Union has established an essential regulatory framework for protection and sustainable management (WFD; 2000/60/EC). In this strategy, reliable and economic bioindicators are a fundamental component. Benthic macroinvertebrates are the group most commonly used as bioindicators through all European countries. However, their conventional assessment currently entails serious cost-efficiency limitations. In this study, we have tested the reliability of metabarcoding as a tool to record river macroinvertebrates using samples from a mock community (in vitro validation) and eDNA extracted for field validation from water from six sites within a north Iberian river (River Nalón, Asturias, Spain). Two markers (V4 region within the nuclear 18S rDNA and a fragment of the mitochondrial COI gene) were amplified and sequenced using an Illumina platform. The molecular technique has proven to be more sensitive than the visual one. A cost-benefit analysis shows that the metabarcoding approach is more expensive than conventional techniques for determining macroinvertebrate communities but requires fewer sampling and identification efforts. Our results suggest metabarcoding is a useful tool for alternative assessment of freshwater quality.</p>
<h1>Ballast Water Exchange and Invasion Risk Posed by Intracoastal Vessel Traffic: An Evaluation Using High Throughput Sequencing.</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.8B02108">doi:10.1021/Acs.Est.8B02108</a></p>
<p>No keywords</p><p>Ballast water remains a potent vector of non-native aquatic species introductions, despite increased global efforts to reduce risk of ballast water mediated invasions. This is particularly true of intracoastal vessel traffic, whose characteristics may limit the feasibility and efficacy of management through ballast water exchange (BWE). Here we utilize high throughput sequencing (HTS) to assess biological communities associated with ballast water being delivered to Valdez, Alaska from multiple source ports along the Pacific Coast of the United States. Our analyses indicate that BWE has a significant but modest effect on ballast water assemblages. Although overall richness was not reduced with exchange, we detected losses of some common benthic coastal taxa (e.g., decapods, mollusks, bryozoans, cnidaria) and gains of open ocean taxa (e.g., certain copepods, diatoms, and dinoflagellates), including some potentially toxic species. HTS-based metabarcoding identified significantly differentiated biodiversity signatures from individual source ports; this signal persisted, though weakened, in vessels undergoing BWE, indicating incomplete faunal turnover associated with management. Our analysis also enabled identification of taxa that may be of particular concern if established in Alaskan waters. While these results reveal a clear effect of BWE on diversity in intracoastal transit, they also indicate continued introduction risk of non-native and harmful taxa.</p>
<h1>Updating Plakobranchus cf. ianthobapsus (Gastropoda, Sacoglossa) host use: Diverse algal-animal interactions revealed by NGS with implications for invasive species management.</h1>
<p><a href="https://doi.org/S1055-7903(18)30054-X">doi:S1055-7903(18)30054-X</a></p>
['Avrainvillea', 'Hawai‘i', 'Metabarcoding', 'rbcL']<p>Sacoglossa, the "sap sucking" sea slugs, are highly specialized herbivores and the only metazoans that exhibit kleptoplasty, the sequestration and retention of chloroplasts from algae. Plakobranchus is one of the most generalistic herbivores within this order, with as many as 12 reported "algal host" (i.e. kleptoplast source) species. However, kleptoplast diversity studies conducted on Plakobranchus to date most likely underestimated the full diversity of kleptoplast sources within the studied populations due to limitations of the molecular techniques employed. Here, we apply a high throughput sequencing technique to assess kleptoplast diversity of Plakobranchus cf. ianthobapsus' from 10 sites across the Main Hawaiian Islands during winter and summer seasons. In so doing, we effectively used P. cf. ianthobapsus as a novel sampling tool to explore diminutive algal communities, including the current distribution of the invasive alga "Avrainvillea amadelpha." Our results show that P. cf. ianthobapsus sequesters chloroplasts from 23 algal species from across the siphonous green algal order Bryopsidales. We identified "Avrainvillea amadelpha" and Codium edule as new host species for P. cf. ianthobapusus, but their rarity among the data suggests they were most likely less preferential as hosts and were possibly utilized due to low abundance or unavailability of more preferable species, and therefore a response to starvation risk. Additionally, the identification of the highly invasive siphonous green alga "A. amadelpha" as a kleptoplast source provides new fine-scale range and distribution data for this problematic species. Overall kleptoplast diversity does not differ among sites, except in a coral-dominated, (i.e. not algal dominated) environment, suggesting that siphonous algal assemblages are common in algal-dominated ecosystems in the Hawaiian Islands. Diversity dissimilarity among seasons was recovered from the majority of sites sampled, supporting the need for seasonal data collection in algal diversity assessments. This case study using metabarcoding of sacoglossan kleptoplasts provides deeper insights into these plant-animal interactions with a better understanding of host use than previous studies using traditional molecular methods and illustrates how algal diversity studies on the scale of plastids can have implications for understanding algal community structure and invasive species dynamics.</p>
<h1>Slippage of degenerate primers can cause variation in amplicon length.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-29364-Z">doi:10.1038/S41598-018-29364-Z</a></p>
<p>No keywords</p><p>It is well understood that homopolymer regions should be avoided for primer binding to prevent off-target amplification. However, in metabarcoding, it is often difficult to avoid primer degeneracy in order to maximize taxa detection. We here investigate primer binding specificity using different primer sets from several invertebrate metabarcoding studies. Our results indicate that primers frequently bound 1-2 bp upstream in taxa where a homopolymer region was present in the amplification direction. Primer binding 1 bp downstream was observed less frequently. This primer slippage leads to taxon-specific length variation in amplicons and subsequent length variation in recovered sequences. Some widely used primer sets were severely affected by this bias, while others were not. While this variation will only have small impacts on the designation of Operational Taxonomic Units (OTUs) by clustering algorithms that ignore terminal gaps, primer sets employed in metabarcoding projects should be evaluated for their sensitivity to slippage. Moreover, steps should be taken to reduce slippage by improving protocols for primer design. For example, the flanking region adjacent to the 3' end of the primer is not considered by current primer development software although GC clamps in this position could mitigate slippage.</p>
<h1>Multifaceted DNA metabarcoding: Validation of a noninvasive, next-generation approach to studying bat populations.</h1>
<p><a href="https://doi.org/10.1111/Eva.12644">doi:10.1111/Eva.12644</a></p>
['DNA metabarcoding', 'bats', 'next‐generation DNA sequencing', 'noninvasive sampling', 'population assessment']<p>As multiple species of bats are currently experiencing dramatic declines in populations due to white-nose syndrome (WNS) and other factors, conservation managers have an urgent need for data on the ecology and overall status of populations of once-common bat species. Standard approaches to obtain data on bat populations often involve capture and handling, requiring extensive expertise and unavoidably resulting in stress to the bats. New methods to rapidly obtain critical data are needed that minimize both the stress on bats and the spread of WNS. Guano provides a noninvasive source of DNA that includes information from the bat, but also dietary items, parasites, and pathogens. DNA metabarcoding is a high-throughput, DNA-based identification technique to assess the biodiversity of environmental or fecal samples. We investigated the use of multifaceted DNA metabarcoding (MDM), a technique combining next-generation DNA sequencing (NGS), DNA barcodes, and bioinformatic analysis, to simultaneously collect data on multiple parameters of interest (bat species composition, individual genotype, sex ratios, diet, parasites, and presence of WNS) from fecal samples using a single NGS run. We tested the accuracy of each MDM assay using samples in which these parameters were previously determined using conventional approaches. We found that assays for bat species identification, insect diet, parasite diversity, and genotype were both sensitive and accurate, the assay to detect WNS was highly sensitive but requires careful sample processing steps to ensure the reliability of results, while assays for nectivorous diet and sex showed lower sensitivity. MDM was able to quantify multiple data classes from fecal samples simultaneously, and results were consistent whether we included assays for a single data class or multiple data classes. Overall, MDM is a useful approach that employs noninvasive sampling and a customizable suite of assays to gain important and largely accurate information on bat ecology and population dynamics.</p>
<h1>Enigmatic Diphyllatea eukaryotes: culturing and targeted PacBio RS amplicon sequencing reveals a higher order taxonomic diversity and global distribution.</h1>
<p><a href="https://doi.org/10.1186/S12862-018-1224-Z">doi:10.1186/S12862-018-1224-Z</a></p>
['Amplicon', 'Collodictyon', 'Diphyllatea', 'PacBio', 'Phylogeny', 'rRNA']<p>The class Diphyllatea belongs to a group of enigmatic unicellular eukaryotes that play a key role in reconstructing the morphological innovation and diversification of early eukaryotic evolution. Despite its evolutionary significance, very little is known about the phylogeny and species diversity of Diphyllatea. Only three species have described morphology, being taxonomically divided by flagella number, two or four, and cell size. Currently, one 18S rRNA Diphyllatea sequence is available, with environmental sequencing surveys reporting only a single partial sequence from a Diphyllatea-like organism. Accordingly, geographical distribution of Diphyllatea based on molecular data is limited, despite morphological data suggesting the class has a global distribution. We here present a first attempt to understand species distribution, diversity and higher order structure of Diphyllatea.</p>
<h1>Microplastic bacterial communities in the Bay of Brest: Influence of polymer type and size.</h1>
<p><a href="https://doi.org/S0269-7491(18)31478-7">doi:S0269-7491(18)31478-7</a></p>
['Bacteria', 'Coastal ecosystem', 'Metabarcoding', 'Microplastics', 'Vibrios']<p>Microplastics (<5 mm) exhibit intrinsic features such as density, hydrophobic surface, or high surface/volume ratio, that are known to promote microbial colonization and biofilm formation in marine ecosystems. Yet, a relatively low number of studies have investigated the nature of microplastic associated bacterial communities in coastal ecosystems and the potential factors influencing their composition and structure. Here, we characterized microplastics collected in the Bay of Brest by manual sorting followed by Raman spectroscopy and studied their associated bacterial assemblages using 16S amplicon high-throughput sequencing. Our methodology allowed discriminating polymer type (polyethylene, polypropylene and polystyrene) within small size ranges (0.3-1 vs. 1-2 vs. 2-5 mm) of microplastics collected. Data showed high species richness and diversity on microplastics compared to surrounding seawater samples encompassing both free living and particle attached bacteria. Even though a high proportion of operational taxonomic units (OTU; 94 ± 4%) was shared among all plastic polymers, polystyrene fragments exhibited distinct bacterial assemblages as compared to polyethylene and polypropylene samples. No effect of microplastic size was revealed regardless of polymer type, site and date of collection. The Vibrio genus was commonly detected in the microplastic fraction and specific PCR were performed to determine the presence of potentially pathogenic Vibrio strains (namely V. aestuarianus and the V. splendidus polyphyletic group). V. splendidus related species harboring putative oyster pathogens were detected on most microplastic pools (77%) emphasizing the need of further research to understand the role of microplastics on pathogen population transport and ultimate disease emergence.</p>
<h1>Supervised machine learning outperforms taxonomy-based environmental DNA metabarcoding applied to biomonitoring.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12926">doi:10.1111/1755-0998.12926</a></p>
['biomonitoring', 'biotic indices', 'environmental DNA', 'predictive models', 'supervised machine learning']<p>Biodiversity monitoring is the standard for environmental impact assessment of anthropogenic activities. Several recent studies showed that high-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) could overcome many limitations of the traditional morphotaxonomy-based bioassessment. Recently, we demonstrated that supervised machine learning (SML) can be used to predict accurate biotic indices values from eDNA metabarcoding data, regardless of the taxonomic affiliation of the sequences. However, it is unknown to which extent the accuracy of such models depends on taxonomic resolution of molecular markers or how SML compares with metabarcoding approaches targeting well-established bioindicator species. In this study, we address these issues by training predictive models upon five different ribosomal bacterial and eukaryotic markers and measuring their performance to assess the environmental impact of marine aquaculture on independent data sets. Our results show that all tested markers are yielding accurate predictive models and that they all outperform the assessment relying solely on taxonomically assigned sequences. Remarkably, we did not find any significant difference in the performance of the models built using universal eukaryotic or prokaryotic markers. Using any molecular marker with a taxonomic range broad enough to comprise different potential bioindicator taxa, SML approach can overcome the limits of taxonomy-based eDNA bioassessment.</p>
<h1>Needle in a haystack? A comparison of eDNA metabarcoding and targeted qPCR for detection of the great crested newt (<i>Triturus cristatus</i>).</h1>
<p><a href="https://doi.org/10.1002/Ece3.4013">doi:10.1002/Ece3.4013</a></p>
['Triturus cristatus', 'environmental DNA', 'great crested newt', 'high‐throughput sequencing', 'metabarcoding', 'ponds', 'real‐time quantitative PCR']<p>Environmental DNA (eDNA) analysis is a rapid, cost-effective, non-invasive biodiversity monitoring tool which utilises DNA left behind in the environment by organisms for species detection. The method is used as a species-specific survey tool for rare or invasive species across a broad range of ecosystems. Recently, eDNA and "metabarcoding" have been combined to describe whole communities rather than focusing on single target species. However, whether metabarcoding is as sensitive as targeted approaches for rare species detection remains to be evaluated. The great crested newt <i>Triturus cristatus</i> is a flagship pond species of international conservation concern and the first UK species to be routinely monitored using eDNA. We evaluate whether eDNA metabarcoding has comparable sensitivity to targeted real-time quantitative PCR (qPCR) for <i>T. cristatus</i> detection. Extracted eDNA samples (<i>N</i> = 532) were screened for <i>T. cristatus</i> by qPCR and analysed for all vertebrate species using high-throughput sequencing technology. With qPCR and a detection threshold of 1 of 12 positive qPCR replicates, newts were detected in 50% of ponds. Detection decreased to 32% when the threshold was increased to 4 of 12 positive qPCR replicates. With metabarcoding, newts were detected in 34% of ponds without a detection threshold, and in 28% of ponds when a threshold (0.028%) was applied. Therefore, qPCR provided greater detection than metabarcoding but metabarcoding detection with no threshold was equivalent to qPCR with a stringent detection threshold. The proportion of <i>T. cristatus</i> sequences in each sample was positively associated with the number of positive qPCR replicates (qPCR score) suggesting eDNA metabarcoding may be indicative of eDNA concentration. eDNA metabarcoding holds enormous potential for holistic biodiversity assessment and routine freshwater monitoring. We advocate this community approach to freshwater monitoring to guide management and conservation, whereby entire communities can be initially surveyed to best inform use of funding and time for species-specific surveys.</p>
<h1>Testing clustering strategies for metabarcoding-based investigation of community-environment interactions.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12922">doi:10.1111/1755-0998.12922</a></p>
['metabarcoding', 'operational taxonomic unit', 'river ecosystem', 'threshold', 'water pollution', 'zooplankton']<p>The degradation of freshwater ecosystems has become a common ecological and environmental problem globally. Owing to the complexity of biological communities, there remain tremendous technical challenges for investigating influence of environmental stressors (e.g., chemical pollution) on biological communities. High-throughput sequencing-based metabarcoding provides a powerful tool to reveal complex interactions between environments and biological communities. Among many technical issues, the clustering strategies for operational taxonomic units (OTUs) which are crucial for assessing biodiversity of communities, may affect final conclusions. Here, we used zooplankton communities along an environmental pollution gradient in the Chaobai River in Northern China to test different clustering strategies, including nonclustering and clustering with varied thresholds. Our results showed that though the number of OTUs estimated by nonclustering strategies and clustering strategies with divergence thresholds of 99%-97% largely varied, they were able to identify the same set of significant environmental and spatial variables responsible for geographical distributions of zooplankton communities. In addition, the ecological conclusions obtained by clustering thresholds of 99%-97% were consistent with nonclustering strategies, where for all eight clustering scenarios we detected that species sorting predicted by environmental variables overrode dispersal as the dominant factor in structuring zooplankton communities. However, clustering with the divergence thresholds of <95% affected the environmental and spatial variables identified. We conclude that both newly developed nonclustering methods and traditional clustering methods with divergence thresholds ≥97% were reliable to reveal mechanisms of complex community-environment interactions, although different clustering strategies could lead to largely varied biodiversity estimates such as those for α-diversity.</p>
<h1>The choice of universal primers and the characteristics of the species mixture determine when DNA metabarcoding can be quantitative.</h1>
<p><a href="https://doi.org/10.1111/Mec.14776">doi:10.1111/Mec.14776</a></p>
['COI', 'diet analysis', 'environmental DNA', 'in silico PCR', 'insects', 'primer bias']<p>DNA metabarcoding is a technique used to survey biodiversity in many ecological settings, but there are doubts about whether it can provide quantitative results, that is, the proportions of each species in the mixture as opposed to a species list. While there are several experimental studies that report quantitative metabarcoding results, there are a similar number that fail to do so. Here, we provide the rationale to understand under what circumstances the technique can be quantitative. In essence, we simulate a mixture of DNA of S species with a defined initial abundance distribution. In the simulated PCR, each species increases its concentration following a certain amplification efficiency. The final DNA concentration will reflect the initial one when the efficiency is similar for all species; otherwise, the initial and final DNA concentrations would be poorly related. Although there are many known factors that modulate amplification efficiency, we focused on the number of primer-template mismatches, arguably the most important one. We used 15 common primers pairs targeting the mitochondrial COI region and the mitogenomes of ca. 1,200 insect species. The results showed that some primers pairs produced quantitative results under most circumstances, whereas some other primers failed to do so. In conclusion, depending on the primer pair used in the PCR amplification and on the characteristics of the mixture analysed (i.e., high species richness, low evenness), DNA metabarcoding can provide a quantitative estimate of the relative abundances of different species.</p>
<h1>Optimization and performance testing of a sequence processing pipeline applied to detection of nonindigenous species.</h1>
<p><a href="https://doi.org/10.1111/Eva.12604">doi:10.1111/Eva.12604</a></p>
['aquatic invasive species', 'biomonitoring', 'clustering', 'high‐throughput sequencing', 'metabarcoding', 'sequence processing']<p>Genetic taxonomic assignment can be more sensitive than morphological taxonomic assignment, particularly for small, cryptic or rare species. Sequence processing is essential to taxonomic assignment, but can also produce errors because optimal parameters are not known a priori. Here, we explored how sequence processing parameters influence taxonomic assignment of 18S sequences from bulk zooplankton samples produced by 454 pyrosequencing. We optimized a sequence processing pipeline for two common research goals, estimation of species richness and early detection of aquatic invasive species (AIS), and then tested most optimal models' performances through simulations. We tested 1,050 parameter sets on 18S sequences from 20 AIS to determine optimal parameters for each research goal. We tested optimized pipelines' performances (detectability and sensitivity) by computationally inoculating sequences of 20 AIS into ten bulk zooplankton samples from ports across Canada. We found that optimal parameter selection generally depends on the research goal. However, regardless of research goal, we found that metazoan 18S sequences produced by 454 pyrosequencing should be trimmed to 375-400 bp and sequence quality filtering should be relaxed (1.5 ≤ maximum expected error ≤ 3.0, Phred score = 10). Clustering and denoising were only viable for estimating species richness, because these processing steps made some species undetectable at low sequence abundances which would not be useful for early detection of AIS. With parameter sets optimized for early detection of AIS, 90% of AIS were detected with fewer than 11 target sequences, regardless of whether clustering or denoising was used. Despite developments in next-generation sequencing, sequence processing remains an important issue owing to difficulties in balancing false-positive and false-negative errors in metabarcoding data.</p>
<h1>Can non-destructive DNA extraction of bulk invertebrate samples be used for metabarcoding?</h1>
<p><a href="https://doi.org/10.7717/Peerj.4980">doi:10.7717/Peerj.4980</a></p>
['Aquatic', 'Bioassessment', 'DNA barcoding', 'Freshwater', 'Insect', 'Macroinvertebrates', 'Metagenomics']<p>High throughput DNA sequencing of bulk invertebrate samples or metabarcoding is becoming increasingly used to provide profiles of biological communities for environmental monitoring. As metabarcoding becomes more widely applied, new reference DNA barcodes linked to individual specimens identified by taxonomists are needed. This can be achieved through using DNA extraction methods that are not only suitable for metabarcoding but also for building reference DNA barcode libraries.</p>
<h1>Tackling critical parameters in metazoan meta-barcoding experiments: a preliminary study based on <i>coxI</i> DNA barcode.</h1>
<p><a href="https://doi.org/10.7717/Peerj.4845">doi:10.7717/Peerj.4845</a></p>
['Amplification bias', 'Biodiversity', 'Carabids', 'DNA metabarcoding', 'GC content', 'High Throughput Sequencing', 'Taxonomic assignment', 'coxI']<p>Nowadays DNA meta-barcoding is a powerful instrument capable of quickly discovering the biodiversity of an environmental sample by integrating the DNA barcoding approach with High Throughput Sequencing technologies. It mainly consists of the parallel reading of informative genomic fragment/s able to discriminate living entities. Although this approach has been widely studied, it still needs optimization in some necessary steps requested in its advanced accomplishment. A fundamental element concerns the standardization of bioinformatic analyses pipelines. The aim of the present study was to underline a number of critical parameters of laboratory material preparation and taxonomic assignment pipelines in DNA meta-barcoding experiments using the cytochrome oxidase subunit-I (<i>coxI</i>) barcode region, known as a suitable molecular marker for animal species identification. We compared nine taxonomic assignment pipelines, including a custom in-house method, based on Hidden Markov Models. Moreover, we evaluated the potential influence of universal primers amplification bias in qPCR, as well as the correlation between GC content with taxonomic assignment results. The pipelines were tested on a community of known terrestrial invertebrates collected by pitfall traps from a chestnut forest in Italy. Although the present analysis was not exhaustive and needs additional investigation, our results suggest some potential improvements in laboratory material preparation and the introduction of additional parameters in taxonomic assignment pipelines. These include the correct setup of OTU clustering threshold, the calibration of GC content affecting sequencing quality and taxonomic classification, as well as the evaluation of PCR primers amplification bias on the final biodiversity pattern. Thus, careful attention and further validation/optimization of the above-mentioned variables would be required in a DNA meta-barcoding experimental routine.</p>
<h1>Metabarcoding analysis on European coastal samples reveals new molecular metazoan diversity.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-27509-8">doi:10.1038/S41598-018-27509-8</a></p>
<p>No keywords</p><p>Although animals are among the best studied organisms, we still lack a full description of their diversity, especially for microscopic taxa. This is partly due to the time-consuming and costly nature of surveying animal diversity through morphological and molecular studies of individual taxa. A powerful alternative is the use of high-throughput environmental sequencing, providing molecular data from all organisms sampled. We here address the unknown diversity of animal phyla in marine environments using an extensive dataset designed to assess eukaryotic ribosomal diversity among European coastal locations. A multi-phylum assessment of marine animal diversity that includes water column and sediments, oxic and anoxic environments, and both DNA and RNA templates, revealed a high percentage of novel 18S rRNA sequences in most phyla, suggesting that marine environments have not yet been fully sampled at a molecular level. This novelty is especially high among Platyhelminthes, Acoelomorpha, and Nematoda, which are well studied from a morphological perspective and abundant in benthic environments. We also identified, based on molecular data, a potentially novel group of widespread tunicates. Moreover, we recovered a high number of reads for Ctenophora and Cnidaria in the smaller fractions suggesting their gametes might play a greater ecological role than previously suspected.</p>
<h1>Advances and challenges in barcoding of microbes, parasites, and their vectors and reservoirs.</h1>
<p><a href="https://doi.org/10.1017/S0031182018000884">doi:10.1017/S0031182018000884</a></p>
['BOLD', 'DNA barcoding', 'MALDI–TOF barcoding', 'R packages', 'eDNA', 'ethics', 'legal issues', 'next-generation sequencing', 'paleoparasitology']<p>DNA barcoding is now a common tool in parasitology and epidemiology, which require good methods for identification not only of parasites and pathogens but vectors and reservoirs. This special issue presents some advances and challenges in barcoding of microbes, parasites, and their vectors and reservoirs. DNA barcoding found new applications in disease ecology, conservation parasitology, environmental parasitology and in paleoparasitology. New technologies such as next-generation sequencing and matrix-assisted laser desorption-ionization time-of-flight have made it now possible to investigate large samples of specimens. By allowing the investigation of parasites at the interface between environment, biodiversity, animal and human health, barcoding and biobanking have important policy outcomes as well as ethics and legal implications. The special issue 'Advances and challenges in the barcoding of parasites, vectors and reservoirs' illustrates some recent advances and proposes new avenues for research in barcoding in parasitology.</p>
<h1>Debugging diversity - a pan-continental exploration of the potential of terrestrial blood-feeding leeches as a vertebrate monitoring tool.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12912">doi:10.1111/1755-0998.12912</a></p>
['high-throughput sequencing', 'iDNA', 'metabarcoding', 'terrestrial haematophagous leeches', 'vertebrate diversity', 'vertebrate monitoring']<p>The use of environmental DNA (eDNA) has become an applicable noninvasive tool with which to obtain information about biodiversity. A subdiscipline of eDNA is iDNA (invertebrate-derived DNA), where genetic material ingested by invertebrates is used to characterize the biodiversity of the species that served as hosts. While promising, these techniques are still in their infancy, as they have only been explored on limited numbers of samples from only a single or a few different locations. In this study, we investigate the suitability of iDNA extracted from more than 3,000 haematophagous terrestrial leeches as a tool for detecting a wide range of terrestrial vertebrates across five different geographical regions on three different continents. These regions cover almost the full geographical range of haematophagous terrestrial leeches, thus representing all parts of the world where this method might apply. We identify host taxa through metabarcoding coupled with high-throughput sequencing on Illumina and IonTorrent sequencing platforms to decrease economic costs and workload and thereby make the approach attractive for practitioners in conservation management. We identified hosts in four different taxonomic vertebrate classes: mammals, birds, reptiles and amphibians, belonging to at least 42 different taxonomic families. We find that vertebrate blood ingested by haematophagous terrestrial leeches throughout their distribution is a viable source of DNA with which to examine a wide range of vertebrates. Thus, this study provides encouraging support for the potential of haematophagous terrestrial leeches as a tool for detecting and monitoring terrestrial vertebrate biodiversity.</p>
<h1>Comparing Sanger sequencing and high-throughput metabarcoding for inferring photobiont diversity in lichens.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-26947-8">doi:10.1038/S41598-018-26947-8</a></p>
<p>No keywords</p><p>The implementation of HTS (high-throughput sequencing) approaches is rapidly changing our understanding of the lichen symbiosis, by uncovering high bacterial and fungal diversity, which is often host-specific. Recently, HTS methods revealed the presence of multiple photobionts inside a single thallus in several lichen species. This differs from Sanger technology, which typically yields a single, unambiguous algal sequence per individual. Here we compared HTS and Sanger methods for estimating the diversity of green algal symbionts within lichen thalli using 240 lichen individuals belonging to two species of lichen-forming fungi. According to HTS data, Sanger technology consistently yielded the most abundant photobiont sequence in the sample. However, if the second most abundant photobiont exceeded 30% of the total HTS reads in a sample, Sanger sequencing generally failed. Our results suggest that most lichen individuals in the two analyzed species, Lasallia hispanica and L. pustulata, indeed contain a single, predominant green algal photobiont. We conclude that Sanger sequencing is a valid approach to detect the dominant photobionts in lichen individuals and populations. We discuss which research areas in lichen ecology and evolution will continue to benefit from Sanger sequencing, and which areas will profit from HTS approaches to assessing symbiont diversity.</p>
<h1>Non-biological synthetic spike-in controls and the AMPtk software pipeline improve mycobiome data.</h1>
<p><a href="https://doi.org/10.7717/Peerj.4925">doi:10.7717/Peerj.4925</a></p>
['Amplicon toolkit', 'Biodiversity', 'Environmental sequencing', 'Fungi', 'High-throughput amplicon sequencing', 'Metabarcoding', 'Mycobiome', 'Synthetic spike-in control', 'rRNA internal transcribed spacer']<p>High-throughput amplicon sequencing (HTAS) of conserved DNA regions is a powerful technique to characterize microbial communities. Recently, spike-in mock communities have been used to measure accuracy of sequencing platforms and data analysis pipelines. To assess the ability of sequencing platforms and data processing pipelines using fungal internal transcribed spacer (ITS) amplicons, we created two ITS spike-in control mock communities composed of cloned DNA in plasmids: a biological mock community, consisting of ITS sequences from fungal taxa, and a synthetic mock community (SynMock), consisting of non-biological ITS-like sequences. Using these spike-in controls we show that: (1) a non-biological synthetic control (e.g., SynMock) is the best solution for parameterizing bioinformatics pipelines, (2) pre-clustering steps for variable length amplicons are critically important, (3) a major source of bias is attributed to the initial polymerase chain reaction (PCR) and thus HTAS read abundances are typically not representative of starting values. We developed AMPtk, a versatile software solution equipped to deal with variable length amplicons and quality filter HTAS data based on spike-in controls. While we describe herein a non-biological SynMock community for ITS sequences, the concept and AMPtk software can be widely applied to any HTAS dataset to improve data quality.</p>
<h1>Counting with DNA in metabarcoding studies: How should we convert sequence reads to dietary data?</h1>
<p><a href="https://doi.org/10.1111/Mec.14734">doi:10.1111/Mec.14734</a></p>
<p>No keywords</p><p>Advances in DNA sequencing technology have revolutionized the field of molecular analysis of trophic interactions, and it is now possible to recover counts of food DNA sequences from a wide range of dietary samples. But what do these counts mean? To obtain an accurate estimate of a consumer's diet should we work strictly with data sets summarizing frequency of occurrence of different food taxa, or is it possible to use relative number of sequences? Both approaches are applied to obtain semi-quantitative diet summaries, but occurrence data are often promoted as a more conservative and reliable option due to taxa-specific biases in recovery of sequences. We explore representative dietary metabarcoding data sets and point out that diet summaries based on occurrence data often overestimate the importance of food consumed in small quantities (potentially including low-level contaminants) and are sensitive to the count threshold used to define an occurrence. Our simulations indicate that using relative read abundance (RRA) information often provides a more accurate view of population-level diet even with moderate recovery biases incorporated; however, RRA summaries are sensitive to recovery biases impacting common diet taxa. Both approaches are more accurate when the mean number of food taxa in samples is small. The ideas presented here highlight the need to consider all sources of bias and to justify the methods used to interpret count data in dietary metabarcoding studies. We encourage researchers to continue addressing methodological challenges and acknowledge unanswered questions to help spur future investigations in this rapidly developing area of research.</p>
<h1>Development of a qPCR assay to detect and quantify ichthyotoxic flagellates along the Norwegian coast, and the first Norwegian record of Fibrocapsa japonica (Raphidophyceae).</h1>
<p><a href="https://doi.org/S1568-9883(18)30051-9">doi:S1568-9883(18)30051-9</a></p>
['454 high throughput environmental sequencing', 'Heterosigma akashiwo', 'Karenia mikimotoi', 'Karlodinium veneficum', 'Molecular monitoring', 'SEM', 'rDNA']<p>Blooms of ichthyotoxic microalgae pose a great challenge to the aquaculture industry world-wide, and there is a need for fast and specific methods for their detection and quantification in monitoring programs. In this study, quantitative real-time PCR (qPCR) assays for the detection and enumeration of three ichthyotoxic flagellates: the dinoflagellate Karenia mikimotoi (Miyake & Kominami ex Oda) Hansen & Moestrup and the two raphidophytes Heterosigma akashiwo (Hada) Hada ex Hara & Chihara and Fibrocapsa japonica Toriumi & Takano were developed. Further, a previously published qPCR assay for the dinoflagellate Karlodinium veneficum (Ballantine) Larsen was used. Monthly samples collected for three years (Aug 2009-Jun 2012) in outer Oslofjorden, Norway were analysed, and the results compared with light microscopy cell counts. The results indicate a higher sensitivity and a lower detection limit (down to 1 cell L<sup>-1</sup>) for both qPCR assays. Qualitative and semi-quantitative results were further compared with those obtained by environmental 454 high throughput sequencing (HTS, metabarcoding) and scanning electron microscopy (SEM) examination from the same samplings. All four species were detected by qPCR and HTS and/or SEM in outer Oslofjorden (Aug 2009-Jun 2012); Karlodinium veneficum was present year-round, whereas Karenia mikimotoi, Heterosigma akashiwo and Fibrocapsa japonica appeared mainly during the autumn in all three years. This is the first observation of Fibrocapsa japonica in Norwegian coastal waters. This species has previously been recorded off the Swedish west coast and German Bight, which may suggest a northward dispersal.</p>
<h1>The effect of filtration method on the efficiency of environmental DNA capture and quantification via metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12899">doi:10.1111/1755-0998.12899</a></p>
['eDNA method development', 'fish monitoring', 'lentic systems', 'prefiltration']<p>Environmental DNA (eDNA) is a promising tool for rapid and noninvasive biodiversity monitoring. eDNA density is low in environmental samples, and a capture method, such as filtration, is often required to concentrate eDNA for downstream analyses. In this study, six treatments, with differing filter types and pore sizes for eDNA capture, were compared for their efficiency and accuracy to assess fish community structure with known fish abundance and biomass via eDNA metabarcoding. Our results showed that different filters (with the exception of 20-μm large-pore filters) were broadly consistent in their DNA capture ability. The 0.45-μm filters performed the best in terms of total DNA yield, probability of species detection, repeatability within pond and consistency between ponds. However performance of 0.45-μm filters was only marginally better than for 0.8-μm filters, while filtration time was significantly longer. Given this trade-off, the 0.8-μm filter is the optimal pore size of membrane filter for turbid, eutrophic and high fish density ponds analysed here. The 0.45-μm Sterivex enclosed filters performed reasonably well and are suitable in situations where on-site filtration is required. Finally, prefilters are applied only if absolutely essential for reducing the filtration time or increasing the throughput volume of the capture filters. In summary, we found encouraging similarity in the results obtained from different filtration methods, but the optimal pore size of filter or filter type might strongly depend on the water type under study.</p>
<h1>Development and implementation of eco-genomic tools for aquatic ecosystem biomonitoring: the SYNAQUA French-Swiss program.</h1>
<p><a href="https://doi.org/10.1007/S11356-018-2172-2">doi:10.1007/S11356-018-2172-2</a></p>
['Aquatic oligochaetes', 'Bioindication', 'Diatoms', 'Metabarcoding', 'Water quality assessment']<p>The effectiveness of environmental protection measures is based on the early identification and diagnosis of anthropogenic pressures. Similarly, restoration actions require precise monitoring of changes in the ecological quality of ecosystems, in order to highlight their effectiveness. Monitoring the ecological quality relies on bioindicators, which are organisms revealing the pressures exerted on the environment through the composition of their communities. Their implementation, based on the morphological identification of species, is expensive because it requires time and experts in taxonomy. Recent genomic tools should provide access to reliable and high-throughput environmental monitoring by directly inferring the composition of bioindicators' communities from their DNA (metabarcoding). The French-Swiss program SYNAQUA (INTERREG France-Switzerland 2017-2019) proposes to use and validate the tools of environmental genomic for biomonitoring and aims ultimately at their implementation in the regulatory bio-surveillance. SYNAQUA will test the metabarcoding approach focusing on two bioindicators, diatoms, and aquatic oligochaetes, which are used in freshwater biomonitoring in France and Switzerland. To go towards the renewal of current biomonitoring practices, SYNAQUA will (1) bring together different actors: scientists, environmental managers, consulting firms, and biotechnological companies, (2) apply this approach on a large scale to demonstrate its relevance, (3) propose robust and reliable tools, and (4) raise public awareness and train the various actors likely to use these new tools. Biomonitoring approaches based on such environmental genomic tools should address the European need for reliable, higher-throughput monitoring to improve the protection of aquatic environments under multiple pressures, guide their restoration, and follow their evolution.</p>
<h1>Estimating intraspecific genetic diversity from community DNA metabarcoding data.</h1>
<p><a href="https://doi.org/10.7717/Peerj.4644">doi:10.7717/Peerj.4644</a></p>
['CO1', 'Ecosystem assessment', 'Exact sequence variant', 'Haplotyping', 'High-throughput sequencing', 'Metabarcoding', 'Population genetics']<p>DNA metabarcoding is used to generate species composition data for entire communities. However, sequencing errors in high-throughput sequencing instruments are fairly common, usually requiring reads to be clustered into operational taxonomic units (OTUs), losing information on intraspecific diversity in the process. While Cytochrome c oxidase subunit I (COI) haplotype information is limited in resolving intraspecific diversity it is nevertheless often useful e.g. in a phylogeographic context, helping to formulate hypotheses on taxon distribution and dispersal.</p>
<h1>Mycobiome of Cysts of the Soybean Cyst Nematode Under Long Term Crop Rotation.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2018.00386">doi:10.3389/Fmicb.2018.00386</a></p>
['Heterodera glycines', 'Mortierellomycotina', 'arbuscular mycorrhizal fungi', 'crop rotation', 'nematophagous fungi', 'next generation sequencing', 'soybean cyst nematode']<p>The soybean cyst nematode (SCN), <i>Heterodera glycines</i> Ichinohe (Phylum Nematoda), is a major pathogen of soybean. It causes substantial yield losses worldwide and is difficult to control because the cyst protects the eggs which can remain viable for nearly a decade. Crop rotation with non-host crops and use of biocontrol organisms such as fungi and bacteria offer promising approaches, but remain hampered by lack of knowledge of the biology of nematode parasitic organisms. We used a high-throughput metabarcoding approach to characterize fungal communities associated with the SCN cyst, a microenvironment in soil that may harbor both nematode parasites and plant pathogens. SCN cysts were collected from a long-term crop rotation experiment in Southeastern Minnesota at three time points over two growing seasons to characterize diversity of fungi inhabiting cysts and to examine how crop rotation and seasonal variation affects fungal communities. A majority of fungi in cysts belonged to Ascomycota and Basidiomycota, but the presence of several early diverging fungal subphyla thought to be primarily plant and litter associated, including Mortierellomycotina and Glomeromycotina (e.g., arbuscular mycorrhizal fungi), suggests a possible role as nematode egg parasites. Species richness varied by both crop rotation and season and was higher in early years of crop rotation and in fall at the end of the growing season. Crop rotation and season also impacted fungal community composition and identified several classes of fungi, including Eurotiomycetes, Sordariomycetes, and Orbiliomycetes (e.g., nematode trapping fungi), with higher relative abundance in early soybean rotations. The relative abundance of several genera was correlated with increasing years of soybean. Fungal communities also varied by season and were most divergent at midseason. The percentage of OTUs assigned to Mortierellomycotina_cls_Incertae_sedis and Sordariomycetes increased at midseason, while Orbiliomycetes decreased at midseason, and Glomeromycetes increased in fall. Ecological guilds of fungi containing an animal-pathogen lifestyle, as well as potential egg-parasitic taxa previously isolated from parasitized SCN eggs, increased at midseason. The animal pathogen guilds included known (e.g., <i>Pochonia chlamydosporia</i>) and new candidate biocontrol organisms. This research advances knowledge of the ecology of nematophagous fungi in agroecosystems and their use as biocontrol agents of the SCN.</p>
<h1>High-throughput terrestrial biodiversity assessments: mitochondrial metabarcoding, metagenomics or metatranscriptomics?</h1>
<p><a href="https://doi.org/10.1080/24701394.2018.1455189">doi:10.1080/24701394.2018.1455189</a></p>
['Arthropods', 'DNA barcodes', 'genome skimming', 'mitogenomics', 'transcriptomics']<p>Consensus on the optimal high-throughput sequencing (HTS) approach to examine biodiversity in mixed terrestrial arthropod samples has not been reached. Metatranscriptomics could increase the proportion of taxonomically informative mitochondrial reads in HTS outputs but has not been investigated for terrestrial arthropod samples. We compared the efficiency of 16S rRNA metabarcoding, metagenomics and metatranscriptomics for detecting species in a mixed terrestrial arthropod sample (pooled DNA/RNA from 38 taxa). 16S rRNA metabarcoding and nuclear rRNA-depleted metatranscriptomics had the highest detection rate with 97% of input species detected. Based on cytochrome c oxidase I, metagenomics had the highest detection rate with 82% of input species detected, but metatranscriptomics produced a larger proportion of reads matching (Sanger) reference sequences. Metatranscriptomics with nuclear rRNA depletion may offer advantages over metabarcoding through reducing the number of spurious operational taxonomic units while retaining high detection rates, and offers natural enrichment of mitochondrial sequences which may enable increased species detection rates compared with metagenomics.</p>
<h1>DNA metabarcoding uncovers fungal diversity of mixed airborne samples in Italy.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0194489">doi:10.1371/Journal.Pone.0194489</a></p>
<p>No keywords</p><p>Fungal spores and mycelium fragments are particles which become and remain airborne and have been subjects of aerobiological studies. The presence and the abundance of taxa in aerobiological samples can be very variable and impaired by changeable climatic conditions. Because many fungi produce mycotoxins and both their mycelium fragments and spores are potential allergens, monitoring the presence of these taxa is of key importance. So far data on exposure and sensitization to fungal allergens are mainly based on the assessment of few, easily identifiable taxa and focused only on certain environments. The microscopic method used to analyze aerobiological samples and the inconspicuous fungal characters do not allow a in depth taxonomical identification. Here, we present a first assessment of fungal diversity from airborne samples using a DNA metabarcoding analysis. The nuclear ITS2 region was selected as barcode to catch fungal diversity in mixed airborne samples gathered during two weeks in four sites of North-Eastern and Central Italy. We assessed the taxonomic composition and diversity within and among the sampled sites and compared the molecular data with those obtained by traditional microscopy. The molecular analyses provide a tenfold more comprehensive determination of the taxa than the traditional morphological inspections. Our results prove that the metabarcoding analysis is a promising approach to increases quality and sensitivity of the aerobiological monitoring. The laboratory and bioinformatic workflow implemented here is now suitable for routine, high-throughput, regional analyses of airborne fungi.</p>
<h1>Automated high throughput animal CO1 metabarcode classification.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-22505-4">doi:10.1038/S41598-018-22505-4</a></p>
<p>No keywords</p><p>We introduce a method for assigning names to CO1 metabarcode sequences with confidence scores in a rapid, high-throughput manner. We compiled nearly 1 million CO1 barcode sequences appropriate for classifying arthropods and chordates. Compared to our previous Insecta classifier, the current classifier has more than three times the taxonomic coverage, including outgroups, and is based on almost five times as many reference sequences. Unlike other popular rDNA metabarcoding markers, we show that classification performance is similar across the length of the CO1 barcoding region. We show that the RDP classifier can make taxonomic assignments about 19 times faster than the popular top BLAST hit method and reduce the false positive rate from nearly 100% to 34%. This is especially important in large-scale biodiversity and biomonitoring studies where datasets can become very large and the taxonomic assignment problem is not trivial. We also show that reference databases are becoming more representative of current species diversity but that gaps still exist. We suggest that it would benefit the field as a whole if all investigators involved in metabarocoding studies, through collaborations with taxonomic experts, also planned to barcode representatives of their local biota as a part of their projects.</p>
<h1>Detailed ecological associations of triatomines revealed by metabarcoding and next-generation sequencing: implications for triatomine behavior and Trypanosoma cruzi transmission cycles.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-22455-X">doi:10.1038/S41598-018-22455-X</a></p>
<p>No keywords</p><p>Trypanosoma cruzi is the agent of Chagas disease, transmitted by hematophagous triatomine vectors. Establishing transmission cycles is key to understand the epidemiology of the disease, but integrative assessments of ecological interactions shaping parasite transmission are still limited. Current approaches also lack sensitivity to assess the full extent of this ecological diversity. Here we developed a metabarcoding approach based on next-generation sequencing to identify triatomine gut microbiome, vertebrate feeding hosts, and parasite diversity and their potential interactions. We detected a dynamic microbiome in Triatoma dimidiata, including 23 bacterial orders, which differed according to blood sources. Fourteen vertebrate species served as blood sources, corresponding to domestic, synantropic and sylvatic species, although four (human, dog, cow and mice) accounted for over 50% of blood sources. Importantly, bugs fed on multiple hosts, with up to 11 hosts identified per bug, indicating very frequent host-switching. A high clonal diversity of T. cruzi was detected, with up to 20 haplotypes per bug. This analysis provided much greater sensitivity to detect multiple blood meals and multiclonal infections with T. cruzi, which should be taken into account to develop transmission networks, and characterize the risk for human infection, eventually leading to a better control of disease transmission.</p>
<h1>Omics Advances in Ecotoxicology.</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.7B06494">doi:10.1021/Acs.Est.7B06494</a></p>
<p>No keywords</p><p>Toxic substances in the environment generate adverse effects at all levels of biological organization from the molecular level to community and ecosystem. Given this complexity, it is not surprising that ecotoxicologists have struggled to address the full consequences of toxic substance release at ecosystem level, due to the limits of observational and experimental tools to reveal the changes in deep structure at different levels of organization. -Omics technologies, consisting of genomics and ecogenomics, have the power to reveal, in unprecedented detail, the cellular processes of an individual or biodiversity of a community in response to environmental change with high sample/observation throughput. This represents a historic opportunity to transform the way we study toxic substances in ecosystems, through direct linkage of ecological effects with the systems biology of organisms. Three recent examples of -omics advance in the assessment of toxic substances are explored here: (1) the use of functional genomics in the discovery of novel molecular mechanisms of toxicity of chemicals in the environment; (2) the development of laboratory pipelines of dose-dependent, reduced transcriptomics to support high-throughput chemical testing at the biological pathway level; and (3) the use of eDNA metabarcoding approaches for assessing chemical effects on biological communities in mesocosm experiments and through direct observation in field monitoring. -Omics advances in ecotoxicological studies not only generate new knowledge regarding mechanisms of toxicity and environmental effect, improving the relevance and immediacy of laboratory toxicological assessment, but can provide a wholly new paradigm for ecotoxicology by linking ecological models to mechanism-based, systems biology approaches.</p>
<h1>Assessing Performance of Spore Samplers in Monitoring Aeromycobiota and Fungal Plant Pathogen Diversity in Canada.</h1>
<p><a href="https://doi.org/E02601-17">doi:E02601-17</a></p>
['HTS', 'air- and rain-borne fungi', 'high-throughput sequencing', 'metabarcodes', 'plant fungal pathogens', 'spore sampler']<p>Spore samplers are widely used in pathogen surveillance but not so much for monitoring the composition of aeromycobiota. In Canada, a nationwide spore-sampling network (AeroNet) was established as a pilot project to assess fungal community composition in air and rain samples collected using three different spore samplers in the summers of 2010 and 2011. Metabarcodes of the internal transcribed spacer (ITS) were exhaustively characterized for three of the network sites, in British Columbia (BC), Québec (QC), and Prince Edward Island (PEI), to compare performance of the samplers. Sampler type accounted for ca. 20% of the total explainable variance in aeromycobiota compositional heterogeneity, with air samplers recovering more Ascomycota and rain samplers recovering more Basidiomycota. Spore samplers showed different abilities to collect 27 fungal genera that are plant pathogens. For instance, <i>Cladosporium</i> spp., <i>Drechslera</i> spp., and <i>Entyloma</i> spp. were collected mainly by air samplers, while <i>Fusarium</i> spp., <i>Microdochium</i> spp., and <i>Ustilago</i> spp. were recovered more frequently with rain samplers. The diversity and abundance of some fungi were significantly affected by sampling location and time (e.g., <i>Alternaria</i> and <i>Bipolaris</i>) and weather conditions (e.g., <i>Mycocentrospora</i> and <i>Leptosphaeria</i>), and depended on using ITS1 or ITS2 as the barcoding region (e.g., <i>Epicoccum</i> and <i>Botrytis</i>). The observation that Canada's aeromycobiota diversity correlates with cooler, wetter conditions and northward wind requires support from more long-term data sets. Our vision of the AeroNet network, combined with high-throughput sequencing (HTS) and well-designed sampling strategies, may contribute significantly to a national biovigilance network for protecting plants of agricultural and economic importance in Canada.<b>IMPORTANCE</b> The current study compared the performance of spore samplers for collecting broad-spectrum air- and rain-borne fungal pathogens using a metabarcoding approach. The results provided a thorough characterization of the aeromycobiota in the coastal regions of Canada in relation to the influence of climatic factors. This study lays the methodological basis to eventually develop knowledge-based guidance on pest surveillance by assisting in the selection of appropriate spore samplers.</p>
<h1>A cross-taxa study using environmental DNA/RNA metabarcoding to measure biological impacts of offshore oil and gas drilling and production operations.</h1>
<p><a href="https://doi.org/S0025-326X(17)30999-2">doi:S0025-326X(17)30999-2</a></p>
['Bacteria 16S', 'Benthic ecology', 'Biomonitoring', 'Eukaryotes 18S', 'Foraminifera 18S', 'High-throughput sequencing']<p>Standardized ecosystem-based monitoring surveys are critical for providing information on marine ecosystem health. Environmental DNA/RNA (eDNA/eRNA) metabarcoding may facilitate such surveys by quickly and effectively characterizing multi-trophic levels. In this study, we assessed the suitability of eDNA/eRNA metabarcoding to evaluate changes in benthic assemblages of bacteria, Foraminifera and other eukaryotes along transects at three offshore oil and gas (O&G) drilling and production sites, and compared these to morphologically characterized macro-faunal assemblages. Bacterial communities were the most responsive to O&G activities, followed by Foraminifera, and macro-fauna (the latter assessed by morphology). The molecular approach enabled detection of hydrocarbon degrading taxa such as the bacteria Alcanivorax and Microbulbifer at petroleum impacted stations. Most identified indicator taxa, notably among macro-fauna, were highly specific to site conditions. Based on our results we suggest that eDNA/eRNA metabarcoding can be used as a stand-alone method for biodiversity assessment or as a complement to morphology-based monitoring approaches.</p>
<h1>Environmental DNA metabarcoding of benthic bacterial communities indicates the benthic footprint of salmon aquaculture.</h1>
<p><a href="https://doi.org/S0025-326X(17)31022-6">doi:S0025-326X(17)31022-6</a></p>
['Bacterial communities', 'Environmental monitoring', 'Salmon aquaculture', 'eDNA metabarcoding']<p>We evaluated benthic bacterial communities as bioindicators in environmental impact assessments of salmon aquaculture, a rapidly growing sector of seafood industry. Sediment samples (n=72) were collected from below salmon cages towards distant reference sites. Bacterial community profiles inferred from DNA metabarcodes were compared to reference data from standard macrofauna biomonitoring surveys of the same samples. Deltaproteobacteria were predominant in immediate vicinity of the salmon cages. Along the transect, significant shifts in bacterial community structures were observed with Gammaproteobacteria dominating the less-impacted sites. Alpha- and beta-diversity measures of bacterial communities correlated significantly with macrofauna diversity metrics and with five ecological status indices. Benthic bacterial communities mirror the reaction of macrofauna bioindicators to environmental disturbances caused by salmon farming. The implementation of bacterial eDNA metabarcoding in future Strategic Framework Directives is an alternative cost-effective high-throughput biomonitoring solution, providing a basis for management strategies in a matter of days rather than months.</p>
<h1>Surface ocean metabarcoding confirms limited diversity in planktonic foraminifera but reveals unknown hyper-abundant lineages.</h1>
<p><a href="https://doi.org/10.1038/S41598-018-20833-Z">doi:10.1038/S41598-018-20833-Z</a></p>
<p>No keywords</p><p>Since the advent of DNA metabarcoding surveys, the planktonic realm is considered a treasure trove of diversity, inhabited by a small number of abundant taxa, and a hugely diverse and taxonomically uncharacterized consortium of rare species. Here we assess if the apparent underestimation of plankton diversity applies universally. We target planktonic foraminifera, a group of protists whose known morphological diversity is limited, taxonomically resolved and linked to ribosomal DNA barcodes. We generated a pyrosequencing dataset of ~100,000 partial 18S rRNA foraminiferal sequences from 32 size fractioned photic-zone plankton samples collected at 8 stations in the Indian and Atlantic Oceans during the Tara Oceans expedition (2009-2012). We identified 69 genetic types belonging to 41 morphotaxa in our metabarcoding dataset. The diversity saturated at local and regional scale as well as in the three size fractions and the two depths sampled indicating that the diversity of foraminifera is modest and finite. The large majority of the newly discovered lineages occur in the small size fraction, neglected by classical taxonomy. These unknown lineages dominate the bulk [>0.8 µm] size fraction, implying that a considerable part of the planktonic foraminifera community biomass has its origin in unknown lineages.</p>
<h1>Effects of predation stress and food ration on perch gut microbiota.</h1>
<p><a href="https://doi.org/10.1186/S40168-018-0400-0">doi:10.1186/S40168-018-0400-0</a></p>
['16S rRNA', 'Food ration', 'Freshwater fish', 'Gut microbial communities', 'Predation stress']<p>Gut microbiota provide functions of importance to influence hosts' food digestion, metabolism, and protection against pathogens. Factors that affect the composition and functions of gut microbial communities are well studied in humans and other animals; however, we have limited knowledge of how natural food web factors such as stress from predators and food resource rations could affect hosts' gut microbiota and how it interacts with host sex. In this study, we designed a two-factorial experiment exposing perch (Perca fluviatilis) to a predator (pike, Esox lucius), and different food ratios, to examine the compositional and functional changes of perch gut microbiota based on 16S rRNA amplicon sequencing. We also investigated if those changes are host sex dependent.</p>
<h1>Species delimitation for the molecular taxonomy and ecology of the widely distributed microbial eukaryote genus <i>Euplotes</i> (Alveolata, Ciliophora).</h1>
<p><a href="https://doi.org/20172159">doi:20172159</a></p>
['Euplotes', 'ecology', 'eukaryotic microbes', 'integrative taxonomy', 'species delimitation']<p>Recent advances in high-throughput sequencing and metabarcoding technologies are revolutionizing our understanding of the diversity and ecology of microbial eukaryotes (protists). The interpretation of protist diversity and the elucidation of their ecosystem function are, however, impeded by problems with species delimitation, especially as it applies to molecular taxonomy. Here, using the ciliate <i>Euplotes</i> as an example, we describe approaches for species delimitation based on integrative taxonomy by using evolutionary and ecological perspectives and selecting the most appropriate metabarcoding gene markers as proxies for species units. Our analyses show that: <i>Euplotes</i> (<i>sensu lato</i>) comprises six distinct clades, mainly as result of ecological speciation; the validity of the genera <i>Euplotes</i> (<i>sensu stricto</i>), <i>Euplotoides</i>, <i>Euplotopsis</i> and <i>Moneuplotes</i> are not supported; the <i>vannus</i>-type group, which includes species without distinct morphological differences, seems to be undergoing incipient speciation and contains cryptic species; the hypervariable V4 region of the small subunit rDNA and D1-D2 region of the large subunit rDNA are the promising candidates for general species delimitation in <i>Euplotes</i>.</p>
<h1>Sorting specimen-rich invertebrate samples with cost-effective NGS barcodes: Validating a reverse workflow for specimen processing.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12751">doi:10.1111/1755-0998.12751</a></p>
['DNA barcoding', 'community ecology', 'insects', 'invertebrates', 'systematics']<p>Biologists frequently sort specimen-rich samples to species. This process is daunting when based on morphology, and disadvantageous if performed using molecular methods that destroy vouchers (e.g., metabarcoding). An alternative is barcoding every specimen in a bulk sample and then presorting the specimens using DNA barcodes, thus mitigating downstream morphological work on presorted units. Such a "reverse workflow" is too expensive using Sanger sequencing, but we here demonstrate that is feasible with an next-generation sequencing (NGS) barcoding pipeline that allows for cost-effective high-throughput generation of short specimen-specific barcodes (313 bp of COI; laboratory cost <$0.50 per specimen) through next-generation sequencing of tagged amplicons. We applied our approach to a large sample of tropical ants, obtaining barcodes for 3,290 of 4,032 specimens (82%). NGS barcodes and their corresponding specimens were then sorted into molecular operational taxonomic units (mOTUs) based on objective clustering and Automated Barcode Gap Discovery (ABGD). High diversity of 88-90 mOTUs (4% clustering) was found and morphologically validated based on preserved vouchers. The mOTUs were overwhelmingly in agreement with morphospecies (match ratio 0.95 at 4% clustering). Because of lack of coverage in existing barcode databases, only 18 could be accurately identified to named species, but our study yielded new barcodes for 48 species, including 28 that are potentially new to science. With its low cost and technical simplicity, the NGS barcoding pipeline can be implemented by a large range of laboratories. It accelerates invertebrate species discovery, facilitates downstream taxonomic work, helps with building comprehensive barcode databases and yields precise abundance information.</p>
<h1>Scaling up: A guide to high-throughput genomic approaches for biodiversity analysis.</h1>
<p><a href="https://doi.org/10.1111/Mec.14478">doi:10.1111/Mec.14478</a></p>
['DNA barcode', 'biodiversity', 'bioinformatics', 'biomonitoring', 'environment', 'high-throughput sequencing', 'metabarcoding', 'metagenomics']<p>The purpose of this review is to present the most common and emerging DNA-based methods used to generate data for biodiversity and biomonitoring studies. As environmental assessment and monitoring programmes may require biodiversity information at multiple levels, we pay particular attention to the DNA metabarcoding method and discuss a number of bioinformatic tools and considerations for producing DNA-based indicators using operational taxonomic units (OTUs), taxa at a variety of ranks and community composition. By developing the capacity to harness the advantages provided by the newest technologies, investigators can "scale up" by increasing the number of samples and replicates processed, the frequency of sampling over time and space, and even the depth of sampling such as by sequencing more reads per sample or more markers per sample. The ability to scale up is made possible by the reduced hands-on time and cost per sample provided by the newest kits, platforms and software tools. Results gleaned from broad-scale monitoring will provide opportunities to address key scientific questions linked to biodiversity and its dynamics across time and space as well as being more relevant for policymakers, enabling science-based decision-making, and provide a greater socio-economic impact. As genomic approaches are continually evolving, we provide this guide to methods used in biodiversity genomics.</p>
<h1>Quantitative species-level ecology of reef fish larvae via metabarcoding.</h1>
<p><a href="https://doi.org/10.1038/S41559-017-0413-2">doi:10.1038/S41559-017-0413-2</a></p>
<p>No keywords</p><p>The larval pool of coral reef fish has a crucial role in the dynamics of adult fish populations. However, large-scale species-level monitoring of species-rich larval pools has been technically impractical. Here, we use high-throughput metabarcoding to study larval ecology in the Gulf of Aqaba, a region that is inhabited by >500 reef fish species. We analysed 9,933 larvae from 383 samples that were stratified over sites, depth and time. Metagenomic DNA extracted from pooled larvae was matched to a mitochondrial cytochrome c oxidase subunit I barcode database compiled for 77% of known fish species within this region. This yielded species-level reconstruction of the larval community, allowing robust estimation of larval spatio-temporal distributions. We found significant correlations between species abundance in the larval pool and in local adult assemblages, suggesting a major role for larval supply in determining local adult densities. We documented larval flux of species whose adults were never documented in the region, suggesting environmental filtering as the reason for the absence of these species. Larvae of several deep-sea fishes were found in shallow waters, supporting their dispersal over shallow bathymetries, potentially allowing Lessepsian migration into the Mediterranean Sea. Our method is applicable to any larval community and could assist coral reef conservation and fishery management efforts.</p>
<h1>Performance of DNA metabarcoding, standard barcoding, and morphological approach in the identification of host-parasitoid interactions.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0187803">doi:10.1371/Journal.Pone.0187803</a></p>
<p>No keywords</p><p>Understanding interactions between herbivores and parasitoids is essential for successful biodiversity protection and monitoring and for biological pest control. Morphological identifications employ insect rearing and are complicated by insects' high diversity and crypsis. DNA barcoding has been successfully used in studies of host-parasitoid interactions as it can substantially increase the recovered real host-parasitoid diversity distorted by overlooked species complexes, or by species with slight morphological differences. However, this approach does not allow the simultaneous detection and identification of host(s) and parasitoid(s). Recently, high-throughput sequencing has shown high potential for surveying ecological communities and trophic interactions. Using mock samples comprising insect larvae and their parasitoids, we tested the potential of DNA metabarcoding for identifying individuals involved in host-parasitoid interactions to different taxonomic levels, and compared it to standard DNA barcoding and morphological approaches. For DNA metabarcoding, we targeted the standard barcoding marker cytochrome oxidase subunit I using highly degenerate primers, 2*300 bp sequencing on a MiSeq platform, and RTAX classification using paired-end reads. Additionally, using a large host-parasitoid dataset from a Central European floodplain forest, we assess the completeness and usability of a local reference library by confronting the number of Barcoding Index Numbers obtained by standard barcoding with the number of morphotypes. Overall, metabarcoding recovery was high, identifying 92.8% of the taxa present in mock samples, and identification success within individual taxonomic levels did not significantly differ among metabarcoding, standard barcoding, and morphology. Based on the current local reference library, 39.4% parasitoid and 90.7% host taxa were identified to the species level. DNA barcoding estimated higher parasitoid diversity than morphotyping, especially in groups with high level of crypsis. This study suggests the potential of metabarcoding for effectively recovering host-parasitoid diversity, together with more accurate identifications obtained from building reliable and comprehensive reference libraries, especially for parasitoids.</p>
<h1>Metagenomic sequencing of environmental DNA reveals marine faunal assemblages from the West Antarctic Peninsula.</h1>
<p><a href="https://doi.org/S1874-7787(17)30258-1">doi:S1874-7787(17)30258-1</a></p>
['Antarctica', 'Benthic invertebrates', 'King crabs', 'Metagenomics', 'eDNA']<p>The West Antarctic Peninsula (WAP) is the fastest warming region in Antarctica where climate impact on the cold-adapted marine ecosystem is already visible. To monitor faunal changes in remote vast bodies of Antarctic waters, efficient and informative tools are essential. High-throughput sequencing of environmental DNA (eDNA) has emerged as one such tool for monitoring biodiversity and ecosystems, as it increases detection sensitivity of taxa, and sampling is often simpler and less costly than traditional collection methods. We collected water samples from four WAP shallow (≤300m) shelf regions, recovered the eDNA therein, and performed metagenomic shotgun sequencing and analyses to determine the effectiveness of this method to assess marine benthic faunal diversity; this includes the detection of deep-water predatory king crabs whose potential shoreward expansion to warming shelves has sparked much concern. Using a customized bioinformatics pipeline, we identified abundant signatures of common benthic invertebrate fauna, endemic notothenioid fishes, as well as lithodid king crabs. We also uncovered species richness and diversity comparable to biological inventories compiled by the use of traditional survey methods, supporting the efficacy of the eDNA shotgun sequencing approach. As the rate of eDNA degradation affects faunal detection sensitivity, we also quantified mitochondrial ND2 gene copies in eDNA derived from a WAP icefish and found ND2 copies persisted to at least 20days in the cold WAP water, much longer than values reported for temperate environments. We propose that eDNA metagenomic sequencing complements traditional sampling, and combining both will enable more inclusive biodiversity detection and faunal change monitoring in the vast Southern Ocean.</p>
<h1>Genetic monitoring of open ocean biodiversity: An evaluation of DNA metabarcoding for processing continuous plankton recorder samples.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12740">doi:10.1111/1755-0998.12740</a></p>
['DNA barcoding', 'biomonitoring', 'environmental DNA', 'high-throughput sequencing']<p>DNA metabarcoding is an efficient method for measuring biodiversity, but the process of initiating long-term DNA-based monitoring programmes, or integrating with conventional programs, is only starting. In marine ecosystems, plankton surveys using the continuous plankton recorder (CPR) have characterized biodiversity along transects covering millions of kilometres with time-series spanning decades. We investigated the potential for use of metabarcoding in CPR surveys. Samples (n = 53) were collected in two Southern Ocean transects and metazoans identified using standard microscopic methods and by high-throughput sequencing of a cytochrome c oxidase subunit I marker. DNA increased the number of metazoan species identified and provided high-resolution taxonomy of groups problematic in conventional surveys (e.g., larval echinoderms and hydrozoans). Metabarcoding also generally produced more detections than microscopy, but this sensitivity may make cross-contamination during sampling a problem. In some samples, the prevalence of DNA from large plankton such as krill masked the presence of smaller species. We investigated adding a fixed amount of exogenous DNA to samples as an internal control to allow determination of relative plankton biomass. Overall, the metabarcoding data represent a substantial shift in perspective, making direct integration into current long-term time-series challenging. We discuss a number of hurdles that exist for progressing DNA metabarcoding from the current snapshot studies to the requirements of a long-term monitoring programme. Given the power and continually increasing efficiency of metabarcoding, it is almost certain this approach will play an important role in future plankton monitoring.</p>
<h1>Fungal Community Structure and As-Resistant Fungi in a Decommissioned Gold Mine Site.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2017.02202">doi:10.3389/Fmicb.2017.02202</a></p>
['arsenic resistant fungi', 'arsenic volatilization', 'fungal community', 'gold mine', 'high throughput sequencing', 'nuclear ribosomal internal transcribed spacers profiling']<p>Although large quantities of heavy metal laden wastes are released in an uncontrolled manner by gold mining activities with ensuing contamination of the surrounding areas, there is scant information on the mycobiota of gold-mine sites. Thus, the present study was aimed to describe the fungal community structure in three differently As- and Hg-polluted soils collected from the Pestarena decommissioned site by using Illumina® metabarcoding. Fungal richness was found to increase as the contamination level increased while biodiversity was not related to the concentrations of inorganic toxicants. Within the phylum Zygomigota which, irrespective of the contamination level, was predominant in all the soils under study, the most abundant genera were <i>Mucor</i> and <i>Mortierella</i>. The relative abundances of Basidiomycota, instead, tended to raise as the contamination increased; within this phylum the most abundant genera were <i>Cryptococcus</i> and <i>Pseudotomentella</i>. The abundance of Ascomycota, ranging from about 8 to 21%, was not related to the contamination level. The relative abundances of those genera (i.e., <i>Penicillium, Trichoderma</i>, and <i>Chaetomium</i>), the cultivable isolates of which exhibited significant As-resistance, were lower than the set threshold (0.5%). Mass balances obtained from As-exposure experiments with these isolates showed that the main mechanisms involved in counteracting the toxicant were accumulation and, above all, volatilization, the respective extents of which ranged from 0.6 to 5.9% and from 6.4 to 31.2% in dependence of the isolate.</p>
<h1>FuzzyID2: A software package for large data set species identification via barcoding and metabarcoding using hidden Markov models and fuzzy set methods.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12738">doi:10.1111/1755-0998.12738</a></p>
['DNA barcoding', 'eDNA', 'fuzzy membership function', 'hidden Markov models', 'high-throughput sequencing (HTS)', 'metabarcoding', 'plant barcodes']<p>Species identification through DNA barcoding or metabarcoding has become a key approach for biodiversity evaluation and ecological studies. However, the rapid accumulation of barcoding data has created some difficulties: for instance, global enquiries to a large reference library can take a very long time. We here devise a two-step searching strategy to speed identification procedures of such queries. This firstly uses a Hidden Markov Model (HMM) algorithm to narrow the searching scope to genus level and then determines the corresponding species using minimum genetic distance. Moreover, using a fuzzy membership function, our approach also estimates the credibility of assignment results for each query. To perform this task, we developed a new software pipeline, FuzzyID2, using Python and C++. Performance of the new method was assessed using eight empirical data sets ranging from 70 to 234,535 barcodes. Five data sets (four animal, one plant) deployed the conventional barcode approach, one used metabarcodes, and two were eDNA-based. The results showed mean accuracies of generic and species identification of 98.60% (with a minimum of 95.00% and a maximum of 100.00%) and 94.17% (with a range of 84.40%-100.00%), respectively. Tests with simulated NGS sequences based on realistic eDNA and metabarcode data demonstrated that FuzzyID2 achieved a significantly higher identification success rate than the commonly used Blast method, and the TIPP method tends to find many fewer species than either FuzztID2 or Blast. Furthermore, data sets with tens of thousands of barcodes need only a few seconds for each query assignment using FuzzyID2. Our approach provides an efficient and accurate species identification protocol for biodiversity-related projects with large DNA sequence data sets.</p>
<h1>DNA metabarcoding for high-throughput monitoring of estuarine macrobenthic communities.</h1>
<p><a href="https://doi.org/10.1038/S41598-017-15823-6">doi:10.1038/S41598-017-15823-6</a></p>
<p>No keywords</p><p>Morphology-based profiling of benthic communities has been extensively applied to aquatic ecosystems' health assessment. However, it remains a low-throughput, and sometimes ambiguous, procedure. Despite DNA metabarcoding has been applied to marine benthos, a comprehensive approach providing species-level identifications for estuarine macrobenthos is still lacking. Here we report a combination of experimental and field studies to assess the aptitude of COI metabarcoding to provide robust species-level identifications for high-throughput monitoring of estuarine macrobenthos. To investigate the ability of metabarcoding to detect all species present in bulk DNA extracts, we contrived three phylogenetically diverse communities, and applied four different primer pairs to generate PCR products within the COI barcode region. Between 78-83% of the species in the contrived communities were recovered through HTS. Subsequently, we compared morphology and metabarcoding-based approaches to determine the species composition from four distinct estuarine sites. Our results indicate that species richness would be considerably underestimated if only morphological methods were used: globally 27 species identified through morphology versus 61 detected by metabarcoding. Although further refinement is required to improve efficiency and output of this approach, here we show the great aptitude of COI metabarcoding to provide high quality and auditable species identifications in estuarine macrobenthos monitoring.</p>
<h1>International Barcode of Life: Focus on big biodiversity in South Africa.</h1>
<p><a href="https://doi.org/10.1139/Gen-2017-0210">doi:10.1139/Gen-2017-0210</a></p>
['ADN environnemental', 'DNA barcoding', 'bioinformatics', 'bioinformatique', 'biomes', 'codage à barres de l’ADN', 'community phylogenetics', 'ecology', 'environmental DNA', 'evolution', 'genome skimming', 'genomics', 'génomique', 'high-throughput sequencing', 'next-generation sequencing', 'phylogénétique des communautés', 'pollination', 'pollinisation', 'research trends', 'séquençage de prochaine génération', 'séquençage à grand débit', 'tendances en recherche', 'écologie', 'écrémage génomique', 'évolution']<p>Participants in the 7th International Barcode of Life Conference (Kruger National Park, South Africa, 20-24 November 2017) share the latest findings in DNA barcoding research and its increasingly diversified applications. Here, we review prevailing trends synthesized from among 429 invited and contributed abstracts, which are collated in this open-access special issue of Genome. Hosted for the first time on the African continent, the 7th Conference places special emphasis on the evolutionary origins, biogeography, and conservation of African flora and fauna. Within Africa and elsewhere, DNA barcoding and related techniques are being increasingly used for wildlife forensics and for the validation of commercial products, such as medicinal plants and seafood species. A striking trend of the conference is the dramatic rise of studies on environmental DNA (eDNA) and on diverse uses of high-throughput sequencing techniques. Emerging techniques in these areas are opening new avenues for environmental biomonitoring, managing species-at-risk and invasive species, and revealing species interaction networks in unprecedented detail. Contributors call for the development of validated community standards for high-throughput sequence data generation and analysis, to enable the full potential of these methods to be realized for understanding and managing biodiversity on a global scale.</p>
<h1>Metabarcoding of freshwater invertebrates to detect the effects of a pesticide spill.</h1>
<p><a href="https://doi.org/10.1111/Mec.14410">doi:10.1111/Mec.14410</a></p>
['DNA barcoding', 'biomonitoring', 'community ecology', 'freshwater ecosystems', 'invertebrates']<p>Biomonitoring underpins the environmental assessment of freshwater ecosystems and guides management and conservation. Current methodology for surveys of (macro)invertebrates uses coarse taxonomic identification where species-level resolution is difficult to obtain. Next-generation sequencing of entire assemblages (metabarcoding) provides a new approach for species detection, but requires further validation. We used metabarcoding of invertebrate assemblages with two fragments of the cox1 "barcode" and partial nuclear ribosomal (SSU) genes, to assess the effects of a pesticide spill in the River Kennet (southern England). Operational taxonomic unit (OTU) recovery was tested under 72 parameters (read denoising, filtering, pair merging and clustering). Similar taxonomic profiles were obtained under a broad range of parameters. The SSU marker recovered Platyhelminthes and Nematoda, missed by cox1, while Rotifera were only amplified with cox1. A reference set was created from all available barcode entries for Arthropoda in the BOLD database and clustered into OTUs. The River Kennet metabarcoding produced matches to 207 of these reference OTUs, five times the number of species recognized with morphological monitoring. The increase was due to the following: greater taxonomic resolution (e.g., splitting a single morphotaxon "Chironomidae" into 55 named OTUs); splitting of Linnaean binomials into multiple molecular OTUs; and the use of a filtration-flotation protocol for extraction of minute specimens (meiofauna). Community analyses revealed strong differences between "impacted" vs. "control" samples, detectable with each gene marker, for each major taxonomic group, and for meio- and macrofaunal samples separately. Thus, highly resolved taxonomic data can be extracted at a fraction of the time and cost of traditional nonmolecular methods, opening new avenues for freshwater invertebrate biodiversity monitoring and molecular ecology.</p>
<h1>Algorithm for post-clustering curation of DNA amplicon data yields reliable biodiversity estimates.</h1>
<p><a href="https://doi.org/10.1038/S41467-017-01312-X">doi:10.1038/S41467-017-01312-X</a></p>
<p>No keywords</p><p>DNA metabarcoding is promising for cost-effective biodiversity monitoring, but reliable diversity estimates are difficult to achieve and validate. Here we present and validate a method, called LULU, for removing erroneous molecular operational taxonomic units (OTUs) from community data derived by high-throughput sequencing of amplified marker genes. LULU identifies errors by combining sequence similarity and co-occurrence patterns. To validate the LULU method, we use a unique data set of high quality survey data of vascular plants paired with plant ITS2 metabarcoding data of DNA extracted from soil from 130 sites in Denmark spanning major environmental gradients. OTU tables are produced with several different OTU definition algorithms and subsequently curated with LULU, and validated against field survey data. LULU curation consistently improves α-diversity estimates and other biodiversity metrics, and does not require a sequence reference database; thus, it represents a promising method for reliable biodiversity estimation.</p>
<h1>Flooding Duration Affects the Structure of Terrestrial and Aquatic Microbial Eukaryotic Communities.</h1>
<p><a href="https://doi.org/10.1007/S00248-017-1085-9">doi:10.1007/S00248-017-1085-9</a></p>
['Aquatic eukaryotes', 'Flooding duration', 'High-throughput sequencing', 'Mesocosm experiment', 'Metabarcoding', 'Terrestrial eukaryotes']<p>The increasing number and duration of inundations is reported to be a consequence of climate change and may severely compromise non-adapted macroorganisms. The effect of flooding events on terrestrial and aquatic microbial communities is, however, less well understood. They may respond to the changed abiotic properties of their native habitat, and the native community may change due to the introduction of alien species. We designed an experiment to investigate the effect of five different flooding durations on the terrestrial and aquatic communities of eukaryotic microorganism, using the AquaFlow mesocosms. With amplicon sequencing of the small subunit (SSU) and internal transcribed spacer (ITS) rRNA gene regions, we analyzed community compositions directly before and after flooding. Subsequently, they were monitored for another 28 days, to determine the sustainability of community changes. Our results revealed a temporary increase in similarity between terrestrial and aquatic communities according to OTU composition (operational taxonomic unit, serves as a proxy for species). Increased similarity was mainly caused by the transmission of OTUs from water to soil. A minority of these were able to persist in soil until the end of the experiment. By contrast, the vast majority of soil OTUs was not transmitted to water. Flooding duration affected the community structure (abundance) more than composition (occurrence). Terrestrial communities responded immediately to flooding and the flooding duration influenced the community changes. Independent from flooding duration, all terrestrial communities recovered largely after flooding, indicating a remarkable resilience to the applied disturbances. Aquatic communities responded immediately to the applied inundations too. At the end of the experiment, they grouped according to the applied flooding duration and the amount of ammonium and chloride that leached from the soil. This indicates a sustained long-term response of the aquatic communities to flooding events.</p>
<h1>Indigenous species barcode database improves the identification of zooplankton.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0185697">doi:10.1371/Journal.Pone.0185697</a></p>
<p>No keywords</p><p>Incompleteness and inaccuracy of DNA barcode databases is considered an important hindrance to the use of metabarcoding in biodiversity analysis of zooplankton at the species-level. Species barcoding by Sanger sequencing is inefficient for organisms with small body sizes, such as zooplankton. Here mitochondrial cytochrome c oxidase I (COI) fragment barcodes from 910 freshwater zooplankton specimens (87 morphospecies) were recovered by a high-throughput sequencing platform, Ion Torrent PGM. Intraspecific divergence of most zooplanktons was < 5%, except Branchionus leydign (Rotifer, 14.3%), Trichocerca elongate (Rotifer, 11.5%), Lecane bulla (Rotifer, 15.9%), Synchaeta oblonga (Rotifer, 5.95%) and Schmackeria forbesi (Copepod, 6.5%). Metabarcoding data of 28 environmental samples from Lake Tai were annotated by both an indigenous database and NCBI Genbank database. The indigenous database improved the taxonomic assignment of metabarcoding of zooplankton. Most zooplankton (81%) with barcode sequences in the indigenous database were identified by metabarcoding monitoring. Furthermore, the frequency and distribution of zooplankton were also consistent between metabarcoding and morphology identification. Overall, the indigenous database improved the taxonomic assignment of zooplankton.</p>
<h1>Insights into the harmful algal flora in northwestern Mediterranean coastal lagoons revealed by pyrosequencing metabarcodes of the 28S rRNA gene.</h1>
<p><a href="https://doi.org/S1568-9883(16)30370-5">doi:S1568-9883(16)30370-5</a></p>
['Harmful algae', 'Metabarcoding', 'Monitoring', 'Phytoplankton diversity']<p>This study investigated the genetic diversity of phytoplankton communities in six shallow lagoons located on the French coast of the northwestern Mediterranean Sea that represented a trophic gradient ranging from oligotrophic to hypereutrophic. The phytoplankton communities were sampled once a month from spring (May) to the beginning of autumn (September/early October) in 2012 and fractionated by size. Metabarcodes were generated from cDNAs by targeting the D1-D2 region of the 28S rRNA gene and pyrosequenced using Roche 454 technology. Examination of the annotated barcodes revealed harmful algal species not previously documented in these lagoons. Three ichthyotoxic species belonging to Pfiesteriaceae were detected: Luciella masanensis was relatively widespread and abundant in many samples, whereas Pfiesteria piscicida and Stoeckeria changwonensis were found as single barcode sequences. Furthermore, a phylogenetic analysis of barcodes annotated as belonging to Pfiesteriaceae suggested the existence of two previously undescribed clades. The other toxic or potentially harmful dinoflagellates detected through rare barcodes were Dinophysis acuminata, Vulcanodinium rugosum, Alexandrium andersonii and A. ostenfeldii. The two most abundant dinoflagellate taxa were Gymnodinium litoralis and Akashiwo sanguinea with respect to sequence numbers. Four diatom species from the genus Pseudo-nitzschia that potentially produce domoic acid were identified (P. galaxiae, P. delicatissima, P. brasiliana and P. calliantha). These observations are discussed in terms of the literature and monitoring records related to the identified taxa in this Mediterranean area.</p>
<h1>The Protistan Microbiome of Grassland Soil: Diversity in the Mesoscale.</h1>
<p><a href="https://doi.org/S1434-4610(17)30034-2">doi:S1434-4610(17)30034-2</a></p>
['Molecular ecology', 'diversity', 'land-use intensity', 'soil', 'spatial distribution', 'taxa-area relationship.']<p>Genomic data for less than one quarter of ∼1.8 million named species on earth exist in public databases like GenBank. Little information exists on the estimated one million small sized (1-100μm) heterotrophic nanoflagellates and ciliates and their taxa-area relationship. We analyzed environmental DNA from 150 geo-referenced grassland plots representing topographical and land-use ranges typical for Central Europe. High through-put barcoding allowed the identification of operational taxonomic units (OTUs) at species level, with high pairwise identity to reference sequences (≥99.7%), but also the identification of sequences at the genus (≥97%) and class (≥80%) taxonomic level. Species richness analyses revealed, on average, 100 genus level OTUs (332 unique individual read (UIR) and 56 class level OTUs per gram of soil sample in the mesoscale (1-1000km). Database shortfalls were highlighted by increased uncertain taxonomic lineages at lower resolution (≥80% sequence identity). No single barcode occurred ubiquitously across all sites. Taxa-area relationships indicated that OTUs spread over the entire mesoscale were more similar than in the local scale and increased land-use (fertilization, mowing and grazing) promoted taxa-area separation. Only a small fraction of sequences strictly matched reference library sequences, suggesting a large protistan "dark matter" in soil which warrants further research.</p>
<h1>Environmental DNA metabarcoding: Transforming how we survey animal and plant communities.</h1>
<p><a href="https://doi.org/10.1111/Mec.14350">doi:10.1111/Mec.14350</a></p>
['bioinformatic pipeline', 'biomonitoring', 'citizen science', 'conservation', 'eDNA', 'ecology', 'invasive species', 'macro-organism', 'species richness']<p>The genomic revolution has fundamentally changed how we survey biodiversity on earth. High-throughput sequencing ("HTS") platforms now enable the rapid sequencing of DNA from diverse kinds of environmental samples (termed "environmental DNA" or "eDNA"). Coupling HTS with our ability to associate sequences from eDNA with a taxonomic name is called "eDNA metabarcoding" and offers a powerful molecular tool capable of noninvasively surveying species richness from many ecosystems. Here, we review the use of eDNA metabarcoding for surveying animal and plant richness, and the challenges in using eDNA approaches to estimate relative abundance. We highlight eDNA applications in freshwater, marine and terrestrial environments, and in this broad context, we distill what is known about the ability of different eDNA sample types to approximate richness in space and across time. We provide guiding questions for study design and discuss the eDNA metabarcoding workflow with a focus on primers and library preparation methods. We additionally discuss important criteria for consideration of bioinformatic filtering of data sets, with recommendations for increasing transparency. Finally, looking to the future, we discuss emerging applications of eDNA metabarcoding in ecology, conservation, invasion biology, biomonitoring, and how eDNA metabarcoding can empower citizen science and biodiversity education.</p>
<h1>From Microscopy to Genomic Approach in Soil Biodiversity Assessment.</h1>
<p><a href="https://doi.org/10.21775/Cimb.027.195">doi:10.21775/Cimb.027.195</a></p>
<p>No keywords</p><p>Soil biota represents a major component of the earth's biodiversity and for over 200 years, the microscopy approach was the only way to explore it. In the last decade, the DNA-based technique has been adopted in soil ecology. Due to the rapid development of cutting-edge technology, the field is transitioning from barcoding individuals to metabarcoding communities. With the advent of next-generation sequencing and a rapid decline in sequencing cost, it has become feasible to assess soil biodiversity at species level. This review article summarizes current approaches in soil biodiversity research along with their advantages and disadvantages.</p>
<h1>DNA in a bottle-Rapid metabarcoding survey for early alerts of invasive species in ports.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0183347">doi:10.1371/Journal.Pone.0183347</a></p>
<p>No keywords</p><p>Biota monitoring in ports is increasingly needed for biosecurity reasons and safeguarding marine biodiversity from biological invasion. Present and future international biosecurity directives can be accomplished only if the biota acquired by maritime traffic in ports is controlled. Methodologies for biota inventory are diverse and now rely principally on extensive and labor-intensive sampling along with taxonomic identification by experts. In this study, we employed an extremely simplified environmental DNA (eDNA) sampling methodology from only three 1-L bottles of water per port, followed by metabarcoding (high-throughput sequencing and DNA-based species identification) using 18S rDNA and Cytochrome oxidase I as genetic barcodes. Eight Bay of Biscay ports with available inventory of fouling invertebrates were employed as a case study. Despite minimal sampling efforts, three invasive invertebrates were detected: the barnacle Austrominius modestus, the tubeworm Ficopomatus enigmaticus and the polychaete Polydora triglanda. The same species have been previously found from visual and DNA barcoding (genetic identification of individuals) surveys in the same ports. The current costs of visual surveys, conventional DNA barcoding and this simplified metabarcoding protocol were compared. The results encourage the use of metabarcoding for early biosecurity alerts.</p>
<h1>Large-scale differences in microbial biodiversity discovery between 16S amplicon and shotgun sequencing.</h1>
<p><a href="https://doi.org/10.1038/S41598-017-06665-3">doi:10.1038/S41598-017-06665-3</a></p>
<p>No keywords</p><p>Modern metagenomic environmental DNA studies are almost completely reliant on next-generation sequencing, making evaluations of these methods critical. We compare two next-generation sequencing techniques - amplicon and shotgun - on water samples across four of Brazil's major river floodplain systems (Amazon, Araguaia, Paraná, and Pantanal). Less than 50% of phyla identified via amplicon sequencing were recovered from shotgun sequencing, clearly challenging the dogma that mid-depth shotgun recovers more diversity than amplicon-based approaches. Amplicon sequencing also revealed ~27% more families. Overall the amplicon data were more robust across both biodiversity and community ecology analyses at different taxonomic scales. Our work doubles the sampling size in similar environmental studies, and novelly integrates environmental data (e.g., pH, temperature, nutrients) from each site, revealing divergent correlations depending on which data are used. While myriad variants on NGS techniques and bioinformatic pipelines are available, our results point to core differences that have not been highlighted in any studies to date. Given the low number of taxa identified when coupling shotgun data with clade-based taxonomic algorithms, previous studies that quantified biodiversity using such bioinformatic tools should be viewed cautiously or re-analyzed. Nonetheless, shotgun has complementary advantages that should be weighed when designing projects.</p>
<h1>Revealing higher than expected meiofaunal diversity in Antarctic sediments: a metabarcoding approach.</h1>
<p><a href="https://doi.org/10.1038/S41598-017-06687-X">doi:10.1038/S41598-017-06687-X</a></p>
<p>No keywords</p><p>An increasing number of studies are showing that Antarctic mega- and macrofauna are highly diverse, however, little is known about meiofaunal biodiversity in sediment communities, which are a vital part of a healthy and functional ecosystem. This is the first study to analyse community DNA (targeting meiofauna) using metabarcoding to investigate biodiversity levels in sediment communities of the Antarctic Peninsula. The results show that almost all of the meiofaunal biodiversity in the benthic habitat has yet to be characterised, levels of biodiversity were higher than expected and similar to temperate regions, albeit with the existence of potentially new and locally adapted species never described before at the molecular level. The Rothera meiofaunal sample sites showed four dominant eukaryotic groups, the nematodes, arthropods, platyhelminthes, and the annelids; some of which could comprise species complexes. Comparisons with deep-sea data from the same region suggest little exchange of Operational Taxonomic Units (OTUs) between depths with the nematodes prevalent at all depths, but sharing the shallow water benthos with the copepods. This study provides a preliminary analysis of benthic Antarctic Peninsula meiofauna using high throughput sequencing which substantiates how little is known on the biodiversity of one of the most diverse, yet underexplored communities of the Antarctic: the benthos.</p>
<h1>Using metagenomics to show the efficacy of forest restoration in the New Jersey Pine Barrens.</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0199">doi:10.1139/Gen-2015-0199</a></p>
['carbon use', 'diversité des communautés telluriques', 'restauration des sols en milieux humides', 'soil community diversity', 'utilisation du carbone', 'wetland soils ecosystem restoration']<p>The Franklin Parker Preserve within the New Jersey Pine Barrens contains 5000 acres of wetlands habitat, including old-growth Atlantic white cedar (or AWC; Chamaecyparis thyoides) swamps, cranberry bogs, and former cranberry bogs undergoing restoration into AWC forests. This study showed that the C-use efficiency was greater in the old-growth AWC soils than in soils from 8-year-old mid-stage restored AWC stands, which were greater than found in soil from 4-year-old AWC stands-the latter two stands being restored from long-term cranberry bogs. A metagenomic analysis of eDNA extracted from these soils showed that the C-cycle trends were associated with increases in the relative numbers of DNA sequences from several copiotrophic bacterial groups (Bacteroidetes and Proteobacteria), complex C-decomposing fungal groups (Sordiomycetes, Mortierellales, and Thelephorales), and collembolan and formicid invertebrates. All groups are indicators of successionally more advanced soils, and critical for soil C-cycle activities. These data suggest that the restoration activities studied are enhancing critical guilds of soil biota, and increasing C-use efficiency in the soils of restored habitats, and that the use of metagenomic analysis of soil eDNA can be used in the development of assessment models for soil recovery of wetlands following restoration.</p>
<h1>Taxa-area relationship of aquatic fungi on deciduous leaves.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0181545">doi:10.1371/Journal.Pone.0181545</a></p>
<p>No keywords</p><p>One of the fundamental patterns in macroecology is the increase in the number of observed taxa with size of sampled area. For microbes, the shape of this relationship remains less clear. The current study assessed the diversity of aquatic fungi, by the traditional approach based on conidial morphology (captures reproducing aquatic hyphomycetes) and next generation sequencing (NGS; captures other fungi as well), on graded sizes of alder leaves (0.6 to 13.6 cm2). Leaves were submerged in two streams in geographically distant locations: the Oliveira Stream in Portugal and the Boss Brook in Canada. Decay rates of alder leaves and fungal sporulation rates did not differ between streams. Fungal biomass was higher in Boss Brook than in Oliveira Stream, and in both streams almost 100% of the reads belonged to active fungal taxa. In general, larger leaf areas tended to harbour more fungi, but these findings were not consistent between techniques. Morphospecies-based diversity increased with leaf area in Boss Brook, but not in Oliveira Stream; metabarcoding data showed an opposite trend. The higher resolution of metabarcoding resulted in steeper taxa-accumulation curves than morphospecies-based assessments (fungal conidia morphology). Fungal communities assessed by metabarcoding were spatially structured by leaf area in both streams. Metabarcoding promises greater resolution to assess biodiversity patterns in aquatic fungi and may be more accurate for assessing taxa-area relationships and local to global diversity ratios.</p>
<h1>Predicting the Ecological Quality Status of Marine Environments from eDNA Metabarcoding Data Using Supervised Machine Learning.</h1>
<p><a href="https://doi.org/10.1021/Acs.Est.7B01518">doi:10.1021/Acs.Est.7B01518</a></p>
<p>No keywords</p><p>Monitoring biodiversity is essential to assess the impacts of increasing anthropogenic activities in marine environments. Traditionally, marine biomonitoring involves the sorting and morphological identification of benthic macro-invertebrates, which is time-consuming and taxonomic-expertise demanding. High-throughput amplicon sequencing of environmental DNA (eDNA metabarcoding) represents a promising alternative for benthic monitoring. However, an important fraction of eDNA sequences remains unassigned or belong to taxa of unknown ecology, which prevent their use for assessing the ecological quality status. Here, we show that supervised machine learning (SML) can be used to build robust predictive models for benthic monitoring, regardless of the taxonomic assignment of eDNA sequences. We tested three SML approaches to assess the environmental impact of marine aquaculture using benthic foraminifera eDNA, a group of unicellular eukaryotes known to be good bioindicators, as features to infer macro-invertebrates based biotic indices. We found similar ecological status as obtained from macro-invertebrates inventories. We argue that SML approaches could overcome and even bypass the cost and time-demanding morpho-taxonomic approaches in future biomonitoring.</p>
<h1>Analysis of phytoplankton assemblage structure in the Mediterranean Sea based on high-throughput sequencing of partial 18S rRNA sequences.</h1>
<p><a href="https://doi.org/S1874-7787(17)30144-7">doi:S1874-7787(17)30144-7</a></p>
['18S rDNA', 'Ecology', 'Eukaryotic diversity', 'Human impact', 'Metabarcoding']<p>Studying taxonomic and ecological diversity of phytoplankton assemblages is often difficult because morphological analysis cannot provide a complete description of their composition. Therefore, more robust and feasible approaches have to be chosen to elucidate the interactions between environmental and human pressures and phytoplankton assemblages. The Ocean Sampling Day (OSD) allowed collecting seawater samples from a wide range of oceanic regions including the Mediterranean Sea. In this study, a total of 754,167 V4-18S ribosomal DNA (rDNA) metabarcodes derived from 20 plankton samples collected at 19 sampling sites across the coastal areas of the Mediterranean Sea were analyzed to explore the relationships between phytoplankton assemblages' composition, sub-regional environmental features and human pressures. We reduced the whole set of autotroph plankton (1398 OTUs) to a smaller number of ecologically relevant entities (205 taxa) and used the latter for analysing the structure of phytoplankton assemblages. Chaetoceros was the only genus occurring in all the samples, while the number of taxa was maximum in the W Mediterranean. Based on the assigned OTUs, the structure of E Mediterranean phytoplankton was the most homogeneous. Further, phytoplankton assemblages from the three Mediterranean sub-regions (Western, Adriatic and Eastern) were significantly different (R=0.25, p=0.0136) based on Jaccard similarity. We also observed that phytoplankton diversity and human impact on marine ecosystems were not significantly related to each other based on Mantel's test.</p>
<h1>Zooplankton community structure along a pollution gradient at fine geographical scales in river ecosystems: The importance of species sorting over dispersal.</h1>
<p><a href="https://doi.org/10.1111/Mec.14199">doi:10.1111/Mec.14199</a></p>
['dispersal', 'environmental pollution', 'eutrophication', 'metabarcoding', 'species sorting', 'zooplankton']<p>The release of anthropogenic pollution into freshwater ecosystems has largely transformed biodiversity and its geographical distribution patterns globally. However, for many communities including ecologically crucial ones such as zooplankton, it is largely unknown how different communities respond to environmental pollution. Collectively, dispersal and species sorting are two competing processes in determining the structure and geographical distribution of zooplankton communities in running water ecosystems such as rivers. At fine geographical scales, dispersal is usually considered as the dominant factor; however, the relative role of species sorting has not been evaluated well, mainly because significant environmental gradients rarely exist along continuously flowing rivers. The Chaobai River in northern China represents a rare system, where a significant environmental gradient exists at fine scales. Here, we employed high-throughput sequencing to characterize complex zooplankton communities collected from the Chaobai River, and tested the relative roles of dispersal and species sorting in determining zooplankton community structure along the pollution gradient. Our results showed distinct patterns of zooplankton communities along the environmental gradient, and chemical pollutant-related factors such as total phosphorus and chlorophyll-a were identified as the major drivers for the observed patterns. Further partial redundancy analyses showed that species sorting overrode the effect of dispersal to shape local zooplankton community structure. Thus, our results reject the dispersal hypothesis and support the concept that species sorting caused by local pollution can largely determine the zooplankton community structure when significant environmental gradients exist at fine geographical scales in highly polluted running water ecosystems.</p>
<h1>Functional metagenomics of oil-impacted mangrove sediments reveals high abundance of hydrolases of biotechnological interest.</h1>
<p><a href="https://doi.org/10.1007/S11274-017-2307-5">doi:10.1007/S11274-017-2307-5</a></p>
['Bioprospecting', 'Fosmid library', 'Hydrolases', 'Mangroves', 'Microbial diversity']<p>Mangroves are located in coastal wetlands and are susceptible to the consequences of oil spills, what may threaten the diversity of microorganisms responsible for the nutrient cycling and the consequent ecosystem functioning. Previous reports show that high concentration of oil favors the incidence of epoxide hydrolases and haloalkane dehalogenases in mangroves. This finding has guided the goals of this study in an attempt to broaden the analysis to other hydrolases and thereby verify whether oil contamination interferes with the prevalence of particular hydrolases and their assigned microorganisms. For this, an in-depth survey of the taxonomic and functional microbial diversity recovered in a fosmid library (Library_Oil Mgv) constructed from oil-impacted Brazilian mangrove sediment was carried out. Fosmid DNA of the whole library was extracted and submitted to Illumina HiSeq sequencing. The resulting Library Oil_Mgv dataset was further compared with those obtained by direct sequencing of environmental DNA from Brazilian mangroves (from distinct regions and affected by distinct sources of contamination), focusing on hydrolases with potential use in biotechnological processes. The most abundant hydrolases found were proteases, esterases and amylases, with similar occurrence profile in all datasets. The main microbial groups harboring such hydrolase-encoding genes were distinct in each mangrove, and in the fosmid library these enzymes were mainly assigned to Chloroflexaceae (for amylases), Planctomycetaceae (for esterases) and Bradyrhizobiaceae (for proteases). Assembly and analysis of Library_Oil Mgv reads revealed three potentially novel enzymes, one epoxide hydrolase, one xylanase and one amylase, to be further investigated via heterologous expression assays.</p>
<h1>DNA metabarcoding to assess indoor fungal communities: Electrostatic dust collectors and Illumina sequencing.</h1>
<p><a href="https://doi.org/S0167-7012(17)30124-0">doi:S0167-7012(17)30124-0</a></p>
['Fungal community', 'Illumina sequencing', 'Indoor dust', 'Metabarcoding', 'qPCR']<p>DNA metabarcoding is increasingly being used to characterize the microbiological composition of both the indoor and outdoor environments of dwellings. Our study aimed to evaluate metabarcoding and bioinformatic analysis resulting from calibrated samples and samples collected by an electrostatic dust collector (EDC) in dwellings with no moisture problems. Thus, the fungal communities of 14 dwellings (eastern France, Franche-Comté region) were analyzed by Illumina MiSeq technology after amplification of the ITS2 region. Using the standard samples of 11 species of yeasts and molds allowed us to validate the Operational taxonomic units (OTU) assignment. These calibrated samples also showed a low amplification bias, a low rate of sequencing errors and the semi-quantitative nature of the technique. Only one species from the calibrated samples (Lichtheimia corymbifera) was less amplified probably due to the presence of two mismatches in its3 primer. EDC analysis identified 3594OTU with 75% of reads corresponding to 30 genera. The main genera are those usually found by culture techniques (Penicillium, Aspergillus and Cladosporium), but findings also indicate others less commonly isolated in culture such as Epicoccum, the fourth detected genus in our study. The type of heating systems was correlated with fungal diversity. We found less diversity in the dwellings with wood heating and larger quantities of Epicoccum nigrum verified by qPCR. DNA metabarcoding analysis applied to EDC seems promising. However, we think that it must be used along with qPCR, to obtain a more global view of microbial ecology and relative quantification of species of interest within communities.</p>
<h1>Metabarcoding monitoring analysis: the pros and cons of using co-extracted environmental DNA and RNA data to assess offshore oil production impacts on benthic communities.</h1>
<p><a href="https://doi.org/10.7717/Peerj.3347">doi:10.7717/Peerj.3347</a></p>
['Bacteria (16S)', 'Benthic ecology', 'Biomonitoring', 'Eukaryotes (18S)', 'High-throughput sequencing', 'Method testing', 'Oil and gas activities', 'eDNA', 'eRNA']<p>Sequencing environmental DNA (eDNA) is increasingly being used as an alternative to traditional morphological-based identification to characterize biological assemblages and monitor anthropogenic impacts in marine environments. Most studies only assess eDNA which, compared to eRNA, can persist longer in the environment after cell death. Therefore, eRNA may provide a more immediate census of the environment due to its relatively weaker stability, leading some researchers to advocate for the use of eRNA as an additional, or perhaps superior proxy for portraying ecological changes. A variety of pre-treatment techniques for screening eDNA and eRNA derived operational taxonomic units (OTUs) have been employed prior to statistical analyses, including removing singleton taxa (i.e., OTUs found only once) and discarding those not present in both eDNA and eRNA datasets. In this study, we used bacterial (16S ribosomal RNA gene) and eukaryotic (18S ribosomal RNA gene) eDNA- and eRNA-derived data from benthic communities collected at increasing distances along a transect from an oil production platform (Taranaki, New Zealand). Macro-infauna (visual classification of benthic invertebrates) and physico-chemical data were analyzed in parallel. We tested the effect of removing singleton taxa, and removing taxa not present in the eDNA and eRNA libraries from the same environmental sample (trimmed by shared OTUs), by comparing the impact of the oil production platform on alpha- and beta-diversity of the eDNA/eRNA-based biological assemblages, and by correlating these to the morphologically identified macro-faunal communities and the physico-chemical data. When trimmed by singletons, presence/absence information from eRNA data represented the best proxy to detect changes on species diversity for both bacteria and eukaryotes. However, assessment of quantitative beta-diversity from read abundance information of bacteria eRNA did not, contrary to eDNA, reveal any impact from the oil production activity. Overall, the data appeared more robust when trimmed by shared OTUs, showing a greater effect of the platform on alpha- and beta-diversity. Trimming by shared OTUs likely removes taxa derived from legacy DNA and technical artefacts introduced through reverse transcriptase, polymerase-chain-reaction and sequencing. Findings from our scoping study suggest that metabarcoding-based biomonitoring surveys should, if funds, time and expertise allow, be assessed using both eDNA and eRNA products.</p>
<h1>The importance of standardization for biodiversity comparisons: A case study using autonomous reef monitoring structures (ARMS) and metabarcoding to measure cryptic diversity on Mo'orea coral reefs, French Polynesia.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0175066">doi:10.1371/Journal.Pone.0175066</a></p>
<p>No keywords</p><p>The advancement of metabarcoding techniques, declining costs of high-throughput sequencing and development of systematic sampling devices, such as autonomous reef monitoring structures (ARMS), have provided the means to gather a vast amount of diversity data from cryptic marine communities. However, such increased capability could also lead to analytical challenges if the methods used to examine these communities across local and global scales are not standardized. Here we compare and assess the underlying biases of four ARMS field processing methods, preservation media, and current bioinformatic pipelines in evaluating diversity from cytochrome c oxidase I metabarcoding data. Illustrating the ability of ARMS-based metabarcoding to capture a wide spectrum of biodiversity, 3,372 OTUs and twenty-eight phyla, including 17 of 33 marine metazoan phyla, were detected from 3 ARMS (2.607 m2 area) collected on coral reefs in Mo'orea, French Polynesia. Significant differences were found between processing and preservation methods, demonstrating the need to standardize methods for biodiversity comparisons. We recommend the use of a standardized protocol (NOAA method) combined with DMSO preservation of tissues for sessile macroorganisms because it gave a more accurate representation of the underlying communities, is cost effective and removes chemical restrictions associated with sample transportation. We found that sequences identified at ≥ 97% similarity increased more than 7-fold (5.1% to 38.6%) using a geographically local barcode inventory, highlighting the importance of local species inventories. Phylogenetic approaches that assign higher taxonomic ranks accrued phylum identification errors (9.7%) due to sparse taxonomic coverage of the understudied cryptic coral reef community in public databases. However, a ≥ 85% sequence identity cut-off provided more accurate results (0.7% errors) and enabled phylum level identifications of 86.3% of the sequence reads. With over 1600 ARMS deployed, standardizing methods and improving databases are imperative to provide unprecedented global baseline assessments of understudied cryptic marine species in a rapidly changing world.</p>
<h1>Effective mosquito and arbovirus surveillance using metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12682">doi:10.1111/1755-0998.12682</a></p>
['\ncytochrome c oxidase subunit I\n', 'Culicidae', 'DNA barcoding', 'bulk sample', 'pooled samples', 'virus']<p>Effective vector and arbovirus surveillance requires timely and accurate screening techniques that can be easily upscaled. Next-generation sequencing (NGS) is a high-throughput technology that has the potential to modernize vector surveillance. When combined with DNA barcoding, it is termed 'metabarcoding.' The aim of our study was to establish a metabarcoding protocol to characterize pools of mosquitoes and screen them for virus. Pools contained 100 morphologically identified individuals, including one Ross River virus (RRV) infected mosquito, with three species present at different proportions: 1, 5, 94%. Nucleic acid extracted from both crude homogenate and supernatant was used to amplify a 269-bp section of the mitochondrial cytochrome c oxidase subunit I (COI) locus. Additionally, a 67-bp region of the RRV E2 gene was amplified from synthesized cDNA to screen for RRV. Amplicon sequencing was performed using an Illumina MiSeq, and bioinformatic analysis was performed using a DNA barcode database of Victorian mosquitoes. Metabarcoding successfully detected all mosquito species and RRV in every positive sample tested. The limits of species detection were also examined by screening a pool of 1000 individuals, successfully identifying the species and RRV from a single mosquito. The primers used for amplification, number of PCR cycles and total number of individuals present all have effects on the quantification of species in mixed bulk samples. Based on the results, a number of recommendations for future metabarcoding studies are presented. Overall, metabarcoding shows great promise for providing a new alternative approach to screening large insect surveillance trap catches.</p>
<h1>Sensitivity and accuracy of high-throughput metabarcoding methods for early detection of invasive fish species.</h1>
<p><a href="https://doi.org/10.1038/Srep46393">doi:10.1038/Srep46393</a></p>
<p>No keywords</p><p>High-throughput DNA metabarcoding has gained recognition as a potentially powerful tool for biomonitoring, including early detection of aquatic invasive species (AIS). DNA based techniques are advancing, but our understanding of the limits to detection for metabarcoding complex samples is inadequate. For detecting AIS at an early stage of invasion when the species is rare, accuracy at low detection limits is key. To evaluate the utility of metabarcoding in future fish community monitoring programs, we conducted several experiments to determine the sensitivity and accuracy of routine metabarcoding methods. Experimental mixes used larval fish tissue from multiple "common" species spiked with varying proportions of tissue from an additional "rare" species. Pyrosequencing of genetic marker, COI (cytochrome c oxidase subunit I) and subsequent sequence data analysis provided experimental evidence of low-level detection of the target "rare" species at biomass percentages as low as 0.02% of total sample biomass. Limits to detection varied interspecifically and were susceptible to amplification bias. Moreover, results showed some data processing methods can skew sequence-based biodiversity measurements from corresponding relative biomass abundances and increase false absences. We suggest caution in interpreting presence/absence and relative abundance in larval fish assemblages until metabarcoding methods are optimized for accuracy and precision.</p>
<h1>Does warming by open-top chambers induce change in the root-associated fungal community of the arctic dwarf shrub Cassiope tetragona (Ericaceae)?</h1>
<p><a href="https://doi.org/10.1007/S00572-017-0767-Y">doi:10.1007/S00572-017-0767-Y</a></p>
['Arctic', 'Cassiope tetragona', 'Climate change', 'Ectomycorrhiza', 'Ericoid mycorrhiza', 'High-throughput sequencing', 'OTCs', 'Open-top chambers', 'Root-associated fungi', 'Svalbard']<p>Climate change may alter mycorrhizal communities, which impact ecosystem characteristics such as carbon sequestration processes. These impacts occur at a greater magnitude in Arctic ecosystems, where the climate is warming faster than in lower latitudes. Cassiope tetragona (L.) D. Don is an Arctic plant species in the Ericaceae family with a circumpolar range. C. tetragona has been reported to form ericoid mycorrhizal (ErM) as well as ectomycorrhizal (ECM) symbioses. In this study, the fungal taxa present within roots of C. tetragona plants collected from Svalbard were investigated using DNA metabarcoding. In light of ongoing climate change in the Arctic, the effects of artificial warming by open-top chambers (OTCs) on the fungal root community of C. tetragona were evaluated. We detected only a weak effect of warming by OTCs on the root-associated fungal communities that was masked by the spatial variation between sampling sites. The root fungal community of C. tetragona was dominated by fungal groups in the Basidiomycota traditionally classified as either saprotrophic or ECM symbionts, including the orders Sebacinales and Agaricales and the genera Clavaria, Cortinarius, and Mycena. Only a minor proportion of the operational taxonomic units (OTUs) could be annotated as ErM-forming fungi. This indicates that C. tetragona may be forming mycorrhizal symbioses with typically ECM-forming fungi, although no characteristic ECM root tips were observed. Previous studies have indicated that some saprophytic fungi may also be involved in biotrophic associations, but whether the saprotrophic fungi in the roots of C. tetragona are involved in biotrophic associations remains unclear. The need for more experimental and microscopy-based studies to reveal the nature of the fungal associations in C. tetragona roots is emphasized.</p>
<h1>Patterns of temporal and enemy niche use by a community of leaf cone moths (Caloptilia) coexisting on maples (Acer) as revealed by metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/Mec.14105">doi:10.1111/Mec.14105</a></p>
['closely related species', 'herbivorous insect', 'metabarcoding', 'parasitoid', 'phenology', 'species coexistence']<p>The diversity of herbivorous insects is often considered a function of host plant diversity. However, recent research has uncovered many examples of closely related herbivores using the same host plant(s), suggesting that partitioning of host plants is not the only mechanism generating diversity. Herbivores sharing hosts may utilize different parts of the same plant, but such resource partitioning is often not apparent; hence, the factors that allow closely related herbivores to coexist are still largely undetermined. We examined whether partitioning of phenology or natural enemies may explain the coexistence of leaf cone moths (Caloptilia; Gracillariidae) associated with maples (Acer; Sapindaceae). Larval activity of 10 sympatric Caloptilia species found on nine maple species was monitored every 2-3 weeks for a total of 13 sampling events, and an exhaustive search for internal parasitoid wasps was conducted using high-throughput sequencing. Blocking primers were used to facilitate the detection of wasp larvae inside moth tissue. We found considerable phenological overlap among Caloptilia species, with two clear peaks in July and September-October. Coexisting Caloptilia species also had largely overlapping parasitoid communities; a total of 13 chalcid and ichneumon wasp species attacked Caloptilia in a nonspecific fashion at an overall parasitism rate of 46.4%. Although coexistence may be facilitated by factors not accounted for in this study, it appears that niche partitioning is not necessary for closely related herbivores to stably coexist on shared hosts. Co-occurrence without resource partitioning may provide an additional axis along which herbivorous insects attain increased species richness.</p>
<h1>Taxonomy-free molecular diatom index for high-throughput eDNA biomonitoring.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12668">doi:10.1111/1755-0998.12668</a></p>
['bioindication', 'environmental DNA', 'metabarcoding', 'water quality']<p>Current biodiversity assessment and biomonitoring are largely based on the morphological identification of selected bioindicator taxa. Recently, several attempts have been made to use eDNA metabarcoding as an alternative tool. However, until now, most applied metabarcoding studies have been based on the taxonomic assignment of sequences that provides reference to morphospecies ecology. Usually, only a small portion of metabarcoding data can be used due to a limited reference database and a lack of phylogenetic resolution. Here, we investigate the possibility to overcome these limitations using a taxonomy-free approach that allows the computing of a molecular index directly from eDNA data without any reference to morphotaxonomy. As a case study, we use the benthic diatoms index, commonly used for monitoring the biological quality of rivers and streams. We analysed 87 epilithic samples from Swiss rivers, the ecological status of which was established based on the microscopic identification of diatom species. We compared the diatom index derived from eDNA data obtained with or without taxonomic assignment. Our taxonomy-free approach yields promising results by providing a correct assessment for 77% of examined sites. The main advantage of this method is that almost 95% of OTUs could be used for index calculation, compared to 35% in the case of the taxonomic assignment approach. Its main limitations are under-sampling and the need to calibrate the index based on the microscopic assessment of diatoms communities. However, once calibrated, the taxonomy-free molecular index can be easily standardized and applied in routine biomonitoring, as a complementary tool allowing fast and cost-effective assessment of the biological quality of watercourses.</p>
<h1>Protistan Biogeography: A Snapshot Across a Major Shipping Corridor Spanning Two Oceans.</h1>
<p><a href="https://doi.org/S1434-4610(16)30095-5">doi:S1434-4610(16)30095-5</a></p>
['Panama Canal', 'dispersal', 'metabarcode', 'protistan communities', 'protistan diversity.', 'protistan ecology']<p>Deciphering patterns of protistan taxa is a crucial step for understanding anthropogenic and environmental impacts on biogeography. We characterized and compared protistan communities from environmental samples collected along a major shipping corridor, the Panama Canal, and the Bocas del Toro archipelago. We used metabarcoding with high throughput sequencing (HTS) with the V4 hypervariable region of the ribosomal gene complex (rDNA). We detected many protistan taxa, including a variety of parasitic and toxic taxa. There were 1,296 OTUs shared across all three regions, with an additional 342-1,526 OTUs occurring across two or more regions, suggesting some mixing within the Caribbean and across the Isthmus. In general, this mixing did not impact community similarity, which was primarily distinct across regions. When OTUs identified as gregarines were analyzed separately, most samples grouped by region and communities were distinct across the Canal. Shipping traffic through the Panama Canal could move some taxa across regions; however, different environmental conditions in the two oceans may limit their establishment. Overall our results suggest that contemporary protistan biogeographic patterns are likely caused by a complex combination of factors, including anthropogenic dispersal and environmental tolerance.</p>
<h1>Revegetation rewilds the soil bacterial microbiome of an old field.</h1>
<p><a href="https://doi.org/10.1111/Mec.14081">doi:10.1111/Mec.14081</a></p>
['\neDNA\n', 'anthropocene', 'ecosystem restoration', 'land degradation', 'microbiome', 'next-generation sequencing']<p>Ecological restoration is a globally important and well-financed management intervention used to combat biodiversity declines and land degradation. Most restoration aims to increase biodiversity towards a reference state, but there are concerns that intended outcomes are not reached due to unsuccessful interventions and land-use legacy issues. Monitoring biodiversity recovery is essential to measure success; however, most projects remain insufficiently monitored. Current field-based methods are hard to standardize and are limited in their ability to assess important components of ecosystems, such as bacteria. High-throughput amplicon sequencing of environmental DNA (metabarcoding of eDNA) has been proposed as a cost-effective, scalable and uniform ecological monitoring solution, but its application in restoration remains largely untested. Here we show that metabarcoding of soil eDNA is effective at demonstrating the return of the native bacterial community in an old field following native plant revegetation. Bacterial composition shifted significantly after 8 years of revegetation, where younger sites were more similar to cleared sites and older sites were more similar to remnant stands. Revegetation of the native plant community strongly impacted on the belowground bacterial community, despite the revegetated sites having a long and dramatically altered land-use history (i.e. >100 years grazing). We demonstrate that metabarcoding of eDNA provides an effective way of monitoring changes in bacterial communities that would otherwise go unchecked with conventional monitoring of restoration projects. With further development, awareness of microbial diversity in restoration has significant scope for improving the efficacy of restoration interventions more broadly.</p>
<h1>The best of both worlds: A combined approach for analyzing microalgal diversity via metabarcoding and morphology-based methods.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0172808">doi:10.1371/Journal.Pone.0172808</a></p>
<p>No keywords</p><p>An increasing number of studies use next generation sequencing (NGS) to analyze complex communities, but is the method sensitive enough when it comes to identification and quantification of species? We compared NGS with morphology-based identification methods in an analysis of microalgal (periphyton) communities. We conducted a mesocosm experiment in which we allowed two benthic grazer species to feed upon benthic biofilms, which resulted in altered periphyton communities. Morphology-based identification and 454 (Roche) pyrosequencing of the V4 region in the small ribosomal unit (18S) rDNA gene were used to investigate the community change caused by grazing. Both the NGS-based data and the morphology-based method detected a marked shift in the biofilm composition, though the two methods varied strongly in their abilities to detect and quantify specific taxa, and neither method was able to detect all species in the biofilms. For quantitative analysis, we therefore recommend using both metabarcoding and microscopic identification when assessing the community composition of eukaryotic microorganisms.</p>
<h1>The importance of molecular markers and primer design when characterizing biodiversity from environmental DNA.</h1>
<p><a href="https://doi.org/10.1139/Gen-2016-0100">doi:10.1139/Gen-2016-0100</a></p>
['ADN environnemental', 'PCR', 'barcoding', 'biodiversity', 'biodiversité', 'codage à barres', 'conception d’amorces', 'environmental DNA', 'high-throughput sequencing', 'metabarcoding', 'métacodage à barres', 'primer design', 'qPCR', 'séquençage à haut débit']<p>Environmental DNA (eDNA) comprises DNA fragments that have been shed into the environment by organisms, and which can be extracted from environmental samples such as water or soil. Characterization of eDNA can allow researchers to infer the presence or absence of species from a particular site without the need to locate and identify individuals, and therefore may provide an extremely valuable tool for quantifying biodiversity. However, as is often the case with relatively new protocols, methodological challenges remain. A number of earlier reviews have discussed these challenges, but none have provided extensive treatment of the critical decisions surrounding molecular markers and primer development for use in eDNA assays. This review discusses a number of options and approaches that can be used when determining which primers and gene regions are most appropriate for either targeted species detection or metabarcoding macro-organisms from eDNA. The latter represents a new field that is growing rapidly, and which has the potential to revolutionize future assessments of community and ecosystem diversity.</p>
<h1>Priming of a DNA metabarcoding approach for species identification and inventory in marine macrobenthic communities.</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0220">doi:10.1139/Gen-2015-0220</a></p>
['DNA barcoding', 'bioassessment', 'codage à barres de l’ADN', 'communauté marine macrobenthique', 'high-throughput sequencing', 'marine macrobenthos', 'séquençage à haut débit', 'évaluation de la biodiversité']<p>In marine and estuarine benthic communities, the inventory and estimation of species richness are often hampered by the need for broad taxonomic expertise across several phyla. The use of DNA metabarcoding has emerged as a powerful tool for the fast assessment of species composition in a diversity of ecological communities. Here, we tested the amplification success of five primer sets targeting different COI-5P regions by 454 pyrosequencing to maximize the recovery of two simulated macrobenthic communities containing 21 species (SimCom1 and SimCom 2). Species identification was first performed against a compiled reference library of macrobenthic species. Reads with similarity results to reference sequences between 70% and 97% were then submitted to GenBank and BOLD to attempt the identification of concealed species in the bulk sample. The combination of at least three primer sets was able to recover more species than any primer set alone, achieving 85% of represented species in SimCom1 and 76% in SimCom2. Our approach was successful to detect low-frequency specimens, as well as concealed species, in the bulk sample, indicating the potential for the application of this approach on marine bioassessment and inventory, including the detection of "hidden" biodiversity that would hardly be possible based on morphology only.</p>
<h1>Annual time-series analysis of aqueous eDNA reveals ecologically relevant dynamics of lake ecosystem biodiversity.</h1>
<p><a href="https://doi.org/10.1038/Ncomms14087">doi:10.1038/Ncomms14087</a></p>
<p>No keywords</p><p>The use of environmental DNA (eDNA) in biodiversity assessments offers a step-change in sensitivity, throughput and simultaneous measures of ecosystem diversity and function. There remains, however, a need to examine eDNA persistence in the wild through simultaneous temporal measures of eDNA and biota. Here, we use metabarcoding of two markers of different lengths, derived from an annual time series of aqueous lake eDNA to examine temporal shifts in ecosystem biodiversity and in an ecologically important group of macroinvertebrates (Diptera: Chironomidae). The analyses allow different levels of detection and validation of taxon richness and community composition (β-diversity) through time, with shorter eDNA fragments dominating the eDNA community. Comparisons between eDNA, community DNA, taxonomy and UK species abundance data further show significant relationships between diversity estimates derived across the disparate methodologies. Our results reveal the temporal dynamics of eDNA and validate the utility of eDNA metabarcoding for tracking seasonal diversity at the ecosystem scale.</p>
<h1>DNA Metabarcoding of Amazonian Ichthyoplankton Swarms.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0170009">doi:10.1371/Journal.Pone.0170009</a></p>
<p>No keywords</p><p>Tropical rainforests harbor extraordinary biodiversity. The Amazon basin is thought to hold 30% of all river fish species in the world. Information about the ecology, reproduction, and recruitment of most species is still lacking, thus hampering fisheries management and successful conservation strategies. One of the key understudied issues in the study of population dynamics is recruitment. Fish larval ecology in tropical biomes is still in its infancy owing to identification difficulties. Molecular techniques are very promising tools for the identification of larvae at the species level. However, one of their limits is obtaining individual sequences with large samples of larvae. To facilitate this task, we developed a new method based on the massive parallel sequencing capability of next generation sequencing (NGS) coupled with hybridization capture. We focused on the mitochondrial marker cytochrome oxidase I (COI). The results obtained using the new method were compared with individual larval sequencing. We validated the ability of the method to identify Amazonian catfish larvae at the species level and to estimate the relative abundance of species in batches of larvae. Finally, we applied the method and provided evidence for strong temporal variation in reproductive activity of catfish species in the Ucayalí River in the Peruvian Amazon. This new time and cost effective method enables the acquisition of large datasets, paving the way for a finer understanding of reproductive dynamics and recruitment patterns of tropical fish species, with major implications for fisheries management and conservation.</p>
<h1>Age-related environmental gradients influence invertebrate distribution in the Prince Charles Mountains, East Antarctica.</h1>
<p><a href="https://doi.org/10.1098/Rsos.160296">doi:10.1098/Rsos.160296</a></p>
['Antarctica', 'environmental DNA', 'gradient', 'high-throughput sequencing', 'invertebrates', 'salinity']<p>The potential impact of environmental change on terrestrial Antarctic ecosystems can be explored by inspecting biodiversity patterns across large-scale gradients. Unfortunately, morphology-based surveys of Antarctic invertebrates are time-consuming and limited by the cryptic nature of many taxa. We used biodiversity information derived from high-throughput sequencing (HTS) to elucidate the relationship between soil properties and invertebrate biodiversity in the Prince Charles Mountains, East Antarctica. Across 136 analysed soil samples collected from Mount Menzies, Mawson Escarpment and Lake Terrasovoje, we found invertebrate distribution in the Prince Charles Mountains significantly influenced by soil salinity and/or sulfur content. Phyla Tardigrada and Arachnida occurred predominantly in low-salinity substrates with abundant nutrients, whereas Bdelloidea (Rotifera) and Chromadorea (Nematoda) were more common in highly saline substrates. A significant correlation between invertebrate occurrence, soil salinity and time since deglaciation indicates that terrain age indirectly influences Antarctic terrestrial biodiversity, with more recently deglaciated areas supporting greater diversity. Our study demonstrates the value of HTS metabarcoding to investigate environmental constraints on inconspicuous soil biodiversity across large spatial scales.</p>
<h1>Environmental DNA metabarcoding reveals primary chemical contaminants in freshwater sediments from different land-use types.</h1>
<p><a href="https://doi.org/S0045-6535(16)31858-6">doi:S0045-6535(16)31858-6</a></p>
['Asia', 'Biomonitoring', 'Ecogenomics', 'Next generation sequencing', 'Protozoa', 'Rivers']<p>Land-use intensification threatens freshwater biodiversity. Freshwater eukaryotic communities are affected by multiple chemical contaminants with a land-use specific manner. However, biodiversities of eukaryotes and their associations with multiple chemical contaminants are largely unknown. This study characterized in situ eukaryotic communities in sediments exposed to mixtures of chemical contaminants and assessed relationships between various environmental variables and eukaryotic communities in sediments from the Nanfei River. Eukaryotic communities in the sediment samples were dominated by Annelida, Arthropoda, Rotifera, Ochrophyta, Chlorophyta and Ciliophora. Alpha-diversities (Shannon entropy) and structures of eukaryotic communities were significantly different between land-use types. According to the results of multiple statistical tests (PCoA, distLM, Mantel and network analysis), dissimilarity of eukaryotic community structures revealed the key effects of pyrethroid insecticides, manganese, zinc, lead, chromium and polycyclic aromatic hydrocarbons (PAHs) on eukaryotic communities in the sediment samples from the Nanfei River. Furthermore, taxa associated with land-use types were identified and several sensitive eukaryotic taxa to some of the primary contaminants were identified as potential indicators to monitor effects of the primary chemical contaminants. Overall, environmental DNA metabarcoding on in situ eukaryotic communities provided a powerful tool for biomonitoring and identifying primary contaminants and their complex effects on benthic eukaryotic communities in freshwater sediments.</p>
<h1>DNA metabarcoding reveals that 200-μm-size-fractionated filtering is unable to discriminate between planktonic microbial and large eukaryotes.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12652">doi:10.1111/1755-0998.12652</a></p>
['\neDNA\n', 'eukaryotic planktonic community', 'high-throughput sequencing', 'size-fractionated filtering', 'subtropical reservoir']<p>Microeukaryotic plankton (0.2-200 μm) are critical components of aquatic ecosystems and key players in global ecological processes. High-throughput sequencing is currently revolutionizing their study on an unprecedented scale. However, it is currently unclear whether we can accurately, effectively and quantitatively depict the microeukaryotic plankton communities using traditional size-fractionated filtering combined with molecular methods. To address this, we analysed the eukaryotic plankton communities both with, and without, prefiltering with a 200 μm pore-size sieve -by using SSU rDNA-based high-throughput sequencing on 16 samples with three replicates in each sample from two subtropical reservoirs sampled from January to October in 2013. We found that ~25% reads were classified as metazoan in both size groups. The species richness, alpha and beta diversity of plankton community and relative abundance of reads in 99.2% eukaryotic OTUs showed no significant changes after prefiltering with a 200 μm pore-size sieve. We further found that both >0.2 μm and 0.2-200 μm eukaryotic plankton communities, especially the abundant plankton subcommunities, exhibited very similar, and synchronous, spatiotemporal patterns and processes associated with almost identical environmental drivers. The lack of an effect on community structure from prefiltering suggests that environmental DNA from larger metazoa is introduced into the smaller size class. Therefore, size-fractionated filtering with 200 μm is insufficient to discriminate between the eukaryotic plankton size groups in metabarcoding approaches. Our results also highlight the importance of sequencing depth, and strict quality filtering of reads, when designing studies to characterize microeukaryotic plankton communities.</p>
<h1>Recommendations for developing and applying genetic tools to assess and manage biological invasions in marine ecosystems.</h1>
<p><a href="https://doi.org/10.1016/J.Marpol.2017.08.014">doi:10.1016/J.Marpol.2017.08.014</a></p>
['Early detection', 'Environmental DNA', 'Good environmental status', 'High throughput sequencing', 'Marine Strategy Framework Directive', 'Marine invasive species', 'Metabarcoding', 'Monitoring', 'Surveillance']<p>The European Union's Marine Strategy Framework Directive (MSFD) aims to adopt integrated ecosystem management approaches to achieve or maintain "Good Environmental Status" for marine waters, habitats and resources, including mitigation of the negative effects of non-indigenous species (NIS). The Directive further seeks to promote broadly standardized monitoring efforts and assessment of temporal trends in marine ecosystem condition, incorporating metrics describing the distribution and impacts of NIS. Accomplishing these goals will require application of advanced tools for NIS surveillance and risk assessment, particularly given known challenges associated with surveying and monitoring with traditional methods. In the past decade, a host of methods based on nucleic acids (DNA and RNA) analysis have been developed or advanced that promise to dramatically enhance capacity in assessing and managing NIS. However, ensuring that these rapidly evolving approaches remain accessible and responsive to the needs of resource managers remains a challenge. This paper provides recommendations for future development of these genetic tools for assessment and management of NIS in marine systems, within the context of the explicit requirements of the MSFD. Issues considered include technological innovation, methodological standardization, data sharing and collaboration, and the critical importance of shared foundational resources, particularly integrated taxonomic expertise. Though the recommendations offered here are not exhaustive, they provide a basis for future intentional (and international) collaborative development of a genetic toolkit for NIS research, capable of fulfilling the immediate and long term goals of marine ecosystem and resource conservation.</p>
<h1>eDNA metabarcoding: a promising method for anuran surveys in highly diverse tropical forests.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12643">doi:10.1111/1755-0998.12643</a></p>
['12S mitochondrial gene', 'amphibians', 'biodiversity', 'monitoring methods', 'next-generation sequencing', 'site occupancy-detection modelling']<p>Understanding the geographical distribution and community composition of species is crucial to monitor species persistence and define effective conservation strategies. Environmental DNA (eDNA) has emerged as a powerful noninvasive tool for species detection. However, most eDNA survey methods have been developed and applied in temperate zones. We tested the feasibility of using eDNA to survey anurans in tropical streams in the Brazilian Atlantic forest and compared the results with short-term visual and audio surveys. We detected all nine species known to inhabit our focal streams with one single visit for eDNA sampling. We found a higher proportion of sequence reads and larger number of positive PCR replicates for more common species and for those with life cycles closely associated with the streams, factors that may contribute to increased release of DNA in the water. However, less common species were also detected in eDNA samples, demonstrating the detection power of this method. Filtering larger volumes of water resulted in a higher probability of detection. Our data also show it is important to sample multiple sites along streams, particularly for detection of target species with lower population densities. For the three focal species in our study, the eDNA metabarcoding method had a greater capacity of detection per sampling event than our rapid field surveys, and thus, has the potential to circumvent some of the challenges associated with traditional approaches. Our results underscore the utility of eDNA metabarcoding as an efficient method to survey anuran species in tropical streams of the highly biodiverse Brazilian Atlantic forest.</p>
<h1>Vertical Structure of Phyllosphere Fungal Communities in a Tropical Forest in Thailand Uncovered by High-Throughput Sequencing.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0166669">doi:10.1371/Journal.Pone.0166669</a></p>
<p>No keywords</p><p>Phyllosphere fungi harbor a tremendous species diversity and play important ecological roles. However, little is known about their distribution patterns within forest ecosystems. We examined how species diversity and community composition of phyllosphere fungi change along a vertical structure in a tropical forest in Thailand. Fungal communities in 144 leaf samples from 19 vertical layers (1.28-34.4 m above ground) of 73 plant individuals (27 species) were investigated by metabarcoding analysis using Ion Torrent sequencing. In total, 1,524 fungal operational taxonomic units (OTUs) were detected among 890,710 reads obtained from the 144 leaf samples. Taxonomically diverse fungi belonging to as many as 24 orders of Ascomycota and 21 orders of Basidiomycota were detected, most of which inhabited limited parts of the lowest layers closest to the forest floor. Species diversity of phyllosphere fungi was the highest in the lowest layers closest to the forest floor, decreased with increasing height, and lowest in the canopy; 742 and 55 fungal OTUs were detected at the lowest and highest layer, respectively. On the layers close to the forest floor, phyllosphere fungal communities were mainly composed of low frequency OTUs and largely differentiated among plant individuals. Conversely, in the canopy, fungal communities consisted of similar OTUs across plant individuals, and as many as 86.1%-92.7% of the OTUs found in the canopy (≥22 m above ground) were also distributed in the lower layers. Overall, our study showed the variability of phyllosphere fungal communities along the vertical gradient of plant vegetation and environmental conditions, suggesting the significance of biotic and abiotic variation for the species diversity of phyllosphere fungi.</p>
<h1>16S rRNA Amplicon Sequencing for Epidemiological Surveys of Bacteria in Wildlife.</h1>
<p><a href="https://doi.org/E00032-16">doi:E00032-16</a></p>
['West Africa', 'bacteria', 'emerging infectious diseases', 'high-throughput sequencing', 'metabarcoding', 'molecular epidemiology', 'next-generation sequencing', 'rodents', 'zoonoses']<p>The human impact on natural habitats is increasing the complexity of human-wildlife interactions and leading to the emergence of infectious diseases worldwide. Highly successful synanthropic wildlife species, such as rodents, will undoubtedly play an increasingly important role in transmitting zoonotic diseases. We investigated the potential for recent developments in 16S rRNA amplicon sequencing to facilitate the multiplexing of the large numbers of samples needed to improve our understanding of the risk of zoonotic disease transmission posed by urban rodents in West Africa. In addition to listing pathogenic bacteria in wild populations, as in other high-throughput sequencing (HTS) studies, our approach can estimate essential parameters for studies of zoonotic risk, such as prevalence and patterns of coinfection within individual hosts. However, the estimation of these parameters requires cleaning of the raw data to mitigate the biases generated by HTS methods. We present here an extensive review of these biases and of their consequences, and we propose a comprehensive trimming strategy for managing these biases. We demonstrated the application of this strategy using 711 commensal rodents, including 208 <i>Mus musculus domesticus</i>, 189 <i>Rattus rattus</i>, 93 <i>Mastomys natalensis</i>, and 221 <i>Mastomys erythroleucus</i>, collected from 24 villages in Senegal. Seven major genera of pathogenic bacteria were detected in their spleens: <i>Borrelia</i>, <i>Bartonella</i>, <i>Mycoplasma</i>, <i>Ehrlichia</i>, <i>Rickettsia</i>, <i>Streptobacillus</i>, and <i>Orientia</i>. <i>Mycoplasma</i>, <i>Ehrlichia</i>, <i>Rickettsia</i>, <i>Streptobacillus</i>, and <i>Orientia</i> have never before been detected in West African rodents. Bacterial prevalence ranged from 0% to 90% of individuals per site, depending on the bacterial taxon, rodent species, and site considered, and 26% of rodents displayed coinfection. The 16S rRNA amplicon sequencing strategy presented here has the advantage over other molecular surveillance tools of dealing with a large spectrum of bacterial pathogens without requiring assumptions about their presence in the samples. This approach is therefore particularly suitable to continuous pathogen surveillance in the context of disease-monitoring programs. <b>IMPORTANCE</b> Several recent public health crises have shown that the surveillance of zoonotic agents in wildlife is important to prevent pandemic risks. High-throughput sequencing (HTS) technologies are potentially useful for this surveillance, but rigorous experimental processes are required for the use of these effective tools in such epidemiological contexts. In particular, HTS introduces biases into the raw data set that might lead to incorrect interpretations. We describe here a procedure for cleaning data before estimating reliable biological parameters, such as positivity, prevalence, and coinfection, using 16S rRNA amplicon sequencing on an Illumina MiSeq platform. This procedure, applied to 711 rodents collected in West Africa, detected several zoonotic bacterial species, including some at high prevalence, despite their never before having been reported for West Africa. In the future, this approach could be adapted for the monitoring of other microbes such as protists, fungi, and even viruses.</p>
<h1>Diversity and temporal patterns of planktonic protist assemblages at a Mediterranean Long Term Ecological Research site.</h1>
<p><a href="https://doi.org/Fiw200">doi:Fiw200</a></p>
['HTS', 'LTER', 'marine protists', 'metabarcoding', 'microbial eukaryote diversity', 'temporal variability']<p>We tracked temporal changes in protist diversity at the Long Term Ecological Research (LTER) station MareChiara in the Gulf of Naples (Mediterranean Sea) on eight dates in 2011 using a metabarcoding approach. Illumina analysis of the V4 and V9 fragments of the 18S rDNA produced 869 522 and 1 410 071 sequences resulting in 6517 and 6519 OTUs, respectively. Marked compositional variations were recorded across the year, with less than 2% of OTUs shared among all samples and similar patterns for the two marker tags. Alveolata, Stramenopiles and Rhizaria were the most represented groups. A comparison with light microscopy data indicated an over-representation of Dinophyta in the sequence dataset, whereas Bacillariophyta showed comparable taxonomic patterns between sequence and light microscopy data. Shannon diversity values were stable from February to September, increasing thereafter with a peak in December. Community variance was mainly explained by seasonality (as temperature), trophic status (as chlorophyll a), and influence of coastal waters (as salinity). Overall, the background knowledge of the system provided a sound context for the result interpretation, showing that LTER sites provide an ideal setting for high-throughput sequencing (HTS) metabarcoding characterisation of protist assemblages and their relationships with environmental variations.</p>
<h1>One step forwards for the routine use of high-throughput DNA sequencing in environmental monitoring. An efficient and standardizable method to maximize the detection of environmental bacteria.</h1>
<p><a href="https://doi.org/10.1002/Mbo3.421">doi:10.1002/Mbo3.421</a></p>
['\nqPCR\n', 'DNA extraction', 'complex matrix', 'environmental bacteria', 'high-throughput sequencing', 'next-generation sequencing', 'tangential flow concentration', 'water']<p>We propose an innovative, repeatable, and reliable experimental workflow to concentrate and detect environmental bacteria in drinking water using molecular techniques. We first concentrated bacteria in water samples using tangential flow filtration and then we evaluated two methods of environmental DNA extraction. We performed tests on both artificially contaminated water samples and real drinking water samples. The efficiency of the experimental workflow was measured through qPCR. The successful applicability of the high-throughput DNA sequencing (HTS) approach was demonstrated on drinking water samples. Our results demonstrate the feasibility of our approach in high-throughput-based studies, and we suggest incorporating it in monitoring strategies to have a better representation of the microbial community. In the recent years, HTS techniques have become key tools in the study of microbial communities. To make the leap from academic laboratories to the routine monitoring (e.g., water treatment plants laboratories), we here propose an experimental workflow suitable for the introduction of HTS as a standard method for detecting environmental bacteria.</p>
<h1>Assessing myxozoan presence and diversity using environmental DNA.</h1>
<p><a href="https://doi.org/S0020-7519(16)30195-3">doi:S0020-7519(16)30195-3</a></p>
['Custom barcoding', 'Faecal DNA', 'Index misassignment', 'Lineage-specific primers', 'Myxosporea', 'Myxozoa', 'Piscivore faeces', 'eDNA']<p>Amplicon sequencing on a High Throughput Sequencing platform (custom barcoding) was used to detect and characterise myxosporean communities in environmental DNA samples from marine and freshwater environments and in faeces of animals that may serve as hosts or whose prey may host myxosporean infections. A diversity of myxozoans in filtered water samples and in faeces of piscivores (otters and great cormorants) was detected, demonstrating the suitability of lineage-specific amplicons for characterising otherwise difficult to sample parasite communities. The importance of using this approach was highlighted by the lack of myxosporean detection using commonly employed, broadly targeted eukaryotic primers. These results suggest that, despite being frequently present in eDNA samples, myxozoans have been generally overlooked in "eukaryote-wide" surveys. Lineage-specific primers, in contrast, detected 107 operational taxonomic units that were assigned to both the "freshwater" and "marine" myxosporean lineages. Only 7% of these OTUs clustered with sequences in GenBank, providing evidence for substantial undescribed myxosporean diversity. Many new operational taxonomic units, including those found in otter faeces, clustered with a clade of myxosporeans previously characterised by sequences from invertebrate hosts and water samples only. Because myxozoan species identification is heavily reliant on molecular signatures, lineage-specific amplicon sequencing offers an effective and non-destructive means of improving our knowledge of myxozoan diversity. In addition, the analysis of myxozoan DNA in faeces of piscivores offers a potentially efficient method of sampling for diversity and revealing life cycles as piscivore activities may integrate myxozoan infections in fish over relatively broad spatial scales.</p>
<h1>Barcoding the food chain: from Sanger to high-throughput sequencing.</h1>
<p>No DOI</p>['DNA barcoding', 'agriculture', 'agro-ecosystems', 'agro-écosystèmes', 'codage à barres de l’ADN', 'food', 'metabarcoding', 'métacodage à barres', 'nourriture']<p>Society faces the complex challenge of supporting biodiversity and ecosystem functioning, while ensuring food security by providing safe traceable food through an ever-more-complex global food chain. The increase in human mobility brings the added threat of pests, parasites, and invaders that further complicate our agro-industrial efforts. DNA barcoding technologies allow researchers to identify both individual species, and, when combined with universal primers and high-throughput sequencing techniques, the diversity within mixed samples (metabarcoding). These tools are already being employed to detect market substitutions, trace pests through the forensic evaluation of trace "environmental DNA", and to track parasitic infections in livestock. The potential of DNA barcoding to contribute to increased security of the food chain is clear, but challenges remain in regulation and the need for validation of experimental analysis. Here, we present an overview of the current uses and challenges of applied DNA barcoding in agriculture, from agro-ecosystems within farmland to the kitchen table.</p>
<h1>Long-term dynamics in microbial eukaryotes communities: a palaeolimnological view based on sedimentary DNA.</h1>
<p><a href="https://doi.org/10.1111/Mec.13893">doi:10.1111/Mec.13893</a></p>
['\neDNA\n', 'climate change', 'eutrophication', 'lake', 'protists']<p>Assessing the extent to which changes in lacustrine biodiversity are affected by anthropogenic or climatic forces requires extensive palaeolimnological data. We used high-throughput sequencing to generate time-series data encompassing over 2200 years of microbial eukaryotes (protists and Fungi) diversity changes from the sedimentary DNA record of two lakes (Lake Bourget in French Alps and Lake Igaliku in Greenland). From 176 samples, we sequenced a large diversity of microbial eukaryotes, with a total 16 386 operational taxonomic units distributed within 50 phylogenetic groups. Thus, microbial groups, such as Chlorophyta, Dinophyceae, Haptophyceae and Ciliophora, that were not previously considered in lacustrine sediment record analyses appeared to be potential biological markers of trophic status changes. Our data suggest that shifts in relative abundance of extant species, including shifts between rare and abundant taxa, drive ecosystem responses to local and global environmental changes. Community structure shift events were concomitant with major climate variations (more particularly in Lake Igaliku). However, this study shows that the impacts of climatic fluctuations may be overpassed by the high-magnitude eutrophication impacts, as observed in the eutrophicated Lake Bourget. Overall, our data show that DNA preserved in sediment constitutes a precious archive of information on past biodiversity changes.</p>
<h1>Fungal diversity and seasonal succession in ash leaves infected by the invasive ascomycete Hymenoscyphus fraxineus.</h1>
<p><a href="https://doi.org/10.1111/Nph.14204">doi:10.1111/Nph.14204</a></p>
['\nHymenoscyphus fraxineus\n', 'ash dieback', 'indigenous fungi', 'internal transcribed spacer (ITS)', 'invasive pathogens', 'metabarcoding']<p>High biodiversity is regarded as a barrier against biological invasions. We hypothesized that the invasion success of the pathogenic ascomycete Hymenoscyphus fraxineus threatening common ash in Europe relates to differences in dispersal and colonization success between the invader and the diverse native competitors. Ash leaf mycobiome was monitored by high-throughput sequencing of the fungal internal transcribed spacer region (ITS) and quantitative PCR profiling of H. fraxineus DNA. Initiation of ascospore production by H. fraxineus after overwintering was followed by pathogen accumulation in asymptomatic leaves. The induction of necrotic leaf lesions coincided with escalation of H. fraxineus DNA levels and changes in proportion of biotrophs, followed by an increase of ubiquitous endophytes with pathogenic potential. H. fraxineus uses high propagule pressure to establish in leaves as quiescent thalli that switch to pathogenic mode once these thalli reach a certain threshold - the massive feedback from the saprophytic phase enables this fungus to challenge host defenses and the resident competitors in mid-season when their density in host tissues is still low. Despite the general correspondence between the ITS-1 and ITS-2 datasets, marker biases were observed, which suggests that multiple barcodes provide better overall representation of mycobiomes.</p>
<h1>Enhancement of the Knowledge on Fungal Communities in Directly Brined Aloreña de Málaga Green Olive Fermentations by Metabarcoding Analysis.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0163135">doi:10.1371/Journal.Pone.0163135</a></p>
<p>No keywords</p><p>Nowadays, our knowledge of the fungal biodiversity in fermented vegetables is limited although these microorganisms could have a great influence on the quality and safety of this kind of food. This work uses a metagenetic approach to obtain basic knowledge of the fungal community ecology during the course of fermentation of natural Aloreña de Málaga table olives, from reception of raw material to edible fruits. For this purpose, samples of brines and fruits were collected from two industries in Guadalhorce Valley (Málaga, Spain) at different moments of fermentation (0, 7, 30 and 120 days). The physicochemical and microbial counts performed during fermentation showed the typical evolution of this type of processes, mainly dominated by yeasts in apparent absence of Enterobacteriaceae and Lactobacillaceae. High-throughput barcoded pyrosequencing analysis of ITS1-5.8S-ITS2 region showed a low biodiversity of the fungal community, with the presence at 97% identity of 29 different fungal genera included in 105 operational taxonomic units (OTUs). The most important genera in the raw material at the moment of reception in the industry were Penicillium, Cladosporium, Malassezia, and Candida, whilst after 4 months of fermentation in brines Zygotorulaspora and Pichia were predominant, whereas in fruits were Candida, Penicillium, Debaryomyces and Saccharomyces. The fungal genera Penicillium, Pichia, and Zygotorulaspora were shared among the three types of substrates during all the course of fermentation, representing the core fungal population for this table olive specialty. A phylogenetic analysis of the ITS sequences allowed a more accurate assignment of diverse OTUs to Pichia manshurica, Candida parapsilosis/C. tropicalis, Candida diddensiae, and Citeromyces nyonensis clades. This study highlights the existence of a complex fungal consortium in olive fermentations including phytopathogenic, saprofitic, spoilage and fermentative genera. Insights into the ecology, identification and quantification of fungi species in olive fermentation will facilitate the design of new strategies to improve the quality and safety of this fermented vegetable. </p>
<h1>A quantitative protocol for DNA metabarcoding of springtails (Collembola).</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0228">doi:10.1139/Gen-2015-0228</a></p>
['16S', 'COX1', 'Collembola', 'collemboles', 'metabarcoding', 'métacodage à barres', 'quantification']<p>We developed a novel protocol with superior quantitative analysis results for DNA metabarcoding of Collembola, a major soil microarthropod order. Degenerate PCR primers were designed for conserved regions in the mitochondrial cytochrome c oxidase subunit I (mtCOI) and 16S ribosomal RNA (mt16S) genes based on published collembolan mitogenomes. The best primer pair was selected based on its ability to amplify each gene, irrespective of the species. DNA was extracted from 10 natural communities sampled in a temperate forest (with typically 25-30 collembolan species per 10 soil samples) and 10 mock communities (with seven cultured collembolan species). The two gene regions were then amplified using the selected primers, ligated with adapters for 454 technology, and sequenced. Examination of the natural community samples showed that 32 and 36 operational taxonomic units (defined at a 90% sequence similarity threshold) were recovered from the mtCOI and mt16S data, respectively, which were comparable to the results of the microscopic identification of 25 morphospecies. Further, sequence abundances for each collembolan species from the mtCOI and mt16S data of the mock communities, after normalization by using a species as the internal control, showed good correlation with the number of individuals in the samples (R = 0.91-0.99), although relative species abundances within a mock community sample estimated from sequences were skewed from community composition in terms of the number of individuals or biomass of the species. Thus, this protocol enables the comparison of collembolan communities in a quantitative manner by metabarcoding. </p>
<h1>First evaluation of foraminiferal metabarcoding for monitoring environmental impact from an offshore oil drilling site.</h1>
<p><a href="https://doi.org/10.1016/J.Marenvres.2016.08.009">doi:10.1016/J.Marenvres.2016.08.009</a></p>
['Benthic ecology', 'Biomonitoring', 'Foraminifera', 'High-throughput sequencing', 'Metabarcoding', 'Oil and gas drilling operations']<p>At present, environmental impacts from offshore oil and gas activities are partly determined by measuring changes in macrofauna diversity. Morphological identification of macrofauna is time-consuming, expensive and dependent on taxonomic expertise. In this study, we evaluated the applicability of using foraminiferal-specific metabarcoding for routine monitoring. Sediment samples were collected along distance gradients from two oil platforms off Taranaki (New Zealand) and their physico-chemical properties, foraminiferal environmental DNA/RNA, and macrofaunal composition analyzed. Macrofaunal and foraminiferal assemblages showed similar shifts along impact gradients, but responded differently to environmental perturbations. Macrofauna were affected by hypoxia, whereas sediment grain size appeared to drive shifts in foraminifera. We identified eight foraminiferal molecular operational taxonomic units that have potential to be used as bioindicator taxa. Our results show that metabarcoding represents an effective tool for assessing foraminiferal communities near offshore oil and gas platforms, and that it can be used to complement current monitoring techniques. </p>
<h1>Deep COI sequencing of standardized benthic samples unveils overlooked diversity of Jordanian coral reefs in the northern Red Sea.</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0208">doi:10.1139/Gen-2015-0208</a></p>
['Autonomous Reef Monitoring Structures', 'COI', 'DNA barcoding', 'Golfe d’Aqaba', 'Gulf of Aqaba', 'codage à barres de l’ADN', 'metabarcoding', 'métacodage à barres', 'structures autonomes de suivi des récifs']<p>High-throughput sequencing (HTS) of DNA barcodes (metabarcoding), particularly when combined with standardized sampling protocols, is one of the most promising approaches for censusing overlooked cryptic invertebrate communities. We present biodiversity estimates based on sequencing of the cytochrome c oxidase subunit 1 (COI) gene for coral reefs of the Gulf of Aqaba, a semi-enclosed system in the northern Red Sea. Samples were obtained from standardized sampling devices (Autonomous Reef Monitoring Structures (ARMS)) deployed for 18 months. DNA barcoding of non-sessile specimens >2 mm revealed 83 OTUs in six phyla, of which only 25% matched a reference sequence in public databases. Metabarcoding of the 2 mm - 500 μm and sessile bulk fractions revealed 1197 OTUs in 15 animal phyla, of which only 4.9% matched reference barcodes. These results highlight the scarcity of COI data for cryptobenthic organisms of the Red Sea. Compared with data obtained using similar methods, our results suggest that Gulf of Aqaba reefs are less diverse than two Pacific coral reefs but much more diverse than an Atlantic oyster reef at a similar latitude. The standardized approaches used here show promise for establishing baseline data on biodiversity, monitoring the impacts of environmental change, and quantifying patterns of diversity at regional and global scales. </p>
<h1>Mycamoeba gemmipara nov. gen., nov. sp., the First Cultured Member of the Environmental Dermamoebidae Clade LKM74 and its Unusual Life Cycle.</h1>
<p><a href="https://doi.org/10.1111/Jeu.12357">doi:10.1111/Jeu.12357</a></p>
['Budding', 'Discosea', 'Longamoebia', 'eukaryotic diversity', 'fungus', 'high throughput sequencing', 'ribosomal genes', 'serial dilution', 'yeast']<p>Since the first environmental DNA surveys, entire groups of sequences called "environmental clades" did not have any cultured representative. LKM74 is an amoebozoan clade affiliated to Dermamoebidae, whose presence is pervasively reported in soil and freshwater. We obtained an isolate from soil that we assigned to LKM74 by molecular phylogeny, close related to freshwater clones. We described Mycamoeba gemmipara based on observations made with light- and transmission electron microscopy. It is an extremely small amoeba with typical lingulate shape. Unlike other Dermamoebidae, it lacked ornamentation on its cell membrane, and condensed chromatin formed characteristic patterns in the nucleus. M. gemmipara displayed a unique life cycle: trophozoites formed walled coccoid stages which grew through successive buddings and developed into branched structures holding cysts. These structures, measuring hundreds of micrometres, are built as the exclusive product of osmotrophic feeding. To demonstrate that M. gemmipara is a genuine soil inhabitant, we screened its presence in an environmental soil DNA diversity survey performed on an experimental setup where pig cadavers were left to decompose in soils to follow changes in eukaryotic communities. Mycamoeba gemmipara was present in all samples, although related reads were uncommon underneath the cadaver.</p>
<h1>Real-time PCR detection of Didemnum perlucidum (Monniot, 1983) and Didemnum vexillum (Kott, 2002) in an applied routine marine biosecurity context.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12581">doi:10.1111/1755-0998.12581</a></p>
['\nDidemnum\n', 'environmental DNA', 'introduced species', 'invasive', 'marine biosecurity monitoring', 'pests', 'real-time PCR']<p>Prevention and early detection are well recognized as the best strategies for minimizing the risks posed by nonindigenous species (NIS) that have the potential to become marine pests. Central to this is the ability to rapidly and accurately identify the presence of NIS, often from complex environmental samples like biofouling and ballast water. Molecular tools have been increasingly applied to assist with the identification of NIS and can prove particularly useful for taxonomically difficult groups like ascidians. In this study, we have developed real-time PCR assays suited to the specific identification of the ascidians Didemnum perlucidum and Didemnum vexillum. Despite being recognized as important global pests, this is the first time specific molecular detection methods have been developed that can support the early identification and detection of these species from a broad range of environmental sample types. These fast, robust and high-throughput assays represent powerful tools for routine marine biosecurity surveillance, as detection and confirmation of the early presence of species could assist in the timely establishment of emergency responses and control strategies. This study applied the developed assays to confirm the ability to detect Didemnid eDNA in water samples. While previous work has focused on detection of marine larvae from water samples, the development of real-time PCR assays specifically aimed at detecting eDNA of sessile invertebrate species in the marine environment represents a world first and a significant step forwards in applied marine biosecurity surveillance. Demonstrated success in the detection of D. perlucidum eDNA from water samples at sites where it could not be visually identified suggests value in incorporating such assays into biosecurity survey designs targeting Didemnid species.</p>
<h1>Scaling up discovery of hidden diversity in fungi: impacts of barcoding approaches.</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0336">doi:10.1098/Rstb.2015.0336</a></p>
['barcoding', 'diversity', 'fungi', 'internal transcribed spacer', 'metabarcoding', 'taxonomy']<p>The fungal kingdom is a hyperdiverse group of multicellular eukaryotes with profound impacts on human society and ecosystem function. The challenge of documenting and describing fungal diversity is exacerbated by their typically cryptic nature, their ability to produce seemingly unrelated morphologies from a single individual and their similarity in appearance to distantly related taxa. This multiplicity of hurdles resulted in the early adoption of DNA-based comparisons to study fungal diversity, including linking curated DNA sequence data to expertly identified voucher specimens. DNA-barcoding approaches in fungi were first applied in specimen-based studies for identification and discovery of taxonomic diversity, but are now widely deployed for community characterization based on sequencing of environmental samples. Collectively, fungal barcoding approaches have yielded important advances across biological scales and research applications, from taxonomic, ecological, industrial and health perspectives. A major outstanding issue is the growing problem of 'sequences without names' that are somewhat uncoupled from the traditional framework of fungal classification based on morphology and preserved specimens. This review summarizes some of the most significant impacts of fungal barcoding, its limitations, and progress towards the challenge of effective utilization of the exponentially growing volume of data gathered from high-throughput sequencing technologies.This article is part of the themed issue 'From DNA barcodes to biomes'.</p>
<h1>Censusing marine eukaryotic diversity in the twenty-first century.</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0331">doi:10.1098/Rstb.2015.0331</a></p>
['18S', 'meiofauna', 'metabarcoding', 'plankton', 'protist']<p>The ocean constitutes one of the vastest and richest biomes on our planet. Most recent estimations, all based on indirect approaches, suggest that there are millions of marine eukaryotic species. Moreover, a large majority of these are small (less than 1 mm), cryptic and still unknown to science. However, this knowledge gap, caused by the lack of diagnostic morphological features in small organisms and the limited sampling of the global ocean, is currently being filled, thanks to new DNA-based approaches. The molecular technique of PCR amplification of homologous gene regions combined with high-throughput sequencing, routinely used to census unculturable prokaryotes, is now also being used to characterize whole communities of marine eukaryotes. Here, we review how this methodological advancement has helped to better quantify the magnitude and patterns of marine eukaryotic diversity, with an emphasis on taxonomic groups previously largely overlooked. We then discuss obstacles remaining to achieve a global understanding of marine eukaryotic diversity. In particular, we argue that 18S variable regions do not provide sufficient taxonomic resolution to census marine life, and suggest combining broad eukaryotic surveys targeting the 18S rRNA region with more taxon-focused analyses of hypervariable regions to improve our understanding of the diversity of species, the functional units of marine ecosystems.This article is part of the themed issue 'From DNA barcodes to biomes'.</p>
<h1>A new way to contemplate Darwin's tangled bank: how DNA barcodes are reconnecting biodiversity science and biomonitoring.</h1>
<p><a href="https://doi.org/10.1098/Rstb.2015.0330">doi:10.1098/Rstb.2015.0330</a></p>
['DNA barcoding', 'biodiversity', 'environment', 'genomics', 'phylogenetics', 'taxonomy']<p>Encompassing the breadth of biodiversity in biomonitoring programmes has been frustrated by an inability to simultaneously identify large numbers of species accurately and in a timely fashion. Biomonitoring infers the state of an ecosystem from samples collected and identified using the best available taxonomic knowledge. The advent of DNA barcoding has now given way to the extraction of bulk DNA from mixed samples of organisms in environmental samples through the development of high-throughput sequencing (HTS). This DNA metabarcoding approach allows an unprecedented view of the true breadth and depth of biodiversity, but its adoption poses two important challenges. First, bioinformatics techniques must simultaneously perform complex analyses of large datasets and translate the results of these analyses to a range of users. Second, the insights gained from HTS need to be amalgamated with concepts such as Linnaean taxonomy and indicator species, which are less comprehensive but more intuitive. It is clear that we are moving beyond proof-of-concept studies to address the challenge of implementation of this new approach for environmental monitoring and regulation. Interpreting Darwin's 'tangled bank' through a DNA lens is now a reality, but the question remains: how can this information be generated and used reliably, and how does it relate to accepted norms in ecosystem study?This article is part of the themed issue 'From DNA barcodes to biomes'.</p>
<h1>Plant diversity accurately predicts insect diversity in two tropical landscapes.</h1>
<p><a href="https://doi.org/10.1111/Mec.13770">doi:10.1111/Mec.13770</a></p>
['Arthropoda', 'biodiversity', 'biomonitoring', 'host specificity', 'insect-plant interactions', 'surrogate species']<p>Plant diversity surely determines arthropod diversity, but only moderate correlations between arthropod and plant species richness had been observed until Basset et al. (Science, 338, 2012 and 1481) finally undertook an unprecedentedly comprehensive sampling of a tropical forest and demonstrated that plant species richness could indeed accurately predict arthropod species richness. We now require a high-throughput pipeline to operationalize this result so that we can (i) test competing explanations for tropical arthropod megadiversity, (ii) improve estimates of global eukaryotic species diversity, and (iii) use plant and arthropod communities as efficient proxies for each other, thus improving the efficiency of conservation planning and of detecting forest degradation and recovery. We therefore applied metabarcoding to Malaise-trap samples across two tropical landscapes in China. We demonstrate that plant species richness can accurately predict arthropod (mostly insect) species richness and that plant and insect community compositions are highly correlated, even in landscapes that are large, heterogeneous and anthropogenically modified. Finally, we review how metabarcoding makes feasible highly replicated tests of the major competing explanations for tropical megadiversity.</p>
<h1>Analysis of Illumina MiSeq Metabarcoding Data: Application to Benthic Indices for Environmental Monitoring.</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_16">doi:10.1007/978-1-4939-3774-5_16</a></p>
['Amplicon sequencing', 'CO1', 'Folmer region', 'Marine benthic macroinvertebrates', 'MiSeq']<p>This protocol details the analysis of Illumina MiSeq amplicon libraries derived from marine benthic macroinvertebrate samples and based on two barcodes of the mitochondrial cytochrome oxidase 1 (CO1) gene: a "short region," covered by overlapping forward and reverse reads and a "long region" for which forward and reverse reads do not overlap. Aside from providing guidelines for analyzing both types of amplicons, we show how amplicon reads can be used for the calculation of benthic indices for environmental monitoring.</p>
<h1>Visualizing Patterns of Marine Eukaryotic Diversity from Metabarcoding Data Using QIIME.</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_15">doi:10.1007/978-1-4939-3774-5_15</a></p>
['Alpha diversity', 'Beta diversity', 'Metabarcoding', 'Principal component analysis', 'QIIME', 'Rarefaction']<p>PCR amplification followed by deep sequencing of homologous gene regions is increasingly used to characterize the diversity and taxonomic composition of marine eukaryotic communities. This approach may generate millions of sequences for hundreds of samples simultaneously. Therefore, tools that researchers can use to visualize complex patterns of diversity for these massive datasets are essential. Efforts by microbiologists to understand the Earth and human microbiomes using high-throughput sequencing of the 16S rRNA gene has led to the development of several user-friendly, open-source software packages that can be similarly used to analyze eukaryotic datasets. Quantitative Insights Into Microbial Ecology (QIIME) offers some of the most helpful data visualization tools. Here, we describe functionalities to import OTU tables generated with any molecular marker (e.g., 18S, COI, ITS) and associated metadata into QIIME. We then present a range of analytical tools implemented within QIIME that can be used to obtain insights about patterns of alpha and beta diversity for marine eukaryotes.</p>
<h1>Preparation of Amplicon Libraries for Metabarcoding of Marine Eukaryotes Using Illumina MiSeq: The Adapter Ligation Method.</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_14">doi:10.1007/978-1-4939-3774-5_14</a></p>
['Eukaryotes', 'Illumina MiSeq', 'Metabarcoding', 'Multiplexing', 'Tags', 'TruSeq']<p>Amplicon-based studies of marine microscopic eukaryotes, also referred to as metabarcoding studies, can be performed to analyze patterns of biodiversity or predator-prey interactions targeting the mitochondrial cytochrome oxidase 1 (CO1) or the small ribosomal subunit (SSU) markers. Because high-throughput sequencing (HTS) Illumina platforms provide millions of reads per run, hundreds of samples may be sequenced simultaneously. This protocol details the preparation of multiplexed amplicon libraries for Illumina MiSeq sequencing. We describe a strategy for sample multiplexing using a combination of tailed PCR primers and ligation of indexed adapters.</p>
<h1>Metabarcoding Marine Sediments: Preparation of Amplicon Libraries.</h1>
<p><a href="https://doi.org/10.1007/978-1-4939-3774-5_12">doi:10.1007/978-1-4939-3774-5_12</a></p>
['Environmental DNA', 'High-throughput sequencing', 'Illumina', 'Marine eukaryotes', 'Metabarcoding', 'Molecular biodiversity', 'NGS']<p>The accurate assessment of community composition and ultimately species identification is of utmost importance in any ecological and evolutionary study. Advances in sequencing technologies have allowed the unraveling of levels of biodiversity never imagined before when applied to large-scale environmental DNA studies (also termed metabarcoding/metagenetics/metasystematics/environmental barcoding). Here, we describe a detailed protocol to assess eukaryotic biodiversity in marine sediments, identifying key steps that should not be neglected when preparing Next-Generation Sequencing (NGS) amplicon libraries: DNA extraction, multiple PCR amplification of DNA barcode markers with index/ tag-primers, and final Illumina MiSeq sequencing library preparation.</p>
<h1>High-throughput metabarcoding of eukaryotic diversity for environmental monitoring of offshore oil-drilling activities.</h1>
<p><a href="https://doi.org/10.1111/Mec.13761">doi:10.1111/Mec.13761</a></p>
['DNA barcoding', 'bioinformatics/phyloinformatics', 'foodwebs', 'microbial biology']<p>As global exploitation of available resources increases, operations extend towards sensitive and previously protected ecosystems. It is important to monitor such areas in order to detect, understand and remediate environmental responses to stressors. The natural heterogeneity and complexity of communities means that accurate monitoring requires high resolution, both temporally and spatially, as well as more complete assessments of taxa. Increased resolution and taxonomic coverage is economically challenging using current microscopy-based monitoring practices. Alternatively, DNA sequencing-based methods have been suggested for cost-efficient monitoring, offering additional insights into ecosystem function and disturbance. Here, we applied DNA metabarcoding of eukaryotic communities in marine sediments, in areas of offshore drilling on the Norwegian continental shelf. Forty-five samples, collected from seven drilling sites in the Troll/Oseberg region, were assessed, using the small subunit ribosomal RNA gene as a taxonomic marker. In agreement with results based on classical morphology-based monitoring, we were able to identify changes in sediment communities surrounding oil platforms. In addition to overall changes in community structure, we identified several potential indicator taxa, responding to pollutants associated with drilling fluids. These included the metazoan orders Macrodasyida, Macrostomida and Ceriantharia, as well as several ciliates and other protist taxa, typically not targeted by environmental monitoring programmes. Analysis of a co-occurrence network to study the distribution of taxa across samples provided a framework for better understanding the impact of anthropogenic activities on the benthic food web, generating novel, testable hypotheses of trophic interactions structuring benthic communities.</p>
<h1>A Bigger Toolbox: Biotechnology in Biodiversity Conservation.</h1>
<p><a href="https://doi.org/S0167-7799(16)30091-9">doi:S0167-7799(16)30091-9</a></p>
['ancient DNA', 'de-extinction', 'environmental DNA', 'genomics', 'metabarcoding', 'next-generation sequencing']<p>Conservation biology needs a bigger toolbox to meet unprecedented challenges. Genomics, fueled by declining sequencing costs, offers novel tools with increased precision for genetic questions previously answered with a few molecular markers, as well as completely new possibilities. Metabarcoding promises quicker, cheaper, and more accurate assessments of biodiversity in groups that are difficult to assess by traditional methods, while sequencing low-quality DNA extends the range of useable materials to include museum specimens, archeological remains, and environmental samples. Genomic and transcriptomic data can be used to assess the potential of populations to adapt to new challenges. In the near future, gene-editing tools may help endangered species cope with change, while gene drives control unwanted species and help wanted ones. De-extinction has become a serious prospect.</p>
<h1>New Primers for Discovering Fungal Diversity Using Nuclear Large Ribosomal DNA.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0159043">doi:10.1371/Journal.Pone.0159043</a></p>
<p>No keywords</p><p>Metabarcoding has become an important tool in the discovery of biodiversity, including fungi, which are the second most speciose group of eukaryotes, with diverse and important ecological roles in terrestrial ecosystems. We have designed and tested new PCR primers that target the D1 variable region of nuclear large subunit (LSU) ribosomal DNA; one set that targets the phylum Ascomycota and another that recovers all other fungal phyla. The primers yield amplicons compatible with the Illumina MiSeq platform, which is cost-effective and has a lower error rate than other high throughput sequencing platforms. The new primer set LSU200A-F/LSU476A-R (Ascomycota) yielded 95-98% of reads of target taxa from environmental samples, and primers LSU200-F/LSU481-R (all other fungi) yielded 72-80% of target reads. Both primer sets have fairly low rates of data loss, and together they cover a wide variety of fungal taxa. We compared our results with these primers by amplifying and sequencing a subset of samples using the previously described ITS3_KYO2/ITS4_KYO3 primers, which amplify the internal transcribed spacer 2 (ITS2) of Ascomycota and Basidiomycota. With approximately equivalent read depth, our LSU primers recovered a greater number and phylogenetic diversity of sequences than the ITS2 primers. For instance, ITS3_KYO2/ITS4_KYO3 primers failed to pick up any members of Eurotiales, Mytilinidiales, Pezizales, Saccharomycetales, or Venturiales within Ascomycota, or members of Exobasidiomycetes, Microbotryomycetes, Pucciniomycetes, or Tremellomycetes within Basidiomycota, which were retrieved in good numbers from the same samples by our LSU primers. Among the OTUs recovered using the LSU primers were 127 genera and 28 species that were not obtained using the ITS2 primers, although the ITS2 primers recovered 10 unique genera and 16 species that were not obtained using either of the LSU primers These features identify the new primer sets developed in this study as useful complements to other universal primers for the study of fungal diversity and community composition. </p>
<h1>The effects of parameter choice on defining molecular operational taxonomic units and resulting ecological analyses of metabarcoding data.</h1>
<p>No DOI</p>['DNA barcoding', 'MOTU', 'codage à barres de l’ADN', 'eDNA', 'ecological simulations', 'metabarcoding', 'métacodage à barres', 'simulations écologiques']<p>The combination of DNA barcoding and high-throughput (next-generation) sequencing (metabarcoding) provides many promises but also serious challenges. Generating a reliable comparable estimate of biodiversity remains a central challenge to the application of the technology. Many approaches have been used to turn millions of sequences into distinct taxonomic units. However, the extent to which these methods impact the outcome of simple ecological analyses is not well understood. Here we performed a simple analysis of dietary overlap by skinks and shrews on Ile Aux Aigrettes, Mauritius. We used a combination of filtering thresholds and clustering algorithms on a COI metabarcoding dataset and demonstrate that all bioinformatics parameters will have interacting effects on molecular operational taxonomic unit (MOTU) recovery rates. These effects generated estimates covering two orders of magnitude. However, the effect on a simple ecological analysis was not large and, despite the wide variation in estimates of niche overlap, the same ecological conclusion was drawn in most cases. We advise that a conservative clustering programme coupled with larger sequence divergences to define a cluster, the removal of singletons, rigorous length filtering, and stringent match criteria for Molecular Identifier tags are preferable to avoid MOTU inflation and that the same parameters be used in all comparative analyses.</p>
<h1>Pollen DNA barcoding: current applications and future prospects.</h1>
<p><a href="https://doi.org/10.1139/Gen-2015-0200">doi:10.1139/Gen-2015-0200</a></p>
['DNA metabarcoding', 'high-throughput sequencing', 'metagenomics', 'métacodage à barres', 'métagénomique', 'next-generation sequencing', 'palynologie', 'palynology', 'pollen', 'séquençage de nouvelle génération', 'séquençage à haut débit']<p>Identification of the species origin of pollen has many applications, including assessment of plant-pollinator networks, reconstruction of ancient plant communities, product authentication, allergen monitoring, and forensics. Such applications, however, have previously been limited by microscopy-based identification of pollen, which is slow, has low taxonomic resolution, and has few expert practitioners. One alternative is pollen DNA barcoding, which could overcome these issues. Recent studies demonstrate that both chloroplast and nuclear barcoding markers can be amplified from pollen. These recent validations of pollen metabarcoding indicate that now is the time for researchers in various fields to consider applying these methods to their research programs. In this paper, we review the nascent field of pollen DNA barcoding and discuss potential new applications of this technology, highlighting existing limitations and future research developments that will improve its utility in a wide range of applications. </p>
<h1>Large-Scale Monitoring of Plants through Environmental DNA Metabarcoding of Soil: Recovery, Resolution, and Annotation of Four DNA Markers.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0157505">doi:10.1371/Journal.Pone.0157505</a></p>
<p>No keywords</p><p>In a rapidly changing world we need methods to efficiently assess biodiversity in order to monitor ecosystem trends. Ecological monitoring often uses plant community composition to infer quality of sites but conventional aboveground surveys only capture a snapshot of the actively growing plant diversity. Environmental DNA (eDNA) extracted from soil samples, however, can include taxa represented by both active and dormant tissues, seeds, pollen, and detritus. Analysis of this eDNA through DNA metabarcoding provides a more comprehensive view of plant diversity at a site from a single assessment but it is not clear which DNA markers are best used to capture this diversity. Sequence recovery, annotation, and sequence resolution among taxa were evaluated for four established DNA markers (matK, rbcL, ITS2, and the trnL P6 loop) in silico using database sequences and in situ using high throughput sequencing of 35 soil samples from a remote boreal wetland. Overall, ITS2 and rbcL are recommended for DNA metabarcoding of vascular plants from eDNA when not using customized or geographically restricted reference databases. We describe a new framework for evaluating DNA metabarcodes and, contrary to existing assumptions, we found that full length DNA barcode regions could outperform shorter markers for surveying plant diversity from soil samples. By using current DNA barcoding markers rbcL and ITS2 for plant metabarcoding, we can take advantage of existing resources such as the growing DNA barcode database. Our work establishes the value of standard DNA barcodes for soil plant eDNA analysis in ecological investigations and biomonitoring programs and supports the collaborative development of DNA barcoding and metabarcoding. </p>
<h1>Resilience of Freshwater Communities of Small Microbial Eukaryotes Undergoing Severe Drought Events.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2016.00812">doi:10.3389/Fmicb.2016.00812</a></p>
['18S rRNA', 'dormancy', 'plankton', 'protist', 'resilience', 'temporal dynamics']<p>Small and shallow aquatic ecosystems such as ponds and streams constitute a significant proportion of continental surface waters, especially in temperate zones. In comparison with bigger lakes and rivers, they harbor higher biodiversity but they also exhibit reduced buffering capacity face to environmental shifts, such that climate global change can affect them in a more drastic way. For instance, many temperate areas are predicted to undergo droughts with increasing frequency in the near future, which may lead to the temporal desiccation of streams and ponds. In this work, we monitored temporal dynamics of planktonic communities of microbial eukaryotes (cell size range: 0.2-5 μm) in one brook and one pond that experienced recurrent droughts from 1 to 5 consecutive months during a temporal survey carried out monthly for 2 years based on high-throughput 18S rDNA metabarcoding. During drought-induced desiccation events, protist communities present in the remaining dry sediment, though highly diverse, differed radically from their planktonic counterparts. However, after water refill, the aquatic protist assemblages recovered their original structure within a month. This rapid recovery indicates that these eukaryotic communities are resilient to droughts, most likely via the entrance in dormancy. This property is essential for the long-term survival and functional stability of small freshwater ecosystems. </p>
<h1>Genetic assessment of meiobenthic community composition and spatial distribution in coastal sediments along northern Gulf of Mexico.</h1>
<p><a href="https://doi.org/10.1016/J.Marenvres.2016.05.011">doi:10.1016/J.Marenvres.2016.05.011</a></p>
['18S rRNA', 'High-throughput sequencing', 'Illumina', 'Meiofauna', 'Metabarcoding', 'Micro-eukaryotes', 'Protists']<p>Meiobenthic (meiofauna and micro-eukaryotes) organisms are important contributors to ecosystem functioning in aquatic environments through their roles in nutrient transport, sediment stability, and food web interactions. Despite their ecological importance, information pertaining to variation of these communities at various spatial and temporal scales is not widely known. Many studies in the Gulf of Mexico (GOM) have focused either on deep sea or continental shelf areas, while little attention has been paid to bays and coastal regions. Herein, we take a holistic approach by using high-throughput sequencing approaches to examine spatial variation in meiobenthic communities within Alabama bays and the coastal northern GOM region. Sediment samples were collected along three transects (Mississippi Sound: MS, FOCAL: FT, and Orange Beach: OB) from September 2010 to April 2012 and community composition was determined by metabarcoding the V9 hypervariable region of the nuclear18S rRNA gene. Results showed that Stramenopiles (diatoms), annelids, arthropods (copepods), and nematodes were the dominate groups within samples, while there was presence of other phyla throughout the dataset. Location played a larger role than time sampled in community composition. However, samples were collected over a short temporal scale. Samples clustered in reference to transect, with the most eastern transect (OB) having a distinct community composition in comparison to the other two transects (MS and FT). Communities also differed in reference to region (Bay versus Shelf). Bulk density and percent inorganic carbon were the only measured environmental factors that were correlated with community composition. </p>
<h1>Vector soup: high-throughput identification of Neotropical phlebotomine sand flies using metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12556">doi:10.1111/1755-0998.12556</a></p>
['DNA barcode', 'French Guiana', 'Psychodidae', 'leishmaniasis', 'mitochondrial 16S rRNA gene', 'vector monitoring']<p>Phlebotomine sand flies are haematophagous dipterans of primary medical importance. They represent the only proven vectors of leishmaniasis worldwide and are involved in the transmission of various other pathogens. Studying the ecology of sand flies is crucial to understand the epidemiology of leishmaniasis and further control this disease. A major limitation in this regard is that traditional morphological-based methods for sand fly species identifications are time-consuming and require taxonomic expertise. DNA metabarcoding holds great promise in overcoming this issue by allowing the identification of multiple species from a single bulk sample. Here, we assessed the reliability of a short insect metabarcode located in the mitochondrial 16S rRNA for the identification of Neotropical sand flies, and constructed a reference database for 40 species found in French Guiana. Then, we conducted a metabarcoding experiment on sand flies mixtures of known content and showed that the method allows an accurate identification of specimens in pools. Finally, we applied metabarcoding to field samples caught in a 1-ha forest plot in French Guiana. Besides providing reliable molecular data for species-level assignations of phlebotomine sand flies, our study proves the efficiency of metabarcoding based on the mitochondrial 16S rRNA for studying sand fly diversity from bulk samples. The application of this high-throughput identification procedure to field samples can provide great opportunities for vector monitoring and eco-epidemiological studies.</p>
<h1>Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea.</h1>
<p><a href="https://doi.org/10.3389/Fmicb.2016.00679">doi:10.3389/Fmicb.2016.00679</a></p>
['Baltic Sea', 'bacterioplankton', 'brackish', 'marine microbiology', 'metabarcoding', 'microbial ecology', 'protists']<p>Microbial plankton form the productive base of both marine and freshwater ecosystems and are key drivers of global biogeochemical cycles of carbon and nutrients. Plankton diversity is immense with representations from all major phyla within the three domains of life. So far, plankton monitoring has mainly been based on microscopic identification, which has limited sensitivity and reproducibility, not least because of the numerical majority of plankton being unidentifiable under the light microscope. High-throughput sequencing of taxonomic marker genes offers a means to identify taxa inaccessible by traditional methods; thus, recent studies have unveiled an extensive previously unknown diversity of plankton. Here, we conducted ultra-deep Illumina sequencing (average 10(5) sequences/sample) of rRNA gene amplicons of surface water eukaryotic and bacterial plankton communities sampled in summer along a 2000 km transect following the salinity gradient of the Baltic Sea. Community composition was strongly correlated with salinity for both bacterial and eukaryotic plankton assemblages, highlighting the importance of salinity for structuring the biodiversity within this ecosystem. In contrast, no clear trends in alpha-diversity for bacterial or eukaryotic communities could be detected along the transect. The distribution of major planktonic taxa followed expected patterns as observed in monitoring programs, but groups novel to the Baltic Sea were also identified, such as relatives to the coccolithophore Emiliana huxleyi detected in the northern Baltic Sea. This study provides the first ultra-deep sequencing-based survey on eukaryotic and bacterial plankton biogeography in the Baltic Sea. </p>
<h1>Metabarcoding reveals environmental factors influencing spatio-temporal variation in pelagic micro-eukaryotes.</h1>
<p><a href="https://doi.org/10.1111/Mec.13709">doi:10.1111/Mec.13709</a></p>
['18S rRNA', 'Gulf of Mexico', 'Illumina', 'diversity', 'high-throughput sequencing']<p>Marine environments harbour a vast diversity of micro-eukaryotic organisms (protists and other small eukaryotes) that play important roles in structuring marine ecosystems. However, micro-eukaryote diversity is not well understood. Likewise, knowledge is limited regarding micro-eukaryote spatial and seasonal distribution, especially over long temporal scales. Given the importance of this group for mobilizing energy from lower trophic levels near the base of the food chain to larger organisms, assessing community stability, diversity and resilience is important to understand ecosystem health. Herein, we use a metabarcoding approach to examine pelagic micro-eukaryote communities over a 2.5-year time series. Bimonthly surface sampling (July 2009 to December 2011) was conducted at four locations within Mobile Bay (Bay) and along the Alabama continental shelf (Shelf). Alpha-diversity only showed significant differences in Shelf sites, with the greatest differences observed between summer and winter. Beta-diversity showed significant differences in community composition in relation to season and the Bay was dominated by diatoms, while the Shelf was characterized by dinoflagellates and copepods. The northern Gulf of Mexico is heavily influenced by the Mobile River Basin, which brings low-salinity nutrient-rich water mostly during winter and spring. Community composition was correlated with salinity, temperature and dissolved silicate. However, species interactions (e.g. predation and parasitism) may also contribute to the observed variation, especially on the Shelf, which warrants further exploration. Metabarcoding revealed clear patterns in surface pelagic micro-eukaryote communities that were consistent over multiple years, demonstrating how these techniques could be greatly beneficial to ecological monitoring and management over temporal scales.</p>
<h1>Metabarcoding improves detection of eukaryotes from early biofouling communities: implications for pest monitoring and pathway management.</h1>
<p><a href="https://doi.org/10.1080/08927014.2016.1186165">doi:10.1080/08927014.2016.1186165</a></p>
['18S ribosomal RNA gene', 'detection tools', 'high-throughput sequencing', 'marine biofilms', 'non-indigenous species']<p>In this experimental study the patterns in early marine biofouling communities and possible implications for surveillance and environmental management were explored using metabarcoding, viz. 18S ribosomal RNA gene barcoding in combination with high-throughput sequencing. The community structure of eukaryotic assemblages and the patterns of initial succession were assessed from settlement plates deployed in a busy port for one, five and 15 days. The metabarcoding results were verified with traditional morphological identification of taxa from selected experimental plates. Metabarcoding analysis identified > 400 taxa at a comparatively low taxonomic level and morphological analysis resulted in the detection of 25 taxa at varying levels of resolution. Despite the differences in resolution, data from both methods were consistent at high taxonomic levels and similar patterns in community shifts were observed. A high percentage of sequences belonging to genera known to contain non-indigenous species (NIS) were detected after exposure for only one day.</p>
<h1>Advances in DNA metabarcoding for food and wildlife forensic species identification.</h1>
<p><a href="https://doi.org/10.1007/S00216-016-9595-8">doi:10.1007/S00216-016-9595-8</a></p>
['Convention on International Trade of Endangered Species', 'Cytochrome c oxidase I', 'Endangered species', 'Next-generation sequencing', 'Wildlife forensic samples']<p>Species identification using DNA barcodes has been widely adopted by forensic scientists as an effective molecular tool for tracking adulterations in food and for analysing samples from alleged wildlife crime incidents. DNA barcoding is an approach that involves sequencing of short DNA sequences from standardized regions and comparison to a reference database as a molecular diagnostic tool in species identification. In recent years, remarkable progress has been made towards developing DNA metabarcoding strategies, which involves next-generation sequencing of DNA barcodes for the simultaneous detection of multiple species in complex samples. Metabarcoding strategies can be used in processed materials containing highly degraded DNA e.g. for the identification of endangered and hazardous species in traditional medicine. This review aims to provide insight into advances of plant and animal DNA barcoding and highlights current practices and recent developments for DNA metabarcoding of food and wildlife forensic samples from a practical point of view. Special emphasis is placed on new developments for identifying species listed in the Convention on International Trade of Endangered Species (CITES) appendices for which reliable methods for species identification may signal and/or prevent illegal trade. Current technological developments and challenges of DNA metabarcoding for forensic scientists will be assessed in the light of stakeholders' needs. </p>
<h1>Fungi Sailing the Arctic Ocean: Speciose Communities in North Atlantic Driftwood as Revealed by High-Throughput Amplicon Sequencing.</h1>
<p><a href="https://doi.org/10.1007/S00248-016-0778-9">doi:10.1007/S00248-016-0778-9</a></p>
['454 sequencing', 'Biosystematics', 'Community ecology', 'Diversity', 'Marine fungi', 'Marine wooden substrates', 'Metabarcoding']<p>High amounts of driftwood sail across the oceans and provide habitat for organisms tolerating the rough and saline environment. Fungi have adapted to the extremely cold and saline conditions which driftwood faces in the high north. For the first time, we applied high-throughput sequencing to fungi residing in driftwood to reveal their taxonomic richness, community composition, and ecology in the North Atlantic. Using pyrosequencing of ITS2 amplicons obtained from 49 marine logs, we found 807 fungal operational taxonomic units (OTUs) based on clustering at 97 % sequence similarity cut-off level. The phylum Ascomycota comprised 74 % of the OTUs and 20 % belonged to Basidiomycota. The richness of basidiomycetes decreased with prolonged submersion in the sea, supporting the general view of ascomycetes being more extremotolerant. However, more than one fourth of the fungal OTUs remained unassigned to any fungal class, emphasising the need for better DNA reference data from the marine habitat. Different fungal communities were detected in coniferous and deciduous logs. Our results highlight that driftwood hosts a considerably higher fungal diversity than currently known. The driftwood fungal community is not a terrestrial relic but a speciose assemblage of fungi adapted to the stressful marine environment and different kinds of wooden substrates found in it.</p>
<h1>Habitat conditions and phenological tree traits overrule the influence of tree genotype in the needle mycobiome-Picea glauca system at an arctic treeline ecotone.</h1>
<p><a href="https://doi.org/10.1111/Nph.13988">doi:10.1111/Nph.13988</a></p>
['Illumina high-throughput sequencing', 'biodiversity', 'internal transcribed spacer (ITS) rDNA', 'metabarcoding', 'microsatellites', 'needle-inhabiting fungal phytobiome', 'plant-fungus interactions', 'tree phenotypic plasticity']<p>Plant-associated mycobiomes in extreme habitats are understudied and poorly understood. We analysed Illumina-generated ITS1 sequences from the needle mycobiome of white spruce (Picea glauca) at the northern treeline in Alaska (USA). Sequences were obtained from the same DNA that was used for tree genotyping. In the present study, fungal metabarcoding and tree microsatellite data were compared for the first time. In general, neighbouring trees shared more fungal taxa with each other than trees growing in further distance. Mycobiomes correlated strongly with phenological host traits and local habitat characteristics contrasting a dense forest stand with an open treeline site. Genetic similarity between trees did not influence fungal composition and no significant correlation existed between needle mycobiome and tree genotype. Our results suggest the pronounced influence of local habitat conditions and phenotypic tree traits on needle-inhabiting fungi. By contrast, the tree genetic identity cannot be benchmarked as a dominant driver for needle-inhabiting mycobiomes, at least not for white spruce in this extreme environment.</p>
<h1>Testing the potential of a ribosomal 16S marker for DNA metabarcoding of insects.</h1>
<p><a href="https://doi.org/10.7717/Peerj.1966">doi:10.7717/Peerj.1966</a></p>
['Biodiversity assessment', 'DNA metabarcoding', 'Freshwater invertebrates', 'High throughput sequencing', 'Illumina sequencing', 'Method testing', 'Primer bias', 'Small ribosomal subunit', 'Stream ecosystems', 'Stream monitoring']<p>Cytochrome c oxidase I (COI) is a powerful marker for DNA barcoding of animals, with good taxonomic resolution and a large reference database. However, when used for DNA metabarcoding, estimation of taxa abundances and species detection are limited due to primer bias caused by highly variable primer binding sites across the COI gene. Therefore, we explored the ability of the 16S ribosomal DNA gene as an alternative metabarcoding marker for species level assessments. Ten bulk samples, each containing equal amounts of tissue from 52 freshwater invertebrate taxa, were sequenced with the Illumina NextSeq 500 system. The 16S primers amplified three more insect species than the Folmer COI primers and amplified more equally, probably due to decreased primer bias. Estimation of biomass might be less biased with 16S than with COI, although variation in read abundances of two orders of magnitudes is still observed. According to these results, the marker choice depends on the scientific question. If the goal is to obtain a taxonomic identification at the species level, then COI is more appropriate due to established reference databases and known taxonomic resolution of this marker, knowing that a greater proportion of insects will be missed using COI Folmer primers. If the goal is to obtain a more comprehensive survey the 16S marker, which requires building a local reference database, or optimised degenerated COI primers could be more appropriate. </p>
<h1>Protist metabarcoding and environmental biomonitoring: Time for change.</h1>
<p><a href="https://doi.org/S0932-4739(16)30003-7">doi:S0932-4739(16)30003-7</a></p>
['Bioindicators', 'Diatoms', 'Environmental DNA', 'Foraminifera', 'Metabarcoding', 'Microbial eukaryotes']<p>High-throughput amplicon sequencing of environmental DNA and/or RNA proved to be a powerful tool to describe protist diversity. This new approach called also the metabarcoding has totally transformed our view of protist diversity, revealing a large number of novel lineages and expanding the range of protist phylogenetic diversity at almost every taxonomic level. However, until now the objectives of the vast majority of metabarcoding studies were purely academic. Practical applications of protist metabarcoding are surprisingly scarce, despite the fact that several groups of protists are commonly used as bioindicators of environmental impacts in freshwater or marine ecosystems. Here, we are reviewing studies that examine the ecological applications of metabarcoding for two groups of well-known protist bioindicators: diatoms and foraminifera. The results of these studies show that despite some biological and technical biases, molecular data quite faithfully reflect the morphology-based biotic indices and provide a similar assessment of ecosystem status. In view of these results, protist metabarcoding appears as a rapid and accurate tool for the evaluation of the quality of aquatic ecosystems. Hence, we plead for integration of protist metabarcoding in future biomonitoring projects as a complement of traditional methods and a source of new biosensors for environmental impact assessment. </p>
<h1>In silico assessment of primers for eDNA studies using PrimerTree and application to characterize the biodiversity surrounding the Cuyahoga River.</h1>
<p><a href="https://doi.org/10.1038/Srep22908">doi:10.1038/Srep22908</a></p>
<p>No keywords</p><p>Analysis of environmental DNA (eDNA) enables the detection of species of interest from water and soil samples, typically using species-specific PCR. Here, we describe a method to characterize the biodiversity of a given environment by amplifying eDNA using primer pairs targeting a wide range of taxa and high-throughput sequencing for species identification. We tested this approach on 91 water samples of 40 mL collected along the Cuyahoga River (Ohio, USA). We amplified eDNA using 12 primer pairs targeting mammals, fish, amphibians, birds, bryophytes, arthropods, copepods, plants and several microorganism taxa and sequenced all PCR products simultaneously by high-throughput sequencing. Overall, we identified DNA sequences from 15 species of fish, 17 species of mammals, 8 species of birds, 15 species of arthropods, one turtle and one salamander. Interestingly, in addition to aquatic and semi-aquatic animals, we identified DNA from terrestrial species that live near the Cuyahoga River. We also identified DNA from one Asian carp species invasive to the Great Lakes but that had not been previously reported in the Cuyahoga River. Our study shows that analysis of eDNA extracted from small water samples using wide-range PCR amplification combined with high-throughput sequencing can provide a broad perspective on biological diversity. </p>
<h1>A metabarcoding framework for facilitated survey of endolithic phototrophs with tufA.</h1>
<p><a href="https://doi.org/10.1186/S12898-016-0068-X">doi:10.1186/S12898-016-0068-X</a></p>
['Algae', 'Amplicon Metagenomics', 'Bryopsidales', 'Calcium carbonate', 'Coral reef', 'Illumina MiSeq', 'Metabarcoding', 'Next generation sequencing', 'Ostreobium', 'Phaeophila', 'Phototrophs', 'Plastid', 'Rhodoliths', 'Ulvophyceae', 'tufA']<p>In spite of their ecological importance as primary producers and microbioeroders of marine calcium carbonate (CaCO3) substrata, endolithic phototrophs spanning both prokaryotic (the cyanobacteria) and eukaryotic algae lack established molecular resources for their facilitated survey with high throughput sequencing. Here, the development of a metabarcoding framework for the elongation factor EF-Ttu (tufA) was tested on four Illumina-sequenced marine CaCO3 microfloras for the characterization of their endolithic phototrophs, especially the abundant bioeroding Ostreobium spp. (Ulvophyceae). The framework consists of novel tufA degenerate primers and a comprehensive database enabling Operational Taxonomic Unit (OTU) identification at multiple taxonomic ranks with percent identity thresholds determined herein.</p>
<h1>Unveiling the Biodiversity of Deep-Sea Nematodes through Metabarcoding: Are We Ready to Bypass the Classical Taxonomy?</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0144928">doi:10.1371/Journal.Pone.0144928</a></p>
<p>No keywords</p><p>Nematodes inhabiting benthic deep-sea ecosystems account for >90% of the total metazoan abundances and they have been hypothesised to be hyper-diverse, but their biodiversity is still largely unknown. Metabarcoding could facilitate the census of biodiversity, especially for those tiny metazoans for which morphological identification is difficult. We compared, for the first time, different DNA extraction procedures based on the use of two commercial kits and a previously published laboratory protocol and tested their suitability for sequencing analyses of 18S rDNA of marine nematodes. We also investigated the reliability of Roche 454 sequencing analyses for assessing the biodiversity of deep-sea nematode assemblages previously morphologically identified. Finally, intra-genomic variation in 18S rRNA gene repeats was investigated by Illumina MiSeq in different deep-sea nematode morphospecies to assess the influence of polymorphisms on nematode biodiversity estimates. Our results indicate that the two commercial kits should be preferred for the molecular analysis of biodiversity of deep-sea nematodes since they consistently provide amplifiable DNA suitable for sequencing. We report that the morphological identification of deep-sea nematodes matches the results obtained by metabarcoding analysis only at the order-family level and that a large portion of Operational Clustered Taxonomic Units (OCTUs) was not assigned. We also show that independently from the cut-off criteria and bioinformatic pipelines used, the number of OCTUs largely exceeds the number of individuals and that 18S rRNA gene of different morpho-species of nematodes displayed intra-genomic polymorphisms. Our results indicate that metabarcoding is an important tool to explore the diversity of deep-sea nematodes, but still fails in identifying most of the species due to limited number of sequences deposited in the public databases, and in providing quantitative data on the species encountered. These aspects should be carefully taken into account before using metabarcoding in quantitative ecological research and monitoring programmes of marine biodiversity. </p>
<h1>MiFish, a set of universal PCR primers for metabarcoding environmental DNA from fishes: detection of more than 230 subtropical marine species.</h1>
<p><a href="https://doi.org/10.1098/Rsos.150088">doi:10.1098/Rsos.150088</a></p>
['MiSeq', 'community ecology', 'environmental DNA', 'metabarcoding', 'mitogenome', 'resource management']<p>We developed a set of universal PCR primers (MiFish-U/E) for metabarcoding environmental DNA (eDNA) from fishes. Primers were designed using aligned whole mitochondrial genome (mitogenome) sequences from 880 species, supplemented by partial mitogenome sequences from 160 elasmobranchs (sharks and rays). The primers target a hypervariable region of the 12S rRNA gene (163-185 bp), which contains sufficient information to identify fishes to taxonomic family, genus and species except for some closely related congeners. To test versatility of the primers across a diverse range of fishes, we sampled eDNA from four tanks in the Okinawa Churaumi Aquarium with known species compositions, prepared dual-indexed libraries and performed paired-end sequencing of the region using high-throughput next-generation sequencing technologies. Out of the 180 marine fish species contained in the four tanks with reference sequences in a custom database, we detected 168 species (93.3%) distributed across 59 families and 123 genera. These fishes are not only taxonomically diverse, ranging from sharks and rays to higher teleosts, but are also greatly varied in their ecology, including both pelagic and benthic species living in shallow coastal to deep waters. We also sampled natural seawaters around coral reefs near the aquarium and detected 93 fish species using this approach. Of the 93 species, 64 were not detected in the four aquarium tanks, rendering the total number of species detected to 232 (from 70 families and 152 genera). The metabarcoding approach presented here is non-invasive, more efficient, more cost-effective and more sensitive than the traditional survey methods. It has the potential to serve as an alternative (or complementary) tool for biodiversity monitoring that revolutionizes natural resource management and ecological studies of fish communities on larger spatial and temporal scales. </p>
<h1>Targeted gene enrichment and high-throughput sequencing for environmental biomonitoring: a case study using freshwater macroinvertebrates.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12488">doi:10.1111/1755-0998.12488</a></p>
['biomonitoring', 'gene enrichment', 'high-throughput sequencing', 'macroinvertebrates', 'metabarcoding']<p>Recent studies have advocated biomonitoring using DNA techniques. In this study, two high-throughput sequencing (HTS)-based methods were evaluated: amplicon metabarcoding of the cytochrome C oxidase subunit I (COI) mitochondrial gene and gene enrichment using MYbaits (targeting nine different genes including COI). The gene-enrichment method does not require PCR amplification and thus avoids biases associated with universal primers. Macroinvertebrate samples were collected from 12 New Zealand rivers. Macroinvertebrates were morphologically identified and enumerated, and their biomass determined. DNA was extracted from all macroinvertebrate samples and HTS undertaken using the illumina miseq platform. Macroinvertebrate communities were characterized from sequence data using either six genes (three of the original nine were not used) or just the COI gene in isolation. The gene-enrichment method (all genes) detected the highest number of taxa and obtained the strongest Spearman rank correlations between the number of sequence reads, abundance and biomass in 67% of the samples. Median detection rates across rare (<1% of the total abundance or biomass), moderately abundant (1-5%) and highly abundant (>5%) taxa were highest using the gene-enrichment method (all genes). Our data indicated primer biases occurred during amplicon metabarcoding with greater than 80% of sequence reads originating from one taxon in several samples. The accuracy and sensitivity of both HTS methods would be improved with more comprehensive reference sequence databases. The data from this study illustrate the challenges of using PCR amplification-based methods for biomonitoring and highlight the potential benefits of using approaches, such as gene enrichment, which circumvent the need for an initial PCR step. </p>
<h1>Molecular diversity and distribution of marine fungi across 130 European environmental samples.</h1>
<p><a href="https://doi.org/10.1098/Rspb.2015.2243">doi:10.1098/Rspb.2015.2243</a></p>
['454 pyrosequencing', 'Dikarya', 'chytrids', 'sediment communities']<p>Environmental DNA and culture-based analyses have suggested that fungi are present in low diversity and in low abundance in many marine environments, especially in the upper water column. Here, we use a dual approach involving high-throughput diversity tag sequencing from both DNA and RNA templates and fluorescent cell counts to evaluate the diversity and relative abundance of fungi across marine samples taken from six European near-shore sites. We removed very rare fungal operational taxonomic units (OTUs) selecting only OTUs recovered from multiple samples for a detailed analysis. This approach identified a set of 71 fungal 'OTU clusters' that account for 66% of all the sequences assigned to the Fungi. Phylogenetic analyses demonstrated that this diversity includes a significant number of chytrid-like lineages that had not been previously described, indicating that the marine environment encompasses a number of zoosporic fungi that are new to taxonomic inventories. Using the sequence datasets, we identified cases where fungal OTUs were sampled across multiple geographical sites and between different sampling depths. This was especially clear in one relatively abundant and diverse phylogroup tentatively named Novel Chytrid-Like-Clade 1 (NCLC1). For comparison, a subset of the water column samples was also investigated using fluorescent microscopy to examine the abundance of eukaryotes with chitin cell walls. Comparisons of relative abundance of RNA-derived fungal tag sequences and chitin cell-wall counts demonstrate that fungi constitute a low fraction of the eukaryotic community in these water column samples. Taken together, these results demonstrate the phylogenetic position and environmental distribution of 71 lineages, improving our understanding of the diversity and abundance of fungi in marine environments. </p>
<h1>Metabarcoding approach for nonindigenous species surveillance in marine coastal waters.</h1>
<p><a href="https://doi.org/S0025-326X(15)30045-X">doi:S0025-326X(15)30045-X</a></p>
['Baltic Sea', 'COI', 'High-throughput sequencing', 'Nonindigenous species']<p>In this study, high-throughput sequencing (HTS) metabarcoding was applied for the surveillance of plankton communities within the southeastern (SE) Baltic Sea coastal zone. These results were compared with those from routine monitoring survey and morphological analyses. Four of five nonindigenous species found in the samples were identified exclusively by metabarcoding. All of them are considered as invasive in the Baltic Sea with reported impact on the ecosystem and biodiversity. This study indicates that, despite some current limitations, HTS metabarcoding can provide information on the presence of exotic species and advantageously complement conventional approaches, only requiring the same monitoring effort as before. Even in the currently immature status of HTS, this combination of HTS metabarcoding and observational records is recommended in the early detection of marine pests and delivery of the environmental status metrics of nonindigenous species. </p>
<h1>High-throughput sequencing and morphology perform equally well for benthic monitoring of marine ecosystems.</h1>
<p><a href="https://doi.org/10.1038/Srep13932">doi:10.1038/Srep13932</a></p>
<p>No keywords</p><p>Environmental diversity surveys are crucial for the bioassessment of anthropogenic impacts on marine ecosystems. Traditional benthic monitoring relying on morphotaxonomic inventories of macrofaunal communities is expensive, time-consuming and expertise-demanding. High-throughput sequencing of environmental DNA barcodes (metabarcoding) offers an alternative to describe biological communities. However, whether the metabarcoding approach meets the quality standards of benthic monitoring remains to be tested. Here, we compared morphological and eDNA/RNA-based inventories of metazoans from samples collected at 10 stations around a fish farm in Scotland, including near-cage and distant zones. For each of 5 replicate samples per station, we sequenced the V4 region of the 18S rRNA gene using the Illumina technology. After filtering, we obtained 841,766 metazoan sequences clustered in 163 Operational Taxonomic Units (OTUs). We assigned the OTUs by combining local BLAST searches with phylogenetic analyses. We calculated two commonly used indices: the Infaunal Trophic Index and the AZTI Marine Biotic Index. We found that the molecular data faithfully reflect the morphology-based indices and provides an equivalent assessment of the impact associated with fish farms activities. We advocate that future benthic monitoring should integrate metabarcoding as a rapid and accurate tool for the evaluation of the quality of marine benthic ecosystems. </p>
<h1>DNA metabarcoding of insects and allies: an evaluation of primers and pipelines.</h1>
<p><a href="https://doi.org/10.1017/S0007485315000681">doi:10.1017/S0007485315000681</a></p>
['Arthropoda', 'COI', 'Illumina MiSeq', 'Malaise trap', 'biodiversity', 'high-throughput sequencing']<p>Metabarcoding, the coupling of DNA-based species identification and high-throughput sequencing, offers enormous promise for arthropod biodiversity studies but factors such as cost, speed and ease-of-use of bioinformatic pipelines, crucial for making the leapt from demonstration studies to a real-world application, have not yet been adequately addressed. Here, four published and one newly designed primer sets were tested across a diverse set of 80 arthropod species, representing 11 orders, to establish optimal protocols for Illumina-based metabarcoding of tropical Malaise trap samples. Two primer sets which showed the highest amplification success with individual specimen polymerase chain reaction (PCR, 98%) were used for bulk PCR and Illumina MiSeq sequencing. The sequencing outputs were subjected to both manual and simple metagenomics quality control and filtering pipelines. We obtained acceptable detection rates after bulk PCR and high-throughput sequencing (80-90% of input species) but analyses were complicated by putative heteroplasmic sequences and contamination. The manual pipeline produced similar or better outputs to the simple metagenomics pipeline (1.4 compared with 0.5 expected:unexpected Operational Taxonomic Units). Our study suggests that metabarcoding is slowly becoming as cheap, fast and easy as conventional DNA barcoding, and that Malaise trap metabarcoding may soon fulfill its potential, providing a thermometer for biodiversity. </p>
<h1>Accurate assessment of the impact of salmon farming on benthic sediment enrichment using foraminiferal metabarcoding.</h1>
<p><a href="https://doi.org/S0025-326X(15)00526-3">doi:S0025-326X(15)00526-3</a></p>
['Aquaculture', 'Benthic ecology', 'Biomonitoring', 'Foraminifera', 'High-throughput sequencing', 'Metabarcoding']<p>Assessing the environmental impact of salmon farms on benthic systems is traditionally undertaken using biotic indices derived from microscopic analyses of macrobenthic infaunal (MI) communities. In this study, we tested the applicability of using foraminiferal-specific high-throughput sequencing (HTS) metabarcoding for monitoring these habitats. Sediment samples and physico-chemical data were collected along an enrichment gradient radiating out from three Chinook salmon (Oncorhynchus tshawytscha) farms in New Zealand. HTS of environmental DNA and RNA (eDNA/eRNA) resulted in 1,875,300 sequences that clustered into 349 Operational Taxonomic Units. Strong correlations were observed among various biotic indices calculated from MI data and normalized fourth-root transformed HTS data. Correlations were stronger using eRNA compared to eDNA data. Quantile regression spline analyses identified 12 key foraminiferal taxa that have potential to be used as bioindicator species. This study demonstrates the huge potential for using this method for biomonitoring of fish-farming and other marine industrial activities. </p>
<h1>The rise and fall of arbuscular mycorrhizal fungal diversity during ecosystem retrogression.</h1>
<p><a href="https://doi.org/10.1111/Mec.13363">doi:10.1111/Mec.13363</a></p>
['Glomeromycota', 'community ecology', 'dune chronosequence', 'ecosystem development', 'nutrient limitation', 'phosphorus']<p>Ecosystem retrogression following long-term pedogenesis is attributed to phosphorus (P) limitation of primary productivity. Arbuscular mycorrhizal fungi (AMF) enhance P acquisition for most terrestrial plants, but it has been suggested that this strategy becomes less effective in strongly weathered soils with extremely low P availability. Using next generation sequencing of the large subunit ribosomal RNA gene in roots and soil, we compared the composition and diversity of AMF communities in three contrasting stages of a retrogressive >2-million-year dune chronosequence in a global biodiversity hotspot. This chronosequence shows a ~60-fold decline in total soil P concentration, with the oldest stage representing some of the most severely P-impoverished soils found in any terrestrial ecosystem. The richness of AMF operational taxonomic units was low on young (1000's of years), moderately P-rich soils, greatest on relatively old (~120 000 years) low-P soils, and low again on the oldest (>2 000 000 years) soils that were lowest in P availability. A similar decline in AMF phylogenetic diversity on the oldest soils occurred, despite invariant host plant diversity and only small declines in host cover along the chronosequence. Differences in AMF community composition were greatest between the youngest and the two oldest soils, and this was best explained by differences in soil P concentrations. Our results point to a threshold in soil P availability during ecosystem regression below which AMF diversity declines, suggesting environmental filtering of AMF insufficiently adapted to extremely low P availability. </p>
<h1>High-throughput monitoring of wild bee diversity and abundance via mitogenomics.</h1>
<p>No DOI</p>['Hymenoptera', 'agri‐environment schemes', 'biodiversity and ecosystem services', 'farmland biodiversity', 'genome skimming', 'metabarcoding', 'metagenomics', 'mitogenomes', 'neonicotinoids', 'pollination']<p>Bee populations and other pollinators face multiple, synergistically acting threats, which have led to population declines, loss of local species richness and pollination services, and extinctions. However, our understanding of the degree, distribution and causes of declines is patchy, in part due to inadequate monitoring systems, with the challenge of taxonomic identification posing a major logistical barrier. Pollinator conservation would benefit from a high-throughput identification pipeline.We show that the metagenomic mining and resequencing of mitochondrial genomes (mitogenomics) can be applied successfully to bulk samples of wild bees. We assembled the mitogenomes of 48 UK bee species and then shotgun-sequenced total DNA extracted from 204 whole bees that had been collected in 10 pan-trap samples from farms in England and been identified morphologically to 33 species. Each sample data set was mapped against the 48 reference mitogenomes.The morphological and mitogenomic data sets were highly congruent. Out of 63 total species detections in the morphological data set, the mitogenomic data set made 59 correct detections (93·7% detection rate) and detected six more species (putative false positives). Direct inspection and an analysis with species-specific primers suggested that these putative false positives were most likely due to incorrect morphological IDs. Read frequency significantly predicted species biomass frequency (<i>R</i><sup>2</sup> = 24·9%). Species lists, biomass frequencies, extrapolated species richness and community structure were recovered with less error than in a metabarcoding pipeline.Mitogenomics automates the onerous task of taxonomic identification, even for cryptic species, allowing the tracking of changes in species richness and distributions. A mitogenomic pipeline should thus be able to contain costs, maintain consistently high-quality data over long time series, incorporate retrospective taxonomic revisions and provide an auditable evidence trail. Mitogenomic data sets also provide estimates of species counts within samples and thus have potential for tracking population trajectories.</p>
<h1>Comparative study of the validity of three regions of the 18S-rRNA gene for massively parallel sequencing-based monitoring of the planktonic eukaryote community.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12459">doi:10.1111/1755-0998.12459</a></p>
['MPS-based environmental surveys', 'identification power', 'marine eukaryote', 'metabarcoding', 'operational taxonomic units']<p>The nuclear 18S-rRNA gene has been used as a metabarcoding marker in massively parallel sequencing (MPS)-based environmental surveys for plankton biodiversity research. However, different hypervariable regions have been used in different studies, and their utility has been debated among researchers. In this study, detailed investigations into 18S-rRNA were carried out; we investigated the effective number of sequences deposited in international nucleotide sequence databases (INSDs), the amplification bias, and the amplicon sequence variability among the three variable regions, V1-3, V4-5 and V7-9, using in silico polymerase chain reaction (PCR) amplification based on INSDs. We also examined the primer universality and the taxonomic identification power, using MPS-based environmental surveys in the Sea of Okhotsk, to determine which region is more useful for MPS-based monitoring. The primer universality was not significantly different among the three regions, but the number of sequences deposited in INSDs was markedly larger for the V4-5 region than for the other two regions. The sequence variability was significantly different, with the highest variability in the V1-3 region, followed by the V7-9 region, and the lowest variability in the V4-5 region. The results of the MPS-based environmental surveys showed significantly higher identification power in the V1-3 and V7-9 regions than in the V4-5 region, but no significant difference was detected between the V1-3 and V7-9 regions. We therefore conclude that the V1-3 region will be the most suitable for future MPS-based monitoring of natural eukaryote communities, as the number of sequences deposited in INSDs increases.</p>
<h1>Standardizing metadata and taxonomic identification in metabarcoding studies.</h1>
<p><a href="https://doi.org/10.1186/S13742-015-0074-5">doi:10.1186/S13742-015-0074-5</a></p>
['Data storage', 'Digital object identifiers (DOI)', 'Environmental metadata', 'High-throughput sequencing (HTS)', 'Interactive database', 'Internal transcribed spacer (ITS)', 'Next-generation sequencing', 'Species hypotheses']<p>High-throughput sequencing-based metabarcoding studies produce vast amounts of ecological data, but a lack of consensus on standardization of metadata and how to refer to the species recovered severely hampers reanalysis and comparisons among studies. Here we propose an automated workflow covering data submission, compression, storage and public access to allow easy data retrieval and inter-study communication. Such standardized and readily accessible datasets facilitate data management, taxonomic comparisons and compilation of global metastudies. </p>
<h1>Environmental DNA COI barcoding for quantitative analysis of protists communities: A test using the Nebela collaris complex (Amoebozoa; Arcellinida; Hyalospheniidae).</h1>
<p><a href="https://doi.org/10.1016/J.Ejop.2015.06.005">doi:10.1016/J.Ejop.2015.06.005</a></p>
['Community ecology', 'Environmental DNA', 'Metabarcoding', 'Protist diversity', 'Sphagnum peatlands', 'Testate amoebae']<p>Environmental DNA surveys are used for screening eukaryotic diversity. However, it is unclear how quantitative this approach is and to what extent results from environmental DNA studies can be used for ecological studies requiring quantitative data. Mitochondrial cytochrome oxidase (COI) is used for species-level taxonomic studies of testate amoebae and should allow assessing the community composition from environmental samples, thus bypassing biases due to morphological identification. We tested this using a COI clone library approach and focusing on the Nebela collaris complex. Comparisons with direct microscopy counts showed that the COI clone library diversity data matched the morphologically identified taxa, and that community composition estimates using the two approaches were similar. However, this correlation was improved when microscopy counts were corrected for biovolume. Higher correlation with biovolume-corrected community data suggests that COI clone library data matches the ratio of mitochondria and that within closely-related taxa the density of mitochondria per unit biovolume is approximately constant. Further developments of this metabarcoding approach including quantifying the mitochondrial density among closely-related taxa, experiments on other taxonomic groups and using high throughput sequencing should make if possible to quantitatively estimate community composition of different groups, which would be invaluable for microbial food webs studies. </p>
<h1>Detecting nuisance species using NGST: Methodology shortcomings and possible application in ballast water monitoring.</h1>
<p><a href="https://doi.org/10.1016/J.Marenvres.2015.07.002">doi:10.1016/J.Marenvres.2015.07.002</a></p>
['Ballast water', 'Environmental DNA', 'Next generation sequencing', 'Species assignation', 'Validation']<p>Detecting the presence of potential invasive species in ballast water is a priority for preventing their spread into new environments. Next generation sequencing technologies are being increasingly used for exploring and assessing biodiversity from environmental samples. Here we apply high throughput sequencing from DNA extracted from ballast water (BW) samples employing two different platforms, Ion Torrent and 454, and compare the putative species catalogues from the resulting Operational Taxonomic Units (OTU). Water samples were taken from the RV Polastern ballast tank in five different days between the second and the twentieth navigation day. Pronounced decrease of oxygen concentration and increase of temperature occurred in the BW during this time, coincident with a progressively higher proportion of unassigned OTU and short reads indicating DNA degradation. Discrepancy between platforms for species catalogues was consistent with previously published bias in AT-rich sequences for Ion Torrent platform. Some putative species detected from the two platforms increased in frequency during the Polarstern travel, which suggests they were alive and therefore tolerant to adverse conditions. OTU assigned to the highly invasive red alga Polysiphonia have been detected at low but increasing frequency from the two platforms. Although in this moment NGST could not replace current methods of sampling, sorting and individual taxonomic identification of BW biota, it has potential as an exploratory methodology especially for detecting scarce species. </p>
<h1>Can DNA-Based Ecosystem Assessments Quantify Species Abundance? Testing Primer Bias and Biomass--Sequence Relationships with an Innovative Metabarcoding Protocol.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0130324">doi:10.1371/Journal.Pone.0130324</a></p>
<p>No keywords</p><p>Metabarcoding is an emerging genetic tool to rapidly assess biodiversity in ecosystems. It involves high-throughput sequencing of a standard gene from an environmental sample and comparison to a reference database. However, no consensus has emerged regarding laboratory pipelines to screen species diversity and infer species abundances from environmental samples. In particular, the effect of primer bias and the detection limit for specimens with a low biomass has not been systematically examined, when processing samples in bulk. We developed and tested a DNA metabarcoding protocol that utilises the standard cytochrome c oxidase subunit I (COI) barcoding fragment to detect freshwater macroinvertebrate taxa. DNA was extracted in bulk, amplified in a single PCR step, and purified, and the libraries were directly sequenced in two independent MiSeq runs (300-bp paired-end reads). Specifically, we assessed the influence of specimen biomass on sequence read abundance by sequencing 31 specimens of a stonefly species with known haplotypes spanning three orders of magnitude in biomass (experiment I). Then, we tested the recovery of 52 different freshwater invertebrate taxa of similar biomass using the same standard barcoding primers (experiment II). Each experiment was replicated ten times to maximise statistical power. The results of both experiments were consistent across replicates. We found a distinct positive correlation between species biomass and resulting numbers of MiSeq reads. Furthermore, we reliably recovered 83% of the 52 taxa used to test primer bias. However, sequence abundance varied by four orders of magnitudes between taxa despite the use of similar amounts of biomass. Our metabarcoding approach yielded reliable results for high-throughput assessments. However, the results indicated that primer efficiency is highly species-specific, which would prevent straightforward assessments of species abundance and biomass in a sample. Thus, PCR-based metabarcoding assessments of biodiversity should rely on presence-absence metrics. </p>
<h1>Toward accurate molecular identification of species in complex environmental samples: testing the performance of sequence filtering and clustering methods.</h1>
<p><a href="https://doi.org/10.1002/Ece3.1497">doi:10.1002/Ece3.1497</a></p>
['18S rRNA', 'OTU', 'biodiversity', 'eDNA', 'high-throughput sequencing', 'metabarcoding']<p>Metabarcoding has the potential to become a rapid, sensitive, and effective approach for identifying species in complex environmental samples. Accurate molecular identification of species depends on the ability to generate operational taxonomic units (OTUs) that correspond to biological species. Due to the sometimes enormous estimates of biodiversity using this method, there is a great need to test the efficacy of data analysis methods used to derive OTUs. Here, we evaluate the performance of various methods for clustering length variable 18S amplicons from complex samples into OTUs using a mock community and a natural community of zooplankton species. We compare analytic procedures consisting of a combination of (1) stringent and relaxed data filtering, (2) singleton sequences included and removed, (3) three commonly used clustering algorithms (mothur, UCLUST, and UPARSE), and (4) three methods of treating alignment gaps when calculating sequence divergence. Depending on the combination of methods used, the number of OTUs varied by nearly two orders of magnitude for the mock community (60-5068 OTUs) and three orders of magnitude for the natural community (22-22191 OTUs). The use of relaxed filtering and the inclusion of singletons greatly inflated OTU numbers without increasing the ability to recover species. Our results also suggest that the method used to treat gaps when calculating sequence divergence can have a great impact on the number of OTUs. Our findings are particularly relevant to studies that cover taxonomically diverse species and employ markers such as rRNA genes in which length variation is extensive. </p>
<h1>Divergence thresholds and divergent biodiversity estimates: can metabarcoding reliably describe zooplankton communities?</h1>
<p><a href="https://doi.org/10.1002/Ece3.1485">doi:10.1002/Ece3.1485</a></p>
['Biodiversity', 'high-throughput sequencing', 'metabarcoding', 'mock community', 'nSSU', 'zooplankton']<p>DNA metabarcoding is a promising method for describing communities and estimating biodiversity. This approach uses high-throughput sequencing of targeted markers to identify species in a complex sample. By convention, sequences are clustered at a predefined sequence divergence threshold (often 3%) into operational taxonomic units (OTUs) that serve as a proxy for species. However, variable levels of interspecific marker variation across taxonomic groups make clustering sequences from a phylogenetically diverse dataset into OTUs at a uniform threshold problematic. In this study, we use mock zooplankton communities to evaluate the accuracy of species richness estimates when following conventional protocols to cluster hypervariable sequences of the V4 region of the small subunit ribosomal RNA gene (18S) into OTUs. By including individually tagged single specimens and "populations" of various species in our communities, we examine the impact of intra- and interspecific diversity on OTU clustering. Communities consisting of single individuals per species generated a correspondence of 59-84% between OTU number and species richness at a 3% divergence threshold. However, when multiple individuals per species were included, the correspondence between OTU number and species richness dropped to 31-63%. Our results suggest that intraspecific variation in this marker can often exceed 3%, such that a single species does not always correspond to one OTU. We advocate the need to apply group-specific divergence thresholds when analyzing complex and taxonomically diverse communities, but also encourage the development of additional filtering steps that allow identification of artifactual rRNA gene sequences or pseudogenes that may generate spurious OTUs. </p>
<h1>Environmental Monitoring: Inferring the Diatom Index from Next-Generation Sequencing Data.</h1>
<p><a href="https://doi.org/10.1021/Es506158M">doi:10.1021/Es506158M</a></p>
<p>No keywords</p><p>Diatoms are widely used as bioindicators for the assessment of water quality in rivers and streams. Classically, the diatom biotic indices are based on the relative abundance of morphologically identified species weighted by their autoecological value. Obtaining such indices is time-consuming, costly, and requires excellent taxonomic expertise, which is not always available. Here we tested the possibility to overcome these limitations using a next-generation sequencing (NGS) approach to identify and quantify diatoms found in environmental DNA and RNA samples. We analyzed 27 river sites in the Geneva area (Switzerland), in order to compare the values of the Swiss Diatom Index (DI-CH) computed either by microscopic quantification of diatom species or directly from NGS data. Despite gaps in the reference database and variations in relative abundance of analyzed species, the diatom index shows a significant correlation between morphological and molecular data indicating similar biological quality status for the majority of sites. This proof-of-concept study demonstrates the potential of the NGS approach for identification and quantification of diatoms in environmental samples, opening new avenues toward the routine application of genetic tools for bioassessment and biomonitoring of aquatic ecosystems. </p>
<h1>Microbial metagenomics in the Baltic Sea: Recent advancements and prospects for environmental monitoring.</h1>
<p><a href="https://doi.org/10.1007/S13280-015-0663-7">doi:10.1007/S13280-015-0663-7</a></p>
<p>No keywords</p><p>Metagenomics refers to the analysis of DNA from a whole community. Metagenomic sequencing of environmental DNA has greatly improved our knowledge of the identity and function of microorganisms in aquatic, terrestrial, and human biomes. Although open oceans have been the primary focus of studies on aquatic microbes, coastal and brackish ecosystems are now being surveyed. Here, we review so far published studies on microbes in the Baltic Sea, one of the world's largest brackish water bodies, using high throughput sequencing of environmental DNA and RNA. Collectively the data illustrate that Baltic Sea microbes are unique and highly diverse, and well adapted to this brackish-water ecosystem, findings that represent a novel base-line knowledge necessary for monitoring purposes and a sustainable management. More specifically, the data relate to environmental drivers for microbial community composition and function, assessments of the microbial biodiversity, adaptations and role of microbes in the nitrogen cycle, and microbial genome assembly from metagenomic sequences. With these discoveries as background, prospects of using metagenomics for Baltic Sea environmental monitoring are discussed. </p>
<h1>Not all are free-living: high-throughput DNA metabarcoding reveals a diverse community of protists parasitizing soil metazoa.</h1>
<p><a href="https://doi.org/10.1111/Mec.13238">doi:10.1111/Mec.13238</a></p>
['454 Metabarcoding', 'apicomplexa', 'high-throughput sequencing', 'potential parasites', 'soil metazoan parasites', 'soil protists']<p>Protists, the most diverse eukaryotes, are largely considered to be free-living bacterivores, but vast numbers of taxa are known to parasitize plants or animals. High-throughput sequencing (HTS) approaches now commonly replace cultivation-based approaches in studying soil protists, but insights into common biases associated with this method are limited to aquatic taxa and samples. We created a mock community of common free-living soil protists (amoebae, flagellates, ciliates), extracted DNA and amplified it in the presence of metazoan DNA using 454 HTS. We aimed at evaluating whether HTS quantitatively reveals true relative abundances of soil protists and at investigating whether the expected protist community structure is altered by the co-amplification of metazoan-associated protist taxa. Indeed, HTS revealed fundamentally different protist communities from those expected. Ciliate sequences were highly over-represented, while those of most amoebae and flagellates were under-represented or totally absent. These results underpin the biases introduced by HTS that prevent reliable quantitative estimations of free-living protist communities. Furthermore, we detected a wide range of nonadded protist taxa probably introduced along with metazoan DNA, which altered the protist community structure. Among those, 20 taxa most closely resembled parasitic, often pathogenic taxa. Therewith, we provide the first HTS data in support of classical observational studies that showed that potential protist parasites are hosted by soil metazoa. Taken together, profound differences in amplification success between protist taxa and an inevitable co-extraction of protist taxa parasitizing soil metazoa obscure the true diversity of free-living soil protist communities. </p>
<h1>obitools: a unix-inspired software package for DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12428">doi:10.1111/1755-0998.12428</a></p>
['PCR errors', 'biodiversity', 'next-generation sequencing', 'sequence analysis', 'taxonomic annotation']<p>DNA metabarcoding offers new perspectives in biodiversity research. This recently developed approach to ecosystem study relies heavily on the use of next-generation sequencing (NGS) and thus calls upon the ability to deal with huge sequence data sets. The obitools package satisfies this requirement thanks to a set of programs specifically designed for analysing NGS data in a DNA metabarcoding context. Their capacity to filter and edit sequences while taking into account taxonomic annotation helps to set up tailor-made analysis pipelines for a broad range of DNA metabarcoding applications, including biodiversity surveys or diet analyses. The obitools package is distributed as an open source software available on the following website: http://metabarcoding.org/obitools. A Galaxy wrapper is available on the GenOuest core facility toolshed: http://toolshed.genouest.org. </p>
<h1>Metabarcoding of benthic eukaryote communities predicts the ecological condition of estuaries.</h1>
<p><a href="https://doi.org/S0269-7491(15)00177-3">doi:S0269-7491(15)00177-3</a></p>
['18S rRNA', 'Biomonitoring', 'DNA', 'Eukaryotes', 'High-throughput sequencing', 'Indicator taxa', 'Metabarcoding', 'Sediments', 'Threshold analysis']<p>DNA-derived measurements of biological composition have the potential to produce data covering all of life, and provide a tantalizing proposition for researchers and managers. We used metabarcoding to compare benthic eukaryote composition from five estuaries of varying condition. In contrast to traditional studies, we found biotic richness was greatest in the most disturbed estuary, with this being due to the large volume of extraneous material (i.e. run-off from aquaculture, agriculture and other catchment activities) being deposited in the system. In addition, we found strong correlations between composition and a number of environmental variables, including nutrients, pH and turbidity. A wide range of taxa responded to these environmental gradients, providing new insights into their sensitivities to natural and anthropogenic stressors. Metabarcoding has the capacity to bolster current monitoring techniques, enabling the decisions regarding ecological condition to be based on a more holistic view of biodiversity. </p>
<h1>The use of genus-specific amplicon pyrosequencing to assess phytophthora species diversity using eDNA from soil and water in Northern Spain.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0119311">doi:10.1371/Journal.Pone.0119311</a></p>
<p>No keywords</p><p>Phytophthora is one of the most important and aggressive plant pathogenic genera in agriculture and forestry. Early detection and identification of its pathways of infection and spread are of high importance to minimize the threat they pose to natural ecosystems. eDNA was extracted from soil and water from forests and plantations in the north of Spain. Phytophthora-specific primers were adapted for use in high-throughput Sequencing (HTS). Primers were tested in a control reaction containing eight Phytophthora species and applied to water and soil eDNA samples from northern Spain. Different score coverage threshold values were tested for optimal Phytophthora species separation in a custom-curated database and in the control reaction. Clustering at 99% was the optimal criteria to separate most of the Phytophthora species. Multiple Molecular Operational Taxonomic Units (MOTUs) corresponding to 36 distinct Phytophthora species were amplified in the environmental samples. Pyrosequencing of amplicons from soil samples revealed low Phytophthora diversity (13 species) in comparison with the 35 species detected in water samples. Thirteen of the MOTUs detected in rivers and streams showed no close match to sequences in international sequence databases, revealing that eDNA pyrosequencing is a useful strategy to assess Phytophthora species diversity in natural ecosystems. </p>
<h1>PhytoREF: a reference database of the plastidial 16S rRNA gene of photosynthetic eukaryotes with curated taxonomy.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12401">doi:10.1111/1755-0998.12401</a></p>
['high-throughput sequencing', 'metabarcoding', 'photosynthesis', 'phytoplankton', 'plastidial 16S rRNA gene', 'protists']<p>Photosynthetic eukaryotes have a critical role as the main producers in most ecosystems of the biosphere. The ongoing environmental metabarcoding revolution opens the perspective for holistic ecosystems biological studies of these organisms, in particular the unicellular microalgae that often lack distinctive morphological characters and have complex life cycles. To interpret environmental sequences, metabarcoding necessarily relies on taxonomically curated databases containing reference sequences of the targeted gene (or barcode) from identified organisms. To date, no such reference framework exists for photosynthetic eukaryotes. In this study, we built the PhytoREF database that contains 6490 plastidial 16S rDNA reference sequences that originate from a large diversity of eukaryotes representing all known major photosynthetic lineages. We compiled 3333 amplicon sequences available from public databases and 879 sequences extracted from plastidial genomes, and generated 411 novel sequences from cultured marine microalgal strains belonging to different eukaryotic lineages. A total of 1867 environmental Sanger 16S rDNA sequences were also included in the database. Stringent quality filtering and a phylogeny-based taxonomic classification were applied for each 16S rDNA sequence. The database mainly focuses on marine microalgae, but sequences from land plants (representing half of the PhytoREF sequences) and freshwater taxa were also included to broaden the applicability of PhytoREF to different aquatic and terrestrial habitats. PhytoREF, accessible via a web interface (http://phytoref.fr), is a new resource in molecular ecology to foster the discovery, assessment and monitoring of the diversity of photosynthetic eukaryotes using high-throughput sequencing. </p>
<h1>Reconstructing long-term human impacts on plant communities: an ecological approach based on lake sediment DNA.</h1>
<p><a href="https://doi.org/10.1111/Mec.13136">doi:10.1111/Mec.13136</a></p>
['anthropocene', 'environmental DNA', 'landscape history', 'metabarcoding', 'paleoecology', 'pastoralism']<p>Paleoenvironmental studies are essential to understand biodiversity changes over long timescales and to assess the relative importance of anthropogenic and environmental factors. Sedimentary ancient DNA (sedaDNA) is an emerging tool in the field of paleoecology and has proven to be a complementary approach to the use of pollen and macroremains for investigating past community changes. SedaDNA-based reconstructions of ancient environments often rely on indicator taxa or expert knowledge, but quantitative ecological analyses might provide more objective information. Here, we analysed sedaDNA to investigate plant community trajectories in the catchment of a high-elevation lake in the Alps over the last 6400 years. We combined data on past and present plant species assemblages along with sedimentological and geochemical records to assess the relative impact of human activities through pastoralism, and abiotic factors (temperature and soil evolution). Over the last 6400 years, we identified significant variation in plant communities, mostly related to soil evolution and pastoral activities. An abrupt vegetational change corresponding to the establishment of an agropastoral landscape was detected during the Late Holocene, approximately 4500 years ago, with the replacement of mountain forests and tall-herb communities by heathlands and grazed lands. Our results highlight the importance of anthropogenic activities in mountain areas for the long-term evolution of local plant assemblages. SedaDNA data, associated with other paleoenvironmental proxies and present plant assemblages, appear to be a relevant tool for reconstruction of plant cover history. Their integration, in conjunction with classical tools, offers interesting perspectives for a better understanding of long-term ecosystem dynamics under the influence of human-induced and environmental drivers. </p>
<h1>Environmental genes and genomes: understanding the differences and challenges in the approaches and software for their analyses.</h1>
<p><a href="https://doi.org/10.1093/Bib/Bbv001">doi:10.1093/Bib/Bbv001</a></p>
['DNA metabarcoding', 'environment', 'genome', 'metagenomics', 'software development']<p>DNA-based taxonomic and functional profiling is widely used for the characterization of organismal communities across a rapidly increasing array of research areas that include the role of microbiomes in health and disease, biomonitoring, and estimation of both microbial and metazoan species richness. Two principal approaches are currently used to assign taxonomy to DNA sequences: DNA metabarcoding and metagenomics. When initially developed, each of these approaches mandated their own particular methods for data analysis; however, with the development of high-throughput sequencing (HTS) techniques they have begun to share many aspects in data set generation and processing. In this review we aim to define the current characteristics, goals and boundaries of each field, and describe the different software used for their analysis. We argue that an appreciation of the potential and limitations of each method can help underscore the improvements required by each field so as to better exploit the richness of current HTS-based data sets. </p>
<h1>Metabarcoding approach for the ballast water surveillance--an advantageous solution or an awkward challenge?</h1>
<p><a href="https://doi.org/S0025-326X(15)00024-7">doi:S0025-326X(15)00024-7</a></p>
['Ballast water management convention', 'COI', 'En-route observation', 'High-throughput sequencing', 'Invasive species', 'RuBisCO']<p>Transfer of organisms with ships' ballast water is recognized as a major pathway of non-indigenous species introduction and addressed in a few recent legislative initiatives. Among other they imply scientific and technical research and monitoring to be conducted in a efficient and reliable way. The recent development of DNA barcoding and metabarcoding technologies opens new opportunities for biodiversity and biosecurity surveillance. In the current study, the performance of metabarcoding approach was assessed in comparison to the conventional (visual) observations, during the en route experimental ballast water survey. Opportunities and limitations of the molecular method were identified from taxonomical datasets rendered by two molecular markers of different degree of universality - the universal cytochrome oxydase sub-unit I gene and a fragment of RuBisCO gene. The cost-efficacy and possible improvements of these methods are discussed for the further successful development and implementation of the approach in ballast water control and NIS surveillance. </p>
<h1>Deciphering diversity and ecological function from marine metagenomes.</h1>
<p>No DOI</p><p>No keywords</p><p>Metagenomic sequencing now represents a common, powerful approach for investigating diversity and functional relationships in marine ecosystems. High-throughput datasets generated from random fragments of environmental DNA can provide a less biased view of organismal abundance (versus PCR-based amplicon sequencing) and enable novel exploration of microbial genomes by recovering genome assemblies from uncultured species, identifying ecological functions, and reconstructing metabolic pathways. This review highlights the current state of knowledge in marine metagenomics, focusing on biological insights gained from recent environmental studies and detailing commonly employed methods for data collection and analysis. </p>
<h1>Next-generation environmental diversity surveys of foraminifera: preparing the future.</h1>
<p>No DOI</p><p>No keywords</p><p>Foraminifera are commonly defined as marine testate protists, and their diversity is mainly assessed on the basis of the morphology of their agglutinated or mineralized tests. Diversity surveys based on environmental DNA (eDNA) have dramatically changed this view by revealing an unexpected diversity of naked and organic-walled lineages as well as detecting foraminiferal lineages in soil and freshwater environments. Moreover, single-cell analyses have allowed discrimination among genetically distinctive types within almost every described morphospecies. In view of these studies, the foraminiferal diversity appeared to be largely underestimated, but its accurate estimation was impeded by the low speed and coverage of a cloning-based eDNA approach. With the advent of high-throughput sequencing (HTS) technologies, these limitations disappeared in favor of exhaustive descriptions of foraminiferal diversity in numerous samples. Yet, the biases and errors identified in early HTS studies raised some questions about the accuracy of HTS data and their biological interpretation. Among the most controversial issues affecting the reliability of HTS diversity estimates are (1) the impact of technical and biological biases, (2) the sensitivity and specificity of taxonomic sequence assignment, (3) the ability to distinguish rare species, and (4) the quantitative interpretation of HTS data. Here, we document the lessons learned from previous HTS surveys and present the current advances and applications focusing on foraminiferal eDNA. We discuss the problems associated with HTS approaches and predict the future trends and avenues that hold promises for surveying foraminiferal diversity accurately and efficiently. </p>
<h1>Multiplex sequencing of pooled mitochondrial genomes-a crucial step toward biodiversity analysis using mito-metagenomics.</h1>
<p><a href="https://doi.org/10.1093/Nar/Gku917">doi:10.1093/Nar/Gku917</a></p>
<p>No keywords</p><p>The advent in high-throughput-sequencing (HTS) technologies has revolutionized conventional biodiversity research by enabling parallel capture of DNA sequences possessing species-level diagnosis. However, polymerase chain reaction (PCR)-based implementation is biased by the efficiency of primer binding across lineages of organisms. A PCR-free HTS approach will alleviate this artefact and significantly improve upon the multi-locus method utilizing full mitogenomes. Here we developed a novel multiplex sequencing and assembly pipeline allowing for simultaneous acquisition of full mitogenomes from pooled animals without DNA enrichment or amplification. By concatenating assemblies from three de novo assemblers, we obtained high-quality mitogenomes for all 49 pooled taxa, with 36 species >15 kb and the remaining >10 kb, including 20 complete mitogenomes and nearly all protein coding genes (99.6%). The assembly quality was carefully validated with Sanger sequences, reference genomes and conservativeness of protein coding genes across taxa. The new method was effective even for closely related taxa, e.g. three Drosophila spp., demonstrating its broad utility for biodiversity research and mito-phylogenomics. Finally, the in silico simulation showed that by recruiting multiple mito-loci, taxon detection was improved at a fixed sequencing depth. Combined, these results demonstrate the plausibility of a multi-locus mito-metagenomics approach as the next phase of the current single-locus metabarcoding method. </p>
<h1>Intracellular diversity of the V4 and V9 regions of the 18S rRNA in marine protists (radiolarians) assessed by high-throughput sequencing.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0104297">doi:10.1371/Journal.Pone.0104297</a></p>
<p>No keywords</p><p>Metabarcoding is a powerful tool for exploring microbial diversity in the environment, but its accurate interpretation is impeded by diverse technical (e.g. PCR and sequencing errors) and biological biases (e.g. intra-individual polymorphism) that remain poorly understood. To help interpret environmental metabarcoding datasets, we investigated the intracellular diversity of the V4 and V9 regions of the 18S rRNA gene from Acantharia and Nassellaria (radiolarians) using 454 pyrosequencing. Individual cells of radiolarians were isolated, and PCRs were performed with generalist primers to amplify the V4 and V9 regions. Different denoising procedures were employed to filter the pyrosequenced raw amplicons (Acacia, AmpliconNoise, Linkage method). For each of the six isolated cells, an average of 541 V4 and 562 V9 amplicons assigned to radiolarians were obtained, from which one numerically dominant sequence and several minor variants were found. At the 97% identity, a diversity metrics commonly used in environmental surveys, up to 5 distinct OTUs were detected in a single cell. However, most amplicons grouped within a single OTU whereas other OTUs contained very few amplicons. Different analytical methods provided evidence that most minor variants forming different OTUs correspond to PCR and sequencing artifacts. Duplicate PCR and sequencing from the same DNA extract of a single cell had only 9 to 16% of unique amplicons in common, and alignment visualization of V4 and V9 amplicons showed that most minor variants contained substitutions in highly-conserved regions. We conclude that intracellular variability of the 18S rRNA in radiolarians is very limited despite its multi-copy nature and the existence of multiple nuclei in these protists. Our study recommends some technical guidelines to conservatively discard artificial amplicons from metabarcoding datasets, and thus properly assess the diversity and richness of protists in the environment. </p>
<h1>An Illumina metabarcoding pipeline for fungi.</h1>
<p><a href="https://doi.org/10.1002/Ece3.1107">doi:10.1002/Ece3.1107</a></p>
['community ecology', 'data pruning', 'high throughput', 'internal transcribed spacer rDNA', 'next-generation sequencing']<p>High-throughput metabarcoding studies on fungi and other eukaryotic microorganisms are rapidly becoming more frequent and more complex, requiring researchers to handle ever increasing amounts of raw sequence data. Here, we provide a flexible pipeline for pruning and analyzing fungal barcode (ITS rDNA) data generated as paired-end reads on Illumina MiSeq sequencers. The pipeline presented includes specific steps fine-tuned for ITS, that are mostly missing from pipelines developed for prokaryotes. It (1) employs state of the art programs and follows best practices in fungal high-throughput metabarcoding; (2) consists of modules and scripts easily modifiable by the user to ensure maximum flexibility with regard to specific needs of a project or future methodological developments; and (3) is straightforward to use, also in classroom settings. We provide detailed descriptions and revision techniques for each step, thus giving the user maximum control over data treatment and avoiding a black-box approach. Employing this pipeline will improve and speed up the tedious and error-prone process of cleaning fungal Illumina metabarcoding data. </p>
<h1>Efficient and sensitive identification and quantification of airborne pollen using next-generation DNA sequencing.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12288">doi:10.1111/1755-0998.12288</a></p>
['DNA metabarcoding', 'molecular identification', 'next-generation sequencing', 'pollen allergy', 'pollen monitoring']<p>Pollen monitoring is an important and widely used tool in allergy research and creation of awareness in pollen-allergic patients. Current pollen monitoring methods are microscope-based, labour intensive and cannot identify pollen to the genus level in some relevant allergenic plant groups. Therefore, a more efficient, cost-effective and sensitive method is needed. Here, we present a method for identification and quantification of airborne pollen using DNA sequencing. Pollen is collected from ambient air using standard techniques. DNA is extracted from the collected pollen, and a fragment of the chloroplast gene trnL is amplified using PCR. The PCR product is subsequently sequenced on a next-generation sequencing platform (Ion Torrent). Amplicon molecules are sequenced individually, allowing identification of different sequences from a mixed sample. We show that this method provides an accurate qualitative and quantitative view of the species composition of samples of airborne pollen grains. We also show that it correctly identifies the individual grass genera present in a mixed sample of grass pollen, which cannot be achieved using microscopic pollen identification. We conclude that our method is more efficient and sensitive than current pollen monitoring techniques and therefore has the potential to increase the throughput of pollen monitoring. </p>
<h1>Next generation restoration genetics: applications and opportunities.</h1>
<p><a href="https://doi.org/10.1016/J.Tplants.2014.03.011">doi:10.1016/J.Tplants.2014.03.011</a></p>
['adaptation', 'genotyping by sequencing', 'metabarcoding', 'next generation sequencing', 'population genomics', 'restoration ecology']<p>Restoration ecology is a young scientific discipline underpinning improvements in the rapid global expansion of ecological restoration. The application of molecular tools over the past 20 years has made an important contribution to understanding genetic factors influencing ecological restoration success. Here we illustrate how recent advances in next generation sequencing (NGS) methods are revolutionising the practical contribution of genetics to restoration. Novel applications include a dramatically enhanced capacity to measure adaptive variation for optimal seed sourcing, high-throughput assessment and monitoring of natural and restored biological communities aboveground and belowground, and gene expression analysis as a measure of genetic resilience of restored populations. Challenges remain in data generation, handling and analysis, and how best to apply NGS for practical outcomes in restoration. </p>
<h1>Environmental monitoring through protist next-generation sequencing metabarcoding: assessing the impact of fish farming on benthic foraminifera communities.</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12261">doi:10.1111/1755-0998.12261</a></p>
['DNA barcoding', 'bioinformatics/phyloinformatics', 'ecotoxicology', 'environmental DNA', 'protists']<p>The measurement of species diversity represents a powerful tool for assessing the impacts of human activities on marine ecosystems. Traditionally, the impact of fish farming on the coastal environment is evaluated by monitoring the dynamics of macrobenthic infaunal populations. However, taxonomic sorting and morphology-based identification of the macrobenthos demand highly trained specialists and are extremely time-consuming and costly, making it unsuitable for large-scale biomonitoring efforts involving numerous samples. Here, we propose to alleviate this laborious task by developing protist metabarcoding tools based on next-generation sequencing (NGS) of environmental DNA and RNA extracted from sediment samples. In this study, we analysed the response of benthic foraminiferal communities to the variation of environmental gradients associated with salmon farms in Scotland. We investigated the foraminiferal diversity based on ribosomal minibarcode sequences generated by the Illumina NGS technology. We compared the molecular data with morphospecies counts and with environmental gradients, including distance to cages and redox used as a proxy for sediment oxygenation. Our study revealed high variations between foraminiferal communities collected in the vicinity of fish farms and at distant locations. We found evidence for species richness decrease in impacted sites, especially visible in the RNA data. We also detected some candidate bioindicator foraminiferal species. Based on this proof-of-concept study, we conclude that NGS metabarcoding using foraminifera and other protists has potential to become a new tool for surveying the impact of aquaculture and other industrial activities in the marine environment.</p>
<h1>Ancient DNA sheds new light on the Svalbard foraminiferal fossil record of the last millennium.</h1>
<p><a href="https://doi.org/10.1111/Gbi.12087">doi:10.1111/Gbi.12087</a></p>
<p>No keywords</p><p>Recent palaeogenetic studies have demonstrated the occurrence of preserved ancient DNA (aDNA) in various types of fossilised material. Environmental aDNA sequences assigned to modern species have been recovered from marine sediments dating to the Pleistocene. However, the match between the aDNA and the fossil record still needs to be evaluated for the environmental DNA approaches to be fully exploited. Here, we focus on foraminifera in sediments up to one thousand years old retrieved from the Hornsund fjord (Svalbard). We compared the diversity of foraminiferal microfossil assemblages with the diversity of aDNA sequenced from subsurface sediment samples using both cloning and high-throughput sequencing (HTS). Our study shows that 57% of the species archived in the fossil record were also detected in the aDNA data. However, the relative abundance of aDNA sequence reads and fossil specimens differed considerably. We also found a limited match between the stratigraphic occurrence of some fossil species and their aDNA sequences, especially in the case of rare taxa. The aDNA data comprised a high proportion of non-fossilised monothalamous species, which are known to dominate in modern foraminiferal communities of the Svalbard region. Our results confirm the relevance of HTS for studying past micro-eukaryotic diversity and provide insight into its ability to reflect fossil assemblages. Palaeogenetic studies including aDNA analyses of non-fossilised groups expand the range of palaeoceanographical proxies and therefore may increase the accuracy of palaeoenvironmental reconstructions. </p>
<h1>Profiling the diversity of microbial communities with single-strand conformation polymorphism (SSCP).</h1>
<p><a href="https://doi.org/10.1007/978-1-62703-712-9_6">doi:10.1007/978-1-62703-712-9_6</a></p>
<p>No keywords</p><p>Genetic fingerprinting techniques for microbial community analysis have evolved over the last decade into standard applications for efficient and fast differentiation of microbial communities based on their diversity. These techniques commonly analyze the diversity of PCR products amplified from extracted environmental DNA usually utilizing primers hybridizing to suspected conserved regions of the targeted genes. In comparison to the more commonly applied terminal restriction fragment length polymorphism (TRFLP) or denaturing gradient gel electrophoresis (DGGE) techniques, the here-described single-strand conformation polymorphism (SSCP) fingerprinting technique features some advantageous key characteristics. (1) Primers for the polymerase chain reaction (PCR) do only need minimal 5'-end alterations; (2) SSCP is adaptable to high throughput applications in automated sequencers; and (3) a second dimension in the SSCP gel electrophoresis can be implemented to obtain high resolution 2D gels. One central key requirement for SSCP gel electrophoresis is a tight temperature control. Gels that run at different temperatures will produce entirely different fingerprints. This can be exploited for an improved analysis of highly diverse communities by running the same template at different temperatures or by 2D-SSCP gel electrophoresis. </p>
<h1>Fifty thousand years of Arctic vegetation and megafaunal diet.</h1>
<p><a href="https://doi.org/10.1038/Nature12921">doi:10.1038/Nature12921</a></p>
<p>No keywords</p><p>Although it is generally agreed that the Arctic flora is among the youngest and least diverse on Earth, the processes that shaped it are poorly understood. Here we present 50 thousand years (kyr) of Arctic vegetation history, derived from the first large-scale ancient DNA metabarcoding study of circumpolar plant diversity. For this interval we also explore nematode diversity as a proxy for modelling vegetation cover and soil quality, and diets of herbivorous megafaunal mammals, many of which became extinct around 10 kyr bp (before present). For much of the period investigated, Arctic vegetation consisted of dry steppe-tundra dominated by forbs (non-graminoid herbaceous vascular plants). During the Last Glacial Maximum (25-15 kyr bp), diversity declined markedly, although forbs remained dominant. Much changed after 10 kyr bp, with the appearance of moist tundra dominated by woody plants and graminoids. Our analyses indicate that both graminoids and forbs would have featured in megafaunal diets. As such, our findings question the predominance of a Late Quaternary graminoid-dominated Arctic mammoth steppe. </p>
<h1>Using environmental DNA to census marine fishes in a large mesocosm.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0086175">doi:10.1371/Journal.Pone.0086175</a></p>
<p>No keywords</p><p>The ocean is a soup of its resident species' genetic material, cast off in the forms of metabolic waste, shed skin cells, or damaged tissue. Sampling this environmental DNA (eDNA) is a potentially powerful means of assessing whole biological communities, a significant advance over the manual methods of environmental sampling that have historically dominated marine ecology and related fields. Here, we estimate the vertebrate fauna in a 4.5-million-liter mesocosm aquarium tank at the Monterey Bay Aquarium of known species composition by sequencing the eDNA from its constituent seawater. We find that it is generally possible to detect mitochondrial DNA of bony fishes sufficient to identify organisms to taxonomic family- or genus-level using a 106 bp fragment of the 12S ribosomal gene. Within bony fishes, we observe a low false-negative detection rate, although we did not detect the cartilaginous fishes or sea turtles present with this fragment. We find that the rank abundance of recovered eDNA sequences correlates with the abundance of corresponding species' biomass in the mesocosm, but the data in hand do not allow us to develop a quantitative relationship between biomass and eDNA abundance. Finally, we find a low false-positive rate for detection of exogenous eDNA, and we were able to diagnose non-native species' tissue in the food used to maintain the mesocosm, underscoring the sensitivity of eDNA as a technique for community-level ecological surveys. We conclude that eDNA has substantial potential to become a core tool for environmental monitoring, but that a variety of challenges remain before reliable quantitative assessments of ecological communities in the field become possible. </p>
<h1>Improved inference of taxonomic richness from environmental DNA.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0071974">doi:10.1371/Journal.Pone.0071974</a></p>
<p>No keywords</p><p>Accurate estimation of biological diversity in environmental DNA samples using high-throughput amplicon pyrosequencing must account for errors generated by PCR and sequencing. We describe a novel approach to distinguish the underlying sequence diversity in environmental DNA samples from errors that uses information on the abundance distribution of similar sequences across independent samples, as well as the frequency and diversity of sequences within individual samples. We have further refined this approach into a bioinformatics pipeline, Amplicon Pyrosequence Denoising Program (APDP) that is able to process raw sequence datasets into a set of validated sequences in formats compatible with commonly used downstream analyses packages. We demonstrate, by sequencing complex environmental samples and mock communities, that APDP is effective for removing errors from deeply sequenced datasets comprising biological and technical replicates, and can efficiently denoise single-sample datasets. APDP provides more conservative diversity estimates for complex datasets than other approaches; however, for some applications this may provide a more accurate and appropriate level of resolution, and result in greater confidence that returned sequences reflect the diversity of the underlying sample. </p>
<h1>Quantifying sequence proportions in a DNA-based diet study using Ion Torrent amplicon sequencing: which counts count?</h1>
<p><a href="https://doi.org/10.1111/1755-0998.12103">doi:10.1111/1755-0998.12103</a></p>
<p>No keywords</p><p>A goal of many environmental DNA barcoding studies is to infer quantitative information about relative abundances of different taxa based on sequence read proportions generated by high-throughput sequencing. However, potential biases associated with this approach are only beginning to be examined. We sequenced DNA amplified from faeces (scats) of captive harbour seals (Phoca vitulina) to investigate whether sequence counts could be used to quantify the seals' diet. Seals were fed fish in fixed proportions, a chordate-specific mitochondrial 16S marker was amplified from scat DNA and amplicons sequenced using an Ion Torrent PGM™. For a given set of bioinformatic parameters, there was generally low variability between scat samples in proportions of prey species sequences recovered. However, proportions varied substantially depending on sequencing direction, level of quality filtering (due to differences in sequence quality between species) and minimum read length considered. Short primer tags used to identify individual samples also influenced species proportions. In addition, there were complex interactions between factors; for example, the effect of quality filtering was influenced by the primer tag and sequencing direction. Resequencing of a subset of samples revealed some, but not all, biases were consistent between runs. Less stringent data filtering (based on quality scores or read length) generally produced more consistent proportional data, but overall proportions of sequences were very different than dietary mass proportions, indicating additional technical or biological biases are present. Our findings highlight that quantitative interpretations of sequence proportions generated via high-throughput sequencing will require careful experimental design and thoughtful data analysis.</p>
<h1>Unveiling the diet of elusive rainforest herbivores in next generation sequencing era? The tapir as a case study.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0060799">doi:10.1371/Journal.Pone.0060799</a></p>
<p>No keywords</p><p>Characterizing the trophic relationships between large herbivores and the outstanding plant diversity in rainforest is a major challenge because of their elusiveness. This is crucial to understand the role of these herbivores in the functioning of the rainforest ecosystems. We tested a non-invasive approach based on the high-throughput sequencing of environmental samples using small plant plastid sequences (the trnL P6 loop) and ribosomal ITS1 primers, referred to as DNA metabarcoding, to investigate the diet of the largest neotropical herbivore, the lowland tapir. Sequencing was performed on plant DNA extracted from tapir faeces collected at the Nouragues station, a protected area of French Guiana. In spite of a limited sampling, our approach reliably provided information about the lowland tapir's diet at this site. Indeed, 95.1% and 74.4% of the plant families and genera identified thanks to the trnL P6 loop, respectively, matched with taxa already known to be consumed by tapirs. With this approach we were able to show that two families and eight new genera are also consumed by the lowland tapir. The taxonomic resolution of this method is limited to the plant family and genera. Complementary barcodes, such as a small portion of ITS1, can be used to efficiently narrow identifications down to the species in some problematic families. We will discuss the remaining limitations of this approach and how useful it is at this stage to unravel the diet of elusive rainforest herbivores and better understand their role as engineers of the ecosystem.</p>
<h1>High-throughput analysis of ammonia oxidiser community composition via a novel, amoA-based functional gene array.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0051542">doi:10.1371/Journal.Pone.0051542</a></p>
<p>No keywords</p><p>Advances in microbial ecology research are more often than not limited by the capabilities of available methodologies. Aerobic autotrophic nitrification is one of the most important and well studied microbiological processes in terrestrial and aquatic ecosystems. We have developed and validated a microbial diagnostic microarray based on the ammonia-monooxygenase subunit A (amoA) gene, enabling the in-depth analysis of the community structure of bacterial and archaeal ammonia oxidisers. The amoA microarray has been successfully applied to analyse nitrifier diversity in marine, estuarine, soil and wastewater treatment plant environments. The microarray has moderate costs for labour and consumables and enables the analysis of hundreds of environmental DNA or RNA samples per week per person. The array has been thoroughly validated with a range of individual and complex targets (amoA clones and environmental samples, respectively), combined with parallel analysis using traditional sequencing methods. The moderate cost and high throughput of the microarray makes it possible to adequately address broader questions of the ecology of microbial ammonia oxidation requiring high sample numbers and high resolution of the community composition.</p>
<h1>Testing three pipelines for 18S rDNA-based metabarcoding of soil faunal diversity.</h1>
<p><a href="https://doi.org/10.1007/S11427-012-4423-7">doi:10.1007/S11427-012-4423-7</a></p>
<p>No keywords</p><p>A number of basic and applied questions in ecology and environmental management require the characterization of soil and leaf litter faunal diversity. Recent advances in high-throughput sequencing of barcode-gene amplicons ('metabarcoding') have made it possible to survey biodiversity in a robust and efficient way. However, one obstacle to the widespread adoption of this technique is the need to choose amongst many candidates for bioinformatic processing of the raw sequencing data. We compare three candidate pipelines for the processing of 18S small subunit rDNA metabarcode data from solid substrates: (i) USEARCH/CROP, (ii) Denoiser/UCLUST, and (iii) OCTUPUS. The three pipelines produced reassuringly similar and highly correlated assessments of community composition that are dominated by taxa known to characterize the sampled environments. However, OCTUPUS appears to inflate phylogenetic diversity, because of higher sequence noise. We therefore recommend either the USEARCH/CROP or Denoiser/UCLUST pipelines, both of which can be run within the QIIME (Quantitative Insights Into Microbial Ecology) environment.</p>
<h1>Incorporating 16S gene copy number information improves estimates of microbial diversity and abundance.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pcbi.1002743">doi:10.1371/Journal.Pcbi.1002743</a></p>
<p>No keywords</p><p>The abundance of different SSU rRNA ("16S") gene sequences in environmental samples is widely used in studies of microbial ecology as a measure of microbial community structure and diversity. However, the genomic copy number of the 16S gene varies greatly - from one in many species to up to 15 in some bacteria and to hundreds in some microbial eukaryotes. As a result of this variation the relative abundance of 16S genes in environmental samples can be attributed both to variation in the relative abundance of different organisms, and to variation in genomic 16S copy number among those organisms. Despite this fact, many studies assume that the abundance of 16S gene sequences is a surrogate measure of the relative abundance of the organisms containing those sequences. Here we present a method that uses data on sequences and genomic copy number of 16S genes along with phylogenetic placement and ancestral state estimation to estimate organismal abundances from environmental DNA sequence data. We use theory and simulations to demonstrate that 16S genomic copy number can be accurately estimated from the short reads typically obtained from high-throughput environmental sequencing of the 16S gene, and that organismal abundances in microbial communities are more strongly correlated with estimated abundances obtained from our method than with gene abundances. We re-analyze several published empirical data sets and demonstrate that the use of gene abundance versus estimated organismal abundance can lead to different inferences about community diversity and structure and the identity of the dominant taxa in microbial communities. Our approach will allow microbial ecologists to make more accurate inferences about microbial diversity and abundance based on 16S sequence data.</p>
<h1>DNA from soil mirrors plant taxonomic and growth form diversity.</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05545.X">doi:10.1111/J.1365-294X.2012.05545.X</a></p>
<p>No keywords</p><p>Ecosystems across the globe are threatened by climate change and human activities. New rapid survey approaches for monitoring biodiversity would greatly advance assessment and understanding of these threats. Taking advantage of next-generation DNA sequencing, we tested an approach we call metabarcoding: high-throughput and simultaneous taxa identification based on a very short (usually <100 base pairs) but informative DNA fragment. Short DNA fragments allow the use of degraded DNA from environmental samples. All analyses included amplification using plant-specific versatile primers, sequencing and estimation of taxonomic diversity. We tested in three steps whether degraded DNA from dead material in soil has the potential of efficiently assessing biodiversity in different biomes. First, soil DNA from eight boreal plant communities located in two different vegetation types (meadow and heath) was amplified. Plant diversity detected from boreal soil was highly consistent with plant taxonomic and growth form diversity estimated from conventional above-ground surveys. Second, we assessed DNA persistence using samples from formerly cultivated soils in temperate environments. We found that the number of crop DNA sequences retrieved strongly varied with years since last cultivation, and crop sequences were absent from nearby, uncultivated plots. Third, we assessed the universal applicability of DNA metabarcoding using soil samples from tropical environments: a large proportion of species and families from the study site were efficiently recovered. The results open unprecedented opportunities for large-scale DNA-based biodiversity studies across a range of taxonomic groups using standardized metabarcoding approaches.</p>
<h1>Towards next-generation biodiversity assessment using DNA metabarcoding.</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05470.X">doi:10.1111/J.1365-294X.2012.05470.X</a></p>
<p>No keywords</p><p>Virtually all empirical ecological studies require species identification during data collection. DNA metabarcoding refers to the automated identification of multiple species from a single bulk sample containing entire organisms or from a single environmental sample containing degraded DNA (soil, water, faeces, etc.). It can be implemented for both modern and ancient environmental samples. The availability of next-generation sequencing platforms and the ecologists' need for high-throughput taxon identification have facilitated the emergence of DNA metabarcoding. The potential power of DNA metabarcoding as it is implemented today is limited mainly by its dependency on PCR and by the considerable investment needed to build comprehensive taxonomic reference libraries. Further developments associated with the impressive progress in DNA sequencing will eliminate the currently required DNA amplification step, and comprehensive taxonomic reference libraries composed of whole organellar genomes and repetitive ribosomal nuclear DNA can be built based on the well-curated DNA extract collections maintained by standardized barcoding initiatives. The near-term future of DNA metabarcoding has an enormous potential to boost data acquisition in biodiversity research.</p>
<h1>Next-generation sequencing technologies for environmental DNA research.</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2012.05538.X">doi:10.1111/J.1365-294X.2012.05538.X</a></p>
<p>No keywords</p><p>Since 2005, advances in next-generation sequencing technologies have revolutionized biological science. The analysis of environmental DNA through the use of specific gene markers such as species-specific DNA barcodes has been a key application of next-generation sequencing technologies in ecological and environmental research. Access to parallel, massive amounts of sequencing data, as well as subsequent improvements in read length and throughput of different sequencing platforms, is leading to a better representation of sample diversity at a reasonable cost. New technologies are being developed rapidly and have the potential to dramatically accelerate ecological and environmental research. The fast pace of development and improvements in next-generation sequencing technologies can reflect on broader and more robust applications in environmental DNA research. Here, we review the advantages and limitations of current next-generation sequencing technologies in regard to their application for environmental DNA analysis.</p>
<h1>Monitoring endangered freshwater biodiversity using environmental DNA.</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2011.05418.X">doi:10.1111/J.1365-294X.2011.05418.X</a></p>
<p>No keywords</p><p>Freshwater ecosystems are among the most endangered habitats on Earth, with thousands of animal species known to be threatened or already extinct. Reliable monitoring of threatened organisms is crucial for data-driven conservation actions but remains a challenge owing to nonstandardized methods that depend on practical and taxonomic expertise, which is rapidly declining. Here, we show that a diversity of rare and threatened freshwater animals--representing amphibians, fish, mammals, insects and crustaceans--can be detected and quantified based on DNA obtained directly from small water samples of lakes, ponds and streams. We successfully validate our findings in a controlled mesocosm experiment and show that DNA becomes undetectable within 2 weeks after removal of animals, indicating that DNA traces are near contemporary with presence of the species. We further demonstrate that entire faunas of amphibians and fish can be detected by high-throughput sequencing of DNA extracted from pond water. Our findings underpin the ubiquitous nature of DNA traces in the environment and establish environmental DNA as a tool for monitoring rare and threatened species across a wide range of taxonomic groups.</p>
<h1>Eukaryotic richness in the abyss: insights from pyrotag sequencing.</h1>
<p><a href="https://doi.org/10.1371/Journal.Pone.0018169">doi:10.1371/Journal.Pone.0018169</a></p>
<p>No keywords</p><p>The deep sea floor is considered one of the most diverse ecosystems on Earth. Recent environmental DNA surveys based on clone libraries of rRNA genes confirm this observation and reveal a high diversity of eukaryotes present in deep-sea sediment samples. However, environmental clone-library surveys yield only a modest number of sequences with which to evaluate the diversity of abyssal eukaryotes.</p>
<h1>Multiple marker parallel tag environmental DNA sequencing reveals a highly complex eukaryotic community in marine anoxic water.</h1>
<p><a href="https://doi.org/10.1111/J.1365-294X.2009.04480.X">doi:10.1111/J.1365-294X.2009.04480.X</a></p>
<p>No keywords</p><p>Sequencing of ribosomal DNA clone libraries amplified from environmental DNA has revolutionized our understanding of microbial eukaryote diversity and ecology. The results of these analyses have shown that protist groups are far more genetically heterogeneous than their morphological diversity suggests. However, the clone library approach is labour-intensive, relatively expensive, and methodologically biased. Therefore, even the most intensive rDNA library analyses have recovered only small samples of much larger assemblages, indicating that global environments harbour a vast array of unexplored biodiversity. High-throughput parallel tag 454 sequencing offers an unprecedented scale of sampling for molecular detection of microbial diversity. Here, we report a 454 protocol for sampling and characterizing assemblages of eukaryote microbes. We use this approach to sequence two SSU rDNA diversity markers-the variable V4 and V9 regions-from 10 L of anoxic Norwegian fjord water. We identified 38 116 V4 and 15 156 V9 unique sequences. Both markers detect a wide range of taxonomic groups but in both cases the diversity detected was dominated by dinoflagellates and close relatives. Long-tailed rank abundance curves suggest that the 454 sequencing approach provides improved access to rare genotypes. Most tags detected represent genotypes not currently in GenBank, although many are similar to database sequences. We suggest that current understanding of the ecological complexity of protist communities, genetic diversity, and global species richness are severely limited by the sequence data hitherto available, and we discuss the biological significance of this high amplicon diversity.</p>
<h1>Accurate quantification of microorganisms in PCR-inhibiting environmental DNA extracts by a novel internal amplification control approach using Biotrove OpenArrays.</h1>
<p><a href="https://doi.org/10.1128/Aem.00796-09">doi:10.1128/Aem.00796-09</a></p>
<p>No keywords</p><p>PCR-based detection assays are prone to inhibition by substances present in environmental samples, thereby potentially leading to inaccurate target quantification or false-negative results. Internal amplification controls (IACs) have been developed to help alleviate this problem but are generally applied in a single concentration, thereby yielding less-than-optimal results across the wide range of microbial gene target concentrations possible in environmental samples (J. Hoorfar, B. Malorny, A. Abdulmawjood, N. Cook, M. Wagner, and P. Fach, J. Clin. Microbiol. 42:1863-1868, 2004). Increasing the number of IACs for each quantitative PCR (qPCR) sample individually, however, typically reduces sensitivity and, more importantly, the reliability of quantification. Fortunately, current advances in high-throughput qPCR platforms offer the possibility of multiple reactions for a single sample simultaneously, thereby allowing the implementation of more than one IAC concentration per sample. Here, we describe the development of a novel IAC approach that is specifically designed for the state-of-the-art Biotrove OpenArray platform. Different IAC targets were applied at a range of concentrations, yielding a calibration IAC curve for each individual DNA sample. The developed IACs were optimized, tested, and validated by using more than 5,000 unique qPCR amplifications, allowing accurate quantification of microorganisms when applied to soil DNA extracts containing various levels of PCR-inhibiting compounds. To our knowledge, this is the first study using a suite of IACs at different target concentrations to monitor PCR inhibition across a wide target range, thereby allowing reliable and accurate quantification of microorganisms in PCR-inhibiting DNA extracts. The developed IAC is ideally suited for high-throughput screenings of, for example, ecological and agricultural samples on next-generation qPCR platforms.</p>
<h1>The ITS region as a target for characterization of fungal communities using emerging sequencing technologies.</h1>
<p><a href="https://doi.org/10.1111/J.1574-6968.2009.01618.X">doi:10.1111/J.1574-6968.2009.01618.X</a></p>
<p>No keywords</p><p>The advent of new high-throughput DNA-sequencing technologies promises to redefine the way in which fungi and fungal communities--as well as other groups of organisms--are studied in their natural environment. With read lengths of some few hundred base pairs, massively parallel sequencing (pyrosequencing) stands out among the new technologies as the most apt for large-scale species identification in environmental samples. Although parallel pyrosequencing can generate hundreds of thousands of sequences at an exceptional speed, the limited length of the reads may pose a problem to the species identification process. This study explores whether the discrepancy in read length between parallel pyrosequencing and traditional (Sanger) sequencing will have an impact on the perceived taxonomic affiliation of the underlying species. Based on all 39,200 publicly available fungal environmental DNA sequences representing the nuclear ribosomal internal transcribed spacer (ITS) region, the results show that the two approaches give rise to quite different views of the diversity of the underlying samples. Standardization of which subregion from the ITS region should be sequenced, as well as a recognition that the composition of fungal communities as depicted through different sequencing methods need not be directly comparable, appear crucial to the integration of the new sequencing technologies with current mycological praxis.</p>
<h1>Biodiversity, chemical diversity and drug discovery.</h1>
<p>No DOI</p><p>No keywords</p><p>Drugs developed from microbial natural products are in the fundaments of modern pharmaceutical companies. Despite decades of research, all evidences suggest that there must remain many interesting natural molecules with potential therapeutic application yet to be discovered. Any efforts to successfully exploit the chemical diversity of microbial secondary metabolites need to rely heavily on a good understanding of microbial diversity, being the working hypothesis that maximizing biological diversity is the key strategy to maximizing chemical diversity. This chapter presents an overview of diverse topics related with this basic principle, always in relation with the discovery of novel secondary metabolites. The types of microorganisms more frequently used for natural products discovery are briefly reviewed, as well as the differences between terrestrial and marine habitats as sources of bioactive secondary metabolite producers. The concepts about microbial diversity as applied to prokaryotes have evolved in the last years, but recent data suggest the existence of true biogeographic patterns of bacterial diversity, which are also discussed. Special attention is dedicated to the existing strategies to exploit the microbial diversity that is not easy to tackle by conventional approaches. This refers explicitly to the current attempts to isolate and cultivate the previously uncultured bacteria, including the application of high throughput techniques. Likewise, the advances of microbial molecular biology has allowed the development of metagenomic approaches, i.e., the expression of biosynthetic pathways directly obtained from environmental DNA and cloned in a suitable host, as another way of accessing microbial genetic resources. Also, approaches relying on the genomics of metabolite producers are reviewed.</p>
